<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-17 09:27:41 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="While recent innovations in spatial biology have driven new insights into how tissue organization is altered in disease, interpreting these datasets in a generalized and scalable fashion remains a challenge. Computational workflows for discovering condition-specific differences in tissue organization typically rely on pairwise comparisons or unsupervised clustering. In many cases, these approaches are computationally expensive, lack statistical rigor, and are insensitive to low-prevalence cellular niches that are nevertheless highly discriminative and predictive of patient outcomes. Here, we present QUICHE – an automated, scalable, and statistically robust method that can be used to discover cellular niches differentially enriched in spatial regions, longitudinal samples, or clinical patient groups. In contrast to existing methods, QUICHE combines local niche detection with interpretable statistical modeling using graph neighborhoods to detect differentially enriched cellular niches, even at low prevalence. Using in silico models and spatial proteomic imaging of human tissues, we demonstrate that QUICHE can accurately detect condition-specific cellular niches occurring at a frequency of 0.5% in fewer than 20% of patient samples, outperforming the next best method which required a patient prevalence of 60% for detection. To validate our approach and understand how tumor structure influences recurrence risk in triple negative breast cancer (TNBC), we used QUICHE to comprehensively profile the tumor microenvironment in a multi-center, spatial proteomics cohort consisting of primary surgical resections, analyzing over 2 million cells from 314 patients across 5 medical centers. We discovered cellular niches that were consistently enriched in key regions of the tumor microenvironment, including the tumor-immune border and extracellular matrix remodeling regions, as well as niches statistically-associated with patient outcomes, including recurrence status and recurrence-free survival. The majority of differential niches (74.2%) were specific to patients that did not relapse and formed a robust interconnected network enriched in monocytes, macrophages, APCs, and CD8T cells with tumor and stroma cells. In contrast, the interaction network for patients that relapsed was notably sparse and enriched in B cells, CD68 macrophages and neutrophils. We validated these findings using two independent cohorts, observing similar cellular interactions and predictive power. Collectively, these results suggest that salient, generalized profiles of productive anti-tumor immune responses are defined by a network of structural engagement between innate and adaptive immunity with tumor and stromal cells, rather than by any single specific cell population. We have made QUICHE freely available as a user-friendly open-source Python package at https://github.com/jranek/quiche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda789c041e695c92206356373353eb906709b05" target='_blank'>
              QUICHE reveals structural definitions of anti-tumor responses in triple negative breast cancer
              </a>
            </td>
          <td>
            Jolene S Ranek, Noah F. Greenwald, Mako Goldston, C. Fullaway, Cameron Sowers, Alex Kong, S. Mourón, M. Quintela-Fandino, Robert West, Michael Angelo
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Small bowel adenocarcinoma (SBA) is a rare malignancy marked with a poor prognosis. The cellular and proteomic heterogeneity within the tumor immune microenvironment (TIME) of SBA is a likely driver of prognosis, disease progression and response to therapy. We have addressed a major gap in knowledge of the TIME in SBA using highly multiplexed, protein imaging of the SBA tumor-immune ecosystem generating a comprehensive, single-cell level, spatial, proteomic atlas of TIME in > 600,000 cells from 136 tumor and matched normal samples from clinically and genomically annotated SBA patients (N=37). The SBA TIME Atlas informs on spatial distribution and interactions of tumor-intrinsic processes, diverse immune cell types, immune checkpoints, and vascularization. Enrichment of proliferating epithelial tissue, stem cells, and likely pro-tumor immune signatures in the tumor niche is contrasted by the representation of naïve and early-effector T-cells in the epithelial compartments of adjacent normal tissues. Epithelial-stem-immune cell spatial architectures within tumor and matched-normal niches were strongly associated with patient survival, suggesting malignancy is driven by spatial architecture beyond the tumor microenvironment. The blueprints of therapeutically actionable immune checkpoints at the interface between epithelial and microenvironmental T-cells as well as macrophages have established a guideline for precision immunotherapies tailored to the TIME composition in SBA. We expect that this SBA atlas will contribute to a deeper understanding of the immune contexture in this rare disease as well as other gastrointestinal cancers and help guide future precision immune-oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, , Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) carries an extremely poor prognosis, in part resulting from cellular heterogeneity that supports overall tumorigenicity. Cancer associated fibroblasts (CAF) are key determinants of PDAC biology and response to systemic therapy. While CAF subtypes have been defined, the effects of patient-specific CAF heterogeneity and plasticity on tumor cell behavior remain unclear. Here, multi-omics was used to characterize the tumor microenvironment (TME) in tumors from patients undergoing curative-intent surgery for PDAC. In these same patients, matched tumor organoid and CAF lines were established to functionally validate the impact of CAFs on the tumor cells. CAFs were found to drive epithelial-mesenchymal transition (EMT) and a switch in tumor cell classificiaton from classical to basal subtype. Furthermore, we identified CAF-specific interleukin 8 (IL-8) as an important modulator of tumor cell subtype. Finally, we defined neighborhood relationships between tumor cell and T cell subsets. Statement of Significance This multidimensional analysis highlights the diverse role CAFs have in influencing other cell types in the TME, including epithelial-derived tumor and infiltrating immune cells. Our methods provide a platform for evaluating emerging therapeutic approaches and for studying mechanisms that dictate tumor behavior in a manner that reflects patient-specific heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d173d5090e9fb969bbd5c7fddf327d5b9643a1d" target='_blank'>
              Cancer associated fibroblasts drive epithelial to mesenchymal transition and classical to basal change in pancreatic ductal adenocarcinoma cells with loss of IL-8 expression
              </a>
            </td>
          <td>
            Samantha Guinn, Brayan Perez, Joseph A Tandurella, Mili Ramani, Jae W. Lee, Daniel J. Zabransky, Emma Kartalia, Jignasha Patel, H. Zlomke, Norman G Nicolson, Sarah M. Shin, Benjamin Barrett, Nicholas Sun, Alexei Hernandez, Erin M Coyne, Courtney D Cannon, Nicole E. Gross, S. Charmsaz, Yeonju Cho, James Leatherman, Melissa R Lyman, Jacob T. Mitchell, Luciane T. Kagohara, Michael G. Goggins, Kelly J. Lafaro, Jin He, Christopher R Shubert, William R Burns, Lei Zheng, E. Fertig, Elizabeth M. Jaffee, Richard A Burkhart, W. Ho, Jacquelyn W. Zimmerman
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Spatial transcriptomics aims to elucidate cell coordination within biological tissues by linking the state of the cell with its local tissue microenvironment. Imaging-based assays are particularly promising for exploring such interdependencies, as they can resolve molecular and cellular features with subcellular resolution in three dimensions. Quantification and analysis of cellular state in such data, however, ultimately depends on the ability to recognize which molecules belong to each cell. Despite computational and experimental progress, this cell segmentation task remains challenging. Here we re-analyze data from multiple tissues and platforms and find that segmentation errors currently confound most downstream analysis of cellular state, including analysis of differential expression, inference of neighboring cell influence, and ligand-receptor interactions. The extent to which mis-segmented molecules impact the results can be striking, often dominating the set of top hits. We show that factorization of molecular neighborhoods can be effective at isolating such molecular admixtures and minimizing their impact on downstream analysis, analogous to doublet filtering of scRNA-seq data. As applications of spatial transcriptomics assays become more widespread, we expect corrections for the confounding effect of segmentation errors to become increasingly important for being able to resolve molecular mechanisms of tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca4acc410c4c078dfd09e8229953bf61e6a7e0c5" target='_blank'>
              Impact of Segmentation Errors in Analysis of Spatial Transcriptomics Data
              </a>
            </td>
          <td>
            Jonathan Mitchel, Teng Gao, Eli Cole, Viktor Petukhov, P. V. Kharchenko
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) provide therapeutic benefit in EGFR-mutated non-small cell lung cancer (NSCLC); however, early resistance develops in some patients. To explore the underlying mechanisms, we performed spatial transcriptomics using Visium HD and StarDist processing on paired pre- and post-treatment samples from the same patients. In early resistance samples, fibroblasts increased in proportion and exhibited enhanced activation following EGFR- TKI treatment, while a distinct tumor cell cluster showed TNF-α/NF-kB pathway activation even prior to treatment. Furthermore, in the pre-treatment sample with early resistance, specific tumor clusters interacted with immune and stromal cells through inflammatory pathways, including MIF/CD74 and PDGF signaling. Immature tertiary lymphoid structures (TLSs) were enriched in early resistance, whereas mature TLSs were observed in long-term responders. These findings highlight the role of tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insights into potential therapeutic strategies to improve treatment outcomes. Disclosure Daisuke Shibahara has received research funding from Eli Lilly Japan K.K. Koichi Azuma has received honoraria from AstraZeneca, MSD, Chugai Pharmaceutical, Ono Pharmaceutical, Bristol-Myers Squibb, Takeda Pharmaceutical, Taiho Pharmaceutical, and Amgen. Yasuto Yoneshima has received honoraria from AstraZeneca. Kentaro Tanaka has received honoraria from Chugai Pharmaceutical, AstraZeneca, Ono Pharmaceutical, Bristol-Myers Squibb, Eli Lilly, Takeda Pharmaceutical, Novartis Pharma K.K., Merck Biopharma, Kyowa Kirin, Daiichi-Sankyo, Pfizer, Amgen, Janssen Pharmaceutical K.K., and MSD, and is an advisory board member for Pfizer and Novartis Pharma K.K. Eiji Iwama has received honoraria from AstraZeneca and Chugai Pharmaceutical. Isamu Okamoto has received honoraria and research funding from Daiichi Sankyo, Chugai Pharmaceutical, Eli Lilly Japan K.K., AstraZeneca, Taiho Pharmaceutical, Boehringer Ingelheim, and Ono Pharmaceutical; honoraria from Takeda Pharmaceutical and Novartis Pharma K.K.; and research funding from Bristol-Meyers Squibb and MSD Oncology. Other authors declare no potential conflicts of interest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc050a906a767a3e545375057173c4f1889efdeb" target='_blank'>
              Spatial dynamics of the tumor microenvironment in emerging resistance to targeted therapy in EGFR-mutated NSCLC
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, Mikihiro Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Spatially non-resolved transcriptomic data identified functionally distinct populations of fibroblasts in health and disease. However, in-depth transcriptional profiling in situ at single-cell resolution has not been possible so far. Here, we studied fibroblast populations in the skin of SSc patients and healthy individuals using cyclic in situ hybridization (cISH) as a novel approach for spatially-resolved transcriptional phenotyping with subcellular resolution. cISH deconvoluted the heterogeneity of 20,979 cells including 3,764 fibroblasts (FB). BANKSY-based spatially-informed clustering identified nine FB subpopulations, with SFRP2+ RetD FB and CCL19+ nonPV FB as novel subpopulations that reside in specific cellular niches and display unique gene expression profiles. SFRP2+ RetD FB and CCL19+ nonPV FB as well as COL8A1+ FB, display altered frequencies in SSc skin and play specific, disease-promoting roles for extracellular matrix release and leukocyte recruitment as revealed by their transcriptional profile, their cellular interactions and ligand-receptor analyses. The frequencies of COL8A1+ FB and their interactions with monocytic cells and B cells are associated with progression of skin fibrosis in SSc. In summary, our spatially-resolved transcriptomic approach identified novel fibroblast subpopulations deregulated in SSc skin with specific pathogenic roles, some of which may potentially serve as biomarkers for progression of skin fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c095f7397487cd95a22547f9e4b6ee14b870509e" target='_blank'>
              Spatially informed phenotyping by cyclic-in-situ-hybridization identifies novel fibroblast populations and their pathogenic niches in systemic sclerosis
              </a>
            </td>
          <td>
            Yi-Nan Li, Tim Filla, A. Györfi, Minrui Liang, Veda Devakumar, Alexandru Micu, Hongtao Chai, C. Bergmann, A. Pecher, Jörg Henes, P. Moinzadeh, Thomas Krieg, Alexander Kreuter, G. Schett, Bernhard Homey, Sascha Dietrich, J. Distler, A. Matei
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Adaptive immunity, mediated by T and B cell responses, is essential for defending against infections and cancers while also being implicated in autoimmune diseases. Tracking T and B cell repertoires in situ at single-cell resolution is essential for understanding adaptive immune responses. To address the lack of tools for in situ single-cell T/BCR (XCR) sequencing, we developed Stereo-XCR-seq, an efficient strategy for retrieving and sequencing TCR and BCR from Stereo-seq cDNA libraries at subcellular resolution. Stereo-XCR-seq provides unbiased full-length XCR reads alongside spatial transcriptomics, enabling the identification of heterogeneous lymphoid aggregates with distinct clonal activities in cancers and inflammatory bowel disease (IBD). We identified plasma cell aggregates that differ from tertiary lymphoid structures (TLSs) in both transcriptomic profiles and clonal activities, with spatial positioning potentially mediating unique immune responses. Collectively, Stereo-XCR-seq enables in situ single-cell profiling of T and B cell clonal activities within tissue microenvironments, providing insights into lymphocyte adaption to environmental stimuli. This technology provides potential for advancing our understanding of tissue immunity and the development of therapeutic strategies for immune disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a087fd6bab927232a0eec4056a80591d4dc5a4" target='_blank'>
              Single cell resolved spatial immune repertoire unveils spatial heterogeneity of lymphoid aggregates in human immune disorders
              </a>
            </td>
          <td>
            Xiaojuan Zhan, Yi Liu, Yanying Guo, Wenwen Zhou, Yixin Yan, Hui Zeng, Xuan Dong, Xiaoyu Chen, Rong Ma, Zhong Liu, Fan Zhu, Xubin Zheng, Xinxing Li, Jinwen Yin, Francis Ka-ming Chan, Chuanyu Liu, Longqi Liu, Xun Xu, Yong Hou, Haoran Tao, Yuliang Dong, Tao Zeng, Young Li, Jingying Zhou, Zexian Zeng, Yu Feng
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics technologies have significantly enhanced resolution and throughput, underscoring an urgent need for systematic benchmarking. To address this, we collected clinical samples from three cancer types – colon adenocarcinoma, hepatocellular carcinoma, and ovarian cancer – and generated serial tissue sections for systematic evaluation. Using these uniformly processed samples, we generated spatial transcriptomics data across five high-throughput platforms with subcellular resolution: Stereo-seq v1.3, Visium HD FFPE, Visium HD FF, CosMx 6K, and Xenium 5K. To establish ground truth datasets, we profiled proteins from adjacent tissue sections corresponding to all five platforms using CODEX and performed single-cell RNA sequencing on the same samples. Leveraging manual cell segmentation and detailed annotations, we systematically assessed each platform’s performance across key metrics, including capture sensitivity, specificity, diffusion control, cell segmentation, cell annotation, spatial clustering, and transcript-protein alignment with adjacent CODEX. The uniformly generated, processed, and annotated multi-omics dataset is valuable for advancing computational method development and biological discoveries. The dataset is accessible via SPATCH, a user-friendly web server for visualization and download (http://spatch.pku-genomics.org/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a092f418d2d0937b601a2e81131e13ca731653a" target='_blank'>
              Systematic Benchmarking of High-Throughput Subcellular Spatial Transcriptomics Platforms
              </a>
            </td>
          <td>
            Pengfei Ren, Rui Zhang, Yunfeng Wang, Peng Zhang, Ce Luo, Suyan Wang, Xiaohong Li, Zongxu Zhang, Yanping Zhao, Yufeng He, Haorui Zhang, Yufeng Li, Zhidong Gao, Xiuping Zhang, Yahui Zhao, Zhihua Liu, Yuanguang Meng, Zhe Zhang, Zexian Zeng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colonic stem cells have a key role in the continuous regeneration of a healthy intestinal epithelium. Despite considerable advances in single-cell omics technologies, the transcriptional heterogeneity of rare cell types such as colonic stem cells, as well as their functional states and niche-specific behaviors, remain poorly characterised. In this study, we leverage a meta-analysis of scRNA-seq and spatial transcriptomic datasets to comprehensively map the heterogeneity of colonic stem cells. We identify multiple, previously underappreciated stem cell states, including distinct quiescent subtypes marked by CDKN1A (P21), CDKN1B (P27), and CDKN1C (P57), proliferative populations defined by MKI67 (Ki67) and LRIG1, and a lineage-committed intermediate subpopulation expressing MUC2. Strikingly, we find that these states can be robustly identified solely by their extracellular matrix (ECM) gene expression signatures, revealing ECM composition as a critical determinant of stem cell identity. Notably, LAMA1 expression is highly specific to the P57+ quiescent population, linking laminin-mediated microenvironments to the active maintenance of deep quiescence, consistent with our recent findings associating LAMA1 with quiescent cell survival. By applying these ECM gene signatures, we delineate discrete “micro-niches” of quiescent stem cells in healthy tissue and provide evidence that analogous states persist in the colorectal cancer (CRC) environment. Extending our approach to an unrelated tissue, the pancreas, we detect parallel quiescent cell subtypes, illustrating the broader applicability of ECM-based signatures. Taken together, our findings redefine the concept of stem cell heterogeneity in the colon, establish ECM-driven gene signatures as a powerful tool for characterizing stem cell states, and offer new perspectives on the niche-dependent regulation of both healthy and cancerous stem cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b79223cf63c16f2be2c9879ace337f25901bae" target='_blank'>
              ECM Signatures Reveal Quiescent Stem Cell Diversity in the Colonic Niche
              </a>
            </td>
          <td>
            Séamus E. Hickey, M. Andreatta, Christina Enright, Emmanuel Boucrot, Patrick Kiely, Siobhán B. Cashman, Saintiago J. Carmona, Kieran McGourty
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In the field of digestive system tumor research, spatial transcriptomics technologies are used to delve into the spatial structure and the spatial heterogeneity of tumors and to analyze the tumor microenvironment (TME) and the inter-cellular interactions within it by revealing gene expression in tumors. These technologies are also instrumental in the diagnosis, prognosis, and treatment of digestive system tumors. This review provides a concise introduction to spatial transcriptomics and summarizes recent advances, application prospects, and technical challenges of these technologies in digestive system tumor research. This review also discusses the importance of combining spatial transcriptomics with single-cell RNA sequencing (scRNA-seq), artificial intelligence, and machine learning in digestive system cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdf975c83e9f742bc7e19f83183283693f91eae" target='_blank'>
              Application of Spatial Transcriptomics in Digestive System Tumors
              </a>
            </td>
          <td>
            Bowen Huang, Yingjia Chen, S. Yuan
          </td>
          <td>2024-12-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) exhibits significant intra-tumoral heterogeneity (ITH) at both morphological and genetic levels, complicating treatment and contributing to disease progression1,2. Among these, rhabdoid ccRCCs stand out as highly aggressive tumors distinguished by cells with unique morphological features and poor clinical outcomes3. However, the correlation between distinct morphological phenotypes, specific molecular alterations, and their influence on tumor behavior remains poorly understood. Understanding the link between clinically relevant morphological features and their molecular underpinnings will be critical for developing more effective treatments targeting key cellular subsets responsible for tumor progression. In this study, we integrated advanced AI-based image analysis with single-cell isolation and multi-omics profiling to dissect the link between clinically relevant morphological and molecular features of ccRCC cells. Using a novel digital pathology workflow, we quantified low-grade, high-grade, and rhabdoid morphologies in ccRCC diagnostic images with unprecedented precision. Subsequently, isolation of two sets of 1,000 morphologically distinct cells for detailed mRNA and protein expression analyses, revealed significant increasing dysregulation associating with higher histopathological grades. Rhabdoid ccRCC cells (grade 4) demonstrated distinct molecular profiles, including upregulated FOXM1-driven proliferation, disrupted cell-matrix interactions, and enhanced immune evasion pathways. Despite high T-cell infiltration in rhabdoid areas, we identified a rhabdoid-specific immunosuppressive network driven by cytokines, IFN-beta, and integrin signaling, likely contributing to T-cell exhaustion. These findings provide a basis for novel therapeutic strategies targeting these pathways in combination with immunotherapy to improve outcomes for patients with aggressive rhabdoid ccRCC. Key Points ccRCC is characterized by well-established morphological heterogeneity but the correlation with the underlying molecular aberrations remained elusive. By integrating AI-based image analysis with single cell isolation and deep multi-omics profiling, we dissect the molecular intricacies of ccRCC, from targeted collection of 1,000 morphologically distinct cells. Our results demonstrate significant dysregulation of gene and protein expression correlating with higher histopathological grades in ccRCC. Aggressive ccRCC cells with rhabdoid differentiation (grade 4) display distinct molecular profiles, as they upregulate FOXM1-mediated proliferation, ECM remodeling and the immune evasion responses, suggesting new therapeutic avenues enhancing ICI efficacy in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7461a1f413e55baf162ee7fd7309ee3cb97ce7a" target='_blank'>
              Deep visual multi-omics profiling reveals mechanisms that underly cancer cell differentiation and aggressiveness in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            H. A. Bolck, Ede Migh, A. Kriston, Natalia Zajac, Susanne Kreutzer, Tiberiu Totu, Peter Leary, Ferenc Kovács, D. Rutishauser, Sybille Pfammatter, Jonas Grossmann, Cassandra Litchfield, Marija Buljan, Niels J Rupp, Péter Horváth, Holger Moch
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is the deadliest cancer worldwide. Therapeutic progress stagnate, highlighting the complexity to replicate NSCLC in preclinical models. Drug discovery studies rely mostly on cancer cells in two-dimension (2D), which poorly predict drug efficacy in patients. There is a growing interest in three-dimensional (3D) preclinical models, such as 3D spheroids, to better model tumor phenotype and improve therapeutic prediction. However, a comprehensive view of 3D culture methods impact on transcriptomes, epigenomes and pharmacological responses and their correlations to NSCLC tumors is still missing. Here, we demonstrate that NSCLC spheroids undergo time-dependent transcriptomic and epigenomic changes, which peak after 3 weeks of culture. While DNA methylome remained stable, chromatin methylation and acetylation marks gained features of advanced NSCLC in a time-dependent manner. Single-cell transcriptomic profiling of spheroids demonstrated that time of 3D culture improved the correlation to NSCLC tumors. Moreover, long-term culture of 3D spheroids increased drug screening predictability, by showing resistance to drugs that failed in NSCLC patients (such as HDAC inhibitors) while demonstrating novel pharmacological vulnerabilities and synergistic interactions (such as combination of PRMT and HDAC inhibitors). Strikingly, reverting 3D spheroids back to 2D culture rapidly reversed transcriptomic, epigenetic and pharmacological signatures acquired after 3 weeks of 3D culture, highlighting the critical impact of cell culture conditions on NSCLC phenotype. Collectively, our findings demonstrate that implementing a time-dependent maturation process into 3D spheroid culture induces chromatin and transcriptomic changes that enhance NSCLC preclinical modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Single-cell RNA sequencing studies have revealed the heterogeneity of cell states present in the rheumatoid arthritis (RA) synovium. However, it remains unclear how these cell types interact with one another in situ and how synovial microenvironments shape observed cell states. Here, we use spatial transcriptomics (ST) to define stable microenvironments across eight synovial tissue samples from six RA patients and characterize the cellular composition of ectopic lymphoid structures (ELS). To identify disease-relevant cellular communities, we developed DeepTopics, a scalable reference-free deconvolution method based on a Dirichlet variational autoencoder architecture. DeepTopics identified 22 topics across tissue samples that were defined by specific cell types, activation states, and/or biological processes. Some topics were defined by multiple colocalizing cell types, such as CD34+ fibroblasts and LYVE1+ macrophages, suggesting functional interactions. Within ELS, we discovered two divergent cellular patterns that were stable across ELS in each patient and typified by the presence or absence of a “germinal-center-like” topic. DeepTopics is a versatile and computationally efficient method for identifying disease-relevant microenvironments from ST data, and our results highlight divergent cellular architectures in histologically similar RA synovial samples that have implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5649472b0dc4cb2ee433d6b1131064e755b734d" target='_blank'>
              Deep topic modeling of spatial transcriptomics in the rheumatoid arthritis synovium identifies distinct classes of ectopic lymphoid structures
              </a>
            </td>
          <td>
            Preethi K. Periyakoil, Melanie H. Smith, Meghana Kshirsagar, Daniel Ramirez, Edward DiCarlo, Susan M. Goodman, Alexander Y. Rudensky, L. Donlin, Christina S. Leslie
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma (ccRCC) is the most common type of kidney cancer, but predicting its behavior remains challenging using standard histopathologic examination. This study introduces a novel approach to predict ccRCC aggressiveness by analyzing the spatial distribution of tumor cells in H&E-stained images. The researchers found that spatial analysis outperformed traditional tumor grading in predicting metastasis, particularly for intermediate-grade tumors. They identified two distinct patient groups based on spatial characteristics, with one group showing greater spatial randomness and a higher association with metastasis. Furthermore, the study revealed a gene expression signature related to the extracellular matrix (matrisome) that correlated with the spatial patterns and aggressive tumor behavior. These findings suggest that analyzing the spatial distribution of ccRCC tumor cells could provide valuable insights into tumor behavior and metastatic potential, potentially improving prognostication and personalized treatment strategies for patients with ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1af154f3c69daea89b74218f36407c18e3de7d" target='_blank'>
              Spatial Distribution of Tumor Cells in Clear Cell Renal Cell Carcinoma Is Associated with Metastasis and a Matrisome Gene Expression Signature
              </a>
            </td>
          <td>
            Prahlad Bhat, P. Tamboli, K. Sircar, Kasthuri Kannan
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Fate-restricted cells can acquire stem cell-like properties through dedifferentiation, enabling them to gain the plasticity required for differentiation into multiple lineages. Tumour plasticity is prominently observed in brain cancers, where transient cell state changes are linked to resistance to conventional therapies. In this study, we demonstrate that a sub-population of dedifferentiated tumour neural stem cells (NSCs) in Drosophila, induced by the knockdown of prospero (pros), can generate its own glial niche. Temporal patterning, known to influence oncogenic competence and tumour malignancy, plays a key role in this process. Specifically, we show that de novo gliogenesis occurs in the more differentiated Syncrip+ (Syp+) NSC population. Modulating Syp levels alters the size of the glial niche, subsequently affecting tumour size. Furthermore, the tumour-associated glial niche expands through cell division and fails to cease proliferation on time due to dysregulated ecdysone signalling, contributing to niche expansion. Our findings reveal that tumours arising via dedifferentiation establish their own supportive glial microenvironment, which sustains tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d30097dbfea218ae747551091d256416f71d4db" target='_blank'>
              Tumour-derived gliogenesis sustains dedifferentiation-dependent tumour growth in the Drosophila CNS
              </a>
            </td>
          <td>
            Edel Alvarez-Ochoa, Qian Dong, Hannah Truong, Louise Y Cheng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Brain metastasis occurs in approximately 16% of metastatic breast cancer patients, and incidence is increasing. Patients with breast cancer brain metastases frequently develop neurological problems including cognitive impairments and seizures. We previously demonstrated accumulation of activated microglia around the tumour site in a breast cancer brain metastasis model, which co-localised with spontaneously occurring local field potential events that resembled interictal epileptic discharges. However, the mechanisms underlying these spatially restricted effects on neuronal excitability are poorly understood. Methods To better understand the underlying cellular changes and potential mechanisms, we used digital spatial profiling of brain cell subtype-specific markers to chart the spatial organisation of neurons, oligodendrocytes, astrocytes, and microglia in relation to metastatic lesion site. We developed a novel chordline-based approach to analyse these spatial changes in biomarker distribution. Results Several protein markers associated with proliferation (Ki67), astrocytes (GFAP) and microglia (CD11b, IBA1, CD45, MSR1) were upregulated in the tumour compared to healthy contralateral brain parenchyma. In contrast, some neuroglial markers (MAP2, Neurofilament light, NeuN, S100B and TMEM119) were lower in the lesion compared to normal tissue. Conclusions Overall, the protein marker changes in the lesion are indicative of neuronal cell loss and changes in the immune microenvironment of the metastatic lesion involving activated microglia, astrocytes and recruitment of peripheral immune cells. Such cellular changes may contribute to the changes in electrical activity resulting from brain metastases observed in mice and patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dbeafc4e0030d90e60f5d024e3f1200854ddc2" target='_blank'>
              Spatial profiling of central nervous system cellular markers in murine breast cancer to brain metastases
              </a>
            </td>
          <td>
            Karen Hogg, Grant Calder, Alastair P. Droop, Ming Yang, Anna Simon, Mark J Hunt, Paul M Kaye, Miles A. Whittington, Sangeeta Chawla, W. Brackenbury
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The kidney is a complex organ requiring tightly coordinated interactions between epithelial, endothelial, and mesenchymal cells during development. Congenital kidney defects can result in kidney disease and renal failure, highlighting the importance of understanding kidney formation mechanisms. Advances in RNA sequencing have revealed remarkable cellular heterogeneity, especially in the kidney stroma, though relationships between stromal, epithelial, and endothelial cells remain unclear. This study presents a comprehensive gene expression atlas of embryonic and postnatal kidneys, integrating single-nucleus and in situ RNA sequencing data. We developed the Kidney Spatial Transcriptome Analysis Tool (KSTAT), enabling researchers to identify cell locations, predict cell-cell communication, and map gene pathway activity. KSTAT revealed significant heterogeneity among embryonic kidney pericytes, providing a critical resource for hypothesis generation and advancing knowledge of kidney development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df4312b6dec5fab934d14ad6ad0cc39d8a07d71" target='_blank'>
              Integration of spatial and single nucleus transcriptomics to map gene expression in the developing mouse kidney
              </a>
            </td>
          <td>
            Christopher P. Chaney, Alexandria N. Fusco, Elyse D. Grilli, Jane N. Warshaw, Peter M. Luo, Ondine Cleaver, Denise K. Marciano, Thomas J. Carroll
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Central Nervous System (CNS) tumors have sophisticated tumor microenvironment (TME) with different cell types such as astrocytes, microglia, neurons, vascular endothelial cells and immune cells. These non-cancerous cells orchestrate the brain TME to regulate cancer progression and therapeutic response. This study aimed to develop a cell composition deconvolution method for CNS tumor and to determine the impact of these cell compositions on patients’ outcomes. Methods We identified the cell type-specific CpG loci using the pairwise differential methylation analysis for 13 major cell types in CNS. Using non-negative least squares (NNLS) methods, we established this cell-type deconvolution approach, MDBrainT, for brain tumors. The predictive accuracy of our MDBrainT model was tested in the DNA methylation profiling of the purified cell samples and compared against another algorithm. Cell composition was predicted by MDBrainT in several brain tumor (glioma, ependymoma, medulloblastoma and ATRT) cohorts and the correlation between cell composition and tumor molecular subtypes and patient outcomes was also assessed. Results Cell type-specific CpG loci for CNS TME was used to build MDBrainT model. Based on these DNA methylation markers, MDBrainT predicted the cell composition in the TME including tumor cells with high accuracy. Endothelial cells were predominately presented in glioblastomas while the percentage of CD8 T cells wassignificantly higher in ATRT. A substantial difference of cell composition was two molecular groups of posterior fossa ependymoma (PFA vs PFB). A higher percentage of cells in TME was usually associated with worse outcomes. Conclusions MDBrainT is a robust algorithm for cell composition prediction for brain TME. Cell composition in brain TME is distinct across different pathological types and molecular subtypes. Key Points MDBrainT is a robust DNA methylation-based deconvolution approach for brain tumor microenvironment (TME). Different molecular subtypes of brain tumors have distinct cell composition patterns. Cell composition in brain TME informs patients of outcomes. Importance of this study DNA methylation is a cell type specific marker that has been utilized for tumor molecular diagnosis, disease progression and therapeutic monitoring. A DNA methylation-based classifier for brain tumors precisely predicts the molecular subtypes but not the cell composition of tumor microenvironment. Brain tumors are a complex cell mixture where tumor microenvironment is critical for tumorigenesis and therapeutic resistance. Here, we developed a novel deconvolution approach (MDBrainT) to predict cell composition for brain tumors. Our model has revealed the heterogeneity of cell composition between tumor types. In addition, tumors with different molecular subtypes have distinct cell composition. Cell percentage in TME also informs patients of outcomes. The tumor microenvironment including cell composition of each patient may direct the different regimen of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1823256c94dd6a377f1b3165f4ae951356ef07" target='_blank'>
              DNA Methylation-Based Cell Type Deconvolution Reveals the Distinct Cell Composition in Brain Tumor Microenvironment
              </a>
            </td>
          <td>
            Feili Liu, Jin Qian, Chenkai Ma
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76feb62f24d1b55375efafabff78309d558b9c49" target='_blank'>
              Spatial interaction mapping of PD-1/PD-L1 in head and neck cancer reveals the role of macrophage-tumour barriers associated with immunotherapy response
              </a>
            </td>
          <td>
            V. Y. Naei, Rafael Tubelleza, J. Monkman, Habib Sadeghirad, Meg L. Donovan, Tony Blick, A. Wicher, Sara Bodbin, Amelie Viratham, Robert Stad, Subham Basu, C. Cooper, Catherine Barnett, K. O’Byrne, Rahul Ladwa, M. Warkiani, Brett G M Hughes, A. Kulasinghe
          </td>
          <td>2025-02-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Purpose Eyelid infiltrative basal cell carcinoma (iBCC) is the most common malignant tumor affecting the ocular adnexa, but studies on metabolic changes within its microenvironment and heterogeneity at the tumor invasive area are limited. This study aims to analyze metabolic differences among iBCC cell types using single-cell and spatial metabolomics analysis and to examine metabolic environment at the tumor invasive area. Methods Single-cell transcriptomic data of human basal cell carcinoma (BCC) were clustered and visualized using Uniform Manifold Approximation and Projection. Metabolic reprogramming was analyzed with single-cell flux estimation analysis. Spatial metabolomics data were obtained with the Timstof Flex MALDI 2 system, and Bruker software was used for region selection. Results Eight cell types were identified within the iBCC microenvironment. Differences between inflammatory cancer-associated fibroblasts and myofibroblastic cancer-associated fibroblasts were analyzed. Metabolic flux analysis showed increased glycolysis, glutamine, heme, and glutathione fluxes in the iBCC microenvironment. Spatial metabolomics revealed high levels of taurine, deoxy-GMP, O-phosphoethanolamine, and pyrithione. Both tumor and invasive regions had significant upregulation of fatty acid pathways, with marked increases in oleic and arachidonic acids at the invasive area. Specific upregulation of UDP-glucuronic acid and high UDP-glucose 6-dehydrogenase (UGDH) expression in the tumor region suggest UXS1 as a potential therapeutic target for iBCC. Conclusions This study establishes a metabolic microenvironment atlas of iBCC, revealing significant metabolic differences and the dominance of lipid and lysosome metabolism. Potential metabolic markers and characteristic substances in the invasive area offer new insights for immunotherapy and the exploration of BCC’s metabolic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322ca4366508114a20cf4530c76f41a9f0ca02cf" target='_blank'>
              Metabolic Atlas of Human Eyelid Infiltrative Basal Cell Carcinoma
              </a>
            </td>
          <td>
            Yanjing Huang, Chengjie He, Qiuling Hu, Zhong Liu, Xingyi Li, Wuyou Gao, Xuanwei Liang, Rongxin Chen, Zhen Mao, Xianchai Lin
          </td>
          <td>2025-01-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Efforts to achieve precise and efficient tumor targeting of highly malignant brain tumors are constrained by the dearth of appropriate models to study the effects and potential side effects of radiation, chemotherapy, and immunotherapy on the most complex human organ, the brain. We established a cerebral organoid model of brain tumorigenesis in an autologous setting by overexpressing c-MYC as one of the most common oncogenes in brain tumors. GFP+/c-MYChigh cells were isolated from tumor organoids and used in two different culture approaches: assembloids comprising of a normal cerebral organoid with a GFP+/c-MYChigh tumor sphere and co-culture of cerebral organoid slices at air-liquid interface with GFP+/c-MYChigh cells. GFP+/c-MYChigh cells used in both approaches exhibited tumor-like properties, including overexpression of the c-MYC oncogene, high proliferative and invasive potential, and an immature phenotype as evidenced by increased expression of Ki-67, VIM, and CD133. Organoids and organoid slices served as suitable scaffolds for infiltrating tumor-like cells. Using our highly reproducible and powerful model system that allows long-term culture, we demonstrated that the migratory and infiltrative potential of tumor-like cells is shaped by the environment in which glia cells provide support to tumor-like cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e83d9fc6e9d50fb29a6be3bfad1aea77a00f111" target='_blank'>
              Human cerebral organoids model tumor infiltration and migration supported by astrocytes in an autologous setting
              </a>
            </td>
          <td>
            Esther Schickel, Tamara Bender, Leon Kaysan, Simone Hufgard, Margot Mayer, David Grosshans, Christiane Thielemann, Insa Schroeder
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Metabolic processes are crucial in immune regulation, yet the impact of metabolic heterogeneity on immunological functions remains unclear. Integrating metabolomics into immunology allows the exploration of the interactions of multilayered features in the biological system and the molecular regulatory mechanism of these features. To elucidate such insight in lung squamous cell carcinoma (LUSC), we analyzed 106 LUSC tumor tissues. We performed high-resolution matrix-assisted laser desorption/ionization (MALDI) mass spectrometry imaging (MSI) to obtain spatial metabolic profiles, and immunohistochemistry to detect tumor-infiltrating T lymphocytes (TILs). Unsupervised k-means clustering and Simpson’s diversity index were employed to assess metabolic heterogeneity, identifying five distinct metabolic tumor subpopulations. Our findings revealed that TILs are specifically associated with metabolite distributions, not randomly distributed. Integrating a validation cohort, we found that heterogeneity-correlated metabolites interact with CD8+ TIL-associated genes, affecting survival. High metabolic heterogeneity was linked to worse survival and lower TIL levels. Pathway enrichment analyses highlighted distinct metabolic pathways in each subpopulation and their potential responses to chemotherapy. This study uncovers the significant impact of metabolic heterogeneity on immune functions in LUSC, providing a foundation for tailoring therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4c98251a8ac1956e7ccdfa63c51218dbe1751b" target='_blank'>
              Metabolic heterogeneity in tumor cells impacts immunology in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Qian Wang, Na Sun, Chaoyang Zhang, Thomas Kunzke, P. Zens, A. Feuchtinger, Sabina Berezowska, Axel Walch
          </td>
          <td>2025-02-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Liposarcoma (LPS) is the most prevalent soft tissue sarcoma. The most common biological subtypes are well-differentiated (WDLPS), a low-grade disease that can evolve to high-grade dedifferentiated LPS (DDLPS), with increased rates of recurrence and metastasis and low response rates to chemotherapy and targeted therapies. Preclinical testing of immunotherapeutics for LPS has been held back by the lack of an immunocompetent mouse model. Here, we present a spontaneous immunocompetent LPS mouse model, ACPP, with targeted deletion of Trp53 and Pten in adipocytes to mimic signaling alterations observed in human LPS. Similar to human LPS, tumors arising in ACPP mice produce WDLPS and DDLPS, along with tumors that exhibit both WD and DD components. Murine and human DDLPS tumors possess transcriptional similarities, including increased expression of oncogenes Cdk4 and Hmga2 and reduced expression of the tumor suppressor Cebpa; further, both mouse and human DDLPS exhibit either high or low T cell infiltration. Syngeneic cell lines derived from spontaneous ACPP DDLPS reliably produce tumors following orthotopic injection, each with distinct growth patterns, aggressiveness and tumor infiltrating lymphocyte profiles. These models provide much needed tools to understand the complex immunobiology of LPS and greatly accelerate the pace of preclinical studies to uncover new therapies for patients with this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/534b0303f4645ca052b41ef2e2a1a4b7d31ee268" target='_blank'>
              An immunocompetent mouse model of liposarcoma
              </a>
            </td>
          <td>
            Amanda Shafer, Emma Kenna, Lexi F Golden, Ahmed M. Elhossiny, Kyle Perry, Jodi Wilkowski, Wei Yan, Brynn Kaczkofsky, J. McGue, Scott C. Bresler, Adam H. Courtney, Jessie M. Dalman, C. Galbán, Wei Jiang, Carlos E. Espinoza, Rashmi Chugh, Matthew K. Iyer, Timothy L. Frankel, M. Pasca di Magliano, Andrzej A. Dlugosz, Christina V. Angeles
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies have enabled new explorations into the spatial organization of tissues and their functional implications. However, one of the most fundamental analyses – comparative analysis of spatial gene expression across phenotypes – remains a formidable challenge. We introduce SpaceExpress, a novel statistical tool for detecting phenotype-associated changes in spatial expression patterns. SpaceExpress employs a neural network to embed multiple ST samples in a common latent space, enabling robust cross-sample comparisons despite structural and technical variations. It then uses spline regression to test differential spatial expression of genes between conditions, with rigorous false discovery control and handling of multiple replicates per condition. It includes visualization tools to help interpret spatial pattern differences. We demonstrate the tool’s effectiveness on synthetic and real ST datasets, revealing mechanistic insights into behavior-related neurogenomic changes in honey bees and mice. Our work extends the highly influential paradigm of differential gene expression analysis to spatial omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17cf03e0e2cb603b45180e44926848307ff9915" target='_blank'>
              SpaceExpress: a method for comparative spatial transcriptomics based on intrinsic coordinate systems of tissues
              </a>
            </td>
          <td>
            Yeojin Kim, Abhishek Ojha, Alex W Schrader, Juyeon Lee, Zijun Wu, Ian M. Traniello, Gene E Robinson, Hee-Sun Han, S. Zhao, Saurabh Sinha
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression.
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, , Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) occurs heterogeneously among malignant carcinoma cells to promote chemoresistance. Identifying the signaling pathways involved will nominate drug combinations to promote chemoresponse, but cell population-level studies are inherently fraught, and single-cell transcriptomics are limited to indirect ontology-based inferences. To understand EMT heterogeneity at a signaling protein level, we combined iterative indirect immunofluorescence imaging of pancreas cancer cells and tumors and mutual information (MI) modeling. Focusing first on MAP kinase pathways, MI predicted that cell-to-cell variation in ERK activity surprisingly dominated control of EMT heterogeneity in response to diverse growth factors and chemotherapeutics, but that JNK compensated when MEK was inhibited. Population-level models could not capture these experimentally validated MI predictions. The dominant role of ERK was predicted by MI even when analyzing seven potential EMT-regulating signaling nodes. More generally, this work provides an approach for studying highly multivariate signaling/phenotype relationships based on protein measurements in any setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a6514737cd599ba0d5ecbb0df4e4d91b74a1b1" target='_blank'>
              ERK plays a conserved dominant role in pancreas cancer cell EMT heterogeneity driven by diverse growth factors and chemotherapies
              </a>
            </td>
          <td>
            Michelle C. Barbeau, Brooke A. Brown, S. Adair, Todd W. Bauer, Matthew J Lazzara
          </td>
          <td>2025-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The advent of spatial transcriptomics and spatial proteomics have enabled profound insights into tissue organization to provide systems-level understanding of diseases. Both technologies currently remain largely independent, and emerging same slide spatial multi-omics approaches are generally limited in plex, spatial resolution, and analytical approaches. We introduce IN-situ DEtailed Phenotyping To High-resolution transcriptomics (IN-DEPTH), a streamlined and resource-effective approach compatible with various spatial platforms. This iterative approach first entails single-cell spatial proteomics and rapid analysis to guide subsequent spatial transcriptomics capture on the same slide without loss in RNA signal. To enable multi-modal insights not possible with current approaches, we introduce k-bandlimited Spectral Graph Cross-Correlation (SGCC) for integrative spatial multi-omics analysis. Application of IN-DEPTH and SGCC on lymphoid tissues demonstrated precise single-cell phenotyping and cell-type specific transcriptome capture, and accurately resolved the local and global transcriptome changes associated with the cellular organization of germinal centers. We then implemented IN-DEPTH and SGCC to dissect the tumor microenvironment (TME) of Epstein-Barr Virus (EBV)-positive and EBV-negative diffuse large B-cell lymphoma (DLBCL). Our results identified a key tumor-macrophage-CD4 T-cell immunomodulatory axis differently regulated between EBV-positive and EBV-negative DLBCL, and its central role in coordinating immune dysfunction and suppression. IN-DEPTH enables scalable, resource-efficient, and comprehensive spatial multi-omics dissection of tissues to advance clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2a15d874ba82ad119e2c09f851c2a8a6c7876e" target='_blank'>
              Same-Slide Spatial Multi-Omics Integration Reveals Tumor Virus-Linked Spatial Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yao Yu Yeo, Yuzhou Chang, Huaying Qiu, Stephanie Pei, Tung Yiu, Hendrik A. Michel, Wenrui Wu, Xiaojie Jin, Shoko Kure, Lindsay Parmelee, Shuli Luo, Precious Cramer, Jia Le Lee, Yang Wang, Jason Yeung, N. Ahmar, Berkay Simsek, R. Mohanna, McKayla Van Orden, Wesley Lu, K. Livak, Shuqiang Li, J. Shahryari, Leandra Kingsley, Reem N Al-Humadi, Sahar Nasr, Dingani Nkosi, Sam Sadigh, Philip Rock, Leonie Frauenfeld, Louisa Kaufmann, Bokai Zhu, Ankit Basak, Nagendra Dhanikonda, Chi Ngai Chan, Jordan E Krull, Ye Won Cho, Chia-Yu Chen, Jia Ying, Joey Lee, Hongbo Wang, Bo Zhao, L. Loo, David M Kim, V. Boussiotis, Baochun Zhang, Alex K. Shalek, Brooke Howitt, S. Signoretti, Christian M. Schürch, Stephan Hodi, R. Burack, Scott J Rodig, Qin Ma, Sizun Jiang
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Peripheral nerve sheath tumors (PNSTs) encompass entities with different cellular differentiation and degrees of malignancy. Spatial heterogeneity complicates diagnosis and grading of PNSTs in some cases. In malignant PNST (MPNST) for example, single cell sequencing data has shown dissimilar differentiation states of tumor cells. Here, we aimed at determining the spatial and biological heterogeneity of PNSTs.


METHODS
We performed spatial transcriptomics on formalin-fixed paraffin-embedded diseased peripheral nerve tissue. We used spatial clustering and weighted correlation network analysis to construct niche-similarity networks and gene expression modules. We determined differential expression in primary pathologies, analysed pathways to investigate the biological significance of identified meta-signatures, integrated the transcriptional data with histological features and existing single cell data, and validated expression data by immunohistochemistry.


RESULTS
We identified distinct transcriptional signatures differentiating PNSTs. Immune cell infiltration, APOD and perineurial fibroblast marker expression highlighted the neurofibroma component of hybrid PNSTs (HPNSTs). While APOD was evenly expressed in neurofibromatous tumor tissue in both, HPNST and pure neurofibromas, perineurial fibroblast markers were evenly expressed in HPNST, but restricted to the periphery in plexiform neurofibromas. Furthermore, we provide a spatial cellular differentiation map for MPNST, locating Schwann cell precursors, neural crest-like cells and those with mesenchymal transition.


CONCLUSIONS
This pilot study shows that applying spatial transcriptomics to PNSTs provides important insight into their biology. It helps establishing new markers and provides spatial information about cellular composition and distribution of cellular differentiation states. By integrating morphological and high-dimensional molecular data it can improve PNSTs classification in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2233f7b03c50daacd9edb143f0325a91952a1c6" target='_blank'>
              Spatially resolved transcriptomics of benign and malignant peripheral nerve sheath tumors.
              </a>
            </td>
          <td>
            J. Bremer, Pamela Franco, J. A. Menstell, S. Tey, K. Zajt, Klimentina Popzhelyazkova, Kay Nolte, Jürgen Schlegel, Maria Teresa Pedro, Anja Osterloh, D. Delev, M. Hohenhaus, Christoph Scholz, O. Schnell, J. Beck, Joachim Weis, D. Heiland
          </td>
          <td>2025-01-23</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Although the epiblast in the embryo has the capacity to generate all tissues of the body, its in vitro counterparts often exhibit differentiation biases, posing significant challenges for both basic research and translational applications involving pluripotent stem cells (PSCs). The origins of these biases remain incompletely understood. In this study, we identify PSC differentiation biases as arising from fluctuations in repressive and activating histone posttranslational modifications, leading to the acquisition of a caudal epiblast-like phenotype. We present a novel approach to overcome this bias using a chemical chromatin restoration (CHR) treatment. This method restores transcriptional programs, chromatin accessibility, histone modification profiles, and differentiation potential, effectively recapitulating the competent anterior epiblast-like state. Furthermore, we propose that a high bivalency state is a defining feature of the anterior human epiblast. We suggest that fluctuations in histone modification marks drive epiblast regionalization, ultimately shaping cellular responses to differentiation cues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fe129f2b61f7c39ce00d29ccb4e09bc4f83868c" target='_blank'>
              Epigenetic restoration of differentiation competency via reversal of epiblast regionalisation
              </a>
            </td>
          <td>
            Magdalena A. Sutcliffe, S. Wingett, Charles A.J. Morris, Eugenia Wong, S. Schoenfelder, Madeline A. Lancaster
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Breast cancer is a significant health challenge worldwide, and disproportionately affects women of African ancestry (AA) who experience higher mortality rates relative to other racial/ethnic groups. Several studies have pointed to biological factors that affect breast cancer outcomes. A recently discovered stromal cell population that expresses PROCR, ZEB1 and PDGFRα (PZP cells) was found to be enriched in normal healthy breast tissue from AA donors, and only in tumor adjacent tissues from donors of European ancestry (EA). Here, we investigated the effect of PZP cells on the acquisition of tumorigenic phenotypes in AA and EA human breast epithelial cells to determine its contribution to the biological basis of outcome disparities. Using 3D cell models of tumor invasion, we find that PZP cells confer invasive capacity to epithelial cells independent of genetic ancestry, wherein cells exhibit leader-follower behaviors during extracellular matrix invasion. Enhanced epithelial invasion stems from a combination of AKT activation and fibronectin deposition by the PZP cells. Although activation of AKT in epithelial cells alone is insufficient to induce invasive behaviors, blocking AKT activation markedly reduces invasive capacity. These findings point to the germaneness of differences in breast biology and the multi-faceted roles and enrichment in PZP cells in AA breast tissue, in furthering current understanding of the molecular basis for worse prognosis for patients of African descent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60dd766ac1428c496c998ec8bd07c1a84a6947" target='_blank'>
              Ancestry-linked stromal variations impact breast epithelial cell invasion
              </a>
            </td>
          <td>
            M. Schmidtmann, Victoria Elliott, James W. Clancy, H. Nakshatri, Crislyn D'Souza-Schorey
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Background Tumour immune macroenvironment is comprised of tumour and surrounding organs responding to tumourigenesis and immunotherapy. The lack of comprehensive analytical methods hinders its application for prediction of survival and treatment response in colorectal cancer (CRC) patients. Methods Cytometry by time‐of‐flight (CyTOF) and RNA‐seq was applied to characterise immune cell heterogeneity in a discovery cohort including tumour, blood and intestinal architecture comprising epithelium, lamina propria, submucosa, muscularis propria of normal bowel and tumour–adjacent bowel tissues. Immunoprofiling was also validated by a validation cohort using single‐cell RNA sequencing, spatial transcription, CyTOF and multiplex immunofluorescent staining. Results Based on cell phenotype and transcription, we identify distinct immunotypes in the CRC macroenvironment including blood, tumour and different intestinal architecture, showing disturbed immune cell compositions, increasing expression of immunosuppressive markers and cell–cell interactions contributing to immunosuppressive regulation. Furthermore, we evaluate immune macroenvironment influencing factors including tertiary lymphoid structures (TLSs), consensus molecular subtypes (CMSs) and immune checkpoint inhibitors (ICIs). TLS presence fuels anti‐tumour immunity by promoting CD8+ T cell infiltration and altering activation or suppression of T cell systematically. TLS presence correlates with patient survival, intrinsic CMS and therapeutic efficacy of ICI. PD‐1 and CD69 expressed in effector memory CD8+ T cells from blood can predict TLS presence in the CRC macroenvironment, serving as potential biomarkers for stratifying CRC patients into immunotherapy. Conclusions Our findings provide insights into the CRC immune macroenvironment, highlighting immune cell suppression and activation in tumourigenesis. Our study illustrates the potential utility of blood for predicting immunotherapy response. Key points Distinct immunotypes are identified in the CRC macroenvironment. TLS and immunotherapy exert influence on the immune macroenvironment. TLS presence correlates with patient survival, CMS and therapeutic efficacy of ICI. PD‐1 and CD69 expressed in CD8+ Tem from blood can predict TLS presence in the CRC macroenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85985e342fbc41d36820a7abb928738245e5ca0b" target='_blank'>
              Immune profiling of the macroenvironment in colorectal cancer unveils systemic dysfunction and plasticity of immune cells
              </a>
            </td>
          <td>
            Haoxian Ke, Peisi Li, Zhihao Li, Xian Zeng, Chi Zhang, Shuzhen Luo, Xiaofang Chen, Xinlan Zhou, Shichen Dong, Shaopeng Chen, Junfeng Huang, Ming Yuan, Runfeng Yu, Shubiao Ye, Tuo Hu, Zhonghui Tang, Dongbin Liu, Kui Wu, Xianrui Wu, Ping Lan
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Novel multiplexed spatial proteomics imaging platforms expose the spatial architecture of cells in the tumor microenvironment (TME). The diverse cell population in the TME, including its spatial context, has been shown to have important clinical implications, correlating with disease prognosis and treatment response. The accelerating implementation of spatial proteomic technologies motivates new statistical models to test if cell-level images associate with patient-level endpoints. Few existing methods can robustly characterize the geometry of the spatial arrangement of cells and also yield both a valid and powerful test for association with patient-level outcomes. We propose a topology-based approach that combines persistent homology with kernel testing to determine if topological structures created by cells predict continuous, binary, or survival clinical endpoints. We term our method TopKAT (Topological Kernel Association Test) and show that it can be more powerful than statistical tests grounded in the spatial point process model, particularly when cells arise along the boundary of a ring. We demonstrate the properties of TopKAT through simulation studies and apply it to two studies of triple negative breast cancer where we show that TopKAT recovers clinically relevant topological structures in the spatial distribution of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f79d159b2d5895452d1adff9339095baad447" target='_blank'>
              Detecting Clinically Relevant Topological Structures in Multiplexed Spatial Proteomics Imaging Using TopKAT
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Amarise Little, Wodan Ling, Ni Zhao, Yen-Chi Chen, Michael C. Wu
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The stimulator of interferon genes (STING) pathway 4 is key to tumour immunity and a studied target for immunotherapy. 5 This study explores the immune landscape of GBM, focusing on spatial relationships between tumour-associated immune cells (TAICs) 6 and STING-expressing cells, uncovering patterns linked to prognosis. We studied 14 recurrent GBM patients using protein composition and Gene Ontology (GO) analysis and analyzed immune pathways in 69 newly diagnosed GBM patients undergoing standard therapy. Spatial analysis of cells was performed using QuPath, CytoMAP, and R, as illustrated in Figure 1A. From our proteomic analysis, we identified protein groups with marked upregulation in patients with favourable responses. A total of 99 proteins were highly upregulated in the favourable group, while 170 were more pronounced in the unfavourable group (Figure 1B). Clusters of downregulated and upregulated proteins were identified using the Benjamini-Hochberg False Discovery Rate. The expression of TMEM173 , the gene encoding STING, was significantly elevated in the favourable group, as measured by protein expression using a mass spectrometer. The results were represented as an abundance ratio, and statistical significance was confirmed using the adjusted p-value. (Figure 1C). GO analysis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c328a57a4b1f1c5d0b9b9ccc1305e50beee1fb97" target='_blank'>
              Spatial distribution of immune cells and their proximity to STING+ cells are associated with survival in glioblastoma
              </a>
            </td>
          <td>
            Hanwool Jeon, Hayeong Kang, Jihyun Im, Suin Jo, Hyunchul Jung, Moinay Kim, Jae Hyun Kim, Eun-Jae Lee, Soyoung Kim, Jeong Hoon Kim, C. Hong, Young-Hoon Kim, Sang Woo Song, Jinha Park, Sang-Yeob Kim, Seungjoo Lee
          </td>
          <td>2025-01-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fef18e249838665646bfdb8d1805895bb4b7093" target='_blank'>
              Characterization of canine tumor-infiltrating leukocyte transcriptomic signatures reveals conserved expression patterns with human osteosarcoma
              </a>
            </td>
          <td>
            Dylan T. Ammons, R. A. Harris, L. Chow, Steven Dow
          </td>
          <td>2025-02-11</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite surgical resection, rigorous endoscopic surveillance, and immunotherapy with the Bacillus Calmette–Guérin (BCG) vaccine, 30% of high-risk bladder cancers recur, and 10% result in fatal outcomes within two years of diagnosis. The global shortage of BCG underscores the urgent need for alternative or complementary therapeutic strategies. To address this, we integrated transcriptomic profiling and targeted genomic sequencing to identify four consensus intrinsic subtypes of bladder cancer. Initially derived from bulk RNA profiling, these subtypes were further validated at the cellular and tissue-compartment levels using single-cell RNA sequencing and spatial transcriptomics. Notably, we identified a subtype of inflamed tumors with enhanced endogenous retroelement expression and increased commensal bacterial presence, which showed the highest responsiveness to BCG therapy. Additionally, we developed a machine learning-based model incorporating composite molecular features to predict recurrence risk, achieving a high accuracy (AUC = 0.90). Our findings establish a molecular precision framework for bladder cancer and nominate novel therapeutic targets to reduce reliance on BCG immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f0d9394677cb2bb295a5f2fe96a5d0c0fc0b84" target='_blank'>
              Genomic and transcriptomic profiling of high-risk bladder cancer reveals diverse molecular and microenvironment ecosystems
              </a>
            </td>
          <td>
            K. Meghani, Yanni Yu, N. Frydenlund, Erik Z. Li, B. Choy, S. Abdulkadir, J. Meeks
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic drug treatments of solid tumors are often undermined by various resistance mechanisms. Identification of drug-resistance phenotypes at the single cell level is challenging because conventional molecular methods are cell-destructive, labor-intensive, and cost-prohibitive. To overcome these challenges, we developed an orthogonal approach to drug-resistance phenotyping, through the use of deep-learning-driven morphology analysis of single, high resolution cell images. Specifically, we trained deep learning-based drug resistance classifiers using cell images from 5 different cell lines that were rendered resistant to 5 different therapeutic agents, using a foundation model framework. With high accuracy, the classifier correctly predicted naive or resistance phenotypes across all cancer types and across all the therapeutic agent types (chemotherapeutic, targeted) tested. These results showed that morphology can capture complex phenotype information in the context of drug treatment. To demonstrate the potential clinical utility of the drug resistance classifier, it was applied to a dissociated tumor biopsy and the resulting phenotype predictions were in close concordance with scRNASeq analysis of the biopsy. Our study highlights the potential of deep-learning-driven morphology analysis to provide complex phenotype information, and ultimately shape oncology drug treatment strategies at the patient-level in a clinical context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e32bdcccdadfa02831a9867b05cc627a1045f4" target='_blank'>
              Deep learning-driven morphology analysis enables label-free classification of therapeutic agentnaive versus resistant cancer cells
              </a>
            </td>
          <td>
            Cyril Y Ramathal, K. Saini, Zhouyang Lian, Christian Corona, T. Pham, Ryan Carelli, Stephane C. Boutet, Manisha Ray, Sunantha Sethuraman, Vivian Prindle, Evelyn Lattmann, Maria-Grazia Molvetti, A. Dzung, Mitchell P. Levesque, Matt Barnes, Andreja Jovic
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8afd5989cd7e1d2eeffdf0537754b1eeb0da52a8" target='_blank'>
              SMORE: spatial motifs reveal patterns in cellular architecture of complex tissues
              </a>
            </td>
          <td>
            Zainalabedin Samadi, Kai Hao, Amjad Askary
          </td>
          <td>2025-01-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Resistance to tumor necrosis factor inhibitor (TNFi) therapy remains a major obstacle in managing ulcerative colitis (UC), affecting up to 50% of patients.1 The complex interplay of immune cells and their spatial organization in the gut mucosa is poorly understood but likely contributes to therapeutic outcomes.1,2 This study utilized advanced multiplexed imaging to construct a spatial atlas of UC, providing insights into cellular interactions driving TNFi resistance. Novel spatial omic technologies show to be able to add additional thus far unknown insights to this question.



 A total of 42 tissue regions from 29 UC patients and 5 healthy controls were analyzed using CODEX multiplexed imaging. Spatially resolved single-cell data, encompassing over 1.7 million cells, were used to identify distinct cell types and map cellular neighborhoods (CNs). Pairwise cell-cell contacts and CN-level functional states were analyzed in TNFi-treated and untreated patients, correlating findings with clinical outcomes.



 TNFi therapy led to normalization of adaptive immune CNs, such as lymphoid aggregates, with reductions in T-cell frequencies and associated contacts. However, innate immune CNs, including neutrophil-enriched granulocyte niches and inflamed stroma, remained unchanged despite treatment. Notably, neutrophil-dominated CNs exhibited persistent expression of pro-inflammatory markers, such as TNFR2 and CD137, implicating these niches in sustaining mucosal inflammation and therapy resistance. Additionally, sex-dependent differences were observed, with higher TNFi responsiveness in females potentially linked to adaptive immune modulation.3



 Neutrophil-dominated inflammatory niches represent a potential mechanism of TNFi resistance in UC.3,4 These findings highlight the importance of targeting innate immune pathways alongside TNFi therapy to improve outcomes. By integrating spatial insights with functional data, this study paves the way for precision medicine approaches in UC management.



 1.R. Ungaro, S. Mehandru, P. B. Allen, L. Peyrin-Biroulet, J.-F. Colombel, Ulcerative colitis.Lancet 389, 1756–1770 (2017).
 2.R. Gaujoux, E. Starosvetsky, N. Maimon, F. Vallania, H. Bar-Yoseph, S. Pressman, R. Weisshof,I. Goren, K. Rabinowitz, M. Waterman, H. Yanai, I. Dotan, E. Sabo, Y. Chowers, P. Khatri,S. S. Shen-Orr; Israeli IBD research Network (IIRN), Cell-centred meta-analysis revealsbaseline predictors of anti-TNFα non-response in biopsy and blood of patients with IBD.Gut 68, 604–614 (2019).
 3.A.T. Mayer, D.R. Holman, A. Sood, U. Tandon, S. S. Bhate, S. Bodapati, G. L. Barlow, J. Chang, S. Black, E. C. Crenshaw, A. N. Koron, S. E. Streett, S. S. Gambhir SS, W. J. Sandborn, B. S. Boland, T. Hastie, R. Tibshirani, J. T. Chang, G. P. Nolan, C. M. Schürch, S. Rogalla “A tissue atlas of ulcerative colitis revealing evidence of sex-dependent differences in disease-driving inflammatory cell types and resistance to TNF inhibitor therapy” Sci Adv. 2023 Jan 20;9(3):eadd1166. doi: 10.1126/sciadv.add1166. Epub 2023 Jan 20.
 4.S. T. Eshghi, J. M. Gubatan, P. Mazrooei, L. Quintanilla, A. Nguyen, A. Au-Yeung, D. R. Holman, C. Takashi, C. Schiffman, B. O’Gorman, M. Keir, S. Ramanujan, S. Rogalla, J. A. Hackney, J. M McBride “Molecular characterization of response to Etrolizumab and anti-TNF reveals treatment resistance in ulcerative colitis is associated with abundance of residual neutrophil subsets and inflammatory fibroblast populations” https://doi.org/10.1101/2024.07.02.601267

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391837f71bfd5dab1136ff60f107c81956417fbb" target='_blank'>
              OP23 Neutrophil Niches and Therapy Resistance in Ulcerative Colitis: Spatial Atlas Insights
              </a>
            </td>
          <td>
            D. Holman, S. Rogalla
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed580d09e5d8a50632bdd53ac296d7f45feff0d1" target='_blank'>
              The microenvironment cell index is a novel indicator for the prognosis and therapeutic regimen selection of cancers
              </a>
            </td>
          <td>
            Xian-Yan Yang, Nian Chen, Qian Wen, Yu Zhou, Tao Zhang, Ji Zhou, Cheng-Hui Liang, Li-Ping Han, Xiao-Ya Wang, Qingmei Kang, Xiaomei Zheng, Xue-Jia Zhai, Hong-Ying Jiang, Tian-Hua Shen, Jin-Wei Xiao, Yu-Xin Zou, Yun Deng, Shuang Lin, Jiang-jie Duan, Jun Wang, Shicang Yu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are lymphoid formations that develop in non-lymphoid tissues during chronic inflammation, autoimmune diseases, and cancer. Accurate identification and quantification of TLS in tissue can provide crucial insights into the immune response of several disease processes including antitumor immune response. TLS are defined as aggregates of T cells, B cells and dendritic cells. In histological tissue sections stained with Hematoxylin and Eosin they are identified as aggregates of 50 or more lymphoid cells, however immunohistochemical analysis are required to confirm presence of distinct immune cell patterns. Assessment of lymphoid aggregates can be done in H&E slides or in slides stained with single or multiplex immunohistochemistry or other tissue based high-plex approaches with key biomarkers such as CD3 (T cells) and CD20 (B cells), however manual assessment of them is time consuming thus limiting its full evaluation. To our knowledge, published algorithms that identify TLS within a tissue are based on histological assessment of H&E slides or through use of ML algorithms trained on images that show the TLS presence in tissues; and quantification and spatial analysis of TLS still remains a challenge. This study aims to develop a robust algorithm to recognize TLS using the spatial coordinates of immune cells in any given tissue. The algorithm uses X and Y coordinates of T and B cells in tissues identified by a pathologist-supervised digital image analysis of multiplex chromogenic immunohistochemistry of tissues stained with CD3 and CD20. The algorithm is flexible to be used for detailed analysis of TLS stages; including other cell types within the definition of TLS; such as dendritic cells (DC) and high endothelial venules (HEV); to assess different stages of TLS formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5237511ad45e3ecdaa3252f9aa2a014ac80b4418" target='_blank'>
              TLS_Finder: An algorithm for Identifying Tertiary Lymphoid Structures Using Immune Cell Spatial Coordinates
              </a>
            </td>
          <td>
            Ayse A Koksoy, Maria E Salvatierra, L. S. Solis Soto, C. Haymaker
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The prevalent intra- and intertumoral heterogeneity results in undesirable prognosis and therapy failure of pancreatic cancer, potentially resulting from cellular senescence. Herein, integrated analysis of bulk and single-cell RNA-seq profiling was conducted to characterize senescence-based heterogeneity in pancreatic cancer. Publicly available bulk and single-cell RNA sequencing from pancreatic cancer patients were gathered from TCGA-PAAD, PACA-AU, PACA-CA, and GSE154778 datasets. The activity of three senescence-related pathways (cell cycle, DNA repair, and inflammation) was scored utilizing ssGSEA algorithm. A series of functional verifications of crucial genes were accomplished in patient tissue and pancreatic cancer cells. Based upon them, unsupervised clustering analysis was executed to classify pancreatic cancer samples into distinct senescence-based clusters at the bulk and single-cell levels. For single-cell transcriptome profiling, cell clustering and annotation were implemented, and malignant cells were recognized utilizing infercnv algorithm. Two senescence-based clusters were established and highly reproducible at the bulk level, with the heterogeneity in prognosis, clinicopathological features, genomic CNVs, oncogenic pathway activity, immune microenvironment and immune checkpoints. Senescence-relevant gene CHGA, UBE2C and MCM10 were proved to correlate with the migration and prognosis of pancreatic cancer. At the single-cell level, seven cell types were annotated, comprising ductal cells 1, ductal cells 2, fibroblasts, macrophages, T cells, stellate cells, and endothelial cells. The senescence-based classification was also proven at the single-cell level. Ductal cells were classified as malignant cells and non-malignant cells. In the tumor microenvironment of malignant cells, hypoxia and angiogenesis affected senescent phenotype. The heterogeneity in senescence was also observed between and within cell types. Altogether, our findings unveil that cellular senescence contributes to intra- and intertumoral heterogeneity in pancreatic cancer, which might facilitate the development of therapeutics and precision therapy in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2742fe352ef25d18eeda7d59fe2c6a8314ffc13" target='_blank'>
              Deciphering the senescence-based tumoral heterogeneity and characteristics in pancreatic cancer: Results from parallel bulk and single-cell transcriptome data.
              </a>
            </td>
          <td>
            Yeting Lu, Shuo Han, Jing Hu, Kaiji Lv, Y. Ruan, Gong Cheng, Jing Zhang, Xiang Wu, Zeming Weng, Xinhua Zhou
          </td>
          <td>2025-01-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Disruptions in uterine tissue function contribute to disorders such as endometriosis, adenomyosis, endometrial cancer, and fibroids, which all significantly impact health and fertility. Advances in transcriptomics, particularly single-cell RNA sequencing, have revolutionized uterine biological research by revealing the cellular heterogeneity and molecular mechanisms underlying disease states. Single-cell RNA sequencing and spatial transcriptomics have mapped endometrial and myometrial cellular landscapes, which helped to identify critical cell types, signaling pathways, and phase-specific dynamics. Said transcriptomic technologies also identified stromal and immune cell dysfunctions, such as fibroblast-to-myofibroblast transitions and impaired macrophage activity, which drive fibrosis, chronic inflammation, and lesion persistence in endometriosis. For endometrial cancer, scRNA-seq uncovered tumor microenvironmental complexities, identifying cancer-associated fibroblast subtypes and immune cell profiles contributing to progression and therapeutic resistance. Similarly, studies on adenomyosis highlighted disrupted signaling pathways, including Wnt and VEGF, and novel progenitor cell populations linked to tissue invasion and neuroinflammation, while single-cell approaches characterized smooth muscle and fibroblast subpopulations in uterine fibroids, elucidating their roles in extracellular matrix remodeling and signaling pathways like ERK and mTOR. Despite challenges such as scalability and reproducibility, single-cell transcriptomic approaches may have potential applications in biomarker discovery, therapeutic target identification, and personalized medicine in gynecological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d46c8d721f3ec856f042d24cd830de014f3393b" target='_blank'>
              Mapping Human Uterine Disorders Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Sandra Boldu-Fernández, Carolina Lliberos, Carlos Simón, A. Mas
          </td>
          <td>2025-01-21</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Attempts to activate an anti-tumor immune response in glioblastoma (GBM) have been met with many challenges due to its inherently immunosuppressive tumor microenvironment. The degree and mechanisms by which molecularly and phenotypically diverse tumor-propagating glioma stem cells (GSCs) contribute to this state are poorly defined. In this study, our multifaceted approach combining bioinformatics analyses of clinical and experimental datasets, single-cell sequencing, and molecular and pharmacologic manipulation of patient-derived cells identified GSCs expressing immunosuppressive effectors mimicking regulatory T cells (Tregs). We show that this Immunosuppressive Treg-Like (ITL) GSC state is specific to the mesenchymal GSC subset and is associated with and driven specifically by TGF-β type II receptor (TGFBR2) in contrast to TGFBR1. Transgenic TGFBR2 expression in patient-derived GBM neurospheres promoted a mesenchymal transition and induced a 6-gene ITL signature consisting of CD274 (PD-L1), NT5E (CD73), ENTPD1 (CD39), LGALS1 (galectin-1), PDCD1LG2 (PD-L2), and TGFB1. This TGFBR2-driven ITL signature was identified in clinical GBM specimens, patient-derived GSCs and systemic mesenchymal malignancies. TGFBR2High GSCs inhibited CD4+ and CD8+ T cell viability and their capacity to kill GBM cells, effects reversed by pharmacologic and shRNA-based TGFBR2 inhibition. Collectively, our data identify an immunosuppressive GSC state that is TGFBR2-dependent and susceptible to TGFBR2-targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7f721f28d28359511d7b452c6e33026ca71e4b" target='_blank'>
              TGFBR2High mesenchymal glioma stem cells phenocopy regulatory T cells to suppress CD4+ and CD8+ T cell function
              </a>
            </td>
          <td>
            Amanda L Johnson, Harmon Khela, Jack Korleski, Sophie Sall, Yunqing Li, Weiqiang Zhou, Karen Smith-Connor, H. Lopez-Bertoni, J. Laterra
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0090834d905d07acf5293829d6a0fb1c2a65490b" target='_blank'>
              Immune infiltrate populations within distinct tumor immune microenvironments predictive of immune checkpoint treatment outcome
              </a>
            </td>
          <td>
            B. Ring, C. Cronister, Huijun Z Ring, Douglas T Ross, Robert S. Seitz
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Therapeutic strategy for efficiently targeting cancer cells needs an in-depth understanding of the cellular and molecular interplay in the tumor microenvironment (TME). TME comprises heterogeneous cells clustered together to translate tumor initiation, migration, and proliferation. The TME mainly comprises proliferating tumor cells, stromal cells, blood vessels, lymphatic vessels, cancer-associated fibroblasts (CAFs), extracellular matrix (ECM), and cancer stem cells (CSC). The heterogeneity and genetic evolution of metastatic tumors can substantially impact the clinical effectiveness of therapeutic agents. Therefore, the therapeutic strategy shall target TME of all metastatic stages. Since the advent of nanotechnology, smart drug delivery strategies are employed to deliver effective drug formulations directly into tumors, ensuring controlled and sustained therapeutic efficacy. The state-of-the-art nano-drug delivery systems are shown to have innocuous modes of action in targeting the metastatic players of TME. Therefore, this review provides insight into the mechanism of cancer metastasis involving invasion, intravasation, systemic transport of circulating tumor cells (CTCs), extravasation, metastatic colonization, and angiogenesis. Further, the novel perspectives associated with current nanotherapeutic strategies are highlighted on different stages of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62c6b188dc4bb363851ee009a8519399e3f5251c" target='_blank'>
              Navigating Tumor Microenvironment Barriers with Nanotherapeutic Strategies for Targeting Metastasis.
              </a>
            </td>
          <td>
            Mahima Rachel Thomas, A. K. Badekila, Vishruta Pai, N. S, Y. Bhandary, Ankit Rai, S. Kini
          </td>
          <td>2025-01-22</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Hypoxia is a common feature in solid tumors that arises when there is insufficient oxygen available. This lack of oxygen causes molecular adaptations required for the tumor cells to survive. Additionally, oxygen‐deprived cancer cells tend to become less responsive to conventional cancer therapies. Hence, hypoxia plays an important role in contributing to tumor aggressiveness and therapy resistance. Hypoxia‐related markers are gaining interest as prognostic and predictive markers for tumor response and treatment strategies. However, the detection of hypoxia in the tumor microenvironment without employing any labeling strategies poses significant challenges. Here, we present a classification model based on lipidomic single‐cell mass spectrometry imaging data to classify hypoxia in breast cancer xenografts. Our approach is based on a classification model built from the lipid profiles of single breast cancer cells cultured under various oxygen conditions. Lipidomic alterations caused by differences in available oxygen concentrations were subsequently used to classify and spatially determine hypoxic regions in breast cancer xenografts without the need for any labeling. This approach, using cells as hypoxia markers, contributes to a better understanding of tumor biology and provides a foundation for improving diagnostic and therapeutic strategies for cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7ec1e82c3348ba2f51d436e568e85c4f135880" target='_blank'>
              Classifying Hypoxia in Breast Cancer Xenografts Using a Single‐Cell Mass Spectrometry Imaging Model
              </a>
            </td>
          <td>
            B. Claes, R. Biemans, N. Lieuwes, Lynn Theunissen, Kristine Glunde, Ludwig J Dubois, R. Heeren, Eva Cuypers
          </td>
          <td>2024-12-29</td>
          <td>Analysis &amp; Sensing</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d149bc202f5c23c61477d92eaa03f63e4391af44" target='_blank'>
              Descart: a method for detecting spatial chromatin accessibility patterns with inter-cellular correlations
              </a>
            </td>
          <td>
            Xiaoyang Chen, Keyi Li, Xiaoqing Wu, Zhen Li, Qun Jiang, Xuejian Cui, Zijing Gao, Yanhong Wu, Rui Jiang
          </td>
          <td>2024-12-30</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gliomas are aggressive tumors with a poor prognosis. The protocols presented here outline the methods used to study tumor progression, the tumor microenvironment (TME), and the effects of experimental treatments. The Sleeping Beauty (SB) transposase system induces tumors de novo to generate mouse models that recapitulate human gliomas. Plasmids are constructed with oncogenic drivers and other genetic alterations of interest. which are recognized by their unique position in between inverted/direct repeat (IR/DR) sequences. Luciferase enzyme is used to monitor the uptake of the plasmid, tumor growth, and response to experimental therapies. The genes of interest are tracked using fluorescent markers. Tumors will arise in immunocompetent hosts, which provides a relevant preclinical platform for analysis of tumor initiation, progression, survival, immune microenvironment, and histopathological features. Once the tumor grows within the desired brain location, it can be harvested to generate cell cultures of neurospheres for future experimentation. The benefit of implantable models generated from SB tumors is that they provide specific anatomical and genetic context, in which specific genetic characteristics can be tracked, as they are co-expressed with fluorescent markers. Post glioma cell implantation, additional analysis of the TME and tumor growth can be performed through immunohistochemistry (IHC) and flow cytometry. © 2025 Wiley Periodicals LLC. Basic Protocol 1: Creation of mouse glioma models by Sleeping-Beauty-mediated transposition Basic Protocol 2: Generation of orthotopic implantable brain tumors and neurospheres Basic Protocol 3: Hematoxylin and eosin staining of glioma tissue samples Basic Protocol 4: Immunohistochemistry of glioma tissue samples Basic Protocol 5: Flow cytometry for immune cell analysis of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/886ac84c05876a63108c36f36b7b7af78f353102" target='_blank'>
              Genetically Engineered and Implantable Mouse Brain Tumor Models: Characterization by Immunohistochemistry and Flow Cytometry.
              </a>
            </td>
          <td>
            Apoorva Mirji, Gurveer Singh, Anzar A. Mujeeb, Brandon L. McClellan, YingXiang Li, Makayla Perez, Maria G Castro
          </td>
          <td>2025-01-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f22ff573111b3740a2cb886527997d19806de8b" target='_blank'>
              Single-cell RNA sequencing reveals tumor heterogeneity in small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Xuesong Xiang, X. Tao, Keqin Hua, Hua Jiang, Jingxin Ding
          </td>
          <td>2025-02-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within spatial patterns remain poorly understood. Here, we present the Triangulation Cellular Community Motif Neural Network (TrimNN), a graph-based deep learning framework designed to identify conserved spatial cell organization patterns, termed Cellular Community (CC) motifs, from spatial transcriptomics and proteomics data. TrimNN employs a semi-divide-and-conquer approach to efficiently detect over-represented topological motifs of varying sizes in a triangulated space. By uncovering CC motifs, TrimNN reveals key associations between spatially distributed cell-type patterns and diverse phenotypes. These insights provide a foundation for understanding biological and disease mechanisms and offer potential biomarkers for diagnosis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c80ec316e2753068ced3597143b442f6a178a7b" target='_blank'>
              TrimNN: Characterizing cellular community motifs for studying multicellular topological organization in complex tissues
              </a>
            </td>
          <td>
            Yang Yu, Shuang Wang, Jinpu Li, Meichen Yu, Kyle McCrocklin, Jing-Qiong Kang, A. Ma, Qin Ma, Dong Xu, Juexin Wang
          </td>
          <td>2024-12-22</td>
          <td>Research Square</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Image-based profiling is rapidly transforming drug discovery, offering unprecedented insights into cellular responses. However, experimental variability hinders accurate identification of mechanisms of action (MoA) and compound targets. Existing methods commonly fail to generalize to novel compounds, limiting their utility in exploring uncharted chemical space. To address this, we present a confounder-aware foundation model integrating a causal mechanism within a latent diffusion model, enabling the generation of balanced synthetic datasets for robust biological effect estimation. Trained on over 13 million Cell Painting images and 107 thousand compounds, our model learns robust cellular phenotype representations, mitigating confounder impact. We achieve state-of-the-art MoA and target prediction for both seen (0.66 and 0.65 ROC-AUC) and unseen compounds (0.65 and 0.73 ROC-AUC), significantly surpassing real and batch-corrected data. This innovative framework advances drug discovery by delivering robust biological effect estimations for novel compounds, potentially accelerating hit expansion. Our model establishes a scalable and adaptable foundation for cell imaging, holding the potential to become a cornerstone in data-driven drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e5181b7f0c0ceeecbeb4bfde45a1c946b950b6" target='_blank'>
              Confounder-aware foundation modeling for accurate phenotype profiling in cell imaging
              </a>
            </td>
          <td>
            G. Papanastasiou, Pedro P. Sanchez, Argyrios Christodoulidis, Guang Yang, W. H. L. Pinaya
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Tumor heterogeneity and interaction with tumor microenvironment play a crucial role in neoadjuvant chemotherapy (NAC) resistance in breast cancer (BRCA). Unraveling this dynamic interaction may help uncover novel therapeutic targets. Here, dynamic changes in tumor states and cellular composition are systemically characterized using 175,825 single-cell transcriptomics from naïve and post-treatment biopsies of BRCA patients receiving NAC. CDK16+ tumors are identified featured with luminal progenitor cell (LPC)-like tumor cells enriched in the triple-negative subtype of BRCA, associated with chemo-resistance. Integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and six independent public gene expression profiles that underwent chemotherapy revealed that POSTN+ cancer-associated fibroblasts (CAFs) are closely localized and interacted with CDK16+ LPC-like tumor cells to promote chemo-resistance. In vivo, CDK16 knockdown in tumor cells combined with chemotherapy significantly enhanced therapeutic efficacy. This in-house scRNA-seq from a mouse model validated that CDK16 knockdown reduced the LPC-like tumor cell signature, and the interaction of tumor featured with LPC-like tumor cells and POSTN+ CAFs. Together, the systematically integrated analyses uncovered an interaction network of CDK16+ tumor and POSTN+ CAFs that contributed to NAC- resistance, providing a new strategy for targeting CDK16 to enhance chemotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5910ba964df4cca18ebc577234ea9a9e6c01d2f" target='_blank'>
              CDK16+ Luminal Progenitor Cell-Like Tumor Cells Interacted with POSTN+ Cancer-Associated Fibroblasts Associate with Chemo-Resistance In Breast Cancer.
              </a>
            </td>
          <td>
            Xinyu Ding, Qi Wu, Yanhua Du, Meng-Meng Ji, Hua Yang, Qingsong Hu, Youqiong Ye
          </td>
          <td>2025-02-10</td>
          <td>Small methods</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3688661c2fe1fbcdd267f213b5e012b7afe3157f" target='_blank'>
              Synergistic strategies for glioblastoma treatment: CRISPR-based multigene editing combined with immune checkpoint blockade
              </a>
            </td>
          <td>
            Xiaolin Liu, Xiao Liu, Xiaonan Luo, Maorong Zhu, Nannan Liu, Juan Li, Qi Zhang, Cheng Zou, Yuxin Wu, Zhen Cao, Shuangxin Ma, Weizhong Wang, Guangzhao Yang, Jintao Gu, Wei Liu, Meng Li, Anan Yin, Yalong He, Wei Lin
          </td>
          <td>2025-02-07</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The evolution of metastasis, the spread of cancer to distal sites within the body, represents a lethal stage of cancer progression. Yet, the evolutionary dynamics that shape the emergence of metastatic disease remain unresolved. Here, using single-cell lineage tracing data in combination with phylogenetic statistical methods, we show that the evolutionary trajectory of metastatic disease is littered with bursts of rapid molecular change as new cellular subpopulations appear, a pattern known as punctuational evolution. Next, by measuring punctuational evolution across the metastatic cascade, we show that punctuational effects are concentrated within the formation of secondary tumours at distal metastatic sites, suggesting that qualitatively different modes of evolution may drive primary and metastatic tumour progression. Taken as a whole, our findings provide empirical evidence for distinct patterns of molecular evolution at early and late stages of metastatic disease and our approach provides a framework to study the evolution of metastasis at a more nuanced level than has been previously possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eea86b0534290bd81992d00ef5caf3aad45d15" target='_blank'>
              Punctuational evolution is pervasive in distal site metastatic colonization
              </a>
            </td>
          <td>
            George Butler, Sarah R. Amend, C. Venditti, Kenneth J Pienta
          </td>
          <td>2025-01-01</td>
          <td>Proceedings of the Royal Society B: Biological Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54d1904727e65e93a89095c3025ebc1f384c120" target='_blank'>
              A genome-wide atlas of human cell morphology.
              </a>
            </td>
          <td>
            Meraj Ramezani, Erin Weisbart, J. Bauman, Avtar Singh, John Yong, Maria Lozada, Gregory P. Way, Sanam L. Kavari, Celeste Diaz, Eddy Leardini, Gunjan Jetley, Jenlu Pagnotta, Marzieh Haghighi, Thiago M. Batista, Joaquín Pérez-Schindler, M. Claussnitzer, Shantanu Singh, B. Cimini, P. Blainey, A. Carpenter, Calvin H Jan, James T Neal
          </td>
          <td>2025-01-27</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, , Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3cbd55a53252e9bb3dfd532de1bccedf85f35d" target='_blank'>
              Tumor-associated-fibrosis and active collagen-CD44 axis characterize a poor-prognosis subtype of gastric cancer and contribute to tumor immunosuppression
              </a>
            </td>
          <td>
            Yingqi Yang, Haohan Sun, Hongkai Yu, Luyao Wang, Chang Gao, Haokun Mei, Xiaomeng Jiang, Minghui Ji
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47537700619a16e63c5ebabefc327d2e5284da8b" target='_blank'>
              Transcriptome analysis of novel B16 melanoma metastatic variants generated by serial intracarotid artery injection
              </a>
            </td>
          <td>
            J. C. Kienzler, Erick Contreras, J. Treger, Linda M. Liau, Geoffrey Owens, Robert M. Prins
          </td>
          <td>2025-01-16</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor interpatient heterogeneity has been well described with two major prognostic subtypes (classical and basal-like). An important intrapatient heterogeneity has been reported but has not yet been studied extensively due to the lack of standardized, reproducible, and easily accessible high-throughput methods. We built an immunohistochemical (IHC) tool capable of differentiating RNA-defined classical and basal-like tumors by selecting relevant antibodies using a multistep process. The successive stages of (i) an in silico selection from a literature review and a bulk transcriptome analysis of 309 PDACs, (ii) a tumor-specific selection from 30 patient-derived xenografts and single-cell data, followed by (iii) the validation on tissue microarrays in 50 PDAC were conducted. We used our final IHC panel on two independent cohorts of resected PDAC (n = 95, whole-slide, n = 148, tissue microarrays) for external validation. After digitization and registration of pathology slides, we performed a tile-based analysis in tumor areas to identify relevant marker combinations. Sequential marker selection led to the following panel: GATA6, CLDN18, TFF1, MUC16, S100A2, KRT17, PanBasal. Four different phenotypes were identified: one classical, one intermediate (KRT17+), and two basal-like (MUC16+ versus S100A2+) with specific biological properties. The presence of a minor basal contingent drastically reduced overall survival [hazard ratio (HR) = 1.90, p = 0.03], even in classical predominant PDACs. Analysis of preneoplastic lesions suggested that pancreatic carcinogenesis might follow a progressive evolution from classical toward a basal through an early intermediate phenotype. In conclusion, our IHC panel redefined and easily assessed the high degree of intra- and intertumoral heterogeneity of PDAC. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9c9cb0346cebac5553bfe30b873ba04769f272" target='_blank'>
              Redefining phenotypic intratumor heterogeneity of pancreatic ductal adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            M. Hilmi, F. Delecourt, J. Raffenne, Taib Bourega, Nelson Dusetti, J. Iovanna, Yuna Blum, Magali Richard, C. Neuzillet, A. Couvelard, Matthieu Tihy, L. de Mestier, Vinciane Rebours, R. Nicolle, Jérôme Cros
          </td>
          <td>2025-02-11</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5eb0294ad5d50d6c71886518bd626ba65245bb" target='_blank'>
              UBE2J1 is identified as a novel plasma cell-related gene involved in the prognosis of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yunjie Tian, Ruoyu Dong, Yingxia Guan, Ying Wang, Wei Zhao, Jun Zhang, Shan Kang
          </td>
          <td>2025-01-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in neoplastic initiation and progression. Epigenetic machinery, governing the expression of core oncogenes and tumor suppressor genes in transformed cells, significantly contributes to tumor development at both primary and distant sites. Recent studies have illuminated how epigenetic mechanisms integrate external cues and downstream signals, altering the phenotype of stromal cells and immune cells. This remolds the area surrounding tumor cells, ultimately fostering an immunosuppressive microenvironment. Therefore, correcting the TME by targeting the epigenetic modifications holds substantial promise for cancer treatment. This review synthesizes recent research that elucidates the impact of specific epigenetic regulations—ranging from DNA methylation to histone modifications and chromatin remodeling—on stromal and immune cells within the TME. Notably, we highlight their functional roles in either promoting or restricting tumor progression. We also discuss the potential applications of epigenetic agents for cancer treatment, envisaging their ability to normalize the ecosystem. This review aims to assist researchers in understanding the dynamic interplay between epigenetics and the TME, paving the way for better epigenetic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea83192c32e6b6228a3b5ff261efd56060404967" target='_blank'>
              Epigenetic Regulation of Stromal and Immune Cells and Therapeutic Targets in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Kang Liu, Yue Li, Minmin Shen, Wei Xu, Shanshan Wu, Xinxin Yang, Bo Zhang, Nengming Lin
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b664fbb9ca2f0d479f54b9221d29d13ae0d1a6" target='_blank'>
              Transcriptional regulatory network analysis identifies GRN as a key regulator bridging chemotherapy and immunotherapy response in small cell lung cancer
              </a>
            </td>
          <td>
            S. Yoo, , S. Karam, Yi Zhong, Li Wang, Feng Jiang, Ranran Kong, Sharon Bikvan, Wenhui Wang, A. Sinha, Charles A. Powell, Jun Zhu, Hideo Watanabe
          </td>
          <td>2025-02-05</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Pineiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small subset within the tumor mass significantly contributing to cancer progression through dysregulation of various oncogenic pathways, driving tumor growth, chemoresistance and metastasis formation. The aggressive behavior of CSCs is guided by several intracellular signaling pathways such as WNT, NF-kappa-B, NOTCH, Hedgehog, JAK-STAT, PI3K/AKT1/MTOR, TGF/SMAD, PPAR and MAPK kinases, as well as extracellular vesicles such as exosomes, and extracellular signaling molecules such as cytokines, chemokines, pro-angiogenetic and growth factors, which finely regulate CSC phenotype. In this scenario, tumor microenvironment (TME) is a key player in the establishment of a permissive tumor niche, where CSCs engage in intricate communications with diverse immune cells. The “oncogenic” immune cells are mainly represented by B and T lymphocytes, NK cells, and dendritic cells. Among immune cells, macrophages exhibit a more plastic and adaptable phenotype due to their different subpopulations, which are characterized by both immunosuppressive and inflammatory phenotypes. Specifically, tumor-associated macrophages (TAMs) create an immunosuppressive milieu through the production of a plethora of paracrine factors (IL-6, IL-12, TNF-alpha, TGF-beta, CCL1, CCL18) promoting the acquisition by CSCs of a stem-like, invasive and metastatic phenotype. TAMs have demonstrated the ability to communicate with CSCs via direct ligand/receptor (such as CD90/CD11b, LSECtin/BTN3A3, EPHA4/Ephrin) interaction. On the other hand, CSCs exhibited their capacity to influence immune cells, creating a favorable microenvironment for cancer progression. Interestingly, the bidirectional influence of CSCs and TME leads to an epigenetic reprogramming which sustains malignant transformation. Nowadays, the integration of biological and computational data obtained by cutting-edge technologies (single-cell RNA sequencing, spatial transcriptomics, trajectory analysis) has significantly improved the comprehension of the biunivocal multicellular dialogue, providing a comprehensive view of the heterogeneity and dynamics of CSCs, and uncovering alternative mechanisms of immune evasion and therapeutic resistance. Moreover, the combination of biology and computational data will lead to the development of innovative target therapies dampening CSC-TME interaction. Here, we aim to elucidate the most recent insights on CSCs biology and their complex interactions with TME immune cells, specifically TAMs, tracing an exhaustive scenario from the primary tumor to metastasis formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ce167799371ac069e6df125e4a468fd20b0208" target='_blank'>
              Cancer stem cells and tumor-associated macrophages as mates in tumor progression: mechanisms of crosstalk and advanced bioinformatic tools to dissect their phenotypes and interaction
              </a>
            </td>
          <td>
            Francesco Verona, Sebastiano Di Bella, Roberto Schirano, Camilla Manfredi, Francesca Angeloro, Giulia Bozzari, M. Todaro, Giuseppe Giannini, Giorgio Stassi, V. Veschi
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background & Aims
Identifying patients with intrahepatic cholangiocarcinoma (ICC) likely to benefit from immunochemotherapy, the new front-line treatment, remains challenging. We aimed to unveil a novel radiotranscriptomic signature that can facilitate treatment response prediction by multi-omics integration and multi-scale modelling.


Methods
We analyzed bulk, single-cell and spatial transcriptomic data comprising 457 ICC patients to identify an immune-related score (IRS), followed by decoding its spatial immune context. We mapped radiomics profiles onto spatial-specific IRS using machine-learning to define a novel radiotranscriptomic signature, followed by multi-scale and multi-cohort validation covering 331 ICC patients. The signature was further explored for the potential therapeutic target from in vitro to in vivo.


Results
We revealed a novel 3-gene (PLAUR, CD40LG and FGFR4) IRS whose down-regulation correlated with better survival and improved sensitivity to immunochemotherapy. We highlighted functional IRS-immune interactions within tumor epithelium, rather than stromal compartment, irrespective of geospatial locations. Machine-learning pipeline identified the optimal 3-feature radiotranscriptomic signature that was well-validated by immunohistochemical assays in molecular cohort, exhibited favorable external prognostic validity with C-index over 0.64 in resection cohort, and predicted treatment response with an area under the curve of up to 0.84 in immunochemotherapy cohort. We also showed that anti-uPAR/PLAUR alone or in combination with anti-programmed cell death protein 1 therapy remarkably curbed tumor growth, using in vitro ICC cell lines and in vivo humanized ICC patient-derived xenograft mouse models.


Conclusions
This proof-of-concept study sheds light on the spatially-resolved radiotranscriptomic signature to improve patient selection for emerging immunochemotherapy and high-order immunotherapy combinations in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, , C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified. Methods Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs). Results Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes. Conclusions CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spine metastases are the most common bone site for breast cancer, with evolving surgery and multidisciplinary care improving outcomes. Current treatments, including chemotherapy and invasive surgery, may damage healthy tissue and may leave residual tumors that lead to recurrence. Cold atmospheric plasma (CAP) offers a non-invasive alternative by delivering reactive oxygen and nitrogen species (RONS) locally to tumor sites, selectively targeting cancer cells while sparing healthy tissue. To assess the impact and selectivity toward tumor cells adjacent to bone-like tissue, we develop a 3D bioprinted tumor-stroma model using a 1% alginate and 7% gelatin cell-laden hydrogel to mimic a bone-like microenvironment. The model co-cultures triple-negative MDA-MB-231 human breast cancer cells with primary human bone marrow mesenchymal stromal cells to simulate tumor-stroma interactions. The effects of CAP treatments are assessed through metabolic activity and viability assays over three days. Results show significant selectivity for cancer cells in both 2D and 3D cultures. CAP minimizes damage to healthy cells, offering the potential for localized treatment over systemic chemotherapies such as doxorubicin. Our novel bioprinted model, combined with a plasma source controlling RONS composition, enables detailed studies of redox-based cancer cell inactivation and highlights CAP as a personalized, non-invasive treatment for bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db15f717a397933ff6371b23deebeffcaae60439" target='_blank'>
              Cold Atmospheric Plasma Selectively Disrupts Breast Cancer Growth in a Bioprinted 3D Tumor Microenvironment Model
              </a>
            </td>
          <td>
            Laura M. Bouret, Jean-Baptiste Billeau, Michael H. Weber, Derek H. Rosenzweig, Stephan Reuter
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state.
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is the most prevalent malignant brain tumor, with an average survival time of 14 to 20 months. Its capacity to invade brain parenchyma leads to the failure of conventional treatments and subsequent tumor recurrence. Recent studies have explored new therapeutic strategies using a chemoattracting gradient to attract GBM cells into a soft hydrogel trap where they can be exposed to higher doses of radiation or chemotherapy. It has been demonstrated in vitro under static conditions, that nanoparticles (NPs) encapsulating the chemoattractant CXCL12 can create a gradient to attract GBM cell. However, GBM cell invasion is also largely dependent on interstitial fluid flow (IFF). In the present study, a custom-made in vitro 3D model with indirect perfusion to mimic IFF at flow rates of 0.5 μL/min and 3 μL/min was used to examine the invasive behavior of F98-rodent-derived and U87-human-derived GBM cells. This model simulated IFF and CXCL12 gradient within an alginate:matrigel-based hydrogel mimicking brain parenchyma. Findings revealed that CXCL12 (1600 ng/mL) released from NPs significantly increased the migration of F98 GBM cells after 72 hours under IFF conditions at both 0.5 and 3 μL/min. In contrast, U87 GBM cells required a higher CXCL12 concentration (2400 ng/mL) and longer incubation time for migration (120 hours). Unlike the F98 cells, U87 GBM cells showed a CXCL12 dose-dependent proliferation. Semi-quantitative qPCR showed higher CXCR4 mRNA levels in F98 cells than in U87 cells. CXCL12 significantly increased intracellular calcium levels via CXCR4 activation, with a 2.3-fold rise in F98 cells compared to U87, consistent with observed cell behavior during perfusion. This highlights the combined influence of IFF and CXCL12 on cell migration, dependent on cell line. This 3D dynamic model is a valuable tool to analyze parameters like interstitial fluid flow (IFF) and chemokine gradients, influenced by GBM tumor diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6bd1496f5c1fbe6eb347fa559aed0f8ce3defbb" target='_blank'>
              CXCL12 impact on glioblastoma cells behaviors under dynamic culture conditions: Insights for developing new therapeutic approaches
              </a>
            </td>
          <td>
            Wiam El Kheir, Sahar Naasri, Bernard Marcos, Nick Virgilio, Benoit Paquette, Nathalie Faucheux, Marc-Antoine Lauzon
          </td>
          <td>2024-12-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune checkpoint (ICP) blockade has shown limited effectiveness in glioblastoma (GBM), particularly in the mesenchymal subtype, where interactions between immune cells and glioblastoma cancer stem cells (GSCs) drive immunosuppression and therapy resistance. Tailoring ICPs specific to GSCs can enhance the antitumor immune response. This study proposes the use of lipid nanoparticles (LNPs) encapsulating CRISPR RNAs as an in vivo screening tool for ICPs in a syngeneic model of mesenchymal GSCs. Using PD-L1 and CD47 to validate the proof of concept, intratumoral administration of LNPs in orthotopic tumors achieved efficient editing of ICPs, leading to enhanced immune cell infiltration within the tumor microenvironment. Targeting CD47 reduced tumor growth, suggesting improved cancer cell sensitization to the immune system post-ICP editing. The study positions LNPs as a robust tool for in vivo validation of ICPs as therapeutic targets in clinically relevant GBM models. LNPs could serve as a screening tool in patient-derived xenografts to identify and optimize ICP combinations, potentially expediting ICP translation and enhancing personalized GBM immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80511bcdd71416b8352607629bb871f6f30a5f0" target='_blank'>
              CD47 Knock-Out Using CRISPR-Cas9 RNA Lipid Nanocarriers Results in Reduced Mesenchymal Glioblastoma Growth In Vivo.
              </a>
            </td>
          <td>
            N. Rouatbi, A. Walters, Alaa Zam, Y. Lim, Alessia Marrocu, Revadee Liam-or, Joanne E. Anstee, James N Arnold, J. Wang, Steven M. Pollard, Khuloud T Al-Jamal
          </td>
          <td>2025-01-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Mouse models that faithfully represent the biology of human brain tumors are critical tools for unraveling the underlying tumor biology and screening for potential precision therapies. This is especially true of rare tumor types, many of which have correspondingly few xenograft or cell lines available. Although our understanding of the specific biological pathways driving cancer has improved significantly, identifying the appropriate progenitor populations to drive oncogenic processes represents a significant barrier to efficient mouse model production. In this issue of Cancer Research, Jessa and colleagues developed an innovative transcription factor fingerprinting method to map the cellular origin of central nervous system neuroblastoma, FOXR2-activated to medial ganglionic eminence-derived interneurons, which could then be efficiently targeted in the developing mouse brain using in utero electroporation. This approach serves as a blueprint for investigating other rare pediatric brain tumors, potentially accelerating progress toward the development of mouse models and identification of effective therapies. See related article by Jessa et al., p. 231.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36b1d9ff773da8434bf45b0c13f61a915472b5f" target='_blank'>
              Transcription Factor Fingerprint Provides Clues for Brain Tumor Cell of Origin.
              </a>
            </td>
          <td>
            Brent A. Orr
          </td>
          <td>2024-01-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Basement membrane (BM) supports and regulates the structural integrity, function and differentiation of epithelial tissues and protects against stromal invasion of breast cancer cells. Here we show that LAMA5, a central BM component, is critical for luminal mammary tumour initiation, and controls tumor progression and phenotype development. LAMA5 is overexpressed in both human and mouse luminal mammary tumours and LAMA5 downregulation attenuates growth of human breast cancer cells. Prepubertal luminal deletion of Lama5 in MMTV-PyMT mice results in marked reduction in emergence of early hyperplasias, along with a shift from mature hormone receptor (HR+) positive luminal epithelial phenotype towards a HR+ progenitor phenotype, and widespread alterations in extracellular matrix and Fibroblast growth factor (Fgf) signalling genes, including overexpression of Fgf receptor 2 (Fgfr2) in the HR+ progenitor cells. Single allele deletion, but not biallelic deletion of Lama5 additionally inhibits growth and progression towards advanced mammary carcinomas. However, inhibition of Fgf receptors restores growth decrease and induces apoptosis in biallelic-Lama5-deleted organoids without affecting wildtype organoids. Our analyses demonstrate a critical role for a BM component LAMA5 in mammary tumour initiation and reveal mechanisms of extracellular matrix-epithelial interplay in breast tumour progression and phenotype maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6489e8a0631d1e53cfa8dbbd16ee60384d5aff0" target='_blank'>
              Tumour-derived LAMA5 is critical for tumour initiation and controls progression and phenotype in luminal breast cancer
              </a>
            </td>
          <td>
            Alexandra Ritchie, Priyal Dhawan, Eveliina Kailari, Jere Lindén, Leander Blaas, V. Izzi, Pekka Katajisto, J. Englund
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
The tumour microenvironment significantly influences the clinical response of patients to therapeutic immune checkpoint inhibition (ICI), but a comprehensive understanding of the underlying immune-regulatory proteome is still lacking.


OBJECTIVES
To decipher targetable biologic processes that determine tumour-infiltrating lymphocytes (TiLs) as a cellular equivalent of clinical response to ICI.


METHODS
We mapped the spatial distribution of proteins in TiL-enriched vs. TiL-low compartments in melanoma by combining microscopy, matrix-assisted laser desorption mass spectrometry imaging and liquid chromatography-mass spectrometry, as well as computational data mining. Pharmacological modulation of sirtuin 1 (SIRT1) activity in syngeneic mouse models was used to evaluate the efficacy of pharmacological SIRT1 activation in two syngeneic melanoma mouse models, one known to be α-programmed cell death protein 1 (PD-1) sensitive and the other α-PD-1 resistant.


RESULTS
Spatial proteomics and gene ontology-based enrichment analysis identified > 145 proteins enriched in CD8high tumour compartments, including negative regulators of mammalian target of rapamycin signalling such as SIRT1. Multiplexed immunohistochemistry confirmed that SIRT1 protein was expressed more in CD8high than in CD8low compartments. Further analysis of bulk and single-cell RNA sequencing data from melanoma tissue samples suggested the expression of SIRT1 by different lymphocyte subpopulations (CD8+ T cells, CD4+ T cells and B cells). Furthermore, we showed in vivo that pharmacological SIRT1 activation increased the immunological effect of α-PD-1 ICI against melanoma cells in mice, which was accompanied by an increase in T-cell infiltration and T-cell-related cytokines, including interferon (IFN)-γ, CCL4, CXCL9, CXCL10 and tumour necrosis factor-α. In silico analysis of large transcriptional data cohorts showed that SIRT1 was positively associated with the proinflammatory T-cell chemokines CXCL9, CXCL10 and IFN-γ, and prolonged overall survival of patients with melanoma.


CONCLUSIONS
Our study deciphers the proteomics landscape in human melanoma, providing important information on the tumour microenvironment and identifying SIRT1 as having important prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e28ba69768cc37ada6cc67883351db03f9a038" target='_blank'>
              Spatial proteomics reveals sirtuin 1 to be a determinant of T-cell infiltration in human melanoma.
              </a>
            </td>
          <td>
            J. Placke, Jenny Bottek, R. Váraljai, B. Shannan, Sarah Scharfenberg, Christoph Krisp, Philippa Spangenberg, Camille Soun, Devon Siemes, Lars Borgards, Franziska Hoffmann, Fang Zhao, Anette Paschen, Hartmut Schlueter, Ferdinand von Eggeling, Iris Helfrich, Florian Rambow, Selma Ugurel, A. Tasdogan, Dirk Schadendorf, D. R. Engel, Alexander Roesch
          </td>
          <td>2024-12-31</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a complex and heterogeneous hematologic malignancy characterized by clonal evolution, genetic instability, and interactions with a supportive tumor microenvironment. These factors contribute to treatment resistance, disease progression, and significant variability in clinical outcomes among patients. This review explores the mechanisms underlying MM progression, including the genetic and epigenetic changes that drive clonal evolution, the role of the bone marrow microenvironment in supporting tumor growth and immune evasion, and the impact of genomic instability. We highlight the critical insights gained from single-cell technologies, such as single-cell transcriptomics, genomics, and multiomics, which have enabled a detailed understanding of MM heterogeneity at the cellular level, facilitating the identification of rare cell populations and mechanisms of drug resistance. Despite the promise of advanced technologies, MM remains an incurable disease and challenges remain in their clinical application, including high costs, data complexity, and the need for standardized bioinformatics and ethical considerations. This review emphasizes the importance of continued research and collaboration to address these challenges, ultimately aiming to enhance personalized treatment strategies and improve patient outcomes in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, 
          </td>
          <td>2025-02-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c65e0e056080142eb716f1c5270f62643243964" target='_blank'>
              From ductal carcinoma in situ to invasive breast cancer: the prognostic value of the extracellular microenvironment
              </a>
            </td>
          <td>
            Taylor S. Hulahan, Peggi M. Angel
          </td>
          <td>2024-12-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="The progression of colorectal cancer is strongly influenced by environmental and genetic conditions. One of the key factors is tumor heterogeneity which is extensively studied by cfDNA and bulk sequencing methods; however, we lack knowledge regarding its effects at the single-cell level. Motivated by this, we aimed to employ an end-to-end single-cell sequencing workflow from tissue-derived sample isolation to exome sequencing. Our main goal was to investigate the heterogeneity patterns by laser microdissecting samples from different locations of a tissue slide. Moreover, by studying healthy colon control, tumor-associated normal, and colorectal cancer tissues, we explored tissue-specific heterogeneity motifs. For completeness, we also compared the performance of the whole-exome bulk, cfDNA, and single-cell sequencing methods based on variation at the level of a single nucleotide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9266548dfc591e54ed5a1417777c06bd133c4150" target='_blank'>
              Investigation of Exome-Wide Tumor Heterogeneity on Colorectal Tissue-Based Single Cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, A. Kalmár, B. Barták, Z. Nagy, Gábor Valcz, Tamás Richárd Linkner, Kristóf Róbert Rada, István Takács, Béla Molnár
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Liver cancer is a leading cause of cancer-related deaths worldwide, highlighting the need for innovative approaches to understand its complex biology and develop effective treatments. While traditional in vivo animal models have played a vital role in liver cancer research, ethical concerns and the demand for more human-relevant systems have driven the development of advanced in vitro models. Spheroids and organoids have emerged as powerful tools due to their ability to replicate tumor microenvironment and facilitate preclinical drug development. Spheroids are simpler 3D culture models that partially recreate tumor structure and cell interactions. They can be used for drug penetration studies and high-throughput screening. Organoids derived from stem cells or patient tissues that accurately emulate the complexity and functionality of liver tissue. They can be generated from pluripotent and adult stem cells, as well as from liver tumor specimens, providing personalized models for studying tumor behavior and drug responses. Liver organoids retain the genetic variability of the original tumor and offer a robust platform for high-throughput drug screening and personalized treatment strategies. However, both organoids and spheroids have limitations, such as the absence of functional vasculature and immune components, which are essential for tumor growth and therapeutic responses. The field of preclinical modeling is evolving, with ongoing efforts to develop more predictive and personalized models that reflect the complexities of human liver cancer. By integrating these advanced in vitro tools, researchers can gain deeper insights into liver cancer biology and accelerate the development of novel treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375c4613d7c88e38aaf9cca61e8b7c2e909f2dd5" target='_blank'>
              Organoids and spheroids: advanced in vitro models for liver cancer research
              </a>
            </td>
          <td>
            M. Pastore, Alessia Giachi, Elena Spínola-Lasso, F. Marra, C. Raggi
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and Aims Total abdominal colectomy (TAC) with a staged ileal pouch-anal anastomosis (IPAA) is a common surgical treatment for ulcerative colitis (UC). However, a significant percentage of patients experience pouch failure, leading to morbidity. This retrospective case-control study identified histopathological features of the TAC specimen associated with pouch failure and investigated the molecular mechanisms of this susceptibility using single-cell spatial transcriptomics. Methods We analyzed a cohort of 417 patients who underwent IPAA between 2000-2010 at the University of Chicago Medical Center for up to 18 years. Histological examination of TAC specimens focused on disease activity, depth of inflammation, and specific features, including granulomas and deep ulcers. A subset of patients was profiled using single-cell spatial transcriptomics to map gene expression and immune cell interactions in relation to the risk of pouch failure. Results The 18-year pouch failure risk was 23%, with post-procedure diagnosis of CD as a major risk factor (HR = 4.3, 95% CI: 2.3–8.1) as well as high-risk histologic features, including deep chronic inflammation (HR = 21, 95% CI: 11-41) and severe disease activity (HR = 14, 95% CI: 5.7-32) in TAC specimens. Spatial transcriptomics showed immune infiltration of T and myeloid cells, reduced myocyte-glial interactions, and cytokine signaling pathways such as IL-10, IL-1β, and type I/II interferons, associated with an increased risk of pouch failure. Conclusion Histological features and spatial molecular profiling are predictive of IPAA failure. These findings support the use of histologic evaluation and targeted molecular analysis of the TAC specimen to identify high-risk patients and improve IPAA outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a586fb545c9fdbb6b8896dfb9ebdcb9f9f1dc37" target='_blank'>
              Histopathologic Evaluation and Single-Cell Spatial Transcriptomics of the Colon Reveal Cellular and Molecular Abnormalities Linked to J-Pouch Failure in Patients with Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            Andrea D. Olivas, Paul Chak Mou Ngai, Emily Schahrer, K. Olortegui, John F. Cursio, Shintaro Akiyama, Eugene B. Chang, Le Shen, Konstantin Umanskiy, David T. Rubin, David Zemmour, Christopher R. Weber
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b01929d83c53a2a6c8d3025f3a6d5a89b435982" target='_blank'>
              Tissue-resident memory CD8 T cell diversity is spatiotemporally imprinted.
              </a>
            </td>
          <td>
            Miguel Reina-Campos, Alexander Monell, Amir Ferry, Vida Luna, Kitty P. Cheung, Giovanni Galletti, Nicole E. Scharping, Kennidy K. Takehara, Sara Quon, Peter P Challita, Brigid Boland, Yun Hsuan Lin, William H. Wong, Cynthia S. Indralingam, Hayley Neadeau, Suzie Alarcón, Gene W. Yeo, John T Chang, Maximilian Heeg, Ananda W Goldrath
          </td>
          <td>2025-01-22</td>
          <td>Nature</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patient-derived organoids represent a novel platform to recapitulate the cancer cells in the patient tissue. While cancer heterogeneity has been extensively studied by a number of omics approaches, little is known about the spatiotemporal kinase activity dynamics. Here we applied a live imaging approach to organoids derived from 10 pancreatic ductal adenocarcinoma (PDAC) patients to comprehensively understand their heterogeneous growth potential and drug responses. By automated wide-area image acquisitions and analyses, the PDAC cells were non-selectively observed to evaluate their heterogeneous growth patterns. We monitored single-cell ERK and AMPK activities to relate cellular dynamics to molecular dynamics. Furthermore, we evaluated two anti-cancer drugs, a MEK inhibitor, PD0325901, and an autophagy inhibitor, hydroxychloroquine (HCQ), by our analysis platform. Our analyses revealed a phase-dependent regulation of PDAC organoid growth, where ERK activity is necessary for the early phase and AMPK activity is necessary for the late stage of organoid growth. Consistently, we found PD0325901 and HCQ target distinct organoid populations, revealing their combination is widely effective to the heterogeneous cancer cell population in a range of PDAC patient-derived organoid lines. Together, our live imaging quantitatively characterized the growth and drug sensitivity of human PDAC organoids at multiple levels: in single cells, single organoids, and individual patients. This study will pave the way for understanding the cancer heterogeneity and promote the development of new drugs that eradicate intractable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824e882598ac674f498c3ffe96037afbee14d231" target='_blank'>
              Quantitative Live Imaging Reveals Phase Dependency of PDAC Patient-Derived Organoids on ERK and AMPK Activity.
              </a>
            </td>
          <td>
            Shoko Tsukamoto, Ye Huaze, Weisheng Zhang, Akihito Machinaga, N. Kakiuchi, Seishi Ogawa, Hiroshi Seno, Shigeki Higashiyama, Michiyuki Matsuda, Toru Hiratsuka
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated. Methods To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays. Results In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1+FOLR2+ tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway. Conclusions We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1+FOLR2+TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cellular activity is spatially organized across different organelles. While several structures are well-characterized, many organelles have unknown roles. Profiling biomolecular composition is key to understanding function but is difficult to achieve in the context of small, dynamic structures. Photoproximity labeling has emerged as a powerful tool for mapping these interaction networks, yet maximizing catalyst localization and reducing toxicity remains challenging in live cell applications. Here, we disclose a new intracellular photocatalyst with minimal cytotoxicity and off-target binding, and we utilize this catalyst for HaloTag-based microenvironment-mapping (μMap) to spatially catalog subnuclear condensates in living cells. We also specifically develop a novel RNA-focused workflow (μMap-seq) to enable parallel transcriptomic and proteomic profiling of these structures. After validating the accuracy of our approach, we generate a spatial map across the nucleolus, nuclear lamina, Cajal bodies, paraspeckles, and PML bodies. These results provide potential new insights into RNA metabolism and gene regulation while significantly expanding the μMap platform for improved live-cell proximity labeling in biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cedc48a841755a5a6c7dc0077077c40278c4804" target='_blank'>
              Parallel Proteomic and Transcriptomic Microenvironment Mapping (μMap) of Nuclear Condensates in Living Cells.
              </a>
            </td>
          <td>
            Steve D. Knutson, C. Pan, Niels Bisballe, Brandon J Bloomer, Philip Raftopolous, Iakovos Saridakis, David W. C. MacMillan
          </td>
          <td>2024-12-21</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Tumor hypoxia reshapes the microenvironment, driving progression, invasion, metastasis, and therapy resistance. Patient-derived tumor organoids, formed under three-dimensional conditions, preserve cellular heterogeneity and nutrient gradients, making them ideal for studying hypoxia-induced tumor responses. This study examines hypoxia-induced changes in lung tumor organoids from two patients, focusing on tumor-associated markers, stem cell markers, and migration capabilities.


RESULTS
Our findings demonstrate that hypoxia distinctively modulates the expression of lung cancer markers thyroid transcription factor-1, cytokeratin 7, and ΔNP63 variant. Hypoxia also induces the upregulation of stem cell-associated markers, resulting in an increased proportion of cancer stem cells. Furthermore, hypoxic lung tumor organoids exhibit unique migratory behavior upon reoxygenation, driven by epithelial-mesenchymal transition and the elevated expression of matrix metalloproteinases 7 and matrix metalloproteinases 9, indicating their enhanced invasive potential.


CONCLUSIONS
These findings highlight the value of lung tumor organoids as models for studying hypoxia's complex role in lung cancer. Hypoxia significantly modulates lung tumor organoids growth, stemness, and migratory behavior, providing critical insights into tumor progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e6f35da0894cddd474f02ff42f085ddcfac6ff" target='_blank'>
              Pre-immunotherapy alters stereotactic ablative radiotherapy-induced systemic T cell responses in early-stage NSCLC
              </a>
            </td>
          <td>
            Chao Liu, Yanjuan Chen, Xiaohui Li, Zhijie Bai, Meilin Jiang, Dongsheng Sheng, W. Zou, Rui Huang, Qingyu Huang, Fuhao Wang, Jingyang Zhu, Huiru Sun, Bing Liu, Zongcheng Li, Bing Sun
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d8247ee8babd323978806a38c5b3ce6dbebe55" target='_blank'>
              Cell-specific transcriptional signatures of vascular cells in Alzheimer’s disease: perspectives, pathways, and therapeutic directions
              </a>
            </td>
          <td>
            Soumilee Chaudhuri, M. Cho, Julia C Stumpff, Paula Bice, Özkan İş, Nilufer Ertekin‐Taner, A. Saykin, K. Nho
          </td>
          <td>2025-01-29</td>
          <td>Molecular Neurodegeneration</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The development of treatment resistance remains universal for patients with metastatic prostate cancer, driven by AR alterations and lineage state transitions. Identifying the evolution of lineage transitions in treatment resistance has been limited by the challenges of collecting serial tissue biopsies on treatment, which can be overcome using blood-based liquid biopsies. Utilizing a novel circulating tumor cell (CTC) isolation approach, we collected 273 CTC samples from 117 patients with metastatic prostate cancer for RNA sequencing. 146 samples from 70 patients had tumor purity comparable to tissue biopsies. We identified four CTC transcriptional phenotypes, mirroring lineage states identified in tissue. Patients with a luminal-B-like CTC phenotype defined by persistent AR signaling and high proliferation, as well as those with a neuroendocrine CTC phenotype, had significantly shorter survival than patients with luminal-A-like and low proliferation phenotypes. In a prospective substudy, pre-treatment CTC luminal-B-like phenotype was associated with early progression on 177Lu-PSMA-617.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="ABSTRACT Hepatocellular carcinoma (HCC) is an aggressive disease with poor prognosis, necessitating preclinical models for evaluating novel therapies. Large-animal models are particularly valuable for assessing locoregional therapies, which are widely employed across HCC stages. This study aimed to develop a large-animal HCC model with tailored tumor mutations. The Oncopig, a genetically engineered pig with inducible TP53R167H and KRASG12D, was used in the study. Hepatocytes were isolated from Oncopigs and exposed to Cre recombinase in vitro to create HCC cells, and additional mutations were introduced by CRISPR/Cas9 knockout of PTEN and CDKN2A. These edits increased Oncopig HCC cell proliferation and migration. Autologous HCC cells with these CRISPR edits were implanted into Oncopigs using two approaches: ultrasound-guided percutaneous liver injections, which resulted in the development of localized intrahepatic masses, and portal vein injections, which led to multifocal tumors that regressed over time. Tumors developed by both approaches harbored PTEN and CDKN2A knockout mutations. This study demonstrates the feasibility of developing genetically tailored HCC tumors in Oncopigs using somatic cell CRISPR editing and autologous implantation, providing a valuable large-animal model for in vivo therapeutic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d71b4f7487651e0c16ec4abf6eae82e588f8ed" target='_blank'>
              Development of a genetically tailored implantation hepatocellular carcinoma model in Oncopigs by somatic cell CRISPR editing
              </a>
            </td>
          <td>
            L. Elkhadragy, Maximillian J. Carlino, Luke R. Jordan, Thomas Pennix, Nahed Ismail, Grace Guzman, Jonathan P Samuelson, Lawrence B. Schook, K. Schachtschneider, Ron C. Gaba
          </td>
          <td>2025-01-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-Yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="BACKGROUND Gastric cancer (GC) poses a substantial risk to human health due to its high prevalence and mortality rates. Nevertheless, current therapeutic strategies remain insufficient. Single-cell RNA sequencing (scRNA-seq) offers the potential to provide comprehensive insights into GC pathogenesis. AIM To explore the distribution and dynamic changes of cell populations in the GC tumor microenvironment using scRNA-seq techniques. METHODS Cancerous tissues and paracancerous tissues were obtained from patients diagnosed with GC at various stages (I, II, III, and IV). Single-cell suspensions were prepared and analyzed using scRNA-seq to examine transcriptome profiles and cell-cell interactions. Additionally, quantitative real-time polymerase chain reaction (qRT-PCR) and flow cytometry were applied for measuring the expression of cluster of differentiation (CD) 2, CD3D, CD3E, cytokeratin 19, cytokeratin 8, and epithelial cell adhesion molecules. RESULTS Transcriptome data from 73645 single cells across eight tissues of four patients were categorized into 25 distinct cell clusters, representing 10 different cell types. Variations were observed in these cell type distribution. The adjacent epithelial cells in stages II and III exhibited a degenerative trend. Additionally, the quantity of CD4 T cells and CD8 T cells were evidently elevated in cancerous tissues. Interaction analysis displayed a remarkable increase in interaction between B cells and other mast cells in stages II, III, and IV of GC. These findings were further validated through qRT-PCR and flow cytometry, demonstrating elevated T cells and declined epithelial cells within the cancerous tissues. CONCLUSION This study provides a comprehensive analysis of cell dynamics across GC stages, highlighting key interactions within the tumor microenvironment. These findings offer valuable insights for developing novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3adb4df70cb2b109a0b602e7b75fd956596d307f" target='_blank'>
              Landscape of four different stages of human gastric cancer revealed by single-cell sequencing
              </a>
            </td>
          <td>
            Xushan Tang, Chun-Lei Xu, Na Li, Jian-Qing Zhang, Yong Tang
          </td>
          <td>2025-02-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38fe8b7dd282acd4da49fd77a60ef2b609353e0" target='_blank'>
              S100A13-driven interaction between pancreatic adenocarcinoma cells and cancer-associated fibroblasts promotes tumor progression through calcium signaling
              </a>
            </td>
          <td>
            Liuyuan Xia, Xin Guo, Dong Lu, Yixin Jiang, Xiaohui Liang, Yiwen Shen, Jia-Yi Lin, Lijun Zhang, Hongzhuan Chen, Jinmei Jin, Xin Luan, Weidong Zhang
          </td>
          <td>2025-01-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is a major challenge that was recognized as one of the first Cancer Grand Challenges, with a call to provide solutions to visualize tumor heterogeneity. The Rosetta team took on this challenge, exploiting advances in spatial-omics approaches centered around mass spectrometry imaging to map tumor heterogeneity at the cellular and molecular scales with different levels of resolution. See related article by Bressan et al., p. 16 See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d308601bb28236e43c3bc271c19d29c8dd8720ed" target='_blank'>
              Visualizing Cancer Heterogeneity at the Molecular and Cellular Levels: Lessons from Rosetta.
              </a>
            </td>
          <td>
            R. Goodwin, John F Marshall, G. Poulogiannis, Mariia Yuneva, K. Brindle, Z. Takáts, Owen J. Sansom, Josephine Bunch, Simon T Barry
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hypoxia, a common feature in many malignancies, is particularly prominent in cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC). Investigating the mechanisms underlying hypoxia is essential for understanding the heterogeneity of CESC and developing personalized therapeutic regimens. Firstly, the CESC-specific hypoxia gene sets shared between single-cell RNA sequencing (scRNA-seq) and bulk data were identified through Weighted Gene Correlation Network Analysis (WGCNA)and FindMarkers analyses. A CESC-specific hypoxia-related score (CSHRS) risk model was constructed using the least absolute shrinkage and selection operator (LASSO)and Cox regression analyses based on these genes. The prognostic differences were analyzed in terms of immune infiltration, mutations, and drug resistance. Finally, a nomogram model was constructed by integrating clinicopathological features to facilitate precision treatment for CESC. This study constructed a CSHRS risk model that divides patients into two groups, and this model can comprehensively evaluate the tumor microenvironment characteristics of CESC, provide accurate prognostic predictions, and offer rational treatment options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7d7f2c99de4e5f9ef9acea8ebc376cd53f13e8" target='_blank'>
              Integrating Hypoxia Signatures from scRNA-seq and Bulk Transcriptomes for Prognosis Prediction and Precision Therapy in Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma
              </a>
            </td>
          <td>
            Kexin Yu, Shibo Zhang, Jiali Shen, Meini Yu, Yangguang Su, Ying Wang, Kun Zhou, Lei Liu, Xiujie Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Thyroid cancer is one of the most common endocrine tumors worldwide, especially among women and the metastatic mechanism of papillary thyroid carcinoma remains poorly understood. Methods Thyroid cancer tissue samples were obtained for single‐cell RNA‐sequencing and spatial transcriptomics, aiming to intratumoral and antimetastatic heterogeneity of advanced PTC. The functions of APOE in PTC cell proliferation and invasion were confirmed through in vivo and in vitro assays. Pseudotime analysis and CellChat were performed to explore the the molecular mechanisms of the APOE in PTC progression. Results We identified a subpopulation of tumor cells with lower expression levels of APOE, associated with advanced stages of PTC and cervical metastasis. APOE overexpression significantly reduced tumor cell proliferation and invasion, both in vitro and in vivo, by activating the ABCA1‐LXR axis. APOE− tumor cells may promote tumor growth by interacting with dendritic cells and CD4+ T cells via CD99‐ rather than CD6‐regulated signaling. We established a machine learning‐based scRNA‐seq data, 13‐gene signature predictive of lymph node metastasis. Conclusions We identified a distinct APOE− tumor cell population associated with cervical metastasis and poor prognosis. Our results and models have potential clinical, prognostic, and therapeutic implications for advanced PTC. Key points A subpopulation of tumor cells with lower expression levels of APOE was strongly associated with more advanced stages and metastasis of PTC. APOE‐negative (APOE −) cellsoverall exhibited weaker interactions with immune cells. A machine‐learning bioinformatics model based on scRNA‐seq data of in‐situ thyroid cancer tissue was established to predict lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170f7e4d14fbccaf4771b3c72a44d532b8d44e86" target='_blank'>
              Single‐cell RNA‐sequencing and spatial transcriptomic analysis reveal a distinct population of APOE − cells yielding pathological lymph node metastasis in papillary thyroid cancer
              </a>
            </td>
          <td>
            Guohui Xiao, Rongli Xie, Jian-hua Gu, Yishu Huang, Min Ding, Dongjie Shen, Jiqi Yan, Jianmin Yuan, Qiong Yang, Wen He, Siyu Xiao, Haizhen Chen, Dan Xu, Jian Wu, Jian Fei
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Urothelial carcinoma (UC) can arise from either the lower urinary tract or the upper tract; they represent different disease entities and require different clinical treatment strategies. A full understanding of the cellular characteristics in UC may guide the development of novel therapies. Here, we performed single-cell transcriptome analysis from four patients with UC of the bladder (UCB), five patients with UC of the ureter (UCU), and four patients with UC of the renal pelvis (UCRP) to develop a comprehensive cell atlas of UC. We found the rare epithelial cell subtype EP9 with epithelial-to-mesenchymal transition (EMT) and cancer stem cell (CSC) features, and specifically expressed SOX6, which was associated with poor prognosis. We also found that ACKR1+ endothelial cells and inflammatory cancer-associated fibroblasts (iCAFs) were more enriched in UCU, which may promote pathogenesis. While ESM1+ endothelial cells may more actively participate in UCB and UCRP tumorigenesis by promoting angiogenesis. Additionally, CD8 + effector T cells were more enriched in UCU and UCRP patients, while Tregs were mainly enriched in UCB tumors. C1QC+ macrophages and LAMP3+ dendritic cells were more enriched in UCB, which is closely related to the formation of the heterogeneous immunosuppressive microenvironment. Furthermore, we found strong interactions between iCAFs, EP9, and Endo_ESM1, and different degrees of activation of the FGF-FGFR3 axis and immune checkpoint pathway were observed in different UC subtypes. Our study elucidated the cellular heterogeneity and the components of the microenvironment in UC arising from the upper and lower urinary tracts and provided novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a7771d79701d4fbee36981bff69a8e0ebdd585" target='_blank'>
              Single-Cell RNA Sequencing Reveals the Tumor Heterogeneity and Immunosuppressive Microenvironment in Urothelial Carcinoma.
              </a>
            </td>
          <td>
            Tianqi Lyu, Kerong Wu, Yincong Zhou, Tong Kong, Lin Li, Kaizhe Wang, Pan Fu, Pengyao Wei, Ming Chen, Jianping Zheng
          </td>
          <td>2024-12-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Rhabdomyosarcoma (RMS) is a highly aggressive soft tissue sarcoma, affecting children and adolescents, with poor prognosis in some patient groups. Better therapeutic regimens and preclinical models to test them in are needed. Multi-receptor tyrosine kinase inhibitors (MRTKIs) are licensed for adult indications and explored in the clinic in sarcoma patients. The MRTKI Regorafenib is currently assessed in the relapse setting in patients with RMS (NCT04625907). Reliable biomarkers of response for MRTKIs are lacking. MRTKIs act not only against the cancer cell, but also the supporting stroma, particularly the vasculature. The embryonic zebrafish is translucent and allows assessment of this interaction with high-throughput in vivo imaging. Methods A new preclinical embryo zebrafish xenograft model was developed using Tg(flk1:GFP) (blood vessel reporter) transgenic zebrafish embryos inoculated in the yolk with fluorescently labelled cells from 7 different RMS cell lines (fusion-positive (FP): Rh4, Rh30, Rh41, RMS-01, fusion-negative (FN): RD, JR1, SMS-CTR), and patient-derived cells IC-pPDX-104 at 50 hours post-fertilization and incubated at 34°C for up to 70 hours. Xenografts and vessel beds were imaged and analysed using custom FIJI pipelines. MRTKIs regorafenib and infigratinib were used at a concentration of 0.1uM added to the fish water 4 hours post cell inoculation. Pro-angiogenic growth factors VEFG-A, FGF-2 and PDGF-BB were measured in conditioned media of each cell line. Results All 7 RMS cell lines and the patient-derived cells engrafted with tumour burden assessment by fluorescent imaging and direct cell counting indicating adequate growth and high cell viability during the observation period. RMS tumours induced neo-vascularisation towards the tumour and increased density of proximal vessel beds. MRTKI treatment revealed a greater tumour-intrinsic sensitivity of FP cells, but identified a significant blockade of neo-vascularisation across all RMS lines, with regorafenib response correlated with secretion of VEGF-A. Conclusion We have developed an embryonic zebrafish xenograft model of RMS, which allows assessment of tumour growth, vascularisation initiation and therapeutic responses to clinically relevant MRTKIs. The identification of VEGF-A secretion as potential biomarker for Regorafenib response and the separation of therapeutic effects on tumour growth and neovascularisation suggests additional value of our model for response prediction to MRTKIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35ede6334422366b807e1f80f1fb08f52de1a53f" target='_blank'>
              A dual readout embryonic zebrafish xenograft model of rhabdomyosarcoma to assess clinically relevant multi-receptor tyrosine kinase inhibitors
              </a>
            </td>
          <td>
            J. Wragg, Emma L. Gray, Rui Monteiro, Jo R. Morris, Andrew D Beggs, Ferenc Müller, S. Gatz
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Idiopathic pulmonary fibrosis (IPF) is a progressive lung disease characterized by repetitive alveolar injuries with excessive deposition of extracellular matrix (ECM) proteins. A crucial need in understanding IPF pathogenesis is identifying cell types associated with histopathological regions, particularly local fibrosis centers known as fibroblast foci. To address this, we integrated published spatial transcriptomics and single-cell RNA sequencing (scRNA-seq) transcriptomics and adopted the Query method and the Overlap method to determine cell type enrichments in histopathological regions. Distinct fibroblast cell types are highly associated with fibroblast foci, and transitional alveolar type 2 and aberrant KRT5-/KRT17+ (KRT: keratin) epithelial cells are associated with morphologically normal alveoli in human IPF lungs. Furthermore, we employed laser capture microdissection-directed mass spectrometry to profile proteins. By comparing with another published similar dataset, common differentially expressed proteins and enriched pathways related to ECM structure organization and collagen processing were identified in fibroblast foci. Importantly, cell type enrichment results from innovative spatial proteomics and scRNA-seq data integration accord with those from spatial transcriptomics and scRNA-seq data integration, supporting the capability and versatility of the entire approach. In summary, we integrated spatial multi-omics with scRNA-seq data to identify disease-associated cell types and potential targets for novel therapies in IPF intervention. The approach can be further applied to other disease areas characterized by spatial heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947f84484c40c13b0563bade74b2ae0f8c1ec3ac" target='_blank'>
              Novel Integration of Spatial and Single-Cell Omics Data Sets Enables Deeper Insights into IPF Pathogenesis
              </a>
            </td>
          <td>
            Fei Wang, Liang Jin, Xue Wang, Baoliang Cui, Yingli Yang, Lori Duggan, Annette Schwartz Sterman, Sarah M. Lloyd, L. Hazelwood, Neha Chaudhary, Bhupinder Bawa, Lucy A. Phillips, Yupeng He, Yu Tian
          </td>
          <td>2025-01-13</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b351a91b1b339944d457e7db859962fb6728ec9" target='_blank'>
              Patient-derived xenografts from circulating cancer stem cells as a preclinical model for personalized pancreatic cancer research
              </a>
            </td>
          <td>
            Benedikt J Wagner, Andreas Ettner-Sitter, Nicolas A Ihlo, Merle Behr, Sebastian Koelbl, Stefan M Brunner, Florian Weber, Bettina M Rau, Hans J Schlitt, Christoph Brochhausen, Rebecca Schoenmehl, Annalena Artinger, D. Schott, M. Pizon, Katharina Pachmann, Thiha Aung, Silke Haerteis, Christina Hackl
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors exhibit extensive genomic, epigenomic, and transcriptional diversity, with significant intratumoral heterogeneity, complicating standard treatment approaches involving radiation (RT) and the DNA-alkylating agent temozolomide (TMZ). In this study, we employed an integrative multi-omics approach, including targeted proteomics, transcriptomics, genomics, and DNA methylation profiling, to investigate the response of a representative panel of GBM patient-derived cancer stem cells (CSCs) to astrocytic differentiation and RT and TMZ treatments. Differentiated CSC progenies retained the expression of key stemness genes and survival pathways, while activating the BMP-Smad signaling pathway and upregulating extracellular matrix components. This was associated with increased resistance to TMZ, though not to RT, across all models. We identified TP53 status as a critical determinant of transcriptional response to both RT and TMZ, which was also modulated by the differentiation state and treatment modality in wildtype (wt) p53 GBM cells. Both mutant and wt p53 models exhibited significant activation of the DNA-damage associated interferon (IFN) response in CSCs and differentiated cells, implicating this pathway in the GBM response to therapy. We observed that activation of NF-κB was positively correlated with the levels of O-6-methylguanine-DNA methyltransferase (MGMT) protein, a direct DNA repair enzyme leading to TMZ resistance, regardless of MGMT promoter methylation status, further supporting the clinical potential for inhibition of NF-kB signaling in GBM treatment. Our integrative analysis of the impact of GBM cell developmental states, in the context of genomic and molecular diversity of patient-derived models, provides valuable insights for pre-clinical studies aimed at optimizing treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a242e45dbcf0e04c69a3eb415e835a6082edb4c" target='_blank'>
              Impact of developmental state, p53 status, and interferon signaling on glioblastoma cell response to radiation and temozolomide treatment
              </a>
            </td>
          <td>
            A. Berezovsky, O. Nuga, Indrani Datta, Kimberly Bergman, Thais S Sabedot, Katherine Gurdziel, S. Irtenkauf, L. Hasselbach, Yuling Meng, Claudius Mueller, Emanuel F. Petricoin, Stephen Brown, N. Purandare, Sidhesh Aras, Tom Mikkelsen, Laila M Poisson, H. Noushmehr, Douglas Ruden, A. deCarvalho
          </td>
          <td>2025-02-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc019fa42801c8582b11a460b1609019f96845de" target='_blank'>
              A minimal gene set characterizes TIL specific for diverse tumor antigens across different cancer types
              </a>
            </td>
          <td>
            Zhen Zeng, Tianbei Zhang, Jiajia Zhang, Shuai Li, Sydney C Connor, Boyang Zhang, Yimin Zhao, Jordan Wilson, Dipika Singh, R. Kulikauskas, C. Church, T. Pulliam, Saumya Jani, P. Nghiem, S. Topalian, Patrick M Forde, Drew M Pardoll, Hongkai Ji, Kelli N. Smith
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90df80a01384ee300b159e30e515e05880476d0f" target='_blank'>
              Artificial intelligence-based spatial analysis of tertiary lymphoid structures and clinical significance for endometrial cancer
              </a>
            </td>
          <td>
            Haruka Suzuki, Kohei Hamada, J. Hamanishi, Akihiko Ueda, R. Murakami, Mana Taki, Rin Mizuno, Koichi Watanabe, Hanako Sato, Y. Hosoe, Hiroaki Ito, Koji Yamanoi, Hiroyuki Yoshitomi, N. Kakiuchi, Ken Yamaguchi, N. Matsumura, Seishi Ogawa, Hideki Ueno, Masaki Mandai
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Cell and tissue movement in development, cancer invasion, and immune response relies on chemical or mechanical guidance cues. In many systems, this behavior is locally guided by self-generated signaling gradients rather than long-range, pre-patterned cues. However, how heterogeneous mixtures of cells interact non-reciprocally and navigate through self-generated gradients remains largely unexplored. Here we introduce a theoretical framework for the self-organized chemotaxis of heterogeneous cell populations. We find that relative chemotactic sensitivity of cell populations controls their long-time coupling and co-migration dynamics, with boundary conditions such as external cell- and attractant reservoirs substantially influencing the migration patterns. We further predict an optimal parameter regime that leads to robust and colocalized migration. We test our theoretical predictions with in vitro experiments demonstrating the co-migration of different immune cell populations, and quantitatively reproduce observed migration patterns under wild-type and perturbed conditions. Interestingly, immune cell co-migration appears close to the predicted optimal regime. Finally, we incorporate mechanical interactions into our framework, revealing a phase diagram that illustrates non-trivial interplays between chemotactic and mechanical non-reciprocity in driving collective migration. Together, our findings suggest that self-generated chemotaxis is a robust strategy for emergent multicellular navigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f57ee1bc013b72202f77f0ea69bbd4c759c18c" target='_blank'>
              Self-generated chemotaxis of mixed cell populations
              </a>
            </td>
          <td>
            Mehmet Can Uçar, Zane Alsberga, Jonna Alanko, M. Sixt, É. Hannezo
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f574d380fa01f497eea49582767fd01ff18dddc" target='_blank'>
              Characterization of stem cell landscape and identification of stemness-relevant prognostic gene signature to aid immunotherapy in breast cancer
              </a>
            </td>
          <td>
            Xiaozhou Yang, Xiaojun Yang, Haili Tang, Xin Chen, Jiangang Wang, Huadong Zhao
          </td>
          <td>2025-01-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon D. Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It is becoming more broadly accepted that human-based models are needed to better understand the complexities of the human nervous system and its diseases. The recently developed human brain organotypic culture model is one highly promising model that requires the involvement of neurosurgeons and neurosurgical patients. Studies have investigated the electrophysiological properties of neurons in such ex vivo human tissues, but the maintenance of other cell types within explanted brain remains largely unknown. Here, using single-nucleus RNA sequencing, we systematically evaluate the transcriptional identities of the various cell types found in six patient samples after fourteen days in culture (83,501 nuclei from day 0 samples and 45,738 nuclei from day 14 samples). We used two pediatric temporal lobectomy samples, an adult frontal cortex sample, two IDH wild-type glioblastoma samples, and one medulloblastoma sample. We found remarkably high correlations of day 14 transcriptional identities to day 0 tissue, especially in tumor cells (r = 0.90 to 0.93), though microglia (r = 0.86), oligodendrocytes (r = 0.80), pericytes (r = 0.77), endothelial cells (r = 0.78), and fibroblasts (r = 0.76) showed strong preservation of their transcriptional profiles as well. Astrocytes and excitatory neurons showed more moderate preservation (r = 0.66 and 0.47, respectively). Because the main difficulty with organotypic brain cultures is the acquisition of human tissue, which is readily available to neurosurgeons, this model is easily accessible to neurosurgeon-scientists and neurosurgeons affiliated with research laboratories. Broad uptake of this more representative model should prompt advances in our understanding of many uniquely human diseases, lead to more reliable clinical trial performance, and ultimately yield better therapies for our patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d272183d3f81493a970c9067c67dfd502ee3e7d3" target='_blank'>
              Cell type transcriptional identities are maintained in cultured ex vivo human brain tissue
              </a>
            </td>
          <td>
            JP McGinnis, Joshua Ortiz-Guzman, Sai Mallannagari, Maria Camila Guevara, B. D. Belfort, Suyang Bao, Snigdha Srivastava, Maria Morkas, Emily Ji, K. Katlowitz, Angela Addison, Evelyne K. Tantry, Melissa M. Blessing, Carrie Mohila, N. Gadgil, Samuel G. McClugage, David F. Bauer, William E. Whitehead, G. Aldave, Omar Tanweer, Naser Jaleel, Ali Jalali, Akash J. Patel, S. Sheth, Howard L. Weiner, Shankar Gopinath, Ganesh Rao, A. Harmanci, Daniel Curry, Benjamin R. Arenkiel
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74606bb02b8c93f71db20ba35bf801348005a77" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Yu-Jia Shiah, Takafumi N. Yamaguchi, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, A. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Personalized precision medicine (PPPM) in cancer immunology and oncology is a rapidly advancing field with significant potential. Gliomas, known for their poor prognosis, rank among the most lethal brain tumors. Despite advancements, there remains a critical need for precise, individualized treatment strategies.We conducted a comprehensive analysis of RNA-seq and microarray data from the TCGA and GEO databases, supplemented by single-cell RNA sequencing (scRNA-seq) data from glioma patients. By integrating single-cell sequencing analysis with foundational experiments, we investigated the molecular variations and cellular interactions within neural glioma cell subpopulations during tumor progression.Our single-cell sequencing analysis revealed distinct gene expression patterns across glioma cell subpopulations. Notably, differentiation trajectory analysis identified NUSAP1 as a key marker for the terminal subpopulation. We found that elevated NUSAP1 expression correlated with poor prognosis, prompting further investigation of its functional role through both cellular and animal studies.NUSAP1-based risk models hold potential as predictive and therapeutic tools for personalized glioma treatment. In-depth exploration of NUSAP1’s mechanisms in glioblastoma could enhance our understanding of its response to immunotherapy, suggesting that targeting NUSAP1 may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5625f39e4872fec67e5df1a449a03f506dae3b79" target='_blank'>
              Single-cell sequencing elucidates the mechanism of NUSAP1 in glioma and its diagnostic and prognostic significance
              </a>
            </td>
          <td>
            Meng-Yu Zhao, Zhao-Lei Shen, Hongzhen Dai, Wan-Yan Xu, Li-Na Wang, Yu- Gu, Jie-Hui Zhao, Tian-Hang Yu, Cun-Zhi Wang, Jia-feng Xu, Guan-Jun Chen, Dong-Hui Chen, Wen-Ming Hong, Fan Zhang
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Racial disparities in cancer outcomes are well documented across tumor types. For patients with breast cancer, Black women are more likely to present with more aggressive molecular features and more likely to die from disease, even after accounting for those features. Recent efforts have been aimed at developing translational model systems for precision medicine strategies, and a major focus has been on patient-derived organoids. Organoids allow for robust in vitro experimental platforms, including drug and CRISPR screens while maintaining more complex cancer and tumor microenvironment subpopulations than cell lines. For results that are broadly translationally relevant, it is important that cancer models are derived from the spectrum of human disease and humans with disease. In this issue of Cancer Research, Madorsky Rowdo and colleagues derive breast cancer organoids from patients with African ancestry and use CRISPR-Cas9 screens to identify novel therapeutic vulnerabilities. These findings demonstrate the promise of representative cancer model systems to facilitate discoveries that are most likely to translate to improved therapy for all patients. See related article by Madorsky Rowdo et al., p. 551.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af090730ccebba2da2c3e50d4cbf6a340cfee095" target='_blank'>
              In Search of Representative Translational Cancer Model Systems.
              </a>
            </td>
          <td>
            Hannah E. Trembath, Philip M. Spanheimer
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadc098eba1b42a6711d8bde36ca00a426896d86" target='_blank'>
              KMO-driven metabolic reconfiguration and its impact on immune cell infiltration in nasopharyngeal carcinoma: a new avenue for immunotherapy
              </a>
            </td>
          <td>
            Nijun Chen, Yuan Zong, Chen Yang, Lei Li, Yang Yi, Jiawen Zhao, Xiaoyu Zhao, Xianfei Xie, Xingmei Sun, Ning Li, Liting Jiang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite many efforts to understand the molecular mechanisms of pancreatic ductal adenocarcinoma (PDAC) treatment resistance, there is still no reliable method for selecting patients who could benefit from standard pharmacological treatment. Here, four PDAC patient-derived xenografts (PDAC-PDXs) with different responses to gemcitabine plus nab-paclitaxel (nanoparticle albumin-bound paclitaxel) were studied to dissect the contribution of both tumor and host microenvironment to treatment response. PDAC-PDXs transplanted into the pancreas of immunodeficient mice retained the main genetic and histopathological characteristics of the original human tumors, including invasiveness and desmoplastic reaction. Response to chemotherapy was associated with a specific 294 stroma gene signature and was not due to the intrinsic responsiveness of tumor cells or differences in drug delivery. Human dataset analysis validated the expression of the 294 stroma gene signature in PDAC clinical samples, confirming PDAC-PDXs as a useful tool to study the biology of tumor-host interactions and to test drug efficacy. In summary, we identified a stroma gene signature that differentiates PDAC-PDXs that are responsive to gemcitabine plus Nab-paclitaxel treatment from those that are not, confirming the active role of the tumor microenvironment in the drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d373e8c04b13d6d54848ed05cf8cc7a31edd7b1" target='_blank'>
              Stroma gene signature predicts responsiveness to chemotherapy in pancreatic ductal adenocarcinoma patient-derived xenograft models.
              </a>
            </td>
          <td>
            A. Anastasia, Laura Formenti, P. Ostano, L. Minoli, A. Resovi, L. Morosi, Claudia Fioravanti, E. Micotti, C. Matteo, Eugenio Scanziani, Giovanna Chiorino, Raffaella Giavazzi, C. Ghilardi, D. Belotti
          </td>
          <td>2025-02-04</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 The reactivation of neurodevelopmental programs in cancer highlights parallel biological processes that occur in both normal development and brain tumors. Achieving a deeper understanding of how dysregulated developmental factors play a role in the progression of brain tumors is therefore crucial for identifying potential targets for therapeutic interventions. Single-cell RNA sequencing (scRNA-Seq) provides an opportunity to understand how developmental programs are dysregulated and reinitiated in brain tumors at single-cell resolution. The aim of this study is to identify developmental origins of brain tumors using scRNA-Seq data.



 Here, we introduce COORS (Cell Of ORigin like CellS), a computational tool trained on developmental human brain single-cell datasets that annotates “developmental-like” cell states in brain tumors. COORS leverages cell type-specific multilayer perceptron models and incorporates a developmental cell type tree that reflects hierarchical relationships and models cell type probabilities.



 Applying COORS to various brain cancer datasets, including medulloblastoma (MB), glioma, and diffuse midline glioma (DMG), we identified developmental-like cells that represent putative cells of origin in these tumors. Our method provides both cell of origin classification and cell age regression, offering insights into the developmental cell types of tumor subgroups. COORS identified outer radial glia (oRG) developmental cells within IDHWT glioma cells whereas oligodendrocyte precursor cells (OPCs) and neuronal-like cells in IDHMut. Interestingly, IDHMut subgroup cells that map to OPC show bimodal distributions, that are both early and late weeks in development. Furthermore, COORS offers a valuable resource by providing novel markers linked to developmental states within of MB, glioma and DMG tumor subgroups.



 Our work adds to our cumulative understanding of brain tumor heterogeneity and helps pave the way for tailored treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e403b1c16cfb5c2e03317b6d1828fc8f0bdaf21" target='_blank'>
              Inferred Developmental Origins of Brain Tumors from Single Cell RNA-Sequencing Data
              </a>
            </td>
          <td>
            Su Wang, R. Curry, Malcolm F McDonald, Hyun Yong Koh, Anders W Erickson, Claudia L Kleinman, Michael D. Taylor, Ganesh Rao, Benjamin Deneen, Arif O. Harmanci, A. Harmanci
          </td>
          <td>2025-01-23</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Genetically engineered mouse models (GEMMs) are instrumental for modelling local and systemic features of complex diseases, such as cancer. Non-invasive, longitudinal cell detection and monitoring in tumors, metastases and/or the micro-environment is paramount to achieve a better spatiotemporal understanding of cancer progression and to evaluate therapies in preclinical studies. Bioluminescent and fluorescent reporters marking tumor cells or their microenvironment are valuable for non-invasive cell detection and monitoring in vivo. Here, we report the generation of a dual reporter allele allowing simultaneous bioluminescence and fluorescence detection of cells that have undergone Cre-Lox recombination in mice. The single copy knock-in allele in the permissive collagen I locus was evaluated in the context of several cancer GEMMs, where Cre expression was achieved genetically or by ectopic virus-mediated delivery. The new reporter allele was also combined with gene-targeted alleles widely used in bone, prostate, brain and pancreas cancer research, as well as with alleles inserted into the commonly used Rosa26 and collagen I loci. This allele is, therefore, a useful addition to the portfolio of reporters to help advance preclinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec9919bb5c906c59e2fc5d046908d89b5d4d491" target='_blank'>
              A new effLuc/Kate dual reporter allele for tumor imaging in mice
              </a>
            </td>
          <td>
            L. Bakiri, M. Tichet, Carolina Marques, M. Thomsen, Elizabeth Allen, Stefanie Stolzlechner, Ke Cheng, Kazuhiko Matsuoka, Massimo Squatrito, Douglas Hanahan, E. Wagner
          </td>
          <td>2025-01-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c81eee6b668168bd971313aa0fe51bd578f98e" target='_blank'>
              Meta-analyses of mouse and human prostate single-cell transcriptomes reveal widespread epithelial plasticity in tissue regression, regeneration, and cancer
              </a>
            </td>
          <td>
            Luis C. Aparicio, Laura Crowley, John R. Christin, Caroline J. Laplaca, H. Hibshoosh, Raul Rabadan, Michael M. Shen
          </td>
          <td>2025-01-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Despite therapeutic advancements, the cellular and molecular heterogeneity of UC hinders effective treatment. We applied machine learning-based approaches to whole scanned images (WSI) of H&E stained slides prepared from formalin-fixed, paraffin embedded intestinal mucosal biopsies from the etrolizumab clinical trials, aiming to identify cellular interactions and molecular signatures that define disease subtypes and treatment responses. We hypothesised that the cellular-basis of non-response could be derived by linking cell-cell relationships with paired transcriptomic profiles.



 We analysed 2297 WSI paired with molecular data from 1272 UC patients from the etrolizumab PhIII clinical trial including various cohorts, treatment arms, and timepoints. Sections were segmented using a convolutional neural network, extracting 8 cell types and their coordinates. For each WSI, ~500 interaction features from an overlaid network were engineered using our developed software, CellMaskProfiler. The topology of the cell-cell interaction networks allowed clustering of WSI with similar profiles. Cell count features and clinical data enabled the annotation of clusters. Paired molecular data was used to perform differential expression.



 Using the cell-cell topology, we identified 7 distinct UC subtypes characterised by specific cellular interactions, molecular profiles and clinical features resolving into, intact, and neutrophil, eosinophil, plasma, or lymphocyte dominated profiles. WSI with a high proportion of epithelial cells and more connections between epithelial cells types were strongly associated with lower Robarts Histology Index, indicating milder disease or intact epithelium, while strong interaction with neutrophils showed the opposite effect. Looking at different treatment arms allowed to relate baseline biopsy WSI characteristics with probability of treatment response highlighting the probability of response based on initial WSI subtype. Using the cellular topology allowed to characterise the transition between diseased and healthier clusters. Adjusting for cellularity effects and comparing baseline data for response at induction revealed genes characterising response. Looking at transitions between clusters revealed similar genes.



 This study introduces a novel feature engineering approach to unravel the cellular and molecular complexity of WSI in UC. The findings underscore the significance of cellular interactions in UC pathology and provide new insights into the molecular mechanisms underlying different disease states and demonstrates that the structural topology (form) derived from WSI can be effectively linked to the transcriptomic profiles (function) to define probability of remission.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b4d67e4560bf9b521f01dabd53045b03a068c1" target='_blank'>
              OP08 Histology-derived cell-cell interaction networks, linked to transcriptome profiling, define path to resolution in UC
              </a>
            </td>
          <td>
            D. Dylus, A. Paun, A. J. Mueller-Breckenridge, A. D. Christ, A. Scherl, M. Leipner
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f63f6a23a3983ad60b3216e3e5876ec300a9938" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7cfa28f38350ee5f17ed88d743b1b33ba38e828" target='_blank'>
              Investigating the effects of stereotactic body radiation therapy on pancreatic tumor hypoxia and microvasculature in an orthotopic mouse model using intravital fluorescence microscopy
              </a>
            </td>
          <td>
            Timothy Samuel, Sara Rapic, Patricia E Lindsay, Ralph S. Dacosta
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis.
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, A. L. Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c43b2e6ebcea9dd9d569a152511cd9284977827" target='_blank'>
              Establishment and characterization of ovarian clear cell carcinoma patient-derived xenografts
              </a>
            </td>
          <td>
            Joseph J. Caumanns, Shang Li, G. Meersma, E. Duiker, A. van der Zee, G. Wisman, Steven de Jong
          </td>
          <td>2025-02-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf42e4d5c5ce1d7e5ee6b6939079dd4741136db7" target='_blank'>
              Revolutionizing ESCC prognosis: the efficiency of tumor-infiltrating immune cells (TIIC) signature score
              </a>
            </td>
          <td>
            Haixia Wang, Shaowei Ma, Zixin Yang, Ren Niu, Haiyong Zhu, Shujun Li, Shao Gao, Zhirong Li, Yanhua Tian
          </td>
          <td>2025-01-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cell therapy has emerged as a promising treatment for hematological malignancies, offering a targeted approach to cancer treatment. Understanding the complexities of CAR-T cell therapy within solid tumors poses challenges due to the intricate interactions within the tumor microenvironment. Mathematical modeling may serve as a valuable tool to unravel the dynamics of CAR-T cell therapy and improve its effectiveness in solid tumors. This study aimed to investigate the impact of spatial aspects in CAR-T therapy of solid tumors, utilizing cellular automata for modeling purposes. Our main objective was to deepen our understanding of treatment effects by analyzing scenarios with different spatial distributions and varying the initial quantities of tumor and CAR-T cells. Tumor geometry significantly influenced treatment efficacy in-silico, with notable differences observed between tumors with block-like arrangements and those with sparse cell distributions, leading to the concept of immune suppression due to geometrical effects. This research delves into the intricate relationship between spatial dynamics and the effectiveness of CAR-T therapy in solid tumors, highlighting the relevance of tumor geometry in the outcome of cellular immunotherapy treatments. Our results provide a basis for improving the efficacy of CAR-T cell treatments by combining them with other ones reducing the density of compact tumor areas and thus opening access ways for tumor killing T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., Departamento de Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell sequencing technology provides apparent advantages in cell population heterogeneity, allowing individuals to better comprehend tissues and organs. Sequencing technology is currently moving beyond the standard transcriptome to the single-cell level, which is likely to bring new insights into the function of breast cells. In this study, we examine the primary cell types involved in breast development, as well as achievements in the study of scRNA-seq in the microenvironment, stressing the finding of novel cell subsets using single-cell approaches and analyzing the problems and solutions to scRNA-seq. Furthermore, we are excited about the field’s promising future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5fc2ec140357aabbc1437326fa1b4a534856413" target='_blank'>
              Advancements in the Application of scRNA-Seq in Breast Research: A Review
              </a>
            </td>
          <td>
            Zhenyu Zhang, Xiaoming Ma, Yongfu La, Xian Guo, M. Chu, P. Bao, Ping Yan, Xiaoyun Wu, Chunnian Liang
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive malignancy with a dismal prognosis and limited therapeutic options. Adoptive cell therapy, which involves isolating and activating a patient's own immune cells, such as tumor-infiltrating lymphocytes (TILs), before re-infusing them, represents a promising experimental approach. However, techniques for adoptive cell transfer in preclinical pancreatic cancer models are not well established. Here, we describe a detailed protocol for adoptive cell therapy using TILs from a syngeneic pancreatic cancer mouse model. The procedure involves implanting live or irradiated mouse pancreatic cancer cells in fluorescence-labeled reporter mice to initiate immune cell influx, then isolating lymphocytes from primary tumors via flow cytometry sorting and/or activating and expanding tumor-reactive T cells ex vivo, and adoptively transferring these activated T cells intraperitoneally into tumor-bearing mice, followed by interleukin-2 administration. Bioluminescent tumor imaging allows for longitudinal monitoring of orthotopic tumor growth and response to therapy, especially evaluating the tumor-specific cytotoxic effects. This approach recapitulates the logistics involved in developing adoptive cell transfer therapies for pancreatic cancer patients. The results demonstrate enhanced antitumor efficacy of adoptively transferred tumor-reactive T cells compared to irrelevant lymphocyte controls. This versatile methodology enables the in vivo study of adoptive immunotherapy in pancreatic cancer as well as the optimization of cell processing parameters and combination treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab513b58164470baa06a3acdd87be8779d1f8715" target='_blank'>
              Flow Cytometry-Based Isolation and Therapeutic Evaluation of Tumor-Infiltrating Lymphocytes in a Mouse Model of Pancreatic Cancer.
              </a>
            </td>
          <td>
            Xiangqin Zhao, Xinbo Gao, Wenwen Yu, Yi Lu, Xuesong Li, Lu Tan, Xiubao Ren, Yifei Wang, Weijie Song, Jihui Hao, Ying Ma
          </td>
          <td>2025-01-17</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epithelial cell collectives migrate through tissue interfaces and crevices to orchestrate development processes, tumor invasion, and wound healing. Naturally, the traversal of cell collective through confining environments involves crowding due to narrowing spaces, which seems tenuous given the conventional inverse relationship between cell density and migration. However, the physical transitions required to overcome such epithelial densification for migration across confinements remain unclear. Here, in a system of contiguous microchannels of varying confinements, we show that epithelial (MCF10A) monolayers accumulate higher cell density and undergo fluid-like shape transitions before entering narrower channels. However, overexpression of breast cancer oncogene ErbB2 did not require such accumulation of cell density to migrate across matrix confinement. While wild-type MCF10A cells migrated faster in narrow channels, this confinement sensitivity was reduced after +ErbB2 mutation or with constitutively active RhoA. This physical interpretation of collective cell migration as density and shape transitions in granular matter could advance our understanding of complex living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4a64576188154027b2ea1715663784bbe17d31" target='_blank'>
              Gradients in cell density and shape transitions drive collective cell migration into confining environments
              </a>
            </td>
          <td>
            Wan-Jung Lin, Hongsheng Yu, Amit Pathak
          </td>
          <td>2024-12-30</td>
          <td>Soft Matter</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c2a353f69d7baf419611ff4e3df0af6476afc0" target='_blank'>
              snRNA-seq of long-preserved FFPE samples from colorectal liver metastasis lesions with diverse prognoses
              </a>
            </td>
          <td>
            Hongyu Chen, Xiang Zhang, Qing Cheng, Xiner Shen, Linghui Zeng, Yongcheng Wang, Longjiang Fan, Weiqin Jiang
          </td>
          <td>2024-12-26</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND/AIM
Cholangiocarcinoma (CCA) is an aggressive cancer with limited effective chemotherapy and targeted therapy options. Existing cell lines and animal models only partially mimic the characteristics of the tumor, highlighting the need for more effective models to study the biology of cancer and drug responses. This study aimed to establish and characterize patient-derived xenograft (PDX) models of CCA.


MATERIALS AND METHODS
Tumor samples from 40 CCA patients were subcutaneously implanted into non-obese diabetic/ShiJic-severe combined immunodeficiency Jcl mice to establish patient-derived xenograft (PDX) models. Successfully engrafted tumors were passaged across three generations. Histological features were analyzed using H&E staining and immunohistochemistry for cytokeratin-19, cytokeratin-7, heppar-1 and arginase-1. Whole exome sequencing (WES) was performed to assess genetic stability and identify somatic mutations.


RESULTS
A total of eight PDX models were successfully created, representing 20% of the total cases. Histological comparisons showed strong concordance between patient tumors and their corresponding xenografts in the eight PDX models across generations. WES analysis confirmed the genetic stability of the PDX models, with significant somatic mutations identified in key genes such as TTN, MUC12, ARID1A, TP53, and RNF43.


CONCLUSION
The CCA PDX model could reflect both the histological and genetic characteristics of the original tumors, providing a valuable tool for studying tumor biology and serving as a preclinical model to develop personalized treatment options for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89744e786f8b599dcfd612b0ab51b9c0dbd2e620" target='_blank'>
              Characterization of Patient-derived Xenograft Models of Liver Fluke-associated Cholangiocarcinoma: From Establishment to Molecular Profiling.
              </a>
            </td>
          <td>
            Hasaya Dokduang, Apiwat Jarernrat, Attapol Titapun, Sirinya Sitthirak, Sureerat Padthaisong, Yingpinyapt Kittirat, Sakkarn Sangkamanon, P. Sa-Ngiamwibool, Arporn Wangwiwatsin, P. Klanrit, N. Namwat, A. Jusakul, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-01-31</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="There is an urgent need of precision medicine for ovarian cancer patients to identify patients who respond to chemotherapy and PARP inhibitors, a therapy targeting homologous recombination deficiency (HRD). Here we established a panel of 37 long-term patient-derived tumor organoids (PDTO) models of various histological subtypes from 224 patients and demonstrated that they mimic the histological and molecular characteristics of original tumors. Screening of chemotherapeutic drugs showed that PDTO exhibit heterogeneous responses, and that response of PDTO from high-grade serous ovarian carcinoma to carboplatin recapitulated patient response to first-line treatment. Additionally, the detection of HRD phenotype of PDTO using functional assay was associated with the results of the HRD test Genomic Instability Scar (GIScar). Although larger-scale investigations are needed to confirm the predictive potential of PDTO, these results provide further evidence of the potential interest of ovarian PDTO for functional precision medicine even if many challenges remain to be addressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9373b0fdccc45bac208c5469698121e6a4da07c" target='_blank'>
              Long-term patient-derived ovarian cancer organoids closely recapitulate tumor of origin and clinical response
              </a>
            </td>
          <td>
            L. Thorel, E. Dolivet, Pierre-Marie Morice, R. Florent, J. Divoux, M. Perréard, L. Lecouflet, G. Desmartin, C. Marde Alagama, F. Giffard, A. Leconte, J. Lequesne, B. Clarisse, M. Briand, A. Traore, C. Villenet, JP. Meneboo, G. Babin, L. Gaichies, S. Martin-Françoise, JF. Le Brun, R. Rouzier, E. Brotin, C. Denoyelle, N. Vigneron, R. Leman, D. Vaur, L. Castéra, C. Blanc-Fournier, N. Elie, B. Plancoulaine, F. Joly, M. Meryet-Figuiere, M. Figeac, L. weiswald, L. Poulain
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K. Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="The mammary microenvironment has been shown to suppress tumor progression by redirecting cancer cells to adopt a normal mammary epithelial progenitor fate in vivo. However, the mechanism(s) by which this alteration occurs has yet to be defined. Here, we test the hypothesis that mitochondrial transfer from normal mammary epithelial cells to breast cancer cells plays a role in this redirection process. We evaluate mitochondrial transfer in 2D and 3D organoids using our unique 3D bioprinting system to produce chimeric organoids containing normal and cancer cells. We demonstrate that breast cancer tumoroid growth is hindered following interaction with mammary epithelial cells in both 2D and 3D environments. Furthermore, we show mitochondrial transfer occurs between donor mammary epithelial cells and recipient cancer cells primarily through tunneling nanotubes (TNTs) with minimal amounts seen from extracellular transfer of mitochondria, likely via extracellular vesicles (EVs). This organelle exchange results in various cellular and metabolic alterations within cancer cells, reducing their proliferative potential, and making them susceptible to microenvironmental control. Our results demonstrate that mitochondrial transfer contributes to microenvironmental redirection of cancer cells through alteration of metabolic and molecular functions of the recipient cancer cells. To the best of our knowledge, this is the first description of a 3D bioprinter-assisted organoid system for studying mitochondrial transfer. These studies are also the first mechanistic insights into the process of mammary microenvironmental redirection of cancer and provide a framework for new therapeutic strategies to control cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74a99d567922b268e37075173798f1b540e4d47" target='_blank'>
              Intercellular mitochondrial transfer contributes to microenvironmental redirection of cancer cell fate.
              </a>
            </td>
          <td>
            J. S. Bjerring, Yara Khodour, Emilee Anne Peterson, Patrick C. Sachs, Robert David Bruno
          </td>
          <td>2025-02-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
To meet their high energy needs, tumor cells undergo aberrant metabolic reprogramming. A tumor cell may expertly modify its metabolic pathways and the differential expression of the genes for metabolic enzymes. The physiological requirements of the host tissue and the tumor cell of origin mostly dictate metabolic adaptation. Ameloblastoma (AB) is a benign odontogenic tumor of epithelial origin. Due to its unrestricted growth potential, local aggressiveness, and high likelihood of recurrence, this condition poses a significant risk to the patient's health. This study aimed to characterize the metabolic heterogeneity at single-cell resolution of AB.


METHODS
Single-cell RNA sequencing (scRNA-seq) was performed on 17,284 cells from three AB donors. Bioinformatic analysis was used to examine differentially expressed genes, subtypes, and regulatory mechanisms when combined with odontogenic keratocyst scRNA-seq data. Based on metabolic pathway gene sets, the metabolic landscape of AB tumor cells was examined.


RESULTS
Using scRNA-seq, we discovered that AB tumor cells had substantial heterogeneity. The biggest contributor to tumor cell metabolic characteristics is determined to be variation in mitochondrial programming and glycolysis. Surprisingly, hypoxia corresponds with both oxidative phosphorylation and glycolysis activity in AB tumor cells at the single-cell level.


CONCLUSION
This study presents a computational framework for defining metabolism using single-cell expression data and identifies oxidative phosphorylation and glycolysis as critical components of metabolism for AB tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c116028597b1c5eaf72b4b7ed869064e15002ef" target='_blank'>
              Metabolic Analysis of Tumor Cells Within Ameloblastoma at the Single-Cell Level.
              </a>
            </td>
          <td>
            Rui-Fang Li, Yi Zhao, Qi-Wen Man
          </td>
          <td>2024-12-31</td>
          <td>Oral diseases</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding how intratumoral immune populations coordinate antitumor responses after therapy can guide treatment prioritization. We systematically analyzed an established immunotherapy, donor lymphocyte infusion (DLI), by assessing 348,905 single-cell transcriptomes from 74 longitudinal bone marrow samples of 25 patients with relapsed leukemia; a subset was evaluated by both protein- and transcriptome-based spatial analysis. In acute myeloid leukemia (AML) DLI responders, we identified clonally expanded ZNF683+ CD8+ cytotoxic T lymphocytes with in vitro specificity for patient-matched AML. These cells originated primarily from the DLI product and appeared to coordinate antitumor immune responses through interaction with diverse immune cell types within the marrow microenvironment. Nonresponders lacked this cross-talk and had cytotoxic T lymphocytes with elevated TIGIT expression. Our study identifies recipient bone marrow microenvironment differences as a determinant of an effective antileukemia response and opens opportunities to modulate cellular therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dc256708884cd08679466f87592f86695fa33d1" target='_blank'>
              Coordinated immune networks in leukemia bone marrow microenvironments distinguish response to cellular therapy.
              </a>
            </td>
          <td>
            K. Maurer, Cameron Y Park, Shouvik Mani, M. Borji, Florian Raths, Kenneth H Gouin, Livius Penter, Yinuo Jin, Jia Yi Zhang, Crystal Shin, James R Brenner, Jackson Southard, Sachi Krishna, Wesley S Lu, Haoxiang Lyu, Domenic Abbondanza, Chanell Mangum, Lars Rønn Olsen, Michael J Lawson, Martin M. Fabani, D. Neuberg, P. Bachireddy, Eli N Glezer, Samouil L. Farhi, Shuqiang Li, K. Livak, Jerome Ritz, Robert J. Soiffer, Catherine J. Wu, Elham Azizi
          </td>
          <td>2025-01-24</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood. Methods We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts. Results Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis. Conclusions DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            Tian-Hang Yu, Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu- Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aa22fb9f1f95d0dbd7d7c63c078e6faf7930b31" target='_blank'>
              Single-cell transcriptomics analysis reveals dynamic changes and prognostic signature in tumor microenvironment of PDAC
              </a>
            </td>
          <td>
            Yongsheng Li, Zhilong Ding, Tingxin Cheng, Yihuai Hu, Fei Zhong, Shiying Ren, Shiyan Wang
          </td>
          <td>2025-02-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly malignant brain tumor with limited treatment options. Polo-like kinase 2 (PLK2), a member of the polo-like kinase family, has been variably implicated in cancer, but its role in GBM has not been fully elucidated. We utilized RNA-seq data from multiple databases, including Gene Expression Omnibus (GEO), the Cancer Genome Atlas (TCGA), and the Chinese Glioma Genome Atlas (CGGA), and conducted experiments on human glioma cell lines to explore PLK2's expression and function. The effects of PLK2 overexpression on GBM cell viability, proliferation, migration, cell cycle, and apoptosis were assessed, and the tumorigenic potential of PLK2 was evaluated in a mouse model. PLK2 was consistently downregulated in GBM tissues compared to normal brain tissues across several datasets. Overexpression of PLK2 in GBM cell lines U87MG and U251 reduced their tumorigenic potential and enhanced cell cycle arrest and apoptosis, with significant reductions observed in apoptosis markers. Our findings suggest that PLK2 may potentially function as a tumor suppressor in GBM. Hence, PLK2 overexpression could potentially be leveraged as a therapeutic strategy to inhibit tumor progression and enhance apoptosis, providing new avenues for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528eb3ce98e6565c84a21860767b84902d6f9be9" target='_blank'>
              A potential tumor suppressor role of PLK2 in glioblastoma.
              </a>
            </td>
          <td>
            X. Xia, Peirui Wang, Hua Xiao, Qishan Ran, Yan Li, Shengtao Yao
          </td>
          <td>2025-02-10</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd76046253f83de84c6fe3da73a993a5cbb4c89" target='_blank'>
              3D Cell Culture Models as a Platform for Studying Tumor Progression, Testing Treatment Responses, and Discovering Biomarkers.
              </a>
            </td>
          <td>
            Peyda Korhan, E. Bagirsakci, Y. O. Islakoglu, N. Atabey
          </td>
          <td>2025-01-09</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies. Methods We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process. Results Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway. Conclusion Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The vast majority of cancers exhibit Somatic Copy Number Alterations (SCNAs)—gains and losses of variable regions of DNA. SCNAs can shape the phenotype of cancer cells, e.g. by increasing their proliferation rates, removing tumor suppressor genes, or immortalizing cells. While many SCNAs are unique to a patient, certain recurring patterns emerge as a result of shared selectional constraints or common mutational processes. To discover such patterns in a robust way, the size of the dataset is essential, which necessitates combining SCNA profiles from different cohorts, a non-trivial task. To achieve this, we developed CNSistent, a Python package for imputation, filtering, consistent segmentation, feature extraction, and visualization of cancer copy number profiles from heterogeneous datasets. We demonstrate the utility of CNSistent by applying it to the publicly available TCGA, PCAWG, and TRACERx cohorts. We compare different segmentation and aggregation strategies on cancer type and subtype classification tasks using deep convolutional neural networks. We demonstrate an increase in accuracy over training on individual cohorts and efficient transfer learning between cohorts. Using integrated gradients we investigate lung cancer classification results, highlighting SOX2 amplifications as the dominant copy number alteration in lung squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94ed57d2a2d247c1baada02015edbbd8a64f902" target='_blank'>
              CNSistent integration and feature extraction from somatic copy number profiles
              </a>
            </td>
          <td>
            Adam Streck, Roland F. Schwarz
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets. Materials and methods In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out. Results Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells. Conclusion This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Brain metastasis (BrM) has been a challenge for lung cancer treatment, but the mechanisms underlying lung cancer BrM remain elusive. This study aims to dissect cellular components and their spatial distribution in human BrM tumors of lung adenocarcinoma (LUAD) and identify potential therapeutic targets. Methods We performed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) on three LUAD BrMs, and validated our findings using public scRNA-seq data of 10 LUAD BrMs. Western blotting, quantitative real-time polymerase chain reaction (qRT-PCR) and functional experiments were employed for experimental studies. Results By combining scRNA-seq and ST, our analysis revealed the inter- and intra-tumoral heterogeneity of cellular components and their spatial localization within LUAD BrMs. Through RNA velocity and transcription factor (TF) regulatory activity analyses, we identified ATF3 as a potential regulator of the mesenchymal-epithelial transition (MET) program, which plays crucial roles in the colonization of tumor cells at metastatic sites. Furthermore, we demonstrated that knockdown of ATF3 significantly inhibited cancer cell proliferation while promoting cancer cell migration. Mechanistically, ATF3 knockdown could reverse the MET program. Additionally, we revealed that LGALS3/ANXA2-mediated cell-cell interaction between macrophage and tumor cells may also promote the MET program. Conclusions Our study provides a single-cell atlas of the cellular composition in BrM of LUAD and identifies ATF3 as a potential therapeutic target for BrM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e885c08703f3bf367c97f32c756f8659002f630" target='_blank'>
              Single-cell and spatial transcriptomics reveal a potential role of ATF3 in brain metastasis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Chaoliang Xu, Jingpiao Bao, Deshen Pan, Kehong Wei, Qing Gao, Weihong Lin, Yujie Ma, Meiqing Lou, Cheng Chang, Deshui Jia
          </td>
          <td>2025-01-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Inflammatory responses occur within the complex spatial context of tissues and organs, and many questions remain about how tissue structure and cellular communication shape their spatiotemporal dynamics. Here, we use a multiplexed RNA in situ hybridization approach, together with analytical tools, to study inflammatory gene expression in the larval zebrafish tailfin in response to a bath of lipopolysaccharide (LPS). We use this model system to address whether spatial structure emerges in the tissue response even absent the spatial variation introduced by a pathogen. We find that epithelial cells in the tailfin express several pro-inflammatory genes, and that across these genes, the uniform stimulus triggers a spatially non-uniform response. We use a graph-based spectral decomposition method to analyze its structure, and find that long modes dominate, creating zones of activation. Overall, these zones account for a majority of the variation in gene expression. Our results show that epithelial cells are important producers of pro-inflammatory effector molecules in this system, and that tissue induces spatial correlations even absent a structured input.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c13871b20a0df0f49db2baf60ca96fdc2c42238" target='_blank'>
              Spatially-structured inflammatory response in the presence of a uniform stimulus
              </a>
            </td>
          <td>
            Elizabeth R. Jerison, Nicolas Romeo, Stephen R. Quake
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="High-Grade Serous Ovarian Cancer (HGSOC) originates from fallopian tube (FT) precursors. However, the molecular changes that occur as precancerous lesions progress to HGSOC are not well understood. To address this, we integrated high-plex imaging and spatial transcriptomics to analyze human tissue samples at different stages of HGSOC development, including p53 signatures, serous tubal intraepithelial carcinomas (STIC), and invasive HGSOC. Our findings reveal immune modulating mechanisms within precursor epithelium, characterized by chromosomal instability, persistent interferon (IFN) signaling, and dysregulated innate and adaptive immunity. FT precursors display elevated expression of MHC-class I, including HLA-E, and IFN-stimulated genes, typically linked to later-stage tumorigenesis. These molecular alterations coincide with progressive shifts in the tumor microenvironment, transitioning from immune surveillance in early STICs to immune suppression in advanced STICs and cancer. These insights identify potential biomarkers and therapeutic targets for HGSOC interception and clarify the molecular transitions from precancer to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b491f761ec399a1258abdf4e675e3bc81f53e5e" target='_blank'>
              Multimodal Spatial Profiling Reveals Immune Suppression and Microenvironment Remodeling in Fallopian Tube Precursors to High-Grade Serous Ovarian Carcinoma.
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, C. Hug, S. Coy, Yu-An Chen, Ino de Bruijn, Natalie Shih, Euihye Jung, R. Pelletier, Mariana Lopez Leon, Gabriel Mingo, Dalia K Omran, Jong Suk Lee, Clarence Yapp, B. Satravada, Ritika Kundra, Yilin Xu, Sabrina Chan, Juliann B. Tefft, J. Muhlich, Sarah H Kim, Stefan M. Gysler, Judith Agudo, James R. Heath, N. Schultz, Charles Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2024-12-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 Background: Locally-advanced rectal cancer (LARC) is traditionally treated with neoadjuvant chemoradiotherapy before surgery, however, response heterogeneity necessitates the development of novel treatment combinations. Immune checkpoint blockade (ICB) with or without radiotherapy (RT) elicits clinical benefit in various solid tumors, yet ICB’s effectiveness so far is limited to colorectal tumors with microsatellite instability and the role of the tumor microenvironment (TME) is unclear. We present a novel orthotopic immunocompetent mouse model of LARC, where combinations of RT and ICB are tested to elucidate the role of TME in the response to treatment. Methods: Autochthonous murine tumor-derived organoids expressing common driver mutations (Apcfl/fl; KrasG12D/+; p53fl/fl; Tgfbr1fl/fl: ‘AKPT’) were orthotopically injected into syngeneic C57Bl/6 immunocompetent mice. Tumor response to RT-ICB was studied on timepoint and aging cohorts of mice with tumors growing in different TME settings: (a) in the rectal submucosa of wildtype mice, (b) subcutaneously in wildtype mice, and (c) in the rectal submucosa of mice lacking the iCCR locus (CCR1, CCR2, CCR3, and CCR5: ‘iCCRKO ’), which inhibits inflammatory monocyte recruitment into the TME. Radiation was delivered as 15Gy in 3# (Mon-Wed-Fri) using a Small Animal Radiation Research Platform on day 13 following tumor engraftment for the orthotopic models and on day 7 for the subcutaneous. PD-1 inhibition (PD-1i) was administered intraperitoneally (10mg/kg twice weekly) 3 days pre-RT until endpoint. Mice in the timepoint experiments were sampled 5 days following RT, while aging mice were sampled at clinical endpoint or 100 days, whichever came first. Results: RT and PD-1i offered significant survival extension and tumor control in the orthotopic wildtype setting (a). Transcriptomic characterization at 5 days revealed that untreated and irradiated tumors harbored a stroma-rich microenvironment, low in immune cells, which PD-1i partly reversed. This phenotypic ‘switch’ was also evident in an aging cohort, suggesting that RT + PD-1i have long-lasting effects on the stroma. In the subcutaneous setting (b), RT significantly increased tumor control but PD-1 blockade offered little additional advantage. Interestingly, subcutaneous AKPT tumors had significantly more CD8+ cells at baseline than orthotopic ones, highlighting the inherent immune permissiveness of the subcutaneous TME. Finally, tumors growing in the iCCRKO setting (c) were more radio-sensitive, leading to 3 instances of tumor clearance: one in the RT group and two in the RT + PD-1i. Conclusion: Our findings demonstrate the synergistic potential of RT and PD-1i to sensitize an inherently radio-resistant orthotopic mouse model of LARC. Tumor location and inflammatory monocytes emerged as key regulators of the immune response to RT-ICB, which could have important implications for LARC modeling. Importantly, we highlight that TME components can facilitate treatment resistance and give rise to targets that can be therapeutically exploited.
 Citation Format: Lydia Melissourgou-Syka, Michael A. Gillespie, Lily V.S. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, Tamsin R.M. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, Sean M. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S.D. Roxburgh.Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P022">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b958d65c1fd0bd54499a71c321762f474a07901" target='_blank'>
              Abstract P022: Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations
              </a>
            </td>
          <td>
            Lydia Melissourgou-Syka, Michael A Gillespie, L. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, T. Lannagan, Eoghan Mulholland, Philip D. Dunne, J. Edwards, S. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S D Roxburgh
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5675284f659a316a50b22f64df578d9b883b676a" target='_blank'>
              Single-cell RNA sequencing highlights the unique tumor microenvironment of small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Tianyou Wang, Li Zhang, Song Mei, Bo Wang, Jiaqi Liu, Weiping Yang, Jiongbo Liao, Chao Wang
          </td>
          <td>2025-01-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Glioma is the most common primary malignant brain tumor. Despite advances in surgical techniques and treatment regimens, the therapeutic effects of glioma remain unsatisfactory. Immunotherapy has brought new hope to glioma patients, but its therapeutic outcomes are limited by the immunosuppressive nature of the tumor microenvironment (TME). This study aimed to reveal the subpopulations and functional characteristics of tumor-associated macrophages (TAMs) and explore the regulatory effects of chlorogenic acid (CHA) on the immune microenvironment, as well as its potential for clinical application. Methods In this study, CHA was used in model mice. ScRNA - seq analysis was conducted to elucidate the differentiation trajectories and functional characteristics of bone marrow - derived monomacrophages (BMDMs) and microglia. A PPI and molecular docking model were constructed using the target prediction database. A case of a patient treated with CHA was reviewed. Results CHA slowed tumor growth in model mice and extended the survival time of mice. It enhanced the antigen - presenting function of macrophages and T - cell immune activation - related gene expression, activated microglia through the JAK - STAT pathway, and improved the antitumor functions. The good affinity of CHA with STAT1 was confirmed. The patient treated with CHA survived for 5 years and 6 months, achieved partial remission (PR) after 9 months of treatment, and remained alive without any new symptoms or toxic side effects. Our study revealed the subtypes and differentiation trajectories of TAMs. CHA significantly improved the immune microenvironment of glioma by modulating the function of BMDMs and microglia. Discussion This study may provide new insights into targeting the regulation of TME and offer theoretical and practical support for the clinical application of CHA. The results demonstrated the potential of CHA in improving the immune microenvironment and antitumor effects, which could have implications for future glioma treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43630a7cbb9932035e6e598a785d760c8dd3e625" target='_blank'>
              ScRNA-seq unveils the functional characteristics of glioma-associated macrophages and the regulatory effects of chlorogenic acid on the immune microenvironment—a study based on mouse models and clinical practice
              </a>
            </td>
          <td>
            Jiachen Wang, Shenglan Li, Yuxiao Chen, Jinyi Chen, Can Wang, Zhuang Kang, Mengqian Huang, Ze-Hao Cai, Yuxiang Fan, Yanjie Lan, Yumeng Yu, Ruijing Bai, Feng Chen, Jiandong Jiang, Wenbin Li
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tissue-resident memory T cells (TRM) provide frontline protection against pathogens and emerging malignancies. Tumor-infiltrating lymphocytes (TIL) with TRM features are associated with improved clinical outcomes. However, the cellular interactions that program TRM differentiation and function are not well understood. Using murine genetic models and targeted spatial transcriptomics, we found that the CD8+ T cell-derived chemokine XCL1 is critical for TRM formation and conventional DC1 (cDC1) supported the positioning of intestinal CD8+ T cells during acute viral infection. In tumors, enforced Xcl1 expression by antigen-specific CD8+ T cells promoted intratumoral cDC1 accumulation and T cell persistence, leading to improved overall survival. Notably, analysis of human TIL and TRM revealed conserved expression of XCL1 and XCL2. Thus, we have shown that the XCL1-XCR1 axis plays a non-cell autonomous role in guiding intestinal CD8+ TRM spatial differentiation and tumor control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2308bb40e506b513d809177fd6a68dab1c1bbff2" target='_blank'>
              The XCL1-XCR1 axis supports intestinal tissue residency and antitumor immunity.
              </a>
            </td>
          <td>
            Amir Ferry, Kianoosh M Mempel, Alexander Monell, Miguel Reina-Campos, Nicole E. Scharping, Maximilian Heeg, Kennidy K. Takehara, Shiruyeh Schokrpur, Ning Kuo, R. Saddawi-Konefka, J. S. Gutkind, Ananda W Goldrath
          </td>
          <td>2025-01-22</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985b60df1dc8419f03f3525f9f1216e748698218" target='_blank'>
              Analysis of a mouse germ cell tumor model establishes pluripotency-associated miRNAs as conserved serum biomarkers for germ cell cancer detection
              </a>
            </td>
          <td>
            A. Loehr, Dennis M. Timmerman, Michelle Liu, A. Gillis, Melia Matthews, Jordana C. B. Bloom, Peter K. Nicholls, David C Page, Andrew D. Miller, Leendert H J Looijenga, Robert S Weiss
          </td>
          <td>2025-02-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d66a87eaf162b506f3d34ebd1845180969ed36" target='_blank'>
              Mitochondrial transcription elongation factor TEFM promotes malignant progression of gliomas
              </a>
            </td>
          <td>
            Yin Wang, Wenxuan Hu, Boya Zhou, Yu Zhao, Yufei Tang, Zhiyong Deng, Minbin Chen
          </td>
          <td>2024-12-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of cancer biology and therapeutics has soared in the past several decades with new therapeutic modalities and options for patients, such as chemoradiotherapy, immunotherapy, and combination therapy. This dramatic success in expanding patient options is primarily attributed to the development of various model systems to elucidate drivers of oncogenesis, tumor maturation and evolution, and response to therapeutics. While mouse models have been a workhorse of cancer research, technological progress in ex vivo patient‐derived tumor models has afforded more tunable and scrutable systems for patient‐predictive platforms and mechanistic study. This review explores the technological innovations in 3D solid tumor models and their applicability to various aspects of cancer biology and the identification of therapeutics. Features of the tumor and tumor microenvironment like spatial heterogeneity, multicellular populations, and genomic variations are addressed and elaborated through the establishment of new in vitro models. The integration of perfusable vasculature with 3D tumor models and the potentially wide‐ranging applications of these more complex platforms in precision medicine and cancer immunotherapy are further addressed. Finally, an outlook on the future of experimental cancer models for both biological investigation and bench‐to‐bedside pipeline development is provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bf00a461fcd8fcd70e5c1de6d55efdf64b0b5d" target='_blank'>
              Advances and Challenges in Human 3D Solid Tumor Models
              </a>
            </td>
          <td>
            Naveen R. Natesh, Shyni Varghese
          </td>
          <td>2025-01-24</td>
          <td>Advanced Functional Materials</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd21edea6ef2a85bfcda6c2395a58850a218d8d2" target='_blank'>
              Patient-derived response estimates from zero-passage organoids of luminal breast cancer
              </a>
            </td>
          <td>
            Róża K. Przanowska, Najwa Labban, Piotr Przanowski, Russell B Hawes, Kristen A Atkins, S. Showalter, Kevin A. Janes
          </td>
          <td>2024-12-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="When confronted with an undesired cell population, such as bacterial infections or tumors, we seek the most effective treatment, designed to eliminate the population as rapidly as possible. A common practice is to monitor the cells short-term response to the treatment, and from that, extrapolate the eventual treatment outcome, i.e. will it eradicate the cells, and if yes at what timescales. Underlying this approach is the assumption that the cells exhibit a homogeneous response to the treatment, and hence the early response patterns can be naturally extended to later times. Recent experiments on cancer cell populations, however, indicate a significant level of cellular heterogeneity, undermining this classic assessment protocol of treatment efficacy. We, therefore, develop here a stochastic framework, to analytically predict the temporal dynamics of a heterogeneous cell population. Quite often, we find, the average cellular parameters, governing the short-term response, fail to predict the actual treatment outcome. In contrast, our analysis, which also incorporates the populations variability, helps identify the relevant statistical parameters, which in turn, enable us to predict the full trajectory of the cell population, and specifically - the likelihood and typical timescales for remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ef92fd76daed3e67a88fcf502b6cc330b90e29" target='_blank'>
              Efficient treatment of heterogeneous malignant cell populations
              </a>
            </td>
          <td>
            Uzi Harush, R. Straussman, B. Barzel
          </td>
          <td>2025-01-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The poor survival of ovarian cancer patients is linked to their high likelihood of relapse. In spite of full apparent macroscopic clearance, tumor recurrences arise from cells that are resistant to primary chemotherapy in the form of minimal residual disease (MRD). MRD exhibits distinct molecular drivers from bulk cancer and therefore necessitates alternative therapeutic strategies. However, there is a lack of 3D models that faithfully recapitulate MRD ex vivo for therapy development. This study constructs microfluidics-based 3D microtumors to generate a clinically-relevant model for ovarian cancer MRD. The microtumors recapitulate the non-genetic heterogeneity of ovarian cancer, capturing the "Oxford Classic" five molecular signatures. Gene expression in the 3D microtumors aligns closely with MRD from ovarian cancer patients and features the upregulation of fatty acid metabolism genes. Finally, the MRD 3D microtumors respond to the approved fatty acid oxidation inhibitor, perhexiline, demonstrating their utility in drug discovery. This system might be used as a drug-testing platform for the discovery of novel MRD-specific therapies in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c92f8e21e6ccaf75d856fcadd4e15f431e68af9" target='_blank'>
              3D Microtumors Representing Ovarian Cancer Minimal Residual Disease Respond to the Fatty Acid Oxidation Inhibitor Perhexiline.
              </a>
            </td>
          <td>
            Xingyun Yang, Mara Artibani, Yongcheng Jin, Aneesh Aggarwal, Yujia Zhang, Sandra Muñoz-Galvan, E. Mikhailova, Lena Rai, Nobina Mukherjee, Ravinash Krishna Kumar, A. Albukhari, Shaohua Ma, Linna Zhou, Ahmed Ashour Ahmed, Hagan Bayley
          </td>
          <td>2025-02-09</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Gastric cancer (GC) ranks as the fifth most prevalent malignant neoplasm globally, with an increased death rate despite recent advancements in research and therapeutic options. Different molecular subtypes of GC have distinct interactions with the immune system, impacting the tumor microenvironment (TME), prognosis, and reaction to immunotherapy. Tumor-infiltrating lymphocytes (TILs) in the TME are crucial for preventing tumor growth and metastasis, as evidenced by research showing that patients with GC who have a significant density of TILs have better survival rates. But cancer cells have evolved a variety of mechanisms to evade immune surveillance, both sialic acid-binding immunoglobulin-like lectin 15 (Siglec-15) and Programmed Death-Ligand 1 (PD-L1) playing a pivotal role in the development of an immunosuppressive TME. They prevent T cell activation and proliferation resulting in a decrease in the immune system’s capacity to recognize and eliminate malignant cells. These immune checkpoint molecules function via different but complementary mechanisms, the expression of Siglec-15 being mutually exclusive with PD-L1 and, therefore, providing a different therapeutic approach. The review explores how TILs affect tumor growth and patient outcomes in GC, with particular emphasis on their interactions within the TME and potential targeting of the PD-L1 and Siglec-15 pathways for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062a7edc3c5fa270e9d454add23d11aa4986c500" target='_blank'>
              Immune Cell Interactions and Immune Checkpoints in the Tumor Microenvironment of Gastric Cancer
              </a>
            </td>
          <td>
            A. Cozac-Szőke, D. Cozac, Anca Negovan, A. Tinca, Alexandra Vilaia, I. Cocuz, Adriana Sabău, Raluca Niculescu, D. Chiorean, Alexandru Nicușor Tomuț, O. S. Cotoi
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is the most common primary liver cancer, with microvascular invasion (MVI) identified as a major predictor of early recurrence. However, the intratumor cellular heterogeneity of MVI, the identification of pertinent biomarkers, and the role of intercellular signalling interactions in MVI progression are unclear. This study aims to explore these aspects using single-cell transcriptomic analysis. Methods The present study utilized single-cell transcriptomic data from public databases to conduct an in-depth transcriptome analysis of tumour tissues and adjacent nontumor tissues from five patients with hepatocellular carcinoma, with a particular focus on samples from three patients exhibiting microvascular invasion. Bioinformatics tools were employed to analyze gene expression patterns and signalling pathways. Results The findings indicated that MVI-positive malignant cells activate multiple signalling pathways to facilitate invasion and metastasis. Specific malignant cell subtypes strongly associated with MVI were identified, exhibiting distinctive gene expression patterns related to proliferation, invasion, and metabolic reprogramming of tumour cells. Further analysis revealed that the laminin and VEGF signalling pathways are crucial for remodelling the tumour microenvironment and angiogenesis associated with MVI. The MARCKSL1 gene was predominantly expressed in MVI-positive malignant cells and may contribute to MVI progression by interacting with the PTN signalling network. Additionally, MARCKSL1 is linked to tumour resistance to multiple anticancer drugs. Discussion This study sheds light on the molecular characteristics and functional heterogeneity of MVI-associated malignant cell subpopulations. The single-cell transcriptome and bioinformatics analyses provided insights into the mechanisms driving MVI, potentially aiding the development of targeted diagnostic and therapeutic strategies. Future research should further validate the role of MARCKSL1 in MVI progression and explore its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e75f99f0ab352845523f0f1e55263a27b76fefc5" target='_blank'>
              Single-cell sequencing reveals cell heterogeneity and aberrantly activated pathways associated with microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jianwei Cui, Fanyi Zeng, Ming Tang, Shiwu Yin
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) is a highly aggressive tumor with one of the highest morbidity and mortality rates in the world. Nucleotide metabolic processes are critical for cancer development, progression, and alteration of the tumor microenvironment. However, the effect of nucleotide metabolism on LUAD remains to be thoroughly investigated. Methods Transcriptomic and clinical data of LUAD were downloaded and organized from TCGA and GEO databases. Genes related to nucleotide metabolism were downloaded from the Msigdb database. Genes associated with LUAD prognosis were identified using univariate COX analysis, and a prognostic risk model was constructed using the machine learning combination of Lasso + Stepcox. The model’s predictive validity was evaluated using KM survival and timeROC curves. Based on the prognostic model, LUAD patients were classified into different nucleotide metabolism subtypes, and the differences between patients of different subtypes were explored in terms of genomic mutations, functional enrichment, tumor immune characteristics, and immunotherapy responses. Finally, the key gene SNRPA was screened, and a series of in vitro experiments were performed on LUAD cell lines to explore the role of SNRPA in LUAD. Result LUAD patients could be accurately categorized into subtypes based on the nucleotide metabolism-related prognostic risk score (NMBRS). There were significant differences in prognosis between patients of different subtypes, and the NMBRS showed high accuracy in predicting the prognosis of LUAD patients. In addition, patients of different subtypes showed significant differences in genomic mutation and functional enrichment and exhibited different anti-tumor immune profiles. Importantly, NMBRS can be used to predict the responsiveness of LUAD patients to immunotherapy. The results of in vitro cellular experiments indicate that SNRPA plays an important role in the development and progression of lung adenocarcinoma. Conclusion This study comprehensively reveals the prognostic value and clinical application of nucleotide metabolism in LUAD. A prognostic signature constructed based on genes related to nucleotide metabolism accurately predicted the prognosis of LUAD patients, and this signature can be used as a guide for LUAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f247cf9ee6d123c8afb18b9aa087b0c11d86c73" target='_blank'>
              Integration of single-cell transcriptomics and bulk transcriptomics to explore prognostic and immunotherapeutic characteristics of nucleotide metabolism in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kai Zhang, Luyao Wang, Huili Chen, Lili Deng, Mengling Hu, Ziqiang Wang, Yiluo Xie, Chaoqun Lian, Xiaojing Wang, Jing Zhang
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple negative breast cancers often contain higher numbers of tumour-infiltrating lymphocytes compared with other breast cancer subtypes, with their number correlating with prolonged survival. Since little is known about tumour-infiltrating lymphocyte trafficking in triple negative breast cancers, we investigated the relationship between tumour-infiltrating lymphocytes and the vascular compartment to better understand the immune tumour microenvironment in this aggressive cancer type. We aimed to identify mechanisms and signaling pathways responsible for immune cell trafficking in triple negative breast cancers, specifically of basal type, that could potentially be manipulated to change such tumours from immune “cold” to “hot” thereby increasing the likelihood of successful immunotherapy in this challenging patient population. We characterised the spatial immune environment in 10 basal breast cancers showing a range of tumour-infiltrating lymphocytes using multiplex fluorescent immunohistochemistry and quantitative digital analysis of CD3+ T cells. We examined their relationship to blood vessels and their activation status as defined by VCAM-1, ICAM-1 and PD-L1. Confirmation of the relationship between tumour-infiltrating lymphocytes and endothelial activation was performed through in silico analysis on TCGA BRCA RNA-seq data (N = 808). Significantly higher CD3+ T cell densities were observed in the stromal compartment compared with the neoplastic cell compartment (P = 0.003). ICAM-1 activated blood vessels were spatially associated with higher CD3+ T cell densities only within 30 microns of blood vessels compared with more distal activated and non-activated blood vessels (P = 0.041). In silico analysis confirmed higher numbers of tumour-infiltrating lymphocytes in basal breast cancers and that higher numbers were significantly associated with endothelial cell activation molecules, co-clustering with upregulated ICAM-1 and VCAM-1 amongst others. PD-L1 was also identified in a subset of blood vessels, suggesting an additional immune regulatory mechanism in endothelial cells. Regulating the activation status of tumour-associated vascular endothelial cells may improve T cell trafficking into basal breast tumours and enhance immunotherapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd5404cd864380509879a211020591331396cbc" target='_blank'>
              Preliminary characterisation of the spatial immune and vascular environment in triple negative basal breast carcinomas using multiplex fluorescent immunohistochemistry
              </a>
            </td>
          <td>
            E. Takano, Metta K Jana, Luis E Lara Gonzalez, J. Pang, R. Salgado, S. Loi, Stephen B Fox
          </td>
          <td>2025-01-10</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e846ee374ca9a20e78989a20462c74023e1477a" target='_blank'>
              Single-cell transcriptomics reveal the prognostic roles of epithelial and T cells and DNA methylation-based prognostic models in pancreatic cancer
              </a>
            </td>
          <td>
            Jing Du, Yaqian Zhao, Jie Dong, Peng Li, Yan Hu, Hailang Fan, Feifan Zhang, Lanlan Sun, Dake Zhang, Yuhua Zhang
          </td>
          <td>2024-12-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024cdb898f44026b61ab427711870c26e06ca884" target='_blank'>
              Identification of Cell Fate Determining Transcription Factors for Generating Brain Endothelial Cells.
              </a>
            </td>
          <td>
            Roya Ramezankhani, J. De Smedt, B. Toprakhisar, Bernard K. van der Veer, Tine Tricot, Gert Vanmarcke, Bradley Balaton, Leo A. van Grunsven, Dr. Massoud Vosough, Y. C. Chai, C. Verfaillie
          </td>
          <td>2025-01-24</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The high plasticity of cells undergoing epithelial-mesenchymal transition (EMT) promotes increased tumor heterogeneity, and its interaction with tumor-associated stromal cells appears to contribute to developing a stemness phenotype. Cells with these characteristics exhibit increased resistance to chemotherapy and radiotherapy, leading to disease relapse and metastasis. Here, we discuss the activation of the Wnt/β-catenin pathway in promoting EMT and stemness within the context of cellular resistance to these therapies. We discuss whether EMT and cancer stem cells (CSCs) function in conjunction, independently, or if a link is connecting their development. We further propose that this pathway is necessary to establish a connection between these two phenotypes. And suggest that it could hinder the rise of CSCs from treatment-induced EMT cells when inhibited. Understanding this cellular phenomenon might allow the development of new targeted therapies to improve clinical responses, particularly in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6af1b3bc1ef6fcb118710249dd08cded2ddaa10" target='_blank'>
              Wnt/β-catenin pathway as a link between therapy resistance-driven epithelial-mesenchymal transition and stemness in colorectal cancer.
              </a>
            </td>
          <td>
            M. R. Rocha, Yuri Kelly Castillo-Medina, Bárbara Martins de Lima Coelho, Luidy Lucas Lopes Rios, J. Morgado-Díaz
          </td>
          <td>2024-12-20</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="INTRODUCTION
Spatial biology is an emerging interdisciplinary field facilitating biological discoveries through the use of spatial omics technologies. Recent advancements in spatial transcriptomics, spatial genomics (e.g. genetic mutations and epigenetic marks), multiplexed immunofluorescence, and spatial metabolomics/lipidomics has enabled high-resolution spatial profiling of gene expression, genetic variation, protein expression, and metabolites/lipids profiles in tissue. These developments contribute to a deeper understanding of the spatial organization within tissue microenvironments at the molecular level.


AREAS COVERED
This report provides an overview of the untargeted, bottom-up mass spectrometry (MS)-based spatial proteomics workflow. It highlights recent progress in tissue dissection, sample processing, bioinformatics, and liquid chromatography (LC)-MS technologies that are advancing spatial proteomics toward cellular resolution.


EXPERT OPINION
The field of untargeted MS-based spatial proteomics is rapidly evolving and holds great promise. To fully realize the potential of spatial proteomics, it is critical to advance data analysis and develop automated and intelligent tissue dissection at the cellular or subcellular level, along with high-throughput LC-MS analyses of thousands of samples. Achieving these goals will necessitate significant advancements in tissue dissection technologies, LC-MS instrumentation, and computational tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750f7b08fc1af849136e70bc860f56ef668a0aa0" target='_blank'>
              Spatial proteomics toward cellular resolution.
              </a>
            </td>
          <td>
            Yumi Kwon, James M Fulcher, L. Paša-Tolić, Wei-Jun Qian
          </td>
          <td>2024-12-22</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e47a9918829003a93f398ed9e563361008b2b513" target='_blank'>
              Resolving tissue complexity by multimodal spatial omics modeling with MISO.
              </a>
            </td>
          <td>
            Kyle Coleman, Amelia Schroeder, Melanie Loth, Daiwei Zhang, Jeong Hwan Park, Ji-Youn Sung, Niklas Blank, Alexis J Cowan, Xuyu Qian, Jianfeng Chen, Jiahui Jiang, Hanying Yan, Laith Z. Samarah, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Joshua D. Rabinowitz, Yanxiang Deng, Edward B. Lee, Alexander Lazar, Jianjun Gao, E. E. Furth, Tae Hyun Hwang, Linghua Wang, Cristoph Thaiss, Jian Hu, Mingyao Li
          </td>
          <td>2025-01-15</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The zebrafish (Danio rerio) is a valuable model organism for studying human biology due to its easy genetic manipulation and small size. It is optically transparent and shares genetic similarities with humans, making it ideal for studying developmental processes, diseases, and drug screening via imaging-based approaches. Solid malignant tumors often contain hypoxic areas that stimulate the release of extracellular vesicles (EVs), lipid-bound structures released by cells into the extracellular space, that facilitate short- and long-range intercellular communication and metastatization. Here we investigate the effects of EVs derived from neuroblastoma (NB), a pediatric solid tumor, on metastatic niche formation using the zebrafish as an in vivo model. Intravascular injection in zebrafish embryos allows a non-invasive visualization of EVs dispersion, uptake, and interactions with host cells. To improve repeatability of our results and ease the injection steps, we used an agarose device replica molded from a custom designed micromilled aluminum mold. We first demonstrated that EVs released under hypoxic conditions promote angiogenesis and are more easily internalized by endothelial cells than those purified from normoxic cells. We also showed that injection of with hypoxic EVs increased macrophages mobilization. We then focused on the caudal hematopoietic tissue (CHT) region of the embryo as a potential metastatic site. After hypoxic EVs injection, we highlighted changes in the expression of mmp-9 and cxcl8b genes. Furthermore, we investigated the ability of NB-derived EVs to prime a metastatic niche by a two-step injection of EVs first, followed by NB cells. Interestingly, we found that embryos injected with hypoxic EVs had more proliferating NB cells than those injected with normoxic EVs. Our findings suggest that EVs released by hypoxic NB cells alter the behavior of recipient cells in the zebrafish embryo and promote metastatic outgrowth. In addition, we demonstrated the ability of the zebrafish embryo to be a suitable model for studying the interactions between EVs and recipient cells in the metastatic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2525012f5692c0555303a6b2e55ae094fb8a3938" target='_blank'>
              Neuroblastoma-derived hypoxic extracellular vesicles promote metastatic dissemination in a zebrafish model
              </a>
            </td>
          <td>
            Anna Fietta, P. Fusco, Giuseppe Germano, S. Micheli, M. Sorgato, G. Lucchetta, Elisa Cimetta
          </td>
          <td>2024-12-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49c3ca0cc5a8e692853a67594899e1a9c9870369" target='_blank'>
              Enterocyte-like differentiation defines metabolic gene signatures of CMS3 colorectal cancers and provides therapeutic vulnerability
              </a>
            </td>
          <td>
            Arezo Torang, Aleksandar B. Kirov, Veerle Lammers, Kate Cameron, Valérie M Wouters, Rene F Jackstadt, T. Lannagan, Joan H de Jong, Jan Koster, Owen J. Sansom, J. Medema
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Introduction SYNGR4 is considered to be one of the causative genes for amyotrophic lateral sclerosis, but its role in breast cancer development has not been revealed. Methods The expression of SYNGR4 in a variety of malignancies including breast cancer was analyzed using Genotype Tissue Expression (GTEx) and the Cancer Genome Atlas (TCGA) databases and verified by specimens collected from our center. The effect of SYNGR4 on breast cancer prognosis was analyzed using bioinformatics and possible pathways by which this molecule affects breast cancer prognosis were explored. The effect of SYNGR4 on immune infiltration of breast cancer was analyzed using GSVA, and the effects of SYNGR4 on breast cancer proliferation, migration, and tumor-associated macrophage polarization in cancer foci were verified by cellular and animal experiments, respectively. Results SYNGR4 is highly expressed in a variety of malignant tumors, including breast cancer, and affects the prognosis of breast cancer patients. This may be a volatile effect through Organelle fission, chromosome segregation, nuclear division, etc. SYNGR4 overexpression affects breast cancer proliferation, migration, and tumor immune infiltration, and promotes breast cancer tumor-associated macrophage polarization toward M2. Discussion SYNGR4 overexpression can affect the prognosis of breast cancer patients by promoting M2 polarization of tumor-associated macrophages in breast cancer, and this molecule may be a novel target for breast cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5674f36bf40855cf25e3d1288b0644284cc29bf3" target='_blank'>
              Unveiling the role of SYNGR4 in breast cancer development: a novel target for immunotherapy
              </a>
            </td>
          <td>
            Jie Ma, Hongtao Wang, Zhengwei Gui, Yuanrong Yang
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Since its first conceptualization over a century ago, the mesenchymal phenotype has traditionally been viewed as either a transient phase between successive epithelial stages or as a feature of cell types primarily devoted to structural support. However, recent findings in cancer research challenge this limited view, demonstrating that mesenchymal traits and hybrid mesenchymal/epithelial states can mark cancer cells with stem cell properties. By analyzing publicly available single-cell transcriptome datasets from early embryonic stages and adult tissues, this study aims to extend this concept beyond pathological contexts, suggesting that a partial or fully mesenchymal phenotype may represent the morphological expression of undifferentiated and multipotent states in both the developing embryo and adult organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b86efcb43fa26b36795e33eead52234f743d4e" target='_blank'>
              Mesenchymal Traits as an Intrinsic Feature of Undifferentiated Cells
              </a>
            </td>
          <td>
            M. Galiè
          </td>
          <td>2024-12-24</td>
          <td>Journal of Developmental Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350ae86f61f0bcf11ab3944cf0a0043eb71b585b" target='_blank'>
              Radiogenomics and machine learning predict oncogenic signaling pathways in glioblastoma
              </a>
            </td>
          <td>
            Abdul Basit Ahanger, Syed Wajid Aalam, Tariq Ahmad Masoodi, Asma Shah, Meraj Alam Khan, Ajaz A. Bhat, Assif Assad, M. Macha, M. Bhat
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The cell fate transition is a fundamental characteristic of living organisms. By introducing external perturbations, it is possible to artificially intervene in cell fate and trigger cell reprogramming. Revealing the general principle underlying the induced phenotypic reshaping of cell populations remains a central focus in the field of cell biology. In this study, we investigate the energetic and dynamic features of induced cell phenotypic transition from differentiated somatic state to pluripotent state by constructing a Boolean genetic network model. The simulation and experimental results highlight the critical role of genetic frustration in initiating cell fate transitions, although the two ending phenotypic states are typically featured by minimal frustration. In addition, the altered gene expression profiles exhibit a scale-free distribution, suggesting that there exist a small number of critical genes responsible for the cell fate transition. This study provides important insights into the dynamic principles governing effective cell reprogramming caused by artificial or exogenous interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e17321e8898e722005a2009e4fe82616f4dc5" target='_blank'>
              Role of genetic frustrations in cell reprogramming
              </a>
            </td>
          <td>
            Yuxiang Yao, Jieying Zhu, Wenfei Li, Duanqing Pei
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [2, 4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>