<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-24 09:27:04 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="While recent innovations in spatial biology have driven new insights into how tissue organization is altered in disease, interpreting these datasets in a generalized and scalable fashion remains a challenge. Computational workflows for discovering condition-specific differences in tissue organization typically rely on pairwise comparisons or unsupervised clustering. In many cases, these approaches are computationally expensive, lack statistical rigor, and are insensitive to low-prevalence cellular niches that are nevertheless highly discriminative and predictive of patient outcomes. Here, we present QUICHE – an automated, scalable, and statistically robust method that can be used to discover cellular niches differentially enriched in spatial regions, longitudinal samples, or clinical patient groups. In contrast to existing methods, QUICHE combines local niche detection with interpretable statistical modeling using graph neighborhoods to detect differentially enriched cellular niches, even at low prevalence. Using in silico models and spatial proteomic imaging of human tissues, we demonstrate that QUICHE can accurately detect condition-specific cellular niches occurring at a frequency of 0.5% in fewer than 20% of patient samples, outperforming the next best method which required a patient prevalence of 60% for detection. To validate our approach and understand how tumor structure influences recurrence risk in triple negative breast cancer (TNBC), we used QUICHE to comprehensively profile the tumor microenvironment in a multi-center, spatial proteomics cohort consisting of primary surgical resections, analyzing over 2 million cells from 314 patients across 5 medical centers. We discovered cellular niches that were consistently enriched in key regions of the tumor microenvironment, including the tumor-immune border and extracellular matrix remodeling regions, as well as niches statistically-associated with patient outcomes, including recurrence status and recurrence-free survival. The majority of differential niches (74.2%) were specific to patients that did not relapse and formed a robust interconnected network enriched in monocytes, macrophages, APCs, and CD8T cells with tumor and stroma cells. In contrast, the interaction network for patients that relapsed was notably sparse and enriched in B cells, CD68 macrophages and neutrophils. We validated these findings using two independent cohorts, observing similar cellular interactions and predictive power. Collectively, these results suggest that salient, generalized profiles of productive anti-tumor immune responses are defined by a network of structural engagement between innate and adaptive immunity with tumor and stromal cells, rather than by any single specific cell population. We have made QUICHE freely available as a user-friendly open-source Python package at https://github.com/jranek/quiche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda789c041e695c92206356373353eb906709b05" target='_blank'>
              QUICHE reveals structural definitions of anti-tumor responses in triple negative breast cancer
              </a>
            </td>
          <td>
            Jolene S Ranek, Noah F. Greenwald, Mako Goldston, C. Fullaway, Cameron Sowers, Alex Kong, S. Mourón, M. Quintela-Fandino, Robert West, Michael Angelo
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="Small bowel adenocarcinoma (SBA) is a rare malignancy marked with a poor prognosis. The cellular and proteomic heterogeneity within the tumor immune microenvironment (TIME) of SBA is a likely driver of prognosis, disease progression and response to therapy. We have addressed a major gap in knowledge of the TIME in SBA using highly multiplexed, protein imaging of the SBA tumor-immune ecosystem generating a comprehensive, single-cell level, spatial, proteomic atlas of TIME in > 600,000 cells from 136 tumor and matched normal samples from clinically and genomically annotated SBA patients (N=37). The SBA TIME Atlas informs on spatial distribution and interactions of tumor-intrinsic processes, diverse immune cell types, immune checkpoints, and vascularization. Enrichment of proliferating epithelial tissue, stem cells, and likely pro-tumor immune signatures in the tumor niche is contrasted by the representation of naïve and early-effector T-cells in the epithelial compartments of adjacent normal tissues. Epithelial-stem-immune cell spatial architectures within tumor and matched-normal niches were strongly associated with patient survival, suggesting malignancy is driven by spatial architecture beyond the tumor microenvironment. The blueprints of therapeutically actionable immune checkpoints at the interface between epithelial and microenvironmental T-cells as well as macrophages have established a guideline for precision immunotherapies tailored to the TIME composition in SBA. We expect that this SBA atlas will contribute to a deeper understanding of the immune contexture in this rare disease as well as other gastrointestinal cancers and help guide future precision immune-oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Single-cell spatial transcriptomics reveals how cells organize in healthy and diseased tissues. From these data, tissue segmentation analysis defines discrete compartments that organize cells into functional multicellular units. Existing methods for automated tissue segmentation rely on spatial smoothing to define spatially coherent regions but often blur the boundaries between adjacent tissue compartments. We describe Tessera, an algorithm that approaches tissue segmentation through a novel approach, dividing the tissue into small multicellular tiles whose edges track with natural tissue boundaries. Tessera achieves this by incorporating successful tools from edge-preserving smoothing, topological data analysis, and morphology-aware agglomerative spatial clustering. We show that Tessera identifies a range of known anatomical structures, in healthy mouse brain and human lymph nodes, and novel disease-associated niches, in human brain and in lung cancer. Tessera is a general-purpose tool that returns spatially coherent spatial structures with accurate boundaries across a range of spatial transcriptomics and proteomics technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfead60198d5f27b8c18dbe0ea5203b263942c83" target='_blank'>
              Accurate tiling of spatial single-cell data with Tessera
              </a>
            </td>
          <td>
            Daniel J. Stein, Miles Tran, I. Korsunsky
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris-II, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) features a complex tumor microenvironment (TME) that significantly influences patient outcomes. Understanding the TME’s cellular composition and interactions is crucial for identifying therapeutic targets to improve treatment. Methods We performed an integrative analysis combining 88 single-cell RNA sequencing (scRNA-seq) samples with 187,520 cells, 20 Visium spatial transcriptomics (ST-seq) samples with 67,933 spots, and 1383 bulk RNA-seq samples and 2 Xenium high-resolution ST-seq samples with 307,679 cells, to delineate and characterize distinct subpopulations of fibroblasts, macrophages, T/NK cells, and B/plasma cells within the PDAC microenvironment. Correlations among major cell types across 12 PDAC datasets were assessed through gene set variation analysis (GSVA). Kaplan-Meier survival analysis was used to evaluate the prognostic significance of cell subtypes. Furthermore, multiplex immunohistochemistry (mIHC) and the Xenium platform was utilized to validate cellular interactions in human PDAC tissues at both protein and RNA levels. Results Six fibroblast subtypes and eight macrophage/monocyte subtypes were identified. POSTN high fibroblasts and SPP1 high macrophages highly infiltrated tumor tissues and were associated with poor prognosis. Most immune cell subtypes mediate adverse prognoses, except for CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells, which are linked to favorable outcomes. ST-seq revealed spatial colocalization of POSTN high Fibro and SPP1 high Macro cells, as well as CCL4 high CD8+ T EFF and IGHG1 high plasma cells. These findings were corroborated by mIHC and validated using Xenium spatial transcriptomics with single-cell resolution, which confirmed the expression and spatial proximity of these markers at both the gene and protein levels. Conclusion This integrative analysis of the PDAC TME underscores the prognostic importance of POSTN high fibroblasts and SPP1 high macrophages, while also highlighting the protective roles of CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells. These insights could improve PDAC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Spatial transcriptomics aims to elucidate cell coordination within biological tissues by linking the state of the cell with its local tissue microenvironment. Imaging-based assays are particularly promising for exploring such interdependencies, as they can resolve molecular and cellular features with subcellular resolution in three dimensions. Quantification and analysis of cellular state in such data, however, ultimately depends on the ability to recognize which molecules belong to each cell. Despite computational and experimental progress, this cell segmentation task remains challenging. Here we re-analyze data from multiple tissues and platforms and find that segmentation errors currently confound most downstream analysis of cellular state, including analysis of differential expression, inference of neighboring cell influence, and ligand-receptor interactions. The extent to which mis-segmented molecules impact the results can be striking, often dominating the set of top hits. We show that factorization of molecular neighborhoods can be effective at isolating such molecular admixtures and minimizing their impact on downstream analysis, analogous to doublet filtering of scRNA-seq data. As applications of spatial transcriptomics assays become more widespread, we expect corrections for the confounding effect of segmentation errors to become increasingly important for being able to resolve molecular mechanisms of tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca4acc410c4c078dfd09e8229953bf61e6a7e0c5" target='_blank'>
              Impact of Segmentation Errors in Analysis of Spatial Transcriptomics Data
              </a>
            </td>
          <td>
            Jonathan Mitchel, Teng Gao, Eli Cole, Viktor Petukhov, P. V. Kharchenko
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) provide therapeutic benefit in EGFR-mutated non-small cell lung cancer (NSCLC); however, early resistance develops in some patients. To explore the underlying mechanisms, we performed spatial transcriptomics using Visium HD and StarDist processing on paired pre- and post-treatment samples from the same patients. In early resistance samples, fibroblasts increased in proportion and exhibited enhanced activation following EGFR- TKI treatment, while a distinct tumor cell cluster showed TNF-α/NF-kB pathway activation even prior to treatment. Furthermore, in the pre-treatment sample with early resistance, specific tumor clusters interacted with immune and stromal cells through inflammatory pathways, including MIF/CD74 and PDGF signaling. Immature tertiary lymphoid structures (TLSs) were enriched in early resistance, whereas mature TLSs were observed in long-term responders. These findings highlight the role of tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insights into potential therapeutic strategies to improve treatment outcomes. Disclosure Daisuke Shibahara has received research funding from Eli Lilly Japan K.K. Koichi Azuma has received honoraria from AstraZeneca, MSD, Chugai Pharmaceutical, Ono Pharmaceutical, Bristol-Myers Squibb, Takeda Pharmaceutical, Taiho Pharmaceutical, and Amgen. Yasuto Yoneshima has received honoraria from AstraZeneca. Kentaro Tanaka has received honoraria from Chugai Pharmaceutical, AstraZeneca, Ono Pharmaceutical, Bristol-Myers Squibb, Eli Lilly, Takeda Pharmaceutical, Novartis Pharma K.K., Merck Biopharma, Kyowa Kirin, Daiichi-Sankyo, Pfizer, Amgen, Janssen Pharmaceutical K.K., and MSD, and is an advisory board member for Pfizer and Novartis Pharma K.K. Eiji Iwama has received honoraria from AstraZeneca and Chugai Pharmaceutical. Isamu Okamoto has received honoraria and research funding from Daiichi Sankyo, Chugai Pharmaceutical, Eli Lilly Japan K.K., AstraZeneca, Taiho Pharmaceutical, Boehringer Ingelheim, and Ono Pharmaceutical; honoraria from Takeda Pharmaceutical and Novartis Pharma K.K.; and research funding from Bristol-Meyers Squibb and MSD Oncology. Other authors declare no potential conflicts of interest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc050a906a767a3e545375057173c4f1889efdeb" target='_blank'>
              Spatial dynamics of the tumor microenvironment in emerging resistance to targeted therapy in EGFR-mutated NSCLC
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, Mikihiro Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Spatial transcriptomics provides insights into tissue architecture by linking gene expression with spatial localization. Current deconvolution methods rely heavily on single-cell RNA sequencing (scRNA-seq) references, which are costly and often unavailable, mainly if the tissue under evaluation is limited, such as in a core biopsy specimen. We present a novel tool, CITEgeist, that deconvolutes spatial transcriptomics data using antibody capture from the same slide as the reference, directly leveraging cell surface protein measurements from the same tissue section. This approach circumvents the limitations of scRNA-seq as a reference, offering a cost-effective and biologically grounded alternative. Our method employs mathematical optimization to estimate cell type proportions and gene expression profiles, incorporating sparsity constraints for robustness and interpretability. Benchmarks against state-of-the-art deconvolution methods show improved accuracy in cell type resolution, particularly in dense tumor microenvironments, while maintaining computational efficiency. This antibody-based tool advances spatial transcriptomics by providing a scalable, accurate, and reference-independent solution for deconvolution in complex tissues. We validate this tool by using a combined approach of simulated data and clinical samples by applying CITEgeist to translational pre-treatment and post-treatment ER+ breast tumors from an ongoing clinical trial, emphasizing the applicability and robustness of CITEgeist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4308d97c43b1d495c969105587ab2b638fc0a951" target='_blank'>
              CITEgeist: Cellular Indexing of Transcriptomes and Epitopes for Guided Exploration of Intrinsic Spatial Trends
              </a>
            </td>
          <td>
            A. Chang, Brent T. Schlegel, N. Carleton, Priscilla F McAuliffe, S. Oesterreich, Russell Schwartz, Adrian V. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Malignant gliomas follow two distinct natural histories: de novo high grade tumors such as glioblastoma, or lower grade tumors with a propensity to transform into high grade disease. Despite differences in tumor genotype, both entities converge on a common histologically aggressive phenotype, and the basis for this progression is unknown. Glioma associated macrophages (GAM) have been implicated in this process, however GAMs are ontologically and transcriptionally diverse, rendering isolation of pathogenic subpopulations challenging. Since macrophage contextual gene programs are orchestrated by transcription factors acting on cis-acting promoters and enhancers in gene regulatory networks (GRN), we hypothesized that functional populations of GAMs can be resolved through GRN inference. Here we show via parallel single cell RNA and ATAC sequencing that a subpopulation of human GAMs can be defined by a GRN centered around the Activator Protein-1 transcription factor FOSL2 preferentially enriched in high grade tumors. Using this GRN we nominate ANXA1 and HMOX1 as surrogate cell surface markers for activation, thus permitting prospective isolation and functional validation in human GAMs. These cells, termed malignancy associated GAMs (mGAMs) are pro-invasive, pro-angiogenic, pro-proliferative, possess intact antigen presentation but skew T-cells towards a CD4+FOXP3+ phenotype under hypoxia. Ontologically, mGAMs share somatic mitochondrial mutations with peripheral blood monocytes, and their presence correlates with high grade disease irrespective of underlying tumor mutation status. Furthermore, spatio-temporally mGAMs occupy distinct metabolic niches; mGAMs directly induce proliferation and mesenchymal transition of low grade glioma cells and accelerate tumor growth in vivo upon co-culture. Finally mGAMs are preferentially enriched in patients with newly transformed regions in human gliomas, supporting the view that mGAMs play a pivotal role in glioma progression and may represent a plausible therapeutic target in human high-grade glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, Nathalie Y.R. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Spatially non-resolved transcriptomic data identified functionally distinct populations of fibroblasts in health and disease. However, in-depth transcriptional profiling in situ at single-cell resolution has not been possible so far. Here, we studied fibroblast populations in the skin of SSc patients and healthy individuals using cyclic in situ hybridization (cISH) as a novel approach for spatially-resolved transcriptional phenotyping with subcellular resolution. cISH deconvoluted the heterogeneity of 20,979 cells including 3,764 fibroblasts (FB). BANKSY-based spatially-informed clustering identified nine FB subpopulations, with SFRP2+ RetD FB and CCL19+ nonPV FB as novel subpopulations that reside in specific cellular niches and display unique gene expression profiles. SFRP2+ RetD FB and CCL19+ nonPV FB as well as COL8A1+ FB, display altered frequencies in SSc skin and play specific, disease-promoting roles for extracellular matrix release and leukocyte recruitment as revealed by their transcriptional profile, their cellular interactions and ligand-receptor analyses. The frequencies of COL8A1+ FB and their interactions with monocytic cells and B cells are associated with progression of skin fibrosis in SSc. In summary, our spatially-resolved transcriptomic approach identified novel fibroblast subpopulations deregulated in SSc skin with specific pathogenic roles, some of which may potentially serve as biomarkers for progression of skin fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c095f7397487cd95a22547f9e4b6ee14b870509e" target='_blank'>
              Spatially informed phenotyping by cyclic-in-situ-hybridization identifies novel fibroblast populations and their pathogenic niches in systemic sclerosis
              </a>
            </td>
          <td>
            Yi-Nan Li, Tim Filla, A. Györfi, Minrui Liang, Veda Devakumar, Alexandru Micu, Hongtao Chai, C. Bergmann, A. Pecher, Jörg Henes, P. Moinzadeh, Thomas Krieg, Alexander Kreuter, G. Schett, Bernhard Homey, Sascha Dietrich, J. Distler, A. Matei
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Adaptive immunity, mediated by T and B cell responses, is essential for defending against infections and cancers while also being implicated in autoimmune diseases. Tracking T and B cell repertoires in situ at single-cell resolution is essential for understanding adaptive immune responses. To address the lack of tools for in situ single-cell T/BCR (XCR) sequencing, we developed Stereo-XCR-seq, an efficient strategy for retrieving and sequencing TCR and BCR from Stereo-seq cDNA libraries at subcellular resolution. Stereo-XCR-seq provides unbiased full-length XCR reads alongside spatial transcriptomics, enabling the identification of heterogeneous lymphoid aggregates with distinct clonal activities in cancers and inflammatory bowel disease (IBD). We identified plasma cell aggregates that differ from tertiary lymphoid structures (TLSs) in both transcriptomic profiles and clonal activities, with spatial positioning potentially mediating unique immune responses. Collectively, Stereo-XCR-seq enables in situ single-cell profiling of T and B cell clonal activities within tissue microenvironments, providing insights into lymphocyte adaption to environmental stimuli. This technology provides potential for advancing our understanding of tissue immunity and the development of therapeutic strategies for immune disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a087fd6bab927232a0eec4056a80591d4dc5a4" target='_blank'>
              Single cell resolved spatial immune repertoire unveils spatial heterogeneity of lymphoid aggregates in human immune disorders
              </a>
            </td>
          <td>
            Xiaojuan Zhan, Yi Liu, Yanying Guo, Wenwen Zhou, Yixin Yan, Hui Zeng, Xuan Dong, Xiaoyu Chen, Rong Ma, Zhong Liu, Fan Zhu, Xubin Zheng, Xinxing Li, Jinwen Yin, Francis Ka-ming Chan, Chuanyu Liu, Longqi Liu, Xun Xu, Yong Hou, Haoran Tao, Yuliang Dong, Tao Zeng, Young Li, Jingying Zhou, Zexian Zeng, Yu Feng
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="Colonic stem cells have a key role in the continuous regeneration of a healthy intestinal epithelium. Despite considerable advances in single-cell omics technologies, the transcriptional heterogeneity of rare cell types such as colonic stem cells, as well as their functional states and niche-specific behaviors, remain poorly characterised. In this study, we leverage a meta-analysis of scRNA-seq and spatial transcriptomic datasets to comprehensively map the heterogeneity of colonic stem cells. We identify multiple, previously underappreciated stem cell states, including distinct quiescent subtypes marked by CDKN1A (P21), CDKN1B (P27), and CDKN1C (P57), proliferative populations defined by MKI67 (Ki67) and LRIG1, and a lineage-committed intermediate subpopulation expressing MUC2. Strikingly, we find that these states can be robustly identified solely by their extracellular matrix (ECM) gene expression signatures, revealing ECM composition as a critical determinant of stem cell identity. Notably, LAMA1 expression is highly specific to the P57+ quiescent population, linking laminin-mediated microenvironments to the active maintenance of deep quiescence, consistent with our recent findings associating LAMA1 with quiescent cell survival. By applying these ECM gene signatures, we delineate discrete “micro-niches” of quiescent stem cells in healthy tissue and provide evidence that analogous states persist in the colorectal cancer (CRC) environment. Extending our approach to an unrelated tissue, the pancreas, we detect parallel quiescent cell subtypes, illustrating the broader applicability of ECM-based signatures. Taken together, our findings redefine the concept of stem cell heterogeneity in the colon, establish ECM-driven gene signatures as a powerful tool for characterizing stem cell states, and offer new perspectives on the niche-dependent regulation of both healthy and cancerous stem cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b79223cf63c16f2be2c9879ace337f25901bae" target='_blank'>
              ECM Signatures Reveal Quiescent Stem Cell Diversity in the Colonic Niche
              </a>
            </td>
          <td>
            Séamus E. Hickey, M. Andreatta, Christina Enright, Emmanuel Boucrot, Patrick Kiely, Siobhán B. Cashman, Saintiago J. Carmona, Kieran McGourty
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In the field of digestive system tumor research, spatial transcriptomics technologies are used to delve into the spatial structure and the spatial heterogeneity of tumors and to analyze the tumor microenvironment (TME) and the inter-cellular interactions within it by revealing gene expression in tumors. These technologies are also instrumental in the diagnosis, prognosis, and treatment of digestive system tumors. This review provides a concise introduction to spatial transcriptomics and summarizes recent advances, application prospects, and technical challenges of these technologies in digestive system tumor research. This review also discusses the importance of combining spatial transcriptomics with single-cell RNA sequencing (scRNA-seq), artificial intelligence, and machine learning in digestive system cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdf975c83e9f742bc7e19f83183283693f91eae" target='_blank'>
              Application of Spatial Transcriptomics in Digestive System Tumors
              </a>
            </td>
          <td>
            Bowen Huang, Yingjia Chen, S. Yuan
          </td>
          <td>2024-12-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Understanding how individual cancer cells adapt to drug treatment is a fundamental challenge limiting precision medicine cancer therapy strategies. While single-cell technologies have advanced our understanding of cellular heterogeneity, efforts to connect the behavior of individual cells to broader tumor drug responses and uncover global trends across diverse systems remain limited. There is a growing availability of single-cell and bulk omics data, but a lack of centralized tools and repositories makes it difficult to study drug response globally, especially at the level of single-cell adaptation. To address this, we present a multimodal framework that integrates bulk and single-cell treated and untreated transcriptomics data to identify drug responsive cell populations in triple-negative breast cancer (TNBC). Our framework leverages population-scale bulk transcriptomics data from TNBC samples to define seven main “identities”, each representing unique combinations of biologically relevant genes. These identities are dynamic and trackable, allowing us to map them onto single cells and uncover global patterns of how cell populations respond to drug treatment. Unlike static classifications, this approach captures the evolving nature of cellular states, revealing that a select few identities dominate and drive population-level responses during treatment. Crucially, our ability to decode these trends through the inherent noise of single-cell data provides a clearer picture of how heterogeneous cell populations adapt to therapy. By identifying the dominant identities and their dynamics, we can better predict how entire tumors respond to treatment. This insight is essential for designing precise combination therapies tailored to the unique heterogeneity of patient tumors, addressing the single-cell variations that ultimately determine therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="T cell phenotypes and kinetics are emerging as crucial factors associated with immunotherapeutic responses in a wide range of solid cancer types. However, challenges remain in understanding the spatial and temporal profiles of T cells with differential phenotypes due to difficulties in single-cell analysis with preserved tissue structures. Here, we provide an optimized 12-marker multiplex immunohistochemical (IHC) panel and single-cell-based quantitative assessment to identify the spatial distributions of T cell phenotypes in formalin-fixed paraffin-embedded sections. This panel revealed differential T cell populations with spatial localizations in human tonsil tissue, where the percentages of CD8+ T cell-expressing programmed death receptor-1 (PD-1), T cell immunoglobulin and mucin domain 3 (TIM3), and other T cell phenotypic markers vary by tonsillar tissue components such as follicles, parenchyma, and epithelium. A specimen from salivary gland adenocarcinoma during hyper-progression, followed by anti-PD-1 treatment, exhibited the exclusion of CD8+ T cells from the intratumoral regions. Although the vast majority of peritumoral CD8+ T cells exhibited proliferative effector T cell phenotypes with PD-1−TIM3−Ki67+CD45RA+, intratumoral CD8+ T cells showed exhausted phenotypes with PD-1+TIM3− and increased Eomes expression, which might be related to poor therapeutic response in this case. To verify these findings in the context of temporal changes, we analyzed six longitudinal samples from a single patient with maxillary sinus cancer, observing increased T cell exhaustion along with metastasis and progression. Together, highly multiplexed IHC can be applied to analyze the spatiotemporal phenotypes of T cells, potentially contributing to the understanding of the mechanisms of resistance to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3aa6e9e33a49a78ba8f7c54e58b7d68564b492" target='_blank'>
              Applications of Multiplex Immunohistochemistry in Evaluating Spatiotemporal Heterogeneity of T Cells
              </a>
            </td>
          <td>
            Mercedes Machuca-Ostos, Tim de Martines, K. Yoshimura, Junichi Mitsuda, Sumiyo Saburi, Alisa Kimura, Hiroki Morimoto, Koichi Yoshizawa, Nana Sakurai, Nanako Murakami, Kayo Kitamoto, Makoto Yasuda, Y. Sugiyama, H. Ogi, Saya Shibata, Aya Miyagawa-Hayashino, Eiichi Konishi, Kyoko Itoh, Takahiro Tsujikawa, Shigeru Hirano
          </td>
          <td>2025-02-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is the deadliest cancer worldwide. Therapeutic progress stagnate, highlighting the complexity to replicate NSCLC in preclinical models. Drug discovery studies rely mostly on cancer cells in two-dimension (2D), which poorly predict drug efficacy in patients. There is a growing interest in three-dimensional (3D) preclinical models, such as 3D spheroids, to better model tumor phenotype and improve therapeutic prediction. However, a comprehensive view of 3D culture methods impact on transcriptomes, epigenomes and pharmacological responses and their correlations to NSCLC tumors is still missing. Here, we demonstrate that NSCLC spheroids undergo time-dependent transcriptomic and epigenomic changes, which peak after 3 weeks of culture. While DNA methylome remained stable, chromatin methylation and acetylation marks gained features of advanced NSCLC in a time-dependent manner. Single-cell transcriptomic profiling of spheroids demonstrated that time of 3D culture improved the correlation to NSCLC tumors. Moreover, long-term culture of 3D spheroids increased drug screening predictability, by showing resistance to drugs that failed in NSCLC patients (such as HDAC inhibitors) while demonstrating novel pharmacological vulnerabilities and synergistic interactions (such as combination of PRMT and HDAC inhibitors). Strikingly, reverting 3D spheroids back to 2D culture rapidly reversed transcriptomic, epigenetic and pharmacological signatures acquired after 3 weeks of 3D culture, highlighting the critical impact of cell culture conditions on NSCLC phenotype. Collectively, our findings demonstrate that implementing a time-dependent maturation process into 3D spheroid culture induces chromatin and transcriptomic changes that enhance NSCLC preclinical modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing studies have revealed the heterogeneity of cell states present in the rheumatoid arthritis (RA) synovium. However, it remains unclear how these cell types interact with one another in situ and how synovial microenvironments shape observed cell states. Here, we use spatial transcriptomics (ST) to define stable microenvironments across eight synovial tissue samples from six RA patients and characterize the cellular composition of ectopic lymphoid structures (ELS). To identify disease-relevant cellular communities, we developed DeepTopics, a scalable reference-free deconvolution method based on a Dirichlet variational autoencoder architecture. DeepTopics identified 22 topics across tissue samples that were defined by specific cell types, activation states, and/or biological processes. Some topics were defined by multiple colocalizing cell types, such as CD34+ fibroblasts and LYVE1+ macrophages, suggesting functional interactions. Within ELS, we discovered two divergent cellular patterns that were stable across ELS in each patient and typified by the presence or absence of a “germinal-center-like” topic. DeepTopics is a versatile and computationally efficient method for identifying disease-relevant microenvironments from ST data, and our results highlight divergent cellular architectures in histologically similar RA synovial samples that have implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5649472b0dc4cb2ee433d6b1131064e755b734d" target='_blank'>
              Deep topic modeling of spatial transcriptomics in the rheumatoid arthritis synovium identifies distinct classes of ectopic lymphoid structures
              </a>
            </td>
          <td>
            Preethi K. Periyakoil, Melanie H. Smith, Meghana Kshirsagar, Daniel Ramirez, Edward DiCarlo, Susan M. Goodman, Alexander Y. Rudensky, L. Donlin, Christina S. Leslie
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma (ccRCC) is the most common type of kidney cancer, but predicting its behavior remains challenging using standard histopathologic examination. This study introduces a novel approach to predict ccRCC aggressiveness by analyzing the spatial distribution of tumor cells in H&E-stained images. The researchers found that spatial analysis outperformed traditional tumor grading in predicting metastasis, particularly for intermediate-grade tumors. They identified two distinct patient groups based on spatial characteristics, with one group showing greater spatial randomness and a higher association with metastasis. Furthermore, the study revealed a gene expression signature related to the extracellular matrix (matrisome) that correlated with the spatial patterns and aggressive tumor behavior. These findings suggest that analyzing the spatial distribution of ccRCC tumor cells could provide valuable insights into tumor behavior and metastatic potential, potentially improving prognostication and personalized treatment strategies for patients with ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1af154f3c69daea89b74218f36407c18e3de7d" target='_blank'>
              Spatial Distribution of Tumor Cells in Clear Cell Renal Cell Carcinoma Is Associated with Metastasis and a Matrisome Gene Expression Signature
              </a>
            </td>
          <td>
            Prahlad Bhat, P. Tamboli, K. Sircar, Kasthuri Kannan
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Brain metastasis occurs in approximately 16% of metastatic breast cancer patients, and incidence is increasing. Patients with breast cancer brain metastases frequently develop neurological problems including cognitive impairments and seizures. We previously demonstrated accumulation of activated microglia around the tumour site in a breast cancer brain metastasis model, which co-localised with spontaneously occurring local field potential events that resembled interictal epileptic discharges. However, the mechanisms underlying these spatially restricted effects on neuronal excitability are poorly understood. Methods To better understand the underlying cellular changes and potential mechanisms, we used digital spatial profiling of brain cell subtype-specific markers to chart the spatial organisation of neurons, oligodendrocytes, astrocytes, and microglia in relation to metastatic lesion site. We developed a novel chordline-based approach to analyse these spatial changes in biomarker distribution. Results Several protein markers associated with proliferation (Ki67), astrocytes (GFAP) and microglia (CD11b, IBA1, CD45, MSR1) were upregulated in the tumour compared to healthy contralateral brain parenchyma. In contrast, some neuroglial markers (MAP2, Neurofilament light, NeuN, S100B and TMEM119) were lower in the lesion compared to normal tissue. Conclusions Overall, the protein marker changes in the lesion are indicative of neuronal cell loss and changes in the immune microenvironment of the metastatic lesion involving activated microglia, astrocytes and recruitment of peripheral immune cells. Such cellular changes may contribute to the changes in electrical activity resulting from brain metastases observed in mice and patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dbeafc4e0030d90e60f5d024e3f1200854ddc2" target='_blank'>
              Spatial profiling of central nervous system cellular markers in murine breast cancer to brain metastases
              </a>
            </td>
          <td>
            Karen Hogg, Grant Calder, Alastair P. Droop, Ming Yang, Anna Simon, Mark J Hunt, Paul M Kaye, Miles A. Whittington, Sangeeta Chawla, W. Brackenbury
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The existence of cancer stem cells (CSCs) in various tumors has become increasingly clear in addition to their prominent role in therapy resistance, metastasis, and recurrence. For early diagnosis, disease progression monitoring, and targeting, there is a high demand for clinical-grade methods for quantitative measurement of CSCs from patient samples. Despite years of active research, standard measurement of CSCs has not yet reached clinical settings, especially in the case of solid tumors. This is because detecting this plastic heterogeneous population of cells is not straightforward. This review summarizes various techniques, highlighting their benefits and limitations in detecting CSCs from patient samples. In addition, methods designed to detect CSCs based on secreted and niche-associated signaling factors are reviewed. Spatial and single-cell methods for analyzing patient tumor tissues and noninvasive techniques such as liquid biopsy and in vivo imaging are discussed. Additionally, methods recently established in laboratories, preclinical studies, and clinical assays are covered. Finally, we discuss the characteristics of an ideal method as we look toward the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a142badfcf7c429680b2b941eddecbf41e3d46a6" target='_blank'>
              Detection of Cancer Stem Cells from Patient Samples
              </a>
            </td>
          <td>
            Sofia Hakala, Anna Hämäläinen, Sanne Sandelin, Nikolaos Giannareas, Elisa Närvä
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Central Nervous System (CNS) tumors have sophisticated tumor microenvironment (TME) with different cell types such as astrocytes, microglia, neurons, vascular endothelial cells and immune cells. These non-cancerous cells orchestrate the brain TME to regulate cancer progression and therapeutic response. This study aimed to develop a cell composition deconvolution method for CNS tumor and to determine the impact of these cell compositions on patients’ outcomes. Methods We identified the cell type-specific CpG loci using the pairwise differential methylation analysis for 13 major cell types in CNS. Using non-negative least squares (NNLS) methods, we established this cell-type deconvolution approach, MDBrainT, for brain tumors. The predictive accuracy of our MDBrainT model was tested in the DNA methylation profiling of the purified cell samples and compared against another algorithm. Cell composition was predicted by MDBrainT in several brain tumor (glioma, ependymoma, medulloblastoma and ATRT) cohorts and the correlation between cell composition and tumor molecular subtypes and patient outcomes was also assessed. Results Cell type-specific CpG loci for CNS TME was used to build MDBrainT model. Based on these DNA methylation markers, MDBrainT predicted the cell composition in the TME including tumor cells with high accuracy. Endothelial cells were predominately presented in glioblastomas while the percentage of CD8 T cells wassignificantly higher in ATRT. A substantial difference of cell composition was two molecular groups of posterior fossa ependymoma (PFA vs PFB). A higher percentage of cells in TME was usually associated with worse outcomes. Conclusions MDBrainT is a robust algorithm for cell composition prediction for brain TME. Cell composition in brain TME is distinct across different pathological types and molecular subtypes. Key Points MDBrainT is a robust DNA methylation-based deconvolution approach for brain tumor microenvironment (TME). Different molecular subtypes of brain tumors have distinct cell composition patterns. Cell composition in brain TME informs patients of outcomes. Importance of this study DNA methylation is a cell type specific marker that has been utilized for tumor molecular diagnosis, disease progression and therapeutic monitoring. A DNA methylation-based classifier for brain tumors precisely predicts the molecular subtypes but not the cell composition of tumor microenvironment. Brain tumors are a complex cell mixture where tumor microenvironment is critical for tumorigenesis and therapeutic resistance. Here, we developed a novel deconvolution approach (MDBrainT) to predict cell composition for brain tumors. Our model has revealed the heterogeneity of cell composition between tumor types. In addition, tumors with different molecular subtypes have distinct cell composition. Cell percentage in TME also informs patients of outcomes. The tumor microenvironment including cell composition of each patient may direct the different regimen of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1823256c94dd6a377f1b3165f4ae951356ef07" target='_blank'>
              DNA Methylation-Based Cell Type Deconvolution Reveals the Distinct Cell Composition in Brain Tumor Microenvironment
              </a>
            </td>
          <td>
            Feili Liu, Jin Qian, Chenkai Ma
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="BACKGROUND
Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity.


METHODS
We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types-CXCR3+CD68+ or CD8+-to profile their spatial relationships with CXCL9+CD68+ cells.


RESULTS
Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely 'cold, tumor-rich' and 'cold, stroma-rich', exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8.


CONCLUSIONS
Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment.
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-18</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (iCCA) may be genomically subclassified by the presence of potentially actionable molecular aberrations, of which pathogenic alterations in isocitrate dehydrogenase (IDH)1 and fibroblast growth factor receptor (FGFR)2 are the most frequently observed. The impact of these molecular alterations on the tumor immune microenvironment remains incompletely understood. Methods: We performed a high-parameter spatial immune phenotyping of iCCA samples with pathogenic FGFR2 or IDH1 alterations and FGFR2/IDH1 wild-type controls at the single-cell level using CO-Detection by indEXing. Results:﻿ A total of 24 tumors were examined. Tumors with FGFR2 alterations were characterized by fewer CD8+ T cells and “M2-like” macrophages but higher levels of polymorphonuclear myeloid-derived suppressor cells as compared to FGFR2 wild-type tumors. Spatial relationships between polymorphonuclear myeloid-derived suppressor cells and multiple other cell types in the tumor microenvironment (including tumor cells, CD4+, and CD8+ T cells) were enriched in tumors with FGFR2 alterations. Tumors with IDH1 mutations had a trend toward more fibroblasts and were characterized by a closer proximity of tumor cells to CD4+ T cells, and between macrophages and multiple structural tumor microenvironment components as compared to other subtypes. Conclusions: iCCAs with pathogenic FGFR2 fusions/rearrangements and IDH1 mutations have distinct immunophenotypes. Tailoring immunotherapeutic approaches to specific molecular subsets could improve treatment outcomes across the divergent molecularly defined iCCA subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343c87af8f0d2b156de3f86c823598f7aa36e2b7" target='_blank'>
              Deep immune profiling of intrahepatic cholangiocarcinoma with CODEX multiplexed imaging
              </a>
            </td>
          <td>
            M. Baretti, Soumya Shekhar, Vaibhav Sahai, Daniel H. Shu, Kathryn Howe, Valerie Gunchick, N. Assarzadegan, Emma Kartalia, Qingfeng Zhu, Elsa Hallab, Archit Sheth-Shah, Aya Kondo, Nilofer S Azad, M. Yarchoan
          </td>
          <td>2025-02-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Purpose Eyelid infiltrative basal cell carcinoma (iBCC) is the most common malignant tumor affecting the ocular adnexa, but studies on metabolic changes within its microenvironment and heterogeneity at the tumor invasive area are limited. This study aims to analyze metabolic differences among iBCC cell types using single-cell and spatial metabolomics analysis and to examine metabolic environment at the tumor invasive area. Methods Single-cell transcriptomic data of human basal cell carcinoma (BCC) were clustered and visualized using Uniform Manifold Approximation and Projection. Metabolic reprogramming was analyzed with single-cell flux estimation analysis. Spatial metabolomics data were obtained with the Timstof Flex MALDI 2 system, and Bruker software was used for region selection. Results Eight cell types were identified within the iBCC microenvironment. Differences between inflammatory cancer-associated fibroblasts and myofibroblastic cancer-associated fibroblasts were analyzed. Metabolic flux analysis showed increased glycolysis, glutamine, heme, and glutathione fluxes in the iBCC microenvironment. Spatial metabolomics revealed high levels of taurine, deoxy-GMP, O-phosphoethanolamine, and pyrithione. Both tumor and invasive regions had significant upregulation of fatty acid pathways, with marked increases in oleic and arachidonic acids at the invasive area. Specific upregulation of UDP-glucuronic acid and high UDP-glucose 6-dehydrogenase (UGDH) expression in the tumor region suggest UXS1 as a potential therapeutic target for iBCC. Conclusions This study establishes a metabolic microenvironment atlas of iBCC, revealing significant metabolic differences and the dominance of lipid and lysosome metabolism. Potential metabolic markers and characteristic substances in the invasive area offer new insights for immunotherapy and the exploration of BCC’s metabolic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322ca4366508114a20cf4530c76f41a9f0ca02cf" target='_blank'>
              Metabolic Atlas of Human Eyelid Infiltrative Basal Cell Carcinoma
              </a>
            </td>
          <td>
            Yanjing Huang, Chengjie He, Qiuling Hu, Zhong Liu, Xingyi Li, Wuyou Gao, Xuanwei Liang, Rongxin Chen, Zhen Mao, Xianchai Lin
          </td>
          <td>2025-01-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3852f7c6358b758c264d481b7403f1d4972dcd97" target='_blank'>
              Immunotherapy in ovarian cancer: spatial functional genomics to unravel resistance mechanisms
              </a>
            </td>
          <td>
            Martina Rausch, Karlotta Bartels, Josef Leibold
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells, immune cells, and stromal cells within the tumor microenvironment (TME) collaboratively influence disease progression and therapeutic responses. The nutrient-limited conditions of the TME, particularly the scarcity of glucose, amino acids, and lipids, challenge cancer cell survival1–4. However, the metabolic constraints faced by immune and stromal cells in comparison to cancer cells, and how these limitations affect therapeutic outcomes, remain poorly understood. Here, we introduce Dual Ribosome Profiling (DualRP), a method that allows for simultaneous analysis of translation and identification of ribosome stalling, revealing amino acid shortages in different cell types within tumors. Using DualRP, we uncover that interactions between cancer cells and fibroblasts trigger an inflammatory response, mitigating amino acid limitations during glucose starvation. In immunocompetent mouse models, we observe that immune checkpoint blockade therapy induces serine and glycine restrictions specifically in T cells, but not in cancer cells. We further demonstrate that these amino acids are essential for optimal T cell function both in vitro and in vivo, highlighting their critical role in effective immunotherapy. Our findings show that therapeutic interventions create distinct metabolic demands across different tumor cell types, with nutrient availability significantly influencing the success of immunotherapy. DualRP’s ability to explore cell type-specific metabolic vulnerabilities offers a promising tool for advancing our understanding of tumor biology and improving therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb1f538143f57c46c3d2552aff719b62450477f" target='_blank'>
              Dual Ribosome Profiling reveals metabolic limitations of cancer and stromal cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Daniela Aviles-Huerta, Del Pizzo Rossella, Alexander Kowar, Ali Hyder Baig, Giuliana Palazzo, Ekaterina Stepanova, C. C. Amaya Ramirez, Sara D’Agostino, Edoardo Ratto, Catarina Pechincha, Nora Siefert, Helena Engel, Shangce Du, Silvia Cadenas-De-Miguel, Beiping Miao, Victor Cruz Vilchez, Karin Müller-Decker, Ilaria Elia, Chong Sun, Wilhelm Palm, Fabricio Loayza-Puch
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Metabolic processes are crucial in immune regulation, yet the impact of metabolic heterogeneity on immunological functions remains unclear. Integrating metabolomics into immunology allows the exploration of the interactions of multilayered features in the biological system and the molecular regulatory mechanism of these features. To elucidate such insight in lung squamous cell carcinoma (LUSC), we analyzed 106 LUSC tumor tissues. We performed high-resolution matrix-assisted laser desorption/ionization (MALDI) mass spectrometry imaging (MSI) to obtain spatial metabolic profiles, and immunohistochemistry to detect tumor-infiltrating T lymphocytes (TILs). Unsupervised k-means clustering and Simpson’s diversity index were employed to assess metabolic heterogeneity, identifying five distinct metabolic tumor subpopulations. Our findings revealed that TILs are specifically associated with metabolite distributions, not randomly distributed. Integrating a validation cohort, we found that heterogeneity-correlated metabolites interact with CD8+ TIL-associated genes, affecting survival. High metabolic heterogeneity was linked to worse survival and lower TIL levels. Pathway enrichment analyses highlighted distinct metabolic pathways in each subpopulation and their potential responses to chemotherapy. This study uncovers the significant impact of metabolic heterogeneity on immune functions in LUSC, providing a foundation for tailoring therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b4c98251a8ac1956e7ccdfa63c51218dbe1751b" target='_blank'>
              Metabolic heterogeneity in tumor cells impacts immunology in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Qian Wang, Na Sun, Chaoyang Zhang, Thomas Kunzke, P. Zens, A. Feuchtinger, Sabina Berezowska, Axel Walch
          </td>
          <td>2025-02-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background: Tumors, as intricate ecosystems, comprise oncocytes and the highly dynamic tumor stroma. Tumor stroma, representing the non-cancerous and non-cellular composition of the tumor microenvironment (TME), plays a crucial role in oncogenesis and progression, through its interactions with biological, chemical, and mechanical signals. This review aims to analyze the challenges of stroma mimicry models, and highlight advanced personalized co-culture approaches for recapitulating tumor stroma using patient-derived tumor organoids (PDTOs). Methods: This review synthesizes findings from recent studies on tumor stroma composition, stromal remodeling, and the spatiotemporal heterogeneities of the TME. It explores popular stroma-related models, co-culture systems integrating PDTOs with stromal elements, and advanced techniques to improve stroma mimicry. Results: Stroma remodeling, driven by stromal cells, highlights the dynamism and heterogeneity of the TME. PDTOs, derived from tumor tissues or cancer-specific stem cells, accurately mimic the tissue-specific and genetic features of primary tumors, making them valuable for drug screening. Co-culture models combining PDTOs with stromal elements effectively recreate the dynamic TME, showing promise in personalized anti-cancer therapy. Advanced co-culture techniques and flexible combinations enhance the precision of tumor-stroma recapitulation. Conclusions: PDTO-based co-culture systems offer a promising platform for stroma mimicry and personalized anti-cancer therapy development. This review underscores the importance of refining these models to advance precision medicine and improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a895a0492110694c74a1077998965d537640b393" target='_blank'>
              Facilitation of Tumor Stroma-Targeted Therapy: Model Difficulty and Co-Culture Organoid Method
              </a>
            </td>
          <td>
            Qiu-Shi Feng, Xiao-Feng Shan, Vicky Yau, Z. Cai, Shang Xie
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 224


 Background:
 The tumor-immune microenvironment (TME) is crucial in the progression of prostate cancer (PCa), yet little is known about its spatial organization and immune composition in treatment-naïve, localized PCa. While immunotherapy has had limited success in metastatic PCa, its efficacy in localized disease remains largely unknown. This study aims to dissect tumor-immune interactions in primary PCa and explore differences in immune landscape between low- and high-Gleason PCa, thus providing biologic insights for immunotherapy actionability.
 Methods:
 We performed spatial profiling of radical prostatectomy (RP) specimens from 28 treatment-naïve prostate cancer patients. These included 14 patients with low-grade disease (Gleason Grade Group 1–2) and 14 patients with high-grade disease (Gleason Grade Group 4–5). Using multiplex cyclic immunofluorescence (CyCIF) and a 27-marker panel, we obtained single-cell resolution across whole-slide images. Tumor and stromal compartments were manually annotated, and immune cell densities and neighborhood were analyzed and compared between low- and high-grade tumors. Spatial analysis focused on identifying immune clusters (IC -suggestive of tertiary lymphoid structures -TLS) and quantified the degree of immune infiltration along with characterization of T cell exhaustion markers such as CD8+PD1+TCF1+ cells.
 Results:
 High-grade tumors exhibited significantly higher infiltration of CD8+ T cells and CD20+ B cells compared to low-grade tumors. Spatial profiling revealed organized ICs, especially in high-grade tumors, which exhibited characteristics consistent with TLS. These ICs were enriched with precursor-exhausted CD8+PD1+TCF1+ T cells (TPEX), suggesting a state of immune exhaustion that may be therapeutically reactivatable. The presence of TLS and immune exhaustion markers in high-grade tumors highlights a more active immune microenvironment than previously appreciated in primary prostate cancer.
 Conclusions:
 Our findings reveal that a subset of high-grade prostate cancers harbor an organized, immune-infiltrated tumor microenvironment, including adaptive immune features such as TLS and CD8+ T cell exhaustion. These immune landscapes, marked by significant CD8+ and CD20+ cell infiltration and spatial clustering, suggest that high-grade prostate tumors may be amenable to immune-based therapies, despite the failure of immune checkpoint blockade in metastatic PCa. These results raise the hypothesis that immunotherapy might be effective in treatment-naïve high-grade prostate cancer. Future efforts should explore whether these immune profiles can be leveraged for risk stratification or as biomarkers for response to immunotherapy in advanced disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d9a96ad448410a624d4884f42c080b637c2227" target='_blank'>
              Whole-slide multiplexed tissue imaging of prostate adenocarcinoma to investigate immune niches in high- versus low-grade tumors.
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, B. Labadie, Aishwarya Atmakuri, K. Chaudagar, Eamon Toye, Andréanne Gagné, , E. Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Systemic sclerosis (SSc) is a prototypical fibrotic disease with high mortality and limited treatment options. Despite advances in single-cell RNA sequencing (scRNA-seq), the comprehensive understanding of cellular heterogeneity and cell-cell interaction within the fibrogenesis microenvironment remains limited. We generated spatially resolved transcriptome maps from healthy and SSc skin and built a scRNA-seq atlas to map the single-cell data to spatial space. This enabled us to identify a fibrotic niche, enriched with fibroblasts and macrophages, which is significantly expanded in SSc and correlated with clinical outcome. We revealed disease-specific cell states of fibroblasts and macrophages, and evaluated their spatial dependency on other cell types. We identified selective expression of ACKR3 in fibroblast progenitors that diminishes with SSc progression, which may serve to regulate CXCL12/CXCR4-mediated macrophage recruitment and fibrotic remodeling. Together, we provided an in-depth description at cellular and spatial levels of fine-tuned regulatory events occurring in SSc, offering spatiotemporal insights. One Sentence Summary Integrated spatial omics provide insight into the cellular and transcriptional landscape in spatially distinct microenvironments, which may drive fibrosis progression in SSc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c9c1b45b9d3f07418bc3c35d63c359a3709dc2" target='_blank'>
              Mapping spatially-resolved transcriptomes in systemic sclerosis
              </a>
            </td>
          <td>
            Zhijian Li, A. R. Rigau, Wenjie Xie, Linlin Huang, Xiaohang Shao, Yi-Nan Li, A. Matei, Wenjing Ye, Hejian Zou, Luca Pinello, J. H. Distler, Rui He, Minrui Liang
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression.
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) occurs heterogeneously among malignant carcinoma cells to promote chemoresistance. Identifying the signaling pathways involved will nominate drug combinations to promote chemoresponse, but cell population-level studies are inherently fraught, and single-cell transcriptomics are limited to indirect ontology-based inferences. To understand EMT heterogeneity at a signaling protein level, we combined iterative indirect immunofluorescence imaging of pancreas cancer cells and tumors and mutual information (MI) modeling. Focusing first on MAP kinase pathways, MI predicted that cell-to-cell variation in ERK activity surprisingly dominated control of EMT heterogeneity in response to diverse growth factors and chemotherapeutics, but that JNK compensated when MEK was inhibited. Population-level models could not capture these experimentally validated MI predictions. The dominant role of ERK was predicted by MI even when analyzing seven potential EMT-regulating signaling nodes. More generally, this work provides an approach for studying highly multivariate signaling/phenotype relationships based on protein measurements in any setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a6514737cd599ba0d5ecbb0df4e4d91b74a1b1" target='_blank'>
              ERK plays a conserved dominant role in pancreas cancer cell EMT heterogeneity driven by diverse growth factors and chemotherapies
              </a>
            </td>
          <td>
            Michelle C. Barbeau, Brooke A. Brown, S. Adair, Todd W. Bauer, Matthew J Lazzara
          </td>
          <td>2025-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Metastasis, the leading cause of cancer-related deaths, involves a complex cascade of events, including extravasation. Despite extensive research into metastasis, the mechanisms underlying extravasation remain unclear. Molecular targeted therapies have advanced cancer treatment, yet their efficacy is limited, prompting exploration into novel therapeutic targets. Here, we showed the association of polyploidy in MDA-MB-231 breast cancer cells and their extravasation, using microfluidic systems to reproduce the in vivo microvascular environment. We observed enhanced extravasation in polyploid cells alongside upregulated expression of genes involved in cell-substrate adhesion and cell mechanical dynamics. These findings offer insights into the relationship between polyploidy and extravasation, highlighting potential targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785941742bc350defbe67d5915862c497379ab77" target='_blank'>
              Polyploidy of MDA-MB-231 cells drives increased extravasation with enhanced cell-matrix adhesion
              </a>
            </td>
          <td>
            Satomi Hirose, Tatsuya Osaki, Roger D. Kamm
          </td>
          <td>2025-01-29</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) are pivotal in tumor growth, recurrence, and resistance to standard therapies. This review delves into CSCs’ ability to self-renew and differentiate, fueling tumor heterogeneity and metastasis. It highlights critical signaling pathways like Wnt, Notch, and Hedgehog, which are essential for targeting CSCs. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, promise to enhance CSC research and management. The review also explores the complex interaction between CSCs and the tumor microenvironment (TME), which influences therapeutic effectiveness. Addressing CSC-driven resistance, the article proposes advanced strategies for overcoming these challenges. It emphasizes the role of artificial intelligence in designing personalized, CSC-targeted therapies, ushering in a new era for precision medicine in oncology. This comprehensive approach aims to improve cancer treatment outcomes through innovative technologies and inter-disciplinary collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Invasion is a prerequisite for metastasis formation. During tumor development, the extracellular matrix (ECM) is remodeled in part through overexpression of type I collagen, increasing tumor microenvironment stiffness, and facilitating cancer dissemination. During breast cancer cell migration, we observed membrane debris left behind, attached to the collagen fibrils, along the migration path. We named these structures collagen-tracks. These collagen-tracks can be deposited in 3D matrices in vitro and in vivo and their formation is stimulated by the interaction between the ECM and matrix receptors, such as the discoidin-domain receptor (DDR1). However, they are different from structures already known to be involved in cell-cell communication such as exosomes and migrasomes, due to their specific nucleic acid and protein contents. When deposited by highly invasive breast cancer cells, internalized collagen-tracks reprogram non-invasive cells into highly pro-metastatic ones by inducing a partial epithelial–mesenchymal transition (EMT). This cell reprogramming is dependent on specific miRNAs present in the collagen-tracks, that are necessary to promote ECM degradation, increase cell motility and invasiveness. Collagen-tracks thus represent a new form of cell-cell communication important for driving tumor invasion that could be targeted to prevent metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607709bacd540192f2a53b41a2c4eccacb7b266e" target='_blank'>
              Cancer cells transfer invasive properties through microRNAs contained in collagen-tracks
              </a>
            </td>
          <td>
            Léa Normand, L. Rouyer, Elodie Richard, Nathalie Allain, Julie Giraud, Sylvaine Di-Tommaso, C. Dourthe, A. Raymond, Jean-William Dupuy, Kevin Moreau, Richard Iggo, Gaetan McGrogan, Nathalie Dugot-Senant, Anthony Bouter, Sisareuth Tan, Alexandre Favereaux, Violaine Moreau, R. Luco, M. Ros, F. Saltel
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the liver’s recognized regenerative potential, the role of the hepatic ductal cells (a.k.a. biliary epithelial cells), its heterogeneity, and functionality remain incompletely understood in this process. This study provides a comprehensive examination of the molecular and cellular mechanisms underpinning liver ductal development and liver regeneration in zebrafish, with a spotlight on the functional roles of her family genes in these processes. Using state-of-the-art knock-in zebrafish models and single-cell transcriptomics we reveal the differential expression patterns of the different her genes, of which her2, her6, and her9, were identified as specific molecular signatures for distinguishing different ductal cell types with unique morphology and spatial distribution. Particularly, her9 serves as a pan-ductal marker and shows responsiveness to the synergistic effect of Notch and BMP signaling. By analyzing multiple single-cell RNA-seq datasets, we identify numerous ductal markers which are functional proteins for ductal integrity, and most notably CRISPR mutagenesis demonstrates that her9 is essential for hepatocyte recovery. Using multiple transgenic and knock-in zebrafish lines and genetic fate mapping, we provide a detailed characterization of the ductal remodeling process under development and extreme loss of intra-hepatic duct, highlighting the remarkable ductal cell plasticity. Single-cell transcriptomics of lineage-traced her9-expressing liver ducts in static and regenerative states uncover distinct cell clusters with unique molecular signatures and morphology, reflecting the liver’s regenerative dynamics and highlight relevant key biological processes that could be leveraged to expedite liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54f77860204b564968d456274cf2fc012ddabcb" target='_blank'>
              Deciphering the zebrafish hepatic duct heterogeneity and cell plasticity using lineage tracing and single-cell transcriptomics
              </a>
            </td>
          <td>
            J. Mi, Lipeng Ren, Ka-Cheuk Liu, Lorenzo Buttò, Daniel Colquhoun, Olov Andersson
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="To overcome the paucity of known tumor-specific surface antigens in pediatric high-grade glioma (pHGG), we contrasted splicing patterns in pHGGs and normal brain samples. Among alternative splicing events affecting extracellular protein domains, the most pervasive alteration was the skipping of ≤30 nucleotide-long microexons. Several of these skipped microexons mapped to L1-IgCAM family members, such as NRCAM. Bulk and single-nuclei short- and long-read RNA-seq revealed uniform skipping of NRCAM microexons 5 and 19 in virtually every pHGG sample. Importantly, the Δex5Δex19 (but not the full-length) NRCAM proteoform was essential for pHGG cell migration and invasion in vitro and tumor growth in vivo. We developed a monoclonal antibody selective for Δex5Δex19 NRCAM and demonstrated that “painting” of pHGG cells with this antibody enables killing by T cells armed with an FcRI-based universal immune receptor. Thus, pHGG-specific NRCAM and possibly other L1-IgCAM proteoforms are promising and highly selective targets for adoptive immunotherapies. Statement of significance Existing targets for chimeric antigen receptors (CAR)-armed T cells are often shared by CNS tumors and normal tissues, creating the potential for on-target/off-tumor toxicities. Here we demonstrate that in CNS tumors of glial origin, cell adhesion molecules have alternatively spliced proteoforms, which could be targeted by highly selective therapeutic antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e54241611f14a49622f3cf892c1a58f5571d1d7" target='_blank'>
              Neuronal cell adhesion molecule (NRCAM) variant defined by microexon skipping is an essential, antigenically distinct, and targetable proteoform in high-grade glioma
              </a>
            </td>
          <td>
            Priyanka Sehgal, Ammar S. Naqvi, Makenna Higgins, Jiageng Liu, Kyra Harvey, Julien Jarroux, Taewoo Kim, Berk Mankaliye, Pamela Mishra, Grace Watterson, Justyn Fine, Jacinta Davis, Katharina E. Hayer, Annette Castro, Adanna Mogbo, Charles Drummer, Dan Martinez, Mateusz Koptyra, Zhiwei Ang, Kai Wang, Alvin Farrel, Mathieu Quesnel-Vallières, Yoseph Barash, Jamie B. Spangler, J. L. Rokita, A. Resnick, Hagen U. Tilgner, Thomas DeRaedt, Daniel J Powell, A. Thomas-Tikhonenko
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Purpose
In glioblastoma, the therapeutically intractable and resistant phenotypes can be derived from glioma stem cells, which often have different underlying mechanisms from non-stem glioma cells. Aberrant signaling across the EGFR-PTEN-AKT-mTOR pathways have been shown as common drivers of glioblastoma. Revealing the inter and intra-cellular heterogeneity within glioma stem cell populations in relations to signaling patterns through these pathways may be key to precision diagnostic and therapeutic targeting of these cells.


Materials and Methods
Single cell parallel proteomic heterogeneity profiling of the EGFR-PTEN-AKT-mTOR pathways was conducted in a panel of fifteen glioma stem cell models derived from patient glioblastoma biopsies.


Results
The analysis included 59,464 data points from 14,866 cells and identified forty-nine molecularly distinct signaling phenotypes. High content bioinformatics resolved two unique patient clusters diverging on EGFR expression and AKT/TORC1 activation. Phenotypic validation indicated drug responsive phenotypes to EGFR blocking in the high EGFR expressing cluster with lower tumor initiating potential in comparison to the AKT/TORC1 activated cluster. High EGFR expression trended with improved patient prognosis while AKT/TORC1 activated samples trended with poorer patient outcomes. Genetic heterogeneity was observed in both clusters with proneural, classical and mesenchymal subtypes observed.


Conclusion
Quantitative single cell heterogeneity profiling reveals divergent EGFR-PTEN-AKT-mTOR pathways of patient derived glioma stem cells, which would inform future research and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0984500cc5c038ee7f954e1d13785419da86e8c6" target='_blank'>
              Single-Cell Heterogeneity of EGFR Pathway Is Linked to Unique Signatures of Drug Response and Malignancy in Patient Derived Glioblastoma Stem Cells.
              </a>
            </td>
          <td>
            Michael D. Masterman-Smith, Nicholas A. Graham, E. Panosyan, J. Mottahedeh, Eric R. Samuels, Araceli Nunez, Tiffany Phillips, Meeryo Choe, T. Cloughesy, Jorge A. Lazareff, Linda M. Liau, William H. Yong, Thomas G. Graeber, H. Kornblum, Ming-Fei Lang, Yanzhao Li, Jing Sun
          </td>
          <td>2025-01-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fef18e249838665646bfdb8d1805895bb4b7093" target='_blank'>
              Characterization of canine tumor-infiltrating leukocyte transcriptomic signatures reveals conserved expression patterns with human osteosarcoma
              </a>
            </td>
          <td>
            Dylan T. Ammons, R. A. Harris, L. Chow, Steven Dow
          </td>
          <td>2025-02-11</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is characterized by a limited infiltration of tumor-specific T cells and anti-tumor T cell activity. Extracellular factors in the PDAC TME have been widely reported to mediate immune suppression, but the contribution from tumor-intrinsic factors is not well understood. The RNA-binding protein, HuR (ELAVL1), is enriched in PDAC and negatively correlates with T cell infiltration. In an immunocompetent Kras-p53-Cre (KPC) orthotopic model of PDAC, we found that genetic disruption of HuR impaired tumor growth. Importantly, we found that HuR depletion in tumors enhanced both T cell number and activation states. Mechanistically, HuR mediated the stabilization of mTOR pathway transcripts, and inhibition of mTOR activity rescued the impaired function of local T cells. Phenotypically, we found that HuR induced T-cell suppression in PDAC, as HuR depletion sensitize PDAC tumors to immune checkpoint blockade, while isogenic, wildtype tumors are resistant. Our findings describe a novel role of HuR in facilitating tumor immune suppression in the PDAC TME by inhibiting T cell infiltration and function, and implicate HuR inhibition as a potential therapeutic combination with immunotherapy. SIGNIFICANCE This study identified a novel mechanism that HuR supports pancreatic tumor growth by restricting T cell infiltration, promoting immune evasion. Our work supports targeting strategies against HuR in PDAC with the goal of enhancing PDAC sensitivity to immune-based cancer therapies, such as checkpoint blockade and T cell transfer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5ca2919cbf47a322acf524c95727ed063316d71" target='_blank'>
              mRNA stability factor HuR promotes immune evasion in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yifei Guo, Jennifer M. Finan, Alexandra Q. Bartlett, Shamaline Sivagnanam, Katie E. Blise, Nell Kirchberger, Konjit Betre, Kevin Hawthorne, Canping Chen, Aaron J. Grossberg, Zheng Xia, L. Coussens, Rosalie C. Sears, Jonathan R. Brody, Robert Eil
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Gamma-delta (γδ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8afd5989cd7e1d2eeffdf0537754b1eeb0da52a8" target='_blank'>
              SMORE: spatial motifs reveal patterns in cellular architecture of complex tissues
              </a>
            </td>
          <td>
            Zainalabedin Samadi, Kai Hao, Amjad Askary
          </td>
          <td>2025-01-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1332adb107deebc84becc80e062e225cebcbced2" target='_blank'>
              Turning attention to tumor–host interface and focus on the peritumoral heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Fang Wang, Jiawei Dong, Yuyun Xu, Jiaqi Jin, Yan Xu, Xiuwei Yan, Zhihui Liu, Hongtao Zhao, Jiheng Zhang, Nan Wang, Xueyan Hu, Xin Gao, Lei Xu, Chengyun Yang, Shuai Ma, Jianyang Du, Ying Hu, Hang Ji, Shaoshan Hu
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="L1TD1 is a pluripotency factor required for embryonic stem cell self-renewal; its expression has also been detected in solid tumors, including embryonal tumors of the central nervous system (CNS). Previously, we showed that L1TD1 expression correlates with metastasis formation and shorter overall survival of medulloblastoma patients. Here, we used affinity purification coupled to mass spectrometry to map the L1TD1 interactome, and global proteomics to assess proteins differentially regulated by L1TD1 expression in patient-derived embryonal CNS tumor cell lines. We identified novel L1TD1 interactors and differentially expressed proteins related to cell proliferation, death and motility. Finally, we demonstrated that L1TD1-overexpressing tumor cells have distinct cell morphology with enhanced filopodial formation, higher cell motility, greater proliferation capability, and reduced sensitivity to cisplatin treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0488d6a2ccbb080a6ebfdc837df22774e9cf9" target='_blank'>
              Integrated interactome, proteomic and functional analyses reveal molecular pathways driving L1TD1-induced aggressiveness in CNS embryonal tumor cells.
              </a>
            </td>
          <td>
            T. Mitsugi, Matt Sherwood, Elisa Helena Farias Jandrey, Amanda Faria Assoni, Joseph Bell, Brandon Coke, Paul Skipp, Rob M. Ewing, Oswaldo Keith Okamoto
          </td>
          <td>2025-02-06</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Resistance to tumor necrosis factor inhibitor (TNFi) therapy remains a major obstacle in managing ulcerative colitis (UC), affecting up to 50% of patients.1 The complex interplay of immune cells and their spatial organization in the gut mucosa is poorly understood but likely contributes to therapeutic outcomes.1,2 This study utilized advanced multiplexed imaging to construct a spatial atlas of UC, providing insights into cellular interactions driving TNFi resistance. Novel spatial omic technologies show to be able to add additional thus far unknown insights to this question.



 A total of 42 tissue regions from 29 UC patients and 5 healthy controls were analyzed using CODEX multiplexed imaging. Spatially resolved single-cell data, encompassing over 1.7 million cells, were used to identify distinct cell types and map cellular neighborhoods (CNs). Pairwise cell-cell contacts and CN-level functional states were analyzed in TNFi-treated and untreated patients, correlating findings with clinical outcomes.



 TNFi therapy led to normalization of adaptive immune CNs, such as lymphoid aggregates, with reductions in T-cell frequencies and associated contacts. However, innate immune CNs, including neutrophil-enriched granulocyte niches and inflamed stroma, remained unchanged despite treatment. Notably, neutrophil-dominated CNs exhibited persistent expression of pro-inflammatory markers, such as TNFR2 and CD137, implicating these niches in sustaining mucosal inflammation and therapy resistance. Additionally, sex-dependent differences were observed, with higher TNFi responsiveness in females potentially linked to adaptive immune modulation.3



 Neutrophil-dominated inflammatory niches represent a potential mechanism of TNFi resistance in UC.3,4 These findings highlight the importance of targeting innate immune pathways alongside TNFi therapy to improve outcomes. By integrating spatial insights with functional data, this study paves the way for precision medicine approaches in UC management.



 1.R. Ungaro, S. Mehandru, P. B. Allen, L. Peyrin-Biroulet, J.-F. Colombel, Ulcerative colitis.Lancet 389, 1756–1770 (2017).
 2.R. Gaujoux, E. Starosvetsky, N. Maimon, F. Vallania, H. Bar-Yoseph, S. Pressman, R. Weisshof,I. Goren, K. Rabinowitz, M. Waterman, H. Yanai, I. Dotan, E. Sabo, Y. Chowers, P. Khatri,S. S. Shen-Orr; Israeli IBD research Network (IIRN), Cell-centred meta-analysis revealsbaseline predictors of anti-TNFα non-response in biopsy and blood of patients with IBD.Gut 68, 604–614 (2019).
 3.A.T. Mayer, D.R. Holman, A. Sood, U. Tandon, S. S. Bhate, S. Bodapati, G. L. Barlow, J. Chang, S. Black, E. C. Crenshaw, A. N. Koron, S. E. Streett, S. S. Gambhir SS, W. J. Sandborn, B. S. Boland, T. Hastie, R. Tibshirani, J. T. Chang, G. P. Nolan, C. M. Schürch, S. Rogalla “A tissue atlas of ulcerative colitis revealing evidence of sex-dependent differences in disease-driving inflammatory cell types and resistance to TNF inhibitor therapy” Sci Adv. 2023 Jan 20;9(3):eadd1166. doi: 10.1126/sciadv.add1166. Epub 2023 Jan 20.
 4.S. T. Eshghi, J. M. Gubatan, P. Mazrooei, L. Quintanilla, A. Nguyen, A. Au-Yeung, D. R. Holman, C. Takashi, C. Schiffman, B. O’Gorman, M. Keir, S. Ramanujan, S. Rogalla, J. A. Hackney, J. M McBride “Molecular characterization of response to Etrolizumab and anti-TNF reveals treatment resistance in ulcerative colitis is associated with abundance of residual neutrophil subsets and inflammatory fibroblast populations” https://doi.org/10.1101/2024.07.02.601267

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391837f71bfd5dab1136ff60f107c81956417fbb" target='_blank'>
              OP23 Neutrophil Niches and Therapy Resistance in Ulcerative Colitis: Spatial Atlas Insights
              </a>
            </td>
          <td>
            D. Holman, S. Rogalla
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed580d09e5d8a50632bdd53ac296d7f45feff0d1" target='_blank'>
              The microenvironment cell index is a novel indicator for the prognosis and therapeutic regimen selection of cancers
              </a>
            </td>
          <td>
            Xian-Yan Yang, Nian Chen, Qian Wen, Yu Zhou, Tao Zhang, Ji Zhou, Cheng-Hui Liang, Li-Ping Han, Xiao-Ya Wang, Qingmei Kang, Xiaomei Zheng, Xue-Jia Zhai, Hong-Ying Jiang, Tian-Hua Shen, Jin-Wei Xiao, Yu-Xin Zou, Yun Deng, Shuang Lin, Jiang-jie Duan, Jun Wang, Shicang Yu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c4186232b1945af1f5d2a2e6f5e0453c760e7a" target='_blank'>
              Periostin-mediated NOTCH1 activation between tumor cells and HSCs crosstalk promotes liver metastasis of small cell lung cancer
              </a>
            </td>
          <td>
            Linlin Lou, Keren Peng, Shumin Ouyang, Wen Ding, Jian-shan Mo, Jiayu Yan, Xiaoxiao Gong, Guopin Liu, Jin-Jian Lu, Peibin Yue, Kai Zhang, Jian Zhang, Yan-dong Wang, Xiao-lei Zhang
          </td>
          <td>2025-01-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2a730533f465c1760e5f39354ae03ee2bf1e3c" target='_blank'>
              Spatial expression of fibroblast activation protein-α in clear cell renal cell carcinomas revealed by multiplex immunoprofiling analysis of the tumor microenvironment
              </a>
            </td>
          <td>
            G. Larrinaga, M. Redrado, Ana Loizaga-Iriarte, Amparo Pérez-Fernández, Aida Santos-Martín, Javier C. Angulo, José A Fernández, Alfonso Calvo, José I. López
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The prevalent intra- and intertumoral heterogeneity results in undesirable prognosis and therapy failure of pancreatic cancer, potentially resulting from cellular senescence. Herein, integrated analysis of bulk and single-cell RNA-seq profiling was conducted to characterize senescence-based heterogeneity in pancreatic cancer. Publicly available bulk and single-cell RNA sequencing from pancreatic cancer patients were gathered from TCGA-PAAD, PACA-AU, PACA-CA, and GSE154778 datasets. The activity of three senescence-related pathways (cell cycle, DNA repair, and inflammation) was scored utilizing ssGSEA algorithm. A series of functional verifications of crucial genes were accomplished in patient tissue and pancreatic cancer cells. Based upon them, unsupervised clustering analysis was executed to classify pancreatic cancer samples into distinct senescence-based clusters at the bulk and single-cell levels. For single-cell transcriptome profiling, cell clustering and annotation were implemented, and malignant cells were recognized utilizing infercnv algorithm. Two senescence-based clusters were established and highly reproducible at the bulk level, with the heterogeneity in prognosis, clinicopathological features, genomic CNVs, oncogenic pathway activity, immune microenvironment and immune checkpoints. Senescence-relevant gene CHGA, UBE2C and MCM10 were proved to correlate with the migration and prognosis of pancreatic cancer. At the single-cell level, seven cell types were annotated, comprising ductal cells 1, ductal cells 2, fibroblasts, macrophages, T cells, stellate cells, and endothelial cells. The senescence-based classification was also proven at the single-cell level. Ductal cells were classified as malignant cells and non-malignant cells. In the tumor microenvironment of malignant cells, hypoxia and angiogenesis affected senescent phenotype. The heterogeneity in senescence was also observed between and within cell types. Altogether, our findings unveil that cellular senescence contributes to intra- and intertumoral heterogeneity in pancreatic cancer, which might facilitate the development of therapeutics and precision therapy in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2742fe352ef25d18eeda7d59fe2c6a8314ffc13" target='_blank'>
              Deciphering the senescence-based tumoral heterogeneity and characteristics in pancreatic cancer: Results from parallel bulk and single-cell transcriptome data.
              </a>
            </td>
          <td>
            Yeting Lu, Shuo Han, Jing Hu, Kaiji Lv, Y. Ruan, Gong Cheng, Jing Zhang, Xiang Wu, Zeming Weng, Xinhua Zhou
          </td>
          <td>2025-01-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef28d07b971b4246515b216a14089f7d091622f6" target='_blank'>
              Single-cell transcriptomics unveils multifaceted immune heterogeneity in early-onset versus late-onset cervical cancer
              </a>
            </td>
          <td>
            Qian Chen, Dongfeng Deng, Hong Zhu, Shan Li
          </td>
          <td>2025-01-14</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The Imaging and Molecular Annotation of Xenografts and Tumors Cancer Grand Challenges team was set up with the objective of developing the "next generation" of pathology and cancer research by using a combination of single-cell and spatial omics tools to produce 3D molecularly annotated maps of tumors. Its activities overlapped, and in some cases catalyzed, a spatial revolution in biology that saw new technologies being deployed to investigate the roles of tumor heterogeneity and of the tumor micro-environment. See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28 See related article by Goodwin et al., p. 34.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c9c61637383cf6c8f3f96d10f0ca3cac35ac4d" target='_blank'>
              Cancer Research in the Age of Spatial Omics: Lessons from IMAXT.
              </a>
            </td>
          <td>
            D. Bressan, Nicholas Walton, Gregory J. Hannon
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Summary Cellular stemness is a cells ability to self-renew and differentiate into specialized cell types. In spatial transcriptomics (ST), researchers can use gene activity to quantify stemness and the associated spatial locations, allowing for the investigation of cellular profiles in their natural context. However, many ST platforms provide data with low resolution, and each data spot (i.e., spatial location) contains multiple cell types, which hinders the investigation of cell type–specific (CTS) behaviors, such as cell stemness, at different spatial locations. We developed a bivariate kernel-weighted regression method with constrained optimization to estimate CTS stemness as a function of spatial location and developed accompanying visualization tools. Through simulation studies and an application to real breast cancer ST data from the 10x Genomics Visium platform, we demonstrated that our method can accurately estimate CTS stemness and help shed light on the interplay among cell type, tissue structure, and stemness. Availability and implementation An R Shiny app implementing the proposed method is available at GitHub (https://github.com/ericli0480/stemness-deconvolution/). Supplementary information Available in a separate PDF file">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72f608c82ad15c7c6476ead9f76633ad0bf3a052" target='_blank'>
              Delineating the Spatial Patterns of Cell Type–Specific Stemness Using a Statistical Deconvolution Model
              </a>
            </td>
          <td>
            Eric J. Li, Liang Li, Ziyi Li
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Although invasiveness is one of the major determinants of the poor glioblastoma (GBM) outcome, the mechanisms of GBM invasion are only partially understood. Among the intrinsic and environmental processes promoting cell-to-cell interaction processes, eventually driving GBM invasion, we focused on the pro-invasive role played by Extracellular Vesicles (EVs), a heterogeneous group of cell-released membranous structures containing various bioactive cargoes, which can be transferred from donor to recipient cells.


METHODS
EVs isolated from patient-derived GBM cell lines and surgical aspirates were assessed for their pro-migratory competence by spheroid migration assays, calcium imaging, and PYK-2/FAK phosphorylation. Brain invasiveness was investigated in human cortical organoids-based assembloids and in vivo orthotopic xenografts. EV molecular features were specified by multiplex bead-based flow cytometry.


RESULTS
Results unveil a self-sustaining mechanism triggering migration through autocrine release and engagement of a specific population of EVs of large size (L-EVs), isolated from either patient-derived cell lines or surgical aspirates. L-EVs act through modulation of calcium transients via Connexin 43-Gap Junctions (Cx43-GJ) and phospho-activation of PYK2. Pre-incubation with blocking antibodies targeting Cx43 hemichannels demonstrated a dose-dependent inhibition of the L-EV-mediated GBM migration. By exploiting patients' surgical aspirates, we show that only L-EVs deriving from tumoral cells, and not those with immune origin, promote tumor migration, impacting more prominently the tumoral cells with mesenchymal subtype.


CONCLUSIONS
We demonstrate that L-EVs released by GBM cells, but not by the immune cells of the tumor microenvironment, represent a relevant and unique autocrine pro-migratory input for the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f87a08f8bbc4e6c1b435e86c434ceec17830431" target='_blank'>
              Extracellular Vesicles Released by Glioblastoma Cancer Cells Drive Tumor Invasiveness via Connexin-43 Gap Junctions.
              </a>
            </td>
          <td>
            M. Tamborini, Valentino Ribecco, Elisabetta Stanzani, Arianna Sironi, Monica Tambalo, Davide Franzone, Elena Florio, Edoardo Fraviga, Chiara Saulle, M. Gagliani, Marco Pizzocri, M. Mattioli, Katia Cortese, Jean X Jiang, Giuseppe Martano, L. Politi, Marco Riva, Federico Pessina, Davide Pozzi, Simona Lodato, L. Passoni, Michela Matteoli
          </td>
          <td>2025-01-30</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Attempts to activate an anti-tumor immune response in glioblastoma (GBM) have been met with many challenges due to its inherently immunosuppressive tumor microenvironment. The degree and mechanisms by which molecularly and phenotypically diverse tumor-propagating glioma stem cells (GSCs) contribute to this state are poorly defined. In this study, our multifaceted approach combining bioinformatics analyses of clinical and experimental datasets, single-cell sequencing, and molecular and pharmacologic manipulation of patient-derived cells identified GSCs expressing immunosuppressive effectors mimicking regulatory T cells (Tregs). We show that this Immunosuppressive Treg-Like (ITL) GSC state is specific to the mesenchymal GSC subset and is associated with and driven specifically by TGF-β type II receptor (TGFBR2) in contrast to TGFBR1. Transgenic TGFBR2 expression in patient-derived GBM neurospheres promoted a mesenchymal transition and induced a 6-gene ITL signature consisting of CD274 (PD-L1), NT5E (CD73), ENTPD1 (CD39), LGALS1 (galectin-1), PDCD1LG2 (PD-L2), and TGFB1. This TGFBR2-driven ITL signature was identified in clinical GBM specimens, patient-derived GSCs and systemic mesenchymal malignancies. TGFBR2High GSCs inhibited CD4+ and CD8+ T cell viability and their capacity to kill GBM cells, effects reversed by pharmacologic and shRNA-based TGFBR2 inhibition. Collectively, our data identify an immunosuppressive GSC state that is TGFBR2-dependent and susceptible to TGFBR2-targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7f721f28d28359511d7b452c6e33026ca71e4b" target='_blank'>
              TGFBR2High mesenchymal glioma stem cells phenocopy regulatory T cells to suppress CD4+ and CD8+ T cell function
              </a>
            </td>
          <td>
            Amanda L Johnson, Harmon Khela, Jack Korleski, Sophie Sall, Yunqing Li, Weiqiang Zhou, Karen Smith-Connor, H. Lopez-Bertoni, J. Laterra
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Several studies have attempted to find the initiating drivers of small intestinal neuroendocrine tumors (SI-NET) development and the molecular mechanisms driving progression and metastatic spread. For gene expression studies using bulk microarrays and RNA sequencing, researchers commonly use normal intestinal mucosa as a control. The intestine is made up of several different cell types, and using bulk RNA-seq may generate findings that reflect factors other than the tumor transformation. This could potentially contribute to the lack of discoverable treatments and prevention strategies for SI-NETs. We performed scRNA-seq on tissue from two patients that had tumor resection surgery and separated the EC cells from the normal intestinal mucosa and used specific markers to compare them against SI-NET tumors cells. We pinpointed new genes associated with chr18 haploinsufficiency but also others with loss or gain of expression that have not previously been associated with SI-NETs, and which could be potential targets for further functional genomic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Hypoxia is a common feature in solid tumors that arises when there is insufficient oxygen available. This lack of oxygen causes molecular adaptations required for the tumor cells to survive. Additionally, oxygen‐deprived cancer cells tend to become less responsive to conventional cancer therapies. Hence, hypoxia plays an important role in contributing to tumor aggressiveness and therapy resistance. Hypoxia‐related markers are gaining interest as prognostic and predictive markers for tumor response and treatment strategies. However, the detection of hypoxia in the tumor microenvironment without employing any labeling strategies poses significant challenges. Here, we present a classification model based on lipidomic single‐cell mass spectrometry imaging data to classify hypoxia in breast cancer xenografts. Our approach is based on a classification model built from the lipid profiles of single breast cancer cells cultured under various oxygen conditions. Lipidomic alterations caused by differences in available oxygen concentrations were subsequently used to classify and spatially determine hypoxic regions in breast cancer xenografts without the need for any labeling. This approach, using cells as hypoxia markers, contributes to a better understanding of tumor biology and provides a foundation for improving diagnostic and therapeutic strategies for cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7ec1e82c3348ba2f51d436e568e85c4f135880" target='_blank'>
              Classifying Hypoxia in Breast Cancer Xenografts Using a Single‐Cell Mass Spectrometry Imaging Model
              </a>
            </td>
          <td>
            B. Claes, R. Biemans, N. Lieuwes, Lynn Theunissen, Kristine Glunde, Ludwig J Dubois, R. Heeren, Eva Cuypers
          </td>
          <td>2024-12-29</td>
          <td>Analysis &amp; Sensing</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d149bc202f5c23c61477d92eaa03f63e4391af44" target='_blank'>
              Descart: a method for detecting spatial chromatin accessibility patterns with inter-cellular correlations
              </a>
            </td>
          <td>
            Xiaoyang Chen, Keyi Li, Xiaoqing Wu, Zhen Li, Qun Jiang, Xuejian Cui, Zijing Gao, Yanhong Wu, Rui Jiang
          </td>
          <td>2024-12-30</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c4d7e98a1c7e44d04ba5fa342a0635434e2cbe" target='_blank'>
              Tumor secretome shapes the immune landscape during cancer progression
              </a>
            </td>
          <td>
            Jianqiang Yang, Sijia Tang, Nabil F. Saba, C. Shay, Yong Teng
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, K. Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f22ff573111b3740a2cb886527997d19806de8b" target='_blank'>
              Single-cell RNA sequencing reveals tumor heterogeneity in small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Xuesong Xiang, X. Tao, Keqin Hua, Hua Jiang, Jingxin Ding
          </td>
          <td>2025-02-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are pivotal biomarkers in tumor metastasis, however, the underlying molecular mechanism of CTCs behavioral heterogeneity during metastasis remains unexplored. Here, an integrative workflow is developed to link behavior characteristics to metabolic profiling within individual CTCs, which simulates the metastatic process on a microfluidic system and combined with single-cell mass spectrometry (MS) detection. Spheroid-derived HCT116 cells are tracked and extracted via a temporary vascular system, revealing various arrest patterns under biomimetic vascular shear flow. Downstream MS analysis characterizes 17 cellular metabolites and associates metabolic profiles with de-adhesion behaviors of the same CTCs, identifying a potential high-metastatic subpopulation with enhanced arrest ability and evaluating critical metabolites involved in metastasis pathways. Additionally, the metastasis-inhibiting effect of anti-tumor drug 5-fluorouracil by reducing high-metastatic cells in spheroids is elucidated. This approach offers a valuable opportunity to dissect the interplay of the metastatic behavior and metabolic profiles of CTCs and foster insights into the molecular mechanisms underlying behavioral phenotypes in the tumor metastasis process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3e81bfd80ccc573739b0a4ae2229795ed81e7" target='_blank'>
              Linking Metastatic Behavior and Metabolic Heterogeneity of Circulating Tumor Cells at Single-Cell Level Using an Integrative Microfluidic System.
              </a>
            </td>
          <td>
            Ying Hou, Jiaxu Lin, Hongren Yao, Zengnan Wu, Yongning Lin, Jinming Lin
          </td>
          <td>2025-02-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954785d9e945d81ca856ba6792275aca4b027da0" target='_blank'>
              Dissecting macrophage heterogeneity and kaempferol in lung adenocarcinoma: a single-cell transcriptomic approach and network pharmacology
              </a>
            </td>
          <td>
            L. Wan, Wentao Hao, Leilei Li, Lin Wang, Yanzheng Song
          </td>
          <td>2025-01-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Despite numerous studies on medulloblastoma (MB) cell heterogeneity, the spatial characteristics of cellular states remain unclear.


METHODS
We analyze single-nucleus and spatial transcriptomes and chromatin accessibility from human MB spanning four subgroups, to identify malignant cell populations and describe the spatial evolutionary trajectories. The spatial CNVs patterns and niches were analyzed to investigate the cellular interactions.


RESULTS
Three main malignant cell populations were identified, including progenitor-like, cycling and differentiated populations. Gene signatures of cell populations strongly correlate to clinical outcomes. These tumor cell populations are geographically organized as stem-like and mature regions, highlighting their spatially heterogeneous nature. Progenitor-like and cycling cells are mainly concentrated in stem-like regions, whereas various differentiated populations are primarily distributed in mature regions. By analyzing chromosomal alterations, we find that stem-like region typically harbors a single pattern of CNVs, reflecting high originality and uniformity, which is in stark contrast to mature region exhibiting multiple patterns with a broader range of biological functions. Projecting cellular state program onto spatial sections fully illustrates the evolution from stem-like region to various functional zones in mature region, which is correlated to microenvironmental components along the paths to maintain stemness or promote differentiation. Conclusions. This multi-omics database comprehensively facilitates the understanding of MB spatial evolutionary organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The exploration of cellular heterogeneity within the tumor microenvironment (TME) via single-cell RNA sequencing (scRNA-seq) is essential for understanding cancer progression and response to therapy. Current scRNA-seq approaches, however, lack spatial context and rely on incomplete datasets of ligand-receptor interactions (LRIs), limiting accurate cell type annotation and cell-cell communication (CCC) inference. This study addresses these challenges using a novel graph neural network (GNN) model that enhances cell type prediction and cell interaction analysis. Our study utilized a dataset consisting of 49,020 cells from 19 patients across three cancer types: Leukemia, Breast Invasive Carcinoma, and Colorectal Cancer. The proposed scGSL model demonstrated robust performance, achieving an average accuracy of 84.83%, precision of 86.23%, recall of 81.51%, and an F1 score of 80.92% across all datasets. These metrics represent a significant enhancement over existing methods, which typically exhibit lower performance metrics. Additionally, by reviewing existing literature on gene interactions within the TME, the scGSL model proves to robustly identify biologically meaningful gene interactions in an unsupervised manner, validated by significant expression differences in key gene pairs across various cancers. The source code and data used in this paper can be found in https://github.com/LiYuechao1998/scGSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd31c962d460de902bace72d06f995f9fc390f7" target='_blank'>
              Graph Structure Learning for Tumor Microenvironment with Cell Type Annotation from non-spatial scRNA-seq data
              </a>
            </td>
          <td>
            Yu-an Huang, Yue C. Li, Hai-Ru You, Jie Pan, Xiyue Cao, Xinyuan Li, Zhi-an Huang, Zhu-Hong You
          </td>
          <td>2025-02-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5992d9581aee52854936b160a4c470689208c2e" target='_blank'>
              Targeting HVEM-GPT2 axis: a novel approach to T cell activation and metabolic reprogramming in non-small cell lung cancer therapy
              </a>
            </td>
          <td>
            Yuanshan Yao, Chunji Chen, Bin Li, Wen Gao
          </td>
          <td>2025-02-04</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The evolution of metastasis, the spread of cancer to distal sites within the body, represents a lethal stage of cancer progression. Yet, the evolutionary dynamics that shape the emergence of metastatic disease remain unresolved. Here, using single-cell lineage tracing data in combination with phylogenetic statistical methods, we show that the evolutionary trajectory of metastatic disease is littered with bursts of rapid molecular change as new cellular subpopulations appear, a pattern known as punctuational evolution. Next, by measuring punctuational evolution across the metastatic cascade, we show that punctuational effects are concentrated within the formation of secondary tumours at distal metastatic sites, suggesting that qualitatively different modes of evolution may drive primary and metastatic tumour progression. Taken as a whole, our findings provide empirical evidence for distinct patterns of molecular evolution at early and late stages of metastatic disease and our approach provides a framework to study the evolution of metastasis at a more nuanced level than has been previously possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eea86b0534290bd81992d00ef5caf3aad45d15" target='_blank'>
              Punctuational evolution is pervasive in distal site metastatic colonization
              </a>
            </td>
          <td>
            George Butler, Sarah R. Amend, C. Venditti, Kenneth J Pienta
          </td>
          <td>2025-01-01</td>
          <td>Proceedings of the Royal Society B: Biological Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Glioblastoma (GBM) is the most common primary malignant brain tumor in adults, characterized by aggressive growth and a dismal prognosis. Interleukin-33 (IL-33) and its receptor ST2 have emerged as regulators of glioma growth, but their exact function in tumorigenesis has not been deciphered. Indeed, previous studies on IL-33 in cancer have yielded somewhat opposing results as to whether it is pro- or anti-tumorigenic. Methods IL-33 expression was assessed in a GBM tissue microarray and public databases. As in vivo models we used orthotopic xenografts of patient-derived GBM cells, and syngenic models with grafted mouse glioma cells. Results We analyzed the role of IL-33 and its receptor ST2 in nonmalignant cells of the glioma microenvironment and found that IL-33 levels are increased in cells surrounding the tumor. Protein complexes of IL-33 and ST2 are mainly found outside of the tumor core. The IL-33-producing cells consist primarily of oligodendrocytes. To determine the function of IL-33 in the tumor microenvironment, we used mice lacking the ST2 receptor. When glioma cells were grafted to ST2-deficient mouse brains, the resulting tumors exhibited a more invasive growth pattern, and are associated with poorer survival, compared to wild-type mice. Tumors in ST2-deficient hosts are more invasive, with increased expression of extracellular matrix remodeling enzymes and enhanced tumor angiogenesis. Furthermore, the absence of ST2 leads to a more immunosuppressive environment. Conclusions Our findings reveal that glia-derived IL-33 and its receptor ST2 participate in modulating tumor invasiveness, tumor vasculature, and immunosuppression in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc52b5f579f7e0f0d9dd54d423d1abfbfb33ee41" target='_blank'>
              Lack of ST2 aggravates glioma invasiveness, vascular abnormality, and immune suppression
              </a>
            </td>
          <td>
            Grzegorz Wicher, Ananya Roy, Alessandra Vaccaro, K. Vemuri, Mohanraj Ramachandran, Tommie Olofsson, Rebeca-Noemi Imbria, M. Belting, Gunnar Nilsson, A. Dimberg, K. Forsberg-Nilsson
          </td>
          <td>2025-01-27</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Anaplastic Thyroid Cancer (ATC) is an aggressive form of cancer with poor prognosis, heavily influenced by its tumor immune microenvironment (TIME). Understanding the cellular and gene expression dynamics within the TIME is crucial for developing targeted therapies. This study analyzes the immune microenvironment of ATC and Papillary Thyroid Cancer (PTC) using single-cell RNA sequencing (scRNA-seq). We performed a comprehensive scRNA-seq analysis on ATC and PTC samples, incorporating cell type annotation, marker gene identification, and clustering based on gene expression. A specific focus was on the prevalence and biomarkers of Pre-Exhausted CD8+ T cells in ATC, utilizing the single-cell tumor immune atlas for immune cell characterization. The scRNA-seq analysis identified distinct immune cell populations and differentially expressed genes in ATC and PTC samples. Notably, Pre-Exhausted CD8+ T cells were found to be prevalent in ATC datasets. Additional immunofluorescence staining and co-culture experiments with the ATC cell line identified GNLY, a member of the saposin-like protein family as a potential biomarker for Pre-Exhausted CD8+ T cells in ATC. This study provides valuable insights into the immune landscape of ATC, emphasizing the prevalence of Pre-Exhausted CD8+ T cells and identifying GNLY as a potential biomarker. Understanding the TIME composition and the role of specific immune cells in cancer progression can inform the development of targeted immunotherapies for ATC. Future research should explore the functional implications of GNLY and other identified biomarkers in modulating the immune response in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bee3b0c483d956dde8850d164e4f812a4b30008" target='_blank'>
              Persistent pre-exhausted CD8+ T cells shape the tumor immune microenvironment in anaplastic thyroid cancer.
              </a>
            </td>
          <td>
            X. Ruan, Mei Tao, Yuanxing Dong, Linfei Hu, Guangwei Xu, Chuanxiang Hu, Yue Huang, Yuqi Wang, Jialong Yu, Wei Luo, Ming Gao, Min Zhao, Xiangqian Zheng
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the pivotal role of cytotoxic T lymphocytes (CTLs) in anti-tumor immunity, a substantial proportion of CTL-rich hepatocellular carcinoma (HCC) patients experience early relapse or immunotherapy resistance. However, spatial immune variations impacting the heterogeneous clinical outcomes of CTL-rich HCCs remain poorly understood. Here, we compared the single-cell and spatial landscapes of 20 CTL-rich HCCs with distinct prognoses using multiplexed in situ staining and validated the prognostic value of myeloid spatial patterns in a cohort of 386 patients. Random forest and Cox regression models identified macrophage aggregation as a distinctive spatial pattern characterizing a subset of CTL-rich HCCs with an immunosuppressive microenvironment and poor prognosis. Integrated analysis of single-cell and spatial transcriptomics, combined with in situ staining validation, revealed that spatial aggregation enhanced pro-tumoral macrophage reprogramming in HCCs, marked by lipid metabolism orientation, M2-like polarization, and increased adjacent CTL exhaustion. This spatial effect on macrophage reprogramming was replicated in HCC-conditioned human macrophage cultures, which showed an enhanced capability to suppress CTLs. Notably, increased macrophage aggregation was associated with higher response rates to anti-PD-1 immunotherapy. These findings suggest that the spatial distribution of macrophages is a biomarker of their functional diversities and microenvironment status, which holds prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a3e561fbd74941c09e71f727074ba04c903f0f" target='_blank'>
              Spatial Organization of Macrophages in CTL Rich Hepatocellular Carcinoma Influences CTL Antitumor Activity.
              </a>
            </td>
          <td>
            Yulan Weng, Lu Wang, Yuting Wang, Junyu Xu, Xiaoli Fan, Shufeng Luo, Qiaomin Hua, Jing Xu, Gaoteng Liu, Kai-Bo Zhao, Chang-An Zhao, Dong-Ming Kuang, Chong Wu, Limin Zheng
          </td>
          <td>2025-01-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3cbd55a53252e9bb3dfd532de1bccedf85f35d" target='_blank'>
              Tumor-associated-fibrosis and active collagen-CD44 axis characterize a poor-prognosis subtype of gastric cancer and contribute to tumor immunosuppression
              </a>
            </td>
          <td>
            Yingqi Yang, Haohan Sun, Hongkai Yu, Luyao Wang, Chang Gao, Haokun Mei, Xiaomeng Jiang, Minghui Ji
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor interpatient heterogeneity has been well described with two major prognostic subtypes (classical and basal-like). An important intrapatient heterogeneity has been reported but has not yet been studied extensively due to the lack of standardized, reproducible, and easily accessible high-throughput methods. We built an immunohistochemical (IHC) tool capable of differentiating RNA-defined classical and basal-like tumors by selecting relevant antibodies using a multistep process. The successive stages of (i) an in silico selection from a literature review and a bulk transcriptome analysis of 309 PDACs, (ii) a tumor-specific selection from 30 patient-derived xenografts and single-cell data, followed by (iii) the validation on tissue microarrays in 50 PDAC were conducted. We used our final IHC panel on two independent cohorts of resected PDAC (n = 95, whole-slide, n = 148, tissue microarrays) for external validation. After digitization and registration of pathology slides, we performed a tile-based analysis in tumor areas to identify relevant marker combinations. Sequential marker selection led to the following panel: GATA6, CLDN18, TFF1, MUC16, S100A2, KRT17, PanBasal. Four different phenotypes were identified: one classical, one intermediate (KRT17+), and two basal-like (MUC16+ versus S100A2+) with specific biological properties. The presence of a minor basal contingent drastically reduced overall survival [hazard ratio (HR) = 1.90, p = 0.03], even in classical predominant PDACs. Analysis of preneoplastic lesions suggested that pancreatic carcinogenesis might follow a progressive evolution from classical toward a basal through an early intermediate phenotype. In conclusion, our IHC panel redefined and easily assessed the high degree of intra- and intertumoral heterogeneity of PDAC. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9c9cb0346cebac5553bfe30b873ba04769f272" target='_blank'>
              Redefining phenotypic intratumor heterogeneity of pancreatic ductal adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            M. Hilmi, F. Delecourt, J. Raffenne, Taib Bourega, Nelson Dusetti, J. Iovanna, Yuna Blum, Magali Richard, C. Neuzillet, A. Couvelard, Matthieu Tihy, L. de Mestier, Vinciane Rebours, R. Nicolle, Jérôme Cros
          </td>
          <td>2025-02-11</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5eb0294ad5d50d6c71886518bd626ba65245bb" target='_blank'>
              UBE2J1 is identified as a novel plasma cell-related gene involved in the prognosis of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yunjie Tian, Ruoyu Dong, Ying Guan, Ying Wang, Wei Zhao, Jun Zhang, Shan Kang
          </td>
          <td>2025-01-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in neoplastic initiation and progression. Epigenetic machinery, governing the expression of core oncogenes and tumor suppressor genes in transformed cells, significantly contributes to tumor development at both primary and distant sites. Recent studies have illuminated how epigenetic mechanisms integrate external cues and downstream signals, altering the phenotype of stromal cells and immune cells. This remolds the area surrounding tumor cells, ultimately fostering an immunosuppressive microenvironment. Therefore, correcting the TME by targeting the epigenetic modifications holds substantial promise for cancer treatment. This review synthesizes recent research that elucidates the impact of specific epigenetic regulations—ranging from DNA methylation to histone modifications and chromatin remodeling—on stromal and immune cells within the TME. Notably, we highlight their functional roles in either promoting or restricting tumor progression. We also discuss the potential applications of epigenetic agents for cancer treatment, envisaging their ability to normalize the ecosystem. This review aims to assist researchers in understanding the dynamic interplay between epigenetics and the TME, paving the way for better epigenetic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea83192c32e6b6228a3b5ff261efd56060404967" target='_blank'>
              Epigenetic Regulation of Stromal and Immune Cells and Therapeutic Targets in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Kang Liu, Yue Li, Minmin Shen, Wei Xu, Shanshan Wu, Xinxin Yang, Bo Zhang, Nengming Lin
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Piñeiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small subset within the tumor mass significantly contributing to cancer progression through dysregulation of various oncogenic pathways, driving tumor growth, chemoresistance and metastasis formation. The aggressive behavior of CSCs is guided by several intracellular signaling pathways such as WNT, NF-kappa-B, NOTCH, Hedgehog, JAK-STAT, PI3K/AKT1/MTOR, TGF/SMAD, PPAR and MAPK kinases, as well as extracellular vesicles such as exosomes, and extracellular signaling molecules such as cytokines, chemokines, pro-angiogenetic and growth factors, which finely regulate CSC phenotype. In this scenario, tumor microenvironment (TME) is a key player in the establishment of a permissive tumor niche, where CSCs engage in intricate communications with diverse immune cells. The “oncogenic” immune cells are mainly represented by B and T lymphocytes, NK cells, and dendritic cells. Among immune cells, macrophages exhibit a more plastic and adaptable phenotype due to their different subpopulations, which are characterized by both immunosuppressive and inflammatory phenotypes. Specifically, tumor-associated macrophages (TAMs) create an immunosuppressive milieu through the production of a plethora of paracrine factors (IL-6, IL-12, TNF-alpha, TGF-beta, CCL1, CCL18) promoting the acquisition by CSCs of a stem-like, invasive and metastatic phenotype. TAMs have demonstrated the ability to communicate with CSCs via direct ligand/receptor (such as CD90/CD11b, LSECtin/BTN3A3, EPHA4/Ephrin) interaction. On the other hand, CSCs exhibited their capacity to influence immune cells, creating a favorable microenvironment for cancer progression. Interestingly, the bidirectional influence of CSCs and TME leads to an epigenetic reprogramming which sustains malignant transformation. Nowadays, the integration of biological and computational data obtained by cutting-edge technologies (single-cell RNA sequencing, spatial transcriptomics, trajectory analysis) has significantly improved the comprehension of the biunivocal multicellular dialogue, providing a comprehensive view of the heterogeneity and dynamics of CSCs, and uncovering alternative mechanisms of immune evasion and therapeutic resistance. Moreover, the combination of biology and computational data will lead to the development of innovative target therapies dampening CSC-TME interaction. Here, we aim to elucidate the most recent insights on CSCs biology and their complex interactions with TME immune cells, specifically TAMs, tracing an exhaustive scenario from the primary tumor to metastasis formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ce167799371ac069e6df125e4a468fd20b0208" target='_blank'>
              Cancer stem cells and tumor-associated macrophages as mates in tumor progression: mechanisms of crosstalk and advanced bioinformatic tools to dissect their phenotypes and interaction
              </a>
            </td>
          <td>
            Francesco Verona, Sebastiano Di Bella, Roberto Schirano, Camilla Manfredi, Francesca Angeloro, Giulia Bozzari, M. Todaro, Giuseppe Giannini, Giorgio Stassi, V. Veschi
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The recent emergence of in vitro microvessel models has boosted research on angiogenesis and vascularization in tissue engineering, due to their superior control over in controlling culture conditions and enabling real-time observation compared to in vivo models. However, conventional two-dimensional (2D) observation and analysis fail to capture the heterogeneity of three-dimensional (3D) morphological dynamics. To overcome this issue, here, a novel morphological registration method for spatiotemporal quantification of angiogenic deformation dynamics by combining confocal microscopy of an engineered microvessel with computer vision techniques has been proposed in this paper. Using a coculture system of a microvessel and pericytes, the spatiotemporal measurements reveals: (i) distinct deformation patterns and growth/regression zonation on the parent vessel and angiogenic sprouts; (ii) spatiotemporal changes in pericyte localization and coverage; and (iii) the enhancing effect of pericyte-microvessel contact on local Notch signaling activation, the distribution of matrix metalloproteinase-1 (MMP-1), the heterogeneity of angiogenic dynamics, and morphological maturation. This pilot system offers a characterization of the comprehensive effects of cocultured cells during angiogenesis and enables the interactive fusion of multimodal data in future studies on vascular morphogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25f119562f7fc234fe1009d70f1dc763d382681" target='_blank'>
              Spatiotemporal analysis of pericyte-induced dynamics in nascent angiogenic morphogenesis and heterogeneity using a microvessel-on-a-chip platform
              </a>
            </td>
          <td>
            Hedele Zeng, Takanori Sano, Jun-ichi Kawabe, Yukiko T. Matsunaga
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background & Aims
Identifying patients with intrahepatic cholangiocarcinoma (ICC) likely to benefit from immunochemotherapy, the new front-line treatment, remains challenging. We aimed to unveil a novel radiotranscriptomic signature that can facilitate treatment response prediction by multi-omics integration and multi-scale modelling.


Methods
We analyzed bulk, single-cell and spatial transcriptomic data comprising 457 ICC patients to identify an immune-related score (IRS), followed by decoding its spatial immune context. We mapped radiomics profiles onto spatial-specific IRS using machine-learning to define a novel radiotranscriptomic signature, followed by multi-scale and multi-cohort validation covering 331 ICC patients. The signature was further explored for the potential therapeutic target from in vitro to in vivo.


Results
We revealed a novel 3-gene (PLAUR, CD40LG and FGFR4) IRS whose down-regulation correlated with better survival and improved sensitivity to immunochemotherapy. We highlighted functional IRS-immune interactions within tumor epithelium, rather than stromal compartment, irrespective of geospatial locations. Machine-learning pipeline identified the optimal 3-feature radiotranscriptomic signature that was well-validated by immunohistochemical assays in molecular cohort, exhibited favorable external prognostic validity with C-index over 0.64 in resection cohort, and predicted treatment response with an area under the curve of up to 0.84 in immunochemotherapy cohort. We also showed that anti-uPAR/PLAUR alone or in combination with anti-programmed cell death protein 1 therapy remarkably curbed tumor growth, using in vitro ICC cell lines and in vivo humanized ICC patient-derived xenograft mouse models.


Conclusions
This proof-of-concept study sheds light on the spatially-resolved radiotranscriptomic signature to improve patient selection for emerging immunochemotherapy and high-order immunotherapy combinations in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Personalized precision medicine (PPPM) in cancer immunology and oncology is a rapidly advancing field with significant potential. Gliomas, known for their poor prognosis, rank among the most lethal brain tumors. Despite advancements, there remains a critical need for precise, individualized treatment strategies. Methods We conducted a comprehensive analysis of RNA-seq and microarray data from the TCGA and GEO databases, supplemented by single-cell RNA sequencing (scRNA-seq) data from glioma patients. By integrating single-cell sequencing analysis with foundational experiments, we investigated the molecular variations and cellular interactions within neural glioma cell subpopulations during tumor progression. Results Our single-cell sequencing analysis revealed distinct gene expression patterns across glioma cell subpopulations. Notably, differentiation trajectory analysis identified NUSAP1 as a key marker for the terminal subpopulation. We found that elevated NUSAP1 expression correlated with poor prognosis, prompting further investigation of its functional role through both cellular and animal studies. Conclusions NUSAP1-based risk models hold potential as predictive and therapeutic tools for personalized glioma treatment. In-depth exploration of NUSAP1’s mechanisms in glioblastoma could enhance our understanding of its response to immunotherapy, suggesting that targeting NUSAP1 may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5625f39e4872fec67e5df1a449a03f506dae3b79" target='_blank'>
              Single-cell sequencing elucidates the mechanism of NUSAP1 in glioma and its diagnostic and prognostic significance
              </a>
            </td>
          <td>
            Meng-Yu Zhao, Zhao-Lei Shen, Hongzhen Dai, Wan-Yan Xu, Li-Na Wang, Yu- Gu, Jie-Hui Zhao, Tian-Hang Yu, Cun-Zhi Wang, Jia-feng Xu, Guan-Jun Chen, Dong-Hui Chen, Wen-Ming Hong, Fan Zhang
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) has emerged as a transformative technology for elucidating cellular organization and tissue architecture. However, a significant challenge remains in identifying pathology-relevant spatial functional landscapes within the tissue microenvironment, primarily due to the limited integration of cell-cell communication dynamics. To address this limitation, SpaDCN, a Spatially Dynamic graph Convolutional Network framework is proposed, which aligns cell-cell communications and gene expression within a spatial context to reveal the spatial functional regions with the coherent cellular organization. To effectively transfer the influence of cell-cell communications on expression variation, SpaDCN respectively generates the node layer and edge layer of spatial graph representation from expression data and the ligand-receptor complex contributions and then employs a dynamic graph convolution to switch the propagation of node graph and edge graph. It is demonstrated that SpaDCN outperforms existing methods in identifying spatial domains and denoising expression across various platforms and species. Notably, SpaDCN excels in identifying marker genes with significant prognostic potential in cancer tissues. In conclusion, SpaDCN offers a powerful and precise tool for spatial domain detection in spatial transcriptomics, with broad applicability across various tissue types and research disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Spine metastases are the most common bone site for breast cancer, with evolving surgery and multidisciplinary care improving outcomes. Current treatments, including chemotherapy and invasive surgery, may damage healthy tissue and may leave residual tumors that lead to recurrence. Cold atmospheric plasma (CAP) offers a non-invasive alternative by delivering reactive oxygen and nitrogen species (RONS) locally to tumor sites, selectively targeting cancer cells while sparing healthy tissue. To assess the impact and selectivity toward tumor cells adjacent to bone-like tissue, we develop a 3D bioprinted tumor-stroma model using a 1% alginate and 7% gelatin cell-laden hydrogel to mimic a bone-like microenvironment. The model co-cultures triple-negative MDA-MB-231 human breast cancer cells with primary human bone marrow mesenchymal stromal cells to simulate tumor-stroma interactions. The effects of CAP treatments are assessed through metabolic activity and viability assays over three days. Results show significant selectivity for cancer cells in both 2D and 3D cultures. CAP minimizes damage to healthy cells, offering the potential for localized treatment over systemic chemotherapies such as doxorubicin. Our novel bioprinted model, combined with a plasma source controlling RONS composition, enables detailed studies of redox-based cancer cell inactivation and highlights CAP as a personalized, non-invasive treatment for bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db15f717a397933ff6371b23deebeffcaae60439" target='_blank'>
              Cold Atmospheric Plasma Selectively Disrupts Breast Cancer Growth in a Bioprinted 3D Tumor Microenvironment Model
              </a>
            </td>
          <td>
            Laura M. Bouret, Jean-Baptiste Billeau, Michael H. Weber, Derek H. Rosenzweig, Stephan Reuter
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, Iben Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state.
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 760


 Background:
 Pancreatic adenosquamous carcinoma (PASC) is a rare histopathological type of pancreatic cancer that is more aggressive than pancreatic ductal adenocarcinoma (PDAC). Although PASC has a unique tumor biology and poorer prognosis, there are no guidelines for differentially treating PASC patients, and its rarity makes it significantly understudied. We used Nanostring GeoMX digital spatial profiling (DSP) to provide a targeted proteomic characterization of key tumor-promoting proteins in PASC and PDAC.
 Methods:
 FFPE tissue from PASC (n=10 primary, n= 4 met) and PDAC (n=21 primary, n=5 met) cases were analyzed using the DSP platform. Sections were stained with an 82-protein cocktail of antibodies conjugated to UV-photocleavable DNA barcodes. A pan-cytokeratin morphology marker was used to select tumor regions of interest (ROI) and inferred surrounding stromal (i.e., pan-CK negative) ROIs. UV excitation of the defined ROIs was used to release the DNA barcodes and were quantified on the NanoString nCounter system. For PASC, squamous enriched ROI were inferred from corresponding p40 IHC. Corresponding clinical and mutational status for all cases were abstracted from medical records.
 Results:
 TP53 (PDAC, 80%; PASC, 83%) and KRAS (PDAC, 76%; PASC, 100%) were the most prevalent mutations
 .
 In comparing the tumor ROIs of PASC and PDAC (all cases), DSP-assessed global expression and activation of EGFR-MAPK signaling is significantly increased in PASC, whereas PI3K-AKT signaling is significantly decreased. Compared to PDAC, EGFR expression and phosphorylation (p-EGFR), as well as p-MEK1, are significantly increased in PASC. PASC showed significantly lower levels of PLCG1 and INPP4B, while the majority of other AKT signaling effectors were unchanged. For DNA damage response signaling, PASC showed significantly higher levels of p-ATM, p-CHK2, and p21, but not p-ATR. Likewise, expression of oncoproteins c-MYC and BCL6 were significantly increased in PASC (vs. PDAC).
 Conclusions:
 Given that the normal pancreas lacks squamous cells, their histological presence is a uniquely aberrant feature that can be used to guide deep spatially-resolved interrogation of complex cellular and molecular programs. Our preliminary findings show that regions of squamous cell differentiation within PASC have intrinsically higher levels of active EGFR signaling. With emerging RAS-targeted agents poised to alter the treatment landscape, high EGFR signaling in PASC suggest a possible compensatory resistance mechanism that may require additional targeting. Likewise, reliance on ATM signaling highlights this as preferential target for DDR-directed therapies. Building on these results with additional multiomic assays will inform strategies for improving the management of patients with PASC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b1608b4f56eb9c6e1216a160d63214aa03c88b" target='_blank'>
              Spatially resolved protein profiling in pancreatic adenosquamous carcinoma.
              </a>
            </td>
          <td>
            Adel Kardosh, Rosalie C. Sears, Zheng Xia, Eizabaru M Sasatomi, Brian Mau, Jonathan R Brody, Melissa Wong, Charles D. Lopez, E. Chen, Guillaume Pegna, Cristiane M Franca, John Muschler, Joanna Pucilowska, Shaun M Goodyear
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
The tumour microenvironment significantly influences the clinical response of patients to therapeutic immune checkpoint inhibition (ICI), but a comprehensive understanding of the underlying immune-regulatory proteome is still lacking.


OBJECTIVES
To decipher targetable biologic processes that determine tumour-infiltrating lymphocytes (TiLs) as a cellular equivalent of clinical response to ICI.


METHODS
We mapped the spatial distribution of proteins in TiL-enriched vs. TiL-low compartments in melanoma by combining microscopy, matrix-assisted laser desorption mass spectrometry imaging and liquid chromatography-mass spectrometry, as well as computational data mining. Pharmacological modulation of sirtuin 1 (SIRT1) activity in syngeneic mouse models was used to evaluate the efficacy of pharmacological SIRT1 activation in two syngeneic melanoma mouse models, one known to be α-programmed cell death protein 1 (PD-1) sensitive and the other α-PD-1 resistant.


RESULTS
Spatial proteomics and gene ontology-based enrichment analysis identified > 145 proteins enriched in CD8high tumour compartments, including negative regulators of mammalian target of rapamycin signalling such as SIRT1. Multiplexed immunohistochemistry confirmed that SIRT1 protein was expressed more in CD8high than in CD8low compartments. Further analysis of bulk and single-cell RNA sequencing data from melanoma tissue samples suggested the expression of SIRT1 by different lymphocyte subpopulations (CD8+ T cells, CD4+ T cells and B cells). Furthermore, we showed in vivo that pharmacological SIRT1 activation increased the immunological effect of α-PD-1 ICI against melanoma cells in mice, which was accompanied by an increase in T-cell infiltration and T-cell-related cytokines, including interferon (IFN)-γ, CCL4, CXCL9, CXCL10 and tumour necrosis factor-α. In silico analysis of large transcriptional data cohorts showed that SIRT1 was positively associated with the proinflammatory T-cell chemokines CXCL9, CXCL10 and IFN-γ, and prolonged overall survival of patients with melanoma.


CONCLUSIONS
Our study deciphers the proteomics landscape in human melanoma, providing important information on the tumour microenvironment and identifying SIRT1 as having important prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e28ba69768cc37ada6cc67883351db03f9a038" target='_blank'>
              Spatial proteomics reveals sirtuin 1 to be a determinant of T-cell infiltration in human melanoma.
              </a>
            </td>
          <td>
            J. Placke, Jenny Bottek, R. Váraljai, B. Shannan, Sarah Scharfenberg, Christoph Krisp, Philippa Spangenberg, Camille Soun, Devon Siemes, Lars Borgards, Franziska Hoffmann, Fang Zhao, Anette Paschen, Hartmut Schlueter, Ferdinand von Eggeling, Iris Helfrich, Florian Rambow, Selma Ugurel, A. Tasdogan, Dirk Schadendorf, D. R. Engel, Alexander Roesch
          </td>
          <td>2024-12-31</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 256


 Background:
 Resistance to Androgen Receptor Pathway Inhibitors (ARPI) in metastatic prostate cancer (mPC) is universal and can be driven by complex genomic alterations. The evolution of lineage state transitions from adenocarcinomas to neuroendocrine prostate cancer (NEPC) has also been shown to drive treatment resistance and poor survival. Identifying the timing and association of genomic mutations with lineage state transitions has been limited by the need for serial tumor biopsies. We report an integrated analysis of clinical next-generation sequencing (NGS) data and mPC lineage states with a novel circulating tumor cell (CTC) RNA sequencing methodology.
 Methods:
 We collected 273 samples from 117 unique patients with mPC in a prospective biomarker trial. CTCs were purified via immunomagnetic capture on an automated microfluidic technology and analyzed via RNA-seq. A subset of patients had a clinical grade genetic test performed on a contemporary metastatic tissue biopsy. We compared CTC gene expression for lineage states (luminal A, luminal B, NEPC) with somatic gene mutation subtypes known to confer resistance to ARPIs (AR, p53, RB, PTEN, HRR genes) and overall survival (OS).
 Results:
 Single sample pathway analysis of high CTC purity samples identified four transcriptional phenotypes: luminal A-like (LumA), luminal B-like (LumB), low proliferation (LP), and neuroendocrine (NE). Compared to patients with low CTC burden/purity (median OS not reached), patients with LumA and LP phenotypes had similar survival, patients with LumB and NE CTC phenotypes had shorter survival (LumB: median OS 6m, HR 9.1 [3.8-21.8], p<0.0001, NE: median OS 3.7m, HR 11.8 [2.4-57.2], p=0.0019). AR alterations were found in samples with prior ARPI exposure and at similar frequencies across all CTC phenotypes but NE where they were absent. RB mutations were enriched in the unfavorable CTC phenotypes (LumB, NE; p=0.0288). Integrating CTC phenotype with presence or absence of at least one high risk genomic alteration (AR, RB, TP53, PTEN), patients with favorable CTC phenotype (Low burden, LP and LumA) and no high risk alterations had the longest median survival (median OS NR), followed by patients with favorable phenotype but a high risk alteration (median OS 12.7m), patients with an unfavorable phenotype (LumB, NE) but no high risk alteration (median OS 8m), and unfavorable phenotype with a high risk alteration (median OS 4m).
 Conclusions:
 Somatic mutations in mCRPC influence lineage state acquisitions and treatment resistance. Lineage state transitions themselves are associated with poor outcomes and decreased survival which is worsened in the presence of high-risk genetic mutations including RB. This data presents potentially targetable patient populations that would benefit from treatment intensification and early disease monitoring for more aggressive mCRPC subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba71a7a1acb2ee64ae5bd588ebd4c00f7170961" target='_blank'>
              Investigating clinical associations between genomic alterations and prostate cancer lineage states using circulating tumor cell RNA sequencing.
              </a>
            </td>
          <td>
            Katharine E Tippins, Joshua M Lang, Marina N. Sharifi, J. Sperger, Shannon R Reese, Amy K Taylor, Viridiana Carreno, Alex H Chang, Meghan Wells, John M Floberg, H. Emamekhoo, Shuang Zhao, D. Kosoff
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="Bladder cancer is one of the most prevalent malignancies, and the mechanisms underlying its progression and the role of the tumor microenvironment (TME) are unclear. Recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) enable detailed analysis of the cellular heterogeneity, gene expression, and cell–cell interactions in bladder diseases. We conducted a comprehensive search for recent articles that have investigated bladder diseases using scRNA-seq and ST. scRNA-seq and ST have led to significant discoveries in bladder disease research. These technologies have enabled the identification of multiple molecular subtypes within individual tumors and of the mechanisms of treatment resistance. Additionally, molecular differences based on gender have been explored, explaining the heterogeneity of the incidence and progression of bladder cancer. These findings deepen our understanding of the pathology of bladder diseases and highlight the transformative potential of scRNA-seq and ST in identifying novel biomarkers and therapeutic targets. Integrating scRNA-seq and ST has considerably enhanced our understanding of tumor heterogeneity and the tumor microenvironment within tissues. These insights may lead to the development of personalized therapies and the improvement of patient outcomes. Several challenges, such as technical limitations and access difficulties, need to be addressed for the future clinical application of these technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6158ee2a6cdd6c17c247d644521b29f60e5d9354" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptome analysis in bladder cancer: Current status and future perspectives
              </a>
            </td>
          <td>
            Kentaro Yoshihara, Kagenori Ito, Takahiro Kimura, Yusuke Yamamoto, Fumihiko Urabe
          </td>
          <td>2025-01-01</td>
          <td>Bladder Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbee10e353cac2b48554f45773102121a37f67" target='_blank'>
              Endothelial-secreted Endocan activates PDGFRA and regulates vascularity and spatial phenotype in glioblastoma
              </a>
            </td>
          <td>
            Soniya Bastola, M. Pavlyukov, Neel Sharma, Yasmin Ghochani, Mayu A. Nakano, S. Muthukrishnan, Sang Yul Yu, Min Soo Kim, Alireza Sohrabi, Natalia P. Biscola, Daisuke Yamashita, Ksenia S. Anufrieva, Tatyana F. Kovalenko, Grace Jung, Tomas Ganz, Beatrice O’Brien, R. Kawaguchi, Yue Qin, S. Seidlits, Alma L. Burlingame, J. Oses-Prieto, Leif A. Havton, Steven A. Goldman, A. Hjelmeland, Ichiro Nakano, H. Kornblum
          </td>
          <td>2025-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a complex and heterogeneous hematologic malignancy characterized by clonal evolution, genetic instability, and interactions with a supportive tumor microenvironment. These factors contribute to treatment resistance, disease progression, and significant variability in clinical outcomes among patients. This review explores the mechanisms underlying MM progression, including the genetic and epigenetic changes that drive clonal evolution, the role of the bone marrow microenvironment in supporting tumor growth and immune evasion, and the impact of genomic instability. We highlight the critical insights gained from single-cell technologies, such as single-cell transcriptomics, genomics, and multiomics, which have enabled a detailed understanding of MM heterogeneity at the cellular level, facilitating the identification of rare cell populations and mechanisms of drug resistance. Despite the promise of advanced technologies, MM remains an incurable disease and challenges remain in their clinical application, including high costs, data complexity, and the need for standardized bioinformatics and ethical considerations. This review emphasizes the importance of continued research and collaboration to address these challenges, ultimately aiming to enhance personalized treatment strategies and improve patient outcomes in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, Travis S. Johnson
          </td>
          <td>2025-02-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A comprehensive spatial distribution of the proteome in human body and cancers is fundamental for understanding human biology and diseases including cancers. Here, we present an anatomically resolved human proteome derived from 1781 benign and malignant samples from 58 major tissue types encompassing 251 specific tissues and 25 carcinomas. Based on a spectral library covering over 75% of the human protein-coding genes with 208 understudied and 82 missing proteins characterized, we quantified over 13,000 proteins in these samples using data-independent acquisition proteomics. This data resource presents the so far most comprehensive quantitative proteomic landscape of human tissues and common carcinomas. It allows systematic evaluation of tissue-specific drug responses, identification of drug candidates that may be repurposed as antineoplastics, and discovery of novel targets for anticancer therapy. This resource, available as an online knowledgebase, refines our knowledge of spatial distribution of the human proteome and tumor-specific protein modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yi Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The progression of colorectal cancer is strongly influenced by environmental and genetic conditions. One of the key factors is tumor heterogeneity which is extensively studied by cfDNA and bulk sequencing methods; however, we lack knowledge regarding its effects at the single-cell level. Motivated by this, we aimed to employ an end-to-end single-cell sequencing workflow from tissue-derived sample isolation to exome sequencing. Our main goal was to investigate the heterogeneity patterns by laser microdissecting samples from different locations of a tissue slide. Moreover, by studying healthy colon control, tumor-associated normal, and colorectal cancer tissues, we explored tissue-specific heterogeneity motifs. For completeness, we also compared the performance of the whole-exome bulk, cfDNA, and single-cell sequencing methods based on variation at the level of a single nucleotide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9266548dfc591e54ed5a1417777c06bd133c4150" target='_blank'>
              Investigation of Exome-Wide Tumor Heterogeneity on Colorectal Tissue-Based Single Cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, A. Kalmár, B. Barták, Z. Nagy, Gábor Valcz, Tamás Richárd Linkner, Kristóf Róbert Rada, István Takács, Béla Molnár
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f7c17606ae67d5ce7fe611db0e7bce958444bb" target='_blank'>
              Decoding the heterogeneous subpopulations of glioblastoma for prognostic stratification and uncovering the promalignant role of PSMC2
              </a>
            </td>
          <td>
            Nini Zhou, Jingsong Yan, Manya Xiong, Shunqin Zhu
          </td>
          <td>2025-02-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1db80b341443231830586b45a4d64283910100" target='_blank'>
              Identification of increased dedifferentiation along the Prom1+ cancer cells in Müllerian adenosarcoma with sarcomatous overgrowth.
              </a>
            </td>
          <td>
            Xingming Ye, Jianfeng Zheng, Dan Hu, Li Liu, Fukun Chen, Xintong Cai, Yangmei Xu, Lifeng Li, Jie Lin, Qinying Liu, Yang Sun
          </td>
          <td>2025-02-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="To better understand the homeostatic mechanisms governing melanocytes, we performed deep phenotyping of clonal expansions of single melanocytes from human skin. In total, we interrogated the mutational landscapes, gene expression profiles, and morphological features of 297 melanocytes from 31 donors. To our surprise, a population of melanocytes with low mutation burden was maintained in sun damaged skin. These melanocytes were more stem-like, smaller, less dendritic and displayed distinct gene expression profiles compared to their counterparts with high mutation burdens. We used single-cell spatial transcriptomics (10X Xenium) to reveal the spatial distribution of melanocytes inferred to have low and high mutation burdens (LowMut and HighMut cells), based on their gene expression profiles. LowMut melanocytes were found in hair follicles as well as in the interfollicular epidermis, whereas HighMut melanocytes resided almost exclusively in the interfollicular epidermis. We propose that melanocytes in the hair follicle occupy a privileged niche, protected from UV radiation, but periodically migrate out of the hair follicle to replenish the interfollicular epidermis after waves of photodamage. More broadly, our study illustrates the advantages of a cell atlas that includes mutational information, as cells can change their cellular states and positional coordinates over time, but mutations are like scars, providing a historical record of the homeostatic processes that were operative on each cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35fcdb72f7ddf292a52ab622c06a7d378c43f17" target='_blank'>
              Somatic mutations distinguish melanocyte subpopulations in human skin
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, Neda Bahrani, Beatrice Weier, Harsh Sharma, Noel Cruz-Pacheco, Min Hu, Kayla Marks, Rebecca G. Zitnay, Aravind K. Bandari, Rojina Nekoonam, I. Yeh, R. Judson-Torres, A. H. Shain
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Patient-derived organoids represent a novel platform to recapitulate the cancer cells in the patient tissue. While cancer heterogeneity has been extensively studied by a number of omics approaches, little is known about the spatiotemporal kinase activity dynamics. Here we applied a live imaging approach to organoids derived from 10 pancreatic ductal adenocarcinoma (PDAC) patients to comprehensively understand their heterogeneous growth potential and drug responses. By automated wide-area image acquisitions and analyses, the PDAC cells were non-selectively observed to evaluate their heterogeneous growth patterns. We monitored single-cell ERK and AMPK activities to relate cellular dynamics to molecular dynamics. Furthermore, we evaluated two anti-cancer drugs, a MEK inhibitor, PD0325901, and an autophagy inhibitor, hydroxychloroquine (HCQ), by our analysis platform. Our analyses revealed a phase-dependent regulation of PDAC organoid growth, where ERK activity is necessary for the early phase and AMPK activity is necessary for the late stage of organoid growth. Consistently, we found PD0325901 and HCQ target distinct organoid populations, revealing their combination is widely effective to the heterogeneous cancer cell population in a range of PDAC patient-derived organoid lines. Together, our live imaging quantitatively characterized the growth and drug sensitivity of human PDAC organoids at multiple levels: in single cells, single organoids, and individual patients. This study will pave the way for understanding the cancer heterogeneity and promote the development of new drugs that eradicate intractable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824e882598ac674f498c3ffe96037afbee14d231" target='_blank'>
              Quantitative Live Imaging Reveals Phase Dependency of PDAC Patient-Derived Organoids on ERK and AMPK Activity.
              </a>
            </td>
          <td>
            Shoko Tsukamoto, Ye Huaze, Weisheng Zhang, Akihito Machinaga, N. Kakiuchi, Seishi Ogawa, Hiroshi Seno, Shigeki Higashiyama, Michiyuki Matsuda, Toru Hiratsuka
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Afatinib-induced tumor and microenvironment modifications in head and neck squamous cell carcinoma were evaluated by spatial transcriptomics in surgical specimens and RNA-sequencing in tumor biopsies of patients included in the EORTC-90111-24111 window-of-opportunity study. The aim was to explore tumor evolution and composition under anti-HER therapy. Based on our previous investigations by RNA-seq on tumor biopsies, surgical slides of ID08 and ID15 from the epithelial-to-mesenchymal (EMT) cluster and ID30 from the non-EMT cluster were investigated with spatial transcriptomics. Dimension reduction in ID30 revealed 14 clusters, with clusters overlapping three tumor nodules and the stroma. Differential expression analysis between tumor nodules showed enrichment of the hallmark EMT genelist, with 123 genes in common between the analyses. These genes were involved in PDGF and MET signaling pathways. By comparing gene expression in paired tumor biopsies and the 123 genes from differential analyses obtained in ID30, a list of 13 genes involved in cancer pathways and EMT emerged, which were also highly expressed in ID08 and ID15. These results show a progressive apparition of genes implicated in EMT, MET, and PDGF pathways in tumors after afatinib. Notably, a list of 13 genes emerged which may contain targets to prevent tumor evolution after anti-HER therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce3fcfb23899394262473ee209f0d7889b6808e" target='_blank'>
              Spatial Transcriptomic Analysis of Surgical Resection Specimens of Primary Head and Neck Squamous Cell Carcinoma Treated with Afatinib in a Window-of-Opportunity Study (EORTC90111-24111)
              </a>
            </td>
          <td>
            S. Beyaert, A. Loriot, J. Machiels, S. Schmitz
          </td>
          <td>2025-02-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated. Methods To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays. Results In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1+FOLR2+ tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway. Conclusions We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1+FOLR2+TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Previous studies revealed that tumor-associated macrophages/microglia (TAMs) promoted glioma invasiveness during tumor progression and after radiotherapy. However, the communication of TAMs with tumor cells remains unclear. This study aimed to examine the role of small extracellular vesicles (sEVs) derived from TAMs in TAMs-mediated brain tumor invasion. This study utilized BV2 and RAW264.7 cell lines representing resident and infiltrating macrophages, respectively, to unveil their effect on tumor cells. Purified sEVs from BV2 and RAW264.7 were validated by nanoparticle track analysis (NTA), transmission electron microscopy (TEM), and western blotting for sEV markers. The effect of sEVs on the murine astrocytoma tumor cell line ALTS1C1 was examined on cell proliferation, migration, and gene expression. The results showed that ALTS1C1 cells effectively engulfed sEVs purified from BV2 and RAW264.7. Only BV2-derived sEVs promoted cell proliferation and were dose-dependent. Further, morphological changes in ALTS1C1 cells were observed after incubation with BV2-derived sEVs, which was associated with enhancing cell migration. BV2-mediated glioma proliferation and mobility were related to the upregulation of vascular endothelial growth factor (VEGF) and downregulation of death effector domain-containing protein (DEDD) gene expression. This study demonstrates the distinct function of sEVs of resident macrophages on glioma cell invasion and reveals the mechanism underlying microglia-mediated tumor progression. These findings suggested resident microglia is the potential therapeutic target for TAMs-induced brain tumor invasiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f062fb3413e27a9de43358674b8a4c6d1e771176" target='_blank'>
              Distinct roles of small extracellular vesicles from resident and infiltrating macrophages on glioma growth and mobility
              </a>
            </td>
          <td>
            Chu-Hsuan Chen, Sheng-Yun Hsu, Wen-Jui Yu, Chi-Shiun Chiang, Ching-Fang Yu
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
Tumor hypoxia reshapes the microenvironment, driving progression, invasion, metastasis, and therapy resistance. Patient-derived tumor organoids, formed under three-dimensional conditions, preserve cellular heterogeneity and nutrient gradients, making them ideal for studying hypoxia-induced tumor responses. This study examines hypoxia-induced changes in lung tumor organoids from two patients, focusing on tumor-associated markers, stem cell markers, and migration capabilities.


RESULTS
Our findings demonstrate that hypoxia distinctively modulates the expression of lung cancer markers thyroid transcription factor-1, cytokeratin 7, and ΔNP63 variant. Hypoxia also induces the upregulation of stem cell-associated markers, resulting in an increased proportion of cancer stem cells. Furthermore, hypoxic lung tumor organoids exhibit unique migratory behavior upon reoxygenation, driven by epithelial-mesenchymal transition and the elevated expression of matrix metalloproteinases 7 and matrix metalloproteinases 9, indicating their enhanced invasive potential.


CONCLUSIONS
These findings highlight the value of lung tumor organoids as models for studying hypoxia's complex role in lung cancer. Hypoxia significantly modulates lung tumor organoids growth, stemness, and migratory behavior, providing critical insights into tumor progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic activating mutations in KRAS can cause complex lymphatic anomalies (CLAs). However, the specific processes that drive KRAS-mediated CLAs have yet to be fully elucidated. Here, we used single-cell RNA sequencing to construct an atlas of normal and KrasG12D-malformed lymphatic vessels. We identified six subtypes of lymphatic endothelial cells (LECs) in the lungs of adult wild-type mice (Ptx3, capillary, collecting, valve, mixed, and proliferating). To determine when the LEC subtypes were specified during development, we integrated our data with data from four stages of development. We found that proliferating and Ptx3 LECs were prevalent during early lymphatic development and that collecting and valve LECs emerged later in development. Additionally, we discovered that the proportion of Ptx3 LECs decreased as the lymphatic network matured but remained high in KrasG12D mice. We also observed that the proportion of collecting and valve LECs was lower in KrasG12D mice than in wild-type mice. Last, we found that immature lymphatic vessels in young mice were more sensitive to the pathologic effects of KrasG12D than mature lymphatic vessels in older mice. Together, our results expand the current model for the development of the lymphatic system and suggest that KRAS mutations impair the maturation of lymphatic vessels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b83680aa397127e25da0c4fb9fc494573ba691b" target='_blank'>
              A single-cell atlas of normal and KRASG12D-malformed lymphatic vessels.
              </a>
            </td>
          <td>
            Lorenzo M. Fernandes, Danielle Griswold-Wheeler, Jeffrey D Tresemer, Angelica Vallejo, Neda Vishlaghi, Benjamin Levi, Abigail Shapiro, Joshua P. Scallan, Michael T. Dellinger
          </td>
          <td>2025-01-28</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d7052c816a7546a33fc12f61bcaaa555f6b028" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomics reveals cellular subtypes and molecular features associated with osteosarcoma prognosis
              </a>
            </td>
          <td>
            Feng Liu, Tingting Zhang, Yongqiang Yang, Kailun Wang, Jinlan Wei, Ji-Hua Shi, Dong Zhang, Xia Sheng, Yi Zhang, Jing Zhou, Faming Zhao
          </td>
          <td>2025-02-17</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-Yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="BACKGROUND Gastric cancer (GC) poses a substantial risk to human health due to its high prevalence and mortality rates. Nevertheless, current therapeutic strategies remain insufficient. Single-cell RNA sequencing (scRNA-seq) offers the potential to provide comprehensive insights into GC pathogenesis. AIM To explore the distribution and dynamic changes of cell populations in the GC tumor microenvironment using scRNA-seq techniques. METHODS Cancerous tissues and paracancerous tissues were obtained from patients diagnosed with GC at various stages (I, II, III, and IV). Single-cell suspensions were prepared and analyzed using scRNA-seq to examine transcriptome profiles and cell-cell interactions. Additionally, quantitative real-time polymerase chain reaction (qRT-PCR) and flow cytometry were applied for measuring the expression of cluster of differentiation (CD) 2, CD3D, CD3E, cytokeratin 19, cytokeratin 8, and epithelial cell adhesion molecules. RESULTS Transcriptome data from 73645 single cells across eight tissues of four patients were categorized into 25 distinct cell clusters, representing 10 different cell types. Variations were observed in these cell type distribution. The adjacent epithelial cells in stages II and III exhibited a degenerative trend. Additionally, the quantity of CD4 T cells and CD8 T cells were evidently elevated in cancerous tissues. Interaction analysis displayed a remarkable increase in interaction between B cells and other mast cells in stages II, III, and IV of GC. These findings were further validated through qRT-PCR and flow cytometry, demonstrating elevated T cells and declined epithelial cells within the cancerous tissues. CONCLUSION This study provides a comprehensive analysis of cell dynamics across GC stages, highlighting key interactions within the tumor microenvironment. These findings offer valuable insights for developing novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3adb4df70cb2b109a0b602e7b75fd956596d307f" target='_blank'>
              Landscape of four different stages of human gastric cancer revealed by single-cell sequencing
              </a>
            </td>
          <td>
            Xushan Tang, Chun-Lei Xu, Na Li, Jian-Qing Zhang, Yong Tang
          </td>
          <td>2025-02-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Rationale: Glioblastoma (GBM) is the most fatal form of brain cancer and its treatment represents a persistent challenge. Mesenchymal stromal cells (MSCs) have been explored as therapeutic tools in cancer management owing to their tumor-homing abilities. However, their clinical application is limited due to the controversial role of MSCs in carcinogenesis. This study investigates how MSCs influence tumor behavior and explores the synergistic anticancer effects in combination with melatonin (Mel). Methods: Orthotopic and subcutaneous GBM xenograft mouse models were used to assess the antitumor effect of Mel pre-treated MSCs (MSC Mel ). Histological, immunohistochemical, and ultrastructural analyses were conducted to identify phenotypic changes in tumors. Through a set of in vitro assays, including direct and indirect co-cultures, dynamic single-cell tracking and tumorsphere assay, we explored the impact of MSC Mel on primary and non-primary GBM cells. Transcriptomic profiling was used to identify genes and pathways modulated by this synergistic therapy. Results: MSC Mel delayed tumor growth in mice and increased collagen deposition. Additionally, MSC Mel showed enhanced capacity to prevent GBM cell migration compared to untreated MSCs. Molecular analysis identified genes and proteins related to cell migration, cytoskeletal dynamics and extracellular matrix remodeling in GBM cells exposed to MSC Mel , including reduced vimentin expression. Finally, a genetic signature associated with the clinical outcomes of GBM patients was identified. Conclusions: Our study demonstrates that melatonin enhances the anticancer properties of MSCs, providing new insights into their interaction with GBM cells and tumor environment. These findings offer valuable guidance for advancing MSC-based therapies in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f57dedd75349ecd50ac992a7b9ebfe735e08696" target='_blank'>
              Glioblastoma progression is hindered by melatonin-primed mesenchymal stromal cells through dynamic intracellular and extracellular reorganizations
              </a>
            </td>
          <td>
            Laura Olmedo-Moreno, Concepción Panadero-Morón, J. M. Sierra-Párraga, Rubén Bueno-Fernández, E. Norton, Y. Aguilera, N. Mellado-Damas, Patricia García-Tárraga, Raquel Morales-Gallel, María Antequera-Martínez, Raúl V. Durán, Jaime Ferrer-Lozano, Germaine Escames, J. García-Verdugo, Alejandro Martín-Montalvo, Hugo Guerrero-Cázares, V. Capilla-González
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38fe8b7dd282acd4da49fd77a60ef2b609353e0" target='_blank'>
              S100A13-driven interaction between pancreatic adenocarcinoma cells and cancer-associated fibroblasts promotes tumor progression through calcium signaling
              </a>
            </td>
          <td>
            Liuyuan Xia, Xin Guo, Dong Lu, Yixin Jiang, Xiaohui Liang, Yiwen Shen, Jia-Yi Lin, Lijun Zhang, Hongzhuan Chen, Jinmei Jin, Xin Luan, Weidong Zhang
          </td>
          <td>2025-01-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed731d817cd7c367b8f1abe8184ac2fe55d110b6" target='_blank'>
              Mapping the developmental trajectory of human astrocytes reveals divergence in glioblastoma.
              </a>
            </td>
          <td>
            Caitlin Sojka, Hsiao-Lin V. Wang, Tarun N. Bhatia, Yangping Li, Pankaj Chopra, Anson Sing, Anna Voss, Alexia King, Feng Wang, Kevin Joseph, V. Ravi, Jeffrey Olson, Kimberly Hoang, E. Nduom, Victor G. Corces, Bing Yao, Steven A. Sloan
          </td>
          <td>2025-01-08</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is a major challenge that was recognized as one of the first Cancer Grand Challenges, with a call to provide solutions to visualize tumor heterogeneity. The Rosetta team took on this challenge, exploiting advances in spatial-omics approaches centered around mass spectrometry imaging to map tumor heterogeneity at the cellular and molecular scales with different levels of resolution. See related article by Bressan et al., p. 16 See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d308601bb28236e43c3bc271c19d29c8dd8720ed" target='_blank'>
              Visualizing Cancer Heterogeneity at the Molecular and Cellular Levels: Lessons from Rosetta.
              </a>
            </td>
          <td>
            R. Goodwin, John F Marshall, G. Poulogiannis, Mariia Yuneva, K. Brindle, Z. Takáts, Owen J. Sansom, Josephine Bunch, Simon T Barry
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hypoxia, a common feature in many malignancies, is particularly prominent in cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC). Investigating the mechanisms underlying hypoxia is essential for understanding the heterogeneity of CESC and developing personalized therapeutic regimens. Firstly, the CESC-specific hypoxia gene sets shared between single-cell RNA sequencing (scRNA-seq) and bulk data were identified through Weighted Gene Correlation Network Analysis (WGCNA)and FindMarkers analyses. A CESC-specific hypoxia-related score (CSHRS) risk model was constructed using the least absolute shrinkage and selection operator (LASSO)and Cox regression analyses based on these genes. The prognostic differences were analyzed in terms of immune infiltration, mutations, and drug resistance. Finally, a nomogram model was constructed by integrating clinicopathological features to facilitate precision treatment for CESC. This study constructed a CSHRS risk model that divides patients into two groups, and this model can comprehensively evaluate the tumor microenvironment characteristics of CESC, provide accurate prognostic predictions, and offer rational treatment options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7d7f2c99de4e5f9ef9acea8ebc376cd53f13e8" target='_blank'>
              Integrating Hypoxia Signatures from scRNA-seq and Bulk Transcriptomes for Prognosis Prediction and Precision Therapy in Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma
              </a>
            </td>
          <td>
            Kexin Yu, Shibo Zhang, Jiali Shen, Meini Yu, Yangguang Su, Ying Wang, Kun Zhou, Lei Liu, Xiujie Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb5d097abc3ea1eadfe8415247762a28c6420b7" target='_blank'>
              Oncolytic reprogramming of tumor microenvironment shapes CD4 T-cell memory via the IL6ra-Bcl6 axis for targeted control of glioblastoma
              </a>
            </td>
          <td>
            Jeffrey M Grimes, Sadashib Ghosh, Shamza Manzoor, Li X Li, Monica M Moran, Jennifer C. Clements, Sherrie D Alexander, James M. Markert, J. Leavenworth
          </td>
          <td>2025-01-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba71338c6c092a7d53ee875fe94f60877fe823a" target='_blank'>
              Single-cell transcriptomics identify a novel macrophage population associated with bone invasion in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Xinzhi Wu, Xueshuai Han, Haibo Zhu, Mingxuan Li, L. Gong, Sicheng Jing, Weiyan Xie, Zhaoqi Liu, Chuzhong Li, Yazhuo Zhang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Background Thyroid cancer is one of the most common endocrine tumors worldwide, especially among women and the metastatic mechanism of papillary thyroid carcinoma remains poorly understood. Methods Thyroid cancer tissue samples were obtained for single‐cell RNA‐sequencing and spatial transcriptomics, aiming to intratumoral and antimetastatic heterogeneity of advanced PTC. The functions of APOE in PTC cell proliferation and invasion were confirmed through in vivo and in vitro assays. Pseudotime analysis and CellChat were performed to explore the the molecular mechanisms of the APOE in PTC progression. Results We identified a subpopulation of tumor cells with lower expression levels of APOE, associated with advanced stages of PTC and cervical metastasis. APOE overexpression significantly reduced tumor cell proliferation and invasion, both in vitro and in vivo, by activating the ABCA1‐LXR axis. APOE− tumor cells may promote tumor growth by interacting with dendritic cells and CD4+ T cells via CD99‐ rather than CD6‐regulated signaling. We established a machine learning‐based scRNA‐seq data, 13‐gene signature predictive of lymph node metastasis. Conclusions We identified a distinct APOE− tumor cell population associated with cervical metastasis and poor prognosis. Our results and models have potential clinical, prognostic, and therapeutic implications for advanced PTC. Key points A subpopulation of tumor cells with lower expression levels of APOE was strongly associated with more advanced stages and metastasis of PTC. APOE‐negative (APOE −) cellsoverall exhibited weaker interactions with immune cells. A machine‐learning bioinformatics model based on scRNA‐seq data of in‐situ thyroid cancer tissue was established to predict lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170f7e4d14fbccaf4771b3c72a44d532b8d44e86" target='_blank'>
              Single‐cell RNA‐sequencing and spatial transcriptomic analysis reveal a distinct population of APOE − cells yielding pathological lymph node metastasis in papillary thyroid cancer
              </a>
            </td>
          <td>
            Guohui Xiao, Rongli Xie, Jian-hua Gu, Yishu Huang, Min Ding, Dongjie Shen, Jiqi Yan, Jianmin Yuan, Qiong Yang, Wen He, Siyu Xiao, Haizhen Chen, Dan Xu, Jian Wu, Jian Fei
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Metastasis remains a leading cause of cancer‐related deaths, defined by a complex, multi‐step process in which tumor cells spread and form secondary growths in distant tissues. Despite substantial progress in understanding metastasis, the molecular mechanisms driving this process and the development of effective therapies remain incompletely understood. Elucidating the molecular pathways governing metastasis is essential for the discovery of innovative therapeutic targets. The rapid advancements in sequencing technologies and the expansion of biological databases have significantly deepened our understanding of the molecular drivers of metastasis and associated drug resistance. This review focuses on the molecular drivers of metastasis, particularly the roles of genetic mutations, epigenetic changes, and post‐translational modifications in metastasis progression. We also examine how the tumor microenvironment influences metastatic behavior and explore emerging therapeutic strategies, including targeted therapies and immunotherapies. Finally, we discuss future research directions, stressing the importance of novel treatment approaches and personalized strategies to overcome metastasis and improve patient outcomes. By integrating contemporary insights into the molecular basis of metastasis and therapeutic innovation, this review provides a comprehensive framework to guide future research and clinical advancements in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/647f5025b06ef5ed04146a75e8b99dcf5629913e" target='_blank'>
              Tumor Metastasis: Mechanistic Insights and Therapeutic Intervention
              </a>
            </td>
          <td>
            Lin Tang, Shao‐Cong Peng, Xiao‐Wan Zhuang, Yan He, Yu‐Xiang Song, Hao Nie, Can-Can Zheng, Zhen‐Yu Pan, Alfred King‐Yin Lam, , Xingyuan Shi, Bin Li, W. Xu
          </td>
          <td>2025-02-17</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a common malignant tumor of the digestive system with a high incidence that seriously threatens patients’ lives and health. However, with the rise and application of new treatments, such as immunotherapy, there are still some restrictions in the treatment and diagnosis of HCC, and the therapeutic effects on patients are not ideal. Methods Two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, encompassing 25,189 cells, were analyzed in the study. We utilized non-negative matrix factorization (NMF) clustering to identify mitophagy patterns in HCC TME cells, including cancer-associated fibroblasts (CAFs), T cells, B cells, and tumor-associated macrophages (TAMs). Cell-to-cell communication was analyzed using the CellChat package, and pseudotime trajectory analysis was performed using the Monocle package. Gene regulatory networks were investigated with the SCENIC package, and survival analyses were conducted with mitophagy-related signatures. Results HCC samples analysis identified 22 clusters, including 7 principal cell types. Complex cell communications were observed among these cell types. Mitophagy-related CAFs, TAMs, CD8+ T cells, and B cells were identified. These subtypes had different biological states, cell-cell communications, and metabolic pathways. Mitophagy levels were elevated in tumor samples. Changes in mitophagy-related genes within specific cell subtypes were associated with different overall survival rates. However, mitophagy did not seem to affect the effectiveness of immunotherapy. Conclusion This study provides evidence that mitophagy within the HCC TME modulates intercellular communication, influencing tumor progression and patient prognosis. Targeting mitophagy may offer a promising approach to improve the long-term prognosis of HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/689de3b101ffbbea61436b63812b3f0a30320588" target='_blank'>
              Single-cell mitophagy patterns within the tumor microenvironment modulate intercellular communication, impacting the progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengyan Li, Wei Chen, Shu Yao, Zuxiang Peng, Hongming Liu, Yongliang Tang, Yi Feng
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60a8517e150bdfc65ec6184359a1270c9f5e213f" target='_blank'>
              Pathways and outputs orchestrated in tumor microenvironment cells by hypoxia-induced tumor-derived exosomes in pan-cancer.
              </a>
            </td>
          <td>
            Ozel Capik, Omer Faruk Karatas
          </td>
          <td>2025-02-10</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cells under high confinement form highly polarized hydrostatic pressure-driven, stable leader blebs that enable efficient migration in low adhesion, environments. Here we investigated the basis of the polarized bleb morphology of metastatic melanoma cells migrating in non-adhesive confinement. Using high-resolution time-lapse imaging and specific molecular perturbations, we found that EGF signaling via PI3K stabilizes and maintains a polarized leader bleb. Protein activity biosensors revealed a unique EGFR/PI3K activity gradient decreasing from rear-to-front, promoting PIP3 and Rac1-GTP accumulation at the bleb rear, with its antagonists PIP2 and RhoA-GTP concentrated at the bleb tip, opposite to the front-to-rear organization of these signaling modules in integrin-mediated mesenchymal migration. Optogenetic experiments showed that disrupting this gradient caused bleb retraction, underscoring the role of this signaling gradient in bleb stability. Mathematical modeling and experiments identified a mechanism where, as the bleb initiates, CD44 and ERM proteins restrict EGFR mobility in a membrane-apposed cortical actin meshwork in the bleb rear, establishing a rear-to-front EGFR-PI3K-Rac activity gradient. Thus, our study reveals the biophysical and molecular underpinnings of cell polarity in bleb-based migration of metastatic cells in non-adhesive confinement, and underscores how alternative spatial arrangements of migration signaling modules can mediate different migration modes according to the local microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c8181984f5d71cab72b56985be9cc1b4aaea645" target='_blank'>
              CD44 and Ezrin restrict EGF receptor mobility to generate a novel spatial arrangement of cytoskeletal signaling modules driving bleb-based migration
              </a>
            </td>
          <td>
            Ankita Jha, Ankit Chandra, Payam E. Farahani, Jared E. Toettcher, J. Haugh, Clare M. Waterman
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc019fa42801c8582b11a460b1609019f96845de" target='_blank'>
              A minimal gene set characterizes TIL specific for diverse tumor antigens across different cancer types
              </a>
            </td>
          <td>
            Zhen Zeng, Tianbei Zhang, Jiajia Zhang, Shuai Li, Sydney C Connor, Boyang Zhang, Yimin Zhao, Jordan Wilson, Dipika Singh, R. Kulikauskas, C. Church, T. Pulliam, Saumya Jani, P. Nghiem, S. Topalian, Patrick M Forde, Drew M Pardoll, Hongkai Ji, Kelli N. Smith
          </td>
          <td>2025-02-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, A. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Simple Summary Immunosenescence is the decrease in the function of the immune system with age. The immune system plays a key role in the development and progression of tumors. Targeting immunosenescence is considered a promising therapeutic approach to improving tumor prognosis. However, few studies have been conducted to reveal immunosenescence biomarkers of lung cancer. Thus, our study aimed to fill this critical knowledge gap by developing an immunosenescence gene set to characterize tumor immune microenvironment and proposing an immunosenescence risk model to understand the roles of immunosenescence in tumor prognosis and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/512de6ce565c6e66f2ab70bd40a4880bf8182c3c" target='_blank'>
              Single-Cell and Bulk Transcriptomics Reveal the Immunosenescence Signature for Prognosis and Immunotherapy in Lung Cancer
              </a>
            </td>
          <td>
            Yakun Zhang, Jiajun Zhou, Yitong Jin, Chenyu Liu, Hanxiao Zhou, Yue Sun, Han Jiang, Jing Gan, Caiyu Zhang, Qianyi Lu, Yetong Chang, Yunpeng Zhang, Xia Li, Shangwei Ning
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff35b3725766793dc6e344fca09fac4e90d00646" target='_blank'>
              LATS1/2 inactivation in the mammary epithelium drives the evolution of a tumor-associated niche.
              </a>
            </td>
          <td>
            Joseph G. Kern, Lina Kroehling, Anthony J Spinella, Stefano Monti, X. Varelas
          </td>
          <td>2025-02-14</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon D. Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="We seek to establish a parsimonious mathematical framework for understanding the interaction and dynamics of the response of pancreatic cancer to the NGC triple chemotherapy regimen (mNab-paclitaxel, gemcitabine, and cisplatin), stromal-targeting drugs (calcipotriol and losartan), and an immune checkpoint inhibitor (anti-PD-L1). We developed a set of ordinary differential equations describing changes in tumor size (growth and regression) under the influence of five cocktails of treatments. Model calibration relies on three tumor volume measurements obtained over a 14-day period in a genetically engineered pancreatic cancer model (KrasLSLG12D-Trp53LSLR172H-Pdx1-Cre). Our model reproduces tumor growth in the control and treatment scenarios with an average concordance correlation coefficient (CCC) of 0.99±0.01. We conduct leave-one-out predictions (average CCC=0.74±0.06), mouse-specific predictions (average CCC=0.75±0.02), and hybrid, group-informed, mouse-specific predictions (average CCC=0.85±0.04). The developed mathematical model demonstrates high accuracy in fitting the experimental tumor data and a robust ability to predict tumor response to treatment. This approach has important implications for optimizing combination NGC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a317eb5daf4775d488a8cf45bdb23487a0a8c3" target='_blank'>
              Modeling tumor dynamics and predicting response to chemo-, targeted-, and immune-therapies in a murine model of pancreatic cancer
              </a>
            </td>
          <td>
            Krithik Vishwanath, Hoon Choi, Mamta Gupta, Rong Zhou, A. Sorace, T. Yankeelov, Ernesto A. B. F. Lima
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d017ae1208a1788cb6dfdbf710e4c68d672f7f57" target='_blank'>
              Network-based transfer of pan-cancer immunotherapy responses to guide breast cancer prognosis
              </a>
            </td>
          <td>
            Xiaobao Ding, Lin Zhang, M. Fan, Lihua Li
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3fe6ab149b32eadab919b62a58d2ecdad6d28ca" target='_blank'>
              Senescence-specific molecular subtypes stratify the hallmarks of the tumor microenvironment and guide precision medicine in bladder cancer
              </a>
            </td>
          <td>
            Luzhe Yan, Haisu Liang, Tiezheng Qi, D. Deng, Jinhui Liu, Yunbo He, Jinbo Chen, Benyi Fan, Yi-min Yao, Kun Wang, Xiongbing Zu, Minfeng Chen, Yuanqing Dai, Jiao Hu
          </td>
          <td>2025-02-19</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract: Background/Objectives: The heterogeneity of the tumor immune microenvironment is a key determinant of tumor oncogenesis. This study aims to evaluate the composition of seven immune cells across 5323 samples from 14 cancers using DNA methylation data. Methods: A deconvolution algorithm was proposed to estimate the composition of seven immune cells using 1256 immune cell population-specific methylation genes. Based on the immune infiltration features of seven immune cell fractions, 42 subtypes of 14 tumors (2–5 subtypes per tumor) were identified. Results: Significant differences in immune cells between subtypes were revealed for each cancer. The study found that the methylation values of the selected specific sites correlated with gene expression in most tumor subtypes. Immune infiltration results were integrated with phenotypic data, including survival data and tumor stages, revealing significant correlations between immune infiltration and phenotypes in some tumors. Subtypes with high proportions of CD4+ T cells, CD8+ T cells, CD56+ NK cells, CD19+ B cells, CD14+ monocytes, neutrophils, and eosinophils were identified, with subtype counts of 9, 24, 22, 13, 19, 9, and 11, respectively. Additionally, 2412 differentially expressed genes between these subtypes and normal tissues were identified. Pathway enrichment analysis revealed that these genes were mainly enriched in pathways related to drug response and chemical carcinogens. Differences in ESTIMATE scores for subtypes of seven tumors and TIDE scores for eight tumors were also observed. Conclusions: This study demonstrates the intra-tumor and inter-tumor immune heterogeneity of pan-cancer through DNA methylation analysis, providing assistance for tumor diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adddec60e9424b127df5b7066697b39696f5a789" target='_blank'>
              Evaluation of Pan-Cancer Immune Heterogeneity Based on DNA Methylation
              </a>
            </td>
          <td>
            Yang Zhou, Jiebiao Liu, Bowen Shi, Te Ma, Peishen Yu, Ji Li, Yue Gu, Yan Zhang
          </td>
          <td>2025-01-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Collective migration of cancer cells is often interpreted using concepts derived from the physics of active matter, but the experimental evidence is mostly restricted to observations made in vitro. Here, we study collective invasion of metastatic cancer cells injected into the mouse deep dermis using intravital multiphoton microscopy combined with a skin window technique and three-dimensional quantitative image analysis. We observe a multicellular but low-cohesive migration mode characterized by rotational patterns which self-organize into antiparallel persistent tracks with orientational nematic order. We analyze the deformations induced by the cells in the extracellular matrix and find broadly distributed strain bands with a prevalence of compression. A model of active nematic hydrodynamics is able to describe several statistical features of the experimentally observed flow, suggesting that collective cancer cell invasion can be interpreted as a nematic active fluid in the turbulent regime. Our results help elucidate the migration patterns of cancer cells in vivo and provide quantitative guidance for the development of realistic in vitro and in silico models for collective cell migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80ea8ee253be29748fc77f13c314bba503b6115" target='_blank'>
              Confined cell migration along extracellular matrix space in vivo.
              </a>
            </td>
          <td>
            O. Chepizhko, Josep-Maria Armengol-Collado, Stephanie Alexander, E. Wagena, B. Weigelin, Luca Giomi, Peter Friedl, Stefano Zapperi, C. A. L. La Porta
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer-associated fibroblasts (bCAFs) comprise inflammatory CAFs (iCAFs), characterized by the secretion of pro-inflammatory cytokines, and myofibroblastic CAFs (myCAFs), distinguished by their high production of extracellular matrix and their immunosuppressive properties. We previously showed that targeting the anti-apoptotic protein MCL-1 in primary culture of bCAF derived directly from human samples reduces their myofibroblastic characteristics. We herein show by single-cell RNA-sequencing analysis of bCAFs that MCL-1 knock down induces a phenotypic shift from wound-myCAF to IL-iCAF, characterized by the upregulation of genes associated with inflammation as well as angiogenesis-related genes. In vitro, genetic and pharmacologic MCL-1 inhibition increases VEGF secretion by bCAFs, enhancing endothelial cell tubulogenesis. In a chicken chorioallantoic membrane (CAM) model in ovo, co-engraftment of breast cancer cells and bCAFs with reduced MCL-1 expression leads to heightened peritumoral vascular density, driven by VEGF. Mechanistically, the pro-angiogenic phenotype revealed by MCL-1 inhibition is dependent on BAX-BAK activity. It results in NF-κB activation, inhibition of which by a IKKβ inhibitor suppresses the transcription of VEGF and pro-inflammatory factors triggered by MCL-1 inhibition in bCAFs. Chemotherapy induces a downregulation of MCL-1 in bCAFs and it promotes a pro-angiogenic phenotype, counteracted by overexpressed MCL-1. Overall, these findings uncover a novel regulatory function of MCL-1 in determining bCAF subpopulation differentiation and highlight its role in modulating their pro-angiogenic properties, in response to treatment in particular.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f78cbc704db86de972c4bf0a205bb9092495fef" target='_blank'>
              MCL-1 as a molecular switch between myofibroblastic and pro-angiogenic features of breast cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Chloé C. Lefebvre, Philippine Giowachini, J. Derrien, Maxime Naour, Isabelle Corre, Elise Douillard, F. Guillonneau, M. Campone, Philippe P. Juin, F. Souazé
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Fistula formation is a common and debilitating complication of Crohn’s disease (CD). Although rare, CD-associated fistula carcinomas present significant diagnostic and prognostic challenges. This study aims to identify disease-specific immune cell subsets in CD-associated fistula carcinomas.



 The study analyzed tissue samples from 10 CD patients with fistula carcinomas, 7 patients with CD-associated fistulas, and 6 patients with sporadic colorectal cancer (CRC). In the tumor samples, the main tumor, infiltration front, and non-involved areas were included. Using a 36-marker panel, the immune landscape was characterized through imaging mass cytometry. The samples were processed, stained, and analyzed for immune cell compositions, cell-cell interactions, and spatial microenvironments.



 The immune infiltrate observed in fistula carcinomas displayed similarities to both CD fistulas and CRC. Fistula-carcinoma samples exhibited elevated levels of neutrophils, B cells, and CD163high macrophages. CRC main tumor samples exhibited a higher presence of intraepithelial CD8+ lymphocytes and CD163low macrophages. The highest levels of Cleaved Caspase-3 were observed in CRC main tumor samples, which exhibited a positive correlation with CD163low macrophages and cytotoxic T cells. Conversely, a negative correlation was observed between cleaved Caspase-3 and cytotoxic T cells in fistula-carcinoma main tumor samples. The analysis of cellular microenvironments and dimensionality reduction clustering based on immune cell frequencies revealed that fistula carcinomas exhibit a unique combination of immune cell characteristics derived from both CD fistulas and CRC.



 The immune landscape of CD-associated fistula carcinomas exhibits characteristics of both CD fistulas and CRC, indicating a complex pathogenesis driven by chronic inflammation. These findings suggest that fistula carcinomas constitute a distinct cancer subtype, necessitating further research to develop targeted therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af6164edea7dd205617f64e1e47fc4cd0fe12b3" target='_blank'>
              P0179 Spatial immune profiling of Crohn’s Disease fistula carcinomas - Defining a distinct cancer subtype
              </a>
            </td>
          <td>
            M. Lehmann, D. Paclik, A. Huck, A. Arnold, C. Kurth-Stavenhagen, M. Vieth, C. Treese, A. A. Kühl, B. Siegmund
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadc098eba1b42a6711d8bde36ca00a426896d86" target='_blank'>
              KMO-driven metabolic reconfiguration and its impact on immune cell infiltration in nasopharyngeal carcinoma: a new avenue for immunotherapy
              </a>
            </td>
          <td>
            Nijun Chen, Yuan Zong, Chen Yang, Lei Li, Yang Yi, Jiawen Zhao, Xiaoyu Zhao, Xianfei Xie, Xingmei Sun, Ning Li, Liting Jiang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2ea126a4602d2754c646d3838a0c077f74d91b" target='_blank'>
              CXCL14 in prostate cancer: complex interactions in the tumor microenvironment and future prospects
              </a>
            </td>
          <td>
            Lei Tang, Xin Chen, Jianquan Hou, Xuedong Wei
          </td>
          <td>2025-01-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9629af3f9c8496403aa9d7445a8f7f79cca8a69" target='_blank'>
              Genes and proteins expression profile of 2D vs 3D cancer models: a comparative analysis for better tumor insights.
              </a>
            </td>
          <td>
            Sunaina Bhuker, Abhinav Kumar Sinha, Anuksha Arora, H. Tuli, Sonal Datta, A. Saini, R. Saini, S. Ramniwas
          </td>
          <td>2025-01-25</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c81eee6b668168bd971313aa0fe51bd578f98e" target='_blank'>
              Meta-analyses of mouse and human prostate single-cell transcriptomes reveal widespread epithelial plasticity in tissue regression, regeneration, and cancer
              </a>
            </td>
          <td>
            Luis C. Aparicio, Laura Crowley, John R. Christin, Caroline J. Laplaca, H. Hibshoosh, Raul Rabadan, Michael M. Shen
          </td>
          <td>2025-01-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f63f6a23a3983ad60b3216e3e5876ec300a9938" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis.
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, A. L. Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Determining mutations in the kinase domain of the epidermal growth factor receptor (EGFR) is critical for the effectiveness of EGFR tyrosine kinase inhibitors (TKIs) in lung cancer. Yet, DNA-based sequencing analysis of tumor samples is time-consuming and only provides gene mutation information on EGFR, making it challenging to design effective EGFR-TKI therapeutic strategies. Here, we present a new image-based method involving the rational design of a quenched probe based on EGFR-TKI to identify mutant proteins, which permits specific and "no-wash" real-time imaging of EGFR in living cells only upon covalent targeting of the EGFR kinase. We also show that the probe enables distinguishing EGFR mutant tumor-bearing mice from wild-type tumor-bearing mice via fluorescence-intensity-based imaging with high signal contrast. More interestingly, the image-based phenotypic approach can be used to predict EGFR mutations in tumors from lung cancer patients with an accuracy of 94%. Notably, when immunohistochemistry analysis is integrated, an improved accuracy of 98% is achieved. These data delineate a drug-based phenotypic imaging approach for in-biopsy visualization and define functional groups of EGFR mutants that can effectively guide EGFR-TKI therapeutic decision-making besides gene mutation analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ccf9820346bfbaf82734dfcdb76ade1b7e0ae9" target='_blank'>
              Image-Based Phenotypic Profiling Enables Rapid and Accurate Assessment of EGFR-Activating Mutations in Tissues from Lung Cancer Patients.
              </a>
            </td>
          <td>
            Qian Lei, Xinglong Zhou, Ying Li, Shuang Zhao, N. Yang, Zhaolin Xiao, Chao Song, Quanwei Yu, Hui Deng
          </td>
          <td>2025-01-02</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The kinetic parameters of cancer population dynamics are critical for developing reliable predictors of tumour growth patterns, extracting metrics for patient stratification and creating algorithms that can forecast clinically significant events. Here, we introduce a model-based Bayesian framework that leverages longitudinal phenotypic (e.g., tumour volume, cell counts) or genotypic (e.g., mutation frequency) data to infer critical parameters of tumour progression within a single patient. Our models uses population genetics to estimate probability distributions for tumour growth rates, initiation and extinction times, pinpointing abrupt shifts in tumour dynamics due to treatment response and revealing associations between drug resistance and pre-existing cancer cell populations. We apply our framework to address pivotal clinical questions across three major cancer types. In colorectal cancer, we use tumour markers data to identify extensive pre-existing RAS-linked resistance to cetuximab. In lung cancer, we use somatic mutation frequencies in citculating tumour DNA to determine prognostic growth rates and develop a test for monitoring minimal residual disease. In chronic lymphocytic leukaemia, we use white blood cell counts to stratify patients by growth patterns and predict time to treatment, advancing adaptive monitoring strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e324d640559e2fae6f8be8e484eb8d56fee86230" target='_blank'>
              Model-based Bayesian inference of cancer dynamics from heterogenous longitudinal data
              </a>
            </td>
          <td>
            Giovanni Santacatterina, R. Bergamin, A. Zucchetto, Roberta Laureana, M. D. del Principe, V. Gattei, Leonardo Egidi, G. Caravagna
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf42e4d5c5ce1d7e5ee6b6939079dd4741136db7" target='_blank'>
              Revolutionizing ESCC prognosis: the efficiency of tumor-infiltrating immune cells (TIIC) signature score
              </a>
            </td>
          <td>
            Haixia Wang, Shaowei Ma, Zixin Yang, Ren Niu, Haiyong Zhu, Shujun Li, Shao Gao, Zhirong Li, Yanhua Tian
          </td>
          <td>2025-01-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Collective cell migration is critical to embryonic development, wound healing, and the immune response, but also drives tumor dissemination. Understanding how cell collectives coordinate migration in vivo has been a challenge, with potential therapeutic benefits that range from addressing developmental defects to designing targeted cancer treatments. The small GTPase Rap1 has emerged as a regulator of both embryogenesis and cancer cell migration. How active Rap1 coordinates downstream signaling functions required for coordinated collective migration is poorly understood. Drosophila border cells undergo a stereotyped and genetically tractable in vivo migration within the developing egg chamber of the ovary. This group of 6-8 cells migrates through a densely packed tissue microenvironment and serves as an excellent model for collective cell migration during development and disease. Rap1, like all small GTPases, has distinct activity state switches that link extracellular signals to organized cell behaviors. Proper regulation of Rap1 activity is essential for successful border cell migration yet the signaling partners and other downstream effectors are poorly characterized. Using the known requirement for Rap1 in border cell migration, we conducted a dominant suppressor screen for genes whose heterozygous loss modifies the migration defects observed upon constitutively active Rap1V12 expression. Here we identified seven genomic regions on the X chromosome that interact with Rap1V12. We mapped three genomic regions to single Rap1-interacting genes, frizzled 4, Ubiquitin specific protease 16/45, and strawberry notch. Thus, this unbiased screening approach identified multiple new candidate regulators of Rap1 activity with roles in collective border cell migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f13ed6794927218b5942e30d7e2b21cc68f3faf" target='_blank'>
              A deficiency screen of the X chromosome for Rap1 GTPase dominant interacting genes in Drosophila border cell migration
              </a>
            </td>
          <td>
            C. L. Messer, Emily Burghardt, J. McDonald
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d719de0ed34b62d63deb744432982b8548a3e52" target='_blank'>
              TGF-β signaling orchestrates cancer-associated fibroblasts in the tumor microenvironment of human hepatocellular carcinoma: unveiling insights and clinical significance
              </a>
            </td>
          <td>
            Junwei Ge, Hongwei Jiang, Junjun Chen, Xuemin Chen, Yue Zhang, Liangrong Shi, Xiao Zheng, Jingting Jiang, Lujun Chen
          </td>
          <td>2025-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Epithelial cell collectives migrate through tissue interfaces and crevices to orchestrate development processes, tumor invasion, and wound healing. Naturally, the traversal of cell collective through confining environments involves crowding due to narrowing spaces, which seems tenuous given the conventional inverse relationship between cell density and migration. However, the physical transitions required to overcome such epithelial densification for migration across confinements remain unclear. Here, in a system of contiguous microchannels of varying confinements, we show that epithelial (MCF10A) monolayers accumulate higher cell density and undergo fluid-like shape transitions before entering narrower channels. However, overexpression of breast cancer oncogene ErbB2 did not require such accumulation of cell density to migrate across matrix confinement. While wild-type MCF10A cells migrated faster in narrow channels, this confinement sensitivity was reduced after +ErbB2 mutation or with constitutively active RhoA. This physical interpretation of collective cell migration as density and shape transitions in granular matter could advance our understanding of complex living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4a64576188154027b2ea1715663784bbe17d31" target='_blank'>
              Gradients in cell density and shape transitions drive collective cell migration into confining environments
              </a>
            </td>
          <td>
            Wan-Jung Lin, Hongsheng Yu, Amit Pathak
          </td>
          <td>2024-12-30</td>
          <td>Soft Matter</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8713ce532252c75bfca85781a29fddd0540c27c7" target='_blank'>
              Brd7 loss reawakens dormant metastasis initiating cells in lung by forging an immunosuppressive niche
              </a>
            </td>
          <td>
            Jayanta Mondal, Junfeng Zhang, Feng Qing, Shunping Li, Dhiraj Kumar, Jason T Huse, Filippo G. Giancotti
          </td>
          <td>2025-02-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K. Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="The mammary microenvironment has been shown to suppress tumor progression by redirecting cancer cells to adopt a normal mammary epithelial progenitor fate in vivo. However, the mechanism(s) by which this alteration occurs has yet to be defined. Here, we test the hypothesis that mitochondrial transfer from normal mammary epithelial cells to breast cancer cells plays a role in this redirection process. We evaluate mitochondrial transfer in 2D and 3D organoids using our unique 3D bioprinting system to produce chimeric organoids containing normal and cancer cells. We demonstrate that breast cancer tumoroid growth is hindered following interaction with mammary epithelial cells in both 2D and 3D environments. Furthermore, we show mitochondrial transfer occurs between donor mammary epithelial cells and recipient cancer cells primarily through tunneling nanotubes (TNTs) with minimal amounts seen from extracellular transfer of mitochondria, likely via extracellular vesicles (EVs). This organelle exchange results in various cellular and metabolic alterations within cancer cells, reducing their proliferative potential, and making them susceptible to microenvironmental control. Our results demonstrate that mitochondrial transfer contributes to microenvironmental redirection of cancer cells through alteration of metabolic and molecular functions of the recipient cancer cells. To the best of our knowledge, this is the first description of a 3D bioprinter-assisted organoid system for studying mitochondrial transfer. These studies are also the first mechanistic insights into the process of mammary microenvironmental redirection of cancer and provide a framework for new therapeutic strategies to control cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74a99d567922b268e37075173798f1b540e4d47" target='_blank'>
              Intercellular mitochondrial transfer contributes to microenvironmental redirection of cancer cell fate.
              </a>
            </td>
          <td>
            J. S. Bjerring, Yara Khodour, Emilee Anne Peterson, Patrick C. Sachs, Robert David Bruno
          </td>
          <td>2025-02-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
To meet their high energy needs, tumor cells undergo aberrant metabolic reprogramming. A tumor cell may expertly modify its metabolic pathways and the differential expression of the genes for metabolic enzymes. The physiological requirements of the host tissue and the tumor cell of origin mostly dictate metabolic adaptation. Ameloblastoma (AB) is a benign odontogenic tumor of epithelial origin. Due to its unrestricted growth potential, local aggressiveness, and high likelihood of recurrence, this condition poses a significant risk to the patient's health. This study aimed to characterize the metabolic heterogeneity at single-cell resolution of AB.


METHODS
Single-cell RNA sequencing (scRNA-seq) was performed on 17,284 cells from three AB donors. Bioinformatic analysis was used to examine differentially expressed genes, subtypes, and regulatory mechanisms when combined with odontogenic keratocyst scRNA-seq data. Based on metabolic pathway gene sets, the metabolic landscape of AB tumor cells was examined.


RESULTS
Using scRNA-seq, we discovered that AB tumor cells had substantial heterogeneity. The biggest contributor to tumor cell metabolic characteristics is determined to be variation in mitochondrial programming and glycolysis. Surprisingly, hypoxia corresponds with both oxidative phosphorylation and glycolysis activity in AB tumor cells at the single-cell level.


CONCLUSION
This study presents a computational framework for defining metabolism using single-cell expression data and identifies oxidative phosphorylation and glycolysis as critical components of metabolism for AB tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c116028597b1c5eaf72b4b7ed869064e15002ef" target='_blank'>
              Metabolic Analysis of Tumor Cells Within Ameloblastoma at the Single-Cell Level.
              </a>
            </td>
          <td>
            Rui-Fang Li, Yi Zhao, Qi-Wen Man
          </td>
          <td>2024-12-31</td>
          <td>Oral diseases</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) drive tumor progression through restructuring of the tumor microenvironment. This investigation aim to elucidate the function of molecular subtypes (MS) derived from cancer cells communication with CAFs, depicting the hallmarks of the tumor microenvironment and precise bladder cancer (BLCA) treatment. The BLCA data from TCGA and several external sources are utilized to generate a novel ligand, receptor, and transcription factor (LRT) associated molecular subtype and their corresponding score (LRT score). The LRT-mediated molecular subtype is identified via unsupervised clustering. LRT score is measured by principal component analysis. Then, the association of LRT clusters to established MS, immunophenotypes, and medical endpoints, together with BLCA treatment strategies is investigated. Two LRT clusters (A and B) are identified. LRT cluster (LRT score) can precisely propose immunophenotypes, classical MS, clinical outcomes, and BLCA therapeutic strategies. Cluster B (Low LRT score) represent a basal subtype and inflamed phenotype specified by high immunity against tumors and unfavorable clinical outcomes. Furthermore, it is highly sensitive to cancer immunotherapy; however, it has low sensitivity to antiangiogenic and targeted therapies. The novel LRT clusters with a strong association with biological characteristics and precise BLCA treatment strategies are derived from the communication between cancer cells and cancer-associated fibroblasts. The LRT may be a useful clinician tool for developing individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f8d25128e68026c13f6a41f5701ec7d66c8485" target='_blank'>
              Cancer Cell and Cancer-Associated Fibroblast Communication-Mediated Molecular Subtypes Portray Non-Inflamed Tumor Microenvironment and Guide the Precision Treatment of Bladder Cancer.
              </a>
            </td>
          <td>
            Shenglin Gao, Chuan Liu, Lixin Mao, Yin Chen, Xiaokai Shi, C. Yue, Shouchun Li, Xihu Qin
          </td>
          <td>2025-02-17</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood. Methods We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts. Results Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis. Conclusions DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            Tian-Hang Yu, Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu- Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3095fb742b0c6045977fafeab8ed1d06964edc6e" target='_blank'>
              FXYD5 regulates gastric cancer cell metastasis and drug resistance by EMT modulation.
              </a>
            </td>
          <td>
            Yuning Mao, Yaohua Hu, Han Meng, Jing Qin, Qingling An, Caiqin Zhang, Chenbo Guo, Yong Zhao, Dengxu Tan, Xu Ge, Changhong Shi
          </td>
          <td>2025-02-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Understanding early tumor formation offers critical insights for cancer prevention at the earliest stages. However, most in vivo studies focus on later stages, providing limited information on initial tumorigenesis. Hypoxia, a common physiological condition, is a primary microenvironmental factor in cancer. While molecular studies indicate significant yet sometimes conflicting roles of hypoxia in cancer progression, experimental data on hypoxia’s impact on tumor formation from a single cell, especially regarding cellular dynamics, remain scarce. To address this gap, we developed an integrated platform for live-cell imaging under hypoxic conditions, enabling, for the first time, observation of 3D tumor spheroid development from a single cell. Our results show that tumorigenesis is markedly accelerated under hypoxia. Beyond linear growth, we demonstrate that directional movement enhances fusion events, leading to larger spheroids that, in turn, stimulate further proliferation — a positive feedback loop between spheroid dynamics and growth. Actin polymerization and cell-cell adhesion are optimized to promote spheroid fusion. Additionally, hypoxia induces cells with outreaching F-actin co-localized with smaller nuclei to bridge gaps between spheroids, facilitating fusion, while the appearance of tiny nuclei suggests genetic instability. These findings provide direct insights into hypoxia’s role in early tumorigenesis, highlighting the potential of modulating dynamic, mechanical, and physiological factors for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0153fc66d9ff4cbf61d2a83aea2c920a18ecaa1" target='_blank'>
              Hypoxia-Induced Active Dynamics Promotes Early Tumorigenesis
              </a>
            </td>
          <td>
            Zhengduo Wang, Li Tian, Bo Li
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly malignant brain tumor with limited treatment options. Polo-like kinase 2 (PLK2), a member of the polo-like kinase family, has been variably implicated in cancer, but its role in GBM has not been fully elucidated. We utilized RNA-seq data from multiple databases, including Gene Expression Omnibus (GEO), the Cancer Genome Atlas (TCGA), and the Chinese Glioma Genome Atlas (CGGA), and conducted experiments on human glioma cell lines to explore PLK2's expression and function. The effects of PLK2 overexpression on GBM cell viability, proliferation, migration, cell cycle, and apoptosis were assessed, and the tumorigenic potential of PLK2 was evaluated in a mouse model. PLK2 was consistently downregulated in GBM tissues compared to normal brain tissues across several datasets. Overexpression of PLK2 in GBM cell lines U87MG and U251 reduced their tumorigenic potential and enhanced cell cycle arrest and apoptosis, with significant reductions observed in apoptosis markers. Our findings suggest that PLK2 may potentially function as a tumor suppressor in GBM. Hence, PLK2 overexpression could potentially be leveraged as a therapeutic strategy to inhibit tumor progression and enhance apoptosis, providing new avenues for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528eb3ce98e6565c84a21860767b84902d6f9be9" target='_blank'>
              A potential tumor suppressor role of PLK2 in glioblastoma.
              </a>
            </td>
          <td>
            X. Xia, Peirui Wang, Hua Xiao, Qishan Ran, Yan Li, Shengtao Yao
          </td>
          <td>2025-02-10</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Cervical cancer is the fourth most common cancer in women globally, and the main cause of the disease has been found to be ongoing HPV infection. Cervical cancer remains the primary cause of cancer-related death despite major improvements in screening and treatment approaches, especially in low- and middle-income nations. Therefore, it is crucial to investigate the tumor microenvironment in advanced cervical cancer in order to identify possible treatment targets. Materials and methods In order to better understand malignant cervical cancer epithelial cells (EPCs), this study used bulk RNA-seq data from UCSC in conjunction with single-cell RNA sequencing data from the ArrayExpress database. After putting quality control procedures into place, cell type identification and clustering analysis using the Seurat software were carried out. To clarify functional pathways, enrichment analysis and differential gene expression were carried out. The CIBERSORT and ESTIMATE R packages were used to evaluate the immune microenvironment characteristics, and univariate and multivariate Cox regression analyses were used to extract prognostic features. Furthermore, assessments of drug sensitivity and functional enrichment were carried out. Results Eight cell types were identified, with EPCs showing high proliferative and stemness features. Five EPC subpopulations were defined, with C1 NNMT+ CAEPCs driving tumor differentiation. A NNMT CAEPCs Risk Score (NCRS) model was developed, revealing a correlation between elevated NCRS scores and adverse patient outcomes characterized by immune evasion. In vitro experiments validated that the prognostic gene PLOD2 significantly enhances proliferation, migration, and invasion of cervical cancer cells. Conclusion This investigation delineated eight cell types and five subpopulations of malignant EPCs in cervical cancer, establishing the C1 NNMT+ CAEPCs as a crucial therapeutic target. The NCRS model demonstrated its prognostic capability, indicating that higher scores are associated with poorer clinical outcomes. The validation of PLOD2 as a prognostic gene highlights its therapeutic potential, underscoring the critical need for integrating immunotherapy and targeted treatment strategies to enhance diagnostic and therapeutic approaches in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a0fda708b879e8e016206b2e92375962d615799" target='_blank'>
              Single-cell RNA sequencing and immune microenvironment analysis reveal PLOD2-driven malignant transformation in cervical cancer
              </a>
            </td>
          <td>
            Zhiheng Lin, Fengxin Wang, Renwu Yin, Shengnan Li, Yuquan Bai, Baofang Zhang, Chenlin Sui, Hengjie Cao, Dune Su, Lianwei Xu, Honghong Wang
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Brain metastasis (BrM) has been a challenge for lung cancer treatment, but the mechanisms underlying lung cancer BrM remain elusive. This study aims to dissect cellular components and their spatial distribution in human BrM tumors of lung adenocarcinoma (LUAD) and identify potential therapeutic targets. Methods We performed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) on three LUAD BrMs, and validated our findings using public scRNA-seq data of 10 LUAD BrMs. Western blotting, quantitative real-time polymerase chain reaction (qRT-PCR) and functional experiments were employed for experimental studies. Results By combining scRNA-seq and ST, our analysis revealed the inter- and intra-tumoral heterogeneity of cellular components and their spatial localization within LUAD BrMs. Through RNA velocity and transcription factor (TF) regulatory activity analyses, we identified ATF3 as a potential regulator of the mesenchymal-epithelial transition (MET) program, which plays crucial roles in the colonization of tumor cells at metastatic sites. Furthermore, we demonstrated that knockdown of ATF3 significantly inhibited cancer cell proliferation while promoting cancer cell migration. Mechanistically, ATF3 knockdown could reverse the MET program. Additionally, we revealed that LGALS3/ANXA2-mediated cell-cell interaction between macrophage and tumor cells may also promote the MET program. Conclusions Our study provides a single-cell atlas of the cellular composition in BrM of LUAD and identifies ATF3 as a potential therapeutic target for BrM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e885c08703f3bf367c97f32c756f8659002f630" target='_blank'>
              Single-cell and spatial transcriptomics reveal a potential role of ATF3 in brain metastasis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Chaoliang Xu, Jingpiao Bao, Deshen Pan, Kehong Wei, Qing Gao, Weihong Lin, Yujie Ma, Meiqing Lou, Cheng Chang, Deshui Jia
          </td>
          <td>2025-01-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This study aimed to explore the impact of intratumoral microorganisms in conjunction with hormone receptors on the tumor microenvironment and their potential role in predicting patient prognosis. Significant bacterial variations were identified within ER, PR, HER2, and triple-negative breast cancer subtypes. Kaplan-Meier survival analysis was employed to identify bacteria associated with patient survival. Further, a humanized immune system mouse model bearing breast cancer xenografts was used to evaluate the effects of Stenotrophomonas maltophilia (SMA) on tumor growth and CD8+ T cell infiltration. Additional validation experiments included fluorescence in situ hybridization for SMA, CD8+ T cell chemotaxis, and intracellular cytokine detection. Lawsonella clevelandensis-A, Diaphorobacter nitroreducens, and SMA were identified as significant prognostic species. Notably, tumor-infiltrating immune cells, particularly CD8+ T cells, exhibited a positive association with the presence of SMA. Experimental validation with clinically isolated SMA further confirmed its positive correlation with CD8+ T cell activation. In vivo findings demonstrated that SMA inhibited tumor growth and promoted CD8+ T cell infiltration, highlighting the complex interactions between intratumoral microbiota and tumor immunity in breast cancer. These insights contribute to the understanding of microbial influences on the tumor microenvironment and suggest potential pathways for improving patient prognosis through microbiota modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fce5a50e9f9799a232183eb87230d236d9adfd" target='_blank'>
              Intratumoral Stenotrophomonas Maltophilia in Breast Cancer: Unraveling the Interplay with Hormone Receptors and Impact on Tumor Immunity
              </a>
            </td>
          <td>
            Qian Zhang, Dujuan Wang, Guangzheng Zhuo, Shilin Wang, Yufen Yuan, Liping Wang, Lili Ji, Yuhang Wan, Guohong Liu, Yunbao Pan
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Bladder cancer was the 10th most commonly diagnosed cancer worldwide in 2020. Extracellular vesicles (EVs) are nano-sized membranous structures secreted by all types of cells into the extracellular space. EVs can transport proteins, lipids, or nucleic acids to specific target cells. What brings more attention and potential implications is the fact that cancer cells secrete more EVs than non-malignant cells. EVs are widely studied for their role in cancer development. This publication summarizes the impact of EVs secreted by urinary bladder cancer cells on urinary bladder cancer development and metastasis. EVs isolated from urinary bladder cancer cells affect other lower-grade cancer cells or normal cells by inducing different metabolic pathways (transforming growth factor β/Smads pathway; phosphoinositide 3-kinase/Akt pathway) that promote epithelial–mesenchymal transition. The cargo carried by EVs can also induce angiogenesis, another critical element in the development of bladder cancer, and modulate the immune system response in a tumor-beneficial manner. In summary, the transfer of substances produced by tumor cells via EVs to the environment influences many stages of tumor progression. An in-depth understanding of the role EVs play in the development of urinary bladder cancer is crucial for the development of future anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d86a935ae8082dfc764f4690e13f477c7e940b8" target='_blank'>
              Extracellular vesicles—a new player in the development of urinary bladder cancer
              </a>
            </td>
          <td>
            Damian Kasiński, K. Szeliski, T. Drewa, M. Pokrywczyńska
          </td>
          <td>2025-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e45f953565d873f4e3de5c8d36330c9836a658" target='_blank'>
              Extracellular vesicles from pancreatic cancer and its tumour microenvironment promote increased Schwann cell migration
              </a>
            </td>
          <td>
            F. C. Wong, Sebastian R Merker, Lisa Bauer, Yi Han, Van Manh Hung Le, Carina Wenzel, Lukas Böthig, Max Heiduk, Pascal Drobisch, V. Rao, Farzaneh Malekian, Ana Mansourkiaei, Christian Sperling, Heike Polster, M. Pecqueux, R. Istvanffy, Linhan Ye, Bo Kong, Daniela E. Aust, Gustavo B Baretton, Lena Seifert, A. Seifert, J. Weitz, I. Demir, Christoph Kahlert
          </td>
          <td>2025-01-25</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Cancer metastasis involves cell migration from their primary organ foci into vascular channels, followed by dissemination to prospective colonization sites. Vascular entry of tumor cells or intravasation involves their breaching stromal and endothelial extracellular matrix (ECM) and the endothelial barriers. How the kinetics of this breach are confounded by chronic inflammatory stresses seen in diabetes and aging remains ill-investigated. To study the problem, a histopathology-motivated, imaging-tractable, microfluidic multi-organ-on-chip platform is constructed, that seamlessly integrates a breast tumor-like compartment: invasive MDA-MB-231 in a 3D Collagen I scaffold, and a flow-implemented vascular channel: immortalized human aortic endothelia (TeloHAEC) on laminin-rich basement membrane (lrBM). The chip showcases the complexity of intravasation, wherein tumor cells and endothelia cooperate to form anastomotic structures, which facilitate cancer cell migration into the vascular channel. Upon entry, cancer cells adhere to and flow within the vascular channel. Exposure to methylglyoxal (MG), a dicarbonyl stressor associated with diabetic circulatory milieu increases cancer cell intravasation and adhesion through the vascular channel. This can be driven by MG-induced endothelial senescence and shedding, but also by the ability of MG to degrade lrBM and pathologically cross-link Collagen I, diminishing cell-ECM adhesion. Thus, dicarbonyl stress attenuates homeostatic barriers to cancer intravasation, exacerbating metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ccd9fea88c6f1d34089c734908b37b46e5726f5" target='_blank'>
              Demonstration of Enhancement of Tumor Intravasation by Dicarbonyl Stress Using a Microfluidic Organ-on-chip.
              </a>
            </td>
          <td>
            Nilesh Kumar, Bidita Samanta, Jyothsna Km, Varun Raghunathan, Prosenjit Sen, R. Bhat
          </td>
          <td>2025-01-02</td>
          <td>Small</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29ddc5fa07210675a87aab892e726cd5b689147c" target='_blank'>
              Nanopore-based random genomic sampling for intraoperative molecular diagnosis
              </a>
            </td>
          <td>
            Francesco E Emiliani, Abdol Aziz Ould Ismail, Edward G. Hughes, Gregory J Tsongalis, George J. Zanazzi, Chun-Chieh Lin
          </td>
          <td>2025-01-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tissue-resident memory T cells (TRM) provide frontline protection against pathogens and emerging malignancies. Tumor-infiltrating lymphocytes (TIL) with TRM features are associated with improved clinical outcomes. However, the cellular interactions that program TRM differentiation and function are not well understood. Using murine genetic models and targeted spatial transcriptomics, we found that the CD8+ T cell-derived chemokine XCL1 is critical for TRM formation and conventional DC1 (cDC1) supported the positioning of intestinal CD8+ T cells during acute viral infection. In tumors, enforced Xcl1 expression by antigen-specific CD8+ T cells promoted intratumoral cDC1 accumulation and T cell persistence, leading to improved overall survival. Notably, analysis of human TIL and TRM revealed conserved expression of XCL1 and XCL2. Thus, we have shown that the XCL1-XCR1 axis plays a non-cell autonomous role in guiding intestinal CD8+ TRM spatial differentiation and tumor control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2308bb40e506b513d809177fd6a68dab1c1bbff2" target='_blank'>
              The XCL1-XCR1 axis supports intestinal tissue residency and antitumor immunity.
              </a>
            </td>
          <td>
            Amir Ferry, Kianoosh M Mempel, Alexander Monell, Miguel Reina-Campos, Nicole E. Scharping, Maximilian Heeg, Kennidy K. Takehara, Shiruyeh Schokrpur, Ning Kuo, R. Saddawi-Konefka, J. S. Gutkind, Ananda W Goldrath
          </td>
          <td>2025-01-22</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: Glioblastoma (GBM), a highly aggressive grade IV astrocytoma, poses a major therapeutic challenge due to the resistance of cancer stem cells (CSCs) existing within its cell population to the conventional therapies. Recently, we reported that RNA interference targeting CSC protection mechanism significantly improved therapeutic efficacy. However, challenges remain, including limited transfection efficiency in neural cells and the difficulty of crossing the blood–brain barrier (BBB). Methods: In this study, we investigated the potential of exosome-mediated delivery of therapeutic cargo to GBM cells by engineering the exosomes to carry green fluorescent protein (GFP) and expressing brain-homing peptide (BHP) on their surface, which has high affinity to the neural cells. Results: We found that BHP-modified exosomes doubled GFP delivery efficacy from 20% to 40%, outperforming traditional transfection methods like lipofection in vitro. In vivo, BHP-modified exosomes demonstrated an ability to cross the BBB and targeted cargo delivery to brain regions following intranasal and subcutaneous administration. Conclusions: These results underscore the potential of engineered exosomes for efficient cargo delivery to enhance therapeutic efficacy against brain tumors and suggest novel avenues for delivering biomolecules to the brain in the treatment of neurological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a92f8555ccb51822da73a1172adbb87fada06ce" target='_blank'>
              Brain-Homing Peptide Expression on the Membrane Enhances the Delivery of Exosomes to Neural Cells and Tissue
              </a>
            </td>
          <td>
            Jonhoi Smith, M. Field, Kiminobu Sugaya
          </td>
          <td>2025-01-04</td>
          <td>Neuroglia</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Most diffuse large B-cell lymphoma (DLBCL) patients treated with immunotherapies such as bispecific antibodies (BsAb) or chimeric antigen receptor (CAR) T cells fail to achieve durable treatment responses, underscoring the need for a deeper understanding of mechanisms that regulate the immune environment and response to treatment. Here, an integrative, multi-omic approach was applied to multiple large independent datasets in order to characterize DLBCL immune environments, and to define their association with tumor cell-intrinsic genomic alterations and outcomes to CD19-directed CAR T-cell and CD20 x CD3 BsAb therapies. This approach effectively segregated DLBCLs into four immune quadrants (IQ) defined by cell-of-origin and immune-related gene set expression scores. These quadrants consisted of activated B cell-like (ABC) hot, ABC cold, germinal center B cell-like (GCB) hot, GCB cold DLBCLs. Recurrent genomic alterations were enriched in each IQ, suggesting that lymphoma cell-intrinsic alterations contribute significantly to orchestrating unique DLBCL immune environments. For instance, SOCS1 loss-of-function mutations were significantly enriched among GCB hot DLBCLs, identifying a putative subset of inflamed DLBCLs that may be inherently susceptible to immunotherapy. In relapsed/refractory DLBCL patients, DLBCL-IQ assignment correlated significantly with clinical benefit with a CD20 x CD3 BsAb (n = 74), but not with CD19-directed CAR T cells (Stanford, n = 51; MSKCC, n = 69). Thus, DLBCL-IQ provides a new framework to conceptualize the DLBCL immune landscape and suggests the endogenous immune environment has a more significant impact on outcomes to BsAb compared to CAR T cell treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107d78b11f1b97cfec5bf7318d162bdc5bba82a4" target='_blank'>
              Integrative genomic analysis of DLBCL identifies immune environments associated with bispecific antibody response.
              </a>
            </td>
          <td>
            Sravya Tumuluru, J. Godfrey, Alan Cooper, Jovian Yu, Xiufen Chen, Brendan W. MacNabb, Girish Venkataraman, Yuanyuan Zha, Benedikt Pelzer, Joo Y. Song, Gerben Duns, B. Sworder, Sandeep S. Raj, Christopher Bolen, E. Penuel, E. Postovalova, N. Kotlov, A. Bagaev, N. Fowler, R. Shouval, Sonali M. Smith, A. Alizadeh, Christian Steidl, Justin Kline
          </td>
          <td>2025-01-27</td>
          <td>Blood</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd21edea6ef2a85bfcda6c2395a58850a218d8d2" target='_blank'>
              Patient-derived response estimates from zero-passage organoids of luminal breast cancer
              </a>
            </td>
          <td>
            Róża K. Przanowska, Najwa Labban, Piotr Przanowski, Russell B Hawes, Kristen A Atkins, S. Showalter, Kevin A. Janes
          </td>
          <td>2024-12-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 574


 Background:
 Small renal cell carcinoma (RCC) is generally nonaggressive but in some cases, small RCCs exhibit distant metastasis. The biology and biomarker of small aggressive RCCs remain unclear, necessitating appropriate evaluation to avoid undertreatment. Previous biomarker studies via genetic profiling of tumor tissues were hindered by RCC heterogeneity. We utilized bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) of blood samples to identify genetic biomarkers and characterize the tumor immune microenvironment in patients with aggressive small RCC.
 Methods:
 We identified four T1a (≤4 cm) clear cell RCCs (ccRCC) with synchronous distant metastasis as aggressive, and four age-, sex-, and tumor size-matched ccRCC patients without metastasis as non-aggressive, from a prospective cohort (ClinicalTrials.gov Identifier: NCT03694912). Two healthy kidney donors served as controls. Using the 10X Genomics platform, we performed scRNA-seq and bulk analysis on ten peripheral blood mononuclear cell (PBMC) samples. Samples were demultiplexed with Souporcell, and sequencing reads were normalized and analyzed using R/Seurat. Cellular components were identified with marker genes, and bulk analysis included differential gene expression and pathway enrichment.
 Results:
 The mean tumor size and age of the eight ccRCC patients were 2.5 cm and 63 years, respectively. Among the four aggressive ccRCCs, two had lung metastasis and two had bone metastasis. We analyzed 32,417 cells, identifying 41 subpopulations. Single-cell analysis revealed variations in T and natural killer (NK) cell proportions among aggressive, non-aggressive, and normal groups, with significant differences. The aggressive group showed increased MYOM2, OVCH1-AS1, and HLA-DQA2 expression. Despite an increased proportion of CD4+ cytotoxic T lymphocytes (CTLs) with disease severity, key cytotoxic and immune regulatory genes like GNLY, CD247, CCL4, NKG7, and TGFB1 were decreased, suggesting compromised antitumor function in aggressive states.
 Conclusions:
 This study offers insights into the molecular and cellular characteristics of aggressive small RCCs via PBMC analysis using single-cell and bulk RNA sequencing. The increased proportion of CD4+ CTLs with decreased expression of critical cytotoxic and immune regulatory genes in aggressive RCCs suggests a mechanism for immune evasion. These findings enhance our understanding of aggressive small RCC biology and highlight potential biomarkers and therapeutic targets for adjuvant treatment. Clinical trial information:
 NCT03694912
 .
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95baae39e5fc19835d7ecae2eef1e8eeffa27434" target='_blank'>
              Identification of potential immune evasion mechanisms in aggressive small renal cell carcinomas through single-cell and bulk RNA sequencing.
              </a>
            </td>
          <td>
            J. Park, Eun Chae Kim, Seol Song, Jongchan Kim, W. S. Jang, Byungjin Hwang, Won Sik Ham
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Epithelial lineage differentiation is pivotal to mammary gland development and it can pause metastasis of breast cancer (BC) by inducing tumor dormancy. To simulate this, we expressed epithelial genes in mesenchymal BC cells. Inducible expression of the epithelial OVOL genes in metastatic BC cells suppressed proliferation and migration. We found that C1ORF116, an OVOL’s target, is susceptible to genetic and epigenetic aberrations in BC. It is regulated by steroids and functions as a putative autophagy receptor that inhibits antioxidants like thioredoxin. Accordingly, boosting epithelialization lowered glutathione, elevated reactive oxygen species and increased both DNA oxidation and double strand breaks. Epithelialization also associated with redistribution of NRF2 and an altered interplay among p38, ATM, and the other kinases regulating the DNA damage response. Hence, hormonal regulation of OVOLs and chronic stress might permit epithelial differentiation and retard exit from dormancy, while altering redox homeostasis and permitting DNA damage accumulation, which may awaken dormant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/276799aa4ea10d7b6d31f8b02d5027bb36127494" target='_blank'>
              Inducible re-epithelialization of cancer cells increases autophagy and DNA damage: implications for breast cancer dormancy
              </a>
            </td>
          <td>
            Diana Drago Garcia, Suvendu Giri, Rishita Chattaerjee, Arturo Simoni Nieves, Maha Abedrabbo, Alessandro Genna, Mary Luz Uribe Rios, Moshit Lindzen, Arunachalam Sekar, Nitin Gupta, Noa Aharoni, Tithi Bhandari, Agalyan Mayalagu, Luisa E. Schwarzmüller, Nooraldeen Tarade, Rong Zhu, Harsha-Raj Mohan-Raju, Feride Karatekin, Francesco Roncato, Y. Eyal-Lubling, T. Keidar, Yam Nof, Nishanth Belugali Nataraj, Karin Shira Bernshtein, Bettina Wagner, Nishanth Ulhas Nair, N. Sanghvi, R. Alon, Rony Seger, Eli Pikarsky, S. Donzelli, Giovanni Blandino, Stefan Wiemann, S. Lev, R. Prywes, D. Barkan, Oscar M Rueda, Carlos Caldas, E. Ruppin, Yosef Shiloh, M. Dahlhoff, Yosef Yarden
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="The poor survival of ovarian cancer patients is linked to their high likelihood of relapse. In spite of full apparent macroscopic clearance, tumor recurrences arise from cells that are resistant to primary chemotherapy in the form of minimal residual disease (MRD). MRD exhibits distinct molecular drivers from bulk cancer and therefore necessitates alternative therapeutic strategies. However, there is a lack of 3D models that faithfully recapitulate MRD ex vivo for therapy development. This study constructs microfluidics-based 3D microtumors to generate a clinically-relevant model for ovarian cancer MRD. The microtumors recapitulate the non-genetic heterogeneity of ovarian cancer, capturing the "Oxford Classic" five molecular signatures. Gene expression in the 3D microtumors aligns closely with MRD from ovarian cancer patients and features the upregulation of fatty acid metabolism genes. Finally, the MRD 3D microtumors respond to the approved fatty acid oxidation inhibitor, perhexiline, demonstrating their utility in drug discovery. This system might be used as a drug-testing platform for the discovery of novel MRD-specific therapies in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c92f8e21e6ccaf75d856fcadd4e15f431e68af9" target='_blank'>
              3D Microtumors Representing Ovarian Cancer Minimal Residual Disease Respond to the Fatty Acid Oxidation Inhibitor Perhexiline.
              </a>
            </td>
          <td>
            Xingyun Yang, Mara Artibani, Yongcheng Jin, Aneesh Aggarwal, Yujia Zhang, Sandra Muñoz-Galvan, E. Mikhailova, Lena Rai, Nobina Mukherjee, Ravinash Krishna Kumar, A. Albukhari, Shaohua Ma, Linna Zhou, Ahmed Ashour Ahmed, Hagan Bayley
          </td>
          <td>2025-02-09</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is the most common primary liver cancer, with microvascular invasion (MVI) identified as a major predictor of early recurrence. However, the intratumor cellular heterogeneity of MVI, the identification of pertinent biomarkers, and the role of intercellular signalling interactions in MVI progression are unclear. This study aims to explore these aspects using single-cell transcriptomic analysis. Methods The present study utilized single-cell transcriptomic data from public databases to conduct an in-depth transcriptome analysis of tumour tissues and adjacent nontumor tissues from five patients with hepatocellular carcinoma, with a particular focus on samples from three patients exhibiting microvascular invasion. Bioinformatics tools were employed to analyze gene expression patterns and signalling pathways. Results The findings indicated that MVI-positive malignant cells activate multiple signalling pathways to facilitate invasion and metastasis. Specific malignant cell subtypes strongly associated with MVI were identified, exhibiting distinctive gene expression patterns related to proliferation, invasion, and metabolic reprogramming of tumour cells. Further analysis revealed that the laminin and VEGF signalling pathways are crucial for remodelling the tumour microenvironment and angiogenesis associated with MVI. The MARCKSL1 gene was predominantly expressed in MVI-positive malignant cells and may contribute to MVI progression by interacting with the PTN signalling network. Additionally, MARCKSL1 is linked to tumour resistance to multiple anticancer drugs. Discussion This study sheds light on the molecular characteristics and functional heterogeneity of MVI-associated malignant cell subpopulations. The single-cell transcriptome and bioinformatics analyses provided insights into the mechanisms driving MVI, potentially aiding the development of targeted diagnostic and therapeutic strategies. Future research should further validate the role of MARCKSL1 in MVI progression and explore its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e75f99f0ab352845523f0f1e55263a27b76fefc5" target='_blank'>
              Single-cell sequencing reveals cell heterogeneity and aberrantly activated pathways associated with microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jianwei Cui, Fanyi Zeng, Ming Tang, Shiwu Yin
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) is a highly aggressive tumor with one of the highest morbidity and mortality rates in the world. Nucleotide metabolic processes are critical for cancer development, progression, and alteration of the tumor microenvironment. However, the effect of nucleotide metabolism on LUAD remains to be thoroughly investigated. Methods Transcriptomic and clinical data of LUAD were downloaded and organized from TCGA and GEO databases. Genes related to nucleotide metabolism were downloaded from the Msigdb database. Genes associated with LUAD prognosis were identified using univariate COX analysis, and a prognostic risk model was constructed using the machine learning combination of Lasso + Stepcox. The model’s predictive validity was evaluated using KM survival and timeROC curves. Based on the prognostic model, LUAD patients were classified into different nucleotide metabolism subtypes, and the differences between patients of different subtypes were explored in terms of genomic mutations, functional enrichment, tumor immune characteristics, and immunotherapy responses. Finally, the key gene SNRPA was screened, and a series of in vitro experiments were performed on LUAD cell lines to explore the role of SNRPA in LUAD. Result LUAD patients could be accurately categorized into subtypes based on the nucleotide metabolism-related prognostic risk score (NMBRS). There were significant differences in prognosis between patients of different subtypes, and the NMBRS showed high accuracy in predicting the prognosis of LUAD patients. In addition, patients of different subtypes showed significant differences in genomic mutation and functional enrichment and exhibited different anti-tumor immune profiles. Importantly, NMBRS can be used to predict the responsiveness of LUAD patients to immunotherapy. The results of in vitro cellular experiments indicate that SNRPA plays an important role in the development and progression of lung adenocarcinoma. Conclusion This study comprehensively reveals the prognostic value and clinical application of nucleotide metabolism in LUAD. A prognostic signature constructed based on genes related to nucleotide metabolism accurately predicted the prognosis of LUAD patients, and this signature can be used as a guide for LUAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f247cf9ee6d123c8afb18b9aa087b0c11d86c73" target='_blank'>
              Integration of single-cell transcriptomics and bulk transcriptomics to explore prognostic and immunotherapeutic characteristics of nucleotide metabolism in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kai Zhang, Luyao Wang, Huili Chen, Lili Deng, Mengling Hu, Ziqiang Wang, Yiluo Xie, Chaoqun Lian, Xiaojing Wang, Jing Zhang
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple negative breast cancers often contain higher numbers of tumour-infiltrating lymphocytes compared with other breast cancer subtypes, with their number correlating with prolonged survival. Since little is known about tumour-infiltrating lymphocyte trafficking in triple negative breast cancers, we investigated the relationship between tumour-infiltrating lymphocytes and the vascular compartment to better understand the immune tumour microenvironment in this aggressive cancer type. We aimed to identify mechanisms and signaling pathways responsible for immune cell trafficking in triple negative breast cancers, specifically of basal type, that could potentially be manipulated to change such tumours from immune “cold” to “hot” thereby increasing the likelihood of successful immunotherapy in this challenging patient population. We characterised the spatial immune environment in 10 basal breast cancers showing a range of tumour-infiltrating lymphocytes using multiplex fluorescent immunohistochemistry and quantitative digital analysis of CD3+ T cells. We examined their relationship to blood vessels and their activation status as defined by VCAM-1, ICAM-1 and PD-L1. Confirmation of the relationship between tumour-infiltrating lymphocytes and endothelial activation was performed through in silico analysis on TCGA BRCA RNA-seq data (N = 808). Significantly higher CD3+ T cell densities were observed in the stromal compartment compared with the neoplastic cell compartment (P = 0.003). ICAM-1 activated blood vessels were spatially associated with higher CD3+ T cell densities only within 30 microns of blood vessels compared with more distal activated and non-activated blood vessels (P = 0.041). In silico analysis confirmed higher numbers of tumour-infiltrating lymphocytes in basal breast cancers and that higher numbers were significantly associated with endothelial cell activation molecules, co-clustering with upregulated ICAM-1 and VCAM-1 amongst others. PD-L1 was also identified in a subset of blood vessels, suggesting an additional immune regulatory mechanism in endothelial cells. Regulating the activation status of tumour-associated vascular endothelial cells may improve T cell trafficking into basal breast tumours and enhance immunotherapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd5404cd864380509879a211020591331396cbc" target='_blank'>
              Preliminary characterisation of the spatial immune and vascular environment in triple negative basal breast carcinomas using multiplex fluorescent immunohistochemistry
              </a>
            </td>
          <td>
            E. Takano, Metta K Jana, Luis E Lara Gonzalez, J. Pang, R. Salgado, S. Loi, Stephen B Fox
          </td>
          <td>2025-01-10</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="TAVO412, a multi-specific antibody targeting epidermal growth factor receptor (EGFR), mesenchymal epithelial transition factor (c-Met), and vascular endothelial growth factor A (VEGF-A), is undergoing clinical development for the treatment of solid tumors. TAVO412 has multiple mechanisms of action for tumor growth inhibition that include shutting down the EGFR, c-Met, and VEGF signaling pathways, having enhanced Fc effector functions, addressing drug resistance that can be mediated by the crosstalk amongst these three targets, as well as inhibiting angiogenesis. TAVO412 demonstrated strong in vivo tumor growth inhibition in 23 cell-line derived xenograft (CDX) models representing diverse cancer types, as well as in 9 patient-derived xenograft (PDX) lung tumor models.Using preclinical CDX data, we established transcriptomic biomarkers based on gene expression profiles that were correlated with anti-tumor response or distinguished between responders and non-responders. Together with specific driver mutation that associated with efficacy and the targets of TAVO412, a set of 21-gene biomarker was identified to predict the efficacy. A biomarker predictor was formulated based on the Linear Prediction Score (LPS) to estimate the probability of patients or tumor model response to TAVO412 treatment. This efficacy predictor for TAVO412 demonstrated 78% accuracy in the CDX training models. The biomarker model was further validated in the PDX data set and resulted in comparable accuracy. In implementing precision medicine by leveraging preclinical model data, a predictive transcriptomic biomarker empowered by next-generation sequencing was identified that could optimize the selection of patients that may benefit most from TAVO412 treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baab869040ba74fc2770d76e5b2793db02fc0b71" target='_blank'>
              Developing transcriptomic biomarkers for TAVO412 utilizing next generation sequencing analyses of preclinical tumor models
              </a>
            </td>
          <td>
            Ying Jin, Peng Chen, Huajun Zhou, Guangmao Mu, Simin Wu, Zhengxia Zha, Bin Ma, Chao Han, Mark L Chiu
          </td>
          <td>2025-02-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e47a9918829003a93f398ed9e563361008b2b513" target='_blank'>
              Resolving tissue complexity by multimodal spatial omics modeling with MISO.
              </a>
            </td>
          <td>
            Kyle Coleman, Amelia Schroeder, Melanie Loth, Daiwei Zhang, Jeong Hwan Park, Ji-Youn Sung, Niklas Blank, Alexis J Cowan, Xuyu Qian, Jianfeng Chen, Jiahui Jiang, Hanying Yan, Laith Z. Samarah, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Joshua D. Rabinowitz, Yanxiang Deng, Edward B. Lee, Alexander Lazar, Jianjun Gao, E. E. Furth, Tae Hyun Hwang, Linghua Wang, Cristoph Thaiss, Jian Hu, Mingyao Li
          </td>
          <td>2025-01-15</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune cell infiltration in tumors has been reported to influence tumor progression and clinical outcomes. Considerable efforts have been made to understand interactions between tumors and the immune system. However, current models are either not comprehensive or limited to short-term studies. Recognizing thedynamic and long-term nature of tumor-immune interactions, an immune-infiltrated cancer spheroid model is developed by continuously perfusing and recirculating immune cells with gravity-driven flow through a tubular blood vessel adjacent to a cancer spheroid. Fibroblasts and pericytes are embedded in the gel matrix to support endothelial cells and enhance the vascular barrier. With continuous monocyte recirculation, monocyte adhesion, transendothelium migration, differentiation, and macrophage recruitment into breast carcinoma and hepatoma spheroids is successfully demonstrated over a week. The macrophage recruitment process is temporally dependent and tissue-specific, leading to the formation of cancer-macrophage heterospheroids. Elevated secretion of granulocyte-macrophage colony-stimulating factor (GM-CSF), which regulates monocyte recruitment and macrophage activation, is observed in the breast carcinoma model. Increased levels of Interleukin 6 (IL-6) and Interleukin 8 (IL-8) are detected, indicating a pro-inflammatory environment associated with tumor progression and metastasis. This platform provides a valuable framework for investigating immune cell infiltration and differentiation within the tumor microenvironment, supporting the advancement of cancer immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b39b565addebf1144f1114f7d8782cc48106cd9" target='_blank'>
              Immune-Infiltrated Cancer Spheroid Model with Vascular Recirculation Reveals Temporally Dependent and Tissue-Specific Macrophage Recruitment.
              </a>
            </td>
          <td>
            Feng Zhang, K. Jozani, Anushree Chakravarty, Dawn Lin, Andrew Hollinger, Shravanthi Rajasekar, Boyang Zhang
          </td>
          <td>2025-02-17</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract
 Round cell soft tissue tumors (RCSTTs) are a diverse group of aggressive malignancies that share overlapping clinical, morphological, and immunohistochemical features, posing significant diagnostic challenges. Recent advances in molecular pathology have revolutionized the understanding and classification of these tumors by identifying tumor-specific genetic alterations. This review highlights the role of molecular insights in complementing histopathology and immunohistochemistry to achieve precise diagnosis, risk stratification, and therapeutic decision-making. Key subtypes, including Ewing sarcoma, CIC- and BCOR-rearranged sarcomas, alveolar rhabdomyosarcoma, desmoplastic small round cell tumor, and extraskeletal myxoid chondrosarcoma, are discussed with emphasis on their molecular signatures and clinical significance. The advent of techniques such as fluorescence in situ hybridization (FISH), reverse transcriptase-polymerase chain reaction (RT-PCR), and next-generation sequencing (NGS) has enabled detection of genetic fusions and aberrations critical for tumor identification and targeted therapies. Integration of molecular pathology with traditional diagnostic approaches is essential for improving diagnostic accuracy, prognostication, and therapeutic strategies, paving the way for personalized medicine in the management of RCSTTs. Full form of ABBREVIATIONS are provided at the end.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4697a589f2a69b32b9998c7bb05dd3c97ac1fa8" target='_blank'>
              Beyond Immunuhistochemistry: Molecular Insights Into Round Cell Soft Tissue Tumors
              </a>
            </td>
          <td>
            Prashant Kumar Madoori, Sujatha Tejavat
          </td>
          <td>2024-12-30</td>
          <td>Perspectives in Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The cell fate transition is a fundamental characteristic of living organisms. By introducing external perturbations, it is possible to artificially intervene in cell fate and trigger cell reprogramming. Revealing the general principle underlying the induced phenotypic reshaping of cell populations remains a central focus in the field of cell biology. In this study, we investigate the energetic and dynamic features of induced cell phenotypic transition from differentiated somatic state to pluripotent state by constructing a Boolean genetic network model. The simulation and experimental results highlight the critical role of genetic frustration in initiating cell fate transitions, although the two ending phenotypic states are typically featured by minimal frustration. In addition, the altered gene expression profiles exhibit a scale-free distribution, suggesting that there exist a small number of critical genes responsible for the cell fate transition. This study provides important insights into the dynamic principles governing effective cell reprogramming caused by artificial or exogenous interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e17321e8898e722005a2009e4fe82616f4dc5" target='_blank'>
              Role of genetic frustrations in cell reprogramming
              </a>
            </td>
          <td>
            Yuxiang Yao, Jieying Zhu, Wenfei Li, Duanqing Pei
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, K. Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df4692fb69e6fa5ba91b09d082c03060cfa77e4" target='_blank'>
              Impact of G1 phase kinetics on the acquisition of stemness in cancer cells: the critical role of cyclin D.
              </a>
            </td>
          <td>
            Yasin Ahmadi, Tahran Faiq, Sakhavat Abolhasani
          </td>
          <td>2025-02-14</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66a1456d830962314edc810676fedd379ab37c8b" target='_blank'>
              TIGIT expression in extrahepatic cholangiocarcinoma and its impact on CD8 + T cell exhaustion: implications for immunotherapy
              </a>
            </td>
          <td>
            Tengqian Tang, Wenhao Wang, Lang Gan, Jie Bai, Dehong Tan, Yan Jiang, Ping Zheng, Weijun Zhang, Yu He, Qianfei Zuo, Leida Zhang
          </td>
          <td>2025-02-12</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 13


 Background:
 Advanced penile cancer is a rare but aggressive malignancy. Over 70% of patients with bulky metastases relapse or have primary refractory disease with current treatments. Immunotherapy represents a promising modality, but response rates remain low. While prior studies explored single immune factors within the tumor microenvironment (TME), understanding complex interactions between T cells and macrophages and in relation to the tumor cells is crucial to best characterize the TME.
 Methods:
 We performed multiplex immunofluorescence (mIF) on 59 PSCC tissues obtained from MD Anderson to analyze expression of 12 key immune cell markers (CD3, CD8, CD68, CD86, CD206, CD163, ARG, CSF1R, MHC-II, PD-1, PD-L1). Spatial analysis and co-clustering of cell phenotypes was performed using Ripley’s K. Overall (OS), recurrence free (RFS), and cancer specific survival (CSS) curves were determined using Kaplan Meier method and tested using logrank and cox regression models.
 Results:
 The median age was 60 (IQR 24-86) and majority were HPV negative (64%). High densities and clustering of cytotoxic CD8+ T cells and CD68+ tumor associated macrophages (TAMs) were each associated with improved OS (126 vs 61 months, p= 0.03; 140 vs 61 months, p= 0.04, respectively]. Conversely, high densities of antigen-experienced CD8+PD-1+ cytotoxic T cells and antigen-experienced CD68+PD-1+ macrophages were associated with worse OS [HR 1.44 (1-2.05), p= 0.04 and HR 1.48 (1.04-2.12), p=0.03 respectively], PFS [HR 1.45 (1.03-2.05), p= 0.03 and HR 1.53 (1.06-2.19), p= 0.02 respectively ] and CSS [HR 1.52 (1.03-2.24), p= 0.03 and HR 1.63 (1.12-2.36), p= 0.01 respectively]. Clustering of CD86+ M1 macrophages was associated with improved OS (104.5 vs 29.1 months, p=0.02), while clustering of CD163+ and CD206+ M2 macrophages correlated with decreased OS (67.8 vs 199.7 months, p=0.03; 100.6 vs NA months, p=0.01 respectively). Bivariate analysis revealed improved OS associated with co-clustering of CD8+ T cells with TAMs (104 vs 42 months, p=0.03). Similarly, co-clustering of CD8+T cells with CK+ tumor cells was associated with improved OS (140 vs 61 months; p=0.02). While co-clustering of CD86+ M1 macrophages to cytokeratin (CK+) tumor cells correlated with improved OS (126 vs 42 months, p=0.03), co-clustering of CD 206+ M2 macrophages to CK+ tumor cells associated with poor RFS (48 vs NA months, p=0.03). Co-clustering of antigen experienced T cells with TAMs was also associated with decreased RFS (29 vs NA months, p =0.03).
 Conclusions:
 Using spatial analysis, we showed that interaction between T cells and macrophages impact clinical outcomes in PSCC. High densities and proximity of CD8+ T cells and M1 macrophages, and low levels of antigen experienced T cells and macrophages were associated with improved survival in PSCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77deac7afc0fa58f9777da7363906a609d78503c" target='_blank'>
              Spatial analysis of the immune microenvironment and identification of phenotypes of prognostic significance in penile squamous cell carcinoma (PSCC).
              </a>
            </td>
          <td>
            Keerthi Gullapalli, Justin W Miller, A. Fazili, Jeffrey S Johnson, Christopher Guske, Junmin Whiting, Youngchul Kim, Hiroko Miyagi, N. Zacharias, Andrew C Johns, C. M. Segura, Jonathan V Nguyen, Priya Rao, Philippe E. Spiess, C. Pettaway, J. Chahoud
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The lysosome plays a vitally crucial role in tumor development and is a major participant in the cell death process, involving aberrant functional and structural changes. However, there are few studies on lysosome-associated genes (LAGs) in lung adenocarcinoma (LUAD). Methods Bulk RNA-seq of LUAD was downloaded from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO). The lysosome risk signature was constructed after univariate and least absolute shrinkage and selection operator (Lasso) cox regression analysis of the TCGA training set, and its capability was validated by additional validation sets from GEO. Single cell sequencing (scRNA) was obtained from GEO to analyze the differences of lysosome risk signature at the single-cell level and the differences in the function and pathway. In vitro experiments have validated the function of CTSH in LUAD. Results The risk signature contained seven key LAGs, and patients were categorized into high- and low-risk groups based on a specific calculation formula. The LAG risk signature, which accurately predicted the prognostic status of LUAD patients, was still regarded as an independent prognostic indicator in multifactorial cox regression analysis. Subsequently, the combination of the signature and key clinical information was used to construct a column-line diagram for clinical assessment, which had a high discriminatory power. Immune infiltration analysis from bulk RNA-seq and scRNA-seq indicated that the low-risk group was immune-activated and had a better benefit in the prediction of immunotherapy. Finally, we validated its role in inhibiting tumor proliferation and metastasis in LUAD cells by knockdown of CTSH. Conclusion We defined a new biomarker that provided unique insights for individualized survival prediction and immunotherapy recommendations for LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ba6fe47f7285dc153bdc2d9a1252702def765c" target='_blank'>
              The lysosome-related characteristics affects the prognosis and tumor microenvironment of lung adenocarcinoma
              </a>
            </td>
          <td>
            Wuguang Chang, Wuyou Gao, Yawei Wu, Bin Luo, Lekai Zhong, Leqi Zhong, Wenqian Lin, Zhesheng Wen, Youfang Chen
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (CAC) is a severe complication of inflammatory bowel disease, particularly ulcerative colitis. Despite its clinical significance, the spatiotemporal dynamics of neoplastic progression in CAC remain poorly understood. This study aimed to elucidate the evolutionary trajectory of CAC using spatial transcriptomics.



 We employed spatial transcriptomics to analyze four tissue samples from three patients with ulcerative colitis at various stages of neoplastic progression. Whole-slide spatial transcriptomics RNA sequencing was performed, enabling simultaneous analysis of gene expression and tissue architecture. Integrated spatial transcriptomic data revealed recurring epithelial programs along the progression of copy number alteration.



 Our analysis using copy number variation (CNV) score and Hematoxylin and Eosin (H&E) staining image revealed progressive changes in gene expression patterns as tissue transitioned from non-tumor to pre-tumor and finally to tumor states. This progression was quantified using beta coefficient values, allowing us to track gradual alterations in gene expression across the disease continuum. We identified a core set of 48 genes that exhibited consistent changes across all four analyzed samples. Among these, 40 genes showed a progressive decrease in expression, while eight genes, including CLDN1, E2F1, and RECQL4, demonstrated a gradual increase in expression as the tissue advanced towards a tumorous state. These upregulated genes pointed to potential oncogenic drivers or compensatory mechanisms activated during tumor development. Pathway analysis of the 48 genes revealed enrichment in key cellular processes, including p53 signaling, transcriptional regulation by E2F factors, and role of PKR in interferon induction and antiviral response. Upstream Regulator analysis revealed YAP1 and TGFB1 as the most significant regulators, highlighting the importance of transcriptional control and growth factor signaling in CAC development.



 This study provides unprecedented insights into the spatiotemporal dynamics of CAC progression. By leveraging spatial transcriptomics, we have mapped the evolutionary trajectory of CAC at high resolution, revealing key molecular events and spatial features associated with disease progression. These findings not only enhance our understanding of CAC pathogenesis but also highlight potential targets for early intervention and personalized treatment strategies.


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5406d2b3a2271be7d5c35c08e3c0a5e93dd4e792" target='_blank'>
              DOP128 Spatial transcriptomic analysis reveals evolutionary dynamics in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y. Kim, Y. Lee, M. Lee, D. K. Yoon, J. Baek, S. Hwang, J. L. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K. Song, Y. Yoon, B. Ye, H. S. Lee
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic cancer is a highly lethal malignancy with few effective treatment options. Connexin 31 (Cx31) is a membrane protein capable of forming hexameric channels to facilitate the exchange of metabolites and signaling molecules. Yet, the contribution of Cx31 to the onset and progression of pancreatic cancer remains to be understood. We analyzed Cx31 expression in pancreatic cancer tissues and cell lines using public databases and experimental models. The correlation between Cx31 expression and clinical outcomes was evaluated. The effects of Cx31 on pancreatic cancer cell proliferation, stemness, migration, chemoresistance, and immune infiltration were investigated. Transcriptome analysis and bioinformatics tools were employed to explore the underlying mechanisms. Cx31 was found to be upregulated in pancreatic cancer tissues compared to normal tissues, and its high expression correlated with shorter overall survival and higher mortality risk. Cx31 promoted acinar-to-ductal metaplasia (ADM), stemness, proliferation, migration, metastasis, and chemoresistance in pancreatic cancer cells. Bioinformatics analysis suggested a positive correlation between Cx31 and stemness-related genes. Cx31 knockdown altered the expression of genes involved in stemness and chemoresistance pathways, such as Wnt and Notch. Additionally, Cx31 was identified as a direct target of the transcription factor FOXM1, which upregulated its expression. Cx31 plays a multifaceted role in pancreatic cancer, influencing processes from initiation to metastasis and chemoresistance. It may serve as a potential therapeutic target to combat the aggressive nature of pancreatic cancer. The FOXM1-Cx31 axis could be a promising target for overcoming treatment resistance in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92baaa0b6ff72194944a8710fcf3d6648990069b" target='_blank'>
              FOXM1-Cx31 Axis Drives Pancreatic Cancer Stem Cell-Like Properties and Chemoresistance.
              </a>
            </td>
          <td>
            Yang Chen, Qihui Sun, Qi Zou, Xiaoqi Zhu, Tingting Wen, Xiaojia Li, Shu Li, Jie He, Fang Wei, Keping Xie
          </td>
          <td>2025-01-06</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
 Understanding the status and function of tumor-infiltrating immune cells is essential for improving immunotherapeutic effects and predicting the clinical response in human patients with carcinoma. However, little is known about tumor-infiltrating immune cells, and the corresponding research results in hepatocellular carcinoma (HCC) are limited.
 AIM
 To investigate potential biomarker genes that are important for the development of HCC and to understand how immune cell subsets react throughout this process.
 METHODS
 Using single-cell RNA sequencing and T-cell receptor sequencing, the heterogeneity and potential functions of immune cell subpopulations from HCC tissue and normal tissue adjacent to carcinoma, as well as their possible interactions, were analyzed.
 RESULTS
 Eight T-cell clusters from patients were analyzed and identified using bioinformatics, including six typical major T-cell clusters and two newly identified T-cell clusters, among which Fc epsilon receptor 1G+ T cells were characterized by the upregulation of Fc epsilon receptor 1G, tyrosine kinase binding protein, and T cell receptor delta constant, whereas metallothionein 1E+ T cells proliferated significantly in tumors. Differentially expressed genes, such as regulator of cell cycle, cysteine and serine rich nuclear protein 1, SMAD7 and metallothionein 1E, were identified as significantly upregulated in tumors and have potential as biomarkers. In association with T-cell receptor analysis, we inferred the clonal expansion characteristics of each T-cell cluster in HCC patients.
 CONCLUSION
 We identified lymphocyte subpopulations and potential biomarker genes critical for HCC development and revealed the clonal amplification of infiltrating T cells. These data provide valuable resources for understanding the response of immune cell subsets in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd51f4bc1931e22c96d2d6a320e4c74a7f118e5" target='_blank'>
              Uncovering immune cell heterogeneity in hepatocellular carcinoma by combining single-cell RNA sequencing with T-cell receptor sequencing
              </a>
            </td>
          <td>
            , , , Jin-Long Tong, Xiao-Yue Li, Hui Dong, Cai-Yun Zhang, Wen-Xian Qian, Xiu-Chang Ma, Chang-Hua Yi, 
          </td>
          <td>2025-02-27</td>
          <td>World Journal of Hepatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Single-cell RNA sequencing (scRNA-seq) has emerged as a pivotal technology for investigating novel therapeutic targets in cancer. Despite its significance, there remains a scarcity of studies utilizing this technology to address treatment strategies specifically tailored for early-stage lung adenocarcinoma (LUAD). Consequently, this study aimed to investigate the tumor microenvironment (TME) characteristics and develop a prognostic model for early-stage LUAD. Methods: The markers identifying cell types were obtained from the CellMarker database and published research. The SCEVAN package was employed for identifying malignant lung epithelial cells. Single-cell downstream analyses were conducted using the SCP package, encompassing gene set enrichment analysis, enrichment analysis, pseudotime trajectory analysis, and differential expression analysis. Calibration curves, receiver operating characteristic curves, and decision curve analysis were employed to assess the performance of the prognostic model for LUAD. Reverse transcription-quantitative polymerase chain reaction (RT-qPCR), western blot, cell transfection, cell proliferation, and cell invasion assays were performed to validate the expression and biological function. Results: Seven cell types were distinguished in the scRNA-seq dataset through the utilization of cell markers documented in published literature. Four subpopulations of early-stage LUAD tumor cells exhibited a high degree of heterogeneity. The prognostic model constructed by PERP and KRT8 showed a great prediction for distinguishing the early-stage LUAD and normal tissues. The validation of PERP and KRT8 expression levels was carried out through both RT-qPCR and western blot analyses. Eventually, in vitro experiments, including CCK8, colony formation, EdU, and transwell assays, confirmed that KRT8 and PERP could promote LUAD cell proliferation and migration. Conclusions: Our study provided a comprehensive characterization of the TME in LUAD through integrative single-cell and bulk transcriptomic analyses. We identified dynamic transitions from normal epithelial cells to tumor cells, revealing the heterogeneity and evolution of malignant LUAD cells. The novel prognostic model based on KRT8 and PERP demonstrated robust predictive performance, offering a promising tool for early-stage LUAD risk stratification. Functional experiments further confirmed that KRT8 and PERP promote tumor proliferation and migration, providing new insights into their roles as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d6e3c6aa9426d7c80e1f402166a26104620a98" target='_blank'>
              Integration of Single-Cell and Bulk Transcriptomes to Identify a Poor Prognostic Tumor Subgroup to Predict the Prognosis of Patients with Early-stage Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Zijian Shi, Linchuang Jia, Baichuan Wang, Shuo Wang, Long He, Yingxi Li, Guixin Wang, Wenbin Song, Xianneng He, Zhaoyi Liu, Cangchang Shi, Yao Tian, Keyun Zhu
          </td>
          <td>2025-01-21</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91aaef8a603816e9532df40a515bf3b479efc54f" target='_blank'>
              Cancer Stem Cell Regulation as a Target of Therapeutic Intervention: Insights into Breast, Cervical and Lung Cancer.
              </a>
            </td>
          <td>
            Sakshi Kevat, Archie Mistry, Naman Oza, Mohit Majmudar, Netra Patel, Rushabh Shah, A. V. Ramachandran, Ritu Chauhan, Shafiul Haque, N. C. Parashar, H. Tuli, Gaurav Parashar
          </td>
          <td>2025-01-23</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>40</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [2, 7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>