<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-24 09:34:29 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="


 Glioblastoma are hallmarked by clonal evolution and transcriptional heterogeneity linked to genetic and environmental diversity resulting in therapy resistance and disease progression. In this study, we integrate heritable molecular barcoding and spatial transcriptomics to enable high-resolution cell lineage tracing in the spatial context of the glioblastoma TME.



 Patient-derived glioblastoma cells from two individual cases were isolated from anatomically and pathologically distinct regions: the primary tumor core, the tumor periphery, and the recurrent lesion. Each subline was transduced with predefined FlexTrack lentiviral plasmids encoding GFP, resulting in uniquely barcoded (9N) cell lines. Immediately before injection, the barcoded glioblastoma populations were pooled and introduced into ex vivo human cortical slices. Live-imaging was used to trace GFP-positive, barcoded cell populations. After incubation, the fresh-frozen slices were processed for spatial transcriptomics and the line-specific barcodes were mapped according to the 3′ poly(A) tails of captured transcripts. Graph neural networks based on single-cell deconvolution were employed to spatially analyze the cellular ecosystems.



 We successfully deconvoluted and reconstructed the trajectories of injected glioblastoma cells linking their transcriptome phenotypes along with clonal identity. Multi-omic integration of spatial copy-number alterations and barcoding based pseudo-trajectories enabled reconstruction of the temporal transcriptional dynamics linked to the heterogeneity of tumor ecosystems.



 This study serves as a proof-of-concept for integrating molecular barcoding and spatial transcriptomics in cortical slice culture. Our approach to molecular barcoding in complex model systems opens new avenues to investigate the contextual dynamics of clonal evolution in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f785ed6bb81bfa9c1d1d1cee449d66b313a3ee5" target='_blank'>
              P02.23.B MOLECULAR BARCODING OF GLIOBLASTOMA IN HUMAN ORGANOTYPIC CORTICAL SLICE CULTURE
              </a>
            </td>
          <td>
            G. Kastner, M. Mühlbauer, J. Jung, S. Langer, L. Müller, A. Baginska, P. Berger, C. Dobersalske, S. Ertmer, F. M. Gerach, N. Neidert, K. Herrmann, L. Rauschenbach, L. van Hijfte, U. Sure, K. D. Kokkaliaris, B. M. Grüner, D. Heiland, B. Scheffler
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ce4a4012ffb45da6b02201b0801273b5519b7e" target='_blank'>
              Integrative single-cell and spatial mapping of oxidative stress response uncovers GCLC+ mesenchymal tumor cell state linked with favorable outcomes in triple negative breast cancer
              </a>
            </td>
          <td>
            Nomeda Girnius, Tuulia Vallius, Wenqing Chen, I.-M. Launonen, S. Palomino-Echeverría, Jia-Ren Lin, Caitlin E. Mills, Silja Kauppila, Pauliina Kronqvist, Antti Ellonen, Merja Perala, Eloise Withnell, Yu-An Chen, M. Secrier, S. Santagata, P. Sorger, Anniina Farkkila
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, Wei-Yi Cheng, James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Diseases such as cancer involve alterations in cell proportions, states and interactions, as well as complex changes in tissue morphology and architecture. Histopathological diagnosis of disease and most multiplexed spatial profiling relies on inspecting thin (4–5 µm) specimens. Here we describe a high-plex cyclic immunofluorescence method for three-dimensional tissue imaging and use it to show that few, if any, cells are intact in conventional thin tissue sections, reducing the accuracy of cell phenotyping and interaction analysis. However, three-dimensional cyclic immunofluorescence of sections eightfold to tenfold thicker enables accurate morphological assessment of diverse protein markers in intact tumor, immune and stromal cells. Moreover, the high resolution of this confocal approach generates images of cells in a preserved tissue environment at a level of detail previously limited to cell culture. Precise imaging of cell membranes also makes it possible to detect and map cell–cell contacts and juxtracrine signaling complexes in immune cell niches. Confocal microscopy enables high-resolution, high-plex 3D cyclic immunofluorescence of 30- to 50-µm-thick tissue sections. The approach allows for rich phenotypic assessments of intact cells and intercellular interactions with subcellular resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed4146e9d8a32db71a92a19484656680fad514b9" target='_blank'>
              Highly multiplexed 3D profiling of cell states and immune niches in human tumors
              </a>
            </td>
          <td>
            Clarence Yapp, A. J. Nirmal, Felix Y. Zhou, A. Y. Wong, Juliann B. Tefft, Yi Daniel Lu, Zhiguo Shang, Z. Maliga, P. Llopis, George F. Murphy, C. G. Lian, G. Danuser, S. Santagata, P. Sorger
          </td>
          <td>2025-09-29</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, , Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. J. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, M. Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5971868e6de8c14556026d2b663acac3f252b79d" target='_blank'>
              Genotype-phenotype correlations in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M.M. Linssen, Jill Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Quantitative investigation of how the spatial organization of cells within the tumor microenvironment associates with disease progression, patient outcomes, and that cell’s phenotypic state remains a key challenge in cancer biology. High-dimensional multiplexed imaging offers an opportunity to explore these relationships at single-cell resolution. Methods We developed a computational pipeline to quantify and analyze the neighborhood profiles of individual cells in multiplexed immunofluorescence images. The pipeline characterizes spatial co-localization patterns within the tumor microenvironment and applies interpretable supervised machine learning models, specifically orthogonal partial least squares analysis (OPLS), to identify spatial relationships predictive of cell states and clinical phenotypes. Results We applied this framework to a previously published non-small cell lung cancer (NSCLC) cohort across four applications. At the cellular level, we identified neighborhood features associated with lymphocyte activation states. At the tumor-immune interface, we demonstrated that the immune cell composition surrounding major histocompatibility complex class I-expressing (MHC I+) tumor cells could distinguish adenocarcinoma from squamous cell carcinoma. At the patient level, spatial features predicted tumor grade. Discussion By integrating cell-segmented imaging data with interpretable modeling, our pipeline reveals key spatial determinants of tumor biology. These findings generate testable mechanistic hypotheses about intercellular interactions and support the development of spatially informed prognostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a735226da4130f5ff72da463b9cc63eeddebebdb" target='_blank'>
              A flexible systems analysis pipeline for elucidating spatial relationships in the tumor microenvironment linked with cellular phenotypes and patient-level features
              </a>
            </td>
          <td>
            Gabriel F. Hanson, Kate A. Goundry, Remziye E. Wessel, Michael G Brown, Timothy N. J. Bullock, Sepideh Dolatshahi
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology. Peritumoural pancreatitis and lobular injury are commonly seen in pancreatic cancer. Here, the authors reveal interactions between invading tumour cells and inflammation-related lobular cells, linked to switching from a basal to classical tumor cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b8f75faab2c4feb9da5df93770ed23963d387e7" target='_blank'>
              Nuclear Irregularity as a Universal Diagnostic Tool in Solid Tumors
              </a>
            </td>
          <td>
            Francesco Hamilton, Kane Foster, Irene Ghobrial
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell fate determination is a fundamental process in multicellular development. In multicellular organisms, cells display plasticity in their fate, allowing them to revert to prior states or adopt alternative differentiation pathways, thereby altering their identity and functional specialization in response to specific stimuli. Investigating cell fate determination and its plasticity enhances our understanding of organ development, tissue homeostasis, and disease pathogenesis and progression, providing novel insights into regenerative medicine strategies. Lineage tracing technologies have fundamentally revolutionized this understanding of cell fate dynamics by enabling the identification and tracking of cells and their progeny in vivo. These technologies have progressed significantly, from the direct observation and manual annotation of cell lineage trees to complex recombinase‐mediated genetic labeling techniques. With the advent of sequencing technologies, the resolution and scale of lineage tracing have also developed toward the single‐cell level in individual organisms. Furthermore, lineage tracing is increasingly expanding to investigate how the tissue microenvironment influences cell fate decisions. Here, the evolution of lineage tracing technologies is introduced and their applications in cell fate determinations across development, regeneration, and diseases contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee37a1eed7bec75acde94433c268cf5bca57e275" target='_blank'>
              Cell Fate Determination and Lineage Tracing: Technological Evolution and Multidimensional Applications
              </a>
            </td>
          <td>
            Hui Chen, Zhicong Liu, Qiang Shu, Bin Zhou
          </td>
          <td>2025-09-29</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121b9f0cf30a6fb662f64416fefc808a183dcc03" target='_blank'>
              Colorectal cancer relies on an immunosuppressive cellular topography and genomic adaptations for establishing brain metastases
              </a>
            </td>
          <td>
            A. Sathe, Mengrui Zhang, Xiangqi Bai, Ji In Kang, Rithika Meka, Huiyun Sun, Susan M. Grimes, Aparajita Khan, Mingen Liu, A. Luksik, M. Lim, Claudia K. Petrisch, Christopher M. Jackson, Hannes Vogel, Jeanne Shen, M. Gephart, Summer S. Han, 
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Adult gliomas are a diverse class of tumors with substantial molecular and pathological variation. While prior single-cell omics studies have characterized intratumoral heterogeneity, they often examined tumors in isolation, thus overlooking the recurrence of tumor states across subtypes. Given their shared environment, understanding the conservation of tumor states across gliomas is essential.



 To address this, we analyzed 310 tumor cores from 284 patients spanning oligodendroglioma, low- and high-grade astrocytoma, glioblastoma, and ependymoma, using single-cell spatial transcriptomics with a 344-gene panel (10x Xenium). This approach yielded a dataset of 2.8 million cells, of which 84% were tumor and 16% represented 18 non-malignant cell types from the tumor microenvironment (TME). To further investigate local cellular interactions, we applied an extended spatial transcriptomics panel (5,096 genes) alongside multiplexed immunofluorescence (57 proteins) to a validation cohort of low-grade astrocytomas.



 Our analysis revealed tumor states recurring across gliomas, albeit in varying proportions, organized into neighborhoods that reflect local cell-cell interactions. For instance, proliferating tumor cells tended to co-localize with oligodendro- and neuro-progenitor-like tumor states, suggesting that these programs may give rise to proliferating cells. Spatial neighborhoods were primarily subtype-specific, indicating that tissue organization and not cell composition alone, underlies glioma heterogeneity. We next assessed the relationship between tumor and immune cells, revealing entity-specific immune microenvironments: astrocytoma and ependymoma exhibited distinct immune responses (T cell- versus myeloid-driven inflammation), whereas oligodendroglioma displayed a low inflammatory landscape. Notably, microglia were consistently observed in proximity to tumor cells across all glioma subtypes, highlighting their conserved role in modulating anti-tumor immunity. Methylation profiling stratified these tumors into 13 molecular subtypes, each associated with distinct neighborhoods. Glioblastoma subtypes RTK1, RTK2, and MES, in particular, displayed unique tissue architectures consistent with their subtype-specific methylation profiles.



 Collectively, these findings link microanatomical features to glioma heterogeneity by connecting tumor states to their microenvironments. Regional differences, such as those between the tumor core and infiltrating edge, shape cellular composition with implications to treatment response. By capturing such spatially dependent patterns, our work provides a framework for advancing the understanding of glioma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fbb58728bc44ec20e86f909612c57947c86652" target='_blank'>
              KS03.6.A THE RECURRENT TISSUE STATES ACROSS ADULT GLIOMAS AS REVEALED BY SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            A. Mathioudaki, Z. Seferbekova, M. Ritter, D. Calafato, G. Rukhovich, F. Hinz, F. Ippen, E. Popova, N. Koeberer, P. Sant, S. Schinkewitsch, J. Malm, S. Dietrich, C. Herold-Mende, N. Etminan, A. Wick, S. Krieg, M. Platten, A. von Deimling, F. Sahm, A. Suwala, M. Gerstung
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1126ef3bc754c9a923945851bb89db5b5275e0ec" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, Jenna Collier, K. H. Hoi, Sören Müller, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Shannon J. Turley, David Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary malignant tumor of the central nervous system. Advances in therapy are hindered by the complex intratumoral heterogeneity of GBM, where distinct malignant and non-malignant cellular states and interactions exist in spatially defined niches of the tumor microenvironment (TME), shaping both tumor behavior and treatment response. In this work, we define GBM biological reprogramming, TME recomposition, and cell-cell interactions in relation to spatially well defined Ivy Glioblastoma Atlas Project regions. Further, we apply a novel spatially informed approach to integrate biological reprogramming, as identified through gene co-expression network modules, with specific changes in cell-cell communications. Our results validate prior findings and offer novel insights into the spatial patterning of transcription factor regulation, cellular interactions, and biological pathway activity, in addition to informing rational combination therapies targeting spatial niche specific vulnerabilities. In addition, our work contributes a novel Visium spatial transcriptomics dataset of 14 samples for use by the GBM research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving as a pivotal technology in studying tumor biology and associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. We use serial 5 μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma samples in tissue microarrays to compare the performance of the ST platforms (CosMx, MERFISH, and Xenium (uni/multi-modal)) in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx, and hematoxylin and eosin staining data. In addition to an objective assessment of automatic cell segmentation and phenotyping, we perform a manual phenotyping evaluation to assess pathologically meaningful comparisons between ST platforms. Here, we show the intricate differences between the ST platforms, reveal the importance of parameters such as probe design in determining the data quality, and suggest reliable workflows for accurate spatial profiling and molecular discovery. Spatial cell distribution within a tissue microenvironment is a rapidly advancing field. Here, authors assess three commercially available single-cell resolution spatial transcriptomics approaches (CosMx, MERFISH, and Xenium) to inform which technology outperforms for immune profiling of solid tumors using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7415db7d075b9d1e62e79a2741843a3cf68a8b2" target='_blank'>
              Comparison of imaging based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia Hernandez, Wei Lu, Khaja B Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, J. Dou, Qingnan Liang, , Ahmed N. Alrawi, Claudio A. Arrechedera, Kimberly S. Ayers, Caludia A. Bedoya, Elizabeth Burton, Connie A. Chon, Randy Chu, S. Crosby, Jonathan Do, Cibelle FP Lima, Fu Szu-Chin, Andy Futreal, Ana L. Garcia, Celia Garica-Prieto, S. Gite, C. Gumbs, Kristin J. Hargraves, Meng He, Chacha Horombe, H. Ibarguen, Stacy Jackson, Jeena Jacob, Isha Khanduri, Walter K. Kinyua, M. Knafl, Wenhua Lang, L. Little, S. Mallampati, Mary GT Mendoza, F. Meric-Bernstam, M. M. Mohammad, Mario LM Piubelli, Sabitha Prabhakaran, K. R. Shaw, Xiaofei Song, Sandesh Subramanya, Baohua Sun, Shumaila Virani, Wan-Jan Wang, Scott E. Woodman, Mingchu Xu, Jianhua Zhang, Qingxiu C. Zhang, Shanyu Zhang, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchéz-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L. Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor thrombus (TT) worsens prognosis and complicates surgery in renal cell carcinoma (RCC), yet its formation mechanisms remain unclear. Here, we perform integrative single-cell and spatial transcriptomic analyses on 71 tissues and 48 sections from RCC patients with or without TT. The cellular and spatial atlas reveals distinct TT-associated tumor microenvironment remodeling characterized by the enrichment of FAP+ fibroblasts. These FAP+ fibroblasts are spatially contiguous to aggressive cancer cells and promote their malignant phenotypes in vitro. Their abundance inversely correlates with functional NK cells, suggesting roles in tumor invasion and immune evasion. Furthermore, single-cell multiomics analysis identifies tumor pericytes as a source of FAP+ fibroblasts and delineates transcription factor dynamics underlying pericyte-fibroblast transition. Finally, high levels of FAP+ fibroblasts are associated with poor prognosis and predict a weaker response to anti-VEGF-based therapy. In conclusion, our study highlights FAP+ fibroblasts as drivers of aggressive RCC with TT, suggesting potential therapeutic targets. Tumour thrombus (TT) complicates renal cell carcinoma (RCC) treatment. Here, authors conduct an integrative single-cell RNA sequencing and spatial transcriptomic analyses on RCC patients with and without TT and identify FAP+ fibroblasts as drivers for TT formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472524a744e590345b74e85358cb939dc059d29" target='_blank'>
              FAP+ fibroblasts orchestrate tumor microenvironment remodeling in renal cell carcinoma with tumor thrombus
              </a>
            </td>
          <td>
            Jiacheng Ma, Yan Huang, Jie Chen, Yang Li, Rongyan Yao, Xiubin Li, Qiyang Liang, Xinran Chen, Cheng Peng, Kan Liu, Yuanjun Zhai, Xu Zhang, Xin Ma, Xiaowen Wang, Qingbo Huang, Fuchu He
          </td>
          <td>2025-10-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Background The tumor microenvironment influences treatment response in ER-positive breast cancer, but what distinguishes responders from non-responders and how it changes during treatment is poorly understood. Methods ER-positive breast tumors treated with neoadjuvant chemotherapy with or without bevacizumab were profiled with bulk proteomics pre- (n = 95), on- (n = 84) and post-treatment (n = 100). A subset of tumors was profiled with spatial single-cell proteomics pre- (n = 13) and on-treatment (n = 11). Cell phenotypes, spatial location and activation states were determined, and cellular colocalization assessed with spatial metrics. Bulk and spatial features were evaluated against treatment response defined by residual cancer burden. Results Treatment with bevacizumab amplified chemotherapy effects on proteomic signaling. The immune contexture shifted from suppressive to supportive during treatment through decreased macrophage, regulatory and anergic T-cell density and increased colocalization between epithelial cells and CD8+, CD4+ T-cells and dendritic cells. At baseline, responders had high density of effector memory T-cells, while non-responders had more naïve T-cells. In addition, responders had increased colocalization of epithelial cells with macrophages, and effector memory T-cells with M1-like macrophages compared to non-responders. Conclusions Spatially distinct tumor-immune microenvironments influence response to neoadjuvant treatment, offering valuable insights for guiding treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Diffuse midline glioma (DIPG/DMG) is a uniformly fatal pediatric brain tumor with no effective cure. Although CAR T-cell therapy shows promise, clinical outcomes remain inconsistent due to limited persistence and premature exhaustion. Reliable predictive biomarkers are lacking, and proposed exhaustion or stemness markers provide limited utility. Here, we systematically compare multiple CAR-T constructs targeting clinically-relevant antigen B7-H3 and identify antigen-independent CAR activation, or tonic signaling, as a key determinant of therapeutic performance. We find that B7-H3 CAR-T cells with restrained tonic signaling display superior tumor killing, persistence, and resistance to exhaustion, along with reduced CAR membrane clustering, in patient-derived DIPG models. Integrated multi-omics and single-cell profiling further reveal a CAR-T tonic signaling–associated gene signature that outperforms conventional exhaustion or stemness markers in predicting therapeutic efficacy across multiple clinical trials, including DIPG and other tumor types. Together, these findings define a mechanistic and predictive framework to guide CAR design and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c036e69ca133347b296f15467d94c4fe32de6f8f" target='_blank'>
              A Tonic Signaling Code Predicts CAR-T Cell Efficacy in Diffuse Midline Glioma
              </a>
            </td>
          <td>
            Emily B. Deng, Xiaowen Zhong, Dazhuan E Xin, Upendra K. Soni, Wenkun Ma, Xiao Huang, Mingjun Cai, Po-Yu Liang, Jun Bai, Qian Qin, Shreya Mishra, Ming Hu, Arman E. Bayat, Jiajie Diao, Mei Xin, Natasha Pillay-Smiley, T. Hummel, Charles B. Stevenson, Jessica B Foster, Peter de Blank, Scott Raskin, C. Koschmann, Joseide A. Cancelas, Yi Zheng, Q. R. Lu
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The immune system is deeply shaped by its anatomical context, with spatial organization emerging as a fundamental principle of immune regulation. Recent advances in spatial omics technologies—encompassing transcriptomics, proteomics, metabolomics, lipidomics, and phosphoproteomics—have revolutionized our ability to study immune processes within intact tissue environments. By preserving spatial coordinates while capturing high‐dimensional molecular data, these technologies offer unprecedented insight into how immune cell states and functions are governed by local cues and tissue architecture. In this review, we provide an overview of the major spatial omics platforms, emphasizing methodologies that have gained traction within the immunology community and in our own research. We then illustrate how these tools have begun to elucidate the logic of immune compartmentalization across anatomically complex tissues. While not exhaustive, we highlight selected examples from the intestine, secondary lymphoid organs, and liver to show how spatial omics has uncovered region‐specific immune programs, microenvironmental niches, and context‐dependent signaling pathways. Together, these studies demonstrate how spatial omics technologies are redefining immunological inquiry—shifting the focus from isolated cell types to their spatially embedded roles in tissue physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e33ddb8a880f7c8c895cc2a4a4ca66e664dd1a05" target='_blank'>
              From Histology to High‐Resolution Mapping: The Rise of Spatial Omics in Immunology
              </a>
            </td>
          <td>
            Marco De Giovanni, D. Inverso, Matteo Iannacone
          </td>
          <td>2025-10-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6a05aa622e28c5112d10c171846406c87af99fc" target='_blank'>
              High resolution spatial profiling of the hematopoietic landscape of the murine lung
              </a>
            </td>
          <td>
            Nicholas Skvir, Alexandra B. Ysasi, Veronica Jarzabek, Anthony K. Yeung, Jiaji G Chen, Ruben Dries, George J. Murphy
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive neuroendocrine malignancy characterized by rapid onset of chemoresistance and poor clinical outcomes. Transcriptional heterogeneity among treatment-naïve SCLC tumors underlies four transcriptional subtypes, each with distinct clinical vulnerabilities. Though previously hypothesized to delineate a distinct subtype, expression of YAP1 is largely absent from treatment-naïve, pure SCLC. To characterize relapsed SCLC, circulating tumor DNA, circulating tumor cells, and core needle biopsies from SCLC patients and preclinical models following resistance to standard-of-care therapies were analyzed. In contrast to treatment-naïve SCLC, these analyses reveal an emergent YAP1-positive cell population that coincides with treatment resistance. These YAP1-positive cells exhibit characteristics of drug tolerant persister cells, including senescence, stemness, and plasticity, as YAP1 positive cells largely abandon features characteristic of SCLC to adopt those of large-cell neuroendocrine carcinoma (LCNEC). As a result of this SCLC-like to LCNEC-like evolution, YAP1-positive cells lack several clinically relevant SCLC surface targets (i.e., DLL3, SEZ6), but are enriched for others (i.e., B7-H3, TROP2). We propose a model where YAP1 expressing cells emerge with SCLC treatment resistance and characterize a tenacious subpopulation capable of diverging from the treatment naïve lineage and adopting features to evade therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Meningiomas represent the most common primary intracranial tumors, accounting for approximately one-third of adult CNS tumors. Despite their frequency, model systems that accurately replicate the complex tumor-microenvironment interactions across meningioma grades remain underdeveloped, limiting translational progress. This study aimed to establish and characterize human meningioma tumor models to interrogate grade-specific cellular behaviors and microenvironmental dynamics



 Patient-derived meningioma cell lines were established through mechanical and enzymatic dissociation of surgically resected tumors, followed by expansion in specialized culture media. GFP transfection was performed to enable live-cell tracking. For ex vivo modeling, human neocortical slices — obtained from access tissue from non-meningioma surgeries, either 2 cm distal to the glioma core or from epilepsy surgeries under neuronavigation guidance — were injected with GFP-labeled meningioma cells. Tumor progression, cluster formation, and tissue integration were assessed over six days using longitudinal live imaging and immunofluorescence staining. Single-cell RNA sequencing (scRNA-seq) was employed to resolve cellular heterogeneity and tumor-microenvironmental interactions. Machine learning-based image segmentation combined with manual curation enabled precise quantification of tumor cell distribution and interaction patterns within the host tissue.



 Grade 3 meningioma cells demonstrated significantly enhanced cluster formation, tissue invasion, and microenvironmental remodeling compared to lower-grade counterparts. Ex vivo models recapitulated patient tumor behaviors, with scRNA-seq revealing upregulation of genes associated with mesenchymal transition, extracellular matrix remodeling, and immune modulation in Grade 3 tumors. Microenvironmental analysis indicated that higher-grade tumors established more extensive cell-cell interaction networks and induced greater astrocytic and microglial activation relative to Grade 1 and 2 models. Furthermore, immune profiling revealed a more immunosuppressive environment, with myeloid-derived suppressor cells, reducing immune surveillance and facilitating tumor evasion. Additionally, there was increased activation of astrocytes and microglia, contributing to a supportive tumor microenvironment.



 Our patient-derived meningioma models reveal grade-specific differences in tumor behavior and TME interactions. Grade 3 meningiomas show aggressive clustering and microenvironmental remodeling, offering insights into malignant progression. These models provide a high-fidelity platform for studying tumor biology and preclinical evaluation of therapies targeting tumor-stroma interactions
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b180fe0ce6bc7373d74452191faf8369d3bd9e2c" target='_blank'>
              P13.17.B HUMAN MENINGIOMA TUMOR MODELS REVEAL GRADE-SPECIFIC MICROENVIRONMENTAL INTERACTIONS
              </a>
            </td>
          <td>
            A. Mohammed, K. Joseph, D. Depro, G. Nishita, V. Jariwala, A. Boecke, M. Al Shhab, I. Vasilikos, J. M. Nakagawa, T. Juratli, R. Roelz, M. Shah, J. Beck, J. Grauvogel, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) is a dynamic process that contributes to breast cancer progression, metastasis, and therapy resistance. EMT also influences the tumor immune microenvironment, shaping immune cell infiltration and function. Although the immunological features of fully epithelial or fully mesenchymal tumor states have been characterized, the immune landscape associated with intermediate or partial EMT states remains poorly understood. To address this gap, we established five single-cell–derived clonal populations from the triple-negative 4T1 mouse mammary tumor cell line, representing a spectrum of EMT phenotypes, and analyzed tumors derived from these clones using single-cell RNA sequencing. We show that tumors derived from these clones retained their relative EMT states in vivo, and that EMT progression is associated with a graded reduction in tumor immunogenicity, accompanied by alterations in immune cell composition and function. Along the EMT spectrum, tumor cells exhibited progressive downregulation of major histocompatibility complex (MHC) class I and II gene expression, along with decreased infiltration of cytotoxic CD8+ T cells, and reduced expression of key effector and trafficking genes (Gzmb, Ccr5, Cxcr6). B cell composition also shifted, with decreased frequencies of IgG1-producing plasma cells and an enrichment of regulatory-like B cells. Natural killer (NK) cells similarly demonstrated progressive functional suppression, marked by reduced expression of cytotoxic molecules and the downregulation of key effector pathways. These findings reveal that EMT is associated with immune cell recruitment and function in a graded manner, providing a rationale for integrating EMT phenotyping into therapeutic strategies to overcome immune resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332a17e9829514c9ccbfb7575277eaaea290a475" target='_blank'>
              Epithelial-Mesenchymal Transition is Associated with Altered Immune Composition and Cytotoxic Function in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Hanxu Lu, Meisam Bagheri, Fred W. Kolling, L. Salas, Xiaofeng Wang, Todd W. Miller, D. Pattabiraman, B. Christensen
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75be2c8e54a33a7fe17a0e30ce626e667d1c6218" target='_blank'>
              Revealing Shared Tumor Microenvironment Dynamics Related to Microsatellite Instability Across Different Cancers Using Cellular Social Network Analysis
              </a>
            </td>
          <td>
            N. Zamanitajeddin, M. Jahanifar, M. Eastwood, G. Gunesli, M. J. Arends, N. Rajpoot
          </td>
          <td>2025-09-29</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Organoid technology has emerged as a revolutionary tool in cancer research, offering physiologically accurate, three-dimensional models that preserve the histoarchitecture, genetic stability, and phenotypic complexity of primary tumors. These self-organizing structures, derived from adult stem cells, induced pluripotent stem cells, or patient tumor biopsies, recapitulate critical aspects of tumor heterogeneity, clonal evolution, and microenvironmental interactions. Organoids serve as powerful systems for modeling tumor progression, assessing drug sensitivity and resistance, and guiding precision oncology strategies. Recent innovations have extended organoid capabilities beyond static culture systems. Integration with microfluidic organoid-on-chip platforms, high-throughput CRISPR-based functional genomics, and AI-driven phenotypic analytics has enhanced mechanistic insight and translational relevance. Co-culture systems incorporating immune, stromal, and endothelial components now permit dynamic modeling of tumor–host interactions, immunotherapeutic responses, and metastatic behavior. Comparative analyses with conventional platforms, 2D monolayers, spheroids, and patient-derived xenografts emphasize the superior fidelity and clinical potential of organoids. Despite these advances, several challenges remain, such as protocol variability, incomplete recapitulation of systemic physiology, and limitations in scalability, standardization, and regulatory alignment. Addressing these gaps with unified workflows, synthetic matrices, vascularized and innervated co-cultures, and GMP-compliant manufacturing will be crucial for clinical integration. Proactive engagement with regulatory frameworks and ethical guidelines will be pivotal to ensuring safe, responsible, and equitable clinical translation. With the convergence of bioengineering, multi-omics, and computational modeling, organoids are poised to become indispensable tools in next-generation oncology, driving mechanistic discovery, predictive diagnostics, and personalized therapy optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3effe5e0ade80f8d6b3e4aa8e2c7994edd86eb" target='_blank'>
              Organoids as Next-Generation Models for Tumor Heterogeneity, Personalized Therapy, and Cancer Research: Advancements, Applications, and Future Directions
              </a>
            </td>
          <td>
            Ayush Madan, Ramandeep Saini, Nainci Dhiman, Shu-Hui Juan, M. K. Satapathy
          </td>
          <td>2025-10-08</td>
          <td>Organoids</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3145b8048f1a33f4e0cf44307241622724c18a05" target='_blank'>
              Mapping CD4+ T Cell Landscapes in Glioblastoma Reveals Effectors and Bystanders
              </a>
            </td>
          <td>
            Cameron M. Hill, Anthony Z. Wang, Jose A. Maldonado, T. Johanns, Allegra A. Petti, Gavin P. Dunn
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Neuroblastoma, a deadly pediatric cancer derived from sympathetic ganglia of the peripheral nervous system, frequently metastasizes, driving poor outcomes. Primary neuroblastomas are well-characterized, but the mechanisms underlying metastasis remain poorly understood. Here, by applying single-cell and spatial multi-omics analyses to primary and metastatic tumors, we found that lymph-node metastases in high-risk neuroblastomas display distinctive cellular heterogeneity and plasticity, marked by mesenchymal-like and stem-like states and heightened epithelial-to-mesenchymal transition activity compared to primary adrenal tumors. Additionally, compared to primary adrenal masses, the metastatic niche display increased immunosuppressive myeloid programs, heightened immune checkpoint signaling, and lymphocyte exhaustion, which are indicative of immune evasion and dysfunction. Notably, metastatic neuroblastomas show elevated eIF4F translation machinery and XPO1 levels. Dual inhibition of eIF4A and XPO1 synergistically halted tumor growth and prolonged survival in xenograft models. Together, our multi-omics studies reveal the molecular and cellular plasticity that contributes to therapy resistance and highlight exploitable therapeutic vulnerabilities in high-risk metastatic neuroblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, Long-Sheng Chang, Brian D Weiss, Q. R. Lu
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Human pancreatic ductal adenocarcinomas (PDAC) are composed of sparse, heterogeneous tumour cell populations embedded within a dense, desmoplastic stroma and are highly resistant to existing therapies. Guided by underlying genomic aberrations, cancer cell states and phenotypes interact with their microenvironment to shape disease trajectory and therapeutic response. To quantify the cellular phenotypic heterogeneity of these complex tumour-microenvironment interactions we designed custom multiplexed histopathology imaging panels against single cell RNA sequencing-defined tumour, immune, and stromal cell compartments. Using imaging mass cytometry (IMC), we profiled the co-localization and organization of 83 cell types and their functional states in 221 resected pancreatic tumours. This identified extensive inter- and intra-patient heterogeneity including the presence of both classical and basal cell types in most patients and a gradient of pancreatic epithelial identity confirmed by scRNAseq. This single cell content defined an expanded spectrum of classifiable intermediate tumour subtypes between classical and basal which have specific associations to neighboring immune and stroma cellular content, ploidy, established broad transcriptional subtypes, heterogeneity, and patient outcome. Eight reproducible cancer microenvironments were quantified, empowering cross-tumour comparisons. Matched whole genome sequencing (WGS) identified aberrations associated with specific tumour phenotypes and microenvironments. These results support asynchronous tumour and microenvironment genotype-phenotype axes which subdivide basal to classical tumour polarization and categorize co-existing microenvironments from stiff, ECM-rich, immune suppressed to immune infiltrated regions alongside fibrovascularized CD105+ stroma. This is based upon spatially heterogeneous co-occurring microenvironment niches and correlated tumour phenotypes which are associated with KRAS or MYC amplification and CDKN2A or LATS2 deletion on one axis, mutation of epigenetic modifiers in intermediate and hybrid subtypes, and RNF43 deletion on the other axis. To deeply profile the signalling pathways and enriched biological processes characteristic of each tissue state, we performed deep proteomics measurements of each microenvironment and tumor phenotype through whole-slide IMC informed laser capture microdissection and mass spectrometry of 273 micro-regions from a subset of patients. Finally, to identify a robust, minimized set of outcome-relevant features, we used a multi-modal technology optimized machine learning model trained to predict overall survival. We compare the predictive potential of WGS and spatial proteomics and show that a combination of genomic and cellular content outperforms clinical features and platform specific models thereby demonstrating the synergistic benefit of integrated multi-modal data. Our findings classify the spatial organization of human PDAC to identify an intertwined tumour/microenvironment axis that is influenced by specific genomic aberrations.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer Gorman, Chengxin Yu, Michael Geuenich, Sheng Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra Wong, Brendon Seale, Zhen Yuan Lin, Edward Chen, Golnaz Abazari, Miralem Mrkjonic, Julie Wilson, Kieran Campbell, Anne-Claude Gingras, Rob Grant, Grainne O'Kane, Faiyez Notta, Steve Gallinger, Hartland Jackson. Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B128.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e2ac6f788886b4d295a71549b94c0503bd2fb" target='_blank'>
              Abstract B128: Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, Miralem Mrkjonic, Julie M. Wilson, Kieran R. Campbell, A. Gingras, Robert C. Grant, G. O’Kane, Faiyez Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCLs) are a heterogeneous group of malignancies that can arise in lymph nodes or extranodal locations, including immune-privileged sites. Here, we applied highly multiplexed spatial transcriptomics and proteomics together with genomic profiling to characterize the immune microenvironment architecture of 78 DLBCL tumors. We define seven distinct cellular niches, each characterized by unique cellular compositions, spatial organizations and patterns of intercellular communication associated with niche-specific phenotypes of both T cells and tumor B cells. Among these, DLBCLs from immune-privileged sites showed abundant T cell infiltration into diffuse niches, where immune cells were intermixed with tumor B cells and bore transcriptional hallmarks of activation and effector function, suggesting that they may be primed for anti-tumor immunity. Spatial characterization of the DLBCL immune microenvironment, therefore, reveals cellular niches that foster divergent patterns of cell–cell interactions contributing to the phenotypic heterogeneity of both niche-resident tumor and immune cells. Analysis of the immune microenvironment of diffuse B cell lymphomas using spatial transcriptomics, proteomics and genomics highlights discrete cellular niches with divergent patterns of cell–cell communication that contribute to the phenotypic heterogeneity of both tumor and immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, A. Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4769ce329e17e111a404f23e5026b957dcf66d76" target='_blank'>
              Spatial Structure of Tumor and Immune Cells Shape Outcomes in ER-positive-HER2-negative and Triple-negative Breast Cancer
              </a>
            </td>
          <td>
            I. Nederlof, Siamak Hajizadeh, Rolf Harkes, Barbara Andrade Barbosa, M. D. Maaker, Ingrid Hoﬂand, Dennis Peters, R. Elens, A. Broeks, E. Werkhoven, Yongsoo Kim, Marieke Ijsselstein, Noel F.C.C. de Miranda, S. Bosma, Nicole C. M. Visser, Malou L. H. Snijders, Joyce Sanders, Roberto Salgado, Philip M P Poortmans, Astrid Scholten, Liesbeth J. Boersma, M. Kok, Harry Bartelink, M. V. D. Vijver, Hugo M. Horlings
          </td>
          <td>2025-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive. Method To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through in vitro co-culture and immunohistochemistry. Result We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface. In vitro functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown. Conclusion We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Tumor-infiltrating lymphocyte (TIL) therapy has demonstrated efficacy in melanoma and is emerging as a promising modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors like glioblastoma is limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas to identify molecular and spatial features associated with successful TIL expansion.



 We conducted comprehensive profiling of 18 TIL products using spectral flow cytometry and T Cell Receptor sequencing (TCRseq). Additionally, 4 tumors yielding ≥10⁸ viable TILs per 0.5 g tumor within two weeks of IL2 Rapid Expansion Protocol cultures, classified as TIL⁺, were analyzed and compared to 4 non-yielding tumors, classified as TIL⁻. Expanded TILs were subjected to spectral flow cytometry and TCRseq. These TIL⁺ and TIL⁻ tumors underwent single-cell RNA-seq (scRNAseq), in situ Xenium spatial transcriptomics using a customized 480-gene panel, and CODEX spatial proteomics.



 The TIL⁺ tumors yielded 0.5-3.2 × 10⁹ cells with 100-1,000-fold expansion. Flow cytometry showed CD4⁺ and CD8⁺ effector memory phenotypes with minimal exhaustion; Tregs were <10% and γδ T cells were detected. TCRseq revealed high frequency αβ clones and diverse γδ repertoires. Expanded γδ T cells had higher D50 and diversity indices, suggesting a broad, non-dominant repertoire. TIL⁺ tumors exhibited elevated IL7R and ACSS3 expression, enriched Th1/pro-inflammatory gene signatures, and lower Treg-associated transcripts. Spatial analysis showed perivascular immune clusters in TIL⁺ tumors, enriched for CD8⁺ T cells and antigen-presenting cells, with local upregulation of IL7R, MHC II, and CXCL13. These areas were associated with vascular markers (e.g., VCAM1) and T cell retention signatures. In contrast, TIL⁻ tumors displayed immune exclusion, neuronal lineage enrichment, and higher expression of immune suppressive genes like TOX and FERMT1. Tumor-connected TAMs expressing SOX2, NES, and AQP4 were more abundant in TIL⁻ tumors, suggesting a barrier to infiltration or active immune suppression.



 TIL⁺ tumors exhibit distinct molecular and spatial features associated with successful TIL expansion, including IL7R-driven T cell responsiveness, mesenchymal metabolic programs, and structured perivascular immune niches. These findings define biomarkers of therapeutic TIL production and inform strategies to stratify patients for TIL⁻ based therapies in glioblastoma and other immune-excluded tumors. Incorporating these biomarkers into patient selection and workflows could improve clinical trial design, product consistency, and therapeutic response rates.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b47fb61580f74a05d0a598bf26243ed8f703f3" target='_blank'>
              KS03.4.A SPATIAL MULTI-OMIC PROFILING REVEALS DETERMINANTS OF TIL EXPANDABILITY IN HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            K. Hotchkiss, K. Zhang, A. Corcoran, E. Owens, P. Noldner, C. Railton, K. Van Batavia, Y. Zhou, J. Jepson, K. Singh, R. McLendon, K. Batich, A. Patel, K. Ayasoufi, M. Brown, E. Calabrese, J. Xie, J. Conejo-Garcia, B. Shaz, J. Hickey, M. Khasraw
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The mortality rate of ovarian cancer remains disproportionately high compared to its incidence. This is partly due to a high level of intra-tumoral heterogeneity that promotes disease recurrence and treatment failure. In this study, we describe degrees of heterogeneity revealed by single-cell whole genome sequencing and spatial transcriptomics of five epithelial ovarian carcinomas. At the cellular level, we describe pseudo-diploid cells that match the malignant cell population in both somatic variant and copy number patterns. At the clonal and subclonal levels, we describe diversification associated with copy number gains and whole genome doubling. In multi-clonal samples, we infer evolutionary relationships from single cell copy number, loss of heterozygosity analysis, and somatic variant detection, and correlate these with tissue histology and gene expression programs. In one sample, we identify functionally consequential copy number alterations that contribute to molecular diversity, cell proliferation, and inflammation in a minor clone that persisted without major expansion alongside a more complex major clone. In another, we describe a complex evolutionary history including a spontaneous reversion of a driver mutation in a secondary clone, which correlated with a switch in oncogenic expression programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM) quiescent (Q) cell populations are hypothesized to contain cancer stem-like cells (CSC) that drive tumor growth, cellular heterogeneity, and recurrence. However, GBM tumors do not neatly resolve into developmental hierarchies and Q stem-like activities are difficult to assess. Here, we evaluated tumor Q subpopulations in patient-derived GBM xenograft tumors using live cell reporters, DNA label retention assays, and single cell genomics. Compared to adult neural stems cells (NSCs), GBM Q populations contain hybrid transcriptional states composed of networks found in both dormant and activated adult NSCs, resulting in constitutive expression of key Q egress transcription factors and their targets (e.g., AP-1 and CCND1/2). As a result, even the longest Q-residing cells (∼12 days) in xenograft tumors continuously cycle and fail to enter dormant Q states. We provide evidence and hypothesize that transient Q states in primary tumors arise as part of distinct proliferative compartments rather than deterministic developmental hierarchies driven by CSC activity. We further speculate that increases in basal translation rates drive Q instability in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038a04e70ede8fe2ebdfee87bc265630d9451dc" target='_blank'>
              Characterization of quiescent subpopulations and proliferative compartments in glioblastoma
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Samantha A. O’Connor, Anoop P. Patel, Christopher L. Plaisier, Patrick J. Paddison
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are central architects of the prostate cancer (PCa) microenvironment, yet their phenotypic diversity and druggable vulnerabilities remain largely uncharted. Here, we present an integrative multi-omics characterization of primary ex vivo CAFs from seven treatment-naïve PCa patients. Using single-cell RNA sequencing (scRNA-seq), we uncover substantial transcriptional heterogeneity among CAFs, with distinct gene expression programs related to extracellular matrix remodeling, inflammation, immune modulation, and metabolic reprogramming. This phenotypic diversity was further supported by variable expression of canonical stromal markers, including FAP, SULF1, VIM, CAV1, and αSMA. Transcription factor network analysis revealed SOX, FOX, and STAT3 family members as key regulators of pro-tumorigenic CAF states. To probe therapeutic vulnerabilities, we performed high-throughput drug sensitivity and resistance testing (DSRT) across 396 oncology compounds. CAFs exhibited broad sensitivity to multikinase inhibitors, with dasatinib, midostaurin, and FGFR inhibitors (AZD4547, erdafitinib) emerging as top stromal-directed candidates. These findings underscore the plasticity of prostate CAFs and reveal actionable vulnerabilities, supporting the development of targeted stromal therapies to disrupt tumor-stroma interactions in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0342abe6ecbf94b88ae457a3f680a1e8d054b454" target='_blank'>
              Molecular profiling of ex vivo prostate cancer CAF models captures stromal heterogeneity and drug vulnerabilities
              </a>
            </td>
          <td>
            Frida Rantanen, A. Murumägi, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-11-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The spontaneous regression of cancer lesions demonstrates the potential of immune surveillance; yet, these transient events have remained inaccessible to systematic study. Using longitudinal intravital microscopy in a carcinogen-induced model of head and neck cancer, we tracked premalignant lesions within the same animals for 24 weeks at single-cell resolution. This approach revealed three trajectories: progression, stability, or regression, and enabled dissection of the immune dynamics underlying each fate. Lesion outcome was dictated by the spatial organization of myeloid-derived antigen-presenting cells: regressing lesions were characterized by dense clusters of myeloid-derived cells associated with CXCL9+/CXCL10+ expression and T cell recruitment, whereas progressing lesions displayed sparse, non-clustered infiltration. Remarkably, transient myeloid clusters arose prior to any detectable lesion formation and consistently marked regions that would later develop into premalignant lesions. These findings identify spatiotemporal myeloid organization as an early determinant of tumor fate and provide a mechanistic framework for predicting and intercepting cancer at its inception. Summary Sentence Early myeloid architecture dictates cancer fate: dense CXCL9⁺/CXCL10⁺ clusters with T-cell enrichment accompany regression, whereas sparse infiltration predicts progression. Transient pre-lesional myeloid clusters emerge at future tumor sites, revealing immune organization as an early determinant of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background : Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods : We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre-and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results : In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions : MIF + tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Recent studies using single-cell RNA sequencing (scRNA-seq) have provided insights into the cellular composition and molecular states in glioblastoma (GBM). However, these studies have not identified significant differences between primary and recurrent GBMs. We hypothesize that other factors, such as spatial organization and cellular interactions, may provide key information driving recurrence in GBM. Spatial transcriptomics techniques, such as MERFISH (Multiplexed Error-Robust Fluorescence In Situ Hybridization), provide single-cell spatial resolution of tissues, enabling the identification of distinct molecular features. MERFISH, in particular, allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, providing a detailed spatial map of the tumor cells and surrounding environment. This technology is invaluable in studying longitudinal GBMs, where changes in tumor organization, immune cell interactions, and mechanisms of therapeutic resistance remain poorly understood.



 We analyzed 102 tumor samples consisting of primary and recurrent GBMs and normal brain tissue using Visium and MERSCOPE technologies. We used supervised artificial intelligence (AI)-based learning, in-silico perturbation and explainable-AI to identify distinct spatial and cell-type-specific transcriptomic reorganization associated with GBM recurrence.



 We developed a spatial transcriptomics atlas of GBM at sub-cellular resolution. Our analysis revealed distinct cellular neighborhoods in the primary and recurrent GBM tumors, with changes leading to the dynamic spatial reorganization of GBM at recurrence. The 3D spatial images acquired provided a comprehensive view of the tumor cells interacting with the tumor microenvironment in situ. Our explainable-AI models successfully identified distinct spatial transcriptomic features associated with recurrence in GBM. The identified spatially distinct cellular neighborhoods reveal dynamic shift in cellular organization following in silico perturbation.



 This study provides a high-resolution spatial map of longitudinal GBM that offers new insights into the dynamic cellular reorganization of GBM at recurrence. Our findings highlight the role of cellular reorganization during tumor recurrence in GBM, a potential avenue for targeted therapeutic intervention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4929df1bf2a8e42843ae0cfe5558a1691cc6412" target='_blank'>
              P04.05.B DEEP LEARNING ANALYSIS OF SPATIALLY RESOLVED SINGLE-CELL TRANSCRIPTOMICS UNCOVERS CELLULAR REORGANIZATION IN GLIOBLASTOMA AT RECURRENCE
              </a>
            </td>
          <td>
            Y. Yabo, E. Grabis, T. Peng, J. Benotmane, J. Kueckelhaus, J. Zhang, G. Villa, N. Neidert, M. Prinz, J. Beck, A. Ahmed, O. Schnell, F. Sahm, V. Venkataramani, F. Ricklefs, R. Sankowski, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The bone marrow (BM) is a complex and compartmentalized tissue where spatial context plays a critical role in regulating cell behavior, signaling, and disease progression. To capture these dynamics, we apply spatial transcriptomics using the Visium Spatial Gene Expression platform on formalin-fixed paraffin-embedded (FFPE) BM sections from both healthy and Multiple Myeloma (MM) mouse models, as well as MM patient samples. Overcoming the technical challenges of working with mineralized long bone tissue, we develop a custom analytical framework integrating spatial and single-cell transcriptomic data to map cellular composition and interactions in situ. This approach enables the spatial characterization of transcriptionally heterogeneous malignant plasma cells (MM-PC) and their surrounding microenvironments. We identify spatially distinct gene programs linked to MM pathogenesis, including signatures of NETosis and IL-17 signalling, which are reduced in MM-PC–rich regions. Additionally, a transition gradient from effector to exhausted T cell phenotype is associated with increased remoteness from MM-PC. These spatial patterns are identified in FFPE BM biopsies from MM patients with varying tumor burdens. In summary, our study demonstrates both the capabilities and limitations of Visium technology in characterizing spatially regulated mechanisms underlying MM pathogenesis. Visium Spatial technology enables the characterization of transcriptionally heterogeneous malignant plasma cells in multiple myeloma, and identify spatially distinct gene programs linked to MM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f354b66d39731b9cc107b87037fbdf9278486f" target='_blank'>
              Characterization of the bone marrow architecture of multiple myeloma using spatial transcriptomics
              </a>
            </td>
          <td>
            E. Muiños-López, Ana Rosa López-Pérez, Laura Sudupe, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, P. Aguirre-Ruiz, P. S. Martín-Uriz, M. Larrayoz, L. Alvarez-Gigli, M. Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, Bruno Paiva, L. Tamariz-Amador, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common histological subtype of renal cancer and remains a clinical challenge due to its frequent resistance to therapy and poor prognosis in advanced stages. Apoptosis, a fundamental tumor-suppressive mechanism, exhibits paradoxical roles in cancer, wherein apoptotic tumor cells can also contribute to immunosuppression and tumor progression. However, the spatial dynamics, transcriptional heterogeneity, and prognostic relevance of apoptosis-related gene programs in ccRCC remain poorly defined. We performed an integrative analysis combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and summary-based Mendelian randomization (SMR) to dissect apoptosis-related malignant cell states in ccRCC. Cancer cells were stratified based on apoptosis gene signatures and CASP9 expression. Cell–cell communication was assessed using CellChat and spatial interaction networks were constructed using RCTD and mistyR. SMR was employed to link genetically regulated CASP9 expression with renal cancer risk. A CASP9-associated prognostic model was developed using LASSO Cox regression and DeepSurv on TCGA and E-MTAB-1980 cohorts. We identified transcriptionally and spatially distinct apoptosis-high and apoptosis-low malignant cell subpopulations. Apoptosis-high tumor cells, characterized by elevated CASP9 expression, preferentially localized near macrophage-enriched stromal regions and exhibited stronger spatial clustering. Ligand–receptor modeling revealed directional signaling via the SPP1–CD44 axis between CASP9-high cancer cells and macrophages. SMR analysis provided genetic evidence supporting CASP9 as a causal gene for renal cancer. CASP9-high cells demonstrated distinct developmental trajectories and formed multicellular spatial modules with macrophages and cycling cells. A five-gene apoptosis-related signature derived from CASP9-stratified tumor cells robustly predicted patient survival across both training and validation cohorts. Low-risk patients exhibited enriched immune infiltration, increased immune checkpoint expression, and enhanced immune pathway activity. Our study reveals that apoptosis, particularly CASP9-driven programs, defines a spatially organized, immunosuppressive malignant cell state in ccRCC. CASP9 acts as both a genetic driver and spatial regulator of tumor–macrophage interactions, contributing to disease progression. The CASP9-associated risk model demonstrates strong prognostic utility and highlights apoptosis as a promising therapeutic axis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22570e710b314543b7842267551dd125b21247d1" target='_blank'>
              Integrative SMR and single cell & spatial analysis reveals the spatial heterogeneity and prognostic value of CASP9-mediated apoptotic pathways in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zijie Yu, Xinghan Yan, Wenchuan Shao, Lingchen Cai, Da Zhong, Xiyi Wei, Ninghong Song
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            Jianyou Liao, Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor in tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors that possesses self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here, we set up ultra-wide field microscopy to image breast cancer cell lines expressing a stemness fluorescent reporter over several days. The fluorescent reporter distinguishes three phenotypes: CSCs, cancer differentiated cells (CDCs), and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation in which CSCs cluster together and are spatially separated from CDCs, forming patterns resembling niches. Surprisingly, single-cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we find that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0816aa85e620c96aaa89badbcea20ed0e370fd6d" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulet, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2025-10-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mass cytometry (CyTOF) and Imaging Mass Cytometry (IMC) provide single-cell resolution for over 50 protein markers, enabling unprecedented exploration of tumour and immune heterogeneity. We conducted a scoping review of 61 original studies (inception–2025), spanning 17 cancer types, to map current applications, analytical strategies, and emerging biological insights. 46 studies used CyTOF alone, 12 employed IMC exclusively, and 3 combined both platforms. Median panel sizes were 33.5 markers for CyTOF and 33 for IMC. While lineage and immune checkpoint markers were universal, phospho-epitopes, metabolic enzymes, and stromal proteins appeared in more focused subsets. Most studies followed a three-step analytical workflow: (i) segmentation or gating, (ii) unsupervised clustering, and (iii) downstream spatial or functional analyses. CyTOF investigations frequently identified exhausted CD8+ T-cell subsets (e.g., PD-1+TIM-3+CD39+), suppressive myeloid populations (e.g., CD163+HLA-DR− macrophages), and metabolically reprogrammed Tregs. IMC studies uncovered spatial patterns predictive of outcome, such as tertiary lymphoid structures (TLSs) and macrophage–T cell exclusion zones. Several studies proposed predictive immune signatures or integrated CyTOF with transcriptomic or spatial datasets. We identified five recurrent immunobiological motifs, CD8+ T-cell bifurcation, CD38+ TAM barriers, TLS maturity, CTLA-4+ NK-cell signatures and metabolically defined niches, highlighting convergent axes of resistance and response. Bioinformatic pipelines converged around FlowSOM or PhenoGraph clustering, CITRUS or elastic-net feature selection, and increasingly, machine learning and agent-based spatial modelling. Collectively, CyTOF and IMC are redefining biomarker discovery, therapeutic stratification, and virtual trial design in oncology, establishing high-dimensional CyTOF as a cornerstone of next-generation precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751da190ee62eb1876b5c48df077ef5372fdfb0" target='_blank'>
              Pan-cancer convergence of tumour–immune microenvironment motifs revealed by CyTOF and imaging mass cytometry
              </a>
            </td>
          <td>
            Alexandre Vallée, Alexandre Drezet, Maxence Arutkin
          </td>
          <td>2025-10-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, N. Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is composed of a diverse and dynamic spectrum of cell types, cellular activities, and cell–cell interactions (CCI). Understanding the complex CCI within the TME is critical for advancing cancer treatment strategies, including modulating or predicting drug responses. Recent advances in omics technologies, including spatial transcriptomics and proteomics, have allowed improved mapping of CCI within the TME. The integration of omics insights from different platforms may facilitate the identification of novel biomarkers and therapeutic targets. This review discusses the latest computational methods for inferring CCIs from different omics data and various CCI and drug databases, emphasizing their applications in predicting drug responses. We also comprehensively summarize recent patents, clinical trials, and publications that leverage these cellular interactions to refine cancer treatment approaches. We believe that the integration of these CCI-focused technologies can improve personalized therapy for cancer patients, thereby optimizing treatment outcomes and paving the way for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb307e16bbb2fc4780518da53d8301520562b427" target='_blank'>
              Cell–cell interactions as predictive and prognostic markers for drug responses in cancer
              </a>
            </td>
          <td>
            Xuehan Lu, Xiao Tan, Eun Ju Kim, Xinnan Jin, Meg L. Donovan, Jazmina L. Gonzalez Cruz, Zherui Xiong, Maria Reyes Becerra de los Reyes Becerra Perez, Jialei Gong, J. Monkman, Divya Agrawal, A. Kulasinghe, Quan Nguyen, Z. Tuong
          </td>
          <td>2025-10-08</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) evolves within a highly interactive tumor microenvironment (TME) that shapes therapeutic response. We utilized mass cytometry to analyze over 10 million cells from 25 ESCC tumors, 24 adjacent nontumor tissues, and 23 peripheral blood samples, employing an extensive panel of 42 immune markers. The resulting atlas reveals a compartmentalized landscape with a reproducible paucity of CD4⁺ and CD8⁺ central memory T cells (TCM) in tumor sites. Reintroduction of patient-derived TCMs restored antitumor immunity in coculture assays, demonstrating their cytotoxic capacity in vitro and suggesting their potential relevance for future therapeutic exploration. Myeloid profiling identified PD-L1⁺ tumor-associated macrophages (TAMs) as correlates of clinical benefit; ex vivo PD-L1 blockade reprogrammed TAMs toward proinflammatory states, indicating pharmacological malleability. Notably, CD39⁺ tumor-infiltrating T cells were consistently associated with favorable prognosis and increased responsiveness to PD-1 blockade across cancer types. The functional inhibition of CD39 impaired cytotoxic T-cell activity, underscoring its dual role as a marker of immune dysfunction and a promising therapeutic target. Collectively, our findings provide a comprehensive immune landscape of ESCC, highlighting key immunological deficits and opportunities for targeted interventions. The insights gained underscore the potential of tailoring immunotherapies to the specific immune profiles of the TME, potentially revolutionizing treatment paradigms for ESCC patients. This study sets the stage for a more nuanced understanding and manipulation of the immune elements critical for optimizing cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74ee06dce5a8743106e3e0c5ec6ea39727b2973" target='_blank'>
              Single-cell atlas of the esophageal squamous cell carcinoma immune ecosystem to predict immunotherapy response
              </a>
            </td>
          <td>
            Xiankai Chen, Yahui Zhao, Yuhao Wang, Xiliang Wang, Yuhao Liu, Zhihua Liu, Yin Li
          </td>
          <td>2025-10-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g. metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved gene expression correlation analyses. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor’s proliferative front, correlating with expression of genes involved in the epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. Preliminary observations from this single sample revealed that high copper concentrations were localized to regions of active tumor growth and were associated with increased expression of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583ee888bce168b26048a39ba23b693a3c315972" target='_blank'>
              Integration of elemental imaging and spatial transcriptomic profiling for proof-of-concept metals-based pathway analysis of colon tumor microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Yunrui Lu, M. Chan, A. Diallo, John P. Zavras, Serin Han, T. Punshon, B. Jackson, L. Vahdat, Xiaoying Liu, V. Mittal, K. Lau, J. Gui, Louis J. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2025-10-01</td>
          <td>Metallomics: Integrated Biometal Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fdc35836d59f5a9b6ad6d520636c7ede83639b" target='_blank'>
              Single-Cell Analysis of NF1-Expressing and NF1-Deficient Schwann and Fibroblast Cells Reveals Divergent Neurofibroma Programs
              </a>
            </td>
          <td>
            Grace M Swanson, Marcia Arenas-Hernandez, Katherine Gurdziel
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular source of chromaffin cell regeneration after birth and its relationship to paraganglioma tumorigenesis remains incompletely defined. Here, we identify a postnatal population of SOX2/SOX10-expressing sustentacular glia-like cells in the organ of Zuckerkandl (OZ) and adrenal gland that give rise to chromaffin cells in vivo. These cells differ transcriptionally from embryonic chromaffin progenitors known as Schwann cell precursors and exhibit a unique progenitor signature. Genetic lineage tracing confirms their postnatal contribution to chromaffin cells, and SOX2+PHOX2B+ transitional cells were observed in both human and mouse OZ and adrenal tissues. Single-nuclei RNA-seq and inferCNA analysis of pheochromocytoma and paraganglioma (PPGL) revealed that while most sustentacular cells exhibit a stromal profile, a subset in VHL-mutated PPGLs harbor the hallmark 3p chromosomal loss shared with chief tumor cells, suggesting a clonal origin. In an additional PPGL, widespread SOX2 expression in PHOX2B+ tumor cells supports this hypothesis. Finally, DLK1-NOTCH signaling was predicted as a central regulator of chromaffin–sustentacular communication, suggesting DLK1 fine-tunes chromaffin regeneration via NOTCH inhibition and may represent a therapeutic target in PPGL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4df7e6c797d0a4c278f8d03ce42f2fec1c35071" target='_blank'>
              Postnatal sustentacular cells as chromaffin progenitors and tumor cells of origin in VHL-related paragangliomas
              </a>
            </td>
          <td>
            Petra Bullova, Peng Cui, Maria Arceo, Jiacheng Zhu, Wenyu Li, Monika Plescher, Valentin Poltorachenko, Katerina Stripling, Christian Santangeli, Lidiya Mykhaylechko, M. Kastriti, Catharina Larsson, C. Juhlin, Michael Mints, Susanne Schlisio
          </td>
          <td>2025-10-15</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            A. Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>3</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02530b52b1f076b640bb9d3473208058e5838dbf" target='_blank'>
              Self-organization of tumor heterogeneity and plasticity
              </a>
            </td>
          <td>
            Carlos Pérez-González, D. Brückner, M. Di-Luoffo, Sophie Richon, R. Goswami, Meryem B Baghdadi, Florence Piastra-Facon, Neta Felsenthal, Réda Bouras, A. Fumagalli, Mirjam C van der Net, M. Gloerich, Salvatore Girardo, Jochen Guck, J. van Rheenen, J. Guillermet-Guibert, É. Hannezo, D. Vignjevic
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most lethal brain tumor in adults with remarkable cellular plasticity and complex anatomical features. While single-cell RNA sequencing (scRNA-seq) has revealed its cellular heterogeneity, the spatial cellular organization within the anatomical features remains elusive.



 Here, we provide a comprehensive spatial transcriptomic dataset comprising 115,914 transcriptomes from 32 genotyped tissue sections.



 Leveraging Xenium in situ technology, we generated spatial transcriptomic data with subcellular-resolution, significantly improving the granularity of our understanding of GBM cellular architecture. In addition, our analyses identified oligodendrocyte subtypes specifically localized to tumor cores and perivascular regions with molecular profiles associated with poor prognosis.



 This publicly accessible GBM atlas provides a valuable resource for understanding the complex cellular architecture of GBM and facilitating the development of novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1717e2c5370026ee08f9c770891fbab0aaa81409" target='_blank'>
              P02.08.A A SPATIALLY RESOLVED HUMAN GLIOBLASTOMA ATLAS REVEALS DISTINCT CELLULAR AND MOLECULAR PATTERNS OF ANATOMICAL NICHES
              </a>
            </td>
          <td>
            H. Park, P. Sonpatki, C. Park, W. Park, N. Shah
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells—fusion products of tumour and normal cells—and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d2d946c12a3009915b3b21748cb811af18ab2d4" target='_blank'>
              Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications
              </a>
            </td>
          <td>
            Francis Yew Fu Tieng, Learn-Han Lee, N. Ab Mutalib
          </td>
          <td>2025-10-06</td>
          <td>Cancer Metastasis Reviews</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6214e46642ec3f3780a0b2d3254221a9b00d6" target='_blank'>
              Comparative single-cell transcriptomics of orthotopic and subcutaneous gastric tumors reveal immune and stromal heterogeneity
              </a>
            </td>
          <td>
            Jinho Jang, Yoojeong Seo, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 The human brain contains a rich milieu of extracellular matrix (ECM) components that promote normal physiology and are often dysregulated in pathologies including the malignant cancer glioblastoma (GBM). We have used in situ single-cell spatial transcriptomic platforms to map the expression patterns of nearly 400 ECM-related genes in multiple GBM samples (IDH1 wild type). Our analysis identifies at least four different GBM cell populations with unique ECM expression profiles that show spatial enrichment in distinct intratumor regions. Spatial mapping also demonstrates largely non-overlapping expression signatures of various ECM components in stromal cell types, particularly in GBM vascular endothelial cells and reactive microglia/macrophages. Comparisons of GBM versus lower-grade II and III astrocytoma samples (IDH1 R132H) identifies differential expression of key ECM components, including elevated levels of select ECM glycoproteins (IGFBP2 and MGP) and ECM-affiliated proteins (ANXA1 and ANXA2). In addition, we detect spatially enriched expression of COL8A1 (collagen), LUM (proteoglycan), and POSTN (ECM glycoprotein) in perivascular stromal cells in GBM but not in lower grade tumors. Computational analysis of putative ligand-receptor interactions reveals novel communication networks between cancer cells and stromal components, particularly in regions of GBM microvascular proliferation and pseudopalisading necrosis. In summary, this comprehensive spatial map provides new insights into ECM control of tumor initiation and progression and identifies potential therapeutic targets within the GBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e40eadeccedca2699836ba1458071cfd535dcbfb" target='_blank'>
              TMIC-33. Single Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Joseph H. McCarty, Arpan De, Santiago A. Forero, Ali Pirani, John Morales
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy, poised to become the second leading cause of cancer-related deaths in North America. The paucity of neoantigen presentation in PDAC has been hypothesized to promote the development of an “immunologically-cold” tumor microenvironment (TME), driving poor clinical responses to immunotherapy. Despite the abundant immune infiltrate observed in PDAC, T cells display a limited cytotoxic capacity to kill malignant tumor cells which further diminishes throughout disease progression. Utilizing single cell RNA-sequencing coupled to T cell receptor profiling, we have characterized T cell populations in clinical specimens of inflammatory pre-malignant intraductal papillary mucinous neoplasia (IPMN) lesions and advanced stage PDAC tumors. Striking phenotypic and clonal dynamic differences were observed in both CD8 and CD4 T cell populations between IPMN and PDAC specimens which underly their distinct functional capabilities in the evolving TME. The spatial interactions and microenvironmental neighborhoods of neoantigen-reactive T cells in PDAC have yet to be explored in-depth and hold unexploited therapeutic potential. Utilizing the 10X Genomics Xenium platform with custom probes designed to hybridize with neo-antigen-specific T cell clones, we are interrogating the cellular interactions which drive T cell clonal expansion to identify therapeutically relevant vulnerabilities in PDAC malignant progression.



 Supported by National Insitutes of Health, Canadian Institutes of Health Research, and Princess Margaret Cancer Center Catalyst Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6400ccbc9c89325532880266eae31b90968879" target='_blank'>
              Defining molecular determinants of T cell spatial localization and clonal response throughout PDAC malignant progression 3030
              </a>
            </td>
          <td>
            Matthew Bianca, S. Lamorte, Teresa Ciudad, Cristiane Naffah de Souza, Rene Quevedo, Tracy Mcgaha
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan T Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are pivotal stromal components that shape the tumor microenvironment through their remarkable heterogeneity and dynamic functions. This review comprehensively examines the biology of CAFs, starting with their diverse cellular origins and induction mechanisms via key signaling pathways. We highlight the spatial‐temporal heterogeneity of CAF subsets and propose an updated classification system that encompasses eight functionally distinct subtypes based on recent single‐cell transcriptomic studies. The tumor‐promoting roles of CAFs are explored in depth, including their contributions to tumor behavior through novel mechanisms such as neutrophil extracellular traps formation, and multidrug resistance via metabolic reprogramming. A major focus is placed on CAF‐mediated immunotherapy resistance through immune checkpoint regulation, recruitment of immunosuppressive cells, and metabolite secretion. This review also highlights key technologies that are advancing CAF research, including patient‐derived 3D models, artificial intelligence‐enhanced spatial multiomics, clustered regularly interspaced short palindromic repeats (CRISPR)‐engineered mouse systems, and machine learning approaches for analyzing CAF heterogeneity and function. Finally, we evaluate therapeutic strategies targeting CAF activation, secreted factors, and immune crosstalk, while critically analyzing ongoing clinical trials. Overall, this review not only clarifies the biology of CAFs but also provides a translational framework for developing next‐generation stromal‐targeted therapies to overcome treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1dae86be5576422570b1acdcb11b97738bdec4b" target='_blank'>
              Cancer‐Associated Fibroblasts: Origin, Classification, Tumorigenicity, and Targeting for Cancer Therapy
              </a>
            </td>
          <td>
            Chang Fan, Wenlong Zhu, Yuan Chen, Wanwan Zhu, Jin Ding
          </td>
          <td>2025-10-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b5dde4ed4b216cb7cf78338e646166d480ca06" target='_blank'>
              Spatial proteomics of ovarian cancer precursors delineates early disease changes and drug targets.
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, W. Schmitt, Eliane T. Taube, Fabian Coscia
          </td>
          <td>2025-11-13</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cells to transition to an alternative phenotype as a means for adaptation, is an increasingly recognized mechanism of tumor evolution and a driver of resistance to anticancer therapies. The most extensively described clinical settings impacted by such molecular phenomena include neuroendocrine transformation in androgen receptor-dependent prostate adenocarcinoma, and adenocarcinoma-to-neuroendocrine and adenocarcinoma-to-squamous transdifferentiation in epidermal growth factor receptor-driven lung adenocarcinoma, affecting 10%–20% of patients treated with targeted therapy. Recent analyses of human tumor samples and in vivo models of histological transformation have led to insights into the biology of lineage plasticity, including biomarkers predictive of high risk of transformation. However, no clinically available therapies aimed to prevent or revert plasticity are currently available. In the present review, we will provide a biological and therapeutic overview of the current understanding of common and divergent molecular drivers of neuroendocrine and squamous transdifferentiation in tumors from different origins, including descriptive analysis of previously known and recently described molecular events associated with histological transformation, and propose evidence-based alternative models of transdifferentiation. A clear definition of the commonalities and differences of transforming tumors in different organs and to different histological fates will be important to translate molecular findings to the clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell Research</td>
          <td>2</td>
          <td>18</td>
        </tr>

        <tr id="Squamous cell cancers are responsible for 1 in 5 cancer deaths and survival improvements lag behind those seen in adenocarcinomas. This disparity is in large part due to the limited impact of immunotherapy due to therapeutic resistance, where less than ten percent of patients respond in early stage squamous cell carcinomas of the Head and Neck. Mechanisms that govern intrinsic resistance remain poorly understood and likely arise during the premalignant or dysplastic state. Here, we generated a dataset of murine and human oral squamous epithelia spanning the earliest premalignant stages through invasive carcinoma. Integrative analysis of single-cell and spatial transcriptomics data across the dysplasia to carcinoma continuum reveals early and sustained shifts in epithelial transcriptomes. Spatially informed cell-cell interaction analysis reveals dysplasia-specific upregulation of wound healing and immune remodeling programs in severe dysplasia and invasive carcinoma in a human sample of oral dysplasia. The presence of these transcriptomic programs within early dysplasia may account for the aggressive clinical presentations of squamous cancers including intrinsic therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A. Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a particularly aggressive brain tumor which is characterized by a low overall survival due to rapid formation of neuron-tumor-networks. The neural epigenetic subtype was linked to increased tumor-neural connectivity but the mechanisms underlying this integration remain poorly understood. Here, we aim to investigate the impact of the microenvironmental ecosystem on neuron-tumor synaptogenesis leveraging a human cortical slice model and single cell spatially resolved transcriptomic analysis.



 A human cortical slice model was employed to visualize and assess tumor-neuronal interactions using advanced retrograde tracing techniques in human derived cell lines (n = 12). This system was based on an EnvA-pseudotyped G-protein-deleted rabies virus. Spatial transcriptomics was conducted using multiplexed error-robust fluorescence in situ hybridization analysis (MERFISH). Additionally, we conducted immunohistochemical staining and confocal microscopy.



 Spatial transcriptomic evaluation of neural high and low tumors revealed distinct microenvironmental ecosystems within the infiltrating rim of the tumors. Neural-high tumors exhibited a higher presence of synaptogenesis-inducing inflammatory myeloid cells driving neuronal synaptogenesis. Retrograde tracing of neural high vs low cell lines confirmed increased synaptic density and neuronal coupling in the neural high phenotype, which were associated with higher invasiveness, supported by stRNA-seq, which identified genes linked to synaptogenesis and connectivity.



 Neural-high GB shows enhanced neuronal connectivity and invasiveness, contributing to accelerated tumor progression, which underlines the prognostic value of epigenetic tumor profiling and opens ways for novel therapeutic approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab32a7b8c0ed1e91d6c055b2bda26b2946992d4" target='_blank'>
              P02.15.B GLIOBLASTOMA IN THE CONNECTOME: NEURAL PHENOTYPES PREDICT TUMOR INTEGRATION AND CONNECTIVITY
              </a>
            </td>
          <td>
            L. Menzl, G. Villa, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74e30f87b2d3e252ed12332d86f09f04feb4c91" target='_blank'>
              CTDP: Identifying cell types associated with disease phenotypes using scRNA-seq data
              </a>
            </td>
          <td>
            Chonghui Liu, Murong Zhou, Zhidong Wang, Chen Yang, Yan Zhang, Zhongjun Jiang, Yu-Hang Yin, Zeyu Luo, Tianjiao Zhang, Guohua Wang, Lei Yuan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cervical biofluids collected during routine Pap smears contain a mixture of epithelial and immune cells that reflect cervical health and disease. This study demonstrates proof-of-concept that multicolor flow cytometry can be applied to these samples to identify and characterize cell populations in normal cervical samples as well as in different stages of cervical cancer. Our analysis identified major cell populations, including different types of epithelial cells and immune cells, and revealed that their composition within the samples changes with disease stage. We observed variability in both cellular composition and marker expression, highlighting the need for additional markers and larger patient cohorts. Nonetheless, this minimally invasive approach could potentially be used to monitor tumor cells, immune responses, and therapeutic outcomes over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/565b70c9fdb10e2ce7ca145493da61430604f525" target='_blank'>
              Noninvasive Cell Population Profiling of Normal and Dysplastic Cervical Biofluids by Multicolor Flow Cytometry as a Promising Tool for Companion Diagnostics
              </a>
            </td>
          <td>
            Christoph Berger, Wolf Dietrich, Manuela Richter, Florian Kellner, Christian Kühne, Katharina Strasser
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The pancreas plays a central role in major human diseases, yet our understanding of its cellular diversity and plasticity remains incomplete. Here, we present a single-cell multiomics atlas of the human pancreas, profiling over four million cells and nuclei from 57 donors across fetal development, adult homeostasis, and type 2 diabetes (T2D). Integrating sc/snRNA-seq, snATAC-seq, VASA-seq, spatial transcriptomics (Xenium), and multiplexed proteomics (CODEX), we resolve gene expression, chromatin accessibility, and spatial organization at high resolution. We identify transcriptionally plastic centroacinar-like cells (pCACs) in adults with fetal-like features, delineate endocrine and exocrine lineage trajectories during development, and uncover HNF1A-defined beta cell epigenetic states. In T2D, we observe shifts in beta cell subtypes and altered regulatory programs. Glucose perturbation of healthy islets reveals cell-type-specific adaptation and stress responses. This atlas provides a foundational framework to understand pancreas biology and the role of cellular plasticity in regeneration and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dff204518163415cbe0e6a53eaee0e00ceaf21d" target='_blank'>
              Latent plasticity of the human pancreas across development, health, and disease
              </a>
            </td>
          <td>
            E. Mereu, Diego Balboa, J. Liebig, Aitor Gonzalez-Herrero, Anna Martinez-Casals, Mariya Mardamshina, Fanny Mollandin, Felix Schicktanz, Luca Tosti, Valerie Vandenbempt, D. Avrahami, Edgar Bernardo, Frida Björklund, R. Chua, M. Engelse, J. García-Hurtado, Nathalie Groen, M. Hanegraaf, Pablo Iañez, K. Jechow, Björn Konukiewitz, Christian Lawerenz, Domenica Marchese, Mauro J. Muraro, Silvia Pellegrini, V. Sordi, Alexander Sudy, Ulrike Taron, F. Ten, Timo Trefzer, Sven Twardziok, Maarten van Agen, F. Carlotti, Eelco J. P. de Koning, Jorge Ferrer, Benjamin Glaser, Holger Heyn, Emma Lundberg, Lorenzo Piemonti, Katja Steiger, Alexander van Oudenaarden, Wilko Weichert, C. Conrad, R. Eils
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fb4029b669b488e3f7a2bf959803989f3d7dae" target='_blank'>
              Single-Cell RNA and ATAC Sequencing Reveal Cellular Heterogeneity and Chromatin Accessibility Dynamics in Young Chinese Breast Cancer Patients across Pre- and Post-Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Xue Han, Bin Wang, Wenting Xiang, Fan Lin, Bo Sun, Beibei Yang, Zhihan Ruan, Shasha Liu, Shuo Li, Hong Liu, Jian Liu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Seminoma is the most common solid malignant tumor of the testis in young males, significantly impacting fertility. Approximately 20% of seminomas metastasize, markedly increasing recurrence risk and compromising quality of life. To investigate the poorly understood mechanisms driving seminoma metastasis, we performed a comprehensive analysis by integrating single-cell RNA sequencing, TCGA data mining, and cell biology assays. We identified significant intratumoral heterogeneity. A subset of tumor cells expressing DPPA4 and PSMA7 showed high stemness, enhanced self-renewal, and association with metastasis. Knockdown of these genes reduced sphere formation, tumor migration and proliferation in Tcam-2 and NCCIT cells. Conversely, tumor cells overexpressing PAGE5 and SAT1 exhibited reduced stemness, migratory capacity and proliferation. In the immune microenvironment, we identified IFNG+ T cells, which recruit and activate other antitumor immune cells. These cells secrete IFN-γ, which promotes tumor differentiation, reduces stemness, and mitigates tumor aggressiveness. Based on these findings, we further developed a molecular panel to aid in the identification of seminomas with a higher risk of metastasis. In conclusion, our study identifies unique molecular signatures that facilitate risk stratification based on metastatic potential, providing valuable insights for improving precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a200ee992d63f7f5d78cc885c7a3fb857907df52" target='_blank'>
              Single-cell analysis unravels divergent gene signatures shaping seminoma stemness and metastasis
              </a>
            </td>
          <td>
            Zhouliang Bian, Biying Chen, Jiahui Guo, Jiamin Zhang, Guangye Du, Shufang He, Haihua Yuan, Yue Zhou, Bin Jiang, Daliu Min, Dingwei Ye, Hengchuan Su, Yanjie Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) poses a great challenge for therapeutics development due to its heterogeneity and the lack of immunocompetent models that faithfully recapitulate human GBM. We sought to generate and characterize syngeneic mouse models for major GBM subtypes.



 Postnatal electroporation using the PiggyBac transposon/transposase for overexpression (OE) and CRISPR/Cas9 for knockout (KO) was used to generate models with common GBM alterations including EGFRvIII-OE/Cdkn2a-KO/Pten-KO, Nf1-KO/Pten-KO/p53-KO, and RasV12-OE/Cdk4-OE/p53-KO. Model characterization was done by HE staining, immunofluorescence staining, RNA sequencing and flow cytometry.



 All models displayed histopathological and molecular features of GBM such as pseudopalisiding necrosis, microvascular proliferation, and cellular atypia, as well as Gfap and Nestin expression. On the transcriptomic level, Nf1/Pten/p53 tumors showed enrichment for the human mesenchymal signature, while RasV12/Cdk4/p53 and EGFRvIII/Cdkn2a/Pten tumors resembled the classical and proneural signatures. Each tumor was comprised of four cellular states, i.e., astrocytic, mesenchymal, OPC-like, and NPC-like, suggestive of intratumoral heterogeneity. On examining the microenvironment, EGFRvIII/Cdkn2a/Pten tumors had a higher relative abundance of T cells (48.6% vs 6.3%, p=0.0084, 48.6% vs 8.8%, p=0.0376,), NK cells, (1.2% vs 0.4%, p=0.1425, 1.2% vs 0.19%, p=0.0315) and dendritic cells (7.5% vs 0.4%, p=0.0272) than the Nf1/Pten/p53 and RasV12/Cdk4/p53 models. RasV12/Cdk4/p53 tumors had greater infiltration of monocyte-derived macrophages than Nf1/Pten/p53 (35.9% vs 4.18%, p=0.0012) and EGFRvIII/Cdkn2a/Pten (35.9% vs 10.4%, p=0.0620) tumors as well as an elevated CD4+/CD8+ T cell ratio. In the Nf1/Pten/p53 model, microglia were the most abundant cells within the CD45+ compartment, surpassing levels observed in EGFRvIII/Cdkn2a/Pten (60.2% vs 4.6%, p=0.0548) and RasV12/Cdk4/p53 (60.2%vs7.5%, p=0.0564) tumors.



 We developed an array of syngeneic mouse models of GBM that recapitulate human inter- and intra- tumoral heterogeneity. Future studies will define the functional states of immune cells in the microenvironment and evaluate the impact of immune cell composition on the response to immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9fb466bb8d2e828409bfc681044d64171c2f26" target='_blank'>
              TMOD-40. Genetic drivers shape tumor phenotype and composition of the tumor microenvironment in syngeneic mouse models of glioblastoma
              </a>
            </td>
          <td>
            J. Pyczek, Shannon Snelling, Xueqing Lun, Bo Young Ahn, Heewon Seo, Miranda Yu, Daniela Quail, A. Morrissy, Jennifer A Chan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an incurable and aggressive brain cancer marked by profound intra-tumoral heterogeneity. Malignant cells exist in four core transcriptional states: stem-like (OPC-like and NPC-like) and more differentiated astrocytic (AC-like) or mesenchymal (MES-like) states. Using spatial transcriptomics, we previously identified a layered tumor structure of these states, with hypoxia emerging as an organizing driver, implicating oxygen availability as a central factor in cell state dynamics. However, how these cellular states interact with each other and with the tumor microenvironment remains incompletely understood.



 To uncover state-specific vulnerabilities for reducing GBM cell state diversity, we conducted a small-molecule screen in gliomasphere models using over 1,600 cysteine-reactive covalent inhibitors, assessing their effects on cell states via RNA-seq. We then employed the Sonar metabolic reporter to visualize NAD+ and NADH levels in gliomaspheres. Through experiments with orthotopic xenografts in mice and co-culture with human cortical organoids, we characterized distinct redox profiles across GBM cell states by single-cell and bulk RNA-seq. Spatial single-cell transcriptomics and mass spectrometry imaging (MSI) metabolomics on adjacent sections of frozen patient samples were used to identify preferential metabolic programs in specific cell states.



 Our small-molecule screen identified compounds that promote differentiation and are enriched for NRF2 activation, a master regulator of the antioxidant response. Spatial single-cell transcriptomics and mass spectrometry metabolomics experiments confirmed that MES-like cells exhibit a heightened glycolytic signature, whereas stem-like states preferentially depend on oxidative phosphorylation (OXPHOS) for their energetic needs. Finally, pharmacologic perturbations revealed that mitochondrial and glycolysis inhibition selectively deplete different cell state populations in gliomaspheres.



 Our findings provide deeper insight into the metabolic underpinnings of GBM cell states and support the rationale for combinatorial therapies aimed at reducing intratumoral heterogeneity that exploit their divergent dependencies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25475481794c097c1d8706111cced1dc0e56d4b" target='_blank'>
              TMET-30. Uncovering the metabolic programs underlying malignant cell state heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Alexander Jucht, Maolin Ge, Rony Chanoch-Myers, Florian Ruiz, Gerard Baquer, Alissa C. Greenwald, Min Yang, Toshiro Hara, Yilin Fan, Channing Pooley, D. Gerovasilis, Charles Couturier, Nathalie Y R Agar, I. Tirosh, L. Bar-Peled, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Low-grade gliomas (LGGs) exhibit low overall mutational burden and an immunosuppressive microenvironment, contributing to immunotherapy resistance. Intratumoral heterogeneity (ITH) further complicates the targeting of tumor-specific antigens (TSAs), yet its influence on the antigenic and immune landscapes in LGG remains understudied, particularly due to reliance on single-biopsy analyses.



 We performed exome and transcriptome sequencing on 70 spatially mapped biopsies from seven IDH-mutant Grade II astrocytoma patients. Tumor purity, immune deconvolution, and neoantigen prediction were conducted to assess regional immune variation and antigenic burden. In one patient (P375), neoantigen-specific CD8+ T cells were isolated using barcoded peptide-HLA multimers. Reactive T-cell receptor (TCR) clonotypes were identified via single-cell TCR sequencing and functionally validated in Jurkat76 and CD8+ T cells.



 Spatial profiling of whole tumors revealed most mutations were biopsy-specific, with a steep drop in mutations shared across multiple regions. RNA-seq–based unsupervised clustering with xCell and DESeq2 revealed two distinct immune microenvironments: immune-hot regions with elevated immune infiltration, and immune-cold regions with minimal immune presence. Notably, mutation-derived n-mer peptides from immune-hot regions showed enhanced predicted immunogenicity relative to those from immune-cold areas. In one patient (P375), neoantigen-specific CD8⁺ TCR clonotypes were identified targeting PRMT5 mutations found in immune-hot regions. A mutant PRMT5-specific TCR demonstrated robust, antigen-dependent activation and dose-dependent cytotoxicity in vitro. This TCR showed high specificity toward the mutant peptide with minimal cross-reactivity to the wild-type sequence. Furthermore, engineered CD8⁺ T cells effectively killed glioma cells endogenously expressing the full-length mutant PRMT5, supporting its candidacy as a viable immunotherapeutic target.



 Our findings reveal significant 3D spatial and immunologic heterogeneity in low-grade gliomas and demonstrate that neoantigens such as mutant PRMT5 can be selectively targeted by highly specific, cytotoxic TCR-engineered CD8⁺ T cells. These results support the potential of personalized TCR-based immunotherapy for low-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa3c48068f63d55f21f844232ecea3f66c8ffb8" target='_blank'>
              EXTH-52. Spatially Resolved Antigenic Landscapes In Low-grade Glioma Reveal Targets For Personalized Immunotherapy
              </a>
            </td>
          <td>
            Darwin W. Kwok, Michael Zhang, Cliff Wang, Maggie Colton Cove, Chibo Hong, Nicholas O. Stevers, Benjamin Lerman, S. Lakshmanachetty, Tyler Borrman, Zheng Pan, Benjamin Yuen, Songming Peng, Diana Nguyen, Michael Martin, S. Hilz, Joanna Phillips, Anny Shai, N. Bush, S. Hervey-Jumper, Michael McDermott, S. Mandl, H. Okada, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72005e917b50c2e5c4b78fb42f30196a93fe8580" target='_blank'>
              Transcriptional Profiling of Commonly Used Liver Cancer Cell Lines Reveals Disease-Specific Modeling Potential and Authentication Concerns
              </a>
            </td>
          <td>
            Nima Sarfaraz, Gail R. Rosen, Trevor Winkler, Risha Patel, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J. Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) arises from precursor lesions, the most common of which are Pancreatic Intraepithelial Neoplasia (PanIN). PanINs are microscopic and cannot be detected in live individuals; as such, PanIN characterization has occurred prevalently in the context of cancer-bearing pancreata (adjacent normal) rather than in healthy tissue. Recently, through a unique collaboration with Gift of Life Michigan, we showed that PanIN prevalence is over 60% across age groups in donor healthy tissues, with high prevalence even in young individuals. Given PanIN prevalence and the rarity of pancreatic cancer, we hypothesized that the majority of PanIN is not fated to progress to malignancy. We postulated that a combination of neoplastic cell-intrinsic factors and components of the microenvironment controls progression to malignancy. We utilized spatial transcriptomics using the Visium platform (n = 14) and scRNASeq (n = 45) to dissect gene expression profiles and characterize spatial domains in donor healthy, adjacent-normal, and tumor tissues. We developed a spatial domain identification workflow that integrates spatial transcriptomics with the scRNASeq atlas, leveraging cell-type deconvolution, spatially informed clustering and pathology annotation. This approach allowed us to study the co-evolution of epithelial and stromal cells over the progression of pancreatic cancer. We were able to delineate distinct epithelial spatial domains—including Acinar, Ductal, and PanIN structures—in both donor and tumor tissues, as well as tumor-specific regions such as glandular tumor and poorly differentiated tumor. Additionally, we identified key stromal niches, including immune-rich regions such as tertiary lymphoid structures (TLS), plasma cell clusters, and macrophage-enriched zones, alongside non-immune fibrotic domains. We began by analyzing the epithelial domains and observed that while all acinar and ductal domains formed transcriptionally distinct clusters, neoplastic cells formed a continuum from sporadic healthy donor-associated PanINs to well- and poorly-differentiated tumors. We identified key genes that exhibit expression changes throughout the progression trajectory. Secondly, we analyzed characteristics of the stroma surrounding sporadic lesions, well-differentiated, and poorly-differentiated tumors. We observed key differences in fibroblasts between healthy donor-PanIN and tumorigenic regions, as fibroblasts surrounding donor-associated PanIN lacked known cancer-associated fibroblast (CAF) markers such as Alpha-Smooth Muscle Actin (α-SMA). We also observed distinct differences in immune cells, where there were more abundant plasma and CD8+ T-cells surrounding donor-associated PanIN, while tumorigenic regions showed macrophage enrichment. We then performed immunostaining and confocal imaging to validate the defined markers and specific cell populations. Our work defines the PanIN microenvironment as distinct from that of PDAC and maps the co-evolution of cancer cells and stroma over the course of carcinogenesis.



 Ahmed M. Elhossiny, Jude Okoye, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, Elana J. Fertig, Eileen S. Carpetner, Timothy Frankel, Marina Pasca di Magliano. Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A052.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f4f9e15e43361f1e76f4bf1e3cc3e1d04354d0" target='_blank'>
              Abstract A052: Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, J. Okoye, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, E. Fertig, Eileen S. Carpetner, Timothy L. Frankel, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer metastasis, the process by which tumor cells spread from the primary site to other organs, is one of the leading causes of death in cancer patients. However, due to its complexity and unpredictability, the study of cancer metastasis remains a significant challenge in medicine. In recent years, with the advancement of high-throughput sequencing technologies and single-cell transcriptomics, we could gain a deeper understanding of the molecular mechanisms and cellular heterogeneity underlying cancer metastasis. There is an accumulating volume of publicly available cancer metastasis research data. Nevertheless, these resources lack proper organization, hindering systematic analysis. In this study, we developed an integrated resource named Panmim (http://www.gdwk-bioinfo.com/pan_metastasis/home) to investigate the immune microenvironment of metastatic tumors. The database currently encompasses 90 single-cell RNA-seq datasets from metastatic cancers of diverse origins, encapsulating 14 distinct metastatic sites and 36 primary cancer types. Panmim facilitates a cellular-level comparison of similarities and differences between primary and metastatic cancers, encompassing pathway analysis, alterations in cellular metabolic pathways, cellular distribution preferences, and various aspects of intercellular communication. Panmim presents the analysis results through an intuitive interactive graphical interface, enabling users to explore and understand the complex biological phenomena during cancer metastasis more conveniently. This resource will become a valuable tool for biologists and bioinformaticians to study the mechanisms of cancer metastasis, providing important data support and scientific basis for the optimization of cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9f11b0d110baaaf542f6916b80b13cbf907d57" target='_blank'>
              Panmim: a resource of pan-cancer metastasis immune microenvironment
              </a>
            </td>
          <td>
            Xuelu Zhang, Simeng Hu, Huanjing Hu, Yongkun Li, Pengyin Nie, Xichen Hu, Ruidong Xue, Xiao Xiang, Lu Zheng
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="


 Immune suppression in glioma is marked by systemic reprogramming of bone marrow-derived cells (BDMCs). Tumor-derived exosomes (TEX) exhibit the requisite biodistribution, genomic contents, and specificity for short and long-range reprogramming of BDMCs. However, in vivo studies have emerged as the major obstacle for discovery of TEX cell-specific glioma immunotherapy. We report a syngeneic murine glioma model capable of tracking endogenously released TEX in real-time.



 We used the RCAS retroviral system develop IDH-mutant and wild-type glioma mouse models with concurrent induction of spontaneous glioma and reporting of GFP-tagged TEX. Tumor and blood samples were analyzed using spectral flow cytometry, multi-plex immunofluorescence and scRNA sequencing to characterize specific TEX-targeted immune cells and their corresponding spatial topography.



 Our multimodal analyses reveal unique genotype-specificity of TEX-targeted immune ontogenies. In IDHm gliomas, myeloid cells were ubiquitously endowed with TEX and exhibited central topography within the tumor. Conversely, IDHwt TEX-containing myeloid cells were over-represented along the tumor periphery. These cells exhibited a dichotomous and polarized pattern of TEX uptake with dorsal (CD68/F4/80) and ventral (CD11b/F480) localization. Over 90% of CD4 T cells engulfed TEX and were highly enriched within the medial leading edge of the tumor. CD8 T cells contained the fewest proportion of TEX and were restrained to the meninges and peri-tumoral white matter. Remarkably, TEX were identified in non-neoplastic contralateral hemispheres and the meninges of tumor-bearing mice. Finally, flow analyses of blood samples detected an increase of TEX in circulating myeloid cells of IDHm tumor-bearing mice, emerging at 2 weeks and peaking at 6 weeks after tumor injection.



 We have developed a novel mouse model that allows unprecedented endogenous tracking of glioma tumor exosomes. Tumor exosomes engage peripheral macrophages and enable recruitment of these cells into the local tumor microenvironment. Our data supports engineering exosomes for glioma immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35dbe1308ea00e1781037da15c0c198dd059c70a" target='_blank'>
              TMOD-05. Glioma Exosomes Mediate Systemic Immune Suppression Through Myeloid Cell Targeting and Recruitment
              </a>
            </td>
          <td>
            Emily Xu, Jonathan Patterson, Jonathan H Sussman, Angelo Angione, Ebrar Akca, N. Amankulor
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="One of the main characteristics of solid tumors, such as melanoma, is an acidic tumor microenvironment. Due to dysregulation of the cancer cell metabolism and an increased production of acidic metabolites, the tumor acidifies its microenvironment. We hypothesize that this has a strong impact on tumor heterogeneity and the formation of phenotypic subpopulations. Cell culture experiments are usually carried out at a physiological pH of 7.4. Here, we show that long-time acidosis results in the formation of a senescent subpopulation in melanoma cells. Interestingly, after reintroduction to physiological pH, these cells lose their senescence-associated attributes. We isolated this subpopulation using β-galactosidase-dependent C12FDG staining and FACS. Live cell imaging, microscopy, and RNA sequencing analysis revealed that these apparent senescent cells show an increased migratory activity. With this study, we demonstrate that the acidic tumor microenvironment drives tumor plasticity in melanoma and results in the formation of a small subpopulation, which promotes metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a15918dc412942e4628009e724e226c928f4b6e" target='_blank'>
              Acidic melanoma microenvironment selects for a senescence-like but also migratory-active subpopulation driving metastatic disease
              </a>
            </td>
          <td>
            Chafia Chiheb, Stefan Fischer, Zubeir El Ahmad, Ingmar Henz, Ines Böhme-Schäfer, Melanie Kappelmann‐Fenzl, A. Bosserhoff
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastomas function as intricate cellular networks that extend into the surrounding brain tissue enabling long distance communication. This malignant connectivity spans from the tumor core to remote infiltration zones, supporting the concept of glioblastoma as a whole-brain disease. With growing ethical considerations in biomedical research and the limitations inherent in cross-species comparisons between animal studies and human glioblastoma models, human tumor platforms remain constrained in their ability to replicate the full infiltration spectrum from the tumor core to single-cell dispersion. Here, we present a three-dimensional, fully human ex-vivo glioblastoma model (,,Core2Edge”) that replicates this extensive infiltration range while preserving the intratumoral heterogeneity of the original tumor. This model involves implanting fluorescently labeled human glioblastoma organoids (GBOs) into organotypic human brain slices, maintaining the genetic integrity and cytoarchitecture of both brain and tumor.



 The Core2Edge model involves a comprehensive workflow completed over 7-12 days. Key steps include brain slice preparation (~4-6 hours, depending on quantity), one day for initial culture before GBO staining and transplantation, a variable culture period (up to ten days), and fixation with different techniques (~8 hours). Following model preparation, downstream analyses such as imaging, sequencing, and proteomics were performed.



 Multiplex immunofluorescence confirmed cell composition consistent with histological staining. Spatial transcriptomics enabled detailed glioblastoma cell subtyping based on Neftel states. Advanced imaging, combining tissue expansion and light-sheet fluorescence microscopy, facilitated super-resolution, three-dimensional visualization of the GBO-brain slice system. This approach allows for the study of initial infiltration steps, in-depth analysis of the invasive front, exploration of cell-cell interactions between tumor cells and the tumor microenvironment, and offers a platform for drug screening and testing, reducing the need for animal models.



 The,,Core2Edge“model represents an innovative ex-vivo platform replicating glioblastoma infiltration and heterogeneity. By preserving genetic integrity and cytoarchitecture, this fully human model bridges critical gaps in glioblastoma research, offering unparalleled insights into tumor behavior and potential therapeutic interventions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7215ed71357bb240e167b383e1fb29c1a6a3d4" target='_blank'>
              P01.24.B CORE2EDGE: A HUMAN GLIOBLASTOMA ORGANOID AND BRAIN SLICE CO-CULTURE MODEL CAPTURING TUMOR INFILTRATION AND TRANSCRIPTIONAL HETEROGENEITY FROM CORE TO SINGLE-CELL DISPERSION
              </a>
            </td>
          <td>
            M. Schneider, A. Potthoff, A. Melhem, B. Pregler, J. Rodriguez-Gatica, J. Thomas, L. Friker, M. Westhoff, V. Ravi, D. Heiland, A. Schlitzer, T. Pietsch, M. Hölzel, A. Waha, H. Vatter, M. K. Schwarz, U. Kubitscheck, U. Herrlinger
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The bone marrow microenvironment (BMME) is essential for hematopoiesis and immunity, yet spatiotemporal single-cell analysis during leukemogenesis remains challenging. We characterized the BMME in femurs from wild-type and chronic myeloid leukemia (CML) mice at 7, 14 and 21 days post-induction by highly multiplexed and 3D microscopy. Using a 54-marker CODEX panel, we profiled 2,033,725 cells in 55 tissue regions of interest and identified 41 cell types through unsupervised clustering and supervised annotation. During leukemic progression, we observed an expansion of myeloid and progenitor cell populations, increased PD-L1+ leukemic cells, the upregulation of PD-1 on CD4+ and CD8+ T cells, and a profound loss of B cells, plasma cells and bone cells. Advanced CML exhibited a striking expansion of immature, pericyte-deficient vasculature that disrupted vascular niches and impaired hematopoietic stem/progenitor cell positioning. Spatial mapping revealed leukemia-specific cellular neighborhoods enriched in PD-1+CD8+ T cells, suggesting localized immune cell exhaustion. Early-stage CML showed increaseds between plasmacytoid dendritic cells and megakaryocytes, whereas advanced CML featured heightened megakaryocyte emperipolesis of non-leukemic granulocytes. Megakaryocytes were morphologically irregular in CML mice and BM trephine biopsies from CML patients. Laser-capture microdissected megakaryocytes from newly diagnosed CML patients had reduced expression of cytoskeleton genes, which was reversed in advanced cases treated with tyrosine kinase inhibitors. 3D imaging revealed vascular disorganization and depleted megakaryocytes in the diaphysis, underscoring region-specific pathology. Together, this study provides a spatiotemporal single-cell atlas of the BMME during leukemic progression, showing how leukemic cells reprogram the niche to support their expansion and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9f9ea320832c546ba7a6ff131bcd995c1006c9" target='_blank'>
              High-Dimensional Spatiotemporal Single-Cell and 3D Atlas of the Bone Marrow Microenvironment during Leukemic Progression
              </a>
            </td>
          <td>
            Lanzhu Li, Isabelle Rottmann, Geoff Ivison, Huan Wei, Jan C. Schroeder, G. Dunkel, Y. Goltsev, Garry P. Nolan, Aaron T. Mayer, Borhan R. Saeed, Bettina Weigelin, Christian M. Schürch
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite significant progress, the treatment of estrogen receptorpositive (ER+) breast cancer remains clinically challenging due to reversible drug resistance and immune evasion. Drug resistance often arises as cells undergo a dynamic epithelial-to-mesenchymal transition (EMT), while elevated PD-L1 levels contribute to immune escape. While these phenotypic features can variably co-occur, the impact of co-occurrence on the availability of synergistic treatment strategies remains unknown. To investigate their interplay, we constructed an ER-EMT-PD-L1 gene regulatory network and simulated these networks as coupled ordinary differential equations with biologically informed parameters, to generate steady-state expression profiles. Our study revealed that the relevant overarching network generated antagonistic epithelial and mesenchymal modules, capable of producing monostable, bistable, and tristable dynamics. We further examined the link between phenotypes and immune evasion by quantifying average PD-L1 expression, and found that epithelial-sensitive states consistently exhibited low PD-L1. In contrast, hybrid- and mesenchymal-resistant states were associated with high PD-L1, highlighting a strong coupling between EMT, resistance, and immune evasion. Extending on these network-level insights, we further used a spatially explicit agent-based model seeded with GRN-derived phenotypes to probe tumor behavior under therapeutic pressure. Simulations revealed that tumor escape required co-occurrence of therapy resistance, motility, and immune suppression, with plasticity and multistability further promoting adaptive persistence. Lastly, we identified combination therapies predicted to constrain malignant diversification and enhanced immune accessibility. Taken together, our modeling work links regulatory dynamics with tumor-level adaptation and underscores potential strategies to therapeutically reprogram cell states toward sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1217f04795a563df5b0ffd6d061c6a9c9b363d93" target='_blank'>
              Lineage of origin-specific developmental programs drive the behaviors of malignant cells in an avian embryo model of human Medulloblastoma subgroups
              </a>
            </td>
          <td>
            Marion Mallet, Florian Martin, Karine Thoinet, Caroline Imbert, Moein Sarhadi, Jérémy Ganofsky, Jacob Torrejon Diaz, T. Fenouil, C. Delloye-Bourgeois, Julien Falk, Cécile Faure-Conter, Ludovic Telley, D. Meyronet, O. Ayrault, Valérie Castellani, S. Tauszig-Delamasure
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chemotherapy resistance remains a major clinical challenge in ovarian cancer treatment, limiting therapeutic efficacy and patient survival. Understanding the molecular mechanisms underlying chemoresistance at the single-cell level is crucial for developing targeted therapeutic strategies. We performed comprehensive single-cell RNA sequencing analysis on primary and chemoresistant ovarian cancer samples to characterize cellular heterogeneity and identify resistance-associated molecular signatures. Cell type identification, differential gene expression analysis, functional enrichment analysis, and immune microenvironment characterization were conducted to elucidate chemoresistance mechanisms. Computational predictions were validated by RT-qPCR analysis of four key resistance genes (BIRC5, ABCB1A, ABCG2, BCL2) in SKOV3 and SKOV3-CDDP cell lines. Our analysis identified nine distinct cell populations within the ovarian cancer microenvironment, revealing extensive transcriptional reprogramming in chemoresistant tumors. Key chemoresistance genes including Abcb1a, Abcg2, Bcl2, Birc5, Nek2, Inhba, Irs1, and Prc1 showed differential expression patterns across cell types. Epithelial cells and proliferating cells exhibited elevated expression of multiple resistance markers, particularly Birc5. Functional analysis revealed enrichment of ribosome biogenesis, protein synthesis machinery, and amoeboid-type cell migration pathways in resistant cells. Macrophage polarization states were significantly altered, and immune checkpoint molecules showed coordinated upregulation, indicating comprehensive immune microenvironment reprogramming. RT-PCR validation confirmed single-cell predictions with significant upregulation in resistant cells: BIRC5 (4.2-fold, p < 0.001), ABCB1A (3.8-fold, p < 0.001), ABCG2 (2.9-fold, p < 0.01), and BCL2 (2.1-fold, p < 0.05). This study provides a comprehensive single-cell atlas of chemoresistance in ovarian cancer, revealing cell type-specific resistance mechanisms and immune evasion strategies. The identified molecular signatures offer potential therapeutic targets for overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f229e22beb03e0ca3194ad4df8f24784e9b8331" target='_blank'>
              Single cell transcriptomic analysis reveals molecular mechanisms of chemoresistance and immune microenvironment remodeling in ovarian cancer
              </a>
            </td>
          <td>
            Xia Hu, Caifang Du, Dongjie Du, Yonghua Zhang, Haijian Gao, Minxian Tao
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Highlights What are the main findings? Poor prognostic ligand–receptor pairs are localized in specific regions within glioblastoma tissues. What is the implication of the main finding? Our findings provide new insights into pathological cell–cell interactions and treatment for glioblastoma. Abstract Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Meningiomas are the most common primary intracranial tumors, yet their cellular diversity, tumor-stroma interactions, and the relationship between neoplastic and normal meningeal cells remain incompletely understood. While recent studies have provided valuable single-cell insights into specific tumor contexts, a unified atlas encompassing both meningioma and healthy meningeal tissue across patients and studies is lacking.



 We constructed MeningioMap, a comprehensive single-cell atlas by harmonizing three publicly available datasets with in-house single-cell RNA sequencing (scRNA-seq) of surgically resected meningiomas and matched adjacent meningeal tissue. The dataset encompasses over 300,000 high-quality cells from 40+ patients, spanning diverse tumor grades, anatomical locations, histopathological subtypes, and genders. Standardized preprocessing, robust batch correction, and integrative clustering pipelines were employed. Preliminary spatial transcriptomic analyses further validated key tumor-stromal interactions predicted from scRNA-seq profile.



 Meningioma delineates a high-resolution taxonomy of malignant and non-malignant cell states within the meningioma ecosystem. We identified distinct malignant subpopulations characterized by specific copy number alterations, transcriptional programs, and clinical associations with tumor grade. Comparative analysis with matched meninges uncovered conserved meningeal lineages and unveiled tumor-specific reprogramming events. Spatial transcriptomics confirmed localized immune remodeling and stromal activation signatures at the tumor-meninges interface. Notably, we identified a novel transitional malignant-stromal phenotype enriched in invasive tumor fronts, suggesting a potential driver of recurrence and therapeutic resistance.



 MeningioMap represents the most extensive single-cell resource for meningioma and meningeal research to date. Beyond offering a foundational reference for tumor heterogeneity and cellular dynamics, it provides critical insights into tumor-microenvironment crosstalk, informing future strategies for precision diagnostics, risk stratification, and targeted therapeutic development. An interactive, publicly accessible portal will facilitate broad scientific and clinical utility.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5ef5fbb1b4ae669071d94a0dad7887a5b662c5" target='_blank'>
              P13.18.A MENINGIOMAP: A SINGLE-CELL ATLAS OF MENINGIOMA ECOSYSTEMS AND MICROENVIRONMENTAL NETWORKS
              </a>
            </td>
          <td>
            A. Boecke, A. Mohammed, D. Das, G. Nishita, R. Roelz, M. Shah, T. Juratli, J. Beck, J. Grauvogel, K. Joseph, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) exhibits marked plasticity and intense microenvironmental crosstalk. We aimed to delineate mesenchymal programs with spatial resolution, clinical relevance, and mechanistic anchors. Methods We integrated single-cell RNA-seq, bulk transcriptomes, and Visium spatial data. After rigorous QC and Harmony integration, we annotated 12 cell states using canonical markers, decoupler-based ORA, and AUCell. Tumor boundaries were defined by inferCNV/CopyKAT; developmental potential by CytoTRACE2 and PHATE. Post-translational modification (PTM) axes were scored from curated gene sets. A cell type-aware GNN linked bulk expression to a patient-similarity graph for survival modeling and gene-level hazard attribution. Network convergence combined bulk WGCNA (TCGA/CGGA), single-cell hdWGCNA, BayesPrism deconvolution, and external GEO validation. Ligand–receptor (LR) signaling was inferred with LIANA+, embedded in a signed causal network, and mapped spatially. ARRDC3 expression was assessed in GBM tissues; U251 gain- and loss-of-function assays evaluated proliferation and migration. Results We resolved major GBM states, including two mesenchymal programs (MES1-like, MES2-like). CNV-high regions marked malignant cores, and CytoTRACE2 identified high-potency niches within MES2-like and Proliferation states along non-linear trajectories. PTM landscapes segregated by state; S-nitrosylation, glycosylation, and lactylation were enriched in mesenchymal programs. A GNN risk score stratified overall survival in TCGA (n=157) and generalized to CGGA-325 (n=85) and CGGA-693 (n=140). MES2-like abundance remained an independent adverse predictor (HR = 2.31; 95% CI, 1.04–5.10). MES2-high tumors upregulated EMT, TNFα/NF-κB, JAK/STAT, hypoxia, angiogenesis, and glycolysis; S-nitrosylation associated with increased hazard. Cross-modal convergence defined a conservative MES2 core enriched for ECM remodeling, collagen modification, focal adhesion, and TGF-β regulation. LR analysis prioritized a TAM-to-MES2 axis (e.g., GRN–TNFRSF1A, ADAM9/10/17–ITGB1, TGFB1–ITGB1/EGFR) converging on a CEBPD-centered module. Spatial mapping localized MES2 hotspots within CNV-defined territories and revealed a TNFRSF1A–CEBPD–ARRDC3 focus at an infiltrative rim. ARRDC3 was upregulated in GBM tissues; in U251 cells, knockdown promoted and overexpression suppressed proliferation and migration, indicating context-dependent roles. Conclusions MES2-like GBM is an ECM-driven, stress-adapted state with strong prognostic impact. We nominate CEBPD and TNFRSF1A/ITGB1 as actionable nodes and identify ARRDC3 as a spatially restricted effector with context-dependent tumor-modulatory functions warranting therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8942e39c98213dbf5399f61de91b7075a89e9eb2" target='_blank'>
              Single-cell/spatial integration reveals an MES2-like glioblastoma program orchestrated by immune communication and regulatory networks
              </a>
            </td>
          <td>
            Chonghui Zhang, Lu Tan, Kaijian Zheng, Yifan Xu, Junshan Wan, Jinpeng Wu, Chao Wang, Pin Guo, Yugong Feng
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal primary brain tumor in adults, characterized by rapid proliferation, an immunologically inert microenvironment, diffuse tumor cell infiltration and therapy resistance. Progress in developing effective glioblastoma treatments is impeded by a lack of mechanistic insight into how GBM cells and the surrounding microenvironment respond to therapeutic interventions. To address this prominent gap, we developed an innovative on-therapy, longitudinal, and multi-regional tissue sampling approach in patients receiving rQNestin34.5v.2, an engineered oncolytic biologic, based on herpes simplex virus 1, to investigate the dynamic evolution of tumor and immune ecosystems during treatment. Using targeted spatial proteomics (CyCIF) alongside spatial metabolomics (MALDI-MSI), we analyzed over 100 samples from six patients, profiling immune, stromal, and malignant cells across both space and time. Our CyCIF data revealed that T and B cells infiltrate and colocalize within rQnestin34.5v.2–positive regions, forming clusters that reflect active immune interactions. These immune-rich zones were inversely correlated with SOX2⁺ and OLIG2⁺ tumor cells. Notably, patients with longer survival showed a progressive increase in CD3⁺ T cells, driven by expanding cytotoxic T cell populations alongside a reduction in CD4⁺ T cells. Despite increased immune infiltration, post-trial tissues displayed an immunosuppressive environment characterized by few cytotoxic T cells and reduced tumor cells. Integrating spatial proteomics and metabolomics, we observed elevated purine metabolites—including ATP, GTP, and xanthine—in regions of heightened immune infiltration. Extracellular ATP and GTP likely facilitate T cell activation and migration through purinergic signaling and GTPase pathways. These metabolic patterns suggest that purine metabolism plays a key role in shaping immunologically active tumor microenvironments following treatment. Together, these findings provide novel mechanistic insights into the evolving tumor-immune landscape during rQnestin34.5v.2 therapy in glioblastoma, highlighting purine metabolism as a potential driver of immune activation within the tumor microenvironment. This integrated spatial approach offers a powerful framework to guide the development of more effective immunotherapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82f2a6552b2a8d376cecc37c28c7caffa62bc8f2" target='_blank'>
              TMIC-70. Integrative spatiotemporal proteomic and metabolomic characterization of immune infiltration following intracranial injection of oncolytic immunotherapy in glioblastoma patients
              </a>
            </td>
          <td>
            Jennifer Gantchev, Gerard Baquer, Peter Zheng, Alicia D’Souza, Alexander L. Ling, Michael S Regan, G. Teamlab, Atul Deshpande, Forest M. White, Kenny Yu, E. Chiocca, Charles Couturier, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is marked by an immunologically inert tumor microenvironment (TME), which may explain why immunotherapies have generally failed to improve survival. While TME features have been linked to clinical outcomes, prior studies were limited by small sample sizes and narrow cell type resolution, precluding a full understanding of TME multicellular dynamics. To address this, we assembled 1,979 clinically annotated GBM RNA-seq profiles and benchmarked six deconvolution algorithms using pseudobulk data from a large GBM single-cell RNA-seq dataset. Bayesian deconvolution (InstaPrism) emerged as a top performer and was used to characterize the TME across 39 cell states. We then stratified tumors by their dominant malignant cell state (AC-like, MES-like, NPC-like, or OPC-like). We identified lymphoid-dominant and myeloid-dominant immune compositions within each GBM subtype. We demonstrate that lymphoid-dominant, immune-activated tumors associate with significantly improved overall survival within the aggressive MES-like group, but not within other groups. Immune activation was defined by a conserved stress-inflammatory module consisting of general inflammatory activation, tissue remodeling, and stress. In tumors dominated by MES-like cell states, immune cells within the TME were driven by robust interferon alpha and gamma gene signatures. Sobel mediation analysis pinpointed Stimulator of interferon genes (STING) upregulation as a potential upstream orchestrator within this group. Interestingly, STING activation showed no beneficial effect in AC-like or NPC-like tumors and was detrimental in OPC-like GBM, highlighting a group-specific biology. To our knowledge, this represents the most comprehensive analysis of the GBM TME to date in terms of patient number (n=1,979) and TME cell type plexy (n=39). The findings offer new understanding of GBM multicellular organization and how different cell types coordinate within the tumor. These data may help design rational combinations of immunomodulatory therapies and guide patient selection for next-generation clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fc74608f0fb0d307e1af10e8fcfdf8bbe8b02e" target='_blank'>
              TMIC-63. Microenvironment archetypes in 1,979 glioblastomas reveal cell state-specific drivers of immune recruitment and survival benefit
              </a>
            </td>
          <td>
            Kevin J. Tu, Zachary J. Reitman
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is one of the most aggressive and deadly brain tumors, with a median survival of ~14 months. Its poor prognosis is largely due to its ability to infiltrate healthy brain tissue both as single cells and in collectives. A deeper understanding of both individual and collective GBM cell motility could reveal new insights for patient-specific therapies. We applied live-cell imaging and mathematical modeling to characterize migration in patient-derived GBM cells, aiming to identify stratification markers and therapeutic vulnerabilities.



 We developed Single-Cell Behavior Live Imaging (ScBLI), an approach integrating live imaging with computational analysis to study the dynamics of GBM and healthy astrocyte (HA) cultures. We analyzed 21 GBM cell lines and one HA culture. Using TrackMate and Cellpose for segmentation and tracking, we extracted data on trajectories, velocities, distances, and morphological features (e.g., volume, circularity). Delta scaling was applied to classify trajectories as subdiffusive, Brownian, or Levy-like. Functional assays evaluated responses to migration stimuli and therapeutics. We also assessed collective migration behavior and integrated these data with single-cell analyses. All findings were correlated with clinical outcomes and transcriptomic profiling.



 We tracked 100 cells per sample, analyzing 2200 trajectories. Each culture contained three distinct migratory subpopulations based on metrics such as displacement, velocity, confinement ratio, and cellular organization. GBM samples were grouped into three behavioral clusters. Delta scaling values (0.28-0.80) identified corresponding migration modes. Higher delta values indicated more structured, Levy-like migration and were associated with shorter patient survival, suggesting a relationship between organized motility and increased invasiveness. Transcriptomic analysis revealed differential gene expression signatures among the clusters, potentially linked to migration phenotypes.



 Our study presents a robust framework to characterize GBM migration dynamics across patient-derived cultures. By integrating individual and collective motility data with clinical and transcriptomic information, we identified three distinct behavioral profiles with prognostic value. Delta scaling proved to be an effective metric for stratifying tumors based on migratory efficiency and aggressiveness. These insights highlight how structured motility may facilitate tumor spread and serve as a functional biomarker. Moreover, the identification of migration-associated gene signatures opens avenues for the development of targeted therapies aimed at disrupting invasive behavior. Ultimately, our findings support the use of dynamic phenotyping in guiding personalized treatment strategies for GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53502cc74a0ab473690a657868f9d225b25d02d6" target='_blank'>
              P12.34.A TRACKING INDIVIDUAL AND COLLECTIVE CELLULAR DIFFUSION IN PATIENT-DERIVED PRIMARY GBM CELL LINES: EXPLORING CELL DIFFUSION PROPERTIES AS A POTENTIAL VULNERABILITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Morelli, G. Ferri, T. Hohmann, F. Lessi, F. Di Lorenzo, A. Pastore, S. Franceschi, F. Pieri, C. Gambacciani, O. Santonocito, Y. Shah, L. Tonello, A. D. Di Stefano, P. Grigolini, L. Palatella, C. Mazzanti
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e28d12e48520ac375b3cbad04e1c6d130662c96" target='_blank'>
              Deciphering Cancer Evolution Through Genomic Profiling of Patient-Derived Xenograft Together with Matched Primary Gallbladder Cancer.
              </a>
            </td>
          <td>
            Vipin Yadav, R. Kilambi, Archana Rastogi, Deepti Sharma, Perumal Nagarajan, Varun Suroliya, Anupam Bhattacharya, Vaishali Yadav, Pramod Gautam, Manju Kashayp, M. K. Kashyap, Vanya Garg, Anupama Kumari, N. Trehanpati, G. Ramakrishna
          </td>
          <td>2025-11-10</td>
          <td>Digestive diseases and sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 The reciprocal interaction between neural and immune components in the tumor microenvironment has emerged as a critical area of study in cancer neuroscience. Recent data has shown the increasing role of myeloid cells in enabling an immunosuppressive microenvironment in glioblastoma. However, the nature and the role of T cells and other immune components forming tumor-immune synapses remain to be fully explored. Here we aim to demonstrate how immunological stimuli and inflammatory changes affect the neural connections within and around glioblastoma cells using advanced spatial transcriptomics.



 The study employs human cortical slice models, inoculated with tumor cells carrying retrograde tracing viruses, specifically a rabies virus, to trace neuron-tumor connections. This was followed by the inoculation of reactive T cells. We used the MERSCOPE system, which allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, spanning multiple Z-depths by combining this cutting-edge technology with new segmentation approaches.



 Our spatial transcriptomic analyses revealed different cellular ecosystems in the human cortical slice model. Enhanced inflammatory stimuli can increase local tumor growth and infiltration and neuronal connectivity. We found that inhibition of T cell activation using JAK-STAT inhibition can reverse the reactive hyperconnectivity. We further identified key transcription regulators defining immune cell dynamics and triggering immunological stimuli within tumor-immune synapses.



 The integrative multi-omic approach using imaging-based analysis and 3D single-cell spatial transcriptomics technique is a powerful tool that allows a comprehensive understanding of how immune cells influence the neural connectivity and synaptic reactivity within the peritumoral microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5833d83bdb7f3aebf492021ffdbee5e8a292416d" target='_blank'>
              P04.16.A DECIPHERING NEURAL-IMMUNE RECIPROCAL INTERACTIONS IN GLIOBLASTOMA USING 3D SINGEL-CELL SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            T. Peng, A. Merdan, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression and treatment resistance are closely linked to cancer stem cells (CSCs), yet their spatial dynamics and regulatory mechanisms remain poorly characterized. This study integrates single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to define CRC stem cell subtypes (Co-CSS), map their interactions within the tumor microenvironment, and identify novel prognostic markers driving clinical outcomes. Methods scRNA-seq was used to analyze cellular heterogeneity in CRC, identifying key Co-CSS. CytoTRACE was applied to assess stemness. ST further mapped the distribution and interactions of Co-CSS with the tumor microenvironment. Gene intersection analysis identified NPDC1 and NSMF, whose high expression correlated with poor prognosis. Patient classification based on NPDC1 and NSMF expression highlighted treatment response variability, suggesting targets for precision medicine. Results The findings indicate that Co-CSS in CRC can modulate the tumor microenvironment and influence the progression of the disease through various ligand-receptor interactions. It is possible that Co-CSS is regulated by fibroblasts through the WNT signaling pathway. The expression levels of NPDC1 and NSMF are significantly correlated with the overall survival of CRC patients, suggesting their potential value as independent prognostic markers. Conclusions This study offers insights into the role of Co-CSS in CRC through ST and gene expression profiling, uncovering novel markers and pathways. It redefines the interplay between Co-CSS and the tumor microenvironment, offering a foundation for future mechanistic studies and therapeutic interventions aimed at targeting Co-CSS to improve CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed26e5557feb0443a11a342bd7bb1b7f2d659826" target='_blank'>
              The role of cancer stem cells in the progression of colorectal cancer and the tumor microenvironment: new perspectives from single-cell RNA sequencing and spatial transcriptomics analyses
              </a>
            </td>
          <td>
            Yuting Wu, Xianglin Liu, Wenqiang Liu, Qingliang Jiang, Yangyang Li, Hengyu Li, Liqiang Hao
          </td>
          <td>2025-10-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4273a0ebbda27839741b0bafaebff9ecc71a2adf" target='_blank'>
              Defining heterogeneity in core regulatory circuitry reveals HOXB3 condensation as a potential target in glioblastoma
              </a>
            </td>
          <td>
            Chuanxia Zhang, Yijing He, Xiudan Zhan, Jiaming Yu, Shao Xu, Qinkai Zhang, Ru Qiu, Xinyue Wang, Zhuoxing Gao, Meng Huang, Wenyong Long, Qing Liu, Wei Zhao
          </td>
          <td>2025-09-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Muscle-invasive (MI) urothelial carcinoma (UC) is a clinically challenging malignancy with a poor prognosis. Understanding the cellular dynamics that drive UC progression is critical for the development of optimized therapeutic strategies. Through integrative analysis of large-scale single-cell transcriptomic datasets from non-muscle-invasive (NMI) and MI tumours and validation with spatial transcriptomic datasets, we systematically characterized immune cell dynamics and cancer cell plasticity during UC progression. Our analysis revealed an immunosuppressive tumour microenvironment and a subset of cancer cells with upregulated major histocompatibility complex II (MHC-II) expression in MI tumours. Notably, MHC-II⁺ cancer cells were induced by interferon-γ signalling, as confirmed through in vitro experiments, and exhibited phenotypic alterations characterized by enhanced proliferative and migratory capacities. Furthermore, MHC-II⁺ cancer cells spatially colocalized with CD8⁺ T cells, regulatory T cells, and SPP1⁺ macrophages, where they engaged with inhibitory receptors on these immune cells, promoted CD8⁺ T cell exhaustion and facilitated immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b9542e0889bc085781e9b9db8fbe77cfa48b4c" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveal cellular heterogeneity and cancer cell-intrinsic major histocompatibility complex II expression in urothelial carcinoma
              </a>
            </td>
          <td>
            S. Xiong, Jian Fan, Cong Huang, Shi-ming He, Yuan Liang, Qi Zhang, Bin Guo, Yucai Wu, Mancheng Xia, Fangzhou Zhao, W. Ci, Liqun Zhou, Yanqing Gong, Xuesong Li
          </td>
          <td>2025-10-20</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a highly aggressive brain tumor with limited therapeutic options and significant immune evasion. Tumor-associated regulatory T cells (Tregs) and macrophages (TAMs) are central orchestrators of the GBM immune microenvironment; however, the dynamics and subtype-specific interactions between these populations during disease progression are still poorly understood. In this study, we utilized complementary mesenchymal and proneural murine GBM models to examine the differentiation trajectories and functional crosstalk of Tregs at single-cell resolution. Single-cell RNA sequencing (scRNA-seq) of tumor-infiltrating Tregs revealed that mesenchymal GBM promoted the expansion of an immunosuppressive Treg population characterized by elevated CCR8 (C-C chemokine receptor type 8), GITR (Glucocorticoid-Induced TNFR-Related protein), and IL1R2 (Interleukin 1 Receptor Type 2), which was less pronounced in proneural GBM. Trajectory analysis uncovered enhanced proneural-to-mesenchymal transition signatures, indicating dynamic Treg adaptation to evolving tumor states. Validation in human spatial transcriptomic GBM datasets revealed that the murine tumor Treg signature was enriched within mesenchymal regions and closely associated with myeloid-dense microenvironments. By leveraging a time-course scRNA-seq dataset from the mesenchymal GBM model, we observed a progressive expansion of the mesenchymal-like Treg signature paralleling tumor evolution and mesenchymal transition. Notably, CellChat-based cell–cell communication analysis demonstrated that Treg–myeloid interactions were not only enhanced over time but also that Treg subsets differentially engaged with immunosuppressive TAMs through unique ligand–receptor pairs, underscoring the dynamic and reciprocal adaptation between regulatory and myeloid compartments. Collectively, our study delineates a model in which GBM progression is shaped by the evolving interplay of Treg and myeloid programs, providing a framework for targeting context-specific immunosuppressive circuits in future immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a27d9dd70e278b9475e8a73470ac145030c21c" target='_blank'>
              IMMU-86. Dynamic Treg-myeloid crosstalk dictates glioblastoma progression
              </a>
            </td>
          <td>
            Sadashib Ghosh, Jeffrey M. Grimes, Stephen Opoku, J. Leavenworth
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Triple-Negative Breast Cancer (TNBC) encompasses a biologically heterogeneous group of tumors, which can be classified into distinct molecular subtypes, namely basal-like 1 (BL1), basal-like 2 (BL2), immunomodulatory (IM), mesenchymal (M), mesenchymal stem-like (MSL), and luminal androgen receptor (LAR), with unique clinical and pathological characteristics. While immune features of these subtypes have been extensively characterized, the integration of non-immune stromal and structural components into our understanding of TNBC biology is only now being fully recognized. This narrative review synthesizes current evidence regarding differences in the non-immune microenvironment across TNBC molecular subtypes, with a focus on cancer-associated fibroblasts (CAFs), vascular features, extracellular matrix (ECM) dynamics, and epithelial–mesenchymal transition (EMT), along with metabolic–hypoxic reprogramming. Data from several studies are integrated to highlight subtype-specific signatures. Differences in stromal architecture and metabolic adaptations, potentially reflecting the underlying molecular heterogeneity, may hold prognostic or predictive significance and could inform personalized therapeutic strategies targeting the tumor–stroma interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089f130c819509e80723565570647cc30d7f85ba" target='_blank'>
              Dissecting the Non-Immune Tumor Microenvironment in Triple-Negative Breast Cancer: Molecular Subtype-Specific Patterns and Prognostic Implications
              </a>
            </td>
          <td>
            A. Syrnioti, Eleni Timotheadou, Vasileios N. Papadopoulos, Georgia Syrnioti, T. Koletsa
          </td>
          <td>2025-11-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioblastoma tumors are characterized by a prominent infiltration of tumor-associated macrophages (TAMs), constituting up to 30-40% of the cellular make-up. These cells typically harbor immunosuppressive features which, in conjunction with T cell paucity, contribute to a tumor microenvironment (TME) uniquely refractory to immunotherapy response. Therefore, due to their critical role in tumor progression and therapy response, TAMs represent a promising therapeutic target within the brain TME.



 To address the challenge of macrophage heterogeneity in glioblastoma, we conducted comprehensive multi-omics analyses including scRNAseq, spatial transcriptomics, ATACseq and lipidomics, that identified distinct TAM subsets within tissue-resident microglia and infiltrating monocyte-derived macrophage (MDM) pools.



 We discovered a pro-tumorigenic and immunosuppressive subset common in both cell lineages characterized by high lipid metabolic activity which we termed lipid-laden macrophages (LLMs). These cells engulf and recycle lipid-rich myelin debris to fuel cancer cell proliferation and thereby promote malignancy. Beyond these functions, LLMs also exhibit immunosuppressive features that are induced by myelin debris uptake. Next, we aim to investigate the relationship between myelin-induced metabolic pathways and the subsequent immunosuppressive phenotype of LLMs. To accomplish this, we will employ a genetic screen on primary murine macrophages using a CRISPR knockout library. This approach will target the cellular metabolome and surfaceome, with the goal of rewiring TAMs toward a more immunogenic phenotype. To enable selective targeting of LLMs, we developed lipid nanoparticle (LNP)-based approaches. Our findings demonstrate that LNPs efficiently deliver cargo to macrophages in the TME, with formulations that can be tailored for enhanced cell type-specific uptake.



 We identified myelin as a key molecule fostering a tumor-permissive brain TME. We are currently harnessing genetic screening together with the development of cell type-specific LNPs which will enable interventions targeting the immuno-metabolic axis, potentially advancing new treatment strategies for glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8404fd7f21668b88fc96f4132337aef2254ebb3" target='_blank'>
              P02.04.A UNCOVERING AND TARGETING METABOLIC REGULATORS OF IMMUNOSUPPRESSIVE LIPID-LADEN MACROPHAGES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Farber, T. van Schaik, D. Aschmann, M. Boon, D. Kloosterman, J. Erbani, L. Akkari
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immunohistochemistry (IHC) remains the gold standard for evaluating protein expression in tumor microenvironment analysis. This approach hinders robust correlation analyses between spatial heterogeneity in the tumor microenvironment and clinical outcomes like disease-free survival (DFS). To address these challenges, we developed an automated pipeline for quantitative IHC feature extraction. Our method integrates deep learning-based tumor segmentation with computational detection of invasive margins at varying distances. Deconvolution algorithms quantify diaminobenzidine (DAB) staining intensity across the tumor body and the invasive margin. Spatial heterogeneous DAB density patterns were subsequently analyzed for DFS correlation. Using 104 patient samples (57 training/47 validation) stained for CD3, CD8, CD31, and HIF-1α, we identified 2 prognostic feature categories (CD3/CD8 aggregated positive areas within the 0.25-mm peripheral zone extending outward from the tumor-invasive front and HIF1-α-positive areas within a 0.75-mm peripheral zone extending outward from the tumor-invasive front). Immune-related features demonstrated C-indices of 0.726 (training) and 0.626 (validation), while hypoxia-associated markers showed C-indices of 0.714 and 0.656, respectively. Integration of these features with pTNM staging enhanced DFS stratification compared to pTNM staging alone, improving C-indices from 0.702 to 0.819 (training) and 0.668 to 0.853 (validation). This automated pipeline addresses critical limitations in traditional IHC analysis by enabling: 1) Objective quantification of spatial DAB heterogeneity. 2) Identification of biologically interpretable prognostic features. 3) Enhanced predictive performance over conventional staging systems. Our findings suggest this methodology could standardize IHC-based prognostic assessments and inform personalized treatment strategies. Further validation in multicenter cohorts is warranted to confirm clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c85bd6804a873bad087f9c925d5c02405f8fb31f" target='_blank'>
              Automatically quantifying spatial heterogeneity of immune and tumor hypoxia environment and predicting disease-free survival for patients with rectal cancer
              </a>
            </td>
          <td>
            Shenlun Chen, Zonglin Liu, Jiazhou Wang, Zhiyuan Zhang, Andre Dekker, L. Wee, P. Kalendralis, Yiqun Sun, Tong Tong
          </td>
          <td>2025-11-18</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (ICC) has a poor prognosis due to late-stage presentation and ineffective systemic therapies. Targeting the tumor microenvironment (TME) in ICC offers new therapeutic possibilities, particularly through tumor-associated macrophages (TAM), which can both promote and inhibit tumor progression. The current study utilized multi-omics analysis to characterize the gene signature of TAM and explore its therapeutic potential in ICC. Methods: Public GEO datasets provided the basis for analysis. Single-cell RNA sequencing (scRNA-seq) data from five ICCs, three adjacent non-tumorous tissues (ANTs), and four healthy liver samples were examined with Python. To validate scRNA-seq findings, bulk RNA-seq data from 27 ICC and 27 matched ANT samples were assessed using R. Differentially expressed genes were identified with adjusted p-values <0.01 and log2-fold changes >1 or <−1. CIBERSORT pipeline analyzed 22 immune cell subtypes in bulk RNA-seq data. STRING database analyzed the contribution of unique TAM-related genes to networks of protein–protein interactions. Results: TAM population demonstrated phenotypic heterogeneity exhibiting partial gene signatures of inflammatory (MS1) and anti-inflammatory (MS2) macrophages. Unique TAM-associated markers, TREM2, CD9, and PRMT10, showed variable expression within the TAM subpopulation. Bulk RNAseq analysis confirmed the scRNA-seq results, highlighting overexpression of TREM2 and CD9 in most ICC samples versus ANT. Immune cell deconvolution revealed decreased MS1 and MS2 macrophages in ICC, and alterations in adaptive immune profile, suggesting immunotolerant TME. STRING database defined TREM2-LGALS3 axis as a potential target for anti-tumor therapies. Conclusions: TAM represents a unique heterogenous population which is primarily found in ICC TME versus ANT or healthy liver tissue The non-uniform expression of unique gene signature demonstrates additional heterogeneity in the TAM subpopulation and suggests that TREM2+ TAM may be desirable targets for anti-TREM2-LGALS3 immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be67d95f17acd65abd3873af222e463c89ca98c4" target='_blank'>
              Defining Gene Signature of Tumor-Associated Macrophages in Intrahepatic Cholangiocarcinoma as Target for Immunotherapy Using Single Cell and Bulk RNA Sequencing
              </a>
            </td>
          <td>
            J. S. Badshah, Ryan M. Lee, Andrea Reitsma, M. Melcher, Olivia M. Martinez, Sheri M. Krams, Daniel J. Delitto, Varvara A. Kirchner
          </td>
          <td>2025-10-29</td>
          <td>Livers</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9eea4d17ccda6da786c821ed59ad42654c9c3e4" target='_blank'>
              Tumor microenvironment in breast cancer: cellular crosstalk, pathways, and therapeutic insights
              </a>
            </td>
          <td>
            Manjari Sharma, Umesh Kumar, Shoorvir Singh
          </td>
          <td>2025-11-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human bone marrow stromal cells (BMSC) include skeletal stem cells with ground-breaking the-rapeutic potential. However, BMSC colonies have very heterogeneous in vivo behaviour, due to their different potency; this unpredictability is the greatest hurdle to the development of skeletal regeneration therapies. Colony-level heterogeneity urges a fundamental question: how is it possible that one colony as a collective unit behaves differently from another one? If cell-to-cell variability were just an uncorrelated random process, a million cells in a transplant-bound colony would be enough to yield statistical homogeneity, hence washing out any colony-level traits. A possible answer is that the differences between two originating cells are transmitted to their progenies and collectively persist through an hereditary mechanism. But non-genetic inheritance remains an elusive notion, both at the experimental and at the theoretical level. Here, we prove that heterogeneity in the lineage topology of BMSC clonal colonies is determined by heritable traits that regulate cell-cycle exit. The cornerstone of this result is the definition of a novel entropy of the colony, which measures the hereditary ramifications in the distribution of inactive cells across different branches of the proliferation tree. We measure the entropy in 32 clonal colonies, obtained from single-cell lineage tracing experiments, and show that in the greatest majority of clones this entropy is decisively smaller than that of the corresponding non-hereditary lineage. This result indicates that hereditary epigenetic factors play a major role in determining cycle exit of bone marrow stromal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b8b3d836cf1a27dbadc5ff0c8ab8409486bfd3" target='_blank'>
              Inheritance entropy quantifies epigenetic regulation of cell-cycle exit in human bone marrow stromal cells
              </a>
            </td>
          <td>
            Alessandro Allegrezza, Riccardo Beschi, Domenico Caudo, A. Cavagna, A. Corsi, A. Culla, Samantha Donsante, Giuseppe Giannicola, I. Giardina, Giorgio Gosti, T. Grigera, S. Melillo, B. Palmisano, L. Parisi, L. Postiglione, Mara Riminucci, Francesco Saverio Rotondi
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="
 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumours defined by their midline localization, early onset, and distinct mutational landscape. The H3.3K27M mutation is found in approximately 80% of DMGs, marking it as a key initiating event. However, H3.3K27M alone does not induce brain tumour formation in experimental models, suggesting that its oncogenic potential may be linked to specific developmental windows and susceptible precursor populations. Identifying the cellular states that are vulnerable to transformation is critical to understanding DMG initiation. To establish a reference framework for these vulnerable states, we generated a single-cell RNA sequencing atlas of normal hindbrain development. Using a Sox2-eGFP reporter mouse and FACS-based enrichment of Sox2+ precursors, we profiled over 120,000 cells across ten developmental stages, capturing critical transitions in neurogenesis and gliogenesis. To reveal regional differences, we integrated the atlas with a similarly generated forebrain atlas, providing a comprehensive map of neurodevelopmental trajectories where DMG mutations might exert their effects. Building on this developmental reference, we utilized inducible mouse models to introduce H3.3K27M mutations in precursors and employ lineage tracing to track their fate over time. When combined with loss of Trp53 and overexpression of PDGFRa, alterations prevalent in DMG, mice develop tumours that resemble the human disease. H3.3K27M-driven tumours consistently localize to the pons, in contrast to H3 wild-type tumours, which occur more variably across the forebrain and hindbrain. Tracking these pontine tumours using serial MRI revealed that tumours arise well before symptom onset and exhibit growth dynamics that vary by location and timing. Earlier-arising tumours are more aggressive and tend to occur in posterior locations. Together, our integrated approach combining lineage tracing, tumour modelling, and a developmental atlas provides a powerful platform to dissect the early events in DMG pathogenesis and identify developmental vulnerabilities that may be targeted for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a1a4d1a35cbc94502034b6562096dc3dd8efc6" target='_blank'>
              STEM-27. TRACING THE ORIGINS OF H3.3K27M MUTANT DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Caitlin Herrington, Evan Lubanszky, Quang Trinh, Luiza Lopes Pontual, Jannine Forst, Cynthia Hawkins, Peter Dirks
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BRCA1 mutation is closely linked to triple-negative breast cancer (TNBC), a subtype characterized by early onset, aggressive clinical course, and marked genomic instability. Although poly ADP ribose polymerase (PARP) inhibitors have expanded treatment options, resistance driven by cancer stem cells (CSCs) and incomplete interception of premalignant progression remain pressing challenges. In a recent study, Liu et al. developed a human induced pluripotent stem cell (iPSC)-based breast cancer model that follows BRCA1‑driven tumorigenesis from mammary differentiation to invasive disease and identify S100P as a BRCA1‑repressed effector associated with cancer stemness. These findings provide a human genetic context to probe early oncogenesis and nominate the S100P–RAGE axis as a precautionary—but promising—target for risk interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfe5bbd7fe04bb4260117969d5b0d5e59d56a2ae" target='_blank'>
              Deciphering BRCA1-driven tumorigenesis through an iPSC-based breast cancer model
              </a>
            </td>
          <td>
            Gaoxiang Zhu, Bing Zhao
          </td>
          <td>2025-10-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            Jeanette Johnson, Daniel R. Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Brain metastases (BM) from lung adenocarcinoma represent a formidable clinical challenge due to limited therapeutic efficacy. To gain insight into complex tumor microenvironment (TME) of BM, we analyzed single-cell RNA-sequencing data comprising 66,769 cells from primary tumors (PTs; 44,355 cells) and matched BM lesions (22,414 cells). Our analysis revealed a profound shift in cellular composition during metastasis: while T cells predominated PT (44.12%), tumor cells became the most abundant population in BM (55.17%), with corresponding reductions in T cells (27.57%) and macrophages (19.3%). Cell–cell communication analysis identified fibroblasts as central mediators in both PT and BM, with a transition from collagen signaling in PT to thrombospondin signaling in BM, facilitating interactions with tumor cells, B cells, and oligodendrocytes. Ingenuity Pathway Analysis revealed enrichment of collagen-related genes in fibroblasts (MFAP4, COL15A1, FBLN1, FBLN2) and Th1/Th2 pathway genes in T cells (IKZF1, SH2D1A, RASGRP1, SKAP1). Subclustering uncovered seven distinct fibroblasts subpopulations and a notable shift from PT-enriched C0 to BM-enriched C1, suggesting dynamic phenotypic adaptation. Pseudotime trajectory analysis further clarified fibroblasts differentiation states. T-cell subclustering revealed a significant increase in immunosuppressive regulatory T cells (84.1%) and proliferating CD4+ T cells (10.5%) in BM, highlighting mechanisms of immune evasion. COL15A1 expression correlated with poor patient survival and emerged as a promising therapeutic candidate, showing increased sensitivity to Dabrafenib, Trametinib, Piperlongumine, Cabozantinib, Staurosporine, and Vemurafenib. Additional gene drug associations were observed, with FBLN1, RASGRP1, MFAP4, SH2D1A, and IKZF1 linked to potential sensitivity to Erlotinib, Afatinib, Fluorouracil, and Brefeldin A. These findings provide a high-resolution map of the BM TME and highlight COL15A1 and related pathways as potential targets for precision therapy in patients with BM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc3a5b204d80d9cd5d66d31480f9cf9dd3274795" target='_blank'>
              TMIC-44. Fibroblast reprogramming and immune evasion define the tumor niche in brain metastases
              </a>
            </td>
          <td>
            Sanskriti Dey, J. Chuang, Chih-Yang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Pineoblastoma (PB) is a clinically aggressive embryonal central nervous system (CNS) tumor composed of distinct molecular subgroups. The cellular and biological basis of PB remains poorly defined, limiting the development of more effective therapeutic strategies. To identify the cellular composition, origins, and drivers of PB, we derived single-cell transcriptomes from patient-derived pineal tumors (n=38) and integrated malignant cells with a novel transcriptional atlas of mouse pineal gland development (E13-P21). Application of a series of computational strategies pinpointed transient, cycling pinealocyte progenitors as the likely cellular origin of PB, irrespective of molecular subgroup. To functionally validate these findings, we generated lineage-specific genetically engineered mouse (GEM) models representing distinct PB subgroups with high temporal, anatomic, and phenotypic fidelity. Transcriptomic analysis further substantiated the accuracy of novel GEM models which converged on cell-cycle dysregulation, highlighting a core pathogenic mechanism across PB subgroups. Relative to the developing pineal gland, photoreceptor transcription factors were aberrantly activated in PB GEM models, suggesting photoreceptor involvement in tumor initiation and/or maintenance. Multi-omic analysis across a large range of diverse CNS cancers revealed an oncogenic photoreceptor program specific to PB, retinoblastoma (RETB), and Group 3 medulloblastoma (G3-MB). This shared transcriptional program was active within respective cellular origins, establishing a unified mechanistic basis across these anatomically distinct CNS tumors. CRISPR-based functional studies confirmed selective dependencies of photoreceptor transcription factors (NRL, CRX, OTX2) in PB, RETB and G3-MB, demonstrating the essentiality of this photoreceptor program. These discoveries not only resolve longstanding uncertainties regarding PB pathogenesis but also highlight conserved molecular signatures and therapeutic vulnerabilities spanning different pediatric CNS malignancies. Our results underscore the potential of targeting developmentally regulated programs and master transcription factors as novel therapeutic strategies for PB, RETB, and G3-MB, offering critical insights and preclinical models for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cee7231cd5056fbbc7fcab0e2a1e377781c0c82" target='_blank'>
              EPCO-05. AN ONCOGENIC PHOTORECEPTOR PROGRAM UNIFIES DISTINCT CENTRAL NERVOUS SYSTEM TUMORS
              </a>
            </td>
          <td>
            B. Gudenas, Shiekh Ahmad, Bernhard Englinger, Anthony P Y Liu, Miao-yun Zhao, Leena Paul, Jennifer L. Hadley, Hong Lin, Yiran Li, Melissa Batts, Priya Mittal, Stephanie C Wu, Sara Lewis, Katie Han, T. Soliman, Laure Janke, David Meredith, Elke Pfaff, J. Gojo, Jennifer Cotter, P. Klimo, F. Boop, Amar Gajjar, G. Robinson, G. Rosén, S. Alexandrescu, David T. W. Jones, Brent A. Orr, F. Swartling, M. Filbin, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 Immunosuppression in glioblastoma is a significant impediment to effectiveness in immunotherapy. We employed 3D-spatially mapped sampling and genome wide correlation analysis of RNA sequencing to identify an immunosuppressive signature comprising validated markers of multiple cell types, with LRRC25 as a key player. We confirmed the robustness of this signature in an independent internal cohort of 3D-spatially mapped GBM patients and TCGA. We observed strong co-variation of over one hundred genes, including markers for myeloid cells, neutrophils, and lymphoid cells (T and NK-cells) along with LRRC25. Published single-cell RNA-seq from glioma validated that LRRC25 is expressed by neutrophils and myeloid cells and absent from lymphoid-cells. The signature may therefore represent multiple interacting cell types. This is strongly supported by signature expression increasing when increasing the proportion of the three cell types bioinformatically by combining defined proportions of each via pseudobulking. LRRC25 expression varies spatially within tumors, as shown in single-cell data, the IVY-GAP laser capture microdissection database, and Visium spatial sequencing technologies. These data support and extend our 3D-spatially mapped samples showing upregulation in the tumor core compared to the periphery. We confirmed LRRC25 expression in myeloid cells in situ by RNAscope co-stained with anti-CD163, and in tumor cells by RNAscope and histopathology. HGG and LGG patients have LRRC25 upregulated locally (intratumoral) and systemically (circulating immune cells) in myeloid, T and NK cells relative to healthy donors. This suggests a potentially actionable role in local and systemic immunosuppression. CRISPRi-mediated LRRC25 knockdown in MACs-sorted patient peripheral myeloid cells enhances tumor-cell killing in T-cell co-culture assays. While significant efforts have focused on mechanisms of myeloid to T-cell immunosuppression, less focus is given to myeloid to NK cell, and neutrophil immunosuppression, despite the fact they exist at levels comparable in the tumor microenvironment. This work will be followed by further validation of functional immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f7703b6455b4170f74078687849aea510338ce" target='_blank'>
              TMIC-110. Role of LRRC25 in local and systemic immunosuppression and its reversal
              </a>
            </td>
          <td>
            Daniel Zamler, S. Lakshmanachetty, Nicholas O. Stevers, Gary Chan, Jerry Liu, Lee Chen, Benjamin Lerman, Darwin W. Kwok, Chibo Hong, Maggie Colton Cove, M. Gallus, Radhika Mathur, Cristina Dabaco, Jessica Rodino, Patrick Schupp, Anny Shai, Joanna Phillips, H. Okada, Michael Oldham, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Gliomas are the most common malignant brain tumors, characterized by their aggressive nature and poor prognosis. Despite advancements in radiation and chemotherapy, the five-year survival rate for patients with glioma remains low. A significant challenge is the lack of experimental models that accurately replicate the tumor microenvironment and cellular heterogeneity of human gliomas, limiting the understanding of disease mechanisms and the development of effective therapies.



 Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids.



 Immunofluorescence, flow cytometry, and gene sequencing revealed that the organoids retained an immune microenvironment consistent with that of the original tumor. Unlike traditional cell lines and animal models, GlioMEs retain the immune microenvironment, allowing faster and more accurate immunotherapy screening. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib.



 In summary, our results demonstrate that GlioME can retain the parental glioma microenvironment in vitro and reliably test various therapeutic drugs. This organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4662e63a3116553f2fb291a5b7c518e12a2b2372" target='_blank'>
              P12.31.B A NOVEL ORGANOID MODEL RETAINING THE GLIOMA MICROENVIRONMENT FOR PERSONALIZED DRUG SCREENING AND THERAPEUTIC EVALUATION
              </a>
            </td>
          <td>
            C. Zheng, Q. Chen, Z. Fang, P. Wang, D. Zhang, Y. Feng, Y. Zhang, T. Jiang, L. Zheng, Z. Bao
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cced907718bd3a02f9c4423bb5be0323614fd5" target='_blank'>
              Long-term clonal analysis using stochastic models reveals heterogeneity and quiescence of hematopoietic stem cells
              </a>
            </td>
          <td>
            Yuri Garcia Vilela, L. Thielecke, A. C. Fassoni, I. Glauche
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Bladder cancer is a significant malignancy, for which prognostic prediction and understanding of the tumor immune microenvironment are crucial. B cells play a key regulatory role in this environment, making their study essential for advancing bladder cancer research. Methods In this study, a multi-omics analysis strategy combining single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq was used to establish single-cell transcriptome profiles of tumor tissues from bladder cancer patients, focusing on B-cell populations and their interactions with other cell types in the tumor microenvironment. Large public databases were used to screen for key prognostic genes associated with bladder cancer B cells, and their biomarker expression was verified by in vitro experiments. Results Based on tumor samples from eight patients with bladder cancer and four normal samples, we selected 84, 967 cells for single-cell sequencing analysis. From these, we identified 10, 967 B cells and identified 508 key genes associated with B cells in bladder cancer from five different B cell subtypes. By integrating a large amount of RNA sequencing data, we identified VCL, FLNA, TAGLN, ACTA2, COL6A2, and CALD1 as potential biomarkers for B-cell-associated bladder cancer, and experimentally verified that these markers were significantly lower in bladder cancer patients than in the normal group, and were effective in predicting the survival rate of the patients and the status of the tumor immune microenvironment. Conclusions Using a combination of transcriptomic and experimental validation at single-cell and batch levels, this study provides insights into the key gene signatures of B cells from patients with bladder cancer and their roles in regulating the tumor immune microenvironment, providing new biomarkers and potential therapeutic targets for predicting patient’ prognosis and immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8bfbe519c06579f4740a61199654ca1af067bc" target='_blank'>
              Comprehensive analysis of single-cell and bulk transcriptomes reveals key B-cell genes and immune microenvironment regulation in bladder cancer
              </a>
            </td>
          <td>
            Lijun Wang, Juan Yang, Zhangxiao Xu, Bo Tao, Yunpeng He, Yuan Zhao, Jian Wu, Yira Ma, Zitao Zhong, Lin Ye
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a highly aggressive brain tumor with poor prognosis and limited response to immunotherapies. Tumor intrinsic features like low mutational burden, and extrinsic factors such as T-cell dysfunction and an immunosuppressive microenvironment, contribute to resistance. To better study immune-tumor interactions in a clinically relevant context, we have developed patient-derived tumor organoids (PDTOs) that support co-culture with autologous and CAR-engineered immune cells, enabling real-time functional analysis.



 PDTOs were generated from freshly resected GB tissue, preserving tumor-specific architecture and molecular features. Patient-matched PBMCs were isolated via density gradient centrifugation, and immune cells were enriched using negative selection with MACS sorting. T cells were expanded ex vivo and characterized by flow cytometry for activation and exhaustion markers, although not yet tested in co-culture. Cytokine secretion was analyzed via ELISA to complement phenotypic profiling. For functional studies, EGFR/EGFRvIII-retargeted CAR-NK cells were generated via lentiviral transduction of primary PBMC-derived NK cells and expanded ex vivo in the presence of (low levels of) IL-15. These cells were co-cultured with 3D tumor spheroids at defined effector-to-target ratios. Immune infiltration and tumor cell killing were assessed using real-time live-cell imaging and fluorescence-based readouts.



 As proof of concept, we co-cultured EGFR-diploid and EGFR-amplified 3D GB spheroids with naïve and CAR-NK cells targeting EGFR derived from a healthy donor. CAR-NK cells showed enhanced cytotoxicity against EGFR-amplified spheroids, demonstrating receptor-dependent immune activation. These results confirm the feasibility of functional immune co-culture assays and support the ongoing integration of this approach into our organoid-based platforms.



 Our organoid-immune cell co-culture system offers a physiologically relevant platform to investigate GB immunobiology. Future applications include drug screening and CRISPR-based perturbation to identify key modulators of immune response, with the goal of informing personalized immunotherapeutic strategies. Observations from the CAR2BRAIN trial (NCT03383978) have shown CD4⁺ and CD8⁺ T cell recruitment following CAR-NK therapy, highlighting the clinical relevance of combinatorial immune approaches and supporting the use of our model to explore such mechanisms ex vivo.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb41d2ad01808f179f77775b88ffca7888d1d44" target='_blank'>
              P18.15.A IMMUNOCOMPETENT GLIOBLASTOMA ORGANOIDS FOR PERSONALIZED IMMUNE PROFILING
              </a>
            </td>
          <td>
            T. Koenig, L. M. Reindl, P. Zeiner, K. Weber, A. Rombach, M. Armbrust, P. Cakmak, W. Wels, T. Broggini, K. Plate, M. Czabanka, J. Steinbach, M. Burger, E. Ullrich, T. Buhlmann, A. Hau
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Malignant peripheral nerve sheath tumors (MPNST) are aggressive sarcomas with limited therapeutic options. Here, we present a novel sporadic murine model of Nf1 wild-type MPNST, driven by conditional expression of oncogenic BrafV600E and Pten loss in the glial lineage using the Plp1::CreERT2 driver. This model allows for highly penetrant and rapid tumor induction through spontaneous formation, localized initiation, or cell transplantation. Comparative analysis with Tyr::CreERT2-driven melanoma revealed striking phenotypic divergence despite shared genetic alterations, underscoring the importance of the cell of origin in shaping tumor identity. In this system, MPNST cells show refractory capacities to induce melanocytic trans-differentiation upon melanoma-promoting signaling cues such as canonical Wnt signaling gain of function (GOF) or increased of levels of the epigenetic mark H3K27Me3 upon Ezh2 GOF. Our findings emphasize the significance of lineage context in tumor initiation and provide a foundation for future mechanistic and therapeutic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b40c771fae54171400436daa93f306b770ba99" target='_blank'>
              A novel murine model for sporadic, malignant peripheral nerve sheath tumors, driven by BrafV600E and Pten loss.
              </a>
            </td>
          <td>
            Julien Debbache, Myriam Gwerder, Elisabeth Rushing, Lukas Sommer
          </td>
          <td>2025-10-09</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 There are four established tumor cell states in glioblastoma including astrocyte-like, oligodendrocyte precursor cell-like, neural progenitor cell-like, and mesenchymal-like states. Single-cell RNA sequencing has showed that they are all present with varying distribution between patients. However, since surgery of glioblastoma follows the principle of maximal safe resection, studies have been mainly limited to the necrotic core and contrast-enhancing zone.



 Here, we investigated the cellular organization of the transcriptional states within and outside of the contrast-enhancing zone. This was achieved through a unique set of supramarginal resected glioblastoma samples using large-scale single-cell RNA sequencing of fresh tissue by systematic multi-region sampling from a large number of spatial locations based on 5-ALA fluorescence.



 We discovered that the mesenchymal state is region-specific, highest in the core, at the center of the tumor, shows a gradual decrease with distance from the core and is absent in the low-fluorescent zone and in the periphery, regardless of direction. More importantly, the mesenchymal signature is not only present in tumor cells but also in various other cell types including microglia, macrophages, oligodendrocytes and astrocytes.



 Altogether, this argues that the mesenchymal state is extrinsic, driven by the macro environment, can be induced in completely different cell types and its regional distribution suggests wound-healing and hypoxia to be the main drivers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d224367def8e1deca00ec99d840b021d59cf13" target='_blank'>
              P02.32.A SINGLE CELL RNA-SEQUENCING OF SUPRAMARGINAL GLIOBLASTOMA RESECTIONS REVEAL THAT THE MESENCHYMAL SIGNATURE IS CONFINED TO THE TUMOR BULK AND IS SHARED ACROSS MALIGNANT AND NON-MALIGNANT CELL TYPES
              </a>
            </td>
          <td>
            J. Kjaer Jacobsen, L. Hu, C. Mannens, K. Siletti, O. Persson, S. Linnarsson
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is an incurable pediatric brain tumor, with radiotherapy offering only transient benefit before inevitable recurrence. While recent studies have revealed the cellular complexity of DMG, the relationship between intratumoral heterogeneity and disease progression — particularly under therapeutic pressure — remains poorly understood. In this study, we integrate longitudinal single-cell- and spatial transcriptomics with single-cell chromatin accessibility profiling and radiomics to investigate how distinct DMG cell states and microenvironmental components evolve in response to treatment, contributing to resistance and the emergence of therapy-persistent niches. We applied this multi-omic approach to a unique cohort of 10 matched diagnostic and autopsy DMG samples obtained from children enrolled in the PNOC023 clinical trial. Spatial transcriptomic profiling revealed that post-treatment autopsy tissues showed consistent enrichment of oligodendrocyte progenitor cell-like malignant populations, suggesting that this stem-like subpopulation is selectively retained or expanded following radiotherapy in DMG. Moreover, integration of single-cell RNA and ATAC sequencing with cerebrospinal fluid microRNA analyses further revealed molecular signatures of resistance and uncovered minimally invasive biomarkers of treatment response. To complement our molecular analyses, we applied radiomic profiling of longitudinal MRI scans to non-invasively monitor changes in tumor heterogeneity over the course of treatment in this patient cohort. In parallel, we developed a data-driven mathematical model using patient-derived DMG xenografts to simulate tumor cell responses to radiotherapy and generate quantitative predictions of preclinical treatment outcomes. We employed this model to explore alternative radiation fractionation schemes and identify optimal dosing strategies that most effectively deplete therapy-resistant tumor cell populations in DMG. Together, these integrated spatial and molecular datasets provide a high-resolution framework for understanding how radiotherapy remodels the DMG tumor ecosystem. By identifying persistent cellular populations and spatial niches that survive treatment, this work lays the groundwork for future precision therapies aimed at targeting the cellular reservoirs that drive DMG recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4395d6f286b7b33609966b9d19ccc4d41894fd90" target='_blank'>
              RBIO-16. Mapping the Spatiotemporal Evolution of Diffuse Midline Glioma Under Radiotherapy
              </a>
            </td>
          <td>
            Costanza Lo Cascio, Shashank Katiyar, Simone Bruno, Carlos A O De Biagi Junior, Danielle Bitterman, Joana G. Marques, Sina Neyazi, Sharmistha Pal, David Kozono, Benjamin H. Kann, Charles Stiles, Sabine Mueller, Franziska Michor, D. Haas-Kogan, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant pediatric brain tumor of the cerebellum characterized by extensive molecular, biological, and clinical heterogeneity. Current genetically engineered mouse models have only partially replicated this diversity, and harbor species-specific limitations owing to fundamental differences between mouse and human development. Therapeutic strategies that accommodate MB heterogeneity are required to improve patient outcomes but necessitate biologically faithful models that are predominantly lacking. To this end, we have established human pluripotent stem cell-derived cerebellar organoids that recapitulate early human cerebellar development and enable deep mechanistic studies into MB oncogenesis. Transcriptomic profiling and immunophenotyping of our organoids throughout developmental timepoints revealed cellular compositions and differentiation trajectories reminiscent of cerebellar development in vivo. As our cerebellar organoids faithfully reproduce putative lineages of origin for distinct MB subgroups, we’ve endogenously Cre-tagged stem cell lines and generated various expression constructs to manipulate known driver genes in specific cellular compartments to generate accurate human cerebellar organoid-based disease models. We furthered the utility of this ex vivo human system by establishing a cerebellar organoid/patient-derived xenograft (PDX) co-culture platform that fosters the sustainability of MB PDXs in culture. Upon co-culturing with cerebellar organoids, we observed sustained viability and proliferation of MB PDXs, both of which are unprecedented in standard in vitro cultures. Further, we observed differential electrophysiological properties in PDX-harboring organoids compared to standalone organoids and identified putative signaling pathways crucial for cellular crosstalk between tumor and their supporting microenvironment. Our ongoing multi-faceted MB modeling strategies leveraging cerebellar organoids engender new experimental opportunities otherwise incompatible with pre-existing approaches, providing an efficient platform for advancing disease biology and identifying therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9c6a6fee26a49ce1740bbf42bd5633cf13397a" target='_blank'>
              TMOD-13. Human pluripotent stem cell-derived cerebellar organoids as a robust platform for studying medulloblastoma pathogenesis
              </a>
            </td>
          <td>
            Stephanie C Wu, Rachel Xu, Yiran Li, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Background Despite notable advances with immune checkpoint inhibitors (ICIs) in esophageal squamous cell carcinoma (ESCC), their clinical efficacy remains limited, largely due to CD8⁺T cell exhaustion. Among these, progenitor exhausted T cells (Tpex) represent a key subset with stem cell–like features that sustain durable anti-tumor immunity. Methods We applied multi-color immunohistochemistry (mIHC) to determine the spatial distribution and clinical significance of Tpex cells within the tumor microenvironment (TME) of ESCC. Publicly available single-cell RNA sequencing (scRNA-seq) datasets were further analyzed to characterize Tpex cell phenotypes, differentiation trajectories, and intercellular communication networks. Results Tpex cells constituted a distinct subset of infiltrating CD8⁺T cells and represented a transitional stage of the exhaustion continuum. A higher degree of Tpex infiltration was significantly associated with improved overall survival in ESCC patients. Moreover, scRNA-seq data from patients treated with PD-1 blockade revealed that responders harbored markedly enriched Tpex populations compared with non-responders. Conclusion Our findings identify Tpex cells as a critical prognostic and immunologically active CD8⁺T cell subset in ESCC. Their abundance and functional engagement are closely associated with favorable clinical outcomes and response to PD-1 blockade. Furthermore, their stem cell-like properties may be pivotal in shaping durable anti-tumor immunity and could provide novel therapeutic targets to enhance the efficacy of PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c5eac76128741742f8a923e04cbd692a6d7928" target='_blank'>
              Progenitor-exhausted T cell as prognostic indicator in esophageal squamous cell carcinoma: illuminating their key contribution to tumor immunity
              </a>
            </td>
          <td>
            Yi Liu, Hongwei Jiang, Zhang Fang, Bin Xu, Junjun Chen, Xiao Zheng, Renhao Geng, Lujun Chen
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) are a promising autologous cell therapy to treat solid tumors. TILs are manufactured by expanding and reinfusing tumor-reactive T cells from tumor biopsies. Efficacy of TIL therapies has been limited by the heterogeneity of expanded TIL products and the high prevalence of dysfunctional exhausted CD8+ T cells (TEX). While a subset of CD8+ TILs co-expressing CD103 and CD39 are enriched for tumor-reactive TILs across multiple cancer types, these cells are often in the TEX state with low proliferative potential. To identify regulators of human TIL proliferation, we screened an open reading frame library encoding for all human transcription factors (TFs). RELB emerged as the dominant driver of human TIL expansion with a skew towards CD8+ cells. TCR diversity was maintained after multiple days of in vitro expansion driven by RELB. Transcriptome profiling of multiple RELB-expressing TIL subtypes revealed a shift towards a memory/costimulatory-like phenotype. Using a HER2-targeting CAR and tumor co-culture model, RELB conferred improved persistence after multiple tumor challenges in vitro and improved solid tumor control in mouse xenografts in vivo. Finally, co-culture of RELB-overexpressing TILs with patient-matched tumor organoids showed an increase in TIL product polyfunctionality, tumor reactivity, and tumor killing. Collectively these results support promoting RELB expression as a strategy for broadly enabling TIL therapy for treating solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0439a6de8243c20d60a46c125842a0d34a0ceda" target='_blank'>
              RELB Reprograms Exhausted Tumor-Infiltrating Lymphocytes for Improved Adoptive Cell Therapy
              </a>
            </td>
          <td>
            Christian D. McRoberts Amador, Rachel E. Conover, Michael C. Brown, Liliana S. Lyniv, P. Noldner, Ying Zhou, Aretha R. Gao, Sean R. McCutcheon, S. Antonia, Charles A. Gersbach
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Nodular lymphocyte-predominant Hodgkin lymphoma (NLPHL) is a rare cancer, and few studies have comprehensively investigated the immune microenvironment and rare lymphocyte-predominant (LP) cells. Here we develop a NLPHL specific lymphocyte-predominant ecotype (LPE) model to identify 34 distinct cell states across 14 cell types that co-occur within 3 LPEs for 171 cases. LPE1 and LPE2 were characterized by immunosuppressive microenvironments with high expression of B2M on LP cells, CD8 T-cell exhaustion, immune checkpoint genes expressed by follicular T-cells, and an improved freedom from progression compared to LPE3 in training (n = 109, with 65% LPE1/2) and validation cohorts (n = 62, with 61% LPE1/2). We validate the co-occurrence and co-localization of cell states using spatial transcriptomics. Protein expression of HLA-I and HLA-II on LP cells and SSTR2 on dendritic cells was predictive of LPE1 (C-statistic=0.69), LPE2 (C-statistic=0.79), and LPE3 (C-statistic=0.60). This study establishes a clinically relevant biologic categorization for NLPHL. Nodular lymphocyte-predominant Hodgkin lymphoma (NLPHL) is a rare cancer. Here, the authors develop a NLPHL specific model to identify 34 distinct cell states across 14 cell types that co-occur within 3 lymphocyte predominant ecotypes (LPEs) for 171 cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687e04fd2e7cfd53da0cba5d8ceaacb2510f65e8" target='_blank'>
              Distinct cell state ecosystems for nodular lymphocyte-predominant Hodgkin lymphoma
              </a>
            </td>
          <td>
            Ajay Subramanian, Shengqin Su, J. Flerlage, S. Alig, S. Younes, L. Marks, Chelsea C. Pinnix, F. Vega, Raphael Steiner, Priyadarshini Kumar, Heidi Mocikova, A. Sýkorová, V. Procházka, Cristiane Milito, Pamela Allen, D. Paulino, Alan Ramsay, T. Flerlage, Monica Palese, Robert West, ChunFang Zhu, Troy Noordenbos, J. Schroers-Martin, Shuchun Zhao, N. J. Park, A. Kalbasi, E. J. Moding, Aaron M. Newman, R.H. Advani, Richard T Hoppe, Maximilian Diehn, Y. Natkunam, A. Alizadeh, M. Binkley
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Traditional two-dimensional (2D) cell cultures are widely used in cancer research but fail to replicate the complex interactions that occur in tumors in vivo. Three-dimensional (3D) models more closely mimic the tumor microenvironment by preserving dynamic cell–matrix communication. In this study, we developed and characterized 3D spheroids from two breast cancer cell lines with distinct metastatic potential (MCF-7 and MDA-MB-231). Significant differences in cell morphology, epithelial-to-mesenchymal transition (EMT) markers expression, and functional behavior compared to 2D cultures were observed. The spheroids also showed the distinct expression of key receptors (ERs, EGFR, IGF1R) and matrix molecules (syndecans and matrix metalloproteinases). Bioinformatic analysis further revealed the clinical relevance of these matrix regulators in breast cancer prognosis. Overall, these findings introduce an informative, matrix-free cell culture platform for studying breast cancer progression and potentially exploring new therapeutic approaches. Abstract Background/Objectives: Conventional two-dimensional (2D) cell cultures offer valuable insights into cancer cell biology; however, they lack in replicating the complex interactions present in solid tumors. Therefore, research has shifted towards the development of three-dimensional (3D) cell models that recapitulate the dynamic cell–cell and cell–matrix interactions within the complex tumor microenvironment (TME), better resembling tumor growth and initial stages of dissemination. Extracellular matrix, a key component within the TME, regulates cell morphology and signaling, influencing key functional properties. Breast cancer remains the most frequently diagnosed cancer type in women and a leading cause of cancer-related mortality. Methods: The aim of the present study was the development of breast cancer cell-derived spheroids, utilizing two breast cancer cell lines with differential estrogen receptor (ER) expression profile, and their characterization in terms of morphology, functional properties, and expression of epithelial-to-mesenchymal transition (EMT) markers and matrix signatures implicated in breast cancer progression. To this end, the ERα-positive MCF-7, and ERβ-positive MDA-MB-231 breast cancer cell lines were utilized. Results: Our findings revealed notable phenotypic transitions between 2D and 3D cultures, which were further supported by differential EMT markers expression. Moreover, spheroids exhibited distinct expression profiles of key receptors [ERs, epidermal growth factor receptor (EGFR) and insulin-like growth factor receptor (IGF1R)] and matrix molecules (syndecans, and matrix metalloproteinases), accompanied by altered functional cell properties. Bioinformatic tools further emphasized the interplay between the studied matrix regulators and their prognostic relevance in breast cancer. Conclusions: Overall, this study introduces a simple yet informative 3D breast cancer model that captures key TME features to better predict cell behavior in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454047e64cbe106ca9e24ee78bd584b4344b2361" target='_blank'>
              Spheroid-Based 3D Models to Decode Cell Function and Matrix Effectors in Breast Cancer
              </a>
            </td>
          <td>
            Sylvia Mangani, Christos Koutsakis, Nikolaos E. Koletsis, Z. Piperigkou, Marco Franchi, Martin Götte, Nikos K. Karamanos
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Background Immunotherapy has transformed cancer treatment, highlighting the importance of effective antitumor immunity to fight cancer. However, its success in pediatric cancer remains limited, underscoring the urgent need to identify new immunotherapeutic targets. In this study, we explored the clinical relevance of B cells and tertiary lymphoid structures (TLS) in neuroblastoma (NB), a pediatric tumor with a heterogeneous immune landscape. Methods We analyzed 87 treatment-naïve NB specimens, spanning both localized and metastatic disease previously characterized for T-cell and dendritic cell (DC) infiltration. B cells were detected by immunohistochemistry, and plasma cells were quantified using multiple immunofluorescence. Spatial organization and functional status of immune cells within TLSs were assessed by imaging mass cytometry using a 29-antibody panel. In parallel, gene expression profiles were obtained through NanoString PanCancer Immune Profiling and further validated using publicly available bulk and single-cell RNA-sequencing data from untreated and treated NB samples. These transcriptomic datasets were used to support protein-level findings and to identify prognostic gene signatures. Results B-cell infiltration in NB tumors strongly correlated with the presence of T cells and DCs at both protein and transcriptomic levels, and was associated with improved prognosis. Similar to other solid tumors, B cells in NB were either scattered throughout the tumor or organized into TLSs of varying maturity. Spatial proteomic and transcriptomic analyses revealed that localized tumors often contain mature TLSs, with functional B cells able to antigen presentation and immunoglobulin expression, alongside high cytotoxic T cells. In contrast, metastatic tumors primarily exhibited immature TLSs, with evidence of B-cell and T-cell dysfunction. Importantly, we identified gene signatures associated with B cells and TLSs that not only predicted survival in NB but were also prognostic in multiple adult cancers. Conclusions Our findings highlight a central role for B cells and TLSs in shaping the immune microenvironment of NB. Their presence and maturation status are linked to clinical outcome, suggesting their potential as prognostic biomarkers and targets for novel immunotherapeutic strategies in pediatric oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dadde0b86ab40cec87507bbdba192005334b0d" target='_blank'>
              Mapping B cells and the immune landscape of tertiary lymphoid structures reveals their clinical impact in neuroblastoma
              </a>
            </td>
          <td>
            O. Melaiu, M. Chierici, Paula Gragera, Nicolò Lazzaro, L. L. Petrilli, J. Wienke, Francisca J. Bergsma, B. M. Verhoeven, C. De Stefanis, Valentina D'Oria, M. C. Benedetti, Giovanni Barillari, R. Alaggio, M. A. De Ioris, M. Vinci, Ninib Baryawno, R. Carsetti, G. Jurman, Jan J. Molenaar, Franco Locatelli, D. Fruci
          </td>
          <td>2025-11-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast Cancer is one of the most prevalent malignancies and the cause of mortality among women. Despite technological advancements, diseases have inherent heterogeneity, especially in the tumor microenvironment (TME), which poses a significant challenge for effective therapy. The study explores the spatial heterogeneity of two key molecular markers, EPCAM (Epithelial Cell Adhesion Molecule) and ERBB2 (HER2).  The research was conducted based on spatial transcriptomics and a graph-based clustering method. EPCAM plays a crucial role in maintaining epithelial integrity, and its dysregulation facilitates tumor invasion and metastasis. On the other hand, ERBB2 is a well-characterized oncogene that drives proliferation in HER2-positive breast cancer. The study leveraged 10x Genomics Visium data and a high-resolution histopathology image to analyze gene expression and spatially variable genes (SVGs) within the tissue. The study identified eight functional spatial domains. ERBB2 expression was localized to tumor cores, which reflects the role as an oncogenic driver. On the other hand, EPCAM exhibited a wider distribution, highlighting the involvement in tumor-stroma interactions and epithelial-mesenchymal transition (EMT). The findings demonstrate the utility of spatial transcriptomics in two aspects. The first is resolving tumor heterogeneity, and the second is to underscore the importance of spatial context in understanding the roles of EPCAM and ERBB2 for cancer progression. The insights contribute to the development of precision medicine approaches that target a specific tumor region for improving the therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b117f399cc68a98eef4707a09f6308f49ac08645" target='_blank'>
              Spatial Heterogeneity of EPCAM and ERBB2 Expression in Breast Cancer: Insights from Spatial Transcriptomics
              </a>
            </td>
          <td>
            Muhammad Afrizal Amrustian, Bita Parga Zen, Aminatus Sa'adah
          </td>
          <td>2025-10-27</td>
          <td>International Journal on Advanced Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and deadliest primary brain tumor, with no effective therapies and a longstanding record of failed phase 3 clinical trials—often without mechanistic insight into why promising approaches fall short. To address this critical gap, we implemented a novel on-therapy, longitudinal, and multi-regional tissue sampling strategy in patients treated with rQNestin34.5v2, an engineered oncolytic biologic based on herpes simplex virus 1, to explore how tumor and immune ecosystems evolve during treatment. Using high-resolution spatial transcriptomics (Xenium) and targeted spatial proteomics (CyCIF), we profiled over 80 spatially and temporally distinct samples from six patients, capturing over half a million immune, stromal, and malignant cells. This unique dataset enabled unprecedented resolution of the dynamic interactions between tumor and immune compartments in situ over the course of therapy. Our preliminary analysis revealed discrete, virus-associated lymphoid-like structures that formed early after treatment initiation. These structures were enriched in T cells, B cells, and myeloid cells, and were associated with localized reductions in tumor cell density. Over time, we observed a shift from an initially inflamed microenvironment to a more immunosuppressive state. By the time of radiographic progression, these lymphoid structures had largely dissipated—underscoring the importance of on-treatment sampling in capturing transient, treatment-responsive immune architectures. While these findings are preliminary, they suggest a transient immune structure linked to early therapeutic response, whose collapse may contribute to treatment resistance. Our study illustrates how integrative, on-therapy spatial profiling can begin to reveal mechanisms of response and resistance in GBM, providing a framework for refining future immunotherapeutic strategies in brain tumors and other treatment-refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf60db76ebc26534935de49d68b0018aa310569" target='_blank'>
              IMMU-67. On-Treatment Spatiotemporal Profiling of Glioblastoma During Oncolytic ImmunoTherapy
              </a>
            </td>
          <td>
            Charles Couturier, Jennifer Gantchev, Gerard Baquer, Peter Zhang, Alicia D’Souza, Michael S Regan, Kaitlyn Bootz, J. Bonney, Md Amin Hossain, G. Teamlab, Atul Deshpande, Forest M. White, Kenny Yu, E. Chiocca, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains among the most aggressive and intractable human cancers, driven by profound intratumoral heterogeneity, dynamic cellular plasticity, and robust mechanisms of immune escape. To overcome these barriers, we developed an integrated discovery and engineering pipeline that combines single-nucleus RNA sequencing (snRNA-seq), antibody generation through both immunization and generative AI design, yeast-based directed evolution, and rationally engineered CAR-T cell architectures. We first assembled a high-resolution snRNA-seq atlas spanning malignant GBM, adult brain, and developmental brain tissues. Cross-comparative analysis revealed PTPRZ1 as a consistently expressed, tumor-enriched surface antigen with minimal expression in normal adult tissues. Further analysis across developmental datasets confirmed PTPRZ1 expression during neurodevelopment, particularly in glioma stem cells and oligodendrocyte progenitor-like states, and showed retention of expression across all major GBM differentiation trajectories. These findings suggest that PTPRZ1 marks a persistent cellular state within the glioma hierarchy, making it a stable and tractable immunotherapeutic target despite GBM’s plasticity. To generate high-affinity binders against PTPRZ1, we pursued two parallel strategies: (1) phage display from immunized hosts followed by affinity maturation via yeast-based directed evolution, and (2) de novo nanobody design using structure-aware generative models trained on antibody sequence and structural datasets. Candidate binders were refined through site-directed mutagenesis to enhance specificity and binding kinetics. Given the known challenges of targeting PTPRZ1, including extracellular domain shedding and glycosylation-dependent surface stability, we engineered a suite of TCR-mimetic CAR-T cells incorporating modular costimulatory domains to enable tunable activation thresholds and improved resistance to exhaustion. These constructs emulate core features of physiological TCR signaling, offering increased antigen sensitivity, functional persistence, and reduced tonic signaling in hostile tumor contexts. This work establishes a modular platform for precision antigen discovery and CAR-T cell engineering in GBM and validates PTPRZ1 as a compelling, developmentally anchored immunotherapeutic target. More broadly, it offers a scalable roadmap for rational design of next-generation cellular therapies against solid tumors that evade conventional targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693e783962995ec2b156273884091139270f0e5f" target='_blank'>
              EXTH-68. Targeting Tumor Heterogeneity And Plasticity in Glioblastoma With Novel CAR T Cells Targeting PTPRZ1
              </a>
            </td>
          <td>
            Aditya Mohan, Barbara Lipes, Charla Gentry, Rushil Yerabelli, Kisha Patel, Bhairavy J Puviindran, E. Blandford, Ariel Gonzalez, Daniel S. Wilkinson, John H. Sampson, Michael Gunn, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Hypoxia is a critical driver of tumor aggressiveness in high-grade gliomas, yet its cell-type-specific effects on immune cell populations within the tumor microenvironment (TME) remain poorly understood.



 We investigated how hypoxia shapes the spatial distribution and functional states of monocyte-derived macrophages (MDMs) and brain-resident microglia (MG) in diffuse astrocytomas and glioblastomas (GB) by using cyclic immunohistochemistry (cIHC), single-cell RNA sequencing, spatial transcriptomics, and in vitro cell culture models.



 Hypoxia induces divergent responses in these myeloid subsets, driving spatial immune patterning. CD163⁻ MDMs dominate hypoxic niches in GB, while MG populations are excluded from these regions, correlating with hypoxia-induced TNF upregulation, stress response signatures, and dampened interferon responses. Conversely, MDMs exhibit a shift towards hypoxia-associated immunosuppressive traits and altered metabolism. GBs display elevated hypoxic intensity compared to diffuse astrocytomas, as supported by hypoxia-response gene expression in TCGA dataset. In vitro, MG show heightened sensitivity to hypoxia-associated acidity compared to MDMs, suggesting together with microglial hypoxia response genes that their exclusion from hypoxic zones results from intrinsic vulnerability.



 Our findings reveal that hypoxia remodels the TME by promoting immunosuppressive MDM accumulation and depletion of MG, creating spatially distinct immune landscapes that may underlie glioma progression. These results highlight hypoxia-driven immune dysregulation as a therapeutic target and underscore the importance of cell-type-specific strategies to counteract TME-driven immunosuppression in malignant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad214ece57abc6b1e7d242c9f8b34d0a20415450" target='_blank'>
              P02.29.B HYPOXIA SHAPES TUMOR IMMUNE MICROENVIRONMENT THROUGH CELL-TYPE DEPENDENT RESPONSES IN DIFFUSE ASTROCYTOMAS
              </a>
            </td>
          <td>
            A. Tiihonen, I. Salonen, I. Koivisto, A. Ritamäki, S. Jaatinen, T. Hyvärinen, J. Lotila, J. Kreutzer, M. Valkonen, G. Karabiyik, S. Mäntylä, M. Mohammadlou, M. Hoikka, J. Beck, R. Rölz, M. Marttinen, M. Nykter, J. Haapasalo, P. Ruusuvuori, S. Parkkila, P. Kallio, S. Hagman, J. Kesseli, V. Ravi, H. Haapasalo, A. Jukkola, K. Joseph, K. J. Rautajoki
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Glioblastoma (GB), IDH-wildtype, exhibits increased incidence and poorer prognosis in elderly patients, yet the biological underpinnings of this age-related disparity remain poorly understood. Since previous studies have suggested only a limited increase in potential GB-driving genetic alterations in the human brain with age, we hypothesize that non-genetic factors may present aging-dependent contributors to GB pathobiology. In particular, age-associated microenvironmental drivers have been revealed in other cancer types, but their relevance in GB remains underexplored. Moreover, recent GB mouse model studies have suggested that poorer survival in older animals is driven by factors distant from the tumor bulk (“extratumoral”), but this has not been examined in humans due to limited analyses of matched non-tumor brain tissue. To address this gap, we performed single-cell RNA sequencing (scRNA-seq) on formalin-fixed paraffin-embedded tumor samples from 36 GB patients, comprising 18 elderly (65–88, median 75 years) and 18 younger (35–55, median 47 years) individuals. Cohorts were balanced by sex and by the three major epigenetic GB subtypes RTK I, RTK II, mesenchymal, as defined by DNA methylation profiling. Whole-exome sequencing was performed to control for mutational events. For half of the patients, tumor-adjacent normal-appearing cortex was histologically selected, and additionally profiled by scRNA-seq to assess potential age-dependent extratumoral changes. Our spatially stratified approach enabled the detection of diverse malignant cell populations within the tumor core, as well as enrichment of non-malignant cell types often underrepresented in tumor-core analyses, including astrocytes, neurons, and OPCs. Preliminary analyses indicate a region-specific increase in potentially tumor-promoting cellular signatures across immune and glial compartments in older patients, such as more reactive myeloid signatures at the tumor border. Ongoing analyses include integration of these findings with published normal brain datasets and correlation with clinical outcomes. Our study thereby provides a comprehensive framework for understanding aging-associated drivers in human GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5116a94bf42e3504a7df9be18b8c861fc00afa09" target='_blank'>
              TMIC-76. Deciphering aging-associated drivers of glioblastoma biology at the tumor core and border
              </a>
            </td>
          <td>
            Laurens Zaschke, S. Schmid, D. Teichmann, Peggy Wolkenstein, Jakob Nückles, Christin Siewert, Brendan Osberg, Huy Philip Dao Trong, J. Onken, A. Koch, Matthias Endres, Leif Ludwig, David Capper, Ilon Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Lineage plasticity enables cancer cells to bypass therapeutic pressure, but the underlying chromatin mechanisms remain incompletely defined. A recent study by Lu et al. uncovers how the transcription factors NKX2-1 and FOXA2 reprogram 3D genome architecture to activate neuroendocrine gene networks during the transition from castration-resistant prostate cancer (CRPC) to neuroendocrine prostate cancer (NEPC) (Lu et al., 2025). Importantly, pharmacological inhibition of CBP/p300 markedly suppresses NEPC growth in vitro and in vivo, revealing a promising therapeutic avenue for advanced disease. These findings illuminate how spatial chromatin reorganization can drive cell fate transitions, positioning 3D genome dynamics as a key determinant of cellular identity in both development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c9ebfa025117c138be3767acc9d75dcb26318e" target='_blank'>
              3D chromatin dynamics in cellular neuroendocrine transformation
              </a>
            </td>
          <td>
            Jianhua Xiong
          </td>
          <td>2025-10-17</td>
          <td>Protein &amp; Cell</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T. Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Advanced melanoma exhibits high primary resistance to immunotherapy and limited long-term response, suggesting the need for sensitive biomarkers and novel therapeutic targets to enhance treatment efficacy. We aimed to identify prognostic biomarkers and therapeutic targets for melanoma immunotherapy. Methods Bulk RNA-sequencing data from public melanoma cohorts were analyzed to classify tumor samples into two clusters based on distinct expression patterns of immune-related hallmarks. Differential gene expression and least absolute shrinkage and selection operator (LASSO) Cox regression analysis identified nine core genes, which were integrated into a prognostic risk model. Single-cell RNA-sequencing analysis was then performed to characterize the immune cell heterogeneity and functional interactions in the tumor microenvironment. Results The single-cell analysis revealed that CST7 (cystatin F)-expressing macrophages/monocytes were enriched in an M1 macrophage signature, indicative of an antitumor phenotype. These immune cells were significantly more frequent in immunotherapy responders than in non-responders. Further investigation identified a signaling interaction between CST7+ macrophages/monocytes and a cytotoxic T-cell subset via the ICOSL (inducible T-cell costimulator ligand)-ICOS (inducible T-cell costimulator) axis, which was more prominent in therapy responders. Conclusions These findings suggest that CST7+ macrophages/monocytes are associated with immunotherapy response by correlating with cytotoxic T-cell activation through the ICOSL/ICOS pathway. The observed interaction highlights the potential for combining immune checkpoint inhibitors (ICIs) with ICOS/ICOSL agonistic antibodies to enhance and sustain long-term immunotherapy efficacy in patients with melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815027c5e50dc7d8dd7946768f3069dc44854a8f" target='_blank'>
              CST7+ macrophages/monocytes in melanoma: single-cell insights into immunotherapy response
              </a>
            </td>
          <td>
            Kaitao Yao, Fan Wu, Daiwei Liu, Boying Zheng, Jing Li, Yang Liu, Li Wang, Liuqing Zheng, Zhanlin Li, Gang Zhou
          </td>
          <td>2025-10-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioma evolution is associated with disease progression and therapeutic resistance. We recently identified a group of MAPK-driven glioma with CDKN2A homozygous deletion that are low-grade and immune enriched, characterized by the presence of antigen presenting cells (APCs) and activated cytotoxic CD8⁺ T cells. Over time, however, these tumors can progress to aggressive, higher-grade tumors. While low-grade tumors may be amenable to immunotherapeutic strategies, it is unknown how the tumor and its microenvironment evolve. To investigate this, we tested human tumors before and after progression within the same patients (n=7) over time (longitudinally) and across patients (n=21) using transcriptomic and proteomic approaches. In neoplastic cells, genetic evolution was accompanied by increased tumor cell proliferation and differentially expressed genes were enriched for telomere organization and maintenance (RTEL1, TERC, HSP90) and response to hypoxia (VEGFA, VEGFC, HIF1A). In parallel, the immune response became predominantly immunosuppressive, marked by an expansion of immunosuppressive macrophages, a reduction in APCs, including dendritic cells, and transcriptional reprogramming of T cells. At progression, T cells displayed increased exhaustion, diminished cytotoxicity (reduced GZMA), and enhanced expression of immunosuppressive mediators (IL-10 and TGF-β). Using spatial proteomics and spatial transcriptomics, we investigated factors potentially driving tumor–immune co-evolution. With disease progression, reductions in APCs and activated CD8 T cells were associated with altered cell-cell signaling. CXCL16–CXCR6 signaling between APCs and T cells was decreased and CXCL14, a chemokine implicated in recruitment of both CD8 T cells and APCs to tumors, was decreased. Indeed, CXCL14 expression correlated with an increased CD11c expressing population of antigen presenting cells. Thus, tumor progression is accompanied by a coordinated evolution of both tumor and immune compartments, driven in part by changes in cell-cell communication within the tumor microenvironment. Ongoing studies are investigating how targeting these communication pathways within the tumor microenvironment may alter tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f32c872c50770144e79a4ef320e0649d8c4401f" target='_blank'>
              TMIC-88. MAPK- driven glioma progression and reprogramming of the tumor-associated immune response
              </a>
            </td>
          <td>
            Elaheh Hashemi, Akshaya R.L, Lin Wang, Viva Voong, Hadeesha Piyadasa, Benjamin Oberlton, Spencer Bergland, Anupam Kumar, R. Prins, Sean C. Bendall, Michael Angelo, Mitchell S. Berger, S. Hervey-Jumper, H. Okada, Aaron A. Diaz, Joanna Phillips
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG‐on‐Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real‐time monitoring using wide‐field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient‐derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma‐on‐Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small (Weinheim an Der Bergstrasse, Germany)</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment has become a central focus in cancer research, offering new insights into cancer behavior and showing strong potential for improving cancer treatment. However, in veterinary oncology, specifically feline mammary tumors, research on the topic lags far behind. This review summarizes current knowledge on key elements of the tumor microenvironment in the feline mammary tumors, including tumor necrosis, fibrosis, angiogenesis, adipose tissue, tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition. Although studies in cats remain limited, insights from human breast cancer are frequently drawn, creating a combined perspective that may help drive new research into the diagnostic, prognostic, and therapeutic potential of the tumor microenvironment in feline mammary tumors. Abstract The tumor microenvironment (TME) comprises neoplastic and stromal cells, and extracellular matrix elements, all engaging in a complex interplay that ultimately dictates tumorigenesis, cancer progression, and therapeutic response. While extensive research on the TME has been conducted in human oncology, data on its veterinary counterpart, particularly in feline mammary tumors (FMTs), are still scarce. In this review, we explore current understanding of feline mammary carcinoma (FMC) microenvironment, focusing on tumor necrosis, fibrosis, angiogenesis, adipose tissue tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition (EMT) and their prognostic implications. In FMC, remodeling of collagen fibers, cancer-associated fibroblasts (CAFs), regulatory T cells (Tregs) and elevated serum leptin have been associated with poor prognosis, whereas stromal cytotoxic T cells correlate with more favorable outcomes. By contrast, findings on necrosis and pro-angiogenic factors remain inconsistent, and research on extracellular vesicles (EVs) is still in its early stages. This review presents insights from human breast cancer (HBC) that further support and elucidate the potential relevance of these TME components. As FMCs are highly aggressive tumors, a deeper understanding of their microenvironment could not only improve prognostic accuracy but also uncover novel therapeutic targets. Furthermore, due to their similarities, FMCs offer a potential valuable spontaneous model for HBC, particularly for the aggressive triple-negative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acfc9c9171773b52e82ceb2afc7e5c11c59650bc" target='_blank'>
              Inside the Tumor: Decoding the Feline Mammary Tumor Microenvironment and Its Prognostic Value—A Review
              </a>
            </td>
          <td>
            J. Rodrigues-Jesus, Ana Canadas-Sousa, Hugo Vilhena, P. Dias-Pereira
          </td>
          <td>2025-10-01</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) is a major challenge in oncology due to its high recurrence rate, highlighting the need to identify recurrence drivers. Based on the data provided by Zhang Zemin’s team in their research, we selected survival metadata from 119 patients with lung squamous cell carcinoma and data on the proportions of 51 immune cell types. We then combined machine learning methods to analyze the intrinsic tumor factors and tumor microenvironment (TME) in the recurrence of lung squamous cell carcinoma (LUSC). Chi-square tests identified immune cell subsets enriched in recurrent patients, such as high-MKI67 Tregs, low-FGFBP2 NK cells, low-FOXP3 Tregs, and low-KLRB1 CD8 + T cells. A random forest algorithm further pinpointed pathological response rate (PRR) as the primary predictive factor, with high-MKI67 Tregs as a key secondary contributor. Univariate and multivariate Cox regression analyses and Kaplan-Meier survival curves confirmed that pathological complete response (pCR) and low Treg_MKI67 expression were associated with better survival (p = 0.0055 and p = 0.011, respectively). Patients achieving pCR and with low Treg_MKI67 expression had superior recurrence-free survival (RFS; 2-year RFS: 96.7% vs. 57.9% for non-pCR and high Treg_MKI67, p = 0.00066). These findings underscore the prognostic value of PRR and TME markers. They also highlight the potential of using single-cell RNA sequencing and machine learning to guide personalized therapies and reduce recurrence risk by targeting Treg activity, laying the groundwork for integrating immune markers into clinical practice to improve LUSC prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07193-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771d9eb2ac87b50db8e745eeabbaa9aea5706912" target='_blank'>
              Which dominates recurrence: tumor or microenvironment?
              </a>
            </td>
          <td>
            Guoqing Lyu, Xin Deng, Man Yang, Yan Guo, Yuan Zhang, Lihua Wang, Yan Huang, Sun Wu, Yanting Liu, Wenting Lv, Jin-Cheng Guo
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 High-grade gliomas (HGGs) exhibit significant intra- and inter-tumoral heterogeneity, posing a major challenge for chimeric antigen receptor (CAR) T cell therapies that target a single antigen. Multi-targeting CAR T cells offer a promising solution by simultaneously engaging multiple tumour-associated antigens. However, current clinical trials in glioblastoma (GBM) focus on a limited set of antigen targets, restricting therapeutic efficacy. To expand the range of targetable antigens, we performed a comprehensive surface proteomic analysis of primary GBM tumours. This led to the development of a library of 70 CAR constructs targeting a refined pool of novel and clinically validated surface proteins. These CARs were screened in a pooled manner to identify optimal candidates, with top hits validated as mono-CARs in two GBM models, both in vitro and in vivo. The top 10 CAR candidates were then tested in multi-target pools of up to five CARs. Additionally, we incorporated unique protein tags, called Procodes, into the transgene of each CAR. This enabled the tracking and analysis of up to 32 unique sub-populations within a single pooled CAR T cell product via spectral flow cytometry, both in vitro and ex vivo. This facilitated a deeper understanding of multi-CAR dynamics, including the impact of antigen abundance, CAR T cell phenotype, and CAR design on therapeutic efficacy. Through this work, we have identified novel antigen combinations that elicit potent anti-tumour activity in vivo. In parallel, we are uncovering key principles that govern multi-CAR functionality. Notably, our findings suggest that there may be an upper limit to the number of CARs that can be co-expressed before T cell function is compromised. Additionally, certain CAR combinations may be incompatible within the same product, further influencing efficacy. These insights provide a foundation for the rational design of next-generation CAR T cell therapies for brain tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab16b89fa310963cab0d03daaa33d32eaa4d46c" target='_blank'>
              IMMU-41. Investigating Multi-Targeting CAR T Cells to Overcome Heterogeneity in Adult Glioblastoma
              </a>
            </td>
          <td>
            Yasmin Nouri, Ryan Cross, Misty Jenkins
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Despite decades of concerted research and clinical efforts, patient outcomes in glioblastoma (GBM) remain dismal. While chimeric antigen receptor (CAR)- T cell therapies in GBM have produced individual reports of efficacy, they have yet to meaningfully alter standard of care. Tumor heterogeneity, both inter- and intra-patient, likely contributes to these variable responses. Single-cell RNA sequencing (scRNA-seq) conducted in our lab has revealed that this transcriptional heterogeneity is organized along distinct cellular state axes. We sought to overcome this heterogeneity by designing CAR-T cells directed against these cellular states. We hypothesize that CAR-T cells targeting a defined GBM cell state will lead to a depletion of those populations. We report the development of a panel of cell state-targeting CAR-T cell candidates engineered for target-specific activity in vitro. To model the three-dimensional tumor environment, we developed patient-derived GBM organoids that recapitulate the cell state landscape seen in primary tumors. We treated GBM organoids with these cell state-targeting CAR-T cells and assessed cell states by scRNA seq. Unexpectedly, we observed convergent transcriptome shifts to a new cell state with overlapping features of mesenchymal (MES) and interferon-induced signatures. To target this dynamic, we designed a novel CAR-T cell directed against this population and observed feedforward target enrichment in response to therapy. We anticipate that our findings will inform rational design of translational CAR-T cell candidates for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae77281b5ad33a2d2f93d13dc0c418dca38401a5" target='_blank'>
              EXTH-37. Evaluating cell state-targeting CAR-T cells in glioblastoma
              </a>
            </td>
          <td>
            S. Kovatsis, C. Mount, E. Boxer, Jun Zhong, S. Dumont, D. Gerovasilis, Jack Lu, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the most lethal malignancies worldwide, with outcomes shaped not only by genetic alterations but also by the complexity of the tumor microenvironment (TME). The TME encompasses stromal and endothelial cells, extracellular matrix components, gut microbiota, and a diverse array of immune cells that dynamically interact to influence tumor initiation, progression, and therapeutic response. This review delineates the immunological landscape of CRC, highlighting the dual functions of innate immune cells—including tumor-associated macrophages, natural killer cells, dendritic cells, neutrophils, and mast cells—and adaptive immune players such as cytotoxic T lymphocytes, helper T-cell subsets, and B/plasma cells. These cellular interactions contribute to the heterogeneity between immunologically “hot” microsatellite instability-high (MSI-H) tumors, which are highly responsive to immunotherapy, and “cold” microsatellite-stable (MSS) tumors, which remain resistant. Key mechanisms of immune evasion, such as cancer immunoediting, checkpoint signaling, and exosome-mediated communication, are examined alongside prognostic tools like the Immunoscore that serve as biomarkers of immune infiltration. Emerging immunotherapeutic strategies, including checkpoint blockade, macrophage reprogramming, natural killer cell agonists, and microbiome modulation, are discussed with emphasis on both their promise and limitations in CRC management. By integrating current insights into immune–tumor interactions, the review underscores opportunities for developing personalized, TME-targeted interventions to improve CRC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1877a469ae4ff5d08c11f55218aceb0c7af5a6c3" target='_blank'>
              Immunological landscape of colorectal cancer: tumor microenvironment, cellular players and immunotherapeutic opportunities
              </a>
            </td>
          <td>
            Adile Buse Andac-Aktas, Gizem Calibasi-Kocal
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The glioblastoma (GBM) tumor immune microenvironment (TIME) is an immunosuppressive barrier to therapeutic innovation, and unsurprisingly GBM responses to immune checkpoint inhibition (ICI) are rare. Here, we define therapeutic vulnerabilities and mechanisms underlying GBM resistance to ICI therapy.



 Integration of spatial and single-cell approaches from human IDH-wildtype GBMs before and after ICI treatment with preclinical genetic, immunologic, and efficacy studies in immunocompetent mouse GBM models.



 Human GBMs with radiographic responses to ICI had lower pre-treatment IL6 levels on spatial profiling than GBMs resistant to ICI. Multiplexed immunofluorescence revealed ICI-responders were enriched in T-cell proteins (CD3, CD4, CD8) and activation markers (CD25) at the time of salvage surgery; meanwhile, non-responders were enriched in myeloid proteins (CD68, CD163, CD11c). Single-cell RNA sequencing analysis of 32,877 cells from human GBMs showed IL6 is predominantly produced by radial glial like cancer stem cells. Interestingly, mice bearing intracranial SB28 GBM allografts universally suppressed intratumor IL6 levels when treated with immunostimulatory gene therapies and mouse genetic experiments confirmed prolonged survival in animals with an IL6-/- background. Mouse survival was improved by combination systemic treatment with anti-PD1 and anti-IL6 but was not improved by either antibody alone or by local anti-IL6 delivery. Mass cytometry revealed combination therapy reprogrammed the GBM TIME by increasing MHCII+ monocytes, CD103+ migratory dendritic cells (DCs), CD11b+ conventional DCs, and effector CD8+ T cells, and by decreasing immunosuppressive Tregs.



 Systemic anti-IL6 in combination with systemic ICI reprograms the GBM TIME to sensitize preclinical models to ICI, shedding light on a therapeutic strategy for GBM treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2360ab571d81347b402e726648495085f63cb772" target='_blank'>
              P02.02.A IL6 UNDERLIES MICROENVIRONMENT IMMUNOSUPPRESSION AND RESISTANCE TO THERAPY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Young, N. Cho, C. G. Lucas, K. Seo, H. Najem, K. Mirchia, W. C. Chen, N. Zakimi, V. Daggubati, T. Casey-Clyde, M. P. Nguyen, A. Chen, J. J. Phillips, T. Ozawa, M. Aghi, J. Taylor, J. DeRisi, A. Bhaduri, A. Heimberger, M. Berger, N. Butowski, M. H. Spitzer, D. Raleigh
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background and Objective Heterogeneity is a hallmark of non-small cell lung cancer (NSCLC), manifesting as molecular, genetic, and phenotypic variations that complicate treatment and drive resistance. Accurate characterization of intratumor heterogeneity (ITH) is crucial for optimizing therapy. This review explores ITH in NSCLC from spatiotemporal perspectives, focusing on advances in imaging-based quantification and its clinical implications. Methods A systematic literature search was conducted across PubMed, Web of Science, EMBASE, and Cochrane Library (up to June 1, 2025). Two authors independently screened titles/abstracts and performed full-text reviews of relevant English-language studies, with findings synthesized through iterative consensus. Key Content and Findings NSCLC heterogeneity manifests as dynamic spatial and temporal variations in genetics, the tumor microenvironment, and metabolism. Imaging ITH (IITH), utilizing techniques such as texture analysis, histogram methods, shape/volume features, and habitat imaging, allows for the quantification of biologically distinct subregions within the tumor. IITH biomarkers have been shown to predict treatment response, survival outcomes, histopathological subtypes, and molecular profiles. A reduction in heterogeneity following therapy is associated with improved survival. Furthermore, integrating IITH data with multi-omics information enhances the accuracy of prognostic models that track tumor evolution. However, several critical challenges remain, including the standardization of image acquisition and reconstruction processes, tumor segmentation variability, feature reproducibility, and the need for prospective validation across multicenter cohorts. Conclusions IITH metrics provide clinically actionable biomarkers for precision oncology. Future work should prioritize multi-omics integration and standardization to advance clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ec988ad23d9f2f5fe78afd2031adf043164b11" target='_blank'>
              A narrative review of imaging intratumor heterogeneity in non-small cell lung cancer: current advances
              </a>
            </td>
          <td>
            Ping Lin, Hongbin Wang, Bingrun Zou, Ziwei Zhu, Weimin Li, Bojiang Chen
          </td>
          <td>2025-10-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86ecdda13a72d7b290cf8fe9dd99c8ce634923c0" target='_blank'>
              Galectin-9-driven immunosuppressive macrophage population shapes the lymph node metastatic microenvironment.
              </a>
            </td>
          <td>
            Yao Zhou, Xiaowei Liu, Guixiu Xiao, Genpeng Li, Xiujing He, Leyi Tang, Xueyan Wang, Yujia Lin, Jianyong Lei, Jing Jing, Hubing Shi
          </td>
          <td>2025-10-25</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, Daria M Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-09</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: Galectin-3 (Gal-3), encoded by LGALS3, is a β-galactoside-binding lectin involved in diverse tumor-associated processes, including immune modulation, cell cycle regulation, and stress adaptation. Despite its known roles in cancer biology, the full extent of its molecular functions and prognostic relevance across tumor types remains incompletely understood. This study aimed to systematically investigate the transcriptomic impact of LGALS3 deletion and assess its clinical significance in cancer. Methods: We analyzed CRISPR-Cas9 knockout transcriptomic data from the SigCom LINCS database to characterize the consensus gene signature associated with LGALS3 loss using functional enrichment analyses. Pan-cancer survival analyses were conducted using TIMER2.0. Differential Gal-3 protein levels in ductal adenocarcinoma and normal pancreatic tissues were evaluated using the Human Protein Atlas. Finally, functional analyses were performed in pancreatic ductal adenocarcinoma (PDAC). Results: LGALS3 deletion across multiple cancer cell lines led to transcriptomic changes involving mitotic progression, stress responses, and axonal guidance pathways. High LGALS3 expression was significantly associated with worse overall survival in lower-grade glioma, PDAC, uveal melanoma, and kidney renal papillary cell carcinoma. LGALS3 knockout in YAPC cells recapitulated the pan-cancer findings, linking LGALS3 to cell morphogenesis and proliferation. Conclusions: These findings identify Galectin-3 as a key regulator of oncogenic programs and a potential prognostic biomarker in PDAC and other malignancies, with implications for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0894155b476f9fae4b8588df639cdd5d50fcb2b" target='_blank'>
              Transcriptomic and Clinical Profiling Reveals LGALS3 as a Prognostic Oncogene in Pancreatic Cancer
              </a>
            </td>
          <td>
            Grazia Scuderi, S. Mijatović, D. Maksimović‐Ivanić, M. Di Rosa, J. Muñóz-Valle, Alexis Missael Vizcaíno-Quirarte, Gian Marco Leone, K. Mangano, P. Fagone, Ferdinando Nicoletti
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Glioblastoma progression features extensive dissemination of cancer cells throughout the brain. Limited by neuroimaging resolution and lack of specific markers, malignant cell infiltration is clinically undetectable. This makes complete tumor removal impossible and relapse inevitable. During their exodus from the tumor core, glioblastoma cells are traversing distinct brain territories influencing their phenotypic cell state. To better understand the cellular and molecular dynamics of glioblastoma invasion, we profiled glioblastoma patient avatars and clinical samples using total, single-cell and spatial transcriptomics. In addition to gene expression data, active intracellular pathways of glioblastoma invasion were screened with high throughput activated kinase assays (Pamgene). We identified that human glioblastoma stem cells exhibiting proneural (PN) and neuron-progenitor like (NPC) transcriptomic traits in vitro formed highly infiltrative and connected xenografts in patient avatars. Consistent with this, PN/NPC glioblastoma stem cells cultured on primary hippocampal neurons exhibited higher connectivity and motility compared to mesenchymal-like cells. Leveraging single-cell and spatial transcriptome analytics of avatar and patient samples, we performed cell clustering, annotation, pseudotime and RNA velocity analyses to associate glioblastoma phenotypic cell states to cellular functions. We found that invasive glioblastoma cells re-activated genes and pathways related to neuron progenitor maturation when transitioning from the tumor core to the brain microenvironment. To verify whether invasive PN/NPC glioblastoma cell integration would alter brain functionality, we performed preclinical live cerebral blood volume recordings using noninvasive brain functional ultrasounds (fUS). As glioblastomas progressed, we detected a gradual alteration of brain functional connectivity. Modifications happened in brain areas featuring minimal, yet consistent glioblastoma cell colonization, identified by post-mortem pathology analyses. Altogether, our data shed light on the importance of glioblastoma cell adaptability and brain integration during the metastatic progression. They also provide a framework for detecting invisible clinical features such as tumor invasion, focusing on brain functional connectivity assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de15ca728123a2849468be8f7f288fcf64e2b6b" target='_blank'>
              ANGI-14. GLIOBLASTOMA PROGRESSION FROM TRANSCRIPTOMIC REGULATION TO BRAIN FUNCTIONAL CONNECTIVITY
              </a>
            </td>
          <td>
            Madeleine Lackman, C. Bergström, Anaïs Virenque, Kato Herman, Aarti Venkatesan, Juli Udayani, Takashi Namba, Analiz Rodriguez, E. Castrén, P. Laakkonen, Sinem Karaman, V. Le Joncour
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1fe8757b9db03b1f80d6119b9cc7fd8b56ac919" target='_blank'>
              Pan-cancer analysis reveals that tumor microbiomes generate specific epitopes through transcriptional reprogramming
              </a>
            </td>
          <td>
            Jinman Fang, Yuxin Zhang, Wei Wang, Xiao Zuo, Tao Chen, Kecheng Li, Huiya Yang, Tianyang Zhu, Yuanliang Zhang, Dandan Wei, Jianfeng Zhang, Jialong Cui, Zhen Tang, Shouwu Zhang, Lirong Wang, Zhilei Li, Dushan Ding, Ruixia Sun, Dandan Zhu, Yin He, Yanfei Zhou, Chenkun Ye, Dandan Chen, Xueting Lang, Long Jiang, Qian Zhao, Lu Zhang, Hongzhi Wang, Yuanwei Zhang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common and aggressive primary brain tumor in adults, marked by extensive inter- and intratumoral heterogeneity and pronounced therapy resistance. Microglia is a major contributor to this complexity and plays a pivotal role in shaping tumor behavior and modulating responses to therapy. Yet its precise phenotypic and functional states across distinct tumor regions and model systems remain poorly defined. In this study, we employ patient-derived organoids (PDOs) alongside matched primary tissue (PT) to characterize microglial diversity and investigate how microglia contributes to the cellular inter- and intratumoral heterogeneity of GBM.



 Fresh tumor tissue was obtained from 13 GBM patients and sampled from six spatially distinct regions per tumor. PDOs were generated from these regions and compared to matched PT to evaluate microglial presence, distribution, and morphology. To assess intertumoral heterogeneity, we compared microglial characteristics across patients, while intratumoral heterogeneity was examined by analyzing differences between regions within individual tumors. Subsequently, PDOs were treated with temozolomide (TMZ), and microglia was visualized and quantified using Iba1 immunofluorescence to evaluate treatment-induced morphological changes and shifts in microglial abundance.



 Microglia in PT exhibited a microglia morphology more similar to PDOs than to healthy brain tissue. Distinct microglial distribution patterns emerged across PDOs, with microglia typically arranged in one of four configurations: (1) perinuclear, (2) ring-like, (3) at the PDO margin, or (4) diffusely dispersed throughout the PDO. In 5/13 PDOs we found significantly higher microglial abundance compared to PT (PPDO 4 = 0.0013, PPDO 7 = 0.0424, PPDO 8 = 0.0153, PPDO 10 = 0.0162, PPDO 12 = 0.0083). Analysis across patients revealed significant intertumoral heterogeneity in microglial characteristics (p = 0.0405). In terms of intratumoral heterogeneity, regional differences in microglial abundance and morphology were observed within individual PDOs, with statistically significant variation only found in PDO 10 (p = 0.0211). Following TMZ treatment, 4/13 PDOs showed a significant recline in microglial abundance (PDO 2: p < 0.0001; PDO 4: p = 0.0041; PDO 7: p = 0.0015; PDO 10: p = 0.0297).



 Our findings reveal significant inter- and intratumoral microglial heterogeneity in PDOs. Distinct microglial distribution patterns and variable responses to TMZ highlight the complex and patient-specific nature of microglial involvement in the GBM microenvironment. These results underscore the relevance of PDOs as a model system for studying tumor-immune interactions and therapeutic responses in a personalized context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a796a165fe58659df81ee201e3cb5e0d22f92cdf" target='_blank'>
              P02.05.B MAPPING MICROGLIAL DIVERSITY AND TUMOR HETEROGENEITY IN PRIMARY GLIOBLASTOMA AND PATIENT-DERIVED ORGANOIDS
              </a>
            </td>
          <td>
            J. Mazzoni, G. Haug, E. Roth, J. Hiebl, J. Messinger, K. Mehling, E. Meller, R. Nickl, C. Monoranu, A. Appelt-Menzel, C. Hagemann, F. Liesche-Starnecker, V. Nickl
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent “small cell” phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb’s phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity. IMPORTANCE Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen. Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J. Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background/Objectives Triple-negative breast cancer (TNBC) tumors are typically heterogeneous, predominantly epithelial tissues with discrete patches of mesenchymal-like TNBC cells, that differ in their invasiveness, proliferation potential and response to treatment. However, the contribution of mesenchymal-like and epithelial TNBC cells in the persistence of chemotherapy resistant disease remains poorly understood. Methods Mesenchymal-like and epithelial TNBC cell types were detected by multiplex fluorescent immunohistochemistry using antibodies against vimentin, Ki67, and Annexin A6 (AnxA6). Chemotherapy drug resistant mesenchymal-like and epithelial TNBC cell populations were established by pulse exposure and stepwise dose escalation and validated by 3D cultures and unbiased antibody arrays. Results Analysis of the response of stage IV TNBC tumors treated with six common chemotherapy regimens resulted in 36% complete response and 64% partial response with residual tumor sizes ranging from 0.5 to 37.0 mm. Treatment of TNBC cells with chemotherapy agents led to distinct resistance signatures including downregulation of survivin and upregulation of M-CSF and CXCL8/IL-8 in model mesenchymal-like, and upregulation of CCL2/MCP-1, CTSS and DKK-1 in model epithelial TNBC cells. The inhibitory phosphorylation of GSK-3β (p-S9) increased in paclitaxel resistant epithelial cells but decreased in resistant mesenchymal-like TNBC cells. Finally, chemotherapy resistance also activated p90 ribosomal S6 kinases (RSK1/2) in both cell types, while activation of mitogen- and stress-activated kinases (MSK1/2) was only observed in chemotherapy resistant epithelial TNBC cells. Conclusions These data reveal that chemotherapy resistance of epithelial and mesenchymal-like TNBC cellular subsets upregulated distinct profiles of proinflammatory and immune cell chemotactic cytokines and modulated the activities of GSK-3β, p90 RSK1/2 and the related MSK1/2. Targeting these factors and/or the associated signaling pathways may help overcome chemotherapy resistance in TNBC. Graphical Abstract TNBC tumors are typically heterogeneous, predominantly epithelial tissues with discrete patches of mesenchymal-like TNBC cells, that differ in their invasiveness, proliferation potential and response to treatment. This study reveals that Chemotherapy resistance in epithelial and mesenchymal-like TNBC cells is maintained by distinct signatures of proinflammatory and immune cell chemotactic cytokines, and activities of GSK-3β and p90 ribosomal S6 kinases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfe5f04ad4ee0a30d00ab780556d22fdbb86b41" target='_blank'>
              Contribution of Mesenchymal-like and Epithelial Cellular Subsets to Chemotherapy Resistance in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            N. Vuong, O. Korolkova, Michael Izban, Nobelle I. Sakwe, Antonisha R. McIntosh, Destiny Ball, Perrin Black, Alayjha D Edwards, Billy R. Ballrad, S. Adunyah, A. Sakwe
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Neoadjuvant treatment has demonstrated clinical benefits in advanced gallbladder cancer (GBC). However, the overall response rate remains suboptimal, and the underlying mechanisms driving treatment efficacy are not fully understood.


MATERIALS AND METHODS
This study aimed to evaluate the therapeutic effects of neoadjuvant chemo-immunotherapy (NAT) combining gemcitabine, nab-paclitaxel, and anti-PD-1 immunotherapy in advanced GBC, and to investigate the associated tumor microenvironment (TME) alterations. Single-cell RNA sequencing and multiplex immunohistochemistry were utilized to analyze the cellular composition of the TME in patients who successfully underwent downstaging.


RESULTS
NAT significantly reshaped the tumor-immune landscape, characterized by an expansion of follicular helper T (Tfh) cells and the formation of tertiary lymphoid structures (TLSs) in 50% of treated tumors. Inflammatory cancer-associated fibroblasts (CAFs) increased and exhibited upregulation of CCL19 and CXCL12, potentially promoting Tfh cell recruitment and TLS formation. Additionally, NAT led to an expansion of GZMB+ cytotoxic CD8+ T cells with an exhausted phenotype in GBC, but not in adjacent normal tissues. The treatment also increased the number and effector functions of natural killer cells while reducing tumor-promoting macrophages and angiogenesis-related CAFs. Furthermore, NAT decreased the cancer stem cell-like subpopulation while increasing a cancer cell subset with enhanced antigen-presenting capacity.


CONCLUSION
This study suggested the potential efficacy of NAT in advanced GBC and revealed alterations in the TME following treatment. The findings provided insights into the mechanisms underlying NAT responses and offered valuable directions for optimizing therapeutic strategies in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363d50ed22da82003f2b54ca7ab634be84e764af" target='_blank'>
              Single-cell profiling unveils the immuno-favorable tumor microenvironment remodeling after successful neoadjuvant therapy for advanced gallbladder cancer.
              </a>
            </td>
          <td>
            Qiuyi Tang, Yi-Jian Zhang, Ziyou Wu, Tian-Yu Zhang, Xiao-ling Song, Yi-di Zhu, Zi-Yi Yang, Cheng Zhao, Jing-wei Zhao, Hua-kai Wang, Yue Yang, Yu Zhang, Wen-Ting Dai, Lin Jiang, Yang-Yang Zhai, Chuan-Xin Huang, Xiang-song Wu, Wei Gong
          </td>
          <td>2025-11-06</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253b52aabc9f910e8e84f8a906624aeebc79b3e8" target='_blank'>
              Single-cell transcriptomic profiling reveals liver fibrosis in colorectal cancer liver metastasis.
              </a>
            </td>
          <td>
            Yiqiao Deng, Chengyao Guo, Xiaomeng Liu, Xin Li, Jianmei Liu, Wenjie Liu, Jinghua Chen, Zhen Huang, Yefan Zhang, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Hongliang Wu, Baocai Xing, Qichen Chen, Hong Zhao
          </td>
          <td>2025-11-14</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841868dae6381828134e074133437d8b76fe3212" target='_blank'>
              Bleb-based extravasation: conserved morphodynamics, divergent calcium control
              </a>
            </td>
          <td>
            Mizuki Morita, Manami Morimoto, Junichi Ikenouchi, Bertrand Pain, Yuji Atsuta, Yoshiki Hayashi, Takayuki Teramoto, Daisuke Saito
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by aggressive biological behavior and poor prognosis. However, TNBC exhibits higher immunogenicity than other breast cancer subtypes, making it more responsive to immunotherapy. Neoadjuvant chemotherapy (NAC) is the standard treatment for early high-risk TNBC; however, reliable biomarkers for predicting NAC response remain elusive. Tumor-infiltrating lymphocytes (TIL) are recognized as predictive markers of NAC response in TNBC, yet discordant cases remain, such as tumors with high TIL levels but poor response. This study aimed to further elucidate the immune environment of TNBC by analyzing TIL, programmed death-ligand 1 (PD-L1) expression, and tumor-stroma ratio using pretreatment biopsy tissue slides from 16 patients with TNBC treated with NAC. Multiplexed immunofluorescence for CD8, FOXP3, CD4, CD20, and CK was employed to investigate immune cell (IC) composition and spatial interactions within the tumor immune microenvironment. Cell to cell distance and comparison of subcellular proportion of IC and were analyzed by treatment response, TIL, and PD-L1 status. Significant differences were found in IC composition and distribution between patients achieving pathologic complete response (pCR) and those with residual disease (non-pCR). The pCR group exhibited significant enrichment of cluster of differentiation CD8 + IC and CD20 + IC in both tumor and stromal regions, suggesting their critical role in mediating an effective NAC response. In contrast, non-pCR cases showed higher proportions of immunosuppressive CD4 + FOXP3 + IC, particularly in the tumor region. High TIL levels were associated with pronounced B-T cell interactions, as evidenced by the significant clustering of CD20 + and CD8 + ICs near tumor cells, highlighting their cooperative role in antitumor immunity. In conclusion, our findings suggest that tumoral CD8 + and CD20 + ICs are pivotal determinants of NAC response in TNBC. The enrichment of CD20 + IC under high-TIL conditions underscores the potential role of B-T cell interactions in shaping immune-mediated chemotherapy responses. These insights provide a foundation for leveraging immune-based biomarkers to stratify patients with TNBC and optimize NAC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efea310e4b9cebac8f99562fb5c52b2d67c2f334" target='_blank'>
              Spatial analysis of tumor immune microenvironment of TNBC with different neoadjuvant chemotherapy outcomes using multiplex Immunofluorescence
              </a>
            </td>
          <td>
            Inho Park, Su-Jin Shin, H. Go, Jiwon Ko, S. Bae, S. Ahn, Joon Jeong, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-10-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Iron metabolism is essential for tumor proliferation and survival, yet its dysregulation can drive mutagenesis and tumor evolution. Glioblastoma (GBM) stem-like cells tolerate high reactive oxygen species (ROS) levels and exploit iron-rich environments to support unchecked proliferation. We hypothesize that iron influx through the leaky GBM vasculature creates a ROS- and iron-rich tumor microenvironment (TME), fostering a mutagenic niche that promotes a mesenchymal transition and enhances tumor invasion.



 We conducted single-nucleus RNA sequencing on in-house GBM samples to assess the interplay between iron metabolism and invasion programs. In vitro validation was performed using patient-derived GBM cell lines exposed to exogenous iron. To model TME influences, we employed ex vivo human cortical slice cultures to track three-dimensional GBM invasion, proliferation, and tissue colonization over six days. Findings were further validated using publicly available GBM datasets.



 Transcriptomic analysis revealed enrichment of iron-handling proteins (FTH1, FTL) specifically in highly invasive tumor cells. Iron supplementation increased migration and invasion rates by 2.5-fold (p<0.01) and augmented cluster formation in human ex vivo models. Notably, iron-exposed GBM cells exhibited upregulation of mesenchymal markers (CD44, CHI3L1) alongside iron metabolism genes. In the TME, microglia serve as primary iron reservoirs and potentially facilitated iron transfer to tumor cells via ferroportin-mediated export.



 Our data uncovers iron dysregulation within the GBM TME as a key driver of mesenchymal transition and invasive tumor behavior. Targeting microenvironmental iron handling pathways, including microglial ferroportin-mediated iron export, emerges as a promising therapeutic avenue to mitigate GBM progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c543d533453be74d11334f79e20f985af8d50e" target='_blank'>
              P02.20.A IRON-DRIVEN MESENCHYMAL TRANSITION ENHANCES INVASION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Petry, O. Buchholz, A. Mohammed, M. Al Shhab, I. Vasilikos, M. Shah, J. Grauvogel, R. Roelz, A. Vlachos, J. Beck, U. G. Hofmann, K. Joseph, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Within the glioblastoma (GBM) microenvironment, macrophages are widely implicated in immunosuppression. The neuro-oncology community is interested in modulating this immune suppression to improve therapies for GBM, but this approach poses significant challenges. Non-coding RNAs present an attractive target for therapeutic modulation of the immune system, and they are implicated, broadly, in macrophage polarization and the pathophysiology of GBM. Here, we discuss the extant literature on non-coding RNAs and how they impact the polarization of macrophages with particular emphasis on their impact within glioblastoma. We further discuss the clinical relevance of these non-coding RNAs and macrophage polarization to disease progression and associated therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5a680937bb5bdab851a816e92267e26c2b891d" target='_blank'>
              Macrophages in Glioblastoma and How Non-Coding RNAs Impact Their Differentiation
              </a>
            </td>
          <td>
            Emily S. Westemeier-Rice, E. Nduom
          </td>
          <td>2025-09-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its highly immunosuppressive tumor microenvironment (TME), which fuels metastasis and resistance to immunotherapy. Through comprehensive analysis of single-cell RNA sequencing datasets, we identified multiple heterogeneous tumor-associated macrophage (TAMs) subpopulations as key regulators of PDAC progression, which coexpress MRC1 and exert their effects by actively suppressing antitumor immune responses. To overcome this barrier, we developed a spatiotemporal macrophage reprogramming platform that leverages STING phase separation to reprogram TAM plasticity and reshape the immune landscape. This system, MRC1-targeting peptide-M@BLZ945 (PMMB), integrates a colony-stimulating factor 1 receptor (CSF-1R) inhibitor and a STING agonist within a macrophage-mimetic nanostructure, enabling sequential, controlled reprogramming of TAMs. By leveraging STING phase separation, PMMB stabilizes TAMs in an antitumor CD80+ phenotype while preventing excessive inflammation, achieving durable immune activation. In preclinical models, PMMB not only suppresses both primary and metastatic PDAC but also enhances CD8+ T cell infiltration, reinvigorates anti-PD-1 therapy responses, and mitigates immune exhaustion. These findings establish spatiotemporal macrophage circuit engineering via STING phase separation as a cross-scale strategy to override PDAC's immune barriers and drive next-generation macrophage-targeted immunotherapy. This study paves the way for rationally designed, precision macrophage modulation strategies in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b13978ef265777f1b6bec7b95ed69bf4845d26e" target='_blank'>
              Engineering a spatiotemporal macrophage circuit via STING phase separation to override immune suppression in pancreatic cancer.
              </a>
            </td>
          <td>
            Xue Yang, Yinlu Wang, Jiaxin Zhou, Siyu Li, Jin Ye, Yu Sun, Mengning He, Kai Fan, Zixin Chen, Fangzheng Tian, Ben Zhao, Jianqiong Zhang, Jinbing Xie, Zebin Xiao, Xiaoyuan Chen, Shenghong Ju
          </td>
          <td>2025-11-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f1e267f0452865e325381ce2fb991c017ccf2" target='_blank'>
              Are Different Populations Fairly Represented in Single-Cell Omic Atlases?
              </a>
            </td>
          <td>
            Catrina Yang, Kavitharini Saravanan, Aryan Saharan, Kuan-lin Huang
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The incidence of early-onset colorectal cancer (CRC), defined as cases diagnosed in individuals under 50, is rising globally. However, its molecular and immune characteristics remain poorly understood. In this study, we analyze single-cell RNA sequencing data from 168 CRC patients, aged 22 to 91, to investigate differences between early-onset and standard-onset CRC. We find a reduced proportion of tumor-infiltrating myeloid cells, a higher burden of copy number variations, and decreased tumor-immune interactions in early-onset CRC. Additionally, immune signatures unique to early-onset CRC are associated with differential responses to immunotherapy, underscoring the need for tailored therapeutic strategies for this group of patients. These findings provide valuable insights into the molecular and immune landscape of early-onset CRC, emphasizing the importance of developing targeted prevention and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da549e2629038aa4c9de27bbf167df537f9f53c" target='_blank'>
              Single-cell integration analysis reveals molecular signatures of the tumor microenvironment in early-onset colorectal cancer
              </a>
            </td>
          <td>
            Ting Peng, Qimin Zhan
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDA) has poor response to chemotherapy with a strong likelihood of cancer recurrence. While previous studies have examined patient samples collected before and after chemotherapy, few have analyzed matched samples from the same patients that allow us to deconvolute interpatient heterogeneity, a large limitation to PDA studies. Here, we study the effects of chemotherapy on tumor cells and their surrounding microenvironment using scRNA sequencing of matched pre- and post- treatment tissue biopsies from 5 patients. We utilized the Seurat v4 pipeline to observe transcriptomic differences and corresponding putative cellular interactions in both treatment states. Unsurprisingly, we found insufficient tumor epithelial cells post-treatment due to the intended cytotoxic effects of chemotherapy, leading us to focus on the tumor microenvironment where we had sufficient cells to analyze. We identified that immune cells in the TME are heterogeneously affected by chemotherapy treatment with patient specific differences, but we did identify a unanimous increase in CXCR4 in CD4 and CD8 positive T cells after treatment. Analysis of CAFs revealed that a subset of fibroblasts lose myofibroblastic features and increasingly acquire 'neural'-like features after treatment. This suggests that chemotherapy-altered fibroblasts may contribute to a pro-tumor microenvironment that facilitates recurrence through neuron–CAF–tumor interactions. Through inferred ligand-receptor pairs identified with the CellChat v2 pipeline, we observed that increased CXCL12 in these neural CAFs interacts with the increased CXCR4 on T-cells. We validated the expression of these genes in patients by staining human patient PDAC tissues (both treated and treatment naïve tissues) for neural genes that were highly enriched post-treatment in our single cell sequencing dataset. We have also begun inhibiting these genes of interest in CAFs and are exploring the effects of these ‘neural’-CAFs on patient-derived tumor organoids in culture.



 Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge Machicado, Richard Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martin Fernandez-Zapico, Mark Truty, Timothy Frankel, Fillip Bednar, Marina Pasca di Magliano, Costas Lyssiotis, Eileen S. Carpenter. An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B079.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e666941a86feb629e0cf1155d415645c6c810d" target='_blank'>
              Abstract B079: An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martín E Fernandez-Zapico, M. Truty, Timothy L. Frankel, Fillip Bednar, Marina Pasca di Magliano, C. Lyssiotis, Eileen S. Carpenter
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, Mariano J. Alvarez, M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women, necessitating the development of novel therapeutic strategies. Experimental animal models that closely mimic human breast cancer are crucial for advancing these therapies. This study utilized the criteria of the tumour, node, metastasis (TNM) staging system and variations in metabolic rates to develop models representing stages II and IV of human breast cancer, using the MBL-6 mouse breast cancer cell line. We assessed tumor growth curves in vivo and investigated distant metastasis to organs such as the liver, lungs, lymph nodes, and spleen. Carcinoma-associated fibroblasts (CAFs) were isolated, and their proliferation rates, inflammatory enzyme expression, and matrix metalloproteinase levels were compared between stages II and IV. By analyzing tumor kinetics and metabolic differences, we were able to predict tumor size and progression at each stage. Our results revealed that CAFs isolated from both stages exhibited similar phenotypic characteristics. However, CAFs from stage II tumors showed higher expression of indoleamine 2,3-dioxygenase 1 (IDO1), while those from stage IV tumors had higher levels of inducible nitric oxide synthase (iNOS). These distinct expression patterns suggest unique microenvironmental features at different stages of tumor progression. Further investigation of the cancer microenvironment may provide valuable insights for selecting targeted therapies and improving disease management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d06703a306e6912c38523fa9c498e367782abad" target='_blank'>
              Stage-specific tumor microenvironment dynamics and cancer-associated fibroblast profiling in MBL-6 mouse models of breast cancer
              </a>
            </td>
          <td>
            L. Langroudi, Maryam Iranpour, Mojtaba Mollaei, Masoud Soleimani, Seyed Mahmoud Hashemi, Zuhair M. Hasan
          </td>
          <td>2025-10-10</td>
          <td>Veterinary Science Development</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a common malignancy with poor prognosis, and exhausted T (TEX) cells, a subset of T cells characterized by progressive loss of effector functions, play a critical role in its progression. This study aimed to investigate the impact of TEX-related genes on immune function, prognosis, and drug sensitivity in GBM through multiomics analysis. Initially, we identified a novel set of TEX-related genes specific to GBM and screened hub genes (CCL5, IL18, CXCR6, FCER1G, TNFSF13B) using conventional statistical methods combined with machine learning. A prognostic risk model was subsequently constructed based on TCGA data and validated in the CGGA cohort. Single-cell and pharmacogenomic analyses revealed significant differences in tumor microenvironment composition and drug sensitivity between risk groups. Notably, Palbociclib emerged as a potential therapeutic agent targeting the novel discovered biomarker CCL5. RT-qPCR results showed that T cells with low CCL5 expression exhibited reduced expression of immune checkpoint-related genes (PD1, TIM3, LAG3) and increased expression of CD28, suggesting enhanced immune function. In conclusion, our findings highlight five hub genes as prognostic markers that could stratify GBM patients with different immune landscapes and levels of drug sensitivity. Furthermore, experimental results suggest that low CCL5 expression could alleviate T cell exhaustion and represent a promising therapeutic target, offering new strategies for improving GBM prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8b3263311121666e03f6883ae24b875c18a35f" target='_blank'>
              Multiomics Investigation of Exhausted T Cells in Glioblastoma Tumor Microenvironment: CCL5 as a Prognostic and Therapeutic Target
              </a>
            </td>
          <td>
            Ruihao Qin, Menglei Hua, Yaru Wang, Qi Zhang, Yong Cao, Yanyan Dai, Chenjing Ma, Xiaohan Zheng, Kaiyuan Ge, Huimin Zhang, Shi Li, Yan Liu, Lei Cao, Liuying Wang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer, leading to 179 000 cancer-related deaths annually. ccRCC initiating cells (CICs) are thought to drive tumor initiation, growth, therapy resistance, and metastasis, yet their molecular characteristics remain poorly defined. This study aims to identify putative CICs and their essential genes using a cell marker-agnostic strategy. Methods We performed a comprehensive analysis of ccRCC transcriptomes at single-cell resolution, developed patient-derived xenograft (PDX) and 3D patient-derived organoids (PDO) models of ccRCC, and conducted functional examinations in these models to investigate our findings. Results Computational modeling of tumor formation using single-cell RNA velocity analysis of five primary and metastatic ccRCC-PDXs revealed a minor cell population as the origin of other tumor cells, representing putative CICs. Pathway and network analyses suggested that a core network of proteins, conventionally known to regulate mitosis, are highly active in CICs and may be essential for their function. These proteins were expressed in PDOs and PDX-derived spheroids established following a CIC enrichment protocol. Spheroid cells exhibited higher tumorigenicity and colony formation ability than parental tumor cells, as confirmed by in-vivo injection in nude mice and in-vitro colony formation assays. Successive in vivo passaging confirmed the self-renewal capacity of spheroid-derived tumors. Pharmacological blockade of candidate proteins elicited dose-dependent inhibitory effect on spheroid and colony formation, with in-vivo validation showing that blocking these proteins significantly delayed tumor growth and more efficiently prevented tumor formation in mice. These results highlight the importance of these proteins in the cancer-initiating abilities of malignant cells. Interestingly, our results suggest that the identified target proteins may elicit their CIC-related function independently from their mitosis regulatory roles. Conclusions We identified and validated essential proteins in RCC-CICs, supported by single-cell transcriptome data and RCC spheroids and PDX models. Targeting the vulnerabilities of RCC-CICs, given their role in tumor initiation, progression, and therapy resistance, offers significant potential for developing new anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edac148fb846da9cb774550aaf710d22fbf68ef2" target='_blank'>
              64defining cancer initiating cells and their vulnerabilities in renal cell carcinoma
              </a>
            </td>
          <td>
            Zohreh Mehrjoo, H. Kuasne, Ariel Madrigal Aguirre, A. Shahini, M. Annis, Anne-Marie Fortier, Tianyuan Lu, Larisa Morales Soto, Hong Zhao, D. Zuo, V. Pilon, M. Dankner, Tamiko Nishimura, Kevin Petrecca, Jonathan Spicer, Peter Siegel, S. Tanguay, H. Najafabadi, Morag Park, Yasser Riazalhosseini
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly lethal brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but many ACT strategies rely on patient tumor tissue to derive tumor-specific antigens, limiting their use in inoperable tumors. Further, tumor heterogeneity—primarily driven by glioma stem cells (GSCs) that survive standard treatments—complicates recurrence. Radial glial cells (RGCs), neurodevelopmental progenitors transcriptionally similar to a subpopulation of GSCs, express developmental antigens (DAs) also found in GBM. We hypothesized that RGCs harbor immunogenic antigens shared with GBM and can serve as a renewable, tumor-independent antigen source for ACT to target GSC cells.



 Using single-cell RNA sequencing, we profiled RGCs cultured from post-natal day 4 (P4) murine cortices and compared their transcriptomes with tumors from murine GBM model KR158B. Shared antigens were screened for immunogenicity using our O.R.A.N. antigen prediction and validation platform. We developed a cDNA antigen library from identified RGC immunogens and used this to generate in vitro transcribed RNA for ACT. T cells were expanded ex vivo using dendritic cells (DCs) pulsed with RGC RNA or total tumor RNA (ttRNA) and tested for cytokine production and tumor-specific killing. These RGC T cells were incorporated into our ACT platform alongside hematopoietic stem cell rescue and serial DC vaccinations. Survival was assessed in GBM-bearing mice. We also tested human RGC antigens against patient-derived xenograft (PDX) GBM models using human T cells and 3D bioprinted immune-tumor interaction assays.



 scRNA-seq analysis revealed a subset of immunogenic antigens shared between RGCs and GBM tumors. RGC T cells demonstrated robust IFNγ and TNFα release, and selectively killed tumor cells in vitro. In vivo, ACT incorporating RGC T cells significantly prolonged survival (p=0.0058) in KR158B GBM murine model, matching the efficacy of ttRNA ACT. Preliminary toxicity studies revealed no overt signs of neurotoxicity or systemic inflammation. In human models, RGC T cells recognized and killed a smaller subset of GBM PDX cells when compared to ttRNA T cells, but showed enriched targeting of GSC-like subpopulations in 3D bioprinted assays.



 RGC-derived antigens represent a potent source of targets for ACT in GBM that can be used to generate a renewable antigen library. This library bypasses the need for tumor resection while maintaining therapeutic efficacy. It offers a novel approach to overcoming initiation of tumor heterogeneity through targeting of neurodevelopmental cell types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2fac66b4517361b730352a54b7bf3ec8852fff" target='_blank'>
              OS05.5.A RADIAL GLIAL CELLS AS SOURCE ANTIGEN FOR ADOPTIVE CELLULAR THERAPY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, J. W. Figg, I. West, M. Englebart, A. Reid, D. Jin, D. Mitchell, C. Flores
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Prostate cancer demonstrates significant metabolic heterogeneity, but its role in therapeutic resistance and disease progression remains unclear. This study investigates the clinical implications of metabolic diversity and identifies potential biomarkers for precision oncology. Methods Multi-omics analyses of TCGA-PRAD and meta-cohorts classified tumors into three metabolic subtypes (C1, C2, C3). Functional studies utilized prostate cancer cell lines with genetic modulation of PDIK1L. Proliferation assays, protein expression analysis, and drug sensitivity evaluations were systematically performed. Results Metabolic subtyping delineated distinct molecular and clinical profiles. The C2 subtype demonstrated elevated genomic instability and heightened sensitivity to PARP inhibitors, characterized by enrichment of glycogen metabolism and TP53-driven oncogenic pathways. Integrative multi-omics and random survival forest analysis prioritized PDIK1L as a C2-specific biomarker, where its overexpression accelerated tumor proliferation and rewired metabolic programs to confer resistance to PARP inhibitors. Conversely, PDIK1L knockdown suppressed proliferation and sensitized cells to therapy, underscoring its role as a dual-functional regulator. Mechanistically, PDIK1L interacted with DNA repair and metabolic adaptation pathways, creating a permissive environment for therapeutic resistance. Combinatorial therapy with Enzalutamide and PARP inhibitors effectively reversed PDIK1L-mediated resistance, restoring drug sensitivity across preclinical models. Independent validation in multi-institutional cohorts confirmed the robustness of metabolic subtyping and PDIK1L’s prognostic value in predicting survival and treatment outcomes. Discussion Metabolic stratification reveals the C2 subtype as a high-risk prostate cancer group with unique therapeutic vulnerabilities. PDIK1L emerges as a dual-functional biomarker driving tumor progression and modulating treatment efficacy, offering a novel target for precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/690ba0cce1e2806df775ce1f7031a2079adc6f15" target='_blank'>
              Metabolic subtyping reveals PDIK1L as a dual-functional regulator of progression and PARP inhibitor sensitivity in prostate cancer
              </a>
            </td>
          <td>
            Zhongyuan Wang, Qintao Ge, Anwaie Aihetaimujiang, Ji Zhang, Jiahe Lu, Jianfeng Yang, Yonghao Chen, Bin Qin, Hai-Lang Zhang, Wen-Hao Xu, Dingwei Ye
          </td>
          <td>2025-11-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer/PCa (mCRPC) is a clinically advanced form of PCa that is associated with increased aggressiveness, cancer stemness, morbidity, and the risk of developing resistance to taxanes, currently the first-line chemotherapy for mCRPC. Clofazimine (CLF) is a potential immunomodulator drug that is FDA-approved for the treatment of leprosy. Recently, using in vitro, in vivo, and ex vivo models, we established the efficacy of CLF in chronic myeloid leukemia and multiple myeloma. Here, we demonstrate that CLF is effective as a single agent and in combination with taxanes in a panel of cell lines representing the diversity of CRPC patients. Using a microfluidic assay, we showed the impact of CLF on cancer cell migration and metastatic potential. Further, we also found that CLF reduces ALDH activity—a marker for cancer ‘stem-like’ cells (CSCs), a subtype of cancer cells with self-renewal and differentiation capacities (epithelial-to-mesenchymal transdifferentiation/EMT). Bulk and single-cell RNAseq followed by functional validation and in silico analysis showed that CLF treatment is associated with apoptosis, ER stress, oxidative phosphorylation, and mitochondrial dysfunction. Most importantly, CLF modulates the expression of several non-coding RNAs, including MALAT1 and NEAT1, that are linked to tumor cell proliferation, cell migration, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b0a0423c975cc7befb26cd26f25ee60d9061c02" target='_blank'>
              Clofazimine Treatment Modulates Key Non-Coding RNAs Associated with Tumor Progression and Drug Resistance in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Jeremiah Pfitzer, Daniel Chinedu Nweze, Suman Mazumder, Robert D. Arnold, P. Mistriotis, Taraswi MITRA GHOSH, A. Mitra
          </td>
          <td>2025-11-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81cff854e87edda1d38e6206799dc3a099940d7f" target='_blank'>
              Notch signaling is a driver of glandular stem cell activity and regenerative migration after damage.
              </a>
            </td>
          <td>
            D. Cinat, Rufina Maturi, J. P. Gunawan, Anne L Jellema-de Bruin, Laura Kracht, Paola Serrano Martinez, Yi Wu, A. Soto-Gamez, M. van Goethem, Inge R. Holtman, S. Pringle, Lara Barazzuol, Rob P. Coppes
          </td>
          <td>2025-11-05</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="
 Glioblastoma, IDH-wildtype, WHO Grade 4 (GB), is the most common primary malignant brain tumor, accounting for approximately 50% of all gliomas. The current standard of care for GB consists of maximal safe surgical resection, with the aim of gross total or supramarginal resection whenever possible. Even when gross total resection (GTR) is achieved, a heterogeneous population of residual tumor cells typically persists along the infiltrative margins. These cells, characterized by distinct genetic, epigenetic, and microenvironmental profiles, serve as a reservoir for treatment-resistant subclones that can drive tumor recurrence. Although some studies have indicated that unresected residual cells in the peritumoral brain zone display distinct alterations compared to cells within the central tumor mass, only limited molecular analyses have been conducted. This study aims to analyze molecular profiles of distinct cell populations in the peritumoral infiltration zone using next-generation sequencing (NGS) to identify molecular features associated with early disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2477cc761f52aa09ce8251efee75bbaa2c524e0" target='_blank'>
              EP12.13 TARGETING THE INFILTRATIVE EDGE OF GLIOBLASTOMA: BIOMARKER INSIGHTS INTO EARLY TUMOR RECURRENCE
              </a>
            </td>
          <td>
            M. Barak, M. Hendrych, J. Brychta, T. Kazda, P. Pospíšil, M. Hermanova, J. Šána, R. Jancalek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="T-cell leukemias and lymphomas (TCLs) form a heterogeneous group of rare and often aggressive malignancies. Due to the rarity and heterogeneity of TCL subtypes, clinical trials are challenging to conduct, making pharmacogenomic studies in cell line panels critical for the discovery of targeted therapeutics. The scarcity of data repositories with integrated multi-omics and drug screening data hinders the preclinical evaluation of drug vulnerabilities and the identification of molecular markers predictive of responses to monotherapies and combinations. To address this gap, we conducted comprehensive pharmacogenomic profiling on a panel of 38 TCL cell lines, representing major clinical TCL subtypes to capture the molecular and phenotypic diversity. The TCL-38 multi-omics data resource includes harmonized genetic, molecular, and epigenetic profiles, with comprehensive annotations, and standardized drug response assessment of each cell line. This resource, together with machine learning predictions, was leveraged to identify TCL subtype-specific therapeutic vulnerabilities, including single-agent sensitivities and synergistic drug combinations, which were linked to genetic or epigenetic features as potential predictive biomarkers. This integrated and openly available resource (https://aittokallio.group/tcl38) could help advance the currently limited treatment options for patients with TCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df15d5c0443d2d2b30f24f2a00bf9e3203f8ea6d" target='_blank'>
              Multi-omics Profiling of T-cell Leukemia and Lymphoma Enables Targeted Therapeutic Discovery.
              </a>
            </td>
          <td>
            A. Ianevski, Kristen Nader, Julia Nguyen, H. Sorger, S. Timonen, Edith Julia, Daniel Pölöske, Katrin Spirk, Christina Wagner, Dennis Jungherz, Minoru Nakano, Sisira Kadambat Nair, Philipp Ianevski, M. Kankainen, Diogo Dias, A. Cichońska, T. Pemovska, C. Pirker, Walter Berger, T. Braun, Richard Moriggl, Emmannuel Bachy, S. Mustjoki, Marco Herling, H. Neubauer, Benjamin Haibe-Kains, T. Aittokallio
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a5395a951267c1cd8002a59d2c110d6faa6ffd" target='_blank'>
              Machine Learning-Driven Discovery of Synergistic Protein Interactions Identifies ATL3 as a Putative Biomarker for Cancer Drug Response
              </a>
            </td>
          <td>
            Priya Ramarao-Milne, A. Kaphle, R. Reguant, Anne Klein, Michael J Kuiper, B. Hosking, Julika Wenzel, H. Al-Mamun, Lujain Elazab, Qing Zhong, R. Reddel, Laurence Wilson, Yatish Jain, N. Twine, Denis C. Bauer
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Pediatric high-grade gliomas (pHGGs) are diffusely invasive and rapidly growing brain malignancies and are among the leading causes of cancer-related deaths in children. However, current treatments for pHGGs are largely ineffective. Thus, alternative therapeutic approaches are urgently needed. Genetic mutations observed in these gliomas include H3 oncohistone mutations and alterations in receptor tyrosine kinases (RTKs), including amplification, mutation, and gene fusions. However, directly targeting RTK variants with tyrosine kinase inhibitors (TKIs) has shown only partial responses in patients, due in part to the molecular and cellular heterogeneity within these tumors. Therefore, we sought to develop new experimental model systems that recapitulate inter- and intra-tumor heterogeneity to enable therapeutic discovery.



 To explore new therapeutic approaches, we created human brain organoid-tumor models of pHGG, in which patient-derived tumor cells are engrafted into ex vivo human cortical brain organoids that mimic the neonatal human brain. These organoid models provide an improved system for examining how the neural CNS microenvironment shapes cellular and molecular heterogeneity among tumor cells over time. We tested these models for responses to small molecule inhibitors and targeted therapies, including approaches that target a range of developmental and oncogenic effector pathways downstream of RTKs. Candidate targets included the YAP and TAZ transcription factors, which cooperate with TEAD co-factors to promote target gene expression, orchestrate stem cell fate decisions, and drive proliferation during embryonic neural development. We examined responses in both tumor and non-tumor cells in the neural microenvironment following YAP/TAZ pharmacologic and genetic inhibition, using single-cell RNA sequencing and immunofluorescence to visualize cellular responses.



 We observed that YAP-TEAD function is required for the growth and progression of pHGGs with oncogenic RTK mutations and H3 oncohistone mutations. Depletion of YAP reduced proliferation, inhibited pHGG stem cell self-renewal, and induced apoptosis in pHGG cells. Treatment with small molecule inhibitors targeting YAP/TAZ and TEAD mimicked the effects of genetic YAP inhibition. Further preclinical work in primary tumor cell cultures and mouse genetic models of pHGG correlated with our findings from organoid models, supporting the potential therapeutic relevance of YAP/TAZ-TEAD inhibition for pHGG treatment.



 Our results show that YAP/TAZ-TEAD inhibition may represent a promising therapeutic avenue for pHGGs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2fbad46fe24df52bf242a29e83b7ce9c58fbe" target='_blank'>
              P01.31.A THE YAP/TAZ-TEAD AXIS PROMOTES TUMOR STEM CELL FUNCTIONALITY AND CELLULAR HETEROGENEITY IN PEDIATRIC HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            R. Read, J. Luis Jimenez Macias, S. Oh, Z. Hu, K. Yin, N. Sele, N. Boon, A. Sing, S. Sloan
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="The capacity of cells to proliferate and survive is central to development and disease. Assays that measure cell fitness are therefore a cornerstone of biology, but traditional techniques lack donor diversity and have high technical variability that impedes scale and reproducibility. To overcome these barriers, we designed and validated a “cell village”-based fitness screening approach using pooled cultures of 12–39 genetically distinct human neural progenitor cell (NPC) lines. We also developed Townlet to establish a foundational statistical framework based on Dirichlet regression for analyzing proportional data from cell villages. Applying these systems, we identified hyper-proliferation in NPCs harboring the autism risk factor chromosome 16p11.2 deletion, mapped common genetic variants near ZFHX3 associated with NPC proliferation rate, and discovered genetic modifiers of lead (Pb) sensitivity implicating ARNT2. Together, these experimental and analytical tools advance a scalable, genetically diverse in vitro platform for dissecting human variation in cell fitness and gene-environment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6971b52e8812bcb489dfce1326931d0b1e99d8ab" target='_blank'>
              Cell villages and Dirichlet modeling map human cell fitness genetics
              </a>
            </td>
          <td>
            Chloe Hanson, Tim Derebenskiy, Ana Rodriguez Vega, Yashika S. Kamte, Rachel G. Fox, Hannah Lambing, Patrick Allard, Harold Pimentel, Michael F. Wells
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f4006d95d1215e3a398196888c62daed834131" target='_blank'>
              A Pan-Cancer Blueprint of Early Tumor Microenvironment Reprogramming
              </a>
            </td>
          <td>
            Bhavya Bhavya, Shivangi Agrawal, Ekta Pathak, Rajeev Mishra
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a highly aggressive brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but current strategies often rely on tumor-derived antigens, limiting their applicability in patients with inoperable tumors. Moreover, GBM recurrence is driven by tumor heterogeneity and glioma stem cells (GSCs) that evade standard therapies. Radial glial cells (RGCs), neurodevelopmental progenitors with transcriptional similarity to GSCs, express developmental antigens (DAs) also found in GBM, suggesting their potential as a tumor-independent, renewable antigen source. To explore this, we used single-cell RNA sequencing to profile RGCs from postnatal day 4 murine cortices and identified a subset of antigens shared with tumors from the KR158B GBM model. Using our O.R.A.N. antigen prediction platform, we screened these for immunogenicity and developed an RGC-derived cDNA antigen library for use in ACT. Polyclonal T cells generated from dendritic cells pulsed with RGC RNA showed robust IFNγ and TNFα release in response to cognate antigen and selectively lysed tumor cells in vitro. Further, RGC T cells were significantly better than total tumor RNA (ttRNA) T cells at killing CD133-enriched tumor cells, indicating a preference of RGC T cells for stemness markers. In GBM-bearing mice, ACT using RGC T cells significantly improved survival (p = 0.0058), achieving outcomes comparable to ttRNA ACT, without overt neurotoxicity or systemic inflammation. In parallel human studies, RGC-derived antigens were tested against patient-derived GBM xenografts. Although RGC T cells targeted a smaller fraction of tumor cells compared to ttRNA T cells, they showed enriched recognition of GSC-like populations in 3D bioprinted immune-tumor assays. These findings support the utility of RGCs as a tumor-independent antigen source capable of driving anti-GBM responses. This renewable library enables ACT even in cases where tumor resection is not possible and provides a strategy for targeting treatment-resistant GSC subpopulations early in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6bde2a5f253652f6585d9d9a61d9726319ee48" target='_blank'>
              CNSC-02. BORN TO KILL: NEURODEVELOPMENTAL CELLS AS RENEWABLE TARGETS IN T CELL THERAPY FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, John Figg, D. Jin, A. Reid, Illeana West, M. Englebart, C. Francis, Jeffery Drake, D. Mitchell, C. Flores
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly metastatic and largely refractory to current therapies, underscoring the need to uncover the molecular drivers of progression to identify targetable vulnerabilities. Here, we found that fibronectin type III domain containing 4 (FNDC4), known for its role in macrophage polarization and metabolic regulation, was elevated in metastatic PDAC cells and correlated with poor patient outcomes. FNDC4 knockdown reduced tumor growth and metastasis in a diverse set of aggressive PDAC models. Mechanistically, FNDC4 enhanced CCAR1 stability, thereby sustaining CCAR1/β-catenin signaling. FNCD4 deficiency led to reduced CCAR1 and β-catenin expression and consequently impaired invasion and colony formation. Moreover, FNDC4 promoted immune evasion by driving macrophage polarization toward a pro-tumorigenic M2 phenotype. FNDC4 loss shifted macrophage polarization toward an anti-tumor profile and increased CD4+ and CD8+ T cell infiltration. Together, the effects of FNDC4 targeting resulted in reduced tumor burden, suppression of metastasis, and improved survival in immunocompetent murine PDAC models. Unexpectedly, FNDC4 localized to the nucleus, pointing to potential intranuclear activity. Transcriptomic and functional analyses further identified CCL5 as a critical downstream effector, required for recruiting CCR5⁺ T-cells and mediating the immune effects of FNDC4 inhibition. Upstream, BHLHE40 directly activated FNDC4 transcription, which was stimulated by induction of epithelial-mesenchymal transition. Importantly, combining FNDC4 inhibition with CLDN18.2 CAR-T cells or chemotherapy resulted in enhanced tumor control compared to monotherapy. Together, these findings underscore the role of FNDC4 in promoting PDAC progression and the potential of FNDC4 as a target for innovative multimodal treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd1f79a5d2ad708f61d899ddd55cfd77459a403b" target='_blank'>
              FNDC4 Drives Metastasis and Immune Evasion in Pancreatic Cancer.
              </a>
            </td>
          <td>
            Jingwei Li, Renfei Wu, Xiaohan Jin, Yuntao Yang, Ke Jiang, Yilin Wang, Shenghui Huang, Serena Tondi, Chu-Hu Lai, Shuang Dong, Berina Sabanovic, Miriam Roberto, Chenlei Wen, Yu Jiang, Da Fu, A. Aicher, Baiyong Shen, Christopher Heeschen
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Introduction Prostate cancer (PCa) is a leading cause of male cancer mortality, with metastasis and immune evasion posing major therapeutic challenges. DOCK3, a guanine nucleotide exchange factor implicated in cytoskeletal dynamics, is poorly characterized in PCa. This study investigates DOCK3’s role in PCa metastasis and tumor immune microenvironment (TIME) remodeling. Methods Multi-omics analyses integrated bulk RNA-seq from TCGA-PRAD (499 tumors/52 normals), scRNA-seq from GEO (45,325 cells), and genomic data. We performed: Differential expression analysis (DESeq2), Immune deconvolution (CIBERSORT,ssGSEA, xCell), WGCNA co-expression networks, Tumor mutational burden (TMB) assessment, Distant metastasis (M1 vs. M0) association studies, scRNA-seq clustering (Harmony/UMAP) and DE testing. Statistical significance thresholds: |log2FC|>1, padj<0.05. Results DOCK3 expression was found to be significantly elevated in metastatic (M1) tumors compared to primary (M0) tumors (p<0.05) and demonstrated a strong positive correlation with a higher tumor mutational burden (TMB) in metastatic samples (p<0.001). Cellular specificity analysis revealed that DOCK3 was exclusively and highly enriched within malignant epithelial and stromal cells, specifically in Cluster 6, where it exhibited a log2 fold-change of 9.13 (padj<1e-200) and was expressed in 54% of cells, compared to a negligible presence in all other clusters. In the tumor microenvironment, elevated DOCK3 expression was associated with a significant increase in cytotoxic immune infiltration, notably of CD8+ T and Natural Killer cells, a finding consistently supported by multiple computational algorithms (all p<0.05). Clinically, a high level of DOCK3 was significantly associated with metastatic status (p<0.01), whereas high expression of CDKN3 was correlated with advanced disease features, including higher Gleason scores (3-5) and T-stage (T2-T4) (p<0.01). Furthermore, significant differences in immune infiltration patterns were observed between clusters. Pathway enrichment analysis of genes co-expressed with DOCK3, identified through the WGCNA Green Module, indicated significant involvement in biological processes such as cytoskeletal reorganization, muscle contraction, and metabolic pathways (FDR<0.01). Conclusion DOCK3 drives PCa metastasis through cytoskeletal dynamics while paradoxically promoting an immunologically active microenvironment. Its tumor-specific expression and association with aggressive clinical features nominate DOCK3 as a novel biomarker for risk stratification and a promising therapeutic target for combinatorial immunotherapy in immunologically “cold” PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbd5c9d8d606fe242b51bce0cd05f9f0b2729c1" target='_blank'>
              DOCK3 orchestrates metastasis and immune microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Jiaxue Han, Ming Zhang, Haipeng Zhou, Qiao Xiong, Xin Zhong, Ping Tan
          </td>
          <td>2025-10-22</td>
          <td>Frontiers in Urology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 While chimeric antigen receptor (CAR) T cell therapy has achieved encouraging remission rates in hematologic malignancies, effective treatment of glioblastoma (GBM), the most common and most lethal primary malignant brain tumor in adults, has yet to be demonstrated. Especially the heterogeneity and plasticity of the tumor antigen landscape in GBM present substantial challenges to therapeutic efficacy. In this study, we identified Podoplanin (PDPN) and GD2 as the most consistently preserved target antigens through a comprehensive screening campaign involving cell lines, both primary and recurrent GBM samples, and publicly available single-cell RNA sequencing datasets. We developed novel PDPN/GD2-dual-targeting CAR T cell strategies and evaluated their potential to overcome GBM heterogeneity in an extensive preclinical program. This included GBM cell lines (GCLs), patient-derived organoids (PDOs), and orthotopic xenograft models using stereotactic injections. We employed multi-parameter flow cytometry, in-depth cytokine profiling (cytometric bead arrays), proliferation (CFSE assays), and cytotoxicity assays (luciferase-based assays, Caspase immunofluorescence staining, BLI imaging). Our CAR T cells exhibited potent anti-tumor responses against multiple GBM cell lines and PDOs. While monospecific CAR T cells already elicited strong anti-tumor effects in PDO co-cultures, dual-targeting CAR T cells significantly enhanced apoptosis in PDOs and induced favorable cytokine and cytotoxic mediator profiles. In vivo, long-term remission was observed following locoregional administration of PDPN- or GD2-specific CAR T cells into the lateral ventricle of GBM-bearing NSG mice. While heterogeneous orthotopic tumors eventually relapsed in these two groups, dual-targeting therapy led to significantly prolonged overall survival and even achieved cure in the majority of mice. Collectively, our findings suggest that this approach can effectively address GBM heterogeneity and overcome previous limitations of single-antigen CAR T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d8a656735c860977724e62d3f48b32dc039fd5" target='_blank'>
              EXTH-88. A novel dual-antigen CAR T cell strategy targeting PDPN and GD2 for the treatment of glioblastoma
              </a>
            </td>
          <td>
            Julian Hübner, V. Nickl, Marah Alsalkini, Rhonda L. McFleder, Fabio Toppeta, Julia Wagner, R. Liguori, Björn Grams, Moritz Meyer-Hofmann, L. Diener, C. Monoranu, C. Hagemann, F. Ferrazzi, Chi Wang Ip, R. Ernestus, H. Einsele, Michael Hudecek, Mario Löhr, Thomas Nerreter
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 The advent of RAS inhibitors in the clinic offers the possibility of revolutionized care for patients with pancreatic ductal adenocarcinoma (PDAC). While RAS(ON) multi-selective inhibitors like the investigational agent daraxonrasib (RMC-6236) and the related preclinical tool compound RMC-7977 are designed to target the KRAS-mutant malignant epithelium, their effects on the surrounding tumor stroma remain underexplored. Prior studies using genetically engineered mouse models (GEMMs) have shown that aberrant oncogenic KRAS signaling is critical for the establishment and maintenance of the tumor microenvironment. In recent years, multiple CAF subtypes have been identified based on gene expression profiles, spatial distribution, and cellular origin. Myofibroblastic CAFs (myCAFs), which are located near the malignant epithelium, are marked by active Sonic Hedgehog (SHH) signaling, while inflammatory CAFs (iCAFs) and antigen-presenting CAFs (apCAFs) reside more peripherally. Here, we investigate how pan-RAS-GTP inhibition by RMC-7977 reshapes the CAF landscape in PDAC and its impact on the extracellular matrix (ECM). We profiled CAF subtype prevalence and transcriptional programs using single-cell RNA sequencing (scRNA-seq) of GEMM-derived tumors treated in vivo with RMC-7977 across multiple timepoints. These findings were corroborated by immunostaining of murine and patient-derived tumor explants treated ex vivo for 2 to 4 days. Tumor stiffness following RMC-7977 treatment was assessed using nanoindentation, and mechanistic studies were performed in vitro using reporter fibroblasts. Preliminary data reveal that short-term RMC-7977 treatment reduces myCAF abundance and proliferation, while iCAF and apCAF populations expand. Notably, supplementation with exogenous SHH restores the original myCAF-rich phenotype even in the presence of RMC-7977. Pan-RAS-GTP inhibition also remodels the ECM, resulting in decreased tumor stiffness. These preclinical findings suggest that systemic pan-RAS-GTP inhibition alters not only the malignant epithelium but also the broader PDAC architecture through effects on stromal components. Understanding these stromal adaptations will enable improved assessment of tumor behavior and the development of more effective combination therapies.



 Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, Kenneth P. Olive, Marie C. Hasselluhn. RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B098.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80ccb92666321bba79e7b5a3b723ec7cf1794a5" target='_blank'>
              Abstract B098: RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer
              </a>
            </td>
          <td>
            Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, K. Olive, Marie C. Hasselluhn
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [19, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>