<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 09:26:18 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="While recent innovations in spatial biology have driven new insights into how tissue organization is altered in disease, interpreting these datasets in a generalized and scalable fashion remains a challenge. Computational workflows for discovering condition-specific differences in tissue organization typically rely on pairwise comparisons or unsupervised clustering. In many cases, these approaches are computationally expensive, lack statistical rigor, and are insensitive to low-prevalence cellular niches that are nevertheless highly discriminative and predictive of patient outcomes. Here, we present QUICHE – an automated, scalable, and statistically robust method that can be used to discover cellular niches differentially enriched in spatial regions, longitudinal samples, or clinical patient groups. In contrast to existing methods, QUICHE combines local niche detection with interpretable statistical modeling using graph neighborhoods to detect differentially enriched cellular niches, even at low prevalence. Using in silico models and spatial proteomic imaging of human tissues, we demonstrate that QUICHE can accurately detect condition-specific cellular niches occurring at a frequency of 0.5% in fewer than 20% of patient samples, outperforming the next best method which required a patient prevalence of 60% for detection. To validate our approach and understand how tumor structure influences recurrence risk in triple negative breast cancer (TNBC), we used QUICHE to comprehensively profile the tumor microenvironment in a multi-center, spatial proteomics cohort consisting of primary surgical resections, analyzing over 2 million cells from 314 patients across 5 medical centers. We discovered cellular niches that were consistently enriched in key regions of the tumor microenvironment, including the tumor-immune border and extracellular matrix remodeling regions, as well as niches statistically-associated with patient outcomes, including recurrence status and recurrence-free survival. The majority of differential niches (74.2%) were specific to patients that did not relapse and formed a robust interconnected network enriched in monocytes, macrophages, APCs, and CD8T cells with tumor and stroma cells. In contrast, the interaction network for patients that relapsed was notably sparse and enriched in B cells, CD68 macrophages and neutrophils. We validated these findings using two independent cohorts, observing similar cellular interactions and predictive power. Collectively, these results suggest that salient, generalized profiles of productive anti-tumor immune responses are defined by a network of structural engagement between innate and adaptive immunity with tumor and stromal cells, rather than by any single specific cell population. We have made QUICHE freely available as a user-friendly open-source Python package at https://github.com/jranek/quiche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda789c041e695c92206356373353eb906709b05" target='_blank'>
              QUICHE reveals structural definitions of anti-tumor responses in triple negative breast cancer
              </a>
            </td>
          <td>
            Jolene S Ranek, Noah F. Greenwald, Mako Goldston, C. Fullaway, Cameron Sowers, Alex Kong, S. Mourón, M. Quintela-Fandino, Robert West, Michael Angelo
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, , Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to randomly generated nucleotide barcodes. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple-negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ba3313bee111d463805c5f2fb966ea7f8b649e" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) carries an extremely poor prognosis, in part resulting from cellular heterogeneity that supports overall tumorigenicity. Cancer associated fibroblasts (CAF) are key determinants of PDAC biology and response to systemic therapy. While CAF subtypes have been defined, the effects of patient-specific CAF heterogeneity and plasticity on tumor cell behavior remain unclear. Here, multi-omics was used to characterize the tumor microenvironment (TME) in tumors from patients undergoing curative-intent surgery for PDAC. In these same patients, matched tumor organoid and CAF lines were established to functionally validate the impact of CAFs on the tumor cells. CAFs were found to drive epithelial-mesenchymal transition (EMT) and a switch in tumor cell classificiaton from classical to basal subtype. Furthermore, we identified CAF-specific interleukin 8 (IL-8) as an important modulator of tumor cell subtype. Finally, we defined neighborhood relationships between tumor cell and T cell subsets. Statement of Significance This multidimensional analysis highlights the diverse role CAFs have in influencing other cell types in the TME, including epithelial-derived tumor and infiltrating immune cells. Our methods provide a platform for evaluating emerging therapeutic approaches and for studying mechanisms that dictate tumor behavior in a manner that reflects patient-specific heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d173d5090e9fb969bbd5c7fddf327d5b9643a1d" target='_blank'>
              Cancer associated fibroblasts drive epithelial to mesenchymal transition and classical to basal change in pancreatic ductal adenocarcinoma cells with loss of IL-8 expression
              </a>
            </td>
          <td>
            Samantha Guinn, Brayan Perez, Joseph A Tandurella, Mili Ramani, Jae W. Lee, Daniel J. Zabransky, Emma Kartalia, Jignasha Patel, H. Zlomke, Norman G Nicolson, Sarah M. Shin, Benjamin Barrett, Nicholas Sun, Alexei Hernandez, Erin M Coyne, Courtney D Cannon, Nicole E. Gross, S. Charmsaz, Yeonju Cho, James Leatherman, Melissa R Lyman, Jacob T. Mitchell, Luciane T. Kagohara, Michael G. Goggins, Kelly J. Lafaro, Jin He, Christopher R Shubert, William R Burns, Lei Zheng, E. Fertig, Elizabeth M. Jaffee, Richard A Burkhart, W. Ho, Jacquelyn W. Zimmerman
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Thyroid cancer progression from curable well-differentiated thyroid carcinoma to highly lethal anaplastic thyroid carcinoma is distinguished by tumor cell de-differentiation and recruitment of a robust stromal infiltrate. Combining an integrated thyroid cancer single-cell sequencing atlas with spatial transcriptomics and bulk RNA-sequencing, we define stromal cell subpopulations and tumor-stromal cross-talk occurring across the histologic and mutational spectrum of thyroid cancer. We identify distinct inflammatory and myofibroblastic cancer-associated fibroblast (iCAF and myCAF) populations and perivascular-like populations. The myCAF population is only found in malignant samples and is associated with tumor cell invasion, BRAFV600E mutation, lymph node metastasis, and disease progression. Tumor-adjacent myCAFs abut invasive tumor cells with a partial epithelial-to-mesenchymal phenotype. Tumor-distant iCAFs infiltrate inflammatory autoimmune thyroid lesions and anaplastic tumors. In summary, our study provides an integrated atlas of thyroid cancer fibroblast subtypes and spatial characterization at sites of tumor invasion and de-differentiation, defining the stromal reorganization central to disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b786943f0e4090faa02f89896c2ed59c657e0d0" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations
              </a>
            </td>
          <td>
            Matthew A. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey Caroland, Megan L. Tigue, H. Hartmann, J. Gallant, Courtney J. Phifer, Andres Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H Belcher, Ken S. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatially non-resolved transcriptomic data identified functionally distinct populations of fibroblasts in health and disease. However, in-depth transcriptional profiling in situ at single-cell resolution has not been possible so far. Here, we studied fibroblast populations in the skin of SSc patients and healthy individuals using cyclic in situ hybridization (cISH) as a novel approach for spatially-resolved transcriptional phenotyping with subcellular resolution. cISH deconvoluted the heterogeneity of 20,979 cells including 3,764 fibroblasts (FB). BANKSY-based spatially-informed clustering identified nine FB subpopulations, with SFRP2+ RetD FB and CCL19+ nonPV FB as novel subpopulations that reside in specific cellular niches and display unique gene expression profiles. SFRP2+ RetD FB and CCL19+ nonPV FB as well as COL8A1+ FB, display altered frequencies in SSc skin and play specific, disease-promoting roles for extracellular matrix release and leukocyte recruitment as revealed by their transcriptional profile, their cellular interactions and ligand-receptor analyses. The frequencies of COL8A1+ FB and their interactions with monocytic cells and B cells are associated with progression of skin fibrosis in SSc. In summary, our spatially-resolved transcriptomic approach identified novel fibroblast subpopulations deregulated in SSc skin with specific pathogenic roles, some of which may potentially serve as biomarkers for progression of skin fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c095f7397487cd95a22547f9e4b6ee14b870509e" target='_blank'>
              Spatially informed phenotyping by cyclic-in-situ-hybridization identifies novel fibroblast populations and their pathogenic niches in systemic sclerosis
              </a>
            </td>
          <td>
            Yi-Nan Li, Tim Filla, A. Györfi, Minrui Liang, Veda Devakumar, Alexandru Micu, Hongtao Chai, C. Bergmann, A. Pecher, Jörg Henes, P. Moinzadeh, Thomas Krieg, Alexander Kreuter, G. Schett, Bernhard Homey, Sascha Dietrich, J. Distler, A. Matei
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics technologies have significantly enhanced resolution and throughput, underscoring an urgent need for systematic benchmarking. To address this, we collected clinical samples from three cancer types – colon adenocarcinoma, hepatocellular carcinoma, and ovarian cancer – and generated serial tissue sections for systematic evaluation. Using these uniformly processed samples, we generated spatial transcriptomics data across five high-throughput platforms with subcellular resolution: Stereo-seq v1.3, Visium HD FFPE, Visium HD FF, CosMx 6K, and Xenium 5K. To establish ground truth datasets, we profiled proteins from adjacent tissue sections corresponding to all five platforms using CODEX and performed single-cell RNA sequencing on the same samples. Leveraging manual cell segmentation and detailed annotations, we systematically assessed each platform’s performance across key metrics, including capture sensitivity, specificity, diffusion control, cell segmentation, cell annotation, spatial clustering, and transcript-protein alignment with adjacent CODEX. The uniformly generated, processed, and annotated multi-omics dataset is valuable for advancing computational method development and biological discoveries. The dataset is accessible via SPATCH, a user-friendly web server for visualization and download (http://spatch.pku-genomics.org/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a092f418d2d0937b601a2e81131e13ca731653a" target='_blank'>
              Systematic Benchmarking of High-Throughput Subcellular Spatial Transcriptomics Platforms
              </a>
            </td>
          <td>
            Pengfei Ren, Rui Zhang, Yunfeng Wang, Peng Zhang, Ce Luo, Suyan Wang, Xiaohong Li, Zongxu Zhang, Yanping Zhao, Yufeng He, Haorui Zhang, Yufeng Li, Zhidong Gao, Xiuping Zhang, Yahui Zhao, Zhihua Liu, Yuanguang Meng, Zhe Zhang, Zexian Zeng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="In the field of digestive system tumor research, spatial transcriptomics technologies are used to delve into the spatial structure and the spatial heterogeneity of tumors and to analyze the tumor microenvironment (TME) and the inter-cellular interactions within it by revealing gene expression in tumors. These technologies are also instrumental in the diagnosis, prognosis, and treatment of digestive system tumors. This review provides a concise introduction to spatial transcriptomics and summarizes recent advances, application prospects, and technical challenges of these technologies in digestive system tumor research. This review also discusses the importance of combining spatial transcriptomics with single-cell RNA sequencing (scRNA-seq), artificial intelligence, and machine learning in digestive system cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdf975c83e9f742bc7e19f83183283693f91eae" target='_blank'>
              Application of Spatial Transcriptomics in Digestive System Tumors
              </a>
            </td>
          <td>
            Bowen Huang, Yingjia Chen, S. Yuan
          </td>
          <td>2024-12-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5139bf1027dffb0022f2f770e24762808926adf5" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa47d77b71a7337ad0ecaef200be807283fa4101" target='_blank'>
              Heterogeneity-driven phenotypic plasticity and treatment response in branched-organoid models of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Papargyriou, Mulham Najajreh, David P Cook, Carlo Maurer, S. Bärthel, H. Messal, Sakthi K Ravichandran, Till Richter, Moritz Knolle, Thomas Metzler, Akul R Shastri, R. Öllinger, Jacob Jasper, Laura Schmidleitner, Surui Wang, C. Schneeweis, H. Ishikawa-Ankerhold, T. Engleitner, Laura Mataite, Mariia Semina, Hussein Trabulssi, S. Lange, Aashreya Ravichandra, Maximilian Schuster, S. Mueller, K. Peschke, A. Schäfer, S. Dobiasch, S.E. Combs, Roland M. Schmid, Andreas R. Bausch, R. Braren, I. Heid, C. Scheel, Günter Schneider, Anja Zeigerer, Malte Luecken, K. Steiger, G. Kaissis, J. van Rheenen, F. Theis, Dieter Saur, R. Rad, Maximilian Reichert
          </td>
          <td>2024-12-10</td>
          <td>Nature biomedical engineering</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) provides an invaluable avenue for examining cell-cell interactions within native tissue environments. The development and evaluation of analytical tools for SRT data necessitate tools for generating synthetic datasets with known ground truth of cell-cell interaction induced features. To address this gap, we introduce sCCIgen, a novel real-data-based simulator tailored to generate high-fidelity SRT data with a focus on cell-cell interactions. sCCIgen preserves transcriptomic and spatial characteristics in SRT data, while comprehensively models various cell-cell interaction features, including cell colocalization, spatial dependence among gene expressions, and gene-gene interactions between nearby cells. We implemented sCCIgen as an interactive, easy-to-use, realistic, reproducible, and well-documented tool for studying cellular interactions and spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b742b7f80c26a09db2a4a0e4218062de306a242d" target='_blank'>
              sCCIgen: A high-fidelity spatially resolved transcriptomics data simulator for cell-cell interaction studies
              </a>
            </td>
          <td>
            Xiaoyu Song, J. C. Chávez‐Fuentes, Weiping Ma, Weijia Fu, Pei Wang, Guo-Cheng Yuan
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The FDA approval of T cell receptor-engineered T cells (TCR-T) for synovial sarcoma demonstrates the potential for adoptive T cell therapies (ACTs) in solid tumors. However, the paucity of tumor-specific targets without expression in normal tissues remains a major bottleneck, especially in rare cancer subtypes. Here, we present a comprehensive computational pipeline called SCAN-ACT that leverages single cell RNA sequencing and multi-omics data from tumor and normal tissues to nominate and prioritize targets for both chimeric antigen receptor (CAR)- and TCR-T cells. For surface membrane targets, SCAN-ACT proposes target pairs for bispecific Boolean logic-gated CAR T cells. For peptide-MHC targets, SCAN-ACT proposes intracellular peptides bound to a diverse set of human leukocyte antigens. We applied the SCAN-ACT pipeline to soft tissue sarcoma (STS), analyzing 986,749 single cells to identify and prioritize 395 monospecific CAR-T targets, 14,192 bispecific CAR-T targets, and 5,020 peptide-MHC targets for TCR-T cells. Selected targets were validated experimentally by protein expression and for peptide-MHC binding. Proposed targets and target pairs reflected the mesenchymal, neuronal, and hematopoietic ontogeny of STS. This work provides a robust data repository along with a web-based and user-friendly set of analysis tools to accelerate ACT development for solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcdecff6378c09bb0e27b421562a20dfc23338c" target='_blank'>
              SCAN-ACT: Adoptive T Cell Therapy Target Discovery Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Stefano Testa, Aastha Pal, Ajay Subramanian, Sushama Varma, Jack Pengfei Tang, Danielle Graham, Sara Arfan, Minggui Pan, Nam Q. Bui, Kristen Ganjoo, Sarah Dry, Paul Huang, Matt van de Rijn, Wei Jiang, Anusha Kalbasi, E. J. Moding
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma (ccRCC) is the most common type of kidney cancer, but predicting its behavior remains challenging using standard histopathologic examination. This study introduces a novel approach to predict ccRCC aggressiveness by analyzing the spatial distribution of tumor cells in H&E-stained images. The researchers found that spatial analysis outperformed traditional tumor grading in predicting metastasis, particularly for intermediate-grade tumors. They identified two distinct patient groups based on spatial characteristics, with one group showing greater spatial randomness and a higher association with metastasis. Furthermore, the study revealed a gene expression signature related to the extracellular matrix (matrisome) that correlated with the spatial patterns and aggressive tumor behavior. These findings suggest that analyzing the spatial distribution of ccRCC tumor cells could provide valuable insights into tumor behavior and metastatic potential, potentially improving prognostication and personalized treatment strategies for patients with ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1af154f3c69daea89b74218f36407c18e3de7d" target='_blank'>
              Spatial Distribution of Tumor Cells in Clear Cell Renal Cell Carcinoma Is Associated with Metastasis and a Matrisome Gene Expression Signature
              </a>
            </td>
          <td>
            Prahlad Bhat, P. Tamboli, K. Sircar, Kasthuri Kannan
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Tumor morphological features from histology images are a cornerstone of clinical pathology, diagnostic biomarkers, and basic cancer biology research. Spatial transcriptomics, which provides spatially resolved gene expression profiles overlaid on histology images, offers a unique opportunity to integrate morphological and expression features, thereby deepening our understanding of tumor biology. However, spatial transcriptomics experiments with patient samples in either clinical trials or clinical care are costly and challenging, whereas histology images are generated routinely and available for many legacy prospective cohorts of disease progression and outcomes in well-annotated cohorts. Inferring spatial transcriptomics profiles computationally from these histology images would significantly expand our understanding of tumor biology, but paired data for training multi-modal spatial-histology models remains limited. Here, we tackle this challenge by incorporating performant foundation models pre-trained on massive datasets of pathology images and single-cell RNA-Seq, respectively, which provide useful embeddings to underpin multi-modal models. To this end, we developed PathOmCLIP, a model trained with contrastive loss to create a joint-embedding space between a histopathology foundation model and a single-cell RNA-seq foundation model. We incorporate a set transformer to gather localized neighborhood tumor architecture following contrastive training, which further enhances performance and is necessary to obtain robust results. We validate PathOmCLIP across five tumor types and achieve significant performance improvements in gene expression prediction tasks over other methods. PathOmCLIP can be applied to many archived histology images, unlocking valuable clinical information and facilitating new biomarker discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0858f170113a2543c26ceee12ee7ba475e2efa6" target='_blank'>
              PathOmCLIP: Connecting tumor histology with spatial gene expression via locally enhanced contrastive learning of Pathology and Single-cell foundation model
              </a>
            </td>
          <td>
            Yongju Lee, Xinhao Liu, Minsheng Hao, Tianyu Liu, Aviv Regev
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Fate-restricted cells can acquire stem cell-like properties through dedifferentiation, enabling them to gain the plasticity required for differentiation into multiple lineages. Tumour plasticity is prominently observed in brain cancers, where transient cell state changes are linked to resistance to conventional therapies. In this study, we demonstrate that a sub-population of dedifferentiated tumour neural stem cells (NSCs) in Drosophila, induced by the knockdown of prospero (pros), can generate its own glial niche. Temporal patterning, known to influence oncogenic competence and tumour malignancy, plays a key role in this process. Specifically, we show that de novo gliogenesis occurs in the more differentiated Syncrip+ (Syp+) NSC population. Modulating Syp levels alters the size of the glial niche, subsequently affecting tumour size. Furthermore, the tumour-associated glial niche expands through cell division and fails to cease proliferation on time due to dysregulated ecdysone signalling, contributing to niche expansion. Our findings reveal that tumours arising via dedifferentiation establish their own supportive glial microenvironment, which sustains tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d30097dbfea218ae747551091d256416f71d4db" target='_blank'>
              Tumour-derived gliogenesis sustains dedifferentiation-dependent tumour growth in the Drosophila CNS
              </a>
            </td>
          <td>
            Edel Alvarez-Ochoa, Qian Dong, Hannah Truong, Louise Y Cheng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Human lung is a complex organ susceptible to various diseases. Single-cell transcriptomic studies provide rich data to targeting specific research questions. Here, we present uniLUNG, the largest lung transcriptomic cell atlas, comprising over 10 million cells across 20 disease states and healthy controls. We ensembled a universal hierarchical annotation framework and conducted a full benchmarking of data integration to define a standardized nomenclature and marker genes for lung cell types. Using uniLUNG, we identified Lym-monocyte and T-like B cells, new cell types in specific lung diseases, confirming their existence by comparing with external single-cell atlases. Additionally, we discovered the NSCLC-like SCLC subpopulation, a transitional malignant cell population associated with the transition from NSCLC to SCLC, which was validated and further characterized in spatial dimensions, revealing its complex role in tumour progression. Overall, uniLUNG represents a comprehensive range of human lung cell diversity, providing valuable data resources and a reliable foundation for lung single-cell research. HIGHLIGHTS The largest scRNA atlas for human lung covers 10 million cells from 20 lung states. A four-level universal cell annotation framework encompasses 120 lung cell types. Comprehensive benchmarking on 18 strategies guides data integration. Specific distribution of Lym-monocytes and T-like B cells in specific lung diseases. The NSCLC-like SCLC subpopulation in transitional events of malignant cells from NSCLC to SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707c7b05c0b869d7c0046d311738148202a1849b" target='_blank'>
              A universal single-cell transcriptomics atlas of human lung decodes multiple pulmonary diseases
              </a>
            </td>
          <td>
            Fanjie Wu, Wenhao Cai, Hai Tang, Shikang Zheng, Haiyue Zhang, Yixin Chen, Yutong Han, Dingli Zhou, Ruihan Wang, Mingli Ye, Renke You, Amin Chen, Jiaqi Li, Xuegong Zhang, Weizhong Li
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The kidney is a complex organ requiring tightly coordinated interactions between epithelial, endothelial, and mesenchymal cells during development. Congenital kidney defects can result in kidney disease and renal failure, highlighting the importance of understanding kidney formation mechanisms. Advances in RNA sequencing have revealed remarkable cellular heterogeneity, especially in the kidney stroma, though relationships between stromal, epithelial, and endothelial cells remain unclear. This study presents a comprehensive gene expression atlas of embryonic and postnatal kidneys, integrating single-nucleus and in situ RNA sequencing data. We developed the Kidney Spatial Transcriptome Analysis Tool (KSTAT), enabling researchers to identify cell locations, predict cell-cell communication, and map gene pathway activity. KSTAT revealed significant heterogeneity among embryonic kidney pericytes, providing a critical resource for hypothesis generation and advancing knowledge of kidney development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df4312b6dec5fab934d14ad6ad0cc39d8a07d71" target='_blank'>
              Integration of spatial and single nucleus transcriptomics to map gene expression in the developing mouse kidney
              </a>
            </td>
          <td>
            Christopher P. Chaney, Alexandria N. Fusco, Elyse D. Grilli, Jane N. Warshaw, Peter M. Luo, Ondine Cleaver, Denise K. Marciano, Thomas J. Carroll
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Exosomes, functional biomarkers involved in cancer progression, have gained widespread attention for promoting tumor formation, growth, and metastasis. Current bulk exosome detections in bodily fluids enable cancer functional analysis, but average secretion levels from cell populations, losing parent cell information and ignoring exosome heterogeneity from diverse cell subgroups, necessitating an effective platform for analyzing single-cell exosome functional heterogeneity. Here, a high-throughput platform is presented, capable of efficient single-cell isolation and multi-color exosome phenotype analysis, as well as quantifying trace exosomes secreted by single cells. Photothermal-driven single-cell chips achieve significant single-cell isolation efficiency (≈97%) within 5 min, facilitating the ultra-high throughput single-cell exosome analysis. By conducting mass spectrometry and protein interaction of breast cancer exosome phenotypic proteins, key exosome phenotypes are identified. Tens of thousands of single cells from breast cancer cell lines, and clinical tissues are analyzed, revealing various subgroup differences. The study finds more CD44 and EGFR co-expressing exosome subgroups in breast cancer cell lines, while immune-evasion PD-L1 high-phenotype exosome subgroups are primarily presented in complex tumor microenvironments, especially in HER2-positive tissues. This platform offers powerful single-cell isolation, exosome quantification, and phenotypic analysis capabilities, making it a powerful tool for advancing single-cell exosome analysis in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e3aa2959506a656b04144e1a13a4a7656680a8" target='_blank'>
              Single-Cell Isolation Chip Integrated with Multicolor Barcode Array for High-Throughput Single-Cell Exosome Profiling in Tissue Samples.
              </a>
            </td>
          <td>
            Chao Wang, Yu Zhang, Jianbo Wang, Yunrui Han, Yihe Wang, Mingyuan Sun, Yanbo Liang, Miao Huang, Yang Yu, Huili Hu, Hong Liu, Lin Han
          </td>
          <td>2024-12-10</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The existence of cancer stem cells (CSCs) in various tumors has become increasingly clear in addition to their prominent role in therapy resistance, metastasis, and recurrence. For early diagnosis, disease progression monitoring, and targeting, there is a high demand for clinical-grade methods for quantitative measurement of CSCs from patient samples. Despite years of active research, standard measurement of CSCs has not yet reached clinical settings, especially in the case of solid tumors. This is because detecting this plastic heterogeneous population of cells is not straightforward. This review summarizes various techniques, highlighting their benefits and limitations in detecting CSCs from patient samples. In addition, methods designed to detect CSCs based on secreted and niche-associated signaling factors are reviewed. Spatial and single-cell methods for analyzing patient tumor tissues and noninvasive techniques such as liquid biopsy and in vivo imaging are discussed. Additionally, methods recently established in laboratories, preclinical studies, and clinical assays are covered. Finally, we discuss the characteristics of an ideal method as we look toward the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a142badfcf7c429680b2b941eddecbf41e3d46a6" target='_blank'>
              Detection of Cancer Stem Cells from Patient Samples
              </a>
            </td>
          <td>
            Sofia Hakala, Anna Hämäläinen, Sanne Sandelin, Nikolaos Giannareas, Elisa Närvä
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hyperactivation of WNT signaling is a well-established hallmark of cancer. Various epithelial cancers express high levels of WNT7B and WNT10A that are not commonly expressed during tissue homeostasis, but rather associate with tissue development and regeneration. Although increased WNT7B/10A expression correlates with aggressive disease and lower patient survival rates, the mechanism by which these WNTs influence cancer progression remains unknown. Here, we use patient-derived organoids to show that tumor-intrinsic expression of WNT7B/10A drives survival and growth of advanced pancreatic ductal adenocarcinoma (PDAC). Bulk and single-cell profiling reveal that WNT7B drives proliferation and promotes expression of a poor prognosis basal-like state by preventing expression of a more differentiated, classical PDAC signature. By generating WNT7B reporter organoids, we show that heterogeneously distributed WNT-high PDAC cells are shifted towards a more basal-like phenotype and stably co-exist with WNT-low/negative lineages. Furthermore, hybrid co-cultures of WNT7B-proficient and -knockout PDAC organoids demonstrate that WNT-sending cells drive survival and proliferation of neighboring WNT-negative cells within the cancer epithelium via short range, cell contact-dependent signaling. In summary, our work uncovers a prominent role of WNT7B/10A in driving PDAC subtype heterogeneity and argues that WNT inhibition may be applied to force a ‘class switch’ to a more differentiated, less aggressive cancer subtype that correlates with improved therapeutical response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedf8f2a45b81078975062b7798dffcff4606bf1" target='_blank'>
              WNT7B drives a program for pancreatic cancer subtype switching and progression
              </a>
            </td>
          <td>
            J. Sprangers, J. Bugter, D. Xanthakis, M. Boekhout, R. Kok, L. Brosens, F. Dijk, M.J. Rodríguez Colman, J. van der Vaart, M. Maurice
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The presence of cancer stem cells is linked to aggressive disease and higher risk of recurrence, and multiple markers have been proposed to detect cancer stem cells. However, a detailed evaluation of the expression patterns and the prognostic value of markers relevant for endometrial cancer is lacking. As organoid models are suggested to be enriched in cancer stem cells, such models may prove valuable to define tissue-specific cancer stem cells. To address this, imaging mass cytometry and multiplex single-cell analyses were performed on an endometrial cancer patient series including both tumor biopsies and corresponding patient-derived organoids. An antibody panel focused on cancer stem cell markers was used to identify cancer stem cell phenotypes. Over 70% of epithelial cells in the tumor biopsies expressed at least one putative cancer stem cell marker. We identified distinct cancer cell phenotypes with heterogeneous expression within individual patients and between patient samples. Few differences in the distribution of cancer cell phenotypes were observed between tumor biopsies and corresponding organoids. Cells expressing aldehyde dehydrogenase 1 (ALDH1) were more prevalent in high-grade tumors, while expression of CD44 was more prevalent in grade 1 tumors. Spatial analysis revealed significantly less interaction between ALDH1- and CD44-expressing cells. Gene expression data was used to further investigate selected markers. CD44 gene expression was associated with a favorable prognosis and was further validated using immunohistochemistry. High expression of CD44 was significantly associated with better survival. The general high expression of proposed stem cell markers may indicate alternative roles for these in endometrial cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/001a7252ff4ba05e3cad7ee7ce84b192c797f212" target='_blank'>
              Multiplex single-cell profiling of putative cancer stem cell markers ALDH1, SOX9, SOX2, CD44, CD133 and CD15 in endometrial cancer.
              </a>
            </td>
          <td>
            Hilde E. Lien, M. Hjelmeland, H. Berg, Rose M Gold, Kathrine Woie, L. A. Akslen, I. Haldorsen, C. Krakstad
          </td>
          <td>2025-01-31</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer (BC) is the second leading cause of cancer-related death among women in the U.S. Organoid models of solid tumors have been shown to faithfully recapitulate aspects of cancer progression such as proliferation and invasion. Although patient-derived organoids (PDOs) and patient-derived xenograft organoids (PDXOs) are pathophysiologically relevant, they are costly to propagate, difficult to manipulate and comprised primarily of the most proliferative cell types within the tumor microenvironment (TME). These limitations prevent their use for elucidating cellular mechanisms of disease progression that depend upon tumor-associated stromal cells which are found within the TME and known to contribute to metastasis and therapy resistance. Here, we report on methods for cultivating epithelial-stromal multicellular 3D cultures. Advantages of these methods include a cost-effective system for rapidly generating organoid-like 3D cultures within scaffold-free environments that can be used to track invasion at single-cell resolution within hydrogel scaffolds. Specifically, we demonstrate how to generate these hetero-multicellular 3D cultures using BT-474 breast cancer cells in combination with fibroblasts (BJ-5ta), monocyte-like cells(THP-1) and/or endothelial cells (EA.hy926). Additionally, differential fluorescent labeling of cell populations enables time-lapse microscopy to define 3D culture assembly and invasion dynamics. Notably, the addition of any two stromal cell combinations to 3D cultures of BT-474 cells significantly reduces circularity of the 3D cultures, consistent of the presence of organoid-like or secondary spheroid structures. In tracker dye experiments, fibroblasts and endothelial cells co-localize in the peripheral organoid-like protrusions and are spatially segregated from the primary BT-474 spheroid. Finally, hetero-multicellular 3D cultures of BT-474 cells have increased hydrogel invasion capacity. Since we observed these protrusive structures in hetero-multicellular 3D cultures of both non-tumorigenic and tumorigenic breast epithelial cells, this work provides an efficient and reproducible method for generating organoid-like 3D cultures in a scaffold-free environment for subsequent analyses of phenotypes associated with solid tumor progression. SUMMARY There is a critical need for 3D cancer models that capture hetero-cellular crosstalk to study cancer metastasis. Our study presents the generation of hetero-multicellular stromal-epithelial in a scaffold and scaffold free environment that can be used to study invasion and cellular spatial distributions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a55555e37181d8e4de2e978898726ac03e16ca" target='_blank'>
              Hetero-multicellular stromal cells incorporate into scaffold-free 3D cultures of epithelial cancer cells to drive invasion
              </a>
            </td>
          <td>
            Elizabeth Ortiz, Kyaw Hsu Thway, Gabriela Ortiz-Soto, Paulina Yao, Jonathan A Kelber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiplexed tissue imaging (MTI) technologies enable high-dimensional spatial analysis of tumor microenvironments but face challenges with technical variability in staining intensities. Existing normalization methods, including z-score, ComBat, and MxNorm, often fail to account for the heterogeneous, right-skewed expression patterns of MTI data, compromising signal alignment and downstream analyses. We present UniFORM, a non-parametric, Python-based pipeline for normalizing both feature- and pixel-level MTI data. UniFORM preserves marker intensity distribution shapes while maintaining positive population proportions without prior assumptions and uses automated normalization. Benchmarking across two distinct MTI platforms and datasets demonstrates that UniFORM outperforms existing methods in mitigating batch effects while maintaining biological signal fidelity. This is evidenced by improved marker distribution alignment, enhanced kBET scores, and improved downstream analyses such as UMAP visualizations and Leiden clustering. UniFORM also introduces a novel guided fine-tuning option for complex and heterogeneous datasets. Although optimized for fluorescence-based platforms, UniFORM provides a scalable and robust solution for MTI data normalization, enabling accurate and biologically meaningful interpretations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbbdcd52a2d1b4eef9c42b637c4ae9fadce92217" target='_blank'>
              UniFORM: Towards Universal Immunofluorescence Normalization for Multiplex Tissue Imaging
              </a>
            </td>
          <td>
            Kunlun Wang, Kaoutar Ait-Ahmad, Samuel D Kupp, Zachary S. Sims, Eric M. Cramer, Z. Sayar, Jessica Yu, Melissa H. Wong, G. Mills, S. E. Eksi, Young Hwan Chang
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer manifests as multiple subtypes with distinct patient outcomes and treatment strategies. Here, we optimized proteomic analysis of Formalin-Fixed Paraffin-Embedded (FFPE) specimens from patients diagnosed with five breast cancer subtypes, luminal A, luminal B, Her2, triple negative (TNBC) and metaplastic breast cancers (MBC), and from disease-free individuals undergoing reduction mammoplasty (RM). We identified and quantified ∼6,000 protein groups (with >2 peptides per protein) with significant changes in over 26% of proteins comparing each cancer subtype with control RM. Stringent statistical filters allowed us to deeply mine 576 significant conserved protein changes shared by all subtypes and protein changes unique to each subtype. The most aggressive subtype, MBC, revealed exacerbated stromal stress responses, as illustrated by a collagenolytic extracellular matrix (ECM) and immune participation biased towards neutrophils and eosinophils. Immunostaining of breast tissue sections confirmed differences across subtypes, in particular, a dramatic upregulation of SERPINH1, neutrophil-specific myeloperoxidase and eosinophil cationic protein in MBC. In summary, deep proteomic, digitalized protein abundance profiles, generated from FFPE breast cancer tissues, revealed significant changes in ECM and cellular proteins providing insight into clinically relevant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6b77dd1484f20af85ff2b87357f38f8c094ddb" target='_blank'>
              Proteomic Analysis of Breast Cancer Subtypes Identifies Stromal Contributions that Dictate Aggressive Malignant Behavior
              </a>
            </td>
          <td>
            JO Burton, Philippe Gascard, Deng Pan, J. Bons, Rosemary Bai, Chira Chen-Tanyolac, Joseph A. Caruso, Christie L Hunter, B. Schilling, T. Tlsty
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel F Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3852f7c6358b758c264d481b7403f1d4972dcd97" target='_blank'>
              Immunotherapy in ovarian cancer: spatial functional genomics to unravel resistance mechanisms
              </a>
            </td>
          <td>
            Martina Rausch, Karlotta Bartels, Josef Leibold
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell SD Roxburgh, Paul G. Horgan, Joanne Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="T-cell-based immunotherapies have revolutionized cancer treatment, yet only a minority of patients respond to these approaches, significantly constrained by the limited knowledge of tumor-specific antigens. Here we present a comprehensive map of T cell targets across 21 cancer types, revealing actionable tumor-specific targets in 86% of tumors. To define the repertoire of actionable T cell targets, we conducted a comprehensive pan-cancer analysis integrating data from 7,473 RNA-Seq datasets, 1,564 immunopeptidomes and 208 cancer single-cell datasets, comparing these against 17,384 normal samples covering 51 tissues. Our analysis uncovered 88 viable surface protein targets and 15,079 tumor-specific HLA-presented antigens, deriving from 11 distinct molecular events, across 21 tumor types, providing a comprehensive resource for T-cell-based immunotherapy development. We highlight 128 promising new tumor targets and validate 20 targets across five antigen classes. Among the antigens uncovered, we highlight 339 previously uncharacterized neoantigens, a new PMEL splicing peptide that we expect to be a superior antigen to the clinical target, novel self-antigens, peptides derived from ORFs previously unknown to encode proteins, and novel tumor-specific microbial targets. These findings significantly expand the therapeutic landscape of T cell therapies. To catalyze therapeutic development, we have made our pan-cancer target atlas and accompanying toolkit available to the scientific community and expect these resources will pave the way for immunotherapies across multiple cancers. Significance We identified 88 viable membrane proteins and 15,079 tumor-specific HLA-presented antigens across 21 cancer types through a pan-cancer analysis of RNA-Seq, immunopeptidomes, and normal tissues. This comprehensive atlas highlights diverse antigen classes and key HLA presentation factors, providing a vital resource to catalyze T-cell based immunotherapy development and enhance understanding of antigen presentation and tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb52513187109defac6834d7dc659b898b773d3" target='_blank'>
              A pan-cancer atlas of T cell targets
              </a>
            </td>
          <td>
            Guangyuan Li, Omar Uriel Guzman-Bringas, Aman Sharma, Zachary Harpaz, Ethan Tardio, A. Natarajan, Xinya Liu, Darren Taylor, Rachel Yamin, Maxence Dellacherie, Michelle Krogsgaard, C. R. Y. Cruz, M. Yarmarkovich
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in tumor progression, recurrence, and resistance to conventional therapies, making them a critical focus in oncology research. This review provides a comprehensive analysis of CSC biology, emphasizing their self-renewal, differentiation, and dynamic interactions with the tumor microenvironment (TME). Key signaling pathways, including Wnt, Notch, and Hedgehog, are discussed in detail to highlight their potential as therapeutic targets. Current methodologies for isolating CSCs are critically examined, addressing their advantages and limitations in advancing precision medicine. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, are explored for their transformative potential in unraveling CSC heterogeneity and informing therapeutic strategies. The review also underscores the pivotal role of the TME in supporting CSC survival, promoting metastasis, and contributing to therapeutic resistance. Challenges arising from CSC-driven tumor heterogeneity and dormancy are analyzed, along with strategies to mitigate these barriers, including novel therapeutics and targeted approaches. Ethical considerations and the integration of artificial intelligence in designing CSC-specific therapies are discussed as essential elements of future research. The manuscript advocates for a multi-disciplinary approach that combines innovative technologies, advanced therapeutics, and collaborative research to address the complexities of CSCs. By bridging existing gaps in knowledge and fostering advancements in personalized medicine, this review aims to guide the development of more effective cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Copy number variations (CNVs), the gain or loss of genomic regions, are associated with different diseases and cancer types, where they are related to tumor progression and treatment outcome. Single cell technologies offer new possibilities to measure CNVs in individual cells, allowing to assess population heterogeneity and to delineate subclonal structures. Single cell whole-genome sequencing is considered the gold-standard for the quantification of CNVs in single cells. However, the majority of existing single cell datasets interrogate gene expression, using scRNA-seq. Consequently, several computational approaches have been developed to identify CNVs from that data modality. Nevertheless, an independent benchmarking of these methods is lacking. We used 15 scRNA-seq datasets and evaluated six popular computational methods in their ability to recover the ground truth CNVs using a large set of performance metrics. Additionally, we explored whether they could correctly identify euploid cells, especially also in fully diploid samples, and subclonal structures in heterogeneous tumor samples. We discovered several dataset-specific factors that influence the performance of the methods, such as the dataset size and the number and type of CNVs in the analyzed sample. We found that the choice of the reference dataset can have a large impact on the performance. Methods which included additional allelic information from the scRNA-seq reads performed more robustly across scenarios, but at the cost of higher runtime. Furthermore, the methods differed substantially in their additional functionalities and resource requirements. We offer a benchmarking pipeline to help identify the optimal CNV calling method for newly generated scRNA-seq datasets, and to benchmark and improve new methods performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3f23333379961a358cfa4029d6bc3607c70188" target='_blank'>
              Benchmarking scRNA-seq copy number variation callers
              </a>
            </td>
          <td>
            Katharina T. Schmid, Aikaterini Symeonidi, Dmytro Hlushchenko, M. Richter, Maria Colomé-Tatché
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Existing single-cell clustering methods are based on gene expressions that are susceptible to dropout events in single-cell RNA sequencing (scRNA-seq) data. To overcome this limitation, we proposed a pathway-based clustering method for single cells (scPathClus). scPathClus first transforms single-cell gene expression matrix into pathway enrichment matrix and generates its latent feature matrix. Based on the latent feature matrix, scPathClus clusters single cells using the method of community detection. Applying scPathClus to PDAC scRNA-seq datasets, we identified two types of cancer-associated fibroblasts (CAFs), termed csCAFs and gapCAFs, which highly expressed complement system and gap junction-related pathways, respectively. Spatial transcriptome analysis revealed that gapCAFs and csCAFs are located at cancer and non-cancer regions, respectively. Pseudotime analysis suggest a potential differentiation trajectory from csCAFs to gapCAFs. Bulk transcriptome analysis showed that gapCAFs-enriched tumors are more endowed with tumor-promoting characteristics and worse clinical outcomes, while csCAFs-enriched tumors confront stronger antitumor immune responses. Compared to established CAF subtyping methods, this method displays better prognostic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e5196bb420146fb22bc069bed5ff6f7bc8606d" target='_blank'>
              Pathway-based clustering identifies two subtypes of cancer-associated fibroblasts associated with distinct molecular and clinical features in pancreatic ductal carcinoma
              </a>
            </td>
          <td>
            Hongjing Ai, R. Nie, Xiaosheng Wang
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The advent of spatial transcriptomics and spatial proteomics have enabled profound insights into tissue organization to provide systems-level understanding of diseases. Both technologies currently remain largely independent, and emerging same slide spatial multi-omics approaches are generally limited in plex, spatial resolution, and analytical approaches. We introduce IN-situ DEtailed Phenotyping To High-resolution transcriptomics (IN-DEPTH), a streamlined and resource-effective approach compatible with various spatial platforms. This iterative approach first entails single-cell spatial proteomics and rapid analysis to guide subsequent spatial transcriptomics capture on the same slide without loss in RNA signal. To enable multi-modal insights not possible with current approaches, we introduce k-bandlimited Spectral Graph Cross-Correlation (SGCC) for integrative spatial multi-omics analysis. Application of IN-DEPTH and SGCC on lymphoid tissues demonstrated precise single-cell phenotyping and cell-type specific transcriptome capture, and accurately resolved the local and global transcriptome changes associated with the cellular organization of germinal centers. We then implemented IN-DEPTH and SGCC to dissect the tumor microenvironment (TME) of Epstein-Barr Virus (EBV)-positive and EBV-negative diffuse large B-cell lymphoma (DLBCL). Our results identified a key tumor-macrophage-CD4 T-cell immunomodulatory axis differently regulated between EBV-positive and EBV-negative DLBCL, and its central role in coordinating immune dysfunction and suppression. IN-DEPTH enables scalable, resource-efficient, and comprehensive spatial multi-omics dissection of tissues to advance clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2a15d874ba82ad119e2c09f851c2a8a6c7876e" target='_blank'>
              Same-Slide Spatial Multi-Omics Integration Reveals Tumor Virus-Linked Spatial Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yao Yu Yeo, Yuzhou Chang, Huaying Qiu, Stephanie Pei, Tung Yiu, Hendrik A. Michel, Wenrui Wu, Xiaojie Jin, Shoko Kure, Lindsay Parmelee, Shuli Luo, Precious Cramer, Jia Le Lee, Yang Wang, Jason Yeung, N. Ahmar, Berkay Simsek, R. Mohanna, McKayla Van Orden, Wesley Lu, K. Livak, Shuqiang Li, J. Shahryari, Leandra Kingsley, Reem N Al-Humadi, Sahar Nasr, Dingani Nkosi, Sam Sadigh, Philip Rock, Leonie Frauenfeld, Louisa Kaufmann, Bokai Zhu, Ankit Basak, Nagendra Dhanikonda, Chi Ngai Chan, Jordan Krull, Ye Won Cho, Chia-Yu Chen, Jia Ying, Joey Lee, Hongbo Wang, Bo Zhao, L. Loo, David M Kim, V. Boussiotis, Baochun Zhang, Alex K. Shalek, Brooke Howitt, S. Signoretti, Christian M. Schürch, Stephan Hodi, R. Burack, Scott J Rodig, Qin Ma, Sizun Jiang
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Invasion is a prerequisite for metastasis formation. During tumor development, the extracellular matrix (ECM) is remodeled in part through overexpression of type I collagen, increasing tumor microenvironment stiffness, and facilitating cancer dissemination. During breast cancer cell migration, we observed membrane debris left behind, attached to the collagen fibrils, along the migration path. We named these structures collagen-tracks. These collagen-tracks can be deposited in 3D matrices in vitro and in vivo and their formation is stimulated by the interaction between the ECM and matrix receptors, such as the discoidin-domain receptor (DDR1). However, they are different from structures already known to be involved in cell-cell communication such as exosomes and migrasomes, due to their specific nucleic acid and protein contents. When deposited by highly invasive breast cancer cells, internalized collagen-tracks reprogram non-invasive cells into highly pro-metastatic ones by inducing a partial epithelial–mesenchymal transition (EMT). This cell reprogramming is dependent on specific miRNAs present in the collagen-tracks, that are necessary to promote ECM degradation, increase cell motility and invasiveness. Collagen-tracks thus represent a new form of cell-cell communication important for driving tumor invasion that could be targeted to prevent metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607709bacd540192f2a53b41a2c4eccacb7b266e" target='_blank'>
              Cancer cells transfer invasive properties through microRNAs contained in collagen-tracks
              </a>
            </td>
          <td>
            Léa Normand, L. Rouyer, Elodie Richard, Nathalie Allain, Julie Giraud, Sylvaine Di-Tommaso, C. Dourthe, A. Raymond, Jean-William Dupuy, Kevin Moreau, Richard Iggo, Gaetan McGrogan, Nathalie Dugot-Senant, Anthony Bouter, Sisareuth Tan, Alexandre Favereaux, Violaine Moreau, R. Luco, M. Ros, F. Saltel
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells (VSMCs), and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and VSMCs in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process. Methods Here, we utilized CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development. Results We identified four distinct fibroblast subpopulations, including a myofibroblast population closely resembling VSMC-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion. Conclusions Our multi-omic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcefa069d1b01c8875c90b650f37b9a970f0c89d" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Peripheral nerve sheath tumors (PNSTs) encompass entities with different cellular differentiation and degrees of malignancy. Spatial heterogeneity complicates diagnosis and grading of PNSTs in some cases. In malignant PNST (MPNST) for example, single cell sequencing data has shown dissimilar differentiation states of tumor cells. Here, we aimed at determining the spatial and biological heterogeneity of PNSTs.


METHODS
We performed spatial transcriptomics on formalin-fixed paraffin-embedded diseased peripheral nerve tissue. We used spatial clustering and weighted correlation network analysis to construct niche-similarity networks and gene expression modules. We determined differential expression in primary pathologies, analysed pathways to investigate the biological significance of identified meta-signatures, integrated the transcriptional data with histological features and existing single cell data, and validated expression data by immunohistochemistry.


RESULTS
We identified distinct transcriptional signatures differentiating PNSTs. Immune cell infiltration, APOD and perineurial fibroblast marker expression highlighted the neurofibroma component of hybrid PNSTs (HPNSTs). While APOD was evenly expressed in neurofibromatous tumor tissue in both, HPNST and pure neurofibromas, perineurial fibroblast markers were evenly expressed in HPNST, but restricted to the periphery in plexiform neurofibromas. Furthermore, we provide a spatial cellular differentiation map for MPNST, locating Schwann cell precursors, neural crest-like cells and those with mesenchymal transition.


CONCLUSIONS
This pilot study shows that applying spatial transcriptomics to PNSTs provides important insight into their biology. It helps establishing new markers and provides spatial information about cellular composition and distribution of cellular differentiation states. By integrating morphological and high-dimensional molecular data it can improve PNSTs classification in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2233f7b03c50daacd9edb143f0325a91952a1c6" target='_blank'>
              Spatially resolved transcriptomics of benign and malignant peripheral nerve sheath tumors.
              </a>
            </td>
          <td>
            J. Bremer, Pamela Franco, J. A. Menstell, S. Tey, K. Zajt, Klimentina Popzhelyazkova, Kay Nolte, Jürgen Schlegel, Maria Teresa Pedro, Anja Osterloh, D. Delev, M. Hohenhaus, Christoph Scholz, O. Schnell, J. Beck, Joachim Weis, D. Heiland
          </td>
          <td>2025-01-23</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer is a significant health challenge worldwide, and disproportionately affects women of African ancestry (AA) who experience higher mortality rates relative to other racial/ethnic groups. Several studies have pointed to biological factors that affect breast cancer outcomes. A recently discovered stromal cell population that expresses PROCR, ZEB1 and PDGFRα (PZP cells) was found to be enriched in normal healthy breast tissue from AA donors, and only in tumor adjacent tissues from donors of European ancestry (EA). Here, we investigated the effect of PZP cells on the acquisition of tumorigenic phenotypes in AA and EA human breast epithelial cells to determine its contribution to the biological basis of outcome disparities. Using 3D cell models of tumor invasion, we find that PZP cells confer invasive capacity to epithelial cells independent of genetic ancestry, wherein cells exhibit leader-follower behaviors during extracellular matrix invasion. Enhanced epithelial invasion stems from a combination of AKT activation and fibronectin deposition by the PZP cells. Although activation of AKT in epithelial cells alone is insufficient to induce invasive behaviors, blocking AKT activation markedly reduces invasive capacity. These findings point to the germaneness of differences in breast biology and the multi-faceted roles and enrichment in PZP cells in AA breast tissue, in furthering current understanding of the molecular basis for worse prognosis for patients of African descent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60dd766ac1428c496c998ec8bd07c1a84a6947" target='_blank'>
              Ancestry-linked stromal variations impact breast epithelial cell invasion
              </a>
            </td>
          <td>
            M. Schmidtmann, Victoria Elliott, James W. Clancy, H. Nakshatri, Crislyn D'Souza-Schorey
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Squamous cell carcinomas (SCC) are often preceded by potentially malignant precursor lesions, most of which remain benign. The terminal exhaustion phenotypes of effector T-cells and the accumulation of myeloid-derived suppressor cells (MDSC) have been thoroughly characterized in established SCC. However, it is unclear what precancerous lesions harbor a bona fide high risk for malignant transformation and how precancerous epithelial dysplasia drives the immune system to the point of no return. Here we show that expression of SRY-box transcription factor 2 (SOX2) in precancerous lesions imparts an irreversible risk that recruits suppressive myeloid cells by promoting the release of CCL2. We developed a unique genetically engineered mouse model (GEMM) to recapitulate the malignant transformation of epithelial dysplasia to SCC in the oral mucosa with high histologic and phenotypic fidelity. Using a combination of longitudinal human specimens and the Sox2-GEMM, we found that the myeloid cells in precancerous epithelial dysplasia exhibit a distinctive dichotomous profile featuring high levels of IL-1α-SLC2A1 and low levels of type-I interferon (IFN-I) signatures, which occurs before SCC emerges histologically. Brief priming of myeloid cells with IL-1α desensitizes them to IFN-I agonists and makes myeloid-derived suppressor cells (MDSC) even more suppressive of T-cell activation. Mechanistically, IL-1 activation represses the expression of DHHC3/7 enzymes, which are responsible for the palmitoylation of stimulator of interferon genes (STING). Early blockade of IL1 signaling using pharmacologic and genetic approaches similarly reduces MDSC and SLC2A1high myeloid cells, suppresses epithelial dysplasia transformation, and extends survival. This work establishes a previously unrecognized SOX2-CCL2-IL1 pathway that leads to irreversible immune escape when precancerous epithelial lesions transform.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68906ee8a14c680b07808e1d20602c4baa417802" target='_blank'>
              SOX2-induced IL1α-mediated immune suppression drives epithelial dysplasia malignant transformation
              </a>
            </td>
          <td>
            Hülya F. Taner, Wang Gong, Z. Fitzsimonds, Zaiye Li, Yuesong Wu, Yumin He, Kohei Okuyama, Wanqing Cheng, Jung Kuczura, Sashider Rajesh, Andriana Manousidaki, Shuo Feng, Miki Lee, Felipe Nör, E. Lanzel, S. Demehri, Peter J. Polverini, Jacques E. Nör, Thomas D. Wang, Jianwen Que, Haitao Wen, Yuying Xie, James J. Moon, Yu Leo Lei
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Novel multiplexed spatial proteomics imaging platforms expose the spatial architecture of cells in the tumor microenvironment (TME). The diverse cell population in the TME, including its spatial context, has been shown to have important clinical implications, correlating with disease prognosis and treatment response. The accelerating implementation of spatial proteomic technologies motivates new statistical models to test if cell-level images associate with patient-level endpoints. Few existing methods can robustly characterize the geometry of the spatial arrangement of cells and also yield both a valid and powerful test for association with patient-level outcomes. We propose a topology-based approach that combines persistent homology with kernel testing to determine if topological structures created by cells predict continuous, binary, or survival clinical endpoints. We term our method TopKAT (Topological Kernel Association Test) and show that it can be more powerful than statistical tests grounded in the spatial point process model, particularly when cells arise along the boundary of a ring. We demonstrate the properties of TopKAT through simulation studies and apply it to two studies of triple negative breast cancer where we show that TopKAT recovers clinically relevant topological structures in the spatial distribution of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f79d159b2d5895452d1adff9339095baad447" target='_blank'>
              Detecting Clinically Relevant Topological Structures in Multiplexed Spatial Proteomics Imaging Using TopKAT
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Amarise Little, Wodan Ling, Ni Zhao, Yen-Chi Chen, Michael C. Wu
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Purpose
In glioblastoma, the therapeutically intractable and resistant phenotypes can be derived from glioma stem cells, which often have different underlying mechanisms from non-stem glioma cells. Aberrant signaling across the EGFR-PTEN-AKT-mTOR pathways have been shown as common drivers of glioblastoma. Revealing the inter and intra-cellular heterogeneity within glioma stem cell populations in relations to signaling patterns through these pathways may be key to precision diagnostic and therapeutic targeting of these cells.


Materials and Methods
Single cell parallel proteomic heterogeneity profiling of the EGFR-PTEN-AKT-mTOR pathways was conducted in a panel of fifteen glioma stem cell models derived from patient glioblastoma biopsies.


Results
The analysis included 59,464 data points from 14,866 cells and identified forty-nine molecularly distinct signaling phenotypes. High content bioinformatics resolved two unique patient clusters diverging on EGFR expression and AKT/TORC1 activation. Phenotypic validation indicated drug responsive phenotypes to EGFR blocking in the high EGFR expressing cluster with lower tumor initiating potential in comparison to the AKT/TORC1 activated cluster. High EGFR expression trended with improved patient prognosis while AKT/TORC1 activated samples trended with poorer patient outcomes. Genetic heterogeneity was observed in both clusters with proneural, classical and mesenchymal subtypes observed.


Conclusion
Quantitative single cell heterogeneity profiling reveals divergent EGFR-PTEN-AKT-mTOR pathways of patient derived glioma stem cells, which would inform future research and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0984500cc5c038ee7f954e1d13785419da86e8c6" target='_blank'>
              Single-Cell Heterogeneity of EGFR Pathway Is Linked to Unique Signatures of Drug Response and Malignancy in Patient Derived Glioblastoma Stem Cells.
              </a>
            </td>
          <td>
            Michael D. Masterman-Smith, Nicholas A. Graham, E. Panosyan, J. Mottahedeh, Eric R. Samuels, Araceli Nunez, Tiffany Phillips, Meeryo Choe, T. Cloughesy, Jorge A. Lazareff, Linda M. Liau, William H. Yong, Thomas G. Graeber, H. Kornblum, Ming-Fei Lang, Yanzhao Li, Jing Sun
          </td>
          <td>2025-01-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff90b5963386b78b07e8f87307689a97c4a0bc10" target='_blank'>
              iFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells.
              </a>
            </td>
          <td>
            Irene Garcia-Gonzalez, Stefano Gambera, Susana F Rocha, Alvaro Regano, Lourdes García-Ortega, Mariya Lytvyn, Luis Diago-Domingo, Maria S. Sanchez-Muñoz, Aroa Garcia-Cabero, Ivana Zagorac, Wen Luo, Macarena De Andrés-Laguillo, Macarena Fernández-Chacón, Verónica Casquero-García, F. F. Lunella, C. Torroja, F. Sánchez-Cabo, Rui Benedito
          </td>
          <td>2024-12-13</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a rare malignancy disproportionately affecting adolescents and young adults with no standard of care. FLC is characterized by thick stroma, which has long suggested an important role of the tumor microenvironment. Over the past decade, several studies have revealed aberrant markers and pathways in FLC. However, a significant drawback of these efforts is that they were conducted on bulk tumor samples. Consequently, identities and roles of distinct cell types within the tumor milieu, and the patterns of intercellular communication, have yet to be explored. In this study we unveil cell-type specific gene signatures, transcription factor networks, and super-enhancers in FLC using a multi-omics strategy that leverages both single-nucleus ATAC-seq and single-nucleus RNA-seq. We also infer completely rewired cell-to-cell communication patterns in FLC including signaling mediated by SPP1-CD44, MIF-ACKR3, GDF15-TGFBR2, and FGF7-FGFR. Finally, we validate findings with loss-of-function studies in several models including patient tissue slices, identifying vulnerabilities that merit further investigation as candidate therapeutic targets in FLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5969e5a104cf835bc8976096535f834d7534207" target='_blank'>
              Single-cell multi-omic analysis of fibrolamellar carcinoma reveals rewired cell-to-cell communication patterns and unique vulnerabilities
              </a>
            </td>
          <td>
            Alaa R. Farghli, Marina Chan, Marc S. Sherman, L. K. Dickerson, Bo Shui, M. Nukaya, Andreas Stephanou, Rosanna K. Ma, Brian J Pepe-Mooney, Colton J. Smith, Donald Long, Paul R Munn, Adrian McNairn, Jennifer K. Grenier, Michael Karski, Sean M. Ronnekleiv-Kelly, V. Pillarisetty, Wolfram Goessling, Taranjit S. Gujral, K. Vakili, Praveen Sethupathy
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gamma-delta (γδ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The tumor microenvironment aids cancer progression by promoting several cancer hallmarks, independent of cancer-related mutations. Biophysical properties of this environment, such as the stiffness of the matrix cells adhere to and local cell density, impact proliferation, apoptosis, and the epithelial to mesenchymal transition (EMT). The latter is rate-limiting step for invasion and metastasis, enhanced in hypoxic tumor environments but hindered by soft matrices and/or high cell densities. As these influences are often studied in isolation, the crosstalk between hypoxia, biomechanical signals, and the classic EMT driver TGF-β is not well mapped, limiting our ability to predict and anticipate cancer cell behaviors in changing tumor environments. To address this, we built a Boolean regulatory network model that integrates hypoxic signaling with a mechanosensitive model of EMT, which includes the EMT-promoting crosstalk of mitogens and biomechanical signals, cell cycle control, and apoptosis. Our model reproduces the requirement of Hif-1α for proliferation, the anti-proliferative effects of strong Hif-1α stabilization during hypoxia, hypoxic protection from anoikis, and hypoxia-driven mechanosensitive EMT. We offer experimentally testable predictions about the effect of VHL loss on cancer hallmarks, with or without secondary oncogene activation. Taken together, our model serves as a predictive framework to synthesize the signaling responses associated with tumor progression and metastasis in healthy vs. mutant cells. Our single-cell model is a key step towards more extensive regulatory network models that cover damage-response and senescence, integrating most cell-autonomous cancer hallmarks into a single model that can, in turn, control the behavior of in silico cells within a tissue model of epithelial homeostasis and carcinoma. Author Summary The cellular environment in and around a tumor can aid cancer progression by promoting several cancer hallmarks. This environment can affect growth and cell death, as well as a phenotype change that renders cells migratory and invasive: the epithelial to mesenchymal transition. Hypoxia (low oxygen availability) is known to promote this transition, while the attachment of cells to soft matrices or high cell density environments hinders it. These influences are often studied in isolation. As a result, their crosstalk is poorly understood. To address this, we have built a network model of cellular regulation that integrates a cell’s responses to hypoxia, the biophysical environment, and growth signals to model cell division, death, and the epithelial to mesenchymal transition in environments cells encounter during metastatic tumor progression. Our model reproduces a wide range of experimental cell responses and offers experimentally testable predictions about the emergence of cancer hallmarks driven mutations that affect the hypoxic response. Our single-cell model is a key step towards more extensive cell-scale models that also include cell aging and damage response. These, in turn, can serve as building blocks of a larger tissue model of healthy vs. cancerous epithelia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b181a7a9ac4c5faba59dfd6491ebe83cfeadbb5" target='_blank'>
              A Boolean network model of hypoxia, mechanosensing and TGF-β signaling captures the role of phenotypic plasticity and mutations in tumor metastasis
              </a>
            </td>
          <td>
            Grant Greene, Ian Zonfa, Erzsébet Ravasz Regan
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="


 Resistance to tumor necrosis factor inhibitor (TNFi) therapy remains a major obstacle in managing ulcerative colitis (UC), affecting up to 50% of patients.1 The complex interplay of immune cells and their spatial organization in the gut mucosa is poorly understood but likely contributes to therapeutic outcomes.1,2 This study utilized advanced multiplexed imaging to construct a spatial atlas of UC, providing insights into cellular interactions driving TNFi resistance. Novel spatial omic technologies show to be able to add additional thus far unknown insights to this question.



 A total of 42 tissue regions from 29 UC patients and 5 healthy controls were analyzed using CODEX multiplexed imaging. Spatially resolved single-cell data, encompassing over 1.7 million cells, were used to identify distinct cell types and map cellular neighborhoods (CNs). Pairwise cell-cell contacts and CN-level functional states were analyzed in TNFi-treated and untreated patients, correlating findings with clinical outcomes.



 TNFi therapy led to normalization of adaptive immune CNs, such as lymphoid aggregates, with reductions in T-cell frequencies and associated contacts. However, innate immune CNs, including neutrophil-enriched granulocyte niches and inflamed stroma, remained unchanged despite treatment. Notably, neutrophil-dominated CNs exhibited persistent expression of pro-inflammatory markers, such as TNFR2 and CD137, implicating these niches in sustaining mucosal inflammation and therapy resistance. Additionally, sex-dependent differences were observed, with higher TNFi responsiveness in females potentially linked to adaptive immune modulation.3



 Neutrophil-dominated inflammatory niches represent a potential mechanism of TNFi resistance in UC.3,4 These findings highlight the importance of targeting innate immune pathways alongside TNFi therapy to improve outcomes. By integrating spatial insights with functional data, this study paves the way for precision medicine approaches in UC management.



 1.R. Ungaro, S. Mehandru, P. B. Allen, L. Peyrin-Biroulet, J.-F. Colombel, Ulcerative colitis.Lancet 389, 1756–1770 (2017).
 2.R. Gaujoux, E. Starosvetsky, N. Maimon, F. Vallania, H. Bar-Yoseph, S. Pressman, R. Weisshof,I. Goren, K. Rabinowitz, M. Waterman, H. Yanai, I. Dotan, E. Sabo, Y. Chowers, P. Khatri,S. S. Shen-Orr; Israeli IBD research Network (IIRN), Cell-centred meta-analysis revealsbaseline predictors of anti-TNFα non-response in biopsy and blood of patients with IBD.Gut 68, 604–614 (2019).
 3.A.T. Mayer, D.R. Holman, A. Sood, U. Tandon, S. S. Bhate, S. Bodapati, G. L. Barlow, J. Chang, S. Black, E. C. Crenshaw, A. N. Koron, S. E. Streett, S. S. Gambhir SS, W. J. Sandborn, B. S. Boland, T. Hastie, R. Tibshirani, J. T. Chang, G. P. Nolan, C. M. Schürch, S. Rogalla “A tissue atlas of ulcerative colitis revealing evidence of sex-dependent differences in disease-driving inflammatory cell types and resistance to TNF inhibitor therapy” Sci Adv. 2023 Jan 20;9(3):eadd1166. doi: 10.1126/sciadv.add1166. Epub 2023 Jan 20.
 4.S. T. Eshghi, J. M. Gubatan, P. Mazrooei, L. Quintanilla, A. Nguyen, A. Au-Yeung, D. R. Holman, C. Takashi, C. Schiffman, B. O’Gorman, M. Keir, S. Ramanujan, S. Rogalla, J. A. Hackney, J. M McBride “Molecular characterization of response to Etrolizumab and anti-TNF reveals treatment resistance in ulcerative colitis is associated with abundance of residual neutrophil subsets and inflammatory fibroblast populations” https://doi.org/10.1101/2024.07.02.601267

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391837f71bfd5dab1136ff60f107c81956417fbb" target='_blank'>
              OP23 Neutrophil Niches and Therapy Resistance in Ulcerative Colitis: Spatial Atlas Insights
              </a>
            </td>
          <td>
            D. Holman, S. Rogalla
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ebbd60130879f07ccf62c91d033d068147d46a" target='_blank'>
              A safe haven for cancer cells: tumor plus stroma control by DYRK1B.
              </a>
            </td>
          <td>
            Miriam Ems, Anna Brichkina, Matthias Lauth
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g., metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved differential expression analysis. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor's proliferative front, reflecting epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. High concentrations of copper were predominantly localized in regions of active tumor growth and associated with upregulation of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7bf64840757cf5e603b33283a2afe6a77fb5a3" target='_blank'>
              Integration of Elemental Imaging and Spatial Transcriptomic Profiling for Proof-of-Concept Metals-Based Pathway Analysis of Colon Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Y. Lu, M. Chan, A. B. Diallo, S. Han, T. Punshon, B. P. Jackson, L. Vahdat, X. Liu, V. Mittal, K. S. Lau, J. Gui, L. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2024-12-10</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tissue homeostasis relies on a precise balance of fate choices between renewal and differentiation, which is dysregulated during tumor initiation. Although much progress has been done over recent years to characterize the dynamics of cellular fate choices at the single cell level, their underlying mechanistic basis often remains unclear. In particular, although physical forces are increasingly characterized as regulators of cell behaviors, a unifying description of how global tissue mechanics interplays with local cellular fate choices is missing. Concentrating on skin epidermis as a paradigm for multilayered tissues with complex fate choices, we develop a 3D vertex-based model with proliferation restrained in the basal layer, showing that mechanics and competition for space naturally gives rise to homeostasis and neutral drift dynamics that are seen experimentally. We then explore the effect of introducing mechanical inhomogeneities, whereby subpopulations have differential tensions. We uncover that relatively small mechanical disparities can be sufficient to heavily tilt cellular towards symmetric renewal and exponential growth. Importantly, the simulations predict that such mechanical inhomogeneities are reflected by distinct morphological changes in single-cell shapes. This led us to derive a master relationship between two very different experimentally measurable parameters, cell shape and long-term clonal dynamics, which we validated using a model of basal cell carcinoma (BCC) consisting in clonal Smoothened overexpression in mouse tail epidermis. Altogether, we propose a theoretical framework to link mechanical forces, quantitative cellular morphologies and cellular fate outcomes in complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215257d8ed053f55e3fe3327a06377e6d6a6e712" target='_blank'>
              Biomechanics of stem cell fate decisions in multilayered tissues
              </a>
            </td>
          <td>
            Preeti Sahu, Sara Canato, Raquel Soares, Adriana Sánchez-Danés, É. Hannezo
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The prevalent intra- and intertumoral heterogeneity results in undesirable prognosis and therapy failure of pancreatic cancer, potentially resulting from cellular senescence. Herein, integrated analysis of bulk and single-cell RNA-seq profiling was conducted to characterize senescence-based heterogeneity in pancreatic cancer. Publicly available bulk and single-cell RNA sequencing from pancreatic cancer patients were gathered from TCGA-PAAD, PACA-AU, PACA-CA, and GSE154778 datasets. The activity of three senescence-related pathways (cell cycle, DNA repair, and inflammation) was scored utilizing ssGSEA algorithm. A series of functional verifications of crucial genes were accomplished in patient tissue and pancreatic cancer cells. Based upon them, unsupervised clustering analysis was executed to classify pancreatic cancer samples into distinct senescence-based clusters at the bulk and single-cell levels. For single-cell transcriptome profiling, cell clustering and annotation were implemented, and malignant cells were recognized utilizing infercnv algorithm. Two senescence-based clusters were established and highly reproducible at the bulk level, with the heterogeneity in prognosis, clinicopathological features, genomic CNVs, oncogenic pathway activity, immune microenvironment and immune checkpoints. Senescence-relevant gene CHGA, UBE2C and MCM10 were proved to correlate with the migration and prognosis of pancreatic cancer. At the single-cell level, seven cell types were annotated, comprising ductal cells 1, ductal cells 2, fibroblasts, macrophages, T cells, stellate cells, and endothelial cells. The senescence-based classification was also proven at the single-cell level. Ductal cells were classified as malignant cells and non-malignant cells. In the tumor microenvironment of malignant cells, hypoxia and angiogenesis affected senescent phenotype. The heterogeneity in senescence was also observed between and within cell types. Altogether, our findings unveil that cellular senescence contributes to intra- and intertumoral heterogeneity in pancreatic cancer, which might facilitate the development of therapeutics and precision therapy in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2742fe352ef25d18eeda7d59fe2c6a8314ffc13" target='_blank'>
              Deciphering the senescence-based tumoral heterogeneity and characteristics in pancreatic cancer: Results from parallel bulk and single-cell transcriptome data.
              </a>
            </td>
          <td>
            Yeting Lu, Shuo Han, Jing Hu, Kaiji Lv, Y. Ruan, Gong Cheng, Jing Zhang, Xiang Wu, Zeming Weng, Xinhua Zhou
          </td>
          <td>2025-01-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4799cdd384521f6d3933fff3e76a51c122a658" target='_blank'>
              Differential somatic coding variant landscapes between laser microdissected luminal epithelial cells from canine mammary invasive ductal solid carcinoma and comedocarcinoma
              </a>
            </td>
          <td>
            Vivi Deckwirth, S. Hundi, M. Hytönen, Sari Hannula, Pekka Ellonen, Pia Björkenheim, A. Sukura, H. Lohi
          </td>
          <td>2024-12-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Disruptions in uterine tissue function contribute to disorders such as endometriosis, adenomyosis, endometrial cancer, and fibroids, which all significantly impact health and fertility. Advances in transcriptomics, particularly single-cell RNA sequencing, have revolutionized uterine biological research by revealing the cellular heterogeneity and molecular mechanisms underlying disease states. Single-cell RNA sequencing and spatial transcriptomics have mapped endometrial and myometrial cellular landscapes, which helped to identify critical cell types, signaling pathways, and phase-specific dynamics. Said transcriptomic technologies also identified stromal and immune cell dysfunctions, such as fibroblast-to-myofibroblast transitions and impaired macrophage activity, which drive fibrosis, chronic inflammation, and lesion persistence in endometriosis. For endometrial cancer, scRNA-seq uncovered tumor microenvironmental complexities, identifying cancer-associated fibroblast subtypes and immune cell profiles contributing to progression and therapeutic resistance. Similarly, studies on adenomyosis highlighted disrupted signaling pathways, including Wnt and VEGF, and novel progenitor cell populations linked to tissue invasion and neuroinflammation, while single-cell approaches characterized smooth muscle and fibroblast subpopulations in uterine fibroids, elucidating their roles in extracellular matrix remodeling and signaling pathways like ERK and mTOR. Despite challenges such as scalability and reproducibility, single-cell transcriptomic approaches may have potential applications in biomarker discovery, therapeutic target identification, and personalized medicine in gynecological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d46c8d721f3ec856f042d24cd830de014f3393b" target='_blank'>
              Mapping Human Uterine Disorders Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Sandra Boldu-Fernández, Carolina Lliberos, Carlos Simón, A. Mas
          </td>
          <td>2025-01-21</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The Imaging and Molecular Annotation of Xenografts and Tumors Cancer Grand Challenges team was set up with the objective of developing the "next generation" of pathology and cancer research by using a combination of single-cell and spatial omics tools to produce 3D molecularly annotated maps of tumors. Its activities overlapped, and in some cases catalyzed, a spatial revolution in biology that saw new technologies being deployed to investigate the roles of tumor heterogeneity and of the tumor micro-environment. See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28 See related article by Goodwin et al., p. 34.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c9c61637383cf6c8f3f96d10f0ca3cac35ac4d" target='_blank'>
              Cancer Research in the Age of Spatial Omics: Lessons from IMAXT.
              </a>
            </td>
          <td>
            D. Bressan, Nicholas Walton, Gregory J. Hannon
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Summary Cellular stemness is a cells ability to self-renew and differentiate into specialized cell types. In spatial transcriptomics (ST), researchers can use gene activity to quantify stemness and the associated spatial locations, allowing for the investigation of cellular profiles in their natural context. However, many ST platforms provide data with low resolution, and each data spot (i.e., spatial location) contains multiple cell types, which hinders the investigation of cell type–specific (CTS) behaviors, such as cell stemness, at different spatial locations. We developed a bivariate kernel-weighted regression method with constrained optimization to estimate CTS stemness as a function of spatial location and developed accompanying visualization tools. Through simulation studies and an application to real breast cancer ST data from the 10x Genomics Visium platform, we demonstrated that our method can accurately estimate CTS stemness and help shed light on the interplay among cell type, tissue structure, and stemness. Availability and implementation An R Shiny app implementing the proposed method is available at GitHub (https://github.com/ericli0480/stemness-deconvolution/). Supplementary information Available in a separate PDF file">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72f608c82ad15c7c6476ead9f76633ad0bf3a052" target='_blank'>
              Delineating the Spatial Patterns of Cell Type–Specific Stemness Using a Statistical Deconvolution Model
              </a>
            </td>
          <td>
            Eric J. Li, Liang Li, Ziyi Li
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Although invasiveness is one of the major determinants of the poor glioblastoma (GBM) outcome, the mechanisms of GBM invasion are only partially understood. Among the intrinsic and environmental processes promoting cell-to-cell interaction processes, eventually driving GBM invasion, we focused on the pro-invasive role played by Extracellular Vesicles (EVs), a heterogeneous group of cell-released membranous structures containing various bioactive cargoes, which can be transferred from donor to recipient cells.


METHODS
EVs isolated from patient-derived GBM cell lines and surgical aspirates were assessed for their pro-migratory competence by spheroid migration assays, calcium imaging, and PYK-2/FAK phosphorylation. Brain invasiveness was investigated in human cortical organoids-based assembloids and in vivo orthotopic xenografts. EV molecular features were specified by multiplex bead-based flow cytometry.


RESULTS
Results unveil a self-sustaining mechanism triggering migration through autocrine release and engagement of a specific population of EVs of large size (L-EVs), isolated from either patient-derived cell lines or surgical aspirates. L-EVs act through modulation of calcium transients via Connexin 43-Gap Junctions (Cx43-GJ) and phospho-activation of PYK2. Pre-incubation with blocking antibodies targeting Cx43 hemichannels demonstrated a dose-dependent inhibition of the L-EV-mediated GBM migration. By exploiting patients' surgical aspirates, we show that only L-EVs deriving from tumoral cells, and not those with immune origin, promote tumor migration, impacting more prominently the tumoral cells with mesenchymal subtype.


CONCLUSIONS
We demonstrate that L-EVs released by GBM cells, but not by the immune cells of the tumor microenvironment, represent a relevant and unique autocrine pro-migratory input for the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f87a08f8bbc4e6c1b435e86c434ceec17830431" target='_blank'>
              Extracellular Vesicles Released by Glioblastoma Cancer Cells Drive Tumor Invasiveness via Connexin-43 Gap Junctions.
              </a>
            </td>
          <td>
            M. Tamborini, Valentino Ribecco, Elisabetta Stanzani, Arianna Sironi, Monica Tambalo, Davide Franzone, Elena Florio, Edoardo Fraviga, Chiara Saulle, M. Gagliani, Marco Pizzocri, M. Mattioli, Katia Cortese, Jean X Jiang, Giuseppe Martano, L. Politi, Marco Riva, Federico Pessina, Davide Pozzi, Simona Lodato, L. Passoni, Michela Matteoli
          </td>
          <td>2025-01-30</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a6df87fa44ba62736b9ada1db0c1fd8750c04f" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Luisa Solis Soto, N. O. Lermi, Max Molina Ayala, Sharia Hernandez Ruiz, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don Gibbons, Ignacio I. Wistuba, Cara Haymaker, Ken Chen
          </td>
          <td>2025-01-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Gliomas are aggressive tumors with a poor prognosis. The protocols presented here outline the methods used to study tumor progression, the tumor microenvironment (TME), and the effects of experimental treatments. The Sleeping Beauty (SB) transposase system induces tumors de novo to generate mouse models that recapitulate human gliomas. Plasmids are constructed with oncogenic drivers and other genetic alterations of interest. which are recognized by their unique position in between inverted/direct repeat (IR/DR) sequences. Luciferase enzyme is used to monitor the uptake of the plasmid, tumor growth, and response to experimental therapies. The genes of interest are tracked using fluorescent markers. Tumors will arise in immunocompetent hosts, which provides a relevant preclinical platform for analysis of tumor initiation, progression, survival, immune microenvironment, and histopathological features. Once the tumor grows within the desired brain location, it can be harvested to generate cell cultures of neurospheres for future experimentation. The benefit of implantable models generated from SB tumors is that they provide specific anatomical and genetic context, in which specific genetic characteristics can be tracked, as they are co-expressed with fluorescent markers. Post glioma cell implantation, additional analysis of the TME and tumor growth can be performed through immunohistochemistry (IHC) and flow cytometry. © 2025 Wiley Periodicals LLC. Basic Protocol 1: Creation of mouse glioma models by Sleeping-Beauty-mediated transposition Basic Protocol 2: Generation of orthotopic implantable brain tumors and neurospheres Basic Protocol 3: Hematoxylin and eosin staining of glioma tissue samples Basic Protocol 4: Immunohistochemistry of glioma tissue samples Basic Protocol 5: Flow cytometry for immune cell analysis of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/886ac84c05876a63108c36f36b7b7af78f353102" target='_blank'>
              Genetically Engineered and Implantable Mouse Brain Tumor Models: Characterization by Immunohistochemistry and Flow Cytometry.
              </a>
            </td>
          <td>
            Apoorva Mirji, Gurveer Singh, Anzar A. Mujeeb, Brandon L. McClellan, YingXiang Li, Makayla Perez, Maria G Castro
          </td>
          <td>2025-01-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite recent therapeutic breakthroughs, cancer patients continue to face high recurrence and mortality rates due to treatment resistance. Cancer stem cells (CSCs), a subpopulation with self-renewal capabilities, are key drivers of refractive disease. This review explores the application of molecular imaging techniques, such as PET and SPECT, for the noninvasive detection of CSCs. By providing real-time monitoring of CSCs, these imaging methods have the potential to predict therapy resistance and guide personalized treatment approaches. Here, we cover the biological characteristics of CSCs, mechanisms of therapy resistance, and the identification and targeting of CSC-specific biomarkers with molecular imaging. Additionally, we address the challenges and opportunities for the clinical translation of CSC imaging, highlighting strategies where CSC imaging can be used to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c598784fb4bf91a45e00e3bbc55f45f0d4b538" target='_blank'>
              Molecular Imaging of Cancer Stem Cells and Their Role in Therapy Resistance
              </a>
            </td>
          <td>
            Sofia N Dos Santos, T. Witney
          </td>
          <td>2025-01-01</td>
          <td>Journal of Nuclear Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases.
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9cdfae4578e26b8066f9cf63fa62b59d86c2b7" target='_blank'>
              Exploring tumor microenvironment interactions and apoptosis pathways in NSCLC through spatial transcriptomics and machine learning.
              </a>
            </td>
          <td>
            Huimin Li, Yuheng Jiao, Yi Zhang, Junzhi Liu, Shuixian Huang
          </td>
          <td>2024-12-19</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Tumor heterogeneity plays a pivotal role in tumor progression and resistance to clinical treatment. Single-cell RNA sequencing (scRNA-seq) enables us to explore heterogeneity within a cell population and identify rare cell types, thereby improving our design of targeted therapeutic strategies. Here, we use a pan-cancer and pan-tissue single-cell transcriptional landscape to reveal heterogeneous expression patterns within malignant cells, precancerous cells, as well as cancer-associated stromal and endothelial cells. We introduce a deep learning framework named Shennong for in silico screening of anticancer drugs for targeting each of the landscape cell clusters. Utilizing Shennong, we could predict individual cell responses to pharmacologic compounds, evaluate drug candidates’ tissue damaging effects, and investigate their corresponding action mechanisms. Prioritized compounds in Shennong's prediction results include FDA-approved drugs currently undergoing clinical trials for new indications, as well as drug candidates reporting anti-tumor activity. Furthermore, the tissue damaging effect prediction aligns with documented injuries and terminated discovery events. This robust and explainable framework has the potential to accelerate the drug discovery process and enhance the accuracy and efficiency of drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50856eac718497c33c44c76e8ec32fca287abdfe" target='_blank'>
              A deep learning framework for in silico screening of anticancer drugs at the single-cell level
              </a>
            </td>
          <td>
            Peijing Zhang, Xueyi Wang, Xufeng Cen, Qi Zhang, Yuting Fu, Yuqing Mei, Xinru Wang, Renying Wang, Jingjing Wang, H. Ouyang, Tingbo Liang, Hongguang Xia, Xiaoping Han, Guoji Guo
          </td>
          <td>2024-12-10</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The development and maturation of B lymphocytes involve intricate orchestrated processes, where dedicated gene regulations (GR) take place within specific microenvironments shaped by both extracellular matrix and neighboring cells. Despite extensive investigations aimed at deepening our comprehension of these mechanisms, there remains a dearth of high-dimensional and integrated analysis concerning B cell heterogeneity, gene regulation, and external factors implicated in B cell development. In this study, we scrutinized single-cell transcriptomic data and B cell receptor (BCR) sequencing data obtained from B cells and their surrounding counterparts in the bone marrow, tonsil, and peripheral blood. A full picture of the GR dynamics, the heterogeneity of conventional B cells and cell-cell interactions (CCIs) along B cell development axis was depicted. We found immature B cells represent the most quiescent stage characterized by the least number of expressed genes and low RNA velocity. The homeostatic proliferation and activation of naïve B cells is niche-confined and individualized, respectively. Two development models for memory B cell subpopulations seem not mutually exclusive and warrant in-depth investigation. Moreover, CCI analysis reveals a pivotal role of myeloid cells and two dominant and stage-dependent CCI categories, TNF and adhesion signaling, in B cell development. Besides, we unexpectedly identified two age-associated B cell subpopulations that respectively express S100A8/S100A9 and C1q and experimentally confirmed the secretion of S100A8/A9 from human B cells in vitro, suggesting a senescence-associated secretion phenotype. Our integrated analysis provides valuable insights into GR dynamics, the evolution of B cells, and potential intercellular communication networks involved in B cell development and revealed novel phenotypes of age-associated B cell aberrance. This study serves as a valuable resource for in-depth exploration of the intricacies of B cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6267c2c995e20da9c44b1ab0873ec2b1eb2767c2" target='_blank'>
              Single-cell analysis reveals multi-faceted features of B cell development together with age-associated B cell subpopulations
              </a>
            </td>
          <td>
            Xiujia Yang, Haipei Tang, Chunhong Lan, Weiting He, Sen Chen, Huikun Zeng, Danfeng Liu, Haoyu Wu, Wenjian Wang, Zhenhai Zhang
          </td>
          <td>2024-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The evolution of metastasis, the spread of cancer to distal sites within the body, represents a lethal stage of cancer progression. Yet, the evolutionary dynamics that shape the emergence of metastatic disease remain unresolved. Here, using single-cell lineage tracing data in combination with phylogenetic statistical methods, we show that the evolutionary trajectory of metastatic disease is littered with bursts of rapid molecular change as new cellular subpopulations appear, a pattern known as punctuational evolution. Next, by measuring punctuational evolution across the metastatic cascade, we show that punctuational effects are concentrated within the formation of secondary tumours at distal metastatic sites, suggesting that qualitatively different modes of evolution may drive primary and metastatic tumour progression. Taken as a whole, our findings provide empirical evidence for distinct patterns of molecular evolution at early and late stages of metastatic disease and our approach provides a framework to study the evolution of metastasis at a more nuanced level than has been previously possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eea86b0534290bd81992d00ef5caf3aad45d15" target='_blank'>
              Punctuational evolution is pervasive in distal site metastatic colonization
              </a>
            </td>
          <td>
            George Butler, Sarah R. Amend, C. Venditti, Kenneth J Pienta
          </td>
          <td>2025-01-01</td>
          <td>Proceedings of the Royal Society B: Biological Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, , Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a critical factor in cancer progression, driving tumor growth, immune evasion, therapeutic resistance, and metastasis. Understanding the dynamic interactions within the TME is essential for advancing cancer management. Molecular imaging provides a non-invasive, real-time, and longitudinal approach to studying the TME, with techniques such as positron emission tomography (PET), magnetic resonance imaging (MRI), and fluorescence imaging offering complementary strengths, including high sensitivity, spatial resolution, and intraoperative precision. Recent advances in imaging probe development have enhanced the ability to target and monitor specific components of the TME, facilitating early cancer diagnosis, therapeutic monitoring, and deeper insights into tumor biology. By integrating these innovations, molecular imaging offers transformative potential for precision oncology, improving diagnostic accuracy and treatment outcomes through a comprehensive assessment of TME dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da9d2c5c64b0421eb31502177573bfeb312021d3" target='_blank'>
              Visualizing the Tumor Microenvironment: Molecular Imaging Probes Target Extracellular Matrix, Vascular Networks, and Immunosuppressive Cells
              </a>
            </td>
          <td>
            Hui-Wen Chan, Deng-Yu Kuo, P. Shueng, Hui-Yen Chuang
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Anaplastic Thyroid Cancer (ATC) is an aggressive form of cancer with poor prognosis, heavily influenced by its tumor immune microenvironment (TIME). Understanding the cellular and gene expression dynamics within the TIME is crucial for developing targeted therapies. This study analyzes the immune microenvironment of ATC and Papillary Thyroid Cancer (PTC) using single-cell RNA sequencing (scRNA-seq). We performed a comprehensive scRNA-seq analysis on ATC and PTC samples, incorporating cell type annotation, marker gene identification, and clustering based on gene expression. A specific focus was on the prevalence and biomarkers of Pre-Exhausted CD8+ T cells in ATC, utilizing the single-cell tumor immune atlas for immune cell characterization. The scRNA-seq analysis identified distinct immune cell populations and differentially expressed genes in ATC and PTC samples. Notably, Pre-Exhausted CD8+ T cells were found to be prevalent in ATC datasets. Additional immunofluorescence staining and co-culture experiments with the ATC cell line identified GNLY, a member of the saposin-like protein family as a potential biomarker for Pre-Exhausted CD8+ T cells in ATC. This study provides valuable insights into the immune landscape of ATC, emphasizing the prevalence of Pre-Exhausted CD8+ T cells and identifying GNLY as a potential biomarker. Understanding the TIME composition and the role of specific immune cells in cancer progression can inform the development of targeted immunotherapies for ATC. Future research should explore the functional implications of GNLY and other identified biomarkers in modulating the immune response in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bee3b0c483d956dde8850d164e4f812a4b30008" target='_blank'>
              Persistent pre-exhausted CD8+ T cells shape the tumor immune microenvironment in anaplastic thyroid cancer.
              </a>
            </td>
          <td>
            X. Ruan, Mei Tao, Yuanxing Dong, Linfei Hu, Guangwei Xu, Chuanxiang Hu, Yue Huang, Yuqi Wang, Jialong Yu, Wei Luo, Ming Gao, Min Zhao, Xiangqian Zheng
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3cbd55a53252e9bb3dfd532de1bccedf85f35d" target='_blank'>
              Tumor-associated-fibrosis and active collagen-CD44 axis characterize a poor-prognosis subtype of gastric cancer and contribute to tumor immunosuppression
              </a>
            </td>
          <td>
            Yingqi Yang, Haohan Sun, Hongkai Yu, Luyao Wang, Chang Gao, Haokun Mei, Xiaomeng Jiang, Minghui Ji
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47537700619a16e63c5ebabefc327d2e5284da8b" target='_blank'>
              Transcriptome analysis of novel B16 melanoma metastatic variants generated by serial intracarotid artery injection
              </a>
            </td>
          <td>
            Jenny C. Kienzler, Erick Contreras, J. Treger, Linda M. Liau, Geoffrey Owens, Robert M. Prins
          </td>
          <td>2025-01-16</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Pineiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cell–cell communications (CCCs) involve signaling from multiple sender cells that collectively impact downstream functional processes in receiver cells. Currently, computational methods are lacking for quantifying the contribution of pairwise combinations of cell types to specific functional processes in receiver cells (e.g. target gene expression or cell states). This limitation has impeded understanding the underlying mechanisms of cancer progression and identifying potential therapeutic targets. Here, we proposed a deep learning-based method, scDCA, to decipher the dominant cell communication assembly (DCA) that have a higher impact on a particular functional event in receiver cells from single-cell RNA-seq data. Specifically, scDCA employed a multi-view graph convolution network to reconstruct the CCCs landscape at single-cell resolution, and then identified DCA by interpreting the model with the attention mechanism. Taking the samples from advanced renal cell carcinoma as a case study, the scDCA was successfully applied and validated in revealing the DCA affecting the crucial gene expression in immune cells. The scDCA was also applied and validated in revealing the DCA responsible for the variation of 14 typical functional states of malignant cells. Furthermore, the scDCA was applied and validated to explore the alteration of CCCs under clinical intervention by comparing the DCA for certain cytotoxic factors between patients with and without immunotherapy. In summary, scDCA provides a valuable and practical tool for deciphering the cell type combinations with the most dominant impact on a specific functional process of receiver cells, which is of great significance for precise cancer treatment. Our data and code are free available at a public GitHub repository: https://github.com/pengsl-lab/scDCA.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c6383c760f6614481b5fab3bdbf070d3264811" target='_blank'>
              scDCA: deciphering the dominant cell communication assembly of downstream functional events from single-cell RNA-seq data
              </a>
            </td>
          <td>
            Boya Ji, Xiaoqi Wang, Xiang Wang, Liwen Xu, Shaoliang Peng
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286075cb8f68cf0d587bedcc36d60116c411594e" target='_blank'>
              Spatially resolved gene expression profiling of tumor microenvironment reveals key steps of lung adenocarcinoma development
              </a>
            </td>
          <td>
            Yuma Takano, Jun Suzuki, Kotaro Nomura, Gento Fujii, Junko Zenkoh, Hitomi Kawai, Yuta Kuze, Y. Kashima, Satoi Nagasawa, Yuka Nakamura, M. Kojima, K. Tsuchihara, Masahide Seki, A. Kanai, D. Matsubara, Takashi Kohno, Masayuki Noguchi, Akihiro Nakaya, Masahiro Tsuboi, G. Ishii, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846ec2188a7971fef054ab5147a97b56d594c528" target='_blank'>
              Crosstalk between cancer-associated fibroblasts and non-neuroendocrine tumor cells in small cell lung cancer involves in glycolysis and antigen-presenting features
              </a>
            </td>
          <td>
            Yuanhua Lu, Hui Li, P. Zhao, Xinyue Wang, Wenjun Shao, Yan Liu, L. Tian, Rui Zhong, Haifeng Liu, Ying Cheng
          </td>
          <td>2024-12-25</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The histopathological neurons in the brain tissue of drug-resistant epilepsy exhibit aberrant cytoarchitecture and imbalanced synaptic circuit function. However, the gene expression changes of these neurons remain unknown, making it difficult for the diagnosis or to dissect the mechanism of drug-resistant epilepsy. By integrating whole-cell patch clamp recording and single-cell RNA sequencing approaches, we identified a transcriptionally distinct subset of cortical pyramidal neurons. These neurons highly expressed genes CDKN1A (P21), CCL2 and NFKBIA that associated with mTOR pathway, inflammatory response and cellular senescence. We confirmed the expression of senescent marker genes in a subpopulation of cortical pyramidal neurons with enlarged soma size in the brain tissue of drug-resistant epilepsy. We further revealed the expression of senescent cell markers P21, P53, COX2, γ-H2AX, β-Gal and reduction of nuclear integrity marker Lamin B1 in histopathological neurons in the brain tissue of drug-resistant epilepsy patients with different pathologies, but not in control brain tissue with no history of epilepsy. Additionally, chronic, but not acute, epileptic seizures induced senescent markers expression in cortical neurons in mouse models of drug-resistant epilepsy. These results provide important molecular markers for histopathological neurons and new insights into the pathophysiological mechanisms of drug-resistant epilepsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532645043d5c919e367cd3f843da422bc996244b" target='_blank'>
              Multimodal single-cell analyses reveal molecular markers of neuronal senescence in human drug-resistant epilepsy
              </a>
            </td>
          <td>
            Qianqian Ge, Jiachao Yang, Fei Huang, Xinyue Dai, Chao Chen, Jinxin Guo, Mi Wang, Mengyue Zhu, Yijie Shao, Yuxian Xia, Yu Zhou, Jieqiao Peng, Suixin Deng, Jiachen Shi, Yiqi Hu, Huiying Zhang, Yi Wang, Xiaoqun Wang, Xiao-Ming Li, Zhong Chen, Yousheng Shu, Jun-Ming Zhu, Jianmin Zhang, Ying Shen, Shumin Duan, Shengjin Xu, Li Shen, Jiadong Chen
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Our research aims to understand the adaptive, ergo potentially metastatic, responses of prostate cancer to changing microenvironments. Emerging evidence implicates a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study revealed that PACC presence in patient prostate tumors at the time of radical prostatectomy was predictive of future metastasis. To test for a causative relationship between PACC state biology and metastasis in prostate cancer, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) from animal models. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Specifically, we applied this approach to analysis of subcutaneous, caudal artery, and intracardiac murine models. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. Furthermore, in vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites. Additional in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence of pro-metastatic mechanisms specific to the PACC state. Implications: Considering that many anti-cancer agents induce the PACC state, our data positions the increased metastatic competency of PACC state cells as an important unforeseen ramification of neoadjuvant regimens, which may help explain clinical correlations between chemotherapy and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c130a571d99ddcd76ddd1df86bc68f827f48ad3" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic.
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T A Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, S. Amend
          </td>
          <td>2024-12-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer, with the worst prognosis among all subtypes. The impact of distinct cell subpopulations within the tumor microenvironment (TME) on TNBC patient prognosis has yet to be clarified. Methods Utilizing single-cell RNA sequencing (scRNA-seq) integrated with bulk RNA sequencing (bulk RNA-seq), we applied Cox regression models to compute hazard ratios, and cross-validated prognostic scoring using a GLMNET-based Cox model. Cell communication analysis was used to elucidate the potential mechanisms of CPVL and MSR1. Ultimately, RNA interference-mediated gene knockdown was utilized to validate the impact of specific genes on the polarization of tumor-associated macrophages (TAMs). Results Our findings revealed that the function of immune cells is more pivotal in prognosis, with TAMs showing the strongest correlation with TNBC patient outcomes, compared with other immune cells. Additionally, we identified CPVL and MSR1 as critical prognostic genes within TAMs, with CPVL expression positively correlated with favorable outcomes and MSR1 expression associated with poorer prognosis. Mechanistically, CPVL may contribute to favorable prognosis by inhibiting the SPP1-CD44 ligand-receptor and promoting CXCL9-CXCR3, C3-C3AR1 ligand-receptor, through which TAMs interact with other cells such as monocytes, neutrophils, and T cells. Moreover, cytokines including IL-18, IFNγR1, CCL20, and CCL2, along with complement-related gene like TREM2 and complement component CFD, may participate in the process of CPVL or MSR1 regulating macrophage polarization. Furthermore, RT-PCR experiments confirmed that CPVL is positively associated with M1-like TAM polarization, while MSR1 is linked to M2-like TAM polarization. Finally, the prognostic significance of these two genes is also validated in HER2-positive breast cancer subtypes. Conclusions CPVL and MSR1 are potential biomarkers for macrophage-mediated TNBC prognosis, suggesting the therapeutic potential of macrophage targeting in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af61ce193245e3b400806d4fcea4eed6dfe425f5" target='_blank'>
              Single-cell Atlas reveals core function of CPVL/MSR1 expressing macrophages in the prognosis of triple-negative breast cancer
              </a>
            </td>
          <td>
            Xinan Wang, Li Lin, Xue Zhang, Minghui Zhang, Zhuo Sun, Yichen Yang, Xiuna Zhang, Yonghui Yuan, Yong Zhang, Hao Chen, Ti Wen
          </td>
          <td>2024-12-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Spine metastases are the most common bone site for breast cancer, with evolving surgery and multidisciplinary care improving outcomes. Current treatments, including chemotherapy and invasive surgery, may damage healthy tissue and may leave residual tumors that lead to recurrence. Cold atmospheric plasma (CAP) offers a non-invasive alternative by delivering reactive oxygen and nitrogen species (RONS) locally to tumor sites, selectively targeting cancer cells while sparing healthy tissue. To assess the impact and selectivity toward tumor cells adjacent to bone-like tissue, we develop a 3D bioprinted tumor-stroma model using a 1% alginate and 7% gelatin cell-laden hydrogel to mimic a bone-like microenvironment. The model co-cultures triple-negative MDA-MB-231 human breast cancer cells with primary human bone marrow mesenchymal stromal cells to simulate tumor-stroma interactions. The effects of CAP treatments are assessed through metabolic activity and viability assays over three days. Results show significant selectivity for cancer cells in both 2D and 3D cultures. CAP minimizes damage to healthy cells, offering the potential for localized treatment over systemic chemotherapies such as doxorubicin. Our novel bioprinted model, combined with a plasma source controlling RONS composition, enables detailed studies of redox-based cancer cell inactivation and highlights CAP as a personalized, non-invasive treatment for bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db15f717a397933ff6371b23deebeffcaae60439" target='_blank'>
              Cold Atmospheric Plasma Selectively Disrupts Breast Cancer Growth in a Bioprinted 3D Tumor Microenvironment Model
              </a>
            </td>
          <td>
            Laura M. Bouret, Jean-Baptiste Billeau, Michael H. Weber, Derek H. Rosenzweig, Stephan Reuter
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatially resolved transcriptomics enable comprehensive measurement of gene expression at subcellular resolution while preserving the spatial context of the tissue microenvironment. While deep learning has shown promise in analyzing SCST datasets, most efforts have focused on sequence data and spatial localization, with limited emphasis on leveraging rich histopathological insights from staining images. We introduce GIST, a deep learning-enabled gene expression and histology integration for spatial cellular profiling. GIST employs histopathology foundation models pretrained on millions of histology images to enhance feature extraction and a hybrid graph transformer model to integrate them with transcriptome features. Validated with datasets from human lung, breast, and colorectal cancers, GIST effectively reveals spatial domains and substantially improves the accuracy of segmenting the microenvironment after denoising transcriptomics data. This enhancement enables more accurate gene expression analysis and aids in identifying prognostic marker genes, outperforming state-of-the-art deep learning methods with a total improvement of up to 49.72%. GIST provides a generalizable framework for integrating histology with spatial transcriptome analysis, revealing novel insights into spatial organization and functional dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eddeb329dda1b1dbe9b8fc2b6a0d8ccb68b0e17" target='_blank'>
              Deep Learning-Enabled Integration of Histology and Transcriptomics for Tissue Spatial Profile Analysis
              </a>
            </td>
          <td>
            Yongxin Ge, Jiake Leng, Ziyang Tang, Kanran Wang, Kaicheng U, Sophia Meixuan Zhang, Sen Han, Yiyan Zhang, Jin-Peng Xiang, Sen Yang, Xiang Liu, Yi Song, Xiyue Wang, Yuchen Li, Junhan Zhao
          </td>
          <td>2024-12-17</td>
          <td>Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5eb0294ad5d50d6c71886518bd626ba65245bb" target='_blank'>
              UBE2J1 is identified as a novel plasma cell-related gene involved in the prognosis of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yunjie Tian, Ruoyu Dong, Yingxia Guan, Ying Wang, Wei Zhao, Jun Zhang, Shan Kang
          </td>
          <td>2025-01-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Perioperative chemotherapy is the standard of care for patients with locally advanced gastric and gastroesophageal junction cancer. Recent evidence demonstrated the addition of programmed cell death protein 1 (PD-1) inhibitors enhanced therapeutic efficacy. However, the mechanisms of response and resistance remain largely undefined. A detailed multiomic investigation is essential to elucidate these mechanisms. Methods We performed whole-exome sequencing, whole-transcriptome sequencing, multiplex immunofluorescence and single-cell RNA sequencing on matched pretreatment and post-treatment samples from 30 patients enrolled in an investigator-initiated Phase 2 clinical trial (NCT04908566). All patients received neoadjuvant PD-1 inhibitors in combination with chemotherapy. A major pathologic response (MPR) was defined as the presence of no more than 10% residual viable tumor cells following treatment. Results Before treatment, the positive ratio of CD3+T cells in both the tumor parenchyma and stroma was significantly higher in the non-MPR group compared with the MPR group (p=0.042 and p=0.013, respectively). Least absolute shrinkage and selection operator regression was employed for feature gene selection and 13 genes were ultimately used to construct a predictive model for identifying MPR after surgery. The model exhibited a perfect area under curve (AUC) of 1.000 (95% CI: 1.000 to 1.000, p<0.001). Post-treatment analysis revealed a significant increase in CD3+T cells, CD8+T cells and NK cells in the tumor stroma of MPR patients. In the tumor parenchyma, aside from a marked increase in CD8+T cells and NK cells, a notable reduction in macrophage was also observed (all p<0.05). Importantly, forkheadbox protein 3 (FOXP3), the principal marker for regulatory T cells (Treg) cells, showed a significant decrease during treatment in MPR patients. FOXP3 expression in the non-MPR group was significantly higher than in the MPR group (p=0.0056) after treatment. Furthermore, single-cell RNA sequencing analysis confirmed that nearly all Treg cells were derived from the non-MPR group. Conclusions Our study highlights the critical role of dynamic changes within the tumor immune microenvironment in predicting the efficacy of neoadjuvant combined immunochemotherapy. We examined the disparities between MPR/non-MPR groups, shedding light on potential mechanisms of immune response and suppression. In addition to bolstering cytotoxic immune responses, specifically targeting Treg cells may be crucial for enhancing treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd7e41ce76cedb5c64fc740c519b9826d4d1968" target='_blank'>
              Multiomics reveals tumor microenvironment remodeling in locally advanced gastric and gastroesophageal junction cancer following neoadjuvant immunotherapy and chemotherapy
              </a>
            </td>
          <td>
            Z. Ji, Xia Wang, Jiaqi Xin, Lijun Ma, Duo Zuo, Hong-li Li, Lan Su, Xinze Lv, S. Ge, Le Zhang, Yong Liu, Yanhui Zhang, Tingting Ding, Ting Deng, Y. Ba, Rui Liu
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
The tumour microenvironment significantly influences the clinical response of patients to therapeutic immune checkpoint inhibition (ICI), but a comprehensive understanding of the underlying immune-regulatory proteome is still lacking.


OBJECTIVES
To decipher targetable biologic processes that determine tumour-infiltrating lymphocytes (TiLs) as a cellular equivalent of clinical response to ICI.


METHODS
We mapped the spatial distribution of proteins in TiL-enriched vs. TiL-low compartments in melanoma by combining microscopy, matrix-assisted laser desorption mass spectrometry imaging and liquid chromatography-mass spectrometry, as well as computational data mining. Pharmacological modulation of sirtuin 1 (SIRT1) activity in syngeneic mouse models was used to evaluate the efficacy of pharmacological SIRT1 activation in two syngeneic melanoma mouse models, one known to be α-programmed cell death protein 1 (PD-1) sensitive and the other α-PD-1 resistant.


RESULTS
Spatial proteomics and gene ontology-based enrichment analysis identified > 145 proteins enriched in CD8high tumour compartments, including negative regulators of mammalian target of rapamycin signalling such as SIRT1. Multiplexed immunohistochemistry confirmed that SIRT1 protein was expressed more in CD8high than in CD8low compartments. Further analysis of bulk and single-cell RNA sequencing data from melanoma tissue samples suggested the expression of SIRT1 by different lymphocyte subpopulations (CD8+ T cells, CD4+ T cells and B cells). Furthermore, we showed in vivo that pharmacological SIRT1 activation increased the immunological effect of α-PD-1 ICI against melanoma cells in mice, which was accompanied by an increase in T-cell infiltration and T-cell-related cytokines, including interferon (IFN)-γ, CCL4, CXCL9, CXCL10 and tumour necrosis factor-α. In silico analysis of large transcriptional data cohorts showed that SIRT1 was positively associated with the proinflammatory T-cell chemokines CXCL9, CXCL10 and IFN-γ, and prolonged overall survival of patients with melanoma.


CONCLUSIONS
Our study deciphers the proteomics landscape in human melanoma, providing important information on the tumour microenvironment and identifying SIRT1 as having important prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e28ba69768cc37ada6cc67883351db03f9a038" target='_blank'>
              Spatial proteomics reveals sirtuin 1 to be a determinant of T-cell infiltration in human melanoma.
              </a>
            </td>
          <td>
            J. Placke, Jenny Bottek, R. Váraljai, B. Shannan, Sarah Scharfenberg, Christoph Krisp, Philippa Spangenberg, Camille Soun, Devon Siemes, Lars Borgards, Franziska Hoffmann, Fang Zhao, Anette Paschen, Hartmut Schlueter, Ferdinand von Eggeling, Iris Helfrich, Florian Rambow, Selma Ugurel, A. Tasdogan, Dirk Schadendorf, D. R. Engel, Alexander Roesch
          </td>
          <td>2024-12-31</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The growth and progression of a neoplasm is a multi–stage process mediated by interactions between tumor cells and their surrounding stroma including various cell types (fibroblasts, endothelial cells, immune cells), cell-free components (cytokines, extracellular vesicles and extracellular matrix), blood and lymphatic vessels that form the tumor microenvironment. During the progression of neoplasms, tumor cells can evade the immune response by suppressing or losing tumor antigens releasing immunosuppressive extracellular vesicles and exosomes, secreting various cytokines, chemokines and other factors, and begin to reprogram and remodel the surrounding stroma. The most relevant areas of TME study are cells and molecules of innate and acquired immunity exhibited both suppressive and stimulating activity against tumor cells due to their high heterogeneity and plasticity. Special attention is paid to immune control points, tumor microbiota, components of intercellular interaction as well as immunotherapy of tumor diseases.
In this paper, a systematic review of scientific publications in PubMed database devoted to the study of TME and published in the period from 2019 to 2024 was carried out. 2547 sources were studied and 68 articles were selected during the screening process. Search queries and keywords included «tumor microenvironment», «immune cells», «tumor», «oncogenesis», «metastasis». The formed group of scientific articles will allow to study in more detail the cellular and molecular
composition of the tumor microenvironment, as well as the main targets for the development of new approaches for malignant neoplasms diagnosis and therapy involving TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/568399709b478d3282ff1a127c4698c1e28c9300" target='_blank'>
              Tumor microenvironment: systematic review
              </a>
            </td>
          <td>
            A. V. Vialichka, D. B. Nizheharodava, Elizaveta M. Nazaranka, Bogdan A. Muzychenka, M. Zafranskaya
          </td>
          <td>2024-12-27</td>
          <td>Journal of the Belarusian State University. Ecology.</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c843d34d2321464470845817a613dfc68c436a" target='_blank'>
              Integrin α8 is a useful cell surface marker of alveolar lipofibroblasts
              </a>
            </td>
          <td>
            A. Fukada, Yasunori Enomoto, R. Horiguchi, Y. Aoshima, S. Meguro, H. Kawasaki, I. Kosugi, T. Fujisawa, N. Enomoto, Naoki Inui, T. Suda, T. Iwashita
          </td>
          <td>2025-01-13</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and Aims Total abdominal colectomy (TAC) with a staged ileal pouch-anal anastomosis (IPAA) is a common surgical treatment for ulcerative colitis (UC). However, a significant percentage of patients experience pouch failure, leading to morbidity. This retrospective case-control study identified histopathological features of the TAC specimen associated with pouch failure and investigated the molecular mechanisms of this susceptibility using single-cell spatial transcriptomics. Methods We analyzed a cohort of 417 patients who underwent IPAA between 2000-2010 at the University of Chicago Medical Center for up to 18 years. Histological examination of TAC specimens focused on disease activity, depth of inflammation, and specific features, including granulomas and deep ulcers. A subset of patients was profiled using single-cell spatial transcriptomics to map gene expression and immune cell interactions in relation to the risk of pouch failure. Results The 18-year pouch failure risk was 23%, with post-procedure diagnosis of CD as a major risk factor (HR = 4.3, 95% CI: 2.3–8.1) as well as high-risk histologic features, including deep chronic inflammation (HR = 21, 95% CI: 11-41) and severe disease activity (HR = 14, 95% CI: 5.7-32) in TAC specimens. Spatial transcriptomics showed immune infiltration of T and myeloid cells, reduced myocyte-glial interactions, and cytokine signaling pathways such as IL-10, IL-1β, and type I/II interferons, associated with an increased risk of pouch failure. Conclusion Histological features and spatial molecular profiling are predictive of IPAA failure. These findings support the use of histologic evaluation and targeted molecular analysis of the TAC specimen to identify high-risk patients and improve IPAA outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a586fb545c9fdbb6b8896dfb9ebdcb9f9f1dc37" target='_blank'>
              Histopathologic Evaluation and Single-Cell Spatial Transcriptomics of the Colon Reveal Cellular and Molecular Abnormalities Linked to J-Pouch Failure in Patients with Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            Andrea D. Olivas, Paul Chak Mou Ngai, Emily Schahrer, K. Olortegui, John F. Cursio, Shintaro Akiyama, Eugene B. Chang, Le Shen, Konstantin Umanskiy, David T. Rubin, David Zemmour, Christopher R. Weber
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Patient-derived organoids represent a novel platform to recapitulate the cancer cells in the patient tissue. While cancer heterogeneity has been extensively studied by a number of omics approaches, little is known about the spatiotemporal kinase activity dynamics. Here we applied a live imaging approach to organoids derived from 10 pancreatic ductal adenocarcinoma (PDAC) patients to comprehensively understand their heterogeneous growth potential and drug responses. By automated wide-area image acquisitions and analyses, the PDAC cells were non-selectively observed to evaluate their heterogeneous growth patterns. We monitored single-cell ERK and AMPK activities to relate cellular dynamics to molecular dynamics. Furthermore, we evaluated two anti-cancer drugs, a MEK inhibitor, PD0325901, and an autophagy inhibitor, hydroxychloroquine (HCQ), by our analysis platform. Our analyses revealed a phase-dependent regulation of PDAC organoid growth, where ERK activity is necessary for the early phase and AMPK activity is necessary for the late stage of organoid growth. Consistently, we found PD0325901 and HCQ target distinct organoid populations, revealing their combination is widely effective to the heterogeneous cancer cell population in a range of PDAC patient-derived organoid lines. Together, our live imaging quantitatively characterized the growth and drug sensitivity of human PDAC organoids at multiple levels: in single cells, single organoids, and individual patients. This study will pave the way for understanding the cancer heterogeneity and promote the development of new drugs that eradicate intractable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824e882598ac674f498c3ffe96037afbee14d231" target='_blank'>
              Quantitative Live Imaging Reveals Phase Dependency of PDAC Patient-Derived Organoids on ERK and AMPK Activity.
              </a>
            </td>
          <td>
            Shoko Tsukamoto, Ye Huaze, Weisheng Zhang, Akihito Machinaga, N. Kakiuchi, Seishi Ogawa, Hiroshi Seno, Shigeki Higashiyama, Michiyuki Matsuda, Toru Hiratsuka
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aede480e99c41ff823fe07aee10fb1b86fa8b" target='_blank'>
              DNA methylation-based analysis reveals accelerated epigenetic aging in giant cell-enriched adult-type glioblastoma
              </a>
            </td>
          <td>
            P. Cakmak, Philipp Jurmeister, I. Divé, P. Zeiner, Joachim P. Steinbach, Tim R Fenton, K. H. Plate, M. Czabanka, Patrick Harter, Katharina J. Weber
          </td>
          <td>2024-12-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ace411cdcea09360bc78194a610964a1bfb00c6" target='_blank'>
              AXL: shapers of tumor progression and immunosuppressive microenvironments
              </a>
            </td>
          <td>
            Yihui Liu, Lei Xu, Yuanyao Dou, Yong He
          </td>
          <td>2025-01-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated. Methods To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays. Results In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1+FOLR2+ tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway. Conclusions We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1+FOLR2+TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cellular activity is spatially organized across different organelles. While several structures are well-characterized, many organelles have unknown roles. Profiling biomolecular composition is key to understanding function but is difficult to achieve in the context of small, dynamic structures. Photoproximity labeling has emerged as a powerful tool for mapping these interaction networks, yet maximizing catalyst localization and reducing toxicity remains challenging in live cell applications. Here, we disclose a new intracellular photocatalyst with minimal cytotoxicity and off-target binding, and we utilize this catalyst for HaloTag-based microenvironment-mapping (μMap) to spatially catalog subnuclear condensates in living cells. We also specifically develop a novel RNA-focused workflow (μMap-seq) to enable parallel transcriptomic and proteomic profiling of these structures. After validating the accuracy of our approach, we generate a spatial map across the nucleolus, nuclear lamina, Cajal bodies, paraspeckles, and PML bodies. These results provide potential new insights into RNA metabolism and gene regulation while significantly expanding the μMap platform for improved live-cell proximity labeling in biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cedc48a841755a5a6c7dc0077077c40278c4804" target='_blank'>
              Parallel Proteomic and Transcriptomic Microenvironment Mapping (μMap) of Nuclear Condensates in Living Cells.
              </a>
            </td>
          <td>
            Steve D. Knutson, C. Pan, Niels Bisballe, Brandon J Bloomer, Philip Raftopolous, Iakovos Saridakis, David W. C. MacMillan
          </td>
          <td>2024-12-21</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc9bd11ddbadc141e7404167a9d1237fb1c3a74" target='_blank'>
              Encompassing view of spatial and single-cell RNA sequencing renews the role of the microvasculature in human atherosclerosis.
              </a>
            </td>
          <td>
            Tore Bleckwehl, Anne Babler, Merel Tebens, Sidrah Maryam, Michael Nyberg, Markus Bosteen, Maurice Halder, Isaac Shaw, Susanne Fleig, Charles Pyke, Henning Hvid, Louise M. Voetmann, J. V. van Buul, J. Sluimer, Vivek Das, Simon J. Baumgart, R. Kramann, Sikander Hayat
          </td>
          <td>2024-12-23</td>
          <td>Nature cardiovascular research</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d8247ee8babd323978806a38c5b3ce6dbebe55" target='_blank'>
              Cell-specific transcriptional signatures of vascular cells in Alzheimer’s disease: perspectives, pathways, and therapeutic directions
              </a>
            </td>
          <td>
            Soumilee Chaudhuri, M. Cho, Julia C Stumpff, Paula Bice, Özkan İş, Nilufer Ertekin‐Taner, A. Saykin, K. Nho
          </td>
          <td>2025-01-29</td>
          <td>Molecular Neurodegeneration</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Viral vectors have successfully modified T cells to express chimeric antigen receptors (CAR), leading to clinical approvals. However, their high cost and regulatory challenges hinder rapid and broad clinical translation. Here, we demonstrate that our lentivirally (LV) manufactured R110-CAR T cells, targeting a leukemia neoepitope, can also be engineered using non-viral Sleeping Beauty (SB) transposition with minimal-sized DNA vectors. Flow cytometry and single-cell sequencing was used to compare the two production modes using healthy donor and CLL patient-derived T cells and a CD19-CAR T cell control. SB products were shifted towards CD8+ subsets with expression of activation/co-inhibition markers (CD69, LAG-3, TIM-3) despite their naïve-like phenotype and lack of antigenic challenge. The CAR binding moiety modulated these patterns with R110-CAR T cells showing more aberrant phenotypes. Moreover, SB engineering resulted in inflammatory signatures along with RIG-I-like and TOLL-like nucleotide sensing potentially resulting from the transfection procedure. Patient-derived products showed significantly fewer CAR-expressing cells, reduced proliferation clusters, and lower T cell diversity, particularly with SB manufacturing, pointing at potential challenges with this method when engineering CLL T cells. Together, our data suggest that the engineering mode may substantially influence T cell properties and that these are further modulated by the CAR binding moiety and the type of T cell donor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091bcf82c5312a9b158fdb202c07c565ad9cabb7" target='_blank'>
              CAR T cell engineering impacts antigen-independent activation and co-inhibition
              </a>
            </td>
          <td>
            Simon Stücheli, Christoph Schultheiss, Paul Schmidt-Barbo, Andreas Zingg, Natascha Franz, Sarah Adamo, Claudia Fischer, H. Läubli, Mascha Binder
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55057507874b2d1db614aaeb55e2719a4aaa942d" target='_blank'>
              Metabolic rewiring in skin epidermis drives tolerance to oncogenic mutations.
              </a>
            </td>
          <td>
            Anupama Hemalatha, Zong-Hu Li, David G. Gonzalez, C. Matte-Martone, Karen Tai, Elizabeth Lathrop, Daniel Gil, S. Ganesan, Lauren E Gonzalez, Melissa C. Skala, Rachel J. Perry, Valentina Greco
          </td>
          <td>2025-01-06</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5afe786b47a61ca4c6c86b698a4b6f0aa1df3" target='_blank'>
              Characterizing tertiary lymphoid structures associated single-cell atlas in breast cancer patients
              </a>
            </td>
          <td>
            Xiaokai Fan, Daqin Feng, Donggui Wei, Anqi Li, Fangyi Wei, Shufang Deng, Muling Shen, Congzhi Qin, Yongjia Yu, Lun Liang
          </td>
          <td>2025-01-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The article provides an overview of the current understanding of the interplay between metabolic pathways and immune function in the context of triple-negative breast cancer (TNBC). It highlights recent advancements in single-cell and spatial transcriptomics technologies, which have revolutionized the analysis of tumor heterogeneity and the immune microenvironment in TNBC. The review emphasizes the crucial role of metabolic reprogramming in modulating immune cell function, discussing how specific metabolic pathways, such as glycolysis, lipid metabolism, and amino acid metabolism, can directly impact the activity and phenotypes of various immune cell populations within the TNBC tumor microenvironment. Furthermore, the article explores the implications of these metabolic-immune interactions for the efficacy of immune checkpoint inhibitor (ICI) therapies in TNBC, suggesting that strategies targeting metabolic pathways may enhance the responsiveness to ICI treatments. Finally, the review outlines future directions and the potential for combination therapies that integrate metabolic modulation with immunotherapeutic approaches, offering promising avenues for improving clinical outcomes for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0255c143266458c046ad8fac98c1a3117d3b268e" target='_blank'>
              Exploring the role of metabolic pathways in TNBC immunotherapy: insights from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Shi-liang Chen, Yi-ran Fei, Xin-xian Cai, Cong Wang, Shi-Yuan Tong, Zhe-zhong Zhang, Yan-xia Huang, Dan-dan Bian, Yi-bo He, Xiao-xiao Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, , Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The molecular non-genetic changes of resistance to sotorasib are currently uncertain. The aim of this study was to generate a sotorasib-resistant cell line via selective pressure and systematically examine the molecular and phenotypic alterations caused by resistance.Mutant NCI-H358 (KRASG12C) were exposed to incremental doses (2–512 nM) of sotorasib. Then, resistant clones were separated by single-cell sorting. Proliferation was analyzed in real-time by xCELLigence; protein profiles were quantified by protein arrays; and mRNA expression profile was measured using the PanCancer Pathways panel by NanoString. In silico analyses were conducted from a database comprising patient-derived xenograft (PDX) models and cell lines resistant to sotorasib. AKT and p38. The synergistic effect of combining AKT, p38, and EGFR inhibitors was assessed using the SynergyFinder platform. Additionally, AKT and p38 genes were silenced using esiRNA.Sotorasib-resistant H358-R cell line displayed markers of the mesenchymal-epithelial transition and loss of cell adhesion. Were identified 30 overexpressed genes in the resistance model, implicating in signaling pathways that leads to AKT activation and heightened protein expression levels of phosphorylated AKT and p38. To identify potential therapeutic strategies for overcoming sotorasib resistance, we investigated the combination of AKT and p38 inhibitors. Notably, combined inhibition of AKT (MK2206) and p38 (adezmapimod) restored sensitivity to sotorasib in resistant cell lines, as did silencing AKT expression.These findings underscore the importance of adaptive mechanisms in sotorasib resistance in NSCLC cells contributing by EMT activation and demonstrates synergic combination with AKT and p38 inhibitors to restore sotorasib sensitivity in KRASG12C cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2b84647241a84761deba81b95c6e8effb443d" target='_blank'>
              Sotorasib resistance triggers epithelial-mesenchymal transition and activates AKT and P38-mediated signaling
              </a>
            </td>
          <td>
            R. A. Megid, G. G. Ribeiro, I. Gomes, A. C. Laus, L. Ferro Leal, Luciane Sussuchi da Silva, A. A. Ariwoola, J. M. Dias, R. Reis, Renato Jose da Silva-Oliveira
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
 Gastric cancer (GC) poses a substantial risk to human health due to its high prevalence and mortality rates. Nevertheless, current therapeutic strategies remain insufficient. Single-cell RNA sequencing (scRNA-seq) offers the potential to provide comprehensive insights into GC pathogenesis.
 AIM
 To explore the distribution and dynamic changes of cell populations in the GC tumor microenvironment using scRNA-seq techniques.
 METHODS
 Cancerous tissues and paracancerous tissues were obtained from patients diagnosed with GC at various stages (I, II, III, and IV). Single-cell suspensions were prepared and analyzed using scRNA-seq to examine transcriptome profiles and cell-cell interactions. Additionally, quantitative real-time polymerase chain reaction (qRT-PCR) and flow cytometry were applied for measuring the expression of cluster of differentiation (CD) 2, CD3D, CD3E, cytokeratin 19, cytokeratin 8, and epithelial cell adhesion molecules.
 RESULTS
 Transcriptome data from 73645 single cells across eight tissues of four patients were categorized into 25 distinct cell clusters, representing 10 different cell types. Variations were observed in these cell type distribution. The adjacent epithelial cells in stages II and III exhibited a degenerative trend. Additionally, the quantity of CD4 T cells and CD8 T cells were evidently elevated in cancerous tissues. Interaction analysis displayed a remarkable increase in interaction between B cells and other mast cells in stages II, III, and IV of GC. These findings were further validated through qRT-PCR and flow cytometry, demonstrating elevated T cells and declined epithelial cells within the cancerous tissues.
 CONCLUSION
 This study provides a comprehensive analysis of cell dynamics across GC stages, highlighting key interactions within the tumor microenvironment. These findings offer valuable insights for developing novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3adb4df70cb2b109a0b602e7b75fd956596d307f" target='_blank'>
              Landscape of four different stages of human gastric cancer revealed by single-cell sequencing
              </a>
            </td>
          <td>
            Xushan Tang, Chun-Lei Xu, Na Li, Jian-Qing Zhang, Yong Tang
          </td>
          <td>2025-02-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38fe8b7dd282acd4da49fd77a60ef2b609353e0" target='_blank'>
              S100A13-driven interaction between pancreatic adenocarcinoma cells and cancer-associated fibroblasts promotes tumor progression through calcium signaling
              </a>
            </td>
          <td>
            Liuyuan Xia, Xin Guo, Dong Lu, Yixin Jiang, Xiaohui Liang, Yiwen Shen, Jia-Yi Lin, Lijun Zhang, Hongzhuan Chen, Jinmei Jin, Xin Luan, Weidong Zhang
          </td>
          <td>2025-01-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The TEM8 protein coded by the ANTXR1 gene represents an emerging biomarker in solid tumors. In addition to the various roles TEM8 plays in oncogenesis, including angiogenesis, epithelial-to-mesenchymal transition, and cell migration, it has also been shown that the overexpression of the ANTXR1 gene in solid tumors correlates with poor prognostic indicators in several solid tumor histologies. As such, TEM8 has been identified as the target of novel oncologic therapies. It is especially attractive given its selective expression on the surface of solid tumor cells and associated stromal cells, such as cancer stem cells, invasive cancer cells, and immune cells, such as macrophages, angiogenic endothelial cells, pericytes, and cancer-associated fibroblasts. Furthermore, TEM8 plays this unique role as a mostly non-mutated gene in solid cancers. Here, we demonstrate that elevated expression of ANTXR1 in bladder cancer showed a statistical difference not only in overall survival (OS) but in progression-free survival (PFS), confirming the prognostic biomarker power of ANTXR1 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dff315ebf7de91444ff283e65fc1f528bbeae62" target='_blank'>
              Elevated expression of ANTXR1 gene in tumors is a poor prognostic biomarker for patients with bladder cancer
              </a>
            </td>
          <td>
            L. S. Franco, S. Arunachalam, A. Chauhan, S. A. Kareff, P. Hallenbeck
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is a major challenge that was recognized as one of the first Cancer Grand Challenges, with a call to provide solutions to visualize tumor heterogeneity. The Rosetta team took on this challenge, exploiting advances in spatial-omics approaches centered around mass spectrometry imaging to map tumor heterogeneity at the cellular and molecular scales with different levels of resolution. See related article by Bressan et al., p. 16 See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d308601bb28236e43c3bc271c19d29c8dd8720ed" target='_blank'>
              Visualizing Cancer Heterogeneity at the Molecular and Cellular Levels: Lessons from Rosetta.
              </a>
            </td>
          <td>
            R. Goodwin, John F Marshall, G. Poulogiannis, Mariia Yuneva, K. Brindle, Z. Takáts, Owen J. Sansom, Josephine Bunch, Simon T Barry
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="It is important to systematically identify tumor suppressor genes (TSGs) to improve our understanding of tumorigenesis and develop strategies for early diagnosis and mitigating disease progression. In the present study, we used an in vivo genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein 9 (Cas9) screen and identified FPS/FES-related (FER) as a TSG. Single-cell RNA sequencing (scRNA-seq) revealed that normal cells with low FER expression exhibited elevated malignant transformation potential and stemness properties. FER knockout promoted the tumorigenic transformation, characterized by high colony-forming efficiency and suspension growth ability, acquired tumorigenicity in vivo, increased metabolic activity, dedifferentiation properties, and immune evasion. Moreover, analysis revealed that low FER expression tumors share molecular phenotypes with FER knockout cells, suggesting the consistent role of FER in tumor initiation and progression. Taken together, our findings not only provide insights into the essential role of FER as a tumor suppressor in tumor initiation and progression but also highlight its potential as a target for future clinical diagnosis. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0ef168e6b6a500c7fdbd949d7e67ca980d8052" target='_blank'>
              CRISPR-Cas9 screening identifies the role of FER as a tumor suppressor.
              </a>
            </td>
          <td>
            Jiaqi Wang, Ran Yang, Feng-Po Wang, Junlei Zhang, Yutong Dong, Jiangjun Wang, Meng Yu, Yixiao Xu, Lianlian Liu, Yuda Cheng, Chen Zhang, Yi Yang, Wubin Yang, Jiali Wang, Guangxing Chen, Yi Huang, Yanping Tian, Rui Jian, Bing Ni, Wei Wu, Y. Ruan
          </td>
          <td>2024-12-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Urothelial carcinoma (UC) can arise from either the lower urinary tract or the upper tract; they represent different disease entities and require different clinical treatment strategies. A full understanding of the cellular characteristics in UC may guide the development of novel therapies. Here, we performed single-cell transcriptome analysis from four patients with UC of the bladder (UCB), five patients with UC of the ureter (UCU), and four patients with UC of the renal pelvis (UCRP) to develop a comprehensive cell atlas of UC. We found the rare epithelial cell subtype EP9 with epithelial-to-mesenchymal transition (EMT) and cancer stem cell (CSC) features, and specifically expressed SOX6, which was associated with poor prognosis. We also found that ACKR1+ endothelial cells and inflammatory cancer-associated fibroblasts (iCAFs) were more enriched in UCU, which may promote pathogenesis. While ESM1+ endothelial cells may more actively participate in UCB and UCRP tumorigenesis by promoting angiogenesis. Additionally, CD8 + effector T cells were more enriched in UCU and UCRP patients, while Tregs were mainly enriched in UCB tumors. C1QC+ macrophages and LAMP3+ dendritic cells were more enriched in UCB, which is closely related to the formation of the heterogeneous immunosuppressive microenvironment. Furthermore, we found strong interactions between iCAFs, EP9, and Endo_ESM1, and different degrees of activation of the FGF-FGFR3 axis and immune checkpoint pathway were observed in different UC subtypes. Our study elucidated the cellular heterogeneity and the components of the microenvironment in UC arising from the upper and lower urinary tracts and provided novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a7771d79701d4fbee36981bff69a8e0ebdd585" target='_blank'>
              Single-Cell RNA Sequencing Reveals the Tumor Heterogeneity and Immunosuppressive Microenvironment in Urothelial Carcinoma.
              </a>
            </td>
          <td>
            Tianqi Lyu, Kerong Wu, Yincong Zhou, Tong Kong, Lin Li, Kaizhe Wang, Pan Fu, Pengyao Wei, Ming Chen, Jianping Zheng
          </td>
          <td>2024-12-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Idiopathic pulmonary fibrosis (IPF) is a progressive lung disease characterized by repetitive alveolar injuries with excessive deposition of extracellular matrix (ECM) proteins. A crucial need in understanding IPF pathogenesis is identifying cell types associated with histopathological regions, particularly local fibrosis centers known as fibroblast foci. To address this, we integrated published spatial transcriptomics and single-cell RNA sequencing (scRNA-seq) transcriptomics and adopted the Query method and the Overlap method to determine cell type enrichments in histopathological regions. Distinct fibroblast cell types are highly associated with fibroblast foci, and transitional alveolar type 2 and aberrant KRT5-/KRT17+ (KRT: keratin) epithelial cells are associated with morphologically normal alveoli in human IPF lungs. Furthermore, we employed laser capture microdissection-directed mass spectrometry to profile proteins. By comparing with another published similar dataset, common differentially expressed proteins and enriched pathways related to ECM structure organization and collagen processing were identified in fibroblast foci. Importantly, cell type enrichment results from innovative spatial proteomics and scRNA-seq data integration accord with those from spatial transcriptomics and scRNA-seq data integration, supporting the capability and versatility of the entire approach. In summary, we integrated spatial multi-omics with scRNA-seq data to identify disease-associated cell types and potential targets for novel therapies in IPF intervention. The approach can be further applied to other disease areas characterized by spatial heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947f84484c40c13b0563bade74b2ae0f8c1ec3ac" target='_blank'>
              Novel Integration of Spatial and Single-Cell Omics Data Sets Enables Deeper Insights into IPF Pathogenesis
              </a>
            </td>
          <td>
            Fei Wang, Liang Jin, Xue Wang, Baoliang Cui, Yingli Yang, Lori Duggan, Annette Schwartz Sterman, Sarah M. Lloyd, L. Hazelwood, Neha Chaudhary, Bhupinder Bawa, Lucy A. Phillips, Yupeng He, Yu Tian
          </td>
          <td>2025-01-13</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbin Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Yingcheng Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Personalized radiotherapy based on the intrinsic sensitivity of individual tumors is anticipated, however, it has yet to be realized. To explore cancer radiosensitivity, analysis in combination with omics data is important. The Cancer Cell Line Encyclopedia (CCLE) provides multi-layer omics data for hundreds of cancer cell lines. However, the radiosensitivity counterpart is lacking. To address this issue, we aimed to establish a database of radiosensitivity, as assessed by the gold standard clonogenic assays, for the CCLE cell lines by collecting data from the literature. A deep learning-based screen of 33,284 papers identified 926 relevant studies, from which SF2 (survival fraction after 2 Gy irradiation) data were extracted. The median SF2 (mSF2) was calculated for each cell line, generating an mSF2 database comprising 285 cell lines from 28 cancer types. The mSF2 showed a normal distribution among higher and lower cancer-type hierarchies, demonstrating a large variation across and within cancer types. In selected cell lines, mSF2 correlated significantly with the single-institution SF2 obtained using standardized experimental protocols or with integral survival, a radiosensitivity index that correlates with clonogenic survival. Notably, the mSF2 for blood cancer cell lines was significantly lower than that for solid cancer cell lines, which is in line with the empirical knowledge that blood cancers are radiosensitive. Furthermore, the CCLE-derived protein levels of NFE2L2 and SQSTM1, which are involved in antioxidant damage responses that confer radioresistance, correlated significantly with mSF2. These results suggest the robustness and potential utility of the mSF2 database, linked to multi-layer omics data, for exploring cancer radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532979073d1bb70a6b201c4bde03ed34d813df58" target='_blank'>
              Establishment of cancer cell radiosensitivity database linked to multi-layer omics data.
              </a>
            </td>
          <td>
            T. Oike, Ryosuke Kambe, Narisa Dewi Maulany Darwis, Atsushi Shibata, Tatsuya Ohno
          </td>
          <td>2024-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Introduction: Prostate cancer, notably prostate adenocarcinoma (PARD), has high incidence and mortality rates. Although typically resistant to immunotherapy, recent studies have found immune targets for prostate cancer. Stratifying patients by molecular subtypes may identify those who could benefit from immunotherapy. Methods: We used single-cell and bulk RNA sequencing data from GEO and TCGA databases. We characterized the tumor microenvironment at the single-cell level, analyzing cell interactions and identifying fibroblasts linked to mitophagy. Target genes were narrowed down at the bulk transcriptome level to construct a PARD prognosis prediction nomogram. Unsupervised consensus clustering classified PARD into subtypes, analyzing differences in clinical features, immune infiltration, and immunotherapy. Furthermore, the cellular functions of the genes of interest were verified in vitro. Results: We identified ten cell types and 160 mitophagy-related single-cell differentially expressed genes (MR-scDEGs). Strong interactions were observed between fibroblasts, endothelial cells, CD8+ T cells, and NK cells. Fibroblasts linked to mitophagy were divided into six subtypes. Intersection of DEGs from three bulk datasets with MR-scDEGs identified 26 key genes clustered into two subgroups. COX regression analysis identified seven prognostic key genes, enabling a prognostic nomogram model. High and low-risk groups showed significant differences in clinical features, immune infiltration, immunotherapy, and drug sensitivity. In prostate cancer cell lines, CAV1, PALLD, and ITGB8 are upregulated, while CLDN7 is downregulated. Knockdown of PALLD significantly inhibits the proliferation and colony-forming ability of PC3 and DU145 cells, suggesting the important roles of this gene in prostate cancer progression. Conclusions: This study analyzed mitophagy-related genes in PARD, predicting prognosis and aiding in subtype identification and immunotherapy response analysis. This approach offers new strategies for treating prostate cancer with specific molecular subtypes and helps develop potential biomarkers for personalized medicine strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab8cd0df51baf0c065fc9d46908e143b732254" target='_blank'>
              An Integrated Approach Utilizing Single-Cell and Bulk RNA-Sequencing for the Identification of a Mitophagy-Associated Genes Signature: Implications for Prognostication and Therapeutic Stratification in Prostate Cancer
              </a>
            </td>
          <td>
            Yuke Zhang, Li Ding, Zhijin Zhang, Liliang Shen, Yadong Guo, Wentao Zhang, Yang Yu, Zhuoran Gu, Ji Liu, Aimaitiaji Kadier, Jiang Geng, Shiyu Mao, Xudong Yao
          </td>
          <td>2025-01-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neuroblastoma, the most common pediatric extracranial solid tumor, arises from the malignant transformation of neural crest progenitors in the peripheral nervous system. Its clinical and genetic heterogeneity poses significant challenges, especially in high-risk patients with metastatic disease. Two plastic neuroblastoma cell phenotypes, adrenergic (ADR) and mesenchymal (MES), have been identified. Notably, MES neuroblastoma cells exhibit increased migration and chemoresistance. Cancer-associated fibroblasts (CAFs) in the tumor microenvironment further promote tumor aggressiveness by enhancing cancer cell proliferation, extracellular matrix remodeling, angiogenesis and metastasis. This study explored the role of non-activated fibroblasts in ADR and MES neuroblastoma cell proliferation, migration and invasion in vitro and in vivo. Results showed that MES and ADR neuroblastoma cells influenced fibroblast activation into CAFs differently, with MES cells promoting a more invasive environment leading to tumor spread. These findings enhance our understanding of how ADR and MES phenotypes contribute to the formation of a pro-metastatic niche by activating fibroblasts in CAFs. This insight could inform new therapeutic strategies targeting the tumor microenvironment to prevent neuroblastoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b082119b7251f72ea43b0b4ec46d8cdafa28403e" target='_blank'>
              Exploring the Role of Fibroblasts in Promoting Neuroblastoma Cell Migration and Invasion
              </a>
            </td>
          <td>
            Diana Corallo, Cristina Nardelli, Marcella Pantile, Sara Menegazzo, Alessandra Biffi, Sanja Aveic
          </td>
          <td>2024-12-08</td>
          <td>Journal of Nanotheranostics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2ea126a4602d2754c646d3838a0c077f74d91b" target='_blank'>
              CXCL14 in prostate cancer: complex interactions in the tumor microenvironment and future prospects
              </a>
            </td>
          <td>
            Lei Tang, Xin Chen, Jianquan Hou, Xuedong Wei
          </td>
          <td>2025-01-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 The reactivation of neurodevelopmental programs in cancer highlights parallel biological processes that occur in both normal development and brain tumors. Achieving a deeper understanding of how dysregulated developmental factors play a role in the progression of brain tumors is therefore crucial for identifying potential targets for therapeutic interventions. Single-cell RNA sequencing (scRNA-Seq) provides an opportunity to understand how developmental programs are dysregulated and reinitiated in brain tumors at single-cell resolution. The aim of this study is to identify developmental origins of brain tumors using scRNA-Seq data.



 Here, we introduce COORS (Cell Of ORigin like CellS), a computational tool trained on developmental human brain single-cell datasets that annotates “developmental-like” cell states in brain tumors. COORS leverages cell type-specific multilayer perceptron models and incorporates a developmental cell type tree that reflects hierarchical relationships and models cell type probabilities.



 Applying COORS to various brain cancer datasets, including medulloblastoma (MB), glioma, and diffuse midline glioma (DMG), we identified developmental-like cells that represent putative cells of origin in these tumors. Our method provides both cell of origin classification and cell age regression, offering insights into the developmental cell types of tumor subgroups. COORS identified outer radial glia (oRG) developmental cells within IDHWT glioma cells whereas oligodendrocyte precursor cells (OPCs) and neuronal-like cells in IDHMut. Interestingly, IDHMut subgroup cells that map to OPC show bimodal distributions, that are both early and late weeks in development. Furthermore, COORS offers a valuable resource by providing novel markers linked to developmental states within of MB, glioma and DMG tumor subgroups.



 Our work adds to our cumulative understanding of brain tumor heterogeneity and helps pave the way for tailored treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e403b1c16cfb5c2e03317b6d1828fc8f0bdaf21" target='_blank'>
              Inferred Developmental Origins of Brain Tumors from Single Cell RNA-Sequencing Data
              </a>
            </td>
          <td>
            Su Wang, R. Curry, Malcolm F McDonald, Hyun Yong Koh, Anders W Erickson, Claudia L Kleinman, Michael D. Taylor, Ganesh Rao, Benjamin Deneen, Arif O. Harmanci, A. Harmanci
          </td>
          <td>2025-01-23</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION Alzheimer’s disease (AD), Dementia with Lewy bodies (DLB), and Parkinson’s disease (PD) represent a spectrum of neurodegenerative disorders (NDDs). Here, we performed the first direct comparison of their transcriptomic landscapes. METHODS We profiled the whole transcriptomes of NDD cortical tissue by snRNA-seq. We used computational analyses to identify common and distinct differentially expressed genes (DEGs), biological pathways, vulnerable and disease-driver cell subtypes, and alteration in cell-to-cell interactions. RESULTS The same vulnerable inhibitory neuron subtype was depleted in both AD and DLB. Potentially disease-driving neuronal cell subtypes were present in both PD and DLB. Cell-cell communication was predicted to be increased in AD but decreased in DLB and PD. DEGs were most commonly shared across NDDs within inhibitory neuron subtypes. Overall, we observed the greatest transcriptomic divergence between AD and PD, while DLB exhibited an intermediate transcriptomic signature. DISCUSSION These results help explain the clinicopathological spectrum of this group of NDDs and provide unique insights into the shared and distinct molecular mechanisms underlying the pathogenesis of NDDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377545b1eb6117f0f10ff3c8fe9499887d75cc9d" target='_blank'>
              Comparative mapping of single-cell transcriptomic landscapes in neurodegenerative diseases
              </a>
            </td>
          <td>
            E. K. Shwab, Zhaohui Man, Daniel C. Gingerich, Julia E. Gamache, M. Garrett, G. Serrano, T. Beach, Gregory E. Crawford, Allison E Ashley-Koch, O. Chiba-Falek
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Background The goal of precision oncology is to find effective therapeutics for every patient. Through the inclusion of emerging therapeutics in a high‐throughput drug screening platform, our functional genomics pipeline inverts the common paradigm to identify patient populations that are likely to benefit from novel therapeutic strategies. Approach Utilizing drug screening data across a panel of 46 cancer cell lines from 11 tumor lineages, we identified an ovarian cancer‐specific sensitivity to the first‐in‐class CRL4 inhibitors KH‐4‐43 and 33‐11. CRL4 (i.e., Cullin‐4 RING E3 ubiquitin ligase) is known to be dysregulated in a variety of cancer contexts, making it an attractive therapeutic target. Unlike proteasome inhibitors that are associated with broad toxicity, CRL4 inhibition offers the potential for tumor‐specific effects. Results We observed that CRL4 inhibition negatively regulates core gene signatures that are upregulated in ovarian tumors and significantly slowed tumor growth as compared to the standard of care, cisplatin, in OVCAR8 xenografts. Building on this, we performed combination drug screening in conjunction with proteomic and transcriptomic profiling to identify ways to improve the antitumor effects of CRL4 inhibition in ovarian cancer models. CRL4 inhibition consistently resulted in activation of the mitogen‐activated protein kinase (MAPK) signaling cascade at both the transcriptomic and protein levels, suggesting that survival signaling is induced in response to CRL4 inhibition. These observations were concordant with the results of the combination drug screens in seven ovarian cancer cell lines that showed CRL4 inhibition cooperates with MEK inhibition. Preclinical studies in OVCAR8 and A2780 xenografts confirmed the therapeutic potential of the combination of KH‐4‐43 and trametinib, which extended overall survival and slowed tumor progression relative to either single agent or the standard of care. Conclusions Together, these data demonstrate the prospective utility of functional modeling pipelines for therapeutic development and underscore the clinical potential of CRL4 inhibition in the ovarian cancer context. Highlights A precision medicine pipeline identifies ovarian cancer sensitivity to CRL4 inhibitors. CRL4 inhibition induces activation of MAPK signalling as identified by RNA sequencing, proteomics, and phosphoproteomics. Inhibitor combinations that target both CRL4 and this CRL4 inhibitor‐induced survival signalling enhance ovarian cancer sensitivity to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c6bcbc9392bde91ee7e96491c8582460ba6dd" target='_blank'>
              Functional genomics pipeline identifies CRL4 inhibition for the treatment of ovarian cancer
              </a>
            </td>
          <td>
            S. Claridge, Shalini Nath, Anneliese Baum, Richard Farias, J. Cavallo, Nile M Rizvi, Lamberto De Boni, Eric Park, Genesis Lara Granados, Matthew Hauesgen, Ruben Fernandez-Rodriguez, Eda N Kozan, E. Kanshin, Khoi Q Huynh, Peng-Jen Chen, Kenneth Wu, Beatrix Ueberheide, Juan Miguel Mosquera, Fred R Hirsch, Robert J. DeVita, O. Elemento, Chantal Pauli, Zhen-Qiang Pan, Benjamin D Hopkins
          </td>
          <td>2025-01-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The kinetic parameters of cancer population dynamics are critical for developing reliable predictors of tumour growth patterns, extracting metrics for patient stratification and creating algorithms that can forecast clinically significant events. Here, we introduce a model-based Bayesian framework that leverages longitudinal phenotypic (e.g., tumour volume, cell counts) or genotypic (e.g., mutation frequency) data to infer critical parameters of tumour progression within a single patient. Our models uses population genetics to estimate probability distributions for tumour growth rates, initiation and extinction times, pinpointing abrupt shifts in tumour dynamics due to treatment response and revealing associations between drug resistance and pre-existing cancer cell populations. We apply our framework to address pivotal clinical questions across three major cancer types. In colorectal cancer, we use tumour markers data to identify extensive pre-existing RAS-linked resistance to cetuximab. In lung cancer, we use somatic mutation frequencies in citculating tumour DNA to determine prognostic growth rates and develop a test for monitoring minimal residual disease. In chronic lymphocytic leukaemia, we use white blood cell counts to stratify patients by growth patterns and predict time to treatment, advancing adaptive monitoring strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e324d640559e2fae6f8be8e484eb8d56fee86230" target='_blank'>
              Model-based Bayesian inference of cancer dynamics from heterogenous longitudinal data
              </a>
            </td>
          <td>
            Giovanni Santacatterina, R. Bergamin, A. Zucchetto, Roberta Laureana, M. D. del Principe, V. Gattei, Leonardo Egidi, G. Caravagna
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6984511f4cd3709e0746596a9a57c5a7832bd7" target='_blank'>
              Ovarian tumor cells gain competitive advantage by actively reducing the cellular fitness of microenvironment cells.
              </a>
            </td>
          <td>
            E. Madan, António M Palma, Vignesh Vudatha, Amit Kumar, P. Bhoopathi, Jochen Wilhelm, T. Bernaś, P. Martin, Gaurav Bilolikar, Aenya Gogna, M. L. Peixoto, Isabelle Dreier, Thais Fenz Araujo, Elena Garre, Anna Gustafsson, K. Dorayappan, Narsimha Mamidi, Zhaoyu Sun, Michail Yekelchyk, Davide Accardi, Amalie Lykke Olsen, Lin Lin, Asaf Ashkenazy Titelman, Michael Bianchi, Phil Jessmon, Elnaz Abbasi Farid, A. Pradhan, L. Neufeld, E. Yeini, Santanu Maji, Christopher J Pelham, Hyobin Kim, Daniel Oh, Hans Olav Rolfsnes, Rita C Marques, Amy Lu, Masaki Nagane, Sahil Chaudhary, Kartik Gupta, Keshav C Gogna, Ana Bigio, Karthikeya Bhoopathi, Padmanabhan Mannangatti, K. G. Achary, Javed Akhtar, S. Belião, Swadesh K. Das, I. Correia, Cláudia L da Silva, A. Fialho, M. Poellmann, Kaila Javius-Jones, Adam Hawkridge, Sanya Pal, Kumari S Shree, E. Rakha, Sambhav Khurana, Gaoping Xiao, Dongyu Zhang, Arjun Rijal, Charles Lyons, Steven R. Grossman, David P. Turner, R. Pillappa, Karanvir Prakash, Gaurav Gupta, Gary L W G Robinson, Jennifer E Koblinski, Hongjun Wang, Gita Singh, Sujay Singh, Sagar Rayamajhi, M. Bacolod, Hope Richards, Sadia Sayeed, Katherine P Klein, David Chelmow, R. Satchi‐Fainaro, K. Selvendiran, Denise Connolly, F. Thorsen, Rolf Bjerkvig, K. Nephew, Michael O Idowu, M. Kühnel, Christopher Moskaluk, Seungpyo Hong, William L Redmond, Göran Landberg, Antonio Lopez-Beltran, Andrew Poklepovic, Arun J Sanyal, Paul B Fisher, George M. Church, U. Menon, Ronny Drapkin, Andrew K Godwin, Yonglun Luo, Maximilian Ackermann, A. Tzankov, Kirsten D. Mertz, Danny Jonigk, Allan Tsung, David Sidransky, Jose Trevino, Arturo P Saavedra, Robert Winn, Kyoung Jae Won, Eduardo Moreno, Rajan Gogna
          </td>
          <td>2024-12-09</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is one of the most prevalent causes of cancer-related morbidity and mortality worldwide. Late-stage detection and the complex molecular mechanisms driving tumor progression contribute significantly to its poor prognosis. Dysregulated R-loops, three-stranded nucleic acid structures associated with genome instability, play a key role in the malignant characteristics of various tumors. However, the detailed role and mechanism of R-loops in HCC progression remain elusive and require further exploration. This study aimed to construct an R-loop scoring signature centered on prognosis and lipid metabolism, thereby enhancing our understanding of HCC progression and identifying potential therapeutic targets. Methods In this study, we utilized the single-cell RNA-sequencing (scRNA-seq) data from HCC patients (GSE149614 and CRA002308) to construct an R-loop scoring model based on the identified R-loop regulator genes (RLRGs) related to HBV infection through WGCNA analysis. We also explored the tumor microenvironment and intercellular communication related to R-loop score. Additionally, a prognostic risk model based on the fatty acid metabolism-associated RLRGs was constructed using data from the TCGA database, and its association with immune infiltration, mutations, and drug sensitivity was analyzed. In vitro and in vivo experiments were performed to investigate the role of RLRG CLTC in lipid metabolism and HCC progression. Results Using scRNA-seq data from HCC, we established an R-loop scoring model based on identified RLRGs related to HBV infection. Moreover, the more suppressive tumor immune microenvironment and stronger intercellular communication were displayed in malignant cells with high R-loop scores. The cell trajectory and cellular metabolism analysis exhibited a significant association between lipid metabolism and RLRGs. Additionally, we constructed a prognostic risk model consisting of 8 RLRGs related to fatty acid metabolism, which effectively evaluated the prognostic value, status of tumor immune microenvironment, gene mutations, and chemotherapeutic drug sensitivity for HCC patients. Notably, validation experiments suggested that CLTC could regulate lipid metabolism through R-loop formation and facilitate tumor progression in HCC. Conclusion Collectively, our study proposes an R-loop scoring model associated with tumor immune microenvironment, lipid metabolism and prognostic value. CLTC, an R-loop regulator, emerges as a promising prognostic biomarker and therapeutic target, offering new insights into potential treatment strategies for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cbaecb70e4d3bf1b08fe7b03162bc2296819cdc" target='_blank'>
              Integrated single-cell and bulk transcriptome analysis of R-loop score-based signature with regard to immune microenvironment, lipid metabolism and prognosis in HCC
              </a>
            </td>
          <td>
            Long Chen, Houzhi Yang, Xianfu Wei, Jianchao Liu, Xiuxin Han, Chao Zhang, Yongheng Liu, Yan Zhang, Yao Xu, Yiqin Li, Guowen Wang, Jinyan Feng
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, , Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is a histologically heterogeneous disease with variable clinical outcome. The role the tumour microenvironment (TME) plays in determining tumour progression is complex and not fully understood. To improve our understanding, it is critical that the TME is studied systematically within clinically annotated patient cohorts with long‐term follow‐up. Here we studied the TME in three clinical cohorts of metastatic CRC with diverse molecular subtype and treatment history. The MISSONI cohort included cases with microsatellite instability that received immunotherapy (n = 59, 24 months median follow‐up). The BRAF cohort included BRAF V600E mutant microsatellite stable (MSS) cancers (n = 141, 24 months median follow‐up). The VALENTINO cohort included RAS/RAF WT MSS cases who received chemotherapy and anti‐EGFR therapy (n = 175, 32 months median follow‐up). Using a Deep learning cell classifier, trained upon >38,000 pathologist annotations, to detect eight cell types within H&E‐stained sections of CRC, we quantified the spatial tissue organisation and colocalisation of cell types across these cohorts. We found that the ratio of infiltrating endothelial cells to cancer cells, a possible marker of vascular invasion, was an independent predictor of progression‐free survival (PFS) in the BRAF+MISSONI cohort (p = 0.033, HR = 1.44, CI = 1.029–2.01). In the VALENTINO cohort, this pattern was also an independent PFS predictor in TP53 mutant patients (p = 0.009, HR = 0.59, CI = 0.40–0.88). Tumour‐infiltrating lymphocytes were an independent predictor of PFS in BRAF+MISSONI (p = 0.016, HR = 0.36, CI = 0.153–0.83). Elevated tumour‐infiltrating macrophages were predictive of improved PFS in the MISSONI cohort (p = 0.031). We validated our cell classification using highly multiplexed immunofluorescence for 17 markers applied to the same sections that were analysed by the classifier (n = 26 cases). These findings uncovered important microenvironmental factors that underpin treatment response across and within CRC molecular subtypes, while providing an atlas of the distribution of 180 million cells in 375 clinically annotated CRC patients. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f2afcc9e64651d9dfd348eba0187246fdd6ea3" target='_blank'>
              Computational pathology applied to clinical colorectal cancer cohorts identifies immune and endothelial cell spatial patterns predictive of outcome
              </a>
            </td>
          <td>
            Nicholas Trahearn, Chirine Sakr, Abhirup Banerjee, Seung Hyun Lee, Ann-Marie Baker, Hemant M. Kocher, V. Angerilli, F. Morano, F. Bergamo, G. Maddalena, R. Intini, C. Cremolini, Giulio Caravagna, Trevor Graham, F. Pietrantonio, S. Lonardi, Matteo Fassan, A. Sottoriva
          </td>
          <td>2025-01-09</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood.We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts.Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis.DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            , Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spatial transcriptomics, proteomics, and epigenomics are innovative technologies which offer an unparalleled resolution and wealth of data in understanding and the interpretation of cellular functions and interactions. These techniques allow researchers to investigate gene and protein expressions at an individual cell level, revealing cellular heterogeneity within, for instance, bioengineered tissues and classifying novel and rare cell populations that could be essential for the function of the tissues and in disease processes. It is possible to analyze thousands of cells simultaneously, which gives thorough insights into the transcriptomic view of complex tissues. Spatial transcriptomics combines gene expressions with spatial information, conserving tissue architecture and making the mapping of gene activity across different tissue regions possible. Despite recent advancements in these technologies, they face certain limitations. Single-cell transcriptomics can suffer from technical noise and dropout events, leading to incomplete data. Its applicability has been limited by the complexity of data integration and interpretation, although better resolution and tools for the interpretation of data are developing fast. Spatial proteomics and spatial epigenomics provide data on the distribution of proteins and the gene regulatory aspects in tissues, respectively. The disadvantages of these approaches include rather costly and time-consuming analyses. Nevertheless, combining these techniques promises a more comprehensive understanding of cell function and tissue organization, which can be predicted to be useful in achieving better knowledge of cell guidance in tissue-engineered constructs and a higher quality of tissue technology products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b910cb889d8cf7dd3b82978f6e3916ced3b58d85" target='_blank'>
              Spatial Transcriptomics, Proteomics, and Epigenomics as Tools in Tissue Engineering and Regenerative Medicine
              </a>
            </td>
          <td>
            M. Lammi, C. Qu
          </td>
          <td>2024-12-01</td>
          <td>Bioengineering</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="Climate change, increasing urbanization and global human mobility have increased the risk of emergent infectious diseases and the risk for their pandemic potential. Consequently, there is a pressing need for methods that can provide rapid insights about the potential enduring effects on the human hosts. In-silico methods can be more suitable than in-vitro and in-vivo methods to identify effects potentially manifesting after a long time, under the hypothesis that the underlying causes can be ascribed to perturbed biomolecular processes. Here we focus on oncogenicity, i.e., the risk of developing cancer subsequent to a viral infection, a phenomenon estimated to account for approximately 15% of global cancer prevalence. We characterize viruses in terms of their oncogenic potential by analyzing a multilayer representation of protein-protein interaction networks reconstructed from the human interactome. Our frame-work facilitates classification and allows us to identify sub-sets of proteins in-volved in oncogenesis, highlighting the mechanisms underpinning viral oncogenicity, including factors like chromatin structure. Our framework serves as a foundational resource for medical research, paving the way for the study of other potential long-term effects associated with emerging viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc7f7112ae9e2f8d9b96d5ea55078b28e336c4a" target='_blank'>
              Unraveling the network signatures of oncogenicity in virus-human protein-protein interactions
              </a>
            </td>
          <td>
            Francesco Zambelli, Vera Pancaldi, Manlio De Domenico
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The development of personalized cancer vaccines relies heavily on accurately identifying neoepitopes capable of eliciting strong immune responses. Traditional methods primarily focus on predicting interactions between major histocompatibility complex (MHC) molecules and peptides, often neglecting the crucial step of T-cell receptor (TCR) binding in the context of anchor motif and hydrophobicity, which are critical variables in the elicitation of robust tumor-specific responses. T cells perpetually monitor lymphoid and peripheral organs for antigens, such as peptides or lipids, that are presented by peptide-MHC (pMHC) molecules in other cells. This oversight limits the efficacy of these vaccines in triggering a robust adaptive immune response. We introduce a stand-alone web-server DeepPROTECTNeo (https://cosmos.iitkgp.ac.in/DeepPROTECTNeo/) that processes Whole Exome Sequencing (WES) or Whole Genome Sequencing (WGS) data and mitigate the process of extracting patient-specific neoantigen candidate leveraging genomic and variational TCR information directly, a pivotal component in the development of adaptive immunity and personalized cancer vaccines. Along with traditional MHC binding and B-cell receptor (BCR) recognition, we introduce a context-aware transformer block leveraging cross-attention mechanisms to enhance the analysis of TCR-epitope interactions, thereby advancing neoepitope discovery with unparalleled precision and facilitating reverse vaccinology (RV)-guided multi-epitope vaccine design. With extensive hyperparameter tuning, our model shows exceptional generalizability in predicting TCR-peptide binding on independent test data with AUC 0.7234 and AUPRC 0.7659, substantially outperforming state-of-the-art methods. Furthermore, our pipeline successfully identified 372 neoepitopes out of 532 unique neoepitopes presented in the TESLA benchmark dataset, with 24 true neoantigens, demonstrating its high efficacy and precision in neoepitope discovery. Highlights DeepPROTECTNeo is an all-in-one platform for neo-epitope identification from raw sequencing reads Incorporate a novel transformer-based architecture for peptide-TCR binding affinity prediction and outperforming current methods Accepts flexible input formats from users to provide comprehensive analysis and has a separate module for predicting peptide-TCR binding Successfully identified 24 true neoepitopes among 37 reported in TESLA using only SNVs, indels">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92b67a359609fc60583af345b0924b4d94b992" target='_blank'>
              DeepPROTECTNeo: A Deep learning-based Personalized and RV-guided Optimization tool for TCR Epitope interaction using Context-aware Transformers
              </a>
            </td>
          <td>
            Debraj Das, Soumyadeep Bhaduri, Avik Pramanick, Pralay Mitra
          </td>
          <td>2025-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="There is an urgent need of precision medicine for ovarian cancer patients to identify patients who respond to chemotherapy and PARP inhibitors, a therapy targeting homologous recombination deficiency (HRD). Here we established a panel of 37 long-term patient-derived tumor organoids (PDTO) models of various histological subtypes from 224 patients and demonstrated that they mimic the histological and molecular characteristics of original tumors. Screening of chemotherapeutic drugs showed that PDTO exhibit heterogeneous responses, and that response of PDTO from high-grade serous ovarian carcinoma to carboplatin recapitulated patient response to first-line treatment. Additionally, the detection of HRD phenotype of PDTO using functional assay was associated with the results of the HRD test Genomic Instability Scar (GIScar). Although larger-scale investigations are needed to confirm the predictive potential of PDTO, these results provide further evidence of the potential interest of ovarian PDTO for functional precision medicine even if many challenges remain to be addressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9373b0fdccc45bac208c5469698121e6a4da07c" target='_blank'>
              Long-term patient-derived ovarian cancer organoids closely recapitulate tumor of origin and clinical response
              </a>
            </td>
          <td>
            L. Thorel, E. Dolivet, Pierre-Marie Morice, R. Florent, J. Divoux, M. Perréard, L. Lecouflet, G. Desmartin, C. Marde Alagama, F. Giffard, A. Leconte, J. Lequesne, B. Clarisse, M. Briand, A. Traore, C. Villenet, JP. Meneboo, G. Babin, L. Gaichies, S. Martin-Françoise, JF. Le Brun, R. Rouzier, E. Brotin, C. Denoyelle, N. Vigneron, R. Leman, D. Vaur, L. Castera, C. Blanc-Fournier, N. Elie, B. Plancoulaine, F. Joly, M. Meryet-Figuiere, M. Figeac, L. weiswald, L. Poulain
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background. Colorectal carcinomas (CRCs) are seldom eradicated by cytotoxic chemotherapy. Cancer cells with stem-like functional properties, often referred to as “cancer stem cells” (CSCs), display preferential resistance to several anti-tumor agents used in cancer chemotherapy, but the molecular mechanisms underpinning their selective survival remain only partially understood. Methods. In this study, we used Transcription Factor Target Genes (TFTG) enrichment analysis to identify transcriptional regulators (activators or repressors) that undergo preferential activation by chemotherapy in CRC cells with a “bottom-of-the-crypt” phenotype (EPCAM+/CD44+/CD166+; CSC-enriched) as compared to CRC cells with a “top-of-the-crypt” phenotype (EPCAM+/CD44neg/CD166neg; CSC-depleted). The two cell populations were purified in parallel by fluorescence-activated cell sorting (FACS) from a patient-derived xenograft (PDX) line representative of a moderately differentiated human CRC, following in vivo chemotherapy with irinotecan (CPT-11). The transcriptional regulators identified as differentially activated were tested for differential expression in normal vs. cancer tissues, and in cell populations enriched in stem/progenitor cell-types as compared to differentiated lineages (goblet cells, enterocytes) in the mouse colon epithelium. Finally, the top candidate was tested for mechanistic contribution to drug-resistance by selective down-regulation using short-hairpin RNAs (shRNAs). Results. Our analysis identified E2F4 and TFDP1, two core components of the DREAM transcriptional repression complex, as transcriptional modulators preferentially activated by irinotecan in EPCAM+/CD44+/CD166+ as compared to EPCAM+/CD44neg/CD166neg cancer cells. The expression levels of both genes (E2F4, TFDP1) were found up-regulated in CRCs as compared to human normal colon tissues, and in a sub-population of mouse colon epithelial cells enriched in stem/progenitor elements (Epcam+/Cd44+/Cd66alow/Kitneg) as compared to other sub-populations enriched in either goblet cells (Epcam+/Cd44+/Cd66alow/Kit+) or enterocytes (Epcam+/Cd44neg/Cd66ahigh). Most importantly, E2F4 down-regulation using shRNAs dramatically enhanced the sensitivity of human CRCs to in vivo treatment with irinotecan, across three independent PDX models. Conclusions. Our data identified E2F4 and the DREAM repressor complex as critical regulators of human CRC resistance to irinotecan, and as candidate targets for the development of chemo-sensitizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3c8fee4cf42a4e76a7d265977fa7fcfba7541" target='_blank'>
              The E2F4 transcriptional repressor is a key mechanistic regulator of colon cancer resistance to irinotecan (CPT-11)
              </a>
            </td>
          <td>
            Junichi Matsubara, Y. F. Li, Sanjay Koul, Junko Mukohyama, Luis E. Valencia Salazar, Taichi Isobe, D. Qian, Michael F. Clarke, Debashis Sahoo, R. Altman, Piero Dalerba
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K. Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
To meet their high energy needs, tumor cells undergo aberrant metabolic reprogramming. A tumor cell may expertly modify its metabolic pathways and the differential expression of the genes for metabolic enzymes. The physiological requirements of the host tissue and the tumor cell of origin mostly dictate metabolic adaptation. Ameloblastoma (AB) is a benign odontogenic tumor of epithelial origin. Due to its unrestricted growth potential, local aggressiveness, and high likelihood of recurrence, this condition poses a significant risk to the patient's health. This study aimed to characterize the metabolic heterogeneity at single-cell resolution of AB.


METHODS
Single-cell RNA sequencing (scRNA-seq) was performed on 17,284 cells from three AB donors. Bioinformatic analysis was used to examine differentially expressed genes, subtypes, and regulatory mechanisms when combined with odontogenic keratocyst scRNA-seq data. Based on metabolic pathway gene sets, the metabolic landscape of AB tumor cells was examined.


RESULTS
Using scRNA-seq, we discovered that AB tumor cells had substantial heterogeneity. The biggest contributor to tumor cell metabolic characteristics is determined to be variation in mitochondrial programming and glycolysis. Surprisingly, hypoxia corresponds with both oxidative phosphorylation and glycolysis activity in AB tumor cells at the single-cell level.


CONCLUSION
This study presents a computational framework for defining metabolism using single-cell expression data and identifies oxidative phosphorylation and glycolysis as critical components of metabolism for AB tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c116028597b1c5eaf72b4b7ed869064e15002ef" target='_blank'>
              Metabolic Analysis of Tumor Cells Within Ameloblastoma at the Single-Cell Level.
              </a>
            </td>
          <td>
            Rui-Fang Li, Yi Zhao, Qi-Wen Man
          </td>
          <td>2024-12-31</td>
          <td>Oral diseases</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Immunotherapy that targets immune checkpoints has achieved revolutionary success, but its application in solid tumors remains limited, highlighting the need for reliable enhancement of the efficacy of immunotherapy. Golgi protein 73 (GP73), a Golgi membrane protein, has been implicated in various cellular processes, including immune regulation. Recent studies suggested that GP73 may play a role in modulating the immune response in patients with cancer. In this study, we investigated the mechanism by which GP73 regulates T-cell-mediated antitumor immunity within the tumor microenvironment. Methods We used T-cell specific GP73 knockout mice to establish MC38 and B16 tumor models to investigate the impact of GP73-deficient T cells on tumor growth. Single-cell sequencing was subsequently employed to classify tumor-infiltrating immune cells and assess changes in cytokines and metabolic genes. Through RNA sequencing, real-time quantitative PCR, western blotting, flow cytometry, seahorse analysis, glucose uptake, and L-lactic acid secretion assays, we explored how GP73 regulates hypoxia-inducible factor 1α (HIF-1α) to influence T-cell antitumor functionality. Furthermore, we established adoptive transfer experiments to study the ability of GP73-overexpressing T cells to combat tumors. Blood samples of patient with clinical tumor were collected to assess the relationship between immunotherapy efficacy and T-cell GP73 levels. Results In this study, the absence of GP73 in mouse T cells promoted tumor growth and metastasis, accompanied by a decrease in the proportion of cytotoxic CD8+T cell subsets infiltrating the tumor and an increase in exhausted CD8+ T-cell subsets. Further analysis revealed that the effector function of CD8+T cells in tumors relies on glycolysis regulated by HIF-1α rather than immune checkpoints. GP73-deficient T cells exhibit severely impaired glycolysis in hypoxic environments, whereas ectopic GP73 expression restores HIF-1α levels. In adoptive immunotherapy, overexpression of GP73 in T cells inhibits tumor growth. In cytotoxicity assays, knockdown of GP73 affected the ability of CD8+T cells to kill target cells. Clinically, tumor immunotherapy partial response patients present significantly elevated levels of GP73 expression in T cells. Conclusions These findings reveal the role of GP73 in regulating T-cell glycolysis and may lead to new therapeutic strategies for the prognosis and treatment of clinical tumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb0f11d29bd0ce3b4fa58aa79f03686d534f043" target='_blank'>
              GP73 reinforces cytotoxic T-cell function by regulating HIF-1α and increasing antitumor efficacy
              </a>
            </td>
          <td>
            Jialong Liu, Chao Feng, Ruzhou Zhao, Hongbin Song, Linfei Huang, Nan Jiang, Xiaopan Yang, Lanlan Liu, Cuijuan Duan, Luming Wan, Qi Gao, Lijuan Sun, Xufeng Hou, Muyi Liu, Yanhong Zhang, Xuemiao Zhang, Dandan Zhang, Yufei Wang, Yong Li, Xueping Ma, Hui Zhong, Min Min, Congwen Wei, Yuan Cao, Xiaoli Yang
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Understanding early tumor formation offers critical insights for cancer prevention at the earliest stages. However, most in vivo studies focus on later stages, providing limited information on initial tumorigenesis. Hypoxia, a common physiological condition, is a primary microenvironmental factor in cancer. While molecular studies indicate significant yet sometimes conflicting roles of hypoxia in cancer progression, experimental data on hypoxia’s impact on tumor formation from a single cell, especially regarding cellular dynamics, remain scarce. To address this gap, we developed an integrated platform for live-cell imaging under hypoxic conditions, enabling, for the first time, observation of 3D tumor spheroid development from a single cell. Our results show that tumorigenesis is markedly accelerated under hypoxia. Beyond linear growth, we demonstrate that directional movement enhances fusion events, leading to larger spheroids that, in turn, stimulate further proliferation — a positive feedback loop between spheroid dynamics and growth. Actin polymerization and cell-cell adhesion are optimized to promote spheroid fusion. Additionally, hypoxia induces cells with outreaching F-actin co-localized with smaller nuclei to bridge gaps between spheroids, facilitating fusion, while the appearance of tiny nuclei suggests genetic instability. These findings provide direct insights into hypoxia’s role in early tumorigenesis, highlighting the potential of modulating dynamic, mechanical, and physiological factors for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0153fc66d9ff4cbf61d2a83aea2c920a18ecaa1" target='_blank'>
              Hypoxia-Induced Active Dynamics Promotes Early Tumorigenesis
              </a>
            </td>
          <td>
            Zhengduo Wang, Li Tian, Bo Li
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor growth and progression involve coordinated regulation by internal, microenvironmental, and systemic signals and often display conspicuous sexual dimorphism. The mechanisms governing the integration and coordination of these signals, along with their sex-based differences, remain largely unknown. Using a Drosophila tumor model originating from nonreproductive tissue, we show that female-biased tumor growth involves multifaceted communications among tumor cells, hemocytes, and neuroendocrine insulin-producing cells (IPCs). Notch-active tumor cells recruit hemocytes carrying the tumor necrosis factor–α (TNF-α) homolog Eiger to the tumor microenvironment (TME), activating the c-Jun N-terminal kinase (JNK) pathway in tumor cells, instigating the sexually dimorphic up-regulation of cytokine Unpaired 2 (Upd2). Upd2, in turn, exerts a distal influence by modulating the release of a Drosophila insulin-like peptide (Dilp2) from IPCs. Dilp2 then activates the insulin signaling in the tumor, thereby fostering sexual-dimorphic tumor growth. Together, these findings reveal a relay mechanism involving the TME and systemic signals that collectively control the sexual dimorphism of tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4504e9cb20bc72b2973f3610b4d0ff83ae5829a" target='_blank'>
              Sex-dimorphic tumor growth is regulated by tumor microenvironmental and systemic signals
              </a>
            </td>
          <td>
            Xianfeng Wang, Hongcun Bao, Yi-Chun Huang, Anindita Barua, Chun-Ming Lai, Jie Sun, Youfang Zhou, Fei Cong, Shangyu Gong, Chih-Hsuan Chang, Wu-Min Deng
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d479e3aee16692284dd5875a578cce134082060b" target='_blank'>
              Cell signaling communication within papillary craniopharyngioma revealed by an integrated analysis of single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Yanfei Jia, Zicheng Zhao, Xiaoyu Zhang, Yunlong Zhao, Songbai Gui, Xiu-An Yang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Biliary tract cancer (BTC) comprises a clinically diverse and genetically heterogeneous group of tumors along the intra- and extrahepatic biliary system (intrahepatic and extrahepatic cholangiocarcinoma) and gallbladder cancer with the common feature of a poor prognosis, despite increasing molecular knowledge of associated genetic aberrations and possible targeted therapies. Therefore, the search for even more precise and individualized therapies is ongoing and preclinical tumor models are central to the development of such new approaches.


AREAS COVERED
The models described in the current review include simple and advanced in vitro and in vivo models, including cell lines, 2D monolayer, spheroid and organoid cultures, 3D bioprinting, patient-derived xenografts, and more recently, machine-perfusion platform-based models of resected liver specimens. All these models have individual advantages, disadvantages and limitations that need to be considered depending on the desired application.


EXPERT OPINION
In addition to potential cost limitations, availability of BTC cell types, time required for model establishment and growth success rate, the individual models differently reflect relevant characteristics such as tumor heterogeneity, spatial tumor-stroma microenvironment interactions, metabolic and nutritional gradients and immunological interactions. Therefore, a consequent combination of different models may be required to improve clinical study outcomes by strengthening the preclinical data basis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22dfdd087063cb506f430d33a11c5c6121ad3df5" target='_blank'>
              The importance of preclinical models for cholangiocarcinoma drug discovery.
              </a>
            </td>
          <td>
            F. J. Krendl, Florian Primavesi, Rupert Oberhuber, Daniel Neureiter, Matthias Ocker, Dino Bekric, T. Kiesslich, C. Mayr
          </td>
          <td>2025-01-22</td>
          <td>Expert opinion on drug discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="While optimizing cancer cell growth conditions, we discovered that cancer stem cells can generate second membranes outside the plasma membranes forming compartments separated from the extracellular matrix. The encapsulating membranes can extend and generate long cytocapsular tubes, wherein multiple cells can migrate. SILAC proteomics of the second cytocapsular membranes identified 400 membrane proteins, and a small subset of them are highly upregulated in cytocapsular cancers compared to normal tissues. The ATP-dependent calcium pump PMCA2 is one of the highest upregulated factors of the cytocapsular membrane, and antibodies serve as biomarkers for malignant tumors, as checked for 293 subtypes of cancers. Cytocapsular tumors have not been described before, possibly because the CC membranes do not exhibit epitopes targeted by conventional methods, and no efforts have been made to search for new cancer specific organelles. Antibodies against PMCA2 can now be used to map cancer evolution pathways in human bodies by comparisons of more than 12 000 annotated specimens from tissue banks worldwide. The current research reveals that the native malignant cancer cell is enclosed in a cytocapsular membrane. With the PMCA2 cancer biomarker available, the development of human cancers can be studied from cancer tissue banks and clinical cancer biopsies with a previously unknown diversity. The emerging knowledge on cancer-driving biomarkers opens doors for new routes in cancer diagnosis, surgery, therapy, and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4e5b795160ba02ecb2832797cada359373d1df" target='_blank'>
              Discovery of the Cytocapsular Membrane as Hallmark of Malignant Tumors.
              </a>
            </td>
          <td>
            Tingfang Yi, G. Wagner
          </td>
          <td>2024-12-16</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Diffuse midline gliomas (DMGs) with histone H3K27M mutations represent a devastating paediatric brain cancer characterized by abysmal prognosis and limited treatment options. The only approved treatment is radiotherapy (RT), but most of the tumours relapse with fatal consequences. In this study, we sought to investigate whether irradiation leads to senescence induction and explore the efficacy of senolytics against DMG. Methods We have characterised the senescent phenotype of five genetically heterogeneous H3K27M-altered human DMG cell lines, combining cellular and/or molecular approaches. The sensitivity of senescent cells to Bcl-xL inhibition has been demonstrated in dose/response curves in vitro and in a PDX model of DMG. Results Here, we show that ionizing radiation induces senescence and SASP responses in both TP53 mutant and wild-type H3K27M-altered human DMG cell lines. We identify Navitoclax as a potent senolytic agent that selectively targets senescent DMG cells into apoptosis by inhibiting Bcl-xL. Related compounds, such as a proteolysis-targeting chimera (PROTAC)-mediated Bcl-xL degradation and a galacto-conjugated form of Navitoclax also show an effective senolytic activity in senescent cancer cells. Finally, we show that a combination therapy of irradiation and Navitoclax results in reduced tumor burden and increased mouse survival in an orthotopic xenograft DMG model. Conclusion These results offer a rationale for further clinical development of senolytic therapies as part of multimodal treatment approaches for DMG patients/ Key Points Ionising irradiation induces senescence in human DMG cells independently of the p53 status. Bcl-xL inhibition results in apoptosis of human DMG senescent cells in synergy with irradiation. Combination of irradiation and BcL-xL inhibition reduces tumourigenesis in a PDX model of DMG. Importance of the Study H3K27M-altered DMG are devastating paediatric tumours with an abysmal prognosis. The only approved treatment is radiotherapy but this is palliative and tumours almost always relapse with fatal consequences for the patients. In this study, we show that radiotherapy results in senescence induction in five genomically heterogeneous human DMG cell lines. We identify that drugs targeting the anti-apoptotic protein Bcl-xL show a strong senolytic activity in conjunction with radiotherapy both in vitro in DMG cells and in vivo in a PDX model of H3K27M-altered DMG. Treatment with Bcl-xL inhibitor Navitoclax, or related compounds targeting Bcl-xL protein degradation or containing a galactose conjugated form of Navitoclax results in DMG cancer cell apoptosis. As several of these inhibitors are currently being tested in ongoing clinical trials against other diseases, our data support the use of Bcl-xL inhibition mediated senolytics as an adjuvant therapy to radiotherapy to potentially improve outcomes in this challenging disease setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b580291efecff13cf295c14d6b2926c855610e2" target='_blank'>
              Targeted inhibition of Bcl-xL following radiation reduces tumourigenesis in preclinical models of H3K27M-altered diffuse midline glioma
              </a>
            </td>
          <td>
            Ashley Vardon, S. Haston, Romain Guiho, Diana Carvahlo, Rebecca Carter, J. Boult, Daniel Gharai, Diwakar Santhakumar, Ielyaas Cloete, James Gray, John Apps, O. Ogunbiyi, Daohong Zhou, Guangrong Zheng, Ramón Martínez-Máñez, Mark Lythgoe, Laura Donovan, A. Carcaboso, Owen Williams, Jesus Gil, Thomas Jacques, Jamie Dean, D. Michod, Chris Jones, Darren Hargrave, J. Martínez-Barbera
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study aimed to explore the impact of intratumoral microorganisms in conjunction with hormone receptors on the tumor microenvironment and their potential role in predicting patient prognosis. Significant bacterial variations were identified within ER, PR, HER2, and triple-negative breast cancer subtypes. Kaplan-Meier survival analysis was employed to identify bacteria associated with patient survival. Further, a humanized immune system mouse model bearing breast cancer xenografts was used to evaluate the effects of Stenotrophomonas maltophilia (SMA) on tumor growth and CD8+ T cell infiltration. Additional validation experiments included fluorescence in situ hybridization for SMA, CD8+ T cell chemotaxis, and intracellular cytokine detection. Lawsonella clevelandensis-A, Diaphorobacter nitroreducens, and SMA were identified as significant prognostic species. Notably, tumor-infiltrating immune cells, particularly CD8+ T cells, exhibited a positive association with the presence of SMA. Experimental validation with clinically isolated SMA further confirmed its positive correlation with CD8+ T cell activation. In vivo findings demonstrated that SMA inhibited tumor growth and promoted CD8+ T cell infiltration, highlighting the complex interactions between intratumoral microbiota and tumor immunity in breast cancer. These insights contribute to the understanding of microbial influences on the tumor microenvironment and suggest potential pathways for improving patient prognosis through microbiota modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fce5a50e9f9799a232183eb87230d236d9adfd" target='_blank'>
              Intratumoral Stenotrophomonas Maltophilia in Breast Cancer: Unraveling the Interplay with Hormone Receptors and Impact on Tumor Immunity
              </a>
            </td>
          <td>
            Qian Zhang, Dujuan Wang, Guangzheng Zhuo, Shilin Wang, Yufen Yuan, Liping Wang, Lili Ji, Yuhang Wan, Guohong Liu, Yunbao Pan
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-Grade Serous Ovarian Cancer (HGSOC) originates from fallopian tube (FT) precursors. However, the molecular changes that occur as precancerous lesions progress to HGSOC are not well understood. To address this, we integrated high-plex imaging and spatial transcriptomics to analyze human tissue samples at different stages of HGSOC development, including p53 signatures, serous tubal intraepithelial carcinomas (STIC), and invasive HGSOC. Our findings reveal immune modulating mechanisms within precursor epithelium, characterized by chromosomal instability, persistent interferon (IFN) signaling, and dysregulated innate and adaptive immunity. FT precursors display elevated expression of MHC-class I, including HLA-E, and IFN-stimulated genes, typically linked to later-stage tumorigenesis. These molecular alterations coincide with progressive shifts in the tumor microenvironment, transitioning from immune surveillance in early STICs to immune suppression in advanced STICs and cancer. These insights identify potential biomarkers and therapeutic targets for HGSOC interception and clarify the molecular transitions from precancer to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b491f761ec399a1258abdf4e675e3bc81f53e5e" target='_blank'>
              Multimodal Spatial Profiling Reveals Immune Suppression and Microenvironment Remodeling in Fallopian Tube Precursors to High-Grade Serous Ovarian Carcinoma.
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, C. Hug, S. Coy, Yu-An Chen, Ino de Bruijn, Natalie Shih, Euihye Jung, R. Pelletier, Mariana Lopez Leon, Gabriel Mingo, Dalia K Omran, Jong Suk Lee, Clarence Yapp, B. Satravada, Ritika Kundra, Yilin Xu, Sabrina Chan, Juliann B. Tefft, J. Muhlich, Sarah H Kim, Stefan M. Gysler, Judith Agudo, James R. Heath, N. Schultz, Charles Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2024-12-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Tumor-initiating cancer stem cells (CSC) pose a challenge in human malignancies since they are largely treatment resistant and can seed local recurrence and metastasis. Epigenetic mechanisms governing cell fate decisions in embryonic and adult stem cells are deregulated in CSCs. This review focuses on the methyltransferase DOT1L, which methylates H3K79 and is a key epigenetic regulator governing embryonic organogenesis and adult tissue stem cell maintenance. DOT1L is overexpressed in many human malignancies, and dysregulated H3K79 methylation is pathogenic in acute myeloid leukemia and several solid tumors. DOT1L regulates core stem cell genes governing CSC self-renewal, tumorigenesis, and multidrug resistance. Recent work has situated DOT1L as an attractive stem cell target in cancer. These reports showed that DOT1L is overexpressed and its protein activated specifically in malignant stem cells compared to bulk tumor cells, making them vulnerable to DOT1L inhibition in vitro and in vivo. While early DOT1L inhibitor clinical trials were limited by inadequate drug bioavailability, accumulating preclinical data indicate that DOT1L critically regulates CSC self-renewal and might be more effective when given with other anticancer therapies. The appropriate combinations of DOT1L inhibitors with other agents and the sequence and timing of drug delivery for maximum efficacy warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a23df3bd0f490217c6606b07d6c41ff2e044139" target='_blank'>
              DOT1L mediates stem cell maintenance and represents a therapeutic vulnerability in cancer.
              </a>
            </td>
          <td>
            Hetakshi P Kurani, Joyce M Slingerland
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a high-throughput transcriptomic approach with the power to identify rare cells, discover new cellular subclusters, and describe novel genes. scRNA-seq can simultaneously reveal dynamic shifts in cellular phenotypes and heterogeneities in cellular subtypes. Since the publication of the first protocol on scRNA-seq in 2009, this evolving technology has continued to improve, through the use of cell-specific barcodes, adoption of droplet-based systems, and development of advanced computational methods. Despite induction of the cellular stress response during the tissue dissociation process, scRNA-seq remains a popular technology, and commercially available scRNA-seq methods have been applied to the brain. Recent advances in spatial transcriptomics now allow the researcher to capture the positional context of transcriptional activity, strengthening our knowledge of cellular organization and cell-cell interactions in spatially intact tissues. A combination of spatial transcriptomic data with proteomic, metabolomic, or chromatin accessibility data is a promising direction for future research. Herein, we provide an overview of the workflow, data analyses methods, and pros and cons of scRNA-seq technology. We also summarize the latest achievements of scRNA-seq in stroke and acute traumatic brain injury, and describe future applications of scRNA-seq and spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0277107d9508977e5eb0c209436914dd7332f9c2" target='_blank'>
              Single-cell RNA sequencing in stroke and traumatic brain injury: Current achievements, challenges, and future perspectives on transcriptomic profiling.
              </a>
            </td>
          <td>
            Ruyu Shi, Huaijun Chen, Wenting Zhang, R. Leak, Dequan Lou, Kong Chen, Jun Chen
          </td>
          <td>2024-12-09</td>
          <td>Journal of cerebral blood flow and metabolism : official journal of the International Society of Cerebral Blood Flow and Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Locally-advanced rectal cancer (LARC) is traditionally treated with neoadjuvant chemoradiotherapy before surgery, however, response heterogeneity necessitates the development of novel treatment combinations. Immune checkpoint blockade (ICB) with or without radiotherapy (RT) elicits clinical benefit in various solid tumors, yet ICB’s effectiveness so far is limited to colorectal tumors with microsatellite instability and the role of the tumor microenvironment (TME) is unclear. We present a novel orthotopic immunocompetent mouse model of LARC, where combinations of RT and ICB are tested to elucidate the role of TME in the response to treatment. Methods: Autochthonous murine tumor-derived organoids expressing common driver mutations (Apcfl/fl; KrasG12D/+; p53fl/fl; Tgfbr1fl/fl: ‘AKPT’) were orthotopically injected into syngeneic C57Bl/6 immunocompetent mice. Tumor response to RT-ICB was studied on timepoint and aging cohorts of mice with tumors growing in different TME settings: (a) in the rectal submucosa of wildtype mice, (b) subcutaneously in wildtype mice, and (c) in the rectal submucosa of mice lacking the iCCR locus (CCR1, CCR2, CCR3, and CCR5: ‘iCCRKO ’), which inhibits inflammatory monocyte recruitment into the TME. Radiation was delivered as 15Gy in 3# (Mon-Wed-Fri) using a Small Animal Radiation Research Platform on day 13 following tumor engraftment for the orthotopic models and on day 7 for the subcutaneous. PD-1 inhibition (PD-1i) was administered intraperitoneally (10mg/kg twice weekly) 3 days pre-RT until endpoint. Mice in the timepoint experiments were sampled 5 days following RT, while aging mice were sampled at clinical endpoint or 100 days, whichever came first. Results: RT and PD-1i offered significant survival extension and tumor control in the orthotopic wildtype setting (a). Transcriptomic characterization at 5 days revealed that untreated and irradiated tumors harbored a stroma-rich microenvironment, low in immune cells, which PD-1i partly reversed. This phenotypic ‘switch’ was also evident in an aging cohort, suggesting that RT + PD-1i have long-lasting effects on the stroma. In the subcutaneous setting (b), RT significantly increased tumor control but PD-1 blockade offered little additional advantage. Interestingly, subcutaneous AKPT tumors had significantly more CD8+ cells at baseline than orthotopic ones, highlighting the inherent immune permissiveness of the subcutaneous TME. Finally, tumors growing in the iCCRKO setting (c) were more radio-sensitive, leading to 3 instances of tumor clearance: one in the RT group and two in the RT + PD-1i. Conclusion: Our findings demonstrate the synergistic potential of RT and PD-1i to sensitize an inherently radio-resistant orthotopic mouse model of LARC. Tumor location and inflammatory monocytes emerged as key regulators of the immune response to RT-ICB, which could have important implications for LARC modeling. Importantly, we highlight that TME components can facilitate treatment resistance and give rise to targets that can be therapeutically exploited.
 Citation Format: Lydia Melissourgou-Syka, Michael A. Gillespie, Lily V.S. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, Tamsin R.M. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, Sean M. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S.D. Roxburgh.Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P022">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b958d65c1fd0bd54499a71c321762f474a07901" target='_blank'>
              Abstract P022: Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations
              </a>
            </td>
          <td>
            Lydia Melissourgou-Syka, Michael A Gillespie, L. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, T. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, S. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S D Roxburgh
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5675284f659a316a50b22f64df578d9b883b676a" target='_blank'>
              Single-cell RNA sequencing highlights the unique tumor microenvironment of small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Tianyou Wang, Li Zhang, Song Mei, Bo Wang, Jiaqi Liu, Weiping Yang, Jiongbo Liao, Chao Wang
          </td>
          <td>2025-01-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
Thyroid Hormones (THs) critically impact human cancer. Although endowed with both tumor-promoting and inhibiting effects in different cancer types, excess of THs has been linked to enhanced tumor growth and progression. Breast cancer depends on the interaction between bulk tumor cells and the surrounding microenvironment in which mesenchymal stem cells (MSCs) exert powerful pro-tumorigenic activities.


METHODS
Primary human MSCs from healthy female donors were co-cultured with DIO2 knock out (D2KO) and wild type (WT) MCF7 breast cancer cells to assess cell growth, migration, invasion and the expression of known epithelial-mesenchymal transition (EMT)- and inflammation-related markers. Furthermore, a surgery-free intraductal delivery model, i.e., the Mouse-INtraDuctal (MIND) injection method, was used as a tool for in vivo characterization of breast tumor formation and progression.


RESULTS
In this study, we uncovered a novel role of THs in regulating the tumor-stroma crosstalk. MCF7 cells enhanced the intracellular activation of THs through the TH-activating enzyme, D2, fostering their EMT properties and the dialogue with MSCs. D2 inactivation reduced the invasiveness of MCF7 cells and their responsiveness to the pro-tumorigenic induction via MSCs, both in vivo and in vitro.


CONCLUSIONS
Thus, we argue that intracellular activation of THs via D2 is a critical requirement for invasive and metastatic conversion of breast cancer cells, advising the blocking of D2 as a potential therapeutic tool for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b5bee8d823debda4075760c0534546bb56c40c" target='_blank'>
              Thyroid Hormone Activation Regulates the Crosstalk between Breast Cancer and Mesenchymal Stem Cells.
              </a>
            </td>
          <td>
            A. Nappi, V. D'Esposito, C. Miro, A. Parascandolo, A. G. Cicatiello, S. Sagliocchi, L. Acampora, Sepehr Torabinejad, F. Restolfer, M. Raia, Melania Murolo, E. Di Cicco, Pietro Formisano, M. Dentice
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a00464522656fb99b3714169ed11d185f90a82" target='_blank'>
              A hybrid epithelial-mesenchymal transition program enables basal epithelial cells to bypass stress-induced stasis and contributes to a metaplastic breast cancer progenitor state
              </a>
            </td>
          <td>
            Joseph A. Caruso, Chira Chen-Tanyolac, T. Tlsty
          </td>
          <td>2024-12-18</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver cancer, with microvascular invasion (MVI) identified as a major predictor of early recurrence. However, the intratumor cellular heterogeneity of MVI, the identification of pertinent biomarkers, and the role of intercellular signalling interactions in MVI progression are unclear. This study aims to explore these aspects using single-cell transcriptomic analysis.The present study utilized single-cell transcriptomic data from public databases to conduct an in-depth transcriptome analysis of tumour tissues and adjacent nontumor tissues from five patients with hepatocellular carcinoma, with a particular focus on samples from three patients exhibiting microvascular invasion. Bioinformatics tools were employed to analyze gene expression patterns and signalling pathways.The findings indicated that MVI-positive malignant cells activate multiple signalling pathways to facilitate invasion and metastasis. Specific malignant cell subtypes strongly associated with MVI were identified, exhibiting distinctive gene expression patterns related to proliferation, invasion, and metabolic reprogramming of tumour cells. Further analysis revealed that the laminin and VEGF signalling pathways are crucial for remodelling the tumour microenvironment and angiogenesis associated with MVI. The MARCKSL1 gene was predominantly expressed in MVI-positive malignant cells and may contribute to MVI progression by interacting with the PTN signalling network. Additionally, MARCKSL1 is linked to tumour resistance to multiple anticancer drugs.This study sheds light on the molecular characteristics and functional heterogeneity of MVI-associated malignant cell subpopulations. The single-cell transcriptome and bioinformatics analyses provided insights into the mechanisms driving MVI, potentially aiding the development of targeted diagnostic and therapeutic strategies. Future research should further validate the role of MARCKSL1 in MVI progression and explore its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e75f99f0ab352845523f0f1e55263a27b76fefc5" target='_blank'>
              Single-cell sequencing reveals cell heterogeneity and aberrantly activated pathways associated with microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jianwei Cui, Fanyi Zeng, Ming Tang, Shiwu Yin
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd21edea6ef2a85bfcda6c2395a58850a218d8d2" target='_blank'>
              Patient-derived response estimates from zero-passage organoids of luminal breast cancer
              </a>
            </td>
          <td>
            Róża K. Przanowska, Najwa Labban, Piotr Przanowski, Russell B Hawes, Kristen A Atkins, S. Showalter, Kevin A. Janes
          </td>
          <td>2024-12-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Growing research evidence indicates a substantial influence of the intra-tumor microbiome on tumor outcome. However, there is currently no consistent criterion for identifying the association of microbes with tumor progression and response to treatment across various types of cancer. In this study, we concentrate on the intra-tumor microbiome and develop the Tumor Microbiome Survival Index (TMSI), a measure indicative of cancer patient’ survival risk. Our indices revealed notable distinctions between two stratified risk groups for each of the 10 cancer types and could precisely predict patients’ overall survival. For each type of cancer, our findings unveiled two distinct gene expression profiles and shed light on the varying patterns of immune and stromal cell enrichment between the two risk groups. Additionally, we noted that the high-TMSI group exhibited substantially elevated IC50 values for a number of drugs, indicating that individuals in the low-TMSI group might experience superior therapeutic effects from chemotherapy. These findings illuminate the complex dynamics between the tumor microbiome, the patient’s immune reaction, and medical outcomes, thus shedding light on microbiome-based personalized therapeutic interventions. Highlights The TMSI model generates a survival risk score based on the intra-tumor microbiome. The TMSI model predicts survival rates accurately across multiple cancer types. Molecular-level analysis highlights distinct immune interactions with tumor microbes. High-TMSI groups showed altered drug sensitivity and implying potential differences in treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/668673bbb1c33f1b3ec9d877b6f8591e892b7705" target='_blank'>
              Intra-tumor microbiome-based tumor survival indices predict immune interaction and drug sensitivity on pan-cancer scale
              </a>
            </td>
          <td>
            Yan Gao, Haohong Zhang, Dongliang Chu, Kang Ning
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024cdb898f44026b61ab427711870c26e06ca884" target='_blank'>
              Identification of Cell Fate Determining Transcription Factors for Generating Brain Endothelial Cells.
              </a>
            </td>
          <td>
            Roya Ramezankhani, J. De Smedt, B. Toprakhisar, Bernard K. van der Veer, Tine Tricot, Gert Vanmarcke, Bradley Balaton, Leo A. van Grunsven, Massoud Vosough, Y. C. Chai, C. Verfaillie
          </td>
          <td>2025-01-24</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="INTRODUCTION
Spatial biology is an emerging interdisciplinary field facilitating biological discoveries through the use of spatial omics technologies. Recent advancements in spatial transcriptomics, spatial genomics (e.g. genetic mutations and epigenetic marks), multiplexed immunofluorescence, and spatial metabolomics/lipidomics has enabled high-resolution spatial profiling of gene expression, genetic variation, protein expression, and metabolites/lipids profiles in tissue. These developments contribute to a deeper understanding of the spatial organization within tissue microenvironments at the molecular level.


AREAS COVERED
This report provides an overview of the untargeted, bottom-up mass spectrometry (MS)-based spatial proteomics workflow. It highlights recent progress in tissue dissection, sample processing, bioinformatics, and liquid chromatography (LC)-MS technologies that are advancing spatial proteomics toward cellular resolution.


EXPERT OPINION
The field of untargeted MS-based spatial proteomics is rapidly evolving and holds great promise. To fully realize the potential of spatial proteomics, it is critical to advance data analysis and develop automated and intelligent tissue dissection at the cellular or subcellular level, along with high-throughput LC-MS analyses of thousands of samples. Achieving these goals will necessitate significant advancements in tissue dissection technologies, LC-MS instrumentation, and computational tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750f7b08fc1af849136e70bc860f56ef668a0aa0" target='_blank'>
              Spatial proteomics toward cellular resolution.
              </a>
            </td>
          <td>
            Yumi Kwon, James M Fulcher, L. Paša-Tolić, Wei-Jun Qian
          </td>
          <td>2024-12-22</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49c3ca0cc5a8e692853a67594899e1a9c9870369" target='_blank'>
              Enterocyte-like differentiation defines metabolic gene signatures of CMS3 colorectal cancers and provides therapeutic vulnerability
              </a>
            </td>
          <td>
            Arezo Torang, Aleksandar B. Kirov, Veerle Lammers, Kate Cameron, Valérie M Wouters, Rene F Jackstadt, T. Lannagan, Joan H de Jong, Jan Koster, Owen J. Sansom, J. Medema
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell phenotype underlies prostate cancer presentation and treatment resistance and can be regulated by epigenomic features. However, the osteotropic tendency of prostate cancer limits access to metastatic tissue, meaning most prior insights into prostate cancer chromatin biology are from preclinical models that do not fully represent disease complexity. Noninvasive chromatin immunoprecipitation of histones in plasma cell-free in humans may enable capture of disparate prostate cancer phenotypes. Here, we analyzed activating promoter- and enhancer-associated H3K4me2 from cfDNA in metastatic prostate cancer enriched for divergent patterns of metastasis and diverse clinical presentation. H3K4me2 density across prostate cancer genes, accessible chromatin, and lineage-defining transcription factor binding sites correlated strongly with circulating tumor DNA (ctDNA) fraction-demonstrating capture of prostate cancer-specific biology and informing the development of a statistical framework to adjust for ctDNA fraction. Chromatin hallmarks mirrored synchronously measured clinico-genomic features: bone versus liver-predominant disease, serum PSA, biopsy-confirmed histopathological subtype, and RB1 deletions convergently indicated phenotype segregation along an axis of differential androgen receptor activity and neuroendocrine identity. Detection of lineage switching after sequential progression on systemic therapy in select patients indicates potential utility for individualized resistance monitoring. Epigenomic footprints of metastasis-induced normal tissue destruction were evident in bulk cfDNA from two patients. Finally, a public epigenomic resource was generated using a distinct chromatin marker that has not been widely investigated in prostate cancer. These results provide insight into the adaptive molecular landscape of aggressive prostate cancer and endorse plasma cfDNA chromatin profiling as a biomarker source and biological discovery tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15869a6834cce04dbe66dca3a25682ba07c367af" target='_blank'>
              Plasma cell-free DNA chromatin immunoprecipitation profiling depicts phenotypic and clinical heterogeneity in advanced prostate cancer.
              </a>
            </td>
          <td>
            J. Sipola, Aslı D Munzur, Edmond M Kwan, Clara C Y Seo, Benjamin J Hauk, Karan Parekh, Yi Jou Ruby Liao, C. Bernales, G. Donnellan, Ingrid Bloise, Emily Fung, Sarah W. S. Ng, Gang Wang, G. Vandekerkhove, M. Nykter, M. Annala, C. Maurice-Dror, Kim Chi, C. Herberts, Alexander W Wyatt, David Y. Takeda
          </td>
          <td>2024-12-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350ae86f61f0bcf11ab3944cf0a0043eb71b585b" target='_blank'>
              Radiogenomics and machine learning predict oncogenic signaling pathways in glioblastoma
              </a>
            </td>
          <td>
            Abdul Basit Ahanger, Syed Wajid Aalam, Tariq Ahmad Masoodi, Asma Shah, Meraj Alam Khan, Ajaz A. Bhat, Assif Assad, M. Macha, M. Bhat
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) remains a significant challenge in oncology due to its aggressive nature and heterogeneity. As one of the deadliest malignancies, ESCC research lags behind other cancer types. The balance between ubiquitination and deubiquitination processes plays a crucial role in cellular functions, with its disruption linked to various diseases, including cancer. Methods Our study utilized diverse analytical approaches, encompassing Cox regression models, single-cell RNA sequencing, intercellular communication analysis, and Gene Ontology enrichment. We also conducted mutation profiling and explored potential immunotherapeutic agents. Furthermore, in vitro cellular experiments and in vivo mouse models were performed to validate findings. These methodologies aimed to establish deubiquitination-related gene signatures (DRGS) for predicting ESCC patient outcomes and response to immunotherapy. Results By integrating datasets from TCGA-ESCC and GSE53624, we developed a DRGS model based on 14 deubiquitination-related genes (DUBGs). This signature effectively forecasts ESCC prognosis, drug responsiveness, and immune cell infiltration patterns. It also influences the mutational landscape of patients. Those classified as high-risk exhibited reduced survival rates, increased genetic alterations, and more complex cellular interactions, potentially explaining their poor outcomes. Notably, in vitro and in vivo experiments identified MTOR, a key component of the signature, as a promising therapeutic target for ESCC treatment. Conclusion Our research highlights the significance of 14 DUBGs in ESCC progression. The risk score derived from this gene set enables clinical stratification of patients into distinct prognostic groups. Moreover, MTOR emerges as a potential target for personalized ESCC therapy, offering new avenues for treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f8ef21367e3b48a6b59d4e0e2f8cebf967d4f23" target='_blank'>
              Leveraging single-cell and multi-omics approaches to identify MTOR-centered deubiquitination signatures in esophageal cancer therapy
              </a>
            </td>
          <td>
            Kang Tian, Ziang Yao, Da Pan
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Novel therapeutic strategies are needed to improve the efficacy of chimeric antigen receptor (CAR) T cells as a treatment of solid tumors. Multiple tumor microenvironmental factors are thought to contribute to resistance to CAR T-cell therapy in solid tumors, and appropriate model systems to identify and examine these factors using clinically relevant biospecimens are limited. In this study, we examined the activity of B7-H3-directed CAR T cells (B7-H3.CAR-T) using 3D microfluidic cultures of patient-derived organotypic tumor spheroids (PDOTS) and then confirmed the activity of B7-H3.CAR T cells in PDOTS. Although B7-H3 expression in PDOTS was associated with B7-H3.CAR-T sensitivity, mechanistic studies revealed dynamic upregulation of co-inhibitory receptors on CAR T-cells following target cell encounter that led to CAR T-cell dysfunction and limited efficacy against B7-H3-expressing tumors. PD-1 blockade restored CAR T-cell activity in monotypic and organotypic tumor spheroids with improved tumor control and upregulation of effector cytokines. Given the emerging role of TANK-binding kinase 1 (TBK1) as an immune evasion gene, we examined the effect of TBK1 inhibition on CAR T-cell efficacy. Similar to PD-1 blockade, TBK1 inhibition restored CAR T-cell activity in monotypic and organotypic tumor spheroids, prevented CAR T-cell dysfunction, and enhanced CAR T-cell proliferation. Inhibition or deletion of TBK1 also enhanced the sensitivity of cancer cells to immune-mediated killing. Taken together, our results demonstrate the feasibility and utility of ex vivo profiling of CAR T cells using PDOTS and suggest that targeting TBK1 could be used to enhance CAR T-cell efficacy by overcoming tumor-intrinsic and -extrinsic resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a1f4efa72ad9a94f46817db042ee70f3758db32" target='_blank'>
              TBK1 Targeting Is Identified as a Therapeutic Strategy to Enhance CAR T-Cell Efficacy Using Patient-Derived Organotypic Tumor Spheroids.
              </a>
            </td>
          <td>
            Yi Sun, Luke Maggs, Apekshya Panda, Samuel J Wright, Angelina M Cicerchia, Anne Jenney, Matthew D Perricone, Caitlin E. Mills, Giulia Cattaneo, Marco Ventin, Feng Chen, Martin Q Rasmussen, Alex Miranda, Or-Yam Revach, Jacy Fang, Amina Fu, Peter J Bowling, T. Sharova, Aleigha Lawless, P. Sorger, N. Bardeesy, Xinhui Wang, K. Flaherty, Genevieve M. Boland, Arnav Mehta, Moshe Sade-Feldman, C. Ferrone, Russel Jenkins
          </td>
          <td>2025-01-09</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T cell therapy has revolutionized the treatment of hematologic malignancies, achieving remarkable clinical success with FDA-approved therapies targeting CD19 and BCMA. However, the extension of these successes to solid tumors remains limited due to several intrinsic challenges, including antigen heterogeneity and immunosuppressive tumor microenvironments. In this review, we provide a comprehensive overview of recent advances in CAR T cell therapy aimed at overcoming these obstacles. We discuss the importance of antigen identification by emphasizing the identification of tumor-specific and tumor-associated antigens and the development of CAR T therapies targeting these antigens. Furthermore, we highlight key structural innovations, including cytokine-armored CARs, protease-regulated CARs, and CARs engineered with chemokine receptors, to enhance tumor infiltration and activity within the immunosuppressive microenvironment. Additionally, novel manufacturing approaches, such as the Sleeping Beauty transposon system, mRNA-based CAR transfection, and in vivo CAR T cell production, are discussed as scalable solution to improve the accessibility of CAR T cell therapies. Finally, we address critical therapeutic limitations, including cytokine release syndrome (CRS), immune effector cell-associated neurotoxicity syndrome (ICANS), and suboptimal persistence of CAR T cells. An examination of emerging strategies for countering these limitations reveals that CRISPR-Cas9-mediated genetic modifications and combination therapies utilizing checkpoint inhibitors can improve CAR T cell functionality and durability. By integrating insights from preclinical models, clinical trials, and innovative engineering approaches, this review addresses advances in CAR T cell therapies and their performance in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb34f51cba32b1d9b69cb2a1ee23055970c02639" target='_blank'>
              Advances in CAR T cell therapy: antigen selection, modifications, and current trials for solid tumors
              </a>
            </td>
          <td>
            Safwaan H. Khan, Yeonjoo Choi, M. Veena, John K. Lee, Daniel Sanghoon Shin
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Abstract The patient‐derived xenograft (PDX) model is a crucial in vivo model extensively employed in cancer research that has been shown to maintain the genomic characteristics and pathological structure of patients across various subtypes, metastatic, and diverse treatment histories. Various treatment strategies utilized in PDX models can offer valuable insights into the mechanisms of tumor progression, drug resistance, and the development of novel therapies. This review provides a comprehensive overview of the establishment and applications of PDX models. We present an overview of the history and current status of PDX models, elucidate the diverse construction methodologies employed for different tumors, and conduct a comparative analysis to highlight the distinct advantages and limitations of this model in relation to other in vivo models. The applications are elucidated in the domain of comprehending the mechanisms underlying tumor development and cancer therapy, which highlights broad applications in the fields of chemotherapy, targeted therapy, delivery systems, combination therapy, antibody–drug conjugates and radiotherapy. Furthermore, the combination of the PDX model with multiomics and single‐cell analyses for cancer research has also been emphasized. The application of the PDX model in clinical treatment and personalized medicine is additionally emphasized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0edb643ec9b4460c4eb4c6d7bacb588877a32d4f" target='_blank'>
              Patient‐derived xenograft model in cancer: establishment and applications
              </a>
            </td>
          <td>
            Ao Gu, Jiatong Li, Meng-Yao Li, Yingbin Liu
          </td>
          <td>2025-01-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Odorant receptors (ORs), which constitute approximately 50% of all human G protein-coupled receptors, are increasingly recognized for their diverse roles beyond odor perception, including functions in various pathological conditions like brain diseases and cancers. However, the roles of ORs in glioblastoma (GBM), the most aggressive primary brain tumor with a median survival of only 15 months, remain largely unexplored. Here, we performed an integrated transcriptomic analysis combining The Cancer Genome Atlas RNA-seq and single-cell RNA sequencing data from GBM patients to uncover cell-type-specific roles of ORs within the tumor and its microenvironment. Our findings reveal that ORs display distinct expression patterns, with OR51E1 enriched in pericytes linked to vascular remodeling and angiogenesis, OR2B11 associated with tumor-associated macrophages supporting immunosuppressive phenotypes, and OR2L13 correlated with synaptic activity in recurrent tumors, potentially mediating treatment-induced neuronal adaptations. These results highlight ORs as potential therapeutic targets, offering new insights into their regulatory roles in GBM progression, immune modulation, and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0d78b05c85d74f15c35ba8438bef22107b35ac7" target='_blank'>
              Comprehensive Transcriptomic Analysis Reveals Cell-Type-Specific Roles of Human Odorant Receptors in Glioblastoma and the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hee Jin Cho, Dong Jun Yeo, HeeWoong Yang, JaeHyung Koo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [5, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>