<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-17 09:28:48 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra‐tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, this study develops engineered GBM organoids (eGBOs) harboring GBM subtype‐specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single‐cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single‐cell trajectory analysis of both eGBO‐derived tumor cells and patient GBM samples reveal the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f073ff296b054e7e4c285a38a9d240d687627e67" target='_blank'>
              Genetically Engineered Brain Organoids Recapitulate Spatial and Developmental States of Glioblastoma Progression
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H Kim, J. Millman
          </td>
          <td>2025-01-21</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Small bowel adenocarcinoma (SBA) is a rare malignancy marked with a poor prognosis. The cellular and proteomic heterogeneity within the tumor immune microenvironment (TIME) of SBA is a likely driver of prognosis, disease progression and response to therapy. We have addressed a major gap in knowledge of the TIME in SBA using highly multiplexed, protein imaging of the SBA tumor-immune ecosystem generating a comprehensive, single-cell level, spatial, proteomic atlas of TIME in > 600,000 cells from 136 tumor and matched normal samples from clinically and genomically annotated SBA patients (N=37). The SBA TIME Atlas informs on spatial distribution and interactions of tumor-intrinsic processes, diverse immune cell types, immune checkpoints, and vascularization. Enrichment of proliferating epithelial tissue, stem cells, and likely pro-tumor immune signatures in the tumor niche is contrasted by the representation of naïve and early-effector T-cells in the epithelial compartments of adjacent normal tissues. Epithelial-stem-immune cell spatial architectures within tumor and matched-normal niches were strongly associated with patient survival, suggesting malignancy is driven by spatial architecture beyond the tumor microenvironment. The blueprints of therapeutically actionable immune checkpoints at the interface between epithelial and microenvironmental T-cells as well as macrophages have established a guideline for precision immunotherapies tailored to the TIME composition in SBA. We expect that this SBA atlas will contribute to a deeper understanding of the immune contexture in this rare disease as well as other gastrointestinal cancers and help guide future precision immune-oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, , Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell spatial transcriptomics reveals how cells organize in healthy and diseased tissues. From these data, tissue segmentation analysis defines discrete compartments that organize cells into functional multicellular units. Existing methods for automated tissue segmentation rely on spatial smoothing to define spatially coherent regions but often blur the boundaries between adjacent tissue compartments. We describe Tessera, an algorithm that approaches tissue segmentation through a novel approach, dividing the tissue into small multicellular tiles whose edges track with natural tissue boundaries. Tessera achieves this by incorporating successful tools from edge-preserving smoothing, topological data analysis, and morphology-aware agglomerative spatial clustering. We show that Tessera identifies a range of known anatomical structures, in healthy mouse brain and human lymph nodes, and novel disease-associated niches, in human brain and in lung cancer. Tessera is a general-purpose tool that returns spatially coherent spatial structures with accurate boundaries across a range of spatial transcriptomics and proteomics technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfead60198d5f27b8c18dbe0ea5203b263942c83" target='_blank'>
              Accurate tiling of spatial single-cell data with Tessera
              </a>
            </td>
          <td>
            Daniel J. Stein, Miles Tran, I. Korsunsky
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d655b9c3b36c5c88b85d107ee0f6e182ac411fa5" target='_blank'>
              A quantitative spatial cell-cell colocalizations framework enabling comparisons between in vitro assembloids and pathological specimens
              </a>
            </td>
          <td>
            Gina Bouchard, Weiruo Zhang, Ilayda Ilerten, Irene Li, Asmita Bhattacharya, Yuanyuan Li, W. Trope, Joseph B Shrager, Calvin J. Kuo, Michael G Ozawa, A. Giaccia, Lu Tian, Sylvia K Plevritis
          </td>
          <td>2025-02-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris-II, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) features a complex tumor microenvironment (TME) that significantly influences patient outcomes. Understanding the TME’s cellular composition and interactions is crucial for identifying therapeutic targets to improve treatment. Methods We performed an integrative analysis combining 88 single-cell RNA sequencing (scRNA-seq) samples with 187,520 cells, 20 Visium spatial transcriptomics (ST-seq) samples with 67,933 spots, and 1383 bulk RNA-seq samples and 2 Xenium high-resolution ST-seq samples with 307,679 cells, to delineate and characterize distinct subpopulations of fibroblasts, macrophages, T/NK cells, and B/plasma cells within the PDAC microenvironment. Correlations among major cell types across 12 PDAC datasets were assessed through gene set variation analysis (GSVA). Kaplan-Meier survival analysis was used to evaluate the prognostic significance of cell subtypes. Furthermore, multiplex immunohistochemistry (mIHC) and the Xenium platform was utilized to validate cellular interactions in human PDAC tissues at both protein and RNA levels. Results Six fibroblast subtypes and eight macrophage/monocyte subtypes were identified. POSTN high fibroblasts and SPP1 high macrophages highly infiltrated tumor tissues and were associated with poor prognosis. Most immune cell subtypes mediate adverse prognoses, except for CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells, which are linked to favorable outcomes. ST-seq revealed spatial colocalization of POSTN high Fibro and SPP1 high Macro cells, as well as CCL4 high CD8+ T EFF and IGHG1 high plasma cells. These findings were corroborated by mIHC and validated using Xenium spatial transcriptomics with single-cell resolution, which confirmed the expression and spatial proximity of these markers at both the gene and protein levels. Conclusion This integrative analysis of the PDAC TME underscores the prognostic importance of POSTN high fibroblasts and SPP1 high macrophages, while also highlighting the protective roles of CCL4 high CD8+ T EFF cells and IGHG1 high plasma cells. These insights could improve PDAC treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cellular organization is central to tissue function and homeostasis, influencing development, disease progression, and therapeutic outcomes. The emergence of spatial omics technologies, including spatial transcriptomics and proteomics, has enabled the integration of molecular and histological features within tissues. Analyzing these multimodal data presents unique challenges, including variable resolutions, imperfect tissue alignment, and limited or variable spatial coverage. To address these issues, we introduce CORAL, a probabilistic deep generative model that leverages graph attention mechanisms to learn expressive, integrated representations of multimodal spatial omics data. CORAL deconvolves low-resolution spatial data into high-resolution single-cell profiles and detects functional spatial domains. It also characterizes cell-cell interactions and elucidates disease-relevant spatial features. Validated on synthetic data and experimental datasets, including Stero-CITE-seq data from mouse thymus, and paired CODEX and Visium data from hepatocellular carcinoma, CORAL demonstrates robustness and versatility. In hepatocellular carcinoma, CORAL uncovered key immune cell subsets that drive the failure of response to immunotherapy, highlighting its potential to advance spatial single-cell analyses and accelerate translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/295042adfe589c5c9974e6c8e7ec5b6246fdd726" target='_blank'>
              Learning single-cell spatial context through integrated spatial multiomics with CORAL
              </a>
            </td>
          <td>
            Siyu He, Matthew Bieniosek, Dongyuan Song, Jingtian Zhou, Benjamin Chidester, Zhenqin Wu, Joseph Boen, Padmanee Sharma, Alexandro E. Trevino, James Zou
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Technologies for estimating RNA expression at high throughput, in intact tissue slices, and with high spatial resolution (spatial transcriptomics; ST) shed new light on how cells communicate and tissues function. A fundamental step common to all ST protocols is quantification, namely segmenting the plane into regions, each approximating a cell, and then collating the molecules inside each region to estimate the cellular expression profile. Despite many advances in this area, a persisting problem is that of wrong assignment of molecules to cells, which limits most current applications to the level of a priori defined cell subsets and complicates the discovery of novel cell states. Here, we develop resolVI, a model that operates downstream of any segmentation algorithm to generate a probabilistic representation, correcting for misassignment of molecules, as well as for batch effects and other nuisance factors. We demonstrate that resolVI improves our ability to distinguish between cell states, to identify subtle expression changes in space, and to perform integrated analysis across datasets. ResolVI is available as open source software within scvi-tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef6d6e98a4c414a78cbaa5ec4c9ee19d233f609" target='_blank'>
              ResolVI - addressing noise and bias in spatial transcriptomics
              </a>
            </td>
          <td>
            Can Ergen, N. Yosef
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Adaptive immunity, mediated by T and B cell responses, is essential for defending against infections and cancers while also being implicated in autoimmune diseases. Tracking T and B cell repertoires in situ at single-cell resolution is essential for understanding adaptive immune responses. To address the lack of tools for in situ single-cell T/BCR (XCR) sequencing, we developed Stereo-XCR-seq, an efficient strategy for retrieving and sequencing TCR and BCR from Stereo-seq cDNA libraries at subcellular resolution. Stereo-XCR-seq provides unbiased full-length XCR reads alongside spatial transcriptomics, enabling the identification of heterogeneous lymphoid aggregates with distinct clonal activities in cancers and inflammatory bowel disease (IBD). We identified plasma cell aggregates that differ from tertiary lymphoid structures (TLSs) in both transcriptomic profiles and clonal activities, with spatial positioning potentially mediating unique immune responses. Collectively, Stereo-XCR-seq enables in situ single-cell profiling of T and B cell clonal activities within tissue microenvironments, providing insights into lymphocyte adaption to environmental stimuli. This technology provides potential for advancing our understanding of tissue immunity and the development of therapeutic strategies for immune disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a087fd6bab927232a0eec4056a80591d4dc5a4" target='_blank'>
              Single cell resolved spatial immune repertoire unveils spatial heterogeneity of lymphoid aggregates in human immune disorders
              </a>
            </td>
          <td>
            Xiaojuan Zhan, Yi Liu, Yanying Guo, Wenwen Zhou, Yixin Yan, Hui Zeng, Xuan Dong, Xiaoyu Chen, Rong Ma, Zhong Liu, Fan Zhu, Xubin Zheng, Xinxing Li, Jinwen Yin, Francis Ka-ming Chan, Chuanyu Liu, Longqi Liu, Xun Xu, Yong Hou, Haoran Tao, Yuliang Dong, Tao Zeng, Young Li, Jingying Zhou, Zexian Zeng, Yu Feng
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine.


METHODS
We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types.


RESULTS
Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells-a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions.


CONCLUSIONS
This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy.
              </a>
            </td>
          <td>
            Z. Abdulrahman, Roderick C Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, Mikihiro Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 There is a growing appreciation for the role that the spatial organization of immune cells within the tumor microenvironment (TME) plays in shaping cellular function and response to therapy. However, the majority of studies have been limited to analyzing immune responses at a single time point, and so, little is known about the spatiotemporal changes that underlie successful tumor rejection after cancer immunotherapy. In this study, we used the CODEX multiplex imaging system with a 32-plex antibody panel to longitudinally characterize the spatial changes in the lymphoid and myeloid cell populations that were previously identified by scRNA-Seq and CyTOF to be associated with complete tumor rejection in mice treated with the combination of α-PD-1 and α-CTLA-4 (combo ICT) in our well-characterized mouse T3 MCA sarcoma model. T3 tumor-bearing mice were treated with control antibody or combo ICT, tumors were harvested at multiple time points (day 7, 9, 10, 11 and 13), fresh frozen, sectioned, and subjected to CODEX multiplex imaging. A total of 42 tumors were imaged with the full range of the tumor-immune boundary. A hierarchical cell clustering approach developed in-house was used to profile 13,051,156 cells resulting in the identification of 13 distinct cell types. Our results show that after ICT the decrease in T3 cells in the tumor tissue starts on day 10 and culminates in the complete elimination of tumor cells by day 13. This immune-mediated response is driven by an increase in CD4+ T cells, which allows them to more frequently co-localize with CD8+ T cells and type 1 conventional dendritic cells (cDC1), forming three-cell clusters with increased cell contacts. This change in cell-to-cell interactions is followed by a broader reorganization of the TME as noted by the expansion of a lymphoid-rich cellular neighborhood (CN) that harbors the majority of CD4+ and CD8+ T cells. Interestingly, this CN becomes a hub for T cell effector functions, characterized by an enrichment of GZM-B+ and KI67+ T cells, and exhibits spatial dynamics throughout tumor regression. While both GZM-B+ and KI67+ T cells are observed in proximity to blood vessels on days 7 and 9, they begin to spatially separate into distinct regions after day 10, indicating that cytotoxic T cells move towards areas with tumor cells, while proliferating T cells are more frequently observed at the periphery. This dynamic behavior was further confirmed by assessing the communication rules between CNs, which showed that the lymphoid-rich CN acts by eliminating layers of tumor cells at the tumor boundary and by modulating nearby myeloid populations. Additionally, integration of CODEX dataset with scRNA-seq revealed CD4+ T cells produce Interferon-gamma and a change in the molecular chemotactic pathways orchestrating lymphoid and myeloid communication. Overall, this study provides a detailed spatial view of the cellular events that result in successful ICT and allows us to propose a computational model of the actions of T cells in coordinating the antitumor response after α-CTLA-4/α-PD1.
 Citation Format: Ruan FV Medrano, Vladimir Sukhov, Maxim Artyomov, Robert Schreiber. Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. We aim to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumours in various organs. Lineage tracing and clonal heterogeneity were evaluated using the state-of-the-art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. To identify melanoma clones across different metastatic sites, specifically subcutaneous, lungs and brain, C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7PV1 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor the tumour growth of mice injected intravenously and intracranially. Subcutaneous tumours were measured by calliper. Mice were euthanized before ethical endpoints. Tumors were harvested and single cell analysis as well as DNA sequencing was performed to identify barcodes expressed by the tumor cells across different tumor-sites. Barcode analysis was conducted both before cell implantation and at tumor harvest. Clonality exhibited significant variability between sites. Single-cell RNA sequencing will unveil states associated with dominant clones present at different metastatic sites. This information will provide a new understanding of melanoma transcriptional phenotype behaviours and biological characteristics at different metastatic sites and in response to therapy.
 Citation Format: Veronica Aedo-Lopez. Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Triple-negative breast cancer is a prevalent breast cancer subtype with the lowest 5-year survival. Several factors contribute to its treatment response, but the inherent molecular and cellular tumor heterogeneity are increasingly acknowledged as crucial determinants. Methods Spatial transcriptomic profiling was performed on FFPE tissues from a retrospective, treatment-naive group of women with differential prognoses (17 with >15 years survival-good prognosis (GPx) and 15 with <3 years survival-poor prognosis (PPx)) using GeoMX® Digital Spatial Profiler. Regions of interest were segmented on pan-cytokeratin and analyzed for tumor and stromal components, probed using GeoMx human whole transcriptome atlas (WTA) panel. Data quality control, normalization, and differential analysis was performed in R using GeomxTools and linear mixed models. Additional analyses including cell-type deconvolution, spatial entropy, functional enrichment, TF-target / ligand-receptor analysis and convolution neural networks were employed to identify significant gene signatures contributing to differential prognosis. Results Here we report on the spatial and molecular heterogeneity underlying differential prognosis. We observe that the state of the epithelia and its microenvironment (TME) are transcriptionally distinct between the two groups. Invasive epithelia in GPx show a significant increase in immune transcripts with the TME exhibiting increased immune cell presence (via IF), while in PPx they are more metabolically and translationally active, with the TME being more mesenchymal/fibrotic. Specifically, pre-cancerous epithelia in PPx display a prescience of aggressiveness as evidenced by increased EMT-signaling. We identify distinct epithelial gene signatures for PPx and GPx, that can, with high accuracy, classify samples at the time of diagnosis and likely inform therapy. Conclusions To the best of our knowledge, this is the first study to leverage spatial transcriptomics for an in-depth delineation of the cellular and molecular underpinnings of differential prognosis in TNBC. Our study highlights the potential of spatial transcriptomics to not only uncover the molecular drivers of differential prognosis in TNBC but also to pave the way for precision diagnostics and tailored therapeutic strategies, transforming the clinical landscape for this aggressive breast cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31351362773ec3359f5dd1e2c1fcb9f73d1b49df" target='_blank'>
              Spatially distinct cellular and molecular landscapes define prognosis in triple negative breast cancer
              </a>
            </td>
          <td>
            Kavitha Mukund, Darya Veraksa, David Frankhouser, Lixin Yang, Jerneja Tomsic, Raju Pillai, Srijan Atti, Zahra Mesrizadeh, D. Schmolze, Xiao-Cheng Wu, Mary-Anne LeBlanc, Lucio Miele, Augusto Ochoa, Victoria L Seewaldt, Shankar Subramaniam
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology and extend it to the surrounding area. In this study, we developed SpatialKNifeY (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. Using spatial transcriptomic data from breast cancer, we applied the SKNY algorithm to identify tumor spatial domains, followed by clustering of the domains, trajectory estimation, and spatial extension to the tumor microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed tumor vascularization and immunodeficiency at mid- and late-stage progression in TME. Furthermore, we applied the SKNY to integrate and cluster the spatial domains of 14 patients with metastatic colorectal cancer, and the clusters were divided based on the TME characteristics. In conclusion, the SKNY facilitates the determination of the functions and mechanisms in the microenvironment and cataloguing of the features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796429c918dab0f1b4d04958c50f7cf17d100c5e" target='_blank'>
              SpatialKNifeY (SKNY): Extending from spatial domain to surrounding area to identify microenvironment features with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Mitsuho Imai, Y. Nakamura, Takayuki Yoshino, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is the deadliest cancer worldwide. Therapeutic progress stagnate, highlighting the complexity to replicate NSCLC in preclinical models. Drug discovery studies rely mostly on cancer cells in two-dimension (2D), which poorly predict drug efficacy in patients. There is a growing interest in three-dimensional (3D) preclinical models, such as 3D spheroids, to better model tumor phenotype and improve therapeutic prediction. However, a comprehensive view of 3D culture methods impact on transcriptomes, epigenomes and pharmacological responses and their correlations to NSCLC tumors is still missing. Here, we demonstrate that NSCLC spheroids undergo time-dependent transcriptomic and epigenomic changes, which peak after 3 weeks of culture. While DNA methylome remained stable, chromatin methylation and acetylation marks gained features of advanced NSCLC in a time-dependent manner. Single-cell transcriptomic profiling of spheroids demonstrated that time of 3D culture improved the correlation to NSCLC tumors. Moreover, long-term culture of 3D spheroids increased drug screening predictability, by showing resistance to drugs that failed in NSCLC patients (such as HDAC inhibitors) while demonstrating novel pharmacological vulnerabilities and synergistic interactions (such as combination of PRMT and HDAC inhibitors). Strikingly, reverting 3D spheroids back to 2D culture rapidly reversed transcriptomic, epigenetic and pharmacological signatures acquired after 3 weeks of 3D culture, highlighting the critical impact of cell culture conditions on NSCLC phenotype. Collectively, our findings demonstrate that implementing a time-dependent maturation process into 3D spheroid culture induces chromatin and transcriptomic changes that enhance NSCLC preclinical modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644ee3c95f246053a935ce676721b85861fe7f5" target='_blank'>
              Time-dependent chromatin maturation during 3D spheroid culture improves preclinical modeling of non-small cell lung cancer
              </a>
            </td>
          <td>
            Anaïs Darracq, Nicolas Sgarioto, Marielle Huot, Gabrielle McInnes, Antoine Meant, Alexandra Langford, Oscar D Villarreal, François Marois, Maxime Caron, P. Saint-Onge, S. Leclerc, G. Cagnone, Gregor Andelfinger, Stéphane Richard, Daniel Sinnett, Serge McGraw, Noël J-M Raynal
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chronic inflammation is a well-established risk factor for cancer, but the underlying molecular mechanisms remain unclear. Using a mouse model of colitis, we demonstrate that colonic stem cells retain an epigenetic memory of inflammation following disease resolution, characterized by a cumulative gain of activator protein 1 (AP-1) transcription factor activity. Further, we develop SHARE-TRACE, a method that enables simultaneous profiling of gene expression, chromatin accessibility and clonal history in single cells, enabling high resolution tracking of epigenomic memory. This reveals that inflammatory memory is propagated cell-intrinsically and inherited through stem cell lineages, with certain clones demonstrating dramatically stronger memory than others. Finally, we show that colitis primes stem cells for amplified expression of regenerative gene programs following oncogenic mutation that accelerate tumor growth. This includes a subpopulation of tumors that have exceptionally high AP-1 activity and the additional upregulation of pro-oncogenic programs. Together, our findings provide a mechanistic link between chronic inflammation and malignancy, revealing how long-lived epigenetic alterations in regenerative tissues may contribute to disease susceptibility and suggesting potential therapeutic strategies to mitigate cancer risk in patients with chronic inflammatory conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Digital high-resolution H&E images provide valuable insights into tumor heterogeneity within the tumor microenvironment (TME). However, the ability to perform detailed cell typing solely based on H&E images remains limited. Recent advances in high-resolution spatial transcriptomics (ST) enable precise characterization of cell types and their spatial relationships within the TME, addressing challenges in understanding tumor heterogeneity. In this study, we developed AI models to predict cell types in breast cancer—including subtypes of lymphocytes that are challenging to differentiate visually—using large-scale image-based ST data aligned with H&E images. Methods: We established a breast cancer image-based ST database comprising 190 samples from 113 breast cancer patients, obtained from surgically resected primary tumors. Image-based ST data were generated using the Xenium platform with a 500-gene panel and matched with high-resolution H&E images. Cell type maps were constructed using reference single-cell RNA-seq data and transferred to ST data. The ST data were spatially registered to the corresponding H&E images, and cell type masks were generated at matching resolutions. AI models were trained to segment cell type masks, including epithelial cells, cancer cells, myeloid cells, fibroblasts, endothelial cells, T cells, and B cells. Additionally, refined cell type predictions were performed for dendritic cells, NK cells, CD4+ T cells, and CD8+ T cells. Model performance was validated using external whole-slide image datasets containing ST data from four independent cohorts. Results: Model performance was assessed using the area under the receiver operating characteristic (AUROC) curve for each cell type mask. Internal validation on four samples yielded AUROC values ranging from 0.90 to 0.95. External validation across independent whole-slide datasets demonstrated AUROC values between 0.88 and 0.97 for all cell type masks. The AI model successfully mapped TME cell types with high resolution, using only H&E images. Conclusion: By employing a self-supervised approach that integrates high-resolution H&E images with image-based ST data, we developed AI-driven tools for TME analysis. These models enable accurate identification of detailed cell types and their spatial relationships within the TME. This approach facilitates large-scale analysis of breast cancer TME and holds potential for advancing our understanding of tumor biology and therapeutic strategies.
 Citation Format: Haenara Shin, Dongjoo Lee, Yooeun Kim, Daeseung Lee, Kwon Joong Na, Chihwan David Cha, Hosub Park, Hongyoon Choi. A self-supervised AI model leveraging spatial omics for analyzing tumor microenvironment heterogeneity in breast cancer only with H&E [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B038.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0b044513a32c1bfdbc6767c40e3363e83eee34" target='_blank'>
              Abstract B038: A self-supervised AI model leveraging spatial omics for analyzing tumor microenvironment heterogeneity in breast cancer only with H&E
              </a>
            </td>
          <td>
            Haenara Shin, Dongjoo Lee, Yooeun Kim, D. Lee, Kwon Joong Na, C. Cha, Hosub Park, Hongyoon Choi
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor dissemination is increasingly recognized to begin early in tumor development. Although most of these early disseminated cells are cleared, some survive and persist below clinical detection, acting as reservoirs for metastatic relapse. Metastatic tumor cells often rely on interactions with local stromal cells to support their colonization. In this study, we propose that pericyte-tumor cell interactions promote dormancy induction in the early metastatic lung, enhancing disseminated tumor cell (DTC) persistence. Extravital imaging demonstrated that DTCs interact with pericytes upon extravasation into the lung. Co-culture experiments were used to assess DTC fate after pericyte contact and revealed that transient contact with pericytes reduced the proliferation of metastatic 4T1 breast cancer cells but had no effect on non-metastatic 67NR cells. In vivo, transient pericyte contact resulted in higher lung metastatic burden, driven by small, non-proliferative lesions (<6 cells), 10 days after intracardiac injection. These lesions exhibited reduced KI67 staining and EdU incorporation compared to those from monocultured cells. We further observed that primary lung pericytes transferred lyso-phospholipids (lyso-PLs) specifically to metastatic 4T1 cells through direct contact. Gene expression analysis indicated that transient pericyte contact activated pathways related to syncytium formation in metastatic cells. In normal physiology, pericytes act in a syncytium to regulate blood flow via mechanosensitive channels in response to blood pressure changes. We hypothesize that tumor cells exploit these mechanosensitive responses to trigger lyso-PL transfer from pericytes. Supporting this, calcium imaging showed higher calcium activity in pericytes co-cultured with 4T1 cells, and calcium channel inhibitors significantly reduced lyso-PL transfer. Pharmacological activation of pericyte calcium channels induced lyso-PL release, which was subsequently taken up by tumor cells. Conditioned medium from activated pericytes, containing free lyso-PLs, recapitulated the reduced proliferation observed in transient co-culture. Finally, we found our pericyte-induced dormancy signature to be associated with tumor dormancy and distant metastasis free survival latency in breast cancer patients. Together, these findings suggest that early DTCs may exploit pericyte signaling mechanisms to enter dormancy, facilitating their persistence at metastatic sites and contributing to future relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fe686f97fcb46c56e3f6bfbef8e78fcca9cb41" target='_blank'>
              Pericyte-tumor crosstalk facilitates metastatic tumor cell latency through PIEZO1-activated lysophospholipid transfer
              </a>
            </td>
          <td>
            Tamara McErlain, Morgan J Glass, Elizabeth C McCulla, Caitlin A Madden, Lauren E Ziemer, Kirsten E Overdahl, Alan K. Jarmusch, Michael J. Kruhlak, A. Chesler, Howard H Yang, Maxwell P Lee, Cristina M Branco, Meera Murgai
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Spatial transcriptomics quantifies gene expression within its spatial context, significantly advancing biomedical research. Understanding gene spatial expression and the organization of multicellular systems is vital for disease diagnosis and studying biological processes. However, existing models often struggle to integrate gene expression data with cellular spatial information effectively. In this study, we introduce SpatialFormer, a hybrid framework combining convolutional networks and transformers to learn single-cell multimodal and multi-scale information in the niche context, including expression data and subcellular gene spatial distribution. Pre-trained on 300 million cell pairs from 12 million spatially resolved single cells across 62 Xenium slides, SpatialFormer merges gene spatial expression profiles with cell niche information via the pair-wise training strategy. Our findings demonstrate that SpatialFormer distills biological signals across various tasks, including single-cell batch correction, cell-type annotation, co-localization detection, and identifying gene pairs that are critical for the immune cell-cell interactions involved in the regulation of lung fibrosis. These advancements enhance our understanding of cellular dynamics and offer new pathways for applications in biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73553bdf8819f572ec3a56141520bc2e3e9f559d" target='_blank'>
              SpatialFormer: Universal Spatial Representation Learning from Subcellular Molecular to Multicellular Landscapes
              </a>
            </td>
          <td>
            Jun Wang, Yuanhua Huang, Ole Winther
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The FDA approval of T cell receptor-engineered T cells (TCR-T) for synovial sarcoma demonstrates the potential for adoptive T cell therapies (ACTs) in solid tumors. However, the paucity of tumor-specific targets without expression in normal tissues remains a major bottleneck, especially in rare cancer subtypes. Here, we present a comprehensive computational pipeline called SCAN-ACT that leverages single cell RNA sequencing and multi-omics data from tumor and normal tissues to nominate and prioritize targets for both chimeric antigen receptor (CAR)- and TCR-T cells. For surface membrane targets, SCAN-ACT proposes target pairs for bispecific Boolean logic-gated CAR T cells. For peptide-MHC targets, SCAN-ACT proposes intracellular peptides bound to a diverse set of human leukocyte antigens. We applied the SCAN-ACT pipeline to soft tissue sarcoma (STS), analyzing 986,749 single cells to identify and prioritize 395 monospecific CAR-T targets, 14,192 bispecific CAR-T targets, and 5,020 peptide-MHC targets for TCR-T cells. Selected targets were validated experimentally by protein expression and for peptide-MHC binding. Proposed targets and target pairs reflected the mesenchymal, neuronal, and hematopoietic ontogeny of STS. This work provides a robust data repository along with a web-based and user-friendly set of analysis tools to accelerate ACT development for solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcdecff6378c09bb0e27b421562a20dfc23338c" target='_blank'>
              SCAN-ACT: Adoptive T Cell Therapy Target Discovery Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Stefano Testa, Aastha Pal, Ajay Subramanian, Sushama Varma, Jack Pengfei Tang, Danielle Graham, Sara Arfan, Minggui Pan, Nam Q. Bui, Kristen Ganjoo, Sarah Dry, Paul Huang, Matt van de Rijn, Wei Jiang, Anusha Kalbasi, E. J. Moding
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The spatial arrangement of cells plays a critical role in determining their functions and interactions within tissues. However, single-cell RNA sequencing (scRNA-seq) dissociates cells from their native tissue context, resulting in a loss of spatial information. Here, we show that complex tissue structures can be reassembled from the gene expression profiles of dissociated cells. To achieve this, we developed LUNA, a generative AI model that reconstructs tissues conditioned solely on gene expressions of cells by learning spatial priors over existing spatially resolved datasets. We show that LUNA effectively reconstructs slices from the MERFISH whole mouse brain atlas with over 1.2 million cells, including cells from cell types never seen during model training. Applying LUNA to the mouse central nervous system scRNA-seq atlas, we show that LUNA is applicable for de novo generation of tissue structures. Additionally, LUNA can infer locations of nuclei lost during cell profiling with Slide-tags technology and correctly places cells belonging to spatially distinct compartments in a human metastatic melanoma sample. We envision that AI-driven tissue reassembly can help to overcome current technological limitations and advance our understanding of tissue organization and function, paving the way towards virtual tissue models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2575a743b02c9647eeddfbc3632592d6900d90aa" target='_blank'>
              Tissue reassembly with generative AI
              </a>
            </td>
          <td>
            Tingyang Yu, C. Ekbote, Nikita Morozov, Jiashuo Fan, Pascal Frossard, Stéphane d’Ascoli, M. Brbić
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Brain metastasis occurs in approximately 16% of metastatic breast cancer patients, and incidence is increasing. Patients with breast cancer brain metastases frequently develop neurological problems including cognitive impairments and seizures. We previously demonstrated accumulation of activated microglia around the tumour site in a breast cancer brain metastasis model, which co-localised with spontaneously occurring local field potential events that resembled interictal epileptic discharges. However, the mechanisms underlying these spatially restricted effects on neuronal excitability are poorly understood. Methods To better understand the underlying cellular changes and potential mechanisms, we used digital spatial profiling of brain cell subtype-specific markers to chart the spatial organisation of neurons, oligodendrocytes, astrocytes, and microglia in relation to metastatic lesion site. We developed a novel chordline-based approach to analyse these spatial changes in biomarker distribution. Results Several protein markers associated with proliferation (Ki67), astrocytes (GFAP) and microglia (CD11b, IBA1, CD45, MSR1) were upregulated in the tumour compared to healthy contralateral brain parenchyma. In contrast, some neuroglial markers (MAP2, Neurofilament light, NeuN, S100B and TMEM119) were lower in the lesion compared to normal tissue. Conclusions Overall, the protein marker changes in the lesion are indicative of neuronal cell loss and changes in the immune microenvironment of the metastatic lesion involving activated microglia, astrocytes and recruitment of peripheral immune cells. Such cellular changes may contribute to the changes in electrical activity resulting from brain metastases observed in mice and patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61dbeafc4e0030d90e60f5d024e3f1200854ddc2" target='_blank'>
              Spatial profiling of central nervous system cellular markers in murine breast cancer to brain metastases
              </a>
            </td>
          <td>
            Karen Hogg, Grant Calder, Alastair P. Droop, Ming Yang, Anna Simon, Mark J Hunt, Paul M Kaye, Miles A. Whittington, Sangeeta Chawla, W. Brackenbury
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of tumors to evolve and adapt by developing subclones in different genetic and epigenetic states is a major challenge in oncology. Traditional tools like multi-regional sequencing used to study tumor evolution and the resultant intra-tumor heterogeneity (ITH) are often impractical because of their resource-intensiveness and limited scalability. Here, we present MorphoITH, a novel framework that leverages histopathology slides to deconvolve molecular ITH through tissue morphology. MorphoITH integrates a self-supervised deep learning similarity measure to capture phenotypic variation across multiple dimensions (cytology, architecture, and microenvironment) with rigorous methods to eliminate spurious sources of variation. Using a prototype of ITH, clear cell renal cell carcinoma (ccRCC), we show that MorphoITH captures clinically-significant biological features, such as vascular architecture and nuclear grades. Furthermore, we find that MorphoITH recognizes differential biological states corresponding to subclonal changes in key driver genes (BAP1/PBRM1/SETD2). Finally, by applying MorphoITH to a multi-regional sequencing experiment, we postulate evolutionary trajectories that largely recapitulate genetic evolution. In summary, MorphoITH provides a scalable phenotypic lens that bridges the gap between histopathology and genomics, advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f0d69b3ba41ef9a508aa6acb9b650626ef267d" target='_blank'>
              MorphoITH: A Framework for Deconvolving Intra-Tumor Heterogeneity Using Tissue Morphology
              </a>
            </td>
          <td>
            Aleksandra Weronika Nielsen, H. E. Manoochehri, Hua Zhong, Vandana Panwar, V. Jarmale, Jay Jasti, Mehrdad Nourani, Dinesh Rakheja, J. Brugarolas, Payal Kapur, Satwik Rajaram
          </td>
          <td>2025-02-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bec213fd287708367fee6bf0c6de9efb0e1c0cb" target='_blank'>
              Natural killer cells occupy unique spatial neighborhoods in HER2- and HER2+ human breast cancers
              </a>
            </td>
          <td>
            Femke A I Ehlers, Katie E. Blise, C. Betts, Shamilene Sivagnanam, L. Kooreman, E. S. Hwang, Gerard M. J. Bos, L. Wieten, L. Coussens
          </td>
          <td>2025-01-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, Abigail Wong-Rolle, Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Myeloid-derived cells and tumor-associated macrophages (TAMs) represent an emerging field in melanoma research. Single-cell sequencing has enhanced our understanding of interactions within the tumor microenvironment (TME). Although there is no current consensus, several solid-tumor associated TAM subtypes have been identified by single-cell RNA sequencing (scRNA-seq). Due to the lack of agreement across models and tumor types, we aimed to characterize the myeloid sub-compartment in melanoma by analyzing compiled public scRNA-seq datasets. In this study, we identified six myeloid cell clusters, designated M_C1 through M_C6, with distinct gene expression profiles indicating differences in structure and function. The frequency of the M_C1 subtype was found to be predictive of overall survival (OS) in brain metastatic melanoma. This finding was externally validated by deconvolution of publicly available bulk RNA sequencing data from a clinical trial of advanced melanoma patients treated with nivolumab, as well as data from The Cancer Genome Atlas (TCGA) for melanoma stages I–IV. The M_C1 cell type appears to be a hybrid of dendritic cells and macrophages, demonstrating prognostic relevance across all melanoma stages and predictive potential for immunotherapy response. These data suggest that M_C1 may have utility as a prognostic biomarker in melanoma and as a target for future cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            , Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Central Nervous System (CNS) tumors have sophisticated tumor microenvironment (TME) with different cell types such as astrocytes, microglia, neurons, vascular endothelial cells and immune cells. These non-cancerous cells orchestrate the brain TME to regulate cancer progression and therapeutic response. This study aimed to develop a cell composition deconvolution method for CNS tumor and to determine the impact of these cell compositions on patients’ outcomes. Methods We identified the cell type-specific CpG loci using the pairwise differential methylation analysis for 13 major cell types in CNS. Using non-negative least squares (NNLS) methods, we established this cell-type deconvolution approach, MDBrainT, for brain tumors. The predictive accuracy of our MDBrainT model was tested in the DNA methylation profiling of the purified cell samples and compared against another algorithm. Cell composition was predicted by MDBrainT in several brain tumor (glioma, ependymoma, medulloblastoma and ATRT) cohorts and the correlation between cell composition and tumor molecular subtypes and patient outcomes was also assessed. Results Cell type-specific CpG loci for CNS TME was used to build MDBrainT model. Based on these DNA methylation markers, MDBrainT predicted the cell composition in the TME including tumor cells with high accuracy. Endothelial cells were predominately presented in glioblastomas while the percentage of CD8 T cells wassignificantly higher in ATRT. A substantial difference of cell composition was two molecular groups of posterior fossa ependymoma (PFA vs PFB). A higher percentage of cells in TME was usually associated with worse outcomes. Conclusions MDBrainT is a robust algorithm for cell composition prediction for brain TME. Cell composition in brain TME is distinct across different pathological types and molecular subtypes. Key Points MDBrainT is a robust DNA methylation-based deconvolution approach for brain tumor microenvironment (TME). Different molecular subtypes of brain tumors have distinct cell composition patterns. Cell composition in brain TME informs patients of outcomes. Importance of this study DNA methylation is a cell type specific marker that has been utilized for tumor molecular diagnosis, disease progression and therapeutic monitoring. A DNA methylation-based classifier for brain tumors precisely predicts the molecular subtypes but not the cell composition of tumor microenvironment. Brain tumors are a complex cell mixture where tumor microenvironment is critical for tumorigenesis and therapeutic resistance. Here, we developed a novel deconvolution approach (MDBrainT) to predict cell composition for brain tumors. Our model has revealed the heterogeneity of cell composition between tumor types. In addition, tumors with different molecular subtypes have distinct cell composition. Cell percentage in TME also informs patients of outcomes. The tumor microenvironment including cell composition of each patient may direct the different regimen of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1823256c94dd6a377f1b3165f4ae951356ef07" target='_blank'>
              DNA Methylation-Based Cell Type Deconvolution Reveals the Distinct Cell Composition in Brain Tumor Microenvironment
              </a>
            </td>
          <td>
            Feili Liu, Jin Qian, Chenkai Ma
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell technologies have enhanced our knowledge of molecular and cellular heterogeneity underlying disease. As the scale of single-cell datasets expands, linking cell-level phenotypic alterations with clinical outcomes becomes increasingly challenging. To address this, we introduce CellPhenoX, an eXplainable machine learning method to identify cell-specific phenotypes that influence clinical outcomes. CellPhenoX integrates classification models, explainable AI techniques, and a statistical framework to generate interpretable, cell-specific scores that uncover cell populations associated with relevant clinical phenotypes and interaction effects. We demonstrated the performance of CellPhenoX across diverse single-cell designs, including simulations, binary disease-control comparisons, and multi-class studies. Notably, CellPhenoX identified an activated monocyte phenotype in COVID-19, with expansion correlated with disease severity after adjusting for covariates and interactive effects. It also uncovered an inflammation-associated gradient in fibroblasts from ulcerative colitis. We anticipate that CellPhenoX holds the potential to detect clinically relevant phenotypic changes in single-cell data with multiple sources of variation, paving the way for translating single-cell findings into clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3660ca5e687eb30732f3def619ad0c7183f84308" target='_blank'>
              CellPhenoX: An eXplainable Cell-specific machine learning method to predict clinical Phenotypes using single-cell multi-omics
              </a>
            </td>
          <td>
            Jade Young, Jun Inamo, Zachary Caterer, Revanth Krishna, Fan Zhang
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity. Methods We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types—CXCR3+CD68+ or CD8+—to profile their spatial relationships with CXCL9+CD68+ cells. Results Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely ‘cold, tumor-rich’ and ‘cold, stroma-rich’, exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8. Conclusions Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a85c2bcf9c97b3611062a72118d56273a4ef13" target='_blank'>
              Cellular interactions within the immune microenvironment underpins resistance to cell cycle inhibition in breast cancers
              </a>
            </td>
          <td>
            Jason I. Griffiths, Patrick A. Cosgrove, Eric F. Medina, Aritro Nath, Jinfeng Chen, Fred Adler, Jeffrey T. Chang, Qamar J Khan, Andrea H Bild
          </td>
          <td>2025-03-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Constructing a comprehensive spatiotemporal map of tumor heterogeneity is essential for understanding tumor evolution, with copy number variation (CNV) as a significant feature. Existing studies often rely on tools originally developed for single-cell data, which fail to utilize spatial information. Our study aims to develop a model that fully leverages spatial omics data to elucidate spatio-temporal changes in tumor evolution. Here, we introduce SCOIGET (Spatial COpy number Inference by Graph on Evolution of Tumor), a novel framework using graph neural networks with graph attention layers to learn spatial neighborhood features of gene expression and infer copy number variations. This approach integrates spatial multi-omics features to create a comprehensive spatial map of tumor heterogeneity. Notably, SCOIGET achieves a substantial reduction in error metrics (e.g., mean squared error, cosine similarity, and distance measures) and produces superior clustering performance as indicated by higher Silhouette Scores compared to existing methods. Our model significantly enhances the efficiency and accuracy of tumor evolution depiction, capturing detailed spatial and temporal changes within the tumor microenvironment. It is versatile and applicable to various downstream tasks, demonstrating strong generalizability across different spatial omics technologies and cancer types. This robust performance improves research efficiency and provides valuable insights into tumor progression. In conclusion, SCOIGET offers an innovative solution by integrating multiple features and advanced algorithms, providing a detailed and accurate representation of tumor heterogeneity and evolution, aiding in the development of personalized cancer treatment strategies. Key Points 1. Innovative Framework for Spatially Contextualized CNV Inference Introduces SCOIGET, a graph-based model that integrates spatial omics data with graph neural networks and attention mechanisms to accurately infer spatial copy number features. 2. Detailed Tumor Heterogeneity Mapping Constructs representations of tumor evolution, capturing heterogeneity and clonal dynamics at both cellular and subcellular resolutions. 3. Versatile and Robust Performance Outperforms existing methods across multiple datasets, demonstrating reliability and adaptability to diverse spatial omics platforms and cancer types. 4. Broad Downstream Applications Enables tasks such as tumor subclone identification, clustering, and evolutionary trajectory analysis, facilitating precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76feb62f24d1b55375efafabff78309d558b9c49" target='_blank'>
              Spatial interaction mapping of PD-1/PD-L1 in head and neck cancer reveals the role of macrophage-tumour barriers associated with immunotherapy response
              </a>
            </td>
          <td>
            V. Y. Naei, Rafael Tubelleza, J. Monkman, Habib Sadeghirad, Meg L. Donovan, Tony Blick, A. Wicher, Sara Bodbin, Amelie Viratham, Robert Stad, Subham Basu, C. Cooper, Catherine Barnett, K. O’Byrne, Rahul Ladwa, M. Warkiani, Brett G M Hughes, A. Kulasinghe
          </td>
          <td>2025-02-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while retaining the crucial spatial context within tissues. However, existing ST platforms suffer from high costs, long turnaround times, low resolution, limited gene coverage, and small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that predicts super-resolution gene expression and automatically annotates cellular-level tissue architecture for large-sized tissues that exceed the capture areas of standard ST platforms. The accuracy of iSCALE were validated by comprehensive evaluations, involving benchmarking experiments, immunohistochemistry staining, and manual annotation by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion associated cellular characteristics that were undetectable by conventional ST experiments. Our results demonstrate iSCALE’s utility in analyzing large-sized tissues with automatic and unbiased tissue annotation, inferring cell type composition, and pinpointing regions of interest for features not discernible through human visual assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca1037b8c19366713347f95c55ebd14edfcd843" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, Jeong Hwan Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Resistance to immune checkpoint inhibitors (ICIs) represents a major barrier to effectively treating patients with NSCLC. Increased tumor heterogeneity has been linked with diminished response to ICIs, but the mechanisms underlying such an effect remain underexplored, in part due to the lack of models for studying the impact of heterogeneity. To address this issue, we have developed a novel murine model of NSCLC that utilizes clonal cell lines to modulate tumor heterogeneity. Specifically, we isolated single cells from a murine NSCLC cell line harboring 3 common driver mutations (Kras G12D Tp53 +/- Lkb1 -/- , or “KPL”). We mutagenized KPL cells with N-methyl-N-nitrosourea in order to increase the tumor mutational burden, and then isolated single cells to derive clonal cell lines. Whole exome sequencing and subsequent phylogenetic analysis of tumor mutations yielded categorization of these clones into 5 clusters. Clones within the same cluster demonstrated comparable in vivo growth kinetics and similar response to anti-PD-1 immunotherapy. These clones exhibited unique tumor immune microenvironments that did not fully correlate with genomic factors such as tumor mutation burden. Notably, heterogeneous polyclonal tumors grew more aggressively than monoclonal tumors, and they exhibited response to anti-PD-1 therapy that differed from the response seen in the component single clones. Tracking of individual clones with fluorescent proteins and mutation tracing illustrated clonal shifts induced by ICI treatment. Our results validate this system as a model of NSCLC that can modify tumor heterogeneity and provide insights into the effects of heterogeneity on ICIs and other immunotherapies.
 Citation Format: Michael Oh, Jensen W Abascal, Linh M Tran, Ramin Salehi-Rad, Raymond J Lim, Camelia Dumitras, Cara Yean, Sunny Xiao, Bitta P Kahangi, William P Crosson, Edgar Perez Reyes, Nalani Coleman, Austin K Rennels, Kostyantyn Krysan, Bin Liu, Steven M Dubinett. A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Stereo-seq is the state-of-the-art technique, which allows capturing of spatial transcriptomic data at sub-cellular resolution and in large scale of view. Stereo-seq FFPE technique is recently launched with the capability to obtain spatially resolved gene expression in FFPE tissue sections. However, the omic information is limited to transcriptome. To delineate spatial molecular hierarchy from transcriptome to phenome, we present the protocol that implements nine-plex immunofluorescence staining with detection of spatial transcriptomics in one FFPE tissue section (short for ST-FFPE-mIF). This protocol details the modified biochemistry procedure and also provides analytical framework to achieve single-cell resolution through nuclear or membrane staining-based cell segmentation. We demonstrated the application of ST-FFPE-mIF in colorectal cancer sample to illustrate how ST-FFPE-mIF facilitates a robust analysis of spatially resolved multi-omic data at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ecff521101f3f5486e05cbeda14ff9f31620ce" target='_blank'>
              ST-FFPE-mIF: Integrating Spatial Transcriptomics and Multiplex Immunofluorescence in Formalin-Fixed Paraffin-Embedded Tissues Using Stereo-seq
              </a>
            </td>
          <td>
            Xue Zhang, Meng Zhang, Yuan Xu, Yangyang Song, Xinfeng Yang, Yanan Wu, Xuelin Zhao, Dongsheng Ran, Xin Liu, Huaqiang Huang, Wenxiao Lei, Hongyan Li, Yongfen Zhang, Feng Xi, Guibo Li, Xing Liu, Luohao Xu, Ao Chen, Sha Liao, Jiajun Zhang
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Systemic sclerosis (SSc) is a prototypical fibrotic disease with high mortality and limited treatment options. Despite advances in single-cell RNA sequencing (scRNA-seq), the comprehensive understanding of cellular heterogeneity and cell-cell interaction within the fibrogenesis microenvironment remains limited. We generated spatially resolved transcriptome maps from healthy and SSc skin and built a scRNA-seq atlas to map the single-cell data to spatial space. This enabled us to identify a fibrotic niche, enriched with fibroblasts and macrophages, which is significantly expanded in SSc and correlated with clinical outcome. We revealed disease-specific cell states of fibroblasts and macrophages, and evaluated their spatial dependency on other cell types. We identified selective expression of ACKR3 in fibroblast progenitors that diminishes with SSc progression, which may serve to regulate CXCL12/CXCR4-mediated macrophage recruitment and fibrotic remodeling. Together, we provided an in-depth description at cellular and spatial levels of fine-tuned regulatory events occurring in SSc, offering spatiotemporal insights. One Sentence Summary Integrated spatial omics provide insight into the cellular and transcriptional landscape in spatially distinct microenvironments, which may drive fibrosis progression in SSc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c9c1b45b9d3f07418bc3c35d63c359a3709dc2" target='_blank'>
              Mapping spatially-resolved transcriptomes in systemic sclerosis
              </a>
            </td>
          <td>
            Zhijian Li, A. R. Rigau, Wenjie Xie, Linlin Huang, Xiaohang Shao, Yi-Nan Li, A. Matei, Wenjing Ye, Hejian Zou, Luca Pinello, J. H. Distler, Rui He, Minrui Liang
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Myeloid cells in the tumor microenvironment (TME) exhibit diverse functions, from pro-tumor to anti-tumor. To elucidate myeloid heterogeneity, we developed a deep-learning model to annotate 20 myeloid subtypes in single-cell data. We identified two macrophage populations, macro_SPP1 and macro_C1QC, which were pan-cancer, enriched in tumors compared to normal tissues, and linked to clinical outcomes. Higher macro_SPP1 abundance correlated with IO non-responders, pre-IO treatment, and lower inflammation, while macro_C1QC showed the opposite trends. We developed and validated gene signatures to quantify these macrophages in bulk RNAseq. Applying these signatures to TCGA and real-world data, we confirmed that macro_SPP1 was associated with worse survival and poor IO response, whereas macro_C1QC was linked to better outcomes. Tumors with chromosome 9p21 deletions had higher macro_SPP1 and fibroblast signatures, suggesting a TME influence on macrophage heterogeneity. Cell-cell communication analysis in single-cell data revealed that fibroblasts sent the strongest signals to macrophages in lung and pancreatic cancers. Fibroblast-derived ligands such as FN1 and collagens exhibited stronger signaling to macro_SPP1 than macro_C1QC, and colocalized with macro_SPP1 spatially. Conversely, macro_SPP1 sent higher total signals to other cells in the TME than macro_C1QC, with SPP1 to fibroblasts and endothelial cells being the most differentiating signal. In summary, we identified two disease-critical macrophage populations: pro-tumor macro_SPP1 and anti-tumor macro_C1QC. We found interactions between macrophages and fibroblasts as key modulators of the TME.
 Citation Format: Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas. Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 CD8+ T cell-tumor cell proximity correlates with immunotherapy response. However, it is unknown whether these cells can be captured as functional clusters from clinical samples. In defined co-cultures, tumor antigen-recognizing T cells outcompeted unmatched T cells in forming heterotypic clusters with tumor cells, showing >10-fold enrichment when comprising only 1% of the total T cell population, which was seen in across cancer cell lines. This prompted us to investigate whether this feature could be used to isolate tumor-reactive CD8+ T cells directly from cancer specimens. By regular and imaging flow cytometry, we could identify heterotypic clusters from 21/21 human melanoma metastases, comprising CD8+ T cells interacting with one or more tumor cells and/or antigen-presenting cells (APCs). These results prompted us to investigate whether these heterotypic T cell clusters could be used to better understand biology and whether they hold therapeutic potential. Regarding biology, we observed that CD8+ T cells from clusters isolated from clinical samples were significantly enriched for tumor-reactive and exhausted gene signatures. Integration with T cell receptor (TCR)-sequencing revealed increased clonality of clustered T cells, indicative of expansion. Whereas single CD8+ T cells showed enrichment of viral antigen-specific signatures, T cells from clusters were enriched for tumor antigen-reactivity signatures. In-depth single cell RNA analyses revealed conjugation of these T cells to tumor cells and different APC types. These were not random events, as both T cell-associated APCs and tumor cells were enriched for specific cell states and associated with differential cell-cell communication. CD8+ T cells carrying identical TCRs showed more exhaustion and co-inhibition when conjugated to APCs than to tumor cells. Regarding therapeutic opportunities, we found that upon rapid expansion (REP), CD8+ TIL from clusters exerted on average 9-fold increased melanoma-killing activity in ex vivo assays, accompanied by enhanced production of cytokines. Upon ACT, T cells from clusters showed superior in vivo killing of matched patient-derived xenografts (PDX), which was associated with more infiltration and activation than for ACT of single T cells. Together, these results demonstrate that tumor-reactive CD8+ T cells are enriched in functional clusters with tumor cells and/or APCs, and that they can be isolated and expanded from clinical samples. Being often excluded during sorting for single-cell studies, these distinct heterotypic CD8+ T cell clusters serve as a valuable source amenable to deciphering functional tumor-immune cell interactions, while they may also be therapeutically explored.
 Citation Format: Daniel S Peeper, Sofia Ibáñez-Molero, Johanna Veldman, Juan Simón-Nieto, Joleen Traets, Austin George, Kelly Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, Winan van Houdt. Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="
 Purpose: The purpose of this study is to determine how T cell engagers (TCEs) impact the localization and phenotype of key T-cell subsets in preclinical solid tumor models. Background: TCEs are a class of immunotherapy that have demonstrated substantial benefit to patients with hematologic cancers and have shown evidence of clinical activity in solid tumor malignancies. Mechanisms of response to TCEs, particularly in solid tumors, are not well understood. Identification of diagnostic strategies to stratify patients that are more likely to respond is expected to enhance the clinical benefit of TCEs. Friedrich et al. (Cancer Cell 2023) found that effector CD8 T cell subsets significantly expanded in TCE responsive but not refractory patients in multiple myeloma using single cell RNA sequencing (scRNAseq). Furthermore, the increased abundance of exhausted T cells at baseline was associated with a worse prognosis. We hypothesize that analogous populations exist in preclinical solid tumor models, which have distinct localization and cellular interactions within the TCE treated tumor microenvironment. Methods: Either hCD3 transgenic C57BL/6J or wildtype FVB mice were implanted with either ID8-LyPD1 or Fo5-HER2 tumors. Mice were grouped at a tumor volume of ∼ 300mm^3 and treated with either 10mpk of LyPD1 TCE, 5mpk of HER2 TCE, or vehicle. scRNAseq data was produced at day 1 and 4 post-treatment according to the 10X gene expression kit protocol and was processed in R using Seurat. Flow cytometry data was generated using commercially available antibodies, acquired on a BD FACSymphony and analyzed using FlowJo. Xenium data was generated using a custom probe panel according to the 10X Xenium protocol and was processed in either Python or R using the Squidpy or Seurat packages. Results: In the ID8-LyPD1 TCE model, we identified proliferative effector (CD8.pe) and terminally differentiated (CD8.Tex) CD8 populations by scRNAseq. The abundance of CD8.pe increased 3-fold and CD8.Tex expanded 2-fold upon TCE treatment. We then validated the expansion of these CD8 subsets by flow cytometry. Notably, increases in the number of CD8.pe per gram of tumor significantly correlated with reductions in tumor volume. Due to the impact of TCE treatment on these CD8 subsets and the importance of CD8.pe in tumor outcomes, we profiled the localization and cellular interactions of CD8s in preclinical solid tumors. Cell-cell communication inference analysis suggested that CD8s were more frequently targets of cellular interactions in TCE treated tumors. Utilizing Xenium spatial transcriptomics we verified that CD8.pe and CD8.Tex were more abundant in TCE treated samples and that their interactions with other key cell subsets were enhanced by TCE treatment. Conclusions: In summary we found that key CD8 subsets were expanded by TCE treatment in preclinical solid tumor models, that the abundance of CD8.pe correlates with reduced tumor burden, and that TCE treatment promotes cellular interactions among CD8s and other key cell subsets in the tumor microenvironment.
 Citation Format: Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa McGinnis, Kevin Walsh, Weilan Ye. T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr PR002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Motivation Spatially resolved omics technologies are enhancing our understanding of tissues architecture. Despite major technological improvements, gaining in spatial resolution becomes experimentally expensive, while generating spatial landscapes at moderate resolution combined with computational methods for depixelating data represent a cost-effective strategy allowing to enlarge the number of experiments to be performed. Results We have developed a computational strategy able to gain several-folds of resolution by inferring pseudo-interstitial pixels from their closest neighbors. This strategy has been validated in the context of public spatial transcriptomics data issued from melanoma, and human brain cortex tissue sections, by improving the identification of distinct tissue substructures. Furthermore, this methodology has been used for enhancing the resolution of consecutive sections collected from human brain organoids, as a way to demonstrate that a moderate resolution technology, combined with spatial depixelation processing allows to properly discern molecular tissue structures even in small tissues. Contact mmendoza@genoscope.cns.fr">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83223293873480f4821a0f7502d4baa16a5d8116" target='_blank'>
              Enhancing spatial omics resolution by pseudo-interstitial pixels inference
              </a>
            </td>
          <td>
            M. Mendoza-Parra, Maximilien Duvina, Ariel Galindo-Albarrán
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary Cancer stem cells (CSCs) are pivotal in tumor growth, recurrence, and resistance to standard therapies. This review delves into CSCs’ ability to self-renew and differentiate, fueling tumor heterogeneity and metastasis. It highlights critical signaling pathways like Wnt, Notch, and Hedgehog, which are essential for targeting CSCs. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, promise to enhance CSC research and management. The review also explores the complex interaction between CSCs and the tumor microenvironment (TME), which influences therapeutic effectiveness. Addressing CSC-driven resistance, the article proposes advanced strategies for overcoming these challenges. It emphasizes the role of artificial intelligence in designing personalized, CSC-targeted therapies, ushering in a new era for precision medicine in oncology. This comprehensive approach aims to improve cancer treatment outcomes through innovative technologies and inter-disciplinary collaboration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="BACKGROUND
Peripheral nerve sheath tumors (PNSTs) encompass entities with different cellular differentiation and degrees of malignancy. Spatial heterogeneity complicates diagnosis and grading of PNSTs in some cases. In malignant PNST (MPNST) for example, single cell sequencing data has shown dissimilar differentiation states of tumor cells. Here, we aimed at determining the spatial and biological heterogeneity of PNSTs.


METHODS
We performed spatial transcriptomics on formalin-fixed paraffin-embedded diseased peripheral nerve tissue. We used spatial clustering and weighted correlation network analysis to construct niche-similarity networks and gene expression modules. We determined differential expression in primary pathologies, analysed pathways to investigate the biological significance of identified meta-signatures, integrated the transcriptional data with histological features and existing single cell data, and validated expression data by immunohistochemistry.


RESULTS
We identified distinct transcriptional signatures differentiating PNSTs. Immune cell infiltration, APOD and perineurial fibroblast marker expression highlighted the neurofibroma component of hybrid PNSTs (HPNSTs). While APOD was evenly expressed in neurofibromatous tumor tissue in both, HPNST and pure neurofibromas, perineurial fibroblast markers were evenly expressed in HPNST, but restricted to the periphery in plexiform neurofibromas. Furthermore, we provide a spatial cellular differentiation map for MPNST, locating Schwann cell precursors, neural crest-like cells and those with mesenchymal transition.


CONCLUSIONS
This pilot study shows that applying spatial transcriptomics to PNSTs provides important insight into their biology. It helps establishing new markers and provides spatial information about cellular composition and distribution of cellular differentiation states. By integrating morphological and high-dimensional molecular data it can improve PNSTs classification in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2233f7b03c50daacd9edb143f0325a91952a1c6" target='_blank'>
              Spatially resolved transcriptomics of benign and malignant peripheral nerve sheath tumors.
              </a>
            </td>
          <td>
            J. Bremer, Pamela Franco, J. A. Menstell, S. Tey, K. Zajt, Klimentina Popzhelyazkova, Kay Nolte, Jürgen Schlegel, Maria Teresa Pedro, Anja Osterloh, D. Delev, M. Hohenhaus, Christoph Scholz, O. Schnell, J. Beck, Joachim Weis, D. Heiland
          </td>
          <td>2025-01-23</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7523e1202c6e36a061489ea3b1a6d30d096a1b89" target='_blank'>
              Deciphering the generation of heterogeneity in esophageal squamous cell carcinoma metastasis via single-cell multiomics analysis
              </a>
            </td>
          <td>
            Kaiwen Sheng, Jun Chen, Ruitang Xu, Haoqiang Sun, Ran Liu, Yongjie Wang, Wenwen Xu, Jiao Guo, Miao Zhang, Shuai Liu, Juan Lei, Yawen Sun, Yang Jia, Dianhao Guo
          </td>
          <td>2025-02-04</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="The epidermal basal layer consists of two distinct progenitor populations contributing differentially to skin homeostasis or wound healing. However, their resilience to oncogenic insults is unknown. Here, we developed mouse models allowing lineage tracing and oncogene expression in both populations. Our findings support the existence of two long-lived and functionally distinct epidermal progenitors and define them as tumour-primed and tumour-resistant. The tumour-primed progenitors drive cutaneous squamous cell carcinoma (cSCC), within days from oncogenic expression resembling the rapid onset of BRAF-inhibitor-driven cSCC. The tumour-resistant progenitors transform gradually, requiring SOX2 activation. Once transformed, they share a transcriptomic profile revealing molecular hallmarks of epidermal transformation independently of the cell-of-origin or the oncogenic driving mutations. Importantly, SOX2 is expressed in a subset of cSCC patients hinting at its cell-of-origin. These highlight the need for an in-depth understanding of the cell-specific vulnerabilities that apply in each tissue to open new avenues for rational treatment. STATEMENT OF SIGNIFICANCE Using in vivo models, we define tumour-primed and tumour-resistant epidermal progenitors and reveal SOX2-driven reprogramming as a driver of tumorigenesis in resistant cells. SOX2 expression in a subset of cSCC patients links to cell-of-origin, uncovering a cell-specific vulnerability that advances our understanding of cancer initiation and progression in epithelial tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c6ee68b135f15e99b33463b50a273c96e7d816" target='_blank'>
              Epidermal tumour-resistant progenitors require SOX2-driven developmental reprogramming to drive tumorigenesis
              </a>
            </td>
          <td>
            Patricia P. Centeno, Christopher Chester, Catriona A. Ford, Rachel A Ridgway, P. Cammareri, Thomas Jamieson, Richard Marais, A. Campbell, Owen J. Sansom
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Tumour immune macroenvironment is comprised of tumour and surrounding organs responding to tumourigenesis and immunotherapy. The lack of comprehensive analytical methods hinders its application for prediction of survival and treatment response in colorectal cancer (CRC) patients. Methods Cytometry by time‐of‐flight (CyTOF) and RNA‐seq was applied to characterise immune cell heterogeneity in a discovery cohort including tumour, blood and intestinal architecture comprising epithelium, lamina propria, submucosa, muscularis propria of normal bowel and tumour–adjacent bowel tissues. Immunoprofiling was also validated by a validation cohort using single‐cell RNA sequencing, spatial transcription, CyTOF and multiplex immunofluorescent staining. Results Based on cell phenotype and transcription, we identify distinct immunotypes in the CRC macroenvironment including blood, tumour and different intestinal architecture, showing disturbed immune cell compositions, increasing expression of immunosuppressive markers and cell–cell interactions contributing to immunosuppressive regulation. Furthermore, we evaluate immune macroenvironment influencing factors including tertiary lymphoid structures (TLSs), consensus molecular subtypes (CMSs) and immune checkpoint inhibitors (ICIs). TLS presence fuels anti‐tumour immunity by promoting CD8+ T cell infiltration and altering activation or suppression of T cell systematically. TLS presence correlates with patient survival, intrinsic CMS and therapeutic efficacy of ICI. PD‐1 and CD69 expressed in effector memory CD8+ T cells from blood can predict TLS presence in the CRC macroenvironment, serving as potential biomarkers for stratifying CRC patients into immunotherapy. Conclusions Our findings provide insights into the CRC immune macroenvironment, highlighting immune cell suppression and activation in tumourigenesis. Our study illustrates the potential utility of blood for predicting immunotherapy response. Key points Distinct immunotypes are identified in the CRC macroenvironment. TLS and immunotherapy exert influence on the immune macroenvironment. TLS presence correlates with patient survival, CMS and therapeutic efficacy of ICI. PD‐1 and CD69 expressed in CD8+ Tem from blood can predict TLS presence in the CRC macroenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85985e342fbc41d36820a7abb928738245e5ca0b" target='_blank'>
              Immune profiling of the macroenvironment in colorectal cancer unveils systemic dysfunction and plasticity of immune cells
              </a>
            </td>
          <td>
            Haoxian Ke, Peisi Li, Zhihao Li, Xian Zeng, Chi Zhang, Shuzhen Luo, Xiaofang Chen, Xinlan Zhou, Shichen Dong, Shaopeng Chen, Junfeng Huang, Ming Yuan, Runfeng Yu, Shubiao Ye, Tuo Hu, Zhonghui Tang, Dongbin Liu, Kui Wu, Xianrui Wu, Ping Lan
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Quiescence, a reversible state of cell-cycle arrest, is an adaptive feature of many adult tissue stem cells, including those in the adult brain. In gliomas, brain tumour stem cells that reside in a quiescent state preferentially survive chemotherapy and radiotherapy, highlighting their critical role in therapy resistance and disease progression. To date, it remains unclear whether the molecular programs governing these states are functionally conserved between neural stem cells and brain tumour stem cells. Here, we establish novel in vitro models to study quiescence and find that glioma stem cells are markedly more resistant to entering quiescence than neural stem cells, suggesting that glioma stem cell quiescence more closely resembles a slow-cycling phenotype or shallow quiescence. Nonetheless, direct comparison of quiescent neural stem cells and quiescent/slow-cycling glioma stem cells, as they transition towards proliferation, reveals conserved gene expression trajectories, indicating shared molecular mechanisms. Furthermore, we find that pathways influencing quiescence in neural stem cells exert similar effects in glioma stem cells, underscoring the functional parallels between these populations. Finally, we identify that inhibition of TGF-β signalling might provide an avenue to improve current standard-of-care treatments by targeting quiescent glioma stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfaba977c1ae379d70e521eff155f6c1e15b6182" target='_blank'>
              Shared molecular regulation of quiescence in neural and glioma stem cells reveals therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Chandra Choudhury, Matthew Singleton, Stephanie Brauer, Dana Friess, Jessica Hart, N. Skarne, Karthik Pullela, Likai Mao, Bryan W Day, Lachlan Harris
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a403bcbf00350f8809a4da6d588cc795243cc7f5" target='_blank'>
              Single-cell analysis reveals transcriptomic features and therapeutic targets in primary pulmonary lymphoepithelioma-like carcinoma
              </a>
            </td>
          <td>
            Binghua Tan, Ke Xu, Y. Lyu, Yicheng Liang, Ruihao Liang, Kai Lei, Jialu Liang, Jing Huang, Kefeng Wang, Duoguang Wu, Wenjian Wang, Xueting Hu, Kexi Wang, Minghui Wang, Huayue Lin
          </td>
          <td>2025-03-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells (VSMCs), and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and VSMCs in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process. Methods Here, we utilized CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development. Results We identified four distinct fibroblast subpopulations, including a myofibroblast population closely resembling VSMC-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion. Conclusions Our multi-omic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcefa069d1b01c8875c90b650f37b9a970f0c89d" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47aaf83b6567ca271ef456f540a98ac5c9738676" target='_blank'>
              Comparative analysis of pediatric SHH medulloblastoma DAOY spheres and adherent monolayers: implications for medulloblastoma research
              </a>
            </td>
          <td>
            Jana Klementová, Šárka Jarošová, Irina Danilová, Markéta Farníková, Josef Novotný, M. Davidkova, Martina Zíková
          </td>
          <td>2025-01-22</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Allograft rejection following solid-organ transplantation is a major cause of graft dysfunction and mortality. Current approaches to diagnosis rely on histology, which exhibits wide diagnostic variability and lacks access to molecular phenotypes that may stratify therapeutic response. Here, we leverage image-based spatial transcriptomics at sub-cellular resolution in longitudinal human cardiac biopsies to characterize transcriptional heterogeneity in 62 adult and pediatric heart transplant (HT) recipients during and following histologically-diagnosed rejection. Across 28 cell types, we identified significant differences in abundance in CD4+ and CD8+ T cells, fibroblasts, and endothelial cells across different biological classes of rejection (cellular, mixed, antibody-mediated). We observed a broad overlap in cellular transcriptional states across histologic rejection severity and biological class and significant heterogeneity within rejection severity grades that would qualify for immunomodulatory treatment. Individuals who had resolved rejection after therapy had a distinct transcriptomic profile relative to those with persistent rejection, including 216 genes across 6 cell types along pathways of inflammation, IL6-JAK-STAT3 signaling, IFNα/IFNγ response, and TNFα signaling. Spatial transcriptomics also identified genes linked to long-term prognostic outcomes post-HT. These results underscore importance of subtyping immunologic states during rejection to stratify immune-cardiac interactions following HT that are therapeutically relevant to short- and long-term rejection-related outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3418175c8e687d05cd4cb3259c7f39ab58b56c25" target='_blank'>
              Dynamic responses to rejection in the transplanted human heart revealed through spatial transcriptomics
              </a>
            </td>
          <td>
            K. Amancherla, Angela M. Taravella Oill, Xavier Bledsoe, A. L. Williams, Nelson Chow, Shilin Zhao, Quanhu Sheng, David W. Bearl, Robert D. Hoffman, J. Menachem, Hasan K. Siddiqi, Douglas M. Brinkley, E. D. Mee, Niran Hadad, Vineet Agrawal, Jeffrey Schmeckpepper, A. Rali, Stacy Tsai, E. Farber-Eger, Quinn S Wells, Jane E. Freedman, Nathan R. Tucker, K. Schlendorf, Eric R. Gamazon, Ravi V. Shah, N. Banovich
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Immune cells exhibit functional heterogeneity beyond what is resolved by classical definitions of subpopulations based on cell surface expression of receptors. To develop efficient and personalized cell-based immunotherapies, we need to resolve this heterogeneity and understand the underlying parameters that dictate cellular responses to specific target cells. For this, new methods are required that can identify and harvest immune cells with specific functions, e.g., high cytotoxic potential, to form clonally expanded cells or to assess molecular or genetic signatures. In this study, we evaluate a system for non-destructive, live cell picking and release in deep, high-aspect ratio microwells and test it for isolation of individual natural killer (NK) cells. We assess its performance at retrieving and releasing beads from microwells and demonstrate its potential for single NK cell isolation with intact viability. We also implement a semi-automated workflow for functional single-cell screening of NK cell behavior in microwell arrays followed by single-cell identification and isolation, demonstrating the potential for functional screening and isolation of serial killing immune cells. Our evaluation concludes that this cell isolation system, in combination with microwell arrays, offers opportunities for improved understanding of NK cell biology with applications towards cell therapy. However, its limited throughput hinders large-scale applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79619eb3501acae9caddc99dc83ce718f2ee3c" target='_blank'>
              Isolation of individual natural killer cells from deep, high-aspect ratio microwell arrays - an evaluative study
              </a>
            </td>
          <td>
            Quentin Verron, Niklas Sandström, Hanna van Ooijen, K. Guldevall, K. Olofsson, T. Frisk, Björn Önfelt
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This study explores the potential of canine patient-derived organoids (PDOs) in identifying mammary tumor-specific weaknesses by comparing tumor cells with matching normal mammary PDOs from the same patient. For this purpose, two customized CRISPR/Cas9 screens were performed to identify genes that are essential for tumor cells to survive but dispensable for normal mammary epithelial cells. CDK2 was subsequently identified as an important vulnerability in PDOs derived from a canine mammary tumor patient, which can be targeted by an existing inhibitor, PF3600. Although we only used tumors and normal PDOs from a single patient, our approach exemplifies how to discover novel, more individualized anti-cancer treatments for dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a341795bab91ddb8ebd61b09b4f9f74bbe497648" target='_blank'>
              Individualized Pooled CRISPR/Cas9 Screenings Identify CDK2 as a Druggable Vulnerability in a Canine Mammary Carcinoma Patient
              </a>
            </td>
          <td>
            Marine Inglebert, Martina Dettwiler, Chang He, Enni Markkanen, Lennart Opitz, A. Naguleswaran, Sven Rottenberg
          </td>
          <td>2025-02-01</td>
          <td>Veterinary Sciences</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Therapeutic drug treatments of solid tumors are often undermined by various resistance mechanisms. Identification of drug-resistance phenotypes at the single cell level is challenging because conventional molecular methods are cell-destructive, labor-intensive, and cost-prohibitive. To overcome these challenges, we developed an orthogonal approach to drug-resistance phenotyping, through the use of deep-learning-driven morphology analysis of single, high resolution cell images. Specifically, we trained deep learning-based drug resistance classifiers using cell images from 5 different cell lines that were rendered resistant to 5 different therapeutic agents, using a foundation model framework. With high accuracy, the classifier correctly predicted naive or resistance phenotypes across all cancer types and across all the therapeutic agent types (chemotherapeutic, targeted) tested. These results showed that morphology can capture complex phenotype information in the context of drug treatment. To demonstrate the potential clinical utility of the drug resistance classifier, it was applied to a dissociated tumor biopsy and the resulting phenotype predictions were in close concordance with scRNASeq analysis of the biopsy. Our study highlights the potential of deep-learning-driven morphology analysis to provide complex phenotype information, and ultimately shape oncology drug treatment strategies at the patient-level in a clinical context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e32bdcccdadfa02831a9867b05cc627a1045f4" target='_blank'>
              Deep learning-driven morphology analysis enables label-free classification of therapeutic agentnaive versus resistant cancer cells
              </a>
            </td>
          <td>
            Cyril Y Ramathal, K. Saini, Zhouyang Lian, Christian Corona, T. Pham, Ryan Carelli, Stephane C. Boutet, Manisha Ray, Sunantha Sethuraman, Vivian Prindle, Evelyn Lattmann, Maria-Grazia Molvetti, A. Dzung, Mitchell P. Levesque, Matt Barnes, Andreja Jovic
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objective
Prostate cancer (PCa) is a complex disease characterized by diverse cellular ecosystems within the tumor microenvironment (TME) and high tumor heterogeneity, which challenges clinically stratified management and reinforces the need for novel strategies to fight against castration-resistant PCa (CRPC).


Methods
We performed single-cell RNA sequencing (scRNA-seq) on 10 untreated primary PCa tissues and integrated public scRNA-seq resources from three normal prostate tissues, two untreated primary PCa tissues, and six CRPC tumors to portray a comprehensive cellular and molecular interaction atlas of PCa. We further integrated the single-cell and bulk transcriptomes of PCa to establish a molecular classification system.


Results
scRNA-seq profiles revealed substantial inter- and intra-tumoral heterogeneity across different cell subpopulations in untreated PCa and CRPC tumors. In the malignant epithelial reservoir, cells evolved along decoupled paths in treatment-naive PCa and CRPC tumors, and distinct transcriptional reprogramming processes were activated, highlighting anti-androgen therapy-induced lineage plasticity. Based on the specifically expressed markers of the epithelial subpopulations, we conducted unsupervised clustering analysis in The Cancer Genome Atlas prostate adenocarcinoma (TCGA-PRAD) cohort and identified three molecularly and clinically distinct subtypes. The C1 subtype, characterized by high enrichment of CRPC-enriched epithelial cells, had a high risk of rapid development of anti-androgen resistance and might require active surveillance and additional promising intervention treatments, such as integrin A3 (ITGA3) + integrin B1 (ITGB1) inhibition. The C2 subtype resembled the immune-modulated subtype that was most likely to benefit from anti-LAG3 immunotherapy. The C3 subtype had a favorable prognosis.


Conclusions
Our study provides a comprehensive and high-resolution landscape of the intricate architecture of the PCa TME, and our trichotomic molecular taxonomy could help facilitate precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96305f2e853a0fad4ae17b8dba8c38f0edd28b14" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomes uncovers clinically relevant molecular subtypes in human prostate cancer.
              </a>
            </td>
          <td>
            Tao Ding, Lina He, Guowen Lin, Lei Xu, Yanjun Zhu, Xinan Wang, Xuefei Liu, Jianming Guo, Fanghong Lei, Zhixiang Zuo, Jianghua Zheng
          </td>
          <td>2025-01-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge.
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Simple Summary Multiple myeloma (MM) is a blood cancer of bone marrow that is difficult to diagnose and treat because it changes over time and has complex interactions with the body’s immune system. This review explores how single-cell technologies help scientists understand multiple myeloma more comprehensively by identifying hidden tumor clones, tracking how the disease evolves, and discerning why some treatments stop working. Researchers have found that certain genetic changes, like mutations or deletion of TP53, loss of chromosome 13, and gain of chromosome 1, make the cancer more resistant to treatment, and that the immune system often fails to fight off MM because there is a suppressive environment around the tumor. New treatments, like Chimeric Antigen Receptor T cell (CAR-T) cell therapy and Bispecific T cell engagers (BiTE), take advantage of the immune microenvironment and are showing promise in targeting myeloma cell directly. However, they are not always effective and can have adverse effects. By studying single-cell resolution data to identify new vulnerabilities in myeloma cells and pathways related to resistance that can be different within the cells in a single patient, new treatment regimens can be identified and tailored for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, Travis S. Johnson
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183356cbf79c71bfdd985789fe82d5c12c548853" target='_blank'>
              Gemistocytic tumor cells programmed for glial scarring characterize T cell confinement in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, Wies R Vallentgoed, R. Head, Rosa Luning, Thierry van den Bosch, Bart A Westerman, Pieter Wesseling, J. Joyce, P. French, R. Debets
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Motivation Despite significant advances in spatial transcriptomics, the analysis of formalin-fixed paraffin-embedded (FFPE) tissues, which constitute most clinically available samples, remains challenging. Additionally, capturing both coding and noncoding RNAs in a spatial context poses significant challenges. We recently introduced Patho-DBiT, a technology designed to address these unmet needs. However, the marked differences between Patho-DBiT and existing spatial transcriptomics protocols necessitate specialized computational tools for comprehensive whole-transcriptome analysis in FFPE samples. Results Here, we present ASTRO, an automated pipeline developed to process spatial transcriptomics data. In addition to supporting standard datasets, ASTRO is optimized for whole-transcriptome analyses of FFPE samples, enabling the detection of various RNA species, including non-coding RNAs such as miRNAs. To compensate for the reduced RNA quality in FFPE tissues, ASTRO incorporates a specialized filtering step and optimizes spatial barcode calling, increasing the mapping rate. These optimizations allow ASTRO to spatially quantify coding and non-coding RNA species in the entire transcriptome and achieve robust performance in FFPE samples. Availability Codes are available at GitHub (https://github.com/gersteinlab/ASTRO).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d569a6b53e774b129f0d53a5ac8f2383815ea030" target='_blank'>
              ASTRO: Automated Spatial Whole-Transcriptome RNA-Expression Workflow
              </a>
            </td>
          <td>
            Dingyao Zhang, Zhiyuan Chu, Yiran Huo, Zhiliang Bai, Rong Fan, Jun Lu, Marc H. Gerstein
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell transcriptomics has revolutionized our understanding of neurodevelopmental cell identities, yet, predicting a cell type’s developmental state from its transcriptome remains a challenge. We perform a meta-analysis of developing human brain datasets comprising over 2.8 million cells, identifying both tissue-level and cell-autonomous predictors of developmental age. While tissue composition predicts age within individual studies, it fails to generalize, whereas specific cell type proportions reliably track developmental time across datasets. Training regularized regression models to infer cell-autonomous maturation, we find that a cell type-agnostic model achieves the highest accuracy (error = 2.6 weeks), robustly capturing developmental dynamics across diverse cell types and datasets. This model generalizes to human neural organoids, accurately predicting normal developmental trajectories (R = 0.91) and disease-induced shifts in vitro. Furthermore, it extends to the developing mouse brain, revealing an accelerated developmental tempo relative to humans. Our work provides a unified framework for comparing neurodevelopment across contexts, model systems, and species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd7e20b6112e2c0e549d107034486f23f4f7693" target='_blank'>
              Cell Type-Agnostic Transcriptomic Signatures Enable Uniform Comparisons of Neurodevelopment
              </a>
            </td>
          <td>
            Sridevi Venkatesan, Jonathan M. Werner, Yun Li, Jesse Gillis
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Spatial transcriptomics (ST) is revolutionizing our understanding of tumor heterogeneity by providing high-resolution, location-specific gene expression mapping across the tumor microenvironment (TME). Despite its transformative potential, ST’s high cost limits its widespread clinical application, particularly for large-scale biomarker discovery. Methods: To address these challenges, we developed Path2Space, a deep learning model trained on ST data, to predict spatial gene expression directly from histopathology slides, offering an accessible alternative to experimental ST assays. First, Path2Space is validated using three independent spatial transcriptomics (ST) breast cancer cohorts, demonstrating robust and consistent gene expression prediction across these datasets. Second, the model’s predictions allow for detailed TME characterization, including the accurate inference of cell-type abundances and immune cell distributions within spatial domains. Thirdly and most importantly, we applied Path2Space to the TCGA breast cancer cohort to assess the prognostic potential of immune and TME spatial patterns. Results: Path2Space achieves unprecedented accuracy and coverage in robustly predicting the spatial expression of 4,500 genes across three independent spatial transcriptomics (ST) cohorts, markedly outperforming current methods by a factor of 10. Applied to the TCGA breast cancer cohort, we identify three patient subsets with distinct immune tumor microenvironments (TME) having distinct survival outcomes. The immune TME of the poor-prognosis subset has dense cancer cell populations with low immune infiltration and marked inactivation of specific key immune interactions. In two trastuzumab-treated breast cancer cohorts, Path2Space identifies spatial domains strongly associated with treatment response, with high tumor-infiltrating lymphocytes (TIL) presence, activation of immune pathways, and robust immune-driven cell-cell interactions, enabling the prediction of patient response more accurately than that based on the measured gene expression. Conclusions: Path2Space represents a groundbreaking approach in spatially mapping the tumor microenvironment directly from histopathology slides, paving the way for the study of the immune TME of different cancer types on an unprecedented scale. By enabling the cost-effective inference of the TME, Path2Space holds potential to transform cancer care across diverse healthcare settings, facilitating the development of targeted treatments and improving patient survival globally.
 Citation Format: Roshan Lodha, Eldad D Shulman, Emma M Campagnolo, Thomas Cantore, Tom Hu, MacLean Nasrallah, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr B105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor cells with organ-specific metastasis traits arise in primary lesions with substantial variations of local niche mechanics owing to intratumoral heterogeneity. However, the roles of mechanically heterogeneous primary tumor microenvironment in metastatic organotropism remain an enigma. This study reports that persistent priming in soft but not stiff niches that mimic primary tumor mechanical heterogeneity induces transcriptional reprogramming reminiscent of neuron and promotes the acquisition of brain metastatic potential. Soft-primed cells generate brain metastases in vivo through enhanced transendothelial migration across blood-brain barrier and brain colonization, which is further supported by the findings that tumor cells residing in local soft niches of primary xenografts exhibit brain metastatic tropism. Mechanistically, soft niches suppress cytoskeleton-nucleus–mediated mechanotransduction, which promotes histone deacetylase 3 activity. Inhibiting histone deacetylase 3 abolishes niche softness-induced brain metastatic ability. Collectively, this study uncovers a previously unappreciated role of local niche softness within primary tumors in brain metastasis, highlighting the significance of primary tumor mechanical heterogeneity in metastatic organotropism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4555c3459d7f13d0f05bcb5e93723f35a50f25" target='_blank'>
              Local soft niches in mechanically heterogeneous primary tumors promote brain metastasis via mechanotransduction-mediated HDAC3 activity
              </a>
            </td>
          <td>
            Kai Tang, Yufan Zheng, Guanshuo Hu, Ying Xin, Keming Li, Cunyu Zhang, Xi Chen, Bai Zhang, Xueyi Li, Bing Hu, Qiong Jia, Yong-ping Zheng, Mo Yang, Youhua Tan
          </td>
          <td>2025-02-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell spatial transcriptomics (scST) methodologies allow, in combination with histological stainings, an unprecedented view on disease progression. To comprehensively analyze scST data, bioinformatic analysis frameworks need to integrate the diverse set of data modalities and, just as importantly, enable the joint analysis of multiple datasets from clinical or experimental cohorts together with its corresponding metadata. Here, we present the InSituPy framework to comprehensively analyze single-cell spatial transcriptomic data from a multi-sample level down to the cellular and subcellular level. The framework contains analysis workflows for the integration of image data as well as pathological and biological expert knowledge. Increasing the accessibility of the data for non-bioinformaticians, the framework opens new ways of generating hypotheses, especially in the context of translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f6840421639f0eeeffb398c6fad844f8d444a6" target='_blank'>
              InSituPy – A framework for histology-guided, multi-sample analysis of single-cell spatial transcriptomics data
              </a>
            </td>
          <td>
            Johannes Wirth, Anna Chernysheva, Birthe Lemke, Isabel Giray, Aitana Egea Lavandera, K. Steiger
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins.


METHODS
We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES-dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP-sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro.


RESULTS
Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft.


CONCLUSIONS
Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce67e86c3ed1eb1595b4d5bed2ad026b261bd59" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets.
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, Alaa Narch, F. Zammarchi, P. V. van Berkel, C. V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2025-01-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabdeb7c0a8725d1b43bd88dd9faad675928ee91" target='_blank'>
              Extraction of a stromal metastatic gene signature in breast cancer via spatial profiling
              </a>
            </td>
          <td>
            G. Bertolazzi, Valeria Cancila, Davide Vacca, Beatrice Belmonte, D. Lecis, Parsa Sirati Moghaddam, A. Di Napoli, M. P. Colombo, G. Pruneri, Giannino Del Sal, Giorgio Scita, Matteo Fassan, A. Vecchione, Silvio Bicciato, Claudio Tripodo
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Disruptions in uterine tissue function contribute to disorders such as endometriosis, adenomyosis, endometrial cancer, and fibroids, which all significantly impact health and fertility. Advances in transcriptomics, particularly single-cell RNA sequencing, have revolutionized uterine biological research by revealing the cellular heterogeneity and molecular mechanisms underlying disease states. Single-cell RNA sequencing and spatial transcriptomics have mapped endometrial and myometrial cellular landscapes, which helped to identify critical cell types, signaling pathways, and phase-specific dynamics. Said transcriptomic technologies also identified stromal and immune cell dysfunctions, such as fibroblast-to-myofibroblast transitions and impaired macrophage activity, which drive fibrosis, chronic inflammation, and lesion persistence in endometriosis. For endometrial cancer, scRNA-seq uncovered tumor microenvironmental complexities, identifying cancer-associated fibroblast subtypes and immune cell profiles contributing to progression and therapeutic resistance. Similarly, studies on adenomyosis highlighted disrupted signaling pathways, including Wnt and VEGF, and novel progenitor cell populations linked to tissue invasion and neuroinflammation, while single-cell approaches characterized smooth muscle and fibroblast subpopulations in uterine fibroids, elucidating their roles in extracellular matrix remodeling and signaling pathways like ERK and mTOR. Despite challenges such as scalability and reproducibility, single-cell transcriptomic approaches may have potential applications in biomarker discovery, therapeutic target identification, and personalized medicine in gynecological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d46c8d721f3ec856f042d24cd830de014f3393b" target='_blank'>
              Mapping Human Uterine Disorders Through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Sandra Boldu-Fernández, Carolina Lliberos, Carlos Simón, A. Mas
          </td>
          <td>2025-01-21</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genetic defects in NOX2 can cause chronic granulomatous disease (CGD), characterized by increased susceptibility to infections and pronounced inflammatory responses that lead to granuloma formation. We developed a CGD model using Ncf2-/- mice through controlled environmental exposure. Unlike in specific pathogen-free environment, these mice spontaneously developed pulmonary granulomas under clean-grade conditions. Employing single-cell RNA sequencing, we observed an expansion of neutrophils and monocyte-derived macrophages (MDMs) within the lung tissue, identifying pro-inflammatory NOS2high-neutrophils and a distinct MDMs subset marked by NOS2 and ARG1. Spatial transcriptomics demonstrated that NOS2high-neutrophils were located at the core area, while the MDMs subset was positioned at the periphery, facilitating extracellular matrix remodeling. Pharmacological targeting of MIF, deletion of the pro-survival gene Morrbid, and elimination of Il1r1 all suppressed granuloma formation by mitigating inflammation. This study underscores how environmental control can establish a natural CGD model, elucidates the mechanisms of granuloma formation, and develops potent therapeutic interventions. Highlights Through installing a mutation in a NOX2 subunit gene Ncf2 and changing the husbandry environment, we generated a natural CGD mouse model with progressive pulmonary granuloma formation. NOS2high neutrophils located in the core region of CGD lung granulomas, exhibit a pro-inflammatory transcriptional profile. Monocyte-derived macrophages marked by Nos2 and Arg1 are located at the periphery of granulomas and exhibit a profibrotic transcriptional profile. The transcriptomic studies assist the development of three effective perturbations for treating the rare disease CGD via targeting Il1r1, Morrbid, and Mif respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ab17761f537ad25af8c4de8bd9c011f1caec1f" target='_blank'>
              Single-cell and spatial transcriptomics reveal the pathogenesis of chronic granulomatous disease in a natural model
              </a>
            </td>
          <td>
            Hanzhi Yu, Guorong Zhang, Yunxi Ma, Jiayu Ding, Jingjing Liu, Zhiliang Zhou, Shaozhuo Jiao, Ge Dong, Zhigang Cai
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatial transcriptomics has transformed our understanding of gene expression by preserving the spatial context within tissues. This review focuses on the application of spatial transcriptomics in human cardiac tissues, exploring current technologies with a focus on commercially available platforms. We also highlight key studies utilizing spatial transcriptomics to investigate cardiac development, electro-anatomy, immunology, and ischemic heart disease. These studies demonstrate how spatial transcriptomics can be used in conjunction with other omics technologies to provide a more comprehensive picture of human health and disease. Despite its transformative potential, spatial transcriptomics comes with several challenges that limit its widespread adoption and broader application. By addressing these limitations and fostering interdisciplinary collaboration, spatial transcriptomics has the potential to become an essential tool in cardiovascular research. We hope this review serves as a practical guide for researchers interested in adopting spatial transcriptomics, particularly those with limited prior experience, by providing insights into current technologies, applications, and considerations for successful implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ad32890339541586531b7c09fb574e34d7781c" target='_blank'>
              Spatial Transcriptomics in Human Cardiac Tissue
              </a>
            </td>
          <td>
            Quynh Nguyen, Lin Wei Tung, Bruce Lin, Raam Sivakumar, F. Sar, G. Singhera, Ying Wang, Jeremy Parker, Stephane Le Bihan, Amrit Singh, Fabio M V Rossi, Colin Collins, Jamil Bashir, Zachary Laksman
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de4d7796ed3b7e2389f61c7edf7191dc271c5b5" target='_blank'>
              State-of-the-Art Liver Cancer Organoids: Modeling Cancer Stem Cell Heterogeneity for Personalized Treatment.
              </a>
            </td>
          <td>
            Julien Giron-Michel, Maël Padelli, Estelle Oberlin, Hind Guenou, Jean-Charles Duclos-Vallée
          </td>
          <td>2025-01-18</td>
          <td>BioDrugs : clinical immunotherapeutics, biopharmaceuticals and gene therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91820cda6d428b7441857912e71a4c3a105594b5" target='_blank'>
              Density and entropy of immune cells within the tumor microenvironment of primary tumors and matched brain metastases
              </a>
            </td>
          <td>
            M. Kleinberger, D. Cifci, C. Paiato, E. Tomasich, M. Mair, A. Steindl, Zoltán Spiró, Z. I. Carrero, L. Berchtold, J. Hainfellner, L. Müllauer, G. Heller, M. Preusser, Jakob N Kather, A. Berghoff
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Motivation Immune cells undergo cytokine-driven polarization in response to diverse stimuli, altering their transcriptional profiles and functional states. This dynamic process is central to immune responses in health and diseases, yet a systematic approach to assess cytokine-driven polarization in single-cell RNA sequencing data has been lacking. Results To address this gap, we developed single-cell unified polarization assessment (Scupa), the first computational method for comprehensive immune cell polarization assessment. Scupa leverages data from the Immune Dictionary, which characterizes cytokine-driven polarization states across 14 immune cell types. By integrating cell embeddings from the single-cell foundation model Universal Cell Embeddings, Scupa effectively identifies polarized cells across different species and experimental conditions. Applications of Scupa in independent datasets demonstrated its accuracy in classifying polarized cells and further revealed distinct polarization profiles in tumor-infiltrating myeloid cells across cancers. Scupa complements conventional single-cell data analysis by providing new insights into dynamic immune cell states, and holds potential for advancing therapeutic insights, particularly in cytokine-based therapies. Availability and implementation The code is available at https://github.com/bsml320/Scupa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078b81c851163489c04007fcd2f4d464e0013dc1" target='_blank'>
              Scupa: single-cell unified polarization assessment of immune cells using the single-cell foundation model
              </a>
            </td>
          <td>
            Wendao Liu, Zhongming Zhao
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0116e059872731a0003137617d95a907ab32d870" target='_blank'>
              Molecular landscape of tumor-associated tissue-resident memory T cells in tumor microenvironment of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Mi-So Park, Hyeonbin Jo, Hyeree Kim, Ji Young Kim, Woong-Yang Park, Yong-Han Paik, Yeup Yoon, Wonseok Kang, Hong-Hee Won
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) data, by providing spatial information, enables simultaneous analysis of gene expression distributions and their spatial patterns within tissue. Clustering or spatial domain detection represents an essential methodology for ST data, facilitating the exploration of spatial organizations with shared gene expression or histological characteristics. Traditionally, clustering algorithms for ST have focused on individual tissue sections. However, the emergence of numerous contiguous tissue sections derived from the same or similar tissue specimens within or across individuals has led to the development of multi-slide clustering methods. In this study, we assess seven single-slide and three multi-slide clustering methods on two simulated datasets and three real datasets. Additionally, we investigate the effectiveness of pre-processing techniques, including spatial coordinate alignment (for example, PASTE) and gene expression batch effect removal (for example, Harmony), on clustering performance. Our study provides a comprehensive comparison of clustering methods for multi-slide ST data, serving as a practical guide for method selection in various scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64827c9f1b2bb235c6fb9cc7eb7915798fc07701" target='_blank'>
              A comprehensive comparison on clustering methods for multi-slide spatially resolved transcriptomics data analysis
              </a>
            </td>
          <td>
            Caiwei Xiong, Huang Shuai, Muqing Zhou, Yiyan Zhang, Wenrong Wu, Xihao Li, Huaxiu Yao, Jiawen Chen, Yun Li
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a6df87fa44ba62736b9ada1db0c1fd8750c04f" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Luisa Solis Soto, N. O. Lermi, Max Molina Ayala, Sharia Hernandez Ruiz, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchéz-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, Cara Haymaker, Ken Chen
          </td>
          <td>2025-01-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c89aac6a693cf7fc84e3a1fc45266835eb27a4d" target='_blank'>
              Fusion of tumor cells and mesenchymal stem/stroma cells: a source of tumor heterogeneity, evolution and recurrence.
              </a>
            </td>
          <td>
            Zhen Liu, Yihao Wang, Zesheng Peng, Hui Li, Haofei Wang, Yuyi Wu, Xiaobing Jiang, Peng Fu
          </td>
          <td>2025-01-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c4d7e98a1c7e44d04ba5fa342a0635434e2cbe" target='_blank'>
              Tumor secretome shapes the immune landscape during cancer progression
              </a>
            </td>
          <td>
            Jianqiang Yang, Sijia Tang, Nabil F. Saba, C. Shay, Yong Teng
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Recent advances in single-cell RNA sequencing (scRNA-seq) technologies have enabled us to clarify gene regulatory networks and immune cell compositions. In this study, integration of large-scale bulk and single-cell datasets refined hepatocellular carcinoma (HCC) classification and shed light on the characteristics of its aggressive subtype.


APPROACH AND RESULTS
Single-cell analysis of 228,564 live cells from six scRNA-seq datasets identified five major clusters of HCC cells with high mitotic activity (Cluster 1), activated Wnt/β-catenin signaling (Cluster 2), elevated glycolysis (Cluster 3) and lipogenesis (Clusters 4 and 5). Aggressive HCC subtype defined in bulk RNA-seq analysis of 691 tumor samples comprised a combination of Cluster 1 with Clusters 3, 4 or 5. Gene regulatory network analysis and gene set enrichment analysis highlighted the essential roles of p53 and MYC in aggressive HCC/Cluster 1, and cell composition analysis elucidated T cell depletion as an immune resistance mechanism. In a syngeneic mouse model, Trp53 knockout and MYC overexpression caused highly mitotic, tumorigenic and metastatic phenotypes, characterized by a macrotrabecular (MT) pattern, vascular encapsulation (VE) and T cell exclusion. Angiogenesis inhibition disrupted MT/VE formation, resulting in T cell recruitment, and its combination with immune checkpoint blockade achieved remission.


CONCLUSIONS
Single-cell analysis has deepened our understanding of the molecular mechanism and tumor microenvironment in aggressive HCC. The combination of targeting tumor vasculature and blocking immune checkpoints represents a promising therapeutic strategy for this subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e969ce26ddc70310682b3f3db6a8aee97b5128" target='_blank'>
              Integrative transcriptome profiling elucidates molecular and immunovascular characteristics of macrotrabecular hepatocellular carcinoma.
              </a>
            </td>
          <td>
            T. Taniai, S. Shimada, Y. Akiyama, Megumi Hatano, Koya Yasukawa, Yosuke Igarashi, Shu Tsukihara, Yoshiaki Tanji, K. Kodera, Kohei Okazaki, K. Haruki, Atsushi Nara, Kohei Yagi, K. Akahoshi, D. Ban, Yasuhiro Asahina, Toru Ikegami, Minoru Tanabe, Shinji Tanaka
          </td>
          <td>2025-02-26</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f7ade1ee5b9d0bf71b29249fe64f13f03ddc98" target='_blank'>
              Characterization of single neurons reprogrammed by pancreatic cancer.
              </a>
            </td>
          <td>
            Vera Thiel, Simon Renders, Jasper Panten, Nicolas Dross, K. Bauer, D. Azorin, Vanessa Henriques, V. Vogel, C. Klein, Aino-Maija Leppä, Isabel Barriuso Ortega, J. Schwickert, I. Ourailidis, Julian Mochayedi, Jan-Philipp Mallm, Carsten Müller-Tidow, Hannah Monyer, J. Neoptolemos, T. Hackert, O. Stegle, D. T. Odom, R. Offringa, A. Stenzinger, Frank Winkler, M. Sprick, A. Trumpp
          </td>
          <td>2025-02-17</td>
          <td>Nature</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="The advent of spatial transcriptomics has dramatically expanded our ability to study the vast network of cell-cell interactions at the molecular level in tissue. Among current methods, sequencing-based approaches have great potential in discovering because of its unbiased capture. In the last couple of years, the spatial resolution for the capture addresses has been significantly improved from 100 μm to <1 μm, well below the size of a mammalian cell. However, the capture efficiency has always been a pain point, ∼one order of magnitude lower than that of single cell RNA sequencing (scRNA-seq). The low capture efficiency limits the depth and breadth of its applications in the study of complex biological systems and diseases. Here, we introduce Salus Spatial transcriptomic system (Salus-STS), which provides ∼1 μm capture resolution and a capture efficiency ∼1 order of magnitude higher than other current methods. Analysis with sub-cellular resolution becomes practical for sequencing-based spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6143f746de652c8a331460d4e5358837484378" target='_blank'>
              Sequencing-based Spatial Transcriptomics with scRNA-seq Sensitivity
              </a>
            </td>
          <td>
            Renjie Liao, Defeng Fu, Zaoxu Xu, Han Liang, Xiaoran Zhou, Yiling Chen, Xueqi Liu, Jiajun Cheng, Ruidong Guo, Chen Li, Huihua Xia, Gailing Li, Diewen Feng, Wei Chen, Yang Chen, Longchao Chen, Zihang Huang, Yang Zhou, Qingbin Chen, Dexia Chen, Shuguang Xuan, Changling Zhang, Yongyi Lu, Hui Wang, Taiqing Feng, Yuanye Bao, Luyang Zhao, Erkai Liu, Gufeng Wang
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND AIMS
The prognoses and therapeutic responses of patients with intrahepatic cholangiocarcinoma (iCCA) depend on spatial interactions among tumor microenvironment (TME) components. However, the spatial TME characteristics of iCCA remain poorly understood. The aim of this study was to generate a comprehensive spatial atlas of iCCA using artificial intelligence-assisted spatial multiomics patterns and to identify spatial features associated with prognosis and immunotherapy.


APPROACH RESULTS
Spatial multiomics, including imaging mass cytometry (IMC, n=155 in-house), spatial proteomics (n=155 in-house), spatial transcriptomics (n=4 in-house), multiplex immunofluorescence (mIF, n=20 in-house), single-cell RNA sequencing (scRNA-seq, n=9 in-house and n=34 public), bulk RNA-seq (n=244 public), and bulk proteomics (n=110 in-house and n=214 public), were employed to elucidate the spatial TME of iCCA. More than 1.06 million cells were resolved, and the findings revealed that spatial topology, including cellular deposition patterns, cellular communities, and intercellular communications, profoundly correlates with the prognosis of iCCA patients. Specifically, CD163hi M2-like resident-tissue macrophages suppress anti-tumor immunity by directly interacting with CD8+ T cells, resulting in poorer patient survival. Additionally, five spatial subtypes with distinct prognoses were identified, and potential therapeutic options were generated for these subtypes. Furthermore, a spatial TME deep learning system was developed to predict the prognosis of iCCA patients with high accuracy from a single 1-mm2 tumor sample.


CONCLUSIONS
This study offers preliminary insights into the spatial TME ecosystem of iCCA, providing valuable foundations for precise patient classification and the development of personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, S. Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass-accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte–SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomics analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in three-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d939bb36944c04a3083bae73bc1b5307c8fa9271" target='_blank'>
              Deciphering Colorectal Cancer–Hepatocyte Interactions: A Multiomics Platform for Interrogation of Metabolic Crosstalk in the Liver–Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The tumor microenvironment plays a pivotal role in cancer progression and therapeutic resistance, with tumor-associated macrophages significantly influencing immune suppression and tumor growth. Colorectal cancers (CRC) classified as Consensus Molecular Subtype 4 (CMS4) and triple-negative breast cancers subsets are particularly characterized by a mesenchymal phenotype, immune exclusion, and extensive macrophage infiltration. This study aimed to investigate how targeting cancer cell stemness with specific inhibitors could modulate macrophage polarization in CRC in vitro and breast cancer in vivo, potentially shifting the immune balance from pro-tumor M2-like to anti-tumor M1-like macrophages. We used four stemness inhibitors—salinomycin, SB-431542, JIB-04, and napabucasin—each targeting different pathways (Wnt/β-catenin, TGF-β, histone demethylation, and STAT3, respectively), to evaluate their effects on CMS4 CRC cell lines (HCT116 and SW620) and human peripheral blood-derived macrophages in an indirect co-culture model. Our results showed that CMS4 CRC cell lines induced distinct macrophage polarization patterns, with HCT116 promoting M2-like macrophages and SW620 leaning toward M1-like profile. Notably, the combination of stemness inhibitors reduced stemness markers (CD133, CD44) in colorectal cancer cells and shifted macrophage polarization toward an M1-like phenotype, particularly in co-culture with HCT116. In vivo studies using the syngeneic immunocompetent EO771 breast cancer mouse model demonstrated that combination of stemness inhibitors increased the M1/M2 macrophage ratio.Our study highlights the dual potential of stemness inhibitors to target both cancer cells and the immune microenvironment. These findings offer promising strategies for enhancing favorable immunomodulation in mesenchymal-like colorectal tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/211f6363e023c7088c47db3138c8836f0d0dca08" target='_blank'>
              Targeting stemness pathways modulates macrophage polarization and reprograms the tumor microenvironment
              </a>
            </td>
          <td>
            Austeja Butkute, Marius Baltramonaitis, Simona Malmige, A. Darinskas, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-03-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies have revolutionized our understanding of cellular ecosystems. However, these technologies face challenges such as sparse gene signals and limited gene detection capacities, which hinder their ability to fully capture comprehensive spatial gene expression profiles. To address these limitations, we propose leveraging single-cell RNA sequencing (scRNA-seq), which provides comprehensive gene expression data but lacks spatial context, to enrich ST profiles. Herein, we introduce SpaIM, an innovative style transfer learning model that utilizes scRNA-seq information to predict unmeasured gene expressions in ST data, thereby improving gene coverage and expressions. SpaIM segregates scRNA-seq and ST data into data-agnostic contents and data-specific styles, with the contents capture the commonalities between the two data types, while the styles highlight their unique differences. By integrating the strengths of scRNA-seq and ST, SpaIM overcomes data sparsity and limited gene coverage issues, making significant advancements over 12 existing methods. This improvement is demonstrated across 53 diverse ST datasets, spanning sequencing- and imaging-based spatial technologies in various tissue types. Additionally, SpaIM enhances downstream analyses, including the detection of ligand-receptor interactions, spatial domain characterization, and identification of differentially expressed genes. Released as open-source software, SpaIM increases accessibility for spatial transcriptomics analysis. In summary, SpaIM represents a pioneering approach to enrich spatial transcriptomics using scRNA-seq data, enabling precise gene expression imputation and advancing the field of spatial transcriptomics research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a9f336c4dffd59ace975ec6b6a1760203849d4" target='_blank'>
              SpaIM: Single-cell Spatial Transcriptomics Imputation via Style Transfer
              </a>
            </td>
          <td>
            Bo Li, Ziyang Tang, Aishwarya Budhkar, Xiang Liu, Tonglin Zhang, Baijian Yang, Jing Su, Qianqian Song
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cellular interactions form the fundamental/core circuits that drive development, physiology, and disease within tissues. Advances in spatial genomics (SG) and artificial intelligence (AI) offer unprecedented opportunities to computationally analyze and predict the behavior of cell intricate networks, and to identify interactions that drive disease states. However, challenges arise in both methodology and scalability: (i) how to computationally characterize complicated cellular interactions of multi-scale nature where chemical genes/circuits in individual cells process information and drive interactions among large numbers of diverse cell types, and (ii) how to scale up the pipeline to accommodate the increasing volumes of SG data that map transcriptome-scale gene expression and spatial proximity across millions of cells. In this paper, we introduce the Cellular Interaction Foundation Model (CI-FM), an AI foundation model functioning to analyze and simulate cellular interactions within living tissues. In the CI-FM pipeline, we explicitly capture and embed cellular interactions within microenvironments by leveraging the powerful and scalable geometric graph neural network model, and optimize the characterization of cellular interactions with a novel self-supervised learning objective – we train it to infer gene expressions of cells based upon their interacting microenvironment. As a result, we construct CI-FM with 100 million parameters by consuming SG data of 23 million cells. Our benchmarking experiments show CI-FM effectively infers gene expressions conditional on the microenvironmental contexts: we achieve a high correlation and a low mismatch error (MSE of 1.1% relative to the square median expression), with 79.4% of cells on average being annotated as the similar cell type based on their predicted and actual expressions. We demonstrate the downstream utility of CI-FM by: (i) applying CI-FM to embed tumor samples to capture cellular interactions within tumor microenvironments (ROC-AUC score of 0.76 on classifying sample conditions via linear probing on embeddings), and identifying shared signatures across samples; and (ii) using CI-FM to simulate changes in microenvironmental composition in response to T cell infiltration, which highlights how CI-FM can be leveraged to model cellular responses to tissue perturbations – an essential step toward constructing “AI virtual tissues”. Our model is open source and publicly accessible at https://huggingface.co/ynyou/CIFM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc301670fcf5e5e9540479c78aa400e91f732606" target='_blank'>
              Building Foundation Models to Characterize Cellular Interactions via Geometric Self-Supervised Learning on Spatial Genomics
              </a>
            </td>
          <td>
            Yuning You, Zitong Jerry Wang, Kevin Fleisher, Rex Liu, Matt Thomson
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell proteomics is an emerging field with significant potential to characterize heterogeneity within biological tissues. It offers complementary insights to single-cell transcriptomics by revealing unbiased proteomic changes downstream of the transcriptome. Recent advancements have focused on enhancing proteome coverage and depth, mostly in cultured cell lines, and a few recent studies have explored the potential of analyzing tissue micro-samples but were limited to homogenous peripheral tissues. In this current work, we utilize the power of spatial single cell-proteomics through immunostaining-guided laser capture microdissection (LCM) coupled with LC-MS to investigate the heterogenous central nervous system. We used this method to compare neuronal populations from cortex and substantia nigra, two brain regions associated with motor and cognitive function and various neurological disorders. Moreover, we used the technique to understand the neuroimmune changes associated with stab wound injury. Finally, we focus our application on the peripheral nervous system, where we compare the proteome of the myenteric plexus cell ganglion to the nerve bundle. This study demonstrates the utility of spatial single-cell proteomics in neuroscience research toward understanding fundamental biology and the molecular drivers of neurological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e060f62a68156851b45076f66f6e5883e7b7f674" target='_blank'>
              Molecularly-guided spatial proteomics captures single-cell identity and heterogeneity of the nervous system
              </a>
            </td>
          <td>
            Sayan Dutta, Marion Pang, G. Coughlin, Sirisha Gudavalli, Michael L Roukes, Tsui-Fen Chou, V. Gradinaru
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dear Editor Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark1,2, with a pervasive presence across cancers3,4. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate ecDNA’s behaviors and functions 5-7. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction while we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma.
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Despite numerous studies on medulloblastoma (MB) cell heterogeneity, the spatial characteristics of cellular states remain unclear.


METHODS
We analyze single-nucleus and spatial transcriptomes and chromatin accessibility from human MB spanning four subgroups, to identify malignant cell populations and describe the spatial evolutionary trajectories. The spatial CNVs patterns and niches were analyzed to investigate the cellular interactions.


RESULTS
Three main malignant cell populations were identified, including progenitor-like, cycling and differentiated populations. Gene signatures of cell populations strongly correlate to clinical outcomes. These tumor cell populations are geographically organized as stem-like and mature regions, highlighting their spatially heterogeneous nature. Progenitor-like and cycling cells are mainly concentrated in stem-like regions, whereas various differentiated populations are primarily distributed in mature regions. By analyzing chromosomal alterations, we find that stem-like region typically harbors a single pattern of CNVs, reflecting high originality and uniformity, which is in stark contrast to mature region exhibiting multiple patterns with a broader range of biological functions. Projecting cellular state program onto spatial sections fully illustrates the evolution from stem-like region to various functional zones in mature region, which is correlated to microenvironmental components along the paths to maintain stemness or promote differentiation. Conclusions. This multi-omics database comprehensively facilitates the understanding of MB spatial evolutionary organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Single cell transcriptomics (scRNAseq) remains the method of choice to quantify and annotate cellular heterogeneity in the tumor and its microenvironment (TME), yet most studies are under-powered to associate changes in tumor heterogeneity with phenotypes such as tumor subtype, grade, and patient age, among others. In this study, we created an integrated atlas of human breast cancer (BC), the largest resource of its kind, totaling > 700,000 cells across 145 patients, and optimized computational methods to benchmark integration performance, and robustly perform hierarchical cell type annotation. By combining single profiles, a higher-resolution annotation of immune, stromal, and epithelial cell types was achieved. Further, using this integrated atlas, differential abundance analyses identified significant changes in immune and stromal cell type composition associated with tumor grade. These changes in TME heterogeneity were not discernible or had effects in the opposite direction when the analysis was limited to individual BC studies, highlighting the need for atlas-based mega-analysis approaches. This highly-resolved integrated scRNAseq BC atlas will be a valuable resource for hypothesis-driven analyses of tumor heterogeneity including our own ongoing analysis of comorbidities in BC.
 Citation Format: Andrew D Chen, Lina Kroehling, Christina Ennis, Gerald Denis, Stefano Monti. Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew D Chen, Lina Kroehling, Christina Ennis, Gerald Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The exploration of cellular heterogeneity within the tumor microenvironment (TME) via single-cell RNA sequencing (scRNA-seq) is essential for understanding cancer progression and response to therapy. Current scRNA-seq approaches, however, lack spatial context and rely on incomplete datasets of ligand-receptor interactions (LRIs), limiting accurate cell type annotation and cell-cell communication (CCC) inference. This study addresses these challenges using a novel graph neural network (GNN) model that enhances cell type prediction and cell interaction analysis. Our study utilized a dataset consisting of 49,020 cells from 19 patients across three cancer types: Leukemia, Breast Invasive Carcinoma, and Colorectal Cancer. The proposed scGSL model demonstrated robust performance, achieving an average accuracy of 84.83%, precision of 86.23%, recall of 81.51%, and an F1 score of 80.92% across all datasets. These metrics represent a significant enhancement over existing methods, which typically exhibit lower performance metrics. Additionally, by reviewing existing literature on gene interactions within the TME, the scGSL model proves to robustly identify biologically meaningful gene interactions in an unsupervised manner, validated by significant expression differences in key gene pairs across various cancers. The source code and data used in this paper can be found in https://github.com/LiYuechao1998/scGSL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd31c962d460de902bace72d06f995f9fc390f7" target='_blank'>
              Graph Structure Learning for Tumor Microenvironment with Cell Type Annotation from non-spatial scRNA-seq data
              </a>
            </td>
          <td>
            Yu-an Huang, Yue C. Li, Hai-Ru You, Jie Pan, Xiyue Cao, Xinyuan Li, Zhi-an Huang, Zhu-Hong You
          </td>
          <td>2025-02-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The upper respiratory tract, organized along the pharyngolaryngeal-to-tracheobronchial axis, is essential for homeostatic functions such as breathing and vocalization. The upper respiratory epithelium is frequently exposed to pollutants and pathogens, making this an area of first-line defense against respiratory injury and infection. The respiratory epithelium is composed of a rich array of specialized cell types, each with unique capabilities in immune defense and injury repair. However, the precise transcriptomic signature and spatial distribution of these cell populations, as well as potential cell subpopulations, have not been well defined. Here, using single cell RNAseq combined with spatial validation, we present a comprehensive atlas of the mouse upper respiratory epithelium. We systematically analyzed our rich RNAseq dataset of the upper respiratory epithelium to reveal 17 cell types, which we further organized into three spatially distinct compartments: the Tmprss11a+ pharyngolaryngeal, the Nkx2-1+ tracheobronchial, and the Dmbt1+ submucosal gland epithelium. We profiled/analyzed the pharyngolaryngeal epithelium, composed of stratified squamous epithelium, and identified distinct regional signatures, including a Keratin gene expression code. In profiling the tracheobronchial epithelium, which is composed of a pseudostratified epithelium-with the exception of the hillock structure-we identified that regional luminal cells, such as club cells and basal cells, show varying gradients of marker expression along the proximal-distal and/or dorsal-ventral axis. Lastly, our analysis of the submucosal gland epithelium, composed of an array of cell types, such as the unique myoepithelial cells, revealed the colorful diversity of between and within cell populations. Our single-cell atlas with spatial validation highlights the distinct transcriptional programs of the upper respiratory epithelium and serves as a valuable resource for future investigations to address how cells behave in homeostasis and pathogenesis. Highlights - Defined three spatially distinct epithelial compartments, Tmprss11a+ pharyngolaryngeal, Nkx2-1+ tracheobronchial, and Dmbt1+ submucosal gland, comprising 17 total cell types - Profiled Keratin gene expression code along proximal-distal and basal-luminal axes and highlighted “stress-induced” Keratins KRT6A and KRT17 at homeostasis - Demarcated expression gradients of Scgb1a1+ and Scgb3a2+ club cells along the proximal-distal axes - Specified submucosal gland cell heterogeneity including Nkx3-1+ mucin-producing cells, with ACTA2+ basal myoepithelial cells exhibiting gene profile for neuroimmune mediated signaling">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be87eb7039355e97c2fe29703d3fe281b26c7b40" target='_blank'>
              A Single-Cell Atlas of the Upper Respiratory Epithelium Reveals Heterogeneity in Cell Types and Patterning Strategies
              </a>
            </td>
          <td>
            Alexander G. Foote, Xin Sun
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic and epigenomic instability are defining features of cancer, driving tumor progression, heterogeneity, and therapeutic resistance. Central to this process are epigenetic echoes, persistent and dynamic modifications in DNA methylation, histone modifications, non-coding RNA regulation, and chromatin remodeling that mirror underlying genomic chaos and actively influence cancer cell behavior. This review delves into the complex relationship between genomic instability and these epigenetic echoes, illustrating how they collectively shape the cancer genome, affect DNA repair mechanisms, and contribute to tumor evolution. However, the dynamic, context-dependent nature of epigenetic changes presents scientific and ethical challenges, particularly concerning privacy and clinical applicability. Focusing on lung cancer, we examine how specific epigenetic patterns function as biomarkers for distinguishing cancer subtypes and monitoring disease progression and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b401f96a57f38873c48e3928dd0d7e44a2cbef" target='_blank'>
              Epigenomic Echoes—Decoding Genomic and Epigenetic Instability to Distinguish Lung Cancer Types and Predict Relapse
              </a>
            </td>
          <td>
            Alexandra A Baumann, Zholdas Buribayev, Olaf Wolkenhauer, Amankeldi A. Salybekov, M. Wolfien
          </td>
          <td>2025-02-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Metastasis drives mortality and morbidity in cancer. While some patients develop broad metastatic disease across multiple organs, others exhibit organ-specific spread. To identify mechanisms underlying metastatic organotropism, we analyzed clinico-genomic data from over 7,000 patients with metastatic cutaneous melanoma in three independent cohorts (one primary discovery and two validation cohorts including a nationwide electronic health record-derived deidentified database), leveraging machine learning approaches to clinical data. We found that female sex and increased tumor mutational burden associate with decreased metastatic potential, while older age associates with more lung and adrenal metastases. Using unsupervised analyses, patients clustered into five metastatic patterns: a “highly metastatic” cluster characterized by involvement of many organs, a “low metastatic” cluster characterized by few metastatic sites (mostly lymph node metastases), and three additional clusters each characterized by metastasis to specific sites (brain, lung, liver). Mutations in B2M and PTEN associated with increased overall metastatic potential. PTEN mutations were also associated with brain metastases but were enriched only in the “highly metastatic” cluster and not the brain-specific cluster. Mutations in GNAQ or GNA11 (GNA) associated with increased liver metastasis. To validate this association, we tested and demonstrated liver tropism in two GNA-mutant genetically engineered cutaneous melanoma mouse models of metastasis. Overall, our study elucidates distinct phenotypes of metastasis in patients with melanoma and identifies novel clinical and genomic associations that illuminate the clinical drivers of metastatic organotropism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7031286fbc9875d02c4079e2f16707bda19ea728" target='_blank'>
              Clinico-genomic features predict distinct metastatic phenotypes in cutaneous melanoma
              </a>
            </td>
          <td>
            Tyler J. Aprati, Chi-Ping Day, Daniel Lee, Alexander Pan, J. Jee, Giuseppe Tarantino, M. Manos, Hannah Faulkner, Marta M. Holovatska, K. Khaddour, Catherine H. Feng, Kelly P. Burke, Marc Glettig, Zoe Weaver Ohler, R. El Meskini, F. S. Hodi, Rizwan Haq, A. Shoushtari, N. Schultz, Jeffrey J. Ishizuka, Alexander Gusev, Maryclare Griffith, Kenneth L. Kehl, David Liu
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell (CART) therapy has demonstrated remarkable efficacy in hematologic malignancies but has struggled to achieve comparable success in solid tumors. A key obstacle is the extracellular matrix (ECM) in solid tumors, which significantly impedes CART cell infiltration. In clinical trials, neuroblastoma (NB) has shown responsiveness to GD2-directed CART therapy, however, the failure of GD2.CARTs to effectively clear bulky disease - characterized by dense ECM - highlights the critical challenge of infiltration. In this study, we demonstrate that GD2.CARTs exhibit a unique infiltration-restriction compared to other CARTs and endogenous T cells. A separate analysis of clinical datasets identified MMP7 and SPP1 (OPN) as candidate genes to improve the infiltration of GD2.CARTs as these were upregulated in tumor-infiltrating leukocytes. MMP-7 and OPN overexpression enhanced CART extravasation (p < .001) and interstitial movement (p < .05) in ECM-dense environments in vitro. Overexpression of either OPN (p < .0001) or MMP-7 (p < .001) improved tumor infiltration in a xenograft model of NB. This resulted in improved tumor control (94% reduction in tumor burden, p < .05) and a survival extension in OPN-GD2.CART treated mice compared to unmodified GD2.CARTs (median of 148 days, p < .05). OPN overexpression did not increase off-target infiltration into healthy tissues or promote tumor metastasis, highlighting its potential for safe therapeutic application. Our study provides a framework for further exploration of gene modifications to improve CART infiltration and efficacy in solid tumors and identifies SPP1 as a candidate gene to improve GD2.CART treatment of bulky tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4a1fc739777ef8383e9e035dee7b2fb590a010" target='_blank'>
              Tunneling CARs: Increasing CAR T tumor infiltration through the overexpression of MMP7 and SPP1
              </a>
            </td>
          <td>
            Stacey N. Van Pelt, Mark White, Candise Tat, Devyn Hooper, Lindsay Talbot, Rohan Fernandes, Cliona Rooney, Bilal Omer
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Crohn's disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn's disease.
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Background Glioma, characterized by its cellular and molecular heterogeneity, presents formidable challenges in treatment strategy and prognostic assessment. The tumor microenvironment (TME) profoundly influences tumor behavior and treatment response, with tumor‐associated neutrophils (TANs) playing a complex but understudied role. This study aimed to investigate the heterogeneity and role of TANs in glioma and to develop a prognostic model. Methods Analysis of scRNA‐seq data identified cellular subpopulations and differentially expressed neutrophil‐related genes (DE‐NRGs). Bulk RNA‐seq was obtained from four independent datasets. Molecular subtypes of glioma samples were determined by consensus clustering. WGCNA was conducted to elucidate the association between gene modules and subtypes. We developed a risk score model. Expression of selected genes was confirmed using immunohistochemistry (IHC). In vitro experiments were also performed for functional verification, including CCK8, EdU, Transwell, and TUNEL assays. Results A total of 108 DE‐NRGs for TANs were identified based on scRNA‐seq data. Two molecular subtypes were characterized, showing significant differences in prognosis and clinical features. Immune‐related analyses demonstrated varied immunological characteristics between subtypes. The risk score model was constructed with 7 genes, including AEBP1, CAVIN1, DCTD, DEPP1, DUSP6, FKBP9, and UGCG. It showed significant prognostic value and was validated across three external datasets. The mutation landscape highlighted higher IDH mutation prevalence in low‐risk groups. Drug sensitivity analysis indicated TMZ resistance in high‐risk groups. In vitro experiments showed that UGCG could promote glioma cell proliferation, migration, and invasion, while decreasing apoptosis. Conclusion This study explored the heterogeneity of TANs and developed a prognostic model, providing insights for prognostic prediction and guiding personalized treatment strategies in glioma. Declaration of Generative AI in Scientific Writing: The authors declare nonuse of generative AI and AI‐assisted technologies in the writing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a8af983a6227909234b80a95e84b835146d4d4" target='_blank'>
              Integrated Analysis to Reveal Heterogeneity of Tumor‐Associated Neutrophils in Glioma
              </a>
            </td>
          <td>
            Wen Wang, Junsheng Li, Qiheng He, Chenglong Liu, Siyu Wang, Zhiyao Zheng, Bojian Zhang, Siqi Mou, Wei Sun, Jizong Zhao
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Glioblastoma (GBM) recurrence remains an unmet medical challenge, largely attributed to resilient glioma cancer stem cells (CSCs) that evade conventional therapies. TCR T cells bear virally infected T cell receptors for targeting antigen-bearing tumor cells. NY-ESO-1-specific T cell receptor (TCR) T cells can target GBM cells. Decitabine (DAC) increases NY-ESO-1 expression, specifically in CSCs, potentially enabling the eradication of this cell population.



 NY-ESO-1 expression was analyzed by immunocytochemistry and RT-PCR. Cytotoxicity assays with T cells and DAC pre-treated tumor cells were assessed by flow cytometry. Luciferase-expressing U251 glioma cells, known to harbor a CSC niche, were intracranially implanted in immunocompromised mice. Mice were treated with DAC before the intracerebroventricular delivery of TCR-T or control T cells. Tumor growth was tracked by bioluminescent imaging (BLI), and organ tissues were harvested for analysis at the survival endpoint.



 Our findings indicate that DAC exposure significantly upregulates NY-ESO-1 messenger RNA and protein expression, alongside increases in MHC class, bolstering the immune response against patient-derived CSCs. DAC significantly sensitizes CSC 38b cells to lysis by TCR-T cells, exhibiting a marked increase in killing efficacy compared to control T cells (36% versus 20%, p=0.02) within 20 hours. In vivo, TCR-T cells delayed tumor progression and extended survival in glioma-bearing mice compared to the control cohorts (median 40 versus 27 days, p = 0.031). Furthermore, TCR-T cells successfully infiltrated tumors with preferential targeting of NY-ESO-1-expressing cancer cells and CD133-positive stem-like cells within the intricate tumor microenvironment.



 Our findings demonstrate the potential of a novel NY-ESO-1-specific TCR-T cell as a promising immunotherapeutic strategy against GBM and its difficult-to-treat CSC population. These findings underscore the need to develop TCR-based therapies further for more advanced, precision immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abee2ab2885700afbd38b00c669afff895861f19" target='_blank'>
              2002 NY-ESO-1-Specific TCR-T Cells Effectively Targets Glioblastoma Cancer Stem Cells
              </a>
            </td>
          <td>
            Michelot Michel, Alan Nguyen, Alexis Parker, Hongqiang Wang, Chen Qian, Tesfahun Admasu, Andre E. Boyke, Seokyoung Yoon, Keith L. Black, Rongfu Wang, John S. Yu
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) exhibits a complex tumor microenvironment with significant cellular heterogeneity, particularly involving cancer-associated fibroblasts that influence tumor behavior and metastasis. This study integrated single-cell RNA sequencing and spatial transcriptomics to dissect fibroblast heterogeneity in CRC. Data processing employed Seurat for quality control, principal component analysis for dimensionality reduction, and t-Distributed Stochastic Neighbor Embedding for visualization. Differentially expressed genes were identified using DESeq2. Immune infiltration was assessed via Single-Sample Gene Set Enrichment Analysis, CIBERSORT, and xCell algorithms. Prognostic genes were identified through univariate Cox regression, followed by consensus clustering and survival analysis. Metabolic pathways were explored using scMetabolism. Experimental validation involved CCK8, scratch, and Transwell assays to evaluate the roles of key genes BGN and CERCAM in CRC cell proliferation and metastasis. Machine learning-driven analysis identified four fibroblast-associated genes (TRIP6, TIMP1, BGN, and CERCAM) demonstrating significant prognostic relevance in CRC. Consensus clustering based on these biomarkers stratified CRC patients into three distinct molecular subtypes (Clusters A-C). Notably, Cluster C exhibited the most unfavorable clinical outcomes coupled with marked upregulation of all four fibroblast-related genes. Comprehensive immune profiling revealed paradoxical features in Cluster C: heightened global immune activation (characterized by substantial leukocyte infiltration) coexisted with specific immunosuppressive elements, including significant enrichment of pro-tumorigenic M0 macrophages, depletion of anti-tumor plasma cells, and resting memory CD4+ T cells, along with coordinated upregulation of multiple immune checkpoint molecules. Computational prediction using the TIDE platform suggested enhanced immunotherapy responsiveness in Cluster C patients. Functional validation demonstrated that knockdown of BGN or CERCAM significantly impaired malignant phenotypes, reducing proliferative capacity, migration potential, and invasive ability. Fibroblasts demonstrate significant heterogeneity within the CRC immune microenvironment, impacting prognosis and therapeutic responses. Key genes BGN and CERCAM emerge as potential immunotherapeutic targets, offering new strategies for precision treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Cancer cells are defined by rapid, unchecked proliferation, a hallmark that distinguishes them from normal cells. This aggressive growth often leads to a nutrient- and oxygen-deprived tumor core, where limited blood supply cannot sufficiently reach. While most cells typically undergo cell death under such glutamine-deficient conditions, a subset of cancer cells survives through specific epigenetic modifications, facilitating metastatic progression. Using spatial transcriptomics analysis, we reveal that colorectal cancer cells exploit the downregulation of GLS2 expression to avoid ferroptosis and support metastasis in the tumor's glutamine-starved core. We further identified an increase in H3K27Me3 modification as a key epigenetic driver enabling these adaptations in glutamine-deficient regions. This hypothesis has been validated in in vivo mouse colon cancer models and in patient samples compared with healthy controls, reinforcing its translational relevance. Importantly, by employing amino acid profiling through 19F NMR spectroscopy, we identified key biomarkers in the tumor microenvironment that may enhance response to immunotherapy and overcome current therapeutic resistance. These findings underscore the potential of precision medicine strategies that target ferroptosis, epigenetic regulation, and tumor-specific biomarkers, offering new avenues for therapeutic intervention in colorectal cancer patients, particularly those with core-originating tumor regions. Our work exemplifies the broader goal of functional and genomic precision medicine in cancer, aligning distinct mechanistic insights with shared therapeutic objectives.
 Citation Format: Ji Min Lee. Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7948bcd29ed4c332a5d0dc90dea3c9f39d631a84" target='_blank'>
              Identifying distinct prognostic and predictive contributions of tumor epithelium versus tumor microenvironment in colorectal cancer
              </a>
            </td>
          <td>
            Mingli Yang, M. Nebozhyn, Michael J. Schell, N. Gandhi, Lance Pflieger, Andrey Loboda, W. J. Pledger, Ramani Soundararajan, Michelle M Maurin, Heiman Wang, Jetsen Rodriguez Silva, Ashley J Alden, Domenico Coppola, Andrew Elliott, George W Sledge, M. Khushman, Emil Lou, Sanjay Goel, Timothy J Yeatman
          </td>
          <td>2025-03-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer is a complex disease driven by dynamic regulatory shifts that cannot be fully captured by individual molecular profiling. We employ a data-driven approach to construct a coarse-grained dynamic network model based on hallmark interactions, integrating stochastic differential equations with gene regulatory network data to explore key macroscopic dynamic changes in tumorigenesis. Our analysis reveals that network topology undergoes significant reconfiguration before hallmark expression shifts, serving as an early indicator of malignancy. A pan-cancer examination across $15$ cancer types uncovers universal patterns, where Tissue Invasion and Metastasis exhibits the most significant difference between normal and cancer states, while the differences in Reprogramming Energy Metabolism are the least pronounced, consistent with the characteristic features of tumor biology. These findings reinforce the systemic nature of cancer evolution, highlighting the potential of network-based systems biology methods for understanding critical transitions in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND AND AIM
Intrahepatic cholangiocarcinoma (ICC) is the second most common primary liver cancer whose incidence is increasing globally. However, the high tumor heterogeneity of ICC restricts the efficacy of available systematic therapies. We aim to dissect the tumor heterogeneity of ICC utilizing high-resolution single-cell RNA sequencing to identify novel therapeutic targets.


METHODS
We performed single-cell RNA sequencing (scRNA-seq) of 26 tumor samples from 23 ICC patients and spatial transcriptomic sequencing of six tumor sections from six ICC patients. Bulk RNA-seq data from two public datasets were used for validation. Additionally, immunohistochemical staining and multiplex immunofluorescence staining were conducted to validate the infiltration and distribution of cells in the tumor microenvironment.


RESULTS
We discovered that malignant cells in ICC samples exhibited a remarkably high degree of tumor heterogeneity. We identified a basal-like tumor cell subpopulation characterized by the expression of basal epithelial related genes including KRT5, KRT6A, and KRT17. The basal-like tumor subpopulation was characterized by activation of MET signaling and extracellular matrix organization associated with tumor invasion and correlated with poor prognosis. Cell-cell communication analysis further showed significant HGF-MET interaction between inflammatory cancer-associated fibroblasts (iCAFs) and basal-like tumor cells. We found that iCAFs were the major source of HGF in tumor environment and contributed to the basal-like phenotype formation of tumor cells by HGF-MET axis.


CONCLUSIONS
We identified an aggressive basal-like tumor cell subpopulation, which correlated with poor prognosis in ICC. The MET pathway contributes to the aggressiveness of basal-like tumor cells and serves as a novel therapeutic target for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f2358565c36d3daf9c70fc71b645b6b73d557a4" target='_blank'>
              SELP+ TEC:CD8+ T cell crosstalk associates with improved radiotherapy efficacy in cervical cancer
              </a>
            </td>
          <td>
            Qingyu Huang, Wenhui Yang, Fuhao Wang, R. Huang, Qian Wang, Xiaohui Li, T. Lei, Shengqin Yue, Wen-Lu Zou, Qi An, Jinbo Yue, Qinyong Hu, Chao Liu
          </td>
          <td>2025-02-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Piñeiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Y. Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small subset within the tumor mass significantly contributing to cancer progression through dysregulation of various oncogenic pathways, driving tumor growth, chemoresistance and metastasis formation. The aggressive behavior of CSCs is guided by several intracellular signaling pathways such as WNT, NF-kappa-B, NOTCH, Hedgehog, JAK-STAT, PI3K/AKT1/MTOR, TGF/SMAD, PPAR and MAPK kinases, as well as extracellular vesicles such as exosomes, and extracellular signaling molecules such as cytokines, chemokines, pro-angiogenetic and growth factors, which finely regulate CSC phenotype. In this scenario, tumor microenvironment (TME) is a key player in the establishment of a permissive tumor niche, where CSCs engage in intricate communications with diverse immune cells. The “oncogenic” immune cells are mainly represented by B and T lymphocytes, NK cells, and dendritic cells. Among immune cells, macrophages exhibit a more plastic and adaptable phenotype due to their different subpopulations, which are characterized by both immunosuppressive and inflammatory phenotypes. Specifically, tumor-associated macrophages (TAMs) create an immunosuppressive milieu through the production of a plethora of paracrine factors (IL-6, IL-12, TNF-alpha, TGF-beta, CCL1, CCL18) promoting the acquisition by CSCs of a stem-like, invasive and metastatic phenotype. TAMs have demonstrated the ability to communicate with CSCs via direct ligand/receptor (such as CD90/CD11b, LSECtin/BTN3A3, EPHA4/Ephrin) interaction. On the other hand, CSCs exhibited their capacity to influence immune cells, creating a favorable microenvironment for cancer progression. Interestingly, the bidirectional influence of CSCs and TME leads to an epigenetic reprogramming which sustains malignant transformation. Nowadays, the integration of biological and computational data obtained by cutting-edge technologies (single-cell RNA sequencing, spatial transcriptomics, trajectory analysis) has significantly improved the comprehension of the biunivocal multicellular dialogue, providing a comprehensive view of the heterogeneity and dynamics of CSCs, and uncovering alternative mechanisms of immune evasion and therapeutic resistance. Moreover, the combination of biology and computational data will lead to the development of innovative target therapies dampening CSC-TME interaction. Here, we aim to elucidate the most recent insights on CSCs biology and their complex interactions with TME immune cells, specifically TAMs, tracing an exhaustive scenario from the primary tumor to metastasis formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ce167799371ac069e6df125e4a468fd20b0208" target='_blank'>
              Cancer stem cells and tumor-associated macrophages as mates in tumor progression: mechanisms of crosstalk and advanced bioinformatic tools to dissect their phenotypes and interaction
              </a>
            </td>
          <td>
            Francesco Verona, Sebastiano Di Bella, Roberto Schirano, Camilla Manfredi, Francesca Angeloro, Giulia Bozzari, M. Todaro, Giuseppe Giannini, Giorgio Stassi, V. Veschi
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad4f13e415f9357404f99d30b4979256aef4341" target='_blank'>
              Prognostic implications and characterization of tumor-associated tertiary lymphoid structures genes in pancreatic cancer
              </a>
            </td>
          <td>
            Enkui Zhang, Yongsu Ma, Zonghao Liu, Jixin Zhang, Weikang Liu, Yiran Chen, Guangnian Liu, Xinxin Liu, Fusheng Zhang, Yu Zhu, Yinmo Yang, Xiaodong Tian
          </td>
          <td>2025-03-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Lung cancer, particularly non-small cell lung cancer (NSCLC), remains a leading cause of cancer-related deaths worldwide due to its complexity and resistance to therapy. This review explores the critical role of stem cells in lung cancer progression, focusing on how genetic mutations and the tumor microenvironment drive cancer stem cell activity, leading to therapy resistance and tumor relapse. By understanding these mechanisms, the research aims to identify potential therapeutic targets and strategies to improve treatment outcomes for NSCLC patients. This work presents the latest developments of novel personalized therapies and enhances the effectiveness of existing treatments, offering new insights and information for patients and the broader research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3b80e9aee70a721cb7f7cf165886e84b84b9b" target='_blank'>
              Oncogenic Mutations and the Tumor Microenvironment: Drivers of Non-Small Cell Lung Cancer Progression
              </a>
            </td>
          <td>
            A. Mitrakas, Christos Kakouratos, I. Lamprou, Erasmia T. Xanthopoulou, M. Koukourakis
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Background & Aims
Identifying patients with intrahepatic cholangiocarcinoma (ICC) likely to benefit from immunochemotherapy, the new front-line treatment, remains challenging. We aimed to unveil a novel radiotranscriptomic signature that can facilitate treatment response prediction by multi-omics integration and multi-scale modelling.


Methods
We analyzed bulk, single-cell and spatial transcriptomic data comprising 457 ICC patients to identify an immune-related score (IRS), followed by decoding its spatial immune context. We mapped radiomics profiles onto spatial-specific IRS using machine-learning to define a novel radiotranscriptomic signature, followed by multi-scale and multi-cohort validation covering 331 ICC patients. The signature was further explored for the potential therapeutic target from in vitro to in vivo.


Results
We revealed a novel 3-gene (PLAUR, CD40LG and FGFR4) IRS whose down-regulation correlated with better survival and improved sensitivity to immunochemotherapy. We highlighted functional IRS-immune interactions within tumor epithelium, rather than stromal compartment, irrespective of geospatial locations. Machine-learning pipeline identified the optimal 3-feature radiotranscriptomic signature that was well-validated by immunohistochemical assays in molecular cohort, exhibited favorable external prognostic validity with C-index over 0.64 in resection cohort, and predicted treatment response with an area under the curve of up to 0.84 in immunochemotherapy cohort. We also showed that anti-uPAR/PLAUR alone or in combination with anti-programmed cell death protein 1 therapy remarkably curbed tumor growth, using in vitro ICC cell lines and in vivo humanized ICC patient-derived xenograft mouse models.


Conclusions
This proof-of-concept study sheds light on the spatially-resolved radiotranscriptomic signature to improve patient selection for emerging immunochemotherapy and high-order immunotherapy combinations in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="This study investigates the complex dynamics of vascular tumors and their interplay with macrophages, key agents of the innate immune response. We model the tumor microenvironment as a multiphase fluid, with each cellular population treated as a distinct, non-mixing phase. The framework also incorporates diffusible species that are critical for processes such as nutrient transport, angio-genesis, chemotaxis, and macrophage activation. Numerical simulations of our model show how phenotypic and spatial heterogeneity in the macrophage population arises and how such heterogeneity impacts a tumor’s growth dynamics. Finally, we propose an immunotherapeutic strategy based on the experimental agent vactosertib which promotes an anti-tumor macrophage phenotype. Our simulations demonstrate an increased density of anti-tumor macrophages over the period of a few months, followed by a relapse period where the tumor regains its original dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8148f78a26bf47463c0acd3c067c50a7bec19c0" target='_blank'>
              Spatio-temporal dynamics of M1 and M2 macrophages in a multiphase model of tumor growth
              </a>
            </td>
          <td>
            I. Lampropoulos, Panayotis Kevrekidis, Christos Zois, Helen Byrne, Mihail E Kavousanakis
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53445f254e578b09b808668681cf7ab5736bb72" target='_blank'>
              Transcriptome combined with single-cell sequencing explored prognostic markers associated with T cell exhaustion characteristics in head and neck squamous carcinoma
              </a>
            </td>
          <td>
            Jie Liu, Penghui Li, Yuanyuan Zhang, Lian Zheng
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) has emerged as a transformative technology for elucidating cellular organization and tissue architecture. However, a significant challenge remains in identifying pathology-relevant spatial functional landscapes within the tissue microenvironment, primarily due to the limited integration of cell-cell communication dynamics. To address this limitation, SpaDCN, a Spatially Dynamic graph Convolutional Network framework is proposed, which aligns cell-cell communications and gene expression within a spatial context to reveal the spatial functional regions with the coherent cellular organization. To effectively transfer the influence of cell-cell communications on expression variation, SpaDCN respectively generates the node layer and edge layer of spatial graph representation from expression data and the ligand-receptor complex contributions and then employs a dynamic graph convolution to switch the propagation of node graph and edge graph. It is demonstrated that SpaDCN outperforms existing methods in identifying spatial domains and denoising expression across various platforms and species. Notably, SpaDCN excels in identifying marker genes with significant prognostic potential in cancer tissues. In conclusion, SpaDCN offers a powerful and precise tool for spatial domain detection in spatial transcriptomics, with broad applicability across various tissue types and research disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5ceb2ab6578aafeb302d4517a191144bf649c1" target='_blank'>
              STAG2 expression imparts distinct therapeutic vulnerabilities in muscle-invasive bladder cancer cells
              </a>
            </td>
          <td>
            Sarah R. Athans, Henry Withers, Aimee Stablewski, Katerina Gurova, Joyce Ohm, Anna Woloszynska
          </td>
          <td>2025-03-01</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Breast cancer is a heterogeneous malignancy where precision medicine approaches have informed treatment decisions; currently, these decisions are largely driven by IHC subtyping based on the expression of targetable receptor proteins (ER, PR, and HER2). Between clinical subtypes, immune cell infiltration and its prognostic impact vary considerably. Tumor-infiltrating lymphocytes (TILs) are most common in triple-negative breast cancers and are associated with improved patient prognosis. In contrast, TILs are less abundant on average in ER+/HER2- tumors, and their presence is much less prognostic. These contrasting data point to distinct roles for the immune system in different breast cancer subtypes, which will be crucial to understand for optimal application of immunotherapy. Using the multi-platform TCGA-BRCA dataset (n=1035 tumors), we evaluated stromal TILs from H&E images and then compared the microenvironments of high-TIL tumors by clinical and molecular subtype using 250+ pre-determined immune signatures calculated from bulk RNA sequencing of the same tumors. We identified unique immune signatures highly expressed in each subtype: granulocytes and interferon signaling were more abundant in luminal/ER+ tumors, while HER2-enriched and basal-like tumors had elevated signature scores related to B and T cells, respectively. Major histocompatibility complex Class I (MHC-I) antigen presentation is another key immune pathway that can vary by tumor subtype. Further, loss or downregulation of MHC-I genes, including HLA-A, is a prevalent mechanism of immune evasion in breast cancer. Using bulk and single-cell RNA sequencing (scRNAseq), we found that ER+/Luminal tumor cells express low levels of HLA-A, while TNBCs/basal-like tumor cells express high levels. HLA-A also has subtype-specific associations with survival, where high HLA-A expression predicts better outcomes in TNBC but worse outcomes in ER+ patients. We therefore hypothesized that MHC-I may have distinct roles in different subtypes of breast cancer. Preliminary data from a cohort of 108 human breast tumor scRNAseq samples indicate that HLA-A expression in ER+/Luminal tumors may be associated with more stromal infiltration, while HLA-A in TNBCs is associated with more immune cell infiltration. Additionally, we hypothesized that ER/PR signaling may play a role in the low levels of HLA-A expression observed in ER+/Luminal tumors. Using a scRNAseq dataset of short-term cultures of ER+ human breast tumors treated with tamoxifen, we found an association between PR levels and induction of HLA-A expression following tamoxifen treatment, such that PR-high tumors were more likely to upregulate HLA-A following tamoxifen. Overall, these data support that the immune microenvironments of breast tumor molecular subtypes are distinct and likely influenced by tumor cell-intrinsic features. These attributes will be important to guide the development and implementation of precision immunotherapy approaches and may have utility in the future as biomarkers to guide treatment strategies.
 Citation Format: Constandina E O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M Felsheim, Aranzazu Fernandez-Martinez, Charles M Perou. Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ea1b0559ba72f593b3a6674f424957c7f5230c" target='_blank'>
              Abstract B016: Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer
              </a>
            </td>
          <td>
            Constandina E. O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M. Felsheim, A. Fernandez-Martinez, C. Perou
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state.
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In this study, we aim to provide a deep understanding of the tumor microenvironment (TME) and its metabolic characteristics in non-small cell lung cancer (NSCLC) through single-cell RNA sequencing (scRNAseq) data obtained from public databases. Given that lung cancer is a leading cause of cancer-related deaths globally and NSCLC accounts for the majority of lung cancer cases, understanding the relationship between TME and metabolic pathways in NSCLC is crucial for developing new treatment strategies.Finally, machine learning algorithms were employed to construct a risk signature with strong predictive power across multiple independent cohorts. After quality control, 29,053 cells were retained, and PCA along with UMAP techniques were used to distinguish 13 primary cell subpopulations. Four highly activated metabolic pathways were identified within malignant cell subpopulations, which were further divided into seven distinct subgroups showing significant differences in differentiation potential and metabolic activity. WGCNA was utilized to identify gene modules and hub genes closely associated with these four metabolic pathways.Our analysis showed that DEGs between tumor and normal tissues were predominantly enriched in immune response and cell adhesion pathways. The comprehensive examination of our model revealed substantial variations in clinical and pathological characteristics, enriched pathways, cancer hallmarks, and immune infiltration scores between high-risk and low-risk groups. Wet lab experiments validated the role of KRT6B in NSCLC, demonstrating that KRT6B expression is elevated and it stimulates the proliferation of cancer cells.These observations not only enhance our understanding of metabolic reprogramming and its biological functions in NSCLC but also provide new perspectives for early detection, prognostic evaluation, and targeted therapy. However, future research should further explore the specific mechanisms of these metabolic pathways and their application potentials in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac23a1537d20deed5cf7808552ddb70f9ad2a85" target='_blank'>
              Metabolic pathway activation and immune microenvironment features in non-small cell lung cancer: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Yanru Liu, Hanmin Liu, Ying Xiong
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 760


 Background:
 Pancreatic adenosquamous carcinoma (PASC) is a rare histopathological type of pancreatic cancer that is more aggressive than pancreatic ductal adenocarcinoma (PDAC). Although PASC has a unique tumor biology and poorer prognosis, there are no guidelines for differentially treating PASC patients, and its rarity makes it significantly understudied. We used Nanostring GeoMX digital spatial profiling (DSP) to provide a targeted proteomic characterization of key tumor-promoting proteins in PASC and PDAC.
 Methods:
 FFPE tissue from PASC (n=10 primary, n= 4 met) and PDAC (n=21 primary, n=5 met) cases were analyzed using the DSP platform. Sections were stained with an 82-protein cocktail of antibodies conjugated to UV-photocleavable DNA barcodes. A pan-cytokeratin morphology marker was used to select tumor regions of interest (ROI) and inferred surrounding stromal (i.e., pan-CK negative) ROIs. UV excitation of the defined ROIs was used to release the DNA barcodes and were quantified on the NanoString nCounter system. For PASC, squamous enriched ROI were inferred from corresponding p40 IHC. Corresponding clinical and mutational status for all cases were abstracted from medical records.
 Results:
 TP53 (PDAC, 80%; PASC, 83%) and KRAS (PDAC, 76%; PASC, 100%) were the most prevalent mutations
 .
 In comparing the tumor ROIs of PASC and PDAC (all cases), DSP-assessed global expression and activation of EGFR-MAPK signaling is significantly increased in PASC, whereas PI3K-AKT signaling is significantly decreased. Compared to PDAC, EGFR expression and phosphorylation (p-EGFR), as well as p-MEK1, are significantly increased in PASC. PASC showed significantly lower levels of PLCG1 and INPP4B, while the majority of other AKT signaling effectors were unchanged. For DNA damage response signaling, PASC showed significantly higher levels of p-ATM, p-CHK2, and p21, but not p-ATR. Likewise, expression of oncoproteins c-MYC and BCL6 were significantly increased in PASC (vs. PDAC).
 Conclusions:
 Given that the normal pancreas lacks squamous cells, their histological presence is a uniquely aberrant feature that can be used to guide deep spatially-resolved interrogation of complex cellular and molecular programs. Our preliminary findings show that regions of squamous cell differentiation within PASC have intrinsically higher levels of active EGFR signaling. With emerging RAS-targeted agents poised to alter the treatment landscape, high EGFR signaling in PASC suggest a possible compensatory resistance mechanism that may require additional targeting. Likewise, reliance on ATM signaling highlights this as preferential target for DDR-directed therapies. Building on these results with additional multiomic assays will inform strategies for improving the management of patients with PASC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b1608b4f56eb9c6e1216a160d63214aa03c88b" target='_blank'>
              Spatially resolved protein profiling in pancreatic adenosquamous carcinoma.
              </a>
            </td>
          <td>
            Adel Kardosh, Rosalie C. Sears, Zheng Xia, Eizabaru M Sasatomi, Brian Mau, Jonathan R Brody, Melissa Wong, Charles D. Lopez, E. Chen, Guillaume Pegna, Cristiane M Franca, John Muschler, Joanna Pucilowska, Shaun M Goodyear
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Amongst the various imaging techniques that provide surrogate tumor radiographic indications to aid in planning, monitoring, and predicting outcomes of therapy, ultrasound-guided photoacoustic imaging (US-PAI) is a promising non-ionizing modality based on endogenous blood (hemoglobin) and blood oxygen saturation (StO₂) contrast. Adaptation of US-PAI to the clinical realm requires macroscopic system configurations for adequate depth visualization. Methods: Here we present a vascular regional analysis (VRA) methodology of obtaining areas of low and high vessel density regions within the tumor (LVD and HVD respectively) by frequency domain filtering of macroscopic PA images. In this work, we evaluated the various vascular and oxygenation profiles of different murine xenografts of pancreatic cancer (AsPC-1, MIA PaCa-2, and BxPC-3) that have varying levels of angiogenic potentials and investigated the effects of receptor tyrosine kinase inhibitor (sunitinib) on the tumor microvessel density and StO₂. Results: The administration of sunitinib resulted in transient deoxygenation and reduction in vessel density within 72 h in two (AsPC-1 and MIA PaCa-2) of the three tumor types. Utilizing VRA, the regional change in StO2 (∆StO2) revealed the preferential targeting of sunitinib in LVD regions in only the AsPC-1 tumors. We also identified the presence of vascular normalization (validated through immunohistochemistry) in the sunitinib treated AsPC-1 tumors at day 8 post-treatment where a significant increases in HVD ∆StO2 (~20%) were seen following the 72-hour time point, indicative of improved vessel flow and functionality. Treated AsPC-1 vasculature displayed increased maturity and functionality compared to non-treated tumors on day 8, while these same metrics showed no conclusive evidence of vascular normalization in MIA PaCa-2 or BxPC-3 tumors. Conclusion: Overall, VRA as a tool to monitor treatment response allowed us to identify time points of vascular remodeling, highlighting its ability to provide insights into the tumor microenvironment for sunitinib treatment and other anti-angiogenic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6192ffe103ebd68ab445fab040a95ceda32e6826" target='_blank'>
              Vascular regional analysis unveils differential responses to anti-angiogenic therapy in pancreatic xenografts through macroscopic photoacoustic imaging
              </a>
            </td>
          <td>
            Allison Sweeney, Andrew Langley, Marvin Xavierselvan, R. T. Shethia, Patrick Solomon, Aayush Arora, S. Mallidi
          </td>
          <td>2025-01-27</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="High-grade gliomas are aggressive, primary, central nervous system tumors with low survival rates due to recurrence and resistance to current therapy models. Recent studies have highlighted the importance between the interaction of glioma cancer cells and cells of the tumor microenvironment (TME). Cancer stem cells and immune cells play a critical role in the TME of gliomas. TMEs in glioma include the perivascular TME, hypoxic TME, and invasive TME, each of which have evolved as our understanding of the involved cellular players has improved. This review discusses the multidimensional aspects of the current targeted therapies and interactions between glioma cells and the TME with specific focus on targeted immunotherapies. Understanding the complexities of the TME and elucidating the various tumor-cell interactions will be critical for facilitating the development of novel precision strategies, ultimately enabling better patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e6f6ef570f7a7d3e8fff16bd7c3861aa6a70" target='_blank'>
              Current Understanding Regarding the Glioma Microenvironment and Impact of the Immune System
              </a>
            </td>
          <td>
            Enes Demir, Deondra A. Montgomery, Ammar Saloum, Nasser Yaghi, Michael Karsy
          </td>
          <td>2025-03-07</td>
          <td>Neuroglia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most lethal brain cancer, with GBM stem cells (GSCs) driving therapeutic resistance and recurrence. Targeting GSCs offers a promising strategy for preventing tumor relapse and improving outcomes. We identify SUV39H1, a histone-3, lysine-9 methyltransferase, as critical for GSC maintenance and GBM progression. SUV39H1 is upregulated in GBM compared with normal brain tissues, with single-cell RNA-seq showing its expression predominantly in GSCs due to super-enhancer-mediated activation. Knockdown of SUV39H1 in GSCs impaired their proliferation and stemness. Whole-cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq data, we further demonstrated that knockdown of SUV39H1 altered chromatin accessibility in key genes associated with these pathways. Chaetocin, an SUV39H1 inhibitor, mimics the effects of SUV39H1 knockdown, reducing GSC stemness and sensitizing cells to temozolomide, a standard GBM chemotherapy. In a patient-derived xenograft model, targeting SUV39H1 inhibits GSC-driven tumor growth. Clinically, high SUV39H1 expression correlates with poor glioma prognosis, supporting its relevance as a therapeutic target. This study identifies SUV39H1 as a crucial regulator of GSC maintenance and a promising therapeutic target to improve GBM treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc58ec84c5d0f0ea9dd38953926222f20c648f30" target='_blank'>
              SUV39H1 maintains cancer stem cell chromatin state and properties in glioblastoma.
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Richard L Carpenter, Y. A. Wang, , Kenneth P Nephew, Jia Shen
          </td>
          <td>2025-03-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ac003cb05c58ff6a6e68723c112b42a007eb0f" target='_blank'>
              Neuronal mimicry in tumors: lessons from neuroscience to tackle cancer
              </a>
            </td>
          <td>
            Hanan Bloomer, Haley B Dame, Savannah R. Parker, Madeleine J Oudin
          </td>
          <td>2025-02-11</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background CD4+ T cells are a highly differentiated cell type that maintain enough transcriptomic plasticity to cycle between activated and memory statuses. How the 1D chromatin state and 3D chromatin architecture support this plasticity is under intensive investigation. Methods Here, we wished to test a commercially available in situ Hi-C kit (Arima Genomics Inc.) to establish whether published performance on limiting cell numbers from clonal cell lines copies across to a primary immune cell type. We achieved comparable contact matrices from 50,000, 250,000, and 1,000,000 memory CD4+ T-cell inputs. We generated multiple Hi-C and RNA-seq libraries from the same biological blood donors under three separate conditions: unstimulated fresh ex vivo, IL-2-only stimulated, and T cell receptor (TCR)+CD28+IL-2-stimulated, conferring increasingly stronger activation signals. We wished to capture the magnitude and progression of 3D chromatin shifts and correlate these to expression changes under the two stimulations. Results Although some genome organization changes occurred concomitantly with changes in gene expression, at least as many changes occurred without corresponding changes in expression. Counter to the hypothesis that topologically associated domains (TADs) are largely invariant structures providing a scaffold for dynamic looping contacts between enhancers and promotors, we found that there were at least as many dynamic TAD changes. Stimulation with IL-2 alone triggered many changes in genome organization, and many of these changes were strengthened by additional TCR and CD28 co-receptor stimulation. Conclusions This suggests a stepwise process whereby mCD4+ T cells undergo sequential buildup of 3D architecture induced by distinct or combined stimuli likely to “prime” or “deprime” them for expression responses to subsequent TCR-antigen ligation or additional cytokine stimulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1aa3f310b47dde1e0fdd44c2b4206007ebe632" target='_blank'>
              Memory CD4+ T cells sequentially restructure their 3D genome during stepwise activation
              </a>
            </td>
          <td>
            Alexander I. Ward, J. I. de las Heras, Eric C. Schirmer, A. Fassati
          </td>
          <td>2025-02-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), a highly aggressive brain tumor characterized by molecular hetero-geneity, necessitates integrative approaches to decipher driver pathways and therapeutic vulnerabilities. Here, we present gtePIDP, a computational framework that systematically integrates multi-omics data (genomic, transcriptomic, epigenetic) to identify Phenotypically Impactful Driver Pathways through dual evaluation of mutation patterns (coverage/exclusivity) and downstream regulatory cascades. Lever-aging mutation profiles and expression data from The Cancer Genome Atlas (TCGA), combined with transcription factor (TF) and miRNA regulatory networks, we constructed a systems biology model to map somatic alterations to phenotypic outcomes. By iteratively pruning candidate driver genes and quantifying resultant perturbations in gene expression and regulatory activities, gtePIDP prioritizes gene sets that maximize mutational significance and phenotypic impact. Our analysis uncov-ered pivotal GBM driver pathways involving TP53, CDKN2A, MDM2, and RB1, which disrupt key cancer-associated regulators (e.g., oncogenic: NFATC2, MIR-370 ; tumor-suppressive: MIR-506, MIR-9, FOXJ2), driving dysregulation of angiogenesis, apoptosis, and proliferation. Notably, we identified ther-apeutic axes such as TP53/MIR-185/VEGFA and MDM2/SMAD4/VEGFA, suggesting anti-angiogenic targeting strategies. Furthermore, a PI3K-Akt signaling regulatory module revealed actionable targets (CDKN2A/CREBBP/OSMR and TP53/MIR-9/FGF12) for intervention. By bridging mutational landscapes with transcriptional/epigenetic dysregulation, gtePIDP advances mechanistic driver pathway discovery and prioritizes targets with clinical relevance. This study highlights the power of multi-omics integration to unravel GBM pathogenesis and accelerate precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e24bbb19a6e12af52d533253b1371130f86db3" target='_blank'>
              Integrating genomic, transcriptomic and epigenetic data to identify phenotypically impactful driver pathways in glioblastoma
              </a>
            </td>
          <td>
            Junhua Zhang
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer metastasis, a process in which cancer cells migrate to secondary sites, accounts for 90% of cancer deaths. While machine learning has been used to predict metastasis in a variety of ways, they tend to be specific to tumor types or classification tasks. Here, we provide a pan-cancer machine learning model that is easily interpretable and directly links the transcriptomic and proteomic profiles of hundreds of cancer cell lines from DepMap with metastasis in a tissue-agnostic manner. We show that, for the relatively small sample size here, linear models perform just as well as non-linear models, including neural networks. We also demonstrate that transcriptomics outperforms proteomics for predictive tasks. However, the combination of both transcriptomics and proteomics leads to an overall increase in performance, indicating that these modalities can complement each other. Finally, we demonstrate that those features identified as most important by the joint omics linear model fit are over-represented in biological pathways related to metastasis, suggesting that the model learns relevant information that may be useful in identifying universal biomarkers or therapeutic targets of cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4778538fa7f9077b250641d66da47d7e300ae2d7" target='_blank'>
              Joint linear modeling of transcriptomics and proteomics is predictive of cancer metastasis
              </a>
            </td>
          <td>
            Ronit Sharma, N. Meimetis, Arjana Begzati, S. Nagar, Benjamin Kellman, Hratch M. Baghdassarian
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472095a4c3c0313c90e7e9724e09004bb9a2fb6" target='_blank'>
              From silent partners to potential therapeutic targets: macrophages in colorectal cancer
              </a>
            </td>
          <td>
            Hayat Khizar, K. Ali, Jianwei Wang
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer exhibits high morbidity and mortality rates, characterized by a heterogeneous genetic landscape and an immunosuppressive tumor microenvironment, despite recent therapeutic advances. High-resolution single-cell sequencing has facilitated a comprehensive analysis of malignant and host cell types, enhancing the understanding of lung cancer’s heterogeneity and adaptability under basal conditions and in response to therapeutic intervention. In this review, we provide an overview of the utilization of single-cell sequencing technology in characterizing the microenvironment of lung cancer, facilitating early diagnosis, monitoring tumor progression, and elucidating mechanisms of drug resistance, thereby offering new insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/787ec1e24f0f9b35e2346f89facf0c57ddb91948" target='_blank'>
              Dynamic roles of tumor-infiltrating B lymphocytes in cancer immunotherapy
              </a>
            </td>
          <td>
            Shishengnan Song, Chong Wang, Yangchao Chen, Xiaorong Zhou, Yi Han, Haijian Zhang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f7c17606ae67d5ce7fe611db0e7bce958444bb" target='_blank'>
              Decoding the heterogeneous subpopulations of glioblastoma for prognostic stratification and uncovering the promalignant role of PSMC2
              </a>
            </td>
          <td>
            Nini Zhou, Jingsong Yan, Manya Xiong, Shunqin Zhu
          </td>
          <td>2025-02-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Glioblastoma (GBM) recurrence is the rule rather than the exception. GBM cells invading healthy brain and the tumor microenvironment are believed to be responsible for tumor recurrence. However, little is known about molecular alterations in the primary tumor that drive this progression. Multi-omics studies have revealed specific tissue domains of GBM that could have higher invasive profiles in primary (pGBM) and recurrent (rGBM) tumors.



 Paraffin-embedded sample blocks from matched primary (n = 8) and recurrent (n = 10) GBM were utilized. Spatial transcriptomics (10x-Visium) and in situ molecular imaging (10x-Xenium) were performed. Matching fresh-frozen samples were processed for single-nucleus RNAseq (10x-Chromium). Visium spots were clustered together, and cells were annotated based on gene expression. Differential gene expression analysis, pathway enrichment analysis, and downstream analyses were performed.



 Spatial transcriptomics spots were grouped by transcriptomic similarity. In pGBM we identified that cluster 2 is significantly enriched with high expression of Neurocan (NCAN) (p < 0.001), a proteoglycan member of the brain extracellular matrix (ECM) that is only expressed during development and is absent in the adult brain. NCAN is known to increase proliferation and migration in glioma cells via the RHOA/ROCK pathway, representing an important target for GBM. Comparing spatial transcriptomic spots with high NCAN vs. Low NCAN shows upregulation of RHOA/ROCK and cell motility genes. In-situ molecular imaging confirms the distribution of ECM and cell motility gene transcripts.



 Patient-matched GBM samples can differ substantially within and between patients, but molecular signatures associated with tumor progression remain constant in subpopulations of cells. NCAN is a brain extracellular matrix component overexpressed in distinct tissue domains in pGBM with implications in cell invasion, representing a potential target that could hinder tumor recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b581086a365c6022e8cfc4df2d851d53411c3c94" target='_blank'>
              494 Multi-Omics Integration and Mapping of Invasion Markers Reveal Tumor-Specific Druggable Targets in Primary Human Glioblastoma
              </a>
            </td>
          <td>
            Paola Suarez-Meade, Jean R. Clemenceau, Isabel Barnfather, Mark Edgar, Aleeshba N. Basil, Tae H. Hwang, A. Quiñones-Hinojosa
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The efficacy of cancer immunotherapy is significantly influenced by the heterogeneity of individual tumors and immune responses. To investigate this phenomenon, a microfluidic platform is constructed for profiling immune-cancer cell interactions at the single-cell proteomics level for the first time. Based on the platform, a comprehensive workflow is proposed for achieving accurate single-cell pairing of an immune cell and a cancer cell with low cell damage and high success rate up to 95%, cell pair co-culture, and real-time microscopic monitoring of the cell-pair interactions, cell pair retrieval, mass spectrometry-based proteomic analysis of singe cell pairs, and decoupling of the proteomic information for each cell within the cell pair with the stable-isotope labeling method. With the workflow, the interactions of single natural killer (NK) cells and single K562 tumor cells are investigated based on real-time images and single cell-pair proteomics. Notably, an identification depth of over 1000 protein groups in a single cell-pair is achieved, leading to the discovery of sub-clusters of NK cells with different functions and the identification of important biomarkers for cancer treatments. This demonstrates the unique capability of the present platform in providing substantial and comprehensive datasets for profiling immune-cancer cell interactions, discovering heterogeneous immune responses, and predicting biomarkers in the study of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4858ff9b0301ae94fe60cd639ea84ada99e086" target='_blank'>
              Single Cell-Pair Proteomics for Decoding Immune-Cancer Cell Interactions.
              </a>
            </td>
          <td>
            Qin-Qin Xu, Yi-Rong Jiang, Jian-Bo Chen, Jie Wu, Yi-Xue Chen, Qian-Xi Fan, Huifeng Wang, Yi Yang, Jian-Zhang Pan, Qun Fang
          </td>
          <td>2025-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D Doherty, Brandon A Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Liver and lung metastases are major drivers of breast cancer-related morbidity, characterized by poor therapeutic outcomes and diminished overall survival. These dynamic tumor microenvironments (TMEs) display distinct, site-specific cellular architectures, including cytotoxic lymphocytes, immune suppressor cells, and stromal components, which together facilitate tumor growth and metastatic spread. Varied responses to immune checkpoint inhibition have also been observed in patients with differential tumor burden, improved responses in those with higher lung metastatic burden and poor responses in those with higher liver metastatic burden. Specific immune populations driving observed differential responses have not been fully characterized. Utilizing syngeneic orthotopic mouse models and FFPE tissue samples from patients with primary and metastatic breast cancer, we investigated the organ-specific cellular composition, cell-cell interactions, and molecular pathways underlying tumor progression, immune escape, and site-specific metastasis. Flow cytometry analysis of orthotopic murine primary breast, breast-to-liver, and breast-to-lung tumors demonstrated that CD3+ lymphocytes initially accumulate in primary tumors soon after tumor inoculation and decline significantly over time. Despite this overall lymphocyte depletion within TMEs, the lungs consistently exhibit the highest lymphocyte levels. In contrast, myeloid-derived suppressor cells (MDSCs) progressively infiltrate both primary and distant sites, exerting immunosuppressive functions that likely promote tumor progression and inhibit immune responses. Both granulocytic and monocytic MDSCs suppress CD8+ T cell proliferation through distinct organ-specific mechanisms, with the strongest suppressive effects observed in the liver TME. Validation of these findings in patient specimens using high-resolution spatial transcriptomics from breast and distant breast metastases using 10X Genomics (Visium and Visium-HD platforms) revealed that liver metastases harbor lower immune-to-stromal ratio with greater MDSCs: CD8+ colocalization, and higher expression of genes associated with proliferation, angiogenesis, anti-apoptosis and antiinflammation relative to primary breast tumors. In addition, spatiotemporal plasticity of breast TMEs is specific to the disease site, and investigation of how these observations could impact tumor progression and response to immune checkpoint inhibition is underway. In summary, tissue-specific programming of MDSCs in breast-to-liver metastasis compared with primary and breast-to-lung lesions leads to reduced T cell infiltration and function, and could contribute to a lack of response to checkpoint inhibition and could guide therapeutic options for patients.
 Citation Format: Batul Al-zubeidy, Edgar Gonzalez, Aaron Baugh, Dominic Zavala, Mathew Jacobo, Michael Press, Evanthia Roussos Torres. Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition? [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A086.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Melanoma brain metastasis is linked to dismal prognosis and low overall survival and is detected in up to 80% of patients at autopsy. Circulating tumor cells (CTC) are the smallest functional units of cancer and precursors of fatal metastasis. We previously used an unbiased multilevel approach to discover a unique ribosomal protein large/small subunit (RPL/RPS) CTC gene signature associated with melanoma brain metastasis. In this study, we hypothesized that CTC-driven melanoma brain metastasis secondary metastasis (“metastasis of metastasis” per clinical scenarios) has targeted organ specificity for the liver. We injected parallel cohorts of immunodeficient and newly developed humanized NBSGW (huNBSGW) mice with cells from CTC-derived melanoma brain metastasis to identify secondary metastatic patterns. We found the presence of a melanoma brain–liver metastasis axis in huNBSGW mice. Furthermore, RNA sequencing analysis of tissues showed a significant upregulation of the RPL/RPS CTC gene signature linked to metastatic spread to the liver. Additional RNA sequencing of CTCs from huNBSGW blood revealed extensive CTC clustering with human B cells in these mice. CTC:B-cell clusters were also upregulated in the blood of patients with primary melanoma and maintained either in CTC-driven melanoma brain metastasis or melanoma brain metastasis CTC–derived cells promoting liver metastasis. CTC-generated tumor tissues were interrogated at single-cell gene and protein expression levels (10x Genomics Xenium and HALO spatial biology platforms, respectively). Collectively, our findings suggest that heterotypic CTC:B-cell interactions can be critical at multiple stages of metastasis. Significance: This study provides important insights into the relevance of prometastatic CTC:B-cell clusters in melanoma progression, extends the importance of the CTC RPL/RPS gene signature beyond primary metastasis/melanoma brain metastasis driving targeted organ specificity for liver metastasis (“metastasis of metastasis”), and identifies new targets for clinical melanoma metastasis therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e29f46961d73d0219c05d2a7a74cfd6126ce9a2" target='_blank'>
              A Melanoma Brain Metastasis CTC Signature and CTC:B-cell Clusters Associate with Secondary Liver Metastasis: A Melanoma Brain–Liver Metastasis Axis
              </a>
            </td>
          <td>
            Tetiana Y Bowley, Mireya C Ortiz, Irina V Lagutina, Mara P. Steinkamp, Bridget N Fahy, Bernard Tawfik, Moises Harari-Turquie, Dario Marchetti
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Real-time imaging and monitoring of tumor boundaries are critical for understanding tumor growth dynamics and ensuring precise treatment. All physiological phenomena, specifically communicational, are inherently bioelectrical. Our original technology of “Skin Electrodynamic Introscopy” (SEI) firstly enabled time-lapse imaging of skin spectral electroimpedance landscape (SEL) with an unprecedented combination of temporal and spatial resolution. SEI enabled the revelation of several new phenomena in collective dynamics: the emergence of dynamic SEL structures in the melanoma region in response to weak short stimuli, such as ischemia and non-thermal EMF and MF. The main objective of this work was to identify criteria for assessing the functional boundaries of a tumor and its peritumor as a dynamic ecosystem. These boundaries are conceptualized as dissipative structures (DS)—self-organizing patterns that arise in nonequilibrium thermodynamic systems, with malignant tumors serving as a striking example. It is shown that the impedance boundaries of the tumor exceed its optical dimensions and can vary at different levels of the intra- and intercellular environment. It is proposed that the impedance boundaries are a quasi-stationary manifestation of entropy/negentropy flow from the tumor. The functional boundary can be assessed by revealing the pre-bifurcation zone of the tumor by measuring the increment of the impedance boundary in response to stimulation resulting from activation of the entropy/negentropy emitters. In this context, the invasive front of the tumor can be detected by the appearance of an antiphase impedance structure at its border. Thus, the previously discovered SEL phenomena, as well as a number of new ones, are considered the first demonstrations of DS at the tissue and cellular level. The tissue DS is determined primarily by increased group sensitivity/state of cells and their components, rather than by individual elements. Thus, insight into test-induced collective cellular behavior allows for a more nuanced (i.e. local and system) understanding of the tissue metabolism compared to traditional reductionist approaches. The dissipative nature of the observed phenomena is most effectively represented in fluctuation and correlation fields of the SEL, reflecting thus the fundamental concept of DS: the ‘Order from chaos’, which is especially relevant to tumor biology. In particular, this was demonstrated as: - Transition from initially chaotic SEL to the tumor specific patterns and back upon cessation of the stimulus; - Abrupt shift in magnitude and depth of the impedance front; - Manifestation of stroma of pre-existing nevus at the mitochondrial fluctuation field; - Emergence of the tumor resistance zone to the influence as a systemic post-effect, etc; Moreover, difference images of the SEL frames made it possible to trace the process of the DS emergence in the form of: - Wave-like dynamics around morphological boundaries of the tumor; - Expansion of initial functional microclusters at the tumor boundary; - Reverse migration of DS towards the negentropy emitters. From the ‘Order from chaos’ perspective, similarity was found between manifestations of weak information (EMF) and energy (ischemia) influence. Herewith we also attempted to estimate the information content of the spatio-temporal version of the ion balance index (as a ratio of the impedance modulus at 2 kHz and 1 MHz). In support of the DS hypothesis, the supplement presents analysis of two experiments: - Papillomatous nevus (as a relevant model for exploring the viral theory of cancer), which revealed similar and complementary effects of test-induced reorganization of the SEL; - Plant tissue (plant leaf), demonstrating the universal nature of the phenomenon of antiphase structuring. In summary, the SEL of melanoma, particularly in the context of the peritumor boundaries, reveals critical insights into how collective cellular dynamics influence tumor progression. The discovery of DS around tumors represents a significant advancement in our understanding of tumor biology. The developed approach and observed effects highlights new potential of advanced imaging techniques, it: - facilitates real-time monitoring and biofeedback in laboratory and clinical settings; - highlights the need for a paradigm shift in cancer research; - challenge traditional views on EMF exposure by demonstrating that non-thermal effects can induce significant biological responses at the tissue level; - calls for investigating tumors as dynamic entities shaped by their interactions with environmental factors, thereby opening up opportunities for innovative treatment approaches that target this adaptive behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad8fec02f869aff60be41da2c4550105b35eaada" target='_blank'>
              Functional Tumor Boundaries Revealed by Dynamic Electroimpedance Imaging Under Weak Stimuli: Dissipative Structures in Melanoma Area
              </a>
            </td>
          <td>
            Yuri F. Babich
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy.The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity.We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered.This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2f30006d88490586d800953c99040977f4d92b" target='_blank'>
              Differential cellular origins of the extracellular matrix of tumor and normal tissues according to colorectal cancer subtypes.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Sangkeun Park, Jun Hyeong Lee, Shin Young Chang, Sang Mi Oh, S. Mun, Junho Kang, Jong-Eun Park, Jung Kyoon Choi, Tae Il Kim, Jin Young Kim, Pilnam Kim
          </td>
          <td>2025-03-03</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analysed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi‐omics approach – encompassing genomic, transcriptomic, proteomic, epigenetic, single‐cell, and spatial analyses – we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF‐β and WNT signalling pathways, which are essential for tumour development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumour microenvironment. Elevated levels of POSTN were found to reorganise the extracellular matrix (ECM), interact with TGF‐β, disrupt cell cycle regulation and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single‐cell and spatial transcriptome data revealed that cancer‐associated fibroblasts (CAFs) secrete TGF‐β1/2, promoting cancer cell metastasis through epithelial–mesenchymal transition (EMT). Our integrated multi‐omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments. This article is protected by copyright. All rights reserved">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a9d23bd90d2baf9dbe230266b6f429b3c96696a" target='_blank'>
              Proteogenomic characterisation of primary oral cancer unveils extracellular matrix remodelling and immunosuppressive microenvironment linked to lymph node metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract A cell fate change such as tumorigenesis incurs critical transition. It remains a longstanding challenge whether the underlying mechanism can be unraveled and a molecular switch that can reverse such transition is found. Here a systems framework, REVERT, is presented with which can reconstruct the core molecular regulatory network model and a reversion switch based on single‐cell transcriptome data over the transition process is identified. The usefulness of REVERT is demonstrated by applying it to single‐cell transcriptome of patient‐derived matched organoids of colon cancer and normal colon. REVERT is a generic framework that can be applied to investigate various cell fate transition phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd1a9318f2abd76a8f7539133f45f2810ab7c74" target='_blank'>
              Attractor Landscape Analysis Reveals a Reversion Switch in the Transition of Colorectal Tumorigenesis
              </a>
            </td>
          <td>
            Dongkwan Shin, Jeong-Ryeol Gong, Seoyoon D Jeong, Youngwon Cho, Hwang‐Phill Kim, Tae-You Kim, Kwang-Hyun Cho
          </td>
          <td>2025-01-22</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Tumor hypoxia reshapes the microenvironment, driving progression, invasion, metastasis, and therapy resistance. Patient-derived tumor organoids, formed under three-dimensional conditions, preserve cellular heterogeneity and nutrient gradients, making them ideal for studying hypoxia-induced tumor responses. This study examines hypoxia-induced changes in lung tumor organoids from two patients, focusing on tumor-associated markers, stem cell markers, and migration capabilities.


RESULTS
Our findings demonstrate that hypoxia distinctively modulates the expression of lung cancer markers thyroid transcription factor-1, cytokeratin 7, and ΔNP63 variant. Hypoxia also induces the upregulation of stem cell-associated markers, resulting in an increased proportion of cancer stem cells. Furthermore, hypoxic lung tumor organoids exhibit unique migratory behavior upon reoxygenation, driven by epithelial-mesenchymal transition and the elevated expression of matrix metalloproteinases 7 and matrix metalloproteinases 9, indicating their enhanced invasive potential.


CONCLUSIONS
These findings highlight the value of lung tumor organoids as models for studying hypoxia's complex role in lung cancer. Hypoxia significantly modulates lung tumor organoids growth, stemness, and migratory behavior, providing critical insights into tumor progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c10484a1b1c15548c942fa338e1afd0e25fd59" target='_blank'>
              Lung tumor organoids migrate as cell clusters containing cancer stem cells under hypoxic condition.
              </a>
            </td>
          <td>
            Yanjiao Li, Jiarong Zou, Yanhua Fang, Jianing Zuo, Ruoyu Wang, Shanshan Liang
          </td>
          <td>2025-02-01</td>
          <td>Biology of the cell</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic activating mutations in KRAS can cause complex lymphatic anomalies (CLAs). However, the specific processes that drive KRAS-mediated CLAs have yet to be fully elucidated. Here, we used single-cell RNA sequencing to construct an atlas of normal and KrasG12D-malformed lymphatic vessels. We identified six subtypes of lymphatic endothelial cells (LECs) in the lungs of adult wild-type mice (Ptx3, capillary, collecting, valve, mixed, and proliferating). To determine when the LEC subtypes were specified during development, we integrated our data with data from four stages of development. We found that proliferating and Ptx3 LECs were prevalent during early lymphatic development and that collecting and valve LECs emerged later in development. Additionally, we discovered that the proportion of Ptx3 LECs decreased as the lymphatic network matured but remained high in KrasG12D mice. We also observed that the proportion of collecting and valve LECs was lower in KrasG12D mice than in wild-type mice. Last, we found that immature lymphatic vessels in young mice were more sensitive to the pathologic effects of KrasG12D than mature lymphatic vessels in older mice. Together, our results expand the current model for the development of the lymphatic system and suggest that KRAS mutations impair the maturation of lymphatic vessels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b83680aa397127e25da0c4fb9fc494573ba691b" target='_blank'>
              A single-cell atlas of normal and KRASG12D-malformed lymphatic vessels.
              </a>
            </td>
          <td>
            Lorenzo M. Fernandes, Danielle Griswold-Wheeler, Jeffrey D Tresemer, Angelica Vallejo, Neda Vishlaghi, Benjamin Levi, Abigail Shapiro, Joshua P. Scallan, Michael T. Dellinger
          </td>
          <td>2025-01-28</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84fc597c397918e1cba818cba7150e5110b7b75" target='_blank'>
              Functional genomic imaging (FGI), a virtual tool for visualization of functional gene expression modules in heterogeneous tumor samples
              </a>
            </td>
          <td>
            Xinlei Chen, Youbin Guo, Xiaorong Gu, Danyi Wen
          </td>
          <td>2025-01-21</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Effective medical therapies for metastatic paraganglioma (mPPGL) are currently lacking, leading to dismal prognosis. Building on our knowledge of molecular mechanisms driving PPGL progression, we assessed the therapeutic potential of targeting two critical processes: PI3K signaling and cell cycle regulation. The efficacy of buparlisib (PI3Ki) and ribociclib (CDK4/6i), individually and combined, was assessed in vitro using PPGL cell lines, rat– and patient-derived primary cells, and in vivo using PPGL cells-derived mouse xenografts. The combination therapy demonstrated superior antitumor activity compared to single agents, particularly in vivo. Mechanistically, the efficacy of the combination therapy was associated to the downregulation of FOXM1-controlled genes implicated in mitotic spindle assembly and chromosomal segregation, leading to mitotic catastrophe. Data mining and qRT-PCR showed this genetic signature to be upregulated in human mPPGLs. This suggests that aggressive PPGLs exhibit heightened vulnerability to dual PI3K and CDK4/6 inhibition, offering a promising therapeutic avenue for these challenging cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/312876965c04c1de5ff4c1cc2f5814c15a87434b" target='_blank'>
              A genetic signature predicts aggressive paraganglioma sensitivity to dual PI3K-CDK4/6 inhibition therapy
              </a>
            </td>
          <td>
            Bhargavi Karna, Federica Gentile, Sebastian Gulde, Daniela De Martino, Julian Maurer, Katyayni Ganesan, Katharina Wang, Nicole Bechmann, E. Rapizzi, Mercedes Robledo, Ester Arroba, Stephan Herzig, S. Nölting, Hermine Mohr, Natalia S Pellegata
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Asymmetric cell division (ACD) is a fundamental process to balance cell proliferation and differentiation during development and in the adult. Cancer stem cells (CSCs), a very small but highly malignant population within many human tumors, are able to provide differentiated progeny by ACD that contribute to the intratumoral heterogeneity, as well as to proliferate without control by symmetric, self-renewing divisions. Thus, ACD dysregulation in CSCs could trigger cancer progression. Here we consistently find low expression levels of RAP2A, the human homolog of the Drosophila ACD regulator Rap2l, in glioblastoma (GBM) patient samples, and observe that scarce levels of RAP2A are associated with poor clinical prognosis in GBM. Additionally, we show that restitution of RAP2A in GBM neurosphere cultures increases the ACD of glioblastoma stem cells (GSCs), decreasing their proliferation and expression of stem cell markers. Our results support that ACD failures in GSCs increases their spread, and that ACD amendment could contribute to reduce the expansion of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/518970f2ca9840fa8703b183e169e114b9e64218" target='_blank'>
              Human RAP2A Homolog of the Drosophila Asymmetric Cell Division Regulator Rap2l Targets the Stemness of Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Maribel Franco, Ricardo Gargini, Víctor M. Barberá, Miguel Saceda, A. Carmena
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 PURPOSE: With expanding immunotherapy options available in many cancers, particularly diffuse large B-cell lymphoma (DLBCL), methods for identifying individualized and subtype-specific immunotherapy sensitivity are increasingly important. Evaluation of immunotherapy within ex vivo settings is challenging due to the complexity in conserving tumor heterogeneity and its surrounding tumor microenvironment (TME). We have developed a combinatorial functional precision medicine platform, Optim.AI, a hybrid experimental-analytics platform that predictively ranks all actionable treatments within a drug set of interest. In prior clinical studies, Optim.AI has been demonstrated to guide cytotoxic and targeted treatments in lymphomas, including DLBCL. However, these clinical studies did not interrogate immunotherapy responses. In this study, we explored the integration of high-content screening workflows with Optim.AI to evaluate immunotherapy-focused drug sets in physiologically relevant multicellular models of DLBCL. METHODS: PBMCs and DLBCL cells were fluorescently labelled to facilitate multicellular tracking for high-content analysis. Optim.AI combinatorial drug sensitivity testing plates were then used to evaluate DLBCL cells, co-cultured with PBMCS, with up to 12 FDA-approved chemotherapies, immunomodulatory agent, monoclonal antibodies, antibody-drug conjugates, and bispecific antibodies. Utilizing real-time fluorescent apoptosis signal as the phenotypic readout, high-content analysis of tumor-specific cell death and apoptosis was evaluated at specific timepoints and used as the dataset of Optim.AI analytics and predictive ranking of combinatorial immunotherapy sensitivities in DLBCL cells. RESULTS: Optim.AI could confirm rituximab (CD20 mAb) sensitivity and optimal antibody-dependent cellular cytotoxicity (ADCC) lysis on CD20+ DLBCL cells at an Effector to Target ratio of 5:1 with a quantifiable phenotypic output at 48h post-treatment. Cytotoxicity studies performed across PBMCs from four different donors also showed negligible potential toxicity from drug candidates on immune cells, indicating that the platform can effectively and stably quantify ADCC. Furthermore, low ranking of tafasitamab (CD19 mAb) based combinations in a CD20+ cell line with low CD19 expression showed low target engagement with tafasitamab, confirming the platform’s ability to detect appropriate immunotherapy specificity. These findings collectively demonstrate the feasibility of pairing Optim.AI with an integrated co-culture model and image analysis pipeline to evaluate and rank immunotherapy-based combinations. CONCLUSIONS: We have demonstrated the integration of a high-content screening cancer-immune cell co-culture system with Optim.AI to evaluate immune-based combination therapies. Application of this approach towards ex vivo patient samples as well as appropriate cancer models will allow for guided immunotherapy-based treatment as well as identification of more effective cancer-specific and subtype-specific immunotherapy combinations.
 Citation Format: Sharon Pei Yi Chan, Jhin Jieh Lim, Masturah Rashid, Edward Kai-Hua Chow. Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A049.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Afatinib-induced tumor and microenvironment modifications in head and neck squamous cell carcinoma were evaluated by spatial transcriptomics in surgical specimens and RNA-sequencing in tumor biopsies of patients included in the EORTC-90111-24111 window-of-opportunity study. The aim was to explore tumor evolution and composition under anti-HER therapy. Based on our previous investigations by RNA-seq on tumor biopsies, surgical slides of ID08 and ID15 from the epithelial-to-mesenchymal (EMT) cluster and ID30 from the non-EMT cluster were investigated with spatial transcriptomics. Dimension reduction in ID30 revealed 14 clusters, with clusters overlapping three tumor nodules and the stroma. Differential expression analysis between tumor nodules showed enrichment of the hallmark EMT genelist, with 123 genes in common between the analyses. These genes were involved in PDGF and MET signaling pathways. By comparing gene expression in paired tumor biopsies and the 123 genes from differential analyses obtained in ID30, a list of 13 genes involved in cancer pathways and EMT emerged, which were also highly expressed in ID08 and ID15. These results show a progressive apparition of genes implicated in EMT, MET, and PDGF pathways in tumors after afatinib. Notably, a list of 13 genes emerged which may contain targets to prevent tumor evolution after anti-HER therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce3fcfb23899394262473ee209f0d7889b6808e" target='_blank'>
              Spatial Transcriptomic Analysis of Surgical Resection Specimens of Primary Head and Neck Squamous Cell Carcinoma Treated with Afatinib in a Window-of-Opportunity Study (EORTC90111-24111)
              </a>
            </td>
          <td>
            S. Beyaert, A. Loriot, J. Machiels, S. Schmitz
          </td>
          <td>2025-02-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Purpose: To look into the influence of DNAJC12 knockdown on the progression and radio-sensitivity of esophageal squamous cell carcinoma (ESCC), with a focus on cellular mechanics and tumor microenvironment interactions. Methods: The TCGA database combined with immunohistochemical staining was used to validate the DNAJC12 expression in ESCC patients from the perspective of the clinic. DNAJC12 knockdown was performed in TE-1 and KYSE-150 cell lines to assess changes in proliferation, migration, invasion, apoptosis, and cellular mechanical properties (e.g., stiffness, adhesion, and contractility). The downstream molecule regulated by DNAJC12 was explored using Western blotting and biomechanical assays. The effect of DNAJC12 knockdown on tumor radiosensitivity was evaluated in vivo, with a focus on tumor stiffness and extracellular matrix (ECM) remodeling under irradiated conditions. Results: Upon analyzing the TCGA database and examining tumor tissue samples from patients, it was discovered that DNAJC12 exhibited high expression levels in tissues of ESCC. Vitro experiments showed that DNAJC12 knockdown significantly decreased cellular proliferation and migration (P < 0.05). Biomechanical assays revealed that DNAJC12 knockdown decreased cellular stiffness and contractility, suggesting a role in regulating cytoskeletal dynamics. Molecular analysis showed downregulation of P-ERK, MMP-2, N-Cadherin, P-P38, Snail, Vimentin, β-Catenin, Fibronectin, and Twist alongside upregulation of E-Cadherin (P < 0.05). Overexpression of SNAI1 could restore the proliferative and migratory capabilities of cells with downregulated DNAJC12. In vivo experiments, knockdown of DNAJC12 resulted in faster tumor growth under irradiated conditions (P < 0.05). Conclusion: DNAJC12 knockdown slows ESCC progression by modulating cellular biomechanical properties and molecular pathways. However, it enhances tumor growth post-radiotherapy, potentially due to altered mechanosensitive signaling and ECM remodeling. These findings highlight the interplay between molecular biology and biomechanics in ESCC progression and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9884bfcb4a1529067272444d45e8b4fe6c21be" target='_blank'>
              Targeting FOXP1 phase separation in small cell lung cancer mechanisms of chemotherapy resistance.
              </a>
            </td>
          <td>
            Yichun Tang, Yuchun Niu, Yi Chen, Xuyang Zhou, Yueyang Hu, Lei Sun, Yan Xiong, Yue Xu, Qiongyao Wang, Yu Wang, Linlang Guo
          </td>
          <td>2025-03-13</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Rationale: Non-small cell lung cancer (NSCLC) is a predominant cause of cancer-related mortality, with its progression and treatment resistance significantly influenced by cancer stem cells (CSCs) and their complex intercellular communication mechanisms. Small extracellular vesicles (sEVs) have emerged as pivotal mediators of intercellular signaling, affecting tumor microenvironment modulation and therapeutic resistance. This study investigates the role of CSC-derived sEVs in transmitting stemness traits through the selective sorting of pyruvate kinase M2 phosphorylated at the Y105 site (pY105-PKM2), mediated by the adaptor protein IQGAP1, which supports CSC maintenance and drug resistance in NSCLC. Methods: In vitro and in vivo experiments, including proteomic and transcriptomic analyses, were conducted to identify key regulators of sEV-mediated signaling. Immunoprecipitation, proximity ligation assays, and immunofluorescence were used to examine the role of IQGAP1 in the sorting of pY105-PKM2 into sEVs. Functional assays, including sphere formation, chemoresistance tests, metabolic assessments, and cell cycle analysis, were conducted to evaluate the effects of sEV-mediated delivery of pY105-PKM2 on recipient cells. Additionally, immunohistochemistry and survival analysis were performed on tumor samples from NSCLC patients to establish clinical correlations. Results: We unveiled a novel mechanism by which CSC-derived sEVs transmit stemness traits to replenish the CSC pool in NSCLC. CSC-derived sEVs were enriched with pY105-PKM2, correlating with enhanced stemness, chemoresistance, and poor clinical outcomes. Mechanistically, IQGAP1 was identified as an adaptor facilitating the selective sorting of pY105-PKM2 into sEVs through interactions with the ESCRT component TSG101. Recipient cells treated with CSC-derived sEVs exhibited metabolic reprogramming, slower cell cycle progression, and enhanced chemoresistance. The synergistic role of IQGAP1 and pY105-PKM2 was confirmed, highlighting their critical contributions to CSC maintenance and malignant progression. Conclusion: This study highlights the critical role of CSC-derived sEVs in NSCLC progression and therapy resistance through the IQGAP1-mediated selective sorting of pY105-PKM2. By uncovering this novel pathway, our findings provide valuable insights into CSC pool replenishment and therapeutic resistance mechanisms in NSCLC, identifying IQGAP1 and pY105-PKM2 as promising therapeutic targets for mitigating CSC-driven malignancy and enhancing treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8844821c21b603caec632dd4e93148e05d6265" target='_blank'>
              A novel pathway for stemness propagation and chemoresistance in non-small cell lung cancer via phosphorylated PKM2-loaded small extracellular vesicles
              </a>
            </td>
          <td>
            Jingyi Wang, Liu Liu, Xinyu Gao, Xiyu Liu, Yitian Dai, Zijun Mao, Shengzhe Huang, Junjian Li, Dongliang Wang, Yu Qi, Yingwen Han, Yunjing Xu, C. Y. Chua, A. Grattoni, Wenhui Xie, Hao Yang, Gang Huang
          </td>
          <td>2025-02-24</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, Jin-yuan Liao, Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as patients with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We describe the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes are stratified along a naive → activated axis; patients with a 'high activated' signature score appear to be undergoing a robust melanoma antigen-driven adaptive immune response and thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise anti-tumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for high-risk melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies unique immune cell signatures associated with metastasis.
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7635dc0474dacb31cc3822f8f3055dfb7b7481" target='_blank'>
              Integrating transcriptomics and scPagwas analysis predicts naïve CD4 T cell-related gene DRAM2 as a potential biomarker and therapeutic target for colorectal cancer
              </a>
            </td>
          <td>
            Rui Feng, Xiaofang Li, Benhua Li, Tiankuo Luan, Jiaming He, Guojing Liu, Jian Yue
          </td>
          <td>2025-02-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f8ae5d4748b0e6bd25320bd0093825226598117" target='_blank'>
              Mapping the topography of spatial gene expression with interpretable deep learning.
              </a>
            </td>
          <td>
            Uthsav Chitra, Brian J. Arnold, Hirak Sarkar, Kohei Sanno, Cong Ma, Sereno Lopez-Darwin, Benjamin J. Raphael
          </td>
          <td>2025-01-23</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) progression is associated with an increase in PROX1+ tumor cells, which exhibit features of CRC stem cells and contribute to metastasis. Here, we aimed to provide a better understanding to the function of PROX1+ cells in CRC, investigating their progeny and their role in therapy resistance. PROX1+ cells in intestinal adenomas of ApcMin/+ mice expressed intestinal epithelial and CRC stem cell markers, and cells with high PROX1 expression could both self-renew tumor stem/progenitor cells and contribute to differentiated tumor cells. Most PROX1-lineage traced tumor cells were stem/progenitor cells, which can supply cells to multiple intestinal tumor cell lineages, whereas most lineage-traced LGR5+ tumor cells were enterocytes, indicating that PROX1+ and LGR5+ tumor stem cells have distinct differentiation programs. Although the PROX1+ tumor cells proliferated slower than PROX1- cells, irradiation increased the proportion of PROX1+ cells in human CRC cell lines, patient-derived organoids, and tumor xenografts. Furthermore, transcripts related to DNA damage repair (DDR) were enriched in PROX1+ vs. PROX1- cells in adenomas and in CRC tumor cells from patients. Experiments with PROX1 silencing and overexpression indicated that PROX1 expression enhances CRC cell colony formation following irradiation. PROX1 interacted with DDR proteins, including components of non-homologous end-joining (NHEJ) and base excision repair, and inhibition of NHEJ repair led to a decreased proportion of PROX1+ cells following irradiation. In conclusion, PROX1+ cells are irradiation-resistant tumor stem/progenitor cells capable of self-renewal and differentiation. DDR inhibitors could represent a strategy to target the treatment-resistant PROX1+ tumor stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a5d95b617651776586e68c84818be489675ad01" target='_blank'>
              A Multipotent PROX1+ Tumor Stem/Progenitor Cell Population Emerges During Intestinal Tumorigenesis and Mediates Radioresistance in Colorectal Cancer.
              </a>
            </td>
          <td>
            Pauliina Kallio, Cinzia Bessone, Fatemeh Seyednasrollah, Jefim Brodkin, Marika Lassila, Jenny Högström, A. González-Loyola, Tatiana V. Petrova, C. Haglund, Kari Alitalo
          </td>
          <td>2025-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of a complex mix of cellular and non-cellular components, including immune cells, stromal cells, extracellular matrix, cytokines, and growth factors. These elements interact with tumor cells to influence tumorigenesis, growth, invasion, and metastasis. Long noncoding RNAs (lncRNAs)—a class of non-coding RNAs longer than 200 nucleotides—have attracted considerable attention for their roles in regulating gene expression at the epigenetic, transcriptional, and post-transcriptional levels. Emerging evidence suggests that lncRNAs are crucial in shaping the TME by modulating processes such as immune evasion, angiogenesis, metabolic reprogramming, and the maintenance of cancer stem cells. This review provides an overview of the current understanding of lncRNAs in the TME, focusing on their involvement in key signaling pathways and cellular interactions that drive tumor progression. We discussed how lncRNAs contribute to extra-cellular matrix remodeling, facilitate communication between tumor and stromal cells, and regulate immune cell infiltration and function within the TME. Additionally, we explore the potential of lncRNAs as biomarkers for early cancer detection and prognosis, as well as their promise as therapeutic targets to disrupt tumor-microenvironment crosstalk. The review also addresses challenges in targeting lncRNAs therapeutically, such as ensuring specificity, minimizing off-target effects, and achieving effective in vivo delivery of lncRNA-targeted therapies. Strategies to overcome these challenges include the development of highly specific lncRNA knockout technologies and the use of advanced delivery systems, such as nanoparticles and viral vectors, to precisely target tumor-associated cells. Overall, this review underscores the significant role of lncRNAs in the TME and their potential as novel tools for enhancing cancer diagnosis and treatment. By elucidating the multifaceted roles of lncRNAs in the TME, we aimed to provide insights that could lead to more effective, targeted therapeutic strategies, ultimately advancing cancer research and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf1f9b2f1f2c54afc524f991ea0518f53bf5a2b" target='_blank'>
              Tumor Microenvironment Dynamics: The Regulatory Influence of Long Non-coding RNAs
              </a>
            </td>
          <td>
            O. Beylerli, Elmar Musaev, Chunlei Wang, Irina Popova, I. Gareev
          </td>
          <td>2025-02-22</td>
          <td>Gene Expression</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive form of primary brain tumor. The infiltrative nature of GBM makes complete surgical resection impossible. The selective forces that govern gliomagenesis are strong, shaping the composition of tumor cells during the initial progression to malignancy with late consequences for invasiveness and therapy response. Here, we developed a mathematical model that incorporates ploidy level and the nature of the brain tissue microenvironment to simulate the growth and invasion of GBM and used the model to make inferences about GBM initiation and response to standard-of-care treatment. The spatial distribution of resource access in the brain was approximated through integration of in silico modeling, multi-omics data, and image analysis of primary and recurrent GBM. The in silico results suggested that high ploidy cells transition faster from oxidative phosphorylation to glycolysis than low ploidy cells because they are more sensitive to hypoxia. Between surgeries, simulated tumors with different ploidy compositions progressed at different rates; however, whether higher ploidy predicted fast recurrence was a function of the brain microenvironment. Historical data supported the dependence on available resources in the brain, as shown by a significant correlation between the median oxygen levels in human tissues and the median ploidy of cancers that arise in the respective tissues. Taken together, these findings suggest that availability of metabolic substrates in the brain drives different cell fate decisions for cells with different ploidy, thereby modulating both gliomagenesis and GBM recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard J Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="INTRODUCTION
Metastasis is a principal cause of patient morbidity and death from solid cancers with current therapies being inadequate.


AREAS COVERED
Detailed genomic analyses document mutational differences between the initial tumor and metastatic clones, posing a challenge to current targeted therapies, which focus predominantly on the phenotype of primary tumors. Considering the diverse signaling cascades and numerous compensatory pathways in metastasis, designing broad-spectrum anti-metastatic therapies remains challenging. Although significant anti-cancer activity is evident in specific patients with advanced cancers and metastases treated with single or combination immunotherapies, there are limitations, i.e. toxicity, immune inhibitory 'cold' tumors and the tumor microenvironment (TME), and intra- and intertumoral heterogeneity. Accordingly, multidisciplinary strategies are required to attack metastases and the TME to obtain optimal therapeutic responses.


EXPERT OPINION
To create potent anti-metastatic agents, defining critical genes/proteins and drugs controlling discrete steps in the metastatic cascade are mandatory. Melanoma differentiation-associated gene-9 (MDA-9), Syndecan Binding Protein (SDCBP) or Syntenin (MDA-9/Syntenin) is robustly expressed and serves essential roles in cancer disease progression through protein-protein interactions with additional metastasis-associated molecules and pathways. The importance of MDA-9/Syntenin in the metastatic process is now established and first-in-class inhibitory molecules look promising with some moving toward clinical evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac5e3595c3552c277cd820f66cb6eb10faa0266" target='_blank'>
              MDA-9/Syntenin as a therapeutic cancer metastasis target: current molecular and preclinical understanding.
              </a>
            </td>
          <td>
            Swadesh K. Das, Paul B Fisher
          </td>
          <td>2025-03-08</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The ESCRT (endosomal sorting complex required for transport) machinery is essential for various cellular processes, yet its role in head and neck squamous cell carcinoma (HNSCC) is poorly understood. We utilized The Cancer Genome Atlas (TCGA) datasets to analyze the expression of ESCRT genes. Bulk RNA-sequencing data and HNSCC tissue microarrays (TMAs) were used to evaluate VPS25 expression and its clinical significance. Single-cell RNA sequencing of tumor tissues and VPS25 knockdown experiments in CAL27 cells were used to investigate its biological functions. Immunohistochemistry, spatial transcriptomics, and immunotherapy datasets highlighted the involvement of VPS25 in immune suppression and its potential as a predictive biomarker. The results demonstrated significant VPS25 overexpression in HNSCC tissues, which correlated with poor clinical outcomes. It promoted tumor cell proliferation and migration while reducing immune cell infiltration in the tumor microenvironment (TME). Additionally, by upregulating PVR expression in tumor cells, VPS25 activated the immunosuppressive PVR-TIGIT signaling axis, thereby facilitating immune evasion. Furthermore, VPS25 emerged as a potential biomarker for predicting immunotherapy response. These findings highlight VPS25 as a pivotal regulator of tumor progression and immune evasion in HNSCC and a promising target for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902195b920715f84e3501a67211a1883f6d279dd" target='_blank'>
              VPS25 Promotes an Immunosuppressive Microenvironment in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Liguo Chen, Yu-Han Fang, Kui-Ming Wang, Wei Zhang, Gang Chen
          </td>
          <td>2025-02-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90df80a01384ee300b159e30e515e05880476d0f" target='_blank'>
              Artificial intelligence-based spatial analysis of tertiary lymphoid structures and clinical significance for endometrial cancer
              </a>
            </td>
          <td>
            Haruka Suzuki, Kohei Hamada, J. Hamanishi, Akihiko Ueda, R. Murakami, Mana Taki, Rin Mizuno, Koichi Watanabe, Hanako Sato, Y. Hosoe, Hiroaki Ito, Koji Yamanoi, Hiroyuki Yoshitomi, N. Kakiuchi, Ken Yamaguchi, N. Matsumura, Seishi Ogawa, Hideki Ueno, Masaki Mandai
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Building a virtual cell requires comprehensive understanding of protein network dynamics of a cell which necessitates large-scale perturbation proteome data and intelligent computational models learned from the proteome data corpus. Here, we generate a large-scale dataset of over 38 million perturbed protein measurements in breast cancer cell lines and develop a neural ordinary differential equation-based foundation model, namely ProteinTalks. During pretraining, ProteinTalks gains a fundamental understanding of cellular protein network dynamics. Our model encodes protein networks and exhibits consistently improved predictive accuracy across various downstream tasks, highlighting its generalization capabilities and adaptability. In cancer cells, ProteinTalks robustly predicts drug efficacy and synergy, identifies novel drug combinations, and, through its interpretability, uncovers resistance-associated proteins. When applied to more complex system, patient-derived tumor xenografts, ProteinTalks predicts potential responses to drugs. Its integration with clinical patient data enhances the prognosis prediction of breast cancer patients. Collectively, we present a foundational model based on proteome dynamics, offering potential for various downstream applications, including drug discovery, and providing a basis for developing virtual cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ace5db272ec20ba27e0261b6cabb0d1bf5841bb" target='_blank'>
              A perturbation proteomics-based foundation model for virtual cell construction
              </a>
            </td>
          <td>
            Rui Sun, Liujia Qian, Yongge Li, Honghan Cheng, Zhangzhi Xue, Xuedong Zhang, Lingling Tan, Yuecheng Zhan, Wenbin Hu, Qi Xiao, Zhiwei Liu, Guangmei Zhang, E. Weinan, Peijie Zhou, Han Wen, Y. Zhu, Tiannan Guo
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Solid tumors develop within a complex environment called the tumor microenvironment (TME), which is sculpted by the presence of other cells, such as cancer-associated fibroblasts (CAFs) and immune cells like macrophages (Mφs). Despite the presence of immune cells, tumor cells orchestrate a tumor-supportive environment through intricate interaction with the components of the TME. However, the specific mechanism by which this intercellular dialogue is regulated is not fully understood. To that end, the development of an organotypic 3D breast TME-on-a-chip (TMEC) model, integrated with single-cell RNA sequencing analysis, is reported to mechanistically evaluate the progression of triple-negative breast cancer (TNBC) cells in the presence of patient-derived CAFs and Mφs. Extensive functional assays, including invasion and morphometric characterization, reveal the synergistic influence of CAFs and Mφs on tumor cells. Furthermore, gene expression and pathway enrichment analyses identify the involvement of the KYNU gene, suggesting a potential immune evasion mechanism through the kynurenine pathway. Lastly, the pharmacological targeting of the identified pathway is investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8de452a3f7784c1593c1ff6c237b52e31623e79" target='_blank'>
              Tumor Microenvironment On-A-Chip and Single-Cell Analysis Reveal Synergistic Stromal-Immune Crosstalk on Breast Cancer Progression.
              </a>
            </td>
          <td>
            Kalpana Ravi, Yining Zhang, Lydia Sakala, Twinkle Jina Minette Manoharan, Barbara Pockaj, Joshua LaBaer, Jin G. Park, Mehdi Nikkhah
          </td>
          <td>2025-03-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Activation of invasion and metastasis is a central hallmark of cancer, contributing to the primary cause of death for cancer patients. In the multistep metastatic process, cancer cells must infiltrate the circulation, survive, arrest at capillary beds, extravasate, and form metastatic clones in distant organs. However, only a small proportion of circulating tumor cells (CTCs) successfully form metastases, with transit of CTCs in the circulation being the rate-limiting step. The fate of CTCs is influenced by the circulating tumor microenvironment (cTME), which encompasses factors affecting their biological behaviors in the circulation. This liquid and flowing microenvironment differs significantly from the primary tumor microenvironment or the pre-metastatic niche. This review summarizes the latest advancements in identifying the biophysical cues, key components, and biological roles of the cTME, highlighting the network among biophysical attributes, blood cells, and non-blood factors in cancer metastasis. In addition to the potential of the cTME as a therapeutic target for inhibiting metastasis, the cTME could also represent as a biomarker for predicting patient outcomes and developing strategies for treating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ebef94ee6fd9d5b0a0f131941d243a6a93e28d" target='_blank'>
              Circulating Tumor Microenvironment in Metastasis.
              </a>
            </td>
          <td>
            Rui Tang, Shujuan Luo, Hui Liu, Yan Sun, Manran Liu, Lu Li, Haoyu Ren, Martin K Angele, N. Börner, Keda Yu, Zufeng Guo, Guobing Yin, Haojun Luo
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbing Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bb874396859196c4013ee0fce50222acff4e7b2" target='_blank'>
              Neutrophils physically interact with tumor cells to form a signaling niche promoting breast cancer aggressiveness.
              </a>
            </td>
          <td>
            Sandra Camargo, Ori Moskowitz, Amir Giladi, Maiia Levinson, Roi Balaban, Shani Gola, Alice Raizman, Kelly Lipczyc, Alon Richter, Noa Keren-Khadmy, Oren Barboy, Yael Dugach, Yaron Carmi, A. Sonnenblick, Merav Cohen
          </td>
          <td>2025-03-07</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b339228b07860b7f7b0ff107251f16ed54dc97" target='_blank'>
              Tumor microenvironment as a novel therapeutic target for lymphoid leukemias.
              </a>
            </td>
          <td>
            Shahrzad Mousavi, Soheil Nouri, Arezoo Sadeghipour, Amir Atashi
          </td>
          <td>2025-02-25</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma is a paediatric extracranial solid cancer that arises from the developing sympathetic nervous system and is characterised by an abnormal distribution of cell types in tumours compared to healthy infant tissues. In this paper, we propose a new mathematical model of cell differentiation during sympathoadrenal development. By performing Bayesian inference of the model parameters using clinical data from patient samples, we show that the model successfully accounts for the observed differences in cell type heterogeneity among healthy adrenal tissues and four common types of neuroblastomas. Using a phenotypically structured model, we show that alterations in healthy differentiation dynamics are related to cell malignancy, and tumour volume growth. We use this model to analyse the evolution of malignant traits in a tumour. Our findings suggest that normal development dynamics make the embryonic sympathetic nervous system more robust to perturbations and accumulation of malignancies, and that the diversity of differentiation dynamics found in the neuroblastoma subtypes lead to unique risk profiles for neuroblastoma relapse after treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9629af3f9c8496403aa9d7445a8f7f79cca8a69" target='_blank'>
              Genes and proteins expression profile of 2D vs 3D cancer models: a comparative analysis for better tumor insights.
              </a>
            </td>
          <td>
            Sunaina Bhuker, Abhinav Kumar Sinha, Anuksha Arora, H. Tuli, Sonal Datta, A. Saini, R. Saini, S. Ramniwas
          </td>
          <td>2025-01-25</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 The reactivation of neurodevelopmental programs in cancer highlights parallel biological processes that occur in both normal development and brain tumors. Achieving a deeper understanding of how dysregulated developmental factors play a role in the progression of brain tumors is therefore crucial for identifying potential targets for therapeutic interventions. Single-cell RNA sequencing (scRNA-Seq) provides an opportunity to understand how developmental programs are dysregulated and reinitiated in brain tumors at single-cell resolution. The aim of this study is to identify developmental origins of brain tumors using scRNA-Seq data.



 Here, we introduce COORS (Cell Of ORigin like CellS), a computational tool trained on developmental human brain single-cell datasets that annotates “developmental-like” cell states in brain tumors. COORS leverages cell type-specific multilayer perceptron models and incorporates a developmental cell type tree that reflects hierarchical relationships and models cell type probabilities.



 Applying COORS to various brain cancer datasets, including medulloblastoma (MB), glioma, and diffuse midline glioma (DMG), we identified developmental-like cells that represent putative cells of origin in these tumors. Our method provides both cell of origin classification and cell age regression, offering insights into the developmental cell types of tumor subgroups. COORS identified outer radial glia (oRG) developmental cells within IDHWT glioma cells whereas oligodendrocyte precursor cells (OPCs) and neuronal-like cells in IDHMut. Interestingly, IDHMut subgroup cells that map to OPC show bimodal distributions, that are both early and late weeks in development. Furthermore, COORS offers a valuable resource by providing novel markers linked to developmental states within of MB, glioma and DMG tumor subgroups.



 Our work adds to our cumulative understanding of brain tumor heterogeneity and helps pave the way for tailored treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e403b1c16cfb5c2e03317b6d1828fc8f0bdaf21" target='_blank'>
              Inferred Developmental Origins of Brain Tumors from Single Cell RNA-Sequencing Data
              </a>
            </td>
          <td>
            Su Wang, R. Curry, Malcolm F McDonald, Hyun Yong Koh, Anders W Erickson, Claudia L Kleinman, Michael D. Taylor, Ganesh Rao, Benjamin Deneen, Arif O. Harmanci, A. Harmanci
          </td>
          <td>2025-01-23</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 On-treatment biomarkers for assessing the response to immune-oncology (I/O) therapy have a better predictive value than baseline markers, prior to treatment1. The Farcast TruTumor histoculture platform that preserves the near native tumor microenvironment has the potential to identify such biomarkers post treatment with I/O therapy2. Accessing fresh tumor sample is often challenging, limiting the clinical utility of such ex-vivo platforms. To address this limitation, we assessed exosomes from explant culture supernatants post-treatment, for identifying response biomarkers in the culture supernatant that mimic the TME response and will seek to validate these in exosomes from matched patient plasma. Head and Neck Squamous Cell Carcinoma (HNSCC) surgical excess samples along with matched blood were used for this study. Explants generated from these samples treated with anti-PD1, Nivolumab (132µg/ml) for 72 hours, and response evaluated using flow cytometry and cytokine release analysis. Exosomes were isolated from culture supernatant and plasma using a combination of precipitation and CD63/81/9 coated magnetic bead-based purification. Exosome size was verified using nano particle tracking and further characterized using multicolor flowcytometry analysis. We observed similar distribution patterns of exosomes from the culture supernatant and matched plasma, with highest proportion of CD63+ exosomes. We evaluated exosomal composition of two samples (S1 and S2) that responded differentially to anti-PD1 treatment. S1 displayed strong anti-PD1 treatment driven T cell reinvigoration phenotype along with tumor cytolytic activity, while S2 did not. There was a treatment induced increase in T cell derived exosome (CD3+, TCex) and Interferon gamma (IFNg) positive TCex in S1 but a decrease of both in S2, mirroring the strong response phenotype in S1. We observed a decrease in PD1/PDL1 ratio in exosomes from the culture supernatant of S1 (∼1.5 fold) indicative of decreased exhaustion. This correlated with increase in IFNg secretion on treatment. The PD1/PDL1 ratio remained unchanged in S2. Additionally, TCex/ M2 like macrophage derived exosome (CD206+, MPex) ratio was higher in S1 (1.8) compared to S2 (0.7) upon treatment, indicative of an M2 like macrophage driven immunosuppressive TME that explained the non-responsive phenotype in S2. A similar picture emerged on immune cell profiling, wherein, on treatment S1 had a higher T cell/ M2 macrophage ratio (18.4) compared to S2 (1.9). The data presented from the TruTumor ex vivo platform illustrates good concordance between treatment induced changes in immune cell derived exosome composition and intra-tumoral immune cell profile. This platform presents a unique opportunity for identification of exosomal biomarkers that could be potentially tracked in periphery. 1. Chen, P.L et al. Cancer Discov. 6, 827–837 (2016) 2. Basak, NP et al. Nat Commun 15, 1585 (2024).
 Citation Format: Kowshik Jaganathan, Moumita Nath, Biswajit Das, Méhul Kapur, Christopher Davitt, Ritu Malhotra, Mohit Malhotra, Nandini Pal Basak, Satish Sankaran. Identification of tumor microenvironment informed exosome biomarkers for predicting response to anti-PD1 therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207b1f32a226a188a0a977b012a6e229f1073271" target='_blank'>
              Abstract B030: Identification of tumor microenvironment informed exosome biomarkers for predicting response to anti-PD1 therapy
              </a>
            </td>
          <td>
            Kowshik Jaganathan, Moumita Nath, Biswajit Das, Méhul Kapur, Christopher Davitt, Ritu Malhotra, Mohit Malhotra, N. Basak, Satish Sankaran
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c81eee6b668168bd971313aa0fe51bd578f98e" target='_blank'>
              Meta-analyses of mouse and human prostate single-cell transcriptomes reveal widespread epithelial plasticity in tissue regression, regeneration, and cancer
              </a>
            </td>
          <td>
            Luis C. Aparicio, Laura Crowley, John R. Christin, Caroline J. Laplaca, H. Hibshoosh, Raul Rabadan, Michael M. Shen
          </td>
          <td>2025-01-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The development of therapies that target cancer stem cells (CSCs) and bulk tumors is both crucial and urgent. Several signaling pathways, like Notch and Hedgehog (Hh), have been strongly associated with CSC stemness maintenance and metastasis. However, the extensive crosstalk present between these two signaling networks complicates the development of long-term therapies that also minimize adverse effects on healthy tissues and are not overcome by therapy resistance from CSCs. The present work aims to overview the roles of Notch and Hh in cancer outburst and the intersection of the two pathways with one another, as well as with other networks, such as Wnt/β-catenin, TGF, and JAK/STAT3, and to explore the shaping of the tumor microenvironment (TME) with specific influence on CSC development and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f140e4496eee327489d912de5427944cb351ca1" target='_blank'>
              Notch and Hedgehog Signaling Unveiled: Crosstalk, Roles, and Breakthroughs in Cancer Stem Cell Research
              </a>
            </td>
          <td>
            Sabina Iluta, Mădălina Nistor, Sanda Buruiana, D. Dima
          </td>
          <td>2025-02-01</td>
          <td>Life</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Antigen processing and presentation via major histocompatibility complex (MHC) molecules are central to immune surveillance. Yet, quantifying the dynamic activity of MHC class I and II antigen presentation remains a critical challenge, particularly in diseases like cancer, infection and autoimmunity where these pathways are often disrupted. Current methods fall short in providing precise, sample-specific insights into antigen presentation, limiting our understanding of immune evasion and therapeutic responses. Here, we present PSAA (PINN-empowered Systems Biology Analysis of Antigen Presentation Activity), which is designed to estimate sample-wise MHC class I and class II antigen presentation activity using bulk, single-cell, and spatially resolved transcriptomics or proteomics data. By reconstructing MHC pathways and employing pathway flux estimation, PSAA offers a detailed, stepwise quantification of MHC pathway activity, enabling predictions of gene-specific impacts and their downstream effects on immune interactions. Benchmarked across diverse omics datasets and experimental validations, PSAA demonstrates a robust prediction accuracy and utility across various disease contexts. In conclusion, PSAA and its downstream functions provide a comprehensive framework for analyzing the dynamics of MHC antigen presentation using omics data. By linking antigen presentation to immune cell activity and clinical outcomes, PSAA both elucidates key mechanisms driving disease progression and uncovers potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/582181263bb354bc38b72cdfdf043ee2535ac41b" target='_blank'>
              A physics informed neural network approach to quantify antigen presentation activities at single cell level using omics data
              </a>
            </td>
          <td>
            Chi Zhang, Jia Wang, Pengtao Dang, Yuhui Wei, Xiao Wang, Julie A. Brothwell, Yifan Sun, Haiqi Zhu, K. So, Jing Liu, Yijie Wang, Xiongbin Lu, Stanley M Spinola, Xinna Zhang, Sha Cao
          </td>
          <td>2025-01-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Grade 3 ependymoma is classified as a fast-growing solid tumor that often occurs in the base of the brain with a median survival of less than two years. Despite aggressive standard treatments (surgery, radiation therapy and chemotherapy), grade 3 ependymoma remains uniformly fatal, especially if the tumor cannot be completely removed or if it spreads to other areas of the central nervous system (CNS). Achieving a responsive therapy with chimeric antigen receptor (CAR) T cell therapy has been a significant challenge due to evasive tumor microenvironment, specifically utilized by slid tumors, to resist therapy. Recent studies have highlighted the substantial role of glioma-associated stromal cells in solid tumors by depositing various collagen subunits, including COL1A1, within the extracellular matrix (ECM). This collagen deposition leads to increased ECM stiffness, modulating the tumor microenvironment, inducing immune suppression (e.g. SPP1 overexpression), and hindering T cell trafficking, ultimately worsening clinical outcomes and patient survival. To gain a better understanding of the role of the tumor microenvironment in CAR T persistence and efficacy, we designed and performed ultra-high plex immunofluorescent staining on tissue biopsies obtained from a patient with grade 3 ependymoma before and after treatment with IL13Ra2 CAR T cells. The patient has been received three CAR T infusions at weekly intervals. The tissues were stained for CD3, CD19 (CAR T cells), CD68, CD163, HLA-DR (macrophages), CD3, CD8 (endogenous T cells), SOX2 (stem cells), SPP1 (suppression) and COL1A1 (extracellular matrix). The presence of CAR T cells in tissue biopsies indicated persistence of IL13Ra2 CAR T cells for more than a year. Our preliminary analysis revealed an increase in the number of HLA-DR+ CD163+ macrophages within cellular neighborhoods associated with CAR T cells (around 50% of CD163+ express HLA-DR after treatment) as compared to baseline tumors, which suggests a dynamic interaction between myeloid subsets and CAR T cells within the tumor microenvironment. Ongoing experiments are currently investigating the presence of antigen presenting natural killer cells (CD56, HLA-DR), proliferative T cells (CD3, Ki67) and inflammatory macrophages and/or T cells (IFNg) to gain a better understanding of IL13Ra2 persistence in brain tumor patients.
 Citation Format: Maryam Aftabizadeh, Brenda Aguilar, Massimo D'Apuzzo, Behnam Badie, Christine E Brown. A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cell therapy has emerged as a promising treatment for hematological malignancies, offering a targeted approach to cancer treatment. Understanding the complexities of CAR-T cell therapy within solid tumors poses challenges due to the intricate interactions within the tumor microenvironment. Mathematical modeling may serve as a valuable tool to unravel the dynamics of CAR-T cell therapy and improve its effectiveness in solid tumors. This study aimed to investigate the impact of spatial aspects in CAR-T therapy of solid tumors, utilizing cellular automata for modeling purposes. Our main objective was to deepen our understanding of treatment effects by analyzing scenarios with different spatial distributions and varying the initial quantities of tumor and CAR-T cells. Tumor geometry significantly influenced treatment efficacy in-silico, with notable differences observed between tumors with block-like arrangements and those with sparse cell distributions, leading to the concept of immune suppression due to geometrical effects. This research delves into the intricate relationship between spatial dynamics and the effectiveness of CAR-T therapy in solid tumors, highlighting the relevance of tumor geometry in the outcome of cellular immunotherapy treatments. Our results provide a basis for improving the efficacy of CAR-T cell treatments by combining them with other ones reducing the density of compact tumor areas and thus opening access ways for tumor killing T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., D. D. Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality worldwide. Non-small cell lung cancer (NSCLC) is the most common subtype of lung cancer and comprises 85% of cases. Despite treatment advances, local control after curative-intent chemoradiation for NSCLC remains suboptimal. Polo-like kinase 4 (PLK4) is a serine-threonine kinase that plays a critical role in the regulation of centrosome duplication and cell cycle progression and is overexpressed in NSCLC, thus, making it a potential therapeutic target. CFI-400945 is an orally available PLK4 inhibitor currently undergoing clinical trial evaluation. As radiation causes cell death primarily by mitotic catastrophe, a process enhanced by alterations in centrosome amplification, we hypothesized that disruption of the mitotic machinery by inhibition of PLK4 would enhance the effects of radiation in NSCLC. PLK4 inhibition by CFI-400945 resulted in radiosensitization of NSCLC cell lines. In contrast, CFI-400945 had no effect on the radiosensitivity of normal lung fibroblasts. PLK4 inhibition did not affect cell-cycle phase distribution prior to radiation, but rather the combination of CFI-400945 and radiation resulted in increased G2/M cell cycle arrest, increased centrosome amplification, and a concomitant increase in cell death through mitotic catastrophe. Lastly, CFI-400945 treatment enhanced the radiation-induced tumor growth delay of NSCLC tumor xenografts. These data indicate that targeting PLK4 is a novel approach to enhance the radiation sensitivity of NSCLC in vitro and in vivo through potentiation of centrosome amplification and cell death through mitotic catastrophe.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca94b40a92c124ccef0743348417bc7fb2223cd8" target='_blank'>
              PLK4 inhibition as a strategy to enhance non-small cell lung cancer radiosensitivity
              </a>
            </td>
          <td>
            Irma G. Dominguez-Vigil, Kishore Banik, Marta Baro, Joseph N. Contessa, Thomas J Hayman
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ad7065cbf4c15026ee68bc1c1e9443de9a4af3" target='_blank'>
              Identification of T cell dysfunction molecular subtypes and exploration of potential immunotherapy targets in BRAF V600E-mutant colorectal cancer
              </a>
            </td>
          <td>
            Tiefeng Gu, Haonan Qi, Jiaqi Wang, Liangwei Sun, Yongqi Su, Hanqing Hu
          </td>
          <td>2025-02-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CAR T cell therapies often lack specificity, leading to issues ranging from inadequate antigen targeting to off-tumor toxicities. To counter that lack of specificity, we expanded tumor targeting capabilities with universal CAR and spatially defined CAR T cell engagement with targets through a combination of synthetic biology and biomaterial approaches. We developed a novel framework, called "In situ Mobilization: Polymer Activated Cell Therapies" (IMPACT) for polymer-mediated, anatomical control of IF-THEN gated CAR T cells. With IMPACT, a regulated payload such as a BiTE or tumor-targeting CAR will only be expressed after engineered cells engage a tumor-localizing polymer ("IF" condition). In this first demonstration of IMPACT, we engineered CAR T cells to respond to fluorescein that is displayed by an injectable polymer that binds to and is retained in fibrin deposits in tumor microenvironments. This interaction then drives selective and conditional expression of a protein within tumors ("THEN" condition). Here, we develop the polymer and CAR T cell infrastructure of IMPACT and demonstrate tumor-localized CAR T cell activation in a murine tumor model after the intravenous administration of polymer and engineered T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bb25c7b8caf9c1e00291f67d305dabd29b4533" target='_blank'>
              Spatial Control of CAR T Cell Activation Using Tumor-Homing Polymers.
              </a>
            </td>
          <td>
            Clinton M Heinze, Trey J. Pichon, Abe Y Wu, Michael L Baldwin, James Matthaei, Kefan Song, Meilyn Sylvestre, Joshua Gustafson, Nathan J White, Michael C Jensen, S. Pun
          </td>
          <td>2025-02-04</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Epitranscriptomic modifications impact RNA stability and function, influencing key biological processes. While initially behind epigenetics, the field has advanced in the last decade, linking these modifications to development and diseases like cancer. High-grade gliomas, such as glioblastoma, remain challenging due to their aggressiveness and lack of effective treatments. Understanding the impact of epitranscriptomic alterations in gliomas may provide new insights into tumor progression and therapeutic strategies. This review provides an overview of RNA modifications in tRNAs, rRNAs, miRNAs, and mRNAs, exploring the role of the proteins involved in epitranscriptomics and their dysregulation in gliomas. We discuss how this imbalance affects pathways in glioma progression and highlight recent advancements in RNA modification-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bcb06b4f2cf9c15faeac2d418f2e3e672130f79" target='_blank'>
              Epitranscriptomics in the Glioma Context: A Brief Overview
              </a>
            </td>
          <td>
            Pablo Santamarina-Ojeda, Agustin F. Fernández, Mario F. Fraga
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d719de0ed34b62d63deb744432982b8548a3e52" target='_blank'>
              TGF-β signaling orchestrates cancer-associated fibroblasts in the tumor microenvironment of human hepatocellular carcinoma: unveiling insights and clinical significance
              </a>
            </td>
          <td>
            Junwei Ge, Hongwei Jiang, Junjun Chen, Xuemin Chen, Yue Zhang, Liangrong Shi, Xiao Zheng, Jingting Jiang, Lujun Chen
          </td>
          <td>2025-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous disease that develops through a stepwise accumulation, yet the underlying mechanisms at single-cell resolution remain unclear. Here, we profiled 751,531 single-cell transcriptomes, spatial transcriptomics, and snMultiomes from 142 multistage samples, revealing the cellular and molecular alterations and dynamic intercellular crosstalk during CRC development. Additionally, we created a CRC sc-eQTL map identifying 16,833 significant pairs across 28 cell subtypes, with over 76% of sc-eQTLs being cell-type-specific and fewer than 15% detectable in bulk datasets. A polygenic risk score (PRS) derived from sc-eQTLs substantially improved CRC risk prediction. We prioritized rs4794979 that is associated with an increased CRC risk (OR=1.11, P=2.04×10-12) by promoting LGALS9 expression mediated by ELK1. Elevated LGALS9 in epithelia interacts with SLC1A5 on fibroblasts, promoting transformation into cancer-associated fibroblasts (CAFs), simultaneously induces CD8+ T cells exhaustion via LGALS9-TIM3 axis, thereby facilitating CRC development. Blocking LGALS9-TIM3 axis enhanced anti-PD-1 therapy to inhibit CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) techniques hold great value in evaluating the heterogeneity and spatial characteristics of hematopoietic cells within tissues. These two techniques are highly complementary, with scRNA-seq offering single-cell resolution and ST retaining spatial information. However, there is an urgent demand for well-organized and user-friendly toolkits capable of handling single-cell and spatial information. Here, we present HemaScope, a specialized bioinformatics toolkit featuring modular designs to analyze scRNA-seq and ST data generated from hematopoietic cells. It enables users to perform quality control, basic analysis, cell atlas construction, cellular heterogeneity exploration, and dynamical examination on scRNA-seq data. Also, it can perform spatial analysis and microenvironment analysis on ST data. Meanwhile, HemaScope takes into consideration hematopoietic cell-specific features, including lineage affiliation evaluation, cell cycle prediction, and marker gene collection. To enhance the user experience, we have deployed the toolkit in user-friendly forms: HemaScopeR (an R package), HemaScopeCloud (a web server), HemaScopeDocker (a Docker image), and HemaScopeShiny (a graphical interface). In case studies, we employed it to construct a cell atlas of human bone marrow, analyze age-related changes, and identify acute myeloid leukemia cells in mice. Moreover, we characterized the microenvironments in angioimmunoblastic T cell lymphoma and primary central nervous system lymphoma, elucidating tumor boundaries. HemaScope is freely available at https://zhenyiwangthu.github.io/HemaScope_Tutorial/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bd105f231e9996b6a3ab55c7d2018c2946edbd" target='_blank'>
              HemaScope: A Tool for Analyzing Single-cell and Spatial Transcriptomics Data of Hematopoietic Cells.
              </a>
            </td>
          <td>
            Zhenyi Wang, Yuxin Miao, Hongjun Li, Wenyan Cheng, Minglei Shi, Lv Gang, Yating Zhu, Junyi Zhang, Tingting Tan, Jin Gu, Michael Q. Zhang, Jianfeng Li, Hai Fang, Zhu Chen, Saijuan Chen
          </td>
          <td>2025-01-25</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Wilms tumor (WT) is the most common childhood renal malignancy, with recurrence linked to poor prognosis. Identifying the molecular features of tumor phenotypes that drive recurrence and discovering novel targets are crucial for improving treatment strategies and enhancing patient outcomes.Single-nuclei RNA sequencing (snRNA-seq), spatial transcriptomics (ST), bulk RNA-seq, and mutation/copy number data were curated from public databases. The Seurat package was used to process snRNA-seq and ST data. Scissor analysis was applied to identify tumor subpopulations associated with poor relapse-free survival (RFS). Univariate Cox and LASSO analyses were utilized to reduce features. A prognostic ensemble machine learning model was developed. Immunohistochemistry was used to validate the expression of key features in tumor tissues. The CellChat and Commot package was utilized to infer cellular interactions. The PERCEPTION computational pipeline was used to predict the response of tumor cells to chemotherapy and targeted therapies.By integrating snRNA-seq and bulk RNA-seq data, we identified a subtype of Scissor+ tumor cells associated with poor RFS, predominantly derived from cap mesenchyme-like blastemal and fibroblast-like tumor subgroups. These cells displayed nephron progenitor signatures and cancer stem cell markers. A prognostic ensemble machine learning model was constructed based on the Scissor+ tumor signature to accurately predict patient RFS. TGFA was identified as the most significant feature in this model and validated by immunohistochemistry. Cellular communication analysis revealed strong associations between Scissor+ tumor cells and cancer-associated fibroblasts (CAFs) through IGF, SLIT, FGF, and PDGF pathways. ST data revealed that Scissor+ tumor cells were primarily located in immune-desert niche surrounded by CAFs. Despite reduced responsiveness to conventional chemotherapy, Scissor+ tumor cells were sensitive to EGFR inhibitors, providing insights into clinical intervention strategies for WT patients at high risk of recurrence.This study identified a relapse-associated tumor subtype resembling nephron progenitor cells, residing in immune-desert niches through interactions with CAFs. The proposed prognostic model could accurately identify patients at high risk of relapse, offering a promising method for clinical risk stratification. Targeting these cells with EGFR inhibitors, in combination with conventional chemotherapy, may provide a potential therapeutic strategy for WT patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05daf7364d02794afe753ef756b89d494ba7b356" target='_blank'>
              Integration of single-nuclei and spatial transcriptomics to decipher tumor phenotype predictive of relapse-free survival in Wilms tumor
              </a>
            </td>
          <td>
            Ran Yang, Lulu Xie, Rui Wang, Yi Li, Yifei Lu, Baihui Liu, Shuyang Dai, Shan Zheng, Kuiran Dong, Rui Dong
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive and deadly malignant glioma of the adult central nervous system (CNS), making the discovery of related genes essential for its treatment and prognosis. Recent study has shown that Tripartite motif (TRIM) is involved in several biological processes, both suppressing tumor formation and facilitating tumor growth and invasion. It may modulate the onset and progression of cancers via many methods. tripartite motif-containing protein 28 (TRIM28) is a member of the TRIM protein family, serving as a multifunctional transcriptional regulator that participates in chromatin remodeling, DNA damage repair, and the maintenance of stem cell pluripotency. Recent research indicates that TRIM28 influences the self-renewal and chemotherapy resistance of tumor stem cells via modulating the expression of tumor stem cell markers, including Prominin-1 (CD133) in GBM. Research demonstrates that TRIM28 directly engages with the promoter region of the CD133 gene via an epigenetic mechanism, inhibits histone acetylation (particularly h3k27ac), and therefore downregulates CD133 expression. The knockdown of trim28 might significantly reduce the stem cell properties of GBM cells, hinder tumor sphere formation, and enhance their sensitivity to temozolomide (TMZ). Subsequent study demonstrated that trim28 recruited histone deacetylase (HDAC) and DNA methyltransferase (DNMT) complexes to the CD133 promoter region, leading to localized chromatin compaction and gene silencing. This mechanism depends on the PhD bromo domain of TRIM28, signifying the specificity of its epigenetic regulatory function. Besides, there are also other mechanism that to do with TRIM28 and GBM. As for the therapeutic potential, TRIM28 can be used as a significant biomarker in GBM. Besides, CD133-targeted chimeric antigen receptor T cell may be a viable therapeutic approach to target CD133+ cancer stem cells in human glioblastoma or other treatment-resistant primary malignancies. Nonetheless, the precise mechanism of TRIM in GBM necessitates further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec94852d4fc4e08a027640077bbdafb4dc3d975b" target='_blank'>
              Research Progress on TRIM28 and GBM
              </a>
            </td>
          <td>
            Haocheng Jia, Qishan Ran
          </td>
          <td>2025-02-26</td>
          <td>International Journal of Biology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Metastatic melanoma is a lethal form of cancer. Immunotherapy is recommended for metastatic melanoma and known to elicit beneficial response even after disease progression per RECIST. On-treatment samples hold irreplaceable predictive value. We collected needle biopsies from the metastases of different body sites from the same patient that progressed upon PD-1 treatment. Leveraging Direct Nuclear Tagmentation and RNA sequencing (DNTR- seq), we conducted single-cell genomic analyses on melanoma metastases and jointly sequenced the transcriptomes of each single melanoma cell. After 3 months of PD-1 inhibitor treatment, single-cell DNA-seq analyses on melanoma metastases from different anatomic locations displayed distinct cancer subclones evidenced by their copy number aberrations. Joint transcriptomic analyses of the dominant genomic subclones from each body site showed converging phenotypes, under the selective pressure of immunotherapy. Our preliminary analyses of on-treatment needle biopsies shed light on the challenges of treating heterogeneous tumors and underscore the pressing need to identify an ideal denominator to reach disease control.
 Citation Format: Muyi Yang, Solrun Kolbeinsdottir, Katarina Hammarlund, Vitali Grozman, Martin Enge, Suzanne Egyhazi-Brage, Hildur Helgadottir. Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A108.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Immune checkpoint inhibition has demonstrated no benefit when combined with chemoradiotherapy for locally advanced HNSCC, raising the possibility that standard therapies compromise the response to immunotherapy. To address this hypothesis, we previously demonstrated that ablating tumor draining lymphatics abolishes the response to immune-oncology therapy. Specifically, we defined that the response to immunotherapy is coordinated by interferon type-I signaling through the cDC1 population in tumor-draining lymph nodes. These findings support the premise that successful tumor responses to immuno-oncology therapies is predicated upon intact and functional, locoregional anatomy. Accordingly, we hypothesize that lymphatic-preserving, tumor-directed immune oncology therapy will promote antitumor immunity by enhancing surveillance along the tumor-immune-lymphatic axis. To explore this, we employ a novel mouse reporter model that allows precise spatiotemporal labeling of immune effector cells. Combined with sentinel lymph node mapping, our reporter model enables selective labeling of immune effectors within the tumor microenvironment and subsequent tracking and characterization of these cells as they migrate to the sentinel lymph node. This approach led us to define the “cancer immune migratome” - the repertoire of immune cells actively migrating from tumors to lymph nodes during an antitumor response. For precise cellular identification, we employed CITE-sequencing, which integrates single-cell RNA expression with surface protein markers, allowing us to comprehensively define the characteristics of migrating immune cells. This work represents the first characterization of immune cell populations migrating from the tumor to the sentinel lymph node, revealing the diversity and dynamics of these populations in antitumor immunity. We then develop a tumor-directed, lymphatic-sparing radiotherapy model to modulate the tumor immune microenvironment without inducing tumor regression. Sequential administration of tumor-directed radiotherapy followed by PD-1 checkpoint inhibition achieves complete and durable responses. Analysis of the cancer immune migratome during immunoradiotherapy revealed a crucial role for an activated population of CCR7+ dendritic cells. These migratory dendritic cells are reprogrammed, via rational sequencing of radiation and immunotherapy, from their canonical role in immune tolerance to actively drive antitumor immunity. This reprogramming enhances T cell priming, clonotypic expansion, and, ultimately, the successful tumor response to immunoradiotherapy. Disruption of sentinel lymphatic channels or selective blockade of CCR7+ DC entry into the SLN abrogates the response to immunoradiotherapy. Overall, this work supports rationally sequencing immune-sensitizing, lymphatic-preserving, tumor-directed radiotherapy followed by immune checkpoint inhibition to optimize tumor response. Our findings support a therapeutic strategy that enhances host response to immunotherapy by directing activated dendritic cells to sentinel lymph nodes.
 Citation Format: Robert Saddawi-Konefka, Riyam Al Msari, Shiqi Tang, Lauren M. Clubb, Santiago Fassardi, Riley Jones, Farhoud Faraji, Shiruyeh Schokrpur, Bryan S. Yung, Michael M. Allevato, Shawn M. Jensen, Bernard A. Fox, Richard Bell, J. Silvio Gutkind, Andrew Sharabi, Joseph A. Califano. Mapping the tumor-sentinel node immune migratome demonstrates a key role for CCR7+ dendritic cells in the successful response to immunoradiotherapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P004">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8227f78d1879affbb26ec2351982ff6e814cbb7b" target='_blank'>
              Abstract P004: Mapping the tumor-sentinel node immune migratome demonstrates a key role for CCR7+ dendritic cells in the successful response to immunoradiotherapy
              </a>
            </td>
          <td>
            R. Saddawi-Konefka, Riyam Al Msari, Shiqi Tang, Lauren M. Clubb, Santiago Fassardi, Riley Jones, F. Faraji, Shiruyeh Schokrpur, Bryan S. Yung, M. Allevato, S. Jensen, B. A. Fox, R. Bell, J. S. Gutkind, Andrew Sharabi, Joseph A Califano
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Uveal melanoma (UVM) is a form of eye cancer with a poor prognosis, particularly in metastatic patients. This study aimed to elucidate the cellular heterogeneity within UVM and identify prognostic biomarkers. Methods We performed single-cell RNA sequencing (scRNA-seq) on primary and metastatic UVM samples. A UVM-specific gene signature was constructed using LASSO regression and validated via ROC curve analysis in the TCGA-UVM and GSE84976 cohorts. AlphaFold 3 was used to predict the 3D structures of key proteins. T-cell populations were analyzed using pseudotime trajectory mapping and interaction network visualization. CRISPR-Cas9 screening analysis was conducted to identify hub genes and cytokine pathways that may serve as therapeutic targets. Additionally, we constructed the Dictionary of Immune Responses to Cytokines at single-cell resolution to evaluate cytokine signatures. Results ScRNA-seq revealed five major cell types within UVMs and subdivided them into seven distinct subtypes. Cytokine signaling analysis revealed differential expression of cytokine signaling in immune-related genes (CSIRGs) across these subtypes in primary and metastatic tumors. The UVM-specific gene signature demonstrated high predictive accuracy in ROC curve analysis and was associated with overall survival in Kaplan–Meier survival analyses. Additionally, AlphaFold 3 predicted the 3D structures of key proteins with high confidence. T-cell population analysis revealed complex developmental pathways and interaction networks in UVM. Myeloid-derived suppressor cells (MDSCs) were found to be increased in metastatic UVM, correlating with the enrichment of GM-CSF. CRISPR-Cas9 screening analysis identified hub genes and cytokine pathways with low gene effect scores across cell lines, indicating their potential importance in UVM. Conclusion This study identified critical cellular subtypes and prognostic biomarkers in UVM, shedding light on targeted therapies. The insights into cytokine signaling and T-cell dynamics within the UVM microenvironment provide a foundation for developing personalized therapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/755f8cb5c84686f4d7dbae1de44193cdc55d1136" target='_blank'>
              Single-cell RNA sequencing and AlphaFold 3 insights into cytokine signaling and its role in uveal melanoma
              </a>
            </td>
          <td>
            Hongyan Sun, Cunzi Li, Zuhui Pu, Ying Lu, Zijing Wu, Lan Zhou, Hongzhan Lin, Yumo Wang, Tao Zi, Lisha Mou, M. Yang
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The ribophorin family, including RPN1, has been associated with tumor progression, but its specific role in pan-cancer dynamics remains unclear. Using data from TCGA, GTEx, and Ualcan databases, we investigated the relationship of RPN1 with prognosis, genomic alterations, and epigenetic modifications across various cancers. Differential analysis revealed elevated RPN1 expression in multiple cancer types, indicating a potential prognostic value. Amplification was the predominant mutation type of RPN1 in pan-cancer, with notable correlations with DNA methylation and copy number variation. Gene set variation analysis identified RPN1's involvement in cancer development, immunity, and metabolism. Additionally, RPN1 expression correlated with the tumor microenvironment, immune response factors, and response to anti-tumor therapies. Functional validation in triple-negative breast cancer, glioblastoma, and bladder cancer cell lines demonstrated the role of RPN1 in tumor cell proliferation and migration. Our findings highlight RPN1 as a potential biomarker for cancer diagnosis and treatment response in pan-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7ca6a31f84637afae275f44ff0e8aae63741e5a" target='_blank'>
              Exploring the impact of RPN1 on tumorigenesis and immune response in cancer.
              </a>
            </td>
          <td>
            P. Luo, Zhimin Li, Haodong He, Yuanbin Tang, Lijun Zeng, Lunqi Luo, Lianjie Ouyang, Meiling Wen, Yuehua Li, Yongjun Jiang
          </td>
          <td>2025-03-13</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mammary microenvironment has been shown to suppress tumor progression by redirecting cancer cells to adopt a normal mammary epithelial progenitor fate in vivo. However, the mechanism(s) by which this alteration occurs has yet to be defined. Here, we test the hypothesis that mitochondrial transfer from normal mammary epithelial cells to breast cancer cells plays a role in this redirection process. We evaluate mitochondrial transfer in 2D and 3D organoids using our unique 3D bioprinting system to produce chimeric organoids containing normal and cancer cells. We demonstrate that breast cancer tumoroid growth is hindered following interaction with mammary epithelial cells in both 2D and 3D environments. Furthermore, we show mitochondrial transfer occurs between donor mammary epithelial cells and recipient cancer cells primarily through tunneling nanotubes (TNTs) with minimal amounts seen from extracellular transfer of mitochondria, likely via extracellular vesicles (EVs). This organelle exchange results in various cellular and metabolic alterations within cancer cells, reducing their proliferative potential, and making them susceptible to microenvironmental control. Our results demonstrate that mitochondrial transfer contributes to microenvironmental redirection of cancer cells through alteration of metabolic and molecular functions of the recipient cancer cells. To the best of our knowledge, this is the first description of a 3D bioprinter-assisted organoid system for studying mitochondrial transfer. These studies are also the first mechanistic insights into the process of mammary microenvironmental redirection of cancer and provide a framework for new therapeutic strategies to control cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74a99d567922b268e37075173798f1b540e4d47" target='_blank'>
              Intercellular mitochondrial transfer contributes to microenvironmental redirection of cancer cell fate.
              </a>
            </td>
          <td>
            J. S. Bjerring, Yara Khodour, Emilee Anne Peterson, Patrick C. Sachs, Robert David Bruno
          </td>
          <td>2025-02-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Glioblastoma (GB), is an incurable brain tumor characterized by extreme malignancy and invasiveness and the cellular mechanisms underlying such a severe phenotype are not completely understood. Although calcium (Ca2+) plays an important part in tumor proliferation and infiltration, it remains unclear whether Ca2+ signaling in GB cells is related to its location within the tumor and on the infiltrative potential of the cells.



 In this study, we developed a stably transfected GL261 cell line that co-expresses a red fluorescent protein for actin cytoskeleton staining and the intracellular Ca2+ sensor, GCaMP6s. By means of intravital two photon imaging we have characterized the morphological and functional properties of cells at different locations within the tumor.



 Our results showed that cells located at the tumor core are densely packed and rounded in shape, contrasting sharply with the polarized morphology observed in the peripheral cells. This anatomical heterogeneity corresponded to notable variations of the physiological phenotype: cells at the tumor core displayed low Ca2+ activity and very limited motility, while peripheral cells displayed intense Ca2+ activity and increased migration rates. Moreover, peripheral cells formed cellular ensemble characterized by synchronized Ca2+ activity accompanied by a directionally biased collective motility. Conclusions: These findings suggest that GB cells manifest activity patterns depending upon their spatial location within the tumor and that these correlate with their migration.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f476d5b2005f0aec9da0be185757bb8f7ffc3130" target='_blank'>
              Heterogeneity of intracellular calcium signaling of Glioblastoma cells depends on intratumoral location and migration state
              </a>
            </td>
          <td>
            V. Pillai, E. Pracucci, Francesco Trovato, Riccardo Parra, S. Landi, G. Ratto
          </td>
          <td>2025-03-05</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ceeafcf1da623087d79a8767b5543bf2a7043f8" target='_blank'>
              Genomic and micro-environmental insights into drug resistance in colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Junjie Kuang, Jun Li, Siwei Zhou, Yi Li, Jinbo Lin, Wei-zeng Huang, Xia Yuan
          </td>
          <td>2025-02-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="Background Head and neck cancers harbor dormant cancer stem cells (CSCs). This study explores how platinum therapy impacts these cells in a non-genetic manner and the role of hypoxia in this process. Previously, we identified a novel population of CSCs exhibiting an “altruistic” phenotype, sacrificing self-renewal to promote niche defense (tumor stemness defense, TSD), potentially protecting a dormant subpopulation of CSCs, the reawakening CSC (R-CSC) retaining stress memory. This TSD phenotype involves the activation of the MYC-HIF2α pathway and, importantly, is linked to a hypoxic tumor microenvironment. We termed these TSD+ CSCs “altruistic cancer stem cells” (A-CSCs). Here we investigated the potential role of tumor hypoxia in the mobilization of TSD+ CSCs to the circulation as a part of niche defense against platinum therapy. Methods We isolated CTCs and primary tumor cells from head and neck squamous cell carcinoma (HNSCC) patients undergoing platinum therapy (n = 14). We analyzed the TSD phenotype and markers of hypoxia in these cells. Additionally, we further characterized a previously reported pre-clinical model of platinum-induced tumor stemness to study the link between hypoxia, TSD+ CSC emergence, and mobilization to the circulation and bone marrow. Results We isolated TSD+ CTCs with a hypoxic signature from eight out of 14 HNSCC patients. These cells displayed increased proliferation and invasion upon cisplatin treatment, suggesting a role in niche defense. Our pre-clinical model confirmed that hypoxia directly correlates with the expansion of TSD+ CSCs and their mobilization into the circulation and bone marrow following cisplatin treatment. We demonstrated the protection of R-CSCs by TSD+ CSCs. Notably, inhibiting hypoxia alone with tirapazamine did not reduce TSD+ CSCs, CTCs, or R-CSCs. However, combining tirapazamine with FM19G11, a MYC-HIF2α pathway inhibitor, significantly reduced the platinum-induced expansion of both TSD+ CSCs, CTCs, and the presence of R-CSCs in the bone marrow. Conclusions This study reveals that HNSCC patients undergoing platinum therapy can harbor TSD+ CTCs exhibiting an altruistic phenotype and a hypoxic signature. Additionally, the pre-clinical study provides a novel non-genetic mechanism of therapy resistance-the altruistic tumor self-defense. The tumor microenvironment, through the emergence of TSD+ CSCs, appears to act collectively to defend the tumor self-identity by hijacking an altruistic stem cell niche defense mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc47428ef4dce00c70a03b6d84031451c648467" target='_blank'>
              Hypoxia-driven mobilization of altruistic cancer stem cells in platinum-treated head and neck cancer
              </a>
            </td>
          <td>
            Lekhika Pathak, Bidisha Pal, Joyeeta Talukdar, Parthajyoti Saikia, Sorra Sandhya, Wale Tasabehji, Hong Li, Jyotirmoy Phukan, Anjan Bhuyan, Sanjukta Patra, Bikul Das
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95e6d9d4fbec0ea9d09f8e2bd810ce1221be487" target='_blank'>
              Unveiling the Role of the Cellular Tumor Microenvironment and the Therapeutic Targets it Provides in Cutaneous T-Cell Lymphoma.
              </a>
            </td>
          <td>
            Nikolaos A. Chinas, S. Kaliampou, V. Nikolaou
          </td>
          <td>2025-03-08</td>
          <td>Current oncology reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="PURPOSE
Digital proteomic profiling was performed to identify spatial context in relationship to patient response and survival after anti-PD-1 therapy in metastatic colorectal cancer (CRC).


EXPERIMENTAL DESIGN
Primary CRCs with deficient mismatch repair (d-MMR) from patients treated with anti-PD-1 antibodies were analyzed (N=30) using Digital Spatial Profiling (GeoMx® nCounter). At the invasive margin, 71 proteins were profiled in 10 regions of interest /slide that were segmented into 3 compartments labeled with pan-cytokeratin (PanCK; epithelia), CD45 (stromal cells) and SYTO13 (nuclei). In an independent cohort (n=13), DSP data and single cell transcriptomic data were analyzed. Differential protein abundance, after Benjamini-Hochberg correction, was examined by response and progression-free survival (PFS) using multivariable Cox regression.


RESULTS
Protein abundance varied significantly between epithelial and stromal compartments. Nonresponders (NR) to anti-PD-1 showed higher fibronectin and smooth muscle actin (SMA) abundance in the epithelial compartment that were associated with significantly shorter PFS (HRadj: 6.49 and 4.52, respectively; p < 0.05). In CD45+ stroma, increased expression of proteins related to T-cells (CD3, CD4), NK cells (CD56), antigen presentation (CD40), immune activation (CD27, ICOS), and apoptosis (GZMA) were found in responders (vs NR) to anti-PD-1; each marker was significantly associated with longer patient PFS (0.02 <HRadj < 0.17; p < 0.05). In a separate cohort, consistent results by compartment were found for fibronectin and CD56. Gene expression data revealed that fibronectin and SMA were primarily derived from cancer-associated fibroblasts.


CONCLUSIONS
Spatially resolved protein profiles within microenvironments of d-MMR CRCs can influence patient response and survival after anti-PD-1 highlighting their potential clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e12e22717ab1e00830f945fd4f97470fd6160b" target='_blank'>
              Spatially resolved, multi-region proteomics for prediction of immunotherapy outcome in deficient mismatch repair metastatic colorectal cancer.
              </a>
            </td>
          <td>
            B. Saberzadeh-Ardestani, Zhenglong Liu, Mariam I Stein, Will A Sherman, Christy E Trussoni, Charles W Abbott, Dongyao Yan, Skyler Smith, K. Shanmugam, Rondell P. Graham, Alos Diallo, Joshua J. Levy, Tamas Ordog, F.A. Sinicrope
          </td>
          <td>2025-02-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d479e3aee16692284dd5875a578cce134082060b" target='_blank'>
              Cell signaling communication within papillary craniopharyngioma revealed by an integrated analysis of single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Yanfei Jia, Zicheng Zhao, Xiaoyu Zhang, Yunlong Zhao, Songbai Gui, Xiu-An Yang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis.We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment.Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis.This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is largely refractory to antibodies against programmed cell death 1 (anti-PD-1) therapy. Fully understanding the cellular heterogeneity and immune adaptations in response to anti-PD-1 therapy is necessary to design more effective immunotherapies for GBM. This study aimed to dissect the molecular mechanisms of specific immunosuppressive subpopulations to drive anti-PD-1 resistance in GBM.


METHODS
We systematically analysed single-cell RNA sequencing and spatial transcriptomics data from GBM tissues receiving anti-PD-1 therapy to characterize the microenvironment alterations. The biological functions of a novel circular RNA (circRNA) were validated both in vitro and in vivo. Mechanically, co-immunoprecipitation, RNA immunoprecipitation and pull-down assays were conducted.


RESULTS
Mesenchymal GBM (MES-GBM) cells, which were associated with a poor prognosis, and secreted phosphoprotein 1 (SPP1)+ myeloid-derived macrophages (SPP1+ MDMs), a unique subpopulation of MDMs with complex functions, preferentially accumulated in non-responders to anti-PD-1 therapy, indicating that MES-GBM cells and SPP1+ MDMs were the main anti-PD-1-resistant cell subpopulations. Functionally, we determined that circular RNA succinate dehydrogenase complex assembly factor 2 (circSDHAF2), which was positively associated with the abundance of these two anti-PD-1-resistant cell subpopulations, facilitated the formation of a regional MES-GBM and SPP1+ MDM cell interaction loop, resulting in a spatially specific adaptive immunosuppressive microenvironment. Mechanically, we found that circSDHAF2 promoted MES-GBM cell formation by stabilizing the integrin alpha 5 (ITGA5) protein through N-glycosylation. Meanwhile, the N-glycosylation of the ITGA5 protein facilitated its translocation into exosomes and subsequent delivery to MDMs to induce the formation of SPP1+ MDMs, which in turn maintained the MES-GBM cell status and induced T-cell dysfunction via the SPP1-ITGA5 pathway, ultimately promoting GBM immune escape. Importantly, our findings demonstrated that antibody-mediated ITGA5 blockade enhanced anti-PD-1-mediated antitumor immunity.


CONCLUSIONS
This work elucidated the potential tissue adaptation mechanism of intratumoral dynamic interactions between MES-GBM cells, MDMs and T cells in anti-PD-1 non-responders and identified the therapeutic potential of targeting ITGA5 to reduce anti-PD-1 resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c2a7db7690cdde92374851555ac5230fd10a64c" target='_blank'>
              Blocking ITGA5 potentiates the efficacy of anti-PD-1 therapy on glioblastoma by remodeling tumor-associated macrophages.
              </a>
            </td>
          <td>
            Rongrong Zhao, Z. Pan, Jiawei Qiu, Boyan Li, Yanhua Qi, Zijie Gao, W. Qiu, Weijie Tang, Xiaofan Guo, Lin Deng, Gang Li, Hao Xue
          </td>
          <td>2025-03-14</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1610b8e09ca06ead7a9826cda40c37396bdae40e" target='_blank'>
              Atlas of multilineage stem cell differentiation reveals TMEM88 as a developmental regulator of blood pressure
              </a>
            </td>
          <td>
            Sophie Shen, Tessa Werner, S. Lukowski, Stacey Andersen, Yuliangzi Sun, W. J. Shim, Dalia Mizikovsky, Sakurako Kobayashi, Jennifer Outhwaite, Han Sheng Chiu, Xiaoli Chen, G. Chapman, E. M. Martin, Di Xia, Duy Pham, Zezhuo Su, Daniel Kim, Pengyi Yang, Men Chee Tan, Enakshi Sinniah, Qiongyi Zhao, Sumedha Negi, Meredith A. Redd, Joseph E Powell, Sally L. Dunwoodie, Patrick P L Tam, Mikael Bodén, J. W. Ho, Quan H Nguyen, N. Palpant
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Inflammatory responses occur within the complex spatial context of tissues and organs, and many questions remain about how tissue structure and cellular communication shape their spatiotemporal dynamics. Here, we use a multiplexed RNA in situ hybridization approach, together with analytical tools, to study inflammatory gene expression in the larval zebrafish tailfin in response to a bath of lipopolysaccharide (LPS). We use this model system to address whether spatial structure emerges in the tissue response even absent the spatial variation introduced by a pathogen. We find that epithelial cells in the tailfin express several pro-inflammatory genes, and that across these genes, the uniform stimulus triggers a spatially non-uniform response. We use a graph-based spectral decomposition method to analyze its structure, and find that long modes dominate, creating zones of activation. Overall, these zones account for a majority of the variation in gene expression. Our results show that epithelial cells are important producers of pro-inflammatory effector molecules in this system, and that tissue induces spatial correlations even absent a structured input.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c13871b20a0df0f49db2baf60ca96fdc2c42238" target='_blank'>
              Spatially-structured inflammatory response in the presence of a uniform stimulus
              </a>
            </td>
          <td>
            Elizabeth R. Jerison, Nicolas Romeo, Stephen R. Quake
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer is the most common type of cancer in women and the second leading cause of death by cancer. Despite recent advances, the mortality rate remains high, underlining the need to develop new therapeutic approaches. The complex interaction between cancer cells and the tumor microenvironment (TME) is crucial in determining tumor progression, therapy response, and patient prognosis. Understanding the role of immune cells in carcinogenesis and tumor progression can help improve targeted therapeutic options, increasing the likelihood of a favorable prognosis. Therefore, this review aims to critically analyze the complex interaction between tumor cells and immune cells, emphasizing the clinical and therapeutic implications. Additionally, we explore advances in immunotherapies, with a focus on immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c335e51d434a9f10b3300d0d306ba2af8135f4e1" target='_blank'>
              Breast Cancer and Tumor Microenvironment: The Crucial Role of Immune Cells
              </a>
            </td>
          <td>
            Tânia Moura, P. Laranjeira, Olga Caramelo, Ana M. Gil, Artur Paiva
          </td>
          <td>2025-02-28</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fc0173898db1615005d0ab09ce022fdfc322eb" target='_blank'>
              The Transcriptomic Landscapes of Appendiceal Primary and Metastatic Tumors are Distinct.
              </a>
            </td>
          <td>
            Margaret A. Park, Richard Jacobson, Maria Genilo-Delgado, Amir Mohammadi, Carlos M. Morán-Segura, Solomon Alhassan, Yukihiro Nakanishi, J. Permuth, I. Imanirad, S. Dineen
          </td>
          <td>2025-02-22</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 842


 Background:
 Advanced Bladder Cancer (BC) presents significant challenges due to its high incidence and limited treatment options. Most patients initially present with non-muscle invasive bladder cancer (NMIBC), managed with transurethral resection (TURBT) and Bacillus Calmette–Guerin (BCG) therapy. However, some cancers recur or develop resistance, progressing to muscle-invasive bladder cancer (MIBC), which is associated with high mortality and limited curative options. While radical cystectomy (RC) is the standard treatment for MIBC, it has considerable morbidity. Thus, there is a critical need for predictive tools to guide treatment decisions and novel therapeutic approaches. Immunotherapies show promise, but their efficacy remains constrained. Analyzing the immune infiltrate in BC may uncover mechanisms of resistance and identify new targets.
 Methods:
 We investigated the immune microenvironment in NMIBC and MIBC using high-dimensional single-cell analysis, including scRNAseq and Imaging Mass Cytometry (Cytomass). Tumor tissues were obtained from BC patients who underwent cystectomy or Trans urethral removal of bladder tumour (TURBT). We enrolled 7 NMIBC and 8 MIBC patients for scRNAseq, isolating CD45+ immune cells and CD45- epithelial and stromal cells using FACS. The samples were processed with 10X technology to reveal transcriptional landscapes. Additionally, we enrolled 5 NMIBC and 5 MIBC patients for Cytomass analysis, where tissue sections were snap-frozen, stained, and analyzed with the Hyperion system. Our primary endpoint was to identify differences in the composition of tumor and immune cells between NMIBC and MIBC. The study was funded by the Next Generation EU Plan - Piano Nazionale di Ripresa e Resilienza (PNRR-MAD-2022-12376756).
 Results:
 The cohort included 15 patients with clinically significant BC who underwent radical cystectomy or TURBT. scRNAseq analysis of patients revealed differences in immune cell abundance and activation states between NMIBC and MIBC samples, identifying distinct cell types and high intra-tumoral heterogeneity. Notable changes were observed tumor-infiltrating B cell lymphocytes, with different abundance in tumors depending on the cell subsets. Immature B cells were enriched in MIBC when compared to NMIBC, indicating a potential role in tumor progression. Cytomass analysis further highlighted modulations in immune cell activation and spatial localization between NMIBC and MIBC.
 Conclusions:
 Our findings demonstrate the utility of single-cell RNA sequencing in elucidating the immune landscape of BC and highlight significant immune changes between MIBC and NMIBC. This data enhances our understanding of immune composition and the role of specific immune subsets in BC, potentially informing clinical risk stratification and treatment decision-making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7de131a381559fb27b1d49711570f15a4787b67" target='_blank'>
              Immune profiling at the single-cell level to evaluate the immune microenvironment in bladder cancer: Insights from non-muscle invasive and muscle-invasive disease.
              </a>
            </td>
          <td>
            P. Avolio, Anna Garbin, Marta Pandini, R. Carriero, Paolo Kunderfranco, M. G. Elefante, P. Colombo, P. Casale, G. Lughezzani, N. Buffi, R. Contieri, Gianluca Basso, Diletta Di Mitri, M. Lazzeri
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="
 Background: Locally-advanced rectal cancer (LARC) is traditionally treated with neoadjuvant chemoradiotherapy before surgery, however, response heterogeneity necessitates the development of novel treatment combinations. Immune checkpoint blockade (ICB) with or without radiotherapy (RT) elicits clinical benefit in various solid tumors, yet ICB’s effectiveness so far is limited to colorectal tumors with microsatellite instability and the role of the tumor microenvironment (TME) is unclear. We present a novel orthotopic immunocompetent mouse model of LARC, where combinations of RT and ICB are tested to elucidate the role of TME in the response to treatment. Methods: Autochthonous murine tumor-derived organoids expressing common driver mutations (Apcfl/fl; KrasG12D/+; p53fl/fl; Tgfbr1fl/fl: ‘AKPT’) were orthotopically injected into syngeneic C57Bl/6 immunocompetent mice. Tumor response to RT-ICB was studied on timepoint and aging cohorts of mice with tumors growing in different TME settings: (a) in the rectal submucosa of wildtype mice, (b) subcutaneously in wildtype mice, and (c) in the rectal submucosa of mice lacking the iCCR locus (CCR1, CCR2, CCR3, and CCR5: ‘iCCRKO ’), which inhibits inflammatory monocyte recruitment into the TME. Radiation was delivered as 15Gy in 3# (Mon-Wed-Fri) using a Small Animal Radiation Research Platform on day 13 following tumor engraftment for the orthotopic models and on day 7 for the subcutaneous. PD-1 inhibition (PD-1i) was administered intraperitoneally (10mg/kg twice weekly) 3 days pre-RT until endpoint. Mice in the timepoint experiments were sampled 5 days following RT, while aging mice were sampled at clinical endpoint or 100 days, whichever came first. Results: RT and PD-1i offered significant survival extension and tumor control in the orthotopic wildtype setting (a). Transcriptomic characterization at 5 days revealed that untreated and irradiated tumors harbored a stroma-rich microenvironment, low in immune cells, which PD-1i partly reversed. This phenotypic ‘switch’ was also evident in an aging cohort, suggesting that RT + PD-1i have long-lasting effects on the stroma. In the subcutaneous setting (b), RT significantly increased tumor control but PD-1 blockade offered little additional advantage. Interestingly, subcutaneous AKPT tumors had significantly more CD8+ cells at baseline than orthotopic ones, highlighting the inherent immune permissiveness of the subcutaneous TME. Finally, tumors growing in the iCCRKO setting (c) were more radio-sensitive, leading to 3 instances of tumor clearance: one in the RT group and two in the RT + PD-1i. Conclusion: Our findings demonstrate the synergistic potential of RT and PD-1i to sensitize an inherently radio-resistant orthotopic mouse model of LARC. Tumor location and inflammatory monocytes emerged as key regulators of the immune response to RT-ICB, which could have important implications for LARC modeling. Importantly, we highlight that TME components can facilitate treatment resistance and give rise to targets that can be therapeutically exploited.
 Citation Format: Lydia Melissourgou-Syka, Michael A. Gillespie, Lily V.S. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, Tamsin R.M. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, Sean M. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S.D. Roxburgh.Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P022">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b958d65c1fd0bd54499a71c321762f474a07901" target='_blank'>
              Abstract P022: Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations
              </a>
            </td>
          <td>
            Lydia Melissourgou-Syka, Michael A Gillespie, L. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, T. Lannagan, Eoghan Mulholland, Philip D. Dunne, J. Edwards, S. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S D Roxburgh
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) presents the most challenging subtype of the disease due to its aggressive nature and lack of targeted therapy options. To identify regulators of TNBC, we conducted a genome-wide CRISPR-knockout screen in both 3D tumor spheroid and 2D cell culture models. The 3D spheroid model displayed unique potential in identifying putative tumor suppressors due to its closer mimicry of in vivo tumor growth conditions. Notably, the chromatin-remodeling SWI/SNF complex emerged as a potent suppressor of tumor spheroid growth. Specifically, loss of the SWI/SNF ATPase subunit SMARCA4 promoted tumor spheroid growth with reduced compactness and enhanced primary tumor growth and metastasis across multiple TNBC models. SMARCA4 was required for the transcription of Rho GTPase activating factor ARHGAP29 by enhancing DNA accessibility through direct binding to its promoter. SMARCA4 loss resulted in reduced ARHGAP29 levels and hyperactive RHOA signaling, subsequently disrupting cell adhesion, facilitating the formation of a loose spheroid structure in vitro, and enhancing breast cancer growth and metastasis in vivo. These results establish SMARCA4 and SWI/SNF as tumor suppressors of TNBC through suppression of RHOA activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7936be7822f9a0773dcd71150fe84ffc4ab837c5" target='_blank'>
              SMARCA4 Inhibits Breast Cancer Progression and Metastasis through RHOA Suppression.
              </a>
            </td>
          <td>
            Zheng Sun, Zhuo Li, Yong Wei, Lillian Xu, Xiang Hang, Yibin Kang
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Immune checkpoint blockade (ICB) therapy has revolutionized cancer treatment, delivering impressive results across various cancers. However, resistance is common, and only a limited number of patients experience lasting responses. This underscores the urgent need to understand resistance mechanisms and develop new strategies to enhance responsiveness. Here we focus on breast cancer, particularly ER+ breast cancer, which is generally poorly immunogenic and responds poorly to ICB. To address this, we analyzed the METABRIC cohort's immunogenic biomarkers and tumor mutational burden (TMB). We found that relapsed ER+ patients treated with endocrine therapy exhibit activated immune pathways. However, high TMB correlates with lower overall survival (OS), suggesting an active but clinically ineffective immune response. To explore the immune-resistant mechanisms, we developed chronically tamoxifen-treated T47D and MCF7 cells (cTMX-T47D and cTMX-MCF7) by exposing them to the drug for over six months. RNA sequencing data revealed increased type I interferon (IFN-I) signaling and innate immunity in these cells, confirmed in vivo in T47D xenografts. Tracing tamoxifen’s effects, we found that IFN-I signaling and ISG induction depend on the cGAS/STING pathway. Short-term tamoxifen exposure induced only modest ISG expression, indicating additional prerequisites during long-term treatment. We identified that the RACK7/KDM5C H3K4 demethylase complex is crucial for tamoxifen-induced ISG upregulation, with both proteins decreasing over time. Knocking down RACK7 intensified tamoxifen's activation of the cGAS/STING signaling and ISG expression. Combined tamoxifen treatment and RACK7 knockdown significantly enhanced these innate immunity effects compared to either treatment alone. Furthermore, tamoxifen’s DNA-damaging ability modulates the tumor immune microenvironment, particularly through mitochondrial DNA (mtDNA)-induced STING upregulation. We discovered key tamoxifen-mediated mechanisms for cGAS/STING-IFN-I pathway activation via STING upregulation, involving the tamoxifen–mtDNA–STING and tamoxifen–RACK7/KDM5C–STING axes. While IFN-I signaling can promote anti-tumor immunity, our tumor/T-cell co-culture experiments revealed that tamoxifen inactivates T-cells through tumor-intrinsic mechanisms, likely involving T-cell exhaustion pathways. We investigated tamoxifen’s effects on immune checkpoint molecules, finding that CEACAM1, the primary ligand for TIM3, was upregulated in cTMX-T47D and cTMX-MCF7 cells. Further analysis of tamoxifen-treated T47D xenografts revealed elevated CEACAM1 levels, suggesting an immunosuppressive mechanism via the CEACAM1-TIM3 pathway. To investigate this axis in vivo, we used syngeneic tumor mouse models with TS/A (ER+) cells in BALB/c mice. Tamoxifen induced CEACAM1, increased CD45+ immune cell infiltration, and caused TIM3+ CD8 T-cell exhaustion, contributing to tumor progression. Blocking the CEACAM1-TIM3 interaction with an anti-TIM3 antibody significantly inhibited tumor growth, highlighting its role in tamoxifen resistance.
 Citation Format: Lan-Hsin Wang, Marvin Angelo E Aberin, Saurabh Singh, Yi-Fu Wang, Ya-Ting Lu, Joy Khag, Weng-Lang Wang, Yao-Ming Chang, Shih-Yu Chen, Chandan Guha, Shu-Ping Wang. Enhancing breast cancer immunotherapy by modulating the cGAS–STING and CEACAM1–TIM3 pathways in the tumor microenvironment [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab23d6c01d007b2032b2ae13c6264807a485ddad" target='_blank'>
              Abstract A066: Enhancing breast cancer immunotherapy by modulating the cGAS–STING and CEACAM1–TIM3 pathways in the tumor microenvironment
              </a>
            </td>
          <td>
            Lan-Hsin Wang, M. A. Aberin, Saurabh Singh, Yi-Fu Wang, Ya-Ting Lu, Joy Khag, Weng-Lang Wang, Yao-Ming Chang, Shih-Yu Chen, Chandan Guha, Shu-Ping Wang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Targeting the tumor microenvironment (TME) to be more immune permissive is a potential strategy for enhancing immunotherapies, such as chimeric antigen receptor T cell (CAR-T) therapy, providing a promising avenue for treating aggressive tumors such as glioblastoma multiforme (GBM). Tertiary lymphoid structures (TLS) are ectopic lymphoid aggregates that arise in response to chronic inflammation and mimic the structure and function of secondary lymphoid organs. The spontaneous presence of TLS in some solid tumors, including GBM, is associated with improved clinical outcome and responsiveness to immunotherapy. I hypothesize that membrane tethered anti-LTβR Sc-Fv (mTa) T cells can function as lymphoid tissue inducer cells (LTi) and interact with lymphoid tissue organizer cells (LTo) in the TME, that express lymphotoxin-β receptor (LTβR), to nucleate TLS. To this end, we screened commercially available murine LTβR agonist antibodies to induce NFκB signaling. When a lead candidate with robust NFkB activation was injected in situ into mice orthotopically injected with a murine glioma cell line, we found multiple TLS-like lymphoid aggregates. We therefore sequenced and cloned a membrane-tethered version of this antibody alongside a CAR construct to generate mTa-CAR-T cells. mTa-CAR T cells could activate LTβR in an NFκB-GFP reporter assay. We proceeded to intratumorally inject these cells into a syngeneic mouse GBM tumor model to evaluate TLS induction and assess the ability of mTa-CAR T cells to improve antitumor efficacy and survival. This study attempts to identify strategies to remodel the TME of GBM and nucleate TLS using cellular therapy to improve the response to CAR-T therapy and survival of GBM patients.
 Citation Format: Nour Amwas, Darya Alizadeh, Christine Brown, Victor Chiu, Colt Egelston, Diana Gumber, Hanmin Wang, John Williams, Leo David Wang. Inducing tumor associated tertiary lymphoid structures using cellular therapy [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A038.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d654bf452e2e4a90e37163a7af80b45f3f9eb5df" target='_blank'>
              Abstract A038: Inducing tumor associated tertiary lymphoid structures using cellular therapy
              </a>
            </td>
          <td>
            Nour Amwas, D. Alizadeh, Christine J Brown, Victor Chiu, Colt A. Egelston, D. Gumber, Hanmin Wang, John Williams, L. Wang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc0740c1e17e6a1efafc307103e61f95f2a3a59" target='_blank'>
              Mathematical modeling of multicellular tumor spheroids quantifies inter-patient and intra-tumor heterogeneity
              </a>
            </td>
          <td>
            Adam A. Malik, Kyle C. Nguyen, John T. Nardini, C. Krona, Kevin B. Flores, Sven Nelander
          </td>
          <td>2025-02-15</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Metastatic cancer remains a leading cause of cancer-related mortality, with a 5-year survival rate of just 8% for pancreatic ductal adenocarcinoma (PDAC). Among patients with metastatic PDAC, those with liver metastases experience significantly worse outcomes compared to the rare cases of isolated lung metastases. Recent findings by Link and colleagues reveal that these distinct metastatic patterns reflect underlying biological differences beyond established molecular subtypes. Specifically, the authors curated a primary organotropism (pORG) gene signature that is enriched in the liver cohorts. In detail they found that high-pORG/liver-avid tumors are characterized by high replication stress, enriched DNA repair pathways, and an immunosuppressive microenvironment, whereas low-pORG/lung-avid tumors display stronger immune infiltration, higher T cell density and reduce the richness of TCR repertoire, and better survival outcomes. These insights suggest that the clinical pattern of metastasis provides meaningful information about tumor biology and prognosis, complementing current subtype classifications in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cee75be70d4938d7f15deed0722cf5fa17de26a" target='_blank'>
              The Biology in the Pattern: Metastatic organotropism and clinical outcome depend on DNA damage response and immune interactions in pancreatic cancer.
              </a>
            </td>
          <td>
            A. Papargyriou, Maximilian Reichert
          </td>
          <td>2025-03-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>