<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-22 09:31:22 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer prognosis relies not only on genetic and molecular biomarkers but also on the spatial organization of tumor and immune cells within the tumor microenvironment. Recent advances in spatial biology, particularly hyperplex immunofluorescence (IMF) imaging, have enabled high-dimensional, quantitative assessment of cell-cell interactions at the protein level. Nearest neighbor analysis (NNA) and proximity analysis have emerged as crucial computational methods for quantifying spatial distributions of tumor, stromal, and immune cells in hyperplex IMF datasets, providing insights into tumor heterogeneity, immune infiltration, and treatment response. This review explores the current state of nearest neighbor and proximity analysis in cancer research, focusing on their applications in prognosis using single-cell spatial proteomics data generated by hyperplex IMF imaging. We summarize key computational approaches, including nearest neighbor distance metrics, Ripley's K-function, Voronoi tessellation, and graph-based models, that characterize spatial architecture within the tumor microenvironment. We highlight recent applications of hyperplex IMF in cancers showcasing how spatial proteomic signatures improve prognostic models. Furthermore, we discuss the integration of machine learning and AI-driven methods to leverage these spatial features for predictive modeling. Despite significant progress, challenges remain, including standardization of methodologies, variability in imaging technologies, and the need for large-scale, high-quality datasets. Addressing these challenges could lead to more accurate risk stratification and personalized treatment strategies. By providing a comprehensive overview of nearest neighbor and proximity analysis in the context of hyperplex IMF-based spatial proteomics, this review aims to bridge the gap between computational methodologies and clinical applications, offering new perspectives on how spatial organization at the protein level influences cancer prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dab8bc0120eec053b87275d8e1359f11409b17b" target='_blank'>
              From Spatial Patterns to Prognosis: Decoding Single-Cell Architecture in Cancer with Hyperplex Immunofluorescence Imaging
              </a>
            </td>
          <td>
            Mohammadreza Azimi
          </td>
          <td>2025-07-28</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, Divij Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z. Lin, Dylan Schaff, Sydney M. Shaffer, Drew M Pardoll, Christina Jackson, Nancy R. Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Intrinsic genetic alterations and dynamic transcriptional changes contribute to the heterogeneity of solid tumors. Lung adenocarcinoma (LUAD) is characterized by its significant histological, cellular and molecular heterogeneity. The present study aimed to study the spatial transcriptomics of primary LUAD with initial hopes to decipher molecular characteristics of subtype transitions in LUAD progression, offering new insights for novel therapeutic strategies. Spatial transcriptomics libraries were first generated from tumor samples collected from patients with LUAD who underwent surgical resection in The Fourth Hospital of Hebei Medical University in 2022 and were sequenced using Illumina NovaSeq 6000 system. The processed data were analyzed for differential gene expressions and networks, and were annotated according to cell type, spatial ligand-receptor interaction and trajectory inference. Our analysis revealed 34 annotated cell types, with cancer-associated fibroblasts (CAFs) being the most abundant, playing a crucial role in tumor microenvironment remodeling and prognosis. We noted significant spatial correlations between various immune cells and found that different histological subtypes displayed unique cell composition profiles, particularly in the micropapillary subtype, which exhibited higher macrophage proportions and distinct gene expression pathways related to extracellular matrix organization and receptor tyrosine kinase signaling. Additionally, we explored the dedifferentiation states within these subtypes, identifying that region with higher dedifferentiation scores corresponded to increased tumor invasiveness and potential drug resistance. Our findings demonstrate dynamic biological changes and dedifferentiation states of tumor subtypes during the progression process. This study reveals important biological processes in tumor development and may offer valuable guidance for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a252a3c50c2377c8fca1733740eaa8f4897e893" target='_blank'>
              Spatial transcriptomics uncovers immune-cell plasticity and dedifferentiation signatures in aggressive lung adenocarcinoma subtypes
              </a>
            </td>
          <td>
            Huiyan Deng, Qingyi Liu, Ziqiang Tian, Guan-Yuan Liu, Shize Wang, Di Liang, Yun Wang, Yaqing Han, Shaonan Xie
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastatic cancer is the final frontier in disease progression with few avenues of treatment open to patients. The failure of therapeutic options at the end-stage disease is confounded by the presence of diverse clones in the metastatic tumor which contributes to intratumoral heterogeneity (ITH), resulting in intrinsic and acquired drug resistance. We aim to elucidate the contribution of ITH to the development of metastasis with a focus on the origins and molecular mechanisms driving ITH, as well as the clinical and technical challenges of acquiring and studying metastatic cohorts in which we can investigate this phenomenon. Bioinformatic approaches that could help in silico analysis will be discussed, which could shed light on how to design new therapeutic strategies for metastatic cancer and give hope to patients living under the shadow of the sword of Damocles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ee12f1d193c46bad8ef5d6e4f9b07aba3289ca" target='_blank'>
              Role of intratumoral heterogeneity in metastatic progression and drug resistance
              </a>
            </td>
          <td>
            Neeha Sinai Borker, Jyothilakshmi Sajimon, Seemadri Subhadarshini, Rifat Aara, M. Jolly, Jyothy S Prabhu, Radhika Nair
          </td>
          <td>2025-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cellular anatomy and signaling vary across niches, which can induce gradated gene expressions in subpopulations of cells. Such spatial transcriptomic gradient (STG) makes a significant source of intra-tumor heterogeneity. We present Local Spatial Gradient Inference (LSGI), a computational framework that systematically identifies spatial locations with prominent, interpretable STGs from spatial transcriptomic (ST) data. We demonstrate LSGI in tumor ST datasets and identify pan-cancer and tumor-type specific pathways with gradated patterns, highlighting the ones related to spatial transcriptional intratumoral heterogeneity. LSGI enables interpretable STG analysis, which can reveal novel insights in tumor biology from the increasingly reported tumor ST datasets. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03716-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2919ee7022acd5a65c198642bf710ee21f0432" target='_blank'>
              LSGI: interpretable spatial gradient analysis for spatial transcriptomics data
              </a>
            </td>
          <td>
            Qingnan Liang, Luisa M. Solis Soto, C. Haymaker, Ken Chen
          </td>
          <td>2025-08-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Meningioma is a common primary intracranial tumor with high recurrence and metastasis rate. Delineating the pathological ecosystem at the brain-tumor interface (BTI) of meningioma is critical for understanding the mechanisms of tumor metastasis and developing effective new therapies. Methods To identify biomarkers of early metastasis and discover potential therapeutic targets, we integrated single-cell transcriptome datasets of meningioma, and identified the cell populations and molecular signatures uniquely present at the BTI. Results A specific BTI-enriched tumor cell population with a pro-EMT (epithelial mesenchymal transition) characteristics was associated with invasion and metastasis, and ANXA2 and COL5A1 were detected as the biomarkers for these BTI-enriched tumor cells. Additionally, we characterized the BTI-specific immunosuppressive microenvironment composed of SPP1+ tumor-associated macrophages, as well as specific endothelial cells (ACKR1high) and pericytes (THY1high) promoting the highly malignant invasive state of angiogenesis. Conclusions Collectively, BTI in meningioma is a metastatic and immunosuppressive zone. We have discovered potential biomarkers that help detect early metastasis and recurrence of meningioma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06935-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea39ae7cd89d7f388a32faad30c0efc42673529" target='_blank'>
              Single-cell transcriptomic analysis reveals metastatic and immunosuppressive characteristics in meningioma brain-tumor interface
              </a>
            </td>
          <td>
            Boya Huang, Jinlian Liang, Xiaogen Tang, Yue Zhu, Lijuan Gao, Dongwei Fu, Cheng Wei, Yifang Li, Chao Ke, Hongyi Zhang, Oscar Junhong Luo
          </td>
          <td>2025-08-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment responses remain highly variable due to the complexity and heterogeneity of the tumor microenvironment. The tumor microenvironment comprises malignant, immune, stromal, and endothelial cells, along with extracellular matrix and soluble factors, all organized into spatially distinct communities that evolve dynamically throughout tumor progression and therapy. These spatial structures orchestrate tumor behavior, immune evasion, and drug resistance. Recent breakthroughs in spatial omics technologies, including spatial transcriptomics and spatial proteomics, have enabled high-resolution, multiplexed mapping of tissue architecture and molecular characteristics. These technologies provide valuable insights into how the spatial organization of cells and signaling networks within the tumor microenvironment influences therapeutic efficacy. Notably, specific structures, such as tertiary lymphoid structures, fibroblast-mediated stromal barriers, and vascular heterogeneity have been identified as spatial determinants of treatment response. By delineating cellular communities and their interactions, spatial omics technologies can reduce intratumoral complexity into clinically interpretable modules. This review summarizes the diversity of these spatial structures and their relationships with treatment outcomes in immunotherapy, chemotherapy, radiotherapy, and targeted therapy. In addition, it highlights present challenges in data integration, analytical standardization, and functional validation, and discusses future directions for incorporating spatial omics technologies into precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572818b2484643c196593baace3df887c52839e" target='_blank'>
              Structural cell communities in the tumor microenvironment: Spatial determinants of therapeutic response
              </a>
            </td>
          <td>
            Xuan Cui, Y. Ni, Xia Lei, Lan Zhao, Cheng Qian, Juanjuan Shan
          </td>
          <td>2025-08-14</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract High‐risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single‐cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy‐treated high‐risk neuroblastomas from archived, formalin‐fixed, paraffin‐embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p‐gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical‐like cells expressed several oncogenic ligand‐encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) incidence is rising, and treatment remains challenging unless surgery is curative. Tumour heterogeneity contributes to resistance against both chemotherapy and immune checkpoint inhibitors, underscoring the need to better understand the complex tumour microenvironment (TME). While tumour models derived from cancer tissue from patients have advanced cancer research, they often fail to capture functional RCC heterogeneity and key TME components. We developed a 3D model system with a high success rate from resected tumour, retaining cancer, stromal, and immune cell populations. This system is fully compatible with advanced imaging technologies, including mass spectrometry imaging (MSI) and live-cell multiplex imaging. By integrating static spatial analysis with dynamic live-cell visualisation, our system provides unique insights into tumour heterogeneity, microenvironment metabolic crosstalk, and real-time cellular responses. Phenotypic characterization of the tumoroids showed strong histological resemblance to the original resected tissue, indicating that the tumoroids are reflective of the tumour in vivo and suitable as a representative model system. Additionally, DESI-MSI revealed distinct lipidomic profiles within patient-derived ccRCC tumoroids, capturing spatial metabolic heterogeneity reflective of the primary tissue. Lipid signatures varied across tumour regions, with phospholipid subclasses distinguishing epithelial, endothelial, and highly proliferative cell populations. Notably, non-clear cell regions exhibited reduced lipid droplet and fatty acid content, aligning with aggressive tumour phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc004e66880e4588257a07a21abe175bda8bb5d0" target='_blank'>
              Kidney tumoroid characterisation by spatial mass spectrometry with same-section multiplex immunofluorescence uncovers tumour microenvironment lipid signatures associated with aggressive tumour phenotypes
              </a>
            </td>
          <td>
            Hazem Abdullah, Greice M Zickuhr, I. Um, Alexander Laird, Peter Mullen, David J. Harrison, A. Dickson
          </td>
          <td>2025-09-16</td>
          <td>npj Imaging</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            Hyun Seung Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="The formation of budding cancer cells at the invasive front of solid tumors is one of the first steps of metastasis. However, this process is still incompletely elucidated. Here, we used spatial molecular imaging to disentangle the complex interactions between cancer cells and the tumor microenvironment at the invasive front of colorectal tumors. Employing a 1000-plex gene panel, we defined all major cell types in tumors and adjacent normal tissue with accurate spatial information. Individual cancer cell clusters were located together, consistent with an expected mutation- and epigenetic-driven clonal evolution. However, cancer cell clusters encompassing budding cells exhibited a markedly different spatial distribution as they also contained cells that were scattered around the periphery of the main cancer cell masses. Moreover, these cells were frequently in contact with cancer-associated fibroblasts (CAFs) and underwent broad gene expression changes, mainly related to epithelial-mesenchymal transition (EMT), remodeling of the extracellular matrix (ECM), and migration. In addition, we defined an 11-gene signature (TYK2, IL2RG, KRT17, HLA-B, NPPC, WIF1, IL32, B2M, CCND1, CRIP1, ITGB1), which characterizes cancer cells en route to metastasis and is associated with inferior outcomes. Collectively, our findings suggest that CAFs induce pro-invasive gene expression changes involved in EMT, ECM remodeling, and migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9341a52e672e1a44fbf627c6049087936b5a159e" target='_blank'>
              Cancer-associated fibroblasts shape the formation of budding cancer cells at the invasive front of human colorectal cancer
              </a>
            </td>
          <td>
            J. L. García-Rodríguez, Ulrik Korsgaard, Stine M Vissing, Thea Petersen Paasch, Mariana Semenova, Simon L Vendelbo, Eva F Jensby, Hannah L Williams, Paul Vinu Salachan, Camilla Blunk Brandt, J. Hanimann, Lin Lin, I. Zlobec, K. D. Sørensen, Jørgen Kjems, Henrik Hager, L. S. Kristensen
          </td>
          <td>2025-09-16</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Tumor heterogeneity in metastatic prostate cancer (mPC) is well established, but comprehensive characterization using routine sampling remains challenging. Autopsy-based research addresses this obstacle by enabling broad tissue collection within individual patients after treatment. In this issue of the JCI, Roudier et al. analyzed samples from a mPC research autopsy cohort, revealing extensive inter- and intratumor heterogeneity across patients and at the cellular level. The authors associated this variability with genomic, phenotypic, and clinical features and explored the importance of tumors expressing both androgen receptor and neuroendocrine markers. Their findings demonstrate heterogeneity across metastatic sites that may influence treatment response and clinical outcomes, informing future therapeutic strategies in mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1162fb7a103df269563289c6dd21cc5f28aaf11" target='_blank'>
              Uncovering phenotypic heterogeneity through research autopsy in lethal prostate cancer
              </a>
            </td>
          <td>
            Sylvie S.W. Chan, Osvaldas Vainauskas, Gerhardt Attard
          </td>
          <td>2025-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Lymph node metastasis markedly worsens prognosis in lung adenocarcinoma (LUAD); however, the evolutionary dynamics and regulatory mechanisms underlying the heterogeneity of malignant epithelial cells during this process remain poorly understood and warrant comprehensive investigation. Methods We performed a comprehensive single-cell transcriptomic analysis of epithelial cells from 18 samples comprising normal lung tissue and lymph node metastases. Malignant epithelial cells were identified via inferred copy number variation (CNV) profiles. Key malignant subpopulations were further characterized through trajectory inference, cell–cell communication mapping, gene set variation analysis (GSVA), and reconstruction of transcription factor regulatory networks. To assess clinical relevance, we developed and validated a prognostic model—termed the EAS score—based on the transcriptional signatures of malignant epithelial subsets, using integrated data from multiple TCGA and GEO cohorts. The functional role of the hub gene SELENBP1 was experimentally validated through quantitative PCR (qPCR), Western blotting, immunohistochemistry (IHC), Transwell migration assays, colony formation assays, flow cytometry, ROS quantification, and subcutaneous tumorigenesis assays in vivo. Results Single-cell transcriptomic analysis identified four distinct malignant epithelial subtypes (Clusters 0–3), each characterized by unique patterns of CNV. Leveraging these defined cellular subpopulations, we constructed a highly accurate model for prognostication in LUAD, enabling reliable classification of patients based on clinical outcomes. Through detailed comparisons between groups with divergent prognostic risks, the study revealed notable differences across the tumor microenvironment (TME), including alterations in pathway activity, gene enrichment distributions, mutation profiles, and anticipated responses to immune checkpoint blockade. In addition, functional validation experiments confirmed that SELENBP1 plays a tumor-suppressive role, further supporting its relevance as a potential intervention target in LUAD. Conclusion This research provides insights into the evolutionary complexity and heterogeneity of malignant epithelial populations in lymph node metastatic sites of LUAD. It also presents a scoring system based on prognostic indicators, which serves as a reliable tool for forecasting patient survival outcomes. Moreover, the discovery of SELENBP1 as a candidate tumor suppressor emphasizes its importance in guiding both clinical risk categorization and the design of personalized treatment strategies for individuals classified as high-risk LUAD cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a2796c042204c38d000db3a271f2db16f9cb8" target='_blank'>
              Single-cell transcriptomic profiling reveals the heterogeneity of epithelial cells in lung adenocarcinoma lymph node metastasis and develops a prognostic signature
              </a>
            </td>
          <td>
            Qiuqiao Mu, Han Zhang, Ying Shi, Mengli Xue, Jingxian Wang, Yun Ding, Lin Tan, Hui Yuan, Xin Li, Daqiang Sun
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC), basal cell carcinoma (BCC), and melanoma, the three major types of skin cancer, account for over 70% of all cancer cases. Despite their prevalence, the skin cancer microenvironment remains poorly characterized, both in the outer skin layer where the cancer originates and at the deeper junctional and dermal layers into which it progresses. To address this, we integrated 12 complementary spatial single-cell technologies to construct orthogonally-validated cell signatures, spatial maps, and interactomes for cSCC, BCC, and melanoma. We comprehensively compared and integrated these spatial methods and provided practical guidelines on experimental design. Integrating four spatial transcriptomics platforms, we found keratinocyte cancer signatures, including six consistently validated gene markers. Spatial integration of transcriptomics, proteomics, and glycomics uncovered cancer communities enriched in melanocyte–fibroblast–T-cell colocalization with altered tyrosine and pyrimidine metabolism. Ligand-receptor analysis across >700 cell-type combinations and >1.5 million interactions highlighted key roles for CD44, integrins, and collagens, with CD44-FGF2 emerging as a potential therapeutic target. We consistently found differential interactions of melanocytes with fibroblasts and T-cells. We validated these interactions using Opal Polaris, RNAScope, and Proximal Ligation Assay. To integrate population-scale data, genetic association mapping in >500,000 individuals suggested SNPs enriched for spatial domains containing melanocytes, dysplastic keratinocytes, and fibroblasts, shedding light on functional mechanisms linking genetic heritability to cells within cancer tissue. This publicly available multiomics resource offers insights into the initiation and progression of the most lethal skin cancer (melanoma) and the most common forms (cSCC and BCC) and can be explored interactively at https://skincanceratlas.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87a9509055032cdb5b434f194c71651a4b05c7db" target='_blank'>
              Integrating 12 Spatial and Single Cell Technologies to Characterise Tumour Neighbourhoods and Cellular Interactions in three Skin Cancer Types
              </a>
            </td>
          <td>
            P. Prakrithi, Laura F Grice, Feng Zhang, Levi Hockey, Samuel X. Tan, Xiao Tan, Zherui Xiong, Onkar Mulay, Andrew Causer, Andrew Newman, Duy Pham, Guiyan Ni, Kelvin Tuong, Xinnan Jin, Eunju Kim, Minh Tran, Hani Vu, Nicholas M Muller, E. Killingbeck, M. Gregory, S. Teoh, Tuan Vo, Min Zhang, M. T. Landi, Kevin M. Brown, M. Iles, Zachary Reitz, K. Devitt, Liuliu Pan, A. Kulasinghe, Y. Kao, Michael Leon, Sarah Murphy, Hiromi Sato, J. G. Cruz, Snehlata Kumari, H. N. Luu, Sarah Warren, Chris L. D. McMillan, J. Henricson, Chris Anderson, David A Muller, Arun Everest-Dass, Blake O’Brien, M. Seviiri, Matthew H. Law, H. P. Soyer, Ian H. Frazer, Youngmi Kim, Mitchell S. Stark, Kiarash Khosrotehrani, Quan Nguyen
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Brain metastasis is a serious and often early complication in lung adenocarcinoma, the most common type of lung cancer. While many studies have examined treated or advanced cases, little is known about the tumor environment at initial diagnosis. This study analyzed tissue from five patients whose brain metastases were the first signs of lung adenocarcinoma. Using spatial proteomic profiling, we compared protein expression between matched brain and lung tumors. Brain metastases showed reduced immune activity and increased markers of cell growth and immune evasion. These findings highlight the distinct biology of brain lesions even before treatment begins and suggest early immune escape. Understanding these early changes may support future efforts to develop site-specific strategies targeting brain metastases at initial presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71912171aef6f8867b2853c36b6318ebf2376ae" target='_blank'>
              Spatial Omics Profiling of Treatment-Naïve Lung Adenocarcinoma with Brain Metastasis as the Initial Presentation
              </a>
            </td>
          <td>
            Seoyeon Gwon, Inju Cho, Jieun Lee, Seung Yun Lee, Kyue-Hee Choi, Taejung Kim
          </td>
          <td>2025-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The intrinsic complexity of biological processes often hides the role of dynamic microenvironmental cues in the development of pathological states. Microphysiological systems (MPSs) are emerging technological platforms that model in vitro dynamics of tissue-specific microenvironments, enabling a holistic understanding of pathophysiology. In our previous works, we engineered and used breast tumor MPS differing in matrix stiffness, pH, and fluid flow mimicking normal and tumor breast tissue. High-dimensional data using two distinctive human breast cell lines (i.e., MDA-MB-231, MCF-7), investigating cell proliferation, epithelial-to-mesenchymal transition (EMT), and breast cancer stem cell markers (B-CSC), were obtained from breast-specific microenvironments. Recognizing that the widespread adoption of MPS requires tailoring its complexity to application demands, we herein report an innovative machine-learning (ML)-based approach to analyze MPS data. This approach uses unsupervised k-means clustering and feature extraction to inform on key markers and specific microenvironments that distinguish invasive from non-invasive breast cell phenotypes. This data-driven approach streamlines future experimental design and emphasizes the translational potential of integrating MPS-derived insights with ML to refine prognostic tools and personalize therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f6b70ba36ca6bdecf1ece254a7da2cd8bdc7c" target='_blank'>
              Data-Driven Sustainable In Vitro Campaigns to Decipher Invasive Breast Cancer Features
              </a>
            </td>
          <td>
            Lekha Shah, V. Breschi, Annalisa Tirella
          </td>
          <td>2025-07-25</td>
          <td>ACS Biomaterials Science & Engineering</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background T-cell activation and clonal expansion are essential to effective immunotherapy responses in non-small cell lung cancer (NSCLC). The distribution of T-cell clones may offer insights into immunogenic mechanisms and imply potential prognostic and predictive information. Methods We analyzed α/β T-cell receptor (TCR) clonality using RNA-sequencing of bulk frozen tumor tissue from 182 patients with NSCLC. The data was integrated with molecular and clinical characteristics, extensive in situ imaging, and spatial sequencing of the tumor immune microenvironment. TCR clonality was also determined in an independent cohort of nine patients with immune checkpoint-treated NSCLC. Results TCR clonality (Gini index) patterns ranged from high T-cell clone diversity with high evenness (low Gini index) to clonal dominance with low evenness (high Gini index). Generally, TCR clonality in cancer was lower than in matched normal lung parenchyma distant from the tumor (p=0.021). The TCR clonality distribution between adenocarcinoma and squamous cell carcinoma was similar; however, smokers showed a higher Gini index. While in the operated patient with NSCLC cohort, TCR clonality was not prognostic, in an immune checkpoint inhibitor-treated cohort, high TCR clonality was associated with better therapy response (p=0.016) and prolonged survival (p=0.003, median survival 13.8 vs 2.9 months). On the genomic level, a higher Gini index correlated strongly with a lower frequency of epidermal growth factor receptor (EGFR) and adenomatous polypsis coli (APC) gene mutations, but a higher frequency of P53 mutations, and a higher tumor mutation burden. In-depth characterization of the tumor tissue revealed that high TCR clonality was associated with an activated, inflamed tumor phenotype (PRF1, GZMA, GZMB, INFG) with exhaustion signatures (LAG3, TIGIT, IDO1, PD-1, PD-L1). Correspondingly, PD-1+, CD3+, CD8A+, CD163+, and CD138+immune cells infiltrated cancer tissue with high TCR clonality. In situ sequencing recovered single dominant T-cell clones within the patient tumor tissue, which were predominantly of the CD8 subtype and localized closer to tumor cells. Conclusion Our robust analysis pipeline characterized diverse TCR repertoires linked to distinct genotypes and immunologic tumor phenotypes. The spatial clustering of expanded T-cell clones and their association with immunological activation underscores a functional, clinically relevant immune response, particularly in patients with NSCLC treated with checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cb95082ff80184987c55803cfe4d35d428a707" target='_blank'>
              Spatial TCR clonality and clonal expansion in the in situ microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Hui Yu, Anastasia Magoulopoulou, Rose-Marie Amini, Maria Chatzinikolaou, Masafumi Horie, A. Lindberg, Artur Mezheyeuski, M. Backman, Andreas Metousis, H. Brunnström, M. Marincevic, J. Botling, J. Mattsson, K. Kärre, Karin Leandersson, Mats Nilsson, Carina Strell, P. Micke
          </td>
          <td>2025-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic crosstalk among diverse cellular populations contributes to shaping a competitive and symbiotic tumor microenvironment (TME) to influence cancer progression and immune responses, highlighting vulnerabilities that can be exploited for cancer therapy. Using a spatial multiomics platform to study the cell-specific metabolic spectrum in hepatocellular carcinoma (HCC), we map the metabolic interactions between different cells in the HCC TME and identify a unique tumor-immune-cancer-associated fibroblast (CAF) "interface" zone, where cell-cell interactions are enhanced and accompanied by significant upregulation of lactic acid and long-chain polyunsaturated fatty acids. Further combining single-cell mass spectrometry imaging of patient-derived tumor organoids, cocultured CAFs, and macrophages, we demonstrate that CAFs increase glycolysis and secrete lactic acid to the surrounding microenvironment to drive immunosuppressive macrophage M2 polarization. These findings facilitate the understanding of cancer-associated metabolic interactions in complex TME and provide clues for targeted clinical therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b292d4bb0787fa6280b9910112e99765dee292d" target='_blank'>
              Integrating spatial omics and single-cell mass spectrometry imaging reveals tumor-host metabolic interplay in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Panpan Chen, Haoyuan Geng, Bangzhen Ma, Yaqi Zhang, Zihan Zhu, Min Li, Shiping Chen, Xiao Wang, Chenglong Sun
          </td>
          <td>2025-07-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Lung cancer, a leading cause of cancer death, displays profound histologic and molecular heterogeneity across adenocarcinoma, squamous, and small‐cell types. Clinically, tumours can shift between these states, reflecting lineage plasticity—the reprogramming of differentiated cells to alternate identities. Pre‐existing genomic/epigenomic diversity and microenvironmental cues supply the substrates and pressures for plasticity from disease onset. This review anchors plasticity within normal lung development to clarify how fate programs are co‐opted to drive progression, immune escape, therapy resistance, and invasion. Main text Focusing on the intricate interplay between lineage dysregulation and tumour progression in lung cancer, this review integrates insights from lung tissue development to explore the pivotal molecules and mechanisms driving lineage plasticity, alterations and migration during lung carcinogenesis and progression. Recent research findings on lung cancer lineage plasticity are synthesised, shedding light on the role of transcriptional and epigenetic regulators in disrupting tumour lineages. Particular emphasis is placed on how tumour microenvironmental factors, such as hypoxia, stromal cells and immune cells, reshape tumour cellular profiles by modulating the epigenomic landscape. Furthermore, this review specifically discusses the impact of epidermal growth factor receptor (EGFR) and KRAS mutations on lung cancer progression and the consequent immune escape mechanisms they engender. Importantly, we highlight that lineage regulation persists throughout tumour development, from the early onset of lung adenocarcinoma (LUAD) to its progression through late‐stage dedifferentiation and metastasis. We evaluate the implications of these factors on treatment resistance in lung cancer and focus on innovative therapeutic strategies targeting lineage plasticity. Conclusions Lineage plasticity spans the entire course of lung cancer, from early tumorigenesis through metastasis to treatment resistance. Lineage transitions that occur during tumour progression arise from specific combinations of genomic and epigenetic alterations and are further shaped by microenvironmental forces such as hypoxia, stromal remodeling, and immune pressure. By summarising current research advancements, we aim to provide new insights for future lung cancer research and to promote the development of more effective therapeutic interventions. Key points Lineage plasticity runs through the entire process of lung cancer progression and drug resistance, and drives early tumorigenesis via lineage imbalance. Certain driver mutations have lineage‐restricted tumorigenic potential, requiring lineage reprogramming for tumor initiation. Lineage transitions in lung cancer require specific genomic and epigenetic alterations. Lineage plasticity insights provide a mechanistic framework linking lung cancer origin, evolution, and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07aa8707388affcb2b558f5da5a8327303f46a8" target='_blank'>
              Persistent lineage plasticity driving lung cancer development and progression
              </a>
            </td>
          <td>
            Fanchen Meng, Jianyu Li, Zhijun Xia, Qinglin Wang, Qinhong Sun, Siwei Wang, Lin Xu, Rong Yin
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Summary Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34+ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV+ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, Paige M. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Circulating tumor cells (CTCs) are the primary mediators of metastatic disease. After leaving the tumor, they can travel through the blood alone or with cluster partners to aid them on their way to distant metastatic sites. While CTCs are critical diagnostic tools for monitoring cancer progression, the biology underlying their behavior is obscured by their rarity and heterogeneity. To unravel the complexity of their actions, we undertook a large-scale longitudinal study to determine the factors that affect their presence, clustering, and their impacts on disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ef013e40043bfc40c0608d2cc2f97f3380a6366" target='_blank'>
              Comprehensive Longitudinal Linear Mixed Modeling of CTCs Illuminates the Role of Trop2, EpCAM, and CD45 in CTC Clustering and Metastasis
              </a>
            </td>
          <td>
            Seth D. Merkley, Huining Kang, U. Brown-Glaberman, Dario Marchetti
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2642fd5880574a79a83aa30d56f3b2134c4d69" target='_blank'>
              Applications of single-cell transcriptomics: updated insights in endometrial cancer.
              </a>
            </td>
          <td>
            Shuyue Xiao, Huixin Li, Jianyao Liu, Xinyi Xie, Hanzi Xu, Zhen Gong, Shanliang Zhong
          </td>
          <td>2025-09-03</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Continuous passaging of cancer cell lines can drive phenotypic and genotypic divergence, potentially compromising the reliability of such models. In this study, we show that two late-passage strains (S1 and S2) of ovarian cancer cell line SKOV3, although authenticated via short tandem repeat (STR) profiling as identical, exhibit substantial differences in morphology, transcriptomic signatures, ability to form 3D cultures and chemotherapeutic responses. Notably, S1 formed compact 3D spheroids and exhibited enhanced epithelial-mesenchymal transition (EMT) pathway activity, whereas S2 displayed a more proliferative, MYC-driven phenotype with larger spheroid structures requiring higher seeding densities. Transcriptomic analysis revealed pathways associated with hypoxia, EMT and angiogenesis in 3D culture, highlighting the complexity introduced by dimensionality in tumour modelling. Critically, S1 showed higher sensitivity to doxorubicin than S2 (IC50 of 0.12 µM versus 1.28 µM, P=0.0001), indicating how clonal evolution can confound drug-response assays. Ultimately, our findings suggest that although STR profiling remains essential for cell line authentication, functionally distinct subpopulations can arise and coexist within the same culture, and their isolation may reveal divergent phenotypes that compromise reproducibility in preclinical cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b19ad530ecaca77ba40f9ce705c8944938d900" target='_blank'>
              Extended passaging of the SKOV3 ovarian cancer cell line leads to two phenotypically different strains
              </a>
            </td>
          <td>
            Eglė Žymantaitė, M. Gabrielaitė, V. Pašukonienė, A. Mlynska
          </td>
          <td>2025-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Spread through air spaces (STAS) represents a novel invasion mechanism in adenocarcinoma that considerably influences lung cancer clinical outcomes; however, studies of its mechanisms at the spatial level are lacking. Methods We used the NanoString GeoMx digital spatial profiling (DSP) technology to conduct a spatial transcriptomic analysis of surgically resected tissues from non-small-cell lung cancer (NSCLC) patients with or without STAS. Results Compared with tumor nests in non-STAS patients, HLA-DRB5 and RASGRF1 were significantly less expressed in compartments of STAS, suggesting their inhibitory roles in the occurrence of STAS. Meanwhile, an increase in CD4 T memory cells and a decrease in B cells were observed in the tumor immune microenvironment of STAS. Furthermore, distinct molecular profiles were observed between tumor cells in tumor nests and in air spaces in STAS patients, which was highlighted by the elevated ITGA2 expression in the air spaces. These results were validated in an independent cohort by multiplex immunofluorescence stainings. Conclusion This study is the first to use DSP to analyze spatial transcriptomic profiles of NSCLC tumor nests and air space tumors, and it identifies potential module features that may be used for STAS identification and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570168f77732c756800b064b7d546cfa69566936" target='_blank'>
              Spatial deciphering of the transcriptomic heterogeneity of tumor spread through air spaces in lung cancer
              </a>
            </td>
          <td>
            Wenhao Wang, Wenhao Zhou, Jingli Fan, Tao Jiang, Guang Yang, Congcong Song, Siwei Xu, Haitao Luo, Huining Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A whole pathology section contains approximately 1,000,000 cells of various types, this large-scale heterogeneity of cells and non-cellular constituents constructs a mutually competitive community. Conventional pixel-based visual processing techniques are insufficient to accurately capture the complexities inherent with cell-entity deployment and formation strategy. Here, we conquered segmentation and classification of all cells on the whole pathology sections from 387 hepatocellular carcinoma (HCC) patients across six cohorts with 57 pathologists assisted. Further, an AI system called Hybrid Graph Neural Network-Transformer system (HGTs) was proposed. It precisely predicted local recurrence of postoperative HCC by analyzing cell interactions across multiple scales, from cell-to-cell, cell-community, to tissue-level interactions. The proposed HGTs outperformed existing SOTA methods, with the C-index improving by 23.1% to reach 0.823, by further integrating multimodal data, including clinical information and immunohistochemical markers. A set of spatial relational biomarkers influencing tumor prognosis was discovered and quantitatively validated. They include the frequency of tumor-lymphocyte and tumor-tumor interactions, the distribution and sparsity of key cellular communities, and the degree of fibrosis in adjacent peritumoral tissues. Utilizing the anti-tumor potential of this marker set, we’re developing therapies to enhance the immune system’s fight against cancer. All cell semantic segmentation datasets and code are publicly available: https://github.com/Yuan1z0825/HGTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2cd5bfdebf32b51dd7e8525e95b35b9df810fb6" target='_blank'>
              Cell graph analysis in hepatocellular carcinoma: predicting local recurrence and identifying spatial relationship biomarkers
              </a>
            </td>
          <td>
            Yizhe Yuan, Ziyin Zhao, Xin Fang, Qing Zhang, Wenqing Zhong, Midie Xu, Gongqi Li, Rushi Jiao, Heng Yu, Ruoxi Wang, Shuyu Liu, Weitao Zu, Bingsen Xue, Yuze Chen, Chengxiang Wang, Ya Zhang, Minghui Liang, Bing Han, Cheng Jin
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) was considered the most aggressive type of primary brain tumor, marked by poor clinical outcomes and a high tendency to relapse. The therapeutic efficacy of GBM was significantly compromised by tumor heterogeneity, dysregulated metabolic pathways, the formation of an immunosuppressive microenvironment, and treatment resistance. Therefore, multi-dimensional therapeutic strategies targeting GBM-specific molecular features, its intrinsic properties, and microenvironmental regulatory networks were considered to potentially provide new breakthroughs for overcoming treatment resistance in GBM. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data processed with the Seurat package to accurately identify cell types. Spatial transcriptomics integrated Multimodal Intersection Analysis, TransferData, and Robust Cell Type Decomposition techniques to characterize the spatial distribution patterns of key cell subtypes. CellChat was employed to assess intercellular communication networks. Furthermore, in vitro experiments confirmed the main regulatory role of YEATS4 (key transcription factor of C2 PCLAF+ subtype) in GBM malignant progression. Results Through scRNA-seq, we identified the C2 PCLAF+ subtype in GBM and analyzed its molecular characteristics and functional role in tumor progression. This subtype exhibited a unique malignant phenotype, marked by significant proliferative activity, characteristic metabolic reprogramming, and dysregulated cell death regulation mechanisms. Spatial transcriptomics revealed its preferential localization within specific tumor niches. Furthermore, the C2 PCLAF+ subtype established a specific interaction with fibroblasts through the MDK-LRP1 ligand–receptor pair. Critically, silencing YEATS4 in vitro significantly inhibited GBM malignancy. Additionally, the prognostic risk score model based on the C2 PCLAF+ subtype demonstrated significant clinical translational value. Conclusion Our study systematically elucidated the malignant characteristics of the C2 PCLAF+ subtype and its molecular mechanisms driving GBM progression. This subtype promoted therapeutic resistance through unique metabolic reprogramming, MDK-LRP1-mediated microenvironmental interactions, and immunosuppressive properties. YEATS4 knockdown effectively suppressed malignant tumor behaviors, highlighting its therapeutic potential. These findings provided novel targeted intervention strategies to address GBM heterogeneity and treatment resistance, offering promising avenues for overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8a996fe1a4f26a1455e98643e994edaf34d934" target='_blank'>
              Single-cell and spatial atlas of glioblastoma heterogeneity: characterizing the PCLAF+ subtype and YEATS4’s oncogenic role
              </a>
            </td>
          <td>
            Siqi Ma, Yuwei Sun, Shaowei Zheng, Yilong Fu, Liangyu Wang, Dun Liu, Henan Jiao, Xuqiang Zhu, Xueyuan Li, Dongming Yan, Di Chen, Zi Ye
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of kidney renal clear cell carcinoma (KIRC) exhibits complex dynamics among immune, stromal, and malignant cells that drive immune escape (IE) mechanisms and influence clinical outcomes. Through single-cell RNA sequencing and high-dimensional weighted gene co-expression network analysis (hdWGCNA), we identified a crucial IE-related gene module most strongly associated with KIRC progression. Partitioning Around Medoids (PAM) clustering delineated two distinct IE patterns, with pattern one demonstrating prolonged patient survival. Employing advanced machine learning (ML) algorithms, we identified Cathepsin S (CTSS) as the most pivotal tumor suppressor, with elevated CTSS expression consistently predicting improved survival across multiple independent cohorts. Functional analyses revealed significant enrichment of CTSS in immune-regulatory pathways, including B/T cell activation and inflammatory activity. Mutation profiling uncovered distinct genomic alterations in 5q35.3 among CTSS-high tumors, while drug response prediction identified eight potential therapeutic agents (e.g., Navitoclax, Ibrutinib) exhibiting enhanced efficacy in these patients. Notably, CTSS expression strongly correlated with immune cell infiltration and established immunotherapy biomarkers, supporting its dual role as both a prognostic indicator and predictor of immune response. This study provides mechanistic insights into IE in KIRC and positions CTSS as a promising biomarker and therapeutic target for precision immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03267-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6364c60b20a894510d01361e9f698e8acfc1b2ab" target='_blank'>
              CTSS in the tumor microenvironment links immune escape and immunotherapy sensitivity in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Hanjing Zhou, Jun Ying, Xuchun Xu, Jian Huang
          </td>
          <td>2025-07-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Uterine leiomyosarcoma (ULSA) is a highly aggressive gynecologic malignancy characterized by early metastasis, profound immunosuppression, and resistance to conventional therapies, including immune checkpoint blockade (ICB). The intricate tumor microenvironment (TME) and cellular heterogeneity driving its progression and therapy resistance remain poorly defined. Methods We performed single-cell RNA sequencing (scRNA-seq) on metastatic lesions (pelvic cavity, rectum, peritoneum, bladder) from a treatment-naïve ULSA patient and compared them to normal uterine myometrium, MMM (n=5). Integrated analyses included cellular composition mapping, copy number variation (CNV) assessment, pseudotemporal trajectory reconstruction, cell-cell communication inference, functional enrichment, and validation via multiplex immunofluorescence (mpIF). Survival correlations were assessed using the TCGA-SARC cohort. Results In this study, the main finding is that the tumor microenvironment (TME) has a strong immunosuppressive effect. Firstly, its characteristic is exhausted CD8+T cells. This study found that as time progresses, the initial cell markers (CCR7, MAL) gradually disappear, while the exhaustion markers (LAG3, HAVCR2, TIGIT) are enriched. This is associated with poor prognosis. Secondly, the M2-polarized macrophages are mainly composed of M2-like tumor-associated macrophages (TAMs) with tumor-promoting characteristics (CD163, FTH1, FTL, TIMP1), and there is a polarization from M1 to M2. Finally, the immature, tumor-promoting N2 neutrophils (CD15+EDARADD+) enriched in the metastatic foci are associated with poor prognosis. The cell communication involves the interaction of MIF-(CD74+CD44) between T/B cells, as well as the role of the CXCL8 signaling axis in promoting angiogenesis, TAM polarization, and immunosuppression. Conclusion For the first time, a comprehensive single-cell map of ULSA was constructed, depicting a metastasis-susceptible cell subset (U11-EDARADD) and an extremely immunosuppressed tumor microenvironment dominated by depleted CD8+T cells, M2 macrophages and N2 neutrophils. These features shed light on the underlying mechanisms of chemotherapy resistance and immunotherapy failure. The biomarkers identified here (EDARADD, CLDN10, TMIGD2) as well as the dysregulated pathways (TGF-β, angiogenesis, MIF signaling) provide possible targets for future development of combined immunotherapy strategies against this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b01f016db1d6adc66dea5efac6b72384978f8f" target='_blank'>
              Single-cell profiling delineates the tumor microenvironment and immunological networks in patient-derived uterine leiomyosarcoma
              </a>
            </td>
          <td>
            Yi Guo, Dongsheng Shen, Yuhang Xiao, Chenghao Wu, Meiyi Chen, Lina Yang, Huaifang Li, Xiaowen Tong, Rujun Chen, Fang Li
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R. Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Pathogen-specific CD4+ T cells undergo dynamic expansion and contraction during infection, ultimately generating memory clones that shape the subsequent immune responses. However, the influence of distinct tissue environments on the differentiation and clonal selection of polyclonal T cells remains unclear, primarily because of the technical challenges in tracking these cells in vivo. To address this question, we generated Tracking Recently Activated Cell Kinetics (TRACK) mice, a dual-recombinase fate-mapping system that enables precise spatial and temporal labeling of recently activated CD4+ T cells. Using TRACK mice during influenza infection, we observed organ-specific clonal selection and transcriptional differentiation in the lungs, mediastinal lymph nodes (medLNs), and spleen. T cell receptor (TCR) sequencing revealed that local antigenic landscapes and clonal identity shape repertoire diversity, resulting in a low clonal overlap between tissues during acute infection. During the effector phase, spleen-derived CD4+ T cells preferentially adopted a stem-like migratory phenotype, whereas those activated in the medLNs predominantly differentiated into T follicular helper (Tfh) cells. Memory formation was associated with increased clonal overlap between lung and medLN-derived cells, whereas splenic clones retained a distinct repertoire. Additionally, memory CD4+ T cells displayed converging antigen specificity across tissues over time. These results highlight the tissue-dependent mechanisms driving clonal selection and functional specialization during infection and underscore how memory development facilitates clonal redistribution and functional convergence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20c9478d5aaeec28777b006d856c3b28d0b0ac81" target='_blank'>
              Tissue-specific clonal selection and differentiation of CD4+ T cells during infection
              </a>
            </td>
          <td>
            R. Parsa, Helder Assis, Tiago B.R. de Castro, Gabriella Lima dos Reis, Harald Hartweger, Angelina M Bilate, Daniel Mucida
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cellular interactions are of fundamental importance, orchestrating organismal development, tissue homeostasis and immunity. Recently, powerful methods that use single-cell genomic technologies to dissect physically interacting cells have been developed. However, these approaches are characterized by low cellular throughput, long processing times and high costs and are typically restricted to predefined cell types. Here we introduce Interact-omics, a cytometry-based framework to accurately map cellular landscapes and cellular interactions across all immune cell types at ultra-high resolution and scale. We demonstrate the utility of our approach to study kinetics, mode of action and personalized response prediction of immunotherapies, and organism-wide shifts in cellular composition and cellular interaction dynamics following infection in vivo. Our scalable framework can be applied a posteriori to existing cytometry datasets or incorporated into newly designed cytometry-based studies to map cellular interactions with a broad range of applications from fundamental biology to applied biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3fa333836bfc60bc802f3df4b761ac2db5035e" target='_blank'>
              Ultra-high-scale cytometry-based cellular interaction mapping
              </a>
            </td>
          <td>
            Dominik Vonficht, L. Jopp-Saile, S. Yousefian, Viktoria Flore, Inés Simó Vesperinas, Ruth Teuber, Bogdan Avanesyan, Yanjiang Luo, Caroline Röthemeier, F. Grünschläger, Mirian Fernández-Vaquero, Vincent Fregona, D. Ordoñez-Rueda, L. Schmalbrock, Luca Deininger, Angelo Jovin Yamachui Sitcheu, Zuguang Gu, Maja C. Funk, R. Mikut, Mathias Heikenwälder, Angelika Eggert, A. von Stackelberg, Sebastian Kobold, Jan Krönke, U. Keller, A. Trumpp, Ahmed N. Hegazy, Cornelia Eckert, Daniel Hübschmann, Simon Haas
          </td>
          <td>2025-08-07</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Significance Androgen deprivation therapy (ADT) is a mainstay in prostate cancer (PCa) treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer directions for PCa modeling and pave the way toward therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b171b1d175becacb6af2e7a071be6d1fa5ee752f" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a tumor microenvironment (TME) composed of a dense extracellular matrix, cancer-associated fibroblasts (CAFs), vasculature, neural elements, and immune cell populations. This complex network promotes tumor proliferation, invasion, metastasis, and resistance to immunotherapy and chemotherapy. The microenvironmental characteristics of the various PDAC subtypes are discussed in this review. And we examines the role of cancer cells in the TME, highlighting their ability to manipulate stromal components to serve as collaborators in tumor progression. Furthermore, we explored the formation mechanism of the immunosuppressive microenvironment in PDAC, paying attention on Inflammation and intrinsic genetic alterations, the regulatory effect of metabolic reprogramming, the contribution of CAFs and the role of immune cells in cancer cell metastasis. This review shows the role of soluble molecules and exosomes in facilitating PDAC progression and immune evasion within the microenvironment. In conclusion, we outline the novel therapeutic strategies that focus on the interaction between cancer cells and their microenvironment, with the objective of offering new insights for future precision medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e8295ec7fa4434bd00f48ee0333dca0ffe9c46" target='_blank'>
              Domestication and feedback: bidirectional hijacking in pancreatic ductal adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Yuxi Qiao, Haobo Yin, Yong Zhang, Jingdong Zhang, Qian Dong
          </td>
          <td>2025-08-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor ( HGF ) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina Calderon, Ian M. Loveless, Oliver G. McDonald, Jennifer P Morton, David Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of tumor cells, stromal cells, infiltrating immune cells and non-cellular components such as extracellular matrix, blood vessels and a wide variety of secreted proteins. Evidence shows that beyond supporting tumor growth, the TME also promotes tumor cell proliferation and invasion and contributes to treatment resistance, ultimately affecting patient prognosis. Cell-to-cell communication within the TME is driven by secreted proteins such as cytokines, chemokines, growth factors and interferons, which are produced not only by tumor cells but also by various stromal cells and immune cells. These proteins form a complex signaling network that promotes tumor cell proliferation and invasion and enables tumors to evade innate and adaptive immune responses. Antibody arrays are a technology that can simultaneously screen hundreds of secreted proteins in complex biological samples, aiding in the exploration of this complex signaling network. By combining high-throughput multiplex immunoassays such as antibody arrays with cellular and molecular biology techniques, researchers have uncovered complex regulatory mechanisms of cytokine networks within the TME. The present review summarized recent findings on the communication between tumor cells and the TME, as well as key secreted proteins essential for tumor progression and the development of therapeutic resistance. In addition, it discusses how high-throughput antibody arrays contribute to our understanding of regulatory networks of secreted proteins in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84b1200ba5d18e4e2abf23d107f5f3c2ed9e0f6" target='_blank'>
              Unraveling tumor cell-tumor microenvironment crosstalk through antibody array technologies (Review)
              </a>
            </td>
          <td>
            Yanlin Wang, Shuhong Luo, Hua Dong, Ruo-Pan Huang
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-09</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Trachea squamous cell carcinoma (TSCC) is a subtype of lung cancer. A thorough investigation of the tumor microenvironment of TSCC is crucial for the development of cancer therapeutics and predicting clinical responses. In this study, we utilized single-cell RNA sequencing to analyze seven TSCC samples (including five malignant and two non-malignant samples) and obtained 70,682 high-quality cells. Based on the expression levels of marker genes, we identified 7 major cell types within the samples. By comparing malignant samples that received chemotherapy with those that did not, we identified critical transcriptional regulators responsible for T cell state transition in response to chemotherapy. Additionally, we found specific transcriptional regulators and differentially expressed genes between malignant and non-malignant groups. We identified more particularly abundant specifical intercellular communication in the malignant sample group and that may significantly influence the progression and spread of cancerous cells. Overall, our study provides the first single-cell atlas that comprehensively explains TSCC development and chemotherapy effects, thereby laying a new molecular foundation for therapeutic research in TSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4d5fafc26e824adc29e95477c2a4bfd6ebf074" target='_blank'>
              Comprehensive single-cell sequencing reveals the tumor microenvironment and tumor-specific characteristics in trachea squamous cell carcinoma
              </a>
            </td>
          <td>
            HongWu Wang, Hongli Li, Heng Zou, Han Meng, Yan Liu, Chengjun Ban, Weixia Yu, Miao Cheng, Jun Teng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Up until present day, chimeric antigen receptor (CAR)-T cell therapy has only been approved for hematological malignancies, as CAR-T cells do not show comparable efficacy in solid tumors. Therefore, understanding the features of the tumor microenvironment (TME), is key to improve efficacy of adoptive cell therapies (ACTs) against solid tumors. In this context, robust workflows, which dissect the complex interactions between CAR-T cells and the TME are still lacking. To address this need, we have established an ex vivo workflow co-culturing tissue slices from patient tumor resections with CAR-T cells. The workflow is composed of assessing several complementary attributes, such as cytokine release via flow cytometry, quantification of cell infiltration into the tumor and assessment of the regions of the tissue slice the CAR-T cell infiltrate into by using the MACSima™ imaging cyclic staining technology. Using this workflow it is possible to observe the behavior of CAR-T cells within the tumor and its TME, their infiltration into distinct tumor compartments, as well as to dissect the underlying molecular mechanisms that drive T cell migration, thanks to MACSima™ multiplexing technology and its ability to image several markers at the same time. Assessment of ovarian carcinoma tissue slices revealed substantial release of specific cytokines and increased infiltration of T cells in the tumor areas when CAR-T cells were added to the tissue slices as compared to non-engineered T cells. The establishment of this novel approach will enable researchers to better characterize the interaction between CAR-T cells and the TME. Tissue slices present an intrinsic heterogeneity, which is indeed an advantage compared to other in vitro models but can turn itself into complex results interpretation. Therefore, we recommend that any conclusion derived from this assay should be verified with complementary models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80f902b452985b81dfcba4e7339eba4450d5adf" target='_blank'>
              Precision-cut tumor tissue slices, a novel tool to study the tumor microenvironment interactions with chimeric antigen receptor (CAR) T cells
              </a>
            </td>
          <td>
            Valeria Durante, Alina Wittwer, Benjamin Theek, Manuel Martinez-Osuna, E. Donnadieu, O. Hardt, Dominik Eckardt, A. Bosio, Sonja Schallenberg, Christoph Herbel
          </td>
          <td>2025-08-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer therapy, yet most patients fail to achieve durable responses. To better understand the tumor microenvironment (TME), we analyze single-cell RNA-seq (~189 K cells) from 36 metastatic melanoma samples, defining 14 cell types, 55 subtypes, and 15 transcriptional hallmarks of malignant cells. Correlations between cell subtype proportions reveal six distinct clusters, with a mature dendritic cell subtype enriched in immunoregulatory molecules (mregDC) linked to naive T and B cells. Importantly, mregDC abundance predicts progression-free survival (PFS) with ICIs and other therapies, especially when combined with the TCF7 + /– CD8 T cell ratio. Analysis of an independent cohort (n = 318) validates mregDC as a predictive biomarker for anti-CTLA-4 plus anti-PD-1 therapies. Further characterization of mregDCs versus conventional dendritic cells (cDC1/cDC2) highlights their unique transcriptional, epigenetic (single-nucleus ATAC-seq data for cDCs from 14 matched samples), and interaction profiles, offering new insights for improving immunotherapy response and guiding future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065ebdb359d600d41a58e73e81b2ef0cb4692c57" target='_blank'>
              Mature and migratory dendritic cells promote immune infiltration and response to anti-PD-1 checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Jie-Ying Yang, Cassia Wang, Doris Fu, Li-Lun Ho, K. Galani, Lee Chen, Jose Gonzalez, Jolene Fu, Amy Y. Huang, Dennie T. Frederick, Liang He, Mukta Asnani, Rahul Tacke, Emily J. Robitschek, Sandeep K Yadav, Wentao Deng, Kelly P. Burke, T. Sharova, Ryan J. Sullivan, Sarah A. Weiss, Kunal Rai, David Liu, Genevieve M. Boland, M. Kellis
          </td>
          <td>2025-09-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Gliomas represent the most frequent primary brain tumors in adults and are marked by considerable molecular and clinical heterogeneity. The 2021 WHO classification of central nervous system tumors identifies three major adult-type diffuse glioma subtypes: astrocytoma IDH-mutant, oligodendroglioma IDHmutant with 1p/19q-codeletion, and glioblastoma IDH-wildtype.1 Despite ongoing advances, treatment resistance and poor prognosis remain major challenges, largely due to intratumoral heterogeneity and the complexity of the tumor microenvironment.2 A promising yet underexplored avenue for patient-centered glioma profiling is the tumor secretome, which includes soluble factors, extracellular vesicles, and metabolites released by both neoplastic and stromal cells.3 Existing studies rely on in vitro models that fail to recapitulate the complexity of tumor-host interactions. To bridge this gap, we established an innovative, standardized ex vivo protocol for secretome isolation directly from glioma tissue. The protocol preserves microenvironmental cues from the operating room to laboratory analysis, ensuring spontaneous secretome release under controlled conditions. Feasibility and performance were assessed in a preliminary cohort of seven adult diffuse glioma patients. Transmission electron microscopy confirmed tissue structural integrity. Proteomic profiling identified 84 cancer-associated proteins within the secretome. Additionally, metabolomic analysis revealed a distinctive signature of small molecules. Notably, glioblastoma IDH-wildtype samples exhibited increased lactate levels, consistent with hypoxia-driven metabolism, and decreased N-acetylaspartate, indicating neuronal loss. Overall, this ex vivo secretome protocol offers a robust and physiologically relevant platform for glioma characterization, with potential applications in biomarker discovery, patient stratification, and the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea6096ea6e8da3b1feda45c80168b298c559e43" target='_blank'>
              P64 | EX VIVO EXTRACTION OF GLIOMA SECRETOME: A NOVEL MICROENVIRONMENT-BASED PROTOCOL FOR METABOLIC AND MOLECULAR PROFILING
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background and Aims The relationship between chronic liver disease and liver cancer remains poorly understood, and treatment options for advanced liver disease remain limited. This study aims to elucidate the dynamic evolution of cellular and molecular alterations from normal liver to diseased liver. Methods Single‐cell RNA sequencing was performed to profile the dynamic cellular variations from normal liver to diseased liver. Bioinformatics analyses were employed to assess tumour heterogeneity and the evolution of tumour microenvironment. Molecular biology validations were performed to investigate the findings. Results Through single‐cell RNA sequencing, we constructed a human liver landscape consisting of more than 130 000 single cells, from normal, cirrhotic tissue, primary liver cancer and paired adjacent tissues. We profiled both inter‐ and intra‐tumour heterogeneity of hepatocellular carcinoma and intrahepatic cholangiocarcinoma. A dual role of the pro‐fibrosis and pro‐carcinogenesis FABP5high macrophage subset was shown to be conserved throughout the progression of liver fibrosis and carcinogenesis; markers, spatial localization, origin, and functional pathways associated with FABP5high macrophages were identified through in vitro and in vivo validation. Metabolic reprogramming of these conserved FABP5high macrophages is implicated in the formation of cirrhotic and cancerous niches. Additionally, we also identified a subset of RGCC+COL4A1+ endothelial cells enriched in HCC tissues, which might experience endothelial‐mesenchymal transition and orchestrate the proliferation and invasion of cancer cells via angiogenesis. Conclusions Our findings provide insights into the evolution from chronic liver disease to liver cancer. These insights will contribute to the development of novel cell subset‐based therapeutics based on immunotherapy and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8b08f4e01c5f59d84102994dc20c75fd98f8a8" target='_blank'>
              A Conserved FABP5high Macrophage Subset Promotes Fibrosis and Carcinogenesis in Advanced Liver Disease
              </a>
            </td>
          <td>
            Wenhao Ge, Lifeng Ma, Yu Huang, W. E., Shujing Lai, Bijun Cui, Jingjing Wang, Yutian Ji, Daiyuan Liu, Song Ye, Yang Kong, Jiarong Zhou, Chengxuan Yu, Yingsheng Wu, Qiang Sun, Dongkai Zhou, Wei Zhu, Yuan Ding, Xiaoping Han, Guoji Guo, Weilin Wang
          </td>
          <td>2025-08-28</td>
          <td>Liver International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The immune system comprises multiple cell lineages and subsets maintained in tissues throughout the lifespan, with unknown effects of tissue and age on immune cell function. Here we comprehensively profiled RNA and surface protein expression of over 1.25 million immune cells from blood and lymphoid and mucosal tissues from 24 organ donors aged 20–75 years. We annotated major lineages (T cells, B cells, innate lymphoid cells and myeloid cells) and corresponding subsets using a multimodal classifier and probabilistic modeling for comparison across tissue sites and age. We identified dominant site-specific effects on immune cell composition and function across lineages; age-associated effects were manifested by site and lineage for macrophages in mucosal sites, B cells in lymphoid organs, and circulating T cells and natural killer cells across blood and tissues. Our results reveal tissue-specific signatures of immune homeostasis throughout the body, from which to define immune pathologies across the human lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae04949cbee52fc4de7a7a9f67a67f4325623fd" target='_blank'>
              Multimodal profiling reveals tissue-directed signatures of human immune cells altered with age
              </a>
            </td>
          <td>
            Steven B. Wells, Daniel B. Rainbow, Michal Mark, P. Szabo, Can Ergen, Daniel P. Caron, A. R. Maceiras, Elior Rahmani, Eli Benuck, Valeh Valiollah Pour Amiri, David Chen, Allon Wagner, Sarah K Howlett, L. Jarvis, Karen L. Ellis, Masaru Kubota, Rei Matsumoto, Krishnaa Mahbubani, Kouresh Saeb-Parsy, Cecilia Dominguez Conde, Laura Richardson, Chuan Xu, Shuang Li, L. Mamanova, L. Bolt, Alicja Wilk, Sarah A. Teichmann, Donna L. Farber, Peter A. Sims, Joanne L. Jones, N. Yosef
          </td>
          <td>2025-08-13</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ebb88f9db19ddf675e686f40d924b6b992289a8" target='_blank'>
              Clonal hematopoiesis of indeterminate potential influences breast cancer outcomes in a genotype-specific manner.
              </a>
            </td>
          <td>
            S. C. Reed, Chad R Potts, Leo Luo, Brad A Davidson, Riley E. Bergman, Justin D J Kemp, Ethan K Fox, Bryce A Thomas, Lauren Ha, Vani Arora, Justin Cartailler, M. E. Sanders, Violeta Sanchez, Paula I. Gonzalez-Ericsson, S. Croessmann, Paula J Hurley, Alexander G. Bick, P. Ferrell, B. H. Park
          </td>
          <td>2025-09-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Background Mucosa-associated invariant T (MAIT) cells represent a unique population of innate-like T lymphocytes capable of detecting non-peptide antigens in the context of monomorphic antigen-presenting molecules. Due to their abundance in barrier tissues, reactivity to local inflammatory cues, and cytotoxic and regulatory functions, MAIT cells are poised to shape the dynamics of various tumor microenvironments. Growing evidence suggests that MAIT cells can exert protumor and/or antitumor effects in cancers arising from or metastasizing to mucosal tissues. However, MAIT cell roles in bladder cancer (BCa) remains unclear. Methods To begin to identify MAIT cells in BCa, we stained bladder tumor biopsies for T cell receptor (TCR) Vα7.2+ cells. We then refined a human MAIT cell signature, which enabled us to interrogate a bulk RNA sequencing dataset and conduct correlation analyses linking intratumoral MAIT cell abundance and mortality from BCa. To extend our work to an in vivo setting, we employed a clinically relevant mouse model in which Mr1+/+ B6-MAITCAST (MAIT-sufficient) and Mr1−/− B6-MAITCAST (MAIT-deficient) mice were exposed to N-butyl-N-(4-hydroxybutyl)nitrosamine, a chemical carcinogen associated with tobacco smoke. In additional experiments, MAIT cells were functionally removed through acetyl-6-formylpterin (Ac-6-FP) administration. Effector and regulatory cell types were phenotyped by flow cytometry, and BCa tumor burden and progression were assessed by MRI and/or H&E and Ki67 staining. Results TCR Vα7.2+ cells were readily detectable in several BCa biopsies, and our bioinformatic analyses correlated heavier MAIT cell presence in BCa tumors with poorer overall survival. Similarly, we found higher tumor burdens in Mr1+/+ B6-MAITCAST mice than in Mr1−/− or Ac-6-FP-treated animals. Bladder MAIT cells from tumor-bearing mice exhibited phenotypic MAIT17 bias based on transcription factors they harbored along with increased interleukin-17A and tumor necrosis factor-α production capacities upon stimulation. Finally, FoxP3+ regulatory T (Treg) cell frequencies were elevated in Mr1+/+ mouse bladder tumors, likely contributing to an immunosuppressive tumor microenvironment, a finding that could be recapitulated in our transcriptomic studies on human BCa. Conclusions MAIT cells are abundant in BCa tumor microenvironments where they potentiate Treg cell accumulation and play protumor roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904ce037c95ea168f087ef993f969eaddff0121a" target='_blank'>
              MAIT cells promote cancer progression and regulatory T cell accumulation in bladder tumor microenvironment
              </a>
            </td>
          <td>
            Inbar A Habaz, Xuejin Ou, Nicole I Wang, Angela Schincaglia, Valeryia Shydlouskaya, Agetha Mahendran, F. Martinez-Santiesteban, Dylan Doolabi, Yige Bao, Ahmad R Movasseghi, Julius Haruna, Nicholas E. Power, Timothy J Scholl, Melissa J Huynh, Alp Sener, S. M. Haeryfar, Dr S M Mansour Haeryfar
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG.
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Recent studies have explored the composition of tumor microenvironment (TME) in diffuse large B-cell lymphoma (DLBCL) However, cell-to-cell interactions, along with the spatial organization of DLBCL TME and their impact on patient outcomes, have remained poorly characterized. We ap-plied multiplexed immunofluorescence, cell phenotyping, and neighborhood analysis to investigate 1,218,756 single cells in 99 samples from patients with primary DLBCL. We identified 17 cell pheno-types and 10 recurrent cellular neighborhoods (RCNs) across samples, subdividing DLBCLs into immune-poor areas and areas with diverse immune cell infiltrates. Avoidance of B cells and PD-1+ T cells was associated with less aggressive clinical characteristics and favorable survival. Like-wise, the proximity of CD8+ T cell-rich and immune-poor RCNs translated to favorable, and the proximity of PD-L1+ B cell-rich and CD8+ T cell-rich RCNs to unfavorable patient outcomes. Our findings provide insights into the spatial interactions and organization of DLBCL TME with implica-tions for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8a36543383d1dc42cdb78f6b491ab4c213fbf1a" target='_blank'>
              Spatial Analysis of the Tumor Microenvironment in Diffuse Large B-Cell Lymphoma Reveals Clinically Relevant Cell Interactions and Recurrent Cellular Neighborhoods.
              </a>
            </td>
          <td>
            M. Autio, S. Leivonen, L. Meriranta, M. Karjalainen-Lindsberg, T. Pellinen, S. Leppä
          </td>
          <td>2025-08-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and heterogeneous brain tumor. Glioma stem-like cells (GSCs) play a central role in tumor progression, therapeutic resistance, and recurrence. Although immune cells are known to shape the GBM microenvironment, the impact of antigen-presenting-cell (APC) signature genes on tumor-intrinsic phenotypes remains underexplored. We analyzed both bulk- and single-cell RNA sequencing datasets of GBM to investigate the association between APC gene expression and tumor-cell states, including stemness and metabolic reprogramming. Signature scores were computed using curated gene sets related to APC activity, KEGG metabolic pathways, and cancer hallmark pathways. Protein–protein interaction (PPI) networks were constructed to examine the links between immune regulators and metabolic programs. The high expression of APC-related genes, such as HLA-DRA, CD74, CD80, CD86, and CIITA, was associated with lower stemness signatures and enhanced inflammatory signaling. These APC-high states (mean difference = –0.43, adjusted p < 0.001) also showed a shift in metabolic activity, with decreased oxidative phosphorylation and increased lipid and steroid metabolism. This pattern suggests coordinated changes in immune activity and metabolic status. Furthermore, TNF-α and other inflammatory markers were more highly expressed in the less stem-like tumor cells, indicating a possible role of inflammation in promoting differentiation. Our findings revealed that elevated APC gene signatures are associated with more differentiated and metabolically specialized GBM cell states. These transcriptional features may also reflect greater immunogenicity and inflammation sensitivity. The APC metabolic signature may serve as a useful biomarker to identify GBM subpopulations with reduced stemness and increased immune engagement, offering potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf0983476db2efab2251dc274b40344cf2a61c3" target='_blank'>
              Elevated Antigen-Presenting-Cell Signature Genes Predict Stemness and Metabolic Reprogramming States in Glioblastoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor heterogeneity encompasses genetic, epigenetic, and phenotypic diversity, impacting treatment response and resistance. Spatial heterogeneity occurs both inter- and intra-lesionally, while temporal heterogeneity results from clonal evolution. High-throughput technologies like next-generation sequencing (NGS) enhance tumor characterization, but conventional biopsies still do not adequately capture genetic heterogeneity. Liquid biopsy (LBx), analyzing circulating tumor DNA (ctDNA), provides a minimally invasive alternative, offering real-time tumor evolution insights and identifying resistance mutations overlooked by tissue biopsies. This study evaluates the capability of LBx to capture tumor heterogeneity by comparing genetic profiles from multiple metastatic lesions and LBx samples. Eight patients from the Augsburger Longitudinal Plasma Study with various types of cancer provided 56 postmortem tissue samples, which were compared against pre-mortem LBx-derived circulating-free DNA sequenced by NGS. Tissue analyses revealed significant mutational diversity (4–12 mutations per patient, VAFs: 1.5–71.4%), with distinct intra- and inter-lesional heterogeneity. LBx identified 51 variants (4–17 per patient, VAFs: 0.2–31.1%), which overlapped with mutations from the tissue samples by 33–92%. Notably, 22 tissue variants were absent in LBx, whereas 18 LBx-exclusive variants were detected (VAFs: 0.2–2.8%). LBx effectively captures tumor heterogeneity, but should be used in conjunction with tissue biopsies for comprehensive genetic profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99ae66d468d6885d0bee2286ab25cb4f8d1e150" target='_blank'>
              Dissecting Tumor Heterogeneity by Liquid Biopsy—A Comparative Analysis of Post-Mortem Tissue and Pre-Mortem Liquid Biopsies in Solid Neoplasias
              </a>
            </td>
          <td>
            Tatiana Mögele, Kathrin Hildebrand, Aziz Sultan, Sebastian Sommer, Lukas Rentschler, Maria Kling, Irmengard Sax, Matthias Schlesner, B. Märkl, M. Trepel, Maximilian Schmutz, Rainer Claus
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Thyroid cancer (TC) is now among the fastest-growing solid tumours, yet therapeutic gains remain limited for poorly differentiated, anaplastic and medullary variants whose median survivals are measured in months. Once guided chiefly by histology and single-gene assays, immunotherapy is being reshaped by single-cell profiling, which exposes the cellular mosaics that arbitrate response and resistance. Dissection of more than 150–000 tumour- and immune-cell transcriptomes has uncovered follicular-like, partial EMT-like and dedifferentiated thyrocyte states embedded within ‘hot’ (CD8hi IFN-γhi), ‘cold’ (CD8lo) and ‘excluded’ (stroma-walled) immune niches; these phenotypes correlate with PD-1/LAG-3 expression, macrophage polarisation and radio-iodine refractoriness. Functional studies reveal that SPP1–CD44 and GAS6–AXL crosstalk licenses epithelial–mesenchymal transition while VSIG4+ macrophages blunt cytotoxic T-cell activity, collectively undermining checkpoint blockade. Spatial transcriptomics corroborates these insights, mapping PD-L1-high tumour islets millimetres from CXCL13-rich tertiary lymphoid structures, whereas CITE-seq quantifies actionable checkpoints and cytokine receptors across patient biopsies. Emerging therapeutics mirror this granular knowledge: combinatorial PD-1 + LAG-3 inhibition, CSF-1R-directed macrophage re-programming and TSH-receptor-targeted CAR-T cells are advancing through early-phase trials, while ex-vivo single-cell pharmacotyping aligns drug cocktails with an individual’s tumour ecosystem. Early lenvatinib-pembrolizumab or selpercatinib-nivolumab trials show ~40% ORR but grade-3 hypertension >60%, prompting staggered-start designs. These advances sharpen pathogenetic resolution, refine patient selection and accelerate translational pipeline design. By integrating single-cell biology, immunology and endocrine oncology, this review identifies diagnostic blind spots, spotlights drug-repurposing opportunities and charts a roadmap toward personalised immunotherapeutic strategies capable of improving outcomes across the diverse spectrum of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6d6beff9a786418bc13757a09189c54eb993ef" target='_blank'>
              Personalised immunotherapy strategies informed by single cell profiling in thyroid cancer: a mini review
              </a>
            </td>
          <td>
            Ruyu Chen, Zhimin Wang, Xueying Chen, Yangling Huang, Fengnuan Zhang, Tianshu Gao
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that MDK expression increases with tumor grade in IDHwildtype gliomas, accompanied by a shift in isoform proportions favoring the canonical MDK transcript. High MDK expression was associated with poor prognosis specifically in GBM, where the MDKhigh subgroup comprised 75% of cases. MDKhigh GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16253-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5822ef3a00105861887610dcb7da2c42fc7f87" target='_blank'>
              Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment
              </a>
            </td>
          <td>
            Mieszko Lachota, K. Zielniok, Agata Gozdz, Patrycja Szpak, Ilona Kalaszczyńska, Radosław Zagożdżon
          </td>
          <td>2025-08-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Tumor‐associated macrophages (TAMs) are prominent constituents of solid tumors, and their prevalence is often associated with poor clinical outcomes. These highly adaptable immune cells undergo dynamic functional changes within the immunosuppressive tumor microenvironment (TME), engaging in reciprocal interactions with malignant cells. This bidirectional communication facilitates concurrent phenotypic transformation: tumor cells shift toward invasive mesenchymal states, whereas TAMs develop immunosuppressive, pro‐tumorigenic traits. Increasing evidence highlights metabolic reprogramming, characterized by dysregulation of lipid metabolism, amino acid utilization, and glycolytic activity, as the fundamental molecular basis orchestrating this pathological symbiosis. However, a comprehensive understanding of how metabolic reprogramming specifically coordinates the mutual polarization of tumor cells and TAMs is lacking. This review thoroughly examines the molecular mechanisms governing this co‐polarization process, detailing critical transcriptional regulators, essential signaling pathways, and the maintenance of adaptive phenotypes within the TME. Furthermore, this review critically assesses promising therapeutic strategies aimed at disrupting this alliance, including the use of metabolically targeted agents, engineered chimeric antigen receptor macrophages, and TAM‐selective nanoparticle delivery systems. These insights provide a crucial foundation for the development of next‐generation cancer immunotherapies focused on reprogramming pathological polarization dynamics to overcome treatment resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7946cf3691363a365260b427222e670775d573c5" target='_blank'>
              Polarization of Tumor Cells and Tumor‐Associated Macrophages: Molecular Mechanisms and Therapeutic Targets
              </a>
            </td>
          <td>
            Guohao Wei, Bin Li, Mengyang Huang, Mengyao Lv, Zihui Liang, Chuandong Zhu, Lilin Ge, Jing Chen
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, Genevieve L. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba0ab80686667b3dcec48b4fa56304b52cf36e" target='_blank'>
              Reproducible single-cell annotation of programs underlying T cell subsets, activation states and functions
              </a>
            </td>
          <td>
            Dylan Kotliar, Michelle Curtis, Ryan Agnew, Kathryn Weinand, A. Nathan, Y. Baglaenko, Kamil Slowikowski, Yu Zhao, P. Sabeti, Deepak A. Rao, Soumya Raychaudhuri
          </td>
          <td>2025-09-03</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has emerged as a powerful tool to map gene expression patterns to the local tissue structure in cancer, enabling unprecedented insights into cellular heterogeneity and tumour microenvironments. As the technology matures, developing new, spatially informed analytical frameworks will be essential to fully leverage its potential to elucidate the complex organisation and emerging properties of cancer tissues. Here, we highlight key challenges in cancer spatial transcriptomics, focusing on three emerging topics: (a) defining cell states, (b) delineating cellular niches and (c) integrating spatial data with other modalities that can pave the way towards clinical translation. We discuss multiple analytical approaches that are currently implemented or could be adapted in the future in order to tackle these challenges, including classical biostatistics methods as well as methods inherited from geospatial analytics or artificial intelligence. In the rapidly expanding landscape of ST, such methodologies lay the foundation for biological discoveries that conceptualise cancer as an evolving system of interconnected niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9031fde1ffd2ce70d2c223e1b5d79202f9418532" target='_blank'>
              Decrypting cancer's spatial code: from single cells to tissue niches.
              </a>
            </td>
          <td>
            Cenk Celik, Shi Pan, Eloise Withnell, Hou Wang Lam, M. Secrier
          </td>
          <td>2025-07-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A. Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dendritic cells (DCs) are pivotal orchestrators of anti-tumor immunity. DC-based anti-tumor treatments are being actively developed, but effective clinical responses have not yet been achieved. Further exploration of DC heterogeneity in the tumor microenvironment (TME) and across cancer types could provide insights for developing DC-based immunotherapies. Here, we integrated single-cell RNA sequencing data of DCs from over 2,500 samples across 33 cancer types and established a comprehensive blueprint of human DCs. Several rare subsets of DCs infiltrated the tumors, including AXL+SIGLEC6+ (AS) DCs and Langerhans cell (LC)-like DCs, and displayed functional potentials marked with distinct transcriptomic characteristics. Computational analyses demonstrated that the LC-like subset could be an additional cellular origin of tumor-enriched LAMP3+ DCs and that distinct cellular origins are associated with the pleiotropic functional potentials of LAMP3+ DCs. Furthermore, this DC atlas enabled development of a machine learning model to guide DC annotation for subsequent single-cell analysis and prioritization of a valuable target for enhancing anti-tumor DC vaccination. This integrative resource provides a panoramic view to unravel the complexity of tumor-infiltrating DCs and offers valuable insights for developing therapies targeting DCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20198842496886b53a9ff7adf340fb98f1bc673" target='_blank'>
              Pan-cancer Analyses Refine the Single-Cell Portrait of Tumor-Infiltrating Dendritic Cells.
              </a>
            </td>
          <td>
            Tianyi Ma, Xiaojing Chu, Jinyu Wang, Xiangjie Li, Yu Zhang, Dan Tong, Wenbin Xu, Guohui Dang, Lu Qi, Yuhui Miao, Zemin Zhang, Si Cheng
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Fibroblasts are critical mediators of tumor progression and metastasis; however, their heterogeneity and specific functions in the context of colorectal cancer (CRC) liver metastases remain incompletely understood.We performed single-cell RNA sequencing (scRNA-seq) analysis on relevant tissue samples. Subsequently, we validated our findings using immunofluorescence assays on histological slides from primary CRC with liver metastasis (mCC) and liver metastatic tumors (mLC). Further investigations included transcriptomic profiling, pseudotime trajectory analysis, multiplex immunofluorescence staining, and cell-cell communication analysis.Our scRNA-seq analysis identified a distinct PRELP-positive cancer-associated fibroblast (CAF) subtype that is associated with liver metastasis and tumor progression. These PRELP+ CAFs were predominantly enriched in mLC, less abundant in mCC, and rare in non-metastatic CRC (nCC). This distribution was confirmed by immunofluorescence. Transcriptomically, PRELP+ CAFs exhibit a unique signature defined by extracellular matrix components (e.g., PRELP, COLEC11, ITGBL1) and the activation of pro-tumor pathways such as TGF-β and Wnt signaling. Pseudotime analysis indicated they represent a terminal fibroblast differentiation state. Spatially, they colocalize with immune cells (T cells, B cells, plasma cells), and communication analysis suggests they foster an immunosuppressive microenvironment via APP-CD74 and collagen-CD44 signaling, thereby promoting immune evasion. The transcription factors NR2F2, JUN, and JUND were identified as key regulators of this CAF subtype.These findings provide crucial new insights into fibroblast heterogeneity within CRC liver metastases. We characterize PRELP+ CAFs as a specialized, terminally differentiated fibroblast population that contributes to immunosuppression and tumor progression, highlighting them as a potential therapeutic target for inhibiting metastatic advancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62bf8481f437e703e061482f9031f26e35dec625" target='_blank'>
              Characterization of the phenotype and function of PRELP+ fibroblast subtype in liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Yuting Dai, Xingying Huang, Min Sun, Shiyu Zhang, Weiqiang Yu, Kongwang Hu, Qiang Wu, Qingfa Wu
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, Felix Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, Nada Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Atypical teratoid rhabdoid tumors (ATRTs) are highly aggressive pediatric central nervous system tumors defined by the inactivation of the SMARCB1 gene. Despite the identification of three distinct molecular subtypes, each defined by unique clinical and molecular characteristics, no subtype-specific therapeutic strategies are currently available. This highlights an urgent need to deepen our understanding of the cellular heterogeneity and developmental origins of ATRTs.


METHODS
We generated a comprehensive single-nucleus transcriptomic atlas of ATRT samples, integrated it with single-nucleus ATAC-seq and spatial transcriptomics data, and validated our findings experimentally using patient-derived ATRT tumoroid models.


RESULTS
Our analyses revealed distinct subtype-specific differentiation trajectories, each resembling different brain progenitor lineages. We identified key transcription factors that appear to drive these developmental pathways. Furthermore, a shared cycling, intermediate precursor cell (IPC)-like cell population, interspersed throughout tumors, was consistently present within all ATRT samples. We demonstrate that these subtype-specific differentiation pathways can be pharmacologically manipulated in patient-derived ATRT tumoroids. By directing tumor cells along their respective subtype-specific trajectories, we were able to induce a shift toward more differentiated, non-proliferative states.


CONCLUSIONS
Collectively, our findings show that ATRTs recapitulate fetal brain signaling programs in a subtype-specific manner. This work provides a framework for understanding ATRT heterogeneity and supports the feasibility of maturation-based therapeutic strategies tailored to the molecular subtype of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f089b7cc6f0aae306ce7c5e4e459d6bc20cd73eb" target='_blank'>
              A cycling, progenitor-like cell population at the base of atypical teratoid rhabdoid tumor subtype differentiation trajectories.
              </a>
            </td>
          <td>
            Enrique Blanco-Carmona, Irene Paassen, Jiayou He, Jeff DeMartino, Annette Büllesbach, Nadia Anderson, Juliane L Buhl, A. Federico, Monika Mauermann, Mariël Brok, Karin Straathof, S. Behjati, Rajeev Vibhakar, A. Donson, Nicholas K Foreman, McKenzie L. Shaw, M. Frühwald, A. Korshunov, M. Hasselblatt, C. Thomas, Niels Franke, M. Kranendonk, E. Hoving, N. Jäger, P. Johann, Stefan M Pfister, Mariella G. Filbin, M. Kool, J. Drost
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The 3D organization of the genome plays a critical role in regulating gene expression, maintaining cellular identity, and mediating responses to environmental cues. Advances in super-resolution microscopy and genomic technologies have enabled unprecedented insights into chromatin architecture at nanoscale resolution. However, the complexity and volume of data generated by these techniques necessitate innovative computational strategies for effective analysis and interpretation. In this review, we explore the transformative role of deep learning in the analysis of 3D genome organization, highlighting how deep learning models are being leveraged to enhance image reconstruction, segmentation, and dynamic tracking in chromatin research. We provide an overview of deep learning-enhanced methodologies that significantly improve spatial and temporal resolution of images, with a special focus on single-molecule localization microscopy. Furthermore, we discuss deep learning’s contribution to segmentation accuracy, and its application in single-particle tracking for dissecting chromatin dynamics at the single-cell level. These advances are complemented by frameworks that enable multimodal integration and interpretability, pushing the boundaries of chromatin biology into clinical diagnostics and personalized medicine. Finally, we discuss emerging clinical applications where deep learning models, based on chromatin imaging, aid in disease stratification, drug response prediction, and early cancer detection. We also address the challenges of data sparsity, model interpretability and propose future directions to decode genome function with higher precision and impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33be16632331d98ff0665f21e81c0af12092a8a7" target='_blank'>
              Deep learning in chromatin organization: from super-resolution microscopy to clinical applications
              </a>
            </td>
          <td>
            Mikhail Rotkevich, Carlotta Viana, M. V. Neguembor, M. Cosma
          </td>
          <td>2025-08-29</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c0faec4d04ddee86bb924ee79c08f0dc4ced64" target='_blank'>
              Cancer-associated fibroblasts drive lung adenocarcinoma progression via THBS2-mediated epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            Yuqing Ren, Ruijie Ming, Anning Zuo, Shutong Liu, Yuhao Ba, Yuyuan Zhang, Yukang Chen, Teng Pan, Peng Luo, Quan Cheng, Jinhai Deng, Yi Yue, Hui-Xian Xu, Siyuan Weng, Xinwei Han, Dongdong Zhou, Zaoqu Liu
          </td>
          <td>2025-09-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed22b8c9ee3077b8f3ac11ad2aba9402eaa560a" target='_blank'>
              From 2D to 3D and beyond: the evolution and impact of in vitro tumor models in cancer research.
              </a>
            </td>
          <td>
            G. Rauner, Piyush B. Gupta, Charlotte Kuperwasser
          </td>
          <td>2025-07-25</td>
          <td>Nature methods</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC), a prevalent and highly lethal malignancy, is notorious for its aggressive nature and inherent tendency to metastasize, posing significant challenges in clinical management and prognosis. Hypoxia, a pivotal characteristic of the tumor microenvironment (TME) in hepatocellular HCC, is intimately linked to disease progression and unfavorable patient outcomes, underscoring its critical role in shaping the malignant behavior of this cancer. Methods Our research leveraged single-cell RNA sequencing technology to dissect the heterogeneity of the HCC TME, focusing on hypoxia-related genes. To probe the effects of hypoxia on HCC invasion, we developed the Cell Hypoxia-Related Prognostic Feature (CHPF). We analyzed transcriptome data from the The Cancer Genome Atlas (TCGA) database and the GSE149614 dataset, employing computational methods such as UMAP, Weighted correlation network analysis (WGCNA), and CellChat to identify hypoxia cells, characterize cell subsets, and elucidate intercellular communications. Results Our analysis revealed significant heterogeneity in hypoxia cell populations within the HCC TME, with distinct expression patterns of hypoxia-related genes in neoplastic and immune cells. Our analysis revealed distinct hypoxia subpopulations within HCC, with significant overexpression of genes like MEG3, KLF6 and JUN in hypoxia cells. We identified a unique hypoxia subpopulation with high invasive potential and constructed a prognostic model based on H2-specific transcription factors including LRP10、MED8、NOL10、NOP58 and REXO4. The model demonstrated significant predictive value for lifespan of patients as verified in the TCGA dataset and an external validation group. Conclusion Key transcription factors like NOP58, MED8 play pivotal roles in hypoxia-induced HCC invasion and metastasis, and a predictive model based on these factors forecasts HCC survival. Our findings provide novel molecular markers and therapeutic targets for HCC, highlighting the importance of considering the hypoxia TME in diagnostic and treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03201-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85919f351fa84b022b25ebecfed4ffc0c988bac6" target='_blank'>
              Deciphering hypoxia’s role in hepatocellular carcinoma prognosis with single-cell approaches
              </a>
            </td>
          <td>
            Jun Liang, Di Chen, Huiyu Liang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The advent of single-cell RNA sequencing (scRNA-seq) has provided unprecedented insights into cancer cellular diversity, enabling a comprehensive understanding of cancer at the single-cell level. However, identifying cancer cells remains challenging due to gene expression variability caused by tumor or tissue heterogeneity, which negatively impacts generalization and robustness. Results We propose CanCellCap, a multi-domain learning framework, to identify cancer cells in scRNA-seq data suitable for all tissues, cancers, and sequencing platforms. Integrating domain adversarial learning and Mixture of Experts, CanCellCap is able to simultaneously extract common and specific patterns in gene expression profiles across different tissues for cancer or normal cells. Moreover, the masking-reconstruction strategy enables CanCellCap to cope with scRNA-seq data from different sequencing platforms. CanCellCap achieves 0.977 average accuracy in cancer cell identification across 13 tissue types, 23 cancer types, and 7 sequencing platforms. It outperforms five state-of-the-art methods on 33 benchmark datasets. Notably, CanCellCap maintains high performance on unseen cancer types, tissue types, and even across species, highlighting its effectiveness in challenging scenarios. It also excels in spatial transcriptomics by accurately identifying cancer spots. Furthermore, CanCellCap demonstrates strong computational efficiency, completing inference on 100,000 cells in a few minutes. In addition, interpretability analyses reveal critical biomarkers and pathways, offering valuable biological insights. Conclusions CanCellCap provides a robust and accurate framework for identifying cancer cells across diverse platforms, tissue types, and cancer types. Its strong generalization to unseen cancers, tissues, and even species, combined with its adaptability to spatial transcriptomics data, underscores its versatility for both research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02337-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46771c12c055de1df0eb19772deecbb02fd302c4" target='_blank'>
              CanCellCap: robust cancer cell capture across tissue types on single-cell RNA-seq data by multi-domain learning
              </a>
            </td>
          <td>
            Jiaxing Bai, Yichun Gao, Feng Zhou, Yushuang He, Chen Lin, Xiaobing Huang, Ying Wang
          </td>
          <td>2025-07-30</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo.
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, Hui Yi Grace Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Advances in treating primary breast tumors and extracranial metastases have inadvertently led to a rise in brain metastases, which are severe and carry bleak prognostic outcomes. Thus, there is an urgent need to investigate the cellular and molecular drivers of metastatic central nervous system (CNS) tumors. The tumor microenvironment (TME) is critically important for metastasis, as metastatic cells must alter their new environment to survive and grow. Our laboratory's recent studies have pinpointed platelet-derived growth factor receptor beta (PDGFRβ) signaling as a critical pathway in promoting breast cancer-associated brain metastases (BCBM). Key findings from this published work involved data emanating from a genetically engineered mouse model (GEMM) with hyperactive PDGFRβ-D849V in the mesenchymal lineage driven by Fsp1-cre. These mice exhibit increased BCBM when injected with murine mammary tumor cells secreting the ligand PDGFB, making them an effective model for studying BCBM. However, hyperactivity of PDGFRβ in these mice occurs in all mesenchymal cells. To investigate cell-specific PDGFRβ functions in the CNS, we analyzed publicly available single-cell RNA-seq data of the adult murine brain. This analysis identified astrocyte populations expressing both Pdgfrb and the Fsp1/S100A4 mesenchymal lineage marker. To probe the role of PDGFRβ in reactive astrocytes, we developed a Gfap-cre GEMM model crossed with our mutant PDGFRβ hyperactivation allele (Gfap-cre;Pdgfrb-D849V). These mice exhibit increased GFAP expression in the absence of tumor and decreased overall survival upon intracardiac injection of PDGFB secreting breast cancer cells. Furthermore, flow cytometry at day 10 post intracardiac injection indicates potential differences in the immune response of the brain TME compared to controls. Current work is focused on identifying specific transcriptomic changes in astrocyte populations due to both PDGFRβ hyperactivation as well as metastatic breast cancer colonization of the brain. Overall, this work aims to identify potential therapeutic targets to treat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065fae0e35727b10f37e95370e754abb9b46417c" target='_blank'>
              BSBM-08 ASTROCYTE PDGFRβ HYPERACTIVATION IN BREAST CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Nathaniel S Grabinski, Johnathon Scheibel, Jesse Reardon, Paul Lee, Jonathan Godbout, S. Sizemore, Jessica Winter, Gina M. Sizemore
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) continues to pose a major challenge in cancer treatment, characterized by low survival rates, particularly in metastatic instances. Although immunotherapy has become a common approach for treating LUAD, its success rate is merely 20%, hindered by the absence of effective biomarkers. This limitation is likely influenced by the tumor's immune microenvironment. CD40LG, an important immune molecule, has emerged as a crucial factor in modulating the tumor immune environment and affecting the outcomes of immunotherapy. Nevertheless, its exact mechanisms in LUAD remain unclear, requiring more research. Methods: Through a correlation analysis of prognosis and clinical traits across multiple cancers, we determined that CD40LG is significant in LUAD. This role was confirmed using various public datasets. We evaluated CD40LG protein expression in 94 LUAD and nearby non-cancerous tissues via immunohistochemistry. To gauge immune cell infiltration, we employed multiplex immunofluorescence staining on tissue microarrays. Additionally, in vitro assays were carried out to explore the effects of CD40LG modulation on the behavior of LUAD cells. Results: Various cancers, including LUAD, exhibited down-regulation of CD40LG, which correlated with a worse prognosis. In LUAD tissues, higher CD40LG expression was associated with longer Progression-Free Survival (PFS) and Overall Survival (OS). Moreover, CD40LG expression negatively correlated with the TNM stage and T stage in LUAD. Elevated CD40LG levels were linked to increased infiltration of CD8+ T cells. In vitro studies showed that modulating CD40LG affected LUAD cell metastasis and proliferation. Conclusion: Our study demonstrates the pivotal role of CD40LG in LUAD, proposing its potential utility as a biomarker for prognosis and immunotherapy. The correlation between CD40LG expression, immune cell infiltration, and clinical outcomes emphasizes its importance in tumor-immune dynamics and the efficacy of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6101ef3e2ce499e361f51adb84fd48dc1ec645f4" target='_blank'>
              CD40LG Downregulation in Lung Adenocarcinoma: A Prognostic Biomarker Linked to Immune Cell Infiltration and Survival Outcomes
              </a>
            </td>
          <td>
            Peng Tang, Ruihao Zhang, Yuqi Zhang, Yuan Ao, Yanan Wang, Guannan Wang, Wenjun Wang, Junmei Xu, Fang Hu, Guangsheng Zhu
          </td>
          <td>2025-08-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d735b44b6ee6328d790d08642f35e3daac2ffe" target='_blank'>
              Integrated in vivo combinatorial functional genomics and spatial transcriptomics of tumours to decode genotype-to-phenotype relationships.
              </a>
            </td>
          <td>
            Marco Breinig, Artem Lomakin, Elyas Heidari, M. Ritter, G. Rukhovich, Lio Böse, Luise Butthof, Lena Wendler-Link, Hendrik Wiethoff, Tanja Poth, Felix Sahm, Peter Schirmacher, Oliver Stegle, M. Gerstung, Darjus F. Tschaharganeh
          </td>
          <td>2025-07-28</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Recent advances in self-supervised foundation models for computational cancer histopathology have shown great promise for predicting clinical phenotypes from routine hematoxylin and eosin (H&E)-stained images. However, application of these models in the context of discovery, for example for identification of novel biomarkers or drug targets, remains challenging as this typically requires linking whole slide imaging (WSI) data to molecular data, which has historically been scarce. Recently, spatial omics (SO) has emerged as a powerful technology for high-throughput profiling of RNAs and proteins directly within cancer tissues. This presents new and exciting opportunities to improve histopathology models for making molecular predictions about the tumor microenvironment. In this work, we develop machine learning models that can generate transcriptomic profiles at two different spatial scales: (1) cellular-level resolution, i.e. SO, and (2) slide-level resolution, i.e. bulk RNA-sequencing. We present a preprocessing and benchmarking framework to enable rigorous comparison of models on an in-house SO dataset, as well as a public bulk RNA-seq dataset (TCGA-COAD/READ), of colorectal cancer. We fine-tune recently published digital pathology foundation models for these tasks, and find that they outperform both general pathology-naïve image foundation models, as well as specialized digital pathology deep learning models. We also find that existing models cannot be simultaneously trained using supervision from multiple spatial resolutions. To address this, we propose a graph neural network (GNN) modeling approach that can model both local interactions between cells within tumor niches, as well as global interactions across tumor regions. We show that our GNN model outperforms existing models for both cell-level and slide-level tasks. This work presents a novel multi-scale framework for predicting transcriptome profiles from WSIs.



 Tianyi Zhang, Malay Singh, Dwivedi Naman, Timothy Antoni, Ju Can, Firas Elbayoumi, Iain Bee Huat Tan, Tiantian He, Hwee Kuan Lee, Shyam Prabhakar, Grace Hui Ting Yeo. Multi-Scale Modeling for Prediction of Spatial Single-Cell and Bulk Transcriptomic Profiles from Whole-Slide Images of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P24.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270a89b81baeb130ef31e873fd453a0b1bfebe9" target='_blank'>
              Abstract P24: Multi-Scale Modeling for Prediction of Spatial Single-Cell and Bulk Transcriptomic Profiles from Whole-Slide Images of Colorectal Cancer
              </a>
            </td>
          <td>
            Tianyi Zhang, Malay Singh, Dwivedi Naman, Timothy Antoni, Ju Can, Firas Elbayoumi, Iain Bee Huat Tan, Tiantian He, Hwee Kuan Lee, Shyam Prabhakar, Grace Hui Ting Yeo
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer in which disease initiation and relapse are driven by leukemic cells with stem-like properties, known as leukemic stem cells (LSCs). The LSC compartment is highly heterogenous and this contributes to differences in therapy response. This heterogeneity is determined by genetic and nongenetic factors including somatic mutations, the cell of origin, transcriptional and epigenetic states as well as phenotypic plasticity. While this complicates the identification and eradication of LSCs, it also presents an opportunity to tailor therapeutic strategies to the phenotypic and functional states of LSCs present in a patient, exploiting their specific vulnerabilities. The emergence of single-cell multiomics technologies has transformed our ability to dissect cellular heterogeneity in AML, enabling simultaneous interrogation of genomic, transcriptomic, epigenomic and proteomic layers and providing high-resolution molecular snapshots of individual cells. In this review, we discuss causes and consequences of LSC heterogeneity, highlight advances in single-cell multiomics technologies to resolve it and outline how they can address shortcomings in our understanding of LSC heterogeneity and plasticity to revolutionize diagnostics and disease monitoring of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07654c74db7467e7588089114a2ee5e54a7d1bf9" target='_blank'>
              Resolving leukemic stem cell heterogeneity and plasticity with single-cell multiomics.
              </a>
            </td>
          <td>
            Frank Y Huang, Andreas Trumpp, Patrick Stelmach
          </td>
          <td>2025-08-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background and purpose The tumor microenvironment (TME) is widely acknowledged as a pivotal regulator of cancer progression. However, the dualistic role of tertiary lymphoid structures (TLSs), which serve as critical immune hubs within the TME, remains incompletely characterized, particularly with respect to their context-dependent capacity to either inhibit or facilitate tumor development. This review aims to synthesize current understanding of the complex interactions between stromal cells and TLSs, addressing existing gaps in mechanistic insight and exploring therapeutic avenues to exploit TLS plasticity. Key reviewed topics The current study critically reviews the mechanisms by which stromal components, including cancer-associated fibroblasts and endothelial cells, contribute to TLS neogenesis through chemokine-mediated recruitment of lymphocytes. Furthermore, it highlights the dual functional roles of TLSs as sites of both anti-tumor immune activation and immunosuppression, notably via the enrichment of regulatory T cells. The clinical implications of mature TLS presence, particularly their association with improved patient prognosis and enhanced therapeutic responsiveness, are also analyzed. Main conclusions TLSs demonstrate a bifunctional nature, wherein their spatial organization and dynamic interactions with stromal elements dictate the balance between immune activation and tolerance within the TME. While mature TLSs are generally correlated with favorable clinical outcomes, their potential to foster immunosuppressive microenvironments necessitates the development of precision-targeted interventions. The interplay between stromal cells and TLSs represents a promising therapeutic axis for modulating the tumor immune milieu. Future perspectives Future research should prioritize strategies aimed at promoting TLS maturation, disrupting immunosuppressive niches, and integrating TLS-modulating agents with existing immunotherapeutic regimens to enhance clinical efficacy. Additionally, the identification of robust biomarkers reflective of TLS functional states and the rigorous validation of stromal-targeted therapies within combinatorial treatment frameworks are imperative for advancing translational applications. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c13fc16e53261c020d877469541bad918b8156" target='_blank'>
              In-depth insight into tumor-infiltrating stromal cells linked to tertiary lymphoid structures and their prospective function in cancer immunotherapy
              </a>
            </td>
          <td>
            Maedeh Radandish, Niloofar Mashhadi, Amir Hossein Aghayan, Motahareh Taghizadeh, Sara Salehianfard, Sheida Yahyazadeh, Omid Vakili, Somayeh Igder
          </td>
          <td>2025-08-10</td>
          <td>Experimental Hematology & Oncology</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="T follicular helper (Tfh) cells are central to the adaptive immune response and exhibit remarkable functional diversity and plasticity. The complex nature of Tfh cell populations, inconsistent findings across experimental systems and potential differences across species have fueled ongoing debate regarding core regulatory pathways that govern Tfh differentiation. Many studies have experimentally investigated individual proteins and circuits involved in Tfh differentiation in limited contexts, each providing only a partial understanding of the process. To address this, we adopted a novel multi-scale network systems approach that incorporates both regulatory and protein-protein interactions. Our approach integrates diverse data types, captures regulation across multiple levels of immune system organization, and recapitulates known drivers. Further, we discover a core Tfh gene set that is conserved across tissue types and disease contexts, and is consistent across data modalities - bulk, single-cell and spatial. While components of this set have been individually reported, a novel aspect of our work lies in the discovery, characterization, and connectivity of this core signature using a single unbiased approach. Using this method, we also uncover a novel function of IL-12, a molecule with reported conflicting functions, in the regulation of Tfh differentiation. Notably, we find that, in both humans and mice, IL-12 is permissive for the differentiation of Tfh precursors, but blocks subsequent differentiation into GC Tfh cells. Overall, this work elucidates novel networks with unexplored roles in governing Tfh cell differentiation across species and tissues, paving the way for novel -therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cdef08e00798c3da4a3f940c2d60360ad82720" target='_blank'>
              A unified network systems approach uncovers a core novel program underlying T follicular helper cell differentiation
              </a>
            </td>
          <td>
            Alisa A. Omelchenko, Syed A. Rahman, Vinayak V. Viswanadham, Grace J. Yuen, Perla M. Del Rio Estrada, Valentino D’Onofrio, Yijia Chen, Na Sun, Hamid Mattoo, Chinmay G. Varma, Gonzalo Salgado, Maribel S. Nava, Lady C. Ruiz, Dafne D. Rivera, S. A. Rios, Sudhir P. Kasturi, Susan P. Ribeiro, M. Shlomchik, Amanda C. Poholek, Shiv S. Pillai, Rebecca A. Elsner, V. Mahajan, Jishnu Das
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is a global health challenge shaped by a complex tumor microenvironment (TME), where myeloid‐driven inflammatory and immunosuppressive processes play central roles. Understanding the spatial and dynamic interactions within the TME is essential for uncovering novel regulatory mechanisms. We integrated spatial transcriptomics, single‐cell RNA‐sequencing, and network pharmacology to develop a comprehensive analytical framework for deconstructing cellular and molecular interactions across spatial domains within the TME. Using this approach, we investigated the potential mechanisms by which the Sparganii Rhizoma‐Curcumae Rhizoma (SCP) Chinese medicine (TCM) herbal pair may influence primary CRC and liver metastases. Our analysis suggests that in primary CRC, SCP may affect neutrophil‐enriched domains through modulation of CEACAM‐related signaling, while in liver metastases, it may influence macrophage‐dominated domains associated with APP signaling. These findings highlight possible domain‐specific molecular interactions and suggest that SCP may regulate intra‐ and inter‐domain signaling patterns relevant to TME remodeling. This study establishes a novel integrative methodology to investigate spatial domain‐specific mechanisms within the TME. Our findings reveal the multi‐dimensional effects of SCP, providing a precision‐targeted, TME‐centric therapeutic strategy for CRC. The framework aligns with TCM principles and offers new avenues for personalized cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64108e2457f88f143ed78e3504c4067d48fd77bb" target='_blank'>
              Spatial Domain‐Based Approach to Analyze the Mechanism of Sparganii Rhizoma‐Curcumae Rhizoma Pair in the Treatment of Colorectal Cancer
              </a>
            </td>
          <td>
            Yuan-Jie Liu, Qian-wen Ye, Yi‐lan Jin, Qian Zhang, Xi Zou, Jie-pin Li
          </td>
          <td>2025-08-01</td>
          <td>Food Science & Nutrition</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Functional cellular heterogeneity in tumours often underlies incomplete response to therapy and relapse. Previously, we demonstrated that the growth of the paediatric brain malignancy, sonic hedgehog subgroup medulloblastoma, is rooted in a dysregulated developmental hierarchy, the apex of which is defined by characteristically quiescent SOX2+ stem-like cells. Integrating gene expression and chromatin accessibility patterns in distinct cellular compartments, we identify the transcription factor Olig2 as regulating the stem cell fate transition from quiescence to activation, driving the generation of downstream neoplastic progenitors. Inactivation of Olig2 blocks stem cell activation and tumour output. Targeting this rare OLIG2-driven proliferative programme with a small molecule inhibitor, CT-179, dramatically attenuates early tumour formation and tumour regrowth post-therapy, and significantly increases median survival in vivo. We demonstrate that targeting transition from quiescence to proliferation at the level of the tumorigenic cell could be a pivotal medulloblastoma treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27908f257548bd97b59f7a3c88e807b37fb0d43b" target='_blank'>
              38 OLIG2 MEDIATES A RARE TARGETABLE STEM CELL FATE TRANSITION IN SONIC HEDGEHOG MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, Heather Whetstone, Chunying Yu, Robert J. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, Benjamin D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Colorectal cancer (CRC) arises in the colorectal tissue driven by genetic disorder or the accumulation of somatic mutations, leading to abnormal epithelial cell growth. In this study, we employed single-nucleus multi-omics analysis, including single-nucleus RNA-seq and single-nucleus ATAC-seq, on over 100,000 high-quality nuclei to investigate the molecular landscape of both primary tissue and patient-derived organoids (PDOs). Our analysis showed that normal PDOs (N-PDOs) derived from tissue adjacent to tumors replicate the cellular composition and differentiation trajectory of colorectal crypts. In contrast, tumor PDOs (T-PDOs) showed patient-specific transcriptomic and epigenomic heterogeneity yet consistently maintained a stem cell-like state. T-PDOs retained the somatic mutation profile of the primary tumor while also exhibiting de novo mutations not detected in either the primary tumor or N-PDOs. Notably, inferred cell-cell interaction analysis highlighted the activin signaling pathway as a potential unique feature of fibroblast-epithelial interactions within the tumor microenvironment. This study provides a comprehensive view of the transition from normal to malignant colorectal epithelium and underscores the utility of PDOs as a faithful model for capturing both conserved and patient-specific features of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044d9b62ee3fa5808d22e4c1cdf93a59f76f395e" target='_blank'>
              Single-cell multi-omics characterize colorectal tumors, adjacent healthy tissue and matched (tumor) organoids identifying CRC-unique features.
              </a>
            </td>
          <td>
            Zhijun Yu, Merel Derksen, Brigit M Te Pas, Sabrina Ladstätter, Rene Overmeer, Peter Brazda, Marc van de Wetering, F. Pourfarzad, Robert G. J. Vries, Wout Megchelenbrink, Christoph Bock, L. Altucci, H. Stunnenberg
          </td>
          <td>2025-08-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Background SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses. Methods Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism. Results SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced. Conclusion SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) have a significant impact on the prognosis and treatment outcomes of Wilms tumor (WT) patients. To explore the key mechanisms underlying WT progression and immune therapy, this study used CIBERSORT to analyze the immune cell infiltration of 120 WT patients. Combined with single-cell RNA sequencing (scRNA-seq) data, the heterogeneity of macrophages in WT and adjacent tissues was revealed, identifying a subpopulation of tissue-resident macrophages with specific expression of BCL2A1. Further validation through immunohistochemistry (IHC) and immunofluorescence (IF) experiments confirmed the presence of BCL2A1⁺tissue-resident macrophages and elevated BCL2A1 expression is associated with advanced tumors and poor prognosis. Functional enrichment analysis suggests that BCL2A1⁺tissue-resident macrophages may promote WT progression through immune regulation and apoptosis pathways. This study is the first to identify the presence of a BCL2A1⁺tissue-resident macrophage subset in WT and reveal its critical role in tumor progression, potentially providing a novel target for personalized immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dca751c6f30ef2c3b2e407c5124f519eaf2a487" target='_blank'>
              The potential role of BCL2A1⁺ tissue-resident macrophages in the prognosis of Wilms tumor
              </a>
            </td>
          <td>
            Wei Wang, Xianwu Yang, Zhihui Zhu, Zhishen Tang, Yingquan Zhuo, Jun Du, Yuxian Zhu, Xi Luo, Jingjing Xiao, Huajian Gu
          </td>
          <td>2025-07-25</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aberrantly elevated Ras signals, triggered by various distinct genetic mutations, are frequent features in myeloid leukemias. Normal hematopoiesis requires perpetual and balanced production of different blood cell lineages by multipotent hematopoietic stem cells (HSC). Stem- and progenitor- cells combine dormancy with proliferative drive and require finely tuned metabolism and protein translation rates. Due to the scarcity of stem cells, it has remained largely unknown how aberrantly elevated Ras signals may impact frequency, lineage potential, and quiescent metabolism in rare HSC. Using single cell proteomics and computational analyses, we characterized the effects of induced oncogenic mutant KRasG12D or overexpression of the Ras activator RasGRP1, compared to normal native hematopoiesis. The two Ras pathway lesions drive shared profound skewing towards and expansion of mature myeloid cells. The resolution of CyTOF unmasked opposing patterns for the HSC- and progenitor- compartments: Overexpression of RasGRP1 induced expansion of both subsets, whereas KRASG12D resulted in depletion. By combining spectral flow with SCENITHÔ, a method to quantitate protein translation as a proxy for metabolic state, we first corroborated that immature cells display low metabolic SCENITHÔ rates. Both RasGRP1 and KRASG12D drive elevated, mean SCENITHÔ signals in immature hematopoietic cells. However, RasGRP1-overexpressing stem cells retain a metabolically quiescent cell-fraction, whereas this fraction is incompatible with KRASG12D. Our temporal proteomics and metabolomics datasets provide mechanistic insights into altered hematopoiesis at single cell resolution and support the idea that the exact identity and duration of signals from Ras lesions has profound impacts on stem cell maintenance and lineage-potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8833491efb56cc3a2592617d3966eb43f088b40f" target='_blank'>
              Single cell proteomics characterization of bone marrow hematopoiesis with distinct Ras pathway lesions.
              </a>
            </td>
          <td>
            Laila Karra, Anna-Marie Finger, Lauren A Shechtman, M. Krush, Rita Meng-Yao Huang, Morgan Prinz, Iliana Tennvooren, Kriti Bahl, Lisiena Hysienaj, Paulina Garcia-Gonzalez, Alexis J. Combes, Hugo Gonzalez, R. Argüello, Mathew Spitzer, Jeroen P. Roose
          </td>
          <td>2025-07-30</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Late recurrence of breast cancer poses a considerable threat to patient survival, often attributed to breast cancer dormancy. Dormancy, characterized by cancer cells remaining quiescent for extended periods, is influenced by genetic factors and modifications that directly impact cellular phenotype. Alterations in gene expression dynamically shape cellular behavior, often mediated through receptor signaling pathways. Moreover, interactions within the tumor microenvironment play a pivotal role, fostering either cancer cell dormancy or promoting their escape from dormancy. This review endeavors to provide a comprehensive synthesis of recent advancements in understanding breast cancer dormancy across genetic, receptor molecular, and cellular dimensions. By elucidating the intricate mechanisms underlying dormancy, we aim to shed light on potential therapeutic strategies to prevent late recurrences. Furthermore, we anticipate future research directions that may uncover novel insights into this complex phenomenon, ultimately improving patient outcomes and refining clinical management strategies for breast cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6a18864245f8cad5e8cc957c1b2db50aef88c" target='_blank'>
              Unraveling the Mystery of Breast Cancer Dormancy: Insights into Genetic, Receptor, and Cellular Interactions Driving Late Recurrence
              </a>
            </td>
          <td>
            Haochen Ma, Bingqiang Zhang, Mengmeng Chen, Zhendi Song, Yi Zhao
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The serrated pathway represents a significant route to colorectal cancer (CRC), accounting for approximately 15–30% of cases, yet the specific epithelial cell subpopulations driving this pathway remain poorly understood. This study explores the causal relationship between serrated epithelial cells and CRC risk using single-cell transcriptomics and Mendelian randomization (MR). Publicly available single-cell RNA sequencing data were utilized to analyze epithelial cell subpopulations in CRC, focusing on specific serrated cells (SSCs). By integrating genome-wide association study data, MR was employed to assess the causal relationship between gene expression patterns and CRC risk. The study found that an increase in SSCs is closely associated with CRC progression. MR analysis revealed a significant correlation between expression changes in specific genes, such as IER3 in SSCs, and CRC risk (p < 0.05). Functional analyses indicated that IER3 may promote malignancy by regulating cell proliferation, adhesion, and immune evasion. Several genetic loci related to SSC gene expression were identified and validated for CRC risk association. This study demonstrates the significant role of serrated epithelial cell subpopulations in CRC development, particularly through key genes such as IER3, providing new perspectives for understanding CRC pathogenesis and future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd087daa1599bd00592ced3c9e805ceb0377196a" target='_blank'>
              Single-Cell Dissection of the Serrated Pathway: Cellular Heterogeneity and Genetic Causality in Colorectal Cancer
              </a>
            </td>
          <td>
            Ming Cen, Yunhan Wen, Zhijun Feng, Yahai Shu, Chuanxia Hu
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cell signaling plays a critical role in regulating cellular state, yet uncovering regulators of signaling pathways and understanding their molecular consequences remains challenging. Here, we present an iterative experimental and computational framework to identify and characterize regulators of signaling proteins, using the mTOR marker phosphorylated RPS6 (pRPS6) as a case study. We present a customized workflow that uses the 10x Flex assay to jointly profile intracellular protein levels, transcriptomes, and CRISPR perturbations in single cells. We use this to generate a “glossary” dataset of paired protein–RNA measurements across targeted perturbations, which we leverage to train a predictive model of pRPS6 levels based solely on transcriptomic data. Applying this model to a genome-wide Perturb-seq dataset enables in silico screening for pRPS6 and nominates novel regulators of mTOR signaling. Experimental validation confirms these predictions and reveals mechanistic diversity among hits, including changes in signaling output driven by anabolic activity, cellular proliferation and multiple stress pathways. Our work demonstrates how integrated experimental and computational approaches provide a scalable framework for multimodal phenotyping and discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a589887aa331a2fbdb712cfa97ab0770a36bd6c" target='_blank'>
              Iterative, multimodal, and scalable single-cell profiling for discovery and characterization of signaling regulators
              </a>
            </td>
          <td>
            John Blair, Alexandra Bradu, Carol Dalgarno, Isabella N. Grabski, Rahul Satija
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09d2a9d5ee2e4cda19d7cf627d28deb93a6522c" target='_blank'>
              Multi-modal analysis of pediatric pilocytic astrocytomas reveals tumor location-associated cellular and transcriptional heterogeneity.
              </a>
            </td>
          <td>
            J. Lammers, Jacob S Rozowsky, F. Calkoen, Thijs J M van den Broek, J. Meesters-Ensing, Tiago Carvalheiro, Tiago P J Rijsdijk, Robyn D Gartrell, D. V. van Vuurden, H. Stunnenberg, E. Hoving, P. Wesseling, J. van der Lugt, M. Kranendonk, Lennart A. Kester
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metaplastic breast cancer (MpBC) is defined by the presence of various morphological elements, typically biphasic, with epithelial (e.g. no-special type (NST), squamous) and mesenchymal (e.g. spindle, chondroid, osteoid) components. The established clonality of the different components favours an evolution model encompassing either a multipotent progenitor, or a linear metaplastic conversion. We used methylation profiling and showed that different morphologies have specific methylation profiles. Furthermore, our spatial transcriptomic approach, using 10× Genomics Visium and trajectory analysis, evidenced that spindle cells form a transition between the originating carcinoma of no-special type (NST) and pleomorphic regions, with osteoid differentiation likely to be an end-stage fate of the chondroid growth pattern, supporting the conversion model of lineage differentiation. We have also identified a series of master transcription factors likely to regulate these processes, and are significantly associated with metaplastic-like clinical features. This data further supports the conversion model of metaplasia and warrants functional analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077285caa17e07f895159bb1ece8c70f06837f91" target='_blank'>
              Integrated profiling of metaplastic breast cancer identifies putative master regulators of intratumoral heterogeneity
              </a>
            </td>
          <td>
            Yufan Feng, Albert Xiong, Onkar Mulay, Anna Sokolova, M. Lim, Benjamin Van Haeringen, Natasha McGuire, Xavier M. de Luca, Peter T Simpson, Quan Nguyen, S. Lakhani, A. M. McCart Reed
          </td>
          <td>2025-08-11</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT The recent introduction of immune checkpoint inhibitor therapy has significantly improved outcomes for patients with colorectal cancer (CRC). The most pronounced clinical benefits were observed in patients with immunogenic microsatellite instable (MSI)/deficient MMR (dMMR) tumors. However, emerging evidence indicates that a subset of patients with microsatellite stable tumors may also respond to therapy. Finding predictive markers to identify these patients is critical. In this study, we analyzed the immunohistochemical expression of immune checkpoints CTLA-4, PD-1, and PD-L1 using multispectral imaging in 151 CRC patients with defined molecular characteristics. Consistent with prior reports, MSI tumors had higher levels of all immune checkpoints analyzed than microsatellite stable tumors. Notably, distinct patterns of immune checkpoint expression were associated with KRAS and BRAF mutation status. KRAS-mutated tumors showed lower, and BRAF-mutated tumors higher, expression of immune checkpoints compared to wild-type/wild-type tumors. The strongest association with KRAS and BRAF mutations was observed for PD-L1 expression. The relationship between PD-L1 and KRAS/BRAF-mutational status was validated in a second cohort of 527 CRC patients, finding the association for PD-L1 expression in both stroma and in tumor cells. Furthermore, the role of BRAF mutation on immunity in CRC was found to be partly independent of MSI status. The strongest prognostic role was found for PD-L1 in stroma, underscoring the clinical significance of this marker. In conclusion, our findings suggest that KRAS and BRAF mutations, alongside MSI, may serve as valuable biomarkers for identifying CRC patient subgroups likely to benefit from immune checkpoint blockade in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f746db1756c17b9f3925528556749bad666f054" target='_blank'>
              Differential distribution of immune checkpoints across molecular subtypes of colorectal cancer
              </a>
            </td>
          <td>
            S. Edin, B. Gylling, Xingru Li, Åsa Stenberg, Anna Löfgren-Burström, B. van Guelpen, Agnes Ling, I. Ljuslinder, R. Palmqvist
          </td>
          <td>2025-08-21</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="ABSTRACT In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes ( CXCL12-CXCR4 , CCL5-SDC4 ), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine. SUMMARY Despite major advances in spatial biology, no framework has yet linked spatially resolved intercellular communication networks, independent of cell types, to clinical outcomes in human disease. Here, we present STARComm, a scalable method that identifies Multicellular Interaction MCIMs (MCIMs). Applying STARComm to minor salivary gland biopsies from patients with chronic graft-versus-host disease (GVHD), we identify MCIMs that not only distinguish healthy from diseased tissue but also stratify patient survival. High-risk MCIMs are enriched for actionable immune and stromal pathways, including those targetable with existing therapies. These findings establish the first outcome-linked spatial communication framework in any human disease and highlight the translational potential of oral tissues as minimally invasive platforms for real-time immune diagnostics, prognostic modeling, and therapeutic screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a093b13aff7d0f5c2b6d575cc2d04f2b86e47ba" target='_blank'>
              STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer
              </a>
            </td>
          <td>
            Khoa Huynh, B. Matuck, Deiziane de Souza, XiuYu Zhang, Giancarlo Fatobene, Luiz Alberto Valente Soares Júnior, L. F. Ferraz da Silva, V. Rocha, K. Byrd, Jinze Liu
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8261c73bf75d788913526d1e389ee8b4f740f28c" target='_blank'>
              Inhibition of ACVR1 in Cancer-Associated Fibroblasts Suppresses Colorectal Cancer Cell Growth.
              </a>
            </td>
          <td>
            Shinya Kato, N. Miyoshi, S. Fujino, Masafumi Horie, Shinichi Yachida, Mitsunobu Takeda, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, Mamoru Uemura, Hirofumi Yamamoto, Masayoshi Yasui, M. Ohue, Y. Doki, H. Eguchi
          </td>
          <td>2025-07-31</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioma intratumoral heterogeneity remains a critical barrier to effective treatment, driving recurrence and resistance to therapy. Emerging evidence suggests that glioma cells acquire neural-like features through paracrine and synaptic communication with neural cells, fostering functional diversity within tumors. While neuro-cancer interactions are implicated in glioma heterogeneity, their precise roles remain incompletely synthesized. Here, by consolidating these discoveries, a conceptual framework is proposed for understanding the glioma intratumoral heterogeneity shaped by neuro-cancer interactions: cellular phenotypes, spanning neurogliomal synapses, glioma networks, and neuronal-like motility; and transcriptional states, which exhibit remarkable resemblance to neural cells. These diverse cellular phenotypes and transcriptional states synergistically fuel glioma progression, invasion, and resistance. By emphasizing the converging phenotypical and transcriptional evidence with spatial context within them, an underexplored but critical role of neuro-cancer interactions are proposed in glioma intratumoral heterogeneity. This provides potential strategies to explore and disrupt these neuro-cancer interactions, offering new insights to address glioma intratumoral heterogeneity for improved therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7eca8bd6e6e1b76e76ba8a64aea76418740245" target='_blank'>
              Neuro-Cancer Interactions Shape Glioma Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Tong Wu, Peiran Zhao, Zheng Chen, Tianlei Ying, Zhihong Nie, Wei Hua, Huiyan Li, Ying Mao, Wenbo Bu
          </td>
          <td>2025-08-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ff9bee2610b6983bce3cc41dfb0a549e1b3ea2" target='_blank'>
              SPP1+ macrophages polarized by lactate confer the progression of hypoxic adaptive tumor cells in brain.
              </a>
            </td>
          <td>
            Jianlei Zhang, Zhihui Li, Jiang Yin, Weina Fan, Hongfan Liao, Jing Dong, Xianfeng Yu, Yabing Cao, Qiong Zhang, Guopei Zheng, Qianming Yao
          </td>
          <td>2025-09-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a4c1ebff0edca5f86402765983e14a517f49f7" target='_blank'>
              Form Follows Function: A New Paradigm of Pancreatic Cancer Progression.
              </a>
            </td>
          <td>
            William Gasper, Giada Pontecorvi, Zhikai Chi, S. Enrico, Lily C. Goodman, Haneen Amarneh, Evelyn Mazzarelli, Joshua Thomas, Lucia Zhang, Rachel Handloser, Seth LeCates, A. L. Tomescu, Francesca Rossi, Myla Sykes, Anjali Bhatt, Nicholas Fazio, Alessandro Bifolco, Nafeesah Fatimah, Austin Seamann, Cheryl M Lewis, Shelby Melton, Herbert J Zeh, Megan B. Wachsmann, Dario Ghersi, Matteo Ligorio
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Hypoxia is a hallmark of the colorectal cancer (CRC) tumor microenvironment (TME) that drives malignant progression, chemoresistance, and immune evasion. However, the cellular heterogeneity underpinning hypoxic responses in CRC and its impact on prognosis remain incompletely understood. Methods We integrated single-cell RNA sequencing data from 15 CRC samples (GSE166555 and GSE221575) to delineate hypoxic and normoxic cell populations and identify hypoxia-related genes (HRGs). Weighted gene co-expression network analysis (WGCNA) and enrichment profiling elucidated key biological processes associated with hypoxia. Cell–cell communication networks were inferred using CellChat, and transcription factor regulatory modules were reconstructed via SCENIC and GRNBoost2. A hypoxia-based prognostic signature was developed from unique H3 cluster genes using univariate Cox and Lasso regression on The Cancer Genome Atlas (TCGA; n = 606) and validated in GSE39582 (n = 579). Drug sensitivity correlations were derived from the GDSCv2 database. Finally, in vitro assays assessed the functional role of GIPC2, a model gene, in CRC cell lines. Results Single-cell profiling uncovered eight distinct hypoxic clusters, with H3 exhibiting the highest hypoxia scores and enrichment in glycoprotein metabolic and angiogenesis pathways. The eight-gene prognostic model stratified patients into high- and low-risk groups with significantly different overall survival in both TCGA (P = 0.0026) and validation cohorts (P = 0.011). Drug analysis highlighted associations of model genes with PI3K/MTOR and apoptosis pathways. GIPC2 knockdown in LS180 and HT-29 cells markedly inhibited proliferation, migration, and invasion, while inducing apoptosis and reversing EMT phenotypes. Conclusions We present a robust hypoxia-related gene signature that accurately predicts CRC patient prognosis and nominate GIPC2 as a potential biomarker and therapeutic target, offering new insights into hypoxia-driven CRC biology and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8575947e15fbc35fb6bf68728fce7870efdbff1" target='_blank'>
              Decoding the hypoxic tumor microenvironment in colorectal cancer for prognostic modeling and therapeutic target discovery
              </a>
            </td>
          <td>
            Xiao-Cui Duan, Yan Zhou, Fan Feng, Hai-Bo Jiang, Mei-Lin Wang, Zhe Han, Hong-Fei Pang, Yu-Hang Liu, Heng-Zhe Jia, Meng- He, Hongpan Xu, Yuan-Yuan Wang
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent advances in single-cell studies have unveiled the extensive transcriptomic heterogeneity of cancer-associated fibroblasts (CAFs), with each subset potentially playing unique roles in the tumor microenvironment. Despite these insights, it remains unclear how to culture different subsets of CAFs in vitro while preserving their in vivo phenotype. The inherent plasticity of CAFs, characterized by their ability to dynamically alter their phenotype in response to various environmental stimuli, presents significant challenges in inducing and maintaining specific CAF states in vitro. In this study, we investigated the effects of cell shape and confinement on two-dimensional culture substrates on the transcriptomic profile and phenotype of cultured CAFs. By inducing cytoskeletal rearrangement and activating mechanotransductory pathways through micropatterning of polyacrylamide hydrogels, we observed significant shape- and confinement-dependent changes in cell morphology. Our results showed that micropatterned CAFs exhibited phenotypic shifts towards more myofibroblastic, desmoplastic and inflammatory subsets. Additionally, micropatterning allowed for control over a range of CAF-specific markers and pathways, significantly altering their transcriptome landscape and behavior. Lastly, we demonstrated that micropatterned and non-micropatterned CAFs responded differently to anti-cancer drugs, underscoring the importance of phenotype-oriented therapy that accounts for CAF plasticity and regulatory networks. Control over CAF morphology through micropatterning presents a unique opportunity to establish highly robust CAF phenotypes in vitro, thereby facilitating a deeper understanding of CAF plasticity and heterogeneity. This approach also holds promise for the development of novel therapeutic targets, as it allows for the precise manipulation of CAF states to better model and study their interactions within the tumor microenvironment.



 Aleksandr Mitriashkin, Josephine Yu Yan Yap , N. Gopalakrishna Iyer , Gianluca Grenci, Eliza Li Shan Fong. Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb0c539e2fb60deec8ba647c172fbc76cfd279" target='_blank'>
              Abstract LT03: Cell confinement by micropatterning induces phenotypic changes in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Aleksandr Mitriashkin, Josephine Yu Yan Yap, N. G. Iyer, Gianluca Grenci, Eliza Li Shan Fong
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Regulating tumor invasion and metastasis is pivotal for improving cancer patient prognosis. While cell migration is a key factor in these processes, the non-targeted effects of chemoradiotherapy on cell motility remain poorly understood. In this study, we employed HeLa-FUCCI cells—a cervical cancer-derived HeLa cell line integrated with the Fluorescent Ubiquitination-Based Cell Cycle Indicator (FUCCI) probe, enabling the visualization of cell cycle phases—to investigate the radiation-induced impacts, including non-targeted effects, on cell migration. To create irradiated (In-field) and non-irradiated (out-of-field) regions, half of the culture dish was shielded with a lead block during irradiation. Cells were then exposed to 2 Gy X-rays, with or without cisplatin. Following irradiation, the cells were subjected to time-lapse imaging at 15 min intervals for 24 h, and the acquired data were analyzed using cell segmentation and tracking algorithms, Cellpose 2.0 and TrackMate 7. Without cisplatin, the migration velocity and total distance traveled of Out-of-field cells were significantly reduced compared to controls, suggesting a suppressive bystander signal. In contrast, with cisplatin treatment, these parameters significantly increased in both In-field and Out-of-field cells. This suggests that chemoradiotherapy may inadvertently enhance tumor cell motility outside the target volume, a critical finding with significant implications for therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186add6c9ae96976df3f063bad12c6cb1e871668" target='_blank'>
              Deep Learning-Based Imaging Analysis Reveals Radiation-Induced Bystander Effects on Cancer Cell Migration and the Modulation by Cisplatin
              </a>
            </td>
          <td>
            Ryosuke Seino, Hisanori Fukunaga
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background The N-formyl peptide receptor family (FPRs) is implicated in the progression of diverse cancer types, yet studies specifically exploring their roles in breast cancer remain scarce. Methods A comprehensive analysis integrating bulk RNA-seq transcriptomics, methylomics, single-cell transcriptomics, and spatial single-cell transcriptomics data was conducted to elucidate the distinctive characteristics of FPRs in breast cancer. This study particularly focused on delineating the e xpression profiles of FPR3 across distinct breast cancer subtypes, while systematically investigating its prognostic implications and association with macrophage polarization patterns in breast cancer patients. Furthermore, molecular docking analysis was performed to screen potential therapeutic compounds targeting FPR3, providing insights into its druggability. Results Notably, FPR3 was found to be highly expressed in macrophages within breast cancer tissues, with a notably elevated level in HER2-positive and triple-negative breast cancer (TNBC) subtypes, both of which are associated with poor prognosis. FPR3 expression inversely correlates with promoter methylation levels. Further analysis of pan-cancer immune infiltration patterns uncovered a striking association between FPR3 and macrophage infiltration, as well as their polarization status. Knockdown of FPR3 expression in macrophages markedly enhanced the expression of IL6, TNF-α, and TGF-β, while significantly reducing IL10 levels, indicative of a shift towards an M1-like macrophage phenotype. Through computational molecular docking analyses, Otamixaban and Rivaroxaban emerged as promising candidate inhibitors of FPR3. Conclusions These findings underscore the profound infiltration of FPR3 + macrophages in breast cancer patients with adverse prognoses, highlighting FPR3 as a potential therapeutic target for intervening breast cancer aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03942-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18854fe793c2d29227b32ba5f47639fad4edcf8c" target='_blank'>
              FPR3 orchestrates macrophage polarization in breast cancer: multi-omics dissection of prognostic relevance and therapeutic targeting
              </a>
            </td>
          <td>
            Ying Chen, Xin Tang, Liran Zhu, Yi Wang, Gaopeng Li, Wulin Yang
          </td>
          <td>2025-08-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is a highly aggressive brain tumor characterized by aberrant angiogenesis and an immunosuppressive microenvironment. Pericytes are aberrantly recruited but their spatiotemporal roles and molecular changes remain unclear. This study investigated platelet-derived growth factor receptor beta-positive (Pdgfrb+) pericyte dynamics and reprogramming in GBM vasculature.


METHODS
We generated GL261-Luc and GL261-CFP glioblastoma cells via lentiviral transduction and established two transgenic models. (1) For pericyte labeling, Ai14 reporter mice was crossed with PDGFRβ-P2A-CreERT2 mice for tdTomato-specific lineage tracing (PT mice). (2) For conditional ablation, we generated inducible Pdgfrb-expressing cell ablation models (PT mice was crossed with ROSA-DTA mice). An intravital imaging platform (FITC-dextran/CFP/tdTomato + two-photon microscopy) tracked pericytes, vessels, and tumor cells, while FACS-sorted Pdgfrb+ cells from GBM and normal brain were analyzed by LC-MS/MS proteomics.


RESULTS
Cre-mediated ablation of Pdgfrb-expressing cells revealed stage-dependent effects on GBM growth: early ablation inhibited progression while late ablation promoted it. Pericytes undergo dual spatial reorganization in GBM: regional enrichment with pre-sprouting accumulation at the tumor-brain interface, and focal positioning with preferential localization at vascular branch points. Concurrently, GBM vasculature displayed simplified branching, dilation, and pericyte remodeling (shorter processes, higher density). Proteomics revealed 1426 altered proteins, with upregulated proliferation pathways (e.g., matrix metallopeptidase 14 [Mmp14], lysyl oxidase like 2 [Loxl2]) and downregulated homeostasis functions (e.g., transforming growth factor beta 1 [Tgfb1]), validated by scRNA-seq in human GBM.


CONCLUSIONS
This study demonstrates that during early GBM progression, pericytes actively drive tumor angiogenesis through molecular reprogramming toward proliferative and pro-angiogenic phenotypes, with the integrated imaging-proteomics framework revealing potential therapeutic targets for disrupting pericyte-mediated vascular remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/531e6a56e9b350ddd48811a1f650e717592a9882" target='_blank'>
              Insight into pericytes in glioblastoma angiogenesis: In vivo tracking by two-photon microscopy and proteomic profiling.
              </a>
            </td>
          <td>
            Qinghong Wang, Chengyan Ma, Xinpei Wang, Mengyuan Li, Xingjiu Yang, Ran Gao
          </td>
          <td>2025-08-07</td>
          <td>Animal models and experimental medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Luminal and basal molecular subtypes were first described in breast cancer, where they show distinct biological properties and clinical outcomes. Recent research has revealed that these subtypes are also present in many other epithelial cancers, beyond their tissue of origin. Basal tumors are often linked to aggressive behavior, poorer prognosis, and potential sensitivity to treatments such as DNA-damaging agents or immunotherapy. Luminal tumors usually have more favorable outcomes and may respond to hormone-related therapies. In this review, we summarize studies published between 2010 and 2024 that have applied luminal/basal subtyping across different cancer types and within single tumor types. Our aim is to provide readers with a clear overview of how these molecular programs are defined, where they occur, and why recognizing them could help improve cancer classification, prognosis, and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e1cd92bd5684128dd1291ba7c9bf8b475416dcc" target='_blank'>
              Luminal and Basal Subtypes Across Carcinomas: Molecular Programs Beyond Tissue of Origin
              </a>
            </td>
          <td>
            Celia Gaona-Romero, M. E. Domínguez-Recio, I. Comino-Méndez, M. V. Ortega-Jiménez, R. Lavado-Valenzuela, Emilio Alba
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The transition from acute kidney injury to chronic kidney disease is characterized by significant changes in the cellular composition and molecular interactions within the kidney. Utilizing high-resolution Xenium and whole transcriptome Visium spatial transcriptomics platforms, we analyze over a million cells on 12 male mouse kidneys across six stages of renal injury and repair. We define and validate 20 major kidney cell populations and delineate distinct cellular neighborhoods through this multimodal spatial analysis. We further reveal a specific fibro-inflammatory niche enriched in failed-repair proximal tubule cells, fibroblasts, and immune cells, with conserved neighborhood gene signatures across mouse and human. Within this niche, we predict Runx2 as a key upstream regulator, along with platelet-derived growth factor and integrin beta-2 signaling pathways shaping the fibrogenic microenvironment. Altogether, our study provides deep insights into the cellular and molecular dynamics during kidney injury and repair and establishes a comprehensive multimodal analytical framework applicable to other spatial omics studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80e9ed63af964434bd5c50df588ce36470352d4" target='_blank'>
              Multimodal spatial transcriptomic characterization of mouse kidney injury and repair
              </a>
            </td>
          <td>
            Xuanyuan Qiao, Haojia Wu, Hemalatha Sundaramoorthi, Pierre Isnard, Changfeng Chen, Waleed Rahmani, Benjamin D. Humphreys
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by rapid proliferation and infiltration of immune cells into tumor microenvironment (TME). The treatment of TNBC still remains challenging due to the lack of expression of effective molecular targets pertaining to the tumor cell itself. In TNBC standard of care, therapies such as chemotherapy, together with recently introduced immunotherapy with checkpoint inhibitors, often do not result in durable clinical response. Therefore, better understanding of complex interactions between tumor cells, immune cells, and stromal cells mediated by multiple cytokines, chemokines, enzymes, and metabolites in TME is crucial for understanding the mechanisms that underlie tumor cell immune evasion strategies. The aim of this review is to give comprehensive overview of immune cell network and their interactions with cells in TME and possibilities for therapeutic targeting of TME in TNBC. We discuss cancer-associated fibroblasts (CAFs) as an important recently characterized player in TNBC with respect to their role in interactions with immune cells and their impact on tumor invasion. Based on the recently accumulated knowledge, therapies targeting immune suppressive mechanisms and CAF-related tumor-promoting mechanisms in TME hold great potential for clinical evaluation in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3efa870eaa016c596531e2952c414f514a615d" target='_blank'>
              The Role of Tumor Microenvironment in Triple-Negative Breast Cancer and Its Therapeutic Targeting
              </a>
            </td>
          <td>
            A. Vuletić, K. Mirjačić Martinović, V. Jurišić
          </td>
          <td>2025-08-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in melanoma growth, resistance to treatment, and relapse, however, their dynamic regulatory behavior remains poorly understood. Vibrational spectroscopy offers a unique, label-free approach to investigate cellular heterogeneity at the molecular level. Here, we explored the biochemical and regulatory dynamics of CSCs identified by using a time-course design, integrating infrared and Raman spectroscopies with cell cycle analysis and immunocytochemistry targeting the checkpoint proteins p16 and p21. CSCs, non-cancer stem cells (NCSCs), and bulk CHL-1 melanoma cells were monitored at 11, 24, 48, and 72 h. CSCs showed a steady S-phase with an early rise in p16 followed by a subsequent increase in p21 expression, indicating a dynamic state of cell cycle checkpoints. In contrast, NCSCs and CHL-1 cells showed more transient p16/p21 expression and CHL-1 exhibited a marked p16 increase at 24 h. Spectroscopic analysis revealed that CSCs exhibited distinct vibrational profiles, predominantly in the nucleic acid-, protein- and lipid-associated regions. These differences were further supported by principal component and hierarchical clustering analyses, which consistently distinguished CSCs from NCSCs. Our findings underline the potential of vibrational spectroscopy to sensitively detect CSC-specific regulatory patterns and support its use in detecting new therapeutic targets in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23a07ebb80708549e93a2a342c6077ba0b56ea8" target='_blank'>
              Vibrational spectroscopy unveils distinct cell cycle features of cancer stem cells in melanoma
              </a>
            </td>
          <td>
            Bensu Rüya Uslu, Berrin Ozdil, E. Tarhan, Serdar Özçelik, H. Aktug, Günnur Güler
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Melanoma brain metastasis (MBM) leads to significant morbidity and mortality. Building on previous findings that WNT5A regulates melanoma metastasis and dormancy in extracranial niches, we hypothesize that WNT5A signaling facilitates melanoma invasion across the blood-brain barrier, while its downregulation drives intracranial metastatic proliferation. To explore this, we conducted functional studies using human melanoma cell lines and an organotypic mouse brain xenograft system. Overexpression or pharmacologic induction of WNT5A increased melanoma migration and invasion toward brain explant conditioned media in transwell assays. Wholemount confocal imaging of brain explant cultures revealed distinct tumor colonization patterns mirroring clinical dormancy and proliferative outgrowth depending on WNT5A level: melanoma overexpressing WNT5A formed single-cell dispersions perivascularly, whereas silencing WNT5A in syngeneic cells produced proliferative clusters. Exogenous recombinant WNT5A also induced G1 cell cycle arrest in FUCCI-transduced melanoma cells, supporting its role in maintaining dormancy. Conversely, the WNT5A antagonist SFRP1 drove melanoma toward increased tumor size and accelerated tumor development. Interestingly, immunohistochemical analysis of postmortem human brain tissue demonstrated robust SFRP1 expression in cerebral endothelium and neurons, indicating a potential source promoting metastatic outgrowth. Integrin profiling revealed elevated expression of adhesion proteins in invasive melanoma compared to WNT5A-silenced syngeneic lines, suggesting an association with enhanced metastatic potential and tumor extravasation. WNT5A overexpression resulted in greater endothelial and extracellular matrix engagement by adhesion assay and live-cell tracking. These interactions were recapitulated in ex vivo xenografts, with WNT5A-high melanoma adhering to endothelial basement laminin. Together, these findings highlight the importance of WNT5A regulation through the metastatic cascade, from endothelial adhesion and establishment of a perivascular dormancy niche to metastatic outgrowth. Further integration of transcriptomics and proteomics on mouse models and clinical MBM samples aim to uncover key modulators of WNT signaling and associated pathways in MBM progression which might be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a305149b00eaf3bc4069c98fdc0c3ac940583e00" target='_blank'>
              BSBM-02 WNT5A MEDIATES MELANOMA BRAIN METASTASIS
              </a>
            </td>
          <td>
            Kevin Zhang, Denys Balandin, M. R. Rocha, Vania Wang, Fan Huang, Cheyenne Palm, Alexis E Carey, Ritvik Koya, Jordina Rincon-Torroella, C. Eberhart, Calixto-Hope G. Lucas, A. Weeraratna
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d94128a4ca71f38899bbd2a679a0555681c49d19" target='_blank'>
              Microenvironment crosstalk and immune evasion of circulating tumor cells: From mechanism to clinical significance.
              </a>
            </td>
          <td>
            Shi Chen, Sirui Zhang, Ying Shi, Xiaowei Liu
          </td>
          <td>2025-08-08</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The cornea, a transparent tissue composed of multiple layers, allows light to enter the eye. Several single-cell RNA-seq analyses have been performed to explore the cell states and to understand the cellular composition of the human cornea. However, the inconsistences in cell state annotations between these studies complicate the application of these findings in corneal studies. To address this, we integrated single-cell RNA-seq data from four published studies and created a human corneal cell state meta-atlas. This meta-atlas was subsequently evaluated in two applications. First, we developed a machine learning pipeline cPredictor, using the human corneal cell state meta-atlas as input, to annotate corneal cell states. We demonstrated the accuracy of cPredictor and its ability to identify novel marker genes and rare cell states in the human cornea. Furthermore, cPredictor revealed the differences of the cell states between pluripotent stem cell-derived corneal organoids and the human cornea. Second, we integrated the single-cell RNA-seq based cell state meta-atlas with chromatin accessibility data, conducting motif-focused and gene regulatory network analyses. These approaches identified distinct transcription factors driving cell states of the human cornea. The novel marker genes and transcription factors were validated by immunohistochemistry. Overall, this study offers a reliable and accessible reference for profiling corneal cell states, which facilitates future research in cornea development, disease and regeneration. Significance statement This study creates a human corneal cell state meta-atlas that provides a common nomenclature of cells in the human cornea, through integrating multiple single-cell RNA-seq analyses. Using this meta-atlas, we developed a machine learning pipeline, cPredictor, to accurately annotate cell states in corneal studies using single-cell RNA-seq. Additionally, we identified distinct transcription factors driving cell states through integrating the atlas with chromatin accessibility data. This meta-atlas and the computational tool cPredictor enable future research in cornea development, disease, and regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e652a5bfb1e59f33d0b9da5cfaa537300d8603" target='_blank'>
              Prediction of cell states and key transcription factors of the human cornea through integrated single-cell omics analyses
              </a>
            </td>
          <td>
            Julian A. Arts, Sofia Fallo, Melanie S. Florencio, J. G. Smits, D. L. Cunha, J. Roubroeks, M. Dickman, V. LaPointe, Rosemary Yu, Huiqing Zhou
          </td>
          <td>2024-12-20</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) and interleukins (ILs) family play crucial roles in pancreatic carcinoma (PC) immune response. However, the correlation between the IL family, CAFs infiltration, and PC prognosis remains uninvestigated. Methods An IL family expression pattern for prognosis was constructed using a stepwise Cox proportional hazards regression model using TCGA data. Clinical data and validations from seven independent public cohort datasets were conducted to confirm the performance of the model. CAFs infiltrating abundance and spatial distribution in PC, and their correlation with patient prognosis were detected. Correlation between IL expression pattern, CAF infiltration, and immunotherapy response was evaluated using clinical tissue samples. Results This study constructed the first IL family expression pattern to predict CAFs infiltration and prognosis in PC. The model was validated using clinical data and a meta-analysis of seven public PC datasets (HR= 1.27). IL high-risk patients had shorter survival, advanced tumors and lymph node metastasis compared to low-risk patients. Patients with unfavorable immunotherapy response had significantly higher IL risk scores (P=0.015). The IL expression pattern distinguished CAFs infiltration characteristics in PC, showing greater infiltration of CAFs, antigen-presenting CAFs (apCAFs) and inflammatory CAFs in the high-risk group. IL high-risk group also exhibited increased apCAF/tumor cell and apCAF/Tregs engagement, resulting in suppressed immune responses, crippled T-cell function and B-cell function, and elevated levels of biomarkers associated with poor immune response. Conclusion This study constructed the first IL expression pattern related to CAFs infiltration, immunotherapy response, and prognosis in PC patients. This might promote precise immunotherapy and facilitate treatment options for PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a6478bb61c907205c88b59fe29e4d7b550ad6eb" target='_blank'>
              Interleukin Family-Based Signature Relates to Cancer-Associated Fibroblasts Spatial Distribution and Immune Therapy Response in Pancreatic Carcinoma
              </a>
            </td>
          <td>
            Yang Cheng, Shuzhe Xiao, Xiangzhao Li, Biao Wang, Yi Lei, Penghui Sun, Li Ma, Yun Zhu
          </td>
          <td>2025-07-29</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Paediatric Acute Myeloid Leukemia is a complex adaptive ecosystem with high morbidity. Current trajectory inference algorithms struggle to predict causal dynamics in AML progression, including relapse and recurrence risk. We propose a symbolic AI and deep learning framework grounded in complexity science, integrating Recurrent Neural Networks, Transformers, and Algorithmic Information Dynamics to model longitudinal single cell transcriptomics and infer complex state transitions in paediatric AML. We identify key plasticity markers as predictive signatures regulating developmental trajectories. These were derived by integrating deep learning with complex systems based network perturbation analysis and dynamical systems theory to infer high dimensional state space attractors steering AML evolution. Findings reveal dysregulated epigenetic and developmental patterning, with AML cells in maladaptive, reprogrammable plastic states, i.e., developmental arrest blocking terminal differentiation. Predictions forecast neurodevelopmental and morphogenetic signatures guiding AML cell fate bifurcations, suggesting ectoderm mesoderm crosstalk during disrupted differentiation. Neuroplasticity and neurotransmission related transcriptional signals implicate a brain immune hematopoietic axis in AML cell fate cybernetics. This is the first study combining RNNs and AID to predict and decipher longitudinal patterns of cell fate transition trajectories in AML. Our complex systems approach enables causal discovery of predictive biomarkers and therapeutic targets for ecosystem engineering, cancer reversion, precision gene editing, and differentiation therapy, with strong translational potential for precision oncology and patient centered care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/068dc3116dfa4da6137175c922c0e16868f58720" target='_blank'>
              Neurosymbolic Learning for Predicting Cell Fate Decisions from Longitudinal Single Cell Transcriptomics in Paediatric Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Abicumaran Uthamacumaran, Hector Zenil
          </td>
          <td>2025-08-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Background Melanoma is one of the most immunogenic malignancies, yet resistance to immune checkpoint inhibitors (ICIs) remains a major obstacle to durable therapeutic success. Emerging evidence indicates that aging‐related processes, including cellular senescence and immunosenescence, can reshape the tumor microenvironment (TME) to favor immune evasion and disease progression. Senescent melanoma and stromal cells secrete a senescence‐associated secretory phenotype (SASP) that alters immune cell recruitment and function, while immunosenescence leads to diminished cytotoxic responses and the accumulation of dysfunctional or suppressive immune subsets. Aim This review explores the interplay between cellular senescence and immunosenescence in melanoma, highlighting their contributions to tumor progression and immunotherapy resistance, and discusses potential strategies to therapeutically target senescence‐related pathways. Methods A systematic review of studies published between 2000 and 2024 was performed using PubMed, Web of Science, and Scopus. Literature included mechanistic investigations of senescence in melanoma, analyses of immunosenescence in cancer patients, and preclinical or translational studies targeting senescence‐related pathways. Results and Conclusions Senescent tumor and stromal cells drive a pro‐inflammatory and immunosuppressive TME through SASP, while aging immune cells exhibit impaired antigen presentation, reduced cytotoxicity, and increased suppressive subsets. These dual processes form a self‐reinforcing cycle of chronic inflammation and immune dysfunction, ultimately undermining the efficacy of ICIs. Targeting senescence, through senolytics, senostatics, or SASP modulators, has shown promise in preclinical models and may restore immune competence in melanoma. However, clinical translation requires further investigation to validate safety and efficacy. Addressing both cellular and immune senescence represents a novel and promising direction to overcome therapeutic resistance and improve melanoma outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bbf2caec95f9026cf513548d86d38efae5d8ce" target='_blank'>
              Cellular Senescence and Immunosenescence in Melanoma: Insights From the Tumor Microenvironment
              </a>
            </td>
          <td>
            Lihua Xiong, Jian Cheng
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent advancements in single-cell sequencing technology have reshaped our understanding of cellular processes. While in the realm of biological research, the understanding of the underlying physical and chemical mechanisms from single-cell omics stands as a promising frontier, yet it is still not quite adequately explored. This knowledge gap stems from the complexities of mapping nonequilibrium physical and chemical principles onto the heterogeneous and complex dynamics of cellular functions. Herein, several key biological processes are highlighted to embark on this challenging journey of harnessing the power of single-cell omics to elucidate the nonequilibrium physical and chemical basis of various biological cell functions, including cell cycle, cell differentiation/reprogramming, cancer progression and metastasis, and embryonic dynamic development. This perspective presents breakthrough insights into cell division and differentiation, highlighting the nonequilibrium landscape and flux as the driving forces governing the cellular function which can be quantified through the single-cell omics data, providing new insights into cellular plasticity and fate determination. In addition, it provides possible early warning signals of cancer formation and metastasis based on omics data. Venturing into the wonders of dynamical development, it shows the uncovered nonequilibrium physicochemical mechanisms determined by the dynamical landscape-flux of embryogenesis from time-resolved single-cell data. This perspective further offers an outlook on challenges and opportunities for the integrations of spatiotemporal multiomics and nonequilibrium physical and chemical theories, in anticipation of a more comprehensive and in-depth understanding of the myriad processes of life. Hence, this perspective summarizes key advances in this emerging field and points to the next opportunities and challenges to fully integrate the potential of single-cell biotechnology and physical chemistry theory in life science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0321194975efe56e0cc49e03ac6617ecc83ce7ca" target='_blank'>
              Quantifying Landscape and Flux from Single-Cell Omics: Unraveling the Physical Mechanisms of Cell Function
              </a>
            </td>
          <td>
            Ligang Zhu, Jin Wang
          </td>
          <td>2025-08-07</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Emerging evidence links alterations in the tumor microbiome to therapeutic responses to immunotherapy. Alterations in β-catenin are among the most frequently observed oncogenic drivers of hepatocarcinogenesis and are associated with T-cell exclusion. However, their effect on the immune cell environment and microbiome in hepatocellular cancer is not well understood. We hypothesized that β-catenin could modulate the immune microenvironment through alterations in the secretome and release of extracellular vesicles (EV) that mediate tumor and immune cell interactions and increase tumor growth within regions with attenuated immune activity and reduced microbial diversity. Methods We used a synthetic transgenic murine model of β-catenin-driven hepatocarcinogenesis to analyze microbiome composition, diversity, and immune cell profiles in vivo. Tumor and stool samples were collected from mice with early- or late-stage hepatocellular carcinoma and were used for profiling. Results The microbiome associated with intrahepatic tumors differs from that in non-tumoral regions, normal liver tissues, and gut tissues. Constitutive β-catenin expression modulates lipopolysaccharide-mediated signaling in macrophages and alters the secretion of immunomodulatory chemokines and cytokines. Tumoral immune cell profiles differed from those in hepatic tissues. EV-mediated signaling between immune cells and epithelial cells with mutant β-catenin and immune cells modulates immune cell populations in vitro and in vivo. Conclusion Mutations in β-catenin can drive immune responses through EV-based tumor cell-immune cell interactions to modulate both the tumor microflora and immune microenvironment. A potential strategy to augment responses to immunotherapy for hepatocellular cancer could target these interactions to restore microbial diversity or immune cell infiltration within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d73e045384f38112d9502014ce3fbc6f907c47" target='_blank'>
              Beta-Catenin Mutations Can Impact on the Interplay Between Tumor and Immune Cells and Hepatic Microbiota in Hepatocellular Cancer
              </a>
            </td>
          <td>
            Yu Ota, Julia Driscoll, Anneliese Hill, Irene K Yan, Tushar Patel
          </td>
          <td>2025-08-01</td>
          <td>Journal of Hepatocellular Carcinoma</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma represents the most common principal malignant bone tumor that predominantly appears among teenagers and children. While multimodal treatment methods have greatly evolved with time, survival for recurrent or metastatic disease remains low due to the resistance that accumulates during treatment. Increasing evidence identifies the tumor microenvironment (TME), in particular cancer-associated fibroblasts (CAFs), as playing an important role in imposing immune suppression, enhancing tumor aggressiveness, and mediating resistance toward immunotherapy and chemotherapy. This article gives an overview of the derivation, phenotypic heterogeneity, and mechanisms of action of CAFs during osteosarcoma, such as facilitating immune escape, survival signaling, drug efflux, regulation of genes through exosomes, and inhibiting ferroptosis. Furthermore, we present existing and new treatment methods that are centered on CAFs, such as suppression of the paracrine pathway (e.g., IL-6/STAT3, TGF-β), depletion of CAFs lineages by targeting fibroblast activation protein (FAP), and conversion toward tumor-restraining CAFs. Other methods that are gaining popularity are targeting CAFs-releasing exosomes and metabolic liabilities. By shedding light on CAFs-based methods for imposing resistance and trying targeted treatments, this review offers insights into novel therapeutic combinations that can overcome treatment barriers and improve survival outcomes in osteosarcoma regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edf6237fc3513d36396d6dcdecd8243a643d651f" target='_blank'>
              Cancer-associated fibroblasts in osteosarcoma: key players in immune escape and targeted therapy
              </a>
            </td>
          <td>
            Junjie Fan, Yujie Jin, Feng Lv, Weidong Wu, Li Sun, Churong Wang
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/781db8d3b9951c3d4f119253d55c69b0ec7522ab" target='_blank'>
              Hybrid In Vivo Breast Cancer Model Reveals Transcriptomic Insights into Cancer Progression with Age.
              </a>
            </td>
          <td>
            Lauren Hawthorne, Gokhan Bahcecioglu, Jun Yang, Erin Howe, Emilija Aleksandrovic, Siyuan Zhang, P. Zorlutuna
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background The interactions among cell subgroups in the cervical cancer immune microenvironment play crucial roles in tumor development, but their causal relationships remain unclear. Methods This study employed Mendelian randomization to analyze causal associations between immune cell subgroups and cervical cancer. Multiple statistical methods, including inverse variance weighted, weighted median, and simple mode approaches, were used to evaluate effect sizes. Hierarchical clustering, UMAP, and t-SNE were applied for cell subgroup classification, combined with MIF signaling pathway analysis for cell-cell interaction networks. Results Most immune cell subgroups showed effect estimates close to 1.000 (95%CI: 0.997–1.002) with statistical significance (p < 0.05). Hierarchical clustering analysis revealed eight major cell populations: regulatory T cells, T cells, epithelial cells, natural killer cells, monocytes, ciliated epithelial cells, B cells, and fibroblasts. Cell-cell interaction network analysis demonstrated extensive connectivity among immune cells and between immune and epithelial cells, with particularly strong interactions between monocytes and other immune cells. MIF signaling pathway analysis further confirmed the close relationship between regulatory T cells and T cells. Conclusion This study systematically revealed the causal associations among cell subgroups in the cervical cancer immune microenvironment using Mendelian randomization, providing new insights into understanding tumor immune microenvironment regulation mechanisms and potentially offering theoretical basis for optimizing cervical cancer immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d095e06d9259f527b20902d8cf1fde0afa31bb99" target='_blank'>
              Association and interaction network analysis of immune cell subgroups in cervical cancer microenvironment based on Mendelian randomization
              </a>
            </td>
          <td>
            Yan Cai, Nanxing Jiang, Yan Wang, Min Ji, Wenzhe Shen, Qi-Ming Wang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034c8133be4ba672c54b97e9a66fb7c40de89b73" target='_blank'>
              Single-cell transcriptomic analysis reveals a metastasis-associated PCSK1+ neuroendocrine subpopulation in pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-08-16</td>
          <td>Journal of neuroendocrinology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb4c318da38745c1257f8f87c4b1e84917526c2d" target='_blank'>
              Spatial-in-Spatial: The Utility of Combining Fluorescence-Guided Multiple Sampling with Spatial-Omics in Human Glioblastoma.
              </a>
            </td>
          <td>
            Shubhang Bhalla, Bethsabe Romero, Yusor Al-Nuaimy, Felix Toussaint, Sina Zoghi, Niel Pacheco-Barrios, Stefan T Prvulovic, C. A. Bowers, S. Piccirillo
          </td>
          <td>2025-07-25</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma is the most prevalent primary brain malignancy, distinguished by its highly invasive behavior and exceptionally high rates of recurrence. Conventional radiation therapy, which employs uniform treatment margins, fails to account for patient-specific anatomical and biological factors that critically influence tumor cell migration. To address this limitation, numerous computational models of glioblastoma growth have been developed, enabling generation of tumor cell distribution maps extending beyond radiographically visible regions and thus informing more precise treatment strategies. However, despite encouraging preliminary findings, the clinical adoption of these growth models remains limited. To bridge this translational gap and accelerate both model development and clinical validation, we introduce PREDICT-GBM, a comprehensive integrated pipeline and dataset for modeling and evaluation. This platform enables systematic benchmarking of state-of-the-art tumor growth models using an expert-curated clinical dataset comprising 255 subjects with complete tumor segmentations and tissue characterization maps. Our analysis demonstrates that personalized radiation treatment plans derived from tumor growth predictions achieved superior recurrence coverage compared to conventional uniform margin approaches for two of the evaluated models. This work establishes a robust platform for advancing and systematically evaluating cutting-edge tumor growth modeling approaches, with the ultimate goal of facilitating clinical translation and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3497012bad0fd14f83f8c07402640538840f77c2" target='_blank'>
              PREDICT-GBM: Platform for Robust Evaluation and Development of Individualized Computational Tumor Models in Glioblastoma
              </a>
            </td>
          <td>
            L. Zimmer, J. Weidner, M. Balcerak, F. Kofler, I. Ezhov, B. Menze, B. Wiestler
          </td>
          <td>2025-09-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Interstitial lung diseases (ILDs) encompass a diverse group of pulmonary disorders, with progressive fibrosis leading to poor prognosis. Here we aimed to identify key molecules involved in progressive fibrosis across various ILDs, using spatial transcriptomics (ST). ST analysis (Visium) was performed on lung cryobiopsy specimens from five patients with various ILDs. Two cases, rich in young fibrotic lesions, as defined by fibroblastic foci and destructive alveolar organization, were selected for spatial high‐dimensional weighted gene coexpression network analysis (hdWGCNA) to identify key gene networks with biological significance in active fibrosis. We utilized public single‐cell RNA sequencing datasets of various ILDs, performed enrichment analysis and trajectory‐based differential expression analysis, and quantified cell–cell communication to evaluate the involvement of the spatially extracted module in fibrosis. Immunohistochemical staining of the extracted molecules was performed. Using hdWGCNA, we identified a distinct gene module (the SM2 module) enriched in young fibrotic lesions. The SM2 module was characterized by distinct features of fibroblast activation that were represented across various lesions. Key hub genes within this module, including COL1A2, COL3A1, COL1A1, and SPARC, formed a robust coexpression network. Immunohistochemical staining showed that SPARC, a component of the SM2 module, was highly expressed in young fibrotic lesions, but not in old scarring lesions, across various ILDs. To assess the prognostic significance of SPARC immunohistochemical expression, we extended our analysis to a cohort of 71 patients with unclassifiable ILDs (uILDs), a particularly heterogeneous subtype with unclear pathogenesis and limited treatment options. Higher SPARC levels in the upper, lower, or both lung lobes in uILD were significantly associated with poor overall survival. In summary, an integrated cross‐disease approach using ST revealed key gene expression patterns central to active fibrosis and successfully identified SPARC as a potentially beneficial prognostic marker. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a55a6327413c2017bbf7b10baf6a8049b2a4f0" target='_blank'>
              Spatial transcriptomics identifies SPARC as a prognostic marker in interstitial lung diseases
              </a>
            </td>
          <td>
            T. Niitsu, T. Kataoka, K. Fukushima, D. Motooka, Shigeyuki Shichino, Yayoi Natsume-Kitatani, H. Kitamura, Takashi Niwa, T. Baba, Daisuke Okuzaki, Atsushi Kumanogoh, Shizuo Akira, K. Okudela, T. Ogura
          </td>
          <td>2025-07-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Lu and colleagues identified profound microbial heterogeneity across individual tumor nodules in multifocal hepatocellular carcinoma, revealing that specific bacterial consortia enrich in metastatic clones and drive epithelial-mesenchymal transition and immune suppression. Their study places the intratumoral microbiome alongside genomic and transcriptomic diversity as a key modulator of intrahepatic tumor evolution. See related article by Lu et al., p. 1630.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb39fed6f77266430fee7056b5354580c4710f" target='_blank'>
              A New Dimension in Tumor Heterogeneity: Intratumoral Bacteria Impact Multifocal Hepatocellular Carcinoma Development.
              </a>
            </td>
          <td>
            A. Dzutsev, Giorgio Trinchieri
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Soft tissue sarcomas (STSs) are rare cancers that respond poorly to current treatments. As the immune system is increasingly recognized as a key player in cancer control, we investigate a specific type of immune cell—natural killer (NK) cells. We analyzed NK cells from blood samples of STS patients and healthy individuals. NK cells from STS patients were less functional, showing reduced degranulation and lower production of IFNγ, both essential for killing cancer cells. We also found alterations in key surface markers, which may explain their impaired function. Among these, CD27 and NKp44 were notably different between patients and healthy individuals and showed strong potential to distinguish between these groups. Higher CD27 levels were associated with NK cells that are less effective at killing cancer cells. Overall, our findings suggest that NK cells from STS patients may have a reduced capacity to fight cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb47c660c210aea5cd2c30473df6b270829f44" target='_blank'>
              Comprehensive Receptor Repertoire and Functional Analysis of Peripheral NK Cells in Soft Tissue Sarcoma Patients
              </a>
            </td>
          <td>
            L. M. Sousa, J. Almeida, Tânia Fortes-Andrade, Patrícia Couceiro, Joana Rodrigues, Rúben Fonseca, M. Santos-Rosa, Paulo Freitas-Tavares, J. Casanova, Paulo Rodrigues-Santos
          </td>
          <td>2025-07-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is a frequently observed phenomenon in aging individuals without apparent illness and exhibits an increased prevalence in cancer patients. Mechanistic studies indicate that mutant immune cells alter the tumor microenvironment, leading to increased inflammation, blood vessel formation, and immune cell exhaustion. Paradoxically, these changes also preserve stem-like T-cell pools that can be utilized by immunotherapy. CH may be incidentally detected in patients whose solid tumors are profiled by next-generation sequencing. Clinically, CH confers higher risks of therapy-related myeloid neoplasms, cardiovascular and inflammatory toxicities, and context-specific changes in treatment efficacy. Moreover, tumorinfiltrating CH independently shortens survival. Two validated risk scores can inform the risk for myeloid malignancy, yet surveillance, cardiometabolic management, and regimen selection still primarily rely on expert consensus. Because CH may be discovered incidentally, rigorous confirmation of variant origin when CH is suspected is essential to avoid misdirected therapy. We propose a pragmatic approach: confirm CH with paired blood sequencing when feasible; integrate high-risk features into risk stratification, counseling, and monitoring for cytopenias and cardiovascular events; and prefer less genotoxic regimens when the oncologic benefit is comparable. Early trials blocking interleukin-1β suggest that targeting inflammation driven by CH may improve outcomes in patients with solid tumors. Prospective studies informed by mutation analysis and tracking clonal changes and inflammatory markers are needed to determine if routine CH assessment can be integrated into precision oncology to improve outcomes for patients with solid tumors and CH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a6d6136853e6a95a267ec620d529e97551fcc6" target='_blank'>
              What's hidden in plain sight? Impact of clonal hematopoiesis on the risk and progression of non-hematologic cancers.
              </a>
            </td>
          <td>
            José C Martínez, Catherine C Coombs
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer has remained one of the leading causes of death worldwide throughout history despite significant advancements in drug development, radiation therapy, and surgery. Traditional chemotherapeutic small molecules are often hindered by narrow therapeutic indices and limited specificity, leading to suboptimal clinical outcomes. On the other hand, more advanced approaches, such as antibody–drug conjugates (ADCs), frequently encounter obstacles, including poor tumor penetration and prohibitive production costs. The tumor-forming and metastatic capacity of cancer further challenges currently available cancer therapies by creating a biochemical milieu known as the tumor microenvironment (TME). Although solid tumor development presents significant obstacles, it also opens new avenues for innovative therapeutic approaches. It is well-documented that as tumors grow beyond 1–2 mm3 in size, they undergo profound changes in their microenvironment, including alterations in oxygen levels, pH, enzymatic activity, surface antigen expression, and the cellular composition of the stroma. These changes create unique opportunities that can be exploited to develop novel and innovative therapeutics. Currently, numerous ADCs, small-molecule–drug conjugates (SMDCs), and prodrugs are being developed to target specific aspects of these microenvironmental changes. In this review, we explore five TME parameters in detail, with a focus on their relevance to specific cancer types, phenotypic identifiers, and preferred methods of therapeutic targeting. Additionally, we examine the chemical moieties available to target these changes, providing a framework for design strategies that exploit the dynamics of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fd1f0a88cf8bb961e52872f9849da348c6d9b7" target='_blank'>
              Leveraging the Tumor Microenvironment as a Target for Cancer Therapeutics: A Review of Emerging Opportunities
              </a>
            </td>
          <td>
            Hakan Guven, Zoltán Székely
          </td>
          <td>2025-07-29</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Medulloblastoma is the most common malignant brain tumor in children and comprises four molecular subtypes—WNT, SHH, Group 3, and Group 4—each with distinct genetic, epigenetic, and metabolic features. Increasing evidence highlights the critical role of metabolic reprogramming and epigenetic alterations in driving tumor progression, therapy resistance, and clinical outcomes. This review aims to explore the interplay between metabolic and epigenetic mechanisms in medulloblastoma, with a focus on their functional roles and therapeutic implications. Methods: A comprehensive literature review was conducted using PubMed and relevant databases, focusing on recent studies examining metabolic pathways and epigenetic regulation in medulloblastoma subtypes. Particular attention was given to experimental findings from in vitro and in vivo models, as well as emerging preclinical therapeutic strategies targeting these pathways. Results: Medulloblastoma exhibits metabolic adaptations such as increased glycolysis, lipid biosynthesis, and altered amino acid metabolism. These changes support rapid cell proliferation and interact with the tumor microenvironment. Concurrently, epigenetic mechanisms—including DNA methylation, histone modification, chromatin remodeling, and non-coding RNA regulation—contribute to tumor aggressiveness and treatment resistance. Notably, metabolic intermediates often serve as cofactors for epigenetic enzymes, creating feedback loops that reinforce oncogenic states. Preclinical studies suggest that targeting metabolic vulnerabilities or epigenetic regulators—and particularly their combination—can suppress tumor growth and overcome resistance mechanisms. Conclusions: The metabolic–epigenetic crosstalk in medulloblastoma represents a promising area for therapeutic innovation. Understanding subtype-specific dependencies and integrating biomarkers for patient stratification could facilitate the development of precision medicine approaches that improve outcomes and reduce long-term treatment-related toxicity in pediatric patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733445d34f0ee82899b45eeeaeabd1e9ff5c7ba8" target='_blank'>
              Deciphering Medulloblastoma: Epigenetic and Metabolic Changes Driving Tumorigenesis and Treatment Outcomes
              </a>
            </td>
          <td>
            Jenny Bonifacio-Mundaca, Sandro Casavilca-Zambrano, Christophe Desterke, Íñigo Casafont, Jorge Mata-Garrido
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Variations in DNA methylation within the DNA damage response (DDR) mechanism could have significant implications for glioma prognosis and immune responses. This study aimed to explore the global DNA methylation landscape of DDR genes in gliomas and identify key epigenetically regulated genes influencing glioma biology and immunity. Methods This study incorporated a range of public and local glioma datasets. Multiple clinical, bioinformatic, and in vitro experimental analyses were conducted to explore clinical and biological aspects. Results Global DNA methylation variations in DDR genes correlated with distinct glioma prognoses, with five CpGs identified as potent predictors. Hierarchical clustering and a risk-score model based on these CpGs unveiled immune-related prognostic subgroups in glioblastomas (GBMs) and lower-grade gliomas (LGGs). NSUN5, epigenetically regulated by one of these CpGs, highlighted the biological significance of the DDR CpG panel. In vitro, NSUN5 displayed tumor-suppressor-like activities in GBM cells, but clinically, it was an unfavorable prognostic marker. Depletion of NSUN5 shifted cytosolic DNA sensing from a STING-dependent (cGAS-STING) pathway to a STING-independent (DNA-PK-HSPA8) pathway, leading to a delayed but more robust type I interferon (IFN) response in GBM cells and enhancing microglial M1 polarization and chemotaxis. This may partially account for the functional discrepancy of NSUN5 observed between experimental and clinical contexts. Conclusion This study highlights the complex interplay between DNA methylation, the DDR mechanism, cytosolic DNA sensing, and glioma immunity. These findings may inspire novel strategies for DNA sensing-based immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06360-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54757ac8e99a0979bc781abac88c0c6433dc38" target='_blank'>
              DNA methylation variations of DNA damage response in glioblastoma: NSUN5 modulates tumor-intrinsic cytosolic DNA-sensing and microglial behavior
              </a>
            </td>
          <td>
            An-An Yin, Yan-Qiu Yao, Yi-Feng Liu, Yu-Sha Ji, Yalong He, Tian-Chi Ma, Wen-Heng Guo, A. Etcheverry, M. Aubry, Jean Mosser, Wei Lin, Yuan-Ming Wu, Kun Chen
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Accumulating evidence suggests that the TP53 mutation, intratumoral microbiome, and tumour microenvironment (TME) are closely linked to tumourigenesis, yet the biological mechanisms underlying these connections remain unclear. To explore this, we collected multi‐omics data—including genome, transcriptome, and tumour microbiome data—from a wide range of cancer types in The Cancer Genome Atlas (TCGA). Through a pan‐cancer analysis, we identified significant correlations between intratumoral microbiota diversity and TP53 mutation status, particularly in hepatocellular carcinoma (HCC) and endometrial cancer (EC). Despite notable differences in microbiota composition between these two cancer types, we consistently observed that TP53 mutations were associated with reduced alpha‐diversity. Additionally, we found that TP53 mutation status significantly influenced stromal components within the TME, such as a strong correlation between decreased endothelial cell abundance and TP53 mutation. Our integrated approach reveals the complex interplay between TP53 and factors regulating the host TME, offering new insights into cancer progression and potential therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db906f017cde5061fad12eabf32c649d299b5278" target='_blank'>
              Pan‐Cancer Integrative Analyses Reveal the Crosstalk Between the Intratumoral Microbiome, TP53 Mutation and Tumour Microenvironment
              </a>
            </td>
          <td>
            Baoling Wang, Bo Zhang, Chun Li
          </td>
          <td>2025-01-01</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) treatment remains challenging due to genetic heterogeneity and resistance mechanisms. To address this, we developed a drug discovery pipeline using patient-derived primary CRC cultures with diverse genomic profiles. These cultures closely resemble certain molecular characteristics of primary and metastatic CRC, highlighting their promise as a translational platform for therapeutic evaluation. Importantly, our engineered model and patient-derived cells reflect the complexity and heterogeneity of primary tumors, not observed with standard immortalized cell lines, offering a more clinically relevant system, although further validation is needed. High-throughput screening (HTS) of 4255 compounds identified 33 with selective efficacy against CRC cells, sparing normal, healthy epithelial cells. Among the tested combinations, everolimus (mTOR inhibitor) and uprosertib (AKT inhibitor) demonstrated promising synergy at clinically relevant concentrations, with favorable therapeutic windows confirmed across tested patient-derived cultures. Notably, this synergy, revealed through advanced models, might have been overlooked in traditional immortalized cell lines, highlighting the translational advantage of patient-derived systems. Furthermore, the integration of machine learning into the HTS pipeline significantly improved scalability, cost-efficiency, and predictive accuracy. Our findings underscore the potential of patient-derived materials combined with machine learning-enhanced drug discovery to advance personalized therapies. Specifically, mTOR-AKT inhibition emerges as a promising strategy for CRC treatment, paving the way for more effective and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b44bc31581c51d6eaf637f51510217abbfb946c" target='_blank'>
              Integrated transcriptomic and functional modeling reveals AKT and mTOR synergy in colorectal cancer
              </a>
            </td>
          <td>
            M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Szklarczyk, Olga Dracz, A. Kurzejamski, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Karolina Pyziak, Joanna Drozdowska, Krzysztof Baczyński, Konrad Wojtowicz, Maurycy Chronowski, Krzysztof Rataj, Magdalena Otrocka, M. Mikula, Paula Feliksiak, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka, Andrzej Mazan
          </td>
          <td>2025-07-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. B. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26a0bcbf54938914a93ab50e2ef8aa94cb7790d" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals the functions of TREM2high macrophages and infarct border dynamics post-myocardial infarction.
              </a>
            </td>
          <td>
            Tao Xiong, Yan Chen, Chang Liu, Yaxiong Li, Yayong Zhang, Qing Chang
          </td>
          <td>2025-08-01</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3685ee163d6c268edbf6962e0dbfe3fe91dd59" target='_blank'>
              Tumor microenvironment differences between diagnostic and relapsed classic Hodgkin lymphoma revealed by scRNA-seq.
              </a>
            </td>
          <td>
            Yifan Yin, Shinya Rai, A. Jiang, Alexander M Xu, Luke O'Brien, A. Telenius, J. Delabie, L. Chong, S. Hung, Akil A Merchant, David W Scott, K. Savage, Tomohiro Aoki, Christian Steidl
          </td>
          <td>2025-09-09</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Immunotherapy has progressively gained prominence as a cornerstone therapeutic modality across diverse oncological contexts, with its clinical efficacy intricately linked to the dynamic interactions between the tumor microenvironment (TME) and neoplastic cells. Central to this paradigm is angiogenesis—a quintessential hallmark of cancer—which not only sustains tumor growth but also orchestrates immunomodulatory networks within the TME, thereby profoundly influencing therapeutic responsiveness. However, in the field of bladder cancer (BC), the relationship among angiogenesis and prognosis, immunotherapy response, and immune cell infiltration remains to be further explored. Methods To systematically uncover this relationship, we carried out an exhaustive assessment of 36 genes linked to angiogenesis (AAGs) and explored the relationship between angiogenesis and transcript, prognostic outcomes, as well as the infiltration of immune cells. By constructing an AAG_score, we quantified the angiogenic subtype characteristics of each patient. Subsequently, we evaluated the value of these characteristics in foreseeing BC prognosis and the response of treatment, and concurrently analyzed the performance of AAGs in diffuse large B-cell lymphoma for comparative study. Through RT-qPCR, CCK8 and other experiments, we verified the role of NID2 in bladder cancer. Results This study explored different types of AAGs mutations in BC samples at the genetic level and elucidated the expression patterns of AAGs. Through in-depth analysis, we identified two distinct molecular subpopulations and found significant associations between AAG mutations and patients’ clinicopathological features, prognosis, and invasive TME. We found that patients with low AAG_score showed increased microsatellite instability, high mutation tendency, and immune motivation, and had a better prognosis. NID2 plays a role in promoting proliferation in bladder cancer and evaluate in diffuse large B-cell lymphoma. In addition, our study found a significant correlation between index of cancer stem cell and AAG_score in drug sensitivity. Conclusion In summary, our study successfully identified prognosis-related AAG characteristics in BC patients. These characteristics not only contribute to a clearer understanding of TME properties but also provide an important basis for exploring more effective immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e0c3a38e058744faab2e7b9e1bb3c40f92cc4d" target='_blank'>
              Angiogenesis-related gene NID2 profiling and immune infiltration in bladder cancer: prognostic implications and immunotherapy response
              </a>
            </td>
          <td>
            Jinxia Cao, Wang Zheng
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background: Anoikis is a form of programmed cell death triggered by the detachment of cells from the extracellular matrix. Anoikis resistance represents a critical factor in tumor metastasis, and elucidating the mechanisms by which epithelial cancer cells evade this process may provide a molecular insight for effectively targeting metastatic progression. Methods: Presented here are an experimental workflow and a detailed protocol to examine anoikis sensitivity in tumor cells both in vitro and in vivo. We described a detachment-induced anoikis model, a three-dimensional spheroid culture system, and an in vivo circulating tumor cell assay, by using the human lung carcinoma cell line A549 as a model system. We detailed the cell culture conditions, materials, and sample preparation, and the evaluation and quantification of anoikis. Together, these methods provide a comprehensive approach for investigating anoikis resistance. Conclusion: This protocol offers valuable insights into the mechanisms underlying anoikis resistance and may facilitate the identification of novel therapeutic targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44845c475c780a2f717aad7ae39727b62d7d7bbc" target='_blank'>
              An experimental workflow for investigating anoikis resistance in cancer metastasis
              </a>
            </td>
          <td>
            Xue Han, Yipan Zheng, Xiaohui Si, Zhe Liu
          </td>
          <td>2025-08-05</td>
          <td>Journal of Biological Methods</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite advances in medicine, cancer remains one of the foremost global health concerns. Conventional treatments like surgery, radiotherapy, and chemotherapy have advanced with the emergence of targeted and immunotherapy approaches. However, therapeutic resistance and relapse remain major barriers to long-term success in cancer treatment, often driven by cancer stem cells (CSCs). These rare, resilient cells can survive therapy and drive tumour regrowth, urging deeper investigation into the mechanisms underlying their persistence. CSCs express ion channels typical of excitable tissues, which, beyond electrophysiology, critically regulate CSC fate. However, the underlying regulatory mechanisms of these channels in CSCs remain largely unexplored and poorly understood. Nevertheless, the therapeutic potential of targeting CSC ion channels is immense, as it offers a powerful strategy to disrupt vital signalling pathways involved in numerous pathological conditions. In this review, we explore the diverse repertoire of ion channels expressed in CSCs and highlight recent mechanistic insights into how these channels modulate CSC behaviours, dynamics, and functions. We present a concise overview of ion channel-mediated CSC regulation, emphasizing their potential as novel diagnostic markers and therapeutic targets, and identifying key areas for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74bc88d91ef29f0fa1f654c524e84092e0b4e373" target='_blank'>
              Deciphering the Role of Functional Ion Channels in Cancer Stem Cells (CSCs) and Their Therapeutic Implications
              </a>
            </td>
          <td>
            Krishna Samanta, Gali Sri Venkata Sai Rishma Reddy, Neeraj Kumar Sharma, Pulak Kar
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Endometrial cancer is the most common gynecological cancer in developed countries, but predicting how it will progress remains challenging. This study focused on a gene called HOXA5, which helps control cell growth and development. We examined tissue samples from 75 patients with endometrial cancer and found that high HOXA5 expression was linked to increased tumor cell growth, but at the same time it was associated with better overall survival. Interestingly, patients with high HOXA5 levels had lower amounts of markers related to blood vessel formation and tissue invasion, which may slow down cancer’s spread. Our findings suggest that HOXA5 may act like a double-edged sword—encouraging cancer cell growth while limiting their ability to spread. This means HOXA5 could be used as a helpful marker to predict patient outcomes and might even be a future target for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26173d0e2faabb8df402157a208bcd6d914752f" target='_blank'>
              HOXA5 as a Dual Modulator of Tumor Biology in Endometrial Cancer
              </a>
            </td>
          <td>
            Yi-Kai Fu, Ching-Yu Shih, Chiao-Yin Cheng, Hua Ho, Yen-Lin Chen
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung adenocarcinoma has become a paradigm for precision oncology and success with targeted therapies. However, large chunks of the pie chart of molecular drivers remain unknown, preventing many patients from similarly benefiting from this strategy. Here, we highlight the side-by-side release of two impactful studies from Mozzarelli and colleagues and DiMarco and colleagues in this issue of Cancer Research. Here, each team generated unique mouse models characterizing a novel and potentially actionable driver and facilitator of acquired drug resistance in lung adenocarcinoma, RIT1M90I. Both study teams describe and characterize their new mouse models, reporting the tumor latency and importantly revealing the sensitivity of RIT1-mutant lung tumors to SHP2 or RAS tricomplex inhibitors. Collectively, these findings highlight the importance of RIT1M90I as an actionable driver of lung adenocarcinoma and a potent mediator of acquired drug resistance. These new RIT1-driven lung cancer models represent valuable tools for the preclinical testing and translation of different therapeutic strategies for this patient population. See related article by Mozzarelli et al., p. 3196 See related article by DiMarco et al., p. 3207.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/802429b265b154e6d63054d90ce37155bed940f9" target='_blank'>
              Another Slice of the Pie Uncovered: RIT1M90I Is an Actionable Driver in Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Cheng Pei Wu, A. Vaishnavi
          </td>
          <td>2025-09-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Malate dehydrogenase 1 (MDH1), an NAD(H)-dependent isoenzyme, is a key component of the malate-aspartate shuttle (MAS). A significant association has been observed between MDH1 expression and various characteristics of the tumor microenvironment across different cancer types. Methods This study provides comprehensive pan-cancer analyses exploring the expression patterns, clinical and pathological correlations, genetic alterations, immunogenomic profiles, single-cell dynamics, alternative splicing signatures, and pharmacological sensitivities related to MDH1. Drug sensitivity profiling and molecular docking techniques have been employed to identify potential anti-cancer compounds targeting MDH1. Experiments have also been conducted to investigate the biological function of MDH1 in lung adenocarcinoma (LUAD) and to confirm the interaction between MDH1 and macrophages using immunofluorescence assays. Results MDH1 expression levels are elevated across a wide range of malignancies, and overexpression of MDH1 was consistently linked to poor prognosis in multiple cancer subtypes. Moreover, MDH1 expression shows complex correlations with various immune cell populations, particularly macrophages, and cohort analysis of both bulk and pan-cancer single-cell immunotherapy data suggest that MDH1 could serve as a predictive marker for immunotherapy responses. Moreover, knockdown of MDH1 suppresses macrophage invasion. To investigate the role of MDH1 in LUAD cells, a potential inhibitor of MDH1 was identified, BI-2536, and has been confirmed to impact MDH1 activity and impede the growth of LUAD cells. Conclusion Our findings indicate that MDH1 may serve as a potential prognostic marker and a promising target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3176f0dff692eaebf853d9454461d636ffecd8e7" target='_blank'>
              Panoramic view of MDH1: driving cancer progression and shaping the tumor immune microenvironment
              </a>
            </td>
          <td>
            Yunchen Lou, Yunwei Lou, Yao Cheng, Beining Xu, Hanbin Chen, Yinwei Dai
          </td>
          <td>2025-08-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis remains a leading cause of morbidity and mortality in patients diagnosed with cancer. A variety of in vitro and in vivo approaches have been employed to study the individual steps of the metastatic cascade. However, these methodologies are sometimes limited in their ability to recapitulate the biological complexity and heterogeneity of human tumor biology. As a result, significant knowledge gaps still exist regarding the development, growth and evolution of treatment resistance in metastatic tumors. In this Perspective, we discuss the benefits and drawbacks of current, widely used techniques to model metastatic disease. We also highlight novel approaches utilized in recent studies to confront the limitations posed by classic modeling techniques. Ultimately, we provide suggestions for ensuring scientific rigor and reproducibility in metastasis studies, and we propose key areas of focus for developing next-generation models of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10d2e8655ad96014a93526cd27ef8255240f4d2b" target='_blank'>
              Modeling metastasis - leveraging novel tools to streamline discovery in advanced cancer.
              </a>
            </td>
          <td>
            Nicole M Eskow, Eva Hernando
          </td>
          <td>2025-08-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor stroma can be actively shaped by tumor cells to adopt pro-tumorigenic properties, making it a focal point of cancer research. Among the stromal components, cancer-associated fibroblasts (CAFs) -- often the most abundant cell type in the tumor microenvironment (TME) -- play a pivotal role in promoting cancer progression through their interactions with tumor cells and other stromal components. Elucidating the mechanisms behind these interactions requires robust methods for isolating and characterizing CAFs, which can also lay the foundation for developing novel CAF-targeted cancer therapies. This study presents a method for isolating fibroblasts from both the tumor and the adjacent normal regions of radical prostatectomy specimens obtained from prostate cancer (PCa) patients. This approach enables the concurrent isolation of CAFs and their normal counterparts (NFs) from the same individual, providing a paired experimental system. Detailed protocols are provided for the maintenance of primary CAFs and NFs for downstream in vitro analyses, as well as for their immortalization to facilitate long-term studies. We also describe functional assays designed to compare the effects of CAFs and NFs on cancer cell behaviors, including proliferation, migration, and anchorage-independent growth. Two complementary approaches are detailed: treatment of cancer cells with fibroblast-derived conditioned media (CM), and direct co-culture of fibroblasts with tumor cells. Together, these methodologies represent a comprehensive toolkit for investigating the dynamic interplay between CAFs and tumor cells, not only in PCa but potentially across a range of human cancers. Moreover, molecular characterization of these interactions may reveal key mediators of CAF-driven oncogenesis, offering promising targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2d8dc58b820e48882f34f8df6b8e5bd76919a2" target='_blank'>
              Isolation, Culture, and Characterization of Prostate Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Somayeh Mirzaaghaei, L. Avalle, Chiara Verrengia, Jahnavi Srivatsa, Laura Conti, Marta Gai, Chiara Fiameni, Paolo Gontero, Umberto Mortara, Luca Molinaro, Mauro Papotti, Valeria Poli
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Abstract Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo– and targeted therapy resistant state even after long term culture ex vivo . These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo– and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background To explore the differences in tumor immune microenvironment between different KRAS mutation subtypes, and to provide a new direction for clinical immunotherapy of different subtypes of colorectal cancer. Methods We examined the spatial distribution of common inflammatory cell markers of CD8 + T cells, CD4 + T cells, natural killer cells, and B cells and macrophages in 55 colorectal cancer patients with different KRAS gene phenotypes using immunohistochemistry and panoramic scanning. We analyzed the relationship between inflammatory cells and clinical information. TCGA and String online databases were used to analyze the relationship between KRAS mutation subtypes and inflammatory cell markers. Results We observed that significant differences in the spatial distribution of the tumor invasion front and the immune cell infiltration within the tumor. Colorectal cancer patients with different KRAS mutations showed different immune infiltration, and the main cells with differences were FOXP3 regulatory T cells and M2 macrophages. Furthermore, these two cell types were strongly associated with the prognosis of KRAS wild-type and KRAS mutant colorectal cancer. Conclusions The complexity of the immune microenvironment cannot be explained by infiltration of specific cell types alone, but may arise from cell-to-cell interactions or changes in the proportion of different immune cells. However, the infiltration of FOXP3 cells and M2 macrophages probably accounts for the differences between KRAS mutant subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03be9a5c418e895db04c9d1a928c80c7d3a288ec" target='_blank'>
              Analysis of the immune microenvironment in colorectal cancer with different KRAS gene subtypes
              </a>
            </td>
          <td>
            Jing Huang, Qian Gong, Qingshu Li, M. Xiao, Ming Li, Shuxian Zhang, Yalan Wang, Yi Tang
          </td>
          <td>2025-08-04</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Discovery of cancer driver pathways is essential for targeted therapies, since these pathways govern tumor progression and treatment resistance. However, their context-specific patterns across populations remain poorly understood. Leveraging pan-cancer genomic data, we apply our two models, EntCDP and ModSDP, to perform stratified analyses from four perspectives: region, tumor type, age group, and risk factors. Our results reveal the regional biases in perturbed pathways, such as PI3K-Akt in Chinese patients and GPCR in American patients with bladder cancer. Subtype comparisons highlight the mTOR signaling in lung adenocarcinoma and the FoxO signaling in lung squamous cell carcinoma. Pediatric-adult comparisons emphasize the enrichment of Ras signaling in pediatric acute myeloid leukemia and PAK signaling in pediatric glioblastoma, respectively. Risk factor associations further link Notch-mediated pathways to alcohol consumption and CDKN-regulated pathways to obesity-related cancers. Our findings demonstrate the utility of stratified driver pathway analysis in uncovering common and specific mechanisms, which can help prioritize context-aware therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e088118e277d04d3b9c6fe00f85dd703f5be1ff1" target='_blank'>
              Multi-context modeling of driver pathways reveals common and specific mechanisms across 23 cancer types
              </a>
            </td>
          <td>
            Wenjia Zhou, Junhua Zhang
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a unique subtype of breast cancer characterized by high invasiveness, high metastasis rates, and poor prognosis, making it an important focus within global malignancies. Due to the absence of estrogen receptor, progesterone receptor, and HER2 expression, TNBC presents significant challenges in treatment. Metastatic progression markedly increases treatment complexity, drastically reducing patient survival rates. The metastatic and drug resistance processes of TNBC involve complex, multi-step biological mechanisms regulated through various molecular mechanisms and signaling pathways within and outside tumor cells. In recent years, immunotherapy has brought new hope for TNBC. Compared to other breast cancer subtypes, TNBC demonstrates higher immunogenicity, often accumulating a higher mutational burden that generates more neoantigens, thus typically resulting in a tumor microenvironment (TME) enriched with tumor-infiltrating lymphocytes (TILs). Additionally, PD-L1 expression is significantly higher in TNBC compared to other subtypes, closely correlating with TIL abundance. These characteristics position TNBC as a strong candidate for immune checkpoint inhibitor (ICI) therapy. Clinical trials have demonstrated promising efficacy of ICIs in TNBC, overturning previous beliefs that breast cancer is generally insensitive to immunotherapy. This review summarizes recent advances regarding resistance types, molecular mechanisms, associated genes and pathways, the role of the tumor microenvironment, and clinical strategies related to immunotherapy resistance in the neoadjuvant setting of TNBC, aiming to provide insights and guidance for future research exploration and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03cacb55534d35ef52652d44531e93baccfb5003" target='_blank'>
              Immunotherapy resistance in triple-negative breast cancer: Molecular mechanisms, tumor microenvironment, and therapeutic implications
              </a>
            </td>
          <td>
            Zijian Zhou, Qin Zhou
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are malignant cells that detach from primary or metastatic tumors and enter the bloodstream. Organoids, as three‐dimensional in vitro models, can mimic the tumor microenvironment and histopathological characteristics, thereby serving as valuable tools in tumor research. CTC‐derived organoids retain tumor heterogeneity and metastatic potential, which provides a unique model for the study of metastatic cascade mechanisms, individualized drug screening, and precision therapy. However, the current research on CTC‐derived organoids faces challenges, such as the scarcity of CTCs and the high technical difficulty in their isolation and enrichment, which leads to a low success rate in constructing organoid models. Moreover, most existing studies focus on a single cancer type and lack systematic integration of full‐process standardization as well as cross‐cancer applicability. In this paper, we review the isolation and enrichment strategies of CTC‐derived organoids along with the techniques for optimizing in vitro culture systems, and discusses their potential applications. This review summarizes the existing results, analyzes the technical bottlenecks, and provides a theoretical basis for the standardized construction and application of CTC‐derived organoids, while promoting their application in tumor precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971e10876cd9eda4f902688d2d69a778549269d1" target='_blank'>
              Circulating Tumor Cell‐Derived Organoids: Current Progress, Applications, and Future
              </a>
            </td>
          <td>
            Tiantian Li, Biao Deng, Shenglong Li, Yanxia Wu, Zhenghao Lu, Zhu Liang
          </td>
          <td>2025-09-01</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Knowledge of tumor cell dynamics can inform prognosis and treatment yet is largely lacking for lung adenocarcinoma in people who have never smoked (NS-LUAD). With RNA-seq data from 684 NS-LUAD and validation in an independent dataset, we identified three subtypes with distinct phenotypic traits and cell compositions. Additional genomic and histological data further characterized the subtypes. ‘Steady’ , marked by low proliferation, high alveolar cell fraction, moderate-to-well differentiation, and fewer driver genes’ alterations, is linked to prolonged survival and low immune evasion. ‘Proliferative ’ shows high proliferation markers, TP53 mutations, and gene fusions. ‘Chaotic’ , with high epithelial-to-mesenchymal transition markers, has the worst prognosis even within stage I tumors. Lacking known molecular or histological characteristics, this aggressive subtype is solely identified by transcriptomic data. A 60-gene signature recapitulates the overall classification and strongly predicts survival even within subgroups based on tumor stage or known genomic features, emphasizing its potential for improving NS-LUAD prognostication in clinical settings. Significance The transcriptome of 684 lung adenocarcinomas in people who have never smoked (NS-LUAD) identifies three subtypes with different cellular dynamics, and genomic and morphologic features. A 60-gene signature accurately stratifies subjects for mortality risk, even in stage I, offering a clinically applicable tool for treatment decision making in NS-LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4f12fa512ef11169cdf3ed0dd46beb501ba7b1a" target='_blank'>
              A prognostic signature for lung adenocarcinoma in people who have never smoked
              </a>
            </td>
          <td>
            Wei Zhao, Tongwu Zhang, Xing Hua, P. Hoang, Mona Miraftab, Monjoy Saha, John P. McElderry, Jian Sang, Olivia W. Lee, Caleb Hartman, Azhar Khandekar, Sunandini Sharma, Frank J. Colón-Matos, Samuel Anyaso-Samuel, Difei Wang, Kristine Jones, A. Hutchinson, B. Hicks, J. Rosenbaum, Xiaoming Zhong, Yang Yang, A. Pesatori, Dario Consonni, D.C. Christiani, Kin Chung Leung, Maria Pik Wong, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Simona Ognjanovic, Milan Savic, Millica Kontic, V. Gaborieau, Paul Brennan, O. Arrieta, Y. Bossé, E. Edell, M. Schabath, Paul Hofman, Luis Mas, Sai S. Yendamuri, Chih-Yi Chen, I-Shou Chang, C. A. Hsiung, Geoffrey Liu, Jacobo Martínez Santamaría, B. G. Rothberg, Karun Mutreja, Scott M Lawrence, Nathaniel Rothman, Ludmil B. Alexandrov, Charles Leduc, M. Baine, Philippe Joubert, L. Sholl, William D. Travis, Robert J Homer, Qing Lan, Stephen J Chanock, Lixing Yang, S. Yang, Jianxin Shi, Marion Landi
          </td>
          <td>2025-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background The progression and prognosis of early‐stage lung adenocarcinoma are closely associated with histologic subtypes, yet the presence of mixed histologic patterns often complicates prognostic assessment. Currently, the correlation between molecular and histologic features remains poorly understood. Methods Formalin‐fixed paraffin‐embedded (FFPE) samples were collected from patients with primary early‐stage lung adenocarcinoma, encompassing three histologic subtypes: well‐differentiated, moderately differentiated, and poorly differentiated. The GeoMx Digital Spatial Profiler platform was utilized to obtain spatial transcriptomic profiling. Regions of interest were carefully selected and further subdivided into three categories of areas of interest, specifically epithelial cell‐enriched regions, macrophage‐enriched regions, and other regions. Multiplex immunofluorescence (mIF) assays were employed to validate the obtained results. Results Distinct molecular characteristics were identified in tumor epithelial‐ and macrophage‐enriched compartments spanning well‐differentiated to poorly differentiated tumors. In poorly differentiated tumors, we observed enrichment of pathways related to humoral immune response, complement activation regulation, and extracellular matrix receptor interaction pathways, all of which are significantly associated with poorer prognosis. We integrated these pathways to develop a composite molecular signature that strongly correlate with adverse prognosis. Conclusions Our results provide new insights into the link between molecular and histologic subtypes in mixed‐type lung adenocarcinomas. Specifically, the identified molecular signatures offer potential biomarkers for predicting disease progression and prognosis, thus facilitating more precise and personalized therapeutic approaches. Key points Poorly differentiated components in mixed‐type early‐stage lung adenocarcinoma (LUAD) are characterized by enrichment of humoral immune response, complementactivation regulation, and extracellular matrix receptor interaction pathways, which areassociated with worse prognosis. A composite molecular signature integrating the keypathways strongly correlates with adverse clinical outcomes, serving as apotential prognostic biomarker. Digital spatial transcriptomics reveals spatially resolvedmolecular heterogeneity in tumor epithelial‐ and macrophage‐enrichedcompartments, bridging the gap between histologic subtypes and molecularmechanisms in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec35bb82932a3767cb2f73edc45c10c5b4e72637" target='_blank'>
              Spatial transcriptomic analysis across histological subtypes reveals molecular heterogeneity and prognostic markers in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Hua Geng, Wenhao Zhou, Haitao Luo, Jiaqian Wang, Shixiong Li, Congcong Song, Yujie Zhao, Meilin Xu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Africa faces the highest mortality rates across eight cancer types. However, cancer studies are biased toward European populations, leading to major concerns that cancer treatments may be ineffective for African patients. Providing a systematic review of African-inclusive whole cancer genome studies, African-derived tumours reveal distinct clinically relevant drivers, molecular taxonomies, and overall increased genomic instability, highlighting challenges associated with non-African-derived computational workflows. We provide a rationale for parallelism strategies to accelerate the processing steps of those distinctly intensive data, allowing for required scalability. Advocating for further resources that capture the rich African ancestral diversity, a concerted global effort will be required to improve and ultimately standardise bioinformatic workflows, thereby enhancing health outcomes for African cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c92b99c837fa3c8e00a120d48c0e8433aaae5c6" target='_blank'>
              Scaling for African Inclusion in High-Throughput Whole Cancer Genome Bioinformatic Workflows
              </a>
            </td>
          <td>
            Jue Jiang, Georgina Samaha, C. Willet, T. Chew, Vanessa M. Hayes, Weerachai Jaratlerdsiri
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Macrophage polarization plays a pivotal role in shaping the tumor microenvironment and influencing rectal cancer progression. However, the metabolic and prognostic regulators governing this process remain largely undefined. Methods We constructed a macrophage polarization gene signature (MPGS) by integrating weighted gene co-expression network analysis (WGCNA) with multiple machine learning algorithms across two independent cohorts: 363 rectal cancer samples from GSE87211 and 177 samples from The Cancer Genome Atlas (TCGA). The prognostic performance of MPGS was evaluated across rectal and multiple other cancer types. Functional analyses, single-cell RNA sequencing, immunohistochemistry of clinical specimens, and in vitro cellular assays were employed to investigate the role of the MPGS hub gene, PYGM, in tumor biology and immune modulation. Results The MPGS exhibited robust prognostic capability and effectively predicted responses to immunotherapy and various chemotherapeutic agents. Both MPGS and its central metabolic component, PYGM, were closely linked to M2 macrophage infiltration, immunosuppressive tumor microenvironments, and poor clinical outcomes in rectal adenocarcinoma. Single-cell transcriptomic analysis revealed that malignant epithelial cells with elevated PYGM expression are metabolically active and closely interact with M2 macrophages. Clinical tissue analyses and functional assays confirmed that PYGM is upregulated in rectal cancer and promotes tumor cell proliferation, migration, and M2 macrophage polarization. Conclusions This study firstly highlights PYGM as a key metabolic and immunological regulator in rectal cancer, with significant prognostic and therapeutic implications. MPGS and PYGM may serve as novel biomarkers for risk stratification and guide personalized treatment strategies in patients with rectal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8198c80f5a26115add8dc0453b94b75563056daf" target='_blank'>
              Machine learning identifies PYGM as a macrophage polarization–linked metabolic biomarker in rectal cancer prognosis
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT The immunoregulatory functions and clinical implications of APOE+ macrophages within the tumour microenvironment of lung adenocarcinoma remain incompletely defined. In this study, single‐cell transcriptome analysis revealed distinct subsets of APOE+ macrophages, and subsequent CellChat analyses highlighted that these cells predominantly interact with other components of the tumour microenvironment via MIF‐(CD74+CXCR4) and MIF‐(CD74+CD44) signalling pathways, thereby contributing to the establishment of an immunosuppressive milieu. Integrating mutation profiles with multiple machine learning techniques, we developed an APOE+ Macrophage‐related Risk Model (ARM) through a combination of RSF and Ridge approaches, achieving the highest prognostic accuracy (C‐index) among all tested algorithms. The robustness and predictive value of the ARM model were validated across seven public cohorts and three prospective immunotherapy cohorts. Patients classified in the low‐ARM group consistently exhibited better prognoses, as demonstrated by survival analyses, ROC curves and PCA discrimination. Additionally, multiplex immunofluorescence analysis in the in‐house cohort confirmed significantly decreased infiltration of CD4+, CD8+ and CD20+ immune cells in the high‐ARM group, further supporting a more pronounced immunosuppressive microenvironment in these patients. Collectively, our findings not only clarify the critical role of APOE+ macrophages in shaping the immune landscape and prognosis of lung adenocarcinoma, but also provide a validated, practical risk model for individualised patient prognostication and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a72e3bbee10a06ad9e3538b663bbfeab95978a5" target='_blank'>
              The Critical Role of APOE+ Macrophages in the Immune Microenvironment and Prognosis of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Xiaofei Wang, Pengpeng Zhang, Wei Ye, Mingjun Du, Chenjun Huang, Jianan Zheng
          </td>
          <td>2025-07-30</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48a5df41c5ba0be3a9d26dc85ce353b479996b4" target='_blank'>
              A patient-derived orthotopic xenograft model unveils metastatic dynamics in head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yuchen Bai, Benjamin Blyth, Jarryd M Boath, Qiji Deng, Ruihong Huang, C. Darido
          </td>
          <td>2025-08-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of >1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485aaa6c6ae7c10944eff4bdd9fd536b93adfcd9" target='_blank'>
              Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution
              </a>
            </td>
          <td>
            R. Edahiro, Go Sato, Tatsuhiko Naito, Y. Shirai, R. Saiki, K. Sonehara, Yoshihiko Tomofuji, Kenichi Yamamoto, S. Namba, Noa Sasa, Genta Nagao, Qingbo S. Wang, Yugo Takahashi, Takanori Hasegawa, T. Kishikawa, Ken Suzuki, Yu-Chen Liu, D. Motooka, Ayako Takuwa, Hiromu Tanaka, S. Azekawa, H. Namkoong, R. Koike, Akinori Kimura, S. Imoto, Satoru Miyano, Takanori Kanai, K. Fukunaga, Mamoru Uemura, T. Morita, Yasuhiro Kato, H. Hirata, Y. Takeda, Y. Doki, H. Eguchi, Daisuke Okuzaki, Shuhei Sakakibara, Seishi Ogawa, Atsushi Kumanogoh, Yukinori Okada
          </td>
          <td>2025-07-28</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281dda7d54e63559872a9e12f7b32f74d4425ae4" target='_blank'>
              The tumor-associated fibroblasts regulate urothelial carcinoma progression.
              </a>
            </td>
          <td>
            Yu Xiao, Junfeng Yang, Mengjie Sun, Yongfu Li, Qinyin Liu, Jinjun Leng, Maolin Yang, Jinrui Wang, Hongju Li, Caifeng Yang, Changfen Luo, Jiahong Li, Longli Kang, Fen Huang, Yanhong Yu, Chuanmao Zhang
          </td>
          <td>2025-09-19</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The process of wound healing is regulated by complex interactions between different cell types across space and time. Through the profiling of individual cells within their complex environment, single-cell transcriptomics methods enable the investigation of cellular heterogeneity, cell communication networks, and cell-cell interactions involved in the wound healing process. However, many single-cell analysis tools are run within a computer coding environment, and their more widespread use by wound healing scientists is thwarted by the apparent lack of bioinformatics expertise. Therefore, a step-by-step workflow is presented showing how to use a graphical coding environment called RStudio to perform a basic single-cell analysis of a temporal mouse excisional skin wound healing dataset. This visual and guided protocol will enable scientists with no bioinformatics background to download a previously published wound healing dataset, perform critical quality control steps, run a standard single-cell analysis workflow including dataset visualizations and cell type annotations using Seurat, run cell subtype analyses, run module scoring analyses, run cell-cell interaction analyses using CellChat, and perform integrative analyses of multiple datasets using Seurat. Narrative explanations are provided for each step in the protocol and graphical results from every line of code are presented to safely guide the user through the workflow. The goal of this visual introduction to a single-cell analysis pipeline is to enable more wound healing scientists to use bioinformatics tools directly in their own laboratories in order to facilitate deeper analyses of their own single-cell datasets as well as more widespread re-analyses of previously published single-cell datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aaa9ce25d59a7c9eba1c49987e0404e67db6a5" target='_blank'>
              Using R, Seurat, and CellChat to Analyze a Single-Cell Transcriptomics Dataset of Mouse Skin Wound Healing.
              </a>
            </td>
          <td>
            Shalyn Keiser, Nichole Botello, Ethan Cruz, Mateusz S. Wietecha
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glutamine metabolic reprogramming is a hallmark of tumor progression and is highly correlated with poor clinical outcomes. The excessive uptake of glutamine by tumor cells is a key factor contributing to widespread invasion, metastasis, and immune suppression. SLC38A2, an amino acid transporter widely expressed on the surface of tumor cells, has not been thoroughly studied regarding its function and prognostic significance in tumor progression. Our objective is to employ bioinformatics methods to conduct a comprehensive and in-depth analysis of SLC38A2 across various cancers, aiming to elucidate its role and prognostic value in tumor biology.By comprehensively incorporating gene expression and clinical data from the TCGA tumor database, GTEx database, Human Protein Atlas, and GEO database, we analyzed the expression profile, mutations, and established prognostic models for SLC38A2 across various cancers. Additionally, we investigated the enrichment of SLC38A2 at the single-cell level in 12 types of cancer and analyzed its temporal expression patterns in different cell subgroups in breast and pancreatic cancer. We also studied the correlation between SLC38A2 and glutathione metabolism.Compared to normal tissues, SLC38A2 exhibits significant differential expression in 15 types of cancer and serves as a prognostic risk factor in BRCA (HR = 1.597, p < 0.05), LUAD (HR = 1.650, p < 0.01), MESO (HR = 2.007, p < 0.05), and PAAD (HR = 1.761, p < 0.05), while acting as a protective factor in KIRC (HR = 0.625, p < 0.05). Furthermore, SLC38A2 is positively correlated with tumor and stromal cells, negatively correlated with immune cell infiltration, and associated with immune exhaustion. In BRCA, SLC38A2 is highly expressed during early differentiation of malignant and stromal cells, and enriched in late differentiation of immune cells. Moreover, the expression of SLC38A2 shows a general positive correlation with glutathione metabolism genes in BRCA, LUAD, MESO, and PAAD, demonstrating diagnostic value.SLC38A2 shows widespread changes in expression patterns within tumor tissues, making it an effective diagnostic and prognostic biomarker. It is enriched in malignant cells and tumor-infiltrating stromal cells, while negatively correlated with the infiltration of many cells involved in anti-tumor immunity. Targeting SLC38A2 presents a viable therapeutic strategy by inhibiting glutamine competition and relieving immune suppression in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cdb69c6df340eda3f029c78020f687d6995d39" target='_blank'>
              Panoramic analysis of the biological function and clinical value of SLC38A2 in human cancers: a study based on pan-cancer and single-cell analysis
              </a>
            </td>
          <td>
            Mao Liao, Yuqing Rao, Molan Li, Jiayang Guo, Kun Guo, Kaiyue Li, Rui Zheng, Yifan Liu, Qianyi Wang, Manni Wang, Duo Chen, Meng Zhang, Yongfeng Wang, Yanzong Zhao, Sheng Li
          </td>
          <td>2025-09-17</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent and malignant tumor of the digestive system, characterized by high incidence and mortality rates. This study aimed to investigate the heterogeneity of the tumor microenvironment (TME) and the involvement of immune cells in CRC. Single-cell RNA sequencing (scRNA-seq) data obtained from the Gene Expression Omnibus (GEO) database were used to analyze and identify six major cell types across normal, core, and border tumor samples. A total of 27414 cells from various regions of patients with CRC were selected for subsequent analyses. Cellular interaction analysis revealed that differential signaling pathways between the TME and normal tissues, with several pathways involving interactions between myeloid cells and epithelial cells. Myeloid cells were extracted and classified into six subtypes based on markers identified in the literature. Monocle3 revealed the trajectory of tumor-associated macrophages (TAMs) and identified genes associated with pseudotime. Single-Cell ENrichment analysis for Interpreting Cellular Heterogeneity (SCENIC) analysis identified specific regulons and target genes associated with TAMs. This study reanalyzed single-cell RNA-sequencing data and provided insights into the heterogeneity of the TME, particularly in relation to the role of TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e37ff9c0af32829265ad92932c051ab5305d1fd" target='_blank'>
              Single-cell RNA sequencing analysis reveals the heterogeneity and effect of TAMs in colorectal cancer.
              </a>
            </td>
          <td>
            Chengang Wang, Ying Qian, Mingsheng Zhang
          </td>
          <td>2025-08-20</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Patient-derived xenografts (PDXs) are frequently used as preclinical models, but their recapitulation of tumour metabolism in patients has not been closely examined. We developed a parallel workflow to analyse [U-13C]glucose tracing and metabolomics data from patient melanomas and matched PDXs. Melanomas from patients have substantial TCA cycle labelling, similar to levels in human brain tumours. Although levels of TCA cycle labelling in PDXs were similar to those in the original patient tumours, PDXs had higher labelling in glycolytic metabolites. Through metabolomics, we observed consistent alterations of 100 metabolites among PDXs and patient tumours that reflected species-specific differences in diet, host physiology and microbiota. Despite these differences, most of nearly 200 PDXs retained a ‘metabolic fingerprint’ largely durable over six passages and often traceable back to the patient tumour of origin. This study identifies both high- and low-fidelity metabolites in the PDX model system, providing a resource for cancer metabolism researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b72e5e72726ae899a2f43f50766abe84e03ec90" target='_blank'>
              Conservation and divergence of metabolic phenotypes between patient tumours and matched xenografts
              </a>
            </td>
          <td>
            A. Rao, Ling Cai, Marelize Snyman, R. E. Walsdorf, Xiangyi Li, Sophia N Wix, Gabrielle Gard, Ariel B Brown, Juliana H. Kim, Joao Santos Patricio, Sarah Muh, Misty Martin Sandoval, Lauren G. Zacharias, Kristen A Heimdal, Wen Gu, Jade Homsi, Brittny Tillman, Rohit Sharma, Travis Vandergriff, Ashley Solmonson, B. Faubert, Thomas P. Mathews, Sean J. Morrison, R. DeBerardinis, Jennifer G Gill
          </td>
          <td>2025-07-29</td>
          <td>Nature Metabolism</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Brain metastases (BM) represent a signiﬁcant unmet clinical need with poor survival and limited drug thera- pies. Recent evidence suggests a subset of BM have a more locally invasive phenotype. We sought to determine whether the tumour leading edge and tumour microenvironment (TME) differed between BM of different phe- notypes using spatial transcriptomics.



 Image guided surgical sampling of the tumour margin in operated BM cases using a previously described method (“Geo-Tagging”). Tumour microarray from FFPE blocks assembled using 2mm dots in triplicate. Spatial transcriptomics analysis using Nanostring GeoMX platform and immune-oncology panel (86 markers). Mart1 (melanoma BM) or CK7 (breast and lung adenocarcinoma BM) was used to delineate the tumour, GFAP for per- itumoral brain (TME) and CD45 for immune compartment. Annotation of tissue with clinical data for overall survival, local recurrence, primary cancer and assessment of BM leading edge using established criteria – dif- fuse or border.



 22 cases were analyzed in triplicate, 5 breast cancer, 8 metastatic melanoma and 9 lung adenocarcinoma. Data was analysed using established bioinformatics pipelines in R. As expected there were signiﬁcant differences in gene expression between all BM of different primary cancers but not in the portion of T cells, macrophages and NK cells. PDL1 was upregulated in the tumour compartment and HLA-E downregulated in the TME com- partment of BM without a clear border. Locally recurrent BM signiﬁcantly overexpressed a number of genes previously established as important in BM, including for adhesion (ICAM1, EPCAM), microglia-tumour interac- tion (STAT3) and local invasion (CCND1, VEGFA).



 BM have different phenotypes that are dependent on both tumour and local microenvironmental factors with the future potential for therapeutic targeting to modify tumour behaviour and reduce local invasion and recur- rence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6992d8a9fcf5e2f41d2333a8c8118644a30147" target='_blank'>
              INVESTIGATING THE TUMOUR MICROENVIRONMENT FOR BRAIN METASTASES USING SPATIAL TRANSCRIPTOMICS AND SURGICALLY GUIDED SAMPLING
              </a>
            </td>
          <td>
            R. Zakaria, Michael Jenkinson, Ian Scott, Frances Greaney Davies, Michael Chvanov, Jamie Soul
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4006665d3be664d3d5517dbddb8bef8da7462da" target='_blank'>
              High-Resolution Spatial Transcriptomics Reveals Fibroblast and Neuroimmune Microenvironments in Endometriosis Lesions.
              </a>
            </td>
          <td>
            Caroline M Haney, Elaheh Alizadeh, Meryl Sullivan, Joshua Lee, Jasmina Kuljancic, W. Flynn, Paul Robson, Brian S. White, Danielle E Luciano, Elise T. Courtois
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Polymerase epsilon mutant (POLE-mut) endometrial cancers (EC) are characterized by a near 100% disease-specific survival rate, even when treated by surgery alone. This spectacular survival, combined with the ultramutated genome and high level of neoantigens in these tumors, indicates a substantial degree of immune control in preventing disease spread and recurrence. Although these features are intriguing, the immune infiltration of POLE-mut EC has predominantly been confined to immunohistochemistry studies. Here, we used state of the art single-cell RNA and TCR sequencing to characterize the immune landscape of POLE-mutant ECs. Moreover, we uniquely analyzed patient blood samples taken two to eight years after curative treatment to assess formation of long-term immune memory in circulation. We identified specialized tumor-infiltrating myeloid subsets at different stages of maturation, an array of lymphocytes ranging from immature to cytotoxic and adaptive natural killer (NK) as well as tumor-reactive exhausted and effector T cells, contributing to a highly inflammatory anti-tumor response. Remarkably, our analysis of blood samples taken years after curative treatment uncovered the presence of tumor-reactive T cell clones that matched the primary tumor. This indicates the formation of systemic long-term memory immune responses in POLE-mut EC survivors. Our study highlights the distinctive immunogenicity of POLE-mut EC and identifies key features associated with persistent anti-tumor immunity that may contribute to prolonged, relapse-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0198b259b8d5d685a1ba64c108955b1306a25438" target='_blank'>
              Spatiotemporal immune landscape and long-term immune memory in POLE-mutant endometrial cancer at the single-cell level.
              </a>
            </td>
          <td>
            K. Brummel, M. Requesens, N. van Rooij, H. Workel, F. Eggink, A. Plat, R. Wardenaar, D. Spierings, F. Foijer, David N. Church, Joost Bart, H. Nijman, M. de Bruyn
          </td>
          <td>2025-08-29</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Liver metastasis remains a major cause of death in colorectal cancer (CRC). Interactions between tumor cells and components of the tumor microenvironment (TME) are integral to the progression of cancer cell migration and metastatic dissemination. Investigating these cellular dynamics is essential for identifying actionable therapeutic targets. Methods This study employed bioinformatics analysis, IHC, WB, and murine in vivo imaging to delineate the role of SPOCD1 in CRC. Additional in vitro co-culture systems, immunofluorescence, and RNA-seq were utilized to determine how SPOCD1 modulated epithelial-mesenchymal transition (EMT) through cancer-associated fibroblasts (CAFs). Mechanistic insight into SPOCD1-mediated transcriptional regulation of LAMA4 was obtained via dual-luciferase reporter assays, ChIP-qPCR, and Co-IP. Results SPOCD1 was found to be markedly upregulated in CRC cells and exhibited potent pro-metastatic activity in vivo. Integration of external datasets with experimental validation revealed a strong correlation between SPOCD1 expression and CAFs infiltration in the TME. Further analyses demonstrated that SPOCD1 enhanced CXCL12 expression in CAFs via upregulation of LAMA4, enabling CXCL12 to engage CXCR4 on CRC cells and activate EMT signaling, thereby driving metastasis. This signaling axis was effectively interrupted by CXCR4 inhibitors. Mechanistically, SPOCD1 promoted LAMA4 expression through DNMT1 recruitment, facilitating DNA methylation-dependent transcriptional regulation. Conclusion This study delineates a SPOCD1-centered interaction network between CRC cells and CAFs in colorectal cancer liver metastasis (CRLM), offering novel molecular candidates for therapeutic intervention in CRLM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06863-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d8174ccdaba2a7e9f4b9613ed271a53eaeae82" target='_blank'>
              Cancer cell SPOCD1 promotes colorectal cancer liver metastasis by activating the CXCL12/CXCR4 signaling pathway in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Geng Peng, Lin Zhong, Lina Luo, Yongle Ju, Yan Lu, L. Ng, D. Foo, Fu Cheng, Luji Huang, M. Ouyang
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Effective modeling of diseases in realistic environments is crucial to improve our understanding of diverse pathologies. In this aspect, organoids offer a more faithful environment than their classical two-dimensional counterparts in vitro. Similarly, syngeneic murine models also allow researchers to investigate more complete tumor-host interactions, such as with the immune system, in vivo. Here we present a complete protocol on extracting fallopian tube epithelial cells, the cell-of-origin of high-grade serous ovarian carcinomas (HGSOC), from a mouse and derive them as primary organoid cultures, as well as their in vitro culture and maintenance. Then, we describe how to use genetic engineering techniques, such as lentiviral or retroviral gene overexpression, as well as CRISPR-Cas9 gene deletion, to modify these lines and transform them into pre-cancerous tumoroids. We also report how to use them for in vitro validation, via the extraction of genetic materials and immunofluorescence staining. Finally, we indicate how to effectively inject these tumoroids in the ovarian bursa, the autochthonous site of HGSOC, for tumor initiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ffbd54e3862714f636347c0b41bf9e48752e632" target='_blank'>
              Development of Mouse-Derived Organoid Lines from Fallopian Tube Epithelial Cells for High Grade Serous Ovarian Carcinoma Modeling.
              </a>
            </td>
          <td>
            Elaine M. Hernández González, Jianmei Hou, Oscar J. Pundel
          </td>
          <td>2025-08-06</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome drives cancer metabolic reprogramming.
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diabetic kidney disease (DKD), a primary cause of end-stage renal disease, results from progressive tissue remodeling and loss of kidney function. While single-cell RNA sequencing has significantly accelerated our understanding of cellular diversity and dynamics in DKD, its lack of spatial resolution limits insights into tissue-specific dysregulation and the microenvironment. Spatial transcriptomics (ST) is an innovative technology that combines gene expression with spatial localization, offering a powerful approach to dissect the molecular mechanisms of DKD. This mini-review introduces how ST has transformed DKD research by enabling spatially resolved analysis of cell interactions and identifying localized molecular alterations in glomeruli and tubules. ST has revealed dynamic intercellular communication within the renal microenvironment, lesion-specific gene expression patterns, and immune infiltration profiles. For example, Slide-seqV2 has highlighted disease-specific cellular neighborhoods and associated signaling networks. Furthermore, ST has pinpointed key genes implicated in disease progression, such as fibrosis-related proteins and transcription factors in tubular damage. By integration of ST with computational tools such as machine learning and network-based analysis can help uncover gene regulatory mechanisms and potential therapeutic targets. However, challenges remain in limited spatial resolution, high data complexity, and computational demands. Addressing these limitations is essential for advancing precision medicine in DKD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/932d98ffb9a94d55ce2246d2f721b7f04a8d8572" target='_blank'>
              Spatial transcriptomics meets diabetic kidney disease: Illuminating the path to precision medicine
              </a>
            </td>
          <td>
            Dan-Dan Liu, Han-Yue Hu, Fei-Fei Li, Qiu-Yue Hu, Ming-Wei Liu, You-Jin Hao, Bo Li
          </td>
          <td>2025-09-15</td>
          <td>World Journal of Diabetes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Neutrophils, key cells of the innate immune system, have recently been identified as major players in tumor immunity. Recent data have shown their pro- and anti-tumor roles within the tumor microenvironment; however, it is not fully understood how neutrophils are directed to play one role or the other. To take the first step toward understanding the factors influencing neutrophil heterogeneity during tumor development, we studied the expression of main pro- and anti-tumor markers in splenic neutrophils from mice bearing tumors of different origins that cause distinct immune responses. We then applied a novel anti-tumor treatment, DNase I, and examined the resulting changes in the expression of these markers. The results of our study enhance the understanding of the role of neutrophils in fighting tumors and suggest novel anti-tumor therapies focused on neutrophil function modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50aa1e8a5a69785659344190f2df844592a7c40c" target='_blank'>
              Neutrophils in Cancer: Phenotypic Heterogeneity Across Tumor Models and Significant Alteration of Splenic Neutrophil Phenotype in Lymphosarcoma RLS40 Model Following DNase I Treatment
              </a>
            </td>
          <td>
            K. Sounbuli, L. A. Alekseeva, A. Sen’kova, O. Markov, I. Savin, M. Zenkova, N.L. Mironova
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Gastrointestinal tract cancer (GIC), including oesophageal cancer (EC), gastric cancer (GC) and colorectal cancer (CRC), is characterised with high global incidence and mortality rates, with similar tumourigenic processes. However, the common and heterogeneous molecular features among GIC at single‐cell level remain poorly characterised. Methods Single‐cell RNA‐seq data of more than one million high‐quality annotated cells from 577 specimens, including 121 ECs, 182 GCs and 254 CRCs, were integrated to systematically decipher the heterogeneous characteristics of GIC. Non‐negative matrix factorisation (NMF) was employed to identify epithelial cell meta‐programs (MPs), and cell–cell communication analysis was conducted to investigate regulatory interactions between the tumour microenvironment (TME) and these MPs. Additionally, cell lineage inference analysis was performed to identify metaplastic signatures in EC and GC. Results We identified 24 consensus MPs from epithelial cells and 42 distinct subtypes from non‐epithelial cells thus offering a comprehensive overview of heterogeneous characteristic in GIC. Notably, we observed that EC exhibited unique features, including heightened activity in stress‐related programs and a more exhausted TME, enriched with CD4+ Tregs and CD8+ exhausted T cells. In contrast, epithelial cells in GC displayed increased expression of epithelial–mesenchymal transition (EMT)‐related signatures and an activated immune phenotype, marked by enrichment of NK cells and CD8+ effector T cells. Moreover, samples with metaplastic signatures in GC and EC showed similarities to CRC, including elevated expression of metabolism‐associated signatures and an abundance of CD4+ helper‐like T cells. Finally, we identified the potential regulatory roles of the TME in shaping epithelial cell behaviour. Conclusions Our findings provide insights into the common and specific cellular and molecular patterns associated with GIC tumourigenesis and TME remodelling. We also elucidate the similarity between GC/EC with metaplastic signature and CRC, which advancing our understanding of these malignancies. Key points A comprehensive single‐cell atlas of gastrointestinal tract cancer (GIC) was constructed. GICs exhibit distinct epithelial features and specific tumour microenvironment (TME) patterns, forming diverse niches. GC and EC exhibiting metaplastic features show elevated metabolism‐associated signatures and share similarities with CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e067271392a0848a314665131e19ea77d53c9ab" target='_blank'>
              Integrative single‐cell transcriptomic analysis deciphers heterogeneous characteristics of gastrointestinal tract cancer
              </a>
            </td>
          <td>
            Chuwen Sun, Tong Li, Xin Jin, Zhihui Xiu, Hang Su, Huanming Yang, Ming Liu, Kui Wu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The intricate relationship between inflammation and cancer progression has been well-documented, with chronic inflammation playing a pivotal role in tumor development and drug resistance. This project investigates how cancer-intrinsic inflammation influences drug response, aiming to identify key regulators that could serve as therapeutic targets to overcome drug resistance in colorectal cancer (CRC). We hypothesize that distinct inflammation types within cancer cells modulate their response to chemotherapy, potentially driving resistance. Our study is structured around two specific aims: First, we investigate the inflammation-type-dependent drug response using single-cell barcoded colon cancer cells. Our preliminary studies demonstrated that CRC cells in an inflamed microenvironment, particularly those with STING pathway activation, exhibit distinct responses to 5-FU treatment compared to non-inflamed cells. This finding underscores the need to consider inflammation as a critical factor in cancer treatment strategies. By characterizing the response of inflamed and non-inflamed cells to various drug treatments at single-cell resolution, we aim to identify the specific inflammation-driven mechanisms that contribute to drug resistance. Second, we focus on the characterization and validation of key regulators identified in inflamed CRC cells. We will validate candidate genes through functional assays, including gene knockdown experiments, to uncover the molecular pathways through which inflammation modulates drug response. By linking the distinct inflammatory states within cancer cells to drug resistance, we aim to identify novel biomarkers and therapeutic targets that could enhance the effectiveness of existing treatments. Our research has the potential to significantly impact the treatment of CRC by providing a deeper understanding of how inflammation drives drug resistance, ultimately guiding the development of more effective, personalized therapies for patients with inflamed tumors. This approach addresses the urgent need for targeted strategies to overcome the challenges posed by inflammation-associated drug resistance in cancer therapy.



 Kerem Fidan, Semih Can Akincilar, Tim Stuart, Vinay Tergaonkar. Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P71.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb480f230d054f0fd849886c3b02be9cc7e768af" target='_blank'>
              Abstract P71: Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding
              </a>
            </td>
          <td>
            Kerem Fidan, S. C. Akıncılar, Tim Stuart, Vinay Tergaonkar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H.A. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b91d10221c135d31eb4279368ba168e3fb144c" target='_blank'>
              Intra-tumoral Preservation of the Stem-like Malignant Cell Proportion in Glioblastoma, Prognostic Impact, and its Radiomethylomic Signatures.
              </a>
            </td>
          <td>
            Yuji Matsumoto, Omkar Singh, Jose A. Garcia, Zied Abdullaev, Nelson F Freeburg, Fanyang Yu, Hamed Akbari, Kyunglok Baik, Jun Guo, Natalie N C Shih, Erik Toorens, T. Ganguly, Dominique Ballinger, D. O’Rourke, S. Mohan, J. Morrissette, Dana Silverbush, Kenneth J Aldape, C. Davatzikos, M. P. Nasrallah
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Spatial omics provides three types of information: location, gene expression, and morphology. While current algorithms typically integrate gene expression with location, they often overlook morphology. Chronic Myelomonocytic Leukemia (CMML) is a rare blood cancer characterized by sustained peripheral blood monocytosis, bone marrow dysplasia, and overlapping features of both myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPN). A higher level of monoblasts, which are an immature monocytes with round nuclei, is a known risk factor in CMML, but the underlying biology and markers remain unclear. In this study, we developed a novel framework that integrates gene expression and morphology to investigate morphogenomic phenotypes in 12 CMML patients and 8 healthy controls. Recognizing that most existing spatial omics workflows are designed for adherent cells and are unsuitable for suspension cells like bone marrow mononuclear cells (BMMC), we first developed a protocol for immobilizing suspension cells. This was followed by MERFISH spatial RNA analysis of 492 genes across approximately 25,000 cells in a 16 mm2 region of interest over 34 hours. We then introduced a Morphology and Expression Data Optimal Clustering (MEDOC) algorithm, which uses adaptive Fourier-domain alignment and graph clustering to analyze cell shapes. In healthy BMMC, MEDOC analysis identified genes correlated with cell shape, findings that were substantiated by single-cell RNA sequencing data, confirming their association with nuclear shape. Leveraging this shape marker, we successfully identified a promonocyte population, a monoblast subtype that was previously difficult to distinguish. Furthermore, MEDOC analysis in CMML patients revealed two distinct nuclear shapes in mature leukemic monocytes: one resembling a normal kidney bean shape and another characterized by dysmorphic features. Thus, our framework offers a powerful tool to uncover the underlying biology of clinically relevant malignant cell morphologies and to enhance CMML patient stratification.



 Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar. INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P08.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4c00943719edf91d07a5ae18534128f66314a" target='_blank'>
              Abstract P08: INTERGRATING CELL SHAPE AND SPATIAL EXPRESSION IDENTIFIES MORPHOGENOMICS STATES IN LEUKEMIA
              </a>
            </td>
          <td>
            Jagadish Sankaran, Ignasius Joanito, Giovani Claresta Wijaya, Ziyue Chen, Joseph Lee, Vaidehi Krishnan, Than Hein, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a heterogeneous illness influenced by intricate tumor-immune interactions and characterized by a dismal prognosis. Macrophage-mediated immunological signaling facilitates tumor proliferation and may associate inflammation in the tumor microenvironment (TME) of CRC with negative outcomes. Notwithstanding therapeutic advancements, resistance to treatment remains a significant obstacle. scRNA-seq offers comprehensive insights into the immune signaling network and immunological dynamics inside the CRC’s TME. Methods We integrated scRNA-seq data from GEO with extensive RNA-seq data from TCGA to elucidate immunological signaling and dynamic cellular variation in the TME of CRC. The analyses encompassed quality control via Seurat, InferCNV, Monocle, CellChat, and SCENIC, differential gene expression, inference of copy number variation (CNV), pseudo time trajectories, and intercellular communication. Prognostic modeling was conducted using Cox regression and LASSO. Immune infiltration and drug sensitivity were evaluated by CIBERSORT, ESTIMATE, xCell, TIDE, and pRRophetic. Functional validation encompassed siRNA knockdown, qRT-PCR, Western blot analysis, and cellular assessment in CRC cell lines. Results We discovered four categories of tumor cells exhibiting variations in cell cycle, stemness, and differentiation. The MKI67 + subpopulation exhibited a heightened dynamic cell state and engaged with macrophages via the MIF-(CD74+CD44) axis to facilitate immunological signaling. HMGA1 is a crucial transcription factor in this fraction, and its knockdown impedes CRC cell proliferation, motility, and invasion. The cancer model utilizing the MKI67 + TCs subpopulation (MTRS) successfully classified patient survival and linked with immune infiltration patterns and medication responses. Enrichment analysis revealed tumor-promoting and immunological signaling networks. Correlation scores suggest that this subpopulation may be linked to inflammation and immunosuppression inside the TME. Conclusion Our research indicates that the C2 MKI67 + TCs subpopulation is a key driver of immune signal transduction in CRC TME, which may induce inflammatory responses through interaction with macrophages, thereby leading to adverse consequences such as cardiac dysfunction. HMGA1 represents a viable target for immunotherapy, and our cancer model derived from this subpopulation offers prognostic significance and direction for immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b53660e6d491a7fc5588044fc19d5176db5aae" target='_blank'>
              Single-cell technology reveals the crosstalk between tumor cells and immune cells: driving immune signal transduction and inflammation-mediated cardiac dysfunction in the tumor microenvironment of colorectal cancer
              </a>
            </td>
          <td>
            Wenyang Nie, Wangzheqi Zhang, Zhikai Xiahou, Yuxuan Meng, Yuhang Liu, Jingwen Zhang, Zhen Wang, Yong Wang
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary This study highlights the importance of evaluating cancer cells’ functional properties and gene expression in 3D breast cancer cell models over traditional 2D culture systems to mimic the tumor environment better so as to determine more efficient therapeutic approaches. By examining triple-negative and luminal A breast cancer subtypes, this research evaluates how the inhibition of the EGFR and IGF-IR pathways—separately and in combination—affects cell behavior and extracellular matrix expression. Our results revealed potential crosstalk between these pathways, influencing key functions such as cell proliferation, migration, and dissemination. Significant results in 3D models were observed, especially in the expression of matrix metalloproteinases. In summary, our findings emphasize the crucial role of EGFR and IGF-IR interactions in breast cancer progression and demonstrate the importance of using physiologically relevant 3D cell platforms in cancer research. Abstract Background: Breast cancer, one of the most researched cancers in oncology, remains the primary cause of cancer-related mortality in women. Its biological complexity, which includes phenotypic, genetic, and microenvironmental aspects, makes modeling and treatment quite difficult. The need for more physiologically realistic models is highlighted by the comparison of two-dimensional (2D) cell cultures with 3D breast-cancer-derived spheroids, which discloses how important pathways such as epidermal growth factor receptor (EGFR) and insulin-like growth factor I receptor (IGF-IR) influence cell behavior and extracellular matrix (ECM) macromolecular expression. Methods: The purpose of this study was to utilize novel 3D cell platforms to assess the effect of inhibiting the EGFR and IGF-IR pathways, alone or in combination, on the functional properties and the expression levels of certain matrix metalloproteinases (MMPs) which are implicated in breast cancer progression (i.e., triple-negative and luminal A breast cancer subtypes) and related with the EGFR and IGF-ΙR molecular network, as also demonstrated through STRING analysis. Results: Our results demonstrated potential crosstalk between EGFR and IGF-IR signaling, which influences cell proliferation and spheroid growth, dissemination, and migration. Significant phenotypic changes proposed between 2D and 3D cell cultures, and alterations in the expression of MMPs, were also recorded. Conclusions: Both breast cancer cell lines retained acknowledged characteristics across the tested models while also exhibiting new, condition-dependent properties. Overall, our findings enhance our understanding on the interplay between the EGFR and IGF-IR pathways and underscore the value of 3D models in revealing key biological processes underlying distinct breast cancer phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb767e92d334db46b0fc194e1bd3fd8b089df1a" target='_blank'>
              From Structure to Function: The Impact of EGFR and IGF-IR in 3D Breast Cancer Spheroids
              </a>
            </td>
          <td>
            Chrisavgi Gourdoupi, Spyros Kremmydas, Sylvia Mangani, Paraskevi Ioannou, Nikolaos A. Afratis, Z. Piperigkou, Nikos K. Karamanos
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="The development and metastasis of gastric cancer are complex processes involving the TME (Tumor microenvironment) and the interactions between various cell types. Here, we investigated the molecular mechanisms and biological processes underlying gastric cancer and its metastasis using single cell RNA-sequencing (scRNA-seq) with the aim of identifying new targets and approaches for clinical treatment. R version 4.4.1 and the SeuratV5 package were used to process 20 scRNA-seq samples sourced from the GEO database. Highly variable genes were selected for GO (Gene Ontology), KEGG (Kyoto Encyclopedia of Genes and Genomes) and GSAV (Gene set variation analysis) enrichment analyses. CytoTRACE, CellChat, and Monocle3 analyses were conducted to investigate cell communication and pseudotemporal dynamics in the PM (Peritoneal Metastasis) and GC (Gastric Cancer) groups. Additionally, the prognostic significance of key genes was assessed by integrating data from the TCGA clinical database. A total of 2,626,594 peritoneal metastasis cells and 17,894 gastric cancer cells were identified, revealing 13 distinct cell clusters. Gene enrichment analyses identified high expression in several pathways (P53, Wnt and JAK-STAT3) in TAMs and mast cells. Cell communication was more robust in the GC group than the PM group, with TAMs (Tumor-associated Macrophages) and mast cells showing elevated expression of the CCL5-CCR1 ligand-receptor signaling axis in both groups. Pseudotemporal analysis demonstrated the differentiation potential of TAMs into mast cells, with APOC1, C1QB, FCN1, FTL, S100A9, CD1C, CD1E and FCER1A identified as the top eight genes driving this process. High expression levels of these genes, along with CCL5 and CCR1, were associated with poor long-term survival in cancer patients. scRNA-seq identified the intricate tumor immune microenvironment, highlighting the pivotal roles of TAMs and mast cells in gastric cancer peritoneal metastasis. The CCL5-CCR1 pathway emerged as a potential immune checkpoint, offering novel insights for future immunotherapeutic and targeted therapeutic strategies in the treatment of gastric cancer peritoneal metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b821ef0a01b53e75ba382be86c9829d902456b2" target='_blank'>
              Comprehensive analysis of the tumor immune microenvironment in gastric cancer and peritoneal metastasis based on single-cell RNA sequencing analysis
              </a>
            </td>
          <td>
            Wenhao Sun, Zhiwei Jiang, Zhengming Deng
          </td>
          <td>2025-09-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The human body contains ~27–36 trillion cells of up to 10,000 cell types (CTs) within a volume of ~62–120 liters (males) and 52–89 liters (females). The Human Reference Atlas (HRA) v2.3 provides a quantitative 3D framework of CTs across 73 reference organs and 1,283 3D anatomical structures (ASs). The HRA Cell Type Population (HRApop) effort quantifies CTs per AS using high-quality single-cell (sc) data processed through scalable, reproducible workflows and cell type annotation (CTann) tools. HRApop v1.0 includes reference CT populations for 73 ASs (112 when sex-specific) using 662 datasets spatially registered to 230 locations across 17 organs (31 when sex-specific). For 558 sc-transcriptomics datasets (11,042,750 cells), CTs and biomarker expression were computed using Azimuth, CellTypist, and popV. To test generalizability, 104 sc-proteomics datasets (16,576,863 cells) were integrated. In total, HRApop includes 27,619,613 cells. HRApop can be used to predict (1) CT populations for 3D volumes in the human body and (2) the spatial origin of a tissue block, given a CT population. Data and code are at cns-iu.github.io/hra-cell-type-populations-supporting-information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebb2aca86dc56659eee58275de4ce878d6c3dc83" target='_blank'>
              Constructing and Using Cell Type Populations of the Human Reference Atlas
              </a>
            </td>
          <td>
            Andreas Bueckle, B. Herr, Lu Chen, Daniel Bolin, Danial Qaurooni, Michael Ginda, Yashvardhan Jain, Aleix Puig-Barbe, Kristin Ardlie, Fusheng Wang, Katy Börner
          </td>
          <td>2025-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7804cba30cd408c643b898725452b0298732929b" target='_blank'>
              A Spatial and Temporal Transcriptomic Atlas of Mouse Intestinal Regeneration.
              </a>
            </td>
          <td>
            Rachel Ofer, Tyler Tran, Ping He, Xia Qiu, Xinyu Hu, Emmanuel Zachariah, Curtis Krier, A. Saxena, Jiekun Yang, Michael Verzi
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal, aggressive malignancy. Little is known about the evolutionary trajectories of HCC and how clinical decision-making could be informed based on biopsies of the initial tumour. Here, we report the whole-exome sequencing of a unique series of resected HCC tumours and matched recurrences. This cohort included patients who received a liver transplant and who were immunosuppressed at time of recurrence, in comparison to patients who underwent liver resection for HCC and immunocompetent at time of recurrence, therefore facilitating analyses of immune selection in driving evolutionary divergence. We find extensive evolutionary divergence between baseline and recurrent tumours, with the majority of mutations in our cohort being private, in the process informing sampling guidelines for precision oncology in this disease. We also find no evidence that immunosuppression relaxes immune selection pressures, given the absence of a genomic footprint reflecting the presentation of neoantigens or altered dynamics of genomic evolution. We attribute this to the presence of genetic lesions that confer the capabilities of immune evasion in these tumours prior to transplantation, and then validate the link between immune selection pressures and the emergence of these lesions in publicly available HCC datasets. Our findings point to HCC as a cancer with extensive evolutionary divergence over time, partly defined by an irreversible, genetically determined trajectory towards immune escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9c9a24625027fee25b759620a813b537ff8ac7" target='_blank'>
              Evolutionary dynamics of recurrent hepatocellular carcinoma under divergent immune selection pressures
              </a>
            </td>
          <td>
            A. Chakravarthy, Elisa Pasini, Xun Zhao, Jeffrey To, Shu Yi (Roxana) Shen, Sandra Fischer, A. Ghanekar, Arndt Vogel, Robert C. Grant, Jennifer J Knox, Gonzalo Sapisochin, Gregory J. Gores, Daniel De Carvalho, Mamatha Bhat
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract The interaction between intratumoral microbiome and the tumor microenvironment (TME) has furthered our understanding of tumor ecology. Yet, the implications of their interaction for lung cancer management remain unclear. In the current work, we collected host transcriptome samples and matched intratumoral microbiome samples, as well as detailed clinical metadata from The Cancer Genome Atlas (TCGA) of 478 patients with lung adenocarcinoma (LUAD). Utilizing the multiomics integration approach, we comprehensively investigated the crosstalk between the TME and intratumoral microbiome in patients with LUAD. First, we developed a prognostic model based on the TME signatures (TMEindex) that clearly distinguished clinical, survival, and response to immunotherapy of patients with LUAD. Additionally, we found profound differences in intratumoral microbiota signatures, including alpha- and beta-diversity, among patients with different survival risks based on the TME signatures. In depth, we detected that genera Luteibacter and Chryseobacterium were strongly negatively and positively associated with patients’ survival risk, respectively, suggesting their opposing roles in cancer progression. Moreover, we developed a model that fused intratumoral microbial abundance information with TME signatures, called intratumoral microbiome-modified TMEindex (IMTMEindex), leading in predicting patient overall survival at 1-, 3-, and 5-years. Future clinical profiling of the specific intratumoral microbes in the TME could improve prognosis, inform immunotherapy, and facilitate the development of novel therapeutics for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ceebaa130c34b48cdab5c0e6efa7c18090d66b0" target='_blank'>
              Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota
              </a>
            </td>
          <td>
            Fei Zhao, Lei Wang, Dongjie Du, Heaven Zhao, Geng Tian, Yufeng Li, Yankun Liu, Zhiwu Wang, Dasheng Liu, Jingwu Li, Lei Ji, Hong Zhao
          </td>
          <td>2025-08-11</td>
          <td>Microbial Cell</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We present a novel classification system for murine thymic epithelial cells (TECs), identifying 11 distinct types, four in the thymic cortex and seven in the medulla, based on their spatial localization and unique ultrastructural features. As key stromal components of the thymic microenvironment, TECs play indispensable roles in T cell development, including thymocyte selection, antigen presentation, and structural support. Our classification spans from the subcapsular cortex to the deep medulla and incorporates microanatomical context, morphology, and functional characteristics, providing a comprehensive and flexible framework to study TEC heterogeneity in relation to thymopoiesis. Aligning with TEC classification in rats and humans, this system highlights conserved spatial organization across species while remaining adaptable for refinement. Each TEC type is distinguished by features such as chromatin organization, cytoplasmic morphology, vacuolar content, and organelle distribution, attributes that suggest distinct functional contributions to various stages of thymocyte maturation. Importantly, the classification is designed for logical expansion both horizontally (inclusion of additional subtypes within the proposed TEC types) and vertically (inclusion of entirely novel TEC types). By integrating detailed morphological observations with testable functional hypotheses, this framework underscores the essential role of TEC diversity in supporting thymic architecture and orchestrating effective T cell output. Overall, it offers a robust foundation for future research into immune development and the pathological consequences of thymic dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a499d2fa9f48a695cdae808a5b371013bf08351d" target='_blank'>
              Mapping murine thymic epithelial cells: functional ultrastructure and implications for thymopoiesis.
              </a>
            </td>
          <td>
            Stepan Vodopyanov, Leslie Gunther-Cummins, Joseph Churaman, Xheni Nishku, T. Poutahidis, A. Hardas, Frank P Macaluso, George S. Karagiannis
          </td>
          <td>2025-08-01</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Waddington’s epigenetic landscape has long served as a conceptual framework for understanding cell fate decisions. The landscape’s geometry encodes the molecular mechanisms that guide the gene expression profiles of uncommitted cells toward terminally differentiated cell types. In this study, we demonstrate that applying the concept of intrinsic dimension to single-cell transcriptomic data can effectively capture trends in expression trajectories, supporting this framework. This approach allows us to define a robust cell potency score without relying on prior biological information. By analyzing an extensive collection of datasets from various species, experimental protocols, and differentiation processes, we validate our method and successfully reproduce established hierarchies of cell type potency. Our work provides a direct link between geometric properties of single-cell expression profiles and the level of differentiation of a cell population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/118f0f4ef095a2b7a235de5cd5cb87bf61bf6aec" target='_blank'>
              The intrinsic dimension of gene expression during cell differentiation
              </a>
            </td>
          <td>
            Marta Biondo, Niccolò Cirone, Filippo Valle, Silvia Lazzardi, M. Caselle, Matteo Osella
          </td>
          <td>2025-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) is an aggressive and heterogeneous tumor with limited therapeutic options. Tumor-associated neutrophils (TANs) play multifaceted roles in the tumor microenvironment (TME) depending on their polarization. Understanding TAN heterogeneity and the mechanisms underlying their tumor-promoting activities is critical for advancing CRC treatment. Methods: This study integrated single-cell RNA sequencing, spatial transcriptomics, bulk RNA sequencing, and in vivo/ex vivo experiments to characterize TANs in CRC. Results: We identified a distinct subpopulation of TANs characterized by high CCCTC-binding factor (CTCF) expression (CTCF+ TANs) enriched in hypoxic tumor regions. CTCF+ TANs exhibit enhanced migratory capacity and IL-1β secretion, correlating with poor prognosis and resistance to immunotherapy. Mechanistically, CTCF regulates the expression of Migration and Invasion Enhancer 1 (MIEN1), which promotes TAN recruitment and migration without triggering inflammation. Functional studies revealed that CTCF+ TANs suppress T-cell immunity, facilitate epithelial-to-mesenchymal transition (EMT), and contribute to CRC progression and metastasis. In vivo, targeting CTCF-MIEN1-IL-1β signaling rescued the immunosuppressive microenvironment and improved the efficacy of anti-PD-L1 therapy. Conclusions: CTCF+ TANs represent a novel TAN subtype that drives CRC progression and immunosuppression via the CTCF-MIEN1-IL-1β axis. These findings highlight the potential of targeting CTCF+ TANs to overcome immunotherapy resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e6c853622163edbe8255deed0daaabb0c23ad1" target='_blank'>
              Comprehensive analytical approach identifies a subtype of CTCF+ tumor-associated neutrophils associated with CRC development and as a biomarker for immunotherapy
              </a>
            </td>
          <td>
            Jie-pin Li, Yuan Liu, Yi Zhang, , Guo Xu, Jin-chen Chong, Yi Yin, Yang Li, Shuang-shuang Wang, , Jun Qian, Shen-lin Liu, Xi Zou, Yu-gen Chen
          </td>
          <td>2025-08-16</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Objectives This study charted how investigations of tumour-microenvironment (TME) dynamics have shaped global thinking on drug resistance in breast cancer over the past three decades. Methods Web of Science Core Collection records (1995–2024) were harvested on 15 May 2025 with a three-block Boolean string integrating disease, microenvironmental, and resistance terms. After duplicate removal, document-type filtering, language restriction, and natural-language adjudication, 1 303 original articles and reviews remained. Bibliometrix (R 4.3.2) generated annual output, citation kinetics, and growth rates; VOSviewer (fractional counting, resolution = 1.0) mapped co-authorship, co-citation, and keyword networks. Results Publication volume expanded at a compound annual rate of 18.7%, progressing through formative (1995–2004), consolidation (2005–2012), and expansionary (2013–2024) epochs. Keyword drift traced a pivot from endocrine-centric lexicons (“tamoxifen”, “estrogen receptor”) to TME-immune discourse (“extracellular vesicles”, “tumour-associated macrophages”, “nanoparticles”); “microenvironment”, “hypoxia”, and “multidrug resistance” emerged as high-betweenness semantic bridges. The United States–China dyad accounted for 56.1% of output and anchored 24.4% of internationally co-authored papers, funnelling methodological innovation into an increasingly multipolar network, here defined as a collaboration graph containing at least four countries that each contribute ≥ 5% of total node-level betweenness centrality. Institutional overlays revealed late-decade ascendancy of East-Asian medical universities, while legacy North-American and European centres maintained brokerage dominance. Conclusion Bibliometric cartography documents a decisive ecological re-framing of breast-cancer drug-resistance research, with immune-stromal crosstalk supplanting receptor-centric models. Cross-border, multi-omics consortia that fuse East-Asian clinical scale with Western technological depth appear best poised to convert TME insights into resistance-reversing interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37738d1f4bd99af794fa1fdf0d88aeebed7041dc" target='_blank'>
              Mapping thirty years of tumour-microenvironment-driven drug resistance in breast cancer: a global bibliometric analysis
              </a>
            </td>
          <td>
            Zuoping Duan, Yonghong Zhou, Yang Wu
          </td>
          <td>2025-08-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Efforts to understand the tumor microenvironment through basic science research and the cancer genome atlas (TCGA) data analysis have led to the creation of unique immune transcriptomic signatures from tumor-infiltrating lymphocytes (TIL). However, no pan-cancer analysis has been conducted to compare the prognostic performance of these signatures using overall survival (OS) or progression-free interval (PFI) as endpoints. We compiled a library of 146 TIL-immune signatures and evaluated gene signature score correlation with OS and PFI for 9,961 available TCGA samples across 33 tumor types. Zhang CD8 TCS demonstrated higher accuracy in prognosticating both OS and PFI across the pan-cancer landscape; however, variability was seen across cancer types and germ cell origin. Cluster analysis compiled a group of six signatures (Oh.Cd8.MAIT, Grog.8KLRB1, Oh.TIL_CD4.GZMK, Grog.CD4.TCF7, Oh.CD8.RPL, Grog.CD4.RPL32) whose association with OS and PFI could potentially be conserved across multiple neoplasms. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04102-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb16ad03a570f27481909ae39fc0049c1c73e2a" target='_blank'>
              A pan-cancer comparative analysis of the cancer genome atlas transcriptomic TIL-immune signatures
              </a>
            </td>
          <td>
            Kyle J. Hitscherich, Darryl Nousome, Aaron J. Dinerman, Victoria Dulemba, Frank J Lowery, N. Nilubol
          </td>
          <td>2025-08-07</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Anaplastic thyroid carcinoma (ATC) is one of the most lethal human malignancies, often evolving from differentiated thyroid carcinoma (DTC) through a poorly understood dedifferentiation process. To elucidate this transition, spatial transcriptomic sequencing (spRNAseq) is performed on seven samples containing coexisting regions of ATC, poorly differentiated thyroid carcinoma, and DTC. SpRNAseq revealed that ATC regions were characterized by upregulated genes involved in immune suppression, angiogenesis, and extracellular matrix remodeling. Whole-exome sequencing and inferCNV analysis confirmed that adjacent DTC regions harbored mutational burdens comparable to those of ATC regions, suggesting early genomic priming for dedifferentiation. Trajectory analysis delineated a stepwise reprogramming process and identified four gene modules associated with the loss of thyroid differentiation, among which PDCD4 and TYMP emerged as key regulators. Notably, TYMP⁺ tumor-associated macrophages (TAMs) were highly enriched in ATC regions and contribute to an immunosuppressive microenvironment. Mechanistic experiments demonstrated that loss of PDCD4 led to eIF4A-dependent overexpression of immunosuppressive effectors, promoting the high infiltration of TYMP⁺TAMs in ATC. These findings support that coexisting DTC regions with ATC-like genomic alterations undergo sequential transcriptomic reprogramming and immune microenvironment remodeling to evolve into a full ATC pathological phenotype, in which PDCD4 loss-induced TAMs formation plays a critical role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4fbade6a7e0bbb1789c00df5213a5311bb8e73a" target='_blank'>
              Spatial Transcriptomics Reveals Transcriptomic and Immune Microenvironment Reprogramming during Thyroid Carcinoma Dedifferentiation.
              </a>
            </td>
          <td>
            K. Ning, Bu Zou, Yongchao Yu, Taonong Cai, Zhenyu Luo, Yu Guo, Yi Wu, Xiujiao Shen, Hao Li, Mengyuan Fang, J. Bu, Han Hong, Zan Jiao, Tong Wu, Yulong Wang, Tianrun Liu, Wei-chao Chen, Wanming Hu, Mingjie Jiang, A. Yang
          </td>
          <td>2025-09-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38cc5302dc11612c06528bfceedae2ff2c387b6" target='_blank'>
              S1PR5 as a prognostic biomarker in colon cancer: insights into efferocytosis-related mechanisms and immune modulation.
              </a>
            </td>
          <td>
            Aijie Pei, Ling Lu, Xiaolin Wu
          </td>
          <td>2025-09-15</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) shows racial disparities, with higher incidence in women of African ancestry (AA) compared to European ancestry (EA). Meta-transcriptomic analysis of TNBC tumor tissues from AA (n = 17) and EA (n = 19) subjects revealed distinct microbial landscapes. Hierarchical clustering based on microbial transcripts separated samples into two groups predominantly defined by racial ancestry. Bacterial genera including Hafnia and Cedecea were more abundant in AA tumors, while Erwinia was higher in EA tumors. Cellular composition analysis by xCell revealed differences in immune cell populations, with AA tumors having higher Th1 cell abundance and EA tumors containing higher macrophage M2 cell abundance. Nonetheless, AA women with high M2 abundance experienced poorer disease-free survival (DFS) than EA women. Integrative analyses revealed that high expression of human SPDYE2B gene was associated with Hafnia abundance and decreased DFS, highlighting complex host-microbe interactions in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4784e682d1bb059c5d6fa5b77baa53f7b2ce04c2" target='_blank'>
              Race-related host and microbe transcriptomic signatures in triple-negative breast cancer
              </a>
            </td>
          <td>
            Roshan Kumar, Susan Duyar-Ayerdi, Aishwarya Sundaresan, V. Srinivasasainagendra, C. Pedamallu, M. Behring, D. Chandrashekar, I. Eltoum, S. Varambally, Hemant K. Tiwari, Sadeep Shrestha, Paul L Auer, L. Chaudhary, John R Kirby, Clayton C. Yates, Upender Manne, A. Ojesina
          </td>
          <td>2025-08-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Background The tertiary lymphoid structures (TLS) within the immune microenvironment of hepatocellular carcinoma (HCC) have been shown to significantly influence patient prognosis. Understanding the mechanisms behind their formation and maturation can help us refine therapeutic strategies and improve treatment outcomes. In certain cancers, intratumoral bacteria have been found to promote the development of TLS. Our study aims to investigate the impact of intratumoral bacteria on TLS-associated immune cells in HCC, as well as their relationship with TLS quantity and maturation status. Methods In this study, we collected samples from 153 patients with hepatocellular carcinoma. We employed fluorescence in situ hybridization (FISH), immunofluorescence (IF), and hematoxylin and eosin (H&E) staining to assess intratumoral bacterial load, as well as to evaluate the number and maturation status of tertiary lymphoid structures. Patient prognosis was also analyzed. Furthermore, we examined the bacterial load and distribution within tumors of patients presenting TLS, and explored the relationship between intratumoral bacteria and TLS-associated immune cell infiltration. Results Among 74 patients with hepatocellular carcinoma, tertiary lymphoid structures were present within the tumors. By integrating the expression profiles of both intratumoral and peritumoral TLS, prognostic analysis revealed that patients with structured tumor microenvironments (TME) and exclusionary TME had better outcomes. The intratumoral bacterial load varied among patients, with higher bacterial burden observed in regions enriched with TLS. Moreover, as TLS matured, the bacterial load within tumors was significantly greater compared to patients lacking TLS. Correspondingly, CD20, a major component of TLS, showed increased expression. These findings suggest that intratumoral bacteria can influence the immune response within the tumor microenvironment and are associated with the maturation of TLS. Conclusion Our study demonstrates that patients receiving structured TME and excluded TME subtypes exhibit superior overall survival (OS) and recurrence-free survival (RFS) following radical surgery. Furthermore, the intratumoral bacterial load showed a significant correlation with CD20+ B-cell density and was strongly correlated with both the number and maturity of TLS. These findings suggest that intratumoral bacteria may influence patient responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51afc77b8d734ec619b2147ecef00cb70414525c" target='_blank'>
              Intratumoral bacterial load and tertiary lymphoid structure density in hepatocellular carcinoma: association and prognostic significance
              </a>
            </td>
          <td>
            Shen Qu, Weili Jia, Xiaoyan Liu, Qianyun Yao, Chao Chen, Zihao Zhao, Ye Nie, Feng Chang, Zexu Yang, Chaosheng Peng, Yangang Wang, Wenjie Song
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Parkinson's disease (PD) is a progressive neurodegenerative disorder with complex and heterogeneous molecular pathology across the brain. However, the full cellular architecture of PD progression remains unresolved. Here, we present a comprehensive single nucleus transcriptomic atlas of PD spanning multiple anatomically and clinically relevant brain regions to capture shared and region-influenced transcriptomic features from 97 deeply phenotyped donors, including individuals across the full spectrum of Braak Lewy body stages. Profiling over 2 million nuclei, we define 62 transcriptionally distinct cell subtypes and uncover widespread, cell-type-specific gene expression changes across neurons, glia, and vascular cells. We identify convergent upregulation of stress-responsive transcriptional programs - such as unfolded protein response, DNA damage repair, and autophagy - across multiple cell types, with key regulators including HSF1, MYC, and FOXO3. Integrative analyses link these transcriptional alterations to PD genetic risk, revealing transcription factor to target gene networks enriched for PD GWAS loci in microglia and neuronal subpopulations. To quantify disease burden at cellular resolution, we introduce a transcriptomic pathology score, revealing early-stage activation in neurons and myeloid cells, followed by delayed engagement of vascular populations. We further demonstrate that microglia undergo dynamic, subtype-specific transitions across disease stages, including early adaptive responses and late-phase stress and proliferative programs. Altered cell-cell communication networks, particularly involving myeloid-neuronal signaling, highlight a progressive rewiring of neuroimmune interactions. This atlas provides a foundational resource for understanding PD progression at single-cell resolution, linking genetic risk to dynamic molecular pathology, and illuminating stage-specific targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f76191da79e75d76b8635e83ff4d068f539756" target='_blank'>
              A Transcriptomic Roadmap of Parkinson’s Disease Progression at Single Cell Resolution
              </a>
            </td>
          <td>
            T. Clarence, N. Masse, Christian Porras, Prashant Nm, Sabina Berretta, Vahram Haroutunian, David A. Davis, William K. Scott, G. Hoffman, J. Fullard, Donghoon Lee, J. Bendl, P. Roussos
          </td>
          <td>2025-07-30</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent rare cancers originating from mesenchymal tissues, featuring over 70 histological subtypes with diverse genetic defects and clinical outcomes. Current treatment options for STS mainly involve surgical procedures and anthracycline-based chemotherapy, which often prove ineffective across multiple STS subtypes. Consequently, there's a pressing need for novel, STS-specific therapies. Targeting cell surface antigens expressed exclusively in STS has emerged as a promising approach. However, the surfaceome of STS remains largely uncharacterized. In our research, we employed mass spectrometry to analyze biotinylated and immunoprecipitated cell surface proteins expressed in STS cell lines and patient samples. Through this method, we've identified several promising cell surface antigens that could potentially serve as therapeutic targets for STS. Further validation of these antigens across various experimental settings is crucial to confirm their efficacy and pave the way for new strategies in STS treatment.



 Phuong Mai Hoang, Wai Khang Yong, Dennis Kappei, Anand D. Jeyasekharan. SURFACEOME PROFILING TO IDENTIFY TARGETABLE CELL SURFACE ANTIGENS IN SOFT TISSUE SARCOMAS [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P61.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6123041a91ea4253d5c71b5fec3661998638d06f" target='_blank'>
              Abstract P61: SURFACEOME PROFILING TO IDENTIFY TARGETABLE CELL SURFACE ANTIGENS IN SOFT TISSUE SARCOMAS
              </a>
            </td>
          <td>
            P. Hoang, Wai Khang Yong, Dennis Kappei, A. Jeyasekharan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND Glioblastoma multiforme (GBM) is the most aggressive and prevalent primary malignant brain tumor in adults, marked by poor prognosis and high invasiveness. Traditional GBM invasion assays, such as those involving mouse brain xenografts, are often time-consuming and limited in efficiency. In this context, stem cell-derived neural organoids (NOs) have emerged as advanced, three-dimensional, human-relevant platforms that mimic the cellular architecture and microenvironment of the human brain. These models provide novel opportunities to investigate glioblastoma stem cell invasion, a critical driver of tumor progression and therapeutic resistance. AIM To evaluate studies using stem cell-derived NOs to model glioblastoma migration/invasion, focusing on methodologies, applications and therapeutic implications. METHODS We conducted a systematic review following PRISMA guidelines, searching PubMed and Scopus for studies published between March 2019 and March 2025 that investigated NOs in the context of glioblastoma invasion/migration. After screening 377 articles based on predefined inclusion and exclusion criteria, 10 original research articles were selected for analysis. Extracted data were categorized into four analytical domains: (1) Tumor model formation; (2) NO characteristics; (3) NO differentiation protocols; and (4) Invasion/migration assessment methodologies. RESULTS The included studies exhibit significant methodological heterogeneity GBM model development, particularly regarding model type, cell source and culture conditions. Most studies (70%) used suspension cell models, while 30% employed spheroids, with most research focusing on patient-derived glioblastoma stem cells. NOs were predominantly generated from human induced pluripotent stem cells using both guided and unguided differentiation protocols. Confocal fluorescence microscopy was the primary method used for assessing invasion, revealing invasion depths of up to 300 μm. Organoid maturity and co-culture duration influenced results, while key factors for model optimization included tumor cell density, organoid age and extracellular matrix composition. Some studies also tested therapeutic strategies such as Zika virus and microRNA modulation. Collectively, findings support the utility of NOs as effective tools for studying GBM behavior and therapeutic responses in a humanized three-dimensional context. CONCLUSION Human NOs represent promising platforms for modeling glioblastoma invasion in a humanized three-dimensional environment. However, a limited number of studies and methodological heterogeneity hinder reproducibility. Protocol standardization is essential to enhance the translational application of these models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e3dbb12a54a93d6feaabcb9aa2928129bd1bc4e" target='_blank'>
              Stem cell-derived neural organoids as platforms to investigate glioblastoma invasion and migration: A systematic review
              </a>
            </td>
          <td>
            A. H. Alves, Nicole Mastandrea Ennes do Valle, Bruno Yukio Yokota-Moreno, Marta Caetano dos Santos Galanciak, Keithy Felix da Silva, J. B. Mamani, Andrea Laurato Sertie, Fernando Anselmo de Oliveira, M. Nucci, Lionel Fernel Gamarra
          </td>
          <td>2025-08-26</td>
          <td>World Journal of Stem Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates that form in non-lymphoid organs, frequently observed in conditions such as cancer, autoimmune diseases, transplant rejection, and chronic inflammation. Growing evidence suggests that TLSs are beneficial for patients’ prognosis with higher TLS density generally correlating with improved therapeutic response and survival outcomes across malignancies and might serve as a novel therapeutic target for cancer immunotherapy. However, the correlation between TLSs and tumor development is still ambiguous. The exact timing of TLS formation during tumorigenesis and their dynamic evolution throughout tumor progression remain under investigation. Recent studies have identified potential strategies for inducing TLSs, but there remains a considerable distance from clinical application. More advanced techniques such as high-resolution spatial multi-omics technologies combined with big data analysis will benefit understanding the complex interactions within TLSs and developing novel immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d68cd72d4e06f1532031fad0eb22d59671e56af" target='_blank'>
              Orchestration of tertiary lymphoid structures: decoding developmental mechanisms for next-generation cancer immunotherapies
              </a>
            </td>
          <td>
            Wenqin Hao, Ying Zhao
          </td>
          <td>2025-09-12</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573d73f70f9bc8245cb6e46c84579fa69539af01" target='_blank'>
              Role of tumor microenvironment in cancer promotion, development of drug resistance and cancer treatment.
              </a>
            </td>
          <td>
            Duaa e Fathah, S. Ejaz
          </td>
          <td>2025-09-15</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Single-cell technologies provide an unprecedented opportunity for dissecting the interplay between the cancer cells and the associated tumor microenvironment, and the produced high-dimensional omics data should also augment existing survival modeling approaches for identifying tumor cell type-specific genes predictive of cancer patient survival. However, there is no statistical model to integrate multiscale data including individual-level survival data, multicellular-level cell composition data and cellular-level single-cell omics covariates. We propose a class of Bayesian generalized promotion time cure models (GPTCMs) for the multiscale data integration to identify cell-type-specific genes and improve cancer prognosis. We demonstrate with simulations in both low- and high-dimensional settings that the proposed Bayesian GPTCMs are able to identify cell-type-associated covariates and improve survival prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e78ba96b363e27ecb62584fc447ab1443df2e1a7" target='_blank'>
              Generalized promotion time cure model: A new modeling framework to identify cell-type-specific genes and improve survival prognosis
              </a>
            </td>
          <td>
            Zhi Zhao, Fatih Kızılaslan, Shixiong Wang, M. Zucknick
          </td>
          <td>2025-08-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma (GBM) exhibits profound plasticity, enabling adaptation to fluctuating microenvironmental stressors such as hypoxia and nutrient deprivation. However, this metabolic rewiring also creates subtype-specific vulnerabilities that may be exploited therapeutically. Here, we investigate whether mitochondrial transplantation using non-neoplastic, human myocyte-derived mitochondria alters the metabolic architecture of GBM cells and modulates their response to ionizing radiation. Using a cell-penetrating peptide–mediated delivery system, we successfully introduced mitochondria into two mesenchymal-subtype GBM cell lines, U3035 and U3046. Transplanted cells exhibited enhanced mitochondrial polarization and respiratory function, particularly in the metabolically flexible U3035 line. Bioenergetic profiling revealed significant increases in basal respiration, spare respiratory capacity, and glycolytic reserve in U3035 cells post-transplantation, whereas U3046 cells showed minimal bioenergetic augmentation. Transcriptomic analyses using oxidative phosphorylation (OXPHOS) and glycolysis gene sets confirmed these functional findings. At baseline, U3035 cells expressed high levels of both glycolytic and OXPHOS genes, while U3046 cells were metabolically suppressed. Following radiation, U3035 cells downregulated key OXPHOS and glycolysis genes, suggesting metabolic collapse. In contrast, U3046 cells transcriptionally upregulated both pathways, indicating compensatory adaptation. These results identify and establish mitochondrial transplantation as a metabolic priming strategy that sensitizes adaptable GBM subtypes like U3035 to therapeutic stress by inducing bioenergetic overextension. Conversely, rigid subtypes like U3046 may require inhibition of post-radiation metabolic compensation for effective targeting. Our findings support a novel stratified approach to GBM treatment which integrates metabolic subtype profiling with bioenergetic modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/783b92f72326d7a0b6481fe182c59e8472a6fd17" target='_blank'>
              Differential Bioenergetic Profile of Human Glioblastoma following Transplantation of Myocyte-derived Mitochondria
              </a>
            </td>
          <td>
            Kent L Marshall, Ethan M Meadows, Alan Mizener, John M Hollander, Christopher P Cifarelli
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d83d30b3f5a1eb25ccb22f23e0bb26ae153bc99" target='_blank'>
              Recent progress in clonal hematopoiesis: expanding the concept.
              </a>
            </td>
          <td>
            S. Chiba
          </td>
          <td>2025-07-31</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The self-organization of cellular collectives is crucial in development and cancer. Multicellular aggregation in cancer is associated with a higher efficiency of metastasis. However, it is not fully understood how mechanochemical microenvironmental cues affect the organization and stability of such ensembles. Here, using a model system of ovarian cancer spheroids, which temporally transit from solid, dysmorphic moruloids to structurally plastic, lumen-containing blastuloids, it is shown that the periodic volume fluctuations observed in blastuloids are driven by lumenal fluid influx and cell-cell junctional states. Furthermore, blastuloid cell states are reprogrammed, which enables them to rapidly recover from even complete structural disintegration and self-organize into fully lumenized ensembles. Using targeted chemical perturbations, two distinct cues are identified that regulate separate transition traits: calcium levels establish cell states cognate to, and pH regulates the fluctuation dynamics of blastuloid phenotypes. The work holds significant implications toward understanding mechanisms governing structural resilience and plasticity in complex cellular assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66fde2e2e92de36b5f741bbc8c6625ef851d5aaf" target='_blank'>
              Distinct Chemical Cues Reprogram Cellular and Multicellular Phenotypes in Ovarian Cancer Spheroids.
              </a>
            </td>
          <td>
            M. Sreepadmanabh, Meenakshi Ganesh, Jimpi Langthasa, R. Bhat, Tapomoy Bhattacharjee
          </td>
          <td>2025-09-16</td>
          <td>Small</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3dcbc9ad911267e37016ce60592b305f39ee376" target='_blank'>
              Decoding immunometabolism with next-generation tools: lessons from dendritic cells and T cells.
              </a>
            </td>
          <td>
            Yi-Hao Wang, Limei Wang, Ping-Chih Ho
          </td>
          <td>2025-09-16</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2093fb2bf2cfa5129572c94a9428ff0b8a8b0c1b" target='_blank'>
              Gene signatures characterizing driver mutations in lung squamous carcinoma are predictive of the progression of pre-cancer lesions.
              </a>
            </td>
          <td>
            Yupei Lin, Venugopalareddy Mekala, Jianrong Li, Xiang Wang, M. Aminu, Jia Wu, Jianjun Zhang, R. Ripley, Christopher I Amos, Chao Cheng
          </td>
          <td>2025-07-30</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Precise control of cell-cell communication networks within brain neurovascular units (NVUs) promotes normal tissue physiology, and dysregulation of these networks can lead to pathologies including intracerebral hemorrhage (ICH). The cellular and molecular mechanisms underlying ICH development and subsequent tissue repair processes remain poorly understood. Here we employed quantitative single cell RNA sequencing coupled with spatial in situ gene expression profiling to characterize NVU signaling pathways associated with ICH in neonatal mouse brain tissue. The initial stages of ICH pathogenesis are characterized by downregulation of extracellular matrix (ECM)-associated signaling factors (Adamtsl2, Htra3, and Lama4) that functionally connect to canonical TGFβ activation and signaling in vascular endothelial cells. Conversely, the progressive resolution of ICH involves upregulation of neuroinflammatory signaling networks (Gas6 and Axl) alongside activation of iron metabolism pathway components (Hmox1, Cp, and Slc40a1) in astrocytes and microglial cells. Integrated computational modeling identifies additional ligand-receptor signaling networks between perivascular glial cells and endothelial cells during both ICH pathogenesis and resolution. Collectively, these findings illuminate the molecular signaling networks that promote NVU maturation and provide novel mechanistic insights into the pathways controlling ICH pathogenesis and repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c716d99938660f993271f57064d7c4a3482d5880" target='_blank'>
              Single Cell RNA Sequencing and Spatial Profiling Identify Mechanisms of Neonatal Brain Hemorrhage Development and Resolution
              </a>
            </td>
          <td>
            Santiago A. Forero, Zhihua Chen, Ali Pirani, Arpan De, Zachary Wise, John E. Morales, Joseph H. McCarty
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Silicosis is a progressive fibrotic lung disease without effective treatment options, and its pathogenesis remains incompletely understood, particularly the role of endothelial cells (ECs). Methods Here, we utilized single-cell RNA sequencing to characterize endothelial responses in lungs from silica-exposed mice. Results We identified two functionally distinct endothelial subpopulations: 1. An inflammatory EC subtype, exhibiting significantly increased abundance and characterized by high expression of neutrophil-recruiting factors such as Spp1 (osteopontin), CCL (C-C motif chemokine ligand), and ESAM (endothelial cell–selective adhesion molecule), suggesting active involvement in neutrophil influx and persistent inflammation. 2. A reparative EC subtype, marked by upregulation of angiogenesis and vascular repair pathways, which exhibited decreased abundance and functional suppression within the silicotic lung microenvironment. Discussion These results indicate a pathological shift toward inflammation-amplifying endothelial cells and impaired reparative capacity during silicosis progression. Our findings provide new mechanistic insights into endothelial cell dysfunction in silicosis and highlight potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c99a0e94845d8f93e9c36e4e7472783551642f63" target='_blank'>
              Single-cell transcriptomics reveals predominantly inflammatory endothelial cell responses and suppressed vascular repair in silicosis
              </a>
            </td>
          <td>
            Dongli Cao, Huashun Cui, Zequan Chen, Heyang Li, Bing Li, Jianhua Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cardiac issues following radiotherapy are increasingly prevalent among patients with thoracic cancer and coronary disease. However, the mechanisms underlying radiotherapy-induced plaque instability and changes in plaque characteristics on imaging remain unclear. This study used single-cell RNA sequencing to identify key features of vulnerable plaques following radiotherapy.


METHODS
We applied dual-mode synchronized optical coherence tomography-intravascular ultrasound imaging in a rabbit model to reveal the characteristics of vulnerable plaques at various time points postradiotherapy. We then conducted single-cell RNA sequencing on atherosclerotic lesions from ApoE-/- (apolipoprotein E-/-) mice across 3 stages: nonirradiated, early irradiation, and advanced irradiation. Bioinformatics was used to analyze cell populations, trajectories, and interaction differences among the groups. This was followed by pathological staining, flow cytometry, gene knockout, and antibody depletion.


RESULTS
In vivo optical coherence tomography-intravascular ultrasound imaging at the initial stage accurately evaluated radiation exposure's effect on the stability of rabbit carotid plaques. High-throughput single-cell transcriptomics mapped the radiotherapy-induced immune system remodeling in atherosclerotic lesions, and identified a granzyme K+ CD8+ (cluster of differentiation 8+) T-cell subpopulation. The infiltrating cells highly expressed the cytotoxic molecule GZMB (granzyme B) in the plaques. The absence of CD8+ T cells or GZMB ameliorated radiotherapy-exacerbated atherosclerotic lesions. This demonstrates the pathogenic function of GZMK+ (granzyme K+) CD8+ cytotoxic T cells in promoting radiotherapy-induced atherosclerotic lesion progression.


CONCLUSIONS
We identified characteristic arterial imaging changes resulting from radiation injury, accurately and dynamically assessing the effects of radiation exposure on plaque stability. Radiation exposure affected plaque stability by modulating GZMK+ CD8+ T-cell activation and GZMB function, suggesting GZMB as a mediator of radio-cardiovascular injury and a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ade88282c0b97eb135a560a965611a6206d8944e" target='_blank'>
              Single-Cell RNA Sequencing Reveals Granzyme K+ CD8+ T Cells as a Target for Mitigating Plaque Instability Following Radiation.
              </a>
            </td>
          <td>
            Manru Shi, Quan Lin, Shaohong Fang, Cheng Ma, Weiwei Xu, Di Sun, Naixin Wang, J. Cui, Zhonghua Tong, Yuwu Chen, Xing Luo, Sining Hu, Tuo Huang, Yunlong He, Yang Cao, Jiayin Liu, Jiaqi Li, Anxin Gu, Xiuwen Lan, Meng Chi, Chao Wang, Jiaqi Gu, Siqi Tian, Jiaao Liu, Zixuan Liu, Maomao Zhang, M. E
          </td>
          <td>2025-09-05</td>
          <td>Journal of the American Heart Association</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>