<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-06 09:30:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17117-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, , James Cai, , S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, , D. Sterman, , Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized clinical oncology; however, the inherent complexity and heterogeneity of cancer present substantial challenges to achieving broad therapeutic efficacy. Tumor heterogeneity manifests not only among different patients but also within individual tumors, further complicating personalized treatment approaches. Single-cell sequencing technologies encompassing genomics, transcriptomics, epigenomics, proteomics, and spatial omics have significantly enhanced our ability to dissect tumor heterogeneity at single-cell resolution with multi-layered depth. These approaches have illuminated tumor biology, immune escape mechanisms, treatment resistance, and patient-specific immune response mechanisms, thereby substantially advancing precision oncology strategies. This review systematically examines recent advances in single-cell multi-omics technologies across various cancer research areas, emphasizing their transformative impacts on understanding tumor heterogeneity, immunotherapy, minimal residual disease monitoring, and neoantigen discovery. Additionally, we discuss current technical and analytical limitations and unresolved questions associated with single-cell technologies. We anticipate single-cell multi-omics technologies will become central to precision oncology, facilitating truly personalized therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3005de124c813e8dd9ae4675ad8451e32f8f1a34" target='_blank'>
              Single-cell multi-omics in cancer immunotherapy: from tumor heterogeneity to personalized precision treatment
              </a>
            </td>
          <td>
            Jiayuan Le, Yating Dian, Deze Zhao, Ziyu Guo, Zehao Luo, Xiang Chen, Furong Zeng, Guangtong Deng
          </td>
          <td>2025-08-25</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew M Pardoll, Christina Jackson, Nancy R Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Abstract Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0987fea6e48fd7cd3a2b34fbb805d81b9c4a798" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, Divij Mathew, Mingshuang Wang, Xinyi Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Intrinsic genetic alterations and dynamic transcriptional changes contribute to the heterogeneity of solid tumors. Lung adenocarcinoma (LUAD) is characterized by its significant histological, cellular and molecular heterogeneity. The present study aimed to study the spatial transcriptomics of primary LUAD with initial hopes to decipher molecular characteristics of subtype transitions in LUAD progression, offering new insights for novel therapeutic strategies. Spatial transcriptomics libraries were first generated from tumor samples collected from patients with LUAD who underwent surgical resection in The Fourth Hospital of Hebei Medical University in 2022 and were sequenced using Illumina NovaSeq 6000 system. The processed data were analyzed for differential gene expressions and networks, and were annotated according to cell type, spatial ligand-receptor interaction and trajectory inference. Our analysis revealed 34 annotated cell types, with cancer-associated fibroblasts (CAFs) being the most abundant, playing a crucial role in tumor microenvironment remodeling and prognosis. We noted significant spatial correlations between various immune cells and found that different histological subtypes displayed unique cell composition profiles, particularly in the micropapillary subtype, which exhibited higher macrophage proportions and distinct gene expression pathways related to extracellular matrix organization and receptor tyrosine kinase signaling. Additionally, we explored the dedifferentiation states within these subtypes, identifying that region with higher dedifferentiation scores corresponded to increased tumor invasiveness and potential drug resistance. Our findings demonstrate dynamic biological changes and dedifferentiation states of tumor subtypes during the progression process. This study reveals important biological processes in tumor development and may offer valuable guidance for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a252a3c50c2377c8fca1733740eaa8f4897e893" target='_blank'>
              Spatial transcriptomics uncovers immune-cell plasticity and dedifferentiation signatures in aggressive lung adenocarcinoma subtypes
              </a>
            </td>
          <td>
            Huiyan Deng, Qingyi Liu, Ziqiang Tian, Guan-Yuan Liu, Shize Wang, Di Liang, Yun Wang, Yaqing Han, Shaonan Xie
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18264-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is an advanced technology that allows for the study of gene activity in cancer tissues while preserving the original tissue structure. Unlike traditional methods that either analyze samples without location information or examine individual cells removed from their natural environment, spatial transcriptomic technology maps where genes are active within intact tumor tissue. This approach enables a greater understanding of how different areas of a tumor vary, how cancer cells interact with surrounding healthy tissue, and how cells are organized spatially within the tumor at high resolution. As spatial technology becomes more widely used, it may improve cancer diagnosis and treatment by helping to create more personalized and effective therapies for individual patients. This review outlines the current and future role of spatial transcriptomics methodologies in the context of genitourinary cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa8ee089aa47992263f8b69f0d5705b49d03f797" target='_blank'>
              Current Role and Future Frontiers of Spatial Transcriptomics in Genitourinary Cancers
              </a>
            </td>
          <td>
            Firas Hatoum, Adnan Fazili, Justin W Miller, Xuefeng Wang, Xiaoqing Yu, Xin Lu, Jeffrey S. Johnson, Philippe E. Spiess, J. Chahoud
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cellular anatomy and signaling vary across niches, which can induce gradated gene expressions in subpopulations of cells. Such spatial transcriptomic gradient (STG) makes a significant source of intra-tumor heterogeneity. We present Local Spatial Gradient Inference (LSGI), a computational framework that systematically identifies spatial locations with prominent, interpretable STGs from spatial transcriptomic (ST) data. We demonstrate LSGI in tumor ST datasets and identify pan-cancer and tumor-type specific pathways with gradated patterns, highlighting the ones related to spatial transcriptional intratumoral heterogeneity. LSGI enables interpretable STG analysis, which can reveal novel insights in tumor biology from the increasingly reported tumor ST datasets. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03716-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2919ee7022acd5a65c198642bf710ee21f0432" target='_blank'>
              LSGI: interpretable spatial gradient analysis for spatial transcriptomics data
              </a>
            </td>
          <td>
            Qingnan Liang, Luisa M. Solis Soto, C. Haymaker, Ken Chen
          </td>
          <td>2025-08-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. Methods We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. Results We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Conclusions Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Meningioma is a common primary intracranial tumor with high recurrence and metastasis rate. Delineating the pathological ecosystem at the brain-tumor interface (BTI) of meningioma is critical for understanding the mechanisms of tumor metastasis and developing effective new therapies. Methods To identify biomarkers of early metastasis and discover potential therapeutic targets, we integrated single-cell transcriptome datasets of meningioma, and identified the cell populations and molecular signatures uniquely present at the BTI. Results A specific BTI-enriched tumor cell population with a pro-EMT (epithelial mesenchymal transition) characteristics was associated with invasion and metastasis, and ANXA2 and COL5A1 were detected as the biomarkers for these BTI-enriched tumor cells. Additionally, we characterized the BTI-specific immunosuppressive microenvironment composed of SPP1+ tumor-associated macrophages, as well as specific endothelial cells (ACKR1high) and pericytes (THY1high) promoting the highly malignant invasive state of angiogenesis. Conclusions Collectively, BTI in meningioma is a metastatic and immunosuppressive zone. We have discovered potential biomarkers that help detect early metastasis and recurrence of meningioma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06935-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea39ae7cd89d7f388a32faad30c0efc42673529" target='_blank'>
              Single-cell transcriptomic analysis reveals metastatic and immunosuppressive characteristics in meningioma brain-tumor interface
              </a>
            </td>
          <td>
            Boya Huang, Jinlian Liang, Xiaogen Tang, Yue Zhu, Lijuan Gao, Dongwei Fu, Cheng Wei, Yifang Li, Chao Ke, Hongyi Zhang, Oscar Junhong Luo
          </td>
          <td>2025-08-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment responses remain highly variable due to the complexity and heterogeneity of the tumor microenvironment. The tumor microenvironment comprises malignant, immune, stromal, and endothelial cells, along with extracellular matrix and soluble factors, all organized into spatially distinct communities that evolve dynamically throughout tumor progression and therapy. These spatial structures orchestrate tumor behavior, immune evasion, and drug resistance. Recent breakthroughs in spatial omics technologies, including spatial transcriptomics and spatial proteomics, have enabled high-resolution, multiplexed mapping of tissue architecture and molecular characteristics. These technologies provide valuable insights into how the spatial organization of cells and signaling networks within the tumor microenvironment influences therapeutic efficacy. Notably, specific structures, such as tertiary lymphoid structures, fibroblast-mediated stromal barriers, and vascular heterogeneity have been identified as spatial determinants of treatment response. By delineating cellular communities and their interactions, spatial omics technologies can reduce intratumoral complexity into clinically interpretable modules. This review summarizes the diversity of these spatial structures and their relationships with treatment outcomes in immunotherapy, chemotherapy, radiotherapy, and targeted therapy. In addition, it highlights present challenges in data integration, analytical standardization, and functional validation, and discusses future directions for incorporating spatial omics technologies into precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572818b2484643c196593baace3df887c52839e" target='_blank'>
              Structural cell communities in the tumor microenvironment: Spatial determinants of therapeutic response
              </a>
            </td>
          <td>
            Xuan Cui, Y. Ni, Xia Lei, Lan Zhao, Cheng Qian, Juanjuan Shan
          </td>
          <td>2025-08-14</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background The presence of tertiary lymphoid structures (TLS) in solid tumors, including Merkel cell carcinoma (MCC), is associated with a better prognosis and a better response to immunotherapy with immune checkpoint inhibition (ICI). The detailed mechanisms by which TLS influence antitumor immune responses are only partially understood. Methods Clinically annotated tumor tissues of 27 patients with MCC were obtained prior to ICI therapy. Tumor samples were subjected to transcriptomic and multiplex immuno-visual profiling, T-cell receptor (TCR) clonotype mapping, as well as—in selected cases—spatial transcriptomics to comprehensively characterize the tumor immune microenvironment. Results Weighted gene co-expression network analysis (WGCNA) of transcriptomic data in combination with topological overlap measures indicated a higher abundance of TLS in tumors of patients with MCC responding to ICI therapy. This concept was substantiated through immunomorphological analyses, revealing mature B-cell follicle-like structures characterized by high endothelial venules (HEVs). Further supporting HEVs as critical entry points for naïve T cells, the presence of TLS was correlated with a pronounced infiltration of CD4+ and CD8+ T cells, exhibiting both naïve and central memory phenotypes. The TCR repertoire of these infiltrates exhibited enhanced richness and diversity with a pronounced reactivity toward Merkel cell polyomavirus-derived T-cell epitopes. Spatially resolved RNA and V(D)J sequencing revealed the expression of genes associated with T-cell recruitment within TLS, alongside the presence of naïve and central memory T-cell markers. Notably, individual clonally expanded TCR transcripts were detected both within TLS and among tumor-infiltrating lymphocytes. The latter were associated with low expression of memory cell markers and high expression of effector cell markers. Additionally, a spatial gradient in the expression of genes linked to immune stress in MCC cells—such as those involved in the interferon-γ response and antigen processing and presentation machinery—originated in proximity to the TLS. Conclusion Our findings are consistent with a key role of TLS in shaping immune interactions within the MCC microenvironment, driving the recruitment of diverse tumor-reactive T cells. These insights hold promise for advancing immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13de7f3d541927495612436f916863e4107c4925" target='_blank'>
              Tertiary lymphoid structures in Merkel cell carcinoma facilitate naïve and central memory T-cell infiltration linked to immunotherapy response
              </a>
            </td>
          <td>
            Nalini Srinivas, Ivelina Spassova, Kuan Cheok Lei, Jiwei Gao, María José Pino, Giovanni Giglio, S. Kitanovski, Mazdak Dalkoohi, E. Livingstone, Ulrike Leiter, Peter Mohr, T. Gambichler, Ingo Stoffels, Selma Ugurel, Phyllis Fung-Yi Cheung, Camilla Engblom, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract High‐risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single‐cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy‐treated high‐risk neuroblastomas from archived, formalin‐fixed, paraffin‐embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p‐gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical‐like cells expressed several oncogenic ligand‐encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of Pathology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-02</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C. Williams, Miguel A Vazquez, John O'Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeffrey B. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, T. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Fibroblasts are critical mediators of tumor progression and metastasis; however, their heterogeneity and specific functions in the context of colorectal cancer (CRC) liver metastases remain incompletely understood. Methods We performed single-cell RNA sequencing (scRNA-seq) analysis on relevant tissue samples. Subsequently, we validated our findings using immunofluorescence assays on histological slides from primary CRC with liver metastasis (mCC) and liver metastatic tumors (mLC). Further investigations included transcriptomic profiling, pseudotime trajectory analysis, multiplex immunofluorescence staining, and cell-cell communication analysis. Results Our scRNA-seq analysis identified a distinct PRELP-positive cancer-associated fibroblast (CAF) subtype that is associated with liver metastasis and tumor progression. These PRELP+ CAFs were predominantly enriched in mLC, less abundant in mCC, and rare in non-metastatic CRC (nCC). This distribution was confirmed by immunofluorescence. Transcriptomically, PRELP+ CAFs exhibit a unique signature defined by extracellular matrix components (e.g., PRELP, COLEC11, ITGBL1) and the activation of pro-tumor pathways such as TGF-β and Wnt signaling. Pseudotime analysis indicated they represent a terminal fibroblast differentiation state. Spatially, they colocalize with immune cells (T cells, B cells, plasma cells), and communication analysis suggests they foster an immunosuppressive microenvironment via APP-CD74 and collagen-CD44 signaling, thereby promoting immune evasion. The transcription factors NR2F2, JUN, and JUND were identified as key regulators of this CAF subtype. Discussion These findings provide crucial new insights into fibroblast heterogeneity within CRC liver metastases. We characterize PRELP+ CAFs as a specialized, terminally differentiated fibroblast population that contributes to immunosuppression and tumor progression, highlighting them as a potential therapeutic target for inhibiting metastatic advancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62bf8481f437e703e061482f9031f26e35dec625" target='_blank'>
              Characterization of the phenotype and function of PRELP+ fibroblast subtype in liver metastatic colorectal cancer
              </a>
            </td>
          <td>
            Yuting Dai, Xingying Huang, Min Sun, Shiyu Zhang, Weiqiang Yu, Kongwang Hu, Qiang Wu, Qingfa Wu
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey I. Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Immunotherapy has been used in the clinical management of TNBC. While BRCA1 mutations are associated with immunotherapy response, the therapeutic outcomes in TNBC patients are not promising. Methods This study integrated spatial, single-cell, and bulk RNA-seq data to explore the role of BRCA1 in reshaping the TNBC microenvironment. Through multi-scale analysis, phenotype changes and potential biomarkers in cancer-associated fibroblasts (CAF) were identified. To validate these findings at the protein level, we employed high-resolution, label-free proteomics sequencing in our in-house cohort, providing critical real-world validation. A predictive system for response to ICIs was constructed through the step-by-step machine learning pipeline. Results Compared to BRCA1 mutant patients, BRCA1 wild-type patients experienced increased T-cell exhaustion and dendritic cell tolerance. We identified a MEG3+ pre-CAF subgroup via pseudo-time analysis. Moreover, ISG15 may serve as an immunoregulatory biomarker, and the proposed predictive model demonstrated potential in forecasting immunotherapy response, although further validation is needed. Conclusions This study highlighted the cellular heterogeneity of TNBC and identified ISG15 as a candidate biomarker potentially associated with treatment response. The ISG15-based predictive system might provide a robust framework for predicting ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125312e82f26c9a44c81f1fdf19e7985af609a32" target='_blank'>
              Resolving tumor microenvironment heterogeneity to forecast immunotherapy response in triple-negative breast cancer through multi-scale analysis
              </a>
            </td>
          <td>
            Shihao Sun, Shuang Chen, Kaiyuan Li, Ge Zhang, Nan Wang, Yijia Xu, Xinxing Wang, Jiangrui Chi, Lin Li, Yi Sun
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, , Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Summary Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34+ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV+ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J Geuenich, , Barbara Gruenwald, A. Borgida, , Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Gliosarcoma (GS) is a rare variant of glioblastoma (GBM), characterized by biphasic glial and sarcomatous histology and poor prognosis. Despite its distinct clinical features, GS remains underrepresented in glioma research due to the lack of biologically faithful and experimentally tractable models. Methods We established patient-derived gliosarcoma organoids (GSOs) from freshly resected tumors using a suspension culture system without enzymatic dissociation. Histological, molecular, and functional properties were evaluated using H&E staining, immunohistochemistry, whole-exome sequencing, 3D invasion assays, and single-cell RNA sequencing (scRNA-seq). Drug response assays were performed using temozolomide and ANA-12 (NTRK2 inhibitor). Results GSOs preserved key histological features, genetic alterations, and tumor microenvironmental cell populations from the original tumors. They stable growth and viability over extended culture periods and maintained integrity following cryopreservation and recovery, supporting their long-term utility. 3D invasion assays further revealed infiltrative behavior, consistent with the aggressive nature of GS. Histological and molecular analyses revealed that GSOs retained glial and mesenchymal differentiation, diverse non-malignant stromal cells, and case-specific somatic alterations. Comparative scRNA-seq revealed distinct transcriptional programs: GSOs were enriched for fibroblast-like and oligodendrocyte progenitor-like states, while glioblastoma organoids (GBOs) displayed astrocyte-like differentiation and high connectivity signatures. Functional assays confirmed consistent sensitivity of GSOs to temozolomide, and selective therapeutic response to NTRK2 inhibition was observed in the GSO harboring an NTRK2 alteration, supporting the utility for genomic-context-guided therapy. Conclusion Together, GSOs constitute a tractable and translationally relevant model that faithfully reflects the cellular complexity, genetic landscape, and tumor-specific heterogeneity. This model addresses a critical gap in GS biology and supports its integration into precision oncology. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06952-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6998dea80dedd424ac624050d4245e9964a32790" target='_blank'>
              Tumor microenvironment-preserving gliosarcoma organoids as an in vitro preclinical platform: a comparative analysis with glioblastoma models
              </a>
            </td>
          <td>
            Junseong Park, Dokyeong Kim, Hyeon-Chun Park, Min-Gyoung Park, Songzi Zhang, Okcho Na, Youn Soo Lee, Minho Lee, Stephan Ahn, Yeun-Jun Chung
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. M. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Significance Androgen deprivation therapy (ADT) is a mainstay in prostate cancer (PCa) treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer directions for PCa modeling and pave the way toward therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b171b1d175becacb6af2e7a071be6d1fa5ee752f" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>169</td>
        </tr>

        <tr id="Abdominal aortic aneurysm (AAA) is a life-threatening vascular disease with no effective pharmacological interventions. While single-cell transcriptomics has advanced our understanding of AAA, it lacks spatial context. Here, we employed Seq-Scope, an ultra-high-resolution spatial transcriptomic technology, to decipher the spatial landscape of angiotensin II–induced AAA in Apoe–/– mice. Our analysis revealed the heterogeneity of macrophages, fibroblasts, and smooth muscle cells (SMCs), with specific responses in different layers of the AAA tissue. SMCs in the inner layers showed associations with Mgp-expressing fibroblasts and GPNMB-expressing macrophages, whereas the outer layers had different dominant cell types. Notably, GPNMB-expressing macrophages were concentrated near SMCs in regions of severe elastic lamina damage. Immunofluorescent staining confirmed their colocalization, and scRNA-seq reanalysis independently validated the presence of GPNMB-high macrophages in AAA tissues, highlighting their involvement in inflammation and tissue remodeling. Moreover, we discovered that macrophage-derived soluble GPNMB induces SMC phenotypic switching, reducing contractile markers while increasing cytokines and metalloproteinases. This effect was partly mediated by CD44 signaling. These findings suggest that GPNMB-high macrophages contribute to AAA development by driving SMC dysfunction. This study highlights the importance of high-resolution spatial transcriptomics in complementing single-cell transcriptomics, offering valuable insights into molecular and cellular responses in the AAA microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5eda91d1a78d03cc2b4d2c27c04a3ac82f92378" target='_blank'>
              Single-cell spatial transcriptomics unravels the cellular landscape of abdominal aortic aneurysm
              </a>
            </td>
          <td>
            Guizhen Zhao, Chun-Seok Cho, Hongyu Liu, Yongha Hwang, Yichen Si, Myungjin Kim, Yongjie Deng, Yang Zhao, Chao Xue, Yanhong Guo, Lin Chang, D. Mizrak, Bo Yang, H. Kang, Jifeng Zhang, Jun Hee Lee, Y. Chen
          </td>
          <td>2025-08-22</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: Primary hepatocellular carcinoma (HCC) and liver metastases differ in origin, progression, and therapeutic response, yet a direct high-resolution spatial comparison of their tumor microenvironments (TMEs) within the liver has not previously been performed. Methods: We applied high-definition spatial transcriptomics to fresh-frozen specimens of one HCC and one liver metastasis (>16,000 genes per sample, >97% mapping rates) as a proof-of-principle two-specimen study, cross-validated in human proteomics and patients’ survival datasets. Transcriptional clustering revealed spatially distinct compartments, rare cell states, and pathway alterations, which were further compared against an independent systemic dataset. Results: HCC displayed an ordered lineage architecture, with transformed hepatocyte-like tumor cells broadly dispersed across the tissue and more differentiated hepatocyte-derived cells restricted to localized zones. By contrast, liver metastases showed two sharply compartmentalized domains: an invasion zone, where proliferative stem-like tumor cells occupied TAM-rich boundaries adjacent to hypoxia-adapted tumor-core cells, and a plasticity zone, which formed a heterogeneous niche of cancer–testis antigen–positive germline-like cells. Across both tumor types, we detected a conserved metabolic program of “porphyrin overdrive,” defined by reduced cytochrome P450 expression, enhanced oxidative phosphorylation gene expression, and upregulation of FLVCR1 and ALOX5, reflecting coordinated rewiring of heme and lipid metabolism. Conclusions: In this pilot study, HCC and liver metastases demonstrated fundamentally different spatial architectures, with metastases uniquely harboring a germline/neural-like plasticity hub. Despite these organizational contrasts, both tumor types converged on a shared program of metabolic rewiring, highlighting potential therapeutic targets that link local tumor niches to systemic host–tumor interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb3ca762aa10606e657e9aeaed628e45b17550f" target='_blank'>
              Spatial Transcriptomics Reveals Distinct Architectures but Shared Vulnerabilities in Primary and Metastatic Liver Tumors
              </a>
            </td>
          <td>
            S. Adapa, Sahanama Porshe, Divya Priyanka Talada, T. Nywening, Mattew L. Anderson, Timothy I. Shaw, 
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina E. Calderon, Ian M Loveless, Oliver G. McDonald, Jennifer P Morton, David Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a tumor microenvironment (TME) composed of a dense extracellular matrix, cancer-associated fibroblasts (CAFs), vasculature, neural elements, and immune cell populations. This complex network promotes tumor proliferation, invasion, metastasis, and resistance to immunotherapy and chemotherapy. The microenvironmental characteristics of the various PDAC subtypes are discussed in this review. And we examines the role of cancer cells in the TME, highlighting their ability to manipulate stromal components to serve as collaborators in tumor progression. Furthermore, we explored the formation mechanism of the immunosuppressive microenvironment in PDAC, paying attention on Inflammation and intrinsic genetic alterations, the regulatory effect of metabolic reprogramming, the contribution of CAFs and the role of immune cells in cancer cell metastasis. This review shows the role of soluble molecules and exosomes in facilitating PDAC progression and immune evasion within the microenvironment. In conclusion, we outline the novel therapeutic strategies that focus on the interaction between cancer cells and their microenvironment, with the objective of offering new insights for future precision medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e8295ec7fa4434bd00f48ee0333dca0ffe9c46" target='_blank'>
              Domestication and feedback: bidirectional hijacking in pancreatic ductal adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Yuxi Qiao, Haobo Yin, Yong Zhang, Jingdong Zhang, Qian Dong
          </td>
          <td>2025-08-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Malignant rhabdoid tumors (MRTs) are aggressive pediatric cancers with poor outcomes. MRTs exhibit low tumor mutational burden, yet recent studies reported immune cell infiltration. Here, we used spatial transcriptomics and multi-omic profiling to study immune cell infiltration in MRT samples. We observed a diverse repertoire of candidate tumor antigens (TAs), IRF1 signaling activity and elevated expression of antigen processing and presentation genes, strongly associated with a “hot” tumor immune microenvironment (TIME). Upregulation of factors involved in skeletal muscle development was observed in some MRTs with higher CD8 + T cell infiltration and the ratio of M1/M2 macrophages was positively correlated with cytotoxic lymphocyte infiltration. We identified genes, such as SPP1, preferentially expressed in tumor-associated macrophages (TAMs) and noted the MITF regulatory network appeared active in the M2-like TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248c19c84a73e2abe2ce78fd660b38edcf7c75f9" target='_blank'>
              Spatial and multi-omic profiling reveals genes and pathways associated with cytotoxic lymphocyte infiltration in malignant rhabdoid tumor
              </a>
            </td>
          <td>
            Marco A. Marra, Dan Jin, Joshua Dubland, E. Mullen, James Geller, J. Bush
          </td>
          <td>2025-08-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of tumor cells, stromal cells, infiltrating immune cells and non-cellular components such as extracellular matrix, blood vessels and a wide variety of secreted proteins. Evidence shows that beyond supporting tumor growth, the TME also promotes tumor cell proliferation and invasion and contributes to treatment resistance, ultimately affecting patient prognosis. Cell-to-cell communication within the TME is driven by secreted proteins such as cytokines, chemokines, growth factors and interferons, which are produced not only by tumor cells but also by various stromal cells and immune cells. These proteins form a complex signaling network that promotes tumor cell proliferation and invasion and enables tumors to evade innate and adaptive immune responses. Antibody arrays are a technology that can simultaneously screen hundreds of secreted proteins in complex biological samples, aiding in the exploration of this complex signaling network. By combining high-throughput multiplex immunoassays such as antibody arrays with cellular and molecular biology techniques, researchers have uncovered complex regulatory mechanisms of cytokine networks within the TME. The present review summarized recent findings on the communication between tumor cells and the TME, as well as key secreted proteins essential for tumor progression and the development of therapeutic resistance. In addition, it discusses how high-throughput antibody arrays contribute to our understanding of regulatory networks of secreted proteins in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84b1200ba5d18e4e2abf23d107f5f3c2ed9e0f6" target='_blank'>
              Unraveling tumor cell-tumor microenvironment crosstalk through antibody array technologies (Review)
              </a>
            </td>
          <td>
            Yanlin Wang, Shuhong Luo, Hua Dong, Ruo-Pan Huang
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36afed9d0acad24d10324fa459b1bfa3a197ed1" target='_blank'>
              Abstract A010: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) is a complex network driving endometrioid endometrial cancer (EC) progression. Previous analyses of the EC TME were often limited by a lack of detailed cellular annotations. Integrating single-cell RNA sequencing (scRNA-seq) data offers a comprehensive approach to reconstruct the TME, aiming to uncover novel mechanisms and therapeutic targets. Methods We integrated public scRNA-seq data from 15 EC and 5 normal endometrium samples. Through detailed cell annotation, we performed comprehensive bioinformatic analyses, including pseudotime trajectories, copy number variation, and cell communication, to investigate mechanisms of EC development. Results We identified nine cell types and characterized subpopulations of epithelial, macrophage, lymphocyte, and stromal fibroblast cells. The SOX9+LGR5- epithelial subtype showed elevated malignancy and NFKB pathway enrichment. The M2_like2 macrophage subtype played a critical role, engaging in robust MIF-(CD74+CD44) mediated communication with SOX9+LGR5- cells. Experimental validation confirmed MIF co-expression with E-cadherin in EC tissues. Furthermore, the transcription factor NFKB2 was found to mediate MIF’s effect on the CD44 receptor in malignant epithelial cells. A pericyte-to-fibroblast transition in stromal cells may also support tumor growth, while an increase in CD8 exhausted/Treg cells and a decrease in cytotoxic CD8 cells suggest potential immune evasion. Conclusion Our single-cell analysis details the EC TME landscape, revealing robust communication between M2_like2 macrophages and SOX9+LGR5- epithelial cells. We highlight a key mechanism where NFKB2 mediates MIF’s pro-tumorigenic effects via the CD44 receptor, offering new insights into EC progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d976acc17e07f02157c06df3f07e8d3258236f" target='_blank'>
              Single-Cell Transcriptomic Analysis Highlights the Impact of NFKB2-Mediated MIF-CD44 Signaling Axis in Endometrioid Endometrial Cancer
              </a>
            </td>
          <td>
            Lu Zhang, Mengjie Yang, Quan Zhang, Yiqian Zhang, Qiyuan Li, Qionghua Chen
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Women's Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Medullary thyroid carcinoma (MTC) is a rare neuroendocrine tumor arising from parafollicular C-cells in the thyroid. Unlike other neuroendocrine tumors, knowledge about the MTC tumor microenvironment and its role in therapy resistance remains limited, mainly due to the lack of reliable in vitro models. We hypothesized that, as observed in other cancer types, MTC cells may exhibit a degree of phenotypic plasticity that could be modulated by various microenvironmental factors.



 The primary aim of this study was to establish patient-derived MTC in vitro models capable of growing under different three-dimensional (3D) culture conditions while retaining the ability to shift toward a more mesenchymal, stem-like phenotype.



 We successfully established eight patient-derived primary MTC cell lines from tumors with different genetic backgrounds and variable degrees of aggressiveness. These lines were characterized in terms of phenotype and responses to different microenvironmental stimuli and targeted therapies through Western Blot, confocal microscopy, ELISA assays, ELDA, RT-qPCR, and proliferation assays.



 Notably, the cells maintained their plasticity over extended passaging when cultured under stem-promoting conditions. In differentiation media, they preserved the ability to secrete high levels of classic MTC markers, such as calcitonin and chromogranin A. Under 3D culture conditions, all cell lines showed a significant increase in stem and neuroendocrine precursor markers, including SOX2, OCT4, TBB3 and FOXA1, along with spatially dependent shifts in Epithelial-to- Mesenchymal Transition (EMT) markers E-cadherin and Vimentin. These phenotypic variations partially recapitulated the molecular alterations observed in MTC tissue samples. Moreover, changes in culture conditions—such as inducing a pseudo- hypoxic state or culturing in non-adherent environments—profoundly affected both cell phenotype and sensitivity to therapeutic agents currently available for MTC treatment, including Pralsetinib, Selpercatinib, Cabozantinib, Lenvatinib, and Vandetanib.



 In conclusion, we present here the generation of a new valid patient- derived in vitro model that can provide new opportunities to explore MTC biology complexity and will allow future translational studies, including those involved in uncovering the mechanisms underlying the insurgence of therapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c689353bda0dac4479e679a592360fdc7492a4" target='_blank'>
              OC48 - Generation of patient-derived in vitro models to unravel medullary thyroid cancer microenvironment and therapy resistance
              </a>
            </td>
          <td>
            V. Ghiandai, V. Cirello, G. Gazzano, G. Dionigi, C. Colombo, L. Fugazzola, L. Persani, E. S. Grassi
          </td>
          <td>2025-09-01</td>
          <td>European Journal of Endocrinology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr PR-11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11ba138005b19c5e9d04c0e07daa4c79af69fe8" target='_blank'>
              Abstract PR-11: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gliomas represent the most frequent primary brain tumors in adults and are marked by considerable molecular and clinical heterogeneity. The 2021 WHO classification of central nervous system tumors identifies three major adult-type diffuse glioma subtypes: astrocytoma IDH-mutant, oligodendroglioma IDHmutant with 1p/19q-codeletion, and glioblastoma IDH-wildtype.1 Despite ongoing advances, treatment resistance and poor prognosis remain major challenges, largely due to intratumoral heterogeneity and the complexity of the tumor microenvironment.2 A promising yet underexplored avenue for patient-centered glioma profiling is the tumor secretome, which includes soluble factors, extracellular vesicles, and metabolites released by both neoplastic and stromal cells.3 Existing studies rely on in vitro models that fail to recapitulate the complexity of tumor-host interactions. To bridge this gap, we established an innovative, standardized ex vivo protocol for secretome isolation directly from glioma tissue. The protocol preserves microenvironmental cues from the operating room to laboratory analysis, ensuring spontaneous secretome release under controlled conditions. Feasibility and performance were assessed in a preliminary cohort of seven adult diffuse glioma patients. Transmission electron microscopy confirmed tissue structural integrity. Proteomic profiling identified 84 cancer-associated proteins within the secretome. Additionally, metabolomic analysis revealed a distinctive signature of small molecules. Notably, glioblastoma IDH-wildtype samples exhibited increased lactate levels, consistent with hypoxia-driven metabolism, and decreased N-acetylaspartate, indicating neuronal loss. Overall, this ex vivo secretome protocol offers a robust and physiologically relevant platform for glioma characterization, with potential applications in biomarker discovery, patient stratification, and the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea6096ea6e8da3b1feda45c80168b298c559e43" target='_blank'>
              P64 | EX VIVO EXTRACTION OF GLIOMA SECRETOME: A NOVEL MICROENVIRONMENT-BASED PROTOCOL FOR METABOLIC AND MOLECULAR PROFILING
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background and Aims The relationship between chronic liver disease and liver cancer remains poorly understood, and treatment options for advanced liver disease remain limited. This study aims to elucidate the dynamic evolution of cellular and molecular alterations from normal liver to diseased liver. Methods Single‐cell RNA sequencing was performed to profile the dynamic cellular variations from normal liver to diseased liver. Bioinformatics analyses were employed to assess tumour heterogeneity and the evolution of tumour microenvironment. Molecular biology validations were performed to investigate the findings. Results Through single‐cell RNA sequencing, we constructed a human liver landscape consisting of more than 130 000 single cells, from normal, cirrhotic tissue, primary liver cancer and paired adjacent tissues. We profiled both inter‐ and intra‐tumour heterogeneity of hepatocellular carcinoma and intrahepatic cholangiocarcinoma. A dual role of the pro‐fibrosis and pro‐carcinogenesis FABP5high macrophage subset was shown to be conserved throughout the progression of liver fibrosis and carcinogenesis; markers, spatial localization, origin, and functional pathways associated with FABP5high macrophages were identified through in vitro and in vivo validation. Metabolic reprogramming of these conserved FABP5high macrophages is implicated in the formation of cirrhotic and cancerous niches. Additionally, we also identified a subset of RGCC+COL4A1+ endothelial cells enriched in HCC tissues, which might experience endothelial‐mesenchymal transition and orchestrate the proliferation and invasion of cancer cells via angiogenesis. Conclusions Our findings provide insights into the evolution from chronic liver disease to liver cancer. These insights will contribute to the development of novel cell subset‐based therapeutics based on immunotherapy and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8b08f4e01c5f59d84102994dc20c75fd98f8a8" target='_blank'>
              A Conserved FABP5high Macrophage Subset Promotes Fibrosis and Carcinogenesis in Advanced Liver Disease
              </a>
            </td>
          <td>
            Wenhao Ge, Lifeng Ma, Yu Huang, W. E., Shujing Lai, Bijun Cui, Jingjing Wang, Yutian Ji, Daiyuan Liu, Song Ye, Yang Kong, Jiarong Zhou, Chengxuan Yu, Yingsheng Wu, Qiang Sun, Dongkai Zhou, Wei Zhu, Yuan Ding, Xiaoping Han, Guoji Guo, Weilin Wang
          </td>
          <td>2025-08-28</td>
          <td>Liver International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Hepatoblastoma (HB), the most common childhood liver cancer, often shows aberrant expression and/or mutations of the β-catenin (B), YAP (Y) and/or NRF2 (N) transcription factors and can be modeled in mouse livers by overexpressing combinations of these (e.g., BY, BN, YN and BYN). Because very few human or murine HB cell lines exist, genetically defined murine HB cell lines would be a significant advance. We describe here the generation of cell lines from primary BN and YN tumors. Unexpectedly, one BN line shows a remarkable ability to trans-differentiate into endothelial cells and can accelerate in vivo tumorigenesis and angiogenesis. These and previously derived BY and BYN lines have similar sensitivities to four drugs used to treat HB. Our generic approach for cell line derivation should allow for the generation of additional lines driven by less common drivers. A collection of such genetically defined and well-characterized cell lines will facilitate studies that are impractical to perform in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31da89c86be409fe01133803515bcf28ea7e0273" target='_blank'>
              Derivation of Genetically Defined Murine Hepatoblastoma Cell Lines with Angiogenic Potential
              </a>
            </td>
          <td>
            Keyao Chen, Ahmet Toksoz, Colin Henchy, Jessica Knapp, Jie Lu, S. Ranganathan, Huabo Wang, E. V. Prochownik
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The immune system comprises multiple cell lineages and subsets maintained in tissues throughout the lifespan, with unknown effects of tissue and age on immune cell function. Here we comprehensively profiled RNA and surface protein expression of over 1.25 million immune cells from blood and lymphoid and mucosal tissues from 24 organ donors aged 20–75 years. We annotated major lineages (T cells, B cells, innate lymphoid cells and myeloid cells) and corresponding subsets using a multimodal classifier and probabilistic modeling for comparison across tissue sites and age. We identified dominant site-specific effects on immune cell composition and function across lineages; age-associated effects were manifested by site and lineage for macrophages in mucosal sites, B cells in lymphoid organs, and circulating T cells and natural killer cells across blood and tissues. Our results reveal tissue-specific signatures of immune homeostasis throughout the body, from which to define immune pathologies across the human lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae04949cbee52fc4de7a7a9f67a67f4325623fd" target='_blank'>
              Multimodal profiling reveals tissue-directed signatures of human immune cells altered with age
              </a>
            </td>
          <td>
            Steven B. Wells, Daniel B. Rainbow, Michal Mark, P. Szabo, Can Ergen, Daniel P. Caron, A. R. Maceiras, Elior Rahmani, Eli Benuck, Valeh Valiollah Pour Amiri, David Chen, Allon Wagner, Sarah K Howlett, L. Jarvis, Karen L. Ellis, Masaru Kubota, Rei Matsumoto, Krishnaa Mahbubani, Kouresh Saeb-Parsy, Cecilia Dominguez Conde, Laura Richardson, Chuan Xu, Shuang Li, L. Mamanova, L. Bolt, Alicja Wilk, Sarah A. Teichmann, Donna L. Farber, Peter A. Sims, Joanne L. Jones, N. Yosef
          </td>
          <td>2025-08-13</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Mucosa-associated invariant T (MAIT) cells represent a unique population of innate-like T lymphocytes capable of detecting non-peptide antigens in the context of monomorphic antigen-presenting molecules. Due to their abundance in barrier tissues, reactivity to local inflammatory cues, and cytotoxic and regulatory functions, MAIT cells are poised to shape the dynamics of various tumor microenvironments. Growing evidence suggests that MAIT cells can exert protumor and/or antitumor effects in cancers arising from or metastasizing to mucosal tissues. However, MAIT cell roles in bladder cancer (BCa) remains unclear. Methods To begin to identify MAIT cells in BCa, we stained bladder tumor biopsies for T cell receptor (TCR) Vα7.2+ cells. We then refined a human MAIT cell signature, which enabled us to interrogate a bulk RNA sequencing dataset and conduct correlation analyses linking intratumoral MAIT cell abundance and mortality from BCa. To extend our work to an in vivo setting, we employed a clinically relevant mouse model in which Mr1+/+ B6-MAITCAST (MAIT-sufficient) and Mr1−/− B6-MAITCAST (MAIT-deficient) mice were exposed to N-butyl-N-(4-hydroxybutyl)nitrosamine, a chemical carcinogen associated with tobacco smoke. In additional experiments, MAIT cells were functionally removed through acetyl-6-formylpterin (Ac-6-FP) administration. Effector and regulatory cell types were phenotyped by flow cytometry, and BCa tumor burden and progression were assessed by MRI and/or H&E and Ki67 staining. Results TCR Vα7.2+ cells were readily detectable in several BCa biopsies, and our bioinformatic analyses correlated heavier MAIT cell presence in BCa tumors with poorer overall survival. Similarly, we found higher tumor burdens in Mr1+/+ B6-MAITCAST mice than in Mr1−/− or Ac-6-FP-treated animals. Bladder MAIT cells from tumor-bearing mice exhibited phenotypic MAIT17 bias based on transcription factors they harbored along with increased interleukin-17A and tumor necrosis factor-α production capacities upon stimulation. Finally, FoxP3+ regulatory T (Treg) cell frequencies were elevated in Mr1+/+ mouse bladder tumors, likely contributing to an immunosuppressive tumor microenvironment, a finding that could be recapitulated in our transcriptomic studies on human BCa. Conclusions MAIT cells are abundant in BCa tumor microenvironments where they potentiate Treg cell accumulation and play protumor roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904ce037c95ea168f087ef993f969eaddff0121a" target='_blank'>
              MAIT cells promote cancer progression and regulatory T cell accumulation in bladder tumor microenvironment
              </a>
            </td>
          <td>
            Inbar A Habaz, Xuejin Ou, Nicole I Wang, Angela Schincaglia, Valeryia Shydlouskaya, Agetha Mahendran, F. Martinez-Santiesteban, Dylan Doolabi, Yige Bao, Ahmad R Movasseghi, Julius Haruna, Nicholas E. Power, Timothy J Scholl, Melissa J Huynh, Alp Sener, S. M. Haeryfar, Dr S M Mansour Haeryfar
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Papillary thyroid carcinoma (PTC) is the most prevalent form of thyroid cancer, yet its cellular heterogeneity and prognostic determinants remain poorly defined. Here, we integrate two single-cell RNA sequencing datasets comprising 20 human thyroid samples to construct a high-resolution cellular atlas of PTC. We identify 29 distinct cellular subpopulations and delineate their composition, dynamics, and interactions in healthy versus tumor tissues. Notably, epithelial and monocyte populations were markedly expanded in PTC, whereas adaptive immune subsets such as B and T cells were diminished. Cell–cell communication analysis revealed enhanced intercellular signaling in the tumor microenvironment, with epithelial and endothelial cells receiving the strongest inputs. Among monocyte-specific transcriptional signatures, we identified 65 prognostic genes via univariate Cox analysis. A LASSO-derived 14-gene risk score robustly stratified patient outcomes, with CSGALNACT1 emerging as a key epithelial-specific, independent prognostic gene. Pseudotime analysis further supported its role in epithelial cell differentiation. Functional validation demonstrated that CSGALNACT1 promotes proliferation in PTC cell lines, suggesting a potential oncogenic function. Immune deconvolution across risk groups revealed substantial divergence in innate and adaptive immune infiltration, indicating a close interplay between tumor-intrinsic transcriptional programs and immune microenvironment remodeling. Collectively, our study provides a comprehensive single-cell framework for PTC, identifies a clinically relevant risk model, and highlights CSGALNACT1 as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62fcd940763c0c7abf39d3fd346b8a069b2e5dc" target='_blank'>
              Comprehensive analysis of the papillary thyroid carcinoma identifies CSGALNACT1 as a proliferation driver and prognostic biomarker
              </a>
            </td>
          <td>
            Yu Zhao, Juan Zhang, Yong Yao, Linna Yu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, , Barbara Gruenwald, A. Borgida, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Monocyte-derived macrophages (MDMs) are widely used to model human macrophage biology in vitro and standardized polarization conditions were proposed to recapitulate the different macrophage activation states. Although surface markers specific for distinct MDM populations have been identified, a systematic analysis of surface marker expression on these consensus MDM populations has not previously been reported. Here, we use mass cytometry to perform an in-depth characterization of MDM surface profiles and determine how these markers evolve with time. We also compared the phenotypes found in vitro with patient-derived tumor-associated macrophages (TAMs) and found that although MDMs and TAMs shared most markers investigated, the cell-surface signatures markedly differed in terms of expression levels and combinations. Our high-dimensional, single-cell analyses clarifies the surface expression profile of the core in vitro differentiated macrophage populations and highlights some limitations of the in vitro system to represent the complexity of in vivo polarized tumor-associated macrophage phenotypes. These findings provide new opportunities to improve the models used to study macrophage biology and give a pathway to improve our understanding of the in vivo complexity of these systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1261e100b522c6e6f592ae9633f6538b46799d" target='_blank'>
              In vitro and in vivo derived macrophages occupy distinct phenotypic states
              </a>
            </td>
          <td>
            Stéphane Chevrier, Vito R. T. Zanotelli, D. Schulz, Mark Robinson, Laurie Ailles, Michael SL Jewett, Graig Gedye, Bernhard Reis, Bernd Bodenmiller
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is one of the leading cause of cancer-associated mortality globally. Its tumor microenvironment consists of various types of cells, which have a great impact on patient response and prognosis. The present study used scRNA-seq technology to investigate gene expression pattern of various cell types in NSCLC and their correlation with immune microenvironment. Methods We acquired NSCLC single-cell RNA sequencing data from GSE117570 and employed UMAP clustering analysis to explore the major cell subpopulations. Additionally, we explored the differential expression of important genes among various cell types and their associations with immune cell infiltration, scores of tumor microenvironment. Results Our results exhibited the heterogenicity of cell types in NSCLC and also indicated more than 60 genes whose expression displayed significant differences between cell groups, such as AP1S1, BTK, FUCA1, NDEL14, TMEM106B and UNC13D. Expression of these genes correlated with immunocyte infiltration, tumor microenvironment scores, indicating their potential roles in tumor progression and therapeutics. Conclusion Our single-cell transcriptomic dissection of NSCLC highlights cellular heterogeneity as well as potential biomarkers associated with immune microenvironment. The findings provide a foundation for the rational development of biomarker-driven personalized treatment and therapy by extending the cell-type specificity of current designs of NSCLC therapeutics. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03494-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfe159d40376ac64bc4970c70d60376979b77ad" target='_blank'>
              Single-cell RNA-seq reveals gene expression heterogeneity in NSCLC and its link to the immune microenvironment
              </a>
            </td>
          <td>
            Jinzhuo Ren, Hua Zhang, Haiyong Wu
          </td>
          <td>2025-09-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by poor prognosis, strong resistance to therapy, and a dense immunosuppressive tumor microenvironment (TME). A small subset of cells known as cancer stem cells (CSCs), or tumor-initiating cells (TICs), are increasingly recognized as key contributors to tumor initiation, metastasis, immune evasion, and treatment failure. These cells are defined by their self-renewal capacity, plasticity, and resistance to chemotherapeutic and targeted therapies. Pancreatic cancer stem cells (PaCSCs) are maintained by specific surface markers (CD44, CD133, EpCAM, ALDH1A1) and regulated by stemness-associated signaling pathways such as Wnt/β-catenin, Notch, Hedgehog, and TGF-β. Their survival is further enhanced by metabolic reprogramming, including shifts between glycolysis and oxidative phosphorylation and the activation of ROS-detoxifying enzymes. Importantly, PaCSCs reside in specialized niches formed by hypoxia, cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), and extracellular matrix (ECM) components that together shield them from immune clearance and promote therapeutic resistance. This review outlines the molecular features and functional roles of PaCSCs, their interaction with the TME, and recent advances in targeting this CSC-stroma network. Promising therapeutic strategies, such as CAR-T/NK cell therapies, epigenetic inhibitors, and combination regimens with checkpoint blockade or stromal modulators, are discussed in the context of ongoing clinical trials. Targeting both CSCs and their supportive microenvironment is emerging as a necessary strategy to overcome resistance and improve clinical outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0765b9737d7d7042e865a1f38d718a7df419afbe" target='_blank'>
              Deciphering the secrets of tumor-initiating cells in pancreatic ductal adenocarcinoma microenvironment: mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Zhou, Jiajun Li, Haoyi Lu, Yanggang Hong
          </td>
          <td>2025-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Although relatively rare, metaplastic breast cancer responds poorly to traditional therapies compared to other subtypes. Accordingly, there is a need for novel animal models to understand its pathogenesis and plasticity. Since alterations in PIK3CA and TP53 genes are common in metaplastic breast cancer, we generated a mouse model of metaplastic breast cancer by driving Pik3ca activation and Trp53 loss in keratin 15‐expressing mammary cells. In this model, male and female mice developed spontaneous mammary lesions, with malignancy reliant on loss of both Trp53 alleles. Importantly, tumors of this model are heterogeneous and resemble the mixed histology of metaplastic breast cancer by exhibiting squamous cell carcinoma, carcinosarcoma, and sarcoma features. We developed mammary cell lines from mouse tumors representing these different histological subtypes. These Pik3ca‐activated tumor cells were more sensitive to alpelisib, a p110α‐selective inhibitor approved by the FDA for the treatment of some PIK3CA mutant cancers, compared to Pik3ca WT cells. Additionally, some of these cell lines expressed the androgen receptor, a hormone receptor targeted in prostate cancer and currently under investigation as a therapeutic target in breast cancer. Transplantation of these cell lines into recipient mice maintained histological heterogeneity. Additionally, transplantation of either Epcam+ or Epcam− sorted cells, representing epithelial cell‐like and nonepithelial cell‐like, respectively, from a carcinosarcoma cell line, initiated tumor formation. Both sorted populations formed tumors with mixed histologic features, demonstrating plasticity arising from different tumor‐initiating components. These new models of metaplastic breast cancer from relevant genetic drivers serve as a platform for identifying mechanisms driving plasticity that could inform therapeutic strategies based on histology and reveal how plasticity alters treatment efficacy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f3608742cdae06c97d01e414cb2b45ba07265d" target='_blank'>
              Activation of PI3K and deletion of p53 in keratin 15‐expressing mouse mammary cells induces tumor heterogeneity and plasticity modeling metaplastic breast cancer
              </a>
            </td>
          <td>
            K. A. Nguyen, Lauren E McLemore, Yao Ke, Lisa N DePledge, Lynelle P Smith, Li Bian, John D. Aleman, David Debretsion, Erica Wong, Nicole Manning, Xiao-Jing Wang, Christian D. Young
          </td>
          <td>2025-08-13</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. I. Johnson, Daniel Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) refer to ectopic lymphoid aggregates that form in non-lymphoid tissues at sites of chronic inflammation including cancers. TLSs have been recognized as significant predictors of the efficacy of immune checkpoint blockade (ICB) therapies and have the potential to elicit robust anti-tumor immune response. However, recent studies have revealed substantial heterogeneity in TLSs across different individuals and cancer types, which directly impacts the effectiveness of anti-tumor immunity. Concretely, the maturation status, localization, and density of TLSs profoundly influence the dynamic interactions among immune cells within these structures, potentially leading to adverse effects. This review provides an in-depth exploration of how the heterogeneity of TLSs influences cellular composition and immune dynamics, with the objective of influencing the efficacy of ICB therapies and modulating prognostic prediction accuracy. Additionally, the potential of combining TLSs with other biomarkers for predicting anti-tumor immunity outcomes is further investigated, alongside the introduction of advanced technologies for evaluating TLS heterogeneity. Collectively, these analyses aim to advance the understanding of TLS heterogeneity and facilitate its translation into clinical and translational medicine applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3182b4ad88a2ceacb5b488328272b01f2977fe0" target='_blank'>
              Dissecting Tertiary Lymphoid Structures in Cancer: Maturation, Localization and Density
              </a>
            </td>
          <td>
            Guang-Liang Su, Meng‐Jie Zhang, Hao Li, Zhijun Sun
          </td>
          <td>2025-08-30</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endothelial cells (ECs) play complex roles across tissues and vessel types. Yet, systematic investigations of EC heterogeneity in the combined context of vessel type and tissue microenvironment are still largely lacking. We integrated over three million single cells of scRNA-seq datasets in 15 human tissues and found that ECs in some tissues (e.g., heart and kidney) exhibited greater tissue specificity, while others displayed more substantial vessel specificity. We developed a computational pipeline to analyze cell-cell communications (CCC) mediated by metabolites or proteins to explore microenvironmental regulation. Interestingly, our results showed that CCC events involving ECs varied vastly across tissues, highlighting tissue-specific EC interactions. Using topic modeling, we identified CCC patterns, termed CCC topics, representing metabolite- and protein-mediated interactions between ECs and other tissue-resident cells. Most CCC topics exhibited high tissue specificity, potentially explaining microenvironmental regulations for EC heterogeneity. The work systematically investigates EC heterogeneity and provides insights into how EC heterogeneity was regulated across diverse tissue microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c584c5af5601779d13578366342256b73ed6c81" target='_blank'>
              Human single-cell atlas analysis reveals heterogeneous endothelial signaling
              </a>
            </td>
          <td>
            Zimo Zhu, Rongbin Zheng, Yang Yu, Lili Zhang, Kaifu Chen
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that MDK expression increases with tumor grade in IDHwildtype gliomas, accompanied by a shift in isoform proportions favoring the canonical MDK transcript. High MDK expression was associated with poor prognosis specifically in GBM, where the MDKhigh subgroup comprised 75% of cases. MDKhigh GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16253-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5822ef3a00105861887610dcb7da2c42fc7f87" target='_blank'>
              Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment
              </a>
            </td>
          <td>
            Mieszko Lachota, K. Zielniok, Agata Gozdz, Patrycja Szpak, Ilona Kalaszczyńska, Radosław Zagożdżon
          </td>
          <td>2025-08-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transition is an essential mechanism that drives the development of distinct cellular phenotypes and tumorigenesis. The expression of various types of testis cancer antigens (TCAs) in breast carcinomas suggests a potential transition to male germ cell features within the tumor. This study explores the cellular populations in breast cancer that express genes associated with male germ cell development. We re-analyzed published datasets to identify the germline-associated genes in breast tumors. We then experimentally validated the expression of the identified genes in 28 breast tissue tumor samples using a quantitative RT-PCR. Based on available datasets, we also performed single-cell RNA sequencing (scRNA-seq) to analyze the tumor heterogeneity and cellular clustering. A total of 455 overexpressed genes were identified that were related to fetal primordial germ cells (PGCs), particularly those in the male gonad. Our examinations showed a significant overexpression of five genes (CCNB1, CCNB2, PTTG1, RACGAP1, and UBE2C) in the tumor samples. The scRNA-seq analysis revealed 14 distinct cell clusters, characterized by different gene expression signatures and cell cycle phases. The breast tumor stromal cells were suggested as the main source of the germline-associated genes. This study provides insights into the molecular mechanisms and pathways involved in germ cell transition in breast carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56e2486a4627d9f2385f5544550c02c8ae1d85a2" target='_blank'>
              Investigating Germ Cell Transition Genes in Breast Cancer: Exploring the Genesis of Cancer Testis-Associated Markers
              </a>
            </td>
          <td>
            Hamid Khodayari, Saeed Khodayari, Mohammad Dashtkoohi, A. Razavi, Seyed Rouhollah Miri, Ahad Mohamadnejad, Marcelo de Castro Santos, Fabio Comuzzi, Reza Shirkoohi, H. Mahmoodzadeh, Karim Nayernia
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, Genevieve L. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Objective
Advanced gastric cancer remains highly refractory to therapy, with limited immunotherapy efficacy due to tumor microenvironment heterogeneity. Primary cilia, microtubule-based organelles involved in tumor progression, remain insufficiently explored in gastric cancer. This study aimed to define primary cilia subtypes and establish prognostic signatures for personalized treatment strategies.


Methods
Bulk transcriptomic data from over 1,500 gastric cancer samples were integrated to define distinct primary cilia subtypes. A primary ciliary phenotype-associated signature (PCS) was established using a multi-machine learning survival framework incorporating ten algorithms. The prognostic predictive value and immunotherapy response prediction capability of PCS were validated across multiple independent cohorts. Single-cell RNA sequencing analysis was performed to identify cellular populations associated with high-PCS phenotype. Causal weighted gene co-expression network analysis (WGCNA) was employed to identify driving factors, followed by functional validation through cell culture experiments and xenograft models.


Results
Two distinct primary cilia subtypes were identified and validated across all cohorts, with C2 patients exhibiting significantly worse overall survival compared to C1 patients. PCS demonstrated robust predictive value for both prognosis and immunotherapy response, with superior accuracy compared to existing models across multiple validation cohorts. High-PCS patients showed reduced tumor purity, increased stromal cell infiltration, and poor response to immunotherapy. Single-cell analysis revealed that fibroblasts had the highest PCS scores and identified a novel secreted modular calcium-binding protein 2 (SMOC2)high myofibroblastic cancer-associated fibroblast (mCAF) population as the key driver of high-PCS phenotype. Functional experiments confirmed that SMOC2 knockdown significantly suppressed gastric cancer cell proliferation, migration, and invasion, while promoting mCAF-to-inflammatory cancer-associated fibroblasts (iCAF) transition.


Conclusions
PCS serves as a robust prognostic biomarker for gastric cancer patients. Additionally, targeting SMOC2 high mCAFs represents a potential therapeutic strategy for patients with high-PCS gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8325aa5e13eb71b8edb3f16c8e7ba4db0f0ba8" target='_blank'>
              SMOC2 high myofibroblastic cancer-associated fibroblast drives primary cilia-associated tumor microenvironment remodeling and poor prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qiqi Xu, Changjiang Yang, Jingyuan Ning, Yunze Niu, Xuesong Zhao, Long Zhao, Caihong Wang, Shan Wang, Yingjiang Ye, Zhanlong Shen
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba0ab80686667b3dcec48b4fa56304b52cf36e" target='_blank'>
              Reproducible single-cell annotation of programs underlying T cell subsets, activation states and functions
              </a>
            </td>
          <td>
            Dylan Kotliar, Michelle Curtis, Ryan Agnew, Kathryn Weinand, A. Nathan, Y. Baglaenko, Kamil Slowikowski, Yu Zhao, P. Sabeti, Deepak A. Rao, Soumya Raychaudhuri
          </td>
          <td>2025-09-03</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, 
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, F. Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, Nada Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite notable advances with immune checkpoint inhibitors (ICIs) in esophageal squamous cell carcinoma (ESCC), their clinical efficacy remains limited, largely due to CD8⁺T cell exhaustion. Among these, progenitor exhausted T cells (Tpex) represent a key subset with stem cell–like features that sustain durable anti-tumor immunity.We applied multi-color immunohistochemistry (mIHC) to determine the spatial distribution and clinical significance of Tpex cells within the tumor microenvironment (TME) of ESCC. Publicly available single-cell RNA sequencing (scRNA-seq) datasets were further analyzed to characterize Tpex cell phenotypes, differentiation trajectories, and intercellular communication networks.Tpex cells constituted a distinct subset of infiltrating CD8⁺T cells and represented a transitional stage of the exhaustion continuum. A higher degree of Tpex infiltration was significantly associated with improved overall survival in ESCC patients. Moreover, scRNA-seq data from patients treated with PD-1 blockade revealed that responders harbored markedly enriched Tpex populations compared with non-responders.Our findings identify Tpex cells as a critical prognostic and immunologically active CD8⁺T cell subset in ESCC. Their abundance and functional engagement are closely associated with favorable clinical outcomes and response to PD-1 blockade. Furthermore, their stem cell-like properties may be pivotal in shaping durable anti-tumor immunity and could provide novel therapeutic targets to enhance the efficacy of PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c5eac76128741742f8a923e04cbd692a6d7928" target='_blank'>
              Progenitor-exhausted T cell as prognostic indicator in esophageal squamous cell carcinoma: illuminating their key contribution to tumor immunity
              </a>
            </td>
          <td>
            Yi Liu, Hongwei Jiang, Zhang Fang, Bin Xu, Junjun Chen, Xiao Zheng, Renhao Geng, Lujun Chen
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="It is well established from in vitro studies of immune cells that stimulation by a wide range of potential signals leads to motility and morphology changes. How these physical behaviors manifest inside a living animal remains unclear due to limitations of conventional imaging and analysis approaches. Here, we establish a quantitative framework for imaging and tracking neutrophil and macrophage dynamics in larval zebrafish, spanning a large fraction of the animal for multi-hour timescales with few-minute temporal resolution. We focus especially on the gut, examining innate immune responses to different preparations of the intestinal microbiome. Using light sheet fluorescence microscopy and trajectory analysis of hundreds of individual cells, we characterize speeds, directional persistence measures, and cellular morphology to reveal distinct population behaviors. Individual immune cells exhibit stable motility phenotypes, favoring predominantly motile or non-motile states rather than frequent transitions between them. Gut architecture constrains migration patterns as demonstrated by preferential anterior-posterior movement and a high probability of cells remaining in the vicinity of the gut throughout the imaging duration. Macrophages display significantly reduced sphericity during motile periods compared to non-motile periods, providing a morphological signature that may enable inference of dynamic behavior from static snapshots. Surprisingly, migration patterns remain consistent across diverse microbial conditions – germ-free, conventionally reared, and colonized by two strains of a zebrafish-native Vibrio species – indicating that tissue structure exerts a stronger influence than bacterial stimuli on immune surveillance dynamics. Previously observed tissue damage by the wild-type Vibrio strain, and the resulting recruitment of immune cells towards the damage site, provided the only microbe-specific cellular behavior. These findings reveal innate immune surveillance as a stereotyped process whose characteristics reflect both cellular decision-making and larger-scale anatomical structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1591d467917ceed9ad4c3b39956f5d44b060babf" target='_blank'>
              Characterizing trajectories of innate immune cells in larval zebrafish
              </a>
            </td>
          <td>
            Piyush Amitabh, Raghuveer Parthasarathy
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Pediatric high‐grade gliomas (pHGG) were first defined as a distinct entity in the 2021 fifth edition of the WHO classification of tumors of the central nervous system. These tumors, designated primarily as Grade 4, include the following subtypes: (1) diffuse midline glioma with H3‐K27 alterations (DMG, H3‐K27M), (2) diffuse hemispheric glioma with H3‐G34 mutations (DHG, H3G34M), and (3) diffuse pediatric‐type high‐grade glioma with wild‐type H3 and isocitrate dehydrogenase (pHGG, H3‐WT/IDH WT). Clinically, pHGGs are known for their poor outcomes and marked tumor heterogeneity. Despite this, the characteristics of the tumor microenvironment (TME) and the processes governing tumor cell lineage transitions remain incompletely understood. In this study, we used single‐cell RNA sequencing (scRNA‐seq) to analyze pHGG tumor cells (excluding infant‐type hemispheric gliomas). Through comprehensive bioinformatic approaches—including cell proportion analysis, Gene Ontology (GO) enrichment, metabolic activity inference via scMetabolism, proliferation gene scoring, stemness assessment by CytoTRACE2, SCENT, and pseudotime trajectory analysis with Monocle2—we thoroughly investigated the TME features and heterogeneity of these aggressive brain tumors. Our findings highlight the presence of oligodendrocyte precursor cell (OPC)‐like subpopulations, with epidermal growth factor receptor (EGFR)‐expressing OPC‐like cells emerging as a potential tumorigenic origin in diffuse midline gliomas due to their distinct stemness properties. Notably, platelet‐derived growth factor receptor alpha (PDGFRA)‐positive cells exhibit high specificity in DMG, suggesting greater diagnostic and therapeutic potential than EGFR. Next‐generation sequencing (NGS) and multiplex immunofluorescence analyses confirmed their distinct expression pattern, supporting PDGFRA as a key molecular marker. Moreover, OPC‐like cells at different differentiation states may drive lineage transitions in DMG. Together, these findings enhance our understanding of pHGG—especially DMG—and point to new avenues for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746a8877d1a92b076c1950fd12d0bc8dc2480909" target='_blank'>
              Single‐Cell Profiling of Pediatric High‐Grade Gliomas Reveals OPC‐Like Subpopulations Driving Tumorigenic Lineage Transitions
              </a>
            </td>
          <td>
            Tian Tian, Chong Huang, Lusheng Li
          </td>
          <td>2025-09-01</td>
          <td>Pediatric Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non‐small cell lung cancer (NSCLC) is the most common lung cancer type and one of the deadliest neoplasias worldwide. NSCLC is histologically classified into adenocarcinoma, squamous cell carcinoma, and other less frequent subtypes. Both subtypes and other solid tumors are increasingly regarded as abnormal organs, highlighting the critical role of the desmoplastic tumor stroma rich in cancer‐associated fibroblasts (CAFs) in driving tumor progression and therapeutic resistance. This tumor stroma resembles a chronic fibrotic wound and is largely formed by activated/myofibroblast‐like α‐SMA+ CAFs (myCAFs), which are strongly associated with immunosuppression and poor prognosis. Despite the dominance of the myCAF phenotype, we reported a decade ago phenotypic alterations in NSCLC with a strong dependence on the histologic subtype. Subsequent studies using functional assays, single‐cell techniques, and in vivo models have refined these initial observations, enhancing our understanding of the biology of both normal fibroblasts/myofibroblasts and CAFs in NSCLC and other cancer types, including their origins, subclassification, and physiopathologic functions. Notably, increasing evidence supports that CAFs can exhibit tumor‐restraining or tumor‐promoting effects, and current therapeutic efforts aim to shift the balance towards tumor‐restraining phenotypes. Here, we review major advances in our understanding of tumor stromagenesis and CAF heterogeneity in both primary tumors and metastasis, including emerging consensus, with a special focus on NSCLC and its frequent dissemination to the brain. We also highlight the critical role of smoking through epigenetic reprogramming of the TGF‐β/SMAD3 pathway. These advances are beginning to delineate how CAF heterogeneity depends on the stage and histologic subtype in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bdccf1258eb4984f4511abb46e8045c6f47cd1e" target='_blank'>
              Stromagenesis and cancer‐associated fibroblast heterogeneity in primary tumors and metastasis: focus in non‐small cell lung cancer
              </a>
            </td>
          <td>
            Alejandro Bernardo, N. Díaz-Valdivia, Patricia Fernández-Nogueira, J. Alcaraz
          </td>
          <td>2025-09-04</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Tumour suppressor genes, exemplified by TP53 (encoding the human p53), function as critical guardians against tumourigenesis. Germline TP53‐inactivating mutations underlie Li‐Fraumeni syndrome, a hereditary cancer predisposition disorder characterised by early‐onset pan‐tissue malignancies. However, the context‐dependent tumour‐suppressive mechanisms of p53 remain incompletely elucidated. This study aims to investigate the disruption of cellular homeostasis and tumourigenic mechanisms following p53 inactivation across distinct cell lineages. Methods Trp53 (encoding mouse p53) knockout mouse model was employed to study molecular alterations under p53‐deficient conditions. Multi‐omics analyses – including single‐cell transcriptomics, single‐cell ATAC‐seq, spatial transcriptomics, whole genome sequencing, and CUT&Tag – were integrated to construct a Trp53 functional cell landscape. Deep learning‐based gene network models were employed to reconstruct p53 regulatory networks and simulate in silico perturbations caused by p53 loss. Results Our analyses revealed transitional dynamics in immune, stromal, and epithelial cells from normal physiology to p53‐deficient states and subsequent tumourigenesis. These transitions implicated critical pathways such as cell cycle regulation, stress response, metabolic reprogramming, and immune modulation, displaying both lineage‐conserved and lineage‐specific features. Tumour‐prone cell populations exhibiting elevated differentiation plasticity were identified across lineages within tumourigenic trajectories. Spatial transcriptomic profiling confirmed the emergence of thymic tumour‐initiating T‐cell clusters characterised by deterministic chromatin architectural disruptions under p53‐loss pressure. Notably, we uncovered a recurrent upregulation signature of ribosomal protein genes as an early pivotal molecular event preceding malignant transformation in p53‐deficient oncogenesis. Finally, we decoded the p53 downstream regulatory network and computationally evaluated the perturbation effects of genetic inactivation at single‐cell resolution. Conclusions Our results elucidate the multiscale consequences of p53 inactivation while providing valuable resources for understanding tumour predisposition associated with p53‐inactivating mutations and developing clinical interception strategies. Key points Construction of a Trp53 functional cell landscape utilising single‐cell multi‐omics and spatial omics technologies. Reconstruction of p53 downstream regulatory relationships with lineage heterogeneity via machine learning‐based gene network modelling. Dissection of shared and lineage‐specific features during cross‐lineage tumourigenesis under p53 deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc305d29bbc69cb0bead2f7a5b3cb8fa23e3cf53" target='_blank'>
              Dissecting cross‐lineage tumourigenesis under p53 inactivation through single‐cell multi‐omics and spatial transcriptomics
              </a>
            </td>
          <td>
            Xinru Wang, Yuqing Mei, Xueyi Wang, Hanyu Wu, Renying Wang, Peijing Zhang, Guodong Zhang, Jiaqi Li, Mengmeng Jiang, Xing Fang, Lifeng Ma, Yuan Liao, Danmei Jia, Haofu Niu, E. Weigao, Haide Chen, Lei Yang, Shuang Zhang, Tingyue Zhang, Yincong Zhou, Qi Zhang, He Huang, H. Ouyang, Ming Chen, Tingbo Liang, Jinrong Peng, Jingjing Wang, Guoji Guo, Xiaoping Han
          </td>
          <td>2025-08-31</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c0faec4d04ddee86bb924ee79c08f0dc4ced64" target='_blank'>
              Cancer-associated fibroblasts drive lung adenocarcinoma progression via THBS2-mediated epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            Yuqing Ren, Ruijie Ming, Anning Zuo, Shutong Liu, Yuhao Ba, Yuyuan Zhang, Yukang Chen, Teng Pan, Peng Luo, Q. Cheng, Jinhai Deng, Yi Yue, Hui-Xian Xu, Siyuan Weng, Xinwei Han, Dongdong Zhou, Zaoqu Liu
          </td>
          <td>2025-09-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) orchestrates a profound disruption of immune balance within the bone marrow (BM) microenvironment, driving disease progression and therapeutic resistance. To better understand these complex immune dynamics, we used high-dimensional mass cytometry (CyTOF) profiling to comprehensively characterize the immune landscape of the BM across different stages of myeloma progression, including MGUS (n = 16), smoldering MM (SMM; n = 25), and active MM, both newly diagnosed (n = 43) and relapsed/refractory (n = 104). Our analysis revealed substantial immune remodeling during disease progression, characterized by adaptive immune suppression and extensive infiltration of innate immune populations. Transformation from MGUS to SMM was marked by significant alterations in central and effector memory T helper cells, effector cytotoxic T cells, and an enrichment of monocytic and neutrophil subsets. Active MM stages were further distinguished by increased expansion of myeloid and monocytic lineages, alongside a pronounced reduction in progenitor and transitional B cells. Correspondence analysis demonstrated that specific immune profiles were significantly associated with clinical outcomes, including progression-free survival and overall survival. This study highlights the potential of CyTOF-based molecular profiling to unravel the intricate immune dynamics of the BM microenvironment across MM disease stages, enhancing our understanding of MM pathogenesis and providing a foundation for identifying prognostic biomarkers and tailoring precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c28c48facee0fbd69c0cc410926fbb87e89302" target='_blank'>
              CyTOF profiling of bone marrow immune dynamics across myeloma stages
              </a>
            </td>
          <td>
            Danka Cholujová, G. Beke, Ľuboš Kľučár, Ľ. Drgoňa, Zuzana Valuskova, M. Leiba, E. Kastritis, David M. Dorfman, Kenneth C. Anderson, J. Jakubikova
          </td>
          <td>2025-08-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo.
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, H. Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background and purpose The tumor microenvironment (TME) is widely acknowledged as a pivotal regulator of cancer progression. However, the dualistic role of tertiary lymphoid structures (TLSs), which serve as critical immune hubs within the TME, remains incompletely characterized, particularly with respect to their context-dependent capacity to either inhibit or facilitate tumor development. This review aims to synthesize current understanding of the complex interactions between stromal cells and TLSs, addressing existing gaps in mechanistic insight and exploring therapeutic avenues to exploit TLS plasticity. Key reviewed topics The current study critically reviews the mechanisms by which stromal components, including cancer-associated fibroblasts and endothelial cells, contribute to TLS neogenesis through chemokine-mediated recruitment of lymphocytes. Furthermore, it highlights the dual functional roles of TLSs as sites of both anti-tumor immune activation and immunosuppression, notably via the enrichment of regulatory T cells. The clinical implications of mature TLS presence, particularly their association with improved patient prognosis and enhanced therapeutic responsiveness, are also analyzed. Main conclusions TLSs demonstrate a bifunctional nature, wherein their spatial organization and dynamic interactions with stromal elements dictate the balance between immune activation and tolerance within the TME. While mature TLSs are generally correlated with favorable clinical outcomes, their potential to foster immunosuppressive microenvironments necessitates the development of precision-targeted interventions. The interplay between stromal cells and TLSs represents a promising therapeutic axis for modulating the tumor immune milieu. Future perspectives Future research should prioritize strategies aimed at promoting TLS maturation, disrupting immunosuppressive niches, and integrating TLS-modulating agents with existing immunotherapeutic regimens to enhance clinical efficacy. Additionally, the identification of robust biomarkers reflective of TLS functional states and the rigorous validation of stromal-targeted therapies within combinatorial treatment frameworks are imperative for advancing translational applications. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c13fc16e53261c020d877469541bad918b8156" target='_blank'>
              In-depth insight into tumor-infiltrating stromal cells linked to tertiary lymphoid structures and their prospective function in cancer immunotherapy
              </a>
            </td>
          <td>
            Maedeh Radandish, Niloofar Mashhadi, Amir Hossein Aghayan, Motahareh Taghizadeh, Sara Salehianfard, Sheida Yahyazadeh, Omid Vakili, Somayeh Igder
          </td>
          <td>2025-08-10</td>
          <td>Experimental Hematology & Oncology</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="T follicular helper (Tfh) cells are central to the adaptive immune response and exhibit remarkable functional diversity and plasticity. The complex nature of Tfh cell populations, inconsistent findings across experimental systems and potential differences across species have fueled ongoing debate regarding core regulatory pathways that govern Tfh differentiation. Many studies have experimentally investigated individual proteins and circuits involved in Tfh differentiation in limited contexts, each providing only a partial understanding of the process. To address this, we adopted a novel multi-scale network systems approach that incorporates both regulatory and protein-protein interactions. Our approach integrates diverse data types, captures regulation across multiple levels of immune system organization, and recapitulates known drivers. Further, we discover a core Tfh gene set that is conserved across tissue types and disease contexts, and is consistent across data modalities - bulk, single-cell and spatial. While components of this set have been individually reported, a novel aspect of our work lies in the discovery, characterization, and connectivity of this core signature using a single unbiased approach. Using this method, we also uncover a novel function of IL-12, a molecule with reported conflicting functions, in the regulation of Tfh differentiation. Notably, we find that, in both humans and mice, IL-12 is permissive for the differentiation of Tfh precursors, but blocks subsequent differentiation into GC Tfh cells. Overall, this work elucidates novel networks with unexplored roles in governing Tfh cell differentiation across species and tissues, paving the way for novel -therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cdef08e00798c3da4a3f940c2d60360ad82720" target='_blank'>
              A unified network systems approach uncovers a core novel program underlying T follicular helper cell differentiation
              </a>
            </td>
          <td>
            Alisa A. Omelchenko, Syed A. Rahman, Vinayak V. Viswanadham, Grace J. Yuen, Perla M. Del Rio Estrada, Valentino D’Onofrio, Yijia Chen, Na Sun, Hamid Mattoo, Chinmay G. Varma, Gonzalo Salgado, Maribel S. Nava, Lady C. Ruiz, Dafne D. Rivera, S. A. Rios, Sudhir P. Kasturi, Susan P. Ribeiro, M. Shlomchik, Amanda C. Poholek, Shiv S. Pillai, Rebecca A. Elsner, V. Mahajan, Jishnu Das
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG-on-Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real-time monitoring using wide-field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient-derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma-on-Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies.
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Intra-tumoral heterogeneity is a prominent characteristic of hepatocellular carcinoma (HCC). However, it remains unexplored whether intra-tumoral transcriptomic differences can capture crucial information regarding HCC evolution and be utilized to derive a predictive signature for patient’s clinical trajectories. Methods We quantified transcriptomic heterogeneity using four multiregional HCC cohorts comprising 172 samples from 37 patients, and validated transcriptomic heterogeneity and spatial dynamics using multiregional single-cell transcriptomic profiling of 110,817 cells from 34 liver specimens. The HCC evolutionary signature (HCCEvoSig) was developed and assessed across six cross-platform HCC cohorts. Results Genes exhibiting high intra- and inter-tumor expression variation were significantly enriched in a gene set associated with HCC prognosis, from which we developed and validated a reproducible and robust transcriptomic signature, HCCEvoSig. Multiregional single-cell data confirmed the high intra- and inter-tumoral heterogeneity of HCCEvoSig genes across different cell types, and importantly, demonstrated that the dysregulation of HCCEvoSig genes exhibited a geospatially gradual transition from the non-tumor region to the tumor border and tumor core, as well as from non-malignant to malignant epithelial cells. HCCEvoSig showed significant positive associations with adverse features of HCC, and a high HCCEvoSig risk score predicted increased risks of disease progression and mortality, independent of established clinicopathological indices. Furthermore, HCCEvoSig outperformed 15 published signatures in discriminative ability and prognostic accuracy, particularly regarding 1-year survival rates. Notably, HCCEvoSig demonstrated predictive utility for responses to immunotherapy and trans-arterial chemoembolization. Additionally, we established a well-calibrated predictive nomogram that integrates HCCEvoSig and TNM stage to generate an individualized numerical probability of mortality. Conclusion Our study reveals that regional transcriptional heterogeneity within tumors is substantial enough to capture survival signals, and the constructed and validated HCCEvoSig provides reliable prognostic information for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efeaa7fe73401eb801ace896c0e22c461a57e69" target='_blank'>
              Spatial heterogeneity reveals an evolutionary signature predicting therapeutic response and clinical outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Shangyi Luo, Li Liu, Yang Sun, Jian Shi, Yajing Zhang
          </td>
          <td>2025-08-18</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Capillary malformation (CM) is a congenital vascular anomaly affecting the skin, mucosa, and brain, yet the understanding of its vascular pathogenesis remains limited. Methods: We applied spatial whole-transcriptome profiling (GeoMx) and gene set enrichment analysis within CM lesions at single vasculature level. Differentially expressed genes were validated by immunofluorescence staining. Phosphoproteomics was profiled to uncover lesion-wide phosphorylation sites on proteins. Single-cell RNA sequencing was performed on CM-derived induced pluripotent stem cells (iPSCs) to determine differentiation trajectories of lesional vascular lineages. In silico gene perturbation was used to predict candidate genes for modulating vascular pathological progression, followed by functional validation in CM iPSC-derived endothelial cells (ECs) using a Tet-on system. Results: A spatial transcriptomic atlas was constructed, and pathological landscape of individual CM vasculature was delineated. CM vessels exhibited hallmarks of endothelial-to-mesenchymal transition (EndMT), including disruption of adherens junctions (AJs), vascular identity transitions, and metabolic remodeling. Phosphoproteomics confirmed that differentially phosphorylated proteins were enriched in EndMT- and AJ-related pathways. Aberrant expression of venous transcriptional factor NR2F2 was observed in lesional ECs and correlated with progressive enlargement from capillaries to larger-caliber vessels containing multiple layers of smooth muscle cells (SMCs). In CM iPSCs, differentiation course yielded reduced ECs but increased SMCs. In silico knockout simulation predicted NR2F2 as a crucial regulator of facilitating SMC phenotype in CM. Consistently, enforced NR2F2 expression during iPSC differentiation suppressed endothelial markers while inducing SMC-associated genes. Conclusions: Single CM vasculature displays pathological hallmarks characterized by EndMT and AJ disruption, leading to progressive vascular remodeling. NR2F2 functions as a central regulatory factor orchestrating the acquisition of the SMC phenotype, thereby representing a potential therapeutic target in CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fa98536c86180270a8a6d151af2637cfd03767" target='_blank'>
              Single-vessel transcriptome map pathological landscapes and reveal NR2F2-mediated smooth muscle cell phenotype acquisition in capillary malformations
              </a>
            </td>
          <td>
            Vi Nguyen, Irving Mao, Siwuxie He, Isabella Castellanos, Mackenzie Azuero, Marcelo L Hochman, Yin Rong, L. Pernomian, Elliott H Chen, Harold I Friedman, Yan-Hua Chen, Qun Lu, Daping Fan, Camilla F Wenceslau, Dong-bao Chen, Stuart Nelson, Anil G Jegga, Yunguan Wang, Wenbin Tan
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. J. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S Torcasso, Shannon Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lysosomes are critical organelles that act as degradation centers and signaling hubs within cells, playing a significant role in various cellular processes and human diseases, including cancer. However, the extent to which they influence the heterogeneity and clinical outcomes of ovarian cancer (OC) remains inadequately understood. In this study, we used consensus clustering to identify two distinct lysosome-related clusters (LCs) in OC by analyzing the expression profiles of OC patients from The Cancer Genome Atlas (TCGA) database. Further analyses revealed the functional characteristics and immune landscapes of these subgroups, providing valuable insights into the tumor immune microenvironment (TIME) and tumor responses to immunotherapy. Additionally, we developed and validated a prognostic model based on differentially expressed genes (DEGs) between the two LCs, demonstrating its effectiveness in predicting patient prognosis, TIME characteristics, and immunotherapy potential in OC. A further investigation explored the relationship between lysosome-associated risk scores, IC50 values of standard antitumor agents, and the expression levels of prognostic genes. Finally, in vitro experiments showed that inhibiting CRHR1, a lysosome-associated prognostic gene, significantly reduced OC cell proliferation, invasion, and migration. In conclusion, our study establishes a novel lysosome-based classification and prognostic framework for OC, offering a practical tool to predict clinical outcomes and guide personalized immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18271-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a65a1574799d7c11e4a3b68fa4b0f39f93a926" target='_blank'>
              Lysosome-related molecular subtypes reveal prognostic signatures and immunotherapeutic implications in ovarian cancer
              </a>
            </td>
          <td>
            Zhou Fang, Yunqing Liu, Yujie Cui, Ke Cao, Yiling Han, Zhijie Gao
          </td>
          <td>2025-09-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Late recurrence of breast cancer poses a considerable threat to patient survival, often attributed to breast cancer dormancy. Dormancy, characterized by cancer cells remaining quiescent for extended periods, is influenced by genetic factors and modifications that directly impact cellular phenotype. Alterations in gene expression dynamically shape cellular behavior, often mediated through receptor signaling pathways. Moreover, interactions within the tumor microenvironment play a pivotal role, fostering either cancer cell dormancy or promoting their escape from dormancy. This review endeavors to provide a comprehensive synthesis of recent advancements in understanding breast cancer dormancy across genetic, receptor molecular, and cellular dimensions. By elucidating the intricate mechanisms underlying dormancy, we aim to shed light on potential therapeutic strategies to prevent late recurrences. Furthermore, we anticipate future research directions that may uncover novel insights into this complex phenomenon, ultimately improving patient outcomes and refining clinical management strategies for breast cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6a18864245f8cad5e8cc957c1b2db50aef88c" target='_blank'>
              Unraveling the Mystery of Breast Cancer Dormancy: Insights into Genetic, Receptor, and Cellular Interactions Driving Late Recurrence
              </a>
            </td>
          <td>
            Haochen Ma, Bingqiang Zhang, Mengmeng Chen, Zhendi Song, Yi Zhao
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers.Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities.A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines.Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The awakening of dormant disseminated cancer cells appears to be responsible for the clinical relapses of patients whose primary tumors have been successfully cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, nonproliferative phenotypic state. The awakening of these cells is not triggered by a cancer cell-autonomous process. Instead, lung inflammation induced by the chemotherapeutic agent bleomycin effectively awakens dormant cancer cells, providing useful models for studying metastatic awakening. Mechanistically, the awakened cells shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire tumorigenicity and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their tumor-initiating ability, doing so without ongoing heterotypic signaling from the lung microenvironment. Epidermal growth factor receptor ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is critical for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0978c66611eb319088a607dd5da99f325c7f34ff" target='_blank'>
              Inflammation awakens dormant cancer cells by modulating the epithelial-mesenchymal phenotypic state.
              </a>
            </td>
          <td>
            Jingwei Zhang, Jingwen Zhang, Longfei Han, Shiyi Wu, Jie Li, E. Eaton, Bingbing Yuan, Ferenc Reinhardt, Hao Li, Patrick C Strasser, Sunny Das, Joana Liu Donaher, Md Imtiaz Khalil, Haiping Jiang, Alexander Deuschel, Danni Lin, Carolin Sebastiany, Mariana Maranga, Salomé Shubitidze, Xiaofei Liu, Arthur W Lambert, Yun Zhang, Yana Liu, L. Sui, Sarah Elmiligy, Umberto Pozza, Rauf Günsay, Ranjan Mishra, José Velarde, Sonia Iyer, Whitney S. Henry, K. Weiskopf, Guihai Feng, Tobiloba E. Oni, R. Watnick, Xin Li, Robert A. Weinberg
          </td>
          <td>2025-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Metaplastic breast cancer (MpBC) is defined by the presence of various morphological elements, typically biphasic, with epithelial (e.g. no-special type (NST), squamous) and mesenchymal (e.g. spindle, chondroid, osteoid) components. The established clonality of the different components favours an evolution model encompassing either a multipotent progenitor, or a linear metaplastic conversion. We used methylation profiling and showed that different morphologies have specific methylation profiles. Furthermore, our spatial transcriptomic approach, using 10× Genomics Visium and trajectory analysis, evidenced that spindle cells form a transition between the originating carcinoma of no-special type (NST) and pleomorphic regions, with osteoid differentiation likely to be an end-stage fate of the chondroid growth pattern, supporting the conversion model of lineage differentiation. We have also identified a series of master transcription factors likely to regulate these processes, and are significantly associated with metaplastic-like clinical features. This data further supports the conversion model of metaplasia and warrants functional analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077285caa17e07f895159bb1ece8c70f06837f91" target='_blank'>
              Integrated profiling of metaplastic breast cancer identifies putative master regulators of intratumoral heterogeneity
              </a>
            </td>
          <td>
            Yufan Feng, Albert Xiong, Onkar Mulay, Anna Sokolova, M. Lim, Benjamin Van Haeringen, Natasha McGuire, Xavier M. de Luca, Peter T Simpson, Quan Nguyen, S. Lakhani, A. M. McCart Reed
          </td>
          <td>2025-08-11</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes ( CXCL12-CXCR4 , CCL5-SDC4 ), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine. SUMMARY Despite major advances in spatial biology, no framework has yet linked spatially resolved intercellular communication networks, independent of cell types, to clinical outcomes in human disease. Here, we present STARComm, a scalable method that identifies Multicellular Interaction MCIMs (MCIMs). Applying STARComm to minor salivary gland biopsies from patients with chronic graft-versus-host disease (GVHD), we identify MCIMs that not only distinguish healthy from diseased tissue but also stratify patient survival. High-risk MCIMs are enriched for actionable immune and stromal pathways, including those targetable with existing therapies. These findings establish the first outcome-linked spatial communication framework in any human disease and highlight the translational potential of oral tissues as minimally invasive platforms for real-time immune diagnostics, prognostic modeling, and therapeutic screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a093b13aff7d0f5c2b6d575cc2d04f2b86e47ba" target='_blank'>
              STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer
              </a>
            </td>
          <td>
            Khoa Huynh, B. Matuck, Deiziane de Souza, XiuYu Zhang, Giancarlo Fatobene, Luiz Alberto Valente Soares Júnior, L. F. Ferraz da Silva, V. Rocha, K. Byrd, Jinze Liu
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly lethal malignancy characterized by a complex pathological mechanism involving multiple genes and progressive stages. The efficacy of targeted and immunotherapy remains limited, highlighting the urgent need for a reliable model to predict prognosis and response to immune checkpoint inhibitors (ICIs). Methods We developed an integrated model based on genes related to autophagy, senescence, dormancy, mitochondrial function, and tumor stemness. The predictive capability of this model for HCC prognosis and ICI response was evaluated. Single-cell transcriptomic analysis and immunocompetent mouse models were further utilized to elucidate the role of model-associated genes in regulating the tumor immune microenvironment. Results A 16-gene integrated model was constructed using genes associated with mitochondrial function, autophagy, dormancy, stemness, and senescence. This model demonstrated robust predictive power for HCC prognosis and ICI responsiveness. Single-cell trajectory analysis revealed that EZH2 plays a crucial role in immune cell infiltration, activation, and HCC progression. Additionally, in vivo mouse models further indicated that EZH2 may regulate CD8+ T cell activation and exhaustion through the MIF-CD74 signaling pathway. Conclusion The integrated model holds potential as a prognostic and predictive tool for HCC immunotherapy. EZH2 may influence CD8+ T cell activation and exhaustion via the MIF-CD74 axis, providing insights for patient stratification and potential therapeutic strategies to enhance immunotherapy efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07071-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7937b56d5b80810f28cf602b4f87ac567637d0a" target='_blank'>
              Integrated single-cell RNA-seq analysis reveals that EZH2 regulates the MIF-CD74 axis to modulate T cell activation and exhaustion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yangyang Zhou, Yi Xu, Mengfan Ye, Jiayi Xing, Di Chen, Al-Ameer Wail Hussein Ahmed, Al-Rahabi Rehab Abdullah Mohammed Ali, Chenwei Pan, Xiangchou Yang, Zan Shen
          </td>
          <td>2025-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human leukocyte antigen (HLA) polymorphism hinders allogeneic therapies. We developed REWRITE, a modular platform for iterative, scar-minimized genome writing of synthetic constructs >100 kb in human pluripotent stem cells (hPSCs). Using REWRITE, we deleted 105–209 kb of the HLA locus and installed synthetic 24 kb or 100 kb HLA haplotypes at native and ectopic sites, and a 62 kb antigen-processing locus. This uncovered heritable epigenetic memory of an active state, whose resolution toward a silenced state is driven by native intergenic DNA. These loci restored inducible expression in key lineages, sparing cells from NK-mediated killing and establishing HLA-matched T-cell tolerance. REWRITE provides an extensible platform for programming multigenic behaviors in human cells — from immune interface design to discovery of principles governing human genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab49fc81bb5b66b5a7cc1e58b9c3258864a600c" target='_blank'>
              Genomic REWRITE of Immune Interfaces Reveals Heritable Epigenetic Memory in Human Stem Cells
              </a>
            </td>
          <td>
            Serena F. Generoso, Sarah Levovitz, Susanna Jaramillo, Minjoo Kim, Sumanth Dara, Shean Fu Phen, Bryan Yi, Tomoki Yanagi, Thomas L. DesMarais, Neta Agmon, Megan S. Hogan, Leslie A. Mitchell, David M. Truong
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Macrophages are key components of the tumor microenvironment and play a critical role in melanoma progression. However, the dynamics of different macrophage subsets, particularly CD68+ and CD163+ populations, in relation to tumor thickness and stage remain insufficiently characterized. Objective: This study aimed to quantitatively assess intratumoral and peritumoral CD68+ and CD163+ macrophages in cutaneous melanoma and to investigate their associations with Breslow thickness, pT stage, and AJCC stage. Methods: We conducted a retrospective cohort study of 126 patients with cutaneous melanoma (AJCC stages IA–IIID). Tumor samples were examined histologically and immunohistochemically for CD68+ and CD163+ macrophages, and quantitative analysis was performed in intratumoral and peritumoral regions. Results: This study included 126 patients with cutaneous melanoma, ranging in stage from IA to IIID. Histopathological analysis revealed that melanoma tissues were primarily composed of irregular clusters of atypical melanocytic cells infiltrating the dermis and subcutaneous fat. Immunohistochemical staining identified CD68+ and CD163+ macrophages both within the tumor and in the surrounding stroma. Macrophage infiltration significantly increased with tumor thickness (Breslow) and progression to more advanced stages. Specifically, at Breslow thickness <1.0 mm, the mean number of CD68+ macrophages in the intratumoral zone was 29.7 ± 4.3 cells, increasing to 70.3 ± 6.4 cells in tumors >4.0 mm. CD163+ macrophages showed similar trends, with a rise from 15.6 ± 2.8 cells at <1.0 mm to 39.8 ± 4.6 cells at >4.0 mm in the intratumoral zone. Additionally, macrophage density was higher in tumors with ulceration, and both CD68+ and CD163+ macrophage numbers increased progressively with tumor stage, particularly in advanced stages. The number of CD68+ macrophages at stage IA in the intratumoral zone was 21.6 ± 3.1 cells and increased to 56.4 ± 6.8 cells at stage IIID, while CD163+ macrophages rose from 13.8 ± 3.2 cells at stage IA to 36.4 ± 4.6 cells at stage IIID. This suggests that macrophage infiltration, particularly CD163+ cells, correlates with melanoma progression. Conclusions: Our findings highlight distinct spatial and phenotypic patterns of macrophage infiltration in melanoma progression. The prominent increase in CD68+ and CD163+macrophages suggests their important role in tumor-associated immunomodulation. Further studies are warranted to elucidate macrophage polarization states and their prognostic and therapeutic implications in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4969416cee26205b7a0763813b82a39e29b3d3" target='_blank'>
              Spatial Distribution and Phenotypic Profiling of Cd68+ and Cd163+ Macrophages in Melanoma Progression: Insights into Tumor Microenvironment Dynamics
              </a>
            </td>
          <td>
            G. Demyashkin, D. Atiakshin, K. Silakov, V. Shchekin, Maxim Bobrov, M. Vadyukhin, Tatyana Borovaya, Ekaterina Blinova, Petr Shegay, Andrei Kaprin
          </td>
          <td>2025-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioma intratumoral heterogeneity remains a critical barrier to effective treatment, driving recurrence and resistance to therapy. Emerging evidence suggests that glioma cells acquire neural-like features through paracrine and synaptic communication with neural cells, fostering functional diversity within tumors. While neuro-cancer interactions are implicated in glioma heterogeneity, their precise roles remain incompletely synthesized. Here, by consolidating these discoveries, a conceptual framework is proposed for understanding the glioma intratumoral heterogeneity shaped by neuro-cancer interactions: cellular phenotypes, spanning neurogliomal synapses, glioma networks, and neuronal-like motility; and transcriptional states, which exhibit remarkable resemblance to neural cells. These diverse cellular phenotypes and transcriptional states synergistically fuel glioma progression, invasion, and resistance. By emphasizing the converging phenotypical and transcriptional evidence with spatial context within them, an underexplored but critical role of neuro-cancer interactions are proposed in glioma intratumoral heterogeneity. This provides potential strategies to explore and disrupt these neuro-cancer interactions, offering new insights to address glioma intratumoral heterogeneity for improved therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7eca8bd6e6e1b76e76ba8a64aea76418740245" target='_blank'>
              Neuro-Cancer Interactions Shape Glioma Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Tong Wu, Peiran Zhao, Zheng Chen, Tianlei Ying, Zhihong Nie, Wei Hua, Huiyan Li, Ying Mao, Wenbo Bu
          </td>
          <td>2025-08-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogenes amplified on extrachromosomal DNA (ecDNA) contribute to treatment resistance and poor survival across cancers. Currently, the spatiotemporal evolution of ecDNA remains poorly understood. In this study, we integrate computational modeling with samples from 94 treatment-naive human glioblastomas (GBM) to investigate the spatiotemporal evolution of ecDNA. We observe oncogene-specific patterns of ecDNA spatial heterogeneity, emerging from random ecDNA segregation and differing fitness advantages. Unlike PDGFRA-ecDNAs, EGFR-ecDNAs often accumulate prior to clonal expansions, conferring strong fitness advantages and reaching high abundances. In corroboration, we observe pretumor ecDNA accumulation in vivo in genetically engineered mouse neural stem cells. Variant and wild-type EGFR-ecDNAs often coexist in GBM. Those variant EGFR-ecDNAs, most commonly EGFRvIII-ecDNA, always derive from preexisting wild-type EGFR-ecDNAs, occur early, and reach high abundance. Our results suggest that the ecDNA oncogenic makeup determines unique evolutionary trajectories. New concepts such as ecDNA clonality and heteroplasmy require a refined evolutionary interpretation of genomic data in a large subset of GBMs.


SIGNIFICANCE
We study spatial patterns of ecDNA-amplified oncogenes and their evolutionary properties in human GBM, revealing an ecDNA landscape and ecDNA oncogene-specific evolutionary histories. ecDNA accumulation can precede clonal expansion, facilitating the emergence of EGFR oncogenic variants, reframing our interpretation of genomic data in a large subset of GBMs. See related article by Korsah et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            ChongFeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The thymidine analog EdU (5-ethynyl-2-deoxyuridine) is incorporated into DNA during replication and has traditionally been used as a marker of S-phase cells. In this study, we discovered that EdU fluorescence images display substantial cell-to-cell variability, which could be classified into multiple clusters by unsupervised machine learning. This suggests that seemingly random EdU patterns contain reproducible, computationally recognizable features. Building on our observation that distinct patterns emerged in response to radiation stress, we investigated whether radioresistant cancer cells exhibit specific EdU signatures. Analysis of PLK1-overexpressing cells, which acquire radioresistance through altered DNA replication, revealed radiation-induced EdU patterns distinct from control cells. Prompted by the observation that these cells displayed markedly enlarged and intensified γ-H2AX foci, a marker of DNA damage, we employed a supervised machine learning model based on γ-H2AX patterns to isolate the radioresistant cell subpopulation. We then extracted the EdU signals from these isolated cells and, through further unsupervised machine learning, successfully identified a characteristic pattern specific to radioresistance. This establishes a machine learning framework capable of extracting universal rules from the dynamic networks that vary among individual cells, which provides a novel platform for a screening system to identify molecules involved in radioresistance, focusing on cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447884e0a398eddbbba5abf08e4dfe806602b07" target='_blank'>
              Machine Learning Extracts Radiation Resistant-Specific EdU Fluorescence Pattern in Cancer Cells.
              </a>
            </td>
          <td>
            M. Ikura, Tsuyoshi Ikura, Kanji Furuya
          </td>
          <td>2025-09-01</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor draining lymph nodes (TDLNs), as secondary lymphoid organs, are pivotal in initiating and regulating adaptive immune responses. Historically, TDLNs were recognized primarily as metastasis gateways in cancer, promoting radical dissection to prevent recurrence. However, emerging preclinical studies reveals their critical role in orchestrating systemic anti-tumor immune responses during cancer therapy, highlighting the dilemma of balancing lymph nodes (LNs) preservation with metastasis control. This review traces the evolving understanding of TDLN biology in oncology, from the era of radical LN dissection to multi-omics-driven insights, and synthesizes their dual roles as immune hubs and metastatic niches across first-line clinical therapies (e.g., immunotherapy, radiotherapy, chemotherapy, targeted therapy, etc.). We further propose the concept of “Lymph Node Multi-modal Protective Research (LNMPR)”, emphasizing the prospective value of integrating multi-omics technologies, including spatial transcriptomics, single-cell profiling, and imaging, to decode LN immune dynamics and optimize therapeutic responses. By bridging mechanistic insights with clinical strategies, LN-centric immune modulation may open up a new path for precise tumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ddb9cb8a82e08359fde743e98ea142d35dd6bc8" target='_blank'>
              From trash to treasure: tumor draining lymph nodes as a multi-omics goldmine in cancer therapy
              </a>
            </td>
          <td>
            Yina Li, Zihan Chen, Zhikun Guo, Jiangnan Yu, Jianan Lu, Lei Wang, Qian Zhou
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) in the tumor microenvironment (TME) act as local hubs for antigen presentation and lymphocyte activation, reinforcing antitumor immunity. They act as ectopic, lymph node-like niches that orchestrate local antitumor immunity through well-defined B-cell follicles, T-cell zones, follicular dendritic cells (FDCs), and high endothelial venules (HEVs). Although various cancer treatment methods can impact TLSs, thereby changing the state of the TME, the dynamic changes in TLSs caused by treatment and the mechanisms therein have not been comprehensively summarized to date. This review highlights the mechanisms of different cancer treatment effects on TLSs dynamic change, including the number and functionality of TLSs within tumors. We also summarize the value of TLSs in predicting prognosis and therapeutic effect, as well as the deficiencies in current research and future development directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b10ea5ef7732254210365008058829408071ec8" target='_blank'>
              Dynamic remodeling of tertiary lymphoid structures in response to cancer therapy: a recent review
              </a>
            </td>
          <td>
            Chongyu Tan, Jinliang Huang, Ning Gao, Bingquan Wu, Matsika Juliet, Jiatong Xiao, Jiao Hu, Ping Liu, Jinbo Chen
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a4c1ebff0edca5f86402765983e14a517f49f7" target='_blank'>
              Form Follows Function: A New Paradigm of Pancreatic Cancer Progression.
              </a>
            </td>
          <td>
            William Gasper, Giada Pontecorvi, Zhikai Chi, S. Enrico, Lily C. Goodman, Haneen Amarneh, Evelyn Mazzarelli, Joshua Thomas, Lucia Zhang, Rachel Handloser, Seth LeCates, A. L. Tomescu, Francesca Rossi, Myla Sykes, Anjali Bhatt, Nicholas Fazio, Alessandro Bifolco, Nafeesah Fatimah, Austin Seamann, Cheryl M Lewis, Shelby Melton, Herbert J. Zeh, Megan B. Wachsmann, Dario Ghersi, Matteo Ligorio
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most frequent brain tumors in children and comprise a heterogeneous group of tumors with different locations, histologic subtypes, ages at presentation, and clinical behavior. Tumors frequently respond to treatment with chemotherapy or radiation therapy, but they can regrow after a period of quiescence, requiring further therapy. Thus, a deeper understanding of the molecular processes involved in these tumors is required to develop therapeutic strategies that are effective against their disease mechanisms. To better understand the cellular behaviors of this heterogenous group of tumors, we have employed single-cell and single-nuclei RNA sequencing technologies to analyze a large-scale dataset (>250,000 cells) of pLGGs. Analysis of this data identified a heterogenous population of cell types and cell states, detecting mature and progenitor-like astrocytes and oligodendrocytes, as well as cells exhibiting senescence or cycling programs. Moreover, we identify a significant immune infiltrate, comprised primarily of microglia. In addition to heterogeneity within pLGG tumors, heterogeneity between LGG subtypes represents another layer that stratifies pLGG biology. We performed a compositional analysis of the cell types present in these tumors and compared transcription signatures and gene expression programs across shared cellular populations of histologically and genetically distinct pLGGs. Finally, we optimized our integration and batch correction analyses by using external 293T cells as spike in controls during our single-cell and single-nuclei data generation steps to determine the most suitable method for batch-effect removal. Our analysis of human pLGGs at the single-cell and single-nuclei resolution provides critical insight into the heterogenous biological activities that constitute these tumors.



 Michelle Boisvert, Ashwyn Perera, Alexandra L. Condurat, John Jeang, Jessica W. Tsai, Dana Novikov, Kevin Zhou, Madison Chacon, Jeromy DiGiacomo, Rushil Kumbhani, Dayle Wang, Michael D. Taylor, Jordan R. Hansford, Louise Ludlow, Nada Jabado, Keith L. Ligon, Rameen Beroukhim, Pratiti Bandopadhayay, David T.W. Jones. Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fff9552d8445c379c8cd3282dd22472d8f4d25" target='_blank'>
              Abstract A014: Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology
              </a>
            </td>
          <td>
            Michelle Boisvert, Ashwyn A. Perera, Alexandra L. Condurat, John Jeang, Jessica W Tsai, Dana Novikov, Kevin N Zhou, Madison S. Chacon, Jeromy J. Digiacomo, Rushil Kumbhani, Dayle K. Wang, Michael D. Taylor, J. Hansford, Louise E. Ludlow, Nada Jabado, Keith L. Ligon, R. Beroukhim, P. Bandopadhayay, David Jones
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>150</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Single-cell sequencing technology has emerged as a powerful tool for investigating the tumor immune microenvironment, enabling a deeper understanding of the heterogeneity of immune cell populations within tumors. Tumor-associated neutrophils (TANs) have become a focal point in the study of the tumor immune microenvironment due to their involvement in tumorigenesis, progression, and drug resistance. Methods: We analyzed scRNA-seq data from metastatic gastric cancer (GC) and bulk RNA-seq data from multiple GC cohorts. Non-negative matrix factorization (NMF) was used to classify molecular subtypes, and a prognostic model was constructed using LASSO Cox regression. Key genes were validated by using qRT-PCR and immunohistochemistry. Molecular docking was utilized to screen small-molecule compounds targeting prognostic genes. Results: scRNA-seq analysis identified neutrophil-associated clusters. Using neutrophil-related genes, we classified two molecular subtypes with distinct immune microenvironment features and established a six-gene prognostic risk model. Internal and external validation confirmed its robust predictive ability. The model also predicted immunotherapy response, with the low-risk group showing better outcomes, preliminarily validated in MSS/pMMR patients treated with Camrelizumab. Additionally, we assessed chemotherapy and targeted drug sensitivities and identified small-molecule compounds targeting the six prognostic genes. Conclusions: This study highlights the significance of TANs in predicting GC prognosis, immunotherapy response, and drug exploration, providing a foundation for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912704394907c4bb279874d4d1dda11a6211972e" target='_blank'>
              Integrating Single-Cell Transcriptomic Data of Metastatic Gastric Cancer to Construct a Tumor-Associated Neutrophil Signature and Clinical Validation in MSS/pMMR Patients
              </a>
            </td>
          <td>
            Lin Yao, Yi-Meng Sun, Meng-Fang Liu, Jiang-Ning Xiang, Shan Wang, Chang-Feng Man, Rong He, Jing Yu, Yu Fan
          </td>
          <td>2025-09-16</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background CD4+ T cells, as critical mediators of immune regulation, have not been fully elucidated in their role in hepatocellular carcinoma (HCC) immune evasion. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data from 16 HCC patients (GSE149614). Cell types were annotated based on cluster-specific differentially expressed genes (DEGs), utilizing the SingleR and PISCES for automated cell type assignment, with reference to established markers from previous studies. Metabolic reprogramming, transcriptional networks, and intercellular communication were assessed via GSVA, SCENIC, and CellChat. A prognostic model integrating TCGA-LIHC and validated in GSE109211 was developed using LASSO Cox regression. Molecular docking and dynamics simulations evaluated the therapeutic potential of prioritized targets. Results This study identifies SPP1+TNFRSF18+ CD4+ T cells as key drivers of HCC immune evasion. This subtype was enriched in tumor tissues, exhibited elevated immune checkpoint gene expression scores, and demonstrated metabolic reprogramming favoring tumor growth via glycolysis and reactive oxygen species (ROS) pathways. CellChat analysis revealed that SPP1-mediated interactions construct a multi-layered immunosuppressive network involving macrophages and CD8+ T cells. A prognostic model based on nine genes (LIG1, NOP56, MIS18A, SMC4, CDCA8, DTYMK, ASF1B, RAD51, LGALS3) was developed using TCGA-LIHC data and validated in an independent cohort (GSE109211), achieving robust predictive performance (AUC > 0.7). Molecular docking and dynamics simulations confirmed the drug ability of these targets, which means these genes may serve as prognostic biomarkers and therapeutic targets. Conclusions Our findings establish SPP1+TNFRSF18+ CD4+ T cells as central regulators of HCC immune suppression, offering novel strategies to enhance HCC immunotherapy. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03556-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708e8db1767d8ebe0afe45a2f9c169a343ebb92f" target='_blank'>
              Identification of immunosuppressive CD4+ T-cell subtype and construction of prognostic model for hepatocellular carcinoma
              </a>
            </td>
          <td>
            Chuxuan Wang, Lianchi Li, Yingrui Lou, Lan Zhang, Feifei Tian, Taoyou Zhou
          </td>
          <td>2025-09-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a devastating malignancy characterized by profound lethality, aggressive local invasion, dismal prognosis, and significant resistance to existing therapies. Two critical biological features underpin the challenges in treating PDAC: extensive perineural invasion (PNI), the process by which cancer cells infiltrate and migrate along nerves, and a profoundly immunosuppressive, or “cold,” tumor microenvironment (TME). PNI is not only a primary route for local tumor dissemination and recurrence but also a major contributor to the severe pain often experienced by patients. Concurrently, the PDAC TME is typified by a dense desmoplastic stroma, hypoxia, and an abundance of immunosuppressive cells—including cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), myeloid-derived suppressor cells (MDSCs), and regulatory T cells (Tregs)—while lacking sufficient infiltration of effector T cells, rendering it largely unresponsive to immunotherapies like checkpoint inhibitors. Although historically studied as separate entities, accumulating evidence reveals a deep-seated and complex bidirectional crosstalk between the neural components involved in PNI and the immune and stromal cells constituting the TME. Key cellular mediators, such as CAFs and TAMs, and shared signaling pathways, including the CXCL12/CXCR4 axis, TGF-β signaling, and neurotrophin pathways (e.g., NGF/TrkA), appear to act as critical nodes, coordinating the progression of PNI while simultaneously shaping and maintaining the immunosuppressive TME. This review synthesizes the current understanding of these intricate neuro-immune interactions in PDAC. We delineate the molecular and cellular mechanisms governing this crosstalk and explore how targeting these shared regulatory networks presents novel therapeutic opportunities, potentially disrupting PNI while concurrently “heating” the cold TME to overcome immunotherapy resistance. Elucidating this interplay is crucial not only for a deeper comprehension of PDAC’s invasive and metastatic mechanisms but also for uncovering new therapeutic vulnerabilities to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6006a1bf0a36f45add16447e3ef30da17be2aa8b" target='_blank'>
              Perineural invasion and the “cold” tumor microenvironment in pancreatic cancer: mechanisms of crosstalk and therapeutic opportunities
              </a>
            </td>
          <td>
            Jianbiao Xu, Hong Yao, Junfeng Wang, Yun Jin, Wei Chang, Lanjiang Li, Lei Zou
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, , Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="The transition from acute kidney injury to chronic kidney disease is characterized by significant changes in the cellular composition and molecular interactions within the kidney. Utilizing high-resolution Xenium and whole transcriptome Visium spatial transcriptomics platforms, we analyze over a million cells on 12 male mouse kidneys across six stages of renal injury and repair. We define and validate 20 major kidney cell populations and delineate distinct cellular neighborhoods through this multimodal spatial analysis. We further reveal a specific fibro-inflammatory niche enriched in failed-repair proximal tubule cells, fibroblasts, and immune cells, with conserved neighborhood gene signatures across mouse and human. Within this niche, we predict Runx2 as a key upstream regulator, along with platelet-derived growth factor and integrin beta-2 signaling pathways shaping the fibrogenic microenvironment. Altogether, our study provides deep insights into the cellular and molecular dynamics during kidney injury and repair and establishes a comprehensive multimodal analytical framework applicable to other spatial omics studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80e9ed63af964434bd5c50df588ce36470352d4" target='_blank'>
              Multimodal spatial transcriptomic characterization of mouse kidney injury and repair
              </a>
            </td>
          <td>
            Xuanyuan Qiao, Haojia Wu, Hemalatha Sundaramoorthi, Pierre Isnard, Changfeng Chen, Waleed Rahmani, Benjamin D. Humphreys
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) critically regulate tumor progression, angiogenesis, metastasis, and therapeutic resistance. This study investigated the characteristics of CAFs in glioblastoma (GBM) and developed a CAF-based risk signature to predict patient prognosis. The single-cell RNA sequencing (scRNA-seq) data were sourced from the Gene Expression Omnibus (GEO) database, whereas the bulk RNA-seq datasets were retrieved from The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA), respectively. The Seurat R package processed scRNA-seq data to identify CAF clusters using established markers. Prognostic genes were screened through univariate Cox regression, with Lasso regression constructing the final risk model. A nomogram incorporating clinical parameters was subsequently developed. Immunohistochemical validation was performed using the Human Protein Atlas (HPA) for the signature genes. The qRT-PCR validation was conducted in U251 and HA cells. ScRNA-seq analysis revealed five CAF clusters in GBM, including three prognostically relevant subtypes. Three key genes were refined to construct a risk signature functionally enriched in the the IL6_JAK_STAT3 signaling, P53 pathway, and inflammatory response. The signature correlated strongly with stromal and immune scores. Multivariate analysis confirmed risk signature independent prognostic value (P < 0.0001), followed by age (P = 0.005). The CAF-derived nomogram demonstrated superior predictive accuracy for 1-/2-year survival compared to clinical factors alone. The signature genes were validated in the HPA database and experimental validation. This study proposes CAF-derived molecular signatures as potential predictors of glioblastoma prognosis worthy of clinical validation. Systematic characterization of CAF heterogeneity may offer insights for interpreting GBM immunotherapy responses, providing a foundation for future exploration of stroma-targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00548-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d7a7b7ca4435c9538e14b7aed765c936957688" target='_blank'>
              Integrated single-cell and bulk transcriptomic profiling reveals cancer-associated fibroblast heterogeneity in glioblastoma and establishes a clinically actionable prognostic model and preliminary experimental validation
              </a>
            </td>
          <td>
            Wenhua Zhang, Yaxiong Li, Conghui Li, Qian Huang
          </td>
          <td>2025-08-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) occurs heterogeneously among carcinoma cells to promote chemoresistance. Identifying the signaling pathways involved will nominate drug combinations to promote chemoresponse, but cell population-level studies can be misleading, and single-cell transcriptomics are limited to indirect ontology-based inferences. To understand EMT heterogeneity at a signaling protein level, we combined iterative indirect immunofluorescence imaging of pancreas cancer cells and tumors and mutual information (MI) analysis. Focusing first on mitogen-activated protein kinase pathways, MI indicated that cell-to-cell variation in ERK activity determined EMT heterogeneity in response to different growth factors and chemotherapeutics but that JNK compensated when MEK was inhibited. Population-level models could not capture these experimentally validated MI inferences. The dominant role of ERK was consistently indicated by MI even when the analysis was expanded to include seven potential EMT-regulating signaling nodes. More generally, this work provides an approach for studying multivariate signaling-phenotype relationships based on protein measurements in any setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3bfa194502b5ce8ba1fe00ed578bfccff400f9" target='_blank'>
              The kinase ERK plays a conserved dominant role in the heterogeneity of epithelial-mesenchymal transition in pancreatic cancer cells.
              </a>
            </td>
          <td>
            Michelle C. Barbeau, Brooke A. Brown, S. Adair, Todd W. Bauer, Matthew J. Lazzara
          </td>
          <td>2025-08-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="PURPOSE
With surgery being the only potential cure for pancreatic cancer, high-risk premalignant pancreatic lesions often go unnoticed by palpation or white light visualization, leading to recurrence. We asked whether near-infrared fluorescence imaging of tumor-associated inflammation could identify high-risk premalignant lesions, leveraging the tumor microenvironment (TME) as a sentinel of local disease and, thus, enhance surgery outcomes.


EXPERIMENTAL DESIGN
Fluorescence-guided surgery was performed on genetically engineered mice (Ptf1a-Cre; LSL-KrasG12D/+; Smad4flox/flox [KSC]) at discrete stages of disease progression, histologically confirmed high-risk, premalignant lesions in postnatal mice to locally advanced pancreatic tumors in adults, using the imaging agent V-1520, a translocator protein (TSPO) ligand. Age-matched wild-type littermates were used as controls, while Ptf1a-Cre; LSL-KrasG12D/+(KC) mice modeled pancreatitis and precursors of low penetrance. Localization of V-1520 and tumor-associated macrophages amongst the TME was detected by immunofluorescence imaging.


RESULTS
V-1520 exhibited robust accumulation in the pancreata of KSC mice from the early postnatal stage. Increased accumulation was observed in the pancreata of adolescent- and adult-aged mice with greater ductal lesion and stromal burden. Confocal microscopy of ex vivo pancreas specimens co-localized V-1520 accumulation primarily with CD68-expressing macrophage in KSC mice. Unlike the pancreata of KSC mice, accumulation of V-1520 did not exceed background levels in the pancreata of KC mice with pancreatitis.


CONCLUSION
V-1520 exhibited differential accumulation in pancreatic cancer-associated inflammation compared to pancreatitis. Given the robust tracer uptake in tissues associated with early yet high-risk lesions, we envision V-1520 could enhance surgical resection and reduce the potential for recurrence from residual disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1713383f0cb9b8f04471c53342e9b58b0edaa9" target='_blank'>
              Preclinical Fluorescence-Guided Imaging Leveraging Surrounding Sentinel Tumor Microenvironment Identifies High-Risk Premalignant Pancreatic Lesions.
              </a>
            </td>
          <td>
            Shilpa Sharma, Xiaoxia Wen, Jianbo Wang, Beibei Huang, Denise Hernandez, Cong-Dat Pham, Zhiwen Liu, S. J. Lin, Aiko Yamaguchi, Dimitra K. Georgiou, R. P. Coll, H. C. Manning
          </td>
          <td>2025-08-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabe0cbcb8c826f9af8109abd06e0ae3e7cf0a6a" target='_blank'>
              Patient-derived organoids reveal hypoxia-driven plasticity and therapeutic vulnerabilities in pheochromocytomas and paragangliomas.
              </a>
            </td>
          <td>
            Xiaojing Wang, Maite Calucho, Cynthia M Estrada-Zuniga, Mehwish Noureen, Hector Gonzalez-Cantú, Samantha Mossuto, Jonathan N Levi, Bethany Landry, Qianjin Guo, Kailee A Rutherford, Zi-Ming Cheng, Andreanne Sannajust, Sapna Raghunathan, Sanskriti Balaji, Viviane Nascimento da Conceicao, Summer Norris, S. Y. Munawar, H. Nguyen, Jacob De Rosa, G. Tomlinson, James X Wu, Michael W. Yeh, M. Livhits, Anand Vaidya, James Powers, Ron Lechan, Art Tischler, Miao Zhang, Paul Graham, Camilo Jimenez, M. Peitzsch, Nicole Bechmann, Graeme Eisenhofer, Yanli Ding, Mio Kitano, A. Soragni, P. Dahia
          </td>
          <td>2025-08-23</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            , Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive and heterogeneous malignancy characterized by marked resistance to standard chemotherapy and poor prognosis. While the advent of immunotherapy has revolutionized the management of several solid tumors, including melanoma, breast cancer, and non-small cell lung cancer, its efficacy in iCCA remains limited. Recent clinical trials have demonstrated the efficacy of durvalumab in combination with chemotherapy for iCCA, leading to its approval as a first-line treatment. However, overall response rates remain low, largely due to its immunosuppressive tumor immune microenvironment (TIME). The immune-cold nature of iCCA is typified by a dominant presence of immunosuppressive cell populations, including M2-polarized tumor-associated macrophages, myeloid-derived suppressor cells, and T regulatory cells. In addition, traditional biomarkers such as PD-L1 expression, tumor mutational burden, and microsatellite instability have shown limited predictive value in iCCA, highlighting the need for novel biomarkers and immunotherapeutic strategies. Emerging approaches aimed at reprogramming the TIME, including combination therapies targeting suppressive cells, stromal remodeling, and novel immune effectors like CAR-T and cancer vaccines, hold significant promise for enhancing therapeutic efficacy. This review summarizes the distinct features of iCCA TIME, key mechanisms of immune evasion, current challenges, and future directions to overcome immune resistance, with the aim of developing personalized immunotherapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1167c3442bd04894cf43de06826ab2ab3325aec4" target='_blank'>
              Immune Landscape of Intrahepatic Cholangiocarcinoma: Evasion and Therapeutic Insights
              </a>
            </td>
          <td>
            Nunzia Porro, Elena Spínola-Lasso, F. Marra, A. Gentilini
          </td>
          <td>2025-09-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/781db8d3b9951c3d4f119253d55c69b0ec7522ab" target='_blank'>
              Hybrid In Vivo Breast Cancer Model Reveals Transcriptomic Insights into Cancer Progression with Age.
              </a>
            </td>
          <td>
            Lauren Hawthorne, Gokhan Bahcecioglu, Jun Yang, Erin Howe, Emilija Aleksandrovic, Siyuan Zhang, P. Zorlutuna
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary People with systemic sclerosis (a chronic disease that causes skin and organ scarring) have a higher risk of developing certain cancers soon after diagnosis, but the reasons remain unclear. To study systemic sclerosis in the laboratory, researchers are using tight skin (TSK) mice, which develop excess scar tissue and immune changes similar to systemic sclerosis. We combined the TSK mouse model with mouse models of skin, breast, and ovarian cancers to test whether the unique environment in TSK mice promotes cancer progression. In all cancer types, TSK mice developed more aggressive tumors. Detailed single-cell analyses showed that TSK tumors had more immune imbalance and more scar-promoting cell types. These findings suggest that the TSK mouse is a powerful tool for uncovering how systemic sclerosis-related tissue changes and immune shifts may fuel cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fece107070df629a4a56795a06c184ea38a1b613" target='_blank'>
              Cancer Growth and Invasion Are Increased in the Tight Skin (TSK) Mouse
              </a>
            </td>
          <td>
            M. Recouvreux, Barbie Taylor-Harding, Amy C. Rowat, Beth Y Karlan, Sandra Orsulic
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma is the most prevalent primary brain malignancy, distinguished by its highly invasive behavior and exceptionally high rates of recurrence. Conventional radiation therapy, which employs uniform treatment margins, fails to account for patient-specific anatomical and biological factors that critically influence tumor cell migration. To address this limitation, numerous computational models of glioblastoma growth have been developed, enabling generation of tumor cell distribution maps extending beyond radiographically visible regions and thus informing more precise treatment strategies. However, despite encouraging preliminary findings, the clinical adoption of these growth models remains limited. To bridge this translational gap and accelerate both model development and clinical validation, we introduce PREDICT-GBM, a comprehensive integrated pipeline and dataset for modeling and evaluation. This platform enables systematic benchmarking of state-of-the-art tumor growth models using an expert-curated clinical dataset comprising 255 subjects with complete tumor segmentations and tissue characterization maps. Our analysis demonstrates that personalized radiation treatment plans derived from tumor growth predictions achieved superior recurrence coverage compared to conventional uniform margin approaches for two of the evaluated models. This work establishes a robust platform for advancing and systematically evaluating cutting-edge tumor growth modeling approaches, with the ultimate goal of facilitating clinical translation and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3497012bad0fd14f83f8c07402640538840f77c2" target='_blank'>
              PREDICT-GBM: Platform for Robust Evaluation and Development of Individualized Computational Tumor Models in Glioblastoma
              </a>
            </td>
          <td>
            L. Zimmer, J. Weidner, M. Balcerak, F. Kofler, I. Ezhov, B. Menze, B. Wiestler
          </td>
          <td>2025-09-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Juvenile idiopathic arthritis (JIA) is the most prevalent chronic inflammatory arthritis of childhood, yet the spatial organization in the synovium remains poorly understood. Here, we perform subcellular-resolution spatial transcriptomic profiling of synovial tissue from patients with active JIA. We identify diverse immune and stromal cell populations and reconstruct spatially defined cellular niches. Applying a newly developed spatial colocalization analysis pipeline, we uncover microanatomical structures, including endothelial-fibroblast interactions mediated by NOTCH signalling, and a CXCL9-CXCR3 signaling axis between inflammatory macrophages and CD8+ T cells, alongside the characterization of other resident macrophage subsets. We also detect and characterize tertiary lymphoid structures marked by CXCL13-CXCR5 and CCL19-mediated signaling from Tph cells and immunoregulatory dendritic cells, analogous to those observed in other autoimmune diseases. Finally, comparative analysis with rheumatoid arthritis reveals JIA-enriched cell states, including NOTCH3+ and CXCL12+ sublining fibroblasts, suggesting potentially differential inflammatory programs in pediatric versus adult arthritis. These findings provide a spatially resolved molecular framework of JIA synovitis and introduce a generalizable computational pipeline for spatial colocalization analysis in tissue inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0075173fcf9156b91c3ae01a15628584112c747" target='_blank'>
              Subcellular spatial transcriptomics reveals immune–stromal crosstalk within the synovium of patients with juvenile idiopathic arthritis
              </a>
            </td>
          <td>
            J. Inamo, R. Fierkens, M. R. CLay, A. H. Jonsson, C. Lin, K. Hayes, N. Rogers, H. Leach, K. Yomogida
          </td>
          <td>2025-08-08</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Over the past decade, patient-derived organoids (PDOs) have emerged as powerful in vitro models that closely recapitulate the histological, genetic, and functional features of their parental primary tissues, representing a ground-breaking tool for cancer research and precision medicine. This advancement has led to the development of living PDO biobanks, collections of organoids derived from a wide range of tumor types and patient populations, which serve as essential platforms for drug screening, biomarker discovery, and functional genomics. The classification and global distribution of these biobanks reflect a growing international effort to standardize protocols and broaden accessibility, supporting both basic and translational research. While their relevance to personalized medicine is increasingly recognized, the establishment and maintenance of PDO biobanks remain technically demanding, particularly in terms of optimizing long-term culture conditions, preserving sample viability, and mimicking the tumor microenvironment. In this context, this review provides an overview of the classification and worldwide distribution of tumor and paired healthy tissue-specific PDO biobanks, explores their translational applications, highlights recent advances in culture systems and media formulations, and discusses current challenges and future perspectives for their integration into clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a9d583775632944bea1fb022512dcc7a1ca6c16" target='_blank'>
              Patient-Derived Organoid Biobanks for Translational Research and Precision Medicine: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            F. J. Di Paola, G. Calafato, P. Piccaluga, G. Tallini, Kerry Jane Rhoden
          </td>
          <td>2025-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>2</td>
          <td>57</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor organoids that can accurately recapitulate the pathophysiological characteristics of original tumor are urgently needed for personalized therapy. However, there are few published studies on patient-derived renal cell carcinoma (RCC) heterogeneous organoids for drug testing to account for patient-specific heterogeneous clinical responses, which has significantly impeded research in the field. Traditional RCC organoid technologies involving matrigel droplets require intensive manual manipulation and are hampered by variability, functional immaturity, low throughput, and limited scale. Here, we applied extrusion-based high-throughput bioprinter to rapidly generate heterogeneous RCC organoids with uniform size, realizing batch automated stable construction and quality control. Bioprinted RCC organoids reserved the pathological morphology and gene mutation/expression characteristics of original tumor and demonstrate interorganoid and interpatient heterogeneity even after long-term cultivation, which are suitable for preclinical patient-specific drug screening testing. Finally, we created multicellular assembloids by reconstituting RCC aggregates with stromal components to generate an organized architecture within vivo-like vascular morphology and spatial tumor microenvironment heterogeneity. Thus, we have demonstrated the wide-ranging biomedical utility of bioprinted organoids in furthering our understanding of the physiological mechanisms of tumors and the development of personalized treatment methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d03f5a0c688aba81960dd8501bdbe82b9c1e8" target='_blank'>
              Bioprinting of patient-derived heterogeneous renal cell carcinoma organoids for personalized therapy.
              </a>
            </td>
          <td>
            Shuangshuang Mao, Ruiyang Xie, Jianzhong Shou, Y. Pang, Wei Sun
          </td>
          <td>2025-08-12</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor-immune interactions are shaped by both antigenic heterogeneity and stochastic perturbations in the tumor microenvironment, yet the mathematical mechanisms underlying immune phase transitions remain poorly understood. We propose a four-compartment dynamical model that incorporates antigen accumulation and immune escape mutations. Bifurcation analysis reveals bistability between immune surveillance and immune escape states, providing a mechanistic explanation for heterogeneous immune outcomes during tumor progression. In the multistable regime, the stable manifold of a saddle point partitions the state space into distinct basins of attraction, determining the long-term fate of the system. We further analyze how stochastic fluctuations in the tumor microenvironment perturb these separatrices, potentially triggering irreversible state transitions. By characterizing the critical noise intensity and estimating the tipping time, we establish a mathematical framework for assessing noise-induced transitions. The model further predicts that increasing tumor cell death can improve system resilience to stochastic perturbations, whereas stronger immune pressure may facilitate immune escape-highlighting the nonlinear and non-monotonic nature of tumor-immune dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358f26ec76382702bbe9f0bdc46b3b599d87dd6f" target='_blank'>
              Bistability and Noise-Induced Evasion in Tumor-Immune Dynamics with Antigen Accumulation and Immune Escape
              </a>
            </td>
          <td>
            Mengfan Tan, Shaoqing Chen, Chunjin Wei, Da Zhou
          </td>
          <td>2025-09-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Head and neck squamous cell carcinoma (HNSCC) was a common malignant tumor, and its recurrence and metastasis during treatment were the main factors affecting the patient's prognosis. This study aimed to explore the evolutionary mechanisms during HNSCC metastasis. Methods This study collected single-cell RNA sequencing (scRNA-seq), spatial transcriptome (ST) data, and fatty acid metabolism-related genes from public databases for pseudo-chronological, differentiation, cell interaction, and pathway analysis. In vivo and in vitro experiments were conducted to study the expression of LGALS1 and its role in HNSCC cells. Results Three specific subclusters in the tumor metastasis process were identified, including primary tumors, transitional tumors, and metastatic tumors. During tumor evolution, fatty acid metabolism was upregulated, and active fatty acid metabolism involving LGALS1 was related to HNSCC cell metastasis. Knocking down LGALS1 significantly inhibited the proliferation, migration, and lymph node metastasis ability of HNSCC cells and changed the expressions of E-cadherin, Snail, and PPARγ at the protein level. Discussion This study described metabolic changes during HNSCC dissemination and revealed the critical role of metastatic tumors in the mechanism of HNSCC metastasis by regulating fatty acid metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc7d9de2a2aec8cef12eb25e91da5524696d3550" target='_blank'>
              Integrated analysis of single cell and spatial transcriptomics revealed a metastasis mechanism mediated by fatty acid metabolism in lymph nodes of head and neck cancer
              </a>
            </td>
          <td>
            Jinru Weng, Jiajie Mao, Yixing Li, Jun Zhao, Xiaolin Nong
          </td>
          <td>2025-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioma, a common primary brain tumor arising from glial cells, presents significant therapeutic challenges due to its diverse and complex biological behavior. While disulfidptosis has recently emerged as a novel form of cell death, the role of disulfidptosis-related genes (DRGs) in glioma and their relationship with the tumor immune microenvironment remains poorly understood. Methods We analyzed DRGs expression patterns in glioma samples and their correlation with patient survival using data from TCGA, GEO, and CGGA databases. We employed ROC curve analysis to evaluate the diagnostic potential of individual DRGs and identify hub genes. We investigated the relationship between hub gene expression and immune cell infiltration through immune infiltration analysis. Additionally, we conducted drug sensitivity analysis and molecular docking to explore associations between hub gene expression and drug responsiveness. Finally, we validated DRGs expression in glioma samples using RT-PCR and tissue microarrays. Results Among the ten DRGs examined, six genes (OXSM, SLC3A2, RPN1, NDUFS1, GYS1, and LRPPRC) showed significant differential expression in glioma tissues, while four (NCKAP1, NDUFA11, NUBPL, and SLC7A11) did not. Survival analysis revealed that elevated expression of OXSM, RPN1, GYS1, SLC3A2, and NDUFA11 correlated with poor patient outcomes. ROC analysis of both TCGA test and GEO validation sets identified OXSM as a hub gene, with AUC values exceeding 0.88. CGGA database analysis demonstrated higher OXSM expression in GBM compared to LGG, with peak expression in WHO grade IV tumors, significantly correlating with reduced overall survival (p < 0.01). Immune infiltration analysis showed that OXSM expression correlated with various immune cell populations, particularly significantly positively correlated with macrophages M0, M2, and T cells gamma delta, and significantly negatively correlated with monocytes. Drug sensitivity analysis indicated that high OXSM expression enhanced sensitivity to several therapeutic agents. RT-PCR and tissue microarray analyses confirmed elevated OXSM expression in high-grade gliomas. Conclusion Our study identifies OXSM as a key DRG hub gene in glioma, with elevated expression strongly correlating with poor patient prognosis. The observed relationship between OXSM expression and immune cell infiltration suggests its potential role in modulating the glioma microenvironment. These findings advance our understanding of glioma’s molecular mechanisms and may inform the development of novel therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03491-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/422822b52751a8402e0fa583f0768bc440735523" target='_blank'>
              Disulfidptosis OXSM serves as a potential prognostic biomarker and correlates with immune infiltrates in glioma
              </a>
            </td>
          <td>
            Jinchao Zhu, Yao Ke, Yijuan Fan, Qingyuan Lin, Weiyao Zhu, Wenhuan Guo, Shengdong Nie
          </td>
          <td>2025-09-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. B. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead4b3d9eb38d6bf256d4b5b9585b58a77f9b5e1" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, Jeong Hwan Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-09-01</td>
          <td>Nature Methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Introduction Osteosarcoma (OS) is a malignant bone tumor with high metastatic potential and poor long-term survival. The tumor immune microenvironment and metabolic reprogramming are increasingly recognized as key drivers of OS progression, yet the molecular links between these systems remain unclear. This study aimed to identify immune-metabolic biomarkers in OS, focusing on pleckstrin (PLEK) as a potential regulatory hub. Methods We conducted differential expression and survival analyses using OS transcriptomic datasets and TCGA/GTEx data. Protein–protein interaction networks, GO/KEGG enrichment, and CytoHubba algorithms identified core hub genes. Tumor-infiltrating immune cells were profiled via TIMER. Single-cell RNA-seq (GSE162454) was used for immune and metabolic landscape mapping. PLEK was further validated by qRT-PCR and Western blot in OS samples, and its function assessed via siRNA knockdown in macrophages within TME co-cultured with OS cells. Cell proliferation, migration, and invasion assays evaluated phenotypic effects in OS cells. Results Nine hub genes were identified, with PLEK significantly upregulated in OS tissues. High PLEK expression correlated with improved survival and increased infiltration of macrophages, dendritic cells, and CD4+ T cells. Single-cell analysis showed PLEK enrichment in macrophage-dominated clusters with active glycolytic and oxidative phosphorylation pathways. Downregulation of PLEK in macrophages enhanced OS cell proliferation, migration and invasion. These findings suggest PLEK is linked to a pro-immune, metabolically active microenvironment and may act as a tumor suppressor. Discussion Our study identifies PLEK as a prognostic biomarker and functional regulator in OS. It promotes an immune-infiltrated, metabolically active tumor microenvironment and is associated with attenuated malignant phenotypes in vitro. These findings highlight PLEK as a promising target for immunometabolic modulation in OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c70b1d04d51bb43e31b94e513209045d07404134" target='_blank'>
              The osteosarcoma immune microenvironment in progression: PLEK as a prognostic biomarker and therapeutic target
              </a>
            </td>
          <td>
            Yunpeng Zou, Jianning Kang, Shaopeng Zhu, Xuechen Ren, Zheng Li, Jiayao Niu, Xuanzhe Qin, Hongbo Li, Lu Xiang, Wei Jiang, Jiangbo Zhong, Ying Zhang, Kai Zhao
          </td>
          <td>2025-08-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with a dismal five-year survival rate of 13%. Most patients present with metastatic disease, and 80% undergoing surgery relapse within two years. Research on PDAC has traditionally focused on primary tumors, thus most treatment derives from primary tumor biology. Recent evidence unveiled significant differences in the tumor immune microenvironment (TIME) between primary and metastatic lesions. Understanding these differences is crucial, as immunotherapies are predominantly administered to patients with refractory metastatic PDAC, yet traditional treatment fails to account for organ specificity and patient TIME signatures. In this study, we wanted to know why the same cancer cell type in different organs would create different microenvironments in the pancreas, liver, and lungs. To do this, we investigated the TIME in early and late-stage metastatic cohorts by multiplex immunohistochemistry (mIHC), transcriptomic assays and spatial analysis. To profile the TIME of different immune and stromal cells, we employed spatial protein expression using mIHC platform for the early cohort. In total, we applied a 30-plex panel to 56 samples. Our analysis revealed a higher density of COL1A1+ fibroblasts and CD163+ macrophages in the liver compared to other sites. Additionally, liver metastases exhibited significantly fewer CD8+ T cells compared to other organs within the tumor core. These results confirmed the earlier studies regarding the TIME of metastatic lesions, while also showing more macrophage and fibrosis in the liver. In the late-stage cohort, similar trends of decreased immune cell presence and exclusion phenotypes were observed using mIHC and Visium platforms. To explore the underlying mechanisms, we performed transcriptomic assays on the immune compartment. Our findings indicated an enrichment of interferon-positive macrophage subsets, particularly in liver metastases compared to primary PDAC. Additionally, we observed a higher presence of exhaustive CD8+ T cells and regulatory T cells in primary sites compared to liver and lung metastases, suggesting that both primary PDAC and liver metastases possess an immunosuppressive environment. Furthermore, spatial proximity analysis demonstrated that CD8+ T cells were closer to tumor cells in lung metastases compared to liver and primary tumors, reinforcing the notion that lung metastases are more immune-active. Pathway analysis revealed downregulation of antigen presentation and lymphocyte proliferation signaling in liver T cells compared to those in the lungs. Notably, immune infiltration was observed in some liver samples, but the percentage was significantly lower compared to lung samples. This divergence in TIME modulation between organs suggests a significant impact on disease progression in PDAC patients. The significant difference in immuno-response across organ shows why it is critical to consider distinct immune landscapes of different organs and patient histories when developing tailored therapies for advanced PDAC patients.



 Yu-Lan Kao, Olivia M. Dres, Blake G. Sells, Varun Shenoy, Xiang Li, Kapur Dhami, Paul M. Grandgenett, Li Ding, Liang-I Kang, David G. DeNardo. Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A084.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846e100bfc6ae1b16ca4e9e6b72859a2dc02155f" target='_blank'>
              Abstract A084: Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Yu-Lan Kao, Olivia M Dres, Blake Sells, Varun V Shenoy, Xiang Li, Kapur B. Dhami, P. Grandgenett, Li Ding, Liang-I Kang, D. DeNardo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Both single nucleotide variants (SNVs) and somatic copy number alterations (SCNAs) accumulate in cancer cells during tumour development, fuelling clonal evolution. However, accurate estimation of clone-specific copy numbers from bulk DNA-sequencing data is challenging. Here we present allele-specific phylogenetic analysis of copy number alterations (ALPACA), a method to infer SNV and SCNA coevolution by leveraging phylogenetic trees reconstructed from multi-sample bulk tumour sequencing data using SNV frequencies. ALPACA estimates the SCNA evolution of simulated tumours with a higher accuracy than current state-of-the-art methods1–4. ALPACA uncovers loss-of-heterozygosity and amplification events in minor clones that may be missed using standard approaches and reveals the temporal order of somatic alterations. Analysing clone-specific copy numbers in TRACERx421 lung tumours5,6, we find evidence of increased chromosomal instability in metastasis-seeding clones and enrichment for losses affecting tumour suppressor genes and amplification affecting CCND1. Furthermore, we identify increased SCNA rates in both tumours with polyclonal metastatic dissemination and tumours with extrathoracic metastases, and an association between higher clone copy number diversity and reduced disease-free survival in patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Immunotherapy has progressively gained prominence as a cornerstone therapeutic modality across diverse oncological contexts, with its clinical efficacy intricately linked to the dynamic interactions between the tumor microenvironment (TME) and neoplastic cells. Central to this paradigm is angiogenesis—a quintessential hallmark of cancer—which not only sustains tumor growth but also orchestrates immunomodulatory networks within the TME, thereby profoundly influencing therapeutic responsiveness. However, in the field of bladder cancer (BC), the relationship among angiogenesis and prognosis, immunotherapy response, and immune cell infiltration remains to be further explored. Methods To systematically uncover this relationship, we carried out an exhaustive assessment of 36 genes linked to angiogenesis (AAGs) and explored the relationship between angiogenesis and transcript, prognostic outcomes, as well as the infiltration of immune cells. By constructing an AAG_score, we quantified the angiogenic subtype characteristics of each patient. Subsequently, we evaluated the value of these characteristics in foreseeing BC prognosis and the response of treatment, and concurrently analyzed the performance of AAGs in diffuse large B-cell lymphoma for comparative study. Through RT-qPCR, CCK8 and other experiments, we verified the role of NID2 in bladder cancer. Results This study explored different types of AAGs mutations in BC samples at the genetic level and elucidated the expression patterns of AAGs. Through in-depth analysis, we identified two distinct molecular subpopulations and found significant associations between AAG mutations and patients’ clinicopathological features, prognosis, and invasive TME. We found that patients with low AAG_score showed increased microsatellite instability, high mutation tendency, and immune motivation, and had a better prognosis. NID2 plays a role in promoting proliferation in bladder cancer and evaluate in diffuse large B-cell lymphoma. In addition, our study found a significant correlation between index of cancer stem cell and AAG_score in drug sensitivity. Conclusion In summary, our study successfully identified prognosis-related AAG characteristics in BC patients. These characteristics not only contribute to a clearer understanding of TME properties but also provide an important basis for exploring more effective immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e0c3a38e058744faab2e7b9e1bb3c40f92cc4d" target='_blank'>
              Angiogenesis-related gene NID2 profiling and immune infiltration in bladder cancer: prognostic implications and immunotherapy response
              </a>
            </td>
          <td>
            Jinxia Cao, Wang Zheng
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Hypoxia-inducible factor (HIF) is a master regulator of cancer cell adaptation to tumor hypoxia and is involved in cancer progression. Single-cell (sc) differences in the HIF response allow for tumor evolution and cause therapy resistance. These sc-differences are usually ascribed to tumor microenvironmental differences and/or clonal (epi)genetic variability. However, the sc-heterogeneity of the HIF response in otherwise identical cells cultured under defined in vitro conditions has not yet been addressed. Therefore, we analyzed the sc-response to hypoxia in nonclonal cell lines and multiple clonal derivatives, including HIF-1α or HIF-2α knockouts. While HIF-1α and HIF-1 target mRNA sc-heterogeneity was slightly higher than global transcription or specific housekeeping messenger RNAs (mRNAs), HIF-2α and especially HIF-2 target mRNA sc-heterogeneity was extraordinary, and remained in independent clones following HIFα knockouts. Unexpectedly, neither HIF-2α mRNA nor nuclear protein levels correlated with target mRNA levels. Unsupervised but not supervised HIF target gene dimensionality reduction revealed the initial sample composition after scRNA-seq, demonstrating that, owing to sc-heterogeneity, individual HIF target genes are not sufficient to unequivocally identify hypoxic cancer cells. In conclusion, the pronounced intrinsic sc-heterogeneity of the HIF response represents a hitherto unrecognized feature of cancer cells that impairs clinical HIF pathway-dependent cancer cell identification and targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b54dd117e53122de89768b0fc8a01ce303d40e2" target='_blank'>
              Inherent single-cell heterogeneity of the transcriptional response to hypoxia in cancer cells
              </a>
            </td>
          <td>
            Małgorzata Wilk, Thomas Knöpfel, Stana M Burger, S. N. Khodo, Roland H. Wenger
          </td>
          <td>2025-08-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung adenocarcinoma has become a paradigm for precision oncology and success with targeted therapies. However, large chunks of the pie chart of molecular drivers remain unknown, preventing many patients from similarly benefiting from this strategy. Here, we highlight the side-by-side release of two impactful studies from Mozzarelli and colleagues and DiMarco and colleagues in this issue of Cancer Research. Here, each team generated unique mouse models characterizing a novel and potentially actionable driver and facilitator of acquired drug resistance in lung adenocarcinoma, RIT1M90I. Both study teams describe and characterize their new mouse models, reporting the tumor latency and importantly revealing the sensitivity of RIT1-mutant lung tumors to SHP2 or RAS tricomplex inhibitors. Collectively, these findings highlight the importance of RIT1M90I as an actionable driver of lung adenocarcinoma and a potent mediator of acquired drug resistance. These new RIT1-driven lung cancer models represent valuable tools for the preclinical testing and translation of different therapeutic strategies for this patient population. See related article by Mozzarelli et al., p. 3196 See related article by DiMarco et al., p. 3207.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/802429b265b154e6d63054d90ce37155bed940f9" target='_blank'>
              Another Slice of the Pie Uncovered: RIT1M90I Is an Actionable Driver in Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Cheng Pei Wu, A. Vaishnavi
          </td>
          <td>2025-09-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, 
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transforming Growth Factor (TGF) and Bone Morphogenetic Protein (BMP) signalling critically influence pancreatic ductal adenocarcinoma (PDAC) progression, with TGF-B paradoxically exerting both tumour-promoting and -suppressive effects. Parallel to this observation, the specific context-dependent, spatial dynamics of these pathways and their interaction with the tumour microenvironment (TME) remain poorly understood. Methods We performed a spatially resolved analysis of PDAC on a multi-region tissue microarray cohort of 117 curatively resected PDAC specimens consisting of tumour centre (TC), tumour front (TF), and stromal(-predominant) tissue cores each. Protein (ID1, pSMAD2) and mRNA (TGF-A, TGF-B1/2, BMP4, GREM1) expression were assessed in each tissue compartment by immunohistochemistry and in situ hybridization, respectively, quantified by digital image analysis, and correlated with clinicopathologic features. Results ID1 was significantly overexpressed in PDAC cells compared to associated stroma (p < 0.01), while pSMAD2 was largely absent in PDAC cells, but preserved among associated stroma compartments, particularly in TF cores (p = 0.04). Higher stromal GREM1 signal correlated with reduced overall tumoural ID1 protein expression (p = 0.02), and TGF-B2high/TGF-Alow stroma was significantly associated with worse survival (p < 0.01). Intratumoural TGF-B2 was inversely correlated with stromal pSMAD2 expression (p = 0.03) and was associated with lymph node involvement (p = 0.02). FOXP3+ regulatory T-cells were significantly reduced in TGF-B2high tumours (p = 0.04), while higher tumoural TGF-B1 exhibited a trend towards increased FOXP3+ cells (p = 0.08). Conclusions Our spatial analysis reveals intratumoural heterogeneity of TGF/BMP signalling and its significance for PDAC progression. Notably, stromal TGF-B2 emerges as a prognostic biomarker, while TGF-B1 and ID1 are implicated in adverse clinical and pathologic features. These findings highlight the importance of TGF/BMP signalling niches in the TME with implications for PDAC biology and can inform the development of future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14751-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0085a0bcef90131114a7aad5d7931cab1f83ce7d" target='_blank'>
              Spatially resolved analysis of TGF/BMP signalling in pancreatic ductal adenocarcinoma by digital pathology identifies patient subgroups with adverse outcome
              </a>
            </td>
          <td>
            Konstantin Bräutigam, Philipp Zens, Stefan Reinhard, Jessica L. Rohrbach, Simon J Leedham, A. Wenning, B. Gloor, V. H. Koelzer, Martin Wartenberg
          </td>
          <td>2025-08-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSC) typically originates in the Fallopian tube epithelium (FTE) and is characterized by genomic instability and near-universal TP53 defects. HGSC is typically diagnosed at an advanced stage and rapid progression once it becomes invasive. To date, immunotherapies have had limited efficacy in HGSC. Most current HGSC research focuses on tumor and immune profiling of fully fledged HGSC, where in the tumor microenvironment (TME) has already been established. Much less is known about the microenvironment of pre-neoplastic lesions and the role of the immune system in controlling HGSC emergence. Hence, there is an urgent need for in-depth research on the molecular interactions between precancerous cells and immune cells to better understand the process of malignant transformation and escape from host immunity.



 By using our genetically defined, syngeneic organoid platform, FTE organoids were engineered to Trp53 -/- ,Ccne1-overexpressing FTE organoids (“PC” organoids) and Brca1 +/- ;Trp53 -/- (PBrca1) and Brca2 -/- ;Trp53 -/- (PBrca2) to model Homologous recombination (HR)-deficient (HRD) and HR-Proficient (HRP) HGSC, respectively. Pre-tumor growth was monitored using an in vivo imaging system (IVIS) following luciferase-labeled precancerous cell ovarian bursal injection. In addition, adaptive transfer and antibody deletion were performed to determine which immune cell type(s) mediate immune surveillance.



 In a previous study, our laboratory found that PC organoids are not tumorigenic in wild-type (WT) C57BL6 mice. In this study, I found that PC cells were eliminated around 1 month after bursal injection in WT mice. Surprisingly, however, they kept growing and eventually formed tumors in Rag2 -/- mice, which lack B and T lymphocytes. Then, T cells were adaptive transfer from WT mice bearing PC cells to Rag2 -/- mice. I observed a marked reduction in PC cell signal as early as 24 hours post-transfer, which suggests T cells play a critical role in the clearance of precancerous cells. I also found compared to PC cell, PBrca1 and PBrca2 FTE organoids, both of which are non-tumorigenic, are eliminated much earlier (in 24 hours) and was completed in rapidly (∼15 days) following bursal injection into both WT and Rag2 -/- mice. Following NK cells deletion in Rag2 -/- mice, I observed that PBrca1 cells successfully established growth, whereas PBrca2 cells exhibited significantly delayed clearance compared to the IgG control group. Additionally, compared to PC cells, the expression of mouse MHC-Ib Qa-1b, which inhibits NK cell–mediated cytotoxicity, was markedly reduced in PBrca1/2 cells. Although other explanations are possible, these data raise the possibility that PBrca1/2 cells elicit a strong NK cell mediated innate immune response.



 This study represents a novel and highly feasible approach to interrogating the pre-cancer TME. My findings demonstrate that HRD (PBrca1/2) and HRP (PC) trigger markedly divergent immune surveillance mechanisms during the pre-malignant phase.



 Jianmei Hou, Benjamin Neel. Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263093496afb42b25c3ac1b6f0bdab9faf970516" target='_blank'>
              Abstract PR003: Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer
              </a>
            </td>
          <td>
            Jianmei Hou, Benjamin Neel
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: While the immunomodulatory function of FAM174B in bladder cancer (BLCA) has yet to be fully elucidated, elucidating its biological mechanisms could potentially enhance immunotherapeutic outcomes for this malignancy. Methods: Bulk RNA-seq data from TCGA and GEO databases were analyzed to investigate FAM174B expression patterns and immune landscape characteristics in pan-cancer. The immunoregulatory role of FAM174B in BLCA was systematically evaluated through immune infiltration analysis, immunomodulator profiling, cancer-immunity cycle assessment, and immune checkpoint examination. Validation was performed using the IMvigor210 immunotherapy cohort and a combined GEO dataset (n=871). A machine learning-based immune-related signature was developed for prognostic and therapeutic response prediction. Results: FAM174B was highly expressed in cancer tissues across multiple human cancer types including BLCA. Specifically, FAM174B negatively correlated with various immunological features including immunoregulators and immune cell infiltration abundances, suggesting that FAM174B remodeled the microenvironment to a non-inflamed phenotype of BLCA. Besides, patients with high-FAM174B expression may process limited sensitivity to immunotherapy and increased likelihood of hyperprogression. Consensus molecular classification analysis indicated that elevated FAM174B is related to a luminal BLCA subtype which was characterized by reduced immune infiltration, inhibited immuno- and chemo-therapeutic responses, yet increased response to angiogenesis inhibitors and targeted therapy. Furthermore, the immune-related signature, formulated through a machine learning-integrated approach, is shown to be a dependable indicator for predicting cancer prognosis and the efficacy of immunotherapy responses for BLCA. Conclusion: Given the pivotal role of FAM174B in shaping the non-inflamed tumor microenvironment of BLCA, therapeutic targeting of FAM174B may represent a promising strategy for BLCA management. Furthermore, FAM174B expression could serve as a potential biomarker for predicting molecular subtypes and treatment responsiveness in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdbef21d4d57034ada40cc74a787f10325419b4" target='_blank'>
              FAM174B remodels the tumor microenvironment, inhibits the infiltration of macrophage, predicts the molecular subtype and therapeutic response of bladder cancer
              </a>
            </td>
          <td>
            Hualin Chen, Lin Ma, Zhigang Ji, Jie Dong
          </td>
          <td>2025-08-11</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor associated macrophages (TAMs) in Head and neck squamous cell carcinoma (HNSCC), particularly M2-polarized subtypes, are pivotal drivers of tumorigenesis, angiogenesis, and metastasis, contributing to adverse clinical outcomes. Current prognostic markers lack precision, underscoring the need for novel biomarkers and risk stratification models. Single-cell RNA sequencing (scRNA-seq) was applied to profile the transcriptional landscape of TAMs in HNSCC at single-cell resolution. 1,208 M2 TAMs were integrated from scRNA-seq data with bulk RNA sequencing to identify molecular signatures. Weighted correlation network analysis (WGCNA) and Uniform Manifold Approximation and Projection (UMAP) analysis were applied to dissect TAMs heterogeneity and interactions within the tumor microenvironment. In vivo experiments validated the efficacy of the prognostic signature model. In this study, high infiltration of M2 TAMs was strongly associated with advanced clinical stages, lymph node metastasis, and reduced overall survival (P<0.001). TCGA datasets were utilized for cross-platform verification. Multivariate Cox regression and survival analyses were performed to establish prognostic relevance. 11 prognostic signature genes (FCGBP, GIMAP5, WIPF1, RASGEF1B, GIMAP7, IGFLR1, GPR35, NCF1, CLECL1, HEXB, IL10) were identified through integrative analysis, which formed the basis of a robust risk stratification model. The distribution of biomarkers in the high-risk group, as determined by the signature we constructed, can serve as a better indicator for assessing poor prognosis. In clinical samples, prognosis signature has the potential to predict the prognosis effectively in patients with HNSCC.M2 TAMs-driven prognostic signature for HNSCC offers a clinically actionable tool for risk stratification and outcome prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec7d002b51c3bdc988873ba7fa649cd3f831557" target='_blank'>
              Combined single-cell RNA-seq and bulk RNA-seq construction of M2 TAMs signature for predicting HNSCC prognosis and immunotherapy
              </a>
            </td>
          <td>
            Jiale Wang, Huan Li, Mingrui Shi, Chenghao Ren, Wu Wei, Qi Zhao, Xinxin He, Zi-Xia Yang, Jianhua Wei, Xinjie Yang
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the deadliest diseases, with a 5-year survival rate of only 13%. While rare mutations have been identified to drive PDAC therapy resistance, undiscovered potent, post-transcriptional regulatory mechanisms could be key to understanding this resistance. The RNA-binding protein HuR (ELAVL1) is known to play a critical role in PDAC oncogenesis by stabilizing mRNA transcripts involved in key oncogenic pathways. Our lab has shown that HuR is overexpressed in PDAC tumor and promotes multiple pro-survival signaling pathways. To study HuR-mediated PDAC progression in immune competent models that reflect the tumor microenvironment, we deleted ELAVL1 via CRISPR-Cas9 in Kras-p53 mutant-driven (KPC) murine PDAC cells and found that HuR-knockout (KO) KPC tumors grew more slowly compared to KPC wild-type (WT) tumors in orthotopic implantation settings. Using flow cytometry and multiplex immunohistochemistry, we found that HuR-WT tumors had significantly less T cell infiltration and greater endothelial cell infiltration compared to HuR-KO tumors. Deep transcriptomic and proteomic profiling revealed that HuR mediates the expression of multiple metabolic pathways and determines mRNA and protein cargo loading in secreted extracellular vesicles (EVs). In fact, when mice bearing HuR-KO tumors were injected with purified EVs from HuR-WT cells, both tumor size and endothelial cell function in leukocyte trafficking were rescued. In the context of T cell function, HuR directly enriched transcripts in the mTOR pathway, and inhibition of mTOR activity in HuR-WT tumor cells rescued T cell function in co-culture. Consistent with these observations linking tumor-intrinsic HuR to dysregulated endothelial and anti-tumor immune function, HuR-KO tumors were significantly more responsive to immune checkpoint blockade therapy as compared to WT-HuR tumors. Similarly, in a Kras-MYC driven mouse model, HuR-WT tumor cells metastasized at significantly higher rates to the liver compared to HuR-KO tumors. Finally, given the recent advancements in KRAS inhibitors, we tested their efficacy in these pre-clinical models, and surprisingly, we observed that HuR-KO tumors were up to >100-fold more resistant to KRAS inhibition than WT tumors. These observations could be explained by HuR expression and activity being closely linked to aberrant KRAS signaling, where loss of HuR leads to the reliance of compensatory pathways and drives PDAC cells away from a KRAS addiction phenotype, thus providing a setting of KRAS inhibitor resistance. Accordingly, surviving PDAC cells post-KRAS inhibitor treatment in vivo lost HuR expression compared to untreated tumors. Together, these findings highlight that HuR is a global post-transcriptional regulator that: 1) reshapes the tumor microenvironment to be immune evasive; 2) is involved in a feed-forward loop enhancing KRAS pathway activation; and 3) facilitates PDAC metastatic potential. Strategies to target HuR activity and use it as a potential biomarker could lead to improved therapeutic outcomes.



 Miffy Guo, Jennifer Finan, Hen Halamish, Katherine Pelz, Robert Eil, Rosalie Sears, Jonathan Brody. HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ebd5486262a2dcbf20ec9a786ef9c644e158e4" target='_blank'>
              Abstract A027: HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies
              </a>
            </td>
          <td>
            Miffy Guo, Jennifer M. Finan, Hen Halamish, Katherine R. Pelz, Robert Eil, Rosalie C. Sears, Jonathan R. Brody
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite advances in understanding the morphological disruptions that lead to defects in palate formation, the precise perturbations within the signaling microenvironment of palatal clefts remain poorly understood. To explore in greater depth the genomic basis of palatal clefts, we designed and implemented the first single cell spatial RNA-sequencing study in a cleft palate model, utilizing the Pax9−/− murine model at multiple developmental timepoints, which exhibits a consistent cleft palate defect. Visium HD, an emerging platform for true single-cell resolution spatially resolved transcriptomics, was employed using custom bins of 2 × 2 μm spatial gene expression data. Validation of spatial gene expression was then validated using custom designed Xenium In Situ mRNA spatial profiling and RNAscope Multiplex assays. Functional enrichment analysis revealed a palate cell-specific perturbation in Wnt signaling effector function in tandem with disrupted expression of extracellular matrix genes in developing mesenchyme. As a key step toward laying the framework for identifying key molecular targets these data can be used for translational studies aimed at developing effective therapies for human palatal clefts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14807-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a33f260de07738fe1e1b7eb3fbbd247023eb604" target='_blank'>
              Single cell spatial transcriptomics links Wnt signaling disruption to extracellular matrix development in a cleft palate model
              </a>
            </td>
          <td>
            J. O. Piña, R. Raju, Evan Stipano, A. Myo, Ziyi Wang, M. Ono, P. Chattaraj, M. Furukawa, Rena N. D’Souza
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo- and targeted therapy resistant state even after long term culture ex vivo. These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo- and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a unique subtype of breast cancer characterized by high invasiveness, high metastasis rates, and poor prognosis, making it an important focus within global malignancies. Due to the absence of estrogen receptor, progesterone receptor, and HER2 expression, TNBC presents significant challenges in treatment. Metastatic progression markedly increases treatment complexity, drastically reducing patient survival rates. The metastatic and drug resistance processes of TNBC involve complex, multi-step biological mechanisms regulated through various molecular mechanisms and signaling pathways within and outside tumor cells. In recent years, immunotherapy has brought new hope for TNBC. Compared to other breast cancer subtypes, TNBC demonstrates higher immunogenicity, often accumulating a higher mutational burden that generates more neoantigens, thus typically resulting in a tumor microenvironment (TME) enriched with tumor-infiltrating lymphocytes (TILs). Additionally, PD-L1 expression is significantly higher in TNBC compared to other subtypes, closely correlating with TIL abundance. These characteristics position TNBC as a strong candidate for immune checkpoint inhibitor (ICI) therapy. Clinical trials have demonstrated promising efficacy of ICIs in TNBC, overturning previous beliefs that breast cancer is generally insensitive to immunotherapy. This review summarizes recent advances regarding resistance types, molecular mechanisms, associated genes and pathways, the role of the tumor microenvironment, and clinical strategies related to immunotherapy resistance in the neoadjuvant setting of TNBC, aiming to provide insights and guidance for future research exploration and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03cacb55534d35ef52652d44531e93baccfb5003" target='_blank'>
              Immunotherapy resistance in triple-negative breast cancer: Molecular mechanisms, tumor microenvironment, and therapeutic implications
              </a>
            </td>
          <td>
            Zijian Zhou, Qin Zhou
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of chronic inflammation-associated cancers (CIACs) is shaped by cycles of injury and maladaptive repair, yet the principles organizing fibrotic stroma in these tumors remain unclear. Here, we applied the concept of hot versus cold fibrosis, originally credentialed in non-cancerous fibrosis of heart and kidney, to lung squamous cell carcinoma (LUSC), a prototypical CIAC. Single-cell transcriptomics of matched tumor and adjacent-normal tissue from 16 treatment-naive LUSC patients identified a cold fibrotic architecture in the LUSC TME: cancer-associated fibroblasts (CAFs) expanded and adopted myofibroblast and stress-response states, while macrophages were depleted. This macrophage-poor, CAF-rich stroma was maintained by CAF autocrine growth factor loops, including TIMP1, INHBA, TGFB1, and GMFB. In parallel, the immune compartment exhibited a hot tumor phenotype with abundant T and B cells, forming spatially distinct but molecularly engaged networks with CAFs. CAF gene programs typifying cold fibrosis in LUSC were conserved in other CIACs, including esophageal and gastric adenocarcinomas. These results redefine desmoplastic regions of tumors through the lens of a non-cancer fibrosis model, demonstrating that conserved stromal circuits constitute therapeutic vulnerabilities in CIACs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ed36e6ea62839a2c887c4f2b23f5bf7ed74c7d" target='_blank'>
              A fibroblast-centric network drives cold fibrosis in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Shoval Miyara, Shachaf Frenkel, Avi Mayo, Philippe Gascard, Michael K. Strasser, David Gibbs, Eviatar Weizman, Itay Ben Shalom, Yan Stein, Deng Pan, Joseph A. Caruso, V. Sangwan, Nicholas Bertos, Julie Bérubé, Sophie Camilleri-Broet, Spyridon Oikonomopoulos, H. Djambazian, K. Umansky, Jacob Elkahal, Shimrit Mayer, Pierre Fiset, Jiannis Ragoussis, Miri Adler, Eldad Tzahor, Sui Huang, Lorenzo Ferri, T. Tlsty, Ruth Scherz-Shouval, U. Alon
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are malignant cells that detach from primary or metastatic tumors and enter the bloodstream. Organoids, as three‐dimensional in vitro models, can mimic the tumor microenvironment and histopathological characteristics, thereby serving as valuable tools in tumor research. CTC‐derived organoids retain tumor heterogeneity and metastatic potential, which provides a unique model for the study of metastatic cascade mechanisms, individualized drug screening, and precision therapy. However, the current research on CTC‐derived organoids faces challenges, such as the scarcity of CTCs and the high technical difficulty in their isolation and enrichment, which leads to a low success rate in constructing organoid models. Moreover, most existing studies focus on a single cancer type and lack systematic integration of full‐process standardization as well as cross‐cancer applicability. In this paper, we review the isolation and enrichment strategies of CTC‐derived organoids along with the techniques for optimizing in vitro culture systems, and discusses their potential applications. This review summarizes the existing results, analyzes the technical bottlenecks, and provides a theoretical basis for the standardized construction and application of CTC‐derived organoids, while promoting their application in tumor precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971e10876cd9eda4f902688d2d69a778549269d1" target='_blank'>
              Circulating Tumor Cell‐Derived Organoids: Current Progress, Applications, and Future
              </a>
            </td>
          <td>
            Tiantian Li, Biao Deng, Shenglong Li, Yanxia Wu, Zhenghao Lu, Zhu Liang
          </td>
          <td>2025-09-01</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ec1e1293acdded1e916941aba4cadc68a25b1c" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution.
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, Ramin Shirali Hz, J. Severens, M. Honders, S. Kloet, Hendrik Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2025-09-15</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Background Cervical cancer remains a significant health burden for women worldwide, with persistent high‐risk HPV infection being a major etiological factor. Despite treatment advances, prognosis for recurrent or metastatic disease remains poor. Pyroptosis, a form of programmed cell death, plays a dual role in tumor immunity, but its implications in cervical cancer are not fully elucidated. This study aims to systematically characterize pyroptosis‐related genes (PRGs) in cervical cancer and explore their prognostic and therapeutic relevance. Methods Multi‐omics data from TCGA and GEO databases were integrated to analyze genetic variations, expression patterns, and prognostic significance of 52 PRGs in cervical cancer. Consensus clustering was used to identify PRG subtypes. A prognostic risk score model was constructed using LASSO‐Cox regression based on differentially expressed genes (DEGs). Functional validation was performed via in vitro and in vivo experiments, including Western blot, CCK‐8, colony formation, transwell assays, and a subcutaneous tumor model. Single‐cell RNA sequencing data (GSE171894, GSE168652) were analyzed to explore LAG3 expression in the tumor immune microenvironment. Results Two distinct PRG subtypes were identified, with subtype A showing immune activation features. A five‐gene prognostic signature (GNAZ, LAG3, IL‐1β, CA2, SPRR3) effectively stratified patients into high‐ and low‐risk groups. Low LAG3 expression was associated with poor prognosis. Functional experiments demonstrated that LAG3 overexpression suppressed cervical cancer cell proliferation, migration, and tumor growth, while its knockdown promoted malignant phenotypes. Single‐cell analysis revealed high LAG3 expression in Treg and CD8⁺ T cells, suggesting its role in immune regulation. Conclusion This study establishes a novel PRG‐based prognostic model and highlights LAG3 as a key tumor suppressor and immune regulator in cervical cancer. These findings provide insights into the interplay between pyroptosis and antitumor immunity, supporting LAG3 as a potential therapeutic target for cervical cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e52f9bb07a90d5e78056514db55c4237275cced" target='_blank'>
              LAG3 as a Tumor Suppressor and Immune Regulator in Cervical Cancer: From Functional Validation to Therapeutic Strategy
              </a>
            </td>
          <td>
            Shijie Yao, Siming Chen, Shimeng Wan, Anjin Wang, Ziyan Liang, Xuelian Liu, Yang Gao, Hongbing Cai
          </td>
          <td>2025-09-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Breast cancer remains the most prevalent malignancy in women worldwide, characterized by remarkable genetic, molecular, and clinical heterogeneity. Traditional preclinical models have significantly advanced our understanding of tumor biology, yet consistently fall short in recapitulating the complexity of the human tumor microenvironment (TME), immune, and metastatic behavior. In recent years, breast cancer-on-a-chip (BCOC) have emerged as powerful microengineered systems that integrate patient-derived cells, stromal and immune components, and physiological stimuli such as perfusion, hypoxia, and acidic milieu within controlled three-dimensional microenvironments. Aim: To comprehensively review the BCOC development and application, encompassing fabrication materials, biological modeling of key subtypes (DCIS, luminal A, triple-negative), dynamic tumor–stroma–immune crosstalk, and organotropic metastasis to bone, liver, brain, lungs, and lymph nodes. Methods: We selected papers from academic trusted databases (PubMed, Web of Science, Google Scholar) by using Breast Cancer, Microfluidic System, and Breast Cancer on a Chip as the main search terms. Results: We critically discuss and highlight how microfluidic systems replicate essential features of disease progression—such as epithelial-to-mesenchymal transition, vascular invasion, immune evasion, and therapy resistance—with unprecedented physiological relevance. Special attention has been paid to the integration of liquid biopsy technologies within microfluidic platforms for non-invasive, real-time analysis of circulating tumor cells, cell-free nucleic acids, and exosomes. Conclusions: In light of regulatory momentum toward reducing animal use in drug development, BCOC platforms stand at the forefront of a new era in precision oncology. By bridging biological fidelity with engineering innovation, these systems hold immense potential to transform cancer research, therapy screening, and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7430cec40e32120230c8d2b9f9df29796d602af0" target='_blank'>
              Microengineered Breast Cancer Models: Shaping the Future of Personalized Oncology
              </a>
            </td>
          <td>
            T. Popoiu, A.M. Cimpean, F. Bojin, Simona Cerbu, Miruna Gug, C. Pîrvu, S. Pantea, Adrian Neagu
          </td>
          <td>2025-09-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f06d8c3c1ce1c0e2c42d69d2aefb0ff5769089e" target='_blank'>
              STAT3 sustains tumorigenicity following mutant KRAS ablation.
              </a>
            </td>
          <td>
            S. D'Amico, V. Kirillov, Jingxuan Liu, Zhijuan Qiu, Xinyuan Lei, Hong Qin, Brian S. Sheridan, Nancy C. Reich
          </td>
          <td>2025-08-26</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Macrophage polarization plays a pivotal role in shaping the tumor microenvironment and influencing rectal cancer progression. However, the metabolic and prognostic regulators governing this process remain largely undefined. Methods We constructed a macrophage polarization gene signature (MPGS) by integrating weighted gene co-expression network analysis (WGCNA) with multiple machine learning algorithms across two independent cohorts: 363 rectal cancer samples from GSE87211 and 177 samples from The Cancer Genome Atlas (TCGA). The prognostic performance of MPGS was evaluated across rectal and multiple other cancer types. Functional analyses, single-cell RNA sequencing, immunohistochemistry of clinical specimens, and in vitro cellular assays were employed to investigate the role of the MPGS hub gene, PYGM, in tumor biology and immune modulation. Results The MPGS exhibited robust prognostic capability and effectively predicted responses to immunotherapy and various chemotherapeutic agents. Both MPGS and its central metabolic component, PYGM, were closely linked to M2 macrophage infiltration, immunosuppressive tumor microenvironments, and poor clinical outcomes in rectal adenocarcinoma. Single-cell transcriptomic analysis revealed that malignant epithelial cells with elevated PYGM expression are metabolically active and closely interact with M2 macrophages. Clinical tissue analyses and functional assays confirmed that PYGM is upregulated in rectal cancer and promotes tumor cell proliferation, migration, and M2 macrophage polarization. Conclusions This study firstly highlights PYGM as a key metabolic and immunological regulator in rectal cancer, with significant prognostic and therapeutic implications. MPGS and PYGM may serve as novel biomarkers for risk stratification and guide personalized treatment strategies in patients with rectal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8198c80f5a26115add8dc0453b94b75563056daf" target='_blank'>
              Machine learning identifies PYGM as a macrophage polarization–linked metabolic biomarker in rectal cancer prognosis
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumoral microbiota, a critical component of the tumor microenvironment, notably impacts tumor progression through various complex mechanisms such as metabolic regulation, immune system remodeling and genotoxicity. The present review focuses on eight prevalent solid tumors (breast, colorectal, lung, pancreatic, gastric, ovarian, prostate cancers and melanoma), detailing the intratumoral microbial compositional heterogeneity within these malignancies. The present review analyzes the heterogeneous carcinogenic mechanisms mediated by these microorganisms, including gene instability, immune microenvironment remodeling and metabolic intervention. The diagnostic value of microbial markers in liquid biopsy and in situ tissue detection is explored, and the potential for synergistic strategies combining microbial-targeted therapy and immunotherapy is discussed. Finally, the review suggests future research directions, such as spatiotemporal dynamic analysis and organoid-microorganism co-culture, offering new insights for precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bce21033da16d36e409d313fbe6c3d34814bec" target='_blank'>
              Tumor-associated microbiota: Multi-cancer landscape, mechanistic insights and clinical translation (Review)
              </a>
            </td>
          <td>
            Jindi Yang, Yige Dong, Yi Chen, Hong Liang, Shengyu Rong, Zhe Liu, Qiulei Lang
          </td>
          <td>2025-09-16</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Clear cell renal cell carcinoma (ccRCC) is characterized by high recurrence and metastasis rates, leading to poor prognosis. Migrasomes, a class of organelles mediating intercellular communication, and long noncoding RNAs (lncRNAs) both play critical roles in tumor progression; however, the prognostic significance of migrasome-associated lncRNAs in ccRCC remains unclear. Methods Migrasome-associated lncRNAs were identified using The Cancer Genome Atlas (TCGA) dataset, and a prognostic risk signature was constructed. The associations between the model and overall survival (OS), functional enrichment, tumor mutation burden (TMB), tumor microenvironment (TME) characteristics, immune evasion, and drug sensitivity were evaluated. Single-cell transcriptomic analysis was performed to determine cell type–specific expression patterns and intercellular communication networks. Functional roles of key lncRNAs were validated in vitro using qRT-PCR, CCK-8 proliferation assays, wound-healing assays, Transwell assays, colony formation assays, immunofluorescence, and Western blotting. Results The risk signature stratified patients into high- and low-risk groups with significantly different survival outcomes. High-risk patients exhibited elevated TMB and enhanced immune evasion potential. Drug sensitivity analysis revealed distinct therapeutic response profiles between the groups. Single-cell transcriptomic analysis uncovered pronounced cellular heterogeneity and TME characteristics associated with the prognostic signature. High-risk cells were predominantly enriched within tumor epithelial clusters and displayed distinct intercellular communication patterns. Knockdown of FOXD2-AS1 markedly suppressed tumor cell proliferation and migration and reduced the expression of migrasome marker proteins. Discussion This study presents a novel migrasome-associated lncRNA risk signature with significant prognostic and therapeutic implications for ccRCC. The signature captures distinct immune, genomic, and pharmacologic features, and its core lncRNAs may promote tumor progression through migrasome-mediated signaling pathways, warranting further mechanistic investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b92d7fd288f8d87a49f47f7d9317755153dc38e" target='_blank'>
              Integrative bulk and single-cell transcriptomic analysis identifies a migrasome-associated lncRNA signature predictive of prognosis and immune landscape in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Junlin Shen, Chun Wang, Mingpeng Zhang, Bin Chen, Liwei Liu, Jing Tian, Z. Shang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent and malignant tumor of the digestive system, characterized by high incidence and mortality rates. This study aimed to investigate the heterogeneity of the tumor microenvironment (TME) and the involvement of immune cells in CRC. Single-cell RNA sequencing (scRNA-seq) data obtained from the Gene Expression Omnibus (GEO) database were used to analyze and identify six major cell types across normal, core, and border tumor samples. A total of 27414 cells from various regions of patients with CRC were selected for subsequent analyses. Cellular interaction analysis revealed that differential signaling pathways between the TME and normal tissues, with several pathways involving interactions between myeloid cells and epithelial cells. Myeloid cells were extracted and classified into six subtypes based on markers identified in the literature. Monocle3 revealed the trajectory of tumor-associated macrophages (TAMs) and identified genes associated with pseudotime. Single-Cell ENrichment analysis for Interpreting Cellular Heterogeneity (SCENIC) analysis identified specific regulons and target genes associated with TAMs. This study reanalyzed single-cell RNA-sequencing data and provided insights into the heterogeneity of the TME, particularly in relation to the role of TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e37ff9c0af32829265ad92932c051ab5305d1fd" target='_blank'>
              Single-cell RNA sequencing analysis reveals the heterogeneity and effect of TAMs in colorectal cancer.
              </a>
            </td>
          <td>
            Chengang Wang, Ying Qian, Mingsheng Zhang
          </td>
          <td>2025-08-20</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Background Glutamine metabolic reprogramming is a hallmark of tumor progression and is highly correlated with poor clinical outcomes. The excessive uptake of glutamine by tumor cells is a key factor contributing to widespread invasion, metastasis, and immune suppression. SLC38A2, an amino acid transporter widely expressed on the surface of tumor cells, has not been thoroughly studied regarding its function and prognostic significance in tumor progression. Our objective is to employ bioinformatics methods to conduct a comprehensive and in-depth analysis of SLC38A2 across various cancers, aiming to elucidate its role and prognostic value in tumor biology. Methods By comprehensively incorporating gene expression and clinical data from the TCGA tumor database, GTEx database, Human Protein Atlas, and GEO database, we analyzed the expression profile, mutations, and established prognostic models for SLC38A2 across various cancers. Additionally, we investigated the enrichment of SLC38A2 at the single-cell level in 12 types of cancer and analyzed its temporal expression patterns in different cell subgroups in breast and pancreatic cancer. We also studied the correlation between SLC38A2 and glutathione metabolism. Results Compared to normal tissues, SLC38A2 exhibits significant differential expression in 15 types of cancer and serves as a prognostic risk factor in BRCA (HR = 1.597, p < 0.05), LUAD (HR = 1.650, p < 0.01), MESO (HR = 2.007, p < 0.05), and PAAD (HR = 1.761, p < 0.05), while acting as a protective factor in KIRC (HR = 0.625, p < 0.05). Furthermore, SLC38A2 is positively correlated with tumor and stromal cells, negatively correlated with immune cell infiltration, and associated with immune exhaustion. In BRCA, SLC38A2 is highly expressed during early differentiation of malignant and stromal cells, and enriched in late differentiation of immune cells. Moreover, the expression of SLC38A2 shows a general positive correlation with glutathione metabolism genes in BRCA, LUAD, MESO, and PAAD, demonstrating diagnostic value. Conclusion SLC38A2 shows widespread changes in expression patterns within tumor tissues, making it an effective diagnostic and prognostic biomarker. It is enriched in malignant cells and tumor-infiltrating stromal cells, while negatively correlated with the infiltration of many cells involved in anti-tumor immunity. Targeting SLC38A2 presents a viable therapeutic strategy by inhibiting glutamine competition and relieving immune suppression in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cdb69c6df340eda3f029c78020f687d6995d39" target='_blank'>
              Panoramic analysis of the biological function and clinical value of SLC38A2 in human cancers: a study based on pan-cancer and single-cell analysis
              </a>
            </td>
          <td>
            Mao Liao, Yuqing Rao, Molan Li, Jiayang Guo, Kun Guo, Kaiyue Li, Rui Zheng, Yifan Liu, Qianyi Wang, Manni Wang, Duo Chen, Meng Zhang, Yongfeng Wang, Yanzong Zhao, Sheng Li
          </td>
          <td>2025-09-17</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b63ca88458c114713519c6871fd49fc0e1a4b977" target='_blank'>
              Mechanical confinement governs phenotypic plasticity in melanoma.
              </a>
            </td>
          <td>
            Miranda V. Hunter, Eshita Joshi, Sydney Bowker, Emily D. Montal, Yilun Ma, Young Hun Kim, Zhifan Yang, Laura Tuffery, Zhuoning Li, Eric Rosiek, A. Browning, Reuben Moncada, Itai Yanai, Helen M Byrne, Mara Monetti, Elisa de Stanchina, Pierre-Jacques Hamard, R.P. Koche, Richard M. White
          </td>
          <td>2025-08-27</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Pancreatic ductal adenocarcinoma (PDAC), one of the most aggressive forms of pancreatic cancer, is associated with poor survival outcomes and currently ranks as the third leading cause of cancer-related death in the United States. Despite its clinical significance, the mechanisms of PDAC development and progression remain, in part, poorly understood. In this study, we provide evidence of a novel role of sorting nexin 10 (SNX10), a member of the sorting nexin family, in the regulation of KRAS-induced pancreatic carcinogenesis. We demonstrate that SNX10 is downregulated in PDAC, especially in advanced cases. Furthermore, mutational analysis revealed SNX10 genetic alterations in PDAC cases. Functional studies demonstrated that SNX10 overexpression in human PDAC cells inhibited cell proliferation and colony formation. Moreover, SNX10 overexpression induced G1-phase cell-cycle arrest and decreased KRAS signaling activity. Using a novel Snx10 knockout mouse crossed with a Kras-driven PDAC model, we observed reduced survival, increased tumor cell proliferation, enhanced aggression, and heightened inflammation. Collectively, these findings highlight SNX10 as a tumor suppressor candidate in PDAC and underscore its promise as a foundation for new therapeutic approaches. Significance: SNX10 plays a crucial role in reducing pancreatic tumorigenesis. This discovery offers valuable insights into PDAC’s biology and the development of new effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6051a1113648fd5e6a89b58480ecc5f2b85ee853" target='_blank'>
              Loss of Sorting Nexin 10 Accelerates KRAS-Induced Pancreatic Tumorigenesis
              </a>
            </td>
          <td>
            Kohinoor Khan, Mohammad Shameem, Ashley N. Sigafoos, Linda Kiey, Catherine E. Hagen, Gopal Ramakrishnan, Leslie Morse, Martin E Fernandez-Zapico, Ricardo A. Battaglino
          </td>
          <td>2025-08-14</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background Metastatic melanoma is a highly aggressive disease with poor survival rates despite recent therapeutic advancements with immunotherapy. The proteomic landscape of advanced melanoma remains poorly understood, especially regarding proteomic heterogeneity across metastases within patients. Methods We collected 83 melanoma metastases from 19 different metastatic sites in 24 patients with advanced metastatic melanoma almost exclusively from the pre‐immunotherapy era, using semi‐rapid autopsies. The metastases were subjected to histopathological evaluation, RNA‐sequencing and mass spectrometry‐based proteomics for protein quantitation and non‐reference peptide (NRP) sequence detection using a proteogenomic data integration approach. Results NRPs associated with mutations frequently occurred in proteins related to focal adhesion, vesicle‐mediated transport, MAPK signalling and immune response pathways across the cohort. Intrapatient heterogeneity was negligible when considering morphology and driver gene mutation status but was substantial at the proteogenomic level. This heterogeneity was not driven by metastasis location, albeit liver metastases exhibited distinct proteogenomic patterns, including upregulation of metabolic pathways. Cluster analysis outlined four proteomic clusters (C1–4) of the metastases, characterised by the upregulation of cell cycle and RNA‐splicing (C1), mitochondrial processes (C3), extracellular matrix (ECM) and immune pathways (C2) and ECM and vesicle‐mediated transport pathways (C4). Around two‐thirds of patients had metastases that had strongly distinct phenotypes. Patients in our cohort whose metastases were primarily assigned to clusters C1 and C3 exhibited shorter overall survival than patients whose metastases were categorised mainly into the C2 and C4 clusters. Conclusion Our unique multi‐metastasis cohort captured the proteogenomic heterogeneity of immunotherapy‐naïve melanoma distant metastases, establishing a foundation for future studies aimed at identifying novel therapeutic targets to complement current immunotherapies. Key points Comprehensive proteogenomic profiling of post‐mortem melanoma metastases, collected primarily before the immunotherapy era. Description of 1177 protein sequence variants predicted by RNA‐Seq and validated via mass spectrometry‐based proteomics. Empirical evidence of prominent intrapatient heterogeneity, driven by heterogeneous protein expression related to cell cycle‐ and mitochondrial processes, immune system and extracellular matrix organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c47896dcec6e305a1baa3208d22fb00cf053050b" target='_blank'>
              Distant metastases of melanoma exhibit varying extent of intrapatient proteogenomic heterogeneity
              </a>
            </td>
          <td>
            B. Szeitz, Y. Hagemeijer, Zoltán G Páhi, Zsuzsanna Ujfaludi, M. Kuras, Jimmy Rodriguez, V. Doma, R. Mohácsi, Magdolna Herold, Zoltán Herold, Z. Horváth, I. Pla, Y. Sugihara, Bo Baldetorp, H. Lindberg, Henriett Oskolás, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Jéssica Guedes, Johan Malm, Aniel Sanchez, Imre M Boros, I. Németh, Victor Guryev, Tibor Pankotai, Krzysztof Pawłowski, E. Wieslander, A. M. Szasz, David Fenyö, Peter Horvatovich, József Timár, Gyorgy Marko-Varga, L. Betancourt
          </td>
          <td>2025-09-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c548af869e91466a22ceb7887ea1b6fdb37dc0" target='_blank'>
              Integrative proteogenomics and forward genetics reveals a novel mitotic vulnerability in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Nicholas J Neill, S. Satpathy, Karsten Krug, Jitendra K Meena, Nivetha Ramesh Babu, Cheyenne Calderon, Desmon Reed, Marcus J Weber, L. Dobrolecki, Alaina Lewis, Christina Sallas, M. Anurag, Kimberly R Holloway, Chen Huang, S. Vasaikar, Maria F Cardenas, Beom-Jun Kim, Doug W. Chan, Shayan C. Avanessian, S. Tyagi, Mayra Orellana, Sufeng Mao, Heyuan Li, Fade Gong, S. Kurley, K. Meerbrey, Calla M Olson, Amritha Nair, Tingting Sun, Hsiang-Ching Chung, Elizabeth A. Bowling, Jarey H Wang, Pengju Zhang, Peng Xiao, Duxiao Yang, F. Stossi, Mei-Yin C Polley, Alexander B. Saltzman, F. Mundt, D. Mani, Michael A Gillette, S. Hilsenbeck, George Miles, Carolina Gutierrez, C. K. Osborne, Charles Y. Lin, N. Gray, Jinpeng Sun, David A. Wheeler, C. Perou, A. Malovannaya, Michael T. Lewis, Bing Zhang, Matthew J Ellis, Steven A. Carr, Thomas F Westbrook
          </td>
          <td>2025-08-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="


 Tumors of the choroid plexus (CP) are a rare and heterogenous group of neoplasms that mostly occur in pediatric patients. CP tumors consist of choroid plexus papilloma (CPP, WHO grade 1), atypical choroid plexus papilloma (aCPP, WHO grade 2), and choroid plexus carcinoma (CPC, WHO grade 3). CPPs are mostly treated by surgery alone with excellent prognosis; however, CPC is an aggressive brain cancer associated with poor outcome, whereas survivors often suffer from debilitating long-term treatment effects. Despite chromosome-wide alterations, drivers of most CP tumors remain elusive except recurrent alterations in TP53. Studies of transcriptional and epigenetic changes may bring biological understanding of these malignancies. Previous studies implicated NOTCH signaling in CP tumors; we developed mouse models of CP tumors driven by NOTCH activation and TRP53 loss, respectively. This work examined the role of the transcription factor SOX2 in CP development and tumorigenesis.



 Multi-omics approaches were used to characterize cellular heterogeneity in NOTCH-driven CP tumors. SOX2 expression and functions in the molecular signature of tumor cells were investigated using genetic models, mouse primary CP tumor cell and human CP tumor cell lines.



 Single-cell transcriptomics and epigenetics methods identified diverse cell populations in tumors that resemble normal CP, such as epithelial and glial groups. Pseudo-time trajectory analysis indicated that NOTCH-driven CP tumor arises from bi-potential glial progenitors and retains a progenitor-like signature characterized by an enhanced SOX2 profile, whereas NOTCH inhibition reduces SOX2 expression in tumor cells. SOX2 inactivation attenuated progenitor-like features and blunted tumor growth. Integrative omics studies revealed SOX2 binding to genes expressed in progenitors in the rhombic lip, including LIM homeobox transcription factors LMX1A and LMX1B. Consistently, SOX2 maintains progenitor identity through regulating their expression in CP tumors and during development, whereas LMX1A and LMX1B support SOX2 functions in tumor cell proliferation. Furthermore, spatial transcriptomics revealed aberrant SOX2 and LMX1A expression in human CP tumors.



 SOX2-LMX1 signaling maintains progenitor identity in CP development and tumor formation.



 Haotian Zhao, Mahek Chaudry, Siya Patel. SOX2 promotes progenitor-like program choroid plexus development and tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B038.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed1e41b8ac58aee3ea432e2ed62e6469c983826" target='_blank'>
              Abstract B038: SOX2 promotes progenitor-like program choroid plexus development and tumorigenesis
              </a>
            </td>
          <td>
            Haotian Zhao, Mahek Chaudry, Siya Patel
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="COL4A1, a key component of the basement membrane, has been increasingly implicated in tumor progression, yet its role in colon cancer remains incompletely understood. In this study, we conducted a comprehensive integrative analysis using transcriptomic data from the TCGA-COAD cohort, combined with functional validation in colon cancer cell lines. Gene set enrichment analysis (GSEA) revealed that high COL4A1 expression was associated with oncogenic pathways including epithelial–mesenchymal transition (EMT), KRAS signaling, and inflammatory responses. Immune infiltration analysis indicated that COL4A1 expression negatively correlated with CD8+ T cell infiltration but positively correlated with macrophage subtypes. Immunophenoscore (IPS) analysis further revealed that tumors with high COL4A1 expression exhibited significantly higher IPS values, suggesting altered immunogenicity. Functional assays demonstrated that COL4A1 knockdown reduced cell proliferation, migration, and invasion in vitro. Co-expression analysis of EMT markers showed strong positive correlations between COL4A1 and mesenchymal genes such as VIM, ZEB1, SNAI1, and FN1, supporting its role in EMT-like phenotypes. Collectively, our findings suggest that COL4A1 may serve as a prognostic biomarker and contributor to tumor progression and immune remodeling in colon cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17230-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecb3d0354dede68cfb4b2e1e68f45fa53690288b" target='_blank'>
              COL4A1 as a potential prognostic biomarker with functional roles in colon cancer progression
              </a>
            </td>
          <td>
            Qiusheng Li, Changjing Wang, Guiying Wang
          </td>
          <td>2025-08-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Brain metastases (BM) represent a signiﬁcant unmet clinical need with poor survival and limited drug thera- pies. Recent evidence suggests a subset of BM have a more locally invasive phenotype. We sought to determine whether the tumour leading edge and tumour microenvironment (TME) differed between BM of different phe- notypes using spatial transcriptomics.



 Image guided surgical sampling of the tumour margin in operated BM cases using a previously described method (“Geo-Tagging”). Tumour microarray from FFPE blocks assembled using 2mm dots in triplicate. Spatial transcriptomics analysis using Nanostring GeoMX platform and immune-oncology panel (86 markers). Mart1 (melanoma BM) or CK7 (breast and lung adenocarcinoma BM) was used to delineate the tumour, GFAP for per- itumoral brain (TME) and CD45 for immune compartment. Annotation of tissue with clinical data for overall survival, local recurrence, primary cancer and assessment of BM leading edge using established criteria – dif- fuse or border.



 22 cases were analyzed in triplicate, 5 breast cancer, 8 metastatic melanoma and 9 lung adenocarcinoma. Data was analysed using established bioinformatics pipelines in R. As expected there were signiﬁcant differences in gene expression between all BM of different primary cancers but not in the portion of T cells, macrophages and NK cells. PDL1 was upregulated in the tumour compartment and HLA-E downregulated in the TME com- partment of BM without a clear border. Locally recurrent BM signiﬁcantly overexpressed a number of genes previously established as important in BM, including for adhesion (ICAM1, EPCAM), microglia-tumour interac- tion (STAT3) and local invasion (CCND1, VEGFA).



 BM have different phenotypes that are dependent on both tumour and local microenvironmental factors with the future potential for therapeutic targeting to modify tumour behaviour and reduce local invasion and recur- rence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6992d8a9fcf5e2f41d2333a8c8118644a30147" target='_blank'>
              INVESTIGATING THE TUMOUR MICROENVIRONMENT FOR BRAIN METASTASES USING SPATIAL TRANSCRIPTOMICS AND SURGICALLY GUIDED SAMPLING
              </a>
            </td>
          <td>
            R. Zakaria, Michael Jenkinson, Ian Scott, Frances Greaney Davies, Michael Chvanov, Jamie Soul
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A major challenge in diagnosing and treating cancer is the infiltrative growth of tumors into surrounding tissues. This microscopic spread of the disease is invisible on most diagnostic imaging modalities and can often only be detected histologically in biopsies. The purpose of this paper is to develop a physically based model of tumor spread that captures the histologically observed behavior in terms of seeding small tumor islets in prostate cancer. The model is based on three elementary events: a tumor cell can move, duplicate, or die. The propensity of each event is given by an Ising-like Hamiltonian that captures correlations between neighboring cells. The model parameters were fitted to clinical data obtained from surgical specimens taken from 23 prostate cancer patients. The results demonstrate that this straightforward physical model effectively describes the distribution of the size and the number of tumor islets in prostate cancer. The simulated tumor islets exhibit a regular, approximately spherical shape, correctly mimicking the shapes observed in histology. This is due to the Ising interaction term between neighboring cells acting as a surface tension that gives rise to regularly shaped islets. The model addresses the important clinical need of calculating the probability of tumor involvement in specific sub-volumes of the prostate, which is required for radiation treatment planning and other applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecde5263e9fd31b166516097969ae84ab74f179" target='_blank'>
              Ising energy model for the stochastic prediction of tumor islets
              </a>
            </td>
          <td>
            Lucas Amoudruz, G. Buti, Luciano Rivetti, Ali Ajdari, Gregory C. Sharp, P. Koumoutsakos, Simon Spohn, Anca-Ligia Grosu, T. Bortfeld
          </td>
          <td>2025-08-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4006665d3be664d3d5517dbddb8bef8da7462da" target='_blank'>
              High-Resolution Spatial Transcriptomics Reveals Fibroblast and Neuroimmune Microenvironments in Endometriosis Lesions.
              </a>
            </td>
          <td>
            Caroline M Haney, Elaheh Alizadeh, Meryl Sullivan, Joshua Lee, Jasmina Kuljancic, W. Flynn, Paul Robson, Brian S. White, Danielle E Luciano, Elise T. Courtois
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Polymerase epsilon mutant (POLE-mut) endometrial cancers (EC) are characterized by a near 100% disease-specific survival rate, even when treated by surgery alone. This spectacular survival, combined with the ultramutated genome and high level of neoantigens in these tumors, indicates a substantial degree of immune control in preventing disease spread and recurrence. Although these features are intriguing, the immune infiltration of POLE-mut EC has predominantly been confined to immunohistochemistry studies. Here, we used state of the art single-cell RNA and TCR sequencing to characterize the immune landscape of POLE-mutant ECs. Moreover, we uniquely analyzed patient blood samples taken two to eight years after curative treatment to assess formation of long-term immune memory in circulation. We identified specialized tumor-infiltrating myeloid subsets at different stages of maturation, an array of lymphocytes ranging from immature to cytotoxic and adaptive natural killer (NK) as well as tumor-reactive exhausted and effector T cells, contributing to a highly inflammatory anti-tumor response. Remarkably, our analysis of blood samples taken years after curative treatment uncovered the presence of tumor-reactive T cell clones that matched the primary tumor. This indicates the formation of systemic long-term memory immune responses in POLE-mut EC survivors. Our study highlights the distinctive immunogenicity of POLE-mut EC and identifies key features associated with persistent anti-tumor immunity that may contribute to prolonged, relapse-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0198b259b8d5d685a1ba64c108955b1306a25438" target='_blank'>
              Spatiotemporal immune landscape and long-term immune memory in POLE-mutant endometrial cancer at the single-cell level.
              </a>
            </td>
          <td>
            K. Brummel, M. Requesens, N. van Rooij, H. Workel, F. Eggink, A. Plat, R. Wardenaar, D. Spierings, F. Foijer, David N. Church, Joost Bart, H. Nijman, M. de Bruyn
          </td>
          <td>2025-08-29</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) is projected to become the 2nd most prevalent cancer-related death in the United States. The lethality of PDA is largely due to a lack of effective treatment options. A principal barrier in PDA treatment is the physiology of the tumors, characterized by a densely fibrotic stroma rich with immune cell infiltration. Most of these immune cells are immunosuppressive myeloid cells, including macrophages, polarized by environmental cues that dictate their function. This polarization has typically been thought of as pro-inflammatory or anti-inflammatory. In PDA, macrophages within the tumor, termed tumor-associated macrophages (TAMs), are strongly immunosuppressive, inhibiting both infiltration and activation of cytotoxic T cells. Further, TAMs have been shown to directly inhibit chemotherapy in PDA. Accordingly, targeting or reprogramming TAMs may sensitize pancreatic cancer to multiple treatment strategies. We and others have shown that the polarization of macrophages to a pro and anti-inflammatory state drives the adaptation of specific metabolic programming. Specifically, anti-inflammatory macrophages, including TAMs, preferentially use mitochondrial metabolism for bioenergetics. Accordingly, this provides a potential avenue to employ metabolic intervention to reprogram or eliminate pro-cancer TAM populations. To examine if targeting mitochondrial metabolism can impair the functional anti-inflammatory properties of macrophages we have generated a model to genetically ablate a transcription factor required for ETC assembly in a myeloid-specific manner. Our preliminary data demonstrate that mice harbor harboring defective oxidative metabolism in myeloid cells have an altered response to pancreatic inflammation. Moving forward with these models we expect to use these models to investigate the role of myeloid mitochondrial metabolism on pancreatic tumorigenesis, immune suppression in the pancreatic tumor microenvironment, and PDA response to therapy.



 Mariam Mohagheghi, Austin Silva, Avary Baker, Ethan Smiggs, Dequina Nicholas, Christopher Halbrook. Targeting myeloid metabolism to remodel the pancreatic tumor immune microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A055.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe3ab58158b9e94e799f7bd263f67fdde61db10" target='_blank'>
              Abstract A055: Targeting myeloid metabolism to remodel the pancreatic tumor immune microenvironment
              </a>
            </td>
          <td>
            Mariam Mohagheghi, Austin D. Silva, Avary S. Baker, Ethan Smiggs, D. Nicholas, Christopher J Halbrook
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a696fa9baeadfe94d2a764547f5853c287c52816" target='_blank'>
              Hedgehog signaling drives glial cell plasticity and oncogenic reprogramming in gastroenteropancreatic neuroendocrine neoplasms.
              </a>
            </td>
          <td>
            Suzann Duan, AnneLeigh B Twer, Ateh Zinkeng, Ricky A. Sontz, Juanita L. Merchant
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 PDAC is defined by a dense, immunosuppressive TME composed of ECM proteins, CAFs, and diverse immune cells. ChIP-seq on resected human PDAC samples identified MICAL2 as a super-enhancer-associated gene and its high expression was corelated with poor survival of patients. MICAL2, a flavin monooxygenase, promotes actin depolymerization and SRF transcription. This study investigates how tumor-intrinsic MICAL2 alters the PDAC TME. Orthotopic implantation of KPC-MICAL2 knockdown (M2KD) inducible cells led to reduced tumor growth. Immunofluorescence, flow cytometry, and atomic force microscopy showed M2KD tumors were more cellular, with marked reduction in PDPN+/α-SMA+ CAFs, collagen, and fibronectin, resulting in reduced stiffness. MICAL2 loss also reprogrammed CAF gene expression and increased infiltration of activated CD8+ T cells, skewing from an M2 to M1 macrophage phenotype. scRNA-seq revealed major shifts in immune populations, including increased cycling T and plasma cells. CD8+ T cell depletion reversed tumor suppression in both immune-hot and immune-cold syngeneic models, and adoptive transfer of MICAL2-KD tumor-infiltrating T cells suppressed the growth of control tumors. RNAScope showed reduced IL-1α, IL-6, TGF-β expression and decreased p38 and STAT3 phosphorylation in MICAL2-KD tumors. To test if MICAL2 regulates PDAC TME transcriptionally, we generated KPCM tumors and lines (by crossing PDX1-Cre; LSL-KRAS G12D/+; P53R172H/WT (KPC) with Mf/f mice) and reintroduced WT MICAL2 and MICAL2 lacking nuclear localization signal (DNLS) or FAD enzymatic domain in KPCM line. DNLS and FAD point mutant MICAL2 cells were deficient in tumor growth, had reduced stromal deposition and SRF activity, and increased CD8+ T cells. Anti-PD-1 significantly reduced M2KD tumor size and 50% of M2KD tumor-bearing mice had complete regression with anti-PD-1 and IL-1α antibodies. MICAL2-ASO treatment in human PDAC ex-vivo slice cultures increased T cell activation and M1 polarization. MICAL2 drives immunosuppression in PDAC and could be a potential therapeutic target.



 Bharti Garg, Evangeline Sari. Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, Alexei Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Herve Tiriac, Andrew M. Lowy. Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a79cb674dc0d1eba8719f4642ee7a1f1f6ec638" target='_blank'>
              Abstract A109: Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy
              </a>
            </td>
          <td>
            Bharti Garg, Evangeline S Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, A. Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Hervé Tiriac, A. Lowy
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Objective
Acral melanoma (AM), a unique subtype prevalent in China, develops on the palms, soles, and nail beds. Despite its distinct clinical and pathological features compared to cutaneous melanoma (CM), the molecular basis underlying these differences remains poorly understood. This study aims to perform a comprehensive comparative transcriptomic analysis of AM and CM at the single-cell level to uncover key molecular distinctions.


Methods
We analyzed single-cell RNA sequencing (scRNA-seq) data from 39 AM patients and 18 CM cases. Single-cell transcriptomic profiling was used to compare tumor cell subpopulations and microenvironmental differences. Bioinformatics tools were employed for cell clustering, differential gene expression analysis, cell-cell communication network inferences, and survival analysis.


Results
AM exhibited a significantly higher proportion of MPZ + melanoma cells, a subpopulation with Schwann cell-like properties associated with poor prognosis. These MPZ + melanoma cells established extensive communication networks with AM-specific immune and stromal components, prompting an immunosuppressive microenvironment and enhancing angiogenic potential. Survival analysis further indicated that the presence of MPZ + melanoma cells is closely linked to worse clinical outcomes in AM patients.


Conclusions
This study provides novel insights into the molecular distinctions between AM and CM, highlighting the critical role of MPZ + melanoma cells in AM progression. These findings enhance our understanding of AM pathophysiology and may contribute to the development of more targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0de1a260d9f15e430b88fcae26a055c0cb7307d" target='_blank'>
              A comparative transcriptomic analysis at single-cell resolution reveals acral melanoma features distinct from cutaneous melanoma.
              </a>
            </td>
          <td>
            Hongyue Zhao, Jie Tian, Hang Li, Binbin Lai
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Breast cancer remains a predominant malignancy among females globally, and the tumor microenvironment (TME) exerts a pivotal role in its progression. Despite notable advancements in diagnostic and therapeutic modalities, resistance to conventional therapies persists as a critical hurdle, underscoring the necessity of exploring TME-related prognostic biomarkers. Methods: To elucidate the role of the TME in breast cancer progression and identify potential prognostic biomarkers, we analyzed RNA-seq data from 1081 breast cancer cases and 99 normal controls to assess tumor-infiltrating immune cells (TICs) and stromal components. Differential gene expression analysis identified genes correlated with ImmuneScore and StromalScore. A protein–protein interaction (PPI) network was constructed, followed by univariate Cox regression to pinpoint survival-associated genes. JCHAIN, significantly linked to survival outcomes, was selected for further investigation. Gene Set Enrichment Analysis (GSEA) and TIC correlation analyses were performed to explore its associations with immune pathways. Additionally, immunohistochemistry (IHC) and multiplexed immunofluorescence (mIF) were performed on 61 clinical samples. Results: High ImmuneScore was associated with improved survival. Joining chain of multimeric IgA and IgM (JCHAIN) expression was notably reduced in tumor tissues, with low expression correlating with poorer prognosis. GSEA highlighted immune-related pathways enriched in high JCHAIN expression groups. TIC analysis revealed positive correlations with CD8+ T cells and M1 macrophages. IHC and mIF validations further confirmed decreased JCHAIN protein expression in tumor tissues, and higher JCHAIN expression was associated with increased M1 macrophage density. Conclusions: JCHAIN serves as a promising prognostic biomarker in breast cancer, reflecting immune activity within the TME, providing valuable insights into immune-stromal interactions and the therapeutic potential of JCHAIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee65d0a814b0c7914ab35b196564a54dac3a09e" target='_blank'>
              Comprehensive Analysis of JCHAIN as a Potential Prognostic Factor for Breast Cancer and an Indicator for Tumor Microenvironment
              </a>
            </td>
          <td>
            Yaqin Shi, Li Lin, Xinyu Zhu, Mengyao Wu, Caihua Xu, Wei Li, Kai Chen
          </td>
          <td>2025-09-26</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d81b6af91799905ae613971966f2b549960f979" target='_blank'>
              The tumour microenvironment in pancreatic cancer - new clinical challenges, but more opportunities.
              </a>
            </td>
          <td>
            Heng-Chung Kung, Kevin W Zheng, Jacquelyn W. Zimmerman, Lei Zheng
          </td>
          <td>2025-10-03</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="SUMMARY Spatiotemporal environmental variation results in marked heterogeneity in bacterial infection progression and disease outcome, with vital consequences for treatment success. For the globally important pathogen Mycobacterium tuberculosis (Mtb), while the pronounced intra-host spatial heterogeneity in lesion immune cell composition and phenotype has been well-described, the highly complex Mtb cell envelope has presented a particular challenge for the required equivalent insight into bacterial heterogeneity. Here, we develop hybridization chain reaction-fluorescence in situ hybridization (HCR-FISH)-based methodology for Mtb mRNA visualization in the context of intact lung and lesion architecture. In combination with a Mtb transcriptional/translational activity reporter, we reveal spatiotemporal differences in gene expression relating to Mtb lipid metabolism, response to key environmental signals, and the ESX-1 type VII secretion system. Our results establish a framework for in situ analysis of Mtb mRNA, opening the path to elucidating critical bacterial drivers that underlie the marked heterogeneity in Mtb-host interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c42f93d5cca6138cd9f381352612e75158b1b" target='_blank'>
              Direct visualization of bacterial transcripts in the infected lung illuminates spatiotemporal environmental adaptation of Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Anna-Lisa E. Lawrence, Shumin Tan
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an aggressive brain tumour conferring poor prognosis and one of the highest recurrence rates despite multimodal treatment. The mechanisms underlying recurrence are not fully understood, but the tumour microenvironment is known to play a key role. Astrocytes, the most abundant glial cells in the brain, have been implicated in supporting tumour progression, yet their metabolic interactions with glioblastoma invasive margin cells remain poorly understood. This study aims to investigate metabolic interactions using single-cell spatial metabolomics to identify potential pathways involved in tumour adaptation and progression.



 The study employed atmospheric pressure matrix-assisted laser desorption/ionisation (AP-MALDI) imaging mass spectrometry at 10 μm spatial resolution to investigate metabolic interactions between primary glioblas- toma invasive margin cells and human cortical astrocytes in a co-culture model designed to mimic post-surgical residual disease.



 Metabolomic analysis revealed distinct metabolic changes in glioblastoma invasive margin (GIN) cells and hu- man cortical astrocytes (HA) under co-culture conditions. In GIN cells, pyroglutamic acid and glutamate levels were altered, along with changes in lipid metabolism, including a reduction in oleamide. In astrocytes, co- culture inﬂuenced nucleotide metabolism, with increased levels of deoxyguanosine, alongside shifts in lipid proﬁles. Findings indicate bidirectional metabolic interactions that may contribute to glioblastoma progres- sion.



 By mapping metabolite spatial distributions at the single-cell level, this study provides novel insights into tumour–astrocyte metabolic interactions at the glioblastoma invasive margin. These ﬁndings highlight poten- tial metabolic dependencies that could be therapeutically targeted to disrupt glioblastoma recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48c731e4f1e7e12000fcb428faa049465c8fd4b" target='_blank'>
              SINGLE-CELL METABOLIC PROﬁLING OF TUMOUR ASTROCYTE INTERACTIONS WITHIN A CO-CULTURE MODEL OF THE GLIOBLASTOMA INVASIVE MARGIN
              </a>
            </td>
          <td>
            Une Kontrimaite, Kei F Carver Wong, Phoebe McCrorie, Sandra Martinez Jarquin, Stuart Smith, Dong-Hyun Kim, Ruman Rahman
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with limited therapeutic options and poor prognosis. For translational research, although patient-derived organoids (PDOs) have emerged as powerful platforms for modeling tumor biology and drug response, conventional tissue-derived organoids are often limited by low cellularity, contamination with non-tumor cells, and the invasiveness of tissue sampling. Here, we report the establishment and characterization of fluid-derived organoids (FDOs) from malignant pleural effusion and ascitic fluid of PDAC patients. Compared to tissue-derived PDOs, FDOs were established more rapidly and less invasively, with higher success rates and reduced contamination by non-tumor cells. FDOs retained key histopathological and molecular features of the primary tumor, including KRAS mutations and epithelial marker expression. Holotomography imaging revealed distinct morphological and subcellular phenotypes of fluid-derived cancer organoids, reflecting their divergence from normal organoids as well as temporal changes associated with treatment response and disease progression. In vivo xenograft assays confirmed the tumorigenic potential of FDOs, particularly those derived from advanced disease. Drug screening demonstrated that FDOs harboring KRAS G12D mutations responded to the KRAS inhibitor MRTX1133. Transcriptomic profiling identified a poor prognosis-associated gene signature that conferred gemcitabine resistance by suppressing apoptosis. Collectively, our findings establish FDOs as clinically relevant, minimally invasive, and scalable models that enable personalized therapy and biomarker discovery in PDAC.



 Hee Seung Lee, Yoo Keung Tae, Sang-Min Kim, Jung-hwan Park, Kyu Min Lim, HyunKi Kim, Jung Hyun Jo. Minimally invasive fluid-derived organoids enable modeling of chemotherapy resistance in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B095.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81dfc735c625271116241a3184d0aee8b2524e42" target='_blank'>
              Abstract B095: Minimally invasive fluid-derived organoids enable modeling of chemotherapy resistance in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hee Seung Lee, Yoo-Keung Tae, Sang-Min Kim, , Kyu Min Lim, HyunKi Kim, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lymph nodes (LNs) play a pivotal role in colorectal cancer (CRC) progression and immunity, yet their molecular and functional diversity remains poorly understood. By analyzing 630 LNs and 88 primary tumors from 200 CRC patients across four independent cohorts using bulk and single-cell RNA sequencing, we identify four non-metastatic negative LNs (NLN) subtypes (NLN_C1-C4) exhibiting obviously different immune function and stromal expansion. NLN_C3/C4 are characterized by diminished T and B cell activity and fibroblast-driven fibrosis, with follicular dendritic cell loss contributing to B cell dysfunction. Immune checkpoint inhibitors partially reverse these effects, restoring FDC and B cell activity. LNs subtypes demonstrate heterogeneity across patients and within individuals, with higher NLN_C3/C4 proportions associated with advanced tumor stages, poorer survival, and recurrence. Here, we report LNs subtypes as critical manifestations of LN heterogeneity in CRC, providing a basis for improved clinical stratification and LN-targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7feee3a87f017d97fabbe5f56625b50a9e843d16" target='_blank'>
              Lymph nodes molecular subtypes unravel lymph nodes heterogeneity and clinical implications in colorectal cancer
              </a>
            </td>
          <td>
            Xu Guan, Yueyang Zhang, Ruifang Sun, Guiyu Wang, Xiaoman Bi, Zhen Zhang, L. Zan, Yongsheng Meng, Yun-Xiang Liu, Jinyu Guo, Jian Ma, Jiale Li, Hanqing Hu, Yixun Zhang, Xin Zhang, Maoxi Liu, Bo Jiang, Haiyi Liu, S. Zou, Yanfeng Xi, HAI-MENG Zhou, Wenqi Bai, Deng Wu, Xishan Wang
          </td>
          <td>2025-08-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>