<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-09 10:11:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a0eaac11594dae27086c42ea3c9abe9629eb38" target='_blank'>
              Spatial multi-omics unveils the monoclonal origin, neuroendocrine plasticity, and microenvironment niches in combined small cell lung cancer
              </a>
            </td>
          <td>
            Zhuo Wang, Qing Luo, Jie Wu, Li Lu, Wenjie Ding, Yichun Zhao, Yongfeng Yu, Ruoran Qiu, Ling Zhu, Xinxing Ouyang, Wendi Xuzhang, Shun Lu, Wei Wei, Qihui Shi, Ziming Li
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are the main constituents of the tumor microenvironment. Several studies have delineated CAF heterogeneity in different types of tumors, however, it is still unknown how the distinct CAF transcriptional profiles are established during tumor progression. We reanalyzed a previously published single-cell RNA-sequencing dataset of MMTV-PyMT tumors at higher resolution using Seurat , and CytoTRACE to characterize CAF subtypes and their differentiation states. Wilcoxon rank sum test was applied for differential gene expression. Multiplex immunostaining (Akoya PhenoImager HT) was performed on 38 murine mammary tumors from MMTV-PyMT mice to identify the distinct CAF subtypes. Whole-slide imaging and spatial analysis were conducted using QuPath and Cellpose , followed by neighborhood clustering and interaction mapping with CytoMAP . Cellular distances from CAFs to immune, tumor, and endothelial cells were quantified using SPIAT and Wilcoxon tests for comparisons. In parallel, human spatial transcriptomics data from the 10X Genomics Xenium platform were integrated for cross-species validation. Here, by single cell RNA-sequencing and multiplex immunostaining, we identify six CAF substates. Spatial analysis on immunostained murine mammary tumors and human spatial transcriptomics data outlined temporal changes in stromal composition and the existence of distinct functional niches enriched with different CAF substates. Immunomodulatory CAFs co-localized with immune cells while myofibroblastic CAFs formed a shield around the tumor core, thus preventing immune infiltration. Our work supports the idea that distinct spatial locations dictate different CAF transcriptional programs. Targeting specific functional niches will ultimately hinder tumor progression by inhibiting signaling between distinct CAF substates and the surrounding tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d802e2b1323bfcdbf953412180c241803eb33a" target='_blank'>
              Spatial and temporal dynamics of cancer-associated fibroblast niches in breast cancer
              </a>
            </td>
          <td>
            Jessica Pantaleo, J. Sjölund, P. Bolivar, M. Bocci, B. Phung, Maria Malmberg, G. Jönsson, Kristian Pietras
          </td>
          <td>2026-01-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>196</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor immune microenvironment (TIME) is closely involved in tumor initiation, malignant progression, immune escape, and response to immunotherapy. With the continued development of high-throughput sequencing technologies, transcriptomic approaches have become essential for examining the cellular and molecular features of the TIME. Bulk RNA sequencing offers tissue-level gene expression profiles and allows the estimation of immune cell composition through computational deconvolution. Single-cell RNA sequencing provides finer resolution, revealing cellular heterogeneity, lineage relationships, and functional states. Spatial transcriptomics (ST) retains the native anatomical context, making it possible to localize gene expression patterns and cell–cell interactions within intact tissues. These approaches, when considered together, have shifted TIME research from averaged measurements toward a more detailed and mechanistic understanding. This review summarizes the principles, applications and limitations of bulk, single-cell and spatial transcriptomic methods, highlighting emerging strategies for integrative analysis. Such multi-scale frameworks are increasingly important for studying immune dynamics and may contribute to the development of more precise biotechnological and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95188583167d2d5ed84b42177b48c6d850371a9" target='_blank'>
              Multi-Scale Transcriptomics Redefining the Tumor Immune Microenvironment
              </a>
            </td>
          <td>
            Jing Sun, Yingxue Xiao, Lingling Xie, Dan Qin, Yue Zou, Yingying Liu, Yitong Zhai, Minyi Zhang, Tong Li, Youjin Hao, Bo Li
          </td>
          <td>2026-01-15</td>
          <td>BioTech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: In contrast to most solid tumors, high immune cell infiltration in clear cell renal cell carcinoma (ccRCC) is associated with poor patient prognosis. The biological mechanisms underlying this paradox remain unclear, particularly regarding tumor cell-microenvironment interactions promoting local invasion and recurrence. This study aimed to identify spatially resolved tumor, immune, and stromal features that define aggressive phenotypes in localized ccRCC. Methods: Multiplex immunofluorescence was performed using a 33-marker panel on 1,728 multi-region tissue cores from 435 surgically treated patients with localized ccRCC. Samples systematically included tumor centers, invasive borders, and adjacent benign tissue. Single-cell analyses quantified immune, stromal, endothelial, and epithelial cell populations within their spatial context. Results: Spatially resolved profiling uncovered a highly aggressive tumor subtype distinguished by fibroblast activation protein (FAP) expression on tumor epithelial cells, a marker typically associated with stromal cells. Tumor-cell-specific FAP expression characterized an epithelial-to-mesenchymal transition (EMT)-like state and was spatially associated with profound immunosuppression, marked by enrichment of regulatory T cells, exhausted CD8+ T cells, and M2-like macrophages, particularly at the invasive border. Tumor-cell FAP promoted invasion and independently predicted significantly poorer recurrence-free survival (RFS), even in early-stage disease (multivariable Cox p = 0.022 for pT1-2), surpassing established biomarkers such as PD-L1 in capturing aggressive biological features. Conclusions: Tumor epithelial FAP expression identifies an aggressive, immune-rich subtype of localized ccRCC, integrating EMT with spatially organized immunosuppression. These findings establish tumor-cell FAP as a promising biomarker with substantial translational potential for patient risk stratification, targeted imaging (FAPI-PET), and FAP-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42470b1512d22d2033f3b6dc6829a3e316db8519" target='_blank'>
              Tumor cell FAP orchestrates EMT and immune suppression in aggressive localized ccRCC
              </a>
            </td>
          <td>
            T. Pellinen, L. Luomala, K. Mattila, A. Hemmes, K. Välimäki, Mariliina Arjama, Oscar Brück, L. Paavolainen, Elisa Kankkunen, H. Nísen, P. Järvinen, Leticia Castillon, Sakari Vanharanta, P. Vainio, Olli Kallioniemi, Panu M. Jaakkola, T. Mirtti
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="While single-cell technologies have advanced bladder cancer research, the microenvironment of lymph node metastases and its role in progression remain unclear. We aimed to elucidate dynamic tumor-microenvironment interactions across metastatic and primary sites, identify poor-prognosis features, and uncover novel therapeutic targets. We profiled matched single-cell and spatial transcriptomic samples from lymph node-metastasized bladder cancer to construct a high-resolution atlas. Bioinformatic analyses were used to inferred ligand-receptor interactions, regulatory networks, and spatial co-localization. Integration with multiple prognostic cohorts revealed key cell subpopulations linked to tumor biology and poor outcomes. Our atlas included 32 samples, 61 cell subtypes, key ligand signals (e.g., TGFB1, OSM, IFNG), 343 transcription factors, 188 co-localization events, over 1,000 spatially validated interactions, and 21 cell niches. Tumor-driven convergence was observed across distinct microenvironments. Notably, a pericyte-like myofibroblast subtype, myoCAF_PLXDC1, was enriched in malignant tissues, associated with poor prognosis, and exhibited enhanced angiogenic but reduced chemotactic activity compared to myoCAF_CCL21. Its phenotype was linked to intracellular regulators (PRRX1 and MXD4) and extracellular signals (IFNG and TGFB1). Spatially, myoCAF_PLXDC1 co-localized with ITGA8+ endothelial cells and enriched in tumor regions. This study offers a comprehensive resource for lymph node-metastasized bladder cancer and identifies PLXDC1+ fibroblasts co-localized with ITGA8+ ECs as potential components of a tumor-specific vascular niche and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d7c9cdf4d9cefce93639f3970e5da305a8fc5c" target='_blank'>
              Single-cell and spatial atlas unveil tumor-specific microenvironment convergence and a prognosis-associated PLXDC1+ myofibroblast population in metastatic bladder cancer
              </a>
            </td>
          <td>
            Ziwei Wang, Jiaying Miao, Maoyu Wang, Yidie Ying, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2025-12-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5486f6aaee8639d9484243cb88e9c2a7fbf4ba3" target='_blank'>
              Cancer-myeloid cell invasive program in pediatric-type diffuse high-grade glioma
              </a>
            </td>
          <td>
            C. Ruiz-Moreno, R. Collot, T. J. van den Broek, E. J. Wehrens, N. Bessler, Gitanjali Dharmadhikari, B. T. Te Pas, Ignacio L Ibarra, D. Metselaar, M. Kranendonk, E. Hoving, J. van der Lugt, F. Calkoen, A. Zomer, Fabian Theis, E. Hulleman, D. V. van Vuurden, A. Rios, H. Stunnenberg
          </td>
          <td>2026-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/342dd98c58335efcd7b4ec06f157649f394ee9d4" target='_blank'>
              Clinical feasibility of spatial transcriptomics using discarded tissue from diagnostic breast biopsies
              </a>
            </td>
          <td>
            Jeffrey P Sheridan, Quynh Sun, Jiaji G Chen, Yibing Wei, Veronica Jarzabek, Anirudh Jaishankar, Iqra Amin, L. Sultan, Thea Nalan, Sarah B. Mueller, Eric J. Burks, Lubna H. Suaiti, Christopher M. Heaphy, Naomi Y. Ko, Ruben Dries
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Understanding the spatial organization of individual cell types within tissue and how this organization is disrupted in disease, is a central question in biology and medicine. Hematoxylin and eosin-stained slides are widely available and provide detailed morphological context, while spatial gene expression profiling offers complementary molecular insights, though it remains costly and limited in accessibility. Predicting gene expression directly from histological images is therefore an attractive goal. However, existing approaches typically rely on small image patches, limiting resolution and the ability to capture fine-grained morphological variation. Here, we introduce a deep learning approach that predicts single-cell gene expression from morphology, matching patch-based methods on spot level prediction tasks. The model recovers biologically meaningful expression patterns across two cancer datasets and distinguishes fine cell populations. This approach enables molecular-level interpretation of standard histological slides at scale, offering new opportunities to study tissue organization and cellular diversity in health and disease. Predicting gene expression from H&E slides is a cost-effective alternative to Spatial Transcriptomics for clinical use. Here the authors introduce a Multiple Instance Learning approach to infer single-cell expression, unlike prior methods which operate at coarser patch level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d903aacfbf3a6dfc2c450d8cb95280e66a5321f" target='_blank'>
              sCellST predicts single-cell gene expression from H& E images
              </a>
            </td>
          <td>
            Loïc Chadoutaud, Marvin Lerousseau, Daniel Herrero-Saboya, Julian Ostermaier, Jacqueline Fontugne, Emmanuel Barillot, Thomas Walter
          </td>
          <td>2026-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6a0c7866dff4b99709467cf25215e8947b79ac" target='_blank'>
              Coevolution of neoplastic and non-neoplastic reactive astrocyte states converges on mesenchymal-like and injury-response programs during murine glioblastoma progression and post-radiotherapy recurrence
              </a>
            </td>
          <td>
            D. Lindgren, R. Rosberg, K. Smolag, E. Johansson, D. Manou, J. Sjölund, S. Braun, B. Phung, K. Harbst, P. Jeannot, Maria Malmberg, C. Ceberg, G. Jönsson, Kristian Pietras, A. Pietras
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Understanding how prostate cancers evolve toward treatment resistance requires experimental systems that both recreate the phenotypic diversity of the disease and resolve lineage dynamics at high resolution. We developed complementary platforms to meet these challenges. The Prostate Mouse Platform (ProMPt) is a genetically scalable organoid- and allograft-based system encompassing over 150 combinatorial genotypes, enabling reconstruction of the principal phenotypic states of prostate cancer, from androgen-responsive tumors to castration-resistant double-negative, hybrid/mixed, and neuroendocrine-like variants. Through multi-omic and functional analyses, ProMPt models capture the context-specific vulnerabilities that arise during adaptation to androgen deprivation or growth-factor loss. To map and track this evolving phenoscape, we designed XCODE, a universal, multiplatform barcoding technology that enables clonal tracing across sequencing-, cytometry-, and imaging-based modalities (CyTOF, Phenocycler, Xenium). XCODE integrates lineage information across bulk, single-cell, and spatial layers, linking histology, clonal dynamics, phenotype, and microenvironmental context. Together, ProMPt and XCODE provide a powerful preclinical framework to reconstruct, resolve, and perturb tumor evolution within a controlled yet clinically relevant landscape. This integrated approach reveals how defined genetic combinations reshape lineage plasticity, signaling networks, and therapeutic responses, exposing new intervention points to prevent or redirect adaptive evolution in advanced prostate cancer.



 Marco Bezzi. From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245085c50a80831901ee082ba85bf39b096b337" target='_blank'>
              Abstract A002: From reconstruction to resolution: platform approaches to engineer the prostate cancer phenoscape and map its evolution
              </a>
            </td>
          <td>
            Marco Bezzi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Deng-Feng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            Sara B. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ee6608293723ba24e369164c3802a52bef8a361" target='_blank'>
              Redefining the topology of the human bone marrow using augmented spatial transcriptomic analysis
              </a>
            </td>
          <td>
            Rosalin Cooper, Emily Thomas, Muhammad Dawood, H. Ryou, Anna M. Sozanska, C. Pescia, Oliver McCallion, Muskaan Gupta, Renuka Teague, J. Hester, F. Issa, Dan J Woodcock, B. Psaila, Adam J. Mead, Jens Rittscher, D. Royston
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98b24512687fbcc11862186cf26ac85b52f95e1" target='_blank'>
              Benchmarking cell type deconvolution in spatial transcriptomics and application to cancer immunotherapy
              </a>
            </td>
          <td>
            Amanda Sun, Tamjeed Azad, Chrysothemis C. Brown, Yuri Pritykin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67c0bdeec747ad1c41e7a933728f867f5f704e68" target='_blank'>
              Spatial pharmaco-multiomics reveals drug distribution, metabolic niches, and spatially constrained resistance in medulloblastoma
              </a>
            </td>
          <td>
            Tuan Vo, Cedric Cui, Aitor Benedicto, Yibi Chen, Andrew Causer, Xiao Tan, Eun Ju Kim, Hani Vu, Albert Xiong, Mark Hodson, Yak Deng, Ana Maria Romero Jimenez, Ning Liu, Brett Hamilton, Thomas Robertson, Laura A. Genovesi, Brandon J. Wainwright, Jenny Fung, John Lee, Quan Nguyen
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c46aa519f5755b995d1d19efbf9695d85981f7" target='_blank'>
              Molecular Profiling and High-Content Drug Screening of Metastatic Colorectal Cancer Organoids Reveal Evolutionary Mechanisms of Pan-KRAS Inhibitor Resistance
              </a>
            </td>
          <td>
            Gizem Damla Yalcin, Kübra Yılmaz, Raffaella Klima, O. Kurtulan, Duriye Ozer Turkay, O. Aktepe, A. B. Doğrul, Cenk Sokmensuer, V. Oter, E. B. Bostancı, Javier Fernandez-Mateos, Suayib Yalcin, Luca Braga, Ahmet Acar
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153f97057ff8573dac73f49cf5c6ecd0096ebd36" target='_blank'>
              Single-cell profiling of synchronous multi-organ metastasis reveals a systemic CD74+ lipid-associated macrophage niche driving polymetastatic breast cancer
              </a>
            </td>
          <td>
            Jacob Insua-Rodríguez, Pascal Naef, Hannah Savage, Isam Adam, Sung Kook Chun, Axel A. Almet, Gautham Prabhakar, Sharon Kwon, Cassandra G. Sweet, Sevilla F. Hennessey, Angela Lincy Prem Antony Samy, Hamad Alshetaiwi, Maren Pein, Sharmila Mallya, Delia F. Tifrea, Declan M. Edwards, Angela Gomez-Arboledas, Zander Esh, Carina Sandoval, Katrina T. Evans, Alex Hsu, Victoria Mai, Ngoc Bao Kim Nguyen, Erika Zagni, Aaron Longworth, Paige V. Halas, Aaron Simon, Cholsoon Jang, A. Mohyeldin, Robert A. Edwards, Qing Nie, Selma Masri, Kai Kessenbrock, Devon A. Lawson
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Neoadjuvant PD-1/PD-L1 blockade yields robust efficacy in advanced cutaneous squamous cell carcinoma (cSCC), yet many patients fail to achieve a complete or major pathologic response. The reasons why some patients experience response but others do not are unclear. Methods We profiled cSCC specimens before, after 1 dose, and after 3–4 doses of PD-1/PD-L1 blockade to uncover resistance mechanisms and predict therapeutic response. In total, 27 patients across three cohorts, including two phase II trials, were studied. We created 1.7 mm tissue-core microarrays and performed single-cell spatial transcriptomics, including spatial clustering, gene-set enrichment, and spatial correlation analyses. Results After profiling all samples, six distinct spatial niches emerged, each differentially enriched in responders versus non-responders. A high antigen presentation niche, B/plasma cell enriched niche, and inflammatory keratinocyte niche were more frequent in responders, whereas proliferative keratinocyte, low antigen presentation myeloid, and fibroblast-rich epithelial–mesenchymal transition niches prevailed in non-responders. Notably, spatial niche profiling on pretreatment samples outperformed PD-L1 status in predicting pathologic response. Each niche displayed unique gene coexpression modules, suggesting niche-specific resistance mechanisms. Individual tumor analyses revealed varied immune evasion strategies, including defective interferon-induced antigen presentation, immunosuppressive myeloid environments, and epithelial–mesenchymal transition. Conclusions Our single-cell spatial transcriptomic approach identifies six spatial niches that predict immunotherapy response better than PD-L1 status using only 1.7 mm tissue cores and may inform the development of biomarkers. Our results further underscore the heterogeneity of resistance mechanisms among cSCC patients, highlighting the need for tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf68ddba765459abfda326b7a403a032b5df3da" target='_blank'>
              Single-cell spatial transcriptomics uncovers niches that govern response to PD-1/PD-L1 blockade in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Maxwell Y Lee, Alistaire R. Sherman, Luis Martinez Ramirez, Mobeen Rahman, Sabrina Zdravkovic, June Ho Shin, Ivan Stepanek, A. D. Colevas, John B. Sunwoo, V. Divi
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer remains a significant global health challenge, with metastatic progression being the leading driver of mortality. Organoid technology provides a tractable, physiologically relevant platform to model key aspects of lung cancer metastasis in vitro. The present review summarized methodologies for constructing and interrogating these models, covering tissue sources, culture modalities, gene editing and in vivo transplantation; applications in studying metastatic mechanisms, drug screening and capturing intra- and intertumoral heterogeneity are also highlighted. Persistent challenges include standardizing derivation and culture conditions, improving preservation of tumor-microenvironmental interactions, expanding immune-competent and vascularized models, and addressing scalability, cost, and regulatory and ethical considerations for clinical translation. Future directions include integrating multi-omics approaches and spatial profiling, leveraging artificial intelligence for image and response analytics, advancing immune-organoid models and establishing shared standards, reference materials and reporting guidelines to enhance reproducibility and clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831c4c96b7e8c9d326d8d56192940951debd787" target='_blank'>
              Recapitulating lung cancer metastasis in vitro: Advances in organoid models and challenges in clinical translation (Review)
              </a>
            </td>
          <td>
            Jian Jiang, Guomin Dong, Zhiyun Zhang, Xiaoyuan Lei, Fanshu Huang, Suoni Li, Jiequn Ma, Jie Bai, Qian Ge, Hui Wei, Zheng Zhao
          </td>
          <td>2026-01-20</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains clinically heterogeneous. We integrated single-cell and spatial transcriptomics with explainable machine learning to define a lethal tumor axis and establish an interpretable prognostic model. From 141,986 high-quality single cells spanning localized, hormone-sensitive, and castration-resistant PCa, we identified a malignant C4 epithelial subpopulation characterized by high chromosomal instability, androgen receptor and cell-cycle activation, and stemness potential. Spatial mapping further revealed immune-enriched yet suppressive niches, where fibroblasts and myeloid cells coexisted with exhausted lymphocytes, reflecting functional immune imbalance. We benchmarked 101 machine learning pipelines, selecting a Lasso plus PLS-Cox model that achieved strong concordance across independent cohorts. The C4-based risk score independently predicted recurrence-free survival after adjustment for age, Gleason score and T stage, and a nomogram combining this score with clinical variables showed good discrimination. SHAP interpretation highlighted MT1M, PCSK1N, and ACSL3 as major risk-driving features. PCSK1N was progressively upregulated from normal prostate to castration-resistant disease and promoted proliferation, clonogenicity, migration and enzalutamide resistance, while its inhibition sensitized organoids and xenografts to AR-targeted therapy. These findings define a C4-centered lethal tumor axis and provide an explainable, experimentally supported framework for prognostic stratification in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhong Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The limited availability of tissue samples from rare tumors poses a major barrier to advances in precise diagnosis, prognostic evaluation, and therapeutic research—challenges exemplified by primary central nervous system diffuse large B cell lymphoma (PCNS-DLBCL). Here, we developed a dual-mode microfluidic immunostaining (Dumi) device, which integrates diagnostic and research workflows into a single, automated platform, reducing tissue section consumption by more than 90%. Using just one to two slides, it enables both regional detection of biomarkers for diagnostic subtyping (≤16) and construction of multiplex tumor microenvironment (TME) maps. Joint analysis of multiregion diagnostic biomarkers in the TME map indicates that tumor cell subpopulations defined by specific diagnostic biomarkers can actively shape their in situ microenvironmental niche. Dumi offers an efficient, cost-effective, and multifunctional immunostaining method, overcoming the limitations of scarce tissue resources and providing a clinically accessible solution to the diagnostic and therapeutic challenges with rare tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/242fa7959d0a90b74624ebf05f15aaa2c75a7e96" target='_blank'>
              Dual-mode microfluidic immunostaining device for diagnostic biomarkers detection and tumor microenvironment evaluation
              </a>
            </td>
          <td>
            Yu Zhang, Yanhua Huang, Jing Jia, Xiaoguang Guo, Fuxiu Liu, Bing Shi, Zhenbao Huang
          </td>
          <td>2026-01-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19bfda47de17fb8ef4014da55fa6d5d86101b2f" target='_blank'>
              Addressing complex autofluorescence signatures in solid tissue samples to enhance full spectrum flow cytometry of non-immune cells
              </a>
            </td>
          <td>
            Christina Gkantsinikoudi, Manuela Terranova-Barberio, Neil P Dufton
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf55c0c7d18000c8ba092357d0dd95271e7c35c" target='_blank'>
              Spatial dissection of ADC/RPT targets defines therapeutic opportunities in rhabdoid tumors
              </a>
            </td>
          <td>
            N. Reitsam, V. Fincke, Maria Daniela Hernandez Ramirez, M. Mucha, Eva Sipos, Lisa Siebenhüter, J. Enke, Maurice Loßner, C. Vokuhl, C. Lapa, M. Hasselblatt, Michael C. Frühwald, B. Märkl, Pascal D. Johann
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Rationale: Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy. The metastasis and poor prognosis of ESCC are closely associated with tumor microenvironment (TME) heterogeneity, which is driven by epithelial-mesenchymal transition (EMT). Clinically, how to diagnose and target EMT progression remains a key challenge for ESCC. Methods: Integration of pathological images and bulk RNA sequencing profiles identified a high-risk subtype exhibiting EMT enrichment and immunosuppression. Single-cell and spatial transcriptomics revealed EMT macrostates and their spatial distribution. The role of CACNA1C in programming malignant phenotype was tested in vitro. A pathological image-based deep learning model successfully predicted the spatial expression distribution of CACNA1C, indicating possible clinical utility. Results: EMT progression comprised three macrostates: the early state (high epithelial and metastatic potential), the stable state (hybrid E/M phenotype and high stemness), and the late state (high mesenchymal and invasive propensity). ITGA3 and ITGB4 antagonistically regulate malignant phenotype in the early state. Notably, suppression of CACNA1C induced transdifferentiation from stable/late-state cells to normal epithelium-like cells. Conclusions: This study provides novel insights into the EMT mechanism in ESCC, proposes an intervention strategy, and emphasizes the promising clinical application of pathological images in EMT assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2bd67040364633d082b64fbfe1d8f04e033d05" target='_blank'>
              Pathogenomic analysis reveals clinically relevant epithelial-mesenchymal plasticity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Ruzhen Chen, Chenyi Xie, Ziyu Ning, Meng Yang, Zezhuo Su, Jiahui Chen, Kunheng Du, Yihuai Hu, Chu Han, Shaojun Zhang, Qingling Zhang, Meng Liu, Zaiyi Liu
          </td>
          <td>2026-01-14</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated stromal cells (TASCs) are key architects of the tumor microenvironment (TME), playing a vital role in tumor development, metastasis, and therapeutic response. Their spatiotemporal heterogeneity, characterized by dynamic phenotypic plasticity, diverse cellular subtypes, and distinct spatial distributions, offers profound insights into tumor behavior and paves the way for innovative therapy development. In particular, stromal–immune interactions reveal the powerful capacity of TASCs to shape the immune landscape, highlighting their potential as targets in immunotherapy. Despite growing evidence in functional diversity, precise mechanisms underlying the temporal evolution and spatial organization of TASCs remain elusive, impeding clinical translation. This review delved into the molecular signatures and functional states of TASCs, emphasizing their roles in tumor dynamics and therapeutic resistance. We also discussed innovative strategies targeting the plasticity of TASCs to reverse immune evasion and potentiate immune-mediated tumor eradication. Future studies should prioritize identifying spatially resolved and mechanically defined biomarkers with multi-omics and machine learning approaches, enabling a comprehensive understanding of TASCs to bridge the gap from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc16bf824dc473946d5d3958736301da12edf085" target='_blank'>
              The Spatiotemporal Heterogeneity of Tumor-Associated Stromal Cells: Reprogramming Plasticity to Unlock Precision Cancer Immunotherapy
              </a>
            </td>
          <td>
            Yingying Lv, Tingfei Duan, Jinling Song, Shutong Liu, Zhaokai Zhou, Yuhao Ba, Siyuan Weng, Anning Zuo, Hui-Xian Xu, Peng Luo, Quan Cheng, Chuhan Zhang, Jingyuan Ning, Yukang Chen, Yuyuan Zhang, Zaoqu Liu, Xinwei Han
          </td>
          <td>2025-12-12</td>
          <td>Cancer Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer and displays distinct clinical and biological behavior compared to breast cancer of no special type. However, current molecular classifications largely overlook its complex spatial organization and tumor microenvironment (TME). Here, we performed spatial transcriptomics on 43 hormone receptor-positive, HER2-negative (HR+/HER2-) ILC tumors with detailed morphological annotation and long-term clinical follow-up. By integrating spatial gene expression with histology and single-cell deconvolution, we characterized the composition and architecture of the TME and revealed high inter- and intratumor heterogeneity. Spatial clustering uncovered cell populations and pathways linked to clinical outcome. We then developed a multimodal classification of ILC by integrating gene expression, morphology, and spatial metrics, identifying four distinct subtypes: normal/stroma-enriched (NSE), proliferative (P), androgen receptor-enriched (ARE), and metabolic/immune-enriched (MIE). These subtypes, collectively termed ILC4TME, reflect the interplay between tumor and microenvironmental features. Gene signatures derived from the spatial data enabled subtype assignment in external bulk RNA-seq and microarray datasets (SCAN-B, METABRIC), revealing reproducible biology and significant associations with survival. In multivariable models, ILC4TME retained prognostic value beyond established gene signatures and clinicopathological variables. Notably, the P subtype was linked to poor prognosis, even in patients treated with endocrine therapy alone, while the NSE subtype was associated with favorable outcomes. Our findings uncover spatial and cellular heterogeneity in ILC that is not captured by existing classification approaches, offering a refined framework for risk stratification and therapeutic targeting based on tumor microenvironment architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/048f8386277c3de4abe3f625fd3318ada9b08489" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment-driven subtypes of invasive lobular carcinoma.
              </a>
            </td>
          <td>
            M. Serra, Mattia Rediti, L. Collet, F. Lifrange, D. Venet, N. Occelli, A. Papagiannis, D. Vincent, G. Rouas, D. Larsimont, M. Vikkula, F. Duhoux, F. Rothé, C. Sotiriou
          </td>
          <td>2026-02-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3083560b7513f7f17aefe85af32208b0613ecab1" target='_blank'>
              Cellular neighborhoods in cancer
              </a>
            </td>
          <td>
            Lichun Ma, Barbara Xiong, Meng Liu, Kai Tan
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The omentum, a visceral adipose tissue with critical metabolic, immunological, and stem cell functions is the preferred site for ovarian cancer metastasis. However, its role in maintaining homeostasis and its responses to metastatic colonization remain incompletely understood. Using single-cell transcriptomics, we profile different anatomical regions of the omentum in patients with benign conditions and metastasis. We catalog the benign omentum and observe a stable cell type composition and a preserved stem and progenitor niche. Upon metastatic colonization, we report on increased immune heterogeneity and a concomitant reduction of mesothelial and progenitor cells. The lesser omentum, which is not routinely removed during surgical debulking, is identified as a premetastatic niche characterized by neutrophil infiltration, extracellular trap formation, and the presence of micrometastases. At established metastatic sites, resident cells exhibit cancer-associated phenotypes with regulatory, anti-adipogenic, and immunosuppressive properties. Cellular reprogramming across the omentum is associated with signaling profiles of tumor cells, suggesting potential influences on both proximal and distal tissue regions. This cell atlas illuminates the cellular and molecular determinants of organ homeostasis and reveals a high degree of plasticity and cellular reprogramming promoted by cancer colonization. The omentum is the main metastatic site for ovarian cancer. Here, the authors present a human cell atlas of both benign and omentum metastatic ovarian cancer, providing insights into established metastatic sites but also of the omentum pre-metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d421861e9c6ef5f5ca4fb4ee26c92b15a98c09c9" target='_blank'>
              Ovarian cancer metastasis to the human omentum disrupts organ homeostasis and induces fundamental tissue reprogramming
              </a>
            </td>
          <td>
            Ulrike Lischetti, Ching-Yeu Liang, Matteo Carrara, R. Coelho, Antoine Hanns, Flavio C. Lombardo, Arthur Dondi, Sandra Goetze, M. Hensler, C. Kurzeder, Céline Montavon, J. Fucikova, Franziska Singer, N. Beerenwinkel, C. Beisel, V. Heinzelmann-Schwarz, Francis Jacob
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="Gastroenteropancreatic neuroendocrine tumors (GEP-NETs) are clinically heterogeneous malignancies whose biology and microenvironmental organization remain poorly understood. Here, we integrated single-nucleus multiomic (snRNA-seq and snATAC-seq) and spatial transcriptomic profiling across 38 well-differentiated pancreatic (pNET) and small-intestinal (siNET) tumors to define conserved malignant programs, their regulatory circuits, and spatial niches. We observed two conserved malignant cell programs spanning a continuous transcriptional spectrum: a neuronal-like program (si-cNMF1/p-cNMF1), and a secretory neuroendocrine program (si-cNMF2/p-cNMF2). Matched chromatin accessibility profiles uncovered distinct, tissue-specific regulatory networks, including MAX::MYC and MITF transcription factor binding motifs in siNETs versus ISL1 and TFAP4 in pNETs, indicating organ-specific epigenetic control. Spatial transcriptomic analyses revealed that si/p-cNMF1-high regions localized to high cell density, immune-rich tumor areas, whereas si/p-cNMF2-high regions occupied stromal and vascularized niches and co-occured with fibroblast and endothelial compartments enriched for TGFB1-ITGB1, VEGFA-FLT1, and LAMA2-ITGA1 signaling. Across both tumor types, the cNMF2 program was enriched in metastatic lesions and was enrichedfor pro-fibrotic and pro-angiogenic gene signatures. Thus, GEP-NETs are organized along a conserved neuronal-to-secretory axis defined by distinct epigenetic programs and spatially coupled to specific microenvironmental niches. This framework unifies NET heterogeneity across organ sites and identifies pathway-specific, microenvironment-linked vulnerabilities for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5c183623da01cf434288165ef5499fec974dff" target='_blank'>
              Conserved Neuronal-like and Secretory Programs Define the Spatial Architecture of Gastroenteropancreatic Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Julie Karam, Samantha E. Hoffman, Amanda E. Garza, Dan Y Gui, Hannah I. Hoffman, Breanna M. Titchen, Yutaro Tanaka, Erica M Pimenta, Theodora Pappa, Laura Valderrábano, Kevin Bi, R. Gillani, Lauren Brais, Erin Shannon, Jason L. Hornick, Jihye Park, Jennifer Chan, E. V. Van Allen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, Weiqi Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with complex molecular mechanisms underlying its pathogenesis. Understanding the transcriptional landscape and cellular heterogeneity within the tumor microenvironment is crucial for identifying potential therapeutic targets and prognostic biomarkers. We performed comprehensive bulk RNA sequencing on lung cancer tissues and adjacent normal samples to identify differentially expressed genes (DEGs). Weighted gene co-expression network analysis (WGCNA) was conducted to identify functionally related gene modules associated with clinical phenotypes. Functional enrichment and pathway analyses were performed to elucidate biological significance. Single-cell RNA sequencing (scRNA-seq) was utilized to characterize cellular heterogeneity and reconstruct pseudotemporal trajectories. RT-qPCR validation was performed on A549 and H1299 lung cancer cell lines to confirm key findings. Bulk RNA-seq analysis identified extensive transcriptional reprogramming in lung cancer with symmetrical distribution of upregulated and downregulated genes. WGCNA revealed multiple co-expression modules significantly correlated with clinical traits, with turquoise and blue modules showing particularly strong associations. Functional enrichment analysis highlighted dysregulation in cell proliferation, immune response, metabolic reprogramming, and developmental signaling pathways including Wnt, Hedgehog, and estrogen signaling. Single-cell analysis identified distinct cellular populations including epithelial cells, immune cells (B cells, NK cells, dendritic cells), fibroblasts, and proliferating cells. Pseudotemporal trajectory analysis revealed dynamic cellular state transitions and identified five key cell cycle regulators (AURKA, FANCD2, HELLS, RRM2, STMN1) with stage-specific expression patterns. RT-qPCR validation confirmed significant upregulation of all five genes in both A549 (4.2 to 6.3-fold increase) and H1299 cells (3.9 to 5.9-fold increase) compared to normal bronchial epithelial cells (all p < 0.001). This integrated multi-omics approach reveals the complex transcriptional landscape and cellular heterogeneity in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebcdfb94400220dc54cbd6a2cfd578841de975f" target='_blank'>
              Integrated transcriptomic and single-cell analysis reveals cell cycle dysregulation and cellular heterogeneity in lung cancer
              </a>
            </td>
          <td>
            Yuzi Zhang, Wencheng Che, Qingchuan Li, Wei Huang
          </td>
          <td>2025-12-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Metastatic triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies. Circulating tumor cells (CTC) are invaluable for monitoring metastatic tumor progression and treatment response but are difficult to capture because of their rarity and heterogeneity. Surface-based staining for live CTCs is essential to preserve RNA quality in single cells, but current markers tend to perform poorly on more mesenchymal tumor cells such as TNBCs. To enhance live TNBC CTC detection, we developed a workflow for live CTC capture and single-cell RNA sequencing (scRNA-seq). Using a mouse model of metastatic TNBC, we identified four new CTC surface markers, AHNAK2, CAVIN1, ODR4, and TRIML2, which specifically stain tumor cells. Combining antibodies against these markers improved CTC detection rates in multiple TNBC mouse models and patient samples. Also, combining these new markers with traditional CTC surface markers enhanced detection sensitivity, achieving the highest CTC coverage. This approach identifies diverse CTC populations, while preserving RNA quality for scRNA-seq, which is essential for understanding and therapeutically targeting metastatic breast cancer. The use of these newly identified CTC markers significantly enhances both detection and live capture of CTCs, paving the way for more effective use of liquid biopsy to monitor patient prognosis and treatment response in clinical settings. Significance: CTCs are a powerful indicator of cancer metastasis; however, their scarcity makes them difficult to isolate. Current markers favor epithelial CTCs over mesenchymal populations. Our workflow for live CTC capture and sequencing enables discovery of new markers for both epithelial and mesenchymal CTCs. When combined with existing markers, we improve live CTC capture for more holistic studies of the metastatic process and offer a scalable method for discovering CTC markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b002ef6362d1a4ee73fe011a8af50d517c31c57" target='_blank'>
              Surface Marker Identification to Capture Live Circulating Tumor Cells in Metastatic Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Bree M Lege, Khushali J. Patel, Brendan Panici, Ping Gong, Michael T. Lewis, M.J. Ellis, Chonghui Cheng
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite significant advances in understanding lung development, the intricate cellular interactions and spatial organization of the developing human lung remain incompletely defined. Spatial transcriptomics enables gene expression profiling within the native tissue context, providing unprecedented insights into complex developmental processes. In this study, we applied the 10X Genomics Visium platform to characterize spatially resolved transcriptional profiles of prenatal human lungs during the pseudoglandular and canalicular stages.Spatial transcriptomic analysis of 12 prenatal lung samples (13–20 weeks gestation) identified 10 distinct transcriptional niches corresponding to unique combinations of epithelial, mesenchymal, endothelial, and immune cell populations. Unsupervised clustering revealed developmental shifts in spot/niche composition from the pseudoglandular to canalicular stage, with a progressive increase in alveolar epithelial spots and a concomitant decline in mesenchymal regions, particularly in peripheral lung areas. Differential gene expression analysis demonstrated stage-specific transcriptional transitions in individual spot types, including downregulation of cell cycle and structural pathways and upregulation of secretory pathways as the lung matures. Spatial organization analysis revealed increasing compartmentalization of pulmonary cell types, highlighting the progressive structuring of the distal lung microenvironment. In summary, this study provides a spatial map of the developing human lung, offering novel insights into pulmonary lineage dynamics and cellular interactions during early organogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4cdf7e14676d1f19fec1630ce3c773e08720b2c" target='_blank'>
              Spatial transcriptomics of developing human lungs defines cellular phenotypes associated with age, lineage and location
              </a>
            </td>
          <td>
            Yue Ren, Soula Danopoulos, Gail H. Deutsch, Ian A. Glass, Thomas J. Mariani, S. Bhattacharya
          </td>
          <td>2026-01-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 The tumor microenvironment (TME) plays a pivotal role in shaping cancer progression, immune evasion, and therapeutic response. However, a systematic understanding of immune and stromal cell heterogeneity across cancer types—and their associations with clinical phenotypes—remains lacking due to limited single‐cell data and the associated clinical metadata. Here, we present a comprehensive pan‐cancer TME cell atlas to date, quantifying the composition of 102 immune and stromal subtypes across 30 cancer types and encompassing over 11,000 bulk RNA‐seq samples from TCGA. To enable this unprecedented resolution, we developed DECEPTICONx, a flexible computational framework that integrates single‐cell RNA‐seq data by constructing high‐resolution expression templates, which are then paired with context‐optimized deconvolution strategies, dynamically selecting the best‐matched template‐deconvolution combinations for each task. By leveraging 535,665 single cells spanning T cells, B cells, fibroblasts, macrophages, and NK cells, we constructed high‐resolution expression templates and systematically deconvolved bulk samples. The resulting atlas revealed robust and novel associations between specific cell subtypes and patient prognosis, immunotherapy response, and inter‐cellular coordination within the TME. In summary, the DECEPTICONx‐based atlas not only advances our understanding of tumor‐immune ecology but also offers a foundation for biomarker discovery and therapeutic development in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaef7a98ed82e86ab1fb8032603b073ca5c5bb6" target='_blank'>
              A Pan‐Cancer Microenvironment Atlas Constructed by DECEPTICONx
              </a>
            </td>
          <td>
            Yida Gu, Fulan Deng, Lianchong Gao, Zhiyu Chen, Jianping Chen, Jiale Wu, Zengqiang Xia, Zhijian Li, Fangkaikai Cheng, Jie Chen, HoiYee Tong, Wantao Chen, Xiaoqing Pan, Xin Zou, Jie Hao
          </td>
          <td>2026-01-17</td>
          <td>iMetaMed</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Regional lymph nodes (LNs) in the thoracic cavity serve as essential immunological hubs that coordinate humoral and cell-mediated responses against the development and progression of non-small cell lung cancer (NSCLC). To investigate immune dysregulation in the non-metastatic regional LNs of patients with aggressive NSCLC, we performed multimodal profiling on 36 LNs from 11 patients undergoing curative-intent resection including CITE-seq, scRNA-seq, and Imaging Mass Cytometry (IMC). Regional N1 LNs from patients with more aggressive disease (stage IB-IIIA) exhibited a significant enrichment of dysfunctional CD8+ T cells and regulatory T cells (Tregs) compared to N2 LNs and LNs from patients with less aggressive disease (stage IA). These immune subsets were spatially co-localized with mature regulatory dendritic cells (mregDCs; CD1c+, TIM3+, LAMP3+), forming an immunosuppressive niche uniquely enriched in the N1 LNs of higher-stage patients. Concurrently, higher-stage N1 LNs contained larger number of decorticated B-cell follicles characterized by decreased encapsulation of the mantle zone layer surrounding the germinal centers. This mantle zone disorganization was associated with increased spatial niches involving Tregs, CD68+ CD163+ TIM3+ Macrophages, CD163+ TIM3dim Monocytic-Myeloid Derived Suppressor Cells (M-MDSC), plasma B cells, and a decrease in spatial niches involving CD4+ T helper cells and fibroblastic reticular cells (FRCs). Together, our findings reveal parallel alterations in humoral and cell-mediated immunity within the regional LNs of patients with aggressive NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a9f04f8595229db875548fb40d4f6cfb792216" target='_blank'>
              Multimodal single-cell and spatial profiling reveals altered T cell-mediated immunity and B-cell follicular architecture in non-metastatic lymph nodes of patients with aggressive non-small cell lung cancer
              </a>
            </td>
          <td>
            Z. Xi, Y. Koga, S. McDermott, E. Kane, R. Pfefferkorn, E. Billatos, P. R. Hosking, J. Beane, E. Burks, S. Mazzilli, K. Suzuki, J. D. Campbell
          </td>
          <td>2026-01-18</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is considered immunologically “cold,” however, we have previously identified an immunogenic subset of high-risk localized PCa with T-cell tumor infiltration within antigen-presenting-cell (APC) niches. Here, we used high-dimensional single-cell spatial proteomics to dissect tumor-infiltrating-lymphocyte in (TIL)-high (H), TIL-low (L), and TIL-excluded (X) PCas, to understand the tumor-specific and conserved mechanisms of immune dysregulation for targeted therapeutics.



 Treatment-naive prostatectomy specimens of all three tumor immune microenvironment (TIME) phenotypes, were assembled into tissue microarrays representing within-tumor and adjacent foci. TIL-X was exclusively observed in cribriform pattern 4 cases. A 40-plex antibody panel made of phenotyping and functional markers (e.g AR) was applied to the TMAs, enabling single-cell annotation of key immune cell types (e.g CD8+ T cells) and assessment of their functional states within distinct spatial microenvironments.



 In TIL-X PCa, immunosuppressive macrophages and DCs were enriched in the tumor nest whereas T cells accumulated at the periphery with minimal tumor infiltration. Tumor adenosine 2A receptor (A2AR) expression was uniquely enriched within tumor, suggesting a metabolically suppressive microenvironment that reinforces immune exclusion. TIL-H tumors had comparable T cell proportions inside and outside the tumor, accompanied by the enrichment of myeloid cells within the tumor. CD8+ T cells expressing high levels of AR and B7H3 frequently co-localized with tumor and APCs, forming APC niches and tertiary lymphoid-like structures. CD8+ARhi and CD8+B7H3hi subsets with elevated exhaustion scores preferentially infiltrated tumor, whereas CD8+ARlo and CD8+B7H3lo subsets with moderate exhaustion levels remained peripheral; defining TIL-H PCa as tumors harboring antigen-experienced but exhausted CD8 T cells in a TIME that supports immune cell engagement but restrains anti-tumor activity. TIL-L PCa had sparse T cell infiltration and enrichment of myeloid cells with moderate AR expression within tumor, suggesting a role of androgen signaling and myeloid cells in shaping the TIME, marked by high PD-1+CD8+ cells and PD-L1+ tumor cells. Thus, TIL-L PCa shows low immune recruitment, yet strong tumor-mediated inhibitory signaling that constrains anti-tumor immunity.



 TILs in TIL-H PCa are phenotypically, functionally, and spatially distinct from those in TIL-L or TIL-X tumors, reflecting tumor-intrinsic immune programs. High-dimensional spatial proteomics reveal differential molecular modules and signaling pathways that define unique and potentially targetable mechanisms of immunosuppression across the three TIME phenotypes; additional spatial transcriptomics validation and dissection experiments are underway. These findings demonstrate that the PCa TIME is diverse yet spatially organized, and that comprehensive understanding of baseline spatial and transcriptional immune states may guide personalization of immunotherapy strategies in PCa.



 Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik Michel, Sabina Signoretti, Yue Sun, Steven Balk, Sizun Jiang, David Einstein Einstein. High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d786dbf212bd5a71f25f7e11f429f8dba252ddc" target='_blank'>
              Abstract PR010: High-dimensional spatial profiling identifies distinct and targetable tumor-infiltrating lymphocyte phenotypes in localized prostate cancer
              </a>
            </td>
          <td>
            Precious Cramer, Yang Wang, Wenrui Wu, Huaying Qiu, Lindsay Parmelee, Berkay Simsek, Gabriel Roberti. De Oliveira, Seifeldin Awad, Yao Yu Yeo, Stephanie Pei Tung. Yiu, Hendrik A. Michel, Sabina Signoretti, Yue Sun, Steven P. Balk, Sizun Jiang, D. Einstein
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer cell invasion into the surrounding muscle capsule, known as extracapsular extension (ECE), is associated with a significantly worse prognosis (5-year survival reduction: 98% to 31%). Using a live ECE mouse xenograft model with DU145 tumor cells, our group previously demonstrated that functional heterogeneity among adhesion subtypes within invasive clusters is essential for muscle invasion. Bulk RNA sequencing from three spatial compartments (pre-invasive, invasive, and post-invasive) revealed distinct gene expression profiles, with hypoxia-related and cell–cell/cell–ECM interaction genes prominently dysregulated in muscle-invasive disease. In this study, we sought to define the single-cell transcriptional heterogeneity across the three ECE compartments and compare these patterns to a human patient-derived xenograft (PDX) model. Spatial transcriptomic analysis (STA) was performed using the Bruker CosMx platform with a 6K gene panel. Data was preprocessed (quality control and dimensional reduction) and analyzed in R 4.4.0 using Seurat v5. Notable downstream analyses included unsupervised Louvain clustering, trajectory inference with Monocle 3, and cluster-specific differential gene expression. In the ECE model, STA resolved at least 11 distinct tumor cell identity clusters previously undetectable by bulk methods and revealed a clear spatial demarcation between pre-invasive and invasive populations crossing the muscle barrier. Trajectory analysis identified a pre-invasive population branching into two invasive fates with unique expression profiles. Dispersion scores, calculated using principal component embeddings, exhibited progressively increasing intracluster heterogeneity along one branch. Strong expression enrichment of the Prolaris 31-gene prognostic panel, a clinically established marker of aggressiveness, was observed at the bifurcation point of tumor evolution. STA of the human PDX model similarly revealed a high-dispersion invasive cluster branching into two fates, one exhibiting decreased heterogeneity consistent with specialization of invasive phenotypes. Together, these findings reveal, for the first time, that tumor invasion of the extracapsular muscle layer induces substantial transcriptional heterogeneity. This dynamic process highlights potential therapeutic vulnerabilities during the transition to extracapsular extension. Targeting tumor cells at this critical juncture may represent a key strategy for preventing ECE and improving prostate cancer outcomes. Supported by P30CA023074, University of Arizona Comprehensive Cancer Center Support Grant.



 Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, Noel A. Warfel, Gregory C. Rogers, Beatrice S. Knudsen, Rashid Sayyid, Kelvin W. Pond, Anne E. Cress. Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcc6a26c8a5d704d47fd40ba9c2b9c0030fcf73d" target='_blank'>
              Abstract B045: Single cell spatial transcriptomics identifies unique cell populations driving muscle invasive prostate adenocarcinoma
              </a>
            </td>
          <td>
            Saptarshi Mallick, Rafael Sainz, Jayati Chakrabarti, Kendra D. Marr, N. Warfel, Gregory C Rogers, Beatrice S. Knudsen, R. Sayyid, Kelvin W. Pond, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="CD8+ T cells are an important weapon in the therapeutic armamentarium against cancer. While CD8+CD103+ T cells with a tissue-resident memory T (TRM) cell phenotype are associated with favorable prognoses, the tumor microenvironment also contains dysfunctional exhausted T (TEX) cells that exhibit a variety of TRM-like features. Here we deconvolute TRM and TEX cells across human cancers, ascribing markers and gene signatures that distinguish these populations and enable their functional distinction. Although TRM cells have superior functionality and are associated with long-term survival post-tumor resection, they are not associated with responsiveness to immune checkpoint blockade. Tumor-associated TEX and TRM cells are clonally distinct, with the latter comprising tumor-independent bystanders and tumor-specific cells segregated from cognate antigen. Intratumoral TRM cells can be forced toward an exhausted fate when chronic antigen stimulation occurs, indicating that the presence or absence of continuous antigen exposure within the microenvironment is the key distinction between tumor-associated TEX and TRM populations. These results highlight unique functions for TRM and TEX cells in tumor control, underscoring the need for distinct strategies to harness these populations for cancer therapies. Here the authors show that tissue-resident memory and exhausted T cells in tumors are distinct populations that are shaped by relative presence or absence of TCR signals, suggesting that a tailored therapeutic strategy is needed to target each subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a712967a1f06be04c88812bdec6b8089c56c0b" target='_blank'>
              Antigen reactivity defines tissue-resident memory and exhausted T cells in tumors
              </a>
            </td>
          <td>
            Thomas N. Burn, Jan Schröder, Luke C. Gandolfo, M. Osman, Elanor N. Wainwright, E. Lam, Keely M. McDonald, Rachel B. Evans, Shihan Li, Daniel Rawlinson, Lachlan Dryburgh, Ali Zaid, Z. Maliga, Dominick Schienstock, Philippa Meiser, H. J. Lee, Hongjin Lai, Marcela L. Moreira, Pirooz Zareie, Louis H-Y. Lee, Lutfi Huq, Susan N. Christo, Justine Seow, Keith A. Ching, Stéphane M. Guillaume, Kathy Knezevic, Simone L. Park, Maximilien Evrard, Jason Waithman, T. Gebhardt, Scott N. Mueller, G. Riddiough, M. V. Perini, S. C. Tsao, Terence P. Speed, P. Sorger, S. Loi, Francis R. Carbone, Stephanie Gras, Timothy S Fisher, Bas J Baaten, M. Dawson, Laura K. Mackay
          </td>
          <td>2025-12-29</td>
          <td>Nature Immunology</td>
          <td>2</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1654ebba084e6a42978b44a5cecff88e9f2e8ee" target='_blank'>
              DISCO-seq: 3D single-cell transcriptomics of intact biological systems
              </a>
            </td>
          <td>
            H. Bhatia, Laurent H. A. Simons, L. B. Kuemmerle, Cristin McCabe, S. Jansen, Zhisong He, Mayar Ali, Denise Jeridi, M. Todorov, Luciano Hoeher, S. Padmarasu, Ha Eun Park, Justus Thevis, L. Bartos, Inderjeet Singh, S. Ussar, Kıvanç Görgülü, Hana Algül, Kenny Roberts, O. Bayraktar, Matthias Brendel, F. Theis, Barbara Treutlein, Aviv Regev, Ali Ertürk
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) maturity and cellular composition shape resistance or sensitivity to immune-checkpoint blockade (ICB) across cancers. Single-cell spatial resolution data for the endothelial compartment of TLSs are lacking; therefore, we investigated the cellular composition, endothelial dynamics, and cell-cell interactions within TLS regions. We applied high-resolution Xenium In Situ spatial transcriptomics to eight samples collected from non-small-cell lung cancers (NSCLC) (four after chemoimmunotherapy, four untreated), extracted 90 TLS regions, generated approximately 320,000 single-cell profiles from those regions, and analyzed the endothelial compartment within each TLS. We independently assessed the exploratory findings using multiplex immunofluorescence and survival analysis in a separate cohort of 45 patients receiving immunotherapy. Spatial mapping revealed a reproducible, B-cell-dominant TLS architecture across all samples. Mature TLSs contained CXCL13-rich follicular cores and were associated with a favorable prognosis. Within the endothelial compartment, high endothelial cells (HECs) forming high-endothelial venules (HEVs) segregated from other endothelial cells; in mature TLSs these HECs exhibited high inhibitor of DNA binding 1 (ID1) expression and an adhesion molecular signature, promoting lymphocyte recruitment. Patients with tumors harboring ID1-high HEVs showed higher response rates and pronounced survival advantage; multivariable analysis confirmed ID1-high status as an independent prognostic factor. Conversely, ICB exposure eventually reduced ID1 expression and adhesion molecule levels in HECs, impaired lymphocyte trafficking, and promoted a shift toward a remodeled endothelial state. ID1expression, linked to HEC differentiation during TLS maturation or ICB exposure, marks a functionally mature, lymphocyte-recruiting HEV that predict durable response to immunotherapy. These findings provide a novel mechanistic framework and practical biomarkers for HEV-centered immuno-oncology strategies in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47785e0dbc1da1e3fb62fce1edd50b239726393" target='_blank'>
              Single-cell spatial analysis identifies ID1-high endothelial cells in tertiary lymphoid structures as predictors of durable response to immunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Kinnosuke Matsumoto, Y. Noda, Kensuke Hachiya, F. Muramatsu, Naoki Okamoto, Weizhen Jia, T. Shiroyama, Masahide Mori, M. Tamiya, Yuhei Kinehara, A. Tamiya, Shigeki Shimizu, Hidekazu Suzuki, Kiyonobu Ueno, T. Niki, Satoshi Tetsumoto, O. Morimura, Akio Osa, Toshiyuki Minami, Satoshi Nojima, Y. Takeda, Y. Shintani, Atsushi Kumanogoh, Nobuyuki Takakura
          </td>
          <td>2026-01-06</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (C-CRC) arises in the context of chronic inflammation, where tumor microenvironment interactions strongly influence progression. Spatial transcriptomics enables characterization of these interactions by retaining tissue architecture and cellular context. We aimed to characterize the cellular and microenvironmental features that underlie tumor development in C-CRC.



 Seven tissue samples from three C-CRC patients were analyzed using consecutive sections processed by 10x Xenium and 10x Visium. Spatial coordinates of Xenium cells were mapped onto Visium spots to enable integrated spatial transcriptomic profiling (Figure 1A).



 Xenium analysis identified 3,506,705 cells with diverse epithelial, fibroblast, and immune cell populations (Figure 1B-C). Tumor cell subclustering revealed two functionally segregated populations: (1) proliferating tumor cells characterized by cell-cycle genes (MKI67, TOP2A) and (2) invasive tumor cells with elevated epithelial mesenchymal transition (EMT)-associated genes (CD44, CXCL1). Cellular neighborhood analysis based on distances between cells identified a distinct microenvironmental niche unique to invasive tumor cells, characterized by spatial proximity of cancer-associated fibroblasts (CAFs) and SPP1+ macrophages (Figure 1D). Pathway enrichment analysis showed selective up-regulation of angiogenesis, hypoxia, and inflammatory pathways in invasive versus proliferating tumor regions (Figure 1E). Receptor–ligand analysis identified TNC–ITGB6 interactions between CAFs and invasive tumor cells (Figure 2A). A differentially expressed gene signature distinguishing invasive from proliferating tumor cells stratified TCGA sporadic CRC samples into invasive-tumor-like and proliferating-tumor-like groups. Invasive-tumor-like tumors predominantly displayed consensus molecular subtype (CMS) 4 characteristics, consistent with the known enrichment of CMS4 in UC-CRC and supporting a role for CAF-mediated interactions in mesenchymal phenotype development (Figure 2B).



 This integrated spatial transcriptomics study identified two distinct tumor populations in C-CRC with segregated microenvironmental contexts. The invasive tumor phenotype is characterized by CAF proximity, SPP1+ macrophages, and TNC–ITGB6-mediated signaling. These findings reveal that tumor-stromal composition and architecture are linked to mesenchymal phenotype development, suggesting the microenvironment as a potential target for biomarker and therapeutic development in C-CRC.
 Conflict of interest:
 Lee, Yoonho: No conflict of interest
 Kim, Yongjae: No conflict of interest
 Baek, Jiwon: No conflict of interest
 Hwang, Sung Wook: no conflicts
 Lee, Jong Lyul: None.
 Park, Sang Hyoung: No conflicts
 Yang, Dong-Hoon: Nothing to declare
 Kim, Jihun: No conflict of interest
 Yang, Suk-Kyun: No conflict of interest
 Han, Buhm: No conflict of interest
 Song, Kyuyoung: No conflict of interest
 Yoon, Yong Sik: No conflict of interest
 Ye, Byong Duk: Byong Duk Ye reports consulting fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Chong Kun Dang Pharm, CJ Red BIO, Curacle, Daewoong Pharm, Dong-A ST, Ferring Korea, Hanmi Pharmaceutical, Imscout, IQVIA, Johnson & Johnson, Johnson & Johnson Korea, Jeil Pharmaceutical Co., Kangstem Biotech, Korea Otsuka Pharm, Korea United Pharm, Lilly Korea, Medtronic Korea, NanoEntek, ORGANOIDSCIENCES Ltd., Pfizer Korea, Samsung Bioepis, Takeda, Takeda Korea and Yuhan speaker fees from AbbVie Korea, BMS Pharmaceutical Korea Ltd., Celltrion, Cornerstones Health, Curacle, Daewoong Pharm, Eisai Korea, Ferring Korea, IQVIA, Johnson & Johnson Korea, Pfizer Korea, Samsung Bioepis, and Takeda Korea and research support from Celltrion and Pfizer Korea.
 Lee, Ho-Su: No conflict of interest


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c70793b20108beca09788feb076288ec350d6d1" target='_blank'>
              DOP110 Spatial mapping of tumor-stromal cell interactions defines mesenchymal phenotypes in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y Lee, Y. Kim, J. Baek, S. Hwang, J. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K. Song, Y. Yoon, B. Ye, H. S. Lee
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b319260cf980924d910407801fa072aa43365f" target='_blank'>
              Multi-omic profiling of pleural effusions: a reservoir-incubator for non-invasive insights into tumor heterogeneity
              </a>
            </td>
          <td>
            Paula Nieto, G. Caratù, M. Mulet, M. Sorolla, Virginia Pajares, Domenica Marchese, Ana Pardessus, Rubén Osuna-Gómez, A. Rafecas-Codern, Pere Serra-Mitjà, J. Porcel, Silvia Vidal, Holger Heyn, Juan C. Nieto
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Malignant glioma is incurable. Using a mouse genetic mosaic system to generate sporadic Trp53,Nf1-null OPCs, we previously identified oligodendrocyte precursor cell (OPC) as a cell-of-origin of glioma. Here, we report that pre-malignant Trp53,Nf1-null OPCs outcompete wildtype counterparts during their expansion. Blocking competition by mutating/strengthening wildtype OPCs impeded both pre-malignant progression and malignant expansion of glioma. “In-tissue” phosphoproteomic profiling revealed an enrichment of phosphopeptides related to RNA splicing and protein translation at the peak of cell competition, suggesting that competitiveness may stem from unique protein species. Among candidates was mTORC1, whose pharmacological inhibition or genetic disruption resulted in a loss of competitiveness in our mouse model. Finally, analysis of patient biopsies and interrogating the role of individual gliomagenic mutations in OPC competition supported its relevance in human gliomas. Together, these findings identified the driving role of competitive interactions among OPCs in gliomagenesis, and suggest unconventional therapeutic strategies to target this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b48b1c55225e03672e68d4c772d4807ab8a0a9" target='_blank'>
              Critical role of cell competition in gliomagenesis
              </a>
            </td>
          <td>
            Ying Jiang, Ryuhjin Ahn, Arthur Huang, Phillippe P. Gonzalez, Jungeun Kim, Guoxin Zhang, Zihao Liu, Zhenqiang He, Lindsey A Dudley, Kunal S. Patel, Godfrey A Dzhivhuho, Sam Crowl, Piotr Przanowski, Luisa Quesada Camacho, Sijie Hao, Jianhao Zeng, S. Hippenmeyer, Mohammad Fallahi-Sichani, Kevin A. Janes, Kristen M. Naegle, M. Hammarskjold, Steven A. Goldman, H. Kornblum, Maojin Yao, Forest White, Hui Zong
          </td>
          <td>2026-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intratumoral heterogeneity (ITH) is a major driver of mortality in breast cancer (BC) patients and a critical factor in the variable therapeutic outcomes observed in BC treatment. Understanding the mechanisms underlying ITH is essential for advancing both clinical and basic BC research. Chromatin accessibility is critical for regulation of gene expression and cellular identity and plays a central role in shaping ITH and tumor evolution. However, studying chromatin accessibility in situ has been challenging due to the availability of technical platforms. Here, we leveraged the spatial ATAC-seq platform to profile the chromatin accessibility landscape of tumors from six BC patients. Our analyses revealed prominent heterogeneity within tumor regulatory modules and spatial variations in immune cell composition and stromal structures, offering a framework for investigation of the molecular architecture underlying ITH. Moreover, we identified two tumor subclones with potential common origin but distinct immune infiltration conferred by regulatory cascades, suggesting that epigenetic regulation may further contribute to the divergent tumor microenvironments and phenotypic diversity of these subclones. Our study provides novel insights into the molecular mechanisms driving ITH and opens up potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b8d2e84c5991207c662ff2b4c48eb4734a447e" target='_blank'>
              Spatial Chromatin Accessibility Analysis of Intratumor Heterogeneity in Breast Cancer.
              </a>
            </td>
          <td>
            Yingying Qian, Miao Zhu, C. Ren, Yeyong Zhou, Jian Xu, Liang Dong, Guangyu Zhang, Cheukfai Li, Jiaoyi Lv, Qiaorui Xing, Guochun Zhang, Guangdun Peng, Ning Liao
          </td>
          <td>2026-01-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spatial organization of the disease microenvironment informs patient prognosis. Key modalities for studying spatial biology include H&E images (WSIs) for tissue structure and spatial transcriptomics (ST) for transcriptome-level programs. Spatial analysis aims to (1) identify markers linked to clinical outcome, (2) understand functional programs driving these associations, and (3) guide targeted therapies. Current research addresses these topics but offers limited explainability across the full morphology - molecular mechanism - outcome axis. Further, given the abundance of WSIs and limited availability of ST, there is a need for analyses integrating these complementary datasets. We present an AI-driven framework combining foundation-model features, multiple-instance learning, unsupervised clustering, and molecular analyses to identify mechanisms underlying outcome associated patterns. Applied to HER2+ breast cancer, we identify CCND1 and PTK6 signaling in tumor regions linked to trastuzumab resistance, consistent with prior studies. Our approach offers interpretable insights for multi-level resistance mechanisms, tissue-specific drug targeting, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a60accd94c10ba89d66c522cd744cd2f973bb5" target='_blank'>
              A multimodal framework to identify molecular mechanisms driving patient group-associated morphology through the integration of spatial transcriptomics and whole slide imaging
              </a>
            </td>
          <td>
            Reva Kulkarni, A. Maddox, Sara Bailey, Arvind Rao
          </td>
          <td>2026-01-16</td>
          <td>Npj Artificial Intelligence</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e234e293312dfb9093dad9f87bd40ef8d26f2e28" target='_blank'>
              Spatial transcriptomics reveals molecular heterogeneity and subtype-specific therapeutic targets in small cell lung cancer.
              </a>
            </td>
          <td>
            Tongji Xie, Le Tang, G. Fan, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-01-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d27ce10a067c8461179490fc90529630fe99743" target='_blank'>
              A modular patient-derived organoid–xenograft platform reveals molecular and clinical trajectories of prostate cancer progression
              </a>
            </td>
          <td>
            R. Parmentier, R. Dolgos, R. Servant, Jing Wang, L. Roma, R. Mevel, D. Bossi, H. Pueschel, Z. Diamantopoulou, T. Vlajnic, F. Stenner, A. J. Templeton, H. Seifert, Nicola Aceto, J-P. Theurillat, Cyril Rentsch, L. Bubendorf, C. Le Magnen
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7315a577afd833fc7a6acdad8a8942079ca758fc" target='_blank'>
              A poly(A) isoform–aware single-cell and spatial atlas defines fibroblast niches in the human bladder
              </a>
            </td>
          <td>
            Briana Santo, Emily E. Fink, Pierre-Emmanuel Desprez, Yi-Chia Lin, M. Eltemamy, Alvin Wee, Ninh B. Le, Alexandra E. Krylova, Uyen Tran, Veena Kochat, Kunal Rai, Douglas W. Strand, Oliver Wessely, Byron H. Lee, Angela H. Ting
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer, with liver metastases significantly worsening outcomes. However, features of the tumor microenvironment (TME) that are distinct between primary and metastatic sites remain poorly defined. Cellular neighborhoods within the TME are recognized as functional units that influence tumor behavior. Conventional spatial methods, which assign equal weights to all cells in a region, fail to capture the nuances of cellular interactions. To address this, we report here the development of Functional Cellular Neighborhood (FunCN) quantification, which integrates both the proportion and proximity of surrounding cells. Applying FunCN to PDAC imaging mass cytometry data, we identified neutrophil-enriched interactions in liver metastases compared to primary tumors, correlating with elevated VISTA expression by tumor cells. Additionally, FunCN clusters around CD8⁺ T cells in pancreas and liver were associated with higher TIGIT and LAG3, respectively. These findings demonstrate the importance of spatial immune landscapes in PDAC and identify potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87979b18b917426107034a7951a93426ded0148e" target='_blank'>
              Computational modeling of cellular influence delineates functionally relevant and distinct cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Yeonju Cho, Jae W. Lee, Sarah M. Shin, Alexei G Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, E. Jaffee, Atul Deshpande, W.J. Ho
          </td>
          <td>2026-01-23</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tissue architecture of classic Hodgkin Lymphoma (CHL) is unique among cancers and characterized by rare malignant Hodgkin and Reed-Sternberg cells that co-evolve with a complex ecosystem of immune cells in the tumor microenvironment (TME). The lack of a comprehensive systems-level interrogation has hindered the description of disease heterogeneity and clinically relevant molecular subtypes. Here, we employed an integrative, multimodal approach to characterize CHL tumors using malignant cell sequencing, spatial transcriptomics and imaging mass cytometry. We identified four molecular subtypes (CST, CN913, STB, and CN2P), each characterized by distinct clinical features, mutational patterns, malignant cell gene expression profiles, and spatial architecture involving immune cell populations. Functional modeling of CSF2RB mutations, a characteristic feature of the CST subtype, revealed dysregulated oncogenic signaling and unique TME crosstalk. These findings highlight the significance of multi-dimensional profiling in elucidating patterns of molecular alterations that drive immune ecosystems and underlie therapeutically exploitable vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6324873c99ed0026f9e046294cb9724bf214ad" target='_blank'>
              Multidimensional Characterization of Tumor-Immune Architecture Reveals Clinically Relevant Classic Hodgkin Lymphoma Subtypes.
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Yu Li, Cecilia Lee, Denise Smorra, L. Hilton, S. Healy, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O Brien, Jasper Ch Wong, Claudia Cassidy, Manabu Fujisawa, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Rangel Patiño, Michael Hong, Shaocheng Wu, Eric Lee, Ali Sakhdari, Katsuyoshi Takata, T. Miyata-Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander Xu, A. Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, R. Morin, A. Alizadeh, K. Savage, David W Scott, Christian Steidl
          </td>
          <td>2026-01-30</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b5cdea0e446c4569bc246289883bf8c4b45dc65" target='_blank'>
              miniMTI: minimal multiplex tissue imaging enhances biomarker expression prediction from histology
              </a>
            </td>
          <td>
            Zachary Sims, Sandhya Govindarajan, Kaoutar Ait-Ahmad, Ç. Ak, Marigold Kuykendall, Gordon B. Mills, Ece Eksi, Young Hwan Chang
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The spatial distribution and functional heterogeneity of tumor-infiltrating lymphocytes (TILs) significantly impact patient outcomes in non-small cell lung cancer (NSCLC). While T cell factor 1 (TCF1) expressing TILs have emerged as key players in sustaining anti-tumor immunity, their subset characterization, localization, and clinical significance within the tumor microenvironment remain poorly defined. Method We performed multiplex immunohistochemistry and immunofluorescence to characterize TCF1+ immune cell subsets, in 102 NSCLC tumors, separately analyzing the tumor center (TC) and invasive front (IF). We integrated this data with publicly available single-cell RNA-sequencing datasets and clinical outcome analyses. Results CD4+ T cells and CD79α+ B cells, dominate the TCF1+ landscape, while CD8+ T cells constitute a minority of TCF1+ immune cells, particularly in the TC. We demonstrated the presence of tumor-infiltrating IgG+/IgA+ plasma cells co-expressing TCF1. PD1+TCF1- cells were more frequent than PD1+TCF1+ cells both in the TC and IF, reflecting that terminally differentiated exhausted TILs predominate within the tumor microenvironment. Survival analyses revealed significantly different prognostic impact of TILs including TCF1-expressing cells based on topography. Multivariate analysis showed that increased CD8+TCF1+ cells (HR: 2.5; p=0.039) and increased TCF1 expression by cancer cells (HR: 2,7; p=0.041) in the TC and CD4+TCF1+ cells (HR: 0.4; p=0.043) in the IF emerged as negative and positive independent prognostic markers for Disease-free survival (DFS), respectively. Integrating PD-L1 expression with TILs, PD-L1 immunopositivity was correlated with increased CD8+ and PD1+TCF1- cell infiltration and was associated with favorable DFS especially in the TC. Conclusions Our findings support a more refined framework for TCF1+ TIL assessment and TCF1 expression across cellular populations in the tumor microenvironment, with implications for prognostication in operable NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb42a95fd19ed986ce09103139534fde5284c2fa" target='_blank'>
              Immunophenotyping TCF1-expressing TILs: spatial profiling and prognostic value in operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Konstantinos Ntostoglou, Georgios Christodoulopoulos, Katie Stoker, Jean Descarpentrie, A. Xagara, D. Chatzidaki, Vasiliki Anastasopoulou, Ilias P. Nikas, A. Ieronimaki, Lucy Booth, Sophia Tsoka, Ioannis Vamvakaris, A. Klinakis, Eleni Patsea, Sophia N. Karagiannis, Ioannis G. Panayiotides, Teresa Frisan, Vassilis Georgoulias, A. Kotsakis, I. Pateras
          </td>
          <td>2026-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0135229fffb59fe03ba95bebb58be45a4b3470c" target='_blank'>
              De novo H3.3K27M-altered Diffuse Midline Glioma in human brainstem organoids to dissect GD2 CAR T cell function
              </a>
            </td>
          <td>
            N. Bessler, A. Wezenaar, H. Ariese, C. Honhoff, E. J. Wehrens, N. Dommann, C. Ruiz Moreno, T. J. van den Broek, R. Collot, D.J. Kloosterman, F. Keramati, M. Roosen, S. de Blank, Esmée J. van Vliet, M. Barrera Román, L. C. Gatti, Ali Ertürk, J. Kuball, Z. Sebestyén, M. Kool, S. Patrizi, E. Miele, A. Künkele, M. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Christian Mayer, H. Stunnenberg, A. Alemany, Maria Alieva, A. Rios
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Neoadjuvant immunotherapy has reshaped the treatment paradigm for esophageal squamous cell carcinoma (ESCC), yet mechanisms of resistance in non-responders remain poorly understood. Specifically, the spatial orchestration of the tumor microenvironment that limits effective antitumor immunity is not fully elucidated. Methods Integrating spatial transcriptomic analysis with large-scale tissue pathology, we mapped the spatial architecture of the immune landscape in immunotherapy-resistant ESCC to identify cellular and molecular determinants of treatment failure. Results We revealed a distinct immune exclusion phenotype in non-responders, characterized by peritumoral CD8+ T cell enrichment coupled with intratumoral depletion. At the invasive front, we identified COL11A1+ cancer-associated fibroblasts (CAFs) and SPP1+ tumor-associated macrophages as spatially correlated markers of immune exclusion, demarcating regions characterized by limited T cell infiltration. Mechanistically, a distinct subpopulation of LAMC2+ tumor cells localized at the tumor boundary acts as the master orchestrator of this barrier. These LAMC2+ cells exhibit aberrant lactate metabolism and elevated stemness, driving CAF activation via Semaphorin 3C secretion. Strikingly, this ‘LAMC2+ boundary tumor cell-immune-privileged niche’ axis exhibits pan-cancer correlations with immunotherapy resistance, positioning LAMC2 as a robust predictive biomarker. Conclusion In this study, we identify the tumor invasive margin as a critical determinant of immunotherapy resistance in ESCC. By defining the specific spatial markers and molecular architecture of an immune-privileged niche associated with the immune-exclusion barrier, our findings demonstrate the value of spatially resolved microenvironmental analysis. Moreover, we propose that therapeutic targeting of this boundary niche represents a promising strategy to overcome resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858298d1227537940c73aee34b8d245e5b4c9498" target='_blank'>
              An Immune-privileged niche mediates immunotherapy resistance in esophageal carcinoma
              </a>
            </td>
          <td>
            Li Wan, Boye Li, Tianyun Ma, Min Wang, Fei Xie, Haotian Ge, Zixuan Xiao, Xinyi Chen, Lingda Meng, Li Zhang, Kunxin Xie, Qilong Wang
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Non-small cell lung cancer is the leading cause of cancer-related mortality worldwide, with lung adenocarcinoma (LUAD) as the most common subtype. Although early-stage disease is often treated surgically, advanced LUAD typically requires chemotherapy, radiation, and/or immunotherapy, largely focused on T cell–mediated responses. Therapeutic efficacy, however, is also shaped by tumor-infiltrating (TI) B cells, whose roles in LUAD remain incompletely understood.



 We performed cytometry by time of flight (CyTOF) using 44 markers on matched tumor, adjacent lung, and peripheral blood samples from 48 LUAD patients to define TI immune landscapes. 66 immune cell subsets were identified, and patients were stratified into four groups based on TI cell composition. Adaptive immune receptor repertoire sequencing (AIRR-seq) of IgM and IgG was conducted on matched samples from 29 patients to assess clonal expansion and affinity maturation. Subisotype-resolved AIRR-seq was additionally performed on tumor samples from 18 patients.



 CyTOF analysis revealed four patient groups with distinct TI immune profiles. AIRR-seq demonstrated increased clonal expansion and affinity maturation in tumors compared to adjacent lung and blood. Tumor-specific IGHV enrichment patterns were observed but were not associated with patient group assignment. Instead, clonal expansion was greatest in tumors with higher lymphocyte proportions. Subisotype-resolved analysis showed enrichment of IGHG2 and IGHG3 in tumors from patients with low TI B cell abundance, whereas IGHG4 was enriched in patients with high TI B cell infiltration and correlated with four CyTOF-defined immune subsets.



 These findings reveal substantial inter-individual variation in TI immune landscapes and highlight distinct B cell repertoire and subisotype features within LUAD tumors. Together, these data suggest that B cell composition and antibody subisotype usage may contribute to immune contexture and could inform the development of more tailored immunotherapeutic strategies in LUAD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6aa20b3c0324cfd852e979d6ecb7e9b1134c03" target='_blank'>
              B cell phenotypes and antibody signatures associate with interpatient variation in the lung adenocarcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Kamille M. Rasche, David Tieri, Samantha M. Morrissey, Hong Li, Fan Zhang, William S. Gibson, EA Ford, Uddalok Jana, Melissa L. Smith, Jun Yan, Corey T. Watson
          </td>
          <td>2026-01-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Distal cholangiocarcinoma (dCCA) arises from the distal bile duct and is anatomically embedded within the pancreatic head, adjacent to abundant autonomic nerve plexuses. This unique location renders dCCA particularly prone to perineural invasion (PNI), a pathological hallmark that contributes to its dismal prognosis. However, the spatial architecture and molecular drivers that orchestrate PNI remain poorly defined. Here, we applied Xenium subcellular resolution spatial transcriptomics platform to profile resected tumor tissues from dCCA patients stratified by PNI status pathologically. A spatially resolved atlas comprising a total of 20 cell types was generated, uncovering enrichment of Schwann cells, type 2 conventional dendritic cells (cDC2), M2-like macrophages, cancer associated fibroblasts (CAFs) and B/plasma cells in PNI-high tumors, along with depletion of exhausted CD8+ T cells. Heterogeneous malignant cells in PNI-high tumors demonstrated activation of extracellular matrix remodeling and axonogenesis pathways, in line with the initial pathological classification. Spatial mapping further revealed distinct PNI-associated niches, notably matrix-producing CAFs (mCAFs)-macrophage clusters exhibiting coordinated enrichment of inflammatory and fibrotic programs. We further identified the LAMB3-DAG1 axis as a potential mediator of dCCA cells-Schwann cell interaction, while the preferential proximity of arteries to Schwann cells suggested additional microenvironmental support for nerve invasion. Collectively, our study provides a comprehensive subcellular atlas of PNI in dCCA, uncovering coordinated epithelial, stromal, and immune remodeling that drives perineural invasion. The identified biomarkers not only hold promise for patient stratification but may also guide intraoperative navigation and surgical margin determination, offering new avenues for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ed53173a0ee6bb5d8c5c5791ee85be378c7900" target='_blank'>
              A subcellular spatial atlas illuminates the microenvironmental remodeling of perineural invasion in distal cholangiocarcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Hao Chen, Huan Li, Jiawei Zhang, Sijia Li, Pengfei Wang, Hao Liu, Cui Ge, Bingjun Tang, Hongfang Yin, Xuedong Wang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Molecular subtyping of bladder cancer (BCa) using bulk transcriptomic data is hindered by tumor microenvironment (TME) contamination and intra-tumoral heterogeneity. These factors also contribute to variable responses to immune checkpoint inhibitor (ICI) therapy. Single-cell RNA sequencing offers a high-resolution approach to overcome these limitations by enabling tumor-specific analysis and improving therapeutic target identification.


MATERIALS AND METHODS
We constructed a large multicenter single-cell atlas of BCa by integrating nine cohorts comprising 123 samples and 842 127 cells. Tumor epithelial cells (n = 265 962) were identified via copy number variation (CNV) analysis using inferCNV. Pseudobulk RNA profiles were generated to simulate conventional bulk samples (Pseudobulkall) and purified tumor profiles (Pseudobulktumor), enabling systematic evaluation of bulk-based classifiers. Inter-tumoral and intra-tumoral heterogeneity programs were defined based on transcriptional profiles and validated through in vitro and in vivo experiments. Additionally, we cross-analyzed five immunotherapy datasets encompassing 13 immune response phenotypes to identify molecular features associated with therapeutic resistance and sensitivity.


RESULTS
Unsupervised clustering identified 11 major cell types and 55 non-epithelial subtypes, with TME heterogeneity driven by tissue origin and malignancy status. Existing molecular classifiers UNC (University of North Carolina at Chapel Hill) showed compromised concordance between Pseudobulkall and Pseudobulktumor, particularly in stroma-rich samples. We defined three inter-tumoral (GEinter2-4) subtypes with distinct CNV and transcription factor signatures that remained stable even at low tumor purity. Seven intra-tumoral (GEintra) programs captured key biological states, including an EMT/Stem-like module associated with immunosuppression and ICI resistance. Mechanistically, TGF-β1 induced expression of EMT/Stem-like genes (e.g., PTHLH), while Pthlh knockdown synergized with anti-PD-L1 therapy in murine models by enhancing T cell cytotoxicity without altering T cell infiltration.


CONCLUSIONS
This study provides a robust single-cell framework for resolving inter- and intra-tumoral heterogeneity in BCa, overcoming limitations of bulk-based subtyping. The TGF-β1-PTHLH axis contributes to ICI resistance, positioning PTHLH as a promising therapeutic target. These findings offer new insights for improving immunotherapeutic strategies in BCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d5eb03c1d8a0a6927854d1055fb19018c50409" target='_blank'>
              Integrated multicenter single-cell atlas of bladder cancer reveals tumor heterogeneity associated with immunotherapy.
              </a>
            </td>
          <td>
            Ziwei Wang, Wei He, Maoyu Wang, Meimian Hua, Yidie Ying, Chen Zhang, Qing Chen, Yue Wang, Xufeng Yu, Yi Wang, Huiqing Wang, Zhensheng Zhang, Shuxiong Zeng, Chuanliang Xu
          </td>
          <td>2026-02-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2b8ad851f4ed7ac71d3b37aff3d0eab8d4c1da" target='_blank'>
              Connecting spatial regions to clinical phenotypes by transferring knowledge from bulk patient data
              </a>
            </td>
          <td>
            Bayarbaatar Amgalan, Ermin Hodzic, Ziynet Nesibe Kesimoglu, M.G. Hirsch, John D. Bridgers, Jan Hoinka, David L Levens, Chi-Ping Day, Teresa M. Przytycka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Terminally exhausted CD8+ T cells (Ttex) are emerging as clinically relevant immune subsets across solid tumors, marked by sustained inhibitory receptor expression, loss of TCF1, and limited proliferative capacity. Once considered functionally inert, Ttex are now recognized for their residual cytotoxic potential and strong associations with tumor immunogenicity, including microsatellite instability (MSI), high tumor mutational burden (TMB), and neoantigen load. Importantly, the prognostic significance of Ttex is highly tumor-context-dependent, shaped by stromal architecture, mutational burden, and progenitor Tpex availability. This review examines the biology, spatial localization, and prognostic value of Ttex, highlighting the Ttex/CD8+ ratio as a promising biomarker in cancers such as colorectal, lung, and esophageal carcinoma. We summarize recent advances in multiplex imaging, digital pathology, and AI-driven quantification that support the clinical integration of Ttex assessment. In addition, we discuss emerging therapeutic strategies targeting Ttex through immune checkpoint combinations, thymocyte selection-associated high mobility group box protein (TOX) and circRNA-mediated reprogramming, and exhaustion-resistant T cell engineering. Finally, we outline translational priorities including assay harmonization, functional validation, and longitudinal profiling to advance Ttex-based precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af1a0b32b60b15c52819ffc31235e60745d2593b" target='_blank'>
              Terminally exhausted CD8+ T cells in solid tumors: biology, biomarker potential and translational tools for precision oncology
              </a>
            </td>
          <td>
            Xuejun Guo, Shuhan Ma, Jingwen Wang, Yilin Fu, Wenxue Ma
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) enables high-resolution transcriptomic analysis but loses spatial context due to tissue dissociation, thereby limiting insights into spatially regulated cell–cell interactions critical for tissue function and disease, including cancer. Consequently, developing approaches capable of extracting spatially relevant intercellular interaction information from scRNA-seq data is of substantial importance for elucidating mechanisms underlying disease initiation and progression. Here, we present DeepPNCC, a novel deep learning framework that reconstructs pseudo-spatial cell-cell interaction networks (PSCCIs) from scRNA-seq data by leveraging latent spatial cues preserved in undissociated cell aggregates. Built on a variational graph autoencoder (VGAE) enhanced with adversarial regularization, DeepPNCC uniquely integrates local adjacency matrices derived from multiplet data, without relying on prior knowledge such as ligand-receptor pairs, to infer global, spatially informed interaction landscapes. DeepPNCC is available as an open-source Python package. DeepPNCC outperforms existing methods in recovering interactions aligned with spatial transcriptomics across mouse brain and breast cancer datasets. In breast cancer, the inferred cell-cell interaction network reveals a closed-loop signaling axis in triple-negative breast cancer, in which cancer-associated fibroblasts (CAFs) and perivascular-like cells (PVL) promote angiogenesis through the ADM-CALCRL-STAT1-CD40/CCL2/ICAM1 pathway. These findings provide important clues for elucidating the regulatory mechanisms of the tumor microenvironment. DeepPNCC provides an efficient tool for spatially informed analysis of scRNA-seq data, enabling the reconstruction of pseudo-spatial cell–cell interaction networks from datasets lacking explicit spatial information. This approach expands the spatial analytical scope of scRNA-seq and facilitates mechanistic dissection of cellular ecosystems in both health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee882b8d93f6b445af77befa6248cb5501417766" target='_blank'>
              DeepPNCC: reconstructing pseudo-spatial cell-cell interaction landscapes from single-cell data to decipher breast cancer pathogenesis
              </a>
            </td>
          <td>
            Xu-hua Li, Xiao-Ling Gao, Deng-Hui Guo, Jie Shen, Si-Rui Li, Na Wang, Xiao-ling Wen, Jiang-Ying Liang, Hong-jiu Wang, Zhen-Zhen Wang
          </td>
          <td>2025-12-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma is a lethal primary brain tumor with poor prognosis. Tumor cells exhibit substantial phenotypic variation, complicating treatment. As functional diversity is driven by underlying transcriptional states, characterizing tumor cell gene expression is essential for understanding tumor biology and therapeutic response. The GL261 tumor cell line is a common pre-clinical model system for investigating glioblastoma pathobiology. However, global gene expression patterns in this model are unknown. Here we describe the use of single-cell RNA sequencing (scRNA-Seq) to investigate transcriptional profiles of 5764 adherent and 4951 neurosphere GL261 cells, generating 133,442,221 sequenced reads. Following Principal Component Analysis (PCA) for dimensionality reduction, we applied Uniform Manifold Approximation and Projection (UMAP) to visualize transcriptionally distinct subpopulations (clusters) of GL261 cells grown adherently or as neurospheres. Highly expressed and differentially expressed genes were identified. Because the neurosphere phenotype is known to be more tumorigenic, we further examined differentially expressed genes with gene ontology expression analysis. We found that upregulated genes in neurosphere cells are associated with angiogenesis, cell adhesion, and cell signaling pathways. In addition, we specifically examined gene expression patterns of matrix metalloproteinases and purinergic receptors, glioblastoma drug targets known to be important for promoting tumor infiltration into adjacent healthy tissue. We found that P2RX7, MMP15 and MMP16 are upregulated in neurosphere cells, indicating a potential role for these genes in tumor formation. Together these results reveal global transcriptional profiles of GL261 cells, establish a resource for further scRNA-Seq-based analyses, and give insight into gene expression changes relevant to glioblastoma tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b536ff0acd69dbca3af6bd3122c752115f69d4a" target='_blank'>
              Single-Cell Transcriptomic Profiling of GL261 Glioblastoma Cells Reveals Gene Expression Signatures Underlying Tumorigenicity
              </a>
            </td>
          <td>
            Colton E Troxel, Ruby A Olvera, Emily A Freko, Suely S C Soeiro, Thandiswa Mdluli, Richard L Daniels
          </td>
          <td>2025-12-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian M. Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d7c1910610648515431116d59c0a3a122ecb43" target='_blank'>
              Multimodal Data Fusion Reveals Morpho-Genetic Variations in Human Cortical Neurons Associated with Tumor Infiltration
              </a>
            </td>
          <td>
            Yufeng Liu, Zhixi Yun, Lingli Zhang, Wen Ye, Kaifeng Chen, Xiefeng Wang, Mengzhu Ou, Jing Rong, Xiaomin Yang, Lei Mao, Liang Chen, Nan Ji, Yongping You, Chiyuan Ma, Ying Mao, Junxia Zhang, Liwei Zhang, Hanchuan Peng
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Multiplexed immunofluorescence (mIF) imaging has revolutionized the study of cellular interactions within tissue microenvironments, enabling complex pattern analysis critical to understanding disease biology. However, current analytical methods assume uniform cellular patterns across tissues, overlooking the spatial heterogeneity that characterizes tumor microenvironments. Here, we introduce ISPat (Informed Spatially aware Patterns), a fully Bayesian framework that identifies both shared and region-specific interaction patterns while integrating domain knowledge to enhance spatial pattern estimation. ISPat models spatial cellular densities through kernel density estimation, then constructs interaction networks from precision matrices that capture conditional dependencies between cell types while controlling for confounding effects. The resulting networks reveal direct cellular relationships, with non-zero precision matrix entries indicating significant interactions. We applied ISPat to analyze 119 pancreatic ductal adenocarcinoma (PDAC) and 53 intraductal papillary mucinous neoplasm (IPMN) patients, partitioning tissues into five regions based on tumor intensity gradients. Our analysis revealed fundamentally distinct immune architectures: PDAC maintains a rigid, stable immunosuppressive microenvironment across tumor heterogeneity gradients, whereas IPMN exhibits dynamic spatial remodeling with marked regional variability. Critically, we identified multiple ligand-receptor (LR) interactions that consistently differ between disease conditions specifically in intermediate tumor intensity regions, while extreme conditions showed no significant differences. These include interactions spanning multiple functional axes of anti-tumor immunity: antigen presentation and T cell activation (APC\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL, THelper\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC, Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC), effector function and tumor cell killing (Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL, CTL\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg), and immune regulation (Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg, Treg\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}APC, THelper\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}Treg). Notably, the APC\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL interaction, fundamental for adaptive immunity activation, differs significantly in high tumor density regions, alongside Epithelial\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\leftrightarrow$$\end{document}CTL interactions critical for direct tumor elimination. These spatially resolved signatures provide quantitative evidence for distinct immune evasion mechanisms and represent promising biomarker candidates for disease classification and risk stratification. Through simulation studies, we demonstrated ISPat’s accuracy in pattern recovery and its computational efficiency, achieving 8-10 fold speedup over comparable methods through variational Bayesian inference. The framework exhibits robust scalability and handles naturally occurring partition size imbalances, making it well-suited for analyzing heterogeneous tissues. Our findings demonstrate that spatial context fundamentally shapes cellular interactions in pancreatic cancer, with critical implications for understanding immune evasion mechanisms and developing spatially informed therapeutic strategies. The identification of differential interactions across antigen presentation, effector function, and immune regulation pathways suggests that therapeutic interventions must address multiple axes of immune dysfunction rather than single targets. ISPat provides a generalizable framework for spatial analysis applicable to emerging technologies, enabling precision oncology approaches guided by spatially resolved biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba722b3f6a63490b9a491569ba714e8cdd49aa3e" target='_blank'>
              Informed spatially aware patterns for multiplexed immunofluorescence data
              </a>
            </td>
          <td>
            Sagnik Bhadury, Michele Peruzzi, Satwik Acharyya, Joel Eliason, M. P. di Magliano, Timothy L. Frankel, V. Ravikumar, S. Krishnan, Arvind Rao
          </td>
          <td>2026-01-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ad1efaa62d98c7f16dd57c71f8980e8844d4c9" target='_blank'>
              Tumor microenvironment signature associated with morphology in systemic ALK-positive anaplastic large cell lymphoma
              </a>
            </td>
          <td>
            Chloé Bessiere, Loélia Babin, Sandra Dailhau, P. Gaulard, Stéphane Pyronnet, Fabienne Meggetto, Laurence Lamant
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell studies have significantly advanced our understanding of the transcriptional and functional heterogeneity in cancers. Recent studies have identified distinct states of cancer, immune, and stromal cells in the tumor microenvironment (TME), with growing evidence highlighting their clinical significance and therapeutic potential. Here, we present CancerSEA-X, an expanded version of CancerSEA that offers a comprehensive atlas of TME cell states. CancerSEA-X integrates 25 cancer cell states, 105 immune cell states, and 26 stromal cell states from systematically curated publications. Combining 239 single-cell datasets across 32 cancer types, encompassing over 9 million cells from 2120 patients, CancerSEA-X provides functional activity spectra and cancer-specific gene associations for these 156 cell states. These cell state-gene relationships were mapped onto networks, providing a systematic view of the TME. To improve usability, we redesigned the user interface to feature cell state characterization, state-gene correlation analysis, and interactive visualization of cell state-gene networks, enabling researchers to comprehensively explore these states and their functional relevance. Overall, CancerSEA-X serves as a valuable platform for investigating TME cell states, deepening our understanding of cancer heterogeneity, and potentially advancing the design of more effective clinical therapies. CancerSEA-X is freely available at http://biocc.hrbmu.edu.cn/CancerState.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc11040a019ecb4b6c2065cd731c8ff500837ec3" target='_blank'>
              CancerSEA-X: A Single-cell Resource for Tumor Microenvironment Cell States Across over 30 Cancer Types.
              </a>
            </td>
          <td>
            Lantian Deng, Wei Liu, Jinyuan Xu, Bo Zhang, Shengyuan He, Kun Liu, Xinxin Zhang, Huating Yuan, Fei Quan, Yun Xiao
          </td>
          <td>2026-01-16</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Immunological intratumor heterogeneity (ImTH) describes the variability in the types, spatial distribution, and functional states of immune cells within tumors. While evidence suggests that ImTH influences tumor progression and therapeutic response, few studies have provided a quantitative characterization of ImTH. Here, we present Scoring Immunological Intratumor Heterogeneity (ScImTH), a novel algorithm that quantifies ImTH by calculating the Shannon entropy of immune cell type proportions within the tumor microenvironment. Using bulk, single-cell, and spatial transcriptomic datasets, we show that reduced ScImTH scores are associated with unfavorable survival outcomes, tumor progression-related molecular and phenotypic features, immunosuppressive states, and resistance to immunotherapy across multiple cancer types. Compared with existing measures of tumor immunity, such as immune score and B-cell receptor diversity, the ScImTH score demonstrated stronger and more consistent associations with clinicopathological features. Notably, the ScImTH score outperformed established biomarkers, including tumor mutational burden and PD-L1 expression, in predicting immunotherapy response. These findings highlight the clinical potential of the ScImTH score as a biomarker for cancer prognosis and immunotherapy stratification. More broadly, our results support the hypothesis that loss of immune diversity is a hallmark of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) exhibits highly variable clinical behavior, ranging from indolent growth to aggressive lymph node (LN) metastasis. However, the specific molecular mechanisms involved in PTC evolution, driven by metabolic reprogramming and tumor-microenvironmental interactions, remain poorly understood. This study aims to elucidate the spatial metabolic and transcriptional mechanisms underlying PTC tumorigenesis and LN metastasis at multiple molecular levels. An integrated spatial multi-omics strategy combining spatial metabolomics and spatial transcriptomics was employed to map the distribution of metabolites and gene expression in heterogeneous PTC tissues. Metabolite profiles and transcriptomic data were analyzed to identify dysregulated pathways and to uncover the key molecular features contributing to PTC tumorigenesis and LN metastasis. Genes related to metastasis-driving metabolites were validated using the TCGA dataset, and further substantiated by zebrafish xenograft models. Spatial mapping revealed unique metabolic and transcriptional signatures across heterogeneous regions of PTC tissue. Key metabolic pathways, such as the arginine–polyamine axis, glycolysis and lipid metabolism, were prominently dysregulated in cancer regions. The surrounding stroma also exhibited metabolic adaptations, like polyamine recycling and glucose storage, which supported tumor growth via metabolic crosstalk. Most importantly, greater spatial heterogeneity, spatial evolutionary routes and 5 potential metastasis-driving metabolites, including FA (22:6), PC (36:4), PC (34:1), N-acetylaspartate and ascorbic acid, were discovered in the primary PTC tissue with LN metastasis. Knockdown of key genes NAT8L and SVCT-2 obviously decreased the metastatic ability of PTC cells. The TCGA database further confirmed that high expression of 10 pro-metastatic metabolite-related genes was significantly associated with poorer prognosis in PTC patients. This integrated spatial multi-omics approach provides novel insights into the molecular mechanisms underlying PTC evolution, potentially guiding the development of more effective diagnostic and therapeutic strategies for PTC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4ee960363a0cd08e85ab4a85deecb06d245c95" target='_blank'>
              Integrated spatial metabolomics and transcriptomics reveal the molecular landscape of papillary thyroid cancer and its lymph node metastasis
              </a>
            </td>
          <td>
            Kening Li, Zongfu Pan, Wenqiao Chang, Yingying Gong, Tong Xu, Bin Lu, Jiuming He, Minghua Ge, Zhuo Tan, Ping Huang
          </td>
          <td>2025-12-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) exhibits pronounced intratumoral heterogeneity and a complex immune microenvironment, which together limit therapeutic efficacy. Identifying actionable biomarkers and mechanisms of immune modulation remains critical for improving patient outcomes. Method We integrated single-cell RNA sequencing data with bulk transcriptomic datasets to comprehensively characterize tumor cell heterogeneity and immune landscape features in HCC. Associations between RDH16 expression and clinicopathological characteristics were evaluated, and in vitro functional assays were conducted. Mendelian randomization analyses were performed to assess causal relationships. Results RDH16 expression was significantly reduced in HCC tissues compared with non-tumor liver tissues and was associated with vascular invasion, elevated alpha-fetoprotein levels, poor tumor differentiation, and advanced T stage. Higher RDH16 expression was consistently associated with improved overall prognosis. Functional assays indicated that RDH16 did not directly affect tumor cell proliferation, migration, or invasion. Notably, RDH16 expression was inversely correlated with infiltration of CD163⁺ M2 macrophages, suggesting a potential immunomodulatory role in the tumor microenvironment. Mendelian randomization analyses further supported a protective effect of higher RDH16 expression against HCC risk. Discussion These findings identify RDH16 as an immune-modulating biomarker in HCC, highlighting its potential role in shaping the tumor immune microenvironment and suggesting new avenues for personalized immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444ab6526b7dd129322dafa940c094f836b618fb" target='_blank'>
              Single-cell transcriptomics reveals a novel mechanism of RDH16 regulating immune infiltration in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhenzhen Zhang, Rui Fan, Jing Ma, Dehui Li, Yan Jiang, Fahui Liu, Qiming Gong
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Breast cancer is genetically and histologically heterogenous, and is influenced by a variety of factors, including the tumor microenvironment (TME). The PAM50 subtypes; Luminal A, Luminal B, Normal-like, Basal-like and Her2-enriched, are associated with different tumor phenotypes and overall survival. The quantity and quality of immune cell infiltration in breast tumors play a key role in cancer development and progression and are associated with survival and treatment response. We used multiplex immunohistochemistry, single-cell RNA sequencing, and two deconvolution algorithms to explore the immune landscape across PAM50 subtypes. Immunostaining of CD3+ T cells, tryptase+ mast cells, CD20+ B cells, CD68+ CD163+ macrophages, and CD66b+ granulocytes revealed marked differences in tumor immune infiltrates according to breast cancer subtypes. Luminal tumors were relatively deprived of T cells and B cells, while exhibiting sparse to moderate amounts of macrophage infiltration. In contrast, Her2-enriched tumors exhibited a moderate immune presence, with sparse to moderate T cell infiltration and moderate infiltration of B cells and macrophages. At the other end of the spectrum, Basal-like tumors stood out for their strikingly rich immune environment and are heavily infiltrated by T cells, B cells, and macrophages. The results from the single-cell and deconvolution analyses confirmed subtype-specific immune microenvironments, which also allowed us to observe increased levels of natural killer (NK) and CD8+ T cells in Her2-enriched and Basal-like subtypes. In conclusion, our findings demonstrate significant differences in the immune tumor microenvironment between the established breast cancer molecular subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c99c23b4dfb7da23c9b0f3a691c05aec9a1b68" target='_blank'>
              Multi-platform analysis of the tumor immune microenvironment associated with breast cancer subtypes
              </a>
            </td>
          <td>
            Lilly Anne Torland, Jürgen Geisler, Y. Azimzade, Astri Frafjord, Hogne Røed Nilsen, Linn Buer, J. Lømo, Ø. Garred, Andliena Tahiri, M. Lyngra, Inger Øynebråten, Alexandre Corthay, Kristine Sahlberg, Vessela N. Kristensen, Xavier Tekpli
          </td>
          <td>2026-01-13</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal solid tumors, largely due to its intricate and immunosuppressive tumor microenvironment (TME). While single-cell sequencing technologies have begun to unravel the cellular heterogeneity of PDAC, a comprehensive understanding of how genetic determinants influence and are influenced by the TME is still lacking. To bridge this knowledge gap, our study employs an integrated multi-omics approach, incorporating single-cell transcriptomics, genomics, and proteomics, complemented by computational biology and machine learning. We aimed to delineate the core molecular drivers of PDAC pathogenesis, with subsequent
 in vitro
 functional validation focusing on the role of UNC93B1 in malignant phenotypes. The ultimate goal of this research is to inform the development of precise therapeutic strategies to enhance patient survival and quality of life.





 We assembled a comprehensive multi-omics dataset, including single-cell RNA-seq data from 22 PDAC samples (GSE154778, GSE212966), bulk transcriptomic cohorts (GSE28735, GSE62452), survival data from the TCGA-PAAD project (n=172), spatial transcriptomics, and genome-wide association study data (bbj-a-140, n=196,187). The single-cell data were processed using Seurat v5, which involved rigorous quality control, batch effect correction with Harmony, unsupervised clustering, and cell type annotation to characterize TME heterogeneity. Genetic susceptibility was mapped onto single-cell data using scPagwas to calculate trait-regulated scores (TRS) and identify trait-associated genes. Co-expression networks were constructed via high-diversity WGCNA (hdWGCNA), and key candidate genes were refined through survival analysis and a machine learning framework integrating LASSO regression, Random Forest, and Support Vector Machine algorithms. The functional role of the pivotal gene, UNC93B1, was systematically investigated through Gene Set Variation Analysis (GSVA), pseudotime trajectory inference (Monocle2), and cell-cell communication analysis (CellChat).
 In vitro
 validation was performed using four PDAC cell lines (PANC-1, BxPC-3, Capan-1, SW1990). Following qPCR confirmation of high UNC93B1 expression, a stable knockdown model (sh-UNC93B1) was generated in Capan-1 cells. Functional consequences were assessed using CCK-8, wound healing, transwell and colony formation assays. A subcutaneous xenograft model was established to evaluate tumor growth
 in vivo
 . Mechanistic insights were gained through flow cytometry for cell cycle analysis and molecular profiling of the cGAS-STING pathway, senescence markers (e.g., p16^INK4a), and epithelial-mesenchymal transition (EMT)-related genes.





 Single-cell transcriptomic profiling delineated nine distinct cell populations within the PDAC TME. hdWGCNA identified three gene modules (8, 11, 16) positively associated with tumorigenesis. The intersection of these modules with differentially expressed genes yielded 320 candidates, which were subsequently filtered to 61 genes significantly linked to patient prognosis (P < 0.05) via Cox regression. Cross-validation across machine learning models and scPagwas analysis converged on UNC93B1 as the sole overlapping gene with consistent diagnostic and prognostic relevance. UNC93B1 was robustly upregulated in tumor tissues across independent datasets (TCGAxGTEx, bulk RNA-seq), a finding corroborated at the protein level by HPA and CPTAC data (P < 0.01). Its expression positively correlated with higher pathological grade and was spatially enriched within tumor regions. Functional enrichment analysis (GSVA) suggested that UNC93B1 is involved in the suppression of the cGAS-STING signaling axis. Pseudotime analysis indicated that UNC93B1 expression escalates along tumor progression trajectories. CellChat suggested strengthened intercellular communication networks in UNC93B1-high cells, particularly modulated by the cGAS-STING pathway.
 In vitro
 , UNC93B1 knockdown in Capan-1 cells significantly attenuated proliferative capacity (22.3% reduction in OD450 at 72h, P < 0.05), migratory ability (29.6% reduction in wound closure, P < 0.05), and clonogenic survival (342 fewer colonies, P < 0.01). Mechanistically, sh-UNC93B1 cells exhibited G1/S phase arrest (8.9% increase, P < 0.05), activation of the STING/IFN-β/CXCL10 cascade, elevated p16^INK4a expression, and a reversal of EMT, evidenced by downregulation of VIM and upregulation of CDH1. Consistently,
 in vivo
 xenograft experiments demonstrated that UNC93B1 silencing markedly impeded tumor growth, concomitant with reduced UNC93B1 protein and enhanced STING pathway activation. Critically, the tumor-suppressive phenotypes induced by UNC93B1 knockdown, including the inhibition of proliferation, migration, and clonogenicity, were largely reversed upon treatment with the selective STING inhibitor H-151, confirming that the observed functional consequences are causally mediated through the activation of the cGAS-STING pathway.




 By integrating multi-omics data, including GWAS and spatial transcriptomics, this study systematically defines a pivotal role for UNC93B1 in PDAC progression. Our findings demonstrate that UNC93B1 is associated with an immunosuppressive TME and facilitates metastatic spread, potentially through inhibiting the cGAS-STING-mediated innate immunity pathway. The strong correlation between UNC93B1 overexpression and adverse clinical outcomes underscores its potential as a dual diagnostic biomarker and therapeutic target. This work not only provides a mechanistic foundation for novel precision immunotherapies in PDAC but also establishes a robust methodological paradigm for multi-omics-driven discovery in oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bb10353febe80e8ef377d1403743c79b95316f" target='_blank'>
              UNC93B1 promotes pancreatic cancer progression through modulation of cGAS–STING signaling
              </a>
            </td>
          <td>
            Hao Yang, Yukun Li, Jing Su, Haiyan Zhang, Wenxin Zhu, Kanger Shen, Wei Xu
          </td>
          <td>2026-02-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo- and immune-resistance. Integrating single-cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype-specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone-selective delivery systems and biomarker-driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single-cell, spatial and functional data to map bone-sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker-guided therapeutic strategies that can be translated into treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review)
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S Vargas Solivan, Kinjal Bhadresha, Sumeyra Kartal, Samantha Nichols, Kanak Parmar, Joseph Twohig, Tri M Truong, C. Chau, Kathleen Kelly, W. Figg, Anish Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lactate accumulation is a central feature of tumor metabolic reprogramming, yet its spatial and cell-type-specific effects in cancer, such as lung adenocarcinoma (LUAD), remain poorly defined. We integrated single-cell transcriptomics, spatial transcriptomics, spatial metabolomics, and immunofluorescence with TCGA survival data and machine-learning models. High-lactate tumors exhibited increased epithelial and fibroblast abundances, whereas T/NK cells and monocytes/macrophages were enriched in low-lactate samples. Spatial metabolomics revealed cell-type–restricted lactate and pyruvate distributions, with endothelial cells showing minimal lactate accumulation. Endothelial subclusters in high-lactate tissues displayed angiogenic and stress-response signatures and were strongly associated with poor prognosis. Multiple machine-learning frameworks—including random forest, elastic-net regression, SVM, ANN, and decision-tree models—consistently identified endothelial and fibroblast programs as key determinants of high-lactate states and adverse clinical outcomes. Collectively, our multi-omics spatial profiling demonstrates that lactate reshapes the LUAD microenvironment by driving angiogenesis, immune suppression, and prognostic stratification, highlighting lactate-centered pathways as promising therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f51e7895cef79efeacf0e1e7ab108d79743ff" target='_blank'>
              Machine learning-enabled spatial multi-omics uncovers lactate-driven targets and tumor microenvironmental reprogramming in cancer
              </a>
            </td>
          <td>
            Yingzheng Tan, Wenliang Tan, Yanchao Liang, Yunzhu Long, Shuanghua Chen, Qihao Hu, Yangjing Ou, Jingli Fu, Huan Chen, Fangyuan Ren, Jun Ye, Qing Zhou, Sheng Li, Xiaojin He, Qianqian Wang, Yueming Shen, Haiyuan Lu, Daichao Wu, An-bo Gao, Xun Chen, Yukun Li
          </td>
          <td>2025-12-30</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Advances in single-cell and spatial transcriptomic technologies have transformed tumor ecosystem profiling at cellular resolution. However, large scale studies on patient cohorts continue to rely on bulk transcriptomic data, where variation in tumor purity obscures tumor-intrinsic transcriptional signals and constrains downstream discovery. Many deconvolution methods report strong performance on synthetic bulk mixtures but fail to generalize to real patient cohorts because of unmodeled biological and technical variation. Here, we introduce TwinPurify, a representation learning framework that adapts the Barlow Twins self-supervised objective, representing a fundamental departure from the deconvolution paradigm. Rather than resolving the bulk mixture into discrete cell-type fractions, TwinPurify instead learns continuous, high-dimensional tumor embeddings by leveraging adjacent-normal profiles within the same cohort as"background"guidance, enabling the disentanglement of tumor-specific signals without relying on any external reference. Benchmarked against multiple large cancer cohorts across RNA-seq and microarray platforms, TwinPurify outperforms conventional representation learning baselines like auto-encoders in recovering tumor-intrinsic and immune signals. The purified embeddings improve molecular subtype and grade classification, enhance survival model concordance, and uncover biologically meaningful pathway activities compared to raw bulk profiles. By providing a transferable framework for decontaminating bulk transcriptomics, TwinPurify extends the utility of existing clinical datasets for molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77029e4cfdc9e970b8e603422677220699d06e9e" target='_blank'>
              TwinPurify: Purifying gene expression data to reveal tumor-intrinsic transcriptional programs via self-supervised learning
              </a>
            </td>
          <td>
            Zhiwei Zheng, Kevin Bryson
          </td>
          <td>2026-01-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef7edb3493fb7ab6d3516e83b3f06b2dd845210" target='_blank'>
              VAETracer: Mutation-Guided Lineage Reconstruction and Generational State Inference from scRNA-seq
              </a>
            </td>
          <td>
            Linnuo Pan, Kaiyu Wang, Luonan Chen
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterised by remarkable cellular heterogeneity, which emerges early from the interplay of oncogenic KRAS signalling and inflammatory injury. However, the transcriptional, metabolic and functional properties of these pre-malignant cell states that initiate and drive PDAC progression remain elusive. Objective This study aimed to identify and functionally characterise the critical premalignant cell states that arise from this heterogeneity, to define novel biomarkers and targets for early intervention. Design Public and in-house scRNA-seq data of pancreatic tumour models were analysed to identify key subpopulations in early cellular heterogeneity. Genetic perturbation in KrasG12D-driven models was performed to assess functional impact. Mechanistic studies used TurboID proximity proteomics, epigenetic profiling and metabolic assays. Clinical relevance was validated in human PDAC cohorts. Results We identified LY6D as a marker of a distinct, gastric-like cell state that emerges early and persists throughout tumourigenesis. The LY6D+ population exhibits conserved stemness and a unique, pan-stage dependency on oxidative phosphorylation (OXPHOS). Genetic ablation of Ly6d specifically impaired the gastric lineage and delayed tumourigenesis, while its overexpression enhanced tumourigenic and metastatic potential. Mechanistically, the glycosylphosphatidylinositol (GPI)-anchored LY6D protein scaffolds a lipid raft-associated kinase network that drives FOSL1-dependent epigenetic-transcriptional reprogramming. In human PDAC, LY6D+ cells harbour stemness and Epithelial-Mesenchymal Transition (EMT) signatures, and high LY6D expression is an independent prognostic marker of poor survival. Conclusion Our work defines the LY6D+ gastric-like cell state as a key driver linking early pre-malignant heterogeneity to PDAC initiation and progression. LY6D represents a pan-stage therapeutic target and a candidate biomarker for early detection and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42670138ad6a4407b9d3c018c7d78a840fd287d" target='_blank'>
              LY6D identifies persistent stem-like cells driving pancreatic tumourigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Xian Wang, Yingying Tang, Shixin Meng, Zhengyan Zhang, Ping Lu, Junyi Xu, Feier Yu, Xueni Wang, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2026-01-16</td>
          <td>Gut</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Hepatocellular carcinoma (HCC) is a leading cause of cancer-related mortality worldwide, largely due to late-stage diagnosis and the limited sensitivity of current biomarkers such as α-fetoprotein (AFP). Early detection requires molecularly defined targets that capture the initial steps of malignant transformation. Single-cell RNA sequencing (scRNA-seq) offers high-resolution insight into tumor heterogeneity and lineage progression to enable the identification of early biomarkers. This study aimed apply scRNA-seq analysis to detect clinically important molecular patterns that define the early stages of malignant transformation in HCC and facilitate the diagnosis of small or ambiguous lesions. Methods Two independent scRNA-seq datasets (GSE149614 and GSE189903) comprising non-tumor and HCC tissues were analyzed. Following batch correction and clustering, hepatocyte subpopulations were characterized by differential expression, pseudotime, and CytoTRACE analyses to reconstruct the trajectory from normal to malignant states. High-dimensional weighted gene co-expression network analysis (hdWGCNA) was used to identify stage-associated modules, while CellChat and protein-protein interaction analyses delineated intercellular signaling networks. Target expression was validated in paired human liver specimens using quantitative immunofluorescence. Results scRNA-seq revealed a continuum of hepatocyte states characterized by progressive stemness and oncogenic pathway activation (MYC, E2F, G2M). Module Hep-M20 exhibited the strongest correlation with tumor stage and identified BSG/CD147 as a central hub gene with monotonic upregulation along pseudotime and strong correlation with stemness potential. CellChat analysis uncovered a cyclophilin (PPIA/PPIB)-dependent tumor-stroma signaling axis that positions BSG/CD147 as the key mediator for intercellular communication between tumor hepatocytes, fibroblasts and T cells. Ex vivo validation confirmed significantly higher BSG/CD147 protein expression in HCC versus background liver (P = 2.9×10⁻ 11 ) with excellent diagnostic accuracy (AUC = 0.93–0.96; sensitivity 86–87%; specificity 93–97%) including in lesions < 2 cm that are frequently indeterminate on conventional imaging. Conclusions This study establishes BSG/CD147 upregulation as an early molecular event in hepatocarcinogenesis that integrates hepatocyte dedifferentiation, microenvironmental signaling, and tumor progression. Strong and specific expression in small lesions < 2 cm underscores potential as a precision biomarker and imaging target for early HCC detection, risk stratification, and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b69381a1a33432cf5fe739884155cd3bedeaf6" target='_blank'>
              Multimodal single-cell network analysis uncovers BSG/CD147 as an early biomarker and signaling hub in hepatocellular carcinoma
              </a>
            </td>
          <td>
            T. D. Wang, , Shuo Feng, Hui Jiang, T. D. Wang, Sangeeta Jaiswal
          </td>
          <td>2026-01-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast tumors have traditionally been categorized into four subtypes. Recent single-cell advancements, however, reveal that tumors can contain multiple subtypes, reflecting intrinsic heterogeneity. Relatedly, whilst the crucial role of cell cycle regulation in cancer development is widely acknowledged, the impact of cell cycle phases on oncogenic signaling within these subtypes remains unresolved. Here, we examine differences in transcriptomic hallmarks across and within cell cycle phases and breast cancer subtypes in two large single-cell cohorts. We show that accounting for cell cycle phases reveals a broader pool of hallmark pathways while preserving those identified when phases are ignored. Moreover, we pinpoint FDA approved drugs that can target hallmarks within specific phase-subtype combinations. This study highlights the critical role of cell cycle phases in advancing our understanding of cancer biology. The distinct mechanisms observed here suggest that targeting these hallmarks may offer a more precise and effective strategy for patient treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680e0867e66e372634861e3e8e8f0639de30e291" target='_blank'>
              Computational decoding of cell-cycle phase effects on cancer hallmarks across breast cancer subtypes
              </a>
            </td>
          <td>
            Miguel Castresana-Aguirre, A. Matikas, Linda S Lindström, N. Tobin
          </td>
          <td>2025-12-24</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is characterized by heterogeneous and plastic cellular populations that adopt transcriptional programs shaped by genetic alterations and microenvironmental cues. Recent studies have identified at least four partially inconvertible cell states-astrocytic-like, neural progenitor-like, oligodendrocyte progenitor-like, and mesenchymal-like-that represent aberrant developmental programs. Expanded analysis further reveals hybrid and intermediate states that form continuous transcriptional and metabolic gradients. These states exhibit spatial organization, assembling into three distinct microanatomical niches: a perivascular niche enriched with mesenchymal-like and oligodendrocyte progenitor-like cells, a hypoxic niche harboring quiescent and stressed cells of all states, and an invasive niche containing astrocyte-like or proneural populations. Niches continuously remodel as cell states transition, migrate, and re-establish new programming in response to angiogenesis, hypoxia, immune infiltration, and neuronal activity. This interplay between states and the microenvironment generates a self-renewing spatial architecture, maintaining expansion at the edge and protection within the core. This review integrates single-cell, single-nucleus, and spatial studies to describe a microenvironmental-driven model of cell state organization. Understanding how these multiscale drives converge to generate a continuum of cell state identities may reveal strategies to disrupt the spatial architecture underlying glioblastoma plasticity and recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c762648021d36dbbe67c2cb45c25c28113bb3f2d" target='_blank'>
              Tumor Microenvironment Shapes the Spatial Organization of Glioblastoma Cell States.
              </a>
            </td>
          <td>
            P. Prakash, James Trippett, Cameron Ehsan, Joseph Namkung, Meeki K. Lad, M. Aghi
          </td>
          <td>2026-01-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Immune cells critically influence pituitary neuroendocrine tumor (PitNET) progression and therapeutic responses. However, their composition and functional dynamics remain unclear. This study aimed to delineate the immune heterogeneity and intercellular communication networks within PitNETs.


METHODS
This study conducted single-cell RNA sequencing (scRNA-seq) on 22 fresh PitNET samples obtained from surgical patients at Beijing Tiantan Hospital between September 2023 and January 2024, with classification based on the expression of key transcription factors: pituitary-specific transcription factor 1 (PIT1), steroidogenic factor 1 (SF1), and T-box transcription factor 19 (TPIT). Following cell type identification, cell-cell communication analysis revealed specific intercellular interactions, which were validated by multiplex immunohistochemistry (mIHC), flow cytometry, and in vitro co-culture assays.


RESULTS
The scRNA-seq analysis revealed significant cellular heterogeneity and a lineage-specific immune landscape, including cancer-associated fibroblasts (CAFs), neutrophils, T cells, natural killer cells, and myeloid cells. Specifically, CAF subset distribution was highly lineage-dependent. Collagen-expressing CAF3 was significantly enriched in the PIT1 lineage compared to both TPIT and SF1 lineages. In contrast, inflammatory CAF5 was predominantly found in the TPIT lineage relative to the PIT1 and SF1 lineages. Furthermore, CD4+ regulatory T (Treg) and T follicular helper (Tfh) cells were significantly enriched in the PIT1 lineage relative to the TPIT and SF1 lineages, as validated by both flow cytometry and mIHC data. Cellular communication analysis revealed notable interactions between Treg/Tfh cells and macrophages/microglia that were mediated by the cytotoxic T-lymphocyte-associated protein 4 (CTLA4)-CD86 pathway. Subsequent mIHC assays confirmed spatial colocalization of Treg cells with macrophages/microglia. Complementing these findings, in vitro co-culture assays demonstrated functional CTLA4-CD86 signaling specifically between Treg cells and macrophages, providing deeper insights into the complex cellular crosstalk within the PitNET tumor microenvironment (TME).


CONCLUSIONS
The three major PitNET lineages exhibited distinct immune cell frequencies, with the PIT1 lineage notably enriched in Treg and Tfh cells, as well as collagen-producing CAF3. Furthermore, a novel immunosuppressive interaction between Treg cells and macrophages, mediated by the CTLA4-CD86 pathway, was identified, suggesting its potential regulatory role within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d53bbb9dbcd32760a571bdb944453b1d4eebc82" target='_blank'>
              Single-cell transcriptomics reveals the landscape of immune phenotypes in pituitary neuroendocrine tumors.
              </a>
            </td>
          <td>
            Zhe Zhang, Hong Su, Jie Yin, Junhua He, Yanan Hu, Peng Li, Pinan Liu
          </td>
          <td>2026-01-09</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, R. Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) represent an interesting component of the tumor microenvironment, playing a crucial role in cancer progression, metastasis, and treatment resistance. This review provides a comprehensive overview of identified subtypes across various cancer types, highlighting their inter-tumor heterogeneity, plasticity, and function of the different CAF phenotypes. With a focus on renal cell carcinoma, we reviewed the correlations between CAF markers, disease progression, and treatment outcomes. This approach underlined the existence of CAF-induced drug resistance and therapeutic strategies that can be utilized to overcome this extrinsic resistance. A major focus of this review is the role of CAFs in inducing drug resistance, a phenomenon observed in both in vivo and in vitro cancer models. We discuss the integration of CAFs in several in vitro co-culture systems cultivated in two- and three-dimensional models, as well as their contributions to deciphering CAF-tumor crosstalk. These platforms better recapitulate the tumor microenvironment, reflecting CAF-induced therapeutic resistance. Ultimately, this review underlines the crucial role of CAF-incorporated co-culture systems in advancing drug development, especially in the context of tissue- and subtype-specific CAF targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9bbc135d7b2fbe946576726ad4f72ed810fe0c" target='_blank'>
              Modeling CAF-tumor interactions to overcome therapy resistance
              </a>
            </td>
          <td>
            Jakub Gubała, Daniel Benamran, P. Tsantoulis, Valentin Mieville, Massimo Valerio, P. Nowak-Sliwinska
          </td>
          <td>2026-01-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A central challenge in cancer research is to identify the secreted factors that sustain tumor cell survival. This is best exemplified in Hodgkin lymphoma, where malignant cells constitute a minor fraction of the tumor and rely on signals from the microenvironment for survival. Using genome-wide transcriptional profiling with spatial and single-cell resolution, we show that the neighborhood around malignant cells forms a distinct niche of 31 non-malignant cell types, enriched in helper T cells and myeloid cells, but depleted of plasma cells. Moreover, our spatial analysis nominates IL13 as a candidate survival factor. Recombinant IL13 augments malignant cell growth in vitro, and genome-wide loss-of-function screens across >1000 human cancer cell lines identify IL4R and IL13RA1, heterodimeric components of the IL13 receptor, as uniquely essential in Hodgkin lymphoma. Importantly, blocking antibodies phenocopy genetic inactivation. Our findings provide a biological rationale for testing IL13-directed therapies, which are already FDA-approved, in Hodgkin lymphoma. Hodgkin Reed Sternberg (HRS) cells and their surrounding microenvironment in Hodgkin lymphoma remain poorly characterized. Here, the authors perform genome-wide transcriptional profiling with spatial and single-cell resolution to explore the cellular and molecular composition of the Hodgkin lymphoma microenvironment and used machine learning to identify IL13 as a potential HRS cell survival factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3242a7ddb3cf6f11fc73d58584fbd06dec0792" target='_blank'>
              Glioma cells achieve malignant progression by fusion with macrophages to gain high SLC7A5 expression
              </a>
            </td>
          <td>
            Yuedong Hu, Fusheng Liu, Rui-fang Mi
          </td>
          <td>2025-12-16</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Somatic mutations accumulate during the first zygotic division and continue throughout an organism’s lifespan. The characteristics and frequency of these mutations are contingent on developmental timing and tissue type, giving rise to somatic mosaicism, defined as the presence of unique genomic alterations across different cells. They serve as endogenous cellular barcodes, enabling detailed reconstruction of cell lineages and clonal dynamics. Although lineage tracing techniques have advanced from early microscopic observation and dye staining to the introduction of artificial barcodes via gene editing, owing to ethical considerations, such genetic manipulations in human developmental research are unavailable. Therefore, spontaneously arising somatic mutations are the most suitable strategy for tracing human lineages. Current approaches can be broadly categorized into two strategies: (i) high-resolution methods, including single-cell clonal expansion or laser-capture microdissection, which construct precise phylogenetic trees based on shared mutation profiles; and (ii) bulk sequencing methods, which infer lineage proximity by comparing variant allele frequencies across samples. As more lineage-tracing studies are being conducted focusing on a wider variety of organs, the integration of such data will make it possible to discover the general principles governing human development. This review highlights how the concept of somatic mutations has been applied across diverse biological contexts and discusses the insights and common principles that can be drawn from these findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b635d70a77d777444fb98a226730074956b4517" target='_blank'>
              Reconstructing developmental lineages: a retrospective approach using somatic mutations and variant allele frequency
              </a>
            </td>
          <td>
            Mahnoor Sajjad, Seong Gyu Kwon
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) harbors a dynamic tumor microenvironment (TME) in which macrophages are highly abundant and plastic. Under physiological conditions, macrophages switch between inflammatory and resolution/tissue-repair programs to maintain homeostasis; however, during hepatocarcinogenesis these programs are reprogrammed into tumor-associated macrophage (TAM) states that foster immune suppression, angiogenesis, and tumor progression. Purpose: To summarize macrophage heterogeneity and polarization mechanisms in HCC, and to highlight omics-informed therapeutic opportunities for targeting TAMs and improving precision immunotherapy. Research Design: This review summarizes physiological macrophage polarization and the mechanistic basis of macrophage reprogramming in the hepatocellular carcinoma immune microenvironment, integrating evidence from recent advances in single-cell sequencing, multi-omics, and spatial transcriptomics, with a focus on macrophage subset diversity, key regulatory pathways governing polarization and function, and emerging macrophage-targeted interventions and biomarkers. Results: Recent single-cell and spatial multi-omics studies reveal substantial TAM heterogeneity and plasticity in HCC. Macrophage-targeted strategies—including TAM depletion, phenotypic reprogramming, and exosome-mediated drug delivery—show encouraging preclinical efficacy. Macrophage-associated prognostic models and biomarkers may support individualized immunotherapeutic approaches. Conclusions: Macrophage polarization in HCC represents a dynamic continuum that is essential for homeostasis but is co-opted by tumors to drive immunosuppression and tissue remodeling. Advances in single-cell and spatial multi-omics are redefining TAM subsets and actionable pathways, enabling more rational macrophage-targeted therapies. However, challenges remain in standardizing TAM definitions, identifying robust predictive biomarkers, minimizing off-target effects, and optimizing combinations with immunotherapy. Integrating longitudinal multi-omics with AI-based modeling may help predict macrophage state transitions, guide patient-specific regimens, and advance precision medicine in HCC. Plain language summary Hepatocellular carcinoma is a highly aggressive cancer. Studies have found that immune cells within tumors—particularly macrophages—play a crucial role in cancer development and therapeutic response. Macrophages have two main “personalities”: the M1 type, which can kill cancer cells, and the M2 type, which supports cancer growth. They shift between these states in response to signals from the tumor microenvironment. Both liver-resident Kupffer cells and infiltrating macrophages are involved in this process. Various molecules, such as cytokines, enzymes, and exosomes, influence their polarization. This polarized state of macrophages, in turn, affects key tumor processes such as stemness, proliferation, metastasis, and angiogenesis. In recent years, scientists have been exploring drugs that specifically target macrophage polarization to suppress liver cancer, offering new directions for future therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9c3d3369daa7c8bd747ff210bf91cca4945570" target='_blank'>
              Macrophage Polarization in the Tumor Microenvironment of Hepatocellular Carcinoma: From Mechanistic Insights to Translational Therapies
              </a>
            </td>
          <td>
            Xiaoqing Fu, Mingquan Pang, Zhixin Wang, Haijiu Wang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Control: Journal of the Moffitt Cancer Center</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC), the most common subtype of lung cancer, presents major clinical challenges owing to its highly complex tumor microenvironment and substantial cellular heterogeneity. A comprehensive understanding of the immune landscape—particularly the phenotypic diversity and functional roles of macrophage subpopulations—is critical for identifying novel therapeutic targets and developing more effective treatment strategies. We constructed a comprehensive single-cell atlas of Stage I NSCLC using single-cell RNA sequencing (scRNA-seq) to characterize immune heterogeneity, with particular emphasis on macrophage subsets. Network pharmacology was employed to investigate the interactions between AHSA1 and bioactive compounds derived from the traditional Chinese medicine formulation Yu Ping Feng San (YPFS). To further evaluate these interactions, molecular docking simulations were performed to assess binding affinities and elucidate the potential regulatory effects on macrophage polarization. ScRNA-seq analysis revealed heterogeneous macrophage subtypes, each exhibiting distinct roles in tumor progression. AHSA1 was identified as a key regulator of macrophage polarization, with higher expression levels associated with unfavorable prognosis. Through network pharmacology and molecular docking, strong binding affinities were observed between AHSA1 and Yu Ping Feng San (YPFS)-derived compounds, including wogonin, kaempferol, and quercetin. Notably, quercetin displayed the highest binding affinity and exerted the strongest inhibitory effect on M2 macrophage polarization. These findings identify AHSA1 as a pivotal regulator of macrophage polarization in NSCLC and position quercetin as a promising macrophage-targeted therapeutic candidate. Furthermore, the results highlight the potential of Yu Ping Feng San (YPFS) as a complementary therapeutic strategy for NSCLC, offering a bridge between traditional Chinese medicine and modern oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28981a554271689b2f20697cc02fddff6f907754" target='_blank'>
              Single-cell sequencing and network pharmacology coupled with molecular docking and experimental validation reveal the effects of YPFS on macrophages in stage I non-small cell lung cancer
              </a>
            </td>
          <td>
            Hongmin Li, Qitai Zhao, Li Yang, Le Wang, Fanglei Ye, Lei-Lei Yang, Peijun Zhang, Yi Zhang
          </td>
          <td>2026-01-27</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, F. Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial proteomics has enabled high-resolution characterization of protein organization within tumor microenvironments, yet most computational approaches implicitly assume spatial homogeneity and focus on clustering rather than diffusion constraints imposed by tissue morphology. Here, we model morphology–protein coupling in triplenegative breast cancer using geographically weighted regression (GWR) applied to 41 publicly available Multiplexed Ion Beam Imaging (MIBI) samples comprising 36 protein markers. Single-cell morphometric features were extracted from MIBI spots and combined with spatial adjacency graphs to model location-specific protein dispersion. Compared with ordinary least squares and ridge regression baselines, GWR consistently demonstrated superior performance across regression metrics, explaining substantially greater spatial variance in protein intensity (+.4 R2 improvements across markers) while reducing mean absolute and squared errors. Information-theoretic analysis showed lower (Aikake Information Criterion Corrected) AICc values for GWR across the majority of markers, indicating improved model fit. Spatial autocorrelation diagnostics further confirmed that GWR residuals exhibited near-random structure, with significant reductions in Moran’s I and Geary’s C relative to global models, demonstrating effective capture of local heterogeneity. Eight proteins with significant spatial autocorrelation, including B7-H3 and -catenin, showed pronounced morphology-dependent dispersion patterns that were not recoverable using global regression. These results demonstrate that explicitly modeling spatial heterogeneity yields more accurate and interpretable representations of protein organization and supports a diffusion-barrier view of pathoproteomics beyond agglomeration alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db951a4a2419b28fdecf539ba062b45ce28797e3" target='_blank'>
              Spatial Regression of Morphology-Protein Coupling in Tumour Proteomics
              </a>
            </td>
          <td>
            Alejandro Leyva, M. Khalid, Khan Niazi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Prostate cancer is genomically heterogeneous with few shared drivers. We characterized tumor-specific DNA methylation programs that define evolutionary subtypes, remodel the tumor–microenvironment ecosystem, and yield liquid-biopsy biomarkers.



 Illumina EPIC-seq methylation was profiled in matched benign/tumor tissues from the UK Prostate ICGC cohort. A covariance-based DMR caller identified robust tumor-specific regions that were intersected with bulk RNA-seq, network analyses, and single-cell/spatial transcriptomics to localize programs in situ. A FACS-sorted, cell-type methylome atlas (luminal, basal, endothelial, fibroblast, immune, smooth muscle) anchored cell-of-origin assignments. Cell-free DNA was deconvolved against this atlas to quantify luminal tumor DNA and circulating microenvironmental DNA (cmDNA). Prognostic associations used penalized Cox models.



 Tumor-specific DMRs separated cancers into two “mevotypes” mirroring genomic evotypes (ERG-fusion vs SPOP-mutant), indicating genomic–epigenomic interplay in evolutionary commitment. Cell-resolved methylomes attributed most tumor DMRs to luminal epithelium and uncovered neuronal/plasticity pathways in luminal cells, while immune/endothelial-assigned DMRs converged on ECM remodeling, angiogenesis, TGF-β, and Rho-GTPase signaling. Single-cell pseudotime and spatial mapping localized neuronal-enriched luminal programs to late-trajectory regions and a CHGA/CHGB-positive duct-adjacent epithelial circuit interfacing with fibroblasts at tumor–stroma borders. Network analysis highlighted MYC-centric modules and EZH2-linked transcription factor contexts, consistent with epigenetic priming toward quasi-neuroendocrine states in treatment-naïve localized adenocarcinoma. In plasma, deconvolution detected both luminal ctDNA and cmDNA; luminal fractions tracked tumor burden in metastatic disease (ichorCNA-concordant), whereas cmDNA—predominantly attributable to immune compartments—suggested an early, tumor-specific microenvironmental response that can precede and is independent of luminal ctDNA shedding. Methylation signatures derived from neuronal and microenvironmental gene sets associated with inferior relapse-free survival.



 Epigenetic drivers in early prostate cancer encode luminal neuronal plasticity and reorganize the tumor niche. A cell-resolved atlas enables cfDNA deconvolution to reveal luminal ctDNA plus cmDNA, supporting liquid-biopsy biomarker strategies for earlier detection and improved risk stratification in prostate cancer.



 Harveer Dev, Toby Milne-Clark, Henson Lee Yu, CRUK-ICGC Prostate Cancer Group. Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2503538f0dcd516b3bf974c3bb6eeaf7a1544a19" target='_blank'>
              Abstract A015: Epigenetic Drivers of Localised Prostate Cancers Reveal Neuronal and Microenvironmental Reprogramming and inform Liquid Biopsy Biomarkers
              </a>
            </td>
          <td>
            H. Dev, Toby Milne-Clark, Henson L. Lee Yu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3789c5447c0f1d5a0fe59b4f852d13e8acfbe641" target='_blank'>
              Unlocking single-cell level and continuous whole-slide insights in spatial transcriptomics with PanoSpace.
              </a>
            </td>
          <td>
            Hui-Feng He, Pai Peng, Shi-Tong Yang, Meng-Guo Wang, Xiao-Fei Zhang, Luonan Chen
          </td>
          <td>2026-01-06</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="HCC is characterized by extensive ECM remodeling, primarily mediated by stromal cells. While cancer-associated fibroblasts are known contributors to tumor fibrosis, their transcriptional diversity and role in immune modulation within the HCC tumor microenvironment remain poorly resolved. Integrative analysis of six public HCC scRNA-seq datasets was employed to identify ECM-active stromal cells. High-ECM cells were re-clustered and fibroblast subtypes were defined through differential expression, pathway enrichment, lineage scoring and cell-cell interaction modeling. Spatial transcriptomic mapping and tissue-level profiling were performed to validate and prioritize ECM-related genes for experimental analysis. COL1A1 and COL3A1 were selected for siRNA-mediated knockdown in HepG2 and HepB3 cells followed by RT-qPCR, Western blot, proliferation, colony formation and wound healing assays. ECM activity analysis identified fibroblasts as the most ECM-enriched stromal population. Reclustering of ECM-high stromal cells identified eight distinct subtypes and targeted fibroblast clustering revealed five functionally diverse states. GO and spatial transcriptomic analysis confirmed subtype-specific functions and localization. In silico tissue profiling further prioritized COL1A1 and COL3A1 as pan-mesenchymal ECM genes enriched in fibrogenic fibroblast subsets. CellChat analysis revealed myofibroblasts and inflammatory CAFs as dominant signal senders. Knockdown reduced COL1A1 and COL3A1 expression at both mRNA and protein levels, and enhanced HCC cell proliferation, migration and colony formation. This study identifies transcriptionally and functionally distinct fibroblast subtypes in HCC and highlights COL1A1 and COL3A1 as key matrix-regulatory genes expressed in fibrogenic stromal subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fd864648cbb63d6bef938be32a938f167aaf27" target='_blank'>
              Single-cell profiling reveals lineage-specific fibroblast stromal subtypes drive ECM remodeling and immune modulation in the hepatocellular carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Zhichao Jiang, Haitao Wang, Han Li, Zhiqiang Chen, Baiyu Sun
          </td>
          <td>2026-01-03</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The various cellular composition of the tumor microenvironment (TME) comprises the fundamental units of tumor tissue. The types of stromal cells in the TME are genetically stable, with reduced risk of tumor recurrence and drug resistance. More and more evidence shows their clinicopathological significance and therapeutic effect in predicting prognosis. Therefore, we performed an integrated analysis of the breast cancer TME, correlating it with genomic landscapes and clinical profiles. In this work, we first conducted unsupervised hierarchical clustering on 830 tumors in the breast cancer cohort. Then, we defined three TME phenotypes and applied principal component analysis to construct a TMEscore for quantifying TME. Analysis revealed that patients stratified into the high TMEscore cohort exhibited superior survival compared to the low‐scoring group. Additionally, a high TMEscore is associated with an improved response to immunotherapy. Through TME gene signature analysis, KIF15 was identified as a pivotal driver of the immunosuppressive microenvironment in breast cancer. KIF15 knockdown may promote dendritic cell infiltration and function, thereby inducing CD8+ T cell recruitment. In summary, the immune microenvironment‐derived TMEscore represents an independent prognostic biomarker in breast cancer, while KIF15 emerges as a crucial molecular determinant of its immunosuppressive niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44851ced88ac81dcb9eaccc6f7fcac4727fd09a" target='_blank'>
              Tumor Microenvironment Characterization Identifies KIF15 as an Immunosuppressive Driver in Breast Cancer
              </a>
            </td>
          <td>
            Bo Zhang, Feiran Wang, Huiwei Huang, Xiang Zhong, Qian Chen, Xiancheng Liu, Sujit Nair
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Penile cancer is a rare malignancy with poor prognosis in advanced and recurrent stages. The currently limited treatment options and poor survival outcomes represent an unmet need for patients with advanced or recurrent penile cancer. Innovation in therapeutic options for patients with penile cancer relies on insight of underlying tumor biology and tumor–immune system interactions. In this paper, we use multiplex immunofluorescence to investigate the immune microenvironment of penile cancer, focusing specifically on geospatial clustering patterns of various immune effector cells within various stages of penile cancer and human papillomavirus status. We identified spatial and clustering patterns of pro-immunogenic macrophages, tumor-associated macrophages, and helper T cells as potential prognosticators of survival in penile squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14fd3b38e31e27ab5192f8dbd3698cf76eb34d9f" target='_blank'>
              Geospatial and Cell Density Analysis Using Multiplex Immunofluorescence Reveals an Important Role of Clustering Patterns of Immunosuppressive Macrophages in Survival Outcomes of Penile Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            A. Fazili, Keerthi Gullapalli, Gabriel Roman Souza, Firas Hatoum, Justin W Miller, Youngchul Kim, Junmin Whiting, Jeffrey S Johnson, J. Dhillon, Jonathan Nguygen, C. M. Segura, Philippe E Spiess, J. Chahoud
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5076c3231fa10b8e1b6c94efd5aa380d1b021cf8" target='_blank'>
              Single-Cell Metabolic Profiling in a Glioblastoma Co-culture Model Using AP-MALDI-based Mass Spectrometry Imaging
              </a>
            </td>
          <td>
            U. Kontrimaite, Kei F. Carver Wong, Sandra Martínez-Jarquín, Phoebe McCrorie, Ruman Rahman, Dong-Hyun Kim
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Medulloblastoma and glioblastoma are the most common malignant primary brain tumors in children and adults, respectively. Tumor-associated macrophages and microglia are key non-cancerous cell types in these tumors. These cells interact with CD24, a so called “don’t eat me signal” expressed on tumor cells, through Siglec-10, a receptor that contributes to immune evasion by promoting an immunosuppressive environment. The CD24/Siglec-10 interaction in context of malignant brain tumors has been scarcely studied.In silico analyses reveal that CD24 gene expression correlates with specific gene signatures associated with prognosis in both medulloblastoma and glioblastoma. In both human- and mouse brain tumors, Siglec-10+ cells co-express the microglia-associated molecule TREM2. Treatment with mitoxantrone as an immunogenic cell-death-inducing cytostatic agent led to a dose-dependent reduction in cell viability and cell surface CD24 levels in both murine and human brain tumor cell cultures. Intratumoral mitoxantrone administration in a murine CD24-high glioma model extended survival, decreased tumor size, reduced Siglec-10+/TREM2+ cell populations, and increased anti-tumor CD8+ cells. These findings suggest that targeting the CD24/Siglec-10 axis with mitoxantrone may modulate the tumor microenvironment and enhance anti-tumor immunity. Keywords: CD24, Siglec-10, Mitoxantrone, Malignant brain tumor, Immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b532e86a6b6dc44d55a8d26467db7ca733b62270" target='_blank'>
              Mitoxantrone alters CD24/Siglec-10 expression in malignant brain tumor models
              </a>
            </td>
          <td>
            Jan Kopecky, Julio Enríquez Pérez, Stevanus Jonathan, Tom Milos, Poi Kwanyuen, J. Biskupiak, E. Visse, K. G. de Oliveira, H. Talbot, M. Cerezo-Magaña, V. Governa, Jinquan Cai, Marie Arsenian Henriksson, Nils Ståhl, David Cederberg, M. Belting, P. Siesjö, A. Darabi
          </td>
          <td>2026-01-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283f47aefbdff4ebd6faa24c4a2c93b8e125039d" target='_blank'>
              Translating spatial transcriptomic signatures in adenosquamous carcinoma into bulk prognostic biomarkers in lung adenocarcinoma: a bottom-up approach.
              </a>
            </td>
          <td>
            K. Hatakeyama, T. Kawata, Koji Muramatsu, T. Oishi, Takeshi Nagashima, Ai Sakai, Terumi Nishimura, K. Ohshima, Sumiko Ohnami, S. Ohnami, Y. Shimoda, A. Naruoka, Koji Maruyama, K. Urakami, Y. Akiyama, Y. Ohde, Takashi Sugino, Ken Yamaguchi
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract The musculoskeletal temporally activated novel gene (Mustn1) is a 9.2-kDa microprotein that has been extensively studied within musculoskeletal tissues. Utilizing open-source, single-cell RNA sequencing datasets, we used a top-down, transcriptional approach that combines systems-wide and targeted-tissue mapping of Mustn1. In doing so, we observed robust mural cell-state colocalization of Mustn1 and Acta2 (encoding alpha smooth muscle actin [aSMA]) within bone, synovium, muscle, and tendon tissues. Mapping Mustn1 within recently documented, uncharacterized cell clusters of Sox9 lineage also revealed overlap with musculoskeletal fibroadipogenic progenitors (FAPs) and mural cells. Overall, these findings demonstrate that aSMA-expressing cells of the periosteum and tendon sheath provide a highly selective cell population to leverage the study of Mustn1 during musculoskeletal development and repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3014e21a242c4726145b1f48455415c76fd1751" target='_blank'>
              Single-cell transcriptional mapping of Mustn1 exhibits consistent mural cell localization across musculoskeletal tissues
              </a>
            </td>
          <td>
            Christopher J Janton, Anne Nichols, Michael Hadjiargyrou
          </td>
          <td>2025-12-18</td>
          <td>JBMR Plus</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial proteomics enables high-resolution mapping of protein expression and can transform our understanding of biology and disease. However, major challenges remain for clinical translation, including cost, complexity and scalability. Here we present H&E to protein expression (HEX), an AI model designed to computationally generate spatial proteomics profiles from standard histopathology slides. Trained and validated on 819,000 histopathology image tiles with matched protein expression from 382 tumor samples, HEX accurately predicts the expression of 40 biomarkers encompassing immune, structural and functional programs. HEX demonstrates substantial performance gains over alternative methods for protein expression prediction from H&E images. We develop a multimodal data integration approach that combines the original H&E image and AI-derived virtual spatial proteomics to enhance outcome prediction. Applied to six independent non-small-cell lung cancer cohorts totaling 2,298 patients, HEX-enabled multimodal integration improved prognostic accuracy by 22% and immunotherapy response prediction by 24–39% compared with conventional clinicopathological and molecular biomarkers. Biological interpretation revealed spatially organized tumor–immune niches predictive of therapeutic response, including the co-localization of T helper cells and cytotoxic T cells in responders, and immunosuppressive tumor-associated macrophage and neutrophil aggregates in non-responders. HEX provides a low-cost and scalable approach to study spatial biology and enables the discovery and clinical translation of interpretable biomarkers for precision medicine. A model trained to generate spatial proteomics profiles and predict the expression of 40 biomarkers directly from histopathology slides is shown to improve survival and treatment response prediction in patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812a5ae318d320a30cb51379e4bd5ce695c6ba0e" target='_blank'>
              AI-enabled virtual spatial proteomics from histopathology for interpretable biomarker discovery in lung cancer
              </a>
            </td>
          <td>
            Zhe Li, Yuchen Li, Jinxi Xiang, Xiyue Wang, Sen Yang, Xiaoming Zhang, Feyisope R. Eweje, Yijiang Chen, Xiangde Luo, Yuanyuan Li, Jonathan Mulholland, Colin Bergstrom, Ted Kim, F. Olguin, S. Willens, Jeffrey J Nirschl, Robert West, J. Neal, M. Diehn, Ruijiang Li
          </td>
          <td>2026-01-01</td>
          <td>Nature Medicine</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a rare genitourinary malignancy, and factors of its tumor microenvironment (TME) could serve as prognostic indicators for tumor recurrence and metastasis. Here, we generated a comprehensive single-cell map of PSCC (66 421 cells) and identified 9 distinct cell populations with samples from nine tumor samples and six adjacent normal samples. Among the malignant cells, SEMA3Chigh Mals was found to be associated with epithelial-mesenchymal transition. T cells in tumor tissues are in a highly exhausted state, while SPP1high TAMs were observed to promote tumor progression. Cancer-associated fibroblasts were found to interact with malignant cells to facilitate EMT through several pathways. Notably, there is a specific type of pericyte called POSTN+ pericytes, which can promote angiogenesis and extracellular matrix remodeling in PSCC. Finally, SEMA3C was identified as an effective biomarker reflecting cancer stage and microvessel density. Overall, we investigated the heterogeneity of TME from a single-cell perspective and demonstrated that SEMA3C serve as an effective biomarker for predicting lymph node metastasis and prognosis in PSCC. These findings may offer valuable insights for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5b537b7833eaaeb7747d5123ca286f841e6ac7" target='_blank'>
              Decoding the Cellular Heterogeneity and Malignant Progression of Human Penile Squamous Cell Carcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Xiheng Hu, Wensheng Shi, Liang Dong, Lingjuan Huang, Xiyuan Zhang, Yiting Feng, Jie Sun, Lanlan Liu, Teng Liu, Jun Fu, Bowen Zhong, Qihao Leng, Xiaohua Wu, Minfeng Chen, Lingfang Li, Yuan Li, Xin Jin, Long Wang, Jian Cao, Xin Li, Mingzhu Yin, Xiang Chen
          </td>
          <td>2026-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596e7d27f8f208d19adb1081b3f37426d428f0a7" target='_blank'>
              PTEN deficiency linked to chromosome 10q loss leads to aggressive NF2 mutant meningioma biology
              </a>
            </td>
          <td>
            Abigail G. Parrish, H. N. Thirimanne, Sonali Arora, C. Herold-Mende, P. Sievers, F. Sahm, Frank Szulzewsky, Eric C. Holland
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Routine histology contains rich prognostic information in stage II/III colorectal cancer, much of which is embedded in complex spatial tissue organisation. We present INSIGHT, a graph neural network that predicts survival directly from routine histology images. Trained and cross-validated on TCGA (n=342) and SURGEN (n=336), INSIGHT produces patient-level spatially resolved risk scores. Large independent validation showed superior prognostic performance compared with pTNM staging (C-index 0.68-0.69 vs 0.44-0.58). INSIGHT spatial risk maps recapitulated canonical prognostic histopathology and identified nuclear solidity and circularity as quantitative risk correlates. Integrating spatial risk with data-driven spatial transcriptomic signatures, spatial proteomics, bulk RNA-seq, and single-cell references revealed an epithelium-immune risk manifold capturing epithelial dedifferentiation and fetal programs, myeloid-driven stromal states including $\mathrm{SPP1}^{+}$ macrophages and $\mathrm{LAMP3}^{+}$ dendritic cells, and adaptive immune dysfunction. This analysis exposed patient-specific epithelial heterogeneity, stratification within MSI-High tumours, and high-risk routes of CDX2/HNF4A loss and CEACAM5/6-associated proliferative programs, highlighting coordinated therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0506faf43ebc888bee635ffd236772260977faab" target='_blank'>
              INSIGHT: Spatially resolved survival modelling from routine histology crosslinked with molecular profiling reveals prognostic epithelial-immune axes in stage II/III colorectal cancer
              </a>
            </td>
          <td>
            Piotr Keller, Mark Eastwood, Zedong Hu, A. Selten, Ruqayya Awan, Gertjan Rasschaert, Sara Verbandt, V. Popovici, Hubert Piessevaux, Hayley T Morris, Petros K Tsantoulis, Thomas Mckee, A. D'Hoore, Cédric Schraepen, X. Sagaert, G. Hertogh, S. Tejpar, Fayyaz Minhas
          </td>
          <td>2025-12-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William C. Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb H Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="The formation of intratumoral spatial niches has been reported for many human malignancies. However, the translational potential of such spatial niches is understudied. Herein, we utilize digital spatial profiling (DSP) to explore the prognostic relevance of spatially defined protein expression in high-risk prostate cancer. A total of 49 patient samples were analyzed for the expression of 46 proteins in 463 regions of interest (ROIs) from the tumor center (n = 198) and the tumor periphery (n = 265) resulting in 21,298 primary data points (mean per patient n = 9.4). Expression data from either the tumor center or the tumor periphery were not found to be prognostic. Protein expression of tumor center and periphery was then integrated into single datapoints by calculating the log2-transformed relative expression between the two niches for each protein and patient. Unsupervised hierarchical clustering of these data yielded two distinct patient subgroups. These clusters did not show a statistically significant correlation with known prognostic parameters yet significantly correlated with progression-free survival (p = 0.014, log-rank, HR 0.43; 95% CI, 0.22–0.86). Our results thus reveal that spatial protein expression contains prognostic information, however, only when expression data from both spatial niches are taken into account. In conclusion, our proof-of-concept study shows that DSP can be exploited for the development of novel prognostic biomarkers that rely on spatially resolved protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b3a4672d8a49cdbb7af4005dce769ab547c41" target='_blank'>
              Prognostic impact of spatial niches in prostate cancer
              </a>
            </td>
          <td>
            Felix Schneider, Sarah Böning, Beatriz Coelho Antunes, A. Kaczorowski, M. Görtz, V. Schütz, Johannes Huber, Albrecht Stenzinger, Markus Hohenfellner, S. Duensing, A. Duensing
          </td>
          <td>2026-01-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Early metastatic spread represents a challenge in fighting small cell lung cancer (SCLC). The molecular mechanisms underlying metastatic dissemination remain unclear in this devastating disease. Invasive traits were investigated in 13 SCLC cell lines using 3D-spheroid formation, sprouting assays, co-cultures and a zebrafish xenograft model. Proteomic analysis was performed to unravel metastatic drivers, which were validated by qPCR, growth factor arrays and specific inhibitors. Overall, 8 cell lines formed spheroids, and half of these displayed invasive sprouting in collagen. The ‘sprouter’ SCLC cells, which all had a YAP1-dominant subtype, showed increased migration in zebrafish larvae and penetrated endothelial cell monolayers to a higher extent, thereby mimicking intra- and extravasation. Proteomics revealed differences in adhesion properties, oncogenic pathways and receptor tyrosine kinase signalling. Sprouter cells showed higher expression levels of mesenchymal cell state markers. Stimulation with fibroblast growth factor 2 (FGF2) further induced invasive sprouting, while blocking the FGF/R axis resulted in a significant reduction of sprouting in vitro and in vivo. The FGF/R axis is a key driver of SCLC metastatic spread in the YAP1-dominant subtype. These data might facilitate the development of potential future therapies targeting FGF/R signalling to prevent SCLC progression and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd5e55dc91efeea2420d071d985826aef86a60b" target='_blank'>
              Fibroblast growth factor signals drive the metastatic behavior in small cell lung cancer
              </a>
            </td>
          <td>
            B. Ernhofer, Anna Solta, Julia Sinner, Z. Megyesfalvi, Abigail J. Deloria, K. Boettiger, Lisa Glatt, Lilla Horvath, C. Sturtzel, A. Wenninger-Weinzierl, Martin Distel, M. Grusch, B. Szeitz, Melinda Rezeli, Clemens Aigner, B. Dome, K. Schelch
          </td>
          <td>2025-12-13</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background While adipose tissue constitutes a substantial proportion of breast composition, the functional characteristics and pathological relevance of the adipocyte microenvironment in breast carcinogenesis remain undercharacterized. This study employs single-nucleus RNA sequencing (snRNA-seq) to establish a comprehensive cellular atlas of adipocyte heterogeneity across molecular subtypes of breast cancer, aiming to elucidate subtype-specific adipocyte contributions to tumor microenvironment modulation. Methods snRNA-seq was performed on breast adipose tissues isolated from individuals without cancer and treatment-naïve breast cancer. Various adipocyte and pre-adipocyte subclusters were identified. Comparative analyses of cellular distribution and transcriptional profiles were performed across disease states and molecular subtypes. Pseudotime, cell communication, and immunofluorescence analyses were further implemented to investigate cellular dynamics and microenvironment interactions. Results snRNA-seq data of 86,529 nuclei were obtained. Three adipocyte and seven pre-adipocyte subclusters were identified, of which Adi_LDLR, Pre_Adi_LDLR, and Pre_Adi_LGR4_TGFBR1 were restricted to cancer-associated adipose (CAAs). Adi_LDLR and Pre_Adi_LDLR were enriched in estrogen receptor (ER)-positive CAAs and related to cell senescence and immunosuppression. Pre_Adi_LGR4_TGFBR1 was predominantly present in triple-negative breast cancer, functionally pro-proliferative, immunosuppressive, and lacked normal adipose function. The immunofluorescence intensity of LDLR (p=0.031) and TGFBR1 (p=0.038) was positively associated with disease recurrence, suggesting the formation of immunosuppressive niches by these cancer-specific adipose subsets in both subtypes. Cell communication analyses revealed a specific (pre-) adipocyte-macrophage interaction via ligand-receptor pairs involved in stromal remodeling and tumor migration for ER-positive tumors, whereas tumor proliferation and metastasis for triple-negative ones likely contribute to tumor progression. Conclusions This study delineated a distinct adipocyte landscape in breast cancer and subtype-specific immunosuppressive niches fostered by CAAs and (pre-) adipocyte-macrophage interactions. These findings provide novel therapeutic targets for microenvironment-directed interventions in breast oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c4c519ddf0de4fc6c0a04e6ac2d29ba332f1031" target='_blank'>
              Decoding adipocyte heterogeneity through single-nucleus transcriptomics unveils subtype-specific adipocytes orchestrate immunosuppressive niches in breast cancer
              </a>
            </td>
          <td>
            Yiwei Tong, Zheng Wang, Niu Qiao, Renhong Huang, Chenghui Wu, Haoyu Wang, X. Fei, Kunwei Shen, Xiao-song Chen
          </td>
          <td>2026-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background and objectives Recent molecular landscape studies characterizing lung cancer mostly utilize single tumor regions, ignoring tumor heterogeneity and evolutionary patterns. To address this issue, the TRAcking NSCLC Evolution through therapy (TRACERx) Consortium established a multiregion sampling of Western lung cancer to investigate its longitudinal evolutionary dynamics. Due to ancestral differences, there are still major gaps in understanding oriental lung cancer. Methods We have established a prospective cohort of patients with non-small cell lung cancer (NSCLC) named Chinese lung cancer evolution and microenvironment tracking under therapy (CLEVER) study. Discussion By acquiring comprehensive genetic variation data and clonal events from multiple regions of the tumor, we aim to decipher the molecular evolution code of Chinese lung cancer. The analysis of cell composition and spatial structure further characterizes the remodeling effects of tumor evolution and neoadjuvant therapy on the tumor microenvironment, helping to identify important markers that influence treatment effectiveness and recurrence. By monitoring mutations using a personalized minimal residual disease (MRD) panel from tumors at baseline, we can trace evolutionary significance and clonality, aiding in therapy decisionmaking. The CLEVER study significantly contributes to the development of personalized treatment strategies for lung cancer patients, enhancing outcomes and tailoring interventions to individual patient profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f2f25eef8688466fb037f4fd6a8d16c5aeeb96" target='_blank'>
              Protocol for Chinese lung cancer evolution and microenvironment tracking under therapy study
              </a>
            </td>
          <td>
            Wenxiang Wang, Yue He, Xiaoqiu Yuan, Lin Weng, Jing Bai, Xu Liu, Li Han, Jun Xu, Shuai Wang, Shanshan Lu, Jian Bai, Hui Quan, Qingyun Liu, Xinrui Li, Yuxuan Xiao, Yun Wang, Qingna Zhang, Na Zhou, Yanyan Hou, Hao Li, Y. Li, Fan Yang, Jun Wang, Kezhong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832c2464c80a294ed1bb7a8527b6613a4dd9fa1e" target='_blank'>
              Spatial gene expression analysis reveals drivers of extremely early lymph node metastasis in breast cancer.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Keiko Kajiya, Erina Ishikawa, A. Kanai, Ayako Suzuki, A. Motoyoshi, Tsuguo Iwatani, Manabu Kubota, Masaru Nakamura, T. Onishi, Akiyoshi Hoshino, Ichiro Maeda, Akihiko Morozumi, Kenji Takatsuka, Junki Koike, M. Seki, K. Tsugawa, Yutaka Suzuki
          </td>
          <td>2026-01-23</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e9617142706d92615a2fad08d2ab214304d6df" target='_blank'>
              Real-time monitoring of pediatric high-grade glioma invasion using organotypic brain cultures reveals a developmentally dependent sensitivity
              </a>
            </td>
          <td>
            Yoav Biala, Ramakrishna Sai kumar Aravapalli, Hala Shalata, Zakhariya Manevitch, Oded Behar
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271f2b42923ccb0c522e0defc7aee243eaf5e39d" target='_blank'>
              Reconstructing clone-resolved transcriptional programs from bulk tumor sequencing
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Kathleen Noller, YunZhou Liu, Archana Balan, Prathima B. Nagendra, Luciane T. Kagohara, E. Fertig, Laura D. Wood, Rachel Karchin
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="ABSTRACT Objective The study is to investigate differential signaling pathways within the tumor microenvironment across molecular subtypes of breast cancer (BC). Methods Single‐cell RNA (scRNA‐seq) sequencing data of BC samples were obtained from the Gene Expression Omnibus database. Cell types were identified using the SingleR package, in conjunction with the analysis of marker genes. Subsequently, Monocle was used for pseudotime analysis of epithelial cells, fibroblasts, and macrophages to characterize their differentiation states. CellChat was employed to study the ligand‐receptor interactions among various cell types across different BC molecular subtypes. In addition, we used common bulk RNA sequencing data from The Cancer Genome Atlas to investigate the correlation between key signaling pathway factors identified by scRNA‐seq and clinical outcomes. Results Inference of copy number variation analysis using T cells revealed significantly elevated copy number variation scores in epithelial cells and fibroblasts. In the communication between epithelial cells and fibroblasts, the ANGPTL pathway is critical in estrogen receptor‐positive breast cancer (ER+BC), while the PTN pathway plays a key role in both ER+BC and human epidermal growth factor receptor 2‐positive breast cancer (HER2+BC), and the GAS pathway is associated with poor prognosis in triple‐negative breast cancer (TNBC). In the interaction between fibroblasts and macrophages, the macrophage subpopulation supporting tumor angiogenesis exhibits significant activity in ER+BC, with the associated SPP1 and GRN pathways strongly influencing tumor progression. The SEMA3 pathway mainly acts through dividing tumor‐associated fibroblast clusters across all BC subtypes. When exploring the role of lymphocytes, the PTN pathway also plays a role in HER2+BC, while in TNBC, CXCL and CD70 pathways are significantly involved in immune response modulation. Conclusion Our comprehensive analysis of cell–cell communication networks among epithelial cells, fibroblasts, macrophages, and lymphocytes across BC subtypes focuses on ligand‐receptor interactions. This study revealed that certain molecules within these networks exhibit significant prognostic value and therapeutic promise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff691131f959402fc8cdba31af35a8d31d30b10a" target='_blank'>
              Single‐Cell RNA Sequencing and Bulk RNA Sequencing Revealed the Interplay Between Intratumoral Heterogeneity and the Tumor Microenvironment in Breast Cancer
              </a>
            </td>
          <td>
            Yunlong Zhao, Xiaoyu Zhang, Yingying Wang, Xiaomin Yu, Fengchun Lv, Mingyu Gong, Xiu-An Yang
          </td>
          <td>2026-02-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 PSMA expression is heterogeneous across advanced prostate cancer, limiting the durability of PSMA-radioligand therapy (RLT). To characterize tumor heterogeneity and identify determinants of RLT resistance, we established the SHAPE cohort (Spatial Heterogeneity Atlas of Prostate Cancer Evolution). SHAPE is a multimodal resource integrating PET imaging, clinical treatment history, histopathology, immunohistochemistry, spatial transcriptomics, and genomic profiling across 46 patient tumor samples collected through rapid autopsy, biopsy, and surgical resection. The cohort spans adenocarcinoma, castration-resistant (CRPC), and neuroendocrine (NEPC) prostate cancer states and includes eight xenograft models.



 Twenty-five samples profiled using the 10x Genomics Visium spatial transcriptomics platform, comprising 195,905 spots, were integrated and analyzed using Seurat (v5) in R. Targeted genomic profiling was performed using the Dana-Farber Cancer Institute OncoPanel platform on 36 samples derived from 7 patients and 2 animal models. Among the cohort, seven cases had prior Lu-PSMA exposure, and ten cases were untreated. Eleven clinical and preclinical samples had PSMA PET/CT, including six with both pre- and post-treatment imaging. Data integration connected structural genomic events with spatial lineage programs and imaging features.



 RLT–treated tumors exhibited a copy-number–dominant profile compared with untreated tumors. Recurrent alterations enriched in treated tumors included PTEN deletion (p<0.01), KLLN deletion (p<0.01), AURKA gain (p<0.01), and ZNF217 gain (p<0.01). Treated tumors also showed higher per-sample copy number loss (47.6 vs 37.5, p<0.05) and amplification burden (7.91 vs 2, p<0.05). In contrast, untreated tumors displayed more single nucleotide variants (14.9 vs 6.3, ns) and indels (4.7 vs 1.9, ns). Spatial transcriptomics provided complementary insights. Treated samples with PSMA SUV below the liver reference demonstrated higher copy-number loss burden (p<0.05) and contained unique transcriptomic clusters enriched for glucose metabolism. These clusters displayed variable expression of classical NEPC markers but consistently upregulated genes involved in glucose uptake and utilization. In preclinical studies, head-to-head FDG and Ga-PSMA imaging of WCM12 xenografts demonstrated that FDG, as a surrogate marker of glucose uptake, was significantly increased following relapse after PSMA-RLT (FC, p<0.05).



 The SHAPE cohort integrates genomic, spatial, and imaging data to characterize tumor heterogeneity in advanced prostate cancer. Genomic profiling identified PTEN/KLLN loss and AURKA/ZNF217 gain as recurrent alterations enriched in Lu-PSMA–exposed tumors, while spatial transcriptomics revealed PSMA-low metastases with distinct. clusters enriched for glucose metabolic programs. Together, SHAPE can inform rational strategies for combination or alternative targeted therapies.



 Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao Wei, Varadha Balaji Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, Francesca Khani, David J. Einstein, Mary-Ellen Taplin, Eliezer Van Allen, Heather A. Jacene7, Himisha Beltran. Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therap [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912633affb19287de4d61286123b6004f19890a4" target='_blank'>
              Abstract PR027: Spatial heterogeneity atlas of prostate cancer evolution (SHAPE): Spatial and genomic landscapes of advanced prostate cancer with and without Lu-PSMA therapy
              </a>
            </td>
          <td>
            Martin K. Bakht, Jacob Egelberg, Yasutaka Yamada, Louise Clark, Xiao X Wei, V. B. Venkadakrishnan, Anthony P. Belanger, Alice Bernard-Tessier, Sarah J. Hill, F. Khani, David J Einstein, M. Taplin, E. V. Van Allen, Heather A. Jacene, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Objective: This study aimed to identify clinically relevant regulators of pancreatic ductal adenocarcinoma (PDAC), a disease characterized by stromal remodeling and immune suppression, and to define their links to malignant progression and microenvironmental reprogramming. Methods: We integrated multi-cohort bulk, single-cell, and spatial transcriptomic datasets and subsequently validated bulk differential expression and network analyses with machine learning-based prioritization in an independent combined cohort (TCGA-PAAD plus GSE62452). Single-cell mapping was used to assess cell-type specificity, positioning candidates along inferCNV- and pseudotime-defined malignant continua. In Visium sections, a DKK1-associated program score quantified intratumoral spatial heterogeneity and informed our analyses of ligand–receptor communication. Bulk immune deconvolution linked gene levels to immune infiltration patterns, and functional assays were used to test the impact of DKK1 knockdown on migration, proliferation, clonogenic growth, and apoptosis in PDAC cells. Results: Four reproducible tumor-associated genes—DKK1, COL10A1, SULF1, and SLC24A3—were prioritized and validated externally. DKK1 was predominantly expressed by epithelial tumor cells and tracked along a malignant progression continuum. Spatially, the DKK1 program localized to epithelial-dominant regions, revealed pronounced intratumoral heterogeneity, and highlighted epithelial–endothelial and endothelial–immune signaling in high-score areas. Immune deconvolution associated higher DKK1 expression with increased myeloid infiltration and reduced cytotoxic lymphocyte signatures. Functionally, DKK1 knockdown impaired migration, proliferation, and clonogenicity while increasing apoptosis. Conclusions: We demonstrate that DKK1 is an epithelial-derived regulator linked to malignant progression and tumor–stroma–immune remodeling, supporting its potential as a biomarker and therapeutic target in PDAC treatment, including rational combinations with stroma-modulating strategies and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a91f34d70864c659b210aeaa2777d7f019925a" target='_blank'>
              Multi-Omic and Spatial Profiling Identifies an Epithelial DKK1 Associated with Microenvironmental Remodeling in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Jiajia Xu, Kaiqiang Qian, Yanyu Ding, Jianghao Cheng, Xu Zhang, Yong Huang, Bo Liu
          </td>
          <td>2026-02-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Selective migration of tumors to particular organs, referred to as organ tropism, is not arbitrary but guided by intricate molecular interactions and a complex network of various secretory molecules. This site‐specific migration is modulated by molecular crosstalk, where cytokines and other effectors help tumor cells adapt to and colonize a new organ by reshaping their epigenetics. It is well established that breast and colorectal cancers, for instance, can epigenetically modify themselves to show a preference for the brain, bones, lungs, and liver but the key modulators are still elusive. Here, we have discussed the key mechanisms, including the critical interacting molecules, cytokines, and pathways that drive this site‐specific tropism‐based colonization. By synthesizing current knowledge, we highlight central players in this process and discuss their potential as therapeutic targets to inhibit metastasis and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3344c28db94b8a33c22ef6448a1314c59d95734d" target='_blank'>
              Unravelling Mechanisms of Cancer Cells' Tropism for Metastasis: A Key to Improved Cancer Therapies
              </a>
            </td>
          <td>
            Sumit Mallick, Jahnvi Hora, T. K. Varun, S. Shenoy P, Bipasha Bose
          </td>
          <td>2025-12-15</td>
          <td>Cell Biology International</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Gliomas, the most prevalent primary malignant brain tumors, pose a significant clinical challenge, particularly glioblastoma (GBM), the most aggressive subtype with a median survival of just 12–15 months post-diagnosis. Molecular profiling has revolutionized GBM classification, revealing distinct subtypes—Proneural, Neural, Classical, and Mesenchymal—each associated with unique genetic and epigenetic signatures. Despite these advances, the heterogeneity within GBM demands more comprehensive approaches for effective stratification and treatment. In this study, we integrated multi-omics data, including mRNA, lncRNA, somatic mutations, and DNA methylation profiles, using consensus clustering across ten algorithms. This approach led to the identification of novel GBM subtypes and prognostic biomarkers, enhancing the precision of molecular classification. Our findings underscore the critical role of multi-omics integration in refining GBM subtypes, offering new avenues for personalized therapeutic strategies in combating this lethal malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ecfdc5e4279034447c32992481870b8e951bb9" target='_blank'>
              Advanced Machine Learning and Multiomics Reveal Tumor Dynamics and Therapeutic Opportunities in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Jiaohong Li, Jinnan Cheng, Yong Wu, Zhiyu Lv, Ping Yuan
          </td>
          <td>2026-01-02</td>
          <td>Cancer Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Rising incidence of early-onset prostate and other solid tumors underscores the need for experimental systems that model how normal tissues traverse premalignant states, acquire mutations, and become therapy-responsive malignancies under authentic immune and stromal pressures. A major barrier has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without catastrophic chromosomal instability. To address this, the Bose Lab developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that redefines discovery for early-onset and understudied patient groups. In this system, mismatch-repair deficiency is induced in non-malignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under physiologic selection. SETs evolve primarily through high-resolution point mutations rather than broad copy-number changes, yielding bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional xenografts, SETs display greater intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under therapeutic pressure. As proof of principle in prostate cancer, endocrine therapy applied to SET pools recovered known determinants of androgen-pathway sensitivity and uncovered new drivers. Loss of ZFHX3, typically obscured within a multigene suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen-receptor inhibition in vivo, whereas KMT2D or CIC alterations mediated resistance. Consistent with model predictions, ZFHX3 loss in patients correlated with significantly improved survival, a finding comparable in magnitude to the most favorable molecular subtypes of advanced prostate cancer. SETs also quantify evolutionary thresholds: in a Pten-null background, approximately 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This defines a measurable axis linking mutation burden, genotype, and tumor incidence. Because SETs generate neoantigen-rich point-mutation landscapes, they can be extended to immunocompetent hosts to study tumor–immune coevolution, early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that enable immune escape. In summary, SETs provide a scalable, evolution-aware platform that connects mutational dynamics to therapeutic vulnerability, enabling identification of lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset prostate cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C.K. Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23678ab77681ac1f2c46c1e6d9b9824f918c99b" target='_blank'>
              Abstract A008: Reconstructing prostate evolution with Stochastically Emergent Tumors (SETs) reveals in vivo therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary malignant tumor of the central nervous system (CNS), characterized by marked molecular heterogeneity, rapid proliferation, extensive invasiveness, and strong resistance to conventional therapies. Despite current standard treatments–surgery, radiotherapy, and temozolomide–median patient survival remains below 15 months. Key contributors to GBM progression include activated glial cells and remodeling of the extracellular matrix. In this study, we investigated the cellular and molecular dynamics of the tumor microenvironment, focusing on astrocytic and microglial responses, and on the modulation of glial markers such as GFAP, Iba1, Connexin 43 (Cx43), and the purinergic receptor P2X4R. Acute and organotypic brain slice cultures were prepared from C57BL/6J mice, into which GL261 glioblastoma cells were injected into the cortex. Tissues were collected 3 and 7 days post-injection to analyze tumor progression. GBM growth was monitored through the proliferation marker Ki67. Histological analysis, confocal immunofluorescence, and western blotting were performed to assess marker expression and localization. GL261 cells engaged with the surrounding neuroglial network, migrating in fascicle-like patterns that extended even into the contralateral hemisphere. This dynamic infiltration was associated with pronounced peritumoral astrocyte activation (increased GFAP) and a robust microglial response, evidenced by morphological changes in Iba1-positive cells. Cx43 expression was altered by tumor invasion, suggesting disrupted intercellular communication, while P2X4R expression dynamics indicated modulation of the local immune response. Consistently, elevated Ki67 levels confirmed the highly proliferative and aggressive behavior of the tumor. Organotypic slice cultures represent a fast and reliable ex vivo platform to investigate GBM invasiveness, migration, and neuroglial interactions, revealing spatiotemporal remodeling of the tumor microenvironment during disease progression. Region-specific glial activation and time-dependent modulation of Cx43 and P2X4R highlight the complex and evolving nature of glial–tumor crosstalk. These findings may help identify potential therapeutic windows and support the development of molecularly targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7731564feaeab85e0322e27b0f7ea6db2c5e59" target='_blank'>
              UNRAVELING GLIAL REMODELING IN SPACE AND TIME DURING GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer progression arises from interactions across multiple biological layers, especially beyond morphological and across molecular layers that remain invisible to image-only models. To capture this broader biological landscape, we present EXAONE Path 2.5, a pathology foundation model that jointly models histologic, genomic, epigenetic and transcriptomic modalities, producing an integrated patient representation that reflects tumor biology more comprehensively. Our approach incorporates three key components: (1) multimodal SigLIP loss enabling all-pairwise contrastive learning across heterogeneous modalities, (2) a fragment-aware rotary positional encoding (F-RoPE) module that preserves spatial structure and tissue-fragment topology in WSI, and (3) domain-specialized internal foundation models for both WSI and RNA-seq to provide biologically grounded embeddings for robust multimodal alignment. We evaluate EXAONE Path 2.5 against six leading pathology foundation models across two complementary benchmarks: an internal real-world clinical dataset and the Patho-Bench benchmark covering 80 tasks. Our framework demonstrates high data and parameter efficiency, achieving on-par performance with state-of-the-art foundation models on Patho-Bench while exhibiting the highest adaptability in the internal clinical setting. These results highlight the value of biologically informed multimodal design and underscore the potential of integrated genotype-to-phenotype modeling for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7747448149656db4d84a5fb568dfd6adf6c8b244" target='_blank'>
              EXAONE Path 2.5: Pathology Foundation Model with Multi-Omics Alignment
              </a>
            </td>
          <td>
            Juseung Yun, Sunwoo Yu, Sumin Ha, Jonghyun Kim, Janghyeon Lee, Jongseong Jang, Soonyoung Lee
          </td>
          <td>2025-12-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4167a37773942f9ca4b809bad0d95f60d49ec113" target='_blank'>
              Impacts of mutation accumulation and order on tumor initiation revealed by engineered murine colorectal cancer organoids
              </a>
            </td>
          <td>
            Yanping Li, Xiaoxin Xie, Daqi Deng, Zhiyuan Sun, Zhaoan Huang, Yisen Tang, Liang Fang, Wei Chen, Qionghua Zhu
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression is strongly shaped by the tumor microenvironment (TME), where complex interactions between epithelial, immune, and stromal cells orchestrate immune suppression and tumor evolution. To dissect these relationships at single-cell resolution, we analyzed CRC scRNA-seq datasets using Seurat for data integration and CellChat for ligand–receptor inference. We identified extensive cellular heterogeneity within the TME, dominated by CMS2/CMS3 epithelial states, SPP1+ tumor-associated macrophages, diverse T-cell subsets, and CXCR4+ B cells. Communication analysis revealed MIF-centered signaling—including MIF–CD74–CXCR4 and MIF–CD74–CD44—as the predominant axis linking tumor epithelial cells with T cells, B cells, and macrophage subpopulations. CMS3 epithelial cells displayed particularly strong connectivity to SPP1+ macrophages and cytotoxic lymphocytes through both MIF- and APP–CD74-mediated pathways. Differential gene expression confirmed elevated levels of MIF, CD74, CD44, and SPP1 in tumor tissues, while pathway enrichment analyses highlighted cytokine signaling, antigen presentation, and chemokine-regulated immune modulation as key biological processes. Collectively, our study provides a high-resolution map of CRC intercellular communication and identifies MIF-CD74-associated signaling as a central immunoregulatory hub with potential relevance for therapeutic targeting and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb818bacc4907287357388f42145a8afa7d517aa" target='_blank'>
              Single-Cell Mapping Reveals MIF-Centered Immunoregulatory Networks in Colorectal Cancer
              </a>
            </td>
          <td>
            Marios Gkoris, Ilias Georgakopoulos-Soares, A. Zaravinos
          </td>
          <td>2026-02-03</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer remains a lethal disease marked by profound therapeutic resistance, largely orchestrated by a complex tumor microenvironment (TME) governed by metabolism-immune crosstalk. This review focuses on the spatiotemporal dynamics of the metabolism-immune axis in ovarian cancer progression and resistance, with particular emphasis on how cutting-edge spatial multi-omics technologies reveal previously unrecognized layers of intratumoral heterogeneity and geographic organization that cannot be captured by bulk analyses. Using tools such as MALDI-MSI, GeoMx DSP, and CODEX, these approaches enable high-resolution, spatially resolved mapping of metabolite gradients (e.g., lactate, lipids, kynurenine), immune cell niches, and immunometabolic checkpoints within distinct tumor regions. Such spatial profiling uncovers how metabolic reprogramming-dysregulated glycolysis, lipid metabolism, and glutaminolysis-drives localized immunosuppression and chemoresistance through compartment-specific interactions among tumor cells, cancer-associated fibroblasts (CAFs), adipocytes, and immune populations. These geographically defined insights reshape our understanding of therapeutic failure and highlight precise, location-aware vulnerabilities. Accordingly, we propose spatially informed therapeutic strategies, including regional glycolysis inhibition, glutaminase blockade, lipid pathway interference, and their rational combination with immune checkpoint inhibitors (ICIs), to disrupt pathogenic metabolic-immune circuits and improve immunotherapy outcomes. Looking ahead, advances in vivo spatial imaging, gut microbiota modulation, and AI-powered integrative multi-omics frameworks promise truly personalized treatment of ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5b7b1f229dc616042558b3b060c33208889755" target='_blank'>
              Spatial profiling of the metabolism-immune axis in ovarian cancer
              </a>
            </td>
          <td>
            Zhi-Bin Wang, Ming-Hui Long, Ping Yu, Ya-Li Wang, Zheng Yang, Ma-Sha Huang
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e712867c904dd12e6307954591e726635cae8e" target='_blank'>
              Differences in T-cell counts and neighborhood patterns in human colorectal adenomas and sessile serrated lesions
              </a>
            </td>
          <td>
            Souvik Seal, Lauren R. Fanning, Evan Bagley, Christine Bookhout, Elizabeth L. Barry, Elizabeth O’Quinn, Dale C. Snover, David N. Lewin, Silvia Guglietta, A. Kourtidis, M. Shrubsole, John A. Baron, Todd A. Mackenzie, Alexander V. Alekseyenko, Kristin Wallace
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare malignancy with poorly understood molecular drivers. Consequently, its cellular composition and tumor microenvironment (TME) remain largely undefined. Here, we performed integrated bulk and single-nucleus multi-omic analyses to map the SNSCC ecosystem. Within the malignant compartment, we identified five distinct populations, with hypoxic (TC1) and proliferative (TC2) subtypes associated with adverse clinical outcomes. Functionally, TC1 cells orchestrate a hypoxia-driven angiogenic program via coordinated secretion of adrenomedullin (ADM), MIF, and VEGFA, promoting endothelial tip cell (EC1) differentiation. Integrative analysis revealed these transcriptional programs are underpinned by tumor-specific chromatin accessibility and DNA hypomethylation, particularly at AP-1-enriched regulatory elements. Mechanistically, in vitro studies confirmed that this response depends on cooperative AP-1 and HIF1A signaling. Furthermore, histological analysis of patient tissues demonstrated spatial co-localization of GLUT1-expressing TC1 cells with DLL4-positive EC1 cells. These findings elucidate the epigenetic landscape underlying tumor-stromal interactions and establish the ADM/VEGFA axis as a critical therapeutic target to disrupt epigenetically controlled angiogenesis in SNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, Yoo-Sam Chung, Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef0494e51b8128b410f9d5b4dc00c303aff4796" target='_blank'>
              Mapping lineage and functional diversity in the high-risk human mammary epithelium
              </a>
            </td>
          <td>
            Matthew Waas, Bowen Zhang, Meinusha Govindarajan, Pirashaanthy Tharmapalan, Abhijith Kuttanamkuzhi, Olivia Drummond Guy, Michael Woolman, H. Berman, Paul D. Waterhouse, Rama Khokha, T. Kislinger
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The immune microenvironment profoundly shapes the progression and therapeutic response of thyroid carcinoma. Through comprehensive analysis of single-cell RNA sequencing data, this review delineates the immune landscapes of papillary thyroid carcinoma (PTC), poorly differentiated thyroid carcinomas (PDTC), and anaplastic thyroid carcinomas (ATC), revealing a differentiation dependent trajectory of tumor immune microenvironment remodeling—from immune activation suppression coexistence in PTC, to immune exclusion in PDTC, and terminal exhaustion in ATC. This single-cell based approach enables high resolution dissection of cellular heterogeneity, immune crosstalk, and spatial organization that are often masked in bulk analyses. Such insights provide a scientific basis for precision immunotherapy, offering guidance for differentiation tailored strategies to overcome immune escape and improve clinical outcomes in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662dd84def56730e4ddb7a83c80225823dda796d" target='_blank'>
              Single-cell sequencing reveals the tumor immune microenvironment in thyroid cancer: a narrow review
              </a>
            </td>
          <td>
            Kangcong Liang, Ziyu Wang, Zhiqiang Zhang, Gengluan Liu, Xidi Wang, Heng Cao, Ming Zhong, Liping Ye, Xin Zhong, Jingyu Xun, Kefeng Lei, Ningning Li
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0b74b19f68c5cff1c8b63cb7ba909481a105a55" target='_blank'>
              Spatial 5mC-seq profiling of embryos and decidua after implantation in mammal
              </a>
            </td>
          <td>
            Xun Shan, Yimin Tang, Jinzhou Hu, Mingyuan Bian, Lei Gao, Jiang Liu
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer liver metastasis (CRLM) is a leading cause of cancer-related death in patients with colorectal cancer (CRC) and remains a major clinical challenge. Emerging evidence indicates that metabolic reprogramming within the tumor microenvironment profoundly influences immune cell behavior and disease progression, yet the specific role of lipid metabolism in macrophage heterogeneity during CRLM is still unclear. We integrated single-cell RNA sequencing and spatial transcriptomics to characterize lipid metabolic states, differentiation trajectories, intercellular communication, and spatial distribution of macrophages across normal colon, primary CRC (pCRC), and CRLM. Lipid metabolic activity was quantified using multiple scoring algorithms, pseudotime analysis was performed with the “Monocle” R package, cell–cell communication was evaluated using the “CellChat” R package, and spatial localization was inferred using the RCTD method and the “MISTy” R package. Macrophages exhibited the highest lipid metabolic activity among immune cells, and this activity was significantly elevated in CRLM compared with pCRC. Subclustering analysis identified three lipid metabolism–related macrophage subsets (CD1C⁺, CXCL10⁺, CX3CR1⁺) enriched in CRLM, among which CD1C⁺ and CXCL10⁺ subsets displayed distinct differentiation dynamics. Cell–cell communication analysis revealed metastasis-specific remodeling, including diversified VEGF signaling in CD1C⁺ macrophages and enhanced SPP1-integrin and C3–(ITGAX + ITGB2) interactions in CXCL10⁺ macrophages. Spatial transcriptomics analysis showed that lipid metabolism activity was highest in macrophage-rich regions of the normal colon, higher in epithelial and tumor regions of pCRC than in fibroblast regions, and highest in hepatocyte regions of CRLM. Moreover, CD1C⁺ macrophages predominantly localized to tumor regions in both pCRC and CRLM and maintained close associations with other immune cells. Lipid metabolism shapes macrophage heterogeneity and function states during CRLM progression, with CD1C⁺ populations emerging as key macrophage subsets linked to metastatic niche remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/227b6dd6f421783cfe0a266a6aeffd63f7f91d00" target='_blank'>
              Integrative single-cell and spatial transcriptomic analysis reveals lipid metabolism–mediated macrophage heterogeneity during colorectal cancer liver metastasis progression
              </a>
            </td>
          <td>
            Xin Qi, Siyu Hou, Shaozhuo Xie, Jiajia Chen
          </td>
          <td>2025-12-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7129daca11db4cd5b5b60ff63c51368361b7db2" target='_blank'>
              Multimodal single-cell profiling reveals crosstalk between macrophages and stromal cells in poor prognostic cholangiocarcinoma patients.
              </a>
            </td>
          <td>
            Lara Heij, Sikander Hayat, Konrad Reichel, Sidrah Maryam, C. O'Rourke, X. Tan, Marlous van den Braber, J. Verhoeff, Maurice Halder, Fabian Peisker, G. Wiltberger, Jan Bednarsch, Daniel Heise, Julia Campello Deierl, Sven A Lang, F. Ulmer, Tom Luedde, E. Dahl, Danny Jonigk, Jochen Nolting, Shivan Sivakumar, J. Siveke, Florian Vondran, Flavio G. Rocha, H. Baba, S. Hartmann, Jesper B. Andersen, Zaynab Hobloss, A. Ghallab, J. Hengstler, J. J. G. Vallejo, R. Kramann, Ulf Neumann
          </td>
          <td>2026-01-28</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly invasive and refractory subtype of breast cancer. Despite the promise of immune checkpoint blockade (ICB) therapy, response rates remain limited. The immune resistance driven by the tumor microenvironment has not yet been understood entirely, which hinders the personalized precision treatment of TNBC. We integrated single-cell RNA data from 12 cohorts with TNBC and performed a multi-omics analysis combining spatial transcriptomics (ST), bulk RNA sequencing, and multiplex immunofluorescence (mIF) staining to identify immune-resistant subpopulations. Cell-to-cell communication was explored based on NicheNet and CellChat, and the function of CAF was verified by gene knockdown and overexpression in human mammary fibroblasts, followed by co-culture experiments with TNBC cell lines. ST and mIF data were used to analyze and verify cellular co-localization, while deconvolution was used to examine the relationship between two-cell characteristics and immunotherapy or antibody–drug conjugates (ADC) agent benefit. We identified CA9+cancer-associated fibroblasts (CA9+CAF) as a key subset enriched in non-responders to ICB that promotes immune resistance by establishing a hypoxic and immunosuppressive microenvironment via abnormal angiogenesis and glycolysis. ST and mIF analyses revealed a strong co-localization and interaction between CA9+CAF and SPP1+tumor-associated macrophages (SPP1+TAM), forming a stroma-myeloid axis that promotes immune escape through VEGFA/NRP2 axis in co-localization core region compared to the boundary. In vitro experiments demonstrated that the over-expression of CA9 in fibroblasts enhanced the proliferation, invasion, and migration of TNBC cells, while CA9 knockdown inhibited the tumorigenic effects. The high CA9+CAF/SPP1+TAM profile indicated a poor prognosis, reduced effector T cell infiltration, and attenuated response to immunotherapy, may benefit from TROP2, MUC1, and NECTIN4-based ADC agents. The result was validated in TNBC samples treated with neoadjuvant immunotherapy from our center. This study unveils the critical immunosuppressive axis orchestrated by CA9+CAF and SPP1+TAM in TNBC, offering novel insights into the stromal regulatory mechanisms driving immune resistance. The cell-to-cell interaction signature holds promise as predictor of immunotherapy response and potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9990df240d9cf9afd4d976bdcb6753a231a07876" target='_blank'>
              CA9+ cancer-associated fibroblasts cooperate with SPP1+ tumor-associated macrophages driving immune resistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Qin Ma, Jing Wang, Qian Jiang
          </td>
          <td>2026-01-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major issue in the sphere of head and neck cancer since it is very heterogeneous, which also leads to the poor treatment results and low survival rates in the advanced stages. Here, the review will discuss how spatial omics methods explore tumor heterogeneity in OSCC, which includes cellular, molecular, and immune microenvironment alterations that occur due to cancer stem cells, stromal interactions, genetic instability, epigenetic reorganization, and metabolic reprogramming. The focus of heterogeneity is given on the contribution of the tumor microenvironment such as immune cells, cancer-associated fibroblasts and extracellular matrix remodeling to the stimulation of progression, metastasis, as well as therapeutic resistance. The introduction of spatial omics technologies, including spatial transcriptomics, proteomics, and metabolomics, has revolutionized the field by preserving tissue architecture, enabling high-resolution mapping of gene expression, protein distribution, and metabolite profiles. Significant developments around the spatial omics technologies are discussed, while how they are used in the identification of ligand-receptor network, signaling pathways, and spatial patterns of heterogeneity in OSCC are described. Integration of multi-omics approaches bridges gaps between transcriptomic, proteomic and metabolic, facilitating the discovery of biomarkers for prognosis, immune evasion mechanisms, and precision therapies targeting epithelial-to-mesenchymal transition and immunosuppressive networks. Despite challenges in data integration, cost, and clinical translation, spatial omics holds promise for personalized oncology, with future directions involving artificial intelligence-driven modeling to enhance diagnostic accuracy and therapeutic efficacy in OSCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00bf591ef53e5e31062caa9f75b7e97f0abcc8b" target='_blank'>
              Spatial Omics in Decoding Oral Squamous Cell Carcinoma Heterogeneity: Microenvironment Crosstalk and Multi-Omics Integration
              </a>
            </td>
          <td>
            Xiaotong You, Yu Liu
          </td>
          <td>2025-12-31</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0d75b8ce4639debe11371b460bb9a8a9f43f83" target='_blank'>
              Deciphering the tumor microenvironment and role of immunotherapy in Diffuse Midline Glioma: a scoping review.
              </a>
            </td>
          <td>
            Christian K. Ramsoomair, Felipe Sarmiento, Deryn Ramsoomair, Manav Daftari, Jiasen He, Benjamin Glazer, M. Monje, Danny Reinberg, Ashish H Shah
          </td>
          <td>2026-02-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While cancer immunotherapies have primarily focused on activation of cytotoxic CD8 cells, CD4 T cell activity is also associated with survival and immunotherapeutic response in numerous cancers. We applied integrated single-cell RNA sequencing and multiplexed protein epitope profiling to breast cancer samples to resolve the complexity of immune cell states within the tumor microenvironment. This approach enhanced phenotypic resolution, identifying three distinct states within the CD4 T follicular helper-like (Tfh) cell cluster. A CXCR4high progenitor state gave rise to two differentiated states: an IGFL2high subset resembling conventional Tfh cells and localised to B cell-rich lymphoid aggregates, and a CD103+ subset, exhibiting features of tissue residency, exhaustion, and cytotoxicity, which co-localised with tumor foci. CD103+ Tfh-like cells were found to interact with CXCL10+ macrophages through production of CCL chemokines and CSF1. A higher CD103+ Tfh to IGFL2high Tfh ratio, together with the selective clonal expansion of the CD103+ subset, was strongly associated with improved tumour immunity and superior responses to anti-PD-1 checkpoint blockade, surpassing the predictive value of exhausted CD8 T cells. These findings integrate Tfh and CD4 with cytotoxic potential in breast cancer, offering new insight into anti-tumor immunity and response to checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570ecff53156bbe18de4ac45af83e6cec5c103da" target='_blank'>
              Proteotranscriptomic Dissection of Breast Cancer T Cell States Identifies CD103+ Tfh-derived Cytotoxic Cells Linked to Immunotherapy Response
              </a>
            </td>
          <td>
            Alexander Swarbrick, G. Al-Eryani, Sophie van der Leij, Etienne Masle-Farquhar, Chia-Ling Chan, K. Harvey, Sunny Z. Wu, D. Roden, Taopeng Wang, John Reeves, Bertrand Yeung, Christopher Goodnow, Cindy Ma, Charles Perou, N. Hacohen, Aziz Al’Khafaji, Mats Nilsson, J. Lundeberg, Marcel Batten, S. Junankar
          </td>
          <td>2026-01-07</td>
          <td>Research Square</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is driven by hematopoietic stem cells (HSCs) carrying leukemia-associated mutations that expand in the bone marrow. Several prior studies have revealed that the spatial organization of hematopoietic cells in the bone marrow impacts clonal behaviors. Specifically, leukemic blasts have been found to expand almost exclusively in a subset of marrow cavities that are undergoing active bone remodeling, but whether these cavities also support the expansion of non-malignant mutant clones has never been visualized. Although it is widely appreciated that systemic inflammation promotes the selection of mutant clones, this view has emerged without considering the potential heterogeneity in the inflammatory landscape shaped by local bone remodeling. Leveraging intravital imaging and a murine model of CHIP (Tet2+/-), we demonstrated transcriptional and functional compartmentalization of the marrow microenvironment. Macrophages within non-resorptive cavities are inherently anti-inflammatory, which suppresses disease-initiating Tet2+/- cells while preserving the healthy counterpart. Time-lapse imaging further revealed non-transient association between Tet2+/- clones and CD206+ macrophages. Spatially resolved single-cell transcriptomic profiling and functional assessment revealed that physiological bone remodeling influences CD206+ macrophage plasticity and cytokine secretion which regulate the clonal burden. Additionally, anti-tumor immunity alteration within the microenvironment occurred as early as the formation of initial clones. Suppressing bone remodeling with zoledronate or targeting macrophage-associated niche factors mitigated clonal development. Collectively, our study reveals a previously unrecognized inflammatory landscape shaped by local bone remodeling. The finding presents targetable mechanisms and warrants further studies on the use and precautions of bone-modulating management in clonal blood disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/097a0fcd3f67aa857a852457d51cc39385e5cd32" target='_blank'>
              Compartmentalized inflammatory landscape and macrophage plasticity regulate Tet2+/- mediated clonal hematopoiesis.
              </a>
            </td>
          <td>
            Kevin Lee, Cih-Li Hong, Wimeth Dissanayake, Gulzada Kulzhanova, Alexander Noel Pfeffer, Haiyin Li, Senthil Sivakumar, Zi Yin, Emily R. Quarato, Lauren Benoodt, Jeevisha Bajaj, Chike Cao, Chia-Lung Wu, L. Calvi, Shu-Chi A. Yeh
          </td>
          <td>2026-01-12</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Ovarian cancer(OC) poses a significant clinical challenge due to its frequent peritoneal dissemination and development of chemotherapy resistance, contributing to poor patient outcomes. Chronic inflammation is a pivotal driver of disease initiation and progression. While traditional population-level studies fail to capture cellular heterogeneity, recent advances in single-cell technologies including scRNA-seq, spatial transcriptomics, and multi-omics, have enabled high-resolution analysis of inflammatory regulation within the ovarian cancer microenvironment(OCME). This review synthesizes how single-cell approaches have elucidated inflammation-driven remodeling of OCME components, immune cell signaling pathways, and dynamic inflammatory-immune interactions. Focusing on the “dual microenvironments” of ovarian cancer, we discuss mechanisms of immune suppression, stromal reprogramming, and therapy resistance, along with current progress and challenges in translating these insights into precise diagnostic, targeted therapeutic, and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b692e5a25b9e7346ecfe0d5da66ab9df1f209d5e" target='_blank'>
              Decoding inflammatory regulation in ovarian cancer at single-cell resolution
              </a>
            </td>
          <td>
            Jianghao Yu, Tingting Zhou, Shuangyu Chen, Jialin Zhu, Lu Xu, Yibo He, Wei Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CD4 T follicular helper (Tfh) cells coordinate humoral immune responses within germinal centers (GC) of lymphoid tissue. Despite their critical roles in vaccination and autoimmunity, the gene expression programs that define functionally distinct human Tfh states— and the molecular pathways engaged by Tfh positioned within the GC niche—remain incompletely understood. This gap has limited translational efforts to monitor or therapeutically target specific Tfh states for clinical benefit. Here, we delineate human CD4 T cell heterogeneity in tonsils and peripheral blood using trimodal single-cell sequencing and spectral flow cytometry to define epigenomic, transcriptional, and proteomic features of distinct Tfh states. Tfh with a GC-like phenotype exhibited markedly increased chromatin accessibility and both mRNA and protein expression of G protein subunit gamma 4 (GNG4). In tonsil, single-cell spatial transcriptomics defined GNG4 expression as a distinguishing feature of activated Tfh states within spatially demarcated GC compartments, with greater specificity than conventionally GC-associated features such as BCL6, TOX2, and S1PR2. In contrast, GNG4− Tfh primarily localized to nonGC regions and exhibited a resting, Th17-polarized phenotype. Together, these data highlight GNG4 as a central feature of activated, GC-positioned Tfh cell identity in humans. One Sentence Summary GNG4 expression defines activated CD4 T follicular helper cells localized to the germinal center of human lymphoid tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e07a3ba4b7925ada4a1cba1ab2a33fb871cfc9" target='_blank'>
              Multimodal analysis defines GNG4 as a distinguishing feature of germinal center-positioned CD4 T follicular helper cells in humans
              </a>
            </td>
          <td>
            Sam Barnett Dubensky, Yutong Zhu, Molly Gallagher, K. Kumashie, Tianyu Lu, Jonathan Tedesco, Nina De Luna, Katherine Premo, Yi Qi, Suzanna Rachimi, Emylette Cruz Cabrera, Bria Fulmer, Ijeoma C. Meremikwu, Ashley Carter, Sarah E. Henrickson, N. Romberg, Amy E. Baxter, Derek A. Oldridge, Laura A. Vella
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Copy-number variation (CNV) is a hallmark of prostate cancer, including PTEN loss and TP53 alterations that shape tumor progression. Whole-genome spatial DNA sequencing technologies are not yet mature or widely accessible. Spatial transcriptomics-based CNV inference (SpatialCNV) produces genome-wide CNV maps and preserves histologic and spatial context. Unlike bulk DNA assays, it enables tracing of tumor subclones across tissue architecture. SpatialCNV can also identify tumor-containing spots when histology annotation is not provided, which is an essential feature for a lot of spatial cancer studies.



 We performed a systematic benchmark across simulated and real prostate spatial transcriptomics datasets, including sections with matched whole-exome truth and standard pathology. We evaluated twelve CNV tools, spanning single-cell-derived and spatial transcriptomics-specific methods. Performance was assessed in two scenarios: without histology (spot-level tumor detection) and with histology (CNV calling). We compared different strategies (stromal/immune, pure-benign epithelium, reference-free, etc) using correlation, sensitivity/specificity, and cross-patient robustness.



 For detecting tumor-containing spots without histology, CopyKAT achieved the most reliable automatic identification when run in a reference-free mode after deconvolution-based selection of epithelial enriched spots. For CNV calling, inferCNV achieved the best performance when using a ‘pure-benign’ reference—histologically benign epithelial spots without CNV changes. We developed an automated pipeline to define pure-benign references and applied it to multi-patient cohorts, revealing spatially coherent clones shared between primary and metastatic tumors.



 SpatialCNV enables genome-wide characterization of prostate cancer clonality and metastatic relevant CNV events. Our benchmark provides pratical guidance and workflows, and supports understanding of prostate cancer clonality and spatial distribution.



 Jintong Shi. Benchmarking SpatialCNV in Prostate Cancer: tools, reference strategies, and workflows across simulated and real spatial transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5278d19e1041ace4cefd0a9530604435e619b0" target='_blank'>
              Abstract A059: Benchmarking SpatialCNV in Prostate Cancer: tools, reference strategies, and workflows across simulated and real spatial transcriptomics
              </a>
            </td>
          <td>
            Jintong Shi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a global clinical challenge due to late diagnosis, high heterogeneity, and poor prognosis. Tumor stemness has emerged as a key factor driving tumor aggressiveness and therapeutic resistance. However, the systematic characterization of high-stemness GC cells and their molecular features remains limited. We integrated single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA-seq data to identify and characterize high-stemness GC cells. Stemness scores were calculated using CytoTRACE, and malignant cells were classified into high stemness (top 25% CytoTRACE-scored cells, HighStem), dynamic transition stemness (DTStem), and low stemness (LowStem) subpopulations based on the quartile method cutoff. ScPagwas and cell–cell communication profiling were used to explore genomic instability, genetic susceptibility, and microenvironmental interactions. HighStem-specific co-expression modules were identified via high-dimensional WGCNA (hdWGCNA), and features were screened using six machine learning algorithms. A benchmark model was constructed for HighStem prediction and interpreted using SHAP analysis. HighStem GC cells exhibited enhanced intercellular signaling, metabolic reprogramming, and stemness-related pathway activity. Five genes—APMAP, MAPRE1, GLB1, TSPAN6, and CDKN2A—were identified as robust HighStem features. Spatial and bulk transcriptomic validation confirmed their tumor-specific expression and prognostic relevance. The Support Vector Machine (SVM) model incorporating these genes achieved high accuracy (AUC = 0.973) in distinguishing HighStem cells, demonstrating strong clinical utility at the scRNA-seq level. In addition, experimental validation through knockdown of core genes (APMAP, CDKN2A, TSPAN6, MAPRE1, and GLB1) in SGC7901 and HGC-27 gastric cancer cell lines revealed a significant reduction in JAK1-STAT3 pathway activity, supporting their functional involvement in tumor stemness regulation. Furthermore, knockdown of these genes increased the sensitivity of GC cells to chemotherapeutic agents like 5-FU and cisplatin, indicating their potential role in chemoresistance. This study provides a comprehensive molecular and functional characterization of high-stemness GC cells. The identified signature genes and predictive models offer novel insights into GC stemness biology and could guide personalized therapeutic strategies. Furthermore, our findings suggest that the core genes identified in this study may serve as potential biomarkers for predicting treatment outcomes and monitoring therapeutic resistance in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c82f67a78d136c3b77ab5af1d7c61164794a27" target='_blank'>
              Comprehensive molecular characterization of high-stemness gastric cancer cells using single-cell transcriptomics, spatial mapping, and machine learning
              </a>
            </td>
          <td>
            Ziyi Wang, Xuehao Li, Jin Wang, Huidong Yu, Defeng Zhao, Yan Xu, Siyu Zhou, Wanfu Men
          </td>
          <td>2025-12-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The regulation of T cell exhaustion within the tumor microenvironment plays a pivotal role in shaping the immune response to cancer and determining the efficacy of immunotherapy. However, the molecular factors governing this process in colon cancer remain poorly understood. This study investigates the expression characteristics and functional significance of the transmembrane protein CD82 in the colon cancer immune microenvironment, with emphasis on its regulatory role in CD8+ T cell exhaustion and clinical outcomes. Publicly available transcriptomic datasets were integrated with multiplex immunohistochemistry on colon cancer tissue microarrays to characterize the cell–type–specific distribution of CD82 and its associations with key markers of T cell dysfunction. CD82 expression was markedly increased in tumor-infiltrating immune and epithelial cells compared with normal tissues, particularly within exhausted CD8+ T cells. Elevated CD82 levels showed strong positive correlations with canonical exhaustion markers such as programmed cell death protein 1 and T cell immunoglobulin and mucin domain-containing protein 3. Multiplex immunohistochemical analysis further revealed that enrichment of CD82-positive epithelial regions and expansion of the CD82+TIM-3+PD-1+CD8+ T cell subset were associated with poor prognosis and were confirmed by multivariate Cox regression as independent risk factors for unfavorable survival. In patients who failed to achieve a complete pathological response following immunotherapy, exhausted CD8+ T cells exhibited significantly higher CD82 expression. Single-cell regulatory network analysis identified BATF and BHLHE40 as potential transcriptional regulators of CD82. Collectively, these findings demonstrate that CD82 promotes CD8+ T cell exhaustion, contributing to tumor progression and immunotherapy resistance in colon cancer. This study provides novel insight into the molecular mechanisms underlying immune dysfunction and offers a potential therapeutic target for reversing immunosuppression and improving immunotherapy efficacy in colon malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254f481971ba1782bc6c638ef396c6a18b14c937" target='_blank'>
              CD82-associated exhausted CD8+ T cells define prognosis and immunotherapy resistance in colon cancer
              </a>
            </td>
          <td>
            Jingfeng Zhang, Hengwei Cui, Shaoxian Wu, Hongwei Shi, Haitao Wang, Jingting Jiang
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Giant cell tumor of bone (GCTB) presents considerable complexity in tumor microenvironment (TME) because of its intricate intercellular heterogeneity and the presence of an immunosuppressive milieu. In order to understand the complex gene expression patterns and cell interactions in GCTB, we carried out a thorough investigation using single‐cell RNA sequencing (scRNA‐seq). Methods We examined scRNA‐seq data from 7091 cells that were collected after surgical removal of GCTB. Following the initial quality control process, 10 separate groups of cells were distinguished, which consisted of dendritic cells, endothelial cells, macrophage cells, mast cells, monocyte cells, neutrophil cells, tumor cells, osteoclast cells, pericyte cells, and T cells. Additional analysis uncovered distinct categories within tumor‐associated macrophages (TAMs), CD8+ T cells, and CD4+ T cells. The differentiation mechanisms of TAMs, CD8+ T cells, and CD4+ T cells were explored using pseudo‐time trajectory analysis. The CellPhoneDB study revealed the interactions between various cell types within the TME of GCTB. Results TAMs have been identified as the main infiltrating cells in GCTB. These TAMs exhibit several subtypes that are characterized by specific marker genes and functional states. The identification of several subgroups within CD8+ T cells that are involved in regulating immunological checkpoints underscores the difficulties encountered when attempting to employ immune checkpoint blockade therapy for GCTB. T cell exhaustion poses a major barrier to the efficacy of antitumor immune responses. Research suggests a strong correlation between TAMs and exhausted T cells (Texs) in the TME. The high number of regulatory T cells (Tregs) highlights the immunosuppressive nature of the immunological environment in GCTB. Significant interactions have been observed between TAMs and tumor cells, highlighting their crucial involvement in immune evasion strategies. Conclusion This scRNA‐seq study provides a general overview of the different cellular compositions and immune interactions within GCTB. The identified subtypes and communication networks provide valuable information about the immunosuppressive environment of GCTB, laying the foundation for prospective therapeutic approaches targeting specific cell types or interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77dc33f2650929ccb5e19eeed4e2c7dc7b4c11a" target='_blank'>
              Single‐Cell Sequencing Reveals the Immunosuppressive Trajectory in the Tumor Microenvironment of Human Giant Cell Tumor of Bone
              </a>
            </td>
          <td>
            Yiming Liu, Wei Luo, Yude Xu, Xiguan Yao, Libing Dai, Qiao Feng, Peigeng Wang, Weichao Yang, Yi Feng, Haixiong Miao, Suixiang Huang, Dongping Ye
          </td>
          <td>2026-01-01</td>
          <td>BioMed Research International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Spectral flow cytometry has ushered in a new era in immunology. Through the improvement of the resolution of surface and intracellular protein expression, this approach enables in depth characterization of rare immune cell subsets, such as innate lymphoid cells (ILCs), in health and disease. Due to their heterogeneity, the identification of ILCs requires the use of many lineage marker antibodies for non-ILC exclusion, together with the analysis of several transcription factor expression profiles for ILC subset distinction. Such intricacies toward their identification and their scarcity in tissues have been key factors directly limiting their characterization, particularly during tumor development and progression. We developed, optimized and validated a 25-parameter spectral flow cytometric panel for the identification of mouse ILC subsets and characterization of their phenotype and proliferation capabilities in mouse mammary tumors. The use of conjugated antibodies coupled to different fluorochromes for the analysis of lineage marker expression further allows the identification and characterization of γδ T cells, CD4
 +
 and CD8
 +
 αβ T cells, as well as CD19
 +
 B cells. Furthermore, we built a bioinformatics pipeline for unbiased immune cell clustering and marker expression analysis. We assessed this panel and downstream bioinformatics analyses on two spectral flow cytometers and found no difference in immune cell identification and clustering save for slight variations in marker intensity, inherent to the specificities of the instrument. These findings highlight the robustness of our developed approach for the identification of innate lymphoid cells in tumors, a method that can be easily implemented for day-to-day analysis of ILCs and other rare immune cell subsets.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01c7f2d47ce720b574e3d5e57f88e874523f9fc" target='_blank'>
              Development and computational analysis of high dimensional spectral flow cytometry data for the resolution of innate lymphoid cells in the mammary tumor microenvironment
              </a>
            </td>
          <td>
            Hobin Seo, Jingna Xue, Qiutong Huang, Megan Kinzel, Amisha Verma, Ngan Huynh, Zahra Jamila Ikra, Douglas J Mahoney, Jongbok Lee, Sorana A. Morrissy, Nicolas Jacquelot
          </td>
          <td>2026-01-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126eb3bb80a65b589c6798760093749f010ab57b" target='_blank'>
              Robust characterization and interpretation of rare pathogenic cell populations from spatial omics using GARDEN.
              </a>
            </td>
          <td>
            Xinming Zhang, Zhuohang Yu, Gaoyang Hao, Qi Yao, Yanmei Hu, Fuzhou Wang, Xingjian Chen, Linjing Liu, Ka-chun Wong, Xiangtao Li
          </td>
          <td>2026-01-17</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Urothelial carcinoma (UC), encompassing both bladder cancer and upper tract UC, is a biologically heterogeneous malignancy that has long complicated patient stratification and therapeutic prediction. Recent genomic and transcriptomic analyses, however, have delineated reproducible molecular subtypes across non‐muscle‐invasive and muscle‐invasive disease. These classification frameworks have refined biological understanding and improved both risk stratification and treatment selection. Emerging evidence also indicates that genomic alterations, such as FGFR3 mutations, may confer distinct immunologic phenotypes within specific transcriptomic contexts, thereby challenging prior assumptions. The advent of immune checkpoint inhibitors has transformed UC management, yet variable response rates underscore the complexity of the tumor microenvironment (TME). Single‐cell and spatial analyses have identified immune‐inflamed, ‐desert, and ‐excluded TME subtypes, with cancer‐associated fibroblasts playing a key role in immune exclusion and therapeutic resistance. Liquid biopsy, particularly circulating tumor DNA, has demonstrated utility for real‐time disease monitoring, minimal residual disease detection, and treatment stratification, as highlighted in trials such as IMvigor010 and IMvigor011. Urinary extracellular vesicles also represent promising non‐invasive biomarkers, though further standardization is required. In addition, novel therapeutic strategies—including antibody‐drug conjugates, gene therapies, and sustained‐release delivery systems—are broadening treatment options across disease stages. Collectively, these translational advances mark a paradigm shift toward precision immuno‐oncology in UC. Nonetheless, successful clinical implementation will require prospective validation, optimized analytical platforms, and sustained physician‐scientist engagement to translate these discoveries into improved patient outcomes. This review provides a comprehensive overview of recent progress in UC translational research, with a focus on genomic and transcriptomic insights, the evolving role of immunotherapy, advances in liquid biopsy, and the development of novel therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44cd39c3a6847a9cce1aefa2b7bb98a79ab335f" target='_blank'>
              Translational Advances in Urothelial Carcinoma: From Bench to Bedside
              </a>
            </td>
          <td>
            T. Iwasawa, T. Tsujino, S. Tsukahara, Hiroshi Fukushima, Daisuke Ito, E. Tomiyama, Fumihiko Urabe
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Urology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The process of migration and colonization is important in evolution; for example, modern humans experienced multiple waves of migrations out of Africa. However, no data cover the spatio-temporal patterns sufficiently to be truly informative. Metastatic cancer provides a unique in vivo model to study these processes through rapid somatic evolution. Here, we apply the high-resolution sampling technique (Dense 3D Crypt-scale Sampling) to analyze hundreds of spatially mapped micro-samples from the primary colorectal cancer and liver metastases in two representative cases. This would be analogous to recording the “out-of-Africa” events in two repeats. Our results support that liver metastases arise from polyphyletic and polyclonal seeding events where multiple, genetically distinct clones colonize a new site together. Following colonization, these multi-clonal populations can evolve into distinct spatial architectures: segregated territories formed by cells with low motility, or highly intermixed patterns driven by high motility. The colonization (or seeding) process begins within the first third of the primary tumor's progression, creating a large number of widespread but clinically undetectable micrometastatic colonies. These findings support a model where metastatic competence is not an intrinsic trait of a single “winner” clone but an emergent property of multiple concurrent clones. Collectively, our work supports metastasis as a multi-stage process initiated early in tumor development, characterized by continuous polyclonal dissemination and the formation of spatially distinct clonal architectures. This general pattern may echo the ecology of migration and colonization in organismal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d8108fc71756559fb44982b7725fb8d93fb854" target='_blank'>
              Full spatio-temporal analyses of migration and colonization in evolution—dense 3D mapping of cancer metastases provides new insights
              </a>
            </td>
          <td>
            Qihang Chen, Senmao Li, Xianrui Wu, Qing Xu, Ranran Zhu, Yongsen Ruan, Ao Lan, Zihan Liu, Jiarui Weng, Yanjiang Zhao, Xiying Xu, Xinyue Qi, Jinhong Lai, Leyi Xiao, Ping Lan, Chung-I Wu, Bingjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 Bladder cancer (BLCA) is a malignant tumor originating from the urothelial lining, characterized by a complex tumor microenvironment (TME) and heterogeneous tumor mutation burden (TMB). Cancer-associated epithelial cells (EpiCs) exhibit substantial heterogeneity during BLCA initiation and progression. Therefore, elucidating the diversity and functional states of EpiCs is essential for improving future diagnostic and therapeutic strategies.



 We integrated multi-omics datasets, including 13 single-cell RNA-seq samples, 514 bulk transcriptome profiles, and 30 whole-exome sequencing (WES) samples, to comprehensively characterize EpiC subtypes. Nonlinear dimensional reduction (UMAP) and clustering analyses were performed to identify major epithelial subsets, followed by secondary clustering. TMB values calculated from self-generated WES data were incorporated into scAB and Ro/e algorithms to determine the TMB-associated epithelial subset, ultimately identifying the key cluster Epi14. Differentially expressed genes (DEGs) of Epi14 were analyzed, and CellChat was used to infer intercellular communication networks. CytoTRACE and Monocle2 were applied to assess stemness potential and differentiation trajectories. Random survival forest (RSF) combined with DEG analysis was used to identify hub genes. Immune infiltration, drug sensitivity, and functional pathway analyses were subsequently conducted. Spatial transcriptomics were deconvoluted using spacexr, CellChat, and PROGENy to map cellular composition, signaling activity, and pathway nodes. Finally, qPCR and Western blot assays were performed to validate hub gene expression in tumor versus adjacent tissues.



 A total of 77,263 cells and 3,000 highly variable genes were included, yielding 32 annotated cell clusters. Secondary clustering combined with WES-derived TMB identified 14 epithelial subpopulations, among which Epi14 was confirmed as the key TMB-associated subset using the Ro/e algorithm. Integration of DEGs, RSF, and multi-cohort datasets revealed ABRACL and ARPC3 as the pivotal hub genes, from which a risk-score model was constructed. Notably, ABRACL expression showed a strong positive association with tumor TMB and exhibited pronounced enrichment in spatial transcriptomic tumor regions.



 By integrating multi-omics and spatial datasets, this study reveals the epithelial heterogeneity of BLCA and identifies ABRACL and ARPC3 as key TMB-associated hub genes within EpiCs. The established risk-score model and validated functional markers provide valuable insights for future mechanistic studies and potential clinical translation in BLCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d01e34654acc5d2fc42e0f3045f827529cbe0f" target='_blank'>
              Integrated single-cell and spatial transcriptomics combined with whole-exome sequencing reveal key hub genes and epithelial heterogeneity in bladder cancer
              </a>
            </td>
          <td>
            Lin Li, Qianyue Li, Jiaxin Liu, Yifan Wang, Yanjun Ma, Lei Tang, Yawei Zhao
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 446


 Background:
 Trophoblast Cell Surface Antigen 2 (TROP2) is a transmembrane glycoprotein that has emerged as a promising therapeutic target due to its frequent overexpression in diverse epithelial malignancies, including gastric cancer. Beyond its role in promoting tumor cell proliferation and survival, TROP2 has been implicated in modulating tumor-immune interactions, raising interest in its potential as a marker for immune-based therapies. However, the relationship between TROP2 expression and the cellular composition of the tumor microenvironment (TME) in gastric cancer remains insufficiently characterized.
 Methods:
 To systematically investigate the influence of TROP2 on the gastric cancer TME, we analyzed 8 publicly available cohorts/trials encompassing 1,351 clinically annotated gastric cancer samples with whole transcriptome sequencing data available. Within each cohort, TROP2 (
 TACSTD2
 ) expression was quantified and stratified into high and low groups using the median of log-transformed TPM values per cohort. We employed the xCell immune deconvolution algorithm to estimate the relative enrichment of immune and stromal cell populations, enabling pan-cohort comparisons of TME composition according to TROP2 status.
 Results:
 Across cohorts, we did not observe any immune or stromal cell populations that were consistently enriched in TROP2-high gastric cancers. However, TROP2-high tumors exhibited a reproducible and statistically significant reduction in several key components of the tumor microenvironment, most notably CD8
 +
 cytotoxic T cells, cancer-associated fibroblasts, hematopoietic stem cells, and myeloid dendritic cells. These depletions were observed with relative consistency across the majority of datasets (at least 5 out of 8 cohorts with p<0.05), while the abundance of other immune and stromal cell types showed no clear or uniform pattern associated with TROP2 expression profile.
 Conclusions:
 Our multi-cohort transcriptomic analysis demonstrates that the microenvironment profile of TROP2-high gastric cancer resembles a "cold" or immune-desert phenotype, characterized by reduced immune activation and diminished stromal support.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd56499b35d2e8cfa85e55b5c305731795e13175" target='_blank'>
              Trop-2 expression and the tumor microenvironment in gastric cancer.
              </a>
            </td>
          <td>
            Joseph J. Zhao, Ammanuel Taye, Raghav Sundar
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Carcinosarcomas (CSs) are aggressive biphasic tumors characterized by epithelial and mesenchymal components, whose histogenesis and differentiation dynamics remain poorly understood. We present single‐nucleus RNA sequencing (snRNA‐seq) analysis of six CSs (five endometrial and one ovarian) and two normal endometrial samples, profiling over 96,298 cells. By integrating transcriptomic data with inferred copy number variations (CNVs), immunohistochemistry (IHC), fluorescence in situ hybridization (FISH), and in situ hybridization (ISH) validation, we resolved the complex cellular architecture of these tumors, identified lineage‐specific programs, and revealed unexpected differentiation trajectories. snRNA‐seq was used to further refine the histopathological classification of three cases by uncovering heterologous differentiation not previously recognized: one rhabdomyogenic, one osteogenic, and, notably, one exhibiting a novel tenogenic program, defined by the expression of SCX, MKX, and TNMD. All CSs displayed a prominent mesenchymal compartment comprising both undifferentiated fibroblast‐like cells and distinct lineage committed populations, including rhabdomyoblasts (Rhab), tenoblasts (Teno), osteoblasts (Osteo), and chondroblasts (Chond). In some tumors, multiple mesenchymal identities co‐existed, and in others, differentiation gradients (e.g. immature versus mature rhabdomyoblasts) were observed. These patterns underscore the cellular plasticity and multilineage potential of the sarcomatous component. Furthermore, the expression of specialized interface markers (COL22A1, NCAM1, ACAN, CHRNG, MUSK) suggests that some tumors use structured developmental programs reminiscent of the muscle–tendon junction, enthesis, or neuromuscular junction. CNV analysis revealed tumor‐specific genomic alterations with clonal and subclonal patterns linked to differentiation state, which were validated by FISH. Altogether, this study demonstrates that CSs are not static biphasic tumors but rather complex ecosystems with extensive developmental plasticity. Our findings redefine their classification and support the use of single‐nucleus approaches to uncover hidden differentiation trajectories in highly heterogeneous cancers, including the discovery of a previously unreported tenogenic lineage. Our results challenge the diagnosis of homologous CS when only morphological criteria are applied. © 2026 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e04245f6b90707319b3f96bbd2d570520f067c" target='_blank'>
              Single‐nucleus RNA sequencing identifies a novel tenogenic heterologous differentiation in endometrial carcinosarcomas: implications for diagnosis and tumor classification
              </a>
            </td>
          <td>
            Silvia González-Martínez, José Palacios, I. Carretero-Barrio, Val Fernández-Lanza, Alfonso Cortés-Salgado, Javier Román, Xavier Matias-Guiu, S. Gatius, Javier Cortés, Belén Pérez-Miés
          </td>
          <td>2026-01-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2092223941ef46884e67c45fd51c2462a350c8db" target='_blank'>
              Unambiguous signatures of malignancies extracted from images of growing cells
              </a>
            </td>
          <td>
            G. Kalweit, M. Kalweit, W. Checinska, M. Saric, R. Berger, E. Bodurova-Spassova, J. Rawluk, N. Talvard-Balland, A. Klett, M. Follo, S. Kreutmair, J. Duque-Afonso, M. LuÌbbert, R. Zeiser, J. Frank, R. Mertelsmann
          </td>
          <td>2026-01-13</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are key drivers of tumor progression in bladder cancer (BLCA), yet their molecular heterogeneity and prognostic utility remain incompletely characterized. Single-cell studies have revealed distinct CAF subpopulations with divergent clinical impacts, necessitating refined prognostic frameworks that capture CAF-mediated progression. Methods We analyzed single-cell RNA sequencing data (GSE267718) from 8 BLCA patients to identify CAF populations and define progression-associated gene signatures. Using 359 TCGA-BLCA samples as the training cohort, we performed non-negative matrix factorization (NMF) consensus clustering on 85 prognostically significant CAF genes, identifying two molecular clusters with distinct survival outcomes. Through LASSO-Cox regression and stepwise selection, we constructed a four-gene Tumor-Progressing Fibroblast Riskscore model comprising FOXA1, TBX3, LRIG1, and RNF11. Model performance was validated in the E-MTAB-4321 cohort (n = 476). Functional validation of RNF11 was performed using shRNA-mediated knockdown in T24 and 5637 bladder cancer cell lines, followed by proliferation, migration, invasion assays, and transcriptomic profiling. Results Single-cell analysis identified 557 differentially expressed genes between non-muscle-invasive bladder cancer and muscle-invasive bladder cancer CAFs. NMF clustering stratified TCGA patients into 2 clusters with significantly different overall survival. The TPFR model showed consistent prognostic performance in both training and validation cohorts, with high-risk patients showing significantly worse survival. Functional enrichment analysis revealed that TPFR scores correlated with ECM-receptor interaction, focal adhesion, and cytoskeletal regulation pathways. Stratified analysis revealed superior model performance in elderly (>60 years), male, and early-stage patients. In particular, RNF11 knockdown significantly reduced proliferation, migration, and invasion in 5637 and T24 cells, while transcriptomic analysis revealed alterations in tumors after RNF11 knockdown including TNF and MAPK signaling pathway, indicating a potential mechanism by which RNF11 regulates bladder cancer progression. Conclusion We established a CAF-based prognostic model that integrates single-cell insights with bulk transcriptomics for robust risk stratification in BLCA. The TPFR model shows clinical utility particularly in elderly and early-stage patients. Functional characterization showed that RNF11 regulates proliferation and migration of bladder cancer. These findings highlight the prognostic value of CAF signatures and provide a framework for precision medicine approaches in bladder cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca640c3b90ed3c9d246d58fff4062a2ad248f2a" target='_blank'>
              A novel tumor-progressing fibroblast signature derived from single-cell RNA sequencing enables prognostic stratification and reveals RNF11 as a functional regulator in bladder cancer
              </a>
            </td>
          <td>
            Miaoyu Zhang, Liangbin Lin, He Xu, Xu Su, Kui Zeng, Fangyang Zhong, Xianchao Su, Jie Chen, Min Cao, Hui Yu, Hui Zhuo
          </td>
          <td>2026-01-15</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Establishing in vitro cancer models that more closely recapitulate patient tumor microenvironmental heterogeneity, including variations in stromal cells and mechanical properties that influence colorectal cancer (CRC) progression, is crucial for advancing CRC research. This study evaluated the ability of 3D engineered CRC-PDX (3D-eCRC-PDX) tissues to recapitulate the heterogeneity found between patient-derived xenograft (PDX) tumors from three CRC patients (stage II, III-B, and IV). To form the 3D-eCRC-PDX tissues, CRC-PDX tumor cells were encapsulated in PEG-fibrinogen hydrogels and maintained for 29 days in vitro. 3D-eCRC-PDX tissues recapitulated key patient-specific tumor characteristics. During long-term culture, 3D-eCRC-PDX tissues mimicked the patient-specific growth rates of the originating CRC-PDX tumors. Importantly, tumor cellular subpopulations, including the ratio of human cancer cells to mouse stromal cells and the ratios of proliferative human cancer cells and CK20+ cells were maintained in 3D-eCRC-PDX tissues, unlike in 2D cell culture. Differences in mechanical stiffness between the originating CRC-PDX tumors were also recapitulated by the 3D-eCRC-PDX tissues. Principal component analysis of transcriptomic data clustered 3D-eCRC-PDX tissues and CRC-PDX tumors together by patient, indicating similar gene expression profiles. These findings highlight the potential of 3D-eCRC-PDX tissues as a promising tool for CRC research, capable of maintaining patient-specific tumor microenvironment heterogeneity. Statement of Significance This study establishes engineered colorectal cancer tissues formed using PEG-fibrinogen and patient-derived xenograft (PDX) tumor cells for modeling inter-patient tumor heterogeneity. In vitro models that preserve patient tumors’ biological and structural heterogeneity are central to the development of more predictive and reproducible preclinical cancer models. Here engineered tissues replicated patient-specific tumor growth dynamics, sustained key cancer and stromal cell subpopulations, recapitulated originating PDX tumor stiffness, and sustained patient-specific patterns of gene expression. This work demonstrates the long-term culture of cells from patient xenografts in engineered colorectal cancer tissues, enabling sustained preservation of critical patient tumor-specific microenvironmental characteristics in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcd7cb41d4ed110d7fe21c7dde9d7a1276178d4" target='_blank'>
              Recapitulating Patient-to-Patient Colorectal Cancer Tumor Heterogeneity Using Patient-Derived Xenograft Cells in an Engineered Tissue Model
              </a>
            </td>
          <td>
            Iman Hassani, Benjamin Anbiah, Yuan Tian, Bulbul Ahmed, William J. Van Der Pol, Elliot J. Lefkowitz, Peyton Kuhlers, Nicole L. Habbit, Martin J. Heslin, Michael W. Greene, E. Lipke
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2227a7d7f1704d0d84703010e3af2b4bec3d96b9" target='_blank'>
              PRDM1 Drives Chemoradiotherapy-associated Enrichment of Adaptive NK Cells in Cervical Cancer
              </a>
            </td>
          <td>
            Meng Wan, Tangwu Zhong, Wenyang Shi, Jianyu Shen, Wei Zhang, Yizhe Sun
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a988a1774cf7ed3f81f590a732823cd8a46b829" target='_blank'>
              Wnt ligands in breast cancer metastasis: unraveling their dual roles and therapeutic potential
              </a>
            </td>
          <td>
            Azita Asadi, Mehran Molavand, Mostafa Vaghari -Tabari, Bahman Yousefi
          </td>
          <td>2026-01-12</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) frequently develops aggressive metastatic disease, yet the cellular features that enable dissemination remain poorly defined. IKKα, a kinase traditionally linked to stress and inflammatory signaling, is increasingly recognized for broader functions in cancer. Here, we show that loss of IKKα unexpectedly promotes metastasis in CRC. Using patient-derived organoids, we find that genetic or pharmacological inhibition of IKKα stabilizes tight-junction components, leading to the emergence of compact epithelial clusters with a heightened ability to spread and colonize the liver. Single-cell transcriptomics reveals expansion of a CDH17⁺/CLDN2⁺ epithelial subpopulation that dominates metastatic lesions, a finding validated by tissue staining. Remarkably, disrupting CLDN2 completely eliminates the metastatic advantage caused by IKKα loss. These results identify a metastasis-competent epithelial state driven by tight-junction remodeling and uncover a vulnerable node that may be exploited therapeutically in aggressive colorectal cancer. The IKKα kinase was previously reported to promote metastasis. Here, the authors reveal that loss of IKKα function promotes colorectal cancer liver metastasis by expanding a CDH17⁺/CLDN2⁺ epithelial subpopulation characterized by tight junction stabilization and collective migration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214214b3f4bf431d9df6a0363a7ab05464474373" target='_blank'>
              Tight junction-high and CDH17-positive cell population is the source of colorectal cancer liver metastases
              </a>
            </td>
          <td>
            Daniel Álvarez-Villanueva, Mariano R Maqueda, Dina Harti, E. Cantón, Evelyn Andrades, Joan Bertran, M. Martínez-Iniesta, Laura Solé, Violeta García‐Hernández, Ángela Montoto, T. Lobo-Jarne, J. Alonso-Marañón, Patricia Herrero-Molinero, Monica Larrubia-Loring, Anna Tramuns, Kieran Wynne, Indrani Bera, D. Matallanas, Alberto Villanueva, A. Bigas, Mar Iglesias, Lluís Espinosa
          </td>
          <td>2026-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="BACKGROUND
The development of gemcitabine resistance significantly diminishes treatment efficacy and worsens prognosis in patients with pancreatic ductal adenocarcinoma (PDAC). A comprehensive understanding of the molecular and cellular mechanisms driving gemcitabine resistance is thus essential for optimizing treatment strategies and improving survival outcomes in PDAC.


METHODS
An integrated analysis of bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics was performed to elucidate the molecular mechanisms of gemcitabine resistance. Tumor-associated epithelial and myeloid cell subsets were identified, and their interactions were characterized using clustering, pseudotime analysis, and cell-to-cell communication networks. Findings were validated through molecular and cellular experiments in PDAC cell lines, as well as multicolor immunofluorescence staining in clinical cohort samples.


RESULTS
The findings revealed that PRRX2+ epithelial cells (EC) and SPP1+ tumor-associated macrophages (TAM) were enriched in gemcitabine-resistant samples, with robust cell-to-cell communication observed between these two populations. PRRX2+ EC cells and SPP1+ TAM cells interacted through the TGFB1/TGFBR2 axis, promoting epithelial-mesenchymal transition (EMT) and extracellular matrix (ECM) remodeling, thereby contributing to drug resistance. The transcription factor PRRX2 was identified as a central regulator of this resistance mechanism, amplifying TGF-β signaling through TGFB1 to induce EMT and activating Fibronectin 1 to remodel the ECM, thereby enhancing an immunosuppressive tumor microenvironment. Furthermore, experimental downregulation of PRRX2 in pancreatic cancer cell lines resulted in reduced tumor cell migration. High expression levels of PRRX2+EC/SPP1+TAM were associated with poor prognosis and may indicate susceptibility to Dasatinib and antibody-drug conjugates (ADCs) targeting TROP2, MUC1, and CEACAM5.


CONCLUSIONS
The results revealed the critical role of PRRX2+EC/SPP1+TAM cell communication in mediating gemcitabine resistance in PDAC. These findings provide new insights into potential therapeutic strategies, highlighting the importance of targeting the PRRX2+EC/SPP1+TAM axis and supporting the use of multidrug regimens to overcome resistance and improve patient outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/add5ea3cff49a3e2c5c93db3e7092d7e12027eec" target='_blank'>
              Single-cell and spatial transcriptomics reveal the interaction between PRRX2-driven epithelial cells and SPP1+ macrophages in mediating gemcitabine resistance in pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Jingmei Liu, Zhilong Li, Guanhua Liu, Zhihua Pei, Weilin Liu, Wenhui Yang, Feng Li, Lijuan Huo
          </td>
          <td>2025-12-26</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer characterized by the absence of estrogen, progesterone, and HER2 receptor expression. This malignancy is often associated with a poor prognosis, early recurrence, and limited treatment options. The tumor microenvironment (TME) in TNBC plays a pivotal role in tumor progression, immune evasion, and therapeutic resistance. In recent years, an increasing body of evidence has highlighted the critical interactions between cancer cells and the components within the TME, including immune cells and soluble components. These interactions influence not only the biological behavior of the tumor but also its response to treatment. Exploring the complex interplay between tumor cells and immune components continues to inform the development of more effective therapeutic approaches. In this study, we provide a synopsis of advancements regarding the TME in TNBC. In light of different cellular compartments, we delineate multiscale interplays within the stroma–tumor symbiosis and highlight their antitumor functions and promising targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bfc269114236644c5ca2675c1d14983686fe40" target='_blank'>
              Prominent protumoral cellular compartments of the tumor microenvironment in triple-negative breast cancer
              </a>
            </td>
          <td>
            Shuai Sun, Pan Zhao, Changrong Wang, Junjun Du, Tingting Zhang, Xiangyun He, Zhibo Zuo, Nan Li, Rongjing Zhou
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c745807b38045a754b3f57a50a75c7b86c914ca" target='_blank'>
              A Scalable Framework for Pan-Cancer Tumor Evolution Analysis Enables Transfer of Progression Mechanisms Across Tumor Entities
              </a>
            </td>
          <td>
            Simon Pfahler, Andreas Lösch, Y. L. Hu, Rudolf Schill, Rainer Spang, T. Wettig
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Single-cell and spatial transcriptomics technologies have transformed the landscape of cardiovascular research, leading to novel insights into cellular heterogeneity and tissue architecture in health and disease. These technologies enable researchers to deconvolute complex tissues and map gene expression patterns within their spatial contexts, providing critical information on the interplay between cell types and pathways affecting tissue regeneration or progression to fibrosis. This review presents an overview of the recently developed applications of single-cell and spatial transcriptomics methods and their impact on cardiovascular research. We discuss the principles underlying emerging solutions to process fixed and low-integrity RNA samples like formalin-fixed paraffin-embedded tissues. In addition, we highlight advances in high-resolution spatial transcriptomics assays, from imaging-based techniques like MERFISH (multiplexed error-robust fluorescence in situ hybridization) and Xenium to sequencing-based platforms like Visium HD, Stereo-seq, and Open-ST, each contributing unique strengths for tissue-level analysis. The integration of these technologies with machine learning and multiomics approaches further enhances the ability to uncover novel biology. These approaches have already led to the discovery of spatially resolved gene expression patterns in human atherosclerosis, hypertrophic cardiomyopathy, myocardial infarction, and myocarditis. These case studies showcase how these methods can be applied to decode the cellular and molecular dynamics of human disease processes, identify potential novel therapeutic targets, and enable predictive modeling of cellular perturbations and patient disease trajectories. We provide a comprehensive overview of the growing data repository landscape, the principles and applications of novel machine learning approaches, which are becoming more and more standard analytical tools. Integrating these areas underscores how recent advancements in single-cell and spatial transcriptomics are offering an increasingly detailed and comprehensive understanding of the cellular landscape of cardiovascular disease while also highlighting the challenges and future directions that will shape innovations in cardiovascular biology and medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31765e8355b6603e6f061451c773b6eba179cda2" target='_blank'>
              Advancing Cardiovascular Research With Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Kai Li, Samaneh Samiei, Daryna Pikulska, Sebastian Foecking, C. Kuppe
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5108b05893ed4ed9a64a7f2a10e9459a930c1b89" target='_blank'>
              Analysis of the transcriptomic and metabolomic landscape of prostate cancer with different anatomical origins using snFLARE-seq and mxFRIZNGRND.
              </a>
            </td>
          <td>
            Dongyin He, Haoran Hu, Kai Xiao, Yuhang Zhang, Yongbing Cheng, Shaozhuo Jiao, Yimei Hao, Yuanyuan Cai, Ziqun Liu, Xinran Yan, Qinsheng Chen, Xiyan Mu, Qi Wang, Shan Peng, Guoqin Sang, Xiaoling Zhi, Yanxia Chang, Qing Ye, Yuyao Yang, Meixia Che, Shengsong Huang, Hongqian Guo, Luonan Chen, Huiru Tang, Xuefeng Qiu, Zhenfei Li
          </td>
          <td>2026-02-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The mechanisms by which a small subset of high-risk localized prostate cance becomes immunogenic is not well understood. The objective of this study is to identify the tumor intrinsic and extrinsic mechanisms associated with immunogenicity in this population with the goal of harnessing this knowledge to further immunotherapy in PCa. Methods We performed Visium and Xenium spatial transcriptomic profiling on radical prostatectomy specimens from untreated, high-risk, localized prostate cancer patients obtained from the UCLA biobank with varying level of intra-tumoral T-cell infiltration (TILs). Validation was performed using OPAL multiplex immune-fluorescence (mxIF) staining for CD4, CD8A, CD68, PIGR (marker of club cells), and PSA on 18 specimens from a separate cohort of untreated, intermediate to high-risk localized prostate cancer. Summary Spatial transcriptomic analysis identified cells with club-like features to be associated with global T cell infiltration. Both benign epithelium and PCa can adopt the club-like features by the expression of PIGR protein. Benign glands with club-like features have reduced androgen receptor activity while club-like PCa are enriched with PTEN loss by IHC. Regardless of cell type, expression of PIGR leads to close contact of T cells and club-like cells. Xenium profiling revealed that most T cells are activated CD4+ stem T cells expressing TCF1 protein that are spatially enriched with myeloid dendritic cells and plasmacytoid dendritic cells. Conclusions We defined a subset of immunogenic localized prostate cancer that expresses PIGR either in benign or tumor epithelium. Epithelial expression of PIGR is associated with TILs, suggesting a non-canonical route for immune recruitment and activation.



 Anson Ku, Rong Rong Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, Ross Lake, Rosina Lis, Nikil Pramod, Shana Trostel, Anna Baj, Etyan Ruppin, Scott Wilkinson, John Fenimore, Huihui Ye, Adam Sowalsky. Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae61f5a94b5b745d520039bbb08b848e6edeb40a" target='_blank'>
              Abstract A035: Tumor PTEN-Loss is Associated with Club-Like Cells Phenotype and CD4 T-Cell Infiltration in High-Risk Localized Prostate Cancer
              </a>
            </td>
          <td>
            A. Ku, R. Huang, Sumeyra Kartel, Xingyu Chen, Kayla Heyward, R. Lake, Rosina T. Lis, Nikil Pramod, S. Trostel, A. Baj, Etyan Ruppin, S. Wilkinson, J. Fenimore, H. Ye, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afcc9cdde9fce44c9f1c91b8acb720e8fa688da9" target='_blank'>
              Adenosine A2B Receptor Promotes Tumor Progression and Metastases in Undifferentiated Pleomorphic Sarcoma.
              </a>
            </td>
          <td>
            Mariella Spalato Ceruso, J. Guégan, A. Bourdon, V. Chaire, Yanina Valverde Timana, Alban Giese, C. Rey, A. Bessede, Raoul Perret, Lucile Vanhersecke, Valérie Velasco, P. Finetti, F. Bertucci, S. Verbeke, A. Italiano
          </td>
          <td>2026-02-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Background/Objectives: Constructing a comprehensive spatiotemporal map of tumor heterogeneity is essential for understanding tumor evolution, with copy number variation (CNV) being a significant feature. Existing studies often rely on tools originally developed for single-cell data, which fail to utilize spatial information, often leading to an incomplete map of clonal architecture. Our study aims to develop a model that fully leverages spatial omics data to elucidate spatio-temporal changes in tumor evolution. Methods: Here, we introduce SCOIGET (Spatial COpy number Inference by Graph on Evolution of Tumor), a novel framework using graph neural networks with graph attention layers to learn spatial neighborhood features of gene expression and infer copy number variations. This approach integrates spatial multi-omics features to create a comprehensive spatial map of tumor heterogeneity. Results: Notably, SCOIGET achieves a substantial reduction in error metrics (e.g., mean squared error, cosine similarity, and distance measures) and produces superior clustering performance, as indicated by higher Silhouette Scores compared to existing methods, validated by both simulated data with spot-level ground truth and patient cohorts. Our model significantly enhances the accuracy of tumor evolution depiction, capturing detailed spatial and temporal changes within the tumor microenvironment. It is versatile and applicable to various downstream tasks, demonstrating strong generalizability across different spatial omics platforms, including 10× Visium and Visium HD and various cancer types, including colorectal cancer and prostate cancer. This robust performance improves research efficiency and provides valuable insights into tumor progression. Conclusions: SCOIGET offers an innovative solution by integrating multiple features and advanced algorithms, providing a detailed and accurate representation of tumor heterogeneity and evolution, aiding in the development of personalized cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6913fca8817dbe011dd13361f41522851e540c4" target='_blank'>
              Toward Graph-Based Decoding of Tumor Evolution: Spatial Inference of Copy Number Variations
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-12-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d17d3f47bf6fa9c2700d80432877f7b37ee2552" target='_blank'>
              Refined cellular activity expression signatures provide a targeted framework to quantify phenotypic intra-tumor heterogeneity in single-cell data
              </a>
            </td>
          <td>
            Angèle Coutant, Lauriane Muller, Cyril Dégletagne, Frédérique Fauvet, Islam El Boudi, Christelle Chassot, J. Valladeau-Guilemond, N. Bendriss-Vermare, A. Morel, Maria Ouzounova, Pierre Saintigny, Alain Puisieux, Arnaud Vigneron, Pierre Martinez
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e2990159413c5229bc09533053b7a0234e2d63" target='_blank'>
              Distinct tumor immune microenvironmental (TIME) landscapes drive divergent immunotherapy responses in glioblastoma.
              </a>
            </td>
          <td>
            Linqian Weng, Krish Skandha Gopalan, Mélanie Guyot, M. Vanmechelen, Pouya Nazari, Marie Duhamel, Clelia Donisi, Carla Pallarés-Moratalla, A. Garg, K. White, A.T. Byrne, D. Lambrechts, F. De Smet, G. Bergers
          </td>
          <td>2026-01-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent epithelial malignancy. Immune-checkpoint inhibitors have reshaped first-line therapy for recurrent/metastatic disease; yet durable benefit is confined to a subset, reflecting myeloid-centric mechanisms—SPP1+ TAM barriers, cDC1/IL-12 insufficiency, and CXCL8–CXCR1/2–driven neutrophil trafficking—distinct from, and complementary to, classical lymphoid exhaustion. In this review we summarize advances from single-cell RNA and ATAC profiling and spatial transcriptomics that resolve macrophage, dendritic-cell and neutrophil programs, and appraise translational opportunities spanning myeloid reprogramming, innate–adaptive combinations and spatial biomarkers. We also discuss enduring challenges—including HPV-status heterogeneity, limited assay standardization and a scarcity of predictive metrics—that temper implementation. By integrating myeloid-informed readouts (e.g., SPP1–TAM burden, cDC1 competency, serum IL-8) with PD-1–based regimens, EGFR-directed antibodies and myeloid checkpoints (CD47–SIRPα, PI3Kγ, CXCR1/2), emerging strategies aim to restore antigen presentation, improve lymphocyte trafficking and remodel tumor–stroma interfaces. Our synthesis provides an appraisal of the evolving landscape of myeloid-informed precision immuno-oncology in HNSCC and outlines pragmatic standards and avenues for clinical translation. We hope these insights will assist researchers and clinicians as they endeavor to implement more effective, individualized regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1e89f3eb622744a2cb356a1f52faae72322d69" target='_blank'>
              Myeloid-driven immunosuppression in head and neck cancer: single-cell ATAC/RNA and spatial transcriptomic perspectives
              </a>
            </td>
          <td>
            Rui Luo, Jianzheng Yang, Zimeng Cao, Bing Li
          </td>
          <td>2025-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) provides valuable insights into the tumor microenvironment by integrating molecular features with spatial context; however, its clinical utility is limited by high costs. To address this, we develop a multi-scale convolutional deep learning framework, HiST, which utilizes ST to learn the relationship between spatially resolved gene expression profiles (GEPs) and histological morphology. HiST accurately predicts tumor regions across multiple cancer types (e.g., breast cancer; area under the curve: 0.96), demonstrating high concordance with pathologist annotations. Moreover, HiST reconstructs spatially resolved GEPs from histological images with an average Pearson correlation coefficient of 0.74 across five cancer types, outperforming existing models by about two-fold. These high-fidelity spatial GEPs enable tumor heterogeneity assessment from histological images, including identification of tumor subtypes with distinct DNA copy number variations. We demonstrate the clinical utility of the predicted GEPs, which robustly stratify patient prognosis across five cancer types from The Cancer Genome Atlas (e.g., breast cancer; concordance index: 0.78). The predicted profiles further facilitate immunotherapy response prediction and enrichment analyses of relevant biological pathways and markers. Collectively, HiST achieves state-of-the-art performance in spatial GEP reconstruction, providing a reliable molecular representation that enhances downstream tasks such as tumor profiling and clinical analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1a859f42b2a4317c7e56d1a0957cb174c649e8f" target='_blank'>
              HiST: Histological Images Reconstruct Tumor Spatial Transcriptomics via MultiScale Fusion Deep Learning.
              </a>
            </td>
          <td>
            Wei Li, Dong Zhang, Eryu Peng, Shijun Shen, Hamid Alinejad-Rokny, Yao Liu, Junke Zheng, Cizhong Jiang, Youqiong Ye
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98cf179f4f4e7762fc39a8377027aab26eafcc9e" target='_blank'>
              Image-Based Profiling of Induced Trophoblast Stem Cells Identifies Signatures Associated with Sex, Schizophrenia Genomic Risk and Placental Stress
              </a>
            </td>
          <td>
            Frank J. Piscotta, Jiyoung Kim, Jisu Ha, Bonna Sheehan, Julia Johnston, John Peters, Thomas M. Hyde, Brady J. Maher, Juan Caicedo, D. Weinberger, Evgeny Shlevkov, Gianluca Ursini
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fa9eb5623540917fa3209ad074268de213852" target='_blank'>
              Molecular and spatial profiling identifies immune endotypes for the stratification of OA patients
              </a>
            </td>
          <td>
            Nicolas Gaigeard, Anaïs Cardon, Romain Guiho, C. Çubuk, Astrid Ouattara, Célia Lamothe, Jordan Brouard, Julien De Lima, Manzoor Ahmed, Mathilde Le Mercier, A. Defois, Lucie Danet, Jimmy Perrot, N. Benzerdjeb, C. Vinatier, R. Danger, L. Delbos, Sophie Brouard, N. Degauque, Simona Pagliuca, Myles J. Lewis, L. Fossati-Jimack, A. Nerviani, D. Waast, Benoit Le Goff, Frédéric Blanchard, David Moulin, Costantino Pitzalis, Jérôme Guicheux, M. Boutet
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Neoadjuvant therapy (NAT) is increasingly used for pancreatic ductal adenocarcinoma (PDAC); yet most patients only achieve partial response. Pathological treatment response grading focuses on assessing residual tumor burden, often overlooking changes in tumor microenvironment (TME). To address this gap, we compared tumor cells and TME of 13 NAT-naïve and 23 post-NAT PDACs using integrated spatial pathomics and transcriptomics, with validation in an independent single-cell spatial dataset. NAT significantly reduced tumor burden (14.7%-6.2%, p = 0.004), but systemic comparison of 13 cytomorphometric features of tumor cells alone did not reliably distinguish between naïve and NAT cases. In contrast, NAT profoundly remodeled TME by increasing cancer-associated fibroblast (CAF) and CD8+ T cell densities, promoting CD8+ T cell-tumor cell proximity and fibrosis, reducing tumor-associated neutrophils, and redistributing tertiary lymphoid structures (TLSs). Spatial transcriptomics shows NAT induced apoptosis, DNA-damage response, and AGC-kinase (S_TK_X) signaling in tumor cells, and upregulated complement pathway, p53 signaling, and cellular senescence program in TME. Cross-platform single-cell spatial analysis revealed decreased regulatory T cells (Treg) and a shift from myofibroblastic (mCAF) to inflammatory CAF (iCAF). Importantly, post-NAT patients with more fibrosis had longer overall survival (p = 0.02), and higher B-cell density showed a favorable trend (p = 0.06). Together, these results suggest that beyond tumor debulking, NAT induces a coordinated TME remodeling characterized by fibroblast reprogramming, matrix fibrosis, and immune spatial reorganization. Incorporating assessment of NAT-induced stromal and immune changes into TRG may improve prognostication and guide more precise therapy in post-NAT PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4f312a97dc39d70b704d780122da11f4cc3af9" target='_blank'>
              Neoadjuvant Therapy-Induced Remodeling in Pancreatic Ductal Adenocarcinoma: Multimodal Spatial Analysis and Prognosis.
              </a>
            </td>
          <td>
            Xiaofei Zhang, Ruoxin Lan, Danting Li, Yongjun Liu, Sonu Kalyan, Momin Iqbal, Nancy Liu, Jerry Zhang, Iman Hanna, Mala Gupta, C. Zhao, Weiguo Liu, Jonathan Melamed, Michael Shusterman, Jessica Widmer, John Allendorf, Yao-zhong Liu
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rationale: Tertiary lymphoid structures (TLSs) play a key role in the adaptive immune response within the local tumor immune microenvironment (TIME) and can predict the clinical outcome of several solid tumors. However, the clinical relevance of TLSs and their formation mechanism in gastrointestinal stromal tumors (GISTs) remain unclear. Methods: Integration analysis was performed on a single-cell RNA sequencing (scRNA-seq) cohort (n = 8), a public transcriptome cohort (n = 65), a public scRNA-seq cohort (n = 7), a tissue cohort (n = 197) and a serum cohort (n = 169) to decode the characteristics of the immunological microenvironment of GIST. The multiplex immunohistochemistry (mIHC) staining, in vitro cell culture, chemotaxis assays and antibody-dependent cellular phagocytosis (ADCP) experiments were used to validate the results of the bioinformatics analysis. Results: Preoperative imatinib targeted therapy significantly enhanced TLS formation in GIST tissues, predicting improved therapeutic efficacy and favorable prognosis. Mechanistically, imatinib remodeled the local microenvironment via tumour-associated macrophages, recruiting B cells via the CXCR4-CXCL12 axis to drive TLS development. Functionally, TLS dominated germinal centre (GC) B-cell differentiation and the formation of IgG+ plasma cells (PCs), which preferentially enhanced the adaptive immune response through the ADCP effect. From a clinical perspective, we identified three distinct GIST immune classes (GICs A-C). GIC-A tumors featured abundant CD20+ B cells and TLS, as well as a favorable prognosis. They were accompanied by enhanced antigen presentation, accumulation of IgG+ PCs, increased immunosuppressive properties and a high frequency of KIT exon 11 mutations. These mutations potentially serve as a predictive biomarker for future targeted and immunotherapies. Furthermore, patients with high serum IgG levels experienced significant therapeutic benefits. Conclusions: Our data show that local adaptive immunity dominated by TLS is a key factor in the efficacy of targeted therapy, and suggest that inducing IgG could be a feasible strategy for improving the prognosis of patients with GIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/485f7a996f0585ddffcf4dd3ac8a1112abd5a345" target='_blank'>
              Enhanced formation of tertiary lymphoid structures shapes the anti-tumor microenvironment in gastrointestinal stromal tumors after imatinib targeted therapy
              </a>
            </td>
          <td>
            Ping Tao, Jiongyuan Wang, Peidang Fan, Wen-Shuai Liu, Wei-Qi Lu, Yujie Hu, Q. Lu, Lijie Ma, Hanxing Tong, Yong Zhang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/706f238b315ed6aac316adf86f94df56f8c2fd50" target='_blank'>
              An integrated single-cell lung cancer atlas reveals distinct fibroblast phenotypes between adenocarcinoma and squamous cell carcinomas.
              </a>
            </td>
          <td>
            Yuta Hirano, Hironori Suzuki, Jun Nakayama, Tomofumi Yamamoto, Eishu Hirata, Jun Araya, Yu Fujita, Yusuke Yamamoto
          </td>
          <td>2026-01-23</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is critically implicated in glioblastoma (GBM) recurrence, therapeutic resistance, and immune evasion. By analyzing both primary and recurrent GBM from Chinese Glioma Genome Atlas (CGGA) and clinical samples, we identified significant differences in TME. To overcome the limitations of traditional murine recurrent GBM model, we successfully established a new murine model and characterized the TME differences between primary and recurrent GBM in vivo. Based on immune chemokine profiling within the CGGA dataset, we selected PLX3397, a Colony stimulating factor 1 receptor (CSF1R) inhibitor, to target recurrent GBM in our murine model. PLX3397 treatment significantly attenuated tumor growth and remodeled TME. Collectively, our study firstly analyzed TME combine a large number of database samples and clinical samples, and made the first application of PLX3397 in a murine recurrent GBM model, thereby providing a novel therapeutic strategy and experimental foundation for recurrent GBM research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a92fc4a3398e8830aed11264d560ebd40da05b6" target='_blank'>
              PLX3397 attenuated tumor growth and remodeled tumor microenvironment of recurrent glioblastoma
              </a>
            </td>
          <td>
            Chen Yang, Linzhi Cai, Xi Xiang, Xiwei Zhu, Chengli Liu, Jianming Liao, Zhan Zhang, Mingchang Li
          </td>
          <td>2025-12-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Most deaths from cancer are caused by their metastases. Along with the ability to evade the immune system and to invade and disrupt very different tissue environments, metastases usually develop resistance to whatever therapeutic approaches are tried. Their extreme adaptability requires a diversity of traits from which natural selection can choose, but current models, such as the epithelial-to-mesenchymal transition (EMT), do not directly address how this diversity is generated. We observed that single cell clones of Panc1 human pancreas cancer cells that had had crispR knock outs (KO) of the gene for activin-like kinase 4 (ALK4) had developed a markedly diverse morphology. Time-lapse and fluorescence microscopy, FACS analysis and mitotic chromosome squashes provided evidence that the cells diversify profoundly in size, behavior and chromosome number. This diversity appears to develop through cytokinesis failure, the generation of very large multinuclear cells that exhibit a high range of nuclear configurations, coupled with continued cell divisions that sometimes generate three, four, and more daughter cells in one cell division. This astounding activity often resulted in cells that were much smaller than the normal Panc1 cells, and indirect evidence suggests they were significantly sub-ploidy, yet underwent regular cell divisions. A subset of the smaller cells were highly motile, and were observed to sometimes merge together or even into larger cells. This phenomenon could provide an unappreciated venue for transferring mutated genes, chromosome fragments, whole chromosomes or sets of chromosomes into an invaded cell. We suggest that these observations may be a serendipitous illustration of the missing link between the genetic mutations implicated in the development of primary carcinoma, and the extreme genomic diversity that characterizes malignant metastases. Graphical Summary Single cell clones of Panc 1 ALK4 KO cultures changed from largely uniformly epithelioid in character (A) to cultures composed of an unexpectedly wide variety of cells, and that this variety is not explained by the traditional “EMT” (epithelial-to-mesenchymal-transition). These cells range from very small to very large, have varying degrees of motility, chromosome number, and cell division behavior. We describe the new morphologies and behaviors, and present time lapse movies and microscopic evidence that help explain the evolution of this variety. These new cell types integrate into colonies where the smallest cells gather into spherical masses (D). We believe that the heterogeneity in genomic content and behavior provides the variety needed for natural selection to select cells that are resistant to most treatments for metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6b16627321b67c2cf57a151fe0aa1bc2c09235" target='_blank'>
              A Model for the Diversity Explosion Fundamental to Metastasis
              </a>
            </td>
          <td>
            Edward Tim O’Brien
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer‐related mortality worldwide, with tumor microenvironment (TME) heterogeneity playing a critical role in disease progression and therapeutic response. Immune escape (IE) mechanisms facilitate tumor evasion from host immune surveillance, yet their characterization at the single‐cell level in CRC is incomplete. This study integrated single‐cell RNA sequencing (scRNA‐seq) and bulk transcriptomic data from multiple public cohorts to systematically explore IE‐related signatures in CRC. We identified major and minor cell populations within the TME and performed differential gene expression analysis. Using high‐dimensional weighted gene coexpression network analysis (hdWGCNA), we identified gene modules correlated with IE activity. Subsequent survival analysis across six independent cohorts revealed Gamma‐glutamylcyclotransferase (GGCT) as a novel prognostic biomarker associated with poor survival. Functional enrichment analysis indicated GGCT′s involvement in critical oncogenic pathways. Furthermore, GGCT expression correlated with altered immune infiltration profiles and stromal components, suggesting its role in modulating the immunosuppressive TME. Additionally, GGCT demonstrated potential predictive value for response to immunotherapy across multiple datasets. Our findings highlight GGCT as a key player in CRC immune evasion and a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcee0e763a97a07d783d6f8f51f97fc2a1182ca" target='_blank'>
              Integrative Multiomics Nominate GGCT as a Crucial Regulator of Immunosuppression in Colorectal Cancer
              </a>
            </td>
          <td>
            Qichao Niu, Yang Liu, Kejin Huang, Lisheng Nie, Shiming Zhao, Shifeng Yang, Changlei Su, Shuyu Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="(Isocitrate dehydrogenase) IDH-mutant astrocytoma is classified as World Health Organization (WHO) grade 2–4 and is second only to IDH wild-type glioblastoma in the incidence of adult glioma. However, few studies use single-cell and spatial transcriptome sequencing to analyze its malignant progression. Intraoperative navigation and yellow fluorescence visualization were utilized to accurately isolate high-grade (WHO grade 3–4) and low-grade (WHO grade 2) samples of IDH-mutant astrocytoma for single-cell and spatial transcriptome sequencing. By combining single-cell, spatial transcriptome, The Cancer Genome Atlas (TCGA), and The Chinese Glioma Genome Atlas (CGGA) data, analyses of survival, enriched pathways, transcription factors, intercellular communication, differentiation trajectories, and immune response were performed to identify the characteristics of a unique subpopulation of high-grade IDH-mutant astrocytoma. Our single-cell RNA sequencing analysis identified a distinct subpopulation (Cluster 7) present in high-grade IDH-mutant astrocytoma, which was localized to the terminus of the pseudotime trajectory. Importantly, this cluster not only exhibited an immunosuppressive phenotype correlated with poor clinical prognosis, but also demonstrated significant enrichment in Developmental Biology and Calcium Signaling pathways. Furthermore, this subpopulation engaged in prominent ligand-receptor interactions, particularly through PTN_PTPRZ1 and MIF_CD74 pairs. Notably, comparative analysis revealed that high-grade astrocytoma displayed both quantitatively and qualitatively enhanced communication networks when compared to their low-grade counterparts. Our single-cell RNA sequencing analysis identifies a distinct tumor cell subpopulation present in high-grade (WHO grade 3–4) adult IDH-mutant astrocytoma. This cluster, which likely arises from malignant progression in adult astrocytoma, may provide new insights for developing therapeutic strategies against this clinically challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586fc47b7312366f86c32ecb5b1181c5a720190b" target='_blank'>
              Single-cell and spatial transcriptome sequencing analysis reveals characteristics of a unique subpopulation in high-grade IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Wei Chen, Min Zheng, Jia-Yin An, Guo-Hao Huang, Jing-Peng Liu, Lin Yang, Peng Ren, Tingting Wang, Jean-Philippe Hugnot, Sheng-Qing Lv
          </td>
          <td>2025-12-29</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2438dcb543ce3b34986c62da919180266191c9c9" target='_blank'>
              Optimised fluorescence-activated nuclei sorting for epigenomic analysis of cortical cell types
              </a>
            </td>
          <td>
            Barry Chioza, Stefania Policicchio, Joe Burrage, Georgina E T Blake, Rosemary A. Bamford, Alice Franklin, Darren Soanes, Philippa M. Wells, Ann Babtie, Marina Flores Payan, Jonathan P. Davies, Anthony Klokkaris, Emma M. Walker, Joy N. Ismail, Paulina Urbanaviciute, S. Marzi, E. Hannon, J. Mill, E. Dempster
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a lethal malignancy lacking reliable biomarkers for precancerous lesion monitoring and intervention. In this study, formalin-fixed, paraffin-embedded archived tissue biopsies from 134 individuals across two cohorts are collected, spanning five stages of ESCC progression. Laser capture microdissection is employed to isolate the epithelial lesion (L) and the adjacent non-lesion (N) tissues. Proteomic profiling reveals a comprehensive spatio-temporal landscape of ESCC progression, encompassing 4461 proteins. Dynamic network biomarker analysis indicates moderate dysplasia (MOD) as the critical turning stage, warranting clinical attention. A seven-protein diagnostic panel (CCDC86, GBP6, PDCD6IP, C19orf53, SF3A3, GMPPB, ARPC5) with an area under the curve (AUC) of 0.956 achieves superior early detection, and the key signatures are validated by immunohistochemistry in an independent cohort. Functional validation using malignantly transformed HET-1A cells, ESCC cells, and mouse xenograft models identifies GBP6 as a novel target: DNA damage-induced progressive loss of GBP6 promotes ESCC progression by accelerating cell cycle and inducing epithelial-mesenchymal transition. Critically, PARP1 inhibition rescues GBP6 loss by suppressing TP63 and prevents ESCC progression. Overall, this study provides a systematic proteomic atlas of ESCC progression, identifies MOD as a pivotal clinical decision point, and proposes PARP1-TP63-GBP6 axis targeting as a novel intervention strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf5c8a9af6fc57009a93bae9671de888e2c59fc" target='_blank'>
              Spatio-Temporal Proteomic Landscape Reveals Early Warning Signals of Esophageal Squamous Cell Carcinoma Progression.
              </a>
            </td>
          <td>
            Xumiao Li, Jie Yuan, Min Gao, Jibin Liu, Qinqin Wang, Yaqi Zhang, Mingtao Cao, Xiaolin Hu, Hui Yang, Jun Li, Chen Li, Xiaoguang Li, Hui Wang
          </td>
          <td>2026-01-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma is the most common form of kidney cancer. Recent advances in spatial and long-read sequencing now allow detailed examination of gene activity within tumours. We used non-passaged, patient-derived tumour organoid models which preserve the structure and cellular composition of the original tumours. Long-read spatial transcriptomics was then applied to study gene expression and transcript variants across organoid regions. We also examined how these patterns change following treatment with the experimental medicine NUC-7738. There are distinct spatial patterns in genes related to protein synthesis and energy metabolism, as well as region-specific differences in transcript isoforms. This work provides new insight into the molecular diversity of kidney cancer and demonstrates how advanced sequencing technologies can be used to study treatment effects in physiologically relevant tumour models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ea98f536949abee4bc7391c3123466548631ed" target='_blank'>
              Long-Read Spatial Transcriptomics of Patient-Derived Clear Cell Renal Cell Carcinoma Organoids Identifies Heterogeneity and Transcriptional Remodelling Following NUC-7738 Treatment
              </a>
            </td>
          <td>
            Hazem Abdullah, Ying Zhang, Kathryn Kirkwood, Alexander Laird, Peter Mullen, David J. Harrison, Mustafa Elshani
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43773501df06a12f0f087012a397c0411bf828e5" target='_blank'>
              Differences in regional developmental origins of glioblastomas revealed by integrated molecular profiling
              </a>
            </td>
          <td>
            Ryan Gensler, A. K. Ahmed, Melanie Alfonzo Horowitz, J. Khalifeh, Marvin Li, Negar Sadeghipour, Mostafa Abdulrahim, Mike Glantz, Sonikpreet Aulakh, Theodore Nicolaides, Calixto-Hope G. Lucas Jr., Srinivasan Yegnasubramanian, Jordina Rincon-Torroella, Chetan Bettegowda
          </td>
          <td>2026-01-19</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b75da29edbdbad60fea18ff4b3049824f5017d" target='_blank'>
              Critical role for a high-plasticity cell state in lung cancer.
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Gary Guzman, Klavdija Krause, Emma Brown, Zeda Zhang, Hannah Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2026-01-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eec9f36f629bd2b6951e1744a883847ea5c28f8" target='_blank'>
              Programming T cells for Early Cancer Detection with Customized Protease-Activatable Receptors
              </a>
            </td>
          <td>
            H. Phuengkham, Ying Chen, Anirudh Sivakumar, Ali H. Zamat, Lena Gamboa, Quoc D. Mac, Hee Jun Lee, Leonard C. Rogers, Jicheng You, Sean A. Steele, Sichen Zhu, Matthew S. Gollins, J. Blazeck, Peng Qiu, G. Kwong
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Radiation-induced senescence (RIS) in glioblastoma (GBM) is an undesirable cell fate that inhibits tumor cell death and supports resistance and outgrowth. While senescence-targeting drugs are promising adjuvants, their clinical application will require proper patient selection based on post-treatment RIS burden. Current methods to evaluate senescence, however, are tissue-based, and given GBM’s difficult anatomical location, post-treatment biopsies are impractical. Therefore, novel and less invasive biomarkers for TIS are urgently needed. To this end, we aimed to identify candidate extracellular vesicle (EV) liquid biomarkers for TIS by profiling senescence-associated cargo changes within GBM EVs. Using a panel of GBM patient-derived cell lines, we show that RIS is the primary functional state following radiation exposure and is associated with significant alterations in the cargo of senescent-derived EVs (senEVs). In particular, senEV transcriptomes have an increased abundance of senescence-associated RNA species and enrichment of senescence-associated gene sets. Most striking, however, was that senEVs are most differentiated by the significant enrichment of a panel of snoRNAs. This signature was conserved in 4/5 GBM models of RIS and was validated by qRT-PCR. Further analysis by mass spectrometry revealed that snoRNAs are likely co-packaged with their associating proteins, as senEVs had concurrent increases in these binding partners. Finally, in a preliminary patient cohort comparing plasma EVs obtained prior to surgery to those obtained after completion of their radiation therapy, we identified increased senescence-associated RNA such as CDKN2B and GLB1 and the snoRNA SNORA49 in post-radiation EVs. Altogether, this data suggests that senEV RNA species, and particularly snoRNAs, are a promising analyte for RIS-biomarker development. With further study, this work may open avenues for a companion diagnostic for senotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97eedb2f9c91a2dd3a0855efa1f8e43a40066349" target='_blank'>
              Molecular profiling of glioblastoma-derived extracellular vesicles identifies small nucleolar RNAs as candidate liquid biomarkers for radiation- induced senescence
              </a>
            </td>
          <td>
            Valerie DeLuca, Nathaniel Hansen, Priya Digumarti, N. Tang, Karen Fink, George J. Snipes, P. Pirrotte, Michael E Berens
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Intratumoral heterogeneity drives immunotherapy resistance, hindering clinical efficacy. Functional assays that capture heterogeneous responses to immunotherapies remain limited. Here, we present a 3D live imaging assay using patient-derived ovarian cancer organoids to quantify natural killer (NK) cell-induced apoptosis across hundreds of individual organoids. Our assay reveals inter- and intratumor response heterogeneity and detects rare populations unresponsive to NK cytotoxicity, enabling the study of subclonal immune responses and underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction Inflammatory breast cancer (IBC) is a rare invasive tumor and characterized by the formation of tumor emboli within dermal lymphatic vessels. The tumor microenvironment (TME) is a key factor in IBC aggressiveness, but its heterogeneity and intercellular role remain incompletely understood. Methods Weighted gene coexpression network analysis (WGCNA) was performed to identify immune infiltration-associated genes. We also depicted cellular communication networks and specific ligand-receptor signaling pathways using the single-cell transcriptomics analysis of IBC and non-IBC samples. Finally, we verified the expression of key proteins by immunohistochemistry. Results Through the intersection of WGCNA module genes and differentially expressed genes between pathological complete response and residual disease samples, GZMB was identified as hub gene which is related to immune infiltration and efficacy of neoadjuvant therapy in IBC. In IBC cohort, patients with high expression of GZMB harbored more immunosuppressive cells thus showed unfavorable prognosis compared with GZMB-low expression group (p<0.05). Subsequent to dimension reduction and clustering, 12 clusters were identified to construct the single-cell atlas between IBC and non-IBC samples. Cellular communication analysis unveiled the heterogeneity of cell communication in IBC. The proportion of immune cells was significantly lower than that of malignant epithelial cells in the cellular composition of IBC. Moreover, it indicated that SPP1 and plasmacytoid dendritic cells were specific in IBC and associated with an immunosuppressive microenvironment in IBC. Immunohistochemical analysis suggested that protein levels of GZMB and SPP1 tended to be higher in the samples from patients with residual disease compared to the patient achieving pathological complete response, though this observation is based on an extremely small sample size. Discussion This study identifies GZMB and SPP1 as potential immunosuppressive-related prognostic biomarkers in IBC patients, reveals the key role of plasmacytoid dendritic cells in remodeling of immunosuppressive microenvironment in IBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282d00d57fceb6947f3f7237a1980645189f7d26" target='_blank'>
              Integrative analysis of bulk and single-cell transcriptomics identifies factors related to immunosuppressive microenvironment to predict unfavorable prognosis in inflammatory breast cancer
              </a>
            </td>
          <td>
            Kexuan Feng, Xiaoduo Li, Xiaoqian Li, Junjie Liu, Rui Zhang, Heyan Chen, Yunpeng Zhang, Jianjun He, Huimin Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell transcriptomics has redefined our understanding of cancer by exposing the complexity of tumor ecosystems and their therapeutic vulnerabilities. scRNA-seq studies have identified lineage hierarchies, immune evasion programs, and resistance-associated states across solid and liquid tumors, informing biomarker development and drug discovery. Advanced computational frameworks integrate these data with longitudinal profiling, RNA velocity, and network diffusion to prioritize targets and predict therapeutic response. Emerging multi-omics approaches further expand the scope of precision oncology by linking genetic alterations, protein-level markers, and spatial context to functional states. This narrative review aims to synthesize current applications of single-cell transcriptomics for target discovery, highlight computational frameworks that translate high-dimensional data into actionable insights, and explore how multi-omics integration is shaping future directions. By bridging molecular complexity with target prioritization, these approaches hold promise for translating single-cell insights into clinically actionable biomarkers and therapeutic strategies for personalized cancer treatment and rational drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc90ad6637cf26b6fef95756c86b88733af331f8" target='_blank'>
              Single-Cell Transcriptomics and Computational Frameworks for Target Discovery in Cancer
              </a>
            </td>
          <td>
            Martina Tarozzi, Nicolas Derus, Stefano Polizzi, C. Sala, Gastone Castellani
          </td>
          <td>2026-02-03</td>
          <td>Targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gastric cancer (GC) exhibits molecular heterogeneity and diverse immune cell infiltration patterns closely associated with patient prognosis. However, a comprehensive understanding of the variations in immune cell phenotypes among different patient subgroups still needs to be improved. In this study, we performed a detailed analysis of the tumor microenvironment in GC by integrating 200,466 single cells from 72 patients across six datasets. We classified patients into immune-deserted, B, T, and myeloid cell subtypes. Using genomic and clinical data from TCGA samples, we identified cellular components associated with tumor histology and genotypes. GC patients were stratified into immune-deserted, B cell, T cell, and myeloid cell subtypes, and we described the pathway and transcription factor activity characteristics of different microenvironment subtypes. Integration of bulk RNA-seq data reveals that fibroblasts and endothelial cells were associated with adverse patient outcomes whereas NK and T cells were notably correlated with improved prognosis. Subsequently, we focused on characterizing cancer-associated fibroblasts (CAFs) and discovered that they acquire new functional properties within the tissue microenvironment, providing evidence of CAF plasticity. We constructed a novel four-gene CAF signature including SPARC, EFEMP1, RGS5 and SERPINE1 which may enhance patient stratification and prognostic prediction of GC patients. qPCR analysis revealed that the significant expressions of SPARC, EFEMP1, RGS5 and SERPINE1 were significantly upregulated in gastric cancer tissues compared to the normal tissues. Our study provides insights into the composition of the tumor microenvironment and construction of a four-gene CAF signature associated with clinical prognosis, offering new perspectives for the clinical management of gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b22dc4e8c3bd98a01915b46ab3333a465c3c7e" target='_blank'>
              Integrative single-cell atlas unveils heterogeneity and prognostic value of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Ziyu Jiang, Xiaomei Zhuang, Junkui Guo, Minyi Zhu, Chunhong Hong, Chunhui Sun, Kaiming Wu, Haofan Yin, Cuncan Deng, P. Jiang
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab2ef7cfea18c9d2b8d0291643654a27ff4767a" target='_blank'>
              ISG15-driven immune modulation and tumor progression in breast cancer metastasis: insights from single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Hua Shao, Hanlu Tang, Huiying Lin, Yongqing Xu
          </td>
          <td>2026-02-04</td>
          <td>BMC medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain invasion of meningioma is a controversial clinicopathological grading criterion that correlates with prognosis and is a stand-alone criterion for WHO grade 2 meningioma. Molecular correlates of meningioma cells and immune cell infiltration at the brain-meningioma border and its possible role in brain invasion are not known. We hypothesized that brain-invasive meningiomas have distinct gene expression profiles in meningioma cell populations and in tumour-associated microglia and macrophages (TAM) populations, most prominently at the brain-meningioma border. We performed spatial transcriptomics using NanoString GeoMx Digital Spatial Profiling to investigate the gene expression profiles of meningioma cell-enriched populations (Iba1-/CD68-) and TAM-enriched populations (Iba1+/CD68+) across 16 tumours. Regions of interest included core regions of meningiomas, the brain-meningioma border, and brain regions with confirmed invasion. Using an 1800 gene panel, we analysed differential gene expression across regions within each tumour and compared brain-invasive and non-invasive meningiomas. Meningioma cell-enriched populations from brain-invasive meningiomas (n = 8) showed significant upregulation of KRT18, implicated in filament reorganization, as well as genes involved in the cell cycle, glycolysis, and the growth factor receptor PDGFRB compared to non-invasive meningiomas (n = 8). Meningioma cell-enriched populations from the brain-meningioma border showed significant upregulation of KRT18, NDUFA4L2, and PKM relative to core areas. In TAM-enriched populations in brain tissue, we found upregulation of the chemokine receptor gene CSF1R relative to TAM-enriched populations in meningioma tissue and increased TAM infiltration in brain-invasive cases. In conclusion, our results demonstrate molecular changes in brain-invasive meningiomas compared to non-invasive meningiomas as well as spatial changes along the core-border axis. The expression pattern of TAMs also changed from meningioma to brain tissue. Additional studies are needed to confirm these findings and further reveal how we can target meningioma brain invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269f040bbd53f8bc31b9e6b6faae7f3967b17a9f" target='_blank'>
              Novel insights into meningioma brain invasion with spatial transcriptomic profiling
              </a>
            </td>
          <td>
            A. Maier, Claes Ottzen Laurentiussen, S. Michaelsen, Lorenzo Perino, T. Stannius, D. Harwood, M. N. Klausen, J. Haslund-Vinding, N. Bager, David Scheie, A. Schmidt, L. Melchior, A. Areškevičiūtė, Frederik Vilhardt, K. Josefsen, D. Raleigh, F. Bagger, Tiit Mathiesen, B. W. Kristensen
          </td>
          <td>2026-01-04</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways. Methods We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes. Results Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells. Discussion In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604fc75ae6b3d011181af39eae23a0a3c99bbefb" target='_blank'>
              MicroRNA spatial profiling for assessing drug efficacy in BRCA1-related triple-negative breast tumors
              </a>
            </td>
          <td>
            Omar N. Mohd, Lin Wang, Brian R. Sardella, David Jou, Gerburg M. Wulf, Frank J. Slack, Yujing J. Heng, Patrick S. Doyle
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d48a09c8bd8763cbb53c10162f6bf4ec34e18bc" target='_blank'>
              Transcriptomic evaluation of immune-infiltrated patient-derived tumor organoids as preclinical models in renal cell carcinoma
              </a>
            </td>
          <td>
            Liangwei Yin, Léonard Lugand, Jules Russick, J. LeMaoult, C. Battail
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Simple Summary Cancer progression is influenced not only by cancer cells themselves but also by surrounding supportive cells in the tumor environment. Among these, cancer-associated fibroblasts play important roles in shaping tumor behavior and immune responses, but their diversity and clinical significance in head and neck cancer are not fully understood. In this study, we analyzed single-cell gene expression data to identify distinct fibroblast subtypes in head and neck squamous cell carcinoma. We further examined whether gene signatures derived from these fibroblast subtypes could predict patient outcomes using large clinical datasets. We found that specific inflammatory fibroblast programs were strongly associated with patient survival and distinct immune environments. Importantly, these fibroblast-related patterns were reproducible across independent patient cohorts and were supported by additional experiments using cultured fibroblasts. Our findings provide new insights into the biological and clinical relevance of fibroblast diversity and suggest that stromal features may complement existing biomarkers to improve risk stratification in head and neck cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39c29db87141bfec7ddb84a086ca849e34133944" target='_blank'>
              Cancer-Associated Fibroblast Heterogeneity Shapes Prognosis and Immune Landscapes in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Hideyuki Takahashi, Hiroyuki Hagiwara, Hiroe Tada, Miho Uchida, T. Matsuyama, K. Chikamatsu
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Osteosarcoma remains lethal for many patients with metastatic or relapsed disease. Post-translational modifications (PTMs) regulate protein signaling and may shape the tumor microenvironment and clinical behavior in osteosarcoma, but PTM-anchored transcriptomic programs are as yet not well defined. Methods We integrated single-cell RNA sequencing from GSE162454 with curated PTM and immune gene sets to build a PTM-related framework for osteosarcoma. Tumor cell differentially expressed genes were intersected with PTM and immune repertoires to derive candidates. A PTM-related prognostic score was trained in TARGET-OS and validated in GSE21257 and GSE16091. Immune infiltration and microenvironment features were profiled using ssGSEA, Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression (ESTIMATE) data, and Tumor Immune Dysfunction and Exclusion (TIDE) scores. Model interpretation used SHapley Additive exPlanations (SHAP) and single-cell localization. GRN was prioritized for exploration of immune correlations and in vitro loss-of-function assays in U2OS and HOS cells. Results The three-way intersection yielded 298 genes. The PTM-related score stratified overall survival in training and validation cohorts and remained independent of clinical covariates. High scores aligned with an immunosuppressed, stroma-rich microenvironment, with lower ImmuneScores and ESTIMATE scores, enrichment of myeloid and regulatory lineages, higher dysfunction and exclusion by TIDE, and reduced cytolytic, interferon, and antigen-presentation programs. SHAP highlighted a compact driver set enriched in malignant and stromal compartments. GRN showed strong contribution and consistent single-cell localization. Elevated GRN correlated with plasmacytoid dendritic cells, myeloid-derived suppressor cells (MDSCs), macrophages, regulatory T cells (Tregs), and multiple inhibitory checkpoints and with diminished immune effector functions. GRN silencing reduced proliferation, clonogenicity, migration, and invasion in osteosarcoma cells. Conclusion A PTM-anchored transcriptomic signature captures prognostic heterogeneity in osteosarcoma and links adverse outcome to an immunosuppressed microenvironment. GRN emerges as a tumor- and stroma-intrinsic mediator of immune suppression and malignant traits and represents a biologically grounded target for future mechanistic and therapeutic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3027626fcc200ffb2a234ce61476d434da4250d4" target='_blank'>
              Single-cell dissection of PTM-related networks reveals an immunosuppressed osteosarcoma ecosystem
              </a>
            </td>
          <td>
            Jingyu Chen, Wei Zhang, Hai Yan, Jinyu Chen, Hanrui Liu, Xingyu Zhou, Haiping Zhang, Dongdong Cheng
          </td>
          <td>2025-12-18</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43834c7632e582a60435566324798a1fd37dbe2b" target='_blank'>
              Fibroblast heterogeneity in 2025
              </a>
            </td>
          <td>
            A. Croft, A. Hackland
          </td>
          <td>2025-12-19</td>
          <td>Nature Reviews Rheumatology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec774215b12f2be4ebfe5634529efaca54fcef6" target='_blank'>
              Toward Personalized Treatment of Urogenital Cancers: The Role of Patient-Derived Organoids.
              </a>
            </td>
          <td>
            S. Sagliocchi, Michele Musone, Stefano Chianese, A. G. Cicatiello, Silvia Del Mastro, Francesco Del Giudice, M. Dentice, F. Crocetto
          </td>
          <td>2026-01-08</td>
          <td>Oncology and therapy</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59359ad678fb5970076eddf100c364fea7a28f75" target='_blank'>
              Adrenergic signaling induces a pro-tumorigenic B cell state in colorectal cancer
              </a>
            </td>
          <td>
            Meike S. Thijssen, Simone L. Schonkeren, Linde Coolkens, Yuchi Zou, Lieve Temmerman, Joëlle de Vaan, M. Idris, Jorunn Vrancken, Kim Wouters, Nathalie Vaes, Marion J. Gijbels, Sharon Scardellato, Erwin Wijnands, Willine J. van de Wetering, F. Verhaegen, L. J. Dubois, Erik Biessen, Carmen López Iglesias, Kim M Smits, A. Bardelli, Stefano Casola, W. Boesmans, V. Melotte
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc6a6f856a7083dd2ac5b388edeaab80c7f5a72" target='_blank'>
              A spatially resolved atlas of gastric cancer characterises a lymphocyte-aggregated region.
              </a>
            </td>
          <td>
            Sen Gao, Shishang Qin, Dongfang Wang, Anqiang Wang, Linna Zhu, Yang Li, Qiang Shi, Hongtao Fan, Yufei Bo, Yunshan Zhong, Yuxuan Sun, Kun Dong, Liqin Fu, Ranran Gao, Yan Wu, Ye Liang, Luyujie Huang, Xueda Hu, Xianwen Ren, Z. Bu, Jiafu Ji, Zemin Zhang
          </td>
          <td>2026-01-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Computational pathology has emerged as an attractive option for improving risk stratification in prostate cancer (PCa), but current approaches either lack interpretability or focus solely on tumor morphology, potentially limiting their utility. We aimed to identify an interpretable, immune microenvironment-derived computational pathology biomarker for PCa.



 We retrospectively identified two independent cohorts (Discovery and Validation) with localized PCa who underwent radical prostatectomy (RP) and had digitized H&E-stained whole-slides images (WSIs) and longitudinal outcomes data. We identified a third cohort from The Cancer Genome Atlas (TCGA) with localized PCa which was treated with RP and had WSIs, bulk RNA sequencing, and whole-exome sequencing. Immune cells were identified from WSIs using a publicly available deep learning model (CellViT), and spatially dense immune clusters were quantified using density-based spatial clustering of applications with noise (DBSCAN). CIBERSORTx was utilized for immune cell deconvolution and TRUST4 for immune receptor repertoire reconstruction from bulk RNA sequencing. Cox proportional hazard regression was used to examine associations between clinicopathologic features and the primary outcome of time to distant metastasis (DM), as well as secondary outcomes of biochemical recurrence (BCR) and overall survival (OS).



 Median follow-up periods for the Discovery (n=272) and Validation (n=218) Cohorts were 12.6 and 8.1 years, respectively. In the Discovery Cohort, median age was 63, 14% had Gleason 8-10 disease, 94% had pT2-T3a disease, and median prostate-specific antigen level at diagnosis was 6.2 ng/mL. Median immune cell proportion was 4.3% (interquartile range 3.0-5.9%), and median immune cluster density was 2.4 per 400 mm2 of tissue (interquartile range 0-7.1). Similar values were observed in the Validation Cohort. In both cohorts, immune cell proportion was not associated with BCR, DM, or OS (P>=0.09). In the Discovery Cohort, while increasing immune cluster was not associated with BCR (P>=0.10), it was independently associated with a decreased risk of DM for Gleason 8-10 (adjusted hazard ratio [AHR] 0.42, 95% confidence interval [CI] 0.19-0.93) but not Gleason 6-7 patients (AHR 1.26, 95% CI 0.77-2.05; Pint=0.020). Similar results were observed in the Validation Cohort, where increasing immune cluster was not associated with BCR (P>=0.10) but was associated with a lower risk of DM in Gleason 8-10 disease (AHR 0.60, 95% CI 0.37-0.98), though not Gleason 6-7 (AHR 1.19, 95% CI 0.74-1.91; Pint=0.043). In TCGA Cohort (n=326), immune cluster was not associated with somatic alterations. For Gleason 8-10 but not Gleason 6-7 disease, high-cluster samples were enriched for CD8+ T cells, activated memory CD4+ T cells, and Tregs (P<=0.037), as well as clonal T cell populations (P<=0.039).



 Our findings nominate immune spatial clustering as a novel, interpretable computational pathology biomarker and provide insights into the unique immune features of high-grade PCa.



 David D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett P. Crowdis, Cora A. Ricker, Seifeldin Awad, Bora Gurel, Jihye Park, Martin T. King, Paul L. Nguyen, Toni K. Choueiri, David J. Einstein, Steven P. Balk, Alok K. Tewari, Johann S. de Bono6, Keyan Salari, Mary-Ellen Taplin, Chin-Lee Wu, Eliezer M. Van Allen. . Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aace1806fe7d01338e5e9857c054699a561213e" target='_blank'>
              Abstract PR001: Immune spatial organization predicts metastasis risk in aggressive localized prostate cancer
              </a>
            </td>
          <td>
            D. Yang, Aya Abdelnaser, Alexander J. Haas, Jeremiah Wala, Alfred A. Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Seifeldin Awad, B. Gurel, Jihye Park, Martin T. King, P. L. Nguyen, T. Choueiri, David J Einstein, Steven P. Balk, A. Tewari, J. D. De Bono, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c25b39df997323e228b3c6d265efc22e9a7958" target='_blank'>
              Multi-omics characterization of IDH-mutant astrocytoma-derived cell lines reveals NOTCH-regulated plastic quiescent astrocyte-like state
              </a>
            </td>
          <td>
            L.Garcia, C.Granotier-Beckers, D.Pineau, D.Sika, S.Hideg, C.Labadie, K.Aguilar Cázarez, J.Kundnani, L.Stuani, L.Gauthier, F.Boussin, M-A.Mouthon, S.Urbach, K.El Koulali, M.Séveno, C.Ripoll, K.Daddi, M.Verreault, A.Idbaih, S.O’Connor, C.Plaisier, L.Zhang, M.Zheng, V.Rigau, H.Duffau, J. Hugnot
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) is an aggressive subtype of breast cancer with limited treatment options and poor prognosis. Coupled with this, TNBC shows a high rate of metastasis which is known to be aided by tumour vasculature. Endothelial cells that form the lining of the tumour vasculature exhibit distinct genotype and phenotype differences compared with normal tissue vasculature. However, little is known about endothelial signatures that drive metastasis from a primary tumour, particularly in the context of immune infiltration. In this study, we utilized GeoMX Digital Spatial Profiling to investigate spatial proteomics differences in endothelial cells between primary and secondary sites of TNBC. By segmenting tissues using epithelial (PanCK), immune (CD45), and endothelial (CD31) markers, we analysed the microvasculature for a panel of 79 target proteins. In paired primary and secondary TNBC tissues, we identified significant downregulation of fibronectin (− log2(Fold-Change) = − 1.7, p < 0.001) in secondary sites. Specifically in epithelial regions, S100B was found to be downregulated in secondary microvasculature when compared to primary tumours. Additionally, metastasis-free primary tissues exhibited upregulated expression of S100B when compared to primary tissues that metastasized. Our study highlights the potential contributions of microvasculature to metastatic progression in TNBC, presenting new opportunities to explore them as potential biomarkers of TNBC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9355568219b2a2ea616006baa27f93ab8c621b2" target='_blank'>
              Digital spatial profiling identifies features of primary and locoregional metastatic vasculature in triple negative breast cancer
              </a>
            </td>
          <td>
            Akhilandeshwari Ravichandran, Kyle Upton, Shiva Taheri, Cheng Liu, Kaltin Ferguson, Mark N. Adams, Laura J. Bray
          </td>
          <td>2026-01-19</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma is the most aggressive adult brain tumour, characterised by resistance to therapy and high recurrence due to diffuse infiltration. We developed a physiologically relevant co-culture model, combining patient-derived glioblastoma cell lines with cortical-like neural spheroids differentiated from human induced pluripotent stem cells. Using high-content imaging, we demonstrate that GBM20 and GBM1 cell lines migrate directionally along axons toward neural spheroids in live imaging assays and infiltrate spheroids extensively in endpoint assays, unlike non-cancerous neural stem cells. A proof-of-principle drug screen identified PF 573228 (FAK inhibitor) and motixafortide (CXCR4 inhibitor) as potent suppressors of GBM20 and GBM1 infiltration, respectively. Bulk RNA sequencing revealed gene expression profiles correlating with invasive behaviour and drug sensitivity. This platform offers a valuable model for studying glioblastoma infiltration along axons and provides proof-of-principle that migration can serve as a measurable and actionable phenotype to screen therapeutic vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72fcee02e1f1c7cefb3147af1fbbfca5819be15" target='_blank'>
              A human iPSC-based neural spheroid platform for modelling glioblastoma infiltration using high-content imaging
              </a>
            </td>
          <td>
            V. Tsang, Federica Riccio, Aimee S Wilson, Hannah Nudds, Jason D Coombes, Heiko Wurdak, Harry Bulstrode, Ivo Lieberam, Davide Danovi
          </td>
          <td>2025-12-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d8e985c420ff8b3595d58e817f6b4a4d574874" target='_blank'>
              ButterflyVI: enabling high-throughput variant interpretation and biomarker discovery with functional genomics
              </a>
            </td>
          <td>
            Debora Sesia, András Hatos, P. Nittis, G. Ciriello
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the primary cause of mortality in patients with prostate cancer (PCa), yet effective treatments remain scarce. Identifying reliable biomarkers and understanding their underlying mechanisms is crucial for advancing clinical management. Firstly, we integrated single-cell and bulk transcriptomic data and employed the Scissor tool to characterize tumor cells with metastatic advantages (termed metastatic cells). Then, independent predictive genes for metastasis were identified through univariate and multivariate regression analyses. The role of hub genes in PCa metastasis was further validated using multiple large datasets, malignant phenotype experiments, in vivo metastatic models, and a clinical-sample-based immunohistochemical cohort. Further, we explored the metabolic characteristics related to hub genes through unbiased functional annotation, and validated the upregulated glycolysis by measuring l-lactic acid production, extracellular acidification rates (ECAR), and oxygen consumption rates (OCR). Finally, multi-omics data were employed to investigate the promoter-methylation-dependent regulation of alpha-2-glycoprotein 1 (AZGP1) transcription, with methylation confirmed through PCa cell-based methylation-specific PCR (MSP) assays. AZGP1 was identified as an independent protective predictor of metastasis, which was validated in vitro and in vivo. Metabolic functional annotation revealed that glycolysis was upregulated in AZGP1-positive luminal cells. Consistently, overexpression of AZGP1 in PCa cells was associated with lower l-lactic acid levels, reduced ECAR, and increased OCR. In addition, DNA methylation at the cg26429636 region was linked to decreased transcriptional expression of AZGP1. MSP assays revealed an unmethylated pattern in PCa cells with high AZGP1 expression, and higher methylation levels in AZGP1-low cells. Promoter methylation of AZGP1 leads to reduced transcriptional expression, thereby promoting glycolysis in tumor cells and facilitating metastasis. The detection of AZGP1 methylation levels offers a valuable reference for dynamic surveillance of PCa metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27afdf89dedd3a210e73d91a40b5e0a60099b14d" target='_blank'>
              Methylation-induced silencing of AZGP1 enhances prostate cancer metastasis by stimulating tumoral glycolysis
              </a>
            </td>
          <td>
            Lu Li, Jinguang Luo, Linyue Zhao, Lu Tian, Jianfeng Wang, Yifei Cheng, Xiao Li
          </td>
          <td>2026-01-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metastasis in colorectal cancer (CRC) is the primary driver of its high mortality rate, with dysregulation of apoptosis and remodeling of the tumor microenvironment (TME) serving as key biological events driving disease progression. However, the intrinsic molecular connections between these processes—particularly the dynamic changes during the progression from primary tumors to liver metastases—remain to be systematically elucidated. This study aims to identify core genes regulating the crosstalk between apoptosis and immunity in CRC using multidimensional computational biology approaches and to explore their clinical value as prognostic biomarkers and potential therapeutic targets. This study integrated single-cell RNA sequencing (scRNA-seq) data with bulk transcriptomic data from The Cancer Genome Atlas (TCGA) database. We first utilized scRNA-seq to dissect cellular heterogeneity and communication networks in CRC primary tumors and liver metastases. Subsequently, weighted gene co-expression network analysis (WGCNA) and differential expression analysis were employed, combined with machine learning algorithms (LASSO, Random Forest, XGBoost), to screen features from apoptosis- and immunity-related gene sets and pinpoint key candidate genes. Finally, the biological functions and clinical significance of the core gene were systematically validated through pan-cancer analysis, immune infiltration analysis, mutation profile analysis, prognostic model construction, and validation via real-time quantitative PCR (RT-qPCR) in clinical samples. Single-cell profiling revealed that colorectal cancer (CRC) liver metastases exhibit a distinct pro-inflammatory immune microenvironment characterized by the enrichment of CD8⁺ effector memory T cells and inflammatory monocytes, along with extensive remodeling of cell-cell communication networks centered on the MIF and CD99 signaling pathways. Through multi-level bioinformatics analysis, we identified the apoptosis-related gene PMAIP1 as a central regulatory hub mediating these processes. Clinical cohort and pan-cancer analyses demonstrated that PMAIP1 is significantly overexpressed across multiple tumor types, particularly in CRC, where its elevated expression serves as an independent predictor of poor prognosis (HR = 0.39, p = 0.028). Further investigation uncovered a critical biological paradox: On the one hand, the high expression of PMAIP1 was positively correlated with the infiltration of core effector immune cells, such as CD8⁺T cells. CD8⁺T cells are the key immune cells mediating tumor cell killing, and their enrichment is expected to form an effective anti-tumor immune barrier, which is consistent with the characteristics of the “pro-inflammatory” immune microenvironment. On the other hand, the high expression of PMAIP1 is also significantly associated with the up-regulation of multiple immune checkpoint molecules, such as PD-1 and PD-L1. The activation of immune checkpoint molecules directly induces the functional exhaustion of infiltrating CD8⁺T cells, which is manifested as decreased proliferation ability, decreased secretion of killer cytokines, and loss of anti-tumor activity. In short, PMAIP1 both “recruits” effector immune cells with antitumor potential and “weakens” these cells by upregulating immune checkpoint molecules, resulting in the coexistence of two contradictory phenotypes of “increased immune cell infiltration” and “decreased immune response efficiency”. Finally, it still promotes the progress of colorectal cancer and is related to the poor prognosis of patients. A nomogram model incorporating PMAIP1 demonstrated strong performance in predicting patient survival, and its overexpression in clinical CRC samples was validated by RT-qPCR. This study systematically delineates the immune landscape of CRC primary tumors and liver metastases, and for the first time reveals the apoptosis gene PMAIP1 as a critical regulatory node linking cellular apoptosis to immune microenvironment remodeling. PMAIP1 not only emerges as a potent, novel biomarker for poor prognosis in CRC but also, through its dual functions in regulating immune infiltration and checkpoints, represents a highly promising target for guiding combined immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13ca61fee4f2179a10131a3116553e22f26a3615" target='_blank'>
              Integrated single-cell and machine learning analysis identifies PMAIP1 as a novel biomarker for predicting prognosis and immunotherapy response in colorectal cancer
              </a>
            </td>
          <td>
            Xiangfang Li, Zihan Cai, Shoupeng Ding, Jinghua Gao, Yuanlin Wang, Yuqian Wu, Huangyan Zhang, Jiachuan Chen, Zongqiang Hu
          </td>
          <td>2025-12-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) exhibits extensive intratumoral heterogeneity on multiple biological scales, contributing to variable clinical outcomes and limiting the effectiveness of conventional TNM staging, which highlights the urgent need for multiscale integrative analytic frameworks. The lipid-deficient de-clear cell differentiated (DCCD) ccRCC subtype, defined by multi-omics analyses, is associated with adverse outcomes even in early-stage disease. Here, we establish a hierarchical cross-scale framework for the preoperative identification of DCCD-ccRCC. At the highest layer, cross-modal mapping transferred molecular signatures to histological and CT phenotypes, establishing a molecular-to-pathology-to-radiology supervisory bridge. Within this framework, each modality-specific model is designed to mirror the inherent hierarchical structure of tumor biology. PathoDCCD captured multi-scale microscopic features, from cellular morphology and tissue architecture to meso-regional organization. RadioDCCD integrated complementary macroscopic information by combining whole-tumor and its habitat-subregions radiomics with a 2D maximal-section heterogeneity metric. These nested models enabled integrated molecular subtype prediction and clinical risk stratification. Across five cohorts totaling 1,659 patients, PathoDCCD reliably recapitulated molecular subtypes, while RadioDCCD provided reliable preoperative prediction. The consistent predictions identified patients with the poorest clinical outcomes. This cross-scale paradigm unifies molecular biology, computational pathology, and quantitative radiology into a biologically grounded strategy for preoperative noninvasive molecular phenotyping of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2906aadb2798289d6c075f86ae197fcc81c4cc58" target='_blank'>
              Multiscale Cross-Modal Mapping of Molecular, Pathologic, and Radiologic Phenotypes in Lipid-Deficient Clear Cell Renal CellCarcinoma
              </a>
            </td>
          <td>
            Ying Cui, Dongzhe Zheng, Ke Yu, Xiyin Zheng, Xiaorui Wang, Xinxiang Li, Yan Gu, Lin Fu, Xinyi Chen, Wenjie Mei, Xin-gui Peng
          </td>
          <td>2025-12-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a self-renewing population capable of fueling long-term tumor growth. In gastric cancer, the identity of CSC populations remains unclear. In this study, we established a gastric CSC population marked by the water channel protein aquaporin-5 (AQP5), which resides in human and mouse pyloric tumors. Using multiple organoid and mouse models, we found a requirement for AQP5+ CSCs in both initiating and sustaining cancer progression and demonstrated that AQP5 expression also directly promotes tumor growth and invasion in a WNT, PI3K (phosphatidylinositol 3-kinase), and MAPK (mitogen-activated protein kinase)-dependent manner. Beyond primary cancers, AQP5 further enriches for a functional CSC population in metastatic tumors. Together, our findings support a CSC model in gastric tumors that may have application for therapeutic strategies targeting CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b857f221b4120247f60ae689da7ba8ea312dc26b" target='_blank'>
              AQP5: A functional gastric cancer stem cell marker in mouse and human tumors.
              </a>
            </td>
          <td>
            H. Lim, Swathi Yada, Kazuhiro Murakami, B. Lee, Sowmya Sagiraju, Phyllis Phuah, Tanysha Chi-Ying Chen, Fidelia B Alvina, S. Tan, Kaushal Krishna Kaslikar, Nur Syahirah Binte Ruhazat, Menaka Priyadharsani Rajapakse, Katzrin Bte Ahmed Murad, Snezhina Kancheva, Liang Thing Tan, Seri Mustafah, Jimmy Bok Yan So, Nick Barker
          </td>
          <td>2025-12-18</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound cellular heterogeneity, the understanding of which is critical for improving prognosis and therapy. Using single-cell RNA sequencing of 32,247 cells from human HCC samples, we characterized the tumor ecosystem and identified five malignant hepatocyte subpopulations with distinct molecular profiles and stage-specific enrichment. Among these, the S100A6⁺ C1 and S100A9⁺ C4 subpopulations were predominantly associated with advanced tumors and actively remodeled the tumor microenvironment through enhanced signaling pathways such as MDK and MIF. We further identified PGAM2 as a key transcriptional regulator in early-stage tumors, whose activity correlated with sialylation—a process linked to immune evasion. Based on these findings, we developed a prognostic model integrating PGAM2 and sialylation-related genes, which robustly stratified patients into high- and low-risk groups with significantly different survival outcomes, immune contextures, and predicted therapeutic responses. Functional experiments validated AGRN, a component of the signature, as a functional driver of HCC proliferation and invasion. Collectively, our results decode the cellular and molecular heterogeneity of HCC, provide a clinically relevant prognostic tool, and highlight potential targets for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d560ef3a66a46cb286a615fdcaac30d8c4ef51e5" target='_blank'>
              Machine learning-driven comprehensive profiling of tumor heterogeneity and sialylation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kaiqiang Tang, Lu Han, Junlin Li, Kang Li
          </td>
          <td>2025-12-17</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic liver disease (CLD) represents a major global public health challenge, necessitating a systematic understanding of its complex immunopathological mechanisms. This review comprehensively summarizes the groundbreaking applications of cutting-edge technologies—including single-cell sequencing, spatial transcriptomics, and organoid models—in chronic liver disease immunology research: Single-cell sequencing resolves immune cell heterogeneity at unprecedented resolution, identifies rare cell subsets, and reveals dynamic changes and regulatory networks through multi-omics integration; Spatial transcriptomics complements this by mapping immune-stromal interactions within structural contexts such as the portal tract, fibrotic septa, and tumor niches, uncovering spatially organized immune evasion mechanisms and microenvironmental remodeling; Organoid technology constructs humanized liver-immune models that recapitulate disease-specific features—such as fibrosis, steatohepatitis, and hepatocellular carcinoma—enabling mechanistic validation, drug screening, and individualized therapeutic exploration. The synergistic integration of multi-omics profiling, spatial mapping, and organoid modeling is driving a paradigm shift in chronic liver disease immunology—transitioning from static cellular descriptions to spatiotemporal mechanism decoding, and from population-level insights to individualized pathophysiology and treatment prediction. These advanced approaches establish a technological foundation for building precision immunotherapeutic strategies tailored to spatiotemporal regulation of the liver immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1390600cdb0085be9c69b75a3de5a6e93ec2fda5" target='_blank'>
              Integrated multi-omics, spatial profiling and organoid modeling drive transformative advances in chronic liver disease and hepatocellular carcinoma immunomicroenvironment research
              </a>
            </td>
          <td>
            Qi Liu, Chenyu Wang, Cheng Ye, Huawei Zhang, Teng Li, Wenjuan Wei, Senyan Wang, Huapeng Zhang
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Bladder cancer (BCA) shows significant prognostic differences between non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) forms. While NMIBC frequently recurs and can progress to invasive disease, reliable biomarkers to monitor this transition are lacking. Extracellular matrix (ECM) remodeling is a critical factor influencing tumor aggressiveness, yet the key regulators of ECM changes across BCA stages remain unclear. In this study, we investigate the role of COL1A2 in ECM-related tumor biology and its potential as a prognostic biomarker for BCA progression. Methods We utilized a multi-step bioinformatics pipeline, analyzing RNA-seq data from TCGA and GEO datasets to identify molecular differences between NMIBC and MIBC. Prognostic markers were prioritized via differential expression analysis, Cox regression, and Kaplan–Meier survival analysis. The regulatory network was explored using protein-protein interaction analysis, and ECM-related activity was quantified through ssGSEA. Cell-type-specific insights were gained through single-cell RNA-seq analysis, and intercellular communication was deciphered using CellChat. Functional validation was performed through in vitro knockdown experiments in BCA cell lines. Results COL1A2 emerged as a key prognostic ECM-related gene associated with MIBC. Single-cell RNA-seq analysis revealed that COL1A2 and ECM components were predominantly enriched in matrix cancer-associated fibroblasts (CAFs), with PTK2 (FAK, focal adhesion kinase) upregulated in epithelial cells undergoing epithelial-mesenchymal transition (EMT). CellChat analysis uncovered a dominant COL1A2-mediated signaling axis from matrix CAFs to EMT epithelial cells via COL1A1/2–SDC1/4 ligand-receptor interactions. Functional assays confirmed that COL1A2 knockdown significantly impaired MIBC cell invasion and migration by suppressing ECM remodeling and EMT. Conclusion Our results suggest that the COL1A2–ECM–FAK signaling axis plays a critical role in MIBC progression, and COL1A2 could serve as a potential biomarker and therapeutic target for muscle-invasive bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda2a0be0d617f715d6d80984a257431f3977b47" target='_blank'>
              Integrative bulk and single-cell transcriptomic analysis reveals COL1A2-driven ECM remodeling and focal adhesion signaling associated with the transition from non-muscle-invasive to muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Menglu Li, Xinwei Liu, Yuhui Xue, Yichen Lu, Zhiqiang Chen, Yuwei Zhang, Weiguo Chen, Shan-Chao Zhao, Ke Wang, Ninghan Feng
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ea403e374bfc46879783639eafd8524e7b9e39" target='_blank'>
              Microenvironment-Inferred Genotyping: An Exclusionary Classifier for EGFR Amplification When DNA Testing Fails
              </a>
            </td>
          <td>
            Elizabeth Baker, Nathan Mehaffy
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d2ad509c801e8dff7695be2327a46d057fed6b" target='_blank'>
              CCR5 expression defines the functional heterogeneity of tumor-infiltrating CD8+ T cells and predicts enhanced antitumor immunity and favorable prognosis in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yilong Li, Kaiyuan Zhang, Lian Wang, Weiping Yao, Wenjing Fei, Keyu Cheng, Yan Li, Guichun Huang
          </td>
          <td>2025-12-23</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment failure from resistance and recurrence remains a major clinical challenge. Therapy-induced senescence (TIS), a state of stable cell cycle arrest with sustained metabolic activity, has emerged as a driver of inflammation, tumor persistence, and relapse. However, the heterogeneity of TIS complicates its detection and targeting. Here, we applied a multi-modal strategy to characterize metabolic alterations in senescent cancer cells induced by doxorubicin or γ-irradiation across three tumor cell lines: MCF7, HeLa, and TPC-1. Mitochondrial dysfunction was assessed using MitoTracker and JC-1 staining, while two-photon excitation fluorescence (TPEF) microscopy enabled label-free visualization of intracellular NAD(P)H and FAD distribution. Lipid remodeling was evaluated by MALDI mass spectrometry imaging, and RNA sequencing was performed on control, senescent, and engulfing-senescent MCF7 cells to identify differentially expressed genes and enriched pathways. Senescent cells displayed mitochondrial dysfunction, with altered NAD(P)H/FAD distribution and decreased membrane potential. TPEF confirmed redistribution of coenzymes, reflecting redox changes. Lipidomics revealed consistent remodeling, notably involving cardiolipin precursors. Transcriptomic profiling showed engulfing-senescent MCF7 cells possess a distinct signature marked by increased lipid metabolism, endocrine signaling, and cell–cell communication. Overall, our findings reveal conserved and cell type–specific metabolic traits of TIS, highlighting metabolic vulnerabilities for senolytic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Primary tracheobronchial tumors (PTBTs) can often be successfully treated by surgery; however, more effective strategies are still warranted for advance tumor cases. The present study aimed to provide a comprehensive understanding of the molecular and cellular features of PTBTs, along with the tumor microenvironment (TME). By performing single-cell RNA, we revealed a distinct TME landscape of PTBTs containing 52,739 single cells. For instance, these three pathological types were found have heterogeneous fibroblast populations, immune cells were more activated in inflammatory myofibroblastic tumor (IMT). Notably, a subpopulation of BIRC5+ cells was identified in primary tracheal squamous cell carcinoma (SCC) and adenoid cystic carcinoma (ACC) samples, which may have similar properties to cancer stem cells (CSCs). Moreover, IMT was characterized by the infiltration of pro-inflammatory immune cells, and might be a benign tumor. Importantly, based on ligand-receptor analysis, the PPIA/CD147 pair was found to be strongly upregulated during tumor and CD8+ T cell interactions. By applying multiplex immunofluorescence for selected markers, we validated the presence of the separate cell entities. Our findings provide deeper insights into the biology of PTBTs, which may be beneficial for advancing the diagnosis and therapy of PTBTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc41a4f5248143a0c3cf35d4c57326e220116bee" target='_blank'>
              A single-cell atlas reveals the tumorigenesis and microenvironment status of primary tracheobronchial tumors
              </a>
            </td>
          <td>
            Chen Shu, Xiyang Tang, Kaifu Zheng, Runze Zhang, Yulong Zhou, Mengchao Li, Weiguang Du, Yuquan Bai, Yang Shen, Yujian Liu, Haitang Yang, Feng Yao, Xiaolong Yan, Nan Ma, Jinbo Zhao
          </td>
          <td>2025-12-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology. CDKN2AIPNL exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze CDKN2AIPNL’s expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease, in which patients each carry distinct molecular features that drive metastatic progression. However, the lack of clinically representative models that recapitulate this disease have limited our understanding in the heterogeneity between patients. LuCaP 147, established from a liver metastasis biopsy, is a prostate cancer transplantable xenograft carrying AR and SPOP mutations with a further hypermutated phenotype, as well as loss of MSH2 and MSH6 expression.



 Using LuCaP 147, we developed a novel patient-derived metastasis (PDM) model in NSG mice via intracardiac injection. The development of metastasis was monitored through weekly bioluminescent imaging. We analyzed the tumor burden and survival rate of the PDM147 models. At the endpoint, long bones, spines and soft tissues were collected. To evaluate the histologic phenotype of PDM147, tissue sections were stained by immunohistochemistry (IHC) for adenocarcinoma markers including AR and neuroendocrine markers (which include SOX2, ASCL1, and SYP).



 After intracardiac injection, tumor burden (BLI) was detected after 5 weeks, and mice developed morbidity after 7-8 weeks. H&E stains showed that PDM147 developed bone metastases in spines, long bones, as well as visceral metastases in liver, brain, spinal cord, kidney and adrenal glands. IHC indicated that metastases were heterogeneous with both AR+/NE- and AR-/NE+ cells. AR+ and SYP+ cells were intermingled in the tumor, but the stains themselves were mutually exclusive between cels. Interestingly, no androgen-deprivation therapy was needed to promote NE differentiation, showcasing the biological novelty [SA([1] of this model.



 PDM147 is a hypermutated tumor with mutually exclusively expressing AR+ and NE+ markers in metastatic sites, including soft tissues and nervous tissues. Future work to define distinct pathways and molecular markers that lead to de novo vs. treatment-induced NE phenotype may offer a better understanding of the biology driving metastatic spread in these similar yet distinct NE subtypes and the role a hypermutator tumor can have in prostate cancer progression. These findings underscore the value of patient-derived models to study mCRPC progression and heterogeneity. Further use of these models for high-throughput drug screens may uncover targeted therapies for advanced prostate cancer.



 Dana S. Vargas Solivan, JuanJuan (Ivy) Yin, Tri Truong, Joseph Twohig, Ross Lake, Adam G. Sowalsky. A patient-derived metastasis model with spontaneous neuroendocrine transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8565cd695486e6d924d7f7c053c2a965518cb8d0" target='_blank'>
              Abstract B080: A patient-derived metastasis model with spontaneous neuroendocrine transformation
              </a>
            </td>
          <td>
            Dana S Vargas Solivan, JuanJuan Yin, Tri M Truong, Joseph Twohig, R. Lake, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efc387a5fab7de78625be6d806bb5f60287b5f7b" target='_blank'>
              Patient-Derived in Vitro Models Reveal Insights into Medullary Thyroid Cancer Microenvironment and Resistance to Tyrosine Kinase Inhibitors.
              </a>
            </td>
          <td>
            E. Grassi, V. Ghiandai, G. Gaudenzi, E. Massardi, V. Cirello, Erika Carbone, Davide Gentilini, Sarah Uraghi, G. Gazzano, G. Dionigi, C. Colombo, Giovanni Vitale, L. Fugazzola, L. Persani
          </td>
          <td>2026-02-06</td>
          <td>Thyroid : official journal of the American Thyroid Association</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="We continue to regard stomach tumors as uniform molecular entities while neglecting microanatomical signaling ecologies that genuinely cause resistance, which is a pressing issue that the manuscript highlights. To drive a transition from descriptive atlases to practical, spatially actionable research, this letter adopts a purposefully critical posture, not one that is hostile. The AREG–SOX9 axis is not just “interesting”; it is a perfect example of why many targeted medicines fall short: we have not figured out where the biology is focused. Through the AREG– ERBB2 axis, amphiregulin (AREG) produced by iCAFs is found to contribute to drug resistance and enhance tumor stemness by upregulating SOX9 expression in tumour cells [ 1 ] . Although this claim is based on correlation evidence that confuses proximity with causality, it is mechanistically reasonable. Spatial–functional corroboration is necessary for a genuinely critical perspective: are AREG-high iCAFs close to ERBB2-active clones? Does SOX9 induction decrease, and ERBB2 inhibitor sensitivity return when AREG is blocked in situ ? The axis remains a theory rather than a clinically validated mechanism, given the absence of direct spatial mapping of ligand–receptor activity (e.g., phospho-ERBB2 colocalized with AREG sources). We suggest using ex vivo slice assays with AREG neutralization ± trastuzumab and in situ proximity ligation/phospho-mapping on identified interface zones as an instantaneous, high-yield test to determine causation. The growth of bacteria-induced stomach cancer cells, elevated β - catenin levels, and development of stem cell-like characteristics all depended on SOX9 [ 2 ] . A compelling multi-hit concept (microbiota → stroma → tumor) is introduced, citing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/079610ca0b7f290f8f651c837134d136fd948c2b" target='_blank'>
              Spatial confinement of the AREG–SOX9 signaling circuit: rethinking tumor–stroma interactions in gastric cancer
              </a>
            </td>
          <td>
            Asra Amjad, Muhammad Talha Usman, Umair Ali, Raghabendra kumar Mahato
          </td>
          <td>2025-12-12</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5407cdb757b46038485ad236efb4e8ad5a8a93" target='_blank'>
              Tumor-associated macrophages promote chemoresistance to Paclitaxel via activating NOTCH2-JAG1 juxtacrine signaling.
              </a>
            </td>
          <td>
            Fazhi Yu, Qin Zhou, Weiqiang Yu, Tong Zhou, Cheng Cao, Yijia Xie, Peng Zhang, Hanyuan Liu, Wei He, Aoxing Cheng, Xiaopeng Ma, Qingfa Wu, Qi Zhao, Jing Guo, Kaiguang Zhang, Ying Zhou, Jue Shi, Zhenye Yang
          </td>
          <td>2026-01-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumorigenesis and metastasis are frequently attributed to the intricate interplay between cancer cells and the tumor microenvironment (TME). Comprehending the mechanisms and key regulators of cancer-immune crosstalk in the TME is imperative for developing efficacious immunotherapy. Through a series of in vivo CRISPR screens, we identified tumor-intrinsic ITGB1 as a critical regulator of triple-negative breast cancer (TNBC) development and deciphered its underlying mechanisms. Tumoral ITGB1 facilitated the establishment of pro-tumorigenic TME by orchestrating tumor-associated myeloid populations. Suppressing ITGB1 favored the enrichment of anti-tumorigenic myeloid cells and enhanced infiltration of CD4 and CD8 T cells, culminating in superior antitumor effects. CRISPR scanning pinpointed a previously unrecognized functional domain essential for ITGB1's pro-tumorigenic activity. This domain is distinct from all known ligand-binding sites in ITGB1. An antibody capable of sterically blocking this domain significantly impaired TNBC progression. These findings position tumoral ITGB1 as a promising therapeutic target for reprogramming the TME from a pro- to an anti-tumorigenic state, thereby effectively inhibiting TNBC development. Our study uncovers a novel mechanism of TNBC development and provides a unique therapeutic strategy for targeting ITGB1 in TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae5ee2f3fdbf08bebdce33dbf8ef1a363061c6f" target='_blank'>
              ITGB1 Regulates Triple-Negative Breast Cancer Development by Modulating the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Nuozi Song, Siqi Chen, Lei Wang, Jessica Dang, Xu Cao, Stephanie Singh, Lu Yang, Jinhui Wang, Steven T. Rosen, Yingyu Wang, Chun-Wei D Chen, Cheng Zhang, Mingye Feng
          </td>
          <td>2026-02-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 The human prostate develops from the urogenital sinus through epithelial bud formation and branching, however the spatial dynamics of progenitor populations during these early stages remain poorly characterized. To addresses this gap, we used a human fetal prostate at 14 weeks of gestation for serial sectioning, yielding approximately 200 sections of 10 µm thickness. Six sections were profiled using spatial transcriptomics with the Visium HD platform, while the remaining sections were H&E-stained for 3D reconstruction of glandular architecture. We mapped transcriptional heterogeneity across developing prostate buds and identified distinct spatial organization of epithelial subdomains with basal-like and luminal-like characteristics. Notably, we also identified a unique cluster of tubule-initiating cells (TIC cells), that have been linked to cancer initiation and androgen-deprivation therapy resistance in adult prostate. This study provides the first high-resolution spatial map of developing human prostate, highlighting the organisation of progenitor populations.



 Emmanouela Perisynaki. 3D spatial mapping of developing human prostate glands [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B059.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9e67e64e1e6f9570a547e8411a3ed76fac26e1" target='_blank'>
              Abstract B059: 3D spatial mapping of developing human prostate glands
              </a>
            </td>
          <td>
            Emmanouela Perisynaki
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Polymorphonuclear myeloid-derived suppressor cells (PMN-MDSCs) are crucial mediators of tumor-induced immunosuppression, while their heterogeneity and spatial dynamics across malignancies remain poorly understood. By integrating single-cell RNA sequencing data from 576 samples across 19 cancer types and spatial transcriptomics data from three distinct malignancies, we identified a PMN-MDSC population. This cell population demonstrated characteristic upregulation of immunosuppressive genes and was associated with poor prognosis across multiple cancer cohorts. Notably, TREM1 was highly expressed in PMN-MDSCs and may mediate immunosuppressive processes. Multiplex immunofluorescence demonstrated that TREM1+ PMN-MDSCs exhibited significantly higher distribution in tumor regions compared to non-tumor tissues. Spatial transcriptomics analysis revealed their co-localization with fibroblasts and exhausted T cells. Moreover, CellChat analysis showed that TREM1+ PMN-MDSCs remodeled the tumor microenvironment through interactions with diverse cellular components. Collectively, our study revealed the conserved immunosuppressive features and spatial interaction networks of TREM1+ PMN-MDSCs from a pan-cancer perspective, highlighting TREM1 as a pivotal therapeutic target to disrupt PMN-MDSC-mediated tumor immune evasion. A pan-cancer analysis reveals TREM1+ PMN-MDSCs as key mediators of immunosuppression by remodeling the tumor microenvironment, highlighting TREM1 as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5174fd91c2ae4f071c06cfa6c978a0e63d85760" target='_blank'>
              Pan-cancer analysis reveals TREM1+ PMN-MDSCs as critical regulators of immune suppression and tumor microenvironment remodeling
              </a>
            </td>
          <td>
            Yangjie Cai, Shanhang Li, Hening Li, Zhuan Zou, Xinda Zheng, Haijun Tang, Mingxiu Yang, Pintian Wang, Weizhen Wu, Hongcai Teng, Kai Luo, Xinyu Huang, Wenyu Feng, Shijie Liao, Juliang He, Yun Liu
          </td>
          <td>2025-12-18</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tight junctions (TJs) are essential for preserving cell polarity and controlling permeability. It has been disclosed that TJ proteins, especially specific claudins (CLDNs), are linked to inflammation and contribute to the emergence of diverse cancers, including brain malignancies. Aggressive gliomas, including glioblastoma multiforme (GBM), remain among the most common and deadly central nervous system (CNS) tumors worldwide, despite considerable advances in diagnostic and therapeutic approaches. These types of tumors are characterized by high rates of recurrence and metastasis, resulting in poor outcomes and prognosis. The pathophysiology of brain cancer is closely linked to CLDNs, as these specific proteins play critical roles in tumor cell proliferation, invasion, and disruption of the blood-brain barrier (BBB). Some studies reported the potential role of CLDNs in glioma progression and other neurological disorders. The purpose of this review is to highlight the significance of CLDNs in CNS tumors, especially their participation in the formation of malignant gliomas. Additionally, the diagnostic and prognostic importance of selected CLDNs has been assessed. Selected CLDNs, such as CLDN3 and CLDN4 promote GBM growth, proliferation and migration. Moreover, overexpression of CLDN3 support progression and metastasis of these malignancies, while reduced expression of CLDN1 and CLDN5 is observed in advanced gliomas. Presented results suggest that CLDNs may serve as biomarkers for diagnosis and prognosis as well as therapeutic targets in CNS tumors. Further investigation is essential to clarify their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a64609eb48722ff52a4aaaa823eb54e52baa236" target='_blank'>
              Claudins proteins in brain tumors: expression patterns and therapeutic target
              </a>
            </td>
          <td>
            A. Romanowicz, M. Łukaszewicz-Zając, B. Mroczko
          </td>
          <td>2025-12-15</td>
          <td>Biochemia Medica</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="BACKGROUND Breast cancer is one of the most prevalent malignancies affecting women worldwide, with approximately 2.3 million new cases diagnosed annually. Breast cancer stem cells (BCSCs) play pivotal roles in tumor initiation, progression, metastasis, therapeutic resistance, and disease recurrence. Cancer stem cells possess self-renewal capacity, multipotent differentiation potential, and enhanced tumorigenic activity, but their molecular characteristics and regulatory mechanisms require further investigation. AIM To comprehensively characterize the molecular features of BCSCs through multi-omics approaches, construct a prognostic prediction model based on stem cell-related genes, reveal cell-cell communication networks within the tumor microenvironment, and provide theoretical foundation for personalized treatment strategies. METHODS Flow cytometry was employed to detect the expression of BCSC surface markers (CD34, CD45, CD29, CD90, CD105). Transcriptomic analysis was performed to identify differentially expressed genes. Least absolute shrinkage and selection operator regression analysis was utilized to screen key prognostic genes and construct a risk scoring model. Single-cell RNA sequencing and spatial transcriptomics were applied to analyze tumor heterogeneity and spatial gene expression patterns. Cell-cell communication network analysis was conducted to reveal interactions between stem cells and the microenvironment. RESULTS Flow cytometric analysis revealed the highest expression of CD105 (96.30%), followed by CD90 (68.43%) and CD34 (62.64%), while CD29 showed lower expression (7.16%) and CD45 exhibited the lowest expression (1.19%). Transcriptomic analysis identified 3837 significantly differentially expressed genes (1478 upregulated and 2359 downregulated). Least absolute shrinkage and selection operator regression analysis selected 10 key prognostic genes, and the constructed risk scoring model effectively distinguished between high-risk and low-risk patient groups (P < 0.001). Single-cell analysis revealed tumor cellular heterogeneity, and spatial transcriptomics demonstrated distinct spatial expression gradients of stem cell-related genes. MED18 gene showed significantly higher expression in malignant tissues (P < 0.001) and occupied a central position in cell-cell communication networks, exhibiting significant correlations with tumor cells, macrophages, fibroblasts, and endothelial cells. CONCLUSION This study comprehensively characterized the molecular features of BCSCs through multi-omics approaches, identified reliable surface markers and key regulatory genes, and constructed a prognostic prediction model with clinical application value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86e7f7dec5d3e190d876efe5f1afc46eeaf94a9" target='_blank'>
              Breast cancer stem cell activity driven by ME18D gene expression in the tumor microenvironment
              </a>
            </td>
          <td>
            De-Yang Guo, Zhang-Yi Liu, Qian-Chuan Yi
          </td>
          <td>2026-01-26</td>
          <td>World Journal of Stem Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720c9004235c7c3aef4fbef8c74bcb96b3903c12" target='_blank'>
              Multiomics analysis reveals that senescent CXCL16+ macrophages promote lung adenocarcinoma progression through TGF-β signalling.
              </a>
            </td>
          <td>
            Zhen-Hu Zhang, , Wei Li, Han Xiao, Dong Wang, Meng-Hang Yang
          </td>
          <td>2026-02-02</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The survival mechanisms of circulating tumor cells (CTCs) remain poorly understood while these rare cell populations transit through the blood stream and colonize distant organs including the brain. Using single-cell RNA-seq of microfluidically-enriched CTCs and patient-matched brain metastatic tumor cells of lung adenocarcinoma (LUAD), we revealed CTC-selective upregulation of core gene signatures associated with ferroptosis. The malignancy of CTCs was confirmed through comparative genomic instability and copy number variation (CNV) analyses of paired CTCs and metastatic tumor cells using both transcriptomic and whole-exome sequencing. Among the transcription factors elevated in CTCs, Chromobox 3 (CBX3) was the top hit, which was tightly correlated with GPX4 expression. Functionally, CBX3 co-operated with EP300 to protect CTCs from ferroptosis by upregulating GPX4 expression. Genetic depletion of CBX3 triggered ferroptosis and substantially reduced tumor cell survival and invasiveness in two independent cancer cell models. Conversely, CBX3 overexpression was sufficient to promote tumor growth, migration, and invasion in vitro, and metastatic progression in vivo. Clinically, CBX3 expression was significantly correlated with TNM stage and was predictive of both progression-free interval and overall survival in LUAD. In a prospective cohort of LUAD and melanoma patients, CTCs co-expressing CBX3 and GPX4 were selectively elevated in the blood samples of metastatic cases compared to the non-metastatic group, highlighting the clinical association between ferroptosis-resistant CTCs and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7fe0bbf1d19b9968391a14825a2e7d78f15cc38" target='_blank'>
              CBX3 confers ferroptosis resistance during blood-borne metastasis
              </a>
            </td>
          <td>
            Chun Wu, Xuefei Liu, Boxi Zhao, Mao Zhao, Binyu Zhang, Guanyin Huang, Yixin Cheng, Shuqian Zheng, Jianyang Hu, Ling Guo, Wei-Jie Guo, Jun Tan, Xin Hong
          </td>
          <td>2026-01-15</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Clear cell renal cell carcinoma (ccRCC) presents a unique immunological paradox: abundant CD8
 +
 tumor-infiltrating lymphocytes (TILs) correlate with poor prognosis. To clarify their functional status and therapeutic potential, we performed single-cell transcriptomic profiling of TILs from 15 patients with ccRCC and functionally validated dominant T cell receptor (TCR) clonotypes using autologous tumor-derived organoids. Single-cell RNA sequencing revealed dynamic shifts in T cell composition, with effector and progenitor-exhausted CD8
 +
 T cells declining and terminally exhausted CD8
 +
 and regulatory CD4
 +
 T cells enriched in advanced tumors. Despite this exhausted phenotype, in an exploratory analysis with five patients, approximately half of the top 20 TCR clonotypes retained anti-tumor reactivity when re-expressed in non-exhausted T cells, as evidenced by TCR-T cell-mediated cytotoxicity and IFN-γ production against autologous organoids. Transcriptomic signatures enabled the development of a penalized logistic regression classifier that distinguished tumor-reactive from bystander T cells with high accuracy, with AUCs of 0.903 (training) and 0.913 (test). Cross-cancer comparison with pancreatic ductal adenocarcinoma (PDAC) datasets revealed limited generalizability, highlighting the need for cancer type-specific models. Notably, ccRCC-specific TILs exhibited mature, functionally differentiated profiles with limited proliferation, consistent with chronic antigen exposure, whereas PDAC-reactive TILs showed highly proliferative and activated phenotypes indicative of ongoing clonal expansion. Collectively, these findings suggest key features of the immune landscape in ccRCC and provide a preliminary, proof-of-concept transcriptomic framework for prioritizing candidate tumor-reactive TCRs. These insights suggest the feasibility of identifying candidate TCRs for future development of TCR-based adoptive T cell therapies in ccRCC and emphasize the importance of integrating single-cell profiling with functional analyses to refine immunotherapeutic strategies. Given the limited sample size, our results should be considered exploratory and hypothesis-generating, and future studies will be required to validate these findings in larger, independent ccRCC cohorts.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62c016264580977157f1e59b0fc3cfcc38dd55" target='_blank'>
              Tumor-reactive TCRs within exhausted TILs reveal cancer type-specific immune landscapes in renal cell carcinoma
              </a>
            </td>
          <td>
            Mitsuru Komahashi, S. Horaguchi, Kayoko Tsuji, Daisuke Hoshino, Takeshi Kishida, Kimitsugu Usui, N. Nakaigawa, Shinya Sato, Hiroshi Hamana, Hiroyuki Kishi, Feifei Wei, Yasunobu Mano, T. Kouro, Shuichiro Uehara, T. Sasada
          </td>
          <td>2026-01-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Motivation Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. Results The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes. Availability The implementation of scSurv is available on GitHub (https://github.com/3254c/scSurv) and Zenodo (https://doi.org/10.5281/zenodo.17793054).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98bd46ab28d9f09fcd249606230d4459ff93bbef" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2025-12-22</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83420d91db40081aef405ab1feed30e61b3d080" target='_blank'>
              Monoallelic expression characterizes a distinct molecular and clinical group of breast tumors
              </a>
            </td>
          <td>
            Mona Arabzadeh, Amartya Singh, Kyle Payne, Shridar Ganesan, Hossein Khiabanian
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Predicting spatial gene expression from H&E histology offers a scalable and clinically accessible alternative to sequencing, but realizing clinical impact requires models that generalize across cancer types and capture biologically coherent signals. Prior work is often limited to per-cancer settings and variance-based evaluation, leaving functional relevance underexplored. We introduce HistoPrism, an efficient transformer-based architecture for pan-cancer prediction of gene expression from histology. To evaluate biological meaning, we introduce a pathway-level benchmark, shifting assessment from isolated gene-level variance to coherent functional pathways. HistoPrism not only surpasses prior state-of-the-art models on highly variable genes , but also more importantly, achieves substantial gains on pathway-level prediction, demonstrating its ability to recover biologically coherent transcriptomic patterns. With strong pan-cancer generalization and improved efficiency, HistoPrism establishes a new standard for clinically relevant transcriptomic modeling from routinely available histology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c31313036e644587313c31ba04ae584ee9cd4119" target='_blank'>
              HistoPrism: Unlocking Functional Pathway Analysis from Pan-Cancer Histology via Gene Expression Prediction
              </a>
            </td>
          <td>
            Susu Hu, Qinghe Zeng, Nithya Bhasker, J. N. Kather, Stefanie Speidel
          </td>
          <td>2026-01-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Gastrointestinal stromal tumors (GISTs) exhibit significant heterogeneity, posing substantial challenges for personalized treatment strategies. Patients often display varied responses to different therapeutic agents and dosages. Currently, the absence of robust and physiologically relevant in vitro models for GISTs impedes accurate prediction of therapeutic efficacy, thereby constraining the advancement of effective treatment strategies. Traditional 2D cell cultures fail to replicate the tumor microenvironment (TME) and lack patient-specific characteristics, limiting their predictive value. In contrast, three-dimensional bioprinting (3DP) technology faithfully recapitulates key histological architecture and molecular features of their parental tumors, enhancing the physiological relevance of in vitro models. We employed patient-derived 3DP-GIST models via 3D bioprinting technology, followed by comprehensive histopathological, genomic, and transcriptomic analyses. Subsequently, we applied clinically approved targeted therapeutic agents to perform drug screening and response prediction on the 3DP-GIST models. The drug sensitivity profiles obtained from these models were then correlated with retrospective clinical data and patient follow-up records to assess the models’ potential in guiding the selection and prediction of effective GIST therapies. In our study, we successfully constructed 12 patient-derived 3DP-GIST models. Histopathological assessments, whole-exome sequencing (WES), and transcriptomic analyses confirmed that these models accurately recapitulate the histological architecture, biomarker expression, and molecular features of their corresponding parental tumors. Transcriptomic profiling further revealed gene expression signatures associated with GIST recurrence risk and imatinib resistance. Importantly, the 3DP-GIST models demonstrated the capacity to provide precise, individualized treatment recommendations within 10 days post-surgery, potentially reducing treatment delays and improving patient outcomes. Overall, the 3DP-GIST model represents a robust and efficient platform for evaluating patient-specific drug sensitivities in vitro, thereby guiding personalized therapeutic strategies for GIST patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492f9dec710186d0bb398ff8467a9fedaf4e7eda" target='_blank'>
              Three-dimensional bioprinting of patient-derived Gastrointestinal stromal tumor: a novel platform for precision oncology and drug response profiling
              </a>
            </td>
          <td>
            Liwei Du, Zicheng Zheng, Yanan Wang, Kai Zhang, Yuce Lu, Minghao Sun, M. Pang, Shangze Jiang, Yixuan He, Shunda Du, Haitao Zhao, Yilei Mao, Huayu Yang, Weiming Kang
          </td>
          <td>2025-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Single-cell genomics has emerged as a transformative approach to unravel the complexity of somatic variation in specific cells within the human body. This field has profound implications for understanding the role of somatic mutations in aging, cardiovascular disease, and tissue-specific pathologies. By focusing on circulating cells and cardiac resident cells, including cardiomyocytes, within the heart, this review explores how single-cell genomics provides insights into cellular heterogeneity and clonal evolution. We discuss the implications of somatic variation for cardiovascular health, highlight technological innovation, and future directions in this rapidly evolving field. This review sheds light on national initiatives, such as the National Institutes of Health Somatic Mosaicism Across Human Tissues Network (SMaHT) and the European Somatic Mutations in Vascular-Wall Function and Age-Associated Disease (EU SOMATICART) project, which aim to generate reference atlases of somatic mutations across human tissues. It also explores challenges and future directions in leveraging single-cell approaches to improve diagnostics and therapeutics in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ec9fb81424f8866ab25b7b24005bdae8410be68" target='_blank'>
              Single-Cell Genomics and Somatic Variation in Circulating and Cardiac Resident Cells.
              </a>
            </td>
          <td>
            N. Hilal, Maniteja Arava, S. Choudhury
          </td>
          <td>2026-01-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc0f9c209c0272c9533b8ac77ff3f95324d7d8be" target='_blank'>
              iPSC-derived human liver organoids uncover ductular reaction as a hallmark of Wolman disease and inform gene therapy design
              </a>
            </td>
          <td>
            D. Selvestrel, Caterina Da Rodda, B. Anfuso, Marine Laurent, Annamaria Antona, Alessia Mattivi, Suresh Velnati, Karin Hofmann, Luciano Conti, D. Bonazza, Fabrizio Zanconati, M. Mastronardi, N. de Manzini, N. Rosso, C. Tiribelli, Marcello Manfredi, Daniela Capello, Philippe Drabent, Luca Fava, Giannino Del Sal, Silvia Palmisano, M. Amendola, Giovanni Sorrentino
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, characterized by one cell engulfing another, play a paradoxical role in cancer metastasis. This review synthesizes their dual nature: while CIC can suppress metastasis by eliminating aberrant cells or activating immune responses, it also drives tumor progression through clonal evolution, metabolic adaptation, and genomic instability. The clinical relevance of CIC varies across cancer types, correlating with both favorable and poor prognoses depending on disease stage and microenvironmental context. Emerging therapeutic strategies aim to harness protective CIC mechanisms or inhibit pro-metastatic pathways, though challenges such as biomarker validation and dynamic tracking persist. Future research integrating advanced technologies will elucidate the spatiotemporal complexity of CIC, enabling tailored interventions to disrupt metastatic cascades. By bridging fundamental biology and clinical translation, CIC represents a promising yet intricate frontier in the battle against metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6d8a02bb8fcc15958d3d8c8939ab24272dbaf6" target='_blank'>
              Cell-in-cell structures and cancer metastasis.
              </a>
            </td>
          <td>
            Yuchen Cao, Jianing Du, Jianguo Lai, Chunjun Liu
          </td>
          <td>2026-01-23</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Patients with von Hippel-Lindau (VHL) disease develop multiple distinct kidney cysts and clear cell renal cell carcinomas (ccRCCs) throughout their lifetime. Although cysts are typically considered precursors of ccRCC, no molecular evidence of cysts to tumor progression was provided to date, due to the lack of an accurate molecular characterization of these lesions. We performed a comprehensive molecular characterization of four kidney cysts and six ccRCCs obtained from the same VHL-disease patient, thus all sharing the same genetic germline and host environment. We combined whole genome sequencing (WGS) and RNA sequencing (RNA-seq) profiling with pathological examinations. Cysts and tumors exhibited distinct transcriptomic profiles. Tumors showed increased glycolysis and response to hypoxia, while cysts were associated with enrichment in extracellular matrix organization and inflammation and displayed elevated expression of nephron distal portion markers. Deconvolution analysis further revealed different stromal and immune microenvironments with cysts enriched in myofibroblasts and plasma cells signatures whereas tumors were associated with tumor-associated macrophages (TAMs) and tumor vasculature signatures. Genomically, most cysts and tumors were clonally independent, harboring distinct somatic single-nucleotide variants. Despite sharing some somatic mutational signatures, tumors and cysts exhibited divergent somatic copy number alterations. Only tumors displayed chromosome 3p loss, resulting in VHL loss of heterozygosity, without other driver mutations. VHL patients develop multiple kidney cysts and solid tumors throughout their lifetime and their clinical management is challenging. This study presents the first in-depth molecular analysis of kidney cysts and ccRCC in a single VHL patient. We observed distinct molecular profiles between cysts and tumors, suggesting independent origins. While preliminary, these findings challenge the assumption that cysts always serve as precursors of ccRCC in VHL disease and underscore the need for larger studies to improve surveillance and management of renal lesions in patients with VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a5cddad02651b1247d054c02f906a3863c26a3" target='_blank'>
              Distinct genomic, microenvironmental, and nephron signatures in VHL kidney cysts and tumors
              </a>
            </td>
          <td>
            I. Rowe, F. Corea, G. Pipitone, G. Scotti, Roberta Lucianò, M. Sant'angelo, N. Tenace, Irene Franco, Annamaria Ferrara, Claudio Doglioni, Maurilio Ponzoni, F. de Cobelli, F. Montorsi, U. Capitanio, Paola Carrera, A. Larcher, Andrea Salonia, Federico Daniela Francesco Giorgia Maria Grazia Annalisa Ra Belladelli Canibus Cei Guazzarotti Patricelli Russ, F. Belladelli, D. Canibus, F. Cei, G. Guazzarotti, M. Patricelli, Annalisa Raucci Russo, Chiara Re, L. Salerno
          </td>
          <td>2026-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Tumor-associated neutrophils (TANs) exhibit highly functional heterogeneity across cancers. Although TANs promote inflammatory responses and contribute to tumor clearance, they frequently undergo context-dependent reprogramming within the tumor microenvironment (TME) into highly immunosuppressive phenotypes that facilitate cancer dissemination and immunotherapy resistance 1,2 . We contend that an underappreciated, upstream determinant of this divergence is the maturation stage of TANs 3,4 . The developmental stage of TANs determines the migration patterns and constrains the functional capacity, and the developmental stage also constrains the extent of TME-driven re-education, together shaping pro-or anti-tumor outcomes 3-5 . In this Perspective, we place maturation at the core of TAN biology and discuss current definitions for TAN developmental stages and the measurable markers that researchers and clinicians can use ( Figure 1 ). In addition, spatial and temporal transitions in TAN maturation stages and the factors that govern these transitions are elucidated. We explain how maturation status shapes TAN function and articulate the key differences between mouse and human TAN maturation systems to highlight the value of human immune system (HIS) mouse models. Based on this framework, functional biomarkers and signatures of TAN maturation are introduced and we show how to embed them into patient stratification and longitudinal monitoring. Finally, we outline immunotherapy strategies targeting TAN maturation, selecting interventions guided by maturation markers to reinforce treatment benefits for">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d662ed678180365badf4da6ebe8288f142877d8a" target='_blank'>
              From bone marrow to the tumor microenvironment: how neutrophil maturation shapes cancer immunity
              </a>
            </td>
          <td>
            Tao Shi, Yiran Cai, Hanbing Wang, Jia Wei
          </td>
          <td>2025-12-15</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Lymph nodes, brain, bone, and liver are recognized as the four most common metastatic sites for lung adenocarcinoma (LUAD). Metastasis to these locations exhibits some common features, such as immune suppression, and distinct tumor microenvironment (TME) heterogeneity involving differentiation of immune cells, impacting treatment efficacy and prognosis. Lymph node metastases are characterized by immune suppression with exhausted CD8+ T cells, expanded regulated T cell (Tregs), M2-polarized macrophages, and high programmed death ligand-1 (PD-L1) expression. Brain metastases display an “immune desert” phenotype due to blood–brain barrier constraints, reduced T-cell infiltration, and microglia-mediated immunosuppression. Bone metastases involve osteoclast activation, RANKL/OPG pathway dysregulation, and metabolic reprogramming, while liver metastases show Kupffer cell-driven PD-L1/ programmed death 1(PD-1) axis suppression and elevated Treg infiltration. Key biomarkers across all types of metastases include PD-L1, cytokine profiles, immune cell ratios, and metabolic markers. Therapeutic strategies focus on combination therapies such as immune checkpoint inhibitors (ICIs) with metabolic modulators, localized drug delivery, and biomarker-guided approaches. Challenges in this field encompass spatial heterogeneity, dynamic TME evolution, and clinical translation barriers. Future research directions highlight spatial transcriptomics, microbiome interactions, and organoid models to optimize personalized immunotherapy. This article aims to provide a comprehensive review of regarding TME alterations across these four main metastatic locations of LUAD. It will also discuss relevant TME biomarkers and their clinical significance on therapeutic response and prognosis. We expect this article to serve as a source of evidence and inspiration for the future development of treatment strategies based on LUAD TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eac0fc7ba7c67303e9257f2a2d91aba7b28ddab" target='_blank'>
              Unraveling the alterations and biomarkers in the tumor microenvironment in lung adenocarcinoma metastases and their indications for therapeutic response and prognosis
              </a>
            </td>
          <td>
            Chao Guan, Xiangmin Li, Xiaoshan Zeng, Zhenguang Du, Zhicheng Zhou, Jungang Zhao, Lele Song, Li Yu
          </td>
          <td>2025-12-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acral melanoma (AM), the predominant melanoma subtype in Asia, responds poorly to immune checkpoint inhibitors, representing a critical unmet medical need. The efficacy of tumor-infiltrating lymphocyte (TIL) therapy in this population is unknown. An Investigator-Initiated Trial evaluates autologous TIL therapy (LM-103) in four Chinese patients with advanced AM, achieving a 75% disease control rate (DCR) and a 25% objective response rate (ORR), including one durable complete response. To define the determinants of response, we performed integrated single-cell RNA and T-cell receptor sequencing on infused TIL products, tumors, and longitudinal peripheral blood. Responders' infused products were significantly enriched for T follicular helper (Tfh) and intermediate exhausted (TEX_int) CD8⁺ T cells, which mediated robust cell-cell signaling networks (e.g., CD40, FASLG). In contrast, the non-responder's product was dominated by terminally exhausted (TEX_term) cells. Clonal tracking revealed that these Tfh and TEX_int subsets possessed higher clonality, and in the complete responder, a dominant clone originating from the TEX_int population persisted systemically by differentiating into a progenitor-like (TEX_prog) state. These findings demonstrate that TIL therapy is clinically active in AM and that durable response is mechanistically linked to the infusion and persistence of Tfh and TEX_int subsets, defining a key cellular and clonal architecture for therapeutic success.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d4d63db111617fa12c56b70ba0807cf90683e2" target='_blank'>
              Single-Cell Dissection of Tumor-Infiltrating Lymphocytes Reveals Cellular Architecture Predictive of Therapeutic Efficacy in Acral Melanoma.
              </a>
            </td>
          <td>
            Chao Zhang, Wanyi Xiao, Hong-xiu Shen, Fenge Li, Ting Li, Weihong Zhang, Haotian Liu, Ziwei Gao, Hongyu Wang, Xiubao Ren, Kexin Chen, Xiangchun Li, Jilong Yang
          </td>
          <td>2026-01-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, Shenghua Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Metabolic rewiring is integral to immune cell activation and tissue inflammation, but its spatial and cell type–specific organization in Crohn’s disease (CD) remains poorly understood. In this study, we aimed to construct a spatially resolved single-cell atlas of immunometabolic states in human ileal CD, spanning active inflammation, remission, and health, to identify activity-linked and persistent metabolic signatures.



 Terminal ileal tissue was obtained from patients with active CD (n = 10), CD in remission (n = 8), and controls (n = 10). Imaging mass cytometry was performed using a 37-marker panel incorporating lineage, structural and 14 metabolic proteins. Data were analyzed at single-cell resolution to define metabolic programs across identified cell clusters and spatial neighborhoods. In parallel, peripheral blood mononuclear cells were profiled by suspension mass cytometry to assess systemic signatures. Comparative and spatial correlation analyses were conducted in R using Phenograph clustering and neighborhood interaction modelling.



 IMC revealed extensive, cell type–resolved metabolic reprogramming in active CD. Inflammation was characterized by reduced mitochondrial and tricarboxylic acid cycle activity (ATP5A, CytC, VDAC1) with a concurrent shift towards the pentose phosphate pathway (G6PD) and dampened mTOR signaling. CD68+ macrophages and neutrophils displayed pronounced PPP activation, whereas B cells showed persistent depression of anaplerosis (GLUD1/2) and lipid uptake (CD36). Spatial neighborhood analysis uncovered compartment-specific metabolic coupling, particularly in remission, where immune-epithelial proximity correlated with restored mitochondrial activity. Peripheral blood exhibited only modest, myeloid-restricted changes, indicating that metabolic rewiring is predominantly tissue-embedded.



 Crohn’s ileitis is defined by spatially organized, cell type–specific metabolic remodeling that persists beyond clinical remission. This immunometabolic atlas provides a framework for identifying tissue-specific therapeutic targets and biomarkers of disease activity in inflammatory bowel disease.
 Conflict of interest:
 Lehmann, Malte: No conflict of interest
 Walling, Sonja: No conflict of interest
 Adrian, Huck: No conflict of interest
 Kunkel, Désirée: No conflict of interest
 Glauben, Rainer: No conflict of interest
 Kühl, Anja A.: No conflict of interest
 Siegmund, Britta: Grant: Pfizer Other: Consultant: Abbvie, Abivax, Boehringer Ingelheim, Bristol Myers Squibb, Dr. Falk Pharma, Eli Lilly, Endpoint Health, Galapagos, Janssen/Johnson & Johnson, Materia Prima, MSD, Pfizer, Takeda, Wedbush Securities. Speaker: Abbvie, AlfaSigma, Bristol Myers Squibb, CED Service GmbH, Dr. Falk Pharma, Eli Lilly, Ferring, Galapagos, Janssen/Johnson & ampJohnson, MD Education, MSD, Pfizer, Tr1xBio.
 Haag, Lea Maxie: No conflict of interest
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55721e835b31a5f5ee95010621f83e8bb19691c1" target='_blank'>
              DOP074 Cell type–specific immunometabolic reprogramming in Crohn’s ileitis
              </a>
            </td>
          <td>
            M. Lehmann, S. Walling, H. Adrian, D. Kunkel, R. Glauben, A. A. Kühl, B. Siegmund, L. Haag
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [5, 5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>