<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-12 09:41:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intertumoral heterogeneity in glioblastoma—driven by both genomic and transcriptomic variation—complicates our understanding of how different tumor cell populations contribute to disease progression. Infiltrating tumor cells, which invade surrounding brain tissue and evade surgical resection, are thought to play a central role in recurrence. To address this, we aimed to characterize the gene expression profiles and cellular states of infiltrative tumor cells in glioblastoma. We performed high-plex spatial transcriptomics using the CosMx Spatial Molecular Imager (NanoString) on tumor tissue from eight glioblastoma patients. Formalin-fixed paraffin-embedded samples were selected to capture both the tumor core and invasive margin. A targeted panel of 1,000 genes enabled spatially resolved gene expression profiling at single-cell resolution, allowing precise identification and localization of malignant and non-malignant cell states. We show that malignant cells can be distinguished from non-malignant populations by using patient-specific clustering. Based on this annotation, we identified several known malignant states—including AC-, OPC-, NPC-, and MES-like cells—as well as a recently characterized glial-progenitor (GPC)-like state. This population co-expressed genes associated with both astrocytic and oligodendrocyte progenitor lineages and was found to be more proliferative than the traditional AC-like state. The GPC-like state was most enriched in the classical glioblastoma subtype and was strongly associated with EGFR amplification or mutation. Spatial analyses investigating malignant differences between tumor and infiltrated tissue showed heterogeneous infiltration patterns across patients. In the most extreme case, the dominant GPC-like population in the tumor core gave way to increased proportions of AC-like cells in infiltrated regions. Our study highlights diverging infiltration patterns across glioblastoma tumors, with indications of a GPC-like to AC-like transition occurring in classical-subtyped tumors. This shift is associated with a decrease in cell proliferation and may have implications for clinical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>198</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7a3ed4eb95927059b414292cf9cce743a10321" target='_blank'>
              Spatiotemporal Mapping of Tertiary Lymphoid Structure Heterogeneity Shapes Immune Niches and Clinical Outcomes in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Liaoliao Gao, Jie Mei, Libing Hong, Yuzhi Jin, Jinlin Cheng, Xuqi Sun, Chuan Liu, Bin Li, Xiaomeng Dai, Bo Lin, Yajie Sun, Peng Zhao, Minshan Chen, Rongping Guo, Shan Xin, Jingping Yun, Inmaculada Martínez-Reyes, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Greater understanding of differential therapeutic sensitivity, specifically to immunotherapy, in small-cell lung cancer (SCLC) is required. We explored SCLC heterogeneity through integrated molecular characterization of tumor tissue samples from 159 treatment-naive patients, utilizing genetic, epigenetic, transcriptional, and proteomic profiling, immunohistochemistry staining for multiple biologically relevant markers including transcriptional subtype-defining proteins, and spatial immune profiling using multiplex immunofluorescence. Multi-omics analysis confirmed high heterogeneity across/within neuroendocrine and non-neuroendocrine subtypes. Methylomics analysis identified four methylome clusters that may enhance subtype prediction, prognosis, and longitudinal monitoring of subtype evolution. Immunohistochemistry analysis showed high MHC-I expression in non-neuroendocrine subtypes, which have greatest potential benefit from adding immunotherapy to chemotherapy; high DLL3 expression associated with neuroendocrine subtypes and an immune-cold tumor microenvironment. Multiplex immunofluorescence demonstrated associations of MHC-I with spatial arrangement and phenotypic features of immune cells in the tumor microenvironment of high-MHC-I-expressing SCLC, providing mechanistic rationale for MHC-I as a potential biomarker of immunotherapy response. This multimodal profiling analysis provides further insights into the biologic complexity of SCLC and highlights potential therapeutic vulnerabilities of distinct disease subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7c241dbb9b8047ecee9d971f81cea0a1e193e3" target='_blank'>
              Multi-omic profiling provides insights into the heterogeneity, microenvironmental features, and biomarker landscape of small-cell lung cancer
              </a>
            </td>
          <td>
            Mingchao Xie, Miljenka Vuko, Shashank Saran, Siyu Liu, Andrew G Chambers, Hana Baakza, Helen K Angell, Felicia Ng, C. Gay, R. Cardnell, F. Segerer, Alma Andoni, Jaime Rodriguez-Canales, Paul M Waring, Markus Schick, J. C. Barrett, L. Byers, Giulia Fabbri
          </td>
          <td>2025-12-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, W. Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cervical cancer (CC), a leading cause of cancer-related deaths among women worldwide, is primarily driven by high-risk human papillomavirus (HPV) infections and comprises two major histological subtypes: adenocarcinoma (AC) and squamous cell carcinoma (SCC). Despite advances in prevention and treatment, the molecular and cellular heterogeneity of these subtypes poses significant challenges to achieving optimal clinical outcomes. Here, we integrate single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to dissect the cellular and spatial heterogeneity of AC and SCC, uncovering distinct tumor microenvironment (TME) dynamics that underlie their divergent clinical behaviors. Our scRNA-seq analysis reveals that AC is enriched in epithelial cells, while SCC exhibits a more immunogenic TME with elevated plasma cells and NK/T cells. Spatial transcriptomics further highlights robust interactions between CD8 + T cells and epithelial subtypes in SCC, contrasting with the stromal-rich, immune-cold phenotype of AC. We identify subtype-specific immune and stromal features, including ICOS+ Tregs, IDO1+ cancer-associated fibroblasts (CAFs), and PLVAP+ endothelial cells, which may drive immune evasion, angiogenesis, and metastasis. These findings provide a comprehensive framework for understanding CC heterogeneity and offer actionable insights for developing subtype-specific therapeutic strategies, such as combining immune checkpoint inhibitors with stromal-targeting agents. This study underscores the potential of spatial multi-omics technologies to advance precision oncology and improve outcomes for cervical cancer patients. Single-cell and spatial transcriptomics uncovered a stark contrast in the TMEs of cervical cancer subtypes: squamous cell carcinoma is more immunogenic, whereas adenocarcinoma is stromal-rich and immune-cold.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) brain metastases have a poor prognosis and limited treatment options, including resistance to radiation therapy. Little is known about the molecular and cellular mechanisms that enable CRC tumor cells to adapt to the brain and establish a supportive tumor microenvironment. To address this gap we used spatial transcriptomics to analyze 51 CRC brain metastases. A subset had matched primary colon tumors and longitudinally paired metastatic resections before and after radiation treatment. We identified the critical spatial cellular features of the tumor epithelium and the surrounding tumor microenvironment that support metastatic growth in the brain. CRC brain metastases developed a stromal microenvironment with abundant fibroblasts and tumor-associated macrophages. A fibroblast–macrophage cellular neighborhood promoted angiogenesis, extracellular matrix remodeling, and immune suppression. Tumor cells showed local adaptations. In endothelial-rich regions, they were proliferative whereas in macrophage-rich regions, they were more differentiated and immune evasive. Compared with paired primary tumors, CRC brain metastases showed increased chromosomal instability, with activation of RNA-processing, stress response, and junctional remodeling pathways. After radiation treatment, resistant clones had increased epithelial–mesenchymal transition, while the immunosuppressive stroma remained intact. We identified tumor-derived MIF, GDF15, PRSS3 and SEMA3C ligands and macrophage-derived SPP1 that have the potential to affect multiple cell types in the metastatic niche. These ligand-receptor interactions drive angiogenesis, stromal activation and immune suppression. In a macrophage–tumor–fibroblast co-culture model, knockout of SPP1 in macrophages led to reduced expression of lipid-metabolism related genes and disrupted tumor-promoting interactions. Together, these results indicate that CRC growth in the brain is sustained by a specific cellular organization with immunosuppressive multicellular interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121b9f0cf30a6fb662f64416fefc808a183dcc03" target='_blank'>
              Colorectal cancer relies on an immunosuppressive cellular topography and genomic adaptations for establishing brain metastases
              </a>
            </td>
          <td>
            A. Sathe, Mengrui Zhang, Xiangqi Bai, Ji In Kang, Rithika Meka, Huiyun Sun, Susan M. Grimes, Aparajita Khan, Mingen Liu, A. Luksik, M. Lim, Claudia K. Petrisch, Christopher M. Jackson, Hannes Vogel, Jeanne Shen, M. Gephart, Summer S. Han, Hanlee P. Ji
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be57704ac8a67d3c2570360452cb391401b265a" target='_blank'>
              A Comprehensive Treatment-Induced Resistance Atlas of Glioblastoma Reveals a Fibrotic Niche Shielding the Tumor from Immunotherapy
              </a>
            </td>
          <td>
            Fei Wang, Run Huang, Hongyi Ling, Yuhan Bai, Chen Yang, Guozhen Zhao, Xin Wu, Wenqian Cao, Yue Lu, Yining Zhang, Dengfeng Lu, Youjia Qiu, Juyi Zhang, Bixi Gao, Chao Ma, Hanhan Dang, Yulian Zhang, Yanbo Yang, Ting Sun, Zhouqing Chen, Zhong Wang
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael M. Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential. Understanding tumor heterogeneity and its impact on prostate cancer progression remains elusive. Here, single nucleus snATAC and snRNA sequencing of a multi-loci sampled cohort of advanced prostate cancer patients identifies an inflammatory-like state underlying metastatic dissemination to the lymph nodes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, Adil Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin C. Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bladder cancer (BCa) is a prevalent malignancy and major cause of cancer-related mortality in men, with clinical outcomes still varying despite advances in personalized treatments. High inter- and intra-tumor heterogeneity significantly contributes to this variability. While traditional high-throughput sequencing has provided insights into BCa mechanisms, driver genes, and clinical strategies, it falls short in completely elucidating cellular heterogeneity. Recently, single-cell sequencing (SCS) technologies have substantially enhanced the detection of tumor heterogeneity by improving sensitivity, accuracy, and efficiency. Single-cell transcriptome sequencing offers unbiased, high-resolution analysis of gene expression patterns at the single-cell level, offering essential insights into BCa pathogenesis. This article reviews advancements in SCS technology and its applications in evaluating tumor heterogeneity, the tumor microenvironment, metastasis, and treatment resistance, offering new perspectives for future BCa research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis. Methods To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations. Results We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts. Conclusions These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19bfda47de17fb8ef4014da55fa6d5d86101b2f" target='_blank'>
              Addressing complex autofluorescence signatures in solid tissue samples to enhance full spectrum flow cytometry of non-immune cells
              </a>
            </td>
          <td>
            Christina Gkantsinikoudi, Manuela Terranova-Barberio, Neil P Dufton
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf55c0c7d18000c8ba092357d0dd95271e7c35c" target='_blank'>
              Spatial dissection of ADC/RPT targets defines therapeutic opportunities in rhabdoid tumors
              </a>
            </td>
          <td>
            N. Reitsam, V. Fincke, Maria Daniela Hernandez Ramirez, M. Mucha, Eva Sipos, Lisa Siebenhüter, J. Enke, Maurice Loßner, C. Vokuhl, Constantin Lapa, M. Hasselblatt, Michael C. Frühwald, Bruno Märkl, Pascal D. Johann
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The global incidence of basal cell carcinoma (BCC) is steadily increasing. Although immunotherapy has emerged as a treatment option, its efficacy remains limited. This underscores the need for further investigation of immune remodeling mechanisms in BCC. Here, we aimed to characterize fibroblast subsets and their interactions with other components of the tumor microenvironment in BCC. We performed single-cell RNA sequencing on nine surgical specimens from six patients, including BCC tumors and adjacent normal tissues. Key findings were further validated using spatial transcriptomics and multiplex immunohistochemistry. We identified the C01_TNC fibroblast cluster as a population of cancer-associated fibroblasts (CAFs). These cells exhibited prominent peritumoral infiltration and were characterized by matrix-associated and inflammatory gene expression profiles. We propose that C01_TNC CAFs contribute to tumor progression, possibly by upregulating collagens that engage integrins on tumor cells, thereby facilitating extracellular matrix remodeling and invasion. Furthermore, C01_TNC CAFs were spatially associated with Tregs and expressed ligands that interact with Treg receptors, suggesting a role in enhancing immunosuppression. Notably, this subset was significantly enriched in peritumoral regions and could establish an immune barrier favoring tumor progression. Our findings shed light on the role of fibroblasts in orchestrating interactions between tumor and immune cells in BCC. This enhanced understanding of the tumor microenvironment may inform novel therapeutic strategies to improve treatment outcomes for patients with BCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254816fb1662c33060d795673833d2ef5dfef42d" target='_blank'>
              Single-cell and spatial transcriptomics reveal TNC-positive cancer-associated fibroblasts that mediate immunosuppression and promote tumor progression in basal cell carcinoma
              </a>
            </td>
          <td>
            Min Luo, Wei Tian, Qianwei Zhuo, Zhenjian Zou, Jinger Lin, Lihang Lin, Rong Li, Yue Han
          </td>
          <td>2025-11-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E. Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Glioblastoma (GBM) is a highly aggressive brain tumor with a median survival of only 15 months. A major challenge in GBM management is the pronounced inter- and intratumoral heterogeneity, which complicates prognosis and therapy. Radiomics, the quantitative extraction of features from medical images, can capture this heterogeneity across the entire tumor volume, but the biological basis of radiographic phenotypes remains poorly understood. Methods We integrated preoperative MRI-based radiomic stratification with multi-platform transcriptomics (bulk RNA-seq, single-cell RNA-seq, and spatial transcriptomics) in IDH-wildtype GBM patients. Unsupervised clustering of radiomic features identified four imaging subtypes. Results Group 4 emerged as a high-risk subtype associated with significantly worse survival and a distinctive MRI pattern of peripheral contrast enhancement. Transcriptomic analyses revealed that Group 4 tumors were enriched in cell-cycle and proliferation markers and exhibited neural stem cell–like gene expression signatures. Single-cell profiling confirmed an elevated proportion of stem-like malignant cells in this subtype. Spatial transcriptomics further demonstrated that these proliferative, stem-like programs were localized predominantly to the tumor periphery, corresponding to the rim-enhancing regions on MRI. Finally, we identified the developmental transcription factor VAX2 as a candidate driver of the Group 4 gene network; functional assays showed that VAX2 promotes GBM cell proliferation in vitro. Discussion Our findings link a radiomics-defined MRI phenotype to specific molecular programs and cell populations in GBM, suggesting that radiomic subtypes can serve as noninvasive biomarkers of tumor biology and highlighting potential therapeutic targets in aggressive, stem-like tumor cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63325a32d6106e3ca9f5bff7f21292a347c5da7b" target='_blank'>
              MRI-based radiomic clustering identifies a glioblastoma subtype enriched for neural stemness and proliferative programs
              </a>
            </td>
          <td>
            Zhongyi Zhang, Yang Liu, Zhicong Zhang, Tian Gui, Youhui Chen, Qian Chen, Xiufu Wu, Li Sun, Sujie Li, Shuyang Wei
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CD8+ T cells are an important weapon in the therapeutic armamentarium against cancer. While CD8+CD103+ T cells with a tissue-resident memory T (TRM) cell phenotype are associated with favorable prognoses, the tumor microenvironment also contains dysfunctional exhausted T (TEX) cells that exhibit a variety of TRM-like features. Here we deconvolute TRM and TEX cells across human cancers, ascribing markers and gene signatures that distinguish these populations and enable their functional distinction. Although TRM cells have superior functionality and are associated with long-term survival post-tumor resection, they are not associated with responsiveness to immune checkpoint blockade. Tumor-associated TEX and TRM cells are clonally distinct, with the latter comprising tumor-independent bystanders and tumor-specific cells segregated from cognate antigen. Intratumoral TRM cells can be forced toward an exhausted fate when chronic antigen stimulation occurs, indicating that the presence or absence of continuous antigen exposure within the microenvironment is the key distinction between tumor-associated TEX and TRM populations. These results highlight unique functions for TRM and TEX cells in tumor control, underscoring the need for distinct strategies to harness these populations for cancer therapies. Here the authors show that tissue-resident memory and exhausted T cells in tumors are distinct populations that are shaped by relative presence or absence of TCR signals, suggesting that a tailored therapeutic strategy is needed to target each subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a712967a1f06be04c88812bdec6b8089c56c0b" target='_blank'>
              Antigen reactivity defines tissue-resident memory and exhausted T cells in tumors
              </a>
            </td>
          <td>
            Thomas N. Burn, Jan Schröder, Luke C. Gandolfo, M. Osman, Elanor N. Wainwright, E. Lam, Keely M. McDonald, Rachel B. Evans, Shihan Li, Daniel Rawlinson, Lachlan Dryburgh, Ali Zaid, Z. Maliga, Dominick Schienstock, Philippa Meiser, H. J. Lee, Hongjin Lai, Marcela L. Moreira, Pirooz Zareie, , Lutfi Huq, Susan N. Christo, Justine Seow, Keith A. Ching, Stéphane M Guillaume, Kathy Knezevic, Simone L. Park, Maximilien Evrard, Jason Waithman, T. Gebhardt, Scott N. Mueller, G. Riddiough, M. V. Perini, S. C. Tsao, Terence P. Speed, P. Sorger, S. Loi, Francis R. Carbone, Stephanie Gras, Timothy S Fisher, Bas J Baaten, M. Dawson, Laura K. Mackay
          </td>
          <td>2025-12-29</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor vascular endothelial cells are essential constituents of the tumor microenvironment, responsible for delivering oxygen and nutrients that are vital for tumor growth and proliferation. As a hallmark of cancer progression, abnormal tumor vasculature contributes to tumor development through multiple mechanisms. Although anti-angiogenic therapies are widely used in the treatment of various cancers, the intrinsic heterogeneity of endothelial cells poses significant challenges regarding therapeutic efficacy. Therefore, further investigation into the heterogeneity of tumor endothelial cells is of paramount importance. The rapid advancement of single-cell sequencing technologies in recent years has facilitated the detailed characterization of heterogeneity among tumor endothelial cells at the single-cell level, thereby fostering a more precise understanding of the functional roles of individual cells within the tumor microenvironment. This technology has become an indispensable tool for investigating the heterogeneity of tumor endothelial cells, offering insights that could inform the refinement of future cancer treatments. In this review, we synthesize findings from the field of single-cell omics to elucidate the heterogeneous characteristics of tumor endothelial cells. We analyze recent advancements in single-cell technology used in the study of tumor cell heterogeneity in terms of both commonalities and distinctive features, covering aspects at the gene and cellular levels. In this review, we provide an overview of recent applications of single-cell sequencing technology in analyzing tumor endothelial cell heterogeneity, offering insights into the development of precise tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a50144e46bbcc6cadd8fab9df5c73d57f4ff4f3" target='_blank'>
              Research Progress Regarding the Use of Single-Cell Sequencing Technology in Analyzing Tumor Endothelial Cell Pathophysiology
              </a>
            </td>
          <td>
            Shu Zhao, S. Liu, Wenxin Shao, Dong Liu
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive adult brain tumor, marked by intratumoral heterogeneity and therapy resistance. Metabolic reprogramming through histone lactylation has been linked to tumor progression and immune suppression. However, the spatial and single-cell landscape of lactylation in GBM and its prognostic significance remain poorly understood. We employed a multi-omics approach integrating bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to investigate lactylation-related signatures in GBM. Differential expression and pathway analyses were performed using GEO and TCGA datasets. Cell clustering, SCENIC transcriptional network inference, CellChat intercellular communication modeling, and pseudotime analysis were conducted. A prognostic risk model was constructed using LASSO-Cox regression based on lactylation-associated genes. Experimental validation was performed using western blotting, immunohistochemistry, and functional assays in GBM cell lines. Lactylation-related genes were significantly upregulated in GBM and associated with poor prognosis and immunosuppressive tumor microenvironments. Single-cell analysis revealed high-lactylation malignant subpopulations enriched in hypoxic tumor cores, exhibiting metabolic reprogramming and enhanced immune evasion. Spatial transcriptomics confirmed the localization of S100A6-high-lactylation GBM cells in aggressive tumor regions. A nine-gene lactylation-based risk model stratified patients into high- and low-risk groups with significantly different survival outcomes (AUC: 0.77–0.87). Experimental knockdown of S100A6 reduced GBM cell proliferation, migration, and invasion. Lactylation defines distinct tumor cell clusters in GBM that are spatially localized, metabolically reprogrammed, and immunosuppressive. The S100A6-associated lactylation signature serves as a robust prognostic biomarker and potential therapeutic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5bf5750b51fd180863b176de6d4f229a2cae4e" target='_blank'>
              Single-cell and spatial transcriptomics reveal lactylation-associated tumor cell clusters and define a prognostic risk model in glioblastoma
              </a>
            </td>
          <td>
            Rui Han, Guangfan Chi, Dongjie Sun, Ziran Xu, Liangfu Zhou, Kan Xu
          </td>
          <td>2025-11-22</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Worldwide, lung cancer stands as the predominant contributor to cancer mortality, characterized by substantial heterogeneity alongside intricate tumor microenvironment interactions. Gaining insight into how immune cells behave across space and time, coupled with profiling key gene expression, offers pathways toward comprehending disease advancement and uncovering therapeutic innovations. Characterization of the lung cancer immune cell landscape was achieved through single-cell RNA sequencing technology. Advanced dimensionality reduction techniques, encompassing UMAP and t-SNE methodologies, enabled multimodal examination of how cells are composed and distributed spatially. Gene Ontology (GO) enrichment alongside pathway analyses were executed to pinpoint biological processes and signaling networks operating within the immune microenvironment. To validate the transcriptomic findings, quantitative real-time PCR (qRT-PCR) was conducted in lung cancer cell lines to assess the relative expression of CXCL12, CXCR4, and PD-L1 using GAPDH as an internal control. Our investigation uncovered an intricate immune cell subpopulation architecture displaying distinctive spatial arrangement patterns. Among various cell types, key genes exhibited particular expression signatures, with tumor-associated macrophages and cytotoxic T cells showing elevated expression levels. Functioning as a principal coordinator of immune cell interactions, the CXCL12-CXCR4 signaling axis stood out prominently. Through pseudotime trajectory examination, we documented evolving patterns in cellular proportions and representative gene expression, reflecting how immune responses transform throughout tumor development. qRT-PCR analysis further confirmed that CXCL12, CXCR4, and PD-L1 were significantly upregulated in lung cancer cell lines, consistent with the single-cell sequencing results. Through investigating immune cell spatiotemporal behaviors and key gene activities, our research illuminates the intricate immune architecture within lung cancer microenvironments. Personalized immunotherapy development will be guided by these insights, which clarify how specific immune cells function while exposing prospective therapeutic intervention points. Additionally, qRT-PCR validation in lung cancer cell lines confirmed the differential expression of CXCL12, CXCR4, and PD-L1, reinforcing the robustness and translational relevance of our single-cell findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73743e96a9fb170b328b9a3bdc455c5b165f804f" target='_blank'>
              Single cell transcriptomic profiling uncovers shared inflammatory pathways and immune dysregulation underlying the asthma to lung cancer transition
              </a>
            </td>
          <td>
            He Huang, Gang Teng, Jing Zhou, Ling Liu, Mei Dong, Wei Chen, Nianzhi Zhang
          </td>
          <td>2025-12-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The human brain contains a milieu of extracellular matrix (ECM) components that promote normal development and physiology. ECM signaling pathways are often dysregulated in brain pathologies including the malignant cancer glioblastoma (GBM). Here, we used single-cell spatial transcriptomic platforms to map the expression of nearly 400 ECM genes in matching non-cancerous brain and GBM samples. At least four different GBM cell populations have been identified that show unique ECM expression profiles and spatial enrichment in distinct intratumor regions. Spatial mapping demonstrates largely non-overlapping expression signatures of ECM components in GBM stromal cell types, particularly in vascular endothelial cells and microglia/macrophages. Comparisons of GBM versus lower grade astrocytoma samples identifies differential expression of key ECM components. Computational analysis reveals novel ECM ligand-receptor networks between GBM and stromal cells. This spatial atlas provides new insights into ECM control of brain tumor initiation and progression and identifies potential targets for therapy in GBM. Using spatial transcriptomics, this study profiled the expression of hundreds of matrisome genes at single-cell resolution in human brain tumors, revealing targetable adhesion and signalling components in glioblastoma and lower-grade gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7358fd5e226c097b2efb1a3758a0fef198cc402b" target='_blank'>
              Single cell spatial profiling of the matrisome identifies region-specific adhesion and signaling networks in glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A Forero, Ali Pirani, John E. Morales, Marisol De La Fuente-Granada, Sumod Sebastian, Jason T Huse, Leomar Ballester, J.S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2025-12-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The bone marrow (BM) is a complex and compartmentalized tissue where spatial context plays a critical role in regulating cell behavior, signaling, and disease progression. To capture these dynamics, we apply spatial transcriptomics using the Visium Spatial Gene Expression platform on formalin-fixed paraffin-embedded (FFPE) BM sections from both healthy and Multiple Myeloma (MM) mouse models, as well as MM patient samples. Overcoming the technical challenges of working with mineralized long bone tissue, we develop a custom analytical framework integrating spatial and single-cell transcriptomic data to map cellular composition and interactions in situ. This approach enables the spatial characterization of transcriptionally heterogeneous malignant plasma cells (MM-PC) and their surrounding microenvironments. We identify spatially distinct gene programs linked to MM pathogenesis, including signatures of NETosis and IL-17 signalling, which are reduced in MM-PC–rich regions. Additionally, a transition gradient from effector to exhausted T cell phenotype is associated with increased remoteness from MM-PC. These spatial patterns are identified in FFPE BM biopsies from MM patients with varying tumor burdens. In summary, our study demonstrates both the capabilities and limitations of Visium technology in characterizing spatially regulated mechanisms underlying MM pathogenesis. Visium Spatial technology enables the characterization of transcriptionally heterogeneous malignant plasma cells in multiple myeloma, and identify spatially distinct gene programs linked to MM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f354b66d39731b9cc107b87037fbdf9278486f" target='_blank'>
              Characterization of the bone marrow architecture of multiple myeloma using spatial transcriptomics
              </a>
            </td>
          <td>
            E. Muiños-López, Ana Rosa López-Pérez, Laura Sudupe, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, P. Aguirre-Ruiz, P. S. Martín-Uriz, M. Larrayoz, L. Alvarez-Gigli, M. Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, L. Tamariz-Amador, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Resistance to immune checkpoint blockade (ICB) remains a major challenge in lung adenocarcinoma (LUAD), with stromal mechanisms underlying CD8⁺ T cell exhaustion still poorly understood. By integrating single-cell, bulk, and spatial transcriptomic datasets using EcoTyper, we identified a distinct immunosuppressive ecotype, EC10, enriched for TGF-β signaling and epithelial–mesenchymal transition. EC10 exhibited spatial co-localization of fibroblasts, malignant epithelial cells, and exhausted CD8+ T cells, and was consistently associated with immune exclusion, poor progression-free survival, and elevated TIDE scores across four ICB-treated LUAD cohorts. Cell–cell communication analyses revealed a dominant TGFB1–SERPINE1 signaling axis originating from fibroblasts, linking stromal remodeling to T cell dysfunction. In contrast, EC12 represented an inflamed, ICB-responsive state enriched in interferon signaling. These findings define EC10 as a spatially organized, fibroblast-driven immunosuppressive ecosystem predictive of ICB resistance, and highlight the therapeutic potential of targeting the TGF-β axis in LUAD. This research identifies a fibroblast-driven immunosuppressive ecosystem in lung adenocarcinoma, linking TGF-β signaling to immune resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130525b051ffbdab87e5edbec0e75b95564de9c8" target='_blank'>
              Fibroblast TGF-β signaling defines spatial tumor ecosystems linked to immune checkpoint blockade resistance
              </a>
            </td>
          <td>
            Seo-Young Lee, Youngjoo Lee
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial Transcriptomics (spTx) offers unprecedented insights into the spatial arrangement of the tumor microenvironment, tumor initiation/progression and identification of new therapeutic target candidates. However, spTx remains complex and unlikely to be routinely used in the near future. Hematoxylin and eosin (H&E) stained histological slides, on the other hand, are routinely generated for a large fraction of cancer patients. Here, we present a novel deep learning-based approach for multiscale integration of spTx with tumor morphology (MISO). We trained MISO to predict spTx from H&E on a new unpublished dataset of 72 10X Genomics Visium samples, and derived a novel estimate of the upper bound on the achievable performance. We demonstrate that MISO enables near single-cell-resolution, spatially-resolved gene expression prediction from H&E. In addition, MISO provides an effective patient representation framework that enables downstream predictive tasks such as molecular phenotyping or MSI prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b66b910856826f08a930be56d475ed3036cfdbc" target='_blank'>
              A deep learning-based multiscale integration of spatial omics with tumor morphology
              </a>
            </td>
          <td>
            B. Schmauch, L. Herpin, Antoine Olivier, Thomas Duboudin, Rémy Dubois, Lucie Gillet, Jean-Baptiste Schiratti, Valentina Di Proietto, Delphine Le Corre, Alexandre Bourgoin, Pr. Julien Taïeb, Pr. Jean-François Emile, Pr. Wolf H. Fridman, Elodie Pronier, P. Laurent-Puig, Eric Y. Durand
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is one of the most aggressive brain tumors because its cells spread widely through the brain, making complete removal impossible. We implanted fluorescently labeled glioma stem-like cells (GSCs) into the mouse brain and observed their infiltrative growth using advanced 3D imaging. Within 6–10 weeks, tumors caused about 15% brain swelling and showed a strong preference for invading the corpus callosum, a major white matter tract. By measuring Sox2-positive cells, we found very high densities in the tumor bulk (over 4000 cells/mm2) but much lower levels at the invasive margin (about 30 cells/mm2) and a clear gradient in the corpus callosum (3010 to 1.5 cells/mm2 across 1 mm). Using CLARITY tissue clearing combined with light-sheet microscopy, we visualized how tumor cells infiltrate along blood vessels and brain structures, adopting distinct shapes and strategies. Single-cell gene analysis of the invading corpus callosum cells identified a 7-gene “invasion signature” (NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, TGFB1), which was linked to worse survival and may represent new therapeutic targets. Abstract Background/Objectives: High-grade gliomas (HGGs), including glioblastomas, are among the most aggressive brain tumors due to their high intratumoral heterogeneity and extensive infiltration. Glioma stem-like cells (GSCs) frequently invade along white matter tracts such as the corpus callosum, but the molecular programs driving this region-specific invasion remain poorly defined. The aim of this study was to identify transcriptional signatures associated with GSC infiltration into the corpus callosum. Methods: We established an orthotopic xenograft model by implanting fluorescently labeled human GSCs into nude mouse brains. Tumor growth and invasion patterns were assessed using tissue clearing, light-sheet fluorescence microscopy, and histological analyses. To characterize region-specific molecular profiles, we performed microfluidic-based single-cell RNA expression analysis of 48 invasion- and stemness-related genes in cells isolated from the tumor bulk (TB) and corpus callosum (CC). Results: By six weeks post-implantation, GSCs displayed marked tropism for the corpus callosum, with distinct infiltration patterns captured by three-dimensional imaging. Single-cell gene expression profiling revealed significant differences in 7 of the 48 genes (14.6%) between TB- and CC-derived GSCs. These genes—NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, and TGFB1—collectively defined a “corpus callosum invasion signature” (CC-Iv). CC-derived cells showed a unimodal, high-expression profile of CC-Iv genes, whereas TB cells exhibited bimodal distributions, suggesting heterogeneous transcriptional states. Importantly, higher CC-Iv expression correlated with worse survival in patients with low-grade gliomas. Conclusions: This multimodal approach identified a corpus callosum-specific invasion signature in glioma stem-like cells, revealing how local microenvironmental cues shape transcriptional reprogramming during infiltration. These findings provide new insights into the spatial heterogeneity of gliomas and highlight potential molecular targets for therapies designed to limit tumor spread through white matter tracts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561590a9b7ca738409c5933b00e64b421edbb629" target='_blank'>
              Innovative In Vivo Imaging and Single Cell Expression from Tumor Bulk and Corpus Callosum Reveal Glioma Stem Cells with Unique Regulatory Programs
              </a>
            </td>
          <td>
            Natália dos Santos, Aline Aquino, Friedrich Preusser, Fabio Rojas Rusak, E. H. Jandrey, Miyuki Uno, T. K. Furuya, C. Lancellotti, M. Maldaun, Roger Chammas, Stephan Preibisch, A. Camargo, C. Masotti, Erico Tosoni Costa
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, A. Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment (TME) has emerged as a promising source of prognostic biomarkers. To fully leverage its potential, analysis methods must capture complex interactions between different cell types. We propose HiGINE -- a hierarchical graph-based approach to predict patient survival (short vs. long) from TME characterization in multiplex immunofluorescence (mIF) images and enhance risk stratification in lung cancer. Our model encodes both local and global inter-relations in cell neighborhoods, incorporating information about cell types and morphology. Multimodal fusion, aggregating cancer stage with mIF-derived features, further boosts performance. We validate HiGINE on two public datasets, demonstrating improved risk stratification, robustness, and generalizability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d207f0757528c944b801713e062bc7358ff14382" target='_blank'>
              From Cells to Survival: Hierarchical Analysis of Cell Inter-Relations in Multiplex Microscopy for Lung Cancer Prognosis
              </a>
            </td>
          <td>
            Olle Edgren Schullerqvist, Jens Baumann, Joakim Lindblad, Love Nordling, Artur Mezheyeuski, Patrick Micke, Natavsa Sladoje
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is an aggressive brain tumor with limited treatment options, and the tumor microenvironment plays a major role in its progression. Among stromal cells, cancer-associated fibroblasts (CAFs) are known to influence tumor growth and immune responses, but their diversity in glioblastoma has not been well understood. In this study, we analyzed both bulk and single-cell RNA sequencing data to identify CAF subtype-associated patterns within glioblastoma. We discovered four CAF populations—immune, inflammatory, antigen-presenting, and myofibroblastic—each associated with poor patient survival and unique immune and metabolic features. We also identified Spleen Tyrosine Kinase (SYK) as a key regulatory gene shared across these CAF subtypes. Our findings highlight the importance of stromal heterogeneity in glioblastoma and suggest that targeting CAF-related signaling, including SYK, may offer new therapeutic opportunities for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eebcce441ee2b3c1609920849b71d1de5f2a30" target='_blank'>
              Dissecting CAF Heterogeneity in Glioblastoma Reveals Prognostic Subtypes and a Central Regulatory Role for Spleen Tyrosine Kinase (SYK)
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Advances in single-cell and spatial transcriptomics have transformed our understanding of tumor cell states and lineage dynamics, enabling high-resolution investigation of cellular heterogeneity and evolution. Motivation Building on Schiffman et al.’s Markovian framework for inferring cell state transitions from single-cell lineage tracing, there is a need for models that integrate spatial context to better capture the dynamics of tumor evolution. Method and results We introduce StateSim, a spatial–temporal model that simulates tumor cell state transitions within tissue, incorporating spatial growth and mutation accumulation (Waclaw, et al. 2015). StateSim assumes that daughter cells are spatial neighbors, shaping the spatial organization of cell states, and models two cell states with distinct birth and death rates. To quantify spatial patterns, we developed StateMap, a network-based approach that maps tumor cell states and their spatial relationships, summarizing transition patterns through network statistics. Applying StateSim under various transition scenarios, we demonstrate that distinct transition modes yield unique spatial statistics, robustly correlated with transition rates (Pearson r = 0.9938, P = 6.87e-10). Using spatial transcriptomics data from glioma samples (Greenwald, et al. 2024), StateMap identified subgroups with distinct transition patterns, further validated by spatial statistics such as bivariate Moran’s I. Approximate Bayesian computation with StateSim enabled inference of posterior probabilities for cell state transition rates and phylogenetic trajectories. Conclusion Our framework enables inference of cell state transitions and in-silico lineage tracing from spatial data, advancing studies of tumor heterogeneity and evolution. Significance This work provides a novel computational approach for integrating spatial and temporal information to decipher the dynamics of cell state transitions and lineage development during cancer evolution, offering new insights into tumor heterogeneity and progression. References 1. Schiffman J.S., et al. ‘Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.’ Nature Genetics 2024; 56(10):2174–2184. 2. Waclaw B., et al. ‘A spatial model predicts that dispersal and cell turnover limit intratumour heterogeneity.’ Nature 2015; 525(7568):261–264. 3. Greenwald A.C., et al. ‘Integrative spatial analysis reveals a multi-layered organization of glioblastoma.’ Cell 2024; 187(10):2485–2501.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c607caf6be05bb9a4600cf6079421f01ab3b8b" target='_blank'>
              Decipher the spatial dynamics of the cell state transition and lineage development during cancer evolution
              </a>
            </td>
          <td>
            Jiabao Li, Jiguang Wang
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0135229fffb59fe03ba95bebb58be45a4b3470c" target='_blank'>
              De novo H3.3K27M-altered Diffuse Midline Glioma in human brainstem organoids to dissect GD2 CAR T cell function
              </a>
            </td>
          <td>
            N. Bessler, A. Wezenaar, H. Ariese, C. Honhoff, E. J. Wehrens, N. Dommann, C. Ruiz Moreno, Thijs J M van den Broek, R. Collot, D.J. Kloosterman, F. Keramati, M. Roosen, S. de Blank, Esmée J. van Vliet, M. Barrera Román, L. C. Gatti, Ali Ertürk, J. Kuball, Z. Sebestyén, M. Kool, S. Patrizi, E. Miele, A. Künkele, M. Kranendonk, Annelisa M. Cornel, Stefan Nierkens, Christian Mayer, H. Stunnenberg, A. Alemany, Maria Alieva, A. Rios
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) exhibits pronounced spatial heterogeneity that limits therapeutic efficacy. The contribution of epithelial–mesenchymal transition (EMT) to regional tumor progression and immune evasion remains incompletely understood. Methods We integrated bulk transcriptomic datasets (TCGA, GSE14520), 34 single-cell RNA-seq samples, and 15 spatial transcriptomic datasets to delineate EMT activity across distinct HCC regions. Immune infiltration profiling, pathway enrichment, and multi-model machine learning were used to identify candidate EMT regulators. Functional validation was performed in Hep3B cells through wound healing, Transwell migration/invasion, and immunofluorescence assays. Results EMT activity was significantly elevated at tumor margins (fold change = 2.7, p < 0.001) and was associated with poorer overall survival (HR = 2.15, 95% CI: 1.41–3.27, p < 0.001). Regions with high EMT signatures showed reduced CD8⁺ T-cell infiltration and increased immunosuppressive cells, including MDSCs, M2 macrophages, and Tregs, along with elevated expression of immune checkpoints (PDCD1, CTLA4, LAG3). Among candidate regulators, UBC was consistently ranked as a top EMT-associated gene across all models. Functional assays confirmed that UBC overexpression enhanced migration, invasion, and vimentin expression, whereas UBC knockdown reversed these effects. Conclusion Through integrative spatial multi-omics and experimental validation, we identify UBC as a key mediator of EMT and immune suppression at HCC margins. These findings provide mechanistic insight into spatial heterogeneity and suggest that targeting UBC could have translational potential for overcoming immune evasion in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15e71d7d57f5aafdc6998bcde18c6af7f7890890" target='_blank'>
              Integrative Spatial Transcriptomics and Experimental Validation Reveal UBC-Mediated EMT Associated with Immune Evasion in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Xiaosong Li, Xian Qin, Kezhi Shi, Guangrui Lu, Guodong Tian, Yue Chen, Rucheng Yao
          </td>
          <td>2025-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly heterogeneous malignancy with poor prognosis, in which tumor metastasis represents a critical event driving disease progression and treatment failure. Understanding the cellular drivers of ICC metastasis and establishing effective risk assessment models are essential for improving patient outcomes. We analyzed the GSE201425 single-cell RNA sequencing dataset to explore the cellular origins and mechanisms of ICC metastasis. Copy number variation (CNV) profiling and clonal evolution analysis were performed to identify malignant epithelial cells. Pseudotime trajectory analysis revealed metastasis-associated epithelial cells (MAECs). Key biomarkers within MAECs were screened, and a Metastasis Index (Met-Index) based on one-class logistic regression was constructed to assess metastasis risk in bulk RNA-seq data from the TCGA-CHOL cohort. Multiplex immunofluorescence staining was performed in 34 clinical ICC specimens. Functional validation was conducted in HuCCT1 cells using siRNA-mediated gene silencing, followed by CCK-8 proliferation assays and wound healing assays. MAECs were identified as a distinct epithelial subpopulation enriched in metastatic ICC samples. Biomarkers including MMP7, FXYD2, and PTHLH were upregulated in MAECs and strongly correlated with metastatic status. The Met-Index effectively stratified patients into high- and low-risk groups, with the high Met-Index group exhibiting significantly worse overall survival (p = 0.005) and progression-free survival (p = 0.015). Immunofluorescence validation confirmed increased expression of MMP7, FXYD2, and PTHLH in metastatic tumors, which correlated with advanced clinicopathological features. Functional assays demonstrated that knockdown of these biomarkers inhibited HuCCT1 cell proliferation and suppressed cell migration capacity. This study delineates the cellular origin and molecular drivers of ICC metastasis at the single-cell level, identifies MAECs as a metastasis-related subpopulation, and establishes a clinically relevant Met-Index with prognostic value. These findings provide potential biomarkers and therapeutic targets for the early identification and management of ICC patients at high risk of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27d855cd27f4e8545b1968d216eb341e556b791" target='_blank'>
              Single-cell multi-omics analysis reveals drivers of intrahepatic cholangiocarcinoma metastasis
              </a>
            </td>
          <td>
            Zetian Zhang, Hao Dou, Shangyumeng Zhao, Jiaqi Wang, Meiru Chen, Xiaoli Xie, Xiangcong Tian, Dongqiang Zhao
          </td>
          <td>2025-11-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression.
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) mutation is a key oncogenic driver in lung adenocarcinoma (LUAD), but its impact on the tumor immune microenvironment (TIME) remains unclear. By integrating single-cell transcriptomes from 153 LUAD samples using machine learning, we generated an atlas of over one million cells that delineates immune heterogeneity. EGFR-mutant tumors exhibited enrichment of TIGIT+regulatory T cells, neutrophils, and macrophages, whereas wild-type tumors contained abundant ZNF683+CD8+tissue-resident memory T cells, diverse memory B cells, and FGFBP2+CD16high natural killer cells, reflecting an immune-active TIME. Non-negative matrix factorization defined five TIME subtypes, with EGFR-mutant patients clustering into immunosuppressive profiles linked to poor prognosis. Flow cytometry and mouse models confirmed the cytotoxic and PD-1 blockade-enhancing functions of FGFBP2+NK cells. These findings reveal distinct TIME landscapes in EGFR-mutant LUAD and illustrate the potential of machine learning-based immunogenomic analysis to inform precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad6284204cf8fc0e7c47d8fa667f142a64e34e5" target='_blank'>
              Machine learning identifies TIME subtypes linking EGFR mutations and immune states in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zetian Gong, Mingjun Du, Ying Li, Bicheng Ye, Yuming Huang, Hui Gong, Wei Wang, Liang Chen, Zongli Ding, Pengpeng Zhang
          </td>
          <td>2025-11-26</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Bladder cancer is a common urinary system malignancy with a complex immune microenvironment that affects treatment outcomes. Single-cell RNA sequencing technology offers new opportunities to understand cellular heterogeneity within tumors, but systematic analysis of immune-related genes in bladder cancer remains limited. We performed multi-omics analysis on bladder cancer samples using TCGA database analysis, single-cell RNA sequencing, cell clustering, immune gene expression analysis, and network pharmacology to study cellular heterogeneity and potential therapeutic targets. We identified 8 upregulated immune genes significantly associated with poor survival: CCL20, IL17RB, IL37, PLAU, INSL4, KNG1, SLURP1, and PAEP. Single-cell analysis revealed major cell types including B cells, T cells, dendritic cells, epithelial cells, fibroblasts, mast cells, monocytes, and smooth muscle cells. We identified 5 distinct fibroblast subgroups and 10 T cell subgroups with different functional characteristics. Network pharmacology analysis of the PLAU gene showed that traditional Chinese medicine compounds had good binding capacity with the PLAU protein, validated by molecular dynamics simulation. This study identified 8 prognosis-related bladder cancer immune genes and revealed tumor microenvironment heterogeneity. The findings, particularly the PLAU therapeutic target, provide important evidence for bladder cancer immunotherapy and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7804e2c979249e5eb150897b03ef7412bb3e32eb" target='_blank'>
              Single cell RNA sequencing decodes cellular heterogeneity and identifies prognostic immune signatures in bladder cancer microenvironment
              </a>
            </td>
          <td>
            Kai Cao, Yunfeng Shi, Bin Wu, Honglei Shi, Zhong Lv, Hongqian Guo, Rong Yang
          </td>
          <td>2025-11-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Çetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cf48b273167de19bd485ee182b5df8f720d9f6b" target='_blank'>
              Spatially distinct chromatin compaction states predict neoadjuvant chemotherapy resistance in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Reut Mealem, T. A. Phillips, Leor Ariel Rose, S. Marcotti, M. Parsons, A. Zaritsky
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Remodeling of the tumor microenvironment (TME) under therapeutic pressure is a critical determinant of treatment response and resistance in hepatocellular carcinoma (HCC). Triple-combination therapy integrating targeted agents, immune checkpoint inhibitors, and radiotherapy (T+I+R) has shown potential synergistic effects in intermediate to advanced HCC, particularly in patients with portal vein tumor thrombus (PVTT), yet the spatial and cellular mechanisms underlying its efficacy remain largely unknown. In this retrospective clinical cohort study, we compared T+I+R with targeted therapy plus radiotherapy (T+R) in advanced HCC, and further employed single-cell spatial transcriptomics and spatial proteomics to generate an integrated multi-omics atlas mapping tumor and stromal compartments, cellular compositions, and intercellular interactions with spatial resolution. Clinically, T+I+R achieved superior tumor shrinkage and disease control compared with T+R. Spatial multi-omics revealed marked region-specific remodeling, with myofibroblastic cancer-associated fibroblasts, angiogenic tip endothelial cells, and conventional dendritic cells enriched at the invasive margin and associated with therapeutic resistance, while CD8+ effector T cells were redistributed away from immunosuppressive niches, a spatial configuration correlating with enhanced response. These findings identify spatial segregation between cytotoxic and suppressive immune elements as a potential hallmark of effective therapy, providing a high-resolution spatial framework for understanding T+I+R induced TME remodeling and offering mechanistic insights to guide biomarker discovery and the optimization of combination strategies in advanced HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6c3edb9469068b18d4bde88276d91593b43ce6" target='_blank'>
              Spatial multi-omics mapping of tumor microanatomy dynamics following radiotherapy combined with targeted-immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Fansen Ji, Haoming Xia, Ying Xiao, Jiawei Zhang, Hao Chen, Bingjun Tang, Huan Li, Hao Liu, Boyang Wu, Xiaojuan Wang, Shizhong Yang, Jiahong Dong
          </td>
          <td>2026-01-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="To identify the cell types that are infected with KSHV and the immune interactions in Kaposi sarcoma (KS) lesions, we performed spatial transcriptomics with seven KS skin tumors. We used a single‐cell RNA‐sequencing reference data set from healthy skin donors with a method to conduct spatially informed cell‐type deconvolution for spatial transcriptomics. This allowed us to predict the relative amounts of each cell type within the patient sample sections. We included custom probes for five KSHV genes that allowed us to measure human and KSHV expression patterns at the same time. We then compared the spatial gene expression data of KS skin samples with six normal skin samples and found higher expression of marker genes corresponding to macrophages/dendritic cells, lymphatic endothelial cells, and vascular endothelial cells in the KS skin lesions when compared to normal skin samples. Our spatial transcriptomic results from thousands of spots across multiple KS tumors indicated a correlation between high levels of STC1 and decreased expression of macrophage markers. Together, these analyses offer potential mechanisms by which KSHV infection may remodel skin tissue, inhibit immune responses against KSHV infection, and confer resistance to anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0e2360131ded34f6dbf590cb05f764fc0e85cc" target='_blank'>
              Spatial Transcriptomics of Patients With Kaposi Sarcoma Identifies Mechanisms of Immune Evasion
              </a>
            </td>
          <td>
            Bahman Afsari, R. Ramaswami, K. Lurain, Takanobu Tagawa, Daphne Knudsen-Palmer, Guruswamy Mahesh, Ameera Mungale, R. Yarchoan, Joseph Ziegelbauer
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Emerging evidence suggests a correlation between CD8+ T cell–tumour cell proximity and anti-tumour immune response1,2. However, it remains unclear whether these cells exist as functional clusters that can be isolated from clinical samples. Here, using conventional and imaging flow cytometry, we show that from 21 out of 21 human melanoma metastases, we could isolate heterotypic clusters, comprising CD8+ T cells interacting with one or more tumour cells and/or antigen-presenting cells (APCs). Single-cell RNA-sequencing analysis revealed that T cells from clusters were enriched for gene signatures associated with tumour reactivity and exhaustion. Clustered T cells exhibited increased TCR clonality indicative of expansion, whereas TCR-matched T cells showed more exhaustion and co-modulation when conjugated to APCs than when conjugated to tumour cells. T cells that were expanded from clusters ex vivo exerted on average ninefold increased killing activity towards autologous melanomas, which was accompanied by enhanced cytokine production. After adoptive cell transfer into mice, T cells from clusters showed improved patient-derived melanoma control, which was associated with increased T cell infiltration and activation. Together, these results demonstrate that tumour-reactive CD8+ T cells are enriched in functional clusters with tumour cells and/or APCs and that they can be isolated and expanded from clinical samples. Typically excluded by single-cell gating in flow cytometry, these distinct heterotypic T cell clusters are a valuable source to decipher functional tumour–immune cell interactions and may also be therapeutically explored. Tumour-reactive CD8+ T cells are enriched in functional clusters with tumour cells and/or antigen-presenting cells and can be isolated and expanded from clinical samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1100b9077e6e7f158509d2bfcdf86085e31d43e6" target='_blank'>
              Tumour-reactive heterotypic CD8 T cell clusters from clinical samples
              </a>
            </td>
          <td>
            Sofía Ibáñez-Molero, J. Veldman, Juan Simon Nieto, Joleen J H Traets, Austin George, K. Hoefakker, Anita Karomi, R. Harkes, Bram van den Broek, Su Min Pack, L. Tas, Nils L Visser, Susan E. van Hal-van Veen, P. Alóndiga-Mérida, M. Alkemade, I. M. Seignette, R. Tissier, M. Nieuwland, M. van Baalen, Joanna Poźniak, Erik Mul, S. Tol, S. Stenqvist, L.M. Nilsson, Jonas A Nilsson, J. Haanen, W. V. van Houdt, D. Peeper
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>2</td>
          <td>59</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a084b4c5d54edf23648139fa8024008066585a8" target='_blank'>
              Beyond genomics: 3D microtumor assays for rapid, clinically relevant functional drug testing
              </a>
            </td>
          <td>
            Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-11-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Glioma remains the most aggressive and therapy-resistant brain tumor, with a highly immunosuppressive tumor microenvironment. The role of inflammatory signaling in glioma progression and treatment response is poorly understood. Methods We performed single-cell RNA sequencing (scRNA-seq) analysis on 93,027 cells from 18 samples. Inflammation-related genes were identified using hdWGCNA and AUCell scoring. Multiple bulk RNA-seq and microarray datasets were integrated for validation. Machine learning algorithms, including CoxBoost, LASSO, and Random Survival Forest, were used to identify prognostic genes. Immune infiltration, immunotherapy response, and mutational landscape were analyzed using established computational tools. Findings COL8A1 was found to be a significant prognostic gene within a highly linked gene module connected to inflammation. Astrocytes, OPCs, and cancerous cells all had high levels of COL8A1 expression. In several cohorts, low survival was linked to high COL8A1 expression. The suppression of tumor migration and proliferation by COL8A1 knockdown was validated by functional tests. Multiple immunotherapy determinants, inhibitory immunological checkpoints, and immune cell infiltration all showed high correlations with COL8A1 expression. Additionally, it accurately forecasted the immune checkpoint blockade response. Through mutational profiling, we identified distinct somatic mutation patterns distinguishing COL8A1-high from COL8A1-low cancers. Conclusion By connecting tumor-intrinsic inflammation to immunological surveillance and treatment resistance, our study identified COL8A1 as a crucial inflammatory hub in glioma. In order to improve the results of immunotherapy for glioma, COL8A1 may be a useful therapeutic target and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf0463c9a68cc9f41847f121143f02d93fcd0752" target='_blank'>
              COL8A1 as a pro-inflammatory mediator bridges immune evasion and therapy resistance in glioma
              </a>
            </td>
          <td>
            Jiachong Wang, Jiale Li, Jun Peng, Chunyuan Zhang, Zigui Chen, Changfeng Miao, Chunhai Tang, Qisheng Luo
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, , S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors.
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c14f73e77a88fcb7e1149959d53563fad60dc97" target='_blank'>
              Liquid Crystal-like self-organization of Glioblastoma Prevents cell density induced migratory arrest
              </a>
            </td>
          <td>
            Urszula Hohmann, Chalid Ghadban, J. Prell, Christian Strauss, Faramarz Dehghani, T. Hohmann
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3242a7ddb3cf6f11fc73d58584fbd06dec0792" target='_blank'>
              Glioma cells achieve malignant progression by fusion with macrophages to gain high SLC7A5 expression
              </a>
            </td>
          <td>
            Yuedong Hu, Fusheng Liu, Rui-fang Mi
          </td>
          <td>2025-12-16</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            , , M. L. Abel, D. S. Vargas Solivan, K. Bhadresha, S. Kartal, S. Nichols, K. Parmar, J. Twohig, T. M. Truong, C. H. Chau, K. Kelly, W. Figg, A. Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>None</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) harbors a dynamic tumor microenvironment (TME) in which macrophages are highly abundant and plastic. Under physiological conditions, macrophages switch between inflammatory and resolution/tissue-repair programs to maintain homeostasis; however, during hepatocarcinogenesis these programs are reprogrammed into tumor-associated macrophage (TAM) states that foster immune suppression, angiogenesis, and tumor progression. Purpose: To summarize macrophage heterogeneity and polarization mechanisms in HCC, and to highlight omics-informed therapeutic opportunities for targeting TAMs and improving precision immunotherapy. Research Design: This review summarizes physiological macrophage polarization and the mechanistic basis of macrophage reprogramming in the hepatocellular carcinoma immune microenvironment, integrating evidence from recent advances in single-cell sequencing, multi-omics, and spatial transcriptomics, with a focus on macrophage subset diversity, key regulatory pathways governing polarization and function, and emerging macrophage-targeted interventions and biomarkers. Results: Recent single-cell and spatial multi-omics studies reveal substantial TAM heterogeneity and plasticity in HCC. Macrophage-targeted strategies—including TAM depletion, phenotypic reprogramming, and exosome-mediated drug delivery—show encouraging preclinical efficacy. Macrophage-associated prognostic models and biomarkers may support individualized immunotherapeutic approaches. Conclusions: Macrophage polarization in HCC represents a dynamic continuum that is essential for homeostasis but is co-opted by tumors to drive immunosuppression and tissue remodeling. Advances in single-cell and spatial multi-omics are redefining TAM subsets and actionable pathways, enabling more rational macrophage-targeted therapies. However, challenges remain in standardizing TAM definitions, identifying robust predictive biomarkers, minimizing off-target effects, and optimizing combinations with immunotherapy. Integrating longitudinal multi-omics with AI-based modeling may help predict macrophage state transitions, guide patient-specific regimens, and advance precision medicine in HCC. Plain language summary Hepatocellular carcinoma is a highly aggressive cancer. Studies have found that immune cells within tumors—particularly macrophages—play a crucial role in cancer development and therapeutic response. Macrophages have two main “personalities”: the M1 type, which can kill cancer cells, and the M2 type, which supports cancer growth. They shift between these states in response to signals from the tumor microenvironment. Both liver-resident Kupffer cells and infiltrating macrophages are involved in this process. Various molecules, such as cytokines, enzymes, and exosomes, influence their polarization. This polarized state of macrophages, in turn, affects key tumor processes such as stemness, proliferation, metastasis, and angiogenesis. In recent years, scientists have been exploring drugs that specifically target macrophage polarization to suppress liver cancer, offering new directions for future therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9c3d3369daa7c8bd747ff210bf91cca4945570" target='_blank'>
              Macrophage Polarization in the Tumor Microenvironment of Hepatocellular Carcinoma: From Mechanistic Insights to Translational Therapies
              </a>
            </td>
          <td>
            Xiaoqing Fu, Mingquan Pang, Zhixin Wang, Haijiu Wang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Control: Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Angioimmunoblastic T-cell lymphoma (AITL) represents the second most prevalent subtype of peripheral T-cell lymphoma, characterized by a dismal prognosis. However, a systematic exploration of tumor microenvironment (TME) features and their prognostic significance in AITL remains notably deficient. To address this knowledge gap, we conducted spatial transcriptome sequencing (ST-SEQ) and whole-exome sequencing in four AITLs and two noncancerous lymph nodes for discovery purposes, complemented by immunohistochemistry analyses on 37 AITL cases for validation. We identified 14 ST clusters, including five neoplastic clusters, wherein a global shift in B-cell phenotypes and enrichment of myeloid cells were observed. These findings underscore a hallmark of exacerbated inflammation and immune dysregulation within the neoplastic TME. Among the 4 ST-sequenced AITLs, 3 expressed high CD40-CD40LG activity, accompanied by the upregulation of immune-suppressive-associated genes, such as CCL17 and PDCD1. Conversely, the remaining patient displayed an uncommon absence of CD40-CD40LG activity but harbored a phagocytosis-associated tumor-associated macrophage (TAM)-enriched TME, which correlated with significantly reduced relapse rates and longer event-free survival (EFS), highlighting the critical value of precise TME stratification in tailoring AITL therapeutic strategies. Finally, trajectory analysis unveiled a distinct trajectory of molecular evolution within this TME landscape. Collectively, our findings illuminate the heterogeneity and prognostic implications of the TME in AITL, providing a robust foundation for the rational design of targeted immunotherapeutic approaches. These insights may substantially advance the development of personalized treatment strategies for AITL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47887a6371994feccf99a507b06079fcceafb4cb" target='_blank'>
              Revealing tumor microenvironmental heterogeneity and prognostic value in angioimmunoblastic T-cell lymphoma via spatial transcriptome sequencing
              </a>
            </td>
          <td>
            Xiang Zhang, Yong Sun, Duo Wu, Fang Yu, Hanjin Yang, X. Ye, Juying Wei, Xuewu Zhang, Yanan Zhu, Yunfei Lv, Zijing Xu, Yuxiang Chen, Hongyan Tong, Jie Jin, Xiaofei Ye, Wenjuan Yu
          </td>
          <td>2026-01-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables simultaneous mapping of tissue morphology and spatially resolved gene expression, offering unique opportunities to study tumor microenvironment heterogeneity. Here, we introduce a computational framework that predicts spatial pathway activity directly from hematoxylin-and-eosin-stained histology images at microscale resolution 55 and 100 um. Using image features derived from a computational pathology foundation model, we found that TGFb signaling was the most accurately predicted pathway across three independent breast and lung cancer ST datasets. In 87-88% of reliably predicted cases, the resulting spatial TGFb activity maps reflected the expected contrast between tumor and adjacent non-tumor regions, consistent with the known role of TGFb in regulating interactions within the tumor microenvironment. Notably, linear and nonlinear predictive models performed similarly, suggesting that image features may relate to pathway activity in a predominantly linear fashion or that nonlinear structure is small relative to measurement noise. These findings demonstrate that features extracted from routine histopathology may recover spatially coherent and biologically interpretable pathway patterns, offering a scalable strategy for integrating image-based inference with ST information in tumor microenvironment studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29b8964f22d869e7cd99fbb783574fbfe821242b" target='_blank'>
              Digital Modeling of Spatial Pathway Activity from Histology Reveals Tumor Microenvironment Heterogeneity
              </a>
            </td>
          <td>
            Ling Liao, Changhuei Yang, Maxim Artyomov, Mark Watson, Adam Kepecs, Haowen Zhou, Alexey Sergushichev, R. Cote
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples are the gold standard for tissue preservation in clinical and research settings. Current single-cell chromatin accessibility technologies cannot resolve cell-type-specific epigenetic profiles in FFPE tissues due to extensive DNA damage. We present scFFPE-ATAC, a high-throughput single-cell chromatin accessibility assay for FFPE samples that integrates an FFPE-adapted Tn5 transposase, ultra-high-throughput DNA barcoding (>56 million barcodes per run), T7 promoter-mediated DNA damage repair, and in vitro transcription. We benchmark scFFPE-ATAC on FFPE mouse spleen and validate its performance against fresh tissue. We apply it to human lymph node samples archived for 8–12 years and to lung cancer FFPE tissues, revealing distinct regulatory trajectories between tumor center and invasive edge. Analysis of archived follicular lymphoma and transformed diffuse large B-cell lymphoma samples identifies relapse- and transformation-associated epigenetic dynamics. scFFPE-ATAC enables retrospective, spatial, and mechanistic epigenetic studies in long-term archived specimens. The study introduces scFFPE-ATAC, a high-throughput single-cell chromatin accessibility profiling method enabling profiling from clinically archived formalin-fixed paraffin-embedded (FFPE) tissues, unlocking retrospective epigenetic insights across cancer and human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42464048e0ef78f0787ee6a93b5eb7756d4e366" target='_blank'>
              scFFPE-ATAC enables high-throughput single cell chromatin accessibility profiling in formalin-fixed paraffin-embedded samples
              </a>
            </td>
          <td>
            Ram Prakash Yadav, Pengwei Xing, Miao Zhao, P. Hollander, C. Strell, Minglu Xie, Maede Salehi, Emma Torell, Tobias Sjöblom, G. Enblad, Rose-Marie Amini, F. Swartling, I. Glimelius, P. Micke, Mats Hellström, Xingqi Chen
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59ad194fb04ebf384288815f79948f2c11cf9f" target='_blank'>
              Humanized glioblastoma patient-derived orthotopic xenografts recreate a locally immunosuppressed human immune ecosystem amenable to immunotherapeutic modulation
              </a>
            </td>
          <td>
            P. M. Moreno-Sanchez, A. Oudin, Batuhan Kısakol, Heiko Dussmann, E. Klein, V. Baus, Camille Rolin, Carole Seguin-Devaux, Mahsa Rezaeipour, Aurélie Poli, A. Michelucci, J. Paggetti, E. Moussay, Jochen H. M. Prehn, Simone P. Niclou, A. Gołȩbiewska
          </td>
          <td>2025-11-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Interactions among tumor, immune, and vascular niches play major roles in glioblastoma (GBM) malignancy and treatment responses. The composition and heterogeneity of extracellular core matrix proteins (CMPs) that mediate such interactions are not well understood. Here, we present an analysis of the clinical relevance of CMP expression in GBM at bulk, single-cell, and spatial anatomical resolution. We show that CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, pro-tumor immune infiltration, and immune checkpoint expression. Matrisome expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells. Finally, we identify a 17-gene CMP signature, termed Matrisome 17 (M17), which is a stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, may predict responses to PD1 blockade. Patient stratification based on matrisome profiles can contribute to the selection and optimization of treatment strategies. A comprehensive genomic analysis of glioblastoma tumors reveals matrisome enrichment associated with mesenchymal phenotypes and pro-tumor immune signatures, a matrisome-based classification of GBM patients, and a prognostic 17-gene matrisome signature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984fecbf9126f6a0596ab278d28e0d91cbca56fc" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Zeynep Dereli, Zheng Yin, Elisabeth Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach‐Carson, Robert C. Rostomily, Anil Korkut
          </td>
          <td>2026-01-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6aac86da3cbb3bd85a8ebe57c0ff220d334a71" target='_blank'>
              Circulating immune profiling reveals impaired monocyte states and trajectories driving immunosuppression in glioblastoma
              </a>
            </td>
          <td>
            A. Scafidi, T. Kaoma, Claudia Cerella, Eleonora Campus, K. Grzyb, Bakhtiyor Nosirov, Frida Lind-Holm Mogensen, E. Klein, Raul Da Costa, A. Skupin, Frank Hertel, G. Berchem, Michel Mittelbronn, A. Cosma, Beatrice Melin, Simone P. Niclou, Petr V Nazarov, A. Gołȩbiewska, Aurélie Poli, A. Michelucci
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Background Polyploid Giant Cancer Cells (PGCCs) are a malformed subpopulation of tumor. They play a crucial role in metastasis, recurrence, and therapy resistance. However, the inconsistent model systems and a lack of standardization have hindered mechanistic understanding and clinical translation. This review highlights the pluralistic research for clinical application by methodically analyzing various model systems used in PGCC research to fill the gap in the literature. Methods As of November 2025, scholarly literature gathered from Google Scholar, PubMed, and ScienceDirect focused on examining the development, characteristics, and functional involvement of PGCCs in cancer. Results In vitro approaches, although limited in their physiological relevance, enable detailed mechanistic studies and facilitate the screening of drugs. Ex vivo tumor explants and organoids preserve patient-specific traits with translational potential, while in vivo models, such as Drosophila and mouse xenografts, provide insight into PGCC function in complex tissue environments. By mapping model capabilities against PGCC research priorities, we demonstrate that no single system comprehensively recapitulates PGCC biology, necessitating integrated, multi-model experimental strategies that we outline in this study. More specifically, integrating patient-derived organoids with lineage-traced xenografts and single-cell omics enables continuous tracking of PGCC development and functional diversity, facilitating mechanistic studies of metastasis, drug resistance, and identification of clinical biomarkers for patient stratification. Conclusion Considering the current lack of PGCC-targeted therapies, the convergence of model modification and the development of single-cell and imaging capabilities indicates significant progress toward therapeutically relevant findings. The ongoing development of these models is thus crucial for translating PGCC biology into predictive diagnoses and effective treatment methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dddf7613800266c8c4bd727d41f553d6bd1312" target='_blank'>
              Modelling the monstrosities: experimental and computational systems for studying polyploid giant cancer cells
              </a>
            </td>
          <td>
            Lakshmi Vineela Nalla, S. N. Gajula
          </td>
          <td>2025-12-10</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pleomorphic rhabdomyosarcoma is a rare soft-tissue tumor that occupies an uncertain middle ground between rhabdomyosarcoma and undifferentiated pleomorphic sarcoma. With its relative rarity, aggressiveness, and lack of detailed characterization, it presents a challenging task for therapeutic treatment. In this case study, we use single-cell transcriptomics to investigate the heterogeneous landscape of pRMS and the tumor microenvironment. We demonstrate that the tumor populations in pRMS have a clear division into myogenic and non-myogenic clusters, with the non-myogenic clusters having more numerous communication links with the immune populations. All pRMS tumor clusters use the MIF-CD74 pathway to suppress the immune response, while APP, PTN, and CXCL12 signaling are employed predominantly by the non-myogenic tumor clusters. The cytotoxic T cells in pRMS bear markers of exhaustion (LAG3, HAVCR2, EOMES), and the macrophages express myeloid checkpoint-related genes (SIGLEC1, SIRPA, CSF1R, HAVCR2). This transcriptomic data suggests that targeting MIF and APP signaling in pRMS may have therapeutic potential; however, studies on multiple-patient cohorts, protein verification, and in vitro and in vivo validation are still needed for clinical actionability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247589e7166c26c19a2c0ff799f9b8b5c5af60e4" target='_blank'>
              Single-Cell Sequencing Reveals Novel Tumor Populations and Their Interplay with the Immune Microenvironment in a Pleomorphic Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Elena E. Kopantseva, A. Ikonnikov, Maxim E. Menyailo, Timur I. Fetisov, A. Korobeynikova, K. Kirsanov, A. Tararykova, B. Bokhyan, Nikolay A. Kozlov, M. Yakubovskaya, E. Denisov
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful technique for understanding cellular diversity, but processing large patient cohorts to identify phenotype-associated cell populations remains challenging. Here, scPER (Estimating cell Proportions using single-cell RNA-seq Reference), a rigorous approach combining adversarial autoencoder and extreme gradient boosting to estimate tumor microenvironment cell compositions and identify phenotype-associated subclusters for bulk RNA-seq samples. Integrating scRNA-seq datasets from diverse studies, scPER constructed comprehensive reference panels and disentangled confounders from true signals. scPER achieved superior accuracy in cellular proportion estimation compared to CIBERSORTx, BayesPrism, Scaden, MuSiC, SCDC, DeSide and ReCIDE. It showed high accuracy in predicting metastatic melanoma immunotherapy response and identified a critical T cell subcluster expressing FCRL3 and SLAMF7. In metastatic urothelial cancer, scPER predicted TGFβ-mediated inhibition of CD4 naïve T cells to diminish PD-L1 checkpoint blockade efficacy. scPER enables robust integration of scRNA-seq datasets to estimate cellular proportions across tumors and identify clinically relevant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390e37b5363dcdab22134564264151d0e0926d0" target='_blank'>
              scPER: A Rigorous Computational Approach to Determine Cellular Subtypes in Tumors Aligned With Cancer Phenotypes From Total RNA Sequencing.
              </a>
            </td>
          <td>
            Bingrui Li, Xiaobo Zhou, R. Kalluri
          </td>
          <td>2025-11-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Immune-mediated glomerulonephritis (IMGN) is a major cause of kidney failure worldwide, yet the precise roles of T cells in its pathogenesis remain poorly understood. However, existing studies lack a comprehensive understanding of the characteristics and functional roles of IMGN T cells in the human context. Addressing this gap is crucial for advancing targeted therapies. By integrating single-cell RNA sequencing (sc-RNA-seq) data from three primary IMGN types—IgA nephropathy (IgAN), lupus nephritis (LN), and membranous nephropathy (MN)—we identified T cell subtype alterations at single-cell resolution. Utilizing advanced sc-RNA-seq computational pipelines, we constructed gene co-expression networks (GCNs), inferred T-cell differentiation trajectories, and assessed metabolic and intercellular signaling features. IMGN kidneys presented expanded T-cell compartments, with significant enrichment of cytotoxic natural killer T (NKT) cells and GZMK⁺ effector memory T (GZMK⁺ Tem) cells. Notably, LTB⁺ memory T cells (LTB⁺ Tm) were selectively elevated in IgAN and LN patients. A coexpression module centered on RGS1 was significantly correlated with 24-h proteinuria (p < 0.001). Metabolic profiling revealed subtype-specific disruptions in the glutathione (GSH) and 3-phospho-D-glyceroyl phosphate (3PD) pathways. Interaction analysis highlighted endothelial cells, mesangial cells, and fibroblasts as key mediators of pathogenic T-cell activation via defined ligand‒receptor pairs. This study provides the first comprehensive single-cell atlas of human IMGN T cells, revealing disease-specific T-cell states, metabolic signatures, and activation mechanisms. Our findings offer new insights into human renal immunopathology and identify promising therapeutic targets for IMGN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355e0f6ab7752fce8d41720415bcb57b5ac71157" target='_blank'>
              Single-cell transcriptomic analysis reveals T cell heterogeneity and metabolic reprogramming in human immune-mediated glomerulonephritis
              </a>
            </td>
          <td>
            Jianbo Qing, Yiting Zhao, W. Cheungpasitporn, Jing Miao, C. Thongprayoon, Henglan Wu, Junnan Wu
          </td>
          <td>2025-11-16</td>
          <td>Autoimmunity</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) is a type of lymphoma that affects the skin and has various subtypes with different features. It can be either slow-growing or aggressive and currently has limited treatment options, causing a significant impact on patients’ quality of life. However, more research is needed to fully understand the underlying causes of the disease. Single-cell sequencing technology shows promise in comprehending the complex genome and microenvironment of CTCL. By analyzing the molecular status of each tumor cell, this approach can provide insights into tumor heterogeneity, microenvironment composition, and cell state transitions, tailoring treatment strategies to individual patients. In this review, we highlight recent research progress in CTCL via single-cell sequencing, focusing on gene alterations and related biomarkers, disease origin and progression, and the tumor microenvironment. We aim to provide examples of single-cell studies that can help inform new drug discovery and contribute to improved treatment strategies for patients with CTCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dcc7ae8a9d6d7170ba10b8b26cd4ca7c7a1a2cc" target='_blank'>
              Single-cell sequencing reveals the complexity of cutaneous T-cell lymphoma
              </a>
            </td>
          <td>
            Jun Du, Hao-Ran Yuan, Zi-Han Lin, Yi-En Pan, Shu-Qi Zhang, Zi-Jin Ye, Jun-Ting Lyu
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Low-grade serous ovarian carcinoma (LGSOC) is a rare, treatment-resistant subtype of ovarian cancer characterized by peritoneal spread. Pressurized Intraperitoneal Aerosol Chemotherapy (PIPAC) delivers chemotherapy directly into the abdomen during serial laparoscopic surgeries, enabling repeated sampling of tumors for longitudinal analysis. Using single-nuclei RNA sequencing on 15 tumor samples across PIPAC cycles in two patients (a responder and a non-responder), we tracked molecular and cellular changes over time. Non-responder tumors showed upregulation of proliferation pathways (E2F, MYC, G2/M), KRAS signaling, epithelial-mesenchymal transition, and unfolded protein response, correlating with resistance. Responder tumors exhibited downregulation of proliferation and stress pathways but activated alternative survival mechanisms (PI3K, Wnt/β-catenin, Notch). Archetype analysis revealed dynamic shifts in metabolic and immune-related subpopulations in the responder tumors, while immunoprofiling showed greater immunosuppression in non-responder tumors. Despite the small cohort, these exploratory observations highlight tumor adaptation and underscore the need for multi-targeted therapies addressing proliferation, stress, survival, and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03dd1cf18442a46d00407d6ee891f98b166026d8" target='_blank'>
              Divergent oncogenic signaling and immune microenvironment changes in low-grade serous ovarian cancer patients undergoing intraperitoneal chemotherapy
              </a>
            </td>
          <td>
            Isaac Bishara, Patrick A. Cosgrove, Sumana Majumdar, Colt A. Egelston, Oscar Colunga Flores, Brad Nakamura, Nora H. Ruel, Paul Frankel, S. Yost, Sue Chang, Alexander Jung, Mihaela Cristea, Mustafa Raoof, Andrea H. Bild, Aritro Nath, T. Dellinger
          </td>
          <td>2025-12-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Single-cell sequencing (SCS) techniques have the potential to offer precise insights into cellular diversity by revealing unique genetic and transcriptomic profiles at the single-cell level. This advanced technology has been used extensively in research, but it has still not translated to clinical diagnostics, despite its potential. SCS provides more accurate granular information about heterogeneous cell populations and the creation of personalized treatment strategies. However, the integration of SCS into clinical practice is challenging. This review discusses the potential of SCS technologies in improving clinical molecular diagnostics in various clinical areas including oncology, genetics and rare diseases, infectious diseases, and autoimmune disorders and inflammation. We review recent advances, current uses, integration challenges, and their contribution to the development of these fields. SCS provides significant opportunities in oncology to analyze tumor heterogeneity and develop personalized treatments. In autoimmune and rare diseases, SCS has helped to define novel biomarkers and understand complex immune pathways. SCS has also been used to understand pathogen diversity and host-pathogen interactions in the context of infectious diseases, leading to targeted therapeutic approaches. Despite this progress, there remain challenges in data analysis, standardization, and routine clinical application. The future of SCS in clinical molecular diagnostics is promising. Further technological and research developments in SCS are expected to increase the precision and personalization of medical diagnostics and treatment. To overcome current limitations, interdisciplinary cooperation and innovative approaches to data analysis are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f80640c729e34d2cf6d8aaba32d154d9a103af" target='_blank'>
              Single-cell sequencing in molecular diagnostics: Transformative yet untapped potential
              </a>
            </td>
          <td>
            D. Elshiekh, Nour Farchoukh, Reema El Hassan, Sara Al-Marzooqi, W. Aldosari, Haissam Abou-Saleh, Amal Al-Haidose, H. Zayed, Atiyeh M. Abdallah
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumor microenvironment (TME) constitutes a highly complex and dynamic ecosystem wherein diverse leukocyte subsets engage in intricate cross-talk that ultimately dictates the balance between tumor progression and suppression. This review provides a comprehensive analysis of the dualistic functions of key immune cell populations-including natural killer (NK) cells, effector T cells, regulatory T cells (Tregs), myeloid-derived suppressor cells (MDSCs), and tumor-associated macrophages (TAMs)-within this milieu. Particular emphasis is placed on the Nigerian context, where unique genetic backgrounds, such as the high prevalence of sickle cell trait, and environmental factors, including endemic infections, significantly shape immune phenotypes and cancer pathogenesis. We explore how cutting-edge single-cell technologies are revolutionizing our understanding of the spatial architecture, functional plasticity, and cellular heterogeneity of these leukocytes in patient cohorts, revealing mechanisms of both immune surveillance and tumor-promoting immunosuppression. The paradoxical roles of these cells elucidate the variable clinical outcomes observed with immunotherapies and underscore the critical need for precision medicine strategies that are tailored to individual immune landscapes. Finally, we discuss emerging therapeutic modalities aimed at selectively depleting pro-tumor subsets like Tregs or MDSCs while activating anti-tumor effectors such as NK cells, which hold substantial promise for reshaping the TME and improving oncology outcomes both in Nigeria and across the globe.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d4138591bf3dda987df758be78f378fdcee1f4" target='_blank'>
              Leukocyte Subset Dynamics in the Tumor Microenvironment: Dual Roles, Single-Cell Insights, and Implications for Cancer Immunotherapy in Nigeria
              </a>
            </td>
          <td>
            Okluchi Hartumu
          </td>
          <td>2025-11-25</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Aberrant tumor glycosylation can alter immune recognition; however, the specific influence of glycan-lectin interactions on tumor progression remains poorly understood. Here, we identify the C-type lectin receptor CD301b (encoded by Mgl2) as a regulator of immune activity within the breast tumor microenvironment (TME). Using a murine triple-negative breast cancer model, we demonstrate that tumors expressing the Tn glycoantigen grow more rapidly, and this growth is facilitated by CD301b+ immune cells. Depletion or genetic loss of CD301b markedly suppressed tumor growth, indicating that CD301b promotes tumor progression through myeloid-tumor interactions. Phenotypic analyses revealed that CD301b+ cells within tumors are type 2 conventional dendritic cells (cDC2s), a subset known to influence immune polarization. Single-cell RNA sequencing of human breast cancers showed that the human ortholog CLEC10A is expressed in cDC2-like dendritic cells and select macrophage subsets, suggesting a conserved role for CD301+ myeloid populations. Transcriptomic profiling of tumors developed in Mgl2-deficient mice revealed a shift toward an inflammatory, immune-activated state consistent with enhanced antitumor immunity. Together, these findings establish a link between tumor glycosylation and lectin signaling of myeloid cells, highlighting CD301b as a potential target for reprogramming the tumor immune microenvironment in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b567312697ac0756ae3899ebeedb84ca70dd0" target='_blank'>
              CD301b lectin expression in the breast tumor microenvironment augments tumor growth
              </a>
            </td>
          <td>
            A. Ozdilek, A. Paschall, Zahra Nawaz, Afaq M. M. Niyas, F. Avci
          </td>
          <td>2025-12-03</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Brain metastasis is a major contributor to melanoma-related mortality, arising from both genomic alterations and dynamic adaptations to the brain microenvironment. Tumor cells must cross the blood–brain barrier and exploit brain-specific signaling and metabolic landscapes to establish metastases. Brain metastases display molecular and phenotypic programs distinct from other organs, reflecting pre-existing and acquired mutations during progression. We analyzed primary melanoma, metastatic melanoma, and metastatic brain tumors genomic datasets to identify mutated proteins and signaling pathways that are common to or distinct among these tumor types. Our comprehensive computational analyses uncovered key pathways and proteins associated with brain metastasis. Brain metastases exhibit unique mutational landscapes and convergent alterations in neurotrophin, MAPK, and PI3K/AKT signaling compared with primary and metastatic melanoma. Proteins especially frequent in brain metastases pathways support survival, proliferation, and adaptation in the brain microenvironment. Pharmacology-wise, rational strategies such as combining c-KIT and VEGFR inhibitors can suppress converging oncogenic pathways and mitigate resistance in advanced melanoma. Co-targeting neurotrophin receptors with MAPK pathway members, particularly in combination with BRAF inhibitors, appears a promising approach for treating melanoma brain metastases. Our findings underscore the molecular heterogeneity of melanoma brain metastases, shaped by interconnected signaling networks, and reveal therapeutic targets that may diverge from those in other metastatic sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d4ecead949fc8a270a159656433a0bf5e5ad895" target='_blank'>
              Cellular Mechanisms Underlying Melanoma Brain Metastasis
              </a>
            </td>
          <td>
            B. R. Yavuz, Hyunbum Jang, Ruth Nussinov
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a leading cause of cancer-related mortality worldwide. Despite neoadjuvant concurrent chemoradiotherapy (nCCRT), approximately half of patients exhibit incomplete pathological response with persistent lymph node (LN) infiltration. The tumor microenvironment (TME), particularly cancer-associated fibroblasts (CAFs) and natural killer (NK) cells, plays crucial roles in treatment outcomes, yet specific cellular signatures associated with LN infiltration status remain poorly characterized. We performed single-cell RNA sequencing analysis on tumor and adjacent normal tissues from 10 ESCC patients who underwent nCCRT followed by surgical resection. Patients were stratified into presence (n = 5) and absence (n = 5) groups based on histological draining LN infiltration status according to ypTNM staging. Cell-type identification, subclustering analysis, differential expression analysis, pathway enrichment, and pseudotime trajectory analysis were performed to characterize infiltration-associated signatures. Analysis of 47,907 cells revealed nine cell populations with significant compositional differences between group. Patients with LN infiltration were characterized by significant enrichment of CLEC2BhimyCAF subtype (10.64% vs. 0.58%), highly expressing NFKB1 and associated with immunosuppressive pathways. Conversely, patients without infiltration showed markedly increased abundance of CD56dim NK subset (19.14% vs. 1.73%), linked to enhanced cytotoxic function. Pseudotime trajectory analysis revealed that cells from different groups occupied opposite ends of trajectories. Treatment-associated cellular signatures extended beyond tumor regions to adjacent normal tissues within the radiation field, indicating regional tissue reprogramming associated with therapeutic exposure. This single-cell analysis reveals that LN infiltration status in ESCC is characterized by distinct TME signatures involving CLEC2BhimyCAF in patients with infiltration and F11RhiCD56dim NK in patients without infiltration. These findings provide new insights into cellular mechanisms underlying different pathological outcomes following nCCRT and offer potential biomarkers for patient stratification and therapeutic targets in ESCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bfb39d676c0c53e60dd4a4ce18c04b4cb25550" target='_blank'>
              Single-cell transcriptomic analysis reveals distinct cancer-associated fibroblast signatures in treatment-refractory esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Sheng-Min Lo, Ting-Shuan Wu, Tzu-Hung Hsiao, Chih-Hung Lin, Li-Wen Lee, Chung-Ping Hsu, Han‐Ni Chuang
          </td>
          <td>2025-12-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Thymic epithelial tumors (TETs) exhibit marked heterogeneity that challenges clinical management and prognostic evaluation. By consensus clustering of immune-related gene expression from public datasets and an in-house validation cohort, we identified two immune subtypes: a lymphocyte-rich subtype (LRS, n = 86) characterized by strong T-cell activity and favorable prognosis, and a myeloid/stromal-rich subtype (MSRS, n = 33) defined by immunosuppressive features, stemness, oncogenic alterations, and poorer outcomes. An APC gene-based classification panel demonstrated robust performance in distinguishing these subtypes, as confirmed by ROC curve analysis and validated in the Ruijin cohort using FFPE-RNA sequencing and multiplex immunofluorescence. Single-cell transcriptomics further delineated subtype-specific tumor microenvironment features and revealed that MSRS tumors suppressed CD8+ T-cell function through MIF-mediated mechanisms. Collectively, these findings establish a clinically relevant immune classification of TETs that integrates molecular, immunological, and clinical features, providing insights into tumor heterogeneity and supporting the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f6f13a57b79f0fdbdce54ed88ac9cc1b39fa61" target='_blank'>
              Immune landscape-driven subtyping reveals distinct microenvironment and prognostic profiles in thymic epithelial tumors
              </a>
            </td>
          <td>
            Yajie Zhang, Tong Lu, Yeke Huang, Xipeng Wang, Xiaoyan Chen, Wei Guo, Youqiong Ye, Hecheng Li
          </td>
          <td>2025-12-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lymph node metastasis is a common metastatic route of gastric cancer. However, the heterogeneity of tumor cells and tumor microenvironment between primary gastric cancer and metastatic lymph nodes, as well as the driving mechanisms of metastasis, remain not fully understood. In this study, single-cell RNA sequencing and spatial transcriptomics analysis were performed on 4 gastric cancer patients (four primary tumors and two paired metastatic lymph nodes). Additionally, the TCGA database was used as an external validation dataset to validate the unfavorable clinical outcomes of necroptosis genes and the correlation between MDK-NCL and immune infiltration. Our single-cell analysis revealed significant heterogeneity in the tumor microenvironment between primary and metastatic lymph nodes of gastric cancer. Pseudotime analysis indicated that gastric cancer cells may follow two potential differentiation trajectories during lymph node metastasis. One type of cells possess the ability to migrate to lymph nodes, while the other type remain in the primary site during tumor progression. We observed that the expression of necroptosis-related gene CHMP3 was significantly associated with lymph node metastasis. Immunofluorescence further suggested upregulated CHMP3 protein expression in metastatic lymph node. Furthermore, we found that MDK-NCL interaction was active in metastatic lymph nodes. External validation using the TCGA cohort indicated that high MDK-NCL expression correlated with an immunosuppressive microenvironment. Finally, through integrated single-cell and spatial analysis, we observed that gastric cancer cells may remodel the tumor microenvironment via the MDK-NCL signaling pathway. In summary, our study revealed the dynamics of tumor cells in lymph node metastasis in GC and identified a subtype of GC cells with potential metastatic capability. Our data suggest that necroptosis and the MDK-NCL signaling pathway may be involved in facilitating lymph node metastasis, providing new insights into the mechanisms of GC progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a384ade012316bb3834d72822d6e02871f918f06" target='_blank'>
              Single-cell and spatial dissection of necroptosis spatiotemporal evolution driving lymph node metastasis in gastric cancer
              </a>
            </td>
          <td>
            Yuhua Hu, Feng-lin Shen, Honghong Zhang, Zhuowen Long, Xiaojun Zhao, Jiale Wen, Yijian Sheng, Junxing Huang, Yan Chen, Qing Guo
          </td>
          <td>2025-11-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f4df7fa2e4800fd55af9dfd1c2bbdecb131c8f" target='_blank'>
              A guide to transcriptomic deconvolution in cancer.
              </a>
            </td>
          <td>
            Yaoyi Dai, Shuai Guo, Yidan Pan, C. Castignani, Matthew D. Montierth, P. van Loo, Wenyi Wang
          </td>
          <td>2025-12-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Cytotoxic CD8 T lymphocytes (CTLs) and natural killer (NK) cells share the common objective of controlling infections and detecting and removing tumor cells, albeit through distinct target recognition mechanisms. Although CTLs belong to the adaptive immune system and NK cells are innate lymphoid cells, they frequently exhibit considerable overlap in their molecular phenotypes. This overlap, as with many others in cell biology, poses challenges for distinguishing cells in the context of single cell transcriptomics. Building on a previous ROC-driven combinatorial approach, we developed a new computational framework for single-cell RNA-seq with the combiroc R package, and in this study we showed that it can identify non-canonical marker combinations for NK cells in Peripheral Blood Mononuclear Cell datasets. These combinatorial markers were in line with the Human Protein Atlas and we validated them through cytometry staining and functional assays. Markers selected with combiroc exhibit exceptional discriminatory power for identifying NK cells, both in blood and solid tumoral tissues. Besides this finding, we showed that our approach vastly optimizes standard differential expression signatures: it reduces dimensionality while improving interpretability and transferability to diagnostic applications, offering a practical solution for refining immune cell identities in complex transcriptomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99789cc7ade66f55d5c28c2b3e98ea1c0506fb12" target='_blank'>
              Single cell RNAseq signatures refined with combiroc enhance identification of NK cells in blood and solid tissues
              </a>
            </td>
          <td>
            I. Ferrari, S. Mazzara, A. Gobbini, Noemi Di Marzo, Mariacristina Crosti, S. Abrignani, Renata Grifantini, M. Bombaci, Riccardo L Rossi
          </td>
          <td>2025-12-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) exhibits substantial genetic and functional heterogeneities. We generated clonal spheroids (CS) from HNSCC cell lines and patient tumor cells, uncovering distinct functional differences. Hyperproliferative spheroids (HRPS) showed increased proliferation and tumorigenic potential, whereas hypoproliferative spheroids (HOPS) demonstrated enhanced migration and resistance to cisplatin and radiation. In patient tumor-derived spheroids, spheroid size distribution analysis across various TNM stages indicated that early-stage tumors (T2N0M0) predominantly formed HRPS, while later-stage tumors (T4N2M0) exhibited a higher prevalence of HOPS. Single-cell RNA sequencing (scRNA-seq) identified six clusters in the HOPS and four in the HRPS, each defined by unique gene expression profiles and pathway enrichment. HOPS were enriched with fibroblast, mesenchymal, and keratinizing epithelial cells, indicating a metastatic phenotype. They also expressed genes associated with stemness (KRT15 and ALDH3A1). Conversely, HRPS was characterized by proliferative genes, including TOP2A, AURKA, CKS2, AREG, and ANLN, suggesting an advanced HNSCC proliferative signature. Despite being generated from only two cells/well, the various clusters in both spheroid types mirrored the diverse tumor microenvironment in HNSCC, including neoplastic, epithelial, basal, fibroblast, and mesenchymal cells. Notably, the gene expression profiles of Cluster 3 in the HOPS and Cluster 1 in the HRPS closely matched the expression patterns observed in the HNSCC-TCGA dataset. These clusters also displayed a high transcriptional correlation between patient tumors and their xenografts, reinforcing the clonal nature of HNSCC’s genetic and functional diversity of HNSCC. This study provides a deeper understanding of the genetic and functional diversity of HNSCC tumors, which can aid in the development of more precise diagnostic tools by identifying distinct tumor subtypes. Distinguishing between hyper- and hypoproliferative spheroids could improve prognostic accuracy, helping to predict tumor behavior and patient outcomes. Moreover, the identification of specific gene expression profiles associated with proliferation, metastasis, and treatment resistance may guide personalized therapeutic strategies, enabling targeted treatment based on the unique tumor characteristics of individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/942cca58b9f76ca4806f68d559d87541dfee33c1" target='_blank'>
              Uncovering functional divergence and cellular clusters with specific gene signatures in HNSCC clonal spheroids
              </a>
            </td>
          <td>
            Jyoti Pandey, Md. Zubbair Malik, Ritis K Shyanti, Palak Parashar, Praveen K. Kujur, Deepali Mishra, Dhanir Tailor, Jee Min Lee, T. Kataria, Deepak Gupta, Hitesh Verma, Anshu Yadav, S. Kateriya, Sanjay V. Malhotra, Vibha Tandon, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Lung cancer is a highly heterogeneous disease, and the tumor microenvironment (TME) characteristics are closely related to disease progression and treatment outcomes. We elucidated the cell type-specific transcriptome landscape of cancer cells and the effect of the TME on lung adenocarcinoma (LUAD). Methods Single-cell RNA sequencing of the LUAD tissues and matched adjacent normal tissues of three patients in the early-stage via histopathological and immunohistochemical confirmation. Results The results revealed the landscape of 68,579 cells in LUAD microenvironment, and highly heterogeneous AT2 cells are crucial source of lung epithelial cell carcinogenesis. Genes KRT81, SPP1, PCDH7, SLC2A1, and TET1 were significantly upregulated in tumor tissues and associated with poor prognosis and survival, providing insights for exploring lung cancer biomarkers in future studies. Trajectory analysis identified ERBB4, SEMA4A, GCNT2 and SOX4 as key factors in AT2 cells that may promote cell proliferation, migration, or epithelial-mesenchymal transition (EMT) during the progression of lung adenocarcinoma (LUAD). We elucidated the transcription factor–target gene regulatory network involving NKX2-1 and TEAD1 in malignant tumor cells derived from AT2 cells. The findings indicated that malignant AT2 cells regulate communication between epithelial and immune cells in the TME by predicted FN1-CD44 and CADM1-CADM1 ligand-receptor interactions, which ultimately suppresses the host immune response. Discussion This comprehensive single-cell analysis increases our understanding of AT2 cells molecular and dynamics of metastatic lung cancer. In summary, single-cell RNA profiling of LUAD offers valuable prognostic insights based on AT2 cell types and identifies potential biomarkers for therapeutic responses, aiding the future development of LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e15868fa220169dfea67c353cad3603481040a" target='_blank'>
              Single-cell RNA sequencing reveals heterogeneity among AT2 epithelial cells in the lung adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Zimeng Cao, Xuefeng Bai, Jing Li, Jinru Cai, Gang Cao, Yuncai Xiao, Wei Zhou
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, Hua-Chang Chen, Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, C. Wu, Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Mechanosensitive ion channel PIEZO2 translates microenvironmental forces into Ca²⁺-dependent signals that shape tumor behavior. Across cancers, PIEZO2 exhibits context-dependent roles, with evidence for both pro- and anti-tumor functions, depending on tissue type, molecular subtype, disease stage, and stromal context. This review synthesizes emerging insights into the multifaceted roles of PIEZO2 in cancer biology, summarize clinical correlations, highlight areas with concordant directionality (e.g., colorectal cancer, medulloblastoma) versus paradoxical findings (e.g., breast cancer across subtypes, gastric cancer by differentiation status), and provide a pragmatic framework for precision, context aware targeting. While PIEZO2 remains attractive for combination strategies (anti-angiogenesis, barrier modulation, dormancy escape, radio/chemo sensitization) and cancer pain interfaces, the field lacks truly PIEZO2 selective small molecule modulators. We propose feasible future priorities encompassing both pharmacologic and non-pharmacologic strategies to accelerate translational applications. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2298c7009cdc540e541e43d7dcb3fb98a9de240" target='_blank'>
              PIEZO2 in tumors: from mechanobiological switches to activity-targeted therapies
              </a>
            </td>
          <td>
            Dan-Xia Huang, Qiu-Zhi Zhou, Hong-Mei Luo, M. Nasb, Yi-Zhou Liu, Yu-Jie Yang, Hong Chen
          </td>
          <td>2025-12-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary malignant tumor of the central nervous system (CNS), characterized by marked molecular heterogeneity, rapid proliferation, extensive invasiveness, and strong resistance to conventional therapies. Despite current standard treatments–surgery, radiotherapy, and temozolomide–median patient survival remains below 15 months. Key contributors to GBM progression include activated glial cells and remodeling of the extracellular matrix. In this study, we investigated the cellular and molecular dynamics of the tumor microenvironment, focusing on astrocytic and microglial responses, and on the modulation of glial markers such as GFAP, Iba1, Connexin 43 (Cx43), and the purinergic receptor P2X4R. Acute and organotypic brain slice cultures were prepared from C57BL/6J mice, into which GL261 glioblastoma cells were injected into the cortex. Tissues were collected 3 and 7 days post-injection to analyze tumor progression. GBM growth was monitored through the proliferation marker Ki67. Histological analysis, confocal immunofluorescence, and western blotting were performed to assess marker expression and localization. GL261 cells engaged with the surrounding neuroglial network, migrating in fascicle-like patterns that extended even into the contralateral hemisphere. This dynamic infiltration was associated with pronounced peritumoral astrocyte activation (increased GFAP) and a robust microglial response, evidenced by morphological changes in Iba1-positive cells. Cx43 expression was altered by tumor invasion, suggesting disrupted intercellular communication, while P2X4R expression dynamics indicated modulation of the local immune response. Consistently, elevated Ki67 levels confirmed the highly proliferative and aggressive behavior of the tumor. Organotypic slice cultures represent a fast and reliable ex vivo platform to investigate GBM invasiveness, migration, and neuroglial interactions, revealing spatiotemporal remodeling of the tumor microenvironment during disease progression. Region-specific glial activation and time-dependent modulation of Cx43 and P2X4R highlight the complex and evolving nature of glial–tumor crosstalk. These findings may help identify potential therapeutic windows and support the development of molecularly targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7731564feaeab85e0322e27b0f7ea6db2c5e59" target='_blank'>
              UNRAVELING GLIAL REMODELING IN SPACE AND TIME DURING GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>

          </td>
          <td>2025-12-12</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is a lethal brain cancer marked by functional plasticity driven by tumor cell-intrinsic mutations and their interplay with developmental programs. To investigate how canonical glioblastoma mutations promote functional plasticity, we have developed an isogenic human neural stem cell (NSC) model of glioblastoma by sequential addition of TERT promoter, TP53, and PDGFRA point mutations. TP53 loss-of-function increases TERT expression during serial mutagenesis, but only triple mutant NSCs reliably form lethal brain tumors in vivo that recapitulate glioblastoma. Tumor cell evolution triggers stress-related metabolic changes and transitions toward a neuronal progenitor network driven by transcription factor INSM1. INSM1 is highly expressed in human glioblastoma tumors and, during cortical development, in intermediate progenitor cells, which give rise to neurons. Remarkably, INSM1 knockdown in triple mutant NSCs and primary glioblastoma cells disrupts oncogenic gene expression and function and inhibits the in vivo tumorigenicity of triple mutant NSCs, highlighting the functional importance of an intermediate progenitor cell-like cell state in glioblastoma pathogenesis. Glioblastoma (GBM) is characterized by a high degree of heterogeneity and plasticity due to interplay with neural developmental programs. Here, the authors develop a model of GBM by introducing sequential oncogenic mutations in human neural stem cells and using this, identify INSM1 as a driver of a neural progenitor gene network promoting tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee12cd01ff0dc670471af7bea5ea93e2f7413e0" target='_blank'>
              INSM1 governs a neuronal progenitor state that drives glioblastoma in a human stem cell model
              </a>
            </td>
          <td>
            Patrick A. DeSouza, Matthew Ishahak, Xuan Qu, Colin R. McCornack, Devi Annamalai, Diane D. Mao, Rajanikanth Vangipurapu, Yiwei Fu, Alexandre T. Vessoni, Ryan Cleary, Rowland H Han, Punn Augsornworawat, Timothy Woodiwiss, Darby Agovino, Braxton Sizemore, Jessica Kline, Maryam Borhani-Haghighi, Hao Chen, Sangami Pugazenthi, Hiroko Yano, Ting Wang, Luis F Z Batista, J. Millman, Albert H. Kim
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aggrephagy plays a pivotal role in ovarian cancer (OC) progression by clearing misfolded proteins. However, the functional role of aggrephagy within the tumor microenvironment (TME) remains elusive. Non-negative matrix factorization (NMF) was employed to analyze 43 aggrephagy-related genes (ARGs) on 65,820 single cells from scRNA-seq data of 12 samples (7 OC tissues and 5 normal tissues). The OC cohort from 9 public databases and TIDE-based immunotherapy scores were used to identify prognosis and immune response patterns of these TME cell clusters. Furthermore, the association between ARGs and CD8+ T cells was validated by the ID8 ovarian cancer mouse subcutaneous tumor model. Fibroblasts, T cells, macrophages and B cells identified from scRNA-seq of OC samples were subjected to biological process enrichment and classified based on ARGs expression patterns. Additionally, it revealed that ARGs may be significantly associated with clinical and biological characteristics of OC and pseudotime trajectories of major TME cell types. Bulk-seq analysis revealed that these aggrephagy-associated TME cell subsets were closely linked to the prognosis of OC patients and exhibited significant heterogeneity in predicting response to immunotherapy. Among them, CAF and CD8+ T cells showed particularly significant immune responsiveness. Notably, CellChat analysis revealed that aggrephagy-associated subtypes of TME cells engage in diverse and extensive interactions with tumor epithelial cells. Meanwhile, significant co-localization of TUBA1B with CD8+ T cells was observed in OC tissues from the mice model. This study is the first to delineate the role of aggrephagy regulators-mediated intercellular communication of the TME in regulating OC growth and antitumor immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae1ad73527bcabf41b52f28eb9547020179867e" target='_blank'>
              Single-Nucleus Multi-Omics Reveals Hypoxia-Driven Angiogenic Programs and Their Epigenetic Control in Sinonasal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, J. Noh, Jaehyun Lee, Geunho Kwon, Myeong S. Yu, , Seung-Jun Lee, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2026-01-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Understanding CD8+ tissue-resident memory T cells (TRM) spatial characteristics in hepatocellular carcinoma (HCC) is challenging, and clarifying the spatial feature changes following immunotherapy represents an urgent research gap. Methods This study employs a multi-omics approach to analyze the spatial distribution and intercellular interactions of TRM cells in HCC tissues using radiomics, single-cell sequencing, and multiplex immunofluorescence histochemistry (m-IHC). Results Our results show that the number of CD8+ TRM cells in HCC increases following immunotherapy. Furthermore, after dividing tumor tissues into the tumor core (TC), invasion margin (IM), and normal tissue (N), a increase in CD8+ TRM cells from the IM to the TC can be observed. Consistent with the results of single-cell sequencing analysis, this change in spatial characteristics may be associated with the interactions between CD8+ TRM cells and CD68+ cells. Discussion Immunotherapy can modify the spatial characteristics of CD8+ TRM cells via regulating their crosstalk with other immune cells, and the spatial distribution of CD8+ TRM cells in the HCC tumor microenvironment (TME) correlates with immune checkpoint blockade (ICB) therapeutic efficacy. Clarifying the mechanisms of action of immunotherapeutic drugs and developing a non-invasive radiomics model to predict CD8⁺ TRM cell dynamics will facilitate the clinical management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384dbc51e0f9efcd6dc85b035d8ba5b7b014c96f" target='_blank'>
              Multi-omics spatial characteristics of CD8+TRM cells in hepatocellular carcinoma and immunotherapy response prediction
              </a>
            </td>
          <td>
            Shipeng Li, Ju Ma, Xinqiang Li, Bingbing Qiao, Liancai Wang, Deyu Li, Hua Zhou, Haixu Xu, Jinzhen Cai
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e088dcd4f395d6c7df482c6c35fcef0137dc41" target='_blank'>
              An epigenetic switch in vascular phenotype augments anti-tumor immunity
              </a>
            </td>
          <td>
            Dae Joong Kim, Mitchell T McGinty, S. Anandh, C. Riedstra, Yuvraj Sethi, Melanie R. Rutkowski, A. C. Dudley
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tracking small-molecule distribution in heterogeneous cell samples at single-cell resolution remains a major analytical challenge. Here, we present a tellurophene-functionalized analogue of the proteasome inhibitor Carfilzomib (TeCar) whose distribution can be followed by mass cytometric (MC) quantification while preserving target engagement and cytotoxicity. Structural and biochemical analyses confirm that TeCar binds the proteasome in a mode comparable to the clinically approved parent compound. Using MC, we demonstrate selective TeCar accumulation in malignant over immune cells within mixed populations, with cancer cells exhibiting 15 to 30-fold higher uptake. Tellurium signal correlates with proteasomal activity, and differential labeling among immune subsets reveals functional heterogeneity not captured by transcriptomics alone. These findings establish tellurophene tagging as a minimally perturbing and broadly applicable strategy for functional distribution studies at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297f01c1c4f8d9a53fbe0c9fdab22012abff3851" target='_blank'>
              Tellurophene-Tagged Carfilzomib Enables Single-Cell Mass Cytometric Mapping of Proteasome Activity.
              </a>
            </td>
          <td>
            Nicole Potter, A. Eddenden, Aleksandra Fomina, Anuroopa Dinesh, H. Jackson, A. McGuigan, M. Groll, Mark Nitz
          </td>
          <td>2025-11-18</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24dfa7ddd758cdfb84fb3b4e9f07b2ebd47b469" target='_blank'>
              Hidden-driver inference reveals synergistic brain-penetrant therapies for medulloblastoma
              </a>
            </td>
          <td>
            Jingjing Liu, Xu Yang, Mingrui Zhu, Xinran Dong, Honglei Zhou, B. Bianski, Barbara Jonchere, Wenwei Lin, Xiang Fu, Lei Yang, Burgess B. Freeman, Abigail S. Wang, Ruilin Jiang, Taosheng Chen, G. Robinson, M. Roussel, T. E. Merchant, Amar Gajjar, Jiyang Yu
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis drives treatment failure and cancer mortality, yet preclinical studies still rely heavily on subcutaneous xenograft models that prioritize tumor growth over metastatic biology. This disconnection from clinical reality significantly contributes to the high failure rate of experimental therapies in trials. Here, current mechanistic insights is integrated into metastasis and critically assess transplantation models to inform rational model selection for metastasis research. Tumor transplantation models exhibit distinct dissemination patterns governed by implantation methodology rather than intrinsic tumor properties. Subcutaneous models, while technically accessible, predominantly assess primary tumor growth and fail to capture critical metastatic steps like intravasation, pre‐metastatic niche formation, and organotropism. Orthotopic transplantation faithfully replicates native tissue microenvironments, enabling simultaneous assessment of tumor growth and metastatic potential. Intravascular models, while inducing rapid colonization, distort natural metastatic progression by skipping early dissemination stages. Metastatic site transplantation isolates microenvironmental impacts on tumor adaptation but fails to capture de novo metastatic initiation. Ultimately, three strategies is proposed for preventing metastasis: Eradicating circulating tumor cells, blocking colonization, and stifling outgrowth. This perspective catalyzes the strategic advancement of tumor metastasis models, thereby strengthening the reliability of preclinical findings and accelerating their clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e9a5079eba6d77348e239cd02e9f7fe18d9f93" target='_blank'>
              A Critical Reassessment of Tumor Metastasis Simulation in Murine Models: Insights Into Methodological Advances and Biological Relevance
              </a>
            </td>
          <td>
            Xiaoxi Li, Weijun Ren, Ling Liu, Chuan Liu, Xiang Chen
          </td>
          <td>2025-11-17</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Although single-cell analyses have advanced our understanding of pancreatic ductal adenocarcinoma (PDAC), most studies to date have focused on primary and metastatic tumors. Here, we map cell composition, phenotypic plasticity, and microenvironmental remodeling from human normal pancreas through preneoplastic lesions to PDAC, with the preneoplastic phase recognized as a critical window for carcinogenesis. We pinpoint genes that are persistently dysregulated throughout malignant transformation and are associated with a poor prognosis. Focusing on ductal and acinar cells as the principal origins of PDAC, we delineate malignant preneoplastic cell clusters that exhibit strong carcinogenic potential. Immune profiling reveals marked expansion and functional reprogramming of macrophages during disease progression. Integrative analysis with human PDAC bulk transcriptomic cohorts identifies candidate compounds, such as Brefeldin A, with potential for intervention in preneoplastic disease. Together, our study elucidates dynamic molecular and cellular mechanisms underlying PDAC carcinogenesis and provides actionable insights for early intervention and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c71ca9f3afc515fdc9293246a8c35d1406ec06" target='_blank'>
              Elucidating Pancreatic Ductal Adenocarcinoma Carcinogenesis at Single-Cell Resolution and Identifying Subtype Specific Drug Candidates
              </a>
            </td>
          <td>
            Jing Chen, Hui Jiang, Hui Chen, Kuan Yang, Kaiyue Yang, Mingyao Sun, Na Lv, Bolin Ren, Xinyi Lin, Xia Li, Yunpeng Zhang, Congxue Hu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CD4 T follicular helper (Tfh) cells coordinate humoral immune responses within germinal centers (GC) of lymphoid tissue. Despite their critical roles in vaccination and autoimmunity, the gene expression programs that define functionally distinct human Tfh states— and the molecular pathways engaged by Tfh positioned within the GC niche—remain incompletely understood. This gap has limited translational efforts to monitor or therapeutically target specific Tfh states for clinical benefit. Here, we delineate human CD4 T cell heterogeneity in tonsils and peripheral blood using trimodal single-cell sequencing and spectral flow cytometry to define epigenomic, transcriptional, and proteomic features of distinct Tfh states. Tfh with a GC-like phenotype exhibited markedly increased chromatin accessibility and both mRNA and protein expression of G protein subunit gamma 4 (GNG4). In tonsil, single-cell spatial transcriptomics defined GNG4 expression as a distinguishing feature of activated Tfh states within spatially demarcated GC compartments, with greater specificity than conventionally GC-associated features such as BCL6, TOX2, and S1PR2. In contrast, GNG4− Tfh primarily localized to nonGC regions and exhibited a resting, Th17-polarized phenotype. Together, these data highlight GNG4 as a central feature of activated, GC-positioned Tfh cell identity in humans. One Sentence Summary GNG4 expression defines activated CD4 T follicular helper cells localized to the germinal center of human lymphoid tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e07a3ba4b7925ada4a1cba1ab2a33fb871cfc9" target='_blank'>
              Multimodal analysis defines GNG4 as a distinguishing feature of germinal center-positioned CD4 T follicular helper cells in humans
              </a>
            </td>
          <td>
            Sam Barnett Dubensky, Yutong Zhu, Molly Gallagher, K. Kumashie, Tianyu Lu, Jonathan Tedesco, Nina De Luna, Katherine Premo, Yi Qi, Suzanna Rachimi, Emylette Cruz Cabrera, Bria Fulmer, Ijeoma C. Meremikwu, Ashley Carter, Sarah E. Henrickson, N. Romberg, Amy E. Baxter, Derek A. Oldridge, Laura A. Vella
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Secreted phosphoprotein 1 (SPP1+) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1+ macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1+ macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1+ macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8+ T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1+ macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neddylation modifications in immune and tumor cells are linked to poor tumor prognosis. This study identifies prognostic genes associated with neddylation-related genes (NRGs) in colorectal cancer (CRC) using single-cell and spatial transcriptome (ST) sequencing, aiming to advance CRC treatment strategies. Datasets included TCGA-CRC (training/internal validation, 7:3 split), GSE28722 (external validation), GSE132257 (scRNA-seq), and GSE226997 (ST). Single-cell analysis annotated seven cell types, with epithelial cells identified as key. Differentially expressed genes (DEGs) from key cells [DEGs(sc)] and bulk analysis of TCGA-CRC [DEGs(bulk)] were intersected with 247 NRGs to yield candidate genes. Regression analyses screened prognostic genes for risk model construction, validated internally and externally. Pseudotime trajectory and ST mapping visualized gene expression, while molecular networks and drug predictions were generated. In scRNA-seq dataset, seven cell types were annotated, and epithelial cells were the key cells. A sum of 32 candidate genes were obtained by intersecting 5,131 DEGs(sc)(key cells), 9,089 DEGs(bulk), and 247 NRGs to produce PSMD12, PSMB2, and FBXL5 as prognostic genes. Both prognostic risk model and nomogram model were predictive of CRC. At the ST samples, PSMD12 was expressed at a low level in all sections, whereas PSMB2 and FBXL5 were expressed at a slightly higher level in the sections. In addition, a lncRNA-miRNA-mRNA network and a drug-prognostic gene network were created, getting some potential drugs like bortezomib. A novel three-gene prognostic model for CRC was developed and validated, offering therapeutic insights through molecular networks and drug predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e622ec21d134087f660327fbc7e9bb587ce7f13c" target='_blank'>
              Integration analysis of single-cell and spatial transcriptomics identifies prognostic genes associated with neddylation in colorectal cancer
              </a>
            </td>
          <td>
            Ziming Zhu, Xinyue Zhang, Song Wang, Yunsi Huang, Xuedong Han, Dongping Lai, Xin Yao, Weixuan Lan, Hui Nong, Wenbin Zeng, Yanhua Mo, Ri'an Xu, Tao Zhang
          </td>
          <td>2025-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Single-cell technologies have revolutionized the understanding of cellular heterogeneity, revealing distinct cell populations and functional states with molecular precision. Traditional approaches, however, are constrained by cellular dimension and throughput scale, in addition to its inability preserve spatial information. Recent innovations now allow high throughput, large-sized cell, and multi-modal profiling of patient samples, overcoming these limitations. In this perspective, three recently developed single-cell technologies is described, highlighting how their enhanced throughput, resolution, and sensitivity provide deeper insights into cellular identity and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/776bc27f6e80cfee3889e04fbb467e4d400dce04" target='_blank'>
              Single-Cell Profiling: Any Scale, Any Size, All at Once.
              </a>
            </td>
          <td>
            Denise Goh, Felicia Wee, R. E. Fincham, Ruisi Li, J. Yeong
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba29f656998508469f49c53f9437fe122dadd98" target='_blank'>
              DYRK1A regulates cancer cell and cancer-associated fibroblast secretomes to foster an immunosuppressive microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Silvia Pascual-Sabater, Sushmita Varhadi, Chiara Di Vona, Marta Celma, Cristina Fillat, S. de la Luna
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa68ef0a898a9de3aed5f44e700f118ea58e14a" target='_blank'>
              A cellular epigenetic classification system for glioblastoma.
              </a>
            </td>
          <td>
            Dana Silverbush, Liv Jürgensen, Nelson F. Freeburg, Channing Pooley, Fabio Boniolo, Federico Gaiti, M. Suvà, V. Hovestadt
          </td>
          <td>2026-01-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Cancer research increasingly relies on large-scale, multimodal datasets that capture the complexity of tumor ecosystems across diverse patients, cancer types, and disease stages. The Human Tumor Atlas Network (HTAN) generates such data, including single-cell transcriptomics, proteomics, and multiplexed imaging. However, the volume and heterogeneity of the data present challenges for researchers seeking to integrate, explore, and analyze these datasets at scale. To this end, HTAN developed a cloud-based infrastructure that transforms clinical and assay metadata into aggregate Google BigQuery tables, hosted through the Institute for Systems Biology Cancer Gateway in the Cloud (ISB-CGC). This infrastructure introduces two key innovations: (1) a provenance-based HTAN ID table that simplifies cohort construction and cross-assay integration, and (2) the novel adaptation of BigQuery’s geospatial functions for use in spatial biology, enabling neighborhood and correlation analysis of tumor microenvironments. We demonstrate these capabilities through R and Python notebooks that highlight use cases such as identifying precancer and organ-specific sample cohorts, integrating multimodal datasets, and analyzing single-cell and spatial data. By lowering technical and computational barriers, this infrastructure provides a cost-effective and intuitive entry point for researchers, highlighting the potential of cloud-based platforms to accelerate cancer discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8baca06d5c64e37cff355e3dd62954386f198bd0" target='_blank'>
              Analysis of clinical, single cell, and spatial data from the Human Tumor Atlas Network (HTAN) with massively distributed cloud-based queries
              </a>
            </td>
          <td>
            David L. Gibbs, Dar’ya Pozhidayeva, Yamina Katariya, Boris Aguilar, Kristen Anton, Clarisse Lau, William Longabaugh, Ino de Bruijn, Alex Lash, M. Nikolov, Jennifer Altreuter, Ashley Clayton, Aditi Gopalan, Adam Taylor, N. Schultz, E. Cerami, V. Thorsson
          </td>
          <td>2025-11-17</td>
          <td>Research Square</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract Background Early tumour vascular invasion contributes to cancer progression. Tip cells, a subset of tumour endothelial cells, significantly decline after anti‐angiogenic therapy. However, their behaviour and the roles of their signature genes during early invasion are incompletely understood. Methods This study employed single‐cell transcriptomic analysis and 10x Genomics Visium spatial transcriptomics on fresh lung tissues from patients with pulmonary nodules and from KrasG12D (K) and KrasG12DTgfbr2−/− (KT) mice. The role of plasma vesicle‐associated protein (PLVAP), a tip cell marker, was further examined using survival databases, immunofluorescence, in vitro co‐culture, cell migration, invasion assays and endothelial tube formation. Results Tip cell proportions were elevated in early‐stage lung adenocarcinoma (LUAD) tissues and KT mice, with evidence suggesting they arise from capillaries type I. PLVAP expression was enriched in tumour endothelial cells, induced by TGFβ1, and negatively correlated with patient prognosis. Functionally, PLVAP promoted endothelial cell invasion, migration and angiogenesis, and regulated tumour cell invasiveness. Intercellular analysis revealed that some tip cells also expressed TGFβ1, which may act on adjacent tumour cells to enhance invasion during early tumour development. Conclusion Tip cells increased during early LUAD progression and likely evolved from capillaries type I. Their marker PLVAP was associated with poor prognosis and pro‐invasive endothelial behaviour. Tumour‐secreted TGFβ1 upregulated PLVAP in endothelial cells, promoting angiogenesis and tumour invasion. Additionally, tip‐cell‐derived TGFβ1 may further stimulate tumour aggressiveness, highlighting a reciprocal interaction that contributes to early tumour progression. Key points Tip cells expand during early LUAD progression and likely originate from capillary type I endothelial cells. Tumour‐derived TGFβ1 induces PLVAP expression in endothelial cells, linking tumour signals to vascular activation. PLVAP enhances endothelial cell migration, invasion and angiogenic capacity. Endothelial PLVAP promotes tumour cell invasiveness, revealing a reciprocal endothelial‐tumour interaction that drives early tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79021428fe4068225ddb196bfe79a293a09eade5" target='_blank'>
              PLVAP mediates the regulation of the tumour microenvironment in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Linshan Xie, Mengting Sun, Yujie Zheng, Zezhong Yu, Hui Kong, Jinjie Yu, Shaohua Lu, Yong Zhang, Jie Hu, Hongyi Xin, Jian Zhou, Xiangdong Wang, Charles A Powell, Fred R Hirsch, Chunxue Bai, Yuanlin Song, Jun Yin, Dawei Yang
          </td>
          <td>2025-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) have emerged as critical prognostic and immunotherapeutic indicators in cancer, with their clinical significance modulated by spatial distribution patterns and density. Here, we performed integrated single-cell and spatial transcriptomic profiling of 30 gastric cancer (GC) specimens stratified by TLS spatial localization patterns. Comparative analysis shows pronounced enrichment of CXCL13+ T lymphocytes (TLCs), CXCR5+ germinal center B lymphocytes (gc_B cells), LAMP3+CD80+ activated dendritic cells (DCs), and SELP+ACKR1+ high endothelial venule (HEV) cells within intratumoral-TLS (iTLS) rich tumors compared to peritumoral-TLS (pTLS) and desert-TLS (dTLS) tumor subtypes. Multimodal cell–cell interaction analysis and functional experiments demonstrate that HEV expressed VCAM1 and ICAM1 recruits and activates CXCL13+ TLC through the CXCL13-ACKR1 pathway, which promotes TLS formation via CXCL13-CXCR5-dependent crosstalk with B lymphocytes. We further develop a single-cell/spatial TLS signature that captures the cellular ecosystem of iTLS-containing tumor, demonstrating predictive value for immunotherapy outcomes in GC patients. Tertiary lymphoid structures (TLS) within the tumour microenvironment are important for prognostic and immunotherapeutic indications in cancer. Here the authors use a spatial and single cell transcriptomics analysis of gastric cancer (GC) samples and show transcriptomic differences grouped by types of TLS and generate a prognostic transcriptomic signature to help predict outcome in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f70b23aaaa2341fd42d68d4bef2be6eb1c78ea" target='_blank'>
              Single-cell and spatial transcriptomics implicate a prognostic function of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yanchun Wang, Guangyu Zhang, Xin Zhang, Gaoke Liu, Lihua Zhang, Lei Chen, Shuli Sang, Sheng Yao, Yang Fei, Zhichao Tian, Yu Zang, Hongwei Wang, Yiping Dong, Chunjie Liu, Xizhao Chen, Yan Liu, Min Zhang
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse gliomas remain lethal primary brain tumors. Immune-checkpoint inhibitors have not delivered durable benefit for most patients, reflecting myeloid-dominant immunosuppression and spatially organized immune exclusion. In this mini-review we summarize ligand–receptor multi-omics—single-cell RNA/CITE-seq, single-cell chromatin accessibility, and spatial proteo-transcriptomics—that resolve microglia- and monocyte-derived TAM programs and malignant state continua, and we appraise translational opportunities spanning TAM reprogramming (CSF1–CSF1R), perivascular SPP1–CD44 disruption, and innate–adaptive combinations targeting CD47–SIRPα, CD39–CD73, and PD-1/PD-L1. We also discuss challenges—including ontogeny-aware state definitions, heteromer-aware databases, chromatin gating of receivers (requiring accessible regulatory DNA for the receptor and its program), spatial registration, and limited assay standardization—that temper implementation. By integrating myeloid-informed readouts (SPP1–TAM burden, CD39–CD73 proximity, HMOX1+ IL-10 niches, serum IL-8), emerging strategies aim to restore antigen presentation, enable effector ingress, and remodel vascular–stromal interfaces. Our synthesis provides an appraisal of reproducible communication architectures in glioma and outlines pragmatic reporting standards and trial-ready pharmacodynamic endpoints for myeloid-informed precision immuno-oncology. We hope these insights will assist researchers and clinicians as they design multi-omics pipelines and interventions to convert suppressive ecosystems into responsive ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39bafbb89665949b18c8b3763ecd3648dcb8152" target='_blank'>
              Mapping TAM–tumor crosstalk in glioma via ligand–receptor multi-omics: mechanisms of immune evasion
              </a>
            </td>
          <td>
            Dong Zhang, Yiming Ma, Daxiong Feng
          </td>
          <td>2025-11-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells.
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="NSCLC accounts for approximately 85% of lung cancers, and Kirsten rat sarcoma (KRAS) is the most commonly mutated gene in NSCLC. Although management of NSCLC has advanced markedly in recent years, its heterogeneity and plasticity remain incompletely understood. Single-cell RNA sequencing (scRNA-seq) has become an essential tool for dissecting cellular heterogeneity in complex tissues such as the lung. However, generating high-quality single-cell suspensions from tumor-bearing lung tissue is challenging, particularly during tumor formation when the tissue becomes fibrotic and heterogeneous. A key requirement for successful scRNA-seq is the preparation of viable single cells with minimal processing-induced stress or damage. Here, we present a detailed step-by-step protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing mouse lung tissue in a KrasG12D; tdTomato reporter mouse model. The procedure includes tissue perfusion, controlled enzymatic digestion, gentle mechanical dissociation, red blood cell lysis, filtration, and viability assessment, and achieves >85% viable cells prior to scRNA-seq. Finally, the tdTomato+ tumor cells are efficiently isolated by FACS and can be detected as distinct clusters by scRNA-seq using this protocol. SUMMARY We present a detailed protocol for isolating high-quality single-cell suspensions from both healthy and tumor-bearing murine lung tissue. This protocol can be applied to scRNA-seq, flow cytometry analysis, and primary cell isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05709be045bd3ac955b0ab18505a25294de0610f" target='_blank'>
              Optimized protocol for processing murine tumor-bearing lung tissue for flow cytometry and single cell RNA-sequencing
              </a>
            </td>
          <td>
            Sicong Wang, Anastasiia Ivanova, Helen P. Makarenkova, Katja A. Lamia
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
The development of gemcitabine resistance significantly diminishes treatment efficacy and worsens prognosis in patients with pancreatic ductal adenocarcinoma (PDAC). A comprehensive understanding of the molecular and cellular mechanisms driving gemcitabine resistance is thus essential for optimizing treatment strategies and improving survival outcomes in PDAC.


METHODS
An integrated analysis of bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics was performed to elucidate the molecular mechanisms of gemcitabine resistance. Tumor-associated epithelial and myeloid cell subsets were identified, and their interactions were characterized using clustering, pseudotime analysis, and cell-to-cell communication networks. Findings were validated through molecular and cellular experiments in PDAC cell lines, as well as multicolor immunofluorescence staining in clinical cohort samples.


RESULTS
The findings revealed that PRRX2+ epithelial cells (EC) and SPP1+ tumor-associated macrophages (TAM) were enriched in gemcitabine-resistant samples, with robust cell-to-cell communication observed between these two populations. PRRX2+ EC cells and SPP1+ TAM cells interacted through the TGFB1/TGFBR2 axis, promoting epithelial-mesenchymal transition (EMT) and extracellular matrix (ECM) remodeling, thereby contributing to drug resistance. The transcription factor PRRX2 was identified as a central regulator of this resistance mechanism, amplifying TGF-β signaling through TGFB1 to induce EMT and activating Fibronectin 1 to remodel the ECM, thereby enhancing an immunosuppressive tumor microenvironment. Furthermore, experimental downregulation of PRRX2 in pancreatic cancer cell lines resulted in reduced tumor cell migration. High expression levels of PRRX2+EC/SPP1+TAM were associated with poor prognosis and may indicate susceptibility to Dasatinib and antibody-drug conjugates (ADCs) targeting TROP2, MUC1, and CEACAM5.


CONCLUSIONS
The results revealed the critical role of PRRX2+EC/SPP1+TAM cell communication in mediating gemcitabine resistance in PDAC. These findings provide new insights into potential therapeutic strategies, highlighting the importance of targeting the PRRX2+EC/SPP1+TAM axis and supporting the use of multidrug regimens to overcome resistance and improve patient outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/add5ea3cff49a3e2c5c93db3e7092d7e12027eec" target='_blank'>
              Single-cell and spatial transcriptomics reveal the interaction between PRRX2-driven epithelial cells and SPP1+ macrophages in mediating gemcitabine resistance in pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Jingmei Liu, Zhilong Li, Guanhua Liu, Zhihua Pei, Weilin Liu, Wenhui Yang, Feng Li, Lijuan Huo
          </td>
          <td>2025-12-26</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a key factor influencing carcinogenesis, disease progression, and therapeutic outcomes. Tumor-associated endothelial cells (TECs), integral components of the TME, form the structural and functional foundation of the vasculature, while also modulating tumor immune responses. TECs regulate immune cell activity and distribution through multiple molecular mechanisms, thereby influencing tumor immunity. Recent evidence highlights TEC-targeted interventions as a promising strategy to enhance immunotherapy efficacy. This review examines the mechanisms through which TECs drive tumorigenesis, progression, and metastasis, and explores the therapeutic potential of TEC-targeted approaches in cancer treatment. It also provides a systematic analysis of TECs’ impact on the immune microenvironment, assessing the potential of TEC-directed therapies to overcome chemoresistance and immunotherapy resistance, thus offering new perspectives for drug development targeting TECs. Finally, the review critically evaluates potential molecular targets within TECs and recent advancements in genetic research for cancer therapy, emphasizing key research priorities and pressing clinical challenges that require urgent focus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5e2322afb5c6ffa7a0363f83b269479ee8db97" target='_blank'>
              Tumor-associated endothelial cells in tumor immune escape and immunotherapy: multifaceted roles and treatment approaches
              </a>
            </td>
          <td>
            Tianyu Zheng, Xinran Yu, Caihong Yu, Wangting Xu, Zhuoyang Fan, Yongjie Zhou, Changyu Li, Juncheng Wan, Chaoqiao Jin, Xuran Jin, Wen Zhang, Zhiping Yan, Peng Luo, Bufu Tang, Xudong Qu
          </td>
          <td>2025-12-24</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) serve as a promising biomarker for tracking cancer metastasis, progression, and recurrence. Liquid biopsy techniques centered on CTC detection have demonstrated considerable potential due to their non-invasive nature and ability to provide real-time monitoring of tumor dynamics. However, conventional bulk CTC analyses fail to capture the intrinsic heterogeneity among CTC populations, obscuring crucial insights into tumor biology. Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of CTC heterogeneity, offering new opportunities for precision oncology and mechanistic studies of tumor progression. Despite these advantages, the existing methodologies for single-CTC sequencing tend to suffer from inefficiencies, including low recovery rates, labor-intensive workflows, and contamination risks associated with multiple manual handling steps. To address these limitations, we present an integrated microfluidic protocol that consolidates CTC enrichment, purification, and single-cell sequencing into a unified workflow. The method employs dynamically controlled magnetic capture within a herringbone-structured chip, where vortex mixing and cumulative immunomagnetic bead binding enable robust, high-throughput CTC isolation with minimal cell damage. Subsequent purification using a leukocyte antibody-coated microfluidic chip effectively removes the non-target cells, further enhancing CTC purity through negative selection. Finally, a high-precision single-cell sequencing chip, designed based on differential flow resistance principles, facilitates efficient single-cell capture and pairing with uniquely barcoded microbeads. This novel platform overcomes the limitations of Poisson distribution-based methods, improving CTC utilization while minimizing microbead consumption and sequencing costs. Our integrated protocol significantly enhances CTC capture efficiency, purity, and single-cell sequencing throughput, making it well-suited for clinical applications and large-scale cancer research. By enabling a more precise and scalable analysis of CTC heterogeneity, this method has the potential to refine early cancer diagnosis, treatment monitoring, and mechanistic studies of metastasis, ultimately advancing the field of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9babafe8a10523678f47d2a96e859b16fafc3f98" target='_blank'>
              Microfluidics-based High-throughput Circulating Tumor Cell Sorting and Single-cell Sequencing Technology.
              </a>
            </td>
          <td>
            Zheyu Liu, Ye Tian, Zian Zhang, Lingling Wu, Liang Dong, Wei Xue
          </td>
          <td>2025-11-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Purpose
Colorectal cancer (CRC) lung metastases exhibit high recurrence rates after resection, underscoring the need for improved therapeutic strategies. This study aimed to characterize the tumor microenvironment (TME) of CRC lung metastases and identify the factors associated with recurrence.


Materials and Methods
Fifteen CRC patients who underwent lung metastasectomy were enrolled. Multiplex immunohistochemistry (IHC), whole exome sequencing, transcriptome profiling, and single-cell RNA sequencing (scRNA-seq) were conducted on matched tumor, adjacent and distant normal lung tissues. Immune cell populations and gene expression profiles were analyzed and correlated with clinical recurrence outcomes.


Results
Exome and transcriptome analyses revealed frequent TP53, KRAS, and APC mutations. Most tumors corresponded to consensus molecular subtypes 2 and 4, characterized by immune-depleted and fibrotic features. Tumors showed downregulation of effector T and NK cell signatures. IHC revealed reduced density and increased distance of CD8+ T cells and macrophages from the epithelial cells. scRNA-seq demonstrated increased regulatory T cells and decreased NK and effector T cells in tumor. Tumor-associated macrophages (TAMs), particularly SPP1 (osteopontin)-expressing subsets, were markedly enriched in tumor and correlated with suppressed effector T cella activity. High SPP1 expression was associated with early recurrence and poor overall survival. Patients with recurrence had higher proportion of PD-1+ CD8+ T cells in adjacent normal tissues.


Conclusion
Immunosuppressive features including enrichment of SPP1+ TAMs and depletion of effector T and NK cells contribute to recurrence after CRC lung metastasectomy. Therapeutic strategies targeting both TAMs and T cells may enhance clinical outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae972bf762bae15538dd4da3de79b673b4d58b2d" target='_blank'>
              Immunosuppressive Tumor Microenvironment in Colorectal Cancer Lung Metastases: Implications for Recurrence After Metastasectomy.
              </a>
            </td>
          <td>
            Minsuk Kwon, Minkyue Shin, M. An, Y.J. Jeon, T. Hong, Jung Kyong Shin, Sung Hee Lim, Yoonah Park, Y. Cho, Seung-Tae Kim, Y. Choi, Jeeyun Lee
          </td>
          <td>2025-12-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Oropharyngeal squamous cell carcinoma (OPC) associated with high-risk human papillomavirus (HPV), particularly HPV-16, generally shows favorable outcomes yet paradoxically exhibits a high incidence of early lymphatic metastasis. The immune mechanisms underlying this phenomenon remain unclear. Methods We conducted an exploratory spatial transcriptomic analysis using the GeoMx Digital Spatial Profiler on formalin-fixed, paraffin-embedded samples from six patients with palatine tonsil-derived OPC. Tumor tissue regions (TTRs) and lymphoid follicular regions (LFRs) were compared according to HPV status and nodal involvement. Results In HPV-positive LFRs, pathways related to B-cell apoptosis appeared downregulated, suggesting prolonged B-cell survival and antigen presentation. Metastasis-negative HPV-positive cases displayed a Th2-skewed immune profile in LFRs, with increased naïve B cells, plasma cells, eosinophils, M2 macrophages, and activated mast cells. In contrast, metastasis-positive cases showed increased T cell activation in LFRs and reduced proliferation-related signaling in TTRs. Pathways involving estrogen signaling and bile acid metabolism were also associated with metastatic behavior. Conclusion These exploratory findings suggest that peritumoral Th2-biased immunity within lymphoid structures may contribute to restraining lymphatic metastasis in HPV-positive OPC. Spatial transcriptomics may provide a high-resolution framework for investigating tumor–immune interactions and generating hypotheses for future mechanistic and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb3a9ec6ff5bf8bd62696a72523c1ce32b123f2" target='_blank'>
              Exploratory spatial transcriptomic profiling of peritumoral Th2 immune polarization in HPV-positive oropharyngeal cancer
              </a>
            </td>
          <td>
            N. Wakisaka, M. Moriyama-Kita, Satoru Kondo, E. Kobayashi, Takayoshi Ueno, Y. Nakanishi, Kazuhira Endo, H. Sugimoto, T. Yoshizaki
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cervical cancer is among the prevalent gynecological malignancies affecting women worldwide. The diverse cellular populations and their intricate communication patterns within the tumor environment significantly influence cancer development and metastasis. Our research focused on elucidating the cellular diversity, intercellular signaling networks, and underlying molecular processes in cervical cancer tissues via single-cell transcriptomic approaches. Publicly available single-cell RNA sequencing (scRNA-seq) data for cervical cancer (GSE236738) were re-analysed to delineate the cellular composition and transcriptional programs of the tumour microenvironment. We applied UMAP and t-SNE algorithms for dimensionality reduction, alongside intercellular communication mapping, MIF pathway analysis, temporal trajectory modeling, and pathway enrichment studies, to comprehensively characterize the cellular architecture and molecular profiles within cervical cancer tissues. In addition, paired tumour and adjacent non-tumour cervical tissues from 20 patients were subjected to quantitative real-time PCR and enzyme-linked immunosorbent assay to confirm the transcriptional and protein-level expression of key genes identified by scRNA-seq. Analysis revealed substantial cellular and molecular diversity within cervical cancer specimens. Ten distinct cellular populations were characterized, including NK lymphocytes, epithelial components, and macrophage subsets, indicating pronounced cellular heterogeneity. Intercellular communication mapping revealed that stromal and epithelial populations are the primary coordinators of cell-to-cell signaling. MIF pathway activity peaked in cancer-associated fibroblasts, which exhibited distinct cell type distributions. Temporal analysis revealed dynamic cellular states and pinpointed critical regulatory elements, including ISG15. Pathway analysis emphasized the importance of neutrophil activity, mitochondrial respiration, and cell adhesion mechanisms in cancer progression. qPCR confirmed that ISG15 and TNFRSF18 were significantly upregulated in tumor tissues, while SDF4 was downregulated and NOC2L showed a moderate increase. ELISA results were consistent with transcript-level findings, demonstrating significant protein overexpression of ISG15 and TNFRSF18 in tumor tissues compared with controls. Our investigation comprehensively characterized the cellular diversity and molecular features of the cervical cancer microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7329b5a877ad7165acf8e0004372f128c5c7552" target='_blank'>
              Mapping cellular interactions and communication landscapes in cervical cancer via single-cell transcriptomics
              </a>
            </td>
          <td>
            Shaoli Zhuang, Xiaoxiao Deng, Furong Fu, Zhang Zhang
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune evasion and epithelial-mesenchymal transition (EMT) are critical mechanisms driving tumor progression and therapy resistance in prostate cancer. In this study, we explored the role of TDP2 in modulating the tumor microenvironment (TME) through single-cell RNA sequencing and pathway enrichment analysis. Our results revealed that epithelial cells with high TDP2 expression extensively interact with myeloid cells, macrophages, and fibroblasts, thereby shaping immune responses and facilitating tumor progression. Specifically, TDP2 overexpression suppressed M1 macrophage polarization and dendritic cell (DC) maturation, leading to reduced CD8 + T cell activation and enhanced immune evasion. Additionally, TDP2-high expression was associated with enriched signaling pathways involved in EMT, including COLLAGEN, GALECTIN, MIDKINE (MK), and ONCOSTATIN M (OSM), which promoted tumor cell migration, invasion, and immune evasion. Survival analyses further demonstrated that high TDP2 expression correlated with poor clinical outcomes in prostate cancer patients. Overall, our findings identify TDP2 as a key regulator within the TME and suggest its potential utility as both a prognostic biomarker and therapeutic target in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c33c0d20e26f20d7665d000d3fb80f5eb193340" target='_blank'>
              TDP2 drives immune evasion and metastatic progression in prostate cancer
              </a>
            </td>
          <td>
            Huan Cao, Anyin Pan, Yuetao Chen, Fei Zheng
          </td>
          <td>2026-01-02</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Programmed cell death protein-1 (PD-1) blockade has shown promising clinical efficacy in hepatocellular carcinoma (HCC), yet the underlying immunological mechanisms governing response and resistance remain unclear. This study aimed to delineate the temporal and spatial dynamics of T-cell clonotypes and their relationship with pathological response in neoadjuvant PD-1 blockade therapy in HCC. By integrating T-cell receptor (TCR) repertoire analysis with transcriptomic profiling, we sought to elucidate the immune landscape alterations associated with treatment outcomes. Methods We analyzed longitudinal, matched tumor and blood samples from our previous clinical trial (NCT05471674), encompassing 114 specimens, including pre-nivolumab tumor biopsies, post-treatment resected tumors, adjacent non-tumorous livers, and peripheral blood collected before, after, and during follow-up from 19 treatment-naïve HCC. TCR clonality, diversity, and transcriptomic signatures were assessed to characterize immune responses. Results Elevated clonality of intratumoral TCR clonotypes (ITCs) post-treatment correlated with increased pathological tumor necrosis, predominantly driven by the most abundant top 1% ITCs. These dominant clonotypes occupied a significantly larger clonal space and demonstrated increased spatial sharing across tumor, non-tumorous liver, and peripheral blood compartments in nivolumab responders (nivo-Rs). Shared clonotypes were positively associated with tumor necrosis extent and cytolytic activity, suggesting active immune engagement. Post-treatment, shared clonotypes exhibited peripheral expansion and greater intratumoral dominance in nivo-Rs, whereas tumor unique intratumoral TCR clonotypes (ITCs) remained prevalent in non-responders (nivo-NRs). These divergent patterns were linked to higher chemokine expression within the tumor microenvironment of nivo-Rs and impaired human leukocyte antigen (HLA) class II antigen presentation in nivo-NRs, indicating distinct immune landscape configurations influencing therapeutic response. Conclusions Our findings demonstrate that the peripheral infiltration and expansion of high-frequency intratumoral T-cell clonotypes are critical drivers of pathological response to neoadjuvant PD-1 blockade in HCC, highlighting potential immune biomarkers and therapeutic targets to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901935af9edd3945285033fa6917d3b60bbe154f" target='_blank'>
              Spatiotemporal dynamics of T cells in peripheral blood and tumor underlying differential responses to neoadjuvant PD-1 blockade in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yajing Zhang, Y. Tsui, J. Lee, T. Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9990df240d9cf9afd4d976bdcb6753a231a07876" target='_blank'>
              CA9+ cancer-associated fibroblasts cooperate with SPP1+ tumor-associated macrophages driving immune resistance in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Qin Ma, Jing Wang, Qian Jiang
          </td>
          <td>2026-01-09</td>
          <td>Cellular and molecular life sciences : CMLS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive tumor type with poor patient outcomes. Most patients present with metastatic disease, which generally has reduced immune infiltration compared to primary tumors. Further work to elucidate the specific cellular features of metastatic PDAC is needed to guide the development of future immunotherapy strategies. Here, we investigated the hypothesis that PDAC tumors harbor distinct immunologic and stromal features depending on their anatomical site. Multiplex immunohistochemistry (mIHC), spatial analysis, and single-cell mass cytometry (CyTOF) uncovered dominant immune and stromal cell populations in tumors derived from 27 primary and 26 liver metastases. Metastatic liver tumors from PDAC patients contained reduced T cell infiltration, fibroblast populations, and collagen accumulation than primary lesions, while CD68+ cells, often co-expressing CCR2, were more abundant. Spatial analyses revealed distinct immune cell communities in primary and metastatic PDAC, whereby CK19+ cells clustered differentially with α-SMA+, CD3+, and CD68+ cells, depending on the tumor site. When comparing tumor-associated regions, the proportion of peritumoral CK19- cells remained consistent, but their composition varied by disease site. CD8+ T cells were significantly less frequent in metastatic tumors, while both CD4+ and CD8+ T cells present in primary tumors expressed more transcription factors (TFs) associated with suppressive properties, including FoxP3 and RORγt. CyTOF revealed that T cells co-expressed multiple inhibitory checkpoint receptors, most notably LAG-3 and PD-1. This report reveals that primary and metastatic tumors from PDAC patients harbor vastly distinct immunologic and stromal features at the protein level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bea286cd97f7f8064af812daefa94a7da6b7d73" target='_blank'>
              High-Dimensional Protein Analysis Uncovers Distinct Immunological and Stromal Features in Primary and Metastatic Pancreatic Ductal Adenocarcinoma.
              </a>
            </td>
          <td>
            E. Greene, N. Horvat, D. Doxie, V. Parihar, Jayden Kim, Cameron J Herting, Erin E. Grundy, Ayana T Ruffin, Alyssa M Krasinskas, Shishir K Maithel, Juan M. Sarmiento, Mihir M. Shah, Mohammad Y Zaidi, Maria Diab, O. Alese, Kavita M. Dhodapkar, H. Kissick, B. El-Rayes, Chrystal M. Paulos, G. Lesinski
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d360e6a196e9de4b8313a49f5e3b66fe3b338c6" target='_blank'>
              Integrative single-cell and spatial transcriptomics with explainable AI reveal lethal prognostic axis in prostate cancer.
              </a>
            </td>
          <td>
            Qintao Ge, Zhenda Wang, Yangyun Wang, Tonghui Chu, Zhongyuan Wang, Wenkai Zhu, Youzhao Zhang, Yonghao Chen, Dingwei Ye, Wenhao Xu, Zhong Wang, Mierxiati Abudurexiti
          </td>
          <td>2026-01-06</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The tumor infiltrating T cells determine whether a patient would respond positively to immunotherapy. This paper has analyzed an mRNA scRNA-seq dataset of melanoma in the context of mapping heterogeneity in T-cells and the question of whether clinical features can shape cell clustering. Unsupervised clustering with k-means based on quality control followed by a description of the major T-cell groups by canonical immune markers. The identification of the differentially expressed genes that were identified was then done by a posteriori information based ranking approach. This study have created a confidence-weighted approach, which is able to merge Spearman and cosine similarity measures with marker-gene rules, to enhance cell-type annotation. Metadata analysis indicated that treatment status is an element that causes the state changes in / T cells pre- and post-checkpoint therapy. Combining graph-based Leiden clustering and k-means, this study refined the definition of the fine-grained T-cell states and proved that, despite the significant heterogeneity of T cells, the strong definition of subclass depends on the combination of several methods of clustering and the use of orthogonal knowledge of experiments. Pseudotime analysis found evidence in gradual transition between naive or early-activated states to cytotoxic or dysfunctional phenotypes that indicate intratumoral T-cell diversity captures a reflection not only of inherent developmental trajectories, but also of the effects due to therapy. Lastly, survival analysis revealed that there was a distinct relationship between clinical response and overall survival. These findings combined narrow the scope of the T-cell picture and offer understanding in the form of transcriptional programs connected with treatment outcomes. Refer to this link for supplementary materials: https://drive.google.com/drive/folders/1UYW-6EMjSUMVOwHxvuYdFSZwREqrGp89">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b245ce41e96da9548625adf2b4b3d69d756bc71f" target='_blank'>
              Single-Cell Transcriptomics Reveals T-Cell State Transitions and Their Association with Immunotherapy Response in the Melanoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Shangyu Ning
          </td>
          <td>2025-12-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Brain metastases are frequent and often lethal complications of advanced cancers. Microglia, resident immune cells of the brain, are known to exert both anti-tumor and pro-tumor functions in late-stage metastases; however, their response during the initial outgrowth of metastatic lesions is not well characterized. Understanding how heterogeneous microglial subgroups are regulated in the developing tumor microenvironment could pave the way for therapeutic strategies to eliminate metastatic tumors at an early stage. In this study, we used a combination of in vivo fate map imaging, single-cell RNA sequencing, and a holographic photoconversion-based technique (Opto-omics) to track tumor fate and early microglial responses over time in the same animals during colonization of disseminated tumor cells. The microglial population was transcriptionally and morphologically heterogeneous, comprising both pro- and anti-tumor subsets. Genetic and pharmacological perturbations revealed that microglial phenotypes could be shifted by inhibiting TGF-β signaling or by deleting the tumor cell surface antigens CD24a and CD47. These findings reveal targetable plasticity in early-stage microglial responses to brain metastasis and suggest that harnessing pro-phagocytic microglial states may offer a therapeutic window before systemic immunosuppression becomes dominant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1778e4e2ee1302e3145504143572e313e56713fc" target='_blank'>
              Microglia Display Heterogeneous Initial Responses to Disseminated Tumor Cells.
              </a>
            </td>
          <td>
            T. Tsuji, Haruka Hirose, Daisuke Sugiyama, Mariko Shindo, Rahadian Yudo Hartantyo, Yutaro Saito, Tsuyako Tatematsu, Shouta Sugio, M. Sanbo, Masumi Hirabayashi, Yasuhiro Kojima, Jun Koseki, K. Hosoya, Hiroshi Yoshida, Tatsuya Ogimoto, Y. Yasuda, Kentaro Hashimoto, Hitomi Ajimizu, Yuichi Sakamori, H. Yoshida, Noritaka Sano, Masahiro Tanji, Hiroaki Ito, Kazuhiro Terada, M. Hamaji, T. Menju, Hiroyuki Konishi, S. Kumagai, Cyrus M. Ghajar, Daisuke Kato, Hiroshi Date, Akihiko Yoshizawa, Yoshiki Arakawa, H. Ozasa, A. Moorhouse, Teppei Shimamura, H. Nishikawa, Toyohiro Hirai, Hiroaki Wake
          </td>
          <td>2025-12-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bc8c7de17d28677187bc796f682ca2aa0c8467e" target='_blank'>
              Single-cell analysis reveals the cellular and transcriptional diversity of thyrocytes in the normal pediatric thyroid
              </a>
            </td>
          <td>
            E. R. Reichenberger, Nicholas E. Bambach, Zach H Spangler, J. Ricarte-Filho, K. Hinkle, A. Isaza, Tricia Bhatti, A. J. Bauer, A. Franco
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading male malignancy worldwide, with metabolic reprogramming being a critical hallmark of its progression. Extracellular vesicles (EVs) derived from tissues directly reflect the tumor microenvironment, offering unique insights into cancer pathophysiology that are unattainable through cell line or biofluid-derived EVs. However, the functional roles of tissue-derived EVs in PCa metabolism remain poorly understood. Leveraging our expertise in murine PCa model establishment and EV isolation from prostate tissue, this study aimed to characterize functional differences between PCa and normal prostate tissue via proteomic analysis of tissue-derived sEVs. We orthotopically implanted luciferase-labeled PCa cells into nude mice to establish an in situ PCa model, confirmed tumor formation via in vivo imaging, and harvested tissues after 4 weeks. sEVs were isolated using ultracentrifugation combined with an iodixanol density cushion and characterized by transmission electron microscopy, nanoparticle tracking analysis, and protein marker profiling. Proteomic analysis identified 28 upregulated and 24 downregulated proteins in PCa-derived sEVs compared to normal controls. Subcellular localization revealed enrichment in the cytoplasm, while pathway analysis highlighted significant involvement in metabolic processes, particularly glycolysis, amino acid biogenesis, carbon metabolism, and pyruvate metabolism. Our study establishes a robust method for isolating prostate tissue sEVs and provides the first evidence that PCa tissue-derived sEVs exhibit profound metabolic pathway alterations. These findings enhance our understanding of PCa progression mechanisms and may facilitate the development of novel diagnostic biomarkers and therapeutic strategies targeting metabolic dysregulation in PCa. SUMMARY: In this study, we created a method to isolate prostate tissue small EVs, based on our knowledge of the murine prostate cancer model building. Our data suggested that prostate tissue small EVs proteins significantly changed in many metabolism pathways, such as Glycolysis, Biogenesis of amino acids, Carbon metabolism and Pyruvate metabolism. In this study, we are the first to report prostate tissue-derived EVs proteins enriched in alterations of cancer metabolism. These differential proteins in PCa tissue EVs reflect metabolic changes in PCa and may provide insights into the development of early diagnostic biomarkers or novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c976599295f2bbc1baf63bb2a61d05088e5e38" target='_blank'>
              Proteomics of Prostate Cancer Tissue Small Extracellular Vesicles Reveal Alteration of Metabolism.
              </a>
            </td>
          <td>
            Lijuan Yu, Ting Ding, R. Olofsson Bagge, Lei Zheng, Xiaoke Hao
          </td>
          <td>2025-11-17</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b43f154dcd18b57efa496cdfa1d3835d1a5b6e" target='_blank'>
              EPI-SauriCas9-based mouse ovarian cancer models recapitulating pten deletion in patients.
              </a>
            </td>
          <td>
            Wutao Chen, Pengju He, Ling Ding, Wei-Hua Lou, Yishu Wang, Weiwei Shi, Zhangzhengyi Fan, Yumeng Sheng, Jing Luo, Zhixing Tan, You Wang, Wen Di, Xiaoping Ke, Bin Yu
          </td>
          <td>2025-12-29</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="T/NK-cell neoplasms represent rare and highly diverse cancers, distinguished by variability in their molecular architecture, local inflammatory milieu, and microenvironmental composition, which collectively underpin the diversity of clinical presentations and outcomes. The neoplastic tissue comprises malignant lymphoma/leukemic cells in concert with a spectrum of stromal elements and the acellular extracellular matrix (ECM), collectively constituting the lymphoma microenvironment (LME). These components engage in dynamic, reciprocal interactions, forming a self-regulating ecosystem capable of responding adaptively to both exogenous and endogenous stimuli. Historically, the LME was largely neglected in considerations of lymphomagenesis; however, emerging evidence highlights its pivotal role in driving core oncogenic processes, including sustained proliferative signaling, angiogenesis, immune evasion, and apoptotic resistance. Deciphering the intricate, multidirectional crosstalk among the cellular and acellular constituents of the T/NK-cell neoplastic microenvironment promises to deepen our understanding of disease biology and may inform the development of novel, mechanism-based therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57f1ac49eb4898f862ff3159155ed73801efa79" target='_blank'>
              The Significance of the Microenvironment in T/Nk-Cell Neoplasms
              </a>
            </td>
          <td>
            Ivan Petković, Michele Ritucci, A. Stojković, S. Stojnev, Aleksandar Popović, I. Conić, Milica Radić, Miljana Džunić, M. Krstić
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Due to their intrinsic heterogeneity and plasticity, pediatric brain tumors present highly complex clinical challenges. To provide insight into the molecular scene of these tumors, we generated spatial transcriptomics data from 19 distinct patients, 7 of which suffered a relapse of the disease. In this cohort, spanning 59 tissue sections across 8 diagnoses and 4 tumor grades, we recovered diagnosis-specific gene expression patterns that could be linked to processes characteristic of developmental stages. We also identified between 2 to 4 spatial archetypal niches per section, which could then be related to 11 main biological themes, and to distinct celltype-like transcriptomic signatures. For example, we noted strong spatial correlations between developmental archetypes and oligodendrocyte lineage signal in pilocytic astrocytoma. Lastly, we utilised spatially inferred copy-number variants for the profiling of relapse-associated tentative tumor clones. These clones were then related via spatial geographical analysis to regions with strong blood vessel signatures. Overall, these results provide vital details into the progression and maintenance of pediatric brain tumors, offering novel edges to be exploited in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69864dd9511e02d9a815817c172642be7190a7d" target='_blank'>
              Surveying the molecular landscape of pediatric brain tumors via spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Javier Escudero Morlanes, Timo-Pekka K Lehto, L. Larsson, L. A. Galicia, A. Mollbrink, Alia Shamikh, Elisa Basmaci, Gabriela Prochazka, T. de Ståhl, Johanna Sandgren, F. Taylan, Bianca Tesi, Ann Nordgren, Andrew Erickson, Alastair D Lamb, Klas Blomgren, Monica Nistér, J. Lundeberg, R. Mirzazadeh, L. Kvastad
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background CD47 is a “don’t eat me” signal that is overexpressed in tumors to evade phagocytosis by tumor associated macrophages (TAM). Investigational agents targeting CD47, such as magrolimab, aim to induce phagocytosis of tumor cells by TAMs. Previously, two key TAM subsets have been identified: C1QC TAMs, which display pro-phagocytic activity, and SPP1 TAMs that express pro-angiogenic markers. We characterize CD47 expression and its relationship with tumor macrophages in solid tumor samples. Patients and methods Resectable tumors from head and neck squamous cell carcinoma (n=36) (HNSCC), breast cancer (n=37) (BC), and colorectal cancer (n=36) (CRC) were evaluated for CD47 expression by immunohistochemistry (IHC), two multiplex immunofluorescence panels were used to characterize TAM markers and T cell markers. RNA sequencing was also performed. Results CD47 protein expression was higher on tumor cells compared to stromal cells across tumor indications tested. Although CRC had the lowest prevalence for CD47 expression in primary tumors, we observed a marked increase in CD47 expression in CRC liver metastases. We developed an SPP1 TAM gene signature and validated a C1QC TAM gene signature to estimate TAM abundances from bulk RNA-Seq data. In the TAM mIF analysis, HNSCC had the highest macrophage density of the indications tested. We observed a positive correlation between a higher C1QC: SPP1 TAM ratio and macrophage phenotype and tumor T cell density. C1QC macrophage signatures correlate with tumor CD47 protein expression in BC and HNSCC samples, suggesting interplay between them. Conclusions We characterized CD47 expression across key solid tumor indications being evaluated clinically using anti-CD47 blockade agents: HNSCC, breast cancer and CRC. Using a CD47 IHC assay, we identified HNSCC as an indication with the highest CD47 expression. In addition, we quantified tumor macrophages using multiplex immunofluorescence (mIF) and determined that HNSCC had the highest density of TAMs. Compared to relatively low CD47 expression in primary CRC tumors, CRC liver metastases had very high CD47 expression. Quantification of TAM signatures and CD47 expression represent key biomarkers to monitor in patient samples during exploration of CD47-blockade agents in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31725c03ca6311073138a838325de55dcf4fe468" target='_blank'>
              CD47 expression in solid tumors correlates with phagocytic tumor-associated macrophage gene signature
              </a>
            </td>
          <td>
            Nicholas van Buuren, Mengshu Xu, Yi Zhang, Paola Correa, Shiva Zaboli, Azadeh Madjidi, Christina Moon, Szu-Wen Liu, Ruidong Li, Kai Sun, Shahed Iqbal, Abhishek Aggarwal, Min Wang, Li Li, Jared M Odegard, Kelli Boyd
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517f57f9e6c335d60cb5a3e0fde4a76539819f12" target='_blank'>
              A Mechanophenotyping chip for high-throughput detection of metastatic bacteria-infected circulating tumor cells.
              </a>
            </td>
          <td>
            Wen Luo, Yanfeng Gao, Shujun Feng, Bangshun He, Jingjing Li, Jingjing Yang, Yi Yin, Meng Wang, Bin Xue, Yi Cao, Tony Y Hu, Yujun Song
          </td>
          <td>2026-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The immune microenvironment profoundly shapes the progression and therapeutic response of thyroid carcinoma. Through comprehensive analysis of single-cell RNA sequencing data, this review delineates the immune landscapes of papillary thyroid carcinoma (PTC), poorly differentiated thyroid carcinomas (PDTC), and anaplastic thyroid carcinomas (ATC), revealing a differentiation dependent trajectory of tumor immune microenvironment remodeling—from immune activation suppression coexistence in PTC, to immune exclusion in PDTC, and terminal exhaustion in ATC. This single-cell based approach enables high resolution dissection of cellular heterogeneity, immune crosstalk, and spatial organization that are often masked in bulk analyses. Such insights provide a scientific basis for precision immunotherapy, offering guidance for differentiation tailored strategies to overcome immune escape and improve clinical outcomes in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662dd84def56730e4ddb7a83c80225823dda796d" target='_blank'>
              Single-cell sequencing reveals the tumor immune microenvironment in thyroid cancer: a narrow review
              </a>
            </td>
          <td>
            Kangcong Liang, Ziyu Wang, Zhiqiang Zhang, Gengluan Liu, Xidi Wang, Heng Cao, Ming Zhong, Liping Ye, Xin Zhong, Jingyu Xun, Kefeng Lei, Ningning Li
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gliomas remain aggressive despite current therapies, highlighting the urgent need for new biomarkers and targets. Transmembrane protein 106A (TMEM106A), implicated as a tumor suppressor in various cancers, has an unclear role in gliomas. We hypothesized that TMEM106A expression associates with tumor aggressiveness and may serve as a prognostic, microenvironmental biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec5d1a0be4912907593445a40e95acffc52750f" target='_blank'>
              TMEM106A as a Macrophage‐Associated Biomarker of Prognosis in IDH‐Wildtype Glioma: Integrative Multi‐Omics and Spatial Analyses
              </a>
            </td>
          <td>
            Wen-Shin Song, Pei-Chi Chang, Dueng-Yuan Hueng, Yao-Feng Li
          </td>
          <td>2025-12-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive cancer thought to arise from impaired myogenesis. This can be substantially overcome by increasing the levels of pannexin 1 (PANX1), a critical component of the myogenic program, but the mechanism involved is unknown. Using RNA-seq, we have previously found that overexpression of PANX1 dramatically reshapes the transcriptomic landscape of RMS including downregulation of a myogenic modulator, APOBEC2 (apolipoprotein B mRNA editing enzyme catalytic subunit 2). Following this clue, we investigated the role of APOBEC2 in the PANX1-mediated suppression of RMS malignancy. Here we show that, using a panel of patient-derived RMS cell lines and tumor specimens, APOBEC2 is expressed in RMS, but that its levels are lower than those in both differentiating myoblasts and skeletal muscle. In most RMS cell lines examined, APOBEC2 accumulates during proliferation and sustains their stem-like characteristics, as evidenced by its ability to promote the growth of spheroids upon increased expression. Yet, ectopic PANX1 expression led to a marked downregulation of APOBEC2 across a large proportion of RMS cell lines assessed. Strikingly, these were the same cells in which PANX1 triggers multinucleation. We further reveal that, like healthy myoblasts progressing through myogenesis, the multinucleation observed here in RMS cells results from cell fusion. Importantly, in RMS cells engineered to overexpress APOBEC2, PANX1 no longer enhances cell fusion, but its other anti-tumorigenic properties are still preserved. Collectively, our data indicate that PANX1 promotes RMS cell fusion by downregulating APOBEC2 expression, driving these tumor cells further into the myogenic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398de031cdf1c2189823064d4c0ea5c0fa3f3f02" target='_blank'>
              Pannexin 1 induces Rhabdomyosarcoma cell fusion by downregulating APOBEC2
              </a>
            </td>
          <td>
            Alexandra Welten, Amit Bera, S. Langlois, X. Xiang, Keshav Gupta, Emily Freeman, Kyle N. Cowan
          </td>
          <td>2025-11-18</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027f986afc66cae82f4bb168ec0c6120d17b62d0" target='_blank'>
              Transgelin defines pro-tumorigenic cancer-associated fibroblasts in pancreatic cancer.
              </a>
            </td>
          <td>
            Xingxing Wang, Keiko Shinjo, Kohei Kumegawa, R. Maruyama, Shinji Mii, Yukihiro Shiraki, Tastunori Nishimura, Y. Murofushi, Miho M. Suzuki, T. Kabasawa, Mitsuru Futakuchi, A. Kanai, Yutaka Suzuki, Atsushi Enomoto, Yutaka Kondo
          </td>
          <td>2025-12-16</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2473c436968a0964a051f1901617b06af4fa0b" target='_blank'>
              Systemic activation and tissue infiltration of CD8 + CX3CR1 + T cells in non-small cell lung cancer treated with neoadjuvant immune checkpoint blockade.
              </a>
            </td>
          <td>
            Sisi Dai, Yongcheng Liu, Tingting Guo, Hong Luo, Guanglei Yang, Songliang Yu, Sifan Zhang, Lili Jiang, Jie Liu, Ye Wang, Daxing Zhu, Xuyu Cai
          </td>
          <td>2025-11-22</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma (DLBCL) is an aggressive cancer with profound heterogeneity. While standard immunochemotherapy is curative for the majority, treatment failure in 30–40% of patients due to refractory or relapsed disease remains a significant clinical challenge, emphasizing the need for better prognostic markers. We investigated the interplay between tumor glycolytic metabolism and the tumor immune microenvironment. We identified four distinct glycolysis-related patterns that were associated with three immune subtypes (known as immune-inflamed, immune-excluded, and immune-desert). Importantly, these patterns stratified clinical outcomes and predicted chemotherapy sensitivity. For example, tumors with an immune-inflamed subtype were more sensitive to doxorubicin. Collectively, our findings propose a metabolism- and immune-based classification framework for DLBCL that may facilitate more precise risk stratification and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894f2d6a04501c694424859815514794b12befc" target='_blank'>
              Glycolysis-Driven Immune Subtypes in the Tumor Microenvironment Determine Clinical Outcomes in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Ying Du, Zhenyu Liu, Chuanyan Liu, Yining Yuan, Mengkun Fang, Tong Zhou, Mengge Pan, Weilu Xu, Xu Liu, Peipei Xu
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Introduction The tumor microenvironment (TME) is a complex ecosystem where cellular interactions are spatially governed. Spatial transcriptomics (ST) technologies, sequencing-based Visium which only have spot level resolution and imaging-based Xenium which are limited by a small RNA panel, probe the complexity, but high dimensional data interpretation remains a challenge [1]. Current analyses rely on pre-defined cell annotation patterns, critically failing to account for higher-order structures in gene co-expression and cell co-localization [2]. Biologically, cellular function is defined by coordinated gene modules, not individual genes, and these relationships exhibit a hierarchical organization across multiple spatial scales. A simple gene expression matrix is insufficient to capture this multi-scale and hierarchical structure of gene–gene and cell–cell interactions, especially given that TME niches are highly variable and context-dependent, difficultly to make a fixed scale definition. Furthermore, the order of cells in discrete gene expression matrices can affect the ability of neural networks to analyze cell–cell interactions. To overcome this, we introduce CoSMIM (Covariance of Spatial Multi-scale Integrated Microenvironment) matrix. Our hypothesis is that gene–gene covariance matrices, computed across multiple scales of neighbor cells, provide a richer representation of the TME’s spatial heterogeneity functional architecture. We use a multi-decoder conditional variational autoencoder(CVAE) to learn a unified latent space for consistent cross-scale modeling. Its architecture can impute genes in targeted spatial panels and infer spatial covariance matrices for non-spatial single-cell data, thus bridging these modalities. Methodology Our framework consists of two core components: the construction of a CoSMIM niche representation and a CVAE generative model for learning a unified representation. The input to our method is ST data from dataset HEST-1 k and 10X human-breast cancer FFPE dataset [3,4] which has various ST platforms such as Visium, Xenium, CosMx, or MERFISH. The specific steps are as follows (Fig. 1b): Definition and construction of multi-scale neighborhoods Tumor spatial heterogeneity is not random but is driven by the organizational patterns of cellular communities at different scales. Precisely defining cellular niches within ST data remains a key challenge. Given the heterogeneity in cell size, defining neighborhoods using a fixed number of cells or a constant radius has limitations [5]. We propose an adaptive approach that constructs neighborhoods through a learned linear combination of different biological scales (Fig. 2&Eq.2). We predefine a series of spatial neighborhood representing distinct biological interaction ranges. The gene expression matrix of whole tissue is \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{X}\in{R}^{n\times g}$\end{document}, \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $n$\end{document} is the number of cells and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{g}$\end{document} is number of genes. Each cell has a muti-scale niches matrix \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} ${E}_k^i\in{R}^{k\times g}\left(\mathrm{e}.\mathrm{g}.\mathrm{k}\in \mathrm{K}\left\{\mathrm{5,15,50}\right\}\right)$\end{document} of \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} nearest neighbor cells (excluding the target cell \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $i$\end{document} itself). Let \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{Cov}\in{\mathrm{R}}^{g\times g}$\end{document} be the target cell \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $i$\end{document}’s gene–gene covariance of \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} nearest neighbor cells. \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\overline{X}$\end{document} global mean gene expression of gene g and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\overline{E_k}$\end{document} is \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} scale niche’s mean gene expression. \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ \mathrm{Co}{v}_k^{troditional}={\sum}_{i=1}^n Co{v}_k\left({E}_i\right)={\sum}_{i=1}^n\frac{1}{k}{\left({E}_i^k-\overline{E_i^k}\right)}^T\left({E}_i^k-\overline{E_i^k}\right) $$\end{document} \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ \mathrm{Co}{v}_k^{our\; method}={\sum}_{i=1}^n\mathrm{Co}{\mathrm{v}}_{\mathrm{k}}\left({E}_i^k\right)={\sum}_{i=1}^n\frac{1}{k}{\left({E}_i^k-\overline{X}\right)}^T\left({E}_i^k-\overline{X}\right) $$\end{document} \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ CoSMIM=\sum_{k={k}_1}^K{w}_k Co{v}_k^{ourmethod},{w}_k\ge 0,\sum_{k={k}_1}^K{w}_k=1(1) $$\end{document} We then evaluate and optimize the selection of these scales \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $K$\end{document} based on the discriminatory power of their corresponding eigenvalues in principal component analysis. These scales correspond to different biological contexts: Small-scale (5–15 cells) Primarily captures direct signaling and short-range paracrine signaling, such as direct contact between a tumor cell and an adjacent immune cell PD-1/PD-L1 or local growth factor support from a Cancer-Associated Fibroblast. Medium-scale (15–50 cells) Reflects the characteristics of a functional niche, such as an immunosuppressive or pro-angiogenic ‘community’ composed of several tumor, immune, and stromal cells. Large-scale (>$50cells) Characterizes the tissue region state, such as broad hypoxic zones, regions of lymphocytic infiltration, or fibrotic stromal areas, which are often governed by diffusive chemical gradients or physical barriers. The scale weights are learned by entropy-regularized log-det minimization: \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ {\mathrm{w}}^{\ast }=\arg \underset{\mathrm{w}\in{\Delta}^{\mathrm{K}-1}}{\min}\left(-\log \left|\Sigma \left(\mathrm{w}\right)+\upvarepsilon \mathrm{I}\right|+\uplambda{\sum}_{\mathrm{k}=1}^{\mathrm{K}}{\mathrm{w}}_{\mathrm{k}}\log{\mathrm{w}}_{\mathrm{k}}\right)\kern0.36em (2) $$\end{document} With \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\varepsilon >0$\end{document} for numerical stability and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\lambda \ge 0$\end{document} (larger \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgre">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a83168d353c1d44c04ce73b829897d6c778eb0" target='_blank'>
              CoSMIM: a covariance-based multi-scale framework for integrated spatial microenvironment analysis
              </a>
            </td>
          <td>
            Bairui Du, H. Kiryu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This review was conducted to address the complexities of breast cancer research, which is often limited by two-dimensional cell cultures that fail to accurately represent the tumor microenvironment. Although three-dimensional (3D) models are generally superior, the absence of standardized methodologies has resulted in inconsistent findings. The objective of this review was to systematically analyze how different 3D culture methods influence the epigenetic reprogramming of tumor cells, focusing on changes in DNA methylation, histone modifications, and chromatin organization, as well as the differential expression of various types of non-coding RNA. Our findings demonstrate that different 3D culture methods are not equivalent and yield distinct epigenetic signatures. This underscores the urgent need to standardize protocols and provide detailed reports on the properties of culture media to enhance reproducibility, which is crucial for identifying new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933fb504bd5970ab97b7de6c042e05e852c76571" target='_blank'>
              Epigenetic and Transcriptional Reprogramming in 3D Culture Models in Breast Cancer
              </a>
            </td>
          <td>
            Laura Cecilia Flores-García, Karla Rubio, Eloísa Ibarra-Sierra, M. B. Silva-Cázares, C. Palma-Flores, C. López-Camarillo
          </td>
          <td>2025-11-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The field effect describes the phenomena where environmental exposures, infection, and genetic predisposition result in molecular changes in cells that predispose them to developing cancer. Though this is a well-established concept in pathology, it remains underexplored in the context of high-resolution omics. We utilized the Diagnostic Evidence Gauge of Single Cells (DEGAS) deep transfer learning framework to analyze prostate cancer spatial transcriptomics to identify cells and tissues that are highly associated with cancer progression. DEGAS highlighted morphologically benign glands that had reduced expression of MSMB, a differentiation marker that is decreased in aggressive tumors. These glands have upregulated genes associated with antigen presentation and aggressive neoplasms. Integration of single-cell transcriptomics and deep learning image analysis separately revealed altered immune-cell infiltration, suggesting a complex interplay in the tumor environment facilitating aggressiveness. We used immunohistochemistry to quantify the MSMB protein (PSP-94) expression on morphologically normal and tumor tissues from patients with and without 5-year distant metastasis. Samples from patients who developed metastasis consistently showed lower fractions of positively stained cells, indicating a subtle yet significant "field effect" in seemingly benign regions. These proteomic results validate the transcriptomic findings and further underscore that inflammatory or immune-related changes in ostensibly normal tissue may contribute to aggressive disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68d7d78a2fef9048f9bcafbf03164f9002e0042" target='_blank'>
              Deep Transfer Learning Links Benign Glands to Prostate Cancer Progression via Transcriptomics.
              </a>
            </td>
          <td>
            Justin L Couetil, Ziyuan Liu, Chao Chen, A. Alomari, Kun Huang, Jie Zhang, Travis S. Johnson
          </td>
          <td>2025-11-29</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb706d249cfb2775d83ca53d2e8588e8894f0cf6" target='_blank'>
              Dual Transcriptional Drivers of Immune Suppression in SACC: NOTCH1 for Cellular and MYB for Humoral Immunity
              </a>
            </td>
          <td>
            Guoliang Yang, Xudong Wang, Tian Ye, Tingyao Ma, Youmei Chen, Fang Nan, Qian Chen, Lu Kong, Xiao-hong Chen
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Precision medicine has evolved through distinct phases, from the origins of the Human Genome Project to mutation‐based targeted therapies. This editorial posits that “stereological cell biomedicine” could be a new approach promoting the development of the next generation of precision medicine. This emerging discipline transitions the focus from genomic data to the multi‐dimensional and spatiotemporal complexity of single cells. Driven by advances in Stereo single‐cell multi‐omics (Stereo Cell‐seq), spatial transcriptomics (Stereo‐seq), and single‐cell surfaceomics (sc‐surfaceome), this approach aims to capture the stereologically dynamic interactions between organelles within a cell and between cells in the tissue. We argue that understanding the spatiotemporal location of molecules, particularly protein interactions at organelle interfaces and on the cell surface, is as critical as their abundance for defining cellular function in health and disease. Integrating these high‐resolution measurements with artificial intelligence and computational modelling will bridge the gap between advanced omics and pathology. Initiatives such as the newly established European Stereo Cell Center (ESCC) signal a global shift towards this new paradigm, which promises to unlock novel diagnostic biomarkers and therapeutic targets for truly multi‐factorial and dynamic precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdcd69ccf472862bb486c6c2a16a93191b0f96" target='_blank'>
              Stereo cell: A new approach to the next generation of clinical precision medicine
              </a>
            </td>
          <td>
            Wanxin Duan, Mingjie Wang, Yifei Liu, Celine Desoyer, Christian Baumgartner, Xiangdong Wang
          </td>
          <td>2025-11-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26bc66a34e745a67c0425c16be84d04301c9194" target='_blank'>
              Multimodal analysis reveals potential association of CDH13 with endothelial cells and its overexpression in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Ke-Jun Wu, Li-Hua Yang, Dong-Ming Li, Rong-Quan He, Di-Yuan Qin, Shi-De Li, Jian-Di Li, Y. Dang, Ming-Jie Li, Qi Li, Jian-jun Li, Lu Zhang, Han He, Ji-Feng He, Gang Chen, Xiao-Bo Jiang
          </td>
          <td>2025-12-12</td>
          <td>European journal of medical research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebcdfb94400220dc54cbd6a2cfd578841de975f" target='_blank'>
              Integrated transcriptomic and single-cell analysis reveals cell cycle dysregulation and cellular heterogeneity in lung cancer.
              </a>
            </td>
          <td>
            Yuzi Zhang, Wencheng Che, Qingchuan Li, Wei Huang
          </td>
          <td>2025-12-30</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Prostate cancer commonly presents as multifocal disease, with distinct tumours that are often genetically independent. Metastases, however, typically derive from a single dominant clone, highlighting the clinical challenge of identifying which lesion drives progression. Focal therapy, an emerging treatment strategy, aims to target only the most aggressive tumour focus or foci, sparing normal prostate structures and reducing morbidity. Yet, the lack of consistent genomic drivers in primary tumours complicates efforts to identify the focus most likely to progress. While prostate cancers are known to harbour widespread methylation alterations, the extent to which these epigenetic changes track with clonal origin remains poorly understood. To address this, we performed multiregional epigenomic sequencing of 189 prostate tissue samples, including 109 tumour foci and 80 non-tumour regions from 44 patients (2-8 tumour samples per case). Differential methylation analysis revealed widespread alterations distinguishing benign from tumour tissue, including a core ‘epigenetic trunk’ of >20,000 recurrently altered CpGs shared across cases. DNA variants called from non-methylation sites of the epigenomic sequencing closely matched variants from WGS in cases with WGS available. Phylogenetic trees constructed from DNA variants and phyloepigenetic trees from methylation sequencing revealed striking patterns. Some cases lacked shared DNA variants across all tumour foci, indicating a polyclonal tumour origin, yet these genetically unrelated foci shared extensive methylation alterations. This suggests widespread methylation convergence occurs in clonally distinct tumour foci. To dissect clonal relationships further, we applied two orthogonal lineage tracing approaches using the methylation sequencing data: fluctuating CpGs (sites that change stochastically over time) and highly entropic 8-mer ‘methylation barcodes’ within the protocadherin gene cluster. Fluctuating CpGs showed concordant states across tumours that shared DNA sequence variants, while the methylation barcodes likewise revealed high similarity among such tumours. Together, these methods support the independent evolutionary origins of polyclonal prostate tumour foci. Pathway analysis of recurrently altered methylation loci highlighted consistent enrichment in epithelial–mesenchymal transition, MYC signalling, DNA damage response, and hormonal signalling across cases, suggesting functional convergence at the pathway level despite divergent genetic origins, consistent with selective pressures acting on shared programmes. Together, these findings demonstrate that DNA methylation alterations represent fundamental and recurrent events in prostate cancer, more consistent than DNA sequence variants across multifocal disease. Lineage-tracing analyses further highlight that methylation processes record tumour evolutionary history, encoding both clonal relationships and convergent biology. These observations may inform methylation-based biomarkers and the rational selection of lesions for focal therapy.



 Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Warren, Andrew Lynch, Daniel Brewer, David Wedge, CRUK-ICGC Prostate Cancer Group, Charles Massie, Harveer Dev. Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90717932598b7c081fea25f61e12e100d9757059" target='_blank'>
              Abstract B003: Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer
              </a>
            </td>
          <td>
            Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Y Warren, Andrew Lynch, Daniel Brewer, D. Wedge, Charles Massie, H. Dev
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Despite the crucial involvement of the EGFR pathway in hepatocellular carcinoma (HCC), the clinical efficacy of EGFR antibodies in HCC remains uncertain. While existing evidence suggests that immune dysfunction and tumor microenvironment alterations may contribute to treatment resistance, the precise markers underlying this phenomenon in HCC warrant further investigation. In this study, we employed patient-derived xenograft (PDX) models generated from 14 HCC patients enrolled in the REHOPE301 cohort to evaluate the sensitivity to nimotuzumab, a humanized anti-EGFR monoclonal antibody. Whole-exome sequencing (WES) and single-cell RNA sequencing (scRNA) were performed on tumor tissues and tumor-infiltrating lymphocytes (TILs) to elucidate the association between TIL characteristics and EGFR antibody response. In addition, immunofluorescence (IF) staining and flow cytometry were used to validate the findings from scRNA. A predictive risk score and nomogram were subsequently developed using LASSO regression analysis. The prognostic performance of this model was evaluated using 2 external datasets (ICGC-JP and GSE141202) through receiver operator characteristic (ROC) curves and calibration curves analyses. Nimotuzumab demonstrated a 50% response rate (7/14) in PDX models. Immune profiling revealed distinct TIL patterns between responders and non-responders. Notably, CD4+CCR7+ T cells were significantly enriched in resistant tumors (p < 0.001) and negatively correlated with the nimotuzumab response (r = -0.767 p = 0.02). IF analysis revealed a higher proportion of CD4+CCR7+ double-positive T cells in the non-responder group compared to responders (p = 0.012). In non-responsive tumors, CD4+CCR7+ T cells exhibited interactions with of macrophages and CD8+PDCD1+ T subsets. The proportion of CD4+CCR7+ T showed negative correlations with active CD8 T infiltrations. A reduced infiltration of CD4+CCR7+ T cells was associated with improved prognosis and enhanced EGFR antibody efficacy across multiple cancer types. Furthermore, a nine-gene signature related to CD4+CCR7+ T cells was identified as a strong prognostic factor in HCC (HR = 5.19, 95% CI: 3.18–8.46, P < 0.001), and was used to construct a nomogram. WES confirmed prognostic gene mutations (VCAN, CAMK4, and CD226) potentially influencing nimotuzumab response. Our findings demonstrate that elevated infiltration of central memory CD4+CCR7+ T cells in HCC correlates with an immunosuppressive tumor microenvironment, which may contribute to impaired efficacy of EGFR-targeted antibodies and worse clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f880c34e4aa66446e6ed1bc0fa4932160c8966" target='_blank'>
              Multi-omics analysis reveals the role of tumor-infiltrating CD4+CCR7+ T cells in EGFR antibody resistance and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zizhong Yang, L. Qiu, G. Jia, Zhuoya Sun, Y. Gong, Yin Chen, Yu Wang, Lai Song, Xiao Zhao, Shunchang Jiao
          </td>
          <td>2025-11-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor-Associated Macrophages play a significant role in the tumor microenvironment of breast cancer, influencing its growth and progression. Their presence is associated with key processes such as angiogenesis and immunomodulation, impacting the immune system response and tumor aggressiveness. This study employed various methods to analyze scRNA-seq and spatial transcriptome data. The scRNA-seq data underwent thorough processing and analysis, with a focus on breast cancer subtypes. The CellChat R package was employed to analyze and visualize cell-cell communication networks within the scRNA-seq data. hdWGCNA, conducted in R, involved multiple steps such as data preprocessing, gene network construction, module identification, module preservation analysis, and functional enrichment analysis. To analyze the pseudotime dynamics of endothelial cells, Monocle3 was utilized. The spatial data for normal skin and leprosy were obtained and analyzed using the SPATA2 and semla R packages, facilitating the integration of spatial and single-cell data. The study utilized scRNA-seq and spatial transcriptome data to comprehensively analyze 40 breast cancer samples, including ER+, HER2+, and TNBC subtypes, alongside normal mammary tissue. Distinct cellular clusters were identified, and myeloid cells were characterized into four subtypes. Comparative analyses across breast cancer subtypes highlighted variations in the expression of the CXCL10 gene, and hdWGCNA provided insights into molecular landscapes. Trajectory inference analysis revealed pseudotime dynamics in myeloid cells, with specific gene expression changes. Spatial transcriptome profiling in TNBC and HER2+ breast cancer showed unique cellular clusters and gene expression patterns within the tumor microenvironment. Spatial and single-cell data integration emphasized the distribution of cell types and spatial enrichment of gene expression signatures.  TAMs emerge as key players engaging in multifaceted interactions with neighboring cell types within the intricate context of breast cancer. These identified genes, particularly CXCL10, assume central roles in regulating various facets of myeloid cell physiology and modulating immune responses. CXCL10 expression increases in TNBC macrophages suggest a potential association with T lymphocyte activity and myeloid cell chemotaxis, implicating its significance in the intricate immune response dynamics within the TNBC microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666c93cdc1f48635e7eb409847dc8e7a85718b81" target='_blank'>
              Spatial profiling of CXCL10 in breast cancer: tumor-associated macrophages' mediation of immune response
              </a>
            </td>
          <td>
            Natália Cordeiro de Albuquerque D’Alva, Ana Kelly da Silva Fernandes Duarte, Leonardo Vinícius da Silva Lima, Eduardo Paulo da Silva Lima, Caio Cesar Fernandes Nobre Porto, Maria Nathália de Menezes, Luis Filipe Pereira Rocha, Leonardo Prudêncio Coutinho de Almeida, Rafael de Oliveira Calaça Farias, Leonardo Barbosa da Costa, Salomão Belfort Sparapan de Melo, Raimundo Rodrigues de França Júnior, Rodolfo Tibério Ferreira Silva, Alberson Maylson Ramos DA Silva, Taina Santos Bezerra
          </td>
          <td>2025-12-04</td>
          <td>CONTRIBUCIONES A LAS CIENCIAS SOCIALES</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC), a prevalent renal malignancy with limited early symptoms, demonstrates immunotherapy sensitivity in advanced stages. This study investigates how exhausted T cells drive ccRCC progression through single-cell transcriptomics. Re-analysis of a single-cell transcriptomic dataset was performed through dimensionality reduction and clustering to annotate cells into distinct populations, followed by comprehensive characterization of cellular composition and subset-specific features. Prognostically relevant genes were identified via differential gene expression (DGE) analysis, and a nomogram prediction model was constructed using Cox regression analysis, with validation through Kaplan–Meier (KM) survival curves and risk score plots. Functional annotation of SOCS3+ exhausted T cells was achieved via gene ontology (GO) and reactome pathway enrichment analyses. Cell–cell communication networks involving SOCS3+ exhausted T cells were delineated using ligand–receptor interaction profiling. Single-cell transcriptomic data were annotated into nine distinct cellular populations, among which SOCS3+ exhausted T cells demonstrated significant prognostic relevance. A nomogram prediction model incorporating SOCS3 and N4BP1 effectively stratified patients into low- and high-risk groups with superior prognostic predictive power compared to conventional parameters. Cell–cell communication analysis revealed that SOCS3+ exhausted T cells interact with myeloid cells through the MIF signaling pathway. Integrated single-cell transcriptomic analysis demonstrates that SOCS3+ exhausted T cells promote tumor resistance to cytotoxic killing and serve as a robust prognostic biomarker in ccRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f553143dc61bf2d891660f6079995a8925cedfb6" target='_blank'>
              Single-cell transcriptomics identifies SOCS3+ exhausted T cells as a biomarker facilitating clear cell renal cell carcinoma progression
              </a>
            </td>
          <td>
            Xiaoqin Jiang, Yifan Xu, Zhenjie Wu, Xinxin Gan, Zhiyuan Zhuo, Guanghui Peng
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Bone metastases are a common and devastating complication of advanced prostate cancer (PCa), with up to 80–90% of men with castration-resistant PCa (CRPC) developing metastases to the bone. The development of bone metastasis serves as a critical indicator of worse long-term clinical outcomes. Understanding the factors driving PCa metastasis to the bone, including through myeloid cell reprogramming within the tumor microenvironment (TME), is critical. Our study examined published single-cell RNA sequencing data from 7 PCa patients with bone metastasis and 7 benign donors, focusing on myeloid cells across tumor, involved marrow, distal marrow, and benign bone factions. We used manual annotation methods and scANVI for whole integrated data annotation. Myeloid cells were sub-classed and manually annotated for classical monocytes (Mono1), intermediate monocytes (Mono2), non-classical monocytes (Mono3), myeloid dendritic cells (mDCs), tumor-infiltrating monocytes (TIMs), tumor-associated macrophages (TAMs), and osteoclasts. We applied pseudotime and used CellRank's Random-Walk Simulation for trajectory inference to determine final terminal states for the myeloid cells, i.e., tumor-associated macrophages (TAMs), myeloid dendritic cells (mDCs), and osteoclasts. Ranked driver gene lists for every sample type (benign, distal, involved, tumor) and cell type (Mono1/2/3, TIM) that were associated with the final terminal states were generated. QIAGEN Ingenuity Pathway Analysis (IPA) and Gene Set Enrichment Analysis (GSEA) were performed on driver gene lists across sample types and cell types, thus identifying upstream regulators influencing terminal state formation. IPA and GSEA identified ATF3, a stress-response regulator, and NF-κB, a central inflammatory mediator, to be significantly associated with TAM differentiation driver genes and were shared across extremes of sample types (benign & tumor) and cell types (Mono1 & TIM). CellOracle's in-silico perturbations were performed to assess the significance of ATF3 in context of the PCa bone TME while visualizing impacts on cell development. Through siRNA suppression of ATF3 in-vitro, followed by flow cytometry and qPCR for knockdown validation, our results show reduced M2/TAM differentiation with decreased associated markers, suggesting a role for ATF3 in myeloid reprogramming that may contribute to TAM-associated immunosuppression and the metastatic disease state



 Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain. Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3000b262dc2818edcc8efcb35668c9389013e" target='_blank'>
              Abstract A019: Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis
              </a>
            </td>
          <td>
            Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinomas (cSCCs) arise from keratinocytes in the skin, but the molecular changes driving this transformation remain unclear. To better understand this process, we perform multi-omic profiling of keratinocytes, actinic keratoses, and cSCCs. Single-cell mutational analyses reveal that most keratinocytes have remarkably low mutation burdens; however, keratinocytes with TP53 or NOTCH1 mutations exhibit substantially higher burdens. These findings suggest that keratinocytes can withstand high dosages of cumulative ultraviolet radiation, but certain pathogenic mutations break these adaptive mechanisms, inducing a mutator phenotype. Mutational profiling of cSCCs adjacent to actinic keratoses reveals TERT promoter and CDKN2A mutations emerge in actinic keratoses, whereas additional mutations that inactivate ARID2 and activate the mitogen-activated protein kinase pathway delineate the transition to cSCC. Surprisingly, actinic keratoses are often not related to their neighboring cSCC. Spatial analyses reveal gene expression heterogeneity, including checkpoint molecule enrichment at invasive fronts, highlighting tumor and immune cell interactions. The evolution of cutaneous squamous cell carcinoma (cSCC) remains poorly understood. Here, the authors employ multi-omics and multi-scale analyses to explore the genetic evolution of keratinocytes to cSCC, finding key pathogenic mutations that break the resistance to ultraviolet radiation as well as spatial heterogeneity patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ace63c32efba22479251c352c77035a045dd5b73" target='_blank'>
              Genetic evolution of keratinocytes to cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, A. K. Bandari, Noel Cruz-Pacheco, H. Sharma, Meng Wang, Albert Xu, Daniel B. Chen, Christopher D. George, Annika L. Marty, Raymond J. Cho, Jeffrey B. Cheng, Drew K Saylor, P. Gerami, I. Yeh, Sarah T. Arron, Boris C. Bastian, A. H. Shain
          </td>
          <td>2025-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) is a leading cause of cancer-related mortality. Protein succinylation, a key post-translational modification, is implicated in tumor progression. However, its comprehensive landscape and clinical significance in LUAD remain largely unexplored. Methods We integrated multi-omics data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) cohorts. A set of core succinylation-related genes was identified through differential expression and univariable Cox regression analyses. Molecular subtypes based on succinylation were determined by principal component analysis (PCA). A succinylation prognostic model was constructed via least absolute shrinkage and selection operator (LASSO) and multivariable Cox regression. The differences of tumor microenvironment (TME), tumor mutation burden and drug sensitivity in different risk groups were further explored. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics revealed effects of succinylation on TME. High-dimensional weighted gene co-expression networks analysis (hdWGCNA) was used to identify potential succinylation-related therapeutic targets. The function of therapeutic targets was further validated through scRNA-seq, spatial transcriptomics, and in vitro experiments. Results We identified 31 core succinylation-related genes and defined three molecular subtypes with distinct prognostic and TME characteristics. A robust 7-gene succinylation-based prognostic signature was developed and validated across 7 independent GEO cohorts, effectively stratifying patients into high- and low-risk groups with significant differences in survival, demonstrating high predictive accuracy, consistency, and clinical utility. The low-risk group exhibited an immunoreactive TME with enhanced immune cell infiltration and superior response to immunotherapy. scRNA-seq and spatial transcriptomics revealed enhanced succinylation in LUAD. Kallikrein-related peptidase 6 (KLK6) was identified as a potential therapeutic target. KLK6 was significantly upregulated in LUAD, correlated with poor prognosis and therapy resistance. KLK6 promoted global succinylation, proliferation, migration, and invasion of LUAD cells in vitro. Mechanistically, KLK6-positive tumor cells might foster an immunosuppressive TME by driving fibroblast-to-myofibroblast differentiation, enhancing extracellular matrix (ECM) deposition, and inhibiting CD8+ T cell infiltration. Conclusion Our study delineates the succinylation landscape in LUAD, establishes a novel prognostic model for risk stratification and immunotherapy prediction. Meanwhile, we identified KLK6 as a potential promoter of tumor progression and immunosuppression. Targeting the succinylation pathway, particularly KLK6, may represent a promising therapeutic strategy for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7908640cb31b86a0484adf705e28a329febb9146" target='_blank'>
              Succinylation heterogeneity in lung adenocarcinoma: from prognostic model to KLK6-driven tumor microenvironment remodeling
              </a>
            </td>
          <td>
            Jichang Liu, Xuehan Zhu, Chenlong Zha, Jiaqi Ding, Chuan Zhang, Yizhe Wang, Tao Yan, Hui Kong, Yong Liu, Jingyu Chen
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a90c7d87d145743b8f163f5d40e3c8d84fc50c6" target='_blank'>
              Single-cell phylodynamics reveal rapid late-stage colorectal cancer expansions
              </a>
            </td>
          <td>
            J. M. Alves, Kylie Chen, Sonia Prado-López, N. Estévez-Gómez, P. Alvariño, Juana Alonso, Monica V. Valecha, Laura Tomás, A. Zwaans, Ugnė Stolz, Tanja Stadler, Débora Chantada, J. Cameselle-Teijeiro, Alexei Drummond, D. Posada
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor characterized by immunosuppressive tumor microenvironment and metabolic reprogramming. Dysregulated lactate metabolism has been observed in the glioma microenvironment, yet the functional roles of lactate accumulation and lactylation modification in driving GBM progression remain poorly understood. We identified core genes associated with lactylation by analyzing gene expression data from GBM tumor tissues and normal tissues, combined with the screening of lactylation-related genes. Utilizing LASSO regression analysis, we developed an optimized five-gene lactylation-related signature, validated within the CGGA cohort. Patients were categorized into high- and low- LRGS groups based on risk scores. Subsequently, we compared clinical outcomes, immune profiles, and therapeutic sensitivities between these two groups. To further elucidate the impact of lactylation on immune cells, we analyzed single-cell sequencing data from the tumor microenvironment of GBM patients. We identified five lactylation-related hub genes strongly associated with GBM and constructed a lactylation-related gene prognostic model. The risk scores of this lactylation-related model showed strong correlations with immune cell infiltration levels. Single-cell sequencing analysis revealed that significant differences in lactylation levels across various immune cell subtypes, with tumor-associated fibroblasts exhibiting markedly higher lactylation activity. Additionally, patients in the low- LRGS group demonstrated enhanced sensitivity to chemotherapy and targeted therapies. The lactylation-related gene signature established in this study demonstrates robust predictive efficacy in glioblastoma, serving not only as a prognostic biomarker but also offering new directions for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006270d0e3c600cb8c7c6b120761a958aa4b1dcf" target='_blank'>
              Prognostic and therapeutic response prediction in glioblastoma multiforme using a lactylation-associated gene signature
              </a>
            </td>
          <td>
            Huan Zhu, Guanjun Li, Tingting Li, Xiangdi Yang, Yongqin Yang, Yifei Li, Zhigang Liu
          </td>
          <td>2025-11-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral heterogeneity poses significant challenges to the efficacy of cancer immunotherapy. Melanoma-associated antigen D4 (MAGED4) has been proposed as a potential immunotherapeutic target in oral squamous cell carcinoma (OSCC). This study aims to investigate the expression of MAGED4, focusing on its intratumoral expression heterogeneity and the underlying epigenetic regulation mechanism. Utilizing public online databases, immunohistochemical analyses of clinical specimens, and single-cell RNA sequencing data, we found that MAGED4 was overexpressed with significant intratumoral heterogeneity in OSCC tissues. Methylation-promoter luciferase reporter assays revealed that MAGED4 transcription was suppressed by DNA methylation at its promoter region. Additionally, co-expression analysis implicated a potential role for histone acetylation in regulating MAGED4. To functionally validate these findings, we treated OSCC cells with the DNA methyltransferase inhibitor 5-aza-2′-deoxycytidine (DAC) and histone deacetylase inhibitors trichostatin A (TSA) and valproic acid (VPA). The triple-drug combination treatment resulted in the most robust reactivation of MAGED4 expression, correlating with promoter DNA demethylation and enhanced acetylation of H3K9 and H3K27 at the MAGED4 promoter. Our findings elucidate critical epigenetic mechanisms contributing to MAGED4 heterogeneity in OSCC and highlight the potential of combination epigenetic therapies to reverse this heterogeneity, thereby providing a foundation for exploring such approaches to improve immunotherapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70fc18a29ffe957dcc14acfa8a9ef8ccd601b26" target='_blank'>
              Intratumoral Heterogeneity of MAGED4 Expression in Oral Squamous Cell Carcinoma: Epigenetic Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Huan Xie, Feng Li, Xiaoqiong Zou, Xiaoqing Yu, Sheng Zhang, Yanjing Wang, Wei-xia Nong, Limin Xie, Yi Wang, Bin Luo, Xiaoxun Xie, Qingmei Zhang
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction The analysis of tumor-infiltrating lymphocytes often requires techniques that expand their numbers, potentially introducing bias. To address this, we performed a detailed analysis of minimally cultured TILs to evaluate whether this approach better preserves their characteristics. Methods The TIL culture method was based solely on tumor tissue with low IL-2 supplementation to minimize artificial alterations. The validity of this approach was confirmed by the correlation between CD3+ T cell percentages in cultures and infiltration patterns observed by immunohistochemistry. Immunophenotyping, cytokine release, and TCR repertoire analysis were used to characterize CD4+ and CD8+ T cell subsets and their molecular features during minimal expansions. Results High TIL infiltration areas did not consistently correspond to an increased presence of any T cell subset; both CD4+ and CD8+ T cells frequently coexisted in these regions. In contrast, low TIL infiltration sections often displayed a higher proportion of CD4+ T cells. An inverse correlation between CD4+ T cell percentages and cytotoxic molecules was observed, indicating reduced cytotoxic activity in low-TIL sections with abundant CD4+ T cells. TCR repertoire analysis revealed differences between T cell subsets: CD4+ T cells were associated with longer TRA CDR3 nt and shorter TRB N(D)N nt lengths, along with lower diversity, while CD8+ T cells did not exhibit significant correlation with any TCR feature. Discussion This study highlights the distinct biological features of CD4⁺ and CD8⁺ TIL populations within the tumor microenvironment that can be preserved using a minimally expanded TIL approach. The observed associations between IHC patterns, T cell subset composition, cytotoxic potential, and TCR repertoire diversity help identify which biopsy regions yield TILs with greater therapeutic potential, thus providing guidance for TIL selection in immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c988d227329c53bf2d2653e81969a85f3a08fee" target='_blank'>
              Minimally expanded breast cancer tumor-infiltrating-lymphocytes provide guidance for therapeutic selection
              </a>
            </td>
          <td>
            A. Aran, Gonzalo Lázaro, Vicente Marco, V. Peg, Maitane Faus, L. Garrigós, J. Pérez-García, Javier Cortés, Mercè Martí
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099b7cbfc744841c3702cbfd3025a09c4f8cf0c1" target='_blank'>
              Targeting the HMGB1-IL32 pathway to alleviate T cell exhaustion in epithelial ovarian cancer.
              </a>
            </td>
          <td>
            Ankita Murmu, Balázs Győrffy
          </td>
          <td>2025-11-15</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15c2c938fad8c796a1fdbc96dde9e4476c7e8f1" target='_blank'>
              Distinct mechanisms of cell in breast cancer and their clinical implication
              </a>
            </td>
          <td>
            Aodi Li, Sishi Liu, Ying Xuan, Ying Liang, Lei Wang, Lei Zhao
          </td>
          <td>2025-12-09</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer brain metastasis (BCBM) is a major cause of poor prognosis in breast cancer, driven by complex molecular mechanisms. Innovative diagnostic and therapeutic strategies are urgently needed. We integrated single-cell RNA sequencing, multiomics profiling from the TCGA database, and machine learning to explore the molecular features of BCBM. Cell composition, gene expression, and subtypes were characterized. Prognostic models were developed, and potential drug targets were computationally identified through analysis of differentially expressed genes and molecular interactions. Experimental validation of these targets was performed using orthotopic implantation of MDA-MB-231-Luc cells in nude mice. scRNA-seq revealed 10 cell types and 1,479 differentially expressed genes, highlighting significant differences between the primary brain cancer and BCBM. Multiomics clustering defined two distinct subtypes (CS1 and CS2) with differential prognosis. A CoxBoost + RSF model identified hub genes (BTG2, PSMB8, SRGN, HLA-DPB1) and demonstrated high predictive accuracy for 3-, 5-, and 10-year survival (AUCs: 0.813, 0.788, and 0.776, respectively). Drug sensitivity analysis highlighted five candidate agents, with molecular docking confirming strong binding affinity to targeted proteins. In vivo experiments confirmed that PSMB8 and HLA-DPB1 promoted brain metastasis, while BTG2 and SRGN suppressed it. High-risk patients exhibited elevated monocyte proportions, which were involved in intercellular interactions. This study delineates the molecular landscape of BCBM, establishes robust prognostic models, and identifies promising therapeutic targets, offering a framework for precision diagnosis and individualized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54597c55097e9960947bcf191d32120feda6fc03" target='_blank'>
              Multiomics analysis and prognostic modeling reveal the molecular features and potential therapeutic targets of breast cancer brain metastasis
              </a>
            </td>
          <td>
            Da He, Guanyou Huang, Xin Jia, Yong Yu, Xuecheng Ge, Liangzhao Chu
          </td>
          <td>2025-12-31</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J. K. Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression is shaped by the tumor microenvironment, particularly tumor-associated macrophages (TAMs), which often adopt immunosuppressive functions. CD300e, a myeloid receptor involved in immune regulation, has an uncharacterized role in CRC. Methods Functional studies were conducted in azoxymethane/dextran sodium sulfate and MC38 murine CRC models using CD300e systemic and myeloid-specific CD300e knockout mice, and adoptive transfer experiments assessed macrophage-intrinsic effects. Human studies included analysis of CD300e expression in matched tumor and normal tissue from patients with CRC and in vitro co-culture of patient-derived colon tumor organoids with monocytes to study CD300e induction and TAM polarization. Results In vivo, CD300e deficiency led to reduced tumor burden, enhanced major histocompatibility complex expression on TAMs, and improved T-cell responses. CD300e-deficient macrophages exhibited increased phagocytic activity, antigen presentation, and support for T-cell proliferation and cytotoxicity. Adoptive transfer confirmed that macrophage-intrinsic CD300e expression is sufficient to suppress T-cell function and promote tumor growth. In patients with CRC, CD300e is selectively upregulated in tumor-infiltrating monocytes and macrophages, driving a suppressive phenotype marked by impaired antigen presentation. Tumor-derived signals in vitro induce CD300e expression and promote a protumorigenic macrophage profile. Conclusions Our findings identify CD300e as a critical regulator of macrophage-mediated immune suppression in CRC and a potential target for reprogramming TAMs to enhance immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c56c500ab6751223d28c46c0f7327dc521a283" target='_blank'>
              CD300e is a driver of the immunosuppressive tumor microenvironment and colorectal cancer progression via macrophage reprogramming
              </a>
            </td>
          <td>
            Annica Barizza, Stefania Vassallo, L. Masatti, Mattia Laffranchi, Sofia Giacometti, S. Lonardi, Mattia Bugatti, Sara Coletta, C. Della Bella, M. D’Elios, Simone Pizzini, A. Rosato, W. Vermi, Matteo Fassan, G. Spolverato, Silvano Sozzani, E. Calura, R. Sommaggio, Gaia Codolo
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Despite the emergence of new therapies such as, immunotherapy, the treatment and diagnosis of Hepatocellular Carcinoma (HCC) still face many challenges, and the therapeutic outcomes for patients remain less than satisfactory. The study identified differentially expressed genes in HCC through differential analysis and then used univariate cox regression analysis to identify genes associated with prognosis. The intersection of these two sets of genes was used to obtain differentially expressed prognostic genes in HCC, which were then subjected to enrichment analysis. We analyzed two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, comprising 24,637 cells. Non-negative Matrix Factorization (NMF) clustering was used to identify cell cycle regulation in HCC tumor microenvironment (TME) cells, including three cell subpopulations: proliferating cells (PC), dendritic cells (DC), and macrophages (MAC). We employed the CellChat package to analyze cell–cell communication, the Monocle package for pseudotime trajectory analysis, and the SCENIC software package to study gene regulatory networks. Survival analysis was also performed using cell cycle-related features. A total of 26 clusters, including 15 major cell types, were identified in the HCC samples. Complex cell–cell communication networks were observed among these cell types. Enrichment analysis revealed that these cells were mainly enriched in pathways related to the cell cycle. The expression of cell cycle-related genes was elevated in tumor samples, and changes in cell cycle-related genes in specific subtypes were associated with different overall survival rates. The study focused on single-cell level data analysis of the cell cycle. The bubble plot results showed that the cell cycle scores were significantly upregulated in the PC, DC, and MAC subpopulations. Further subtyping revealed that these subtypes exhibited distinct biological states, cell–cell communication, and metabolic pathways. This study demonstrates that cell cycle regulation and cell–cell communication within the HCC tumor microenvironment impact tumor progression and patient prognosis. Cell cycle dysregulation in TME cells correlates with poor prognosis and immunotherapy efficacy, suggesting cell cycle targeting as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3531834465d59ab5f37ade029201ae0f13d144" target='_blank'>
              Impact of single-cell cell cycle regulation of intercellular communication on the prognosis of hepatocellular carcinoma in the tumor microenvironment
              </a>
            </td>
          <td>
            Cong Hu, Rui Deng, Shuxiong Nong, Xinglang Mou
          </td>
          <td>2025-11-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tissues are shaped by extracellular signaling fields which convey information between cells. The cellular composition of tissues, and the extracellular signaling within the tissue, are innately spatially structured. Modern spatialomics data provide unprecedented measurement of ligand and receptor expressivity in situ from tissue sections. Here, we show that by adapting generalizable geospatial statistical models to spatialomics data, we are able to reveal statistically-detailed portraits of morphogenic field interactions within tissues and thereby approach a richer set of biologic questions than is typically pursued. The general methods piloted here can readily be applied to spatialomics data from diverse platforms with no need to alter data collection techniques. Our results demonstrate that the application of spatial statistical modeling to spatialomics data opens many avenues for future experimentation that will be valuable to fundamental biology and to regenerative medicine. Graphical Abstract Highlights Tissue biology & regenerative medicine requires analysis of tissue morphogen fields and morphogenic interactions Spatial statistics can be used to model continuous morphogenic interaction fields in tissues from discrete spatialomics data">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dab79968c0de2a95d7970f88d53167ba3435c46" target='_blank'>
              Spatial Modeling of Tissues for Morphogenic Field Analysis
              </a>
            </td>
          <td>
            Aaron Osgood-Zimmerman, M. Raredon
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly lethal malignancy with poor prognosis due to its resistance to conventional chemotherapy and limited number of targetable mutations. The recurrent loss-of-function mutation of the tumor suppressor ARID1A represents a key vulnerability potentially exploitable via synthetic lethality. To advance research in this area, we successfully established and characterized two novel CCA cell lines, SiSP-K01 and SiSP-K05, derived from moderately differentiated intrahepatic CCA. Both models were confirmed to carry ARID1A heterozygous loss and complete protein depletion, authentically reproducing the target genotype. Comprehensive molecular and phenotypic profiling (including karyotype, growth kinetics, 100% in vivo tumorigenesis, and tumor marker expression) confirmed their fidelity to the patient tissue and CCA lineage. Importantly, exome and transcriptome analysis not only validated the cell lines but also revealed an associated derangement of the PI3K/AKT signaling pathway. Based on this molecular finding, we performed drug screening and demonstrated that these ARID1A-deficient CCA cell lines are hypersensitive to inhibition by both PI3K/AKT and PARP inhibitors. The SiSP-K01 and SiSP-K05 cell lines are therefore critical new preclinical models suitable for functional studies of ARID1A deficiency and the development of targeted therapies for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0e9dfbc0a68397e20306d980742627a07920fa" target='_blank'>
              Establishment of novel cholangiocarcinoma cell lines with ARID1A deficiency and preclinical validation of synthetic lethality therapies
              </a>
            </td>
          <td>
            Sunisa Prasopporn, G. Sittithumcharee, Jantappapa Chanthercrob, Somchai Limsrichamrern, Arada Hirankitti, Pimkanya More-krong, Sakda Sathirareuangchai, A. Chaiboonchoe, Somponnat Sampattavanich, Seiji Okada, Siwanon Jirawatnotai
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Cancer remains a leading cause of death worldwide, often because tumors can evade the immune system and become resistant to treatments. In this study, we investigated a protein called HMGA1, which is known to be present in high amounts in many cancers but whose broader role was not fully understood. By analyzing data from thousands of patients across numerous cancer types, we discovered that HMGA1 is frequently overactive due to specific chemical changes on its DNA. High levels of this protein were linked to poorer patient survival. We found that HMGA1 helps tumors hide from the body’s immune defenses and makes them resistant to a common class of drugs known as AKT inhibitors. We confirmed this drug resistance in breast cancer cells. Our work identifies HMGA1 as a key driver of cancer aggression and suggests that measuring its levels could help predict patient outcomes and guide more effective treatment strategies, particularly for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e03334f80f9abed34dd7f9721e333bfb33a958" target='_blank'>
              Promoter Hypomethylation Unleashes HMGA1 to Orchestrate Immune Evasion and Therapy Resistance Across Cancers
              </a>
            </td>
          <td>
            Iram Shahzadi, T. Ahsan, Shoaib Anwaar, Wajid Zaman, Houjun Xia
          </td>
          <td>2025-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, J. Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Endometriosis involves ectopic growth of endometrial-like tissue, yet the spatial transcriptomic and metabolic landscape of ovarian endometriomas remains poorly understood. This investigation presents a comprehensive multi-omics analysis of ovarian endometriomas incorporating single-cell RNA sequencing in conjunction with Digital Spatial Profiler-Whole Transcriptome Atlas for spatial transcriptomics, and non-targeted Matrix-Assisted Laser Desorption/Ionization-Mass Spectrometry Imaging for spatially resolved metabolomics. Our integrated analysis confirms the importance of cell adhesion, ECM-receptor interaction, and focal adhesion pathways in disease context. We identify XBP1, VCAN, and CLDN7 as key markers in epithelial cells, and THBS1 in perivascular cells. Spatially resolved metabolomics further reveals altered activity of cytochrome P450 enzymes, lipoprotein particles, and cholesterol metabolism in mesenchymal regions, along with several undefined metabolites enriched in epithelial areas of endometriomas compared to ovarian cortex controls. These findings reveal potential markers and metabolic pathways linked to ovarian endometriomas, offering a foundation for future diagnostic and therapeutic strategies. Ovarian endometriomas, with distinct microenvironment and heightened hormonal sensitivity, are recognized as precursors of ovarian carcinomas. This study decodes ovarian endometriomas by integrating single-cell and spatial transcriptomics with spatial metabolomics to reveal key markers and altered pathways, offering new avenues for diagnosis and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b76d5485f1fbcc4c032549f15e895ea9152fd283" target='_blank'>
              Single-cell and spatially resolved omics reveal transcriptional and metabolic signatures of ovarian endometriomas
              </a>
            </td>
          <td>
            Yujuan Qi, Xia Chen, Sen Zheng, Tiantian Wu, Zhenbei Li, Jie Cheng, Xinhui Yang, Wei Tao, Qiuru Huang, Juan Gu, Qingqing Sun, Ning Chen, Xiangnan Cao, Jiaxin Li, Lei He, Chenyu Wang, Xinda Wang, Qingqing Hu, Qiushi Xia, Yi Zhang, Jiangming Reng, Weiyi Qian, Lingyi Kong, Yuqi Huang, Yanting Wang, Chen Qiao, Xinyuan Zhao, Ying Zheng, Mei Xu, Bo Zheng, Yijuan Cao, Jun Yu
          </td>
          <td>2025-11-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Computational pathology enables the automatic tissue analysis of Whole Slide Images (WSIs), offering unmatched possibilities to capture quantitative tumor microenvironment (TME) characteristics that are essential for patients' prognosis and therapy response. The existing clinical and digital biomarkers do not encompass the morphometric features and spatial interactions between vascular networks and immunological compartment in the TME. To address this challenge, this work presents a high- throughput quantitative framework for automatic segmentation and assessment of aberrant phenotypes of blood vessels and immune cell clusters in hematoxylin & eosin-stained WSIs, to construct the Vascular-Immune Pathomic (VIPath) biomarker. For our study, we utilized three public datasets of Colon Adenocarcinoma (COAD) and Stomach Adenocarcinoma (STAD) from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) projects: TCGA-COAD, TCGA-STAD, and CPTAC-COAD. Additionally, we collected two in-house gastric cancer cohorts of 80 and 51 patients : DBGC-M0 and DBGC-M1. The VIPath biomarker was incorporated in a Cox proportional hazards model trained on the TCGA-COAD. Then, it was validated for predicting Overall Survival (OS) in TCGA-STAD, DBGC-M0, and DBGC-M1, Disease Free Survival in CPTAC-COAD and second-line therapy Progression-free Survival (PFS-2) in DBGC-M1. VIPath encompasses features from both vascular and immunological compartments interacting in the TME. Results proved that VIPath was capable to significantly stratify risk groups for OS TCGA-STAD (p=0.018), OS DBGC-M0 (p=0.029), OS DBGC-M1 (p=0.014), and PFS-2 DBGC-M1 (p$< $0.005). Furthermore, when inserted in a Cox model, it led to an improvement of C-index and R2 over all other considered prognostic factors, i.e., p-TNM, ECOG, MSI, HER2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f526aa6f7bc4b1f2859a4decf8cee5c58634b7e" target='_blank'>
              Automated Pathomic Analysis of Angiogenesis and Immune Profiles Unveils an Interpretable Prognostic Biomarker in Colon and Gastric Cancers.
              </a>
            </td>
          <td>
            Michela Prunella, Nicola Altini, Rosalba D'Alessandro, Annalisa Schirizzi, Giampiero De Leonardis, Graziana Arborea, M. Savino, Anna Maria Valentini, R. Armentano, Angela Dalia Ricci, Claudio Lotesoriere, Raffaele Carli, M. Dotoli, Gianluigi Giannelli, Vitoantonio Bevilacqua
          </td>
          <td>2025-11-24</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Lung cancer is one of the most prevalent malignant tumors with high morbidity and mortality rates worldwide. Extensive multi-omics analyses have revealed significant intratumoral heterogeneity even within the same histopathological subtype. However, a database that systematically integrates multi-omics data for lung cancer research has long been lacking. Here, we developed LOSTdb, a molecular subtype annotation system for lung cancer that integrates multi-omics data and metadata. LOSTdb comprises 295 multi-omics datasets, including bulk RNA-seq, genomic, proteomic, methylation, and scRNA-seq data, with over 10,000 manually curated metadata entries. This resource encompasses high-quality clinical specimens, mouse models, and cell lines, totaling 34,393 samples and more than 1.2 million single cells. Each omics sample was annotated with both literature-based classical subtypes and NMF-derived meta-program (MP) subtypes. The platform supports cross-searching of omics and metadata at the gene and dataset levels, offers multiple visualization and analysis methods, and includes five tool modules, enabling functions such as integrated analysis, significance analysis between metadata as well as between genes and metadata, and target prediction for lung cancer molecular subtypes, serving as an essential tool for lung cancer precision medicine. LOSTdb is a user-friendly interactive database freely accessible at http://lostdbcancer.com:8080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d313178158ab200d4da77fbc83c8812b7dd83e" target='_blank'>
              LOSTdb: a manually curated multi-omics database for lung cancer research
              </a>
            </td>
          <td>
            Hao Luo, Yunhao Yang, Zhipeng Gong, Lunxu Liu, Yaohui Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a highly heterogeneous disease with diverse biological behaviors and clinical outcomes, which complicates diagnosis and treatment strategies. Recent research has demonstrated that LUAD can originate from multiple alveolar cell types, including GRAMD2+ alveolar type I (AT1) cells and SFTPC+ alveolar type II (AT2) cells, each giving rise to tumors with distinct histological features and transcriptomic profiles. Activation of the KRASG12D oncogene in either AT1 or AT2 cells leads to the development of multifocal LUAD, but it is not fully understood how the cell of origin affects tumor characteristics observable through imaging techniques. In this study, we investigated the influence of cell origin on radiomic signatures by comparing LUAD models derived from SFTPC+ AT2 cells (SKTG) and GRAMD2+ AT1 cells (GKTG). Using fast spin echo (FSE) MRI sequences, we identified distinct and cell-of-origin–specific radiomic profiles, with the SKTG tumors showing a greater number of significantly (p < 0.05) different features, indicating unique imaging phenotypes between the two tumor types. Notably, SKTG tumors exhibited increasing radiomic heterogeneity over time, which may correlate with a more aggressive tumor behavior and progression compared to GKTG tumors. These findings underscore the diagnostic and prognostic potential of radiomics to non-invasively differentiate LUAD lesions based on their cellular origin. By capturing tumor heterogeneity and progression longitudinally through imaging, radiomic analysis offers promising avenues for personalized treatment planning and improved patient management tailored to AT1- versus AT2-derived LUAD. This approach may ultimately enhance early detection, therapeutic response monitoring, and clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a0a3915571fd6a837e7c2df7c6727de519df23" target='_blank'>
              Multimodal Radiomic Profiling Can Distinguish AT1- and AT2- Cell of Origin Lung Adenocarcinoma Models
              </a>
            </td>
          <td>
            Sophia Hai, Minxiao Yang, Xiaomeng Lei, Dighvijay Giri, S. Cen, Crystal Marconett, B. Varghese
          </td>
          <td>2025-11-18</td>
          <td>2025 21st International Symposium on Biomedical Image Processing and Analysis (SIPAIM)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Objective This study aimed to investigate cell-type-specific RNA methylation patterns in primary breast cancers and metastatic lymph nodes and explore their roles in cancer progression and immune regulation. Methods Using single-cell RNA sequencing data from five breast cancer patients (primary cancers and matched metastatic lymph nodes), we calculated RNA methylation scores per cell using the AUCell method. Cell types were annotated using Seurat. Differentially expressed gene analysis, pseudotime trajectory assessment, copy number variation analysis, and cell–cell communication analyses were performed between high- and low-methylation groups. Validation was conducted using bulk RNA sequencing data from The Cancer Genome Atlas. Results Metastatic lymph nodes showed significantly lower RNA methylation scores than primary cancers, particularly in T cells and B cells. Low methylation was correlated with enhanced cell differentiation, increased malignancy in epithelial cells, and reduced immune cell communication. Monocytes exhibited opposite trends. Differentially expressed genes from methylation groups stratified The Cancer Genome Atlas patients into two clusters with distinct survival and immune infiltration profiles. Conclusions RNA methylation is suppressed during breast cancer metastasis, affecting cell differentiation, immune communication, and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a57d08895a4d132dadd9681e650758bc323c10" target='_blank'>
              Single-cell analysis reveal cell-type-specific RNA methylation patterns in breast cancer
              </a>
            </td>
          <td>
            Shengtao Huang, Yichen Lin, Xiaobin Liu, Kun Zhang, Qiuju Han, Ruilei Liu, Yong Zhuang
          </td>
          <td>2025-12-01</td>
          <td>The Journal of International Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Lung cancer symptoms typically do not appear until advanced stages, leading to late diagnoses. This delay is a major contributor to its poor prognosis, resulting in lung cancer being the leading cause of all cancer-related deaths worldwide. Essential biological understanding of how lung cancers arise and progress is still lacking. Studying premalignant lesions that eventually develop into invasive carcinomas helps to bridge this knowledge gap. We are focusing on adenocarcinomas (ACs), the most common subtype of lung cancer, and especially, activating mutations in the Kirsten rat sarcoma virus oncogene (KRAS), which is among the most common oncogenic mutations and has previously been reported in premalignant lung lesions. Given that KRAS signaling is growth-promoting, it is reasonable to assume that KRAS mut promotes tumor formation and progression. However, the extent of its influence, the timing of when it arises, and particularly its connection to histologic and phenotypic changes driving tumor evolution, remain unclear.



 We have collected a cohort from biopsy material, initially collected for diagnostic purposes from AC patients at varying stages of tumor evolution, with some matched ‘normal-histology’ samples when available. In total, tissue from 38 patients with AC premalignancies is available for investigation. To map KRAS mutations in situ, we employed a BaseScope assay with commercial detection probes against the different KRAS mut transcripts. To map KRAS statuses to cellular phenotypes in the microenvironment, we are running the 6K discovery panel on the CosMx platform (Bruker) on a subset of patients, which is a highly sensitive spatial transcriptomics technique at single-cell resolution. Spatial Mass Spectrometry (MS) tracking metabolic features on the timsTOF flex MALDI-2 (Bruker) is also being performed on a subset of patients.



 Initial tests of the BaseScope assay have revealed specific, yet not sensitive, signals of KRAS mut transcripts in premalignant lesions. While advanced cancers display distinct signals, premalignant lesions have fewer, if any, signals. This might be due to the lower expression levels of KRAS at the earlier stages. To address this, we are now applying a BaseScope assay with signal amplification to be able to track KRAS expressed at lower levels. Preliminary analyses of the Spatial MS matrix have revealed highly individual metabolomics features, with great inter-patient heterogeneity. This pattern is also reflected in bulk RNA-seq data, revealing expression profiles more similar across, e.g., Adenocarcinoma in Situ (AiS) and invasive AC from the same patient, than between the AiSs from different patients.



 By applying this state-of-the-art multi-omics approach to premalignant adenocarcinomas, we hope to identify key regulators of early lung cancer progression and to translate these findings to clinically relevant basic molecular diagnostics.



 Amanda Lindberg, Bartosz Sobocki, Patrick Micke, Carina Strell. In situ KRAS-mapping and spatial-omics characterization in lung premalignancy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b558a4354aae9f9807afda6b85aec2e424e2cc" target='_blank'>
              Abstract A002:
 In situ
 KRAS-mapping and spatial-omics characterization in lung premalignancy
              </a>
            </td>
          <td>
            A. Lindberg, B. Sobocki, P. Micke, C. Strell
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b536ff0acd69dbca3af6bd3122c752115f69d4a" target='_blank'>
              Single-Cell Transcriptomic Profiling of GL261 Glioblastoma Cells Reveals Gene Expression Signatures Underlying Tumorigenicity.
              </a>
            </td>
          <td>
            Colton E Troxel, Ruby A Olvera, Emily A Freko, Suely S C Soeiro, Thandiswa Mdluli, Richard L Daniels
          </td>
          <td>2025-12-16</td>
          <td>Cellular and molecular neurobiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a highly heterogeneous disease, and about 15–30% of cases are driven by the HER2 receptor, a protein that can be specifically targeted by modern therapies. Traditionally, cancer cells are grown in flat layers on plastic, but this does not reproduce the complexity of a real tumor. Three-dimensional (3D) culture systems, where cells grow as spherical clusters called “spheroids,” provide a more realistic environment that mimics how cells interact in the body. In this study, we generated 3D spheroids from two commonly used HER2-positive breast cancer cell lines, SKBR3 and BT474. Although both cell types formed viable spheroids, their shapes and internal organization were very different: SKBR3 spheroids were loose and irregular, while BT474 spheroids were compact and highly spherical. We examined how HER2 was distributed within these structures and found that its signal decreased toward the spheroid core, especially in BT474. We also analyzed signaling proteins and cellular structures, discovering differences in cell–cell adhesion, mitochondrial features, and key molecular pathways. Overall, our results show that the 3D architecture of breast cancer spheroids strongly influences how cells organize, signal, and maintain HER2 expression. This system provides a powerful tool to study drug responses and to improve strategies for personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d155c78e81cc0e3e7e461d59374a103c979710df" target='_blank'>
              3D Breast Cancer Spheroids Reveal Architecture-Dependent HER2 Expression and Signaling
              </a>
            </td>
          <td>
            Pietro Arnaldi, Valentina Delli Zotti, G. Bellese, M. Gagliani, Paola Orecchia, P. Castagnola, Katia Cortese
          </td>
          <td>2025-11-24</td>
          <td>Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tissue-resident memory T (TRM) cells, memory T cells that stably occupy tissues and contribute to immunosurveillance, induce favorable survival outcomes in solid tumors. While TRM cells have been observed in lymph nodes, their phenotype and prognostic significance in diffuse large B-cell lymphoma (DLBCL) remains uncharacterized. In this study, CD103+ T cells were quantified in DLBCL samples by immunofluorescence (IF) staining of tissue biopsies and flow cytometry of cell disaggregates, and linked with clinical outcomes. Across two patient cohorts, CD103+ T cells were identified in both nodal and extranodal DLBCL, and higher CD103+ T cell levels correlated with superior clinical outcomes. Single-cell RNA sequencing (scRNAseq) revealed ITGAE expressing T cells in both malignant and reactive lymph node (rLN) samples. However, transcriptional profiles differed, as a canonical TRM population was observed in the malignant setting. This TRM cluster was enriched for genes associated with cytotoxicity and activation and was validated in an external CITEseq dataset. Flow cytometry additionally confirmed protein expression of TRM markers (CXCR6, CD39, and PD-1) on CD69+CD103+ T cells. We assessed functional activity in co-culture experiments of CD103+ versus CD103- T cells with autologous CD20+ B cells, where CD103+ T cells displayed enhanced killing. CD103+ TRM cells in DLBCL represent a prognostically favorable population with an activated/cytotoxic T cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75e4cde340cefbbda9de07c5cdbbe7ec6ec4213" target='_blank'>
              Phenotypic characterization and prognostic impact of CD103+ tissue-resident memory T cells in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Gillian Savage, K. Milne, B. Collinge, Yifan Yin, Pauline Loos, Mary E Warren, Céline M. Laumont, Danielle Keith, Brad H Nelson, Leandro Venturutti, Andrew P. Weng, D. Scott, Christian Steidl, L. Evgin
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="

 Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood.




 We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with
 in vitro
 and
 in vivo
 experiments, was employed to identify and functionally validate key regulatory genes.





 Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity
 in vitro
 and
 in vivo
 .




 This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 A crucial adaptability trait of melanoma cells that persist under all selection pressures, including therapies, is their ability to efficiently switch from proliferation to quiescence. With the goal of improving therapeutic strategies for preventing melanoma recurrence, herein we describe an approach to modeling this trait in cell culture. Body imposes a variety of selection pressures and bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. The rationale for the choice of model as cell culture is that it is ideal for modeling the realistic depth of quiescence and for evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cancer cells. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. A lack of glutamine proved to be a strong and prolonged selection pressure for the highly metastatic human melanoma cell line A375SM, killing more than 99% of cells and selecting rare cells based on their ability to survive in deep quiescence. After 4 weeks, cells gradually exited quiescence and proliferated indefinitely. Furthermore, we modeled even deeper quiescence lasting longer than 4 weeks by increasing the severity of the selection pressure using dialyzed serum in medium. The cells selected in this manner were much more resistant to paclitaxel than was their parental cell line. We obtained similar results with the highly metastatic mouse melanoma cell line B16-BL6. We conclude that our approach is suitable for modeling abnormal deep quiescence in melanoma that drives the disease, including recurrence and metastasis. The main strength of the approach is that, besides modeling genetic alterations, it incorporates suitable selection pressures and the time element that are important in cancer evolution. Although there is a limitation of lack of body-like microenvironments in cell culture, we believe that abnormal quiescent cancer cell state being modeled here is capable of overcoming restraints imposed by the body. We will also discuss our published and new observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive triple-negative breast cancer cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line, modeled as above for melanoma cells, suggest that they are stem-like quiescent cells. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells prior to recurrence. This approach may significantly improve how we investigate deep intrinsic resistance and strategies to overcome it.



 Balraj Singh, Vanessa N. Sarli, Nikil Erry, Anthony Lucci. A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4de6eb92fbb6a26e62be45c5389b5a53529d5" target='_blank'>
              Abstract A010: A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis
              </a>
            </td>
          <td>
            Balraj Singh, V. Sarli, Nikil Erry, Anthony Lucci
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362676e5f313e967da9a02b557ed5fbcda507b0" target='_blank'>
              Single-cell and spatial transcriptome-based metabolism-immunity interaction network and therapeutic target discovery of matrine in cervical cancer.
              </a>
            </td>
          <td>
            Hong Zou, Wang Xiong, Yumei He
          </td>
          <td>2025-11-24</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is associated with an increased risk of cutaneous squamous cell carcinoma (CSCC). A comprehensive understanding of the cellular heterogeneity of HPV-positive and -negative CSCC is crucial for improving diagnosis and preventing tumor progression. We conducted an integrated analysis of single-cell RNA and spatial transcriptomic data from different skin tissue sources to map the cellular landscape of the tumor microenvironment (TME) in both HPV positive and negative CSCC. Results were validated through multiplex immunohistochemistry (mIHC) and in vitro experiments. We identified 10 major cell types in CSCC and normal skin samples, including epithelial cells, myeloid cells, T cells, fibroblasts, endothelial cells, B cells, smooth muscle cells, mast cells, melanocytes, and hair follicle cells. Notably, fibroblasts were found to be associated with tumor progression in CSCC with or without HPV infected. We further identified eight major CAF subtypes in CSCC, with iCAFs-CXCL2 promoting tumor progression, while iCAFs-PLA2G2A acted to suppress tumor growth. The MDK-ITGA6 pair was found to mediate interactions between fibroblasts and epithelial cells in CSCC. mIHC analysis confirmed elevated expression of MDK and ITGA6 in CSCC samples. Additionally, cell co-culture experiments confirmed that MDK-mediated CAFs were shown to enhance tumor cell migration and invasion in CSCC. Our findings provide a comprehensive cellular atlas of CSCC, highlighting the association of CAFs in HPV infection and tumor progression of CSCC. These results also offer potential diagnostic and prognostic biomarkers for CSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65aef27a5d8cbdec4a99704bf580728fefb52e8e" target='_blank'>
              Multi-omics analysis unveils the role of cancer-associated fibroblasts in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Wang, Tingrui Li, Yichao Jin, Jingjing Chen, XinYang, Zhen Guan, Mei Jin, Jingxian Zhang, Liangheng Xu, Sizhen Tao, Chunguang Li, Chunping Ao
          </td>
          <td>2025-11-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Why and how does cancer start? Building from a Symposium at the 2025 Society of Toxicology meeting, we convened a group of international experts to answer this seemingly simple question. As experimental evidence has evolved, perspectives on cancers' origins have shifted from the accumulation of DNA mutations in single cells to complex processes involving signals from an altered tissue microenvironment which promote tumorigenesis. Carcinogen exposures impact the biology of the microenvironment in complex and tissue-specific ways. These changes can include the infiltration of inflammatory cells that produce growth factors, neo-angiogenesis, morphological changes, and immune tolerance that avoids immune-mediated elimination. In this in-depth review, we discuss the evidence linking chemical-driven microenvironmental changes in the development of a range of solid and liquid tumors. We discuss specific phenotypic alterations, such as selection pressure driving clonal expansion and cellular plasticity and reacquisition of stem cell states, linked to carcinogen-induced changes in the microenvironment. We describe assays and biomarkers which can allow us to experimentally assess links between chemical exposures, the microenvironment, and cancer phenotypes. We end by discussing how understanding the role of the microenvironment and malignancy in toxicology is essential for accurate cancer hazard evaluation, development of next-generation risk assessment frameworks, identifying new strategies for cancer prevention, and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64691a327a7b8227d277fcb6bbfe9d59f21fba7" target='_blank'>
              Rethinking the Microenvironment's Role in Chemical-Induced Malignancy.
              </a>
            </td>
          <td>
            William H. Bisson, Richard A Currie, Emilia L Lim, Coraline Mlynarczyk, Erik J Tokar, Annamaria Colacci, Justin A. Colacino
          </td>
          <td>2025-12-17</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Adamantinomatous craniopharyngioma (ACP), a benign yet highly recurrent and therapy-resistant intracranial tumor, remains a considerable clinical challenge because of its complex pathological structure, infiltrative growth, and limited treatment options. Here, integrated spatially resolved multiomics is employed-including single-cell spatial transcriptomics via CosMx SMI and spatially resolved metabolomics via AFADESI-MSI, accompanied by bulk metabolomics and functional validation-to unravel the driving factors of ACP progression and recurrence. Analysis results reveal three interdependent biological hallmarks: first, the spatial segregation and molecular heterogeneity of 10 distinct tumor epithelial cell subpopulations within the ACP, each of which presents unique transcriptional signatures; second, in tumor regions and recurrent tumor epithelium tissues, stronger transporter-mediated choline/ethanolamine uptake from cystic fluid and significant upregulation of phosphatidylcholine (PC) and phosphatidylethanolamine (PE) synthesis is observed, creating the enhanced "cystic fluid-tumor cell" and "choline/ethanolamine-PC/PE" metabolic axis, and demonstrating the spatial metabolic remodeling of ACP; and third, this metabolic axis directly couples to autophagy activation of corresponding regions in ACP tissue, which is validated by multi-immunohistochemistry for Beclin1 and GABARAP. Together, these findings reveal metabolic remodeling and autophagic activation as critical drivers of ACP progression and recurrence and provide an opportunity for precise biomarker-driven treatment of this intractable tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4bf376edd94fbdf6f86d786c026d6877b062546" target='_blank'>
              Spatially Resolved Multiomics Reveals Metabolic Remodeling and Autophagy Activation in Adamantinomatous Craniopharyngiomas.
              </a>
            </td>
          <td>
            Dongting Chen, Yahui Gao, Yulin Wang, Ting Lei, Zheng Qu, Yuhan An, Jiaxu Fu, Xin Li, Fangjun Liu, Yan Li
          </td>
          <td>2026-01-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene-specific expansion patterns were evident among the most frequent CH lesions, with DNMT3A-mutant clones exhibiting impaired hematopoietic differentiation and TET2-mutant clones showing multi-lineage propagation. Notably, identical CH clones were detected in 41% of corresponding lymphomas, displaying distinct clonal dynamics: tumor-promoting CH (expansion in B-NHL; 10/16 clones; mainly TP53) and tumor-infiltrating CH (no expansion; mainly DNMT3A). Moreover, we identified lymphoma-associated mutations in flow-sorted hematopoietic progenitors from patients with indolent but not aggressive B-NHL and observed a stepwise accumulation of mutations along the lymphoid differentiation path. Single-cell genotyping confirmed the presence of mutated progenitors in 3 follicular, 2 mantle cell and 2 marginal zone lymphoma patients, providing direct evidence of a pre-neoplastic state in disease pathogenesis. Our findings offer novel insight into the cellular origin of nodal B-NHLs and highlight a previously underappreciated role for early clonal events involving the stem/progenitor cell compartment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec5b65b719f8d9e5db3ee2069637056c360c3b" target='_blank'>
              Clonal Hematopoiesis and Lymphoma-Associated Mutations in Hematopoietic Progenitors in B-Cell Non-Hodgkin Lymphoma.
              </a>
            </td>
          <td>
            Laura Wiegand, Patricia Silva, Daniel Noerenberg, F. Christen, Klara Kopp, B. Locher, P. Löwe, Marlon Tilgner, Robert Altwasser, Vanessa Storzer, Catarina M Stein, F. Briest, C. M. Arends, Mareike Frick, J. Ihlow, A. Dolnik, N. Ishaque, U. Keller, Il-Kang Na, L. Penter, Lars Bullinger, Raphael Hablesreiter, Frederik Damm
          </td>
          <td>2026-01-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) with highly expressed secreted phosphoprotein 1 (SPP1) carry immunosuppressive property as a potential target for tumor metastasis. However, the mechanisms regulating SPP1 + TAMs in head and neck squamous cell carcinoma (HNSCC) remain poorly understood. This study employs a combination of single-cell and bulk RNA sequencing bioinformatics analysis to confirm the impact of TAMs with high levels of SPP1 on patient prognosis. Additionally, Key genes linked to SPP1 + macrophages were identified using weighted gene co-expression network analysis. A prognostic model was built using the Random Forest algorithm. Here we show, P4HA1 is strongly correlated with SPP1 + macrophages and holds significant value in predicting patient prognosis and diagnosis. In vitro experiments demonstrated that TAMs educated by HNSCC cells with knockdown P4HA1 expressed lower SPP1 level compared to the control group. Furthermore, Gene Set Variation Analysis and Gene Set Enrichment Analysis indicated that P4HA1 mediates the hypoxia pathway in HNSCC. In xenografts model, P4HA1 knockdown effectively suppressed tumor malignant progress, confirming that P4HA1 was positive correlation with SPP1 + TAMs and could mediate tumor hypoxia pathways. Overall, this study identified P4HA1 as a key gene involved in regulating SPP1 + TAMs through modulating hypoxia, providing a potential macrophage-centered therapeutic target in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a63e2de7b9c3162e04da7d6e387827ec9ac28c" target='_blank'>
              P4HA1 facilitates SPP1+ tumor-associated macrophages by activating the hypoxia pathway in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yiding Liu, Bing Yan, Hui Dong, Yi Li, Yuying Zhang, Yi Wang, Ni Kou, Lu Gao
          </td>
          <td>2025-12-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with an increasing incidence correlated with age and diverse subtypes exhibiting distinct prognoses. The tumor microenvironment (TME) in BC is complicated. It is now believed that BC may acquire invasive characteristics and even extra proliferative ability from the TME through various mechanisms. However, most studies predominantly focus on the heterogeneity of tumor cells in BC, lacking a comprehensive depiction of intercellular communication within BC. Therefore, the present study aimed to elucidate cellular communication in the TME by integrated bioinformatic analysis of bulk mRNA and single-cell mRNA sequencing, combined with certain validation in clinical samples. We first utilized single-cell sequencing data from GSE176078 to find out the most important cell communication pairs for the tumor microenvironment in BC then we conducted bulk-sequencing analysis to identify the differential expressed genes. Through correlation analysis, we sort out the top five most relevant genes to the most important cell communication pairs. We then validated the expression of the key genes of the aforementioned cell communication pairs and the five differentially expressed genes by qPCR on clinical samples. Furthermore, we analyzed the immunological relevance of these genes via a novel approach at single-cell resolution. The results of single-cell analysis indicated that the CXC12-CXCR4 ligand-receptor pair in the CXCL pathway and the FGF7-FGFR1 ligand-receptor pair in the FGF pathway are the most important cell communication pairs of the TME in BC. Subsequent bulk sequencing analysis showed that CHRDL1, SCARA5, LYVE1, PI16, and SAA2 were the most important differentially expressed genes linked to these cell communication pairs. In addition, we validated the expression of the key genes of the two cell communication pairs and the five genes in clinical samples, observing that the trends fitted the computational results. Finally, we studied the association of these genes with immune cell infiltration at single-cell level and had it cross-validated in bulk sequencing data, finding out that there were significant connections. In the tumor microenvironment of breast cancer, intercellular communication pairs of different cell types and molecules can exacerbate the development of breast cancer. among them, through the present study, we found that CXCL12-CXCR4 and FGF7-FGFR1 are the most important. Also, most significantly differentially expressed genes including CHRDL1, SCARA5, LYVE1, PI16, and SAA2 seemed to play a critical role in these mechanisms and immune cell infiltration, shaping the TME of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2625cd3b8acf01f328a363b0abd8247b6fa35d" target='_blank'>
              Integrated bioinformatics analysis to elucidate cellular communication in the microenvironment of breast cancer
              </a>
            </td>
          <td>
            Man Wang, Hua Li, Xingong Zhang, Zhijian Zha, Mengjia Zhao, Qi Wang, Danfeng Zhang
          </td>
          <td>2025-11-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND: The spatial organization of immune cells within tumors is an important area of investigation in studies of interactions between tumor cells and the tumor microenvironment. This is particularly relevant because some immune cells function through direct contact with their target cells, whereas others communicate over a distance via paracrine signaling involving cytokines. Thus, the topography of tumor cells and microenvironmental cells may determine the possibility and nature of intercellular interactions and thereby influence the functional effects of immune cells. AIM: This study aimed to compare the spatial transcriptomic profiles of tumor and stromal regions enriched in macrophages in triple-negative breast cancer. METHODS: Eight patients with triple-negative breast cancer were included. Spatial transcriptomic analysis was performed on formalin-fixed, paraffin-embedded tissue sections using high-throughput RNA sequencing with the 10x Visium platform. The annotated spots enriched in intraepithelial and stromal macrophages were used for downstream bioinformatic analysis. RESULTS: A total of 437 differentially expressed genes were identified between the two groups of spots containing macrophages with distinct spatial localization. Spots with intraepithelial macrophages were characterized by activation of processes related to cytokine and chemokine signaling, regulation of regulatory T-cell differentiation, organization of cell – cell contacts, wound healing, and inhibition of viral activity. Spots enriched in stromal macrophages demonstrated activation of biological processes associated with the regulation of angiogenesis, cell migration and recruitment, cell adhesion, and stromal remodeling. CONCLUSION: Macrophage topography within primary tumors of triple-negative breast cancer is associated with their functional characteristics. These fundamental findings may be useful for developing prognostic criteria and therapeutic approaches aimed at modulating the tumor microenvironment to improve long-term outcomes in patients with triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1996d432009ab729b97a00ec19d18afda8c70949" target='_blank'>
              Spatial Characterization of Macrophage-Enriched Tumor Regions in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Patskan, A. Kalinchuk, Elisaveta Tsarenkova, Evgeniia Grigoryeva, I. Larionova, Natalia Popova, L. Tashireva
          </td>
          <td>2025-12-02</td>
          <td>Genes &amp; Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background High endothelial venules (HEVs) are specialized blood vessels found primarily in lymphoid tissues, notably at the cortico-paracortical junction of lymph nodes (LNs). They play a key role in lymphocyte trafficking, facilitating the entry of lymphocytes into peripheral lymphoid organs. Objective Recent evidence indicates the presence of HEVs within tumor tissues, where they form part of tertiary lymphoid structures. This review aims to explore the structural and functional alterations of HEVs in tumors and LNs, and investigate their role in cancer prognosis and response to immunotherapy. Methods We conducted a comprehensive review of recent studies examining HEVs in tumors and LNs, with a focus on their involvement in immune responses and cancer progression. Additionally, the use of MECA-79 as a specific HEV marker was considered as a potential therapeutic target. Results HEVs within tumors have been associated with improved immune surveillance and better prognosis in certain cancers. The presence of HEVs in tumors correlates with enhanced immune cell infiltration and may influence the effectiveness of immunotherapy treatments. Targeting HEVs, particularly through MECA-79, holds promise as a novel therapeutic strategy to modulate tumor immunity. Conclusion Targeting HEVs in tumor tissues represents a promising avenue for cancer therapy. Further research is needed to optimize strategies for modulating HEV function and to better understand their role in immune responses to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701871a3989e04068f26894b4756f3bd07c359ea" target='_blank'>
              Targeting high endothelial venules: potential strategies for cancer treatment
              </a>
            </td>
          <td>
            Weichang Yang, Guofeng Zhu, Shanshan Cai, Hongquan Xing, Aoyu Yang, Xiaoqun Ye
          </td>
          <td>2025-12-01</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5da205fa89f01ecfcaf3e7f0ca0c06e5fa11fb" target='_blank'>
              Cellular turnover can increase or decrease the mutant burden in expanding cell population
              </a>
            </td>
          <td>
            S. S. Mondal, N. L. Komarova, Dominik Wodarz
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25364dc0260b8fa11f45e2ccc5d9dc6f7b06b29" target='_blank'>
              SLC2A1+ tumour-associated macrophages spatially control CD8+ T cell function and drive resistance to immunotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Lei Wang, Han Chu, Degao Chen, Yuxuan Wei, Jia Jia, Liqi Li, Linfeng He, Lina Peng, Fangfang Liu, Shanshan Huang, Zheng Jin, Dong Zhou, WenFeng Fang, Tao Jiang, Shouxia Xu, Xiaofang Ding, Haoyang Cai, Xin-dong Liu, Q. Jia, Bo Zhu, Qian Chu
          </td>
          <td>2026-01-07</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver carcinoma with high lethality. Both of hepatitis B virus (HBV) and Clonorchis sinensis (C. sinensis) are critical infectious contributors to HCC development. However, the inter-tumor heterogeneity and tumor microenvironment (TME) of HCC patients with different infectious background remain largely unknown. We compiled a cohort of 269 primary HCC patients to assess the clinical impact of C. sinensis and HBV infections on patient prognosis. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST-seq) analyses were performed on tumor and adjacent normal samples from C. sinensis-associated HCC (CP), and double-infection HCC (DP) patients. Additionally, we integrated publicly available scRNA-seq and ST-seq datasets from HBV-associated (HP) patients. Immunofluorescence, immunohistochemistry and in vitro experiments were conducted to validate inter-tumor heterogeneity among the three HCC subtypes. C. sinensis infection is significantly associated with poorer prognosis in HCC patients. Multi-omics analyses revealed distinct inter-tumor heterogeneity in epithelial, immune, and stromal compartments across different HCC subtypes. Tumor cells in the DP group exhibited more malignant marker expression, higher copy number variation scores, increased activation of p53 pathway, and worse survival outcomes. Compared with other HCC subtypes, the TME in DP samples was enriched with SPP1+ macrophages, exhausted CD8+ T cells and COL1A1+ fibroblasts. In contrast, the CP and HP groups showed higher proportions of M2-like macrophages and ENPP2+ liver vascular endothelial cells, respectively. These findings decipher the cellular signatures and their interactions within the TME, shedding light on the inter-tumoral heterogeneity driven by different infections, and the development of targeted therapies for infectious HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3fb4e883deefe93c32ac0665db0506da077e80" target='_blank'>
              Integrating single cell- and spatial- resolved transcriptomics unravels the inter-tumor heterogeneity and immunosuppressive landscape in HBV- and Clonorchis sinensis-associated hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiayun Chen, Wenmin Lu, Yanni Lou, Jing Liu, Xiwen Liao, Yunmeng Bai, Guangqing Cheng, Guangzhi Zhu, Ji Feng, Junqi Liu, Zhaoji Liu, Liqun Jia, Jing Zhou, Tao Peng, Guo-Dong Lu, Jigang Wang
          </td>
          <td>2026-01-05</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The lung is a structurally and immunologically complex organ, constantly exposed to airborne microbes, allergens, and pollutants. Understanding how diverse pulmonary immune cells respond to these challenges is critical for advancing respiratory disease research and identifying appropriate therapeutic interventions. Flow cytometry remains a cornerstone of immune profiling, and advances in high‐parameter spectral cytometry have significantly expanded its analytical capabilities. However, challenges, such as poor tissue dissociation, spectral overlap, and loss of spatial information, can hinder the comprehensive interpretation of the lung environment. To address these limitations, we developed an optimized spectral flow cytometry platform, utilizing 5‐laser Cytek Aurora spectral cytometers, for deep immunophenotyping of murine lung tissue. Our approach integrates in vivo CD45 antibody labeling—administered intravenously and oropharyngeally—to distinguish circulating, airway, and interstitial immune populations, preserving spatial context in single‐cell suspensions. We utilize complementary 25+ parameter panels targeting myeloid and lymphoid compartments, built on a shared backbone to enable consistent classification across datasets. Refined tissue processing protocols support optimal recovery of representative lung cell populations, and overnight intracellular staining enhances marker resolution. Using this platform, we reliably resolve stromal, endothelial, and epithelial cells alongside immune subsets—including macrophages, monocytes, dendritic cells, eosinophils, neutrophils, T and B cells, innate lymphoid cells (ILCs), and natural killer (NK) cells—subclassified by activation, function, and tissue residency. Validation with an influenza A virus model confirmed expected dynamic immune responses and revealed previously unrecognized populations. This spatially informed approach enables high‐resolution interrogation of pulmonary immunity in health and disease. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91eea359003311f7d1a7abfa2e8821468360d814" target='_blank'>
              Spatially Resolved Immune Profiling of the Murine Lung Using In Vivo Labeling and Spectral Cytometry
              </a>
            </td>
          <td>
            Rebecca Palmer, Katherine R Pilkington, Kit Moloney-Geany, Kathryn E Hally, K. Hilligan
          </td>
          <td>2025-12-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Gliomas are the most aggressive primary malignancies of the central nervous system (CNS) and exhibit marked heterogeneity that is closely associated with metabolic reprogramming. Emerging evidence underscores the pivotal role of lactylation modifications in shaping the tumor microenvironment (TME) and facilitating glioma progression. This study aimed to systematically identify key lactylation-related genes (LRGs), elucidate their functional roles and associated pathways, and explore their potential as novel therapeutic targets using multi-omics data.


METHODS
We combined various datasets from the TCGA, GEO, and CGGA databases, including RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics. Key LRGs were identified through a multi-step analytical pipeline that involved processing scRNA-seq data using (Seurat, scoring), cell-type-specific lactylation scoring (AUCell), high-dimensional weighted gene co-expression network analysis (hdWGCNA) and applying rigorous machine learning-based feature selection utilizing 10 algorithms and 101 combinatorial strategies. We comprehensively assessed the prognostic value associated with the immune microenvironment, and spatiotemporal heterogeneity of the prioritized RAN. Functional validation was executed using shRNA-mediated knockdown in glioma cell lines, including LN229, U87, and U251, while evaluating proliferation (CCK-8, colony formation, EdU), migration (wound healing), invasion (Transwell), and pathway activity (using western blot).


RESULTS
scRNA-seq analysis revealed distinct lactylation enrichment patterns across glioma cell types, with malignant cells exhibiting the highest scores. hdWGCNA identified a gene module (royal blue) strongly correlated with lactylation activity (correlation = 0.75). The intersection of this module with a curated set of LRGs yielded 22 candidate genes. Subsequent machine learning analysis using (ENet, α = 0.4) prioritized six core LRGs (PDAP1, ALYREF, CBX3, MAGOH, RAN, TMSB4X). RAN, an understudied gene in glioma, was selected for further investigation. High RAN expression correlated significantly with poor patient prognosis, reduced immune cell infiltration (assessed by ESTIMATE, CIBERSORT, xCell, ssGSEA), and distinct spatiotemporal heterogeneity within tumors (analyzed using spatial transcriptomics, Monocle2). Glioma cell invasion, migration, colony formation, and proliferation were all markedly inhibited by RAN knockdown. Mechanistically, reduced p-AKT levels following knockdown and functional rescue with a PI3K/AKT activator (SC79) indicate that RAN increased these malignant traits by activating the PI3K/AKT signaling pathway.


CONCLUSION
Our study established lactylation modifications as a crucial regulator of the TME and glioma progression. Through integrative multi-omics analysis and robust machine learning techniques, we determined that RAN was a novel lactylation-associated gene. RAN is a potent, independent prognostic biomarker that promotes glioma malignancy via the PI3K/AKT pathway. Our results demonstrate RAN as a prospective therapeutic target and establish a novel framework for individualized therapy for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab709820271e6bd43618ee5a794c6475cc3e5b17" target='_blank'>
              Comprehensive Multi-Omics Analysis Identifies Lactylation-Related Gene RAN as a Novel Prognostic Biomarker and Therapeutic Target in Glioma.
              </a>
            </td>
          <td>
            Shunda null Wang, Fan Tang, Hong Chen, Jingyue null Pang, Ying null Zhang, Jianjing null Yang, Yue null Zhang
          </td>
          <td>2025-12-18</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6c50025097225b7066bd7530506f979302dfaf" target='_blank'>
              The subclonal footprint of pervasive early dissemination in pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, A. Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Catia Gaspar, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Ayah Elqaderi, Maryam Monajemzadeh, Yifan Wang, Stanley W. K. Ng, S. Holter, Barbara T. Grünwald, Julie M. Wilson, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Masoom A. Haider, G. O’Kane, Jennifer J. Knox, Stephen Gallinger, F. Notta
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Li-Fraumeni Syndrome (LFS), caused by pathogenic germline Tp53 mutations, is a pediatric cancer predisposition disorder associated with markedly elevated risk of early-onset tumors, particularly rhabdomyosarcoma (RMS). Our study examined how developmental disruption may prime the tissues of LFS individuals for tumorigenesis. Single-nuclei RNA sequencing (snRNA-seq) was performed on RMS tumors, distal muscle from mice matched to RMS tumors, and cancer-free WT and LFS mouse muscle to define lineage-specific transcriptomic signatures of LFS-RMS. Embryonic limb buds were collected at embryonic days 10, 12, 14, and 16, subjected to snRNA-seq, and used to reconstruct developmental trajectories. These lineage-specific signatures were mapped back onto both WT and LFS embryonic lineages to determine whether oncogenic programs were detectable in utero and to identify the timing of their emergence. A stress-primed transcriptional program was identified as a defining feature of LFS tissues. This program distinguished precancer from postcancer states, becoming amplified in both RMS tumors and distal muscle matched to tumors. In LFS embryos, stage-specific alterations were observed in cell type proportions and maturation, including the emergence of unique myogenic and mesenchymal subpopulations absent in both WT embryos and postnatal muscle, reflecting developmental plasticity restricted to the LFS embryonic context. A lag in the transition to mature myogenic cells was detected at E14, preceded by a spike in the LFS stress-primed signature at E12 during windows of active lineage specification and proliferation. Pathway analyses of lineage-specific signatures indicated widespread dysregulation, with particularly strong changes in RNA-associated and developmental programs, further defining this stress-sensitive transcriptional state. These findings provide the first single-nuclei resolution map of embryogenesis in LFS and provide evidence that LFS-associated oncogenic programs are prenatally encoded. By linking altered lineage dynamics, developmental plasticity, and stress-primed transcriptional states to the emergence of RMS, this work establishes a developmental framework for understanding pediatric cancer predisposition. The results highlight potential windows for early detection and suggest opportunities to target embryonically primed programs for intervention in LFS.



 Ashby Kissoondoyal, Paula R. Quaglietta, Brianne Laverty, David Malkin. Single-nuclei profiling of LFS development reveals tumour susceptibility [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4bd18cd0e2b5ff6a30ceebfb643ff450b1c37a" target='_blank'>
              Abstract B033: Single-nuclei profiling of LFS development reveals tumour susceptibility
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, Paula R. Quaglietta, B. Laverty, D. Malkin
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Homozygous deletion of the 9p21.3 genomic locus spanning the CDKN2A/B and MTAP genes is an event affecting 15% of cancers. While CDKN2A is a well-established tumor suppressor gene, the role of MTAP in tumorigenesis varies across cancer types. MTAP codes for methylthioadenosine phosphorylase, a key enzyme in the methionine salvage pathway, and its loss has been associated with several downstream synthetic vulnerabilities. Despite multiple efforts to exploit MTAP loss for targeted therapies, none of these efforts have yielded substantial results in clinical trials. In this review, we consolidate the existing literature along with our systematic analysis to provide an updated perspective on the incidence of MTAP loss in different cancers and elucidate its impact on metabolism, immune microenvironment, and tumor progression. In addition, we summarize the therapeutic strategies that have been investigated preclinically on MTAP-null tumors before and after the advent of functional genomic screening tools. We further assess the current landscape of clinical trials investigating MTAP-targeted inhibitors, evaluating their limitations and potential avenues for improvement. The insights gained from this review will inform future research directions beyond the promising PRMT5/MAT2A axis for rational combination therapies that would work synergistically to eradicate this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846311f0527655d852715f68535274117ddc12d9" target='_blank'>
              MTAP-Null Tumors: A Comprehensive Review on Synthetic Vulnerabilities and Therapeutic Strategies
              </a>
            </td>
          <td>
            Bavani Subramaniam, Wai Chin Chong, Aylar Babaei, M. Bornhorst, Chunchao Zhang, R. Packer, Javad Nazarian
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Emerging evidence suggests that oral squamous cell carcinoma (OSCC) harbors distinct microbial communities, yet their influence on tumor immunobiology remains unclear. This study investigated the prognostic value of intratumoral microbiota and their role in modulating CD8⁺ T-cell function. Using TCGA datasets, a 19-microorganism prognostic signature was constructed via Cox and LASSO regression and validated with Kaplan–Meier survival analysis. Comprehensive bioinformatics analyses, including GSEA/GSVA, CIBERSORT, ESTIMATE, pharmacogenomics, and mutational profiling, were performed to explore underlying mechanisms. The microbial signature demonstrated strong predictive performance, with higher risk scores significantly associated with reduced overall survival (p < 0.001). High-risk tumors exhibited enrichment of PI3K/AKT/mTOR signaling, metabolic reprogramming, and epithelial–mesenchymal transition, alongside an immunosuppressive microenvironment characterized by CD8⁺ T-cell depletion, M0 macrophage infiltration, and upregulation of immunosuppressive markers including CD276 and TGF-β1. Conversely, immune-activating checkpoints such as PD-1 and CTLA-4 were elevated in low-risk tumors. Notably, periodontal pathogens negatively correlated with immune effector activity, and TP53 mutations were more frequent in high-risk cases (82% vs. 67%). This study identified intratumoral microbial signatures as independent prognostic biomarkers and validated their reproducibility in an external cohort. Our findings support a microbiota–immune axis contributing to immune evasion in OSCC, offering novel avenues for prognostic stratification and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea87f1d3cad21e45f622b4777a7389fe481a0a6c" target='_blank'>
              Intratumoral microbiota drive immune evasion and disease progression in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaolong Zang, Xiaoxia Li, Rongxin Sun, Xiaojie Zhang, Zhiyong Li, Zijian Cheng
          </td>
          <td>2025-12-01</td>
          <td>BDJ Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cervical cancer remains a major global burden largely caused by persistent infection with high risk human papillomavirus (HPV). Biological differences between HPV clade A7 and HPV clade A9 may influence tumor programs and clinical outcomes. To propose pharmacological candidates for repositioning, we applied an expression-based drug repurposing approach using the OCTAD (Open Cancer Therapeutic Discovery) framework. Disease transcriptional signatures were constructed for both HPV clades and compared with drug perturbation profiles to identify compounds showing inverse associations with the tumor related expression patterns, restricting the analysis to Food and Drug Administration (FDA) approved agents. The screening identified 41 and 52 candidates for HPV clade A7 and HPV clade A9, respectively, and stronger transcriptomic reversal was associated with higher drug sensitivity in relevant cell lines. These candidates were enriched for pharmacologic classes such as histone deacetylase inhibitors, estrogen pathway modulators, and statins. Additional enriched categories also emerged, including antimetabolites, protein kinase inhibitors, proteasome inhibitors, antimalarials, and antimicrobial agents, several of which already show experimental activity in cervical cancer models. These findings reveal both shared and clade-associated vulnerabilities in HPV-driven cervical cancer and demonstrate the utility of expression-based repurposing for generating actionable hypotheses. The resulting drug lists provide a concise, biologically grounded resource to guide preclinical validation and rational exploration in cervical cancer HPV positive models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df5682b494766817b7e18f24bc15ba93bfc7c7a" target='_blank'>
              Drug Repositioning for HPV Clade-Specific Cervicouterine Cancer Using the OCTAD Pipeline
              </a>
            </td>
          <td>
            Joel Ruiz-Hernández, G. Anda-Jáuregui, Enrqiue Hernandez-Lemus
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2a1ea73f6fab9f1966b90164d74f94fad3f917" target='_blank'>
              Large scale quantification of natural killer cell-induced apoptosis in patient-derived organoids reveals intratumoral response heterogeneity.
              </a>
            </td>
          <td>
            Marisa Mercadante, Armin Scheben, Jacob Estrada, Jan Savas-Carstens, William Sullivan, Nicholas Housel, Maria Sapar, Tatiana Volpari, Jax Hebner, Tomasz Rusielewicz, Frederick J. Monsma, Stefan Semrau, Yinan Wang, Laura A. Martin
          </td>
          <td>2025-12-26</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nowadays, chronic benign and malignant prostatic diseases are prevalent, costly, and impose a significant burden. Benign prostatic hyperplasia (BPH), a common condition in the aging population, often coexists with localized prostate cancer (PCa). These diseases likely share underlying molecular mechanisms, which remain poorly understood. The exploration of novel cell subpopulations and specific biomarkers for accurate diagnosis and treatment of prostatic diseases is ongoing and holds great clinical promise. Prostate cell proliferation and immune inflammation are key contributors to the progression of BPH and PCa, involving various prostate and immune cell subpopulations. This raises important questions about how specific cell types drive phenotypic heterogeneity. Advanced single-cell RNA sequencing (scRNA-seq), a cutting-edge technology, offers unparalleled insights at the single-cell level. Similar to a microscope that identifies cell types within tissue samples, scRNA-seq elucidates cellular heterogeneity and diversity within single cell populations, positioning itself as a future-leading sequencing technology. Considering that BPH and PCa share androgen-dependent growth, chronic inflammation and specific microenvironmental changes, this review discusses recent discoveries of novel cell subpopulations and molecular signatures in BPH and PCa that can be dissected by scRNA-seq. It aims to help researchers better understand the molecular pathogenesis of these conditions while offering new therapeutic possibilities for clinical management of benign and malignant prostatic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65bfb63670355f27eb34fa433fbe18e5d3f994e" target='_blank'>
              Advances in the identification of novel cell signatures in benign prostatic hyperplasia and prostate cancer using single-cell RNA sequencing
              </a>
            </td>
          <td>
            Yu Pan, Qingqing Song, B. Lai, He Ma
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis is a leading cause of mortality in breast cancer patients, yet the signaling promoting metastatic dissemination is not completely understood. Prior literature implicates neuronal innervation in tumor progression, including recent studies in breast cancer progression with the 4T1 and PyMT cell line orthotopic injection models. Our experiments address the immune limitations of these studies with an alternative model to elucidate neuronal control of metastatic breast cancer by using a MMTV-PyMT transplant model and resiniferatoxin (RTX) for denervation. To this end, we generated a robust array of spontaneous MMTV-PyMT tumors with various histological subtypes. These tumors were transplanted into RTX challenged MMTV-Cre mice. In contrast to previous literature, denervation did not impact survival or tumor growth. Interestingly, we noticed a slight reduction in the percent of the solid poorly differentiated tumors with a corresponding increase in tumors that contained a mixed pathology in RTX challenged mice. Strikingly, and consistent with prior work, we noted a reduction in metastasis with denervation. Together, these data suggest neuronal innervation promotes metastasis without impacting tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a7eaed2f0618eb9d5a5d855a2c066e128ecb44" target='_blank'>
              Neuronal innervation of breast cancer promotes metastatic dissemination
              </a>
            </td>
          <td>
            Anthony J. Schulte, E. Andrechek
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer is an increasing public health burden, including in Sub-Saharan African (SSA) populations, where cancer incidence is predicted to increase by around 140% between 2022 and 2050. These rates require a better understanding of the epidemiological, clinical, and genetic/molecular characteristics of cancer in SSA populations. There is an urgent need to improve the genomic characterization of SSA tumour samples and also to establish suitable in vitro models for hypothesis testing. In fact, even though thousands of cancer cell lines (CCLs) have been established employing different methods of cell immortalization and have been included in deep molecular characterization panels, SSA ancestry is limited to only ~6% (mostly African Americans, who represent limited diversity in the context of the African continent) of publicly available CCLs. This disparity needs to be addressed by using next-generation immortalization methods such as conditional reprogramming to establish CCLs derived from SSA cancer patients that also represent the diversity within the African continent. Research in SSA oncobiology has the potential to add essential information to better understand the diverse molecular pathways leading to cancer and to find promising therapeutic avenues. We also discuss the challenges to conducting oncobiology studies with cell modelling derived from SSA patients in low-to-middle-income African countries, such as Portuguese-speaking African countries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4a79cfc8bc4d250ec22de8c16984bfdedfb3a8" target='_blank'>
              Addressing Ancestral Underrepresentation in Oncobiology: The Need for Sub-Saharan African-Specific In Vitro Models
              </a>
            </td>
          <td>
            Carla S. dos Santos, A. C. Magalhães, Ricardo J. Pinto, Carla Carrilho, Cláudia Pereira, Fernando Miguel, Pamela Borges, L. Santos, Luísa Pereira
          </td>
          <td>2025-11-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Studies show that only 20–30% of untreated ductal carcinoma in situ (DCIS) cases will evolve into invasive breast cancer. Thus, many patients are subject to overtreatment and its associated morbidities. The ecological context of the tumor microenvironment may impact how DCIS evolves. We applied landscape ecology statistics to hematoxylin and eosin (H&E) images from DCIS to test whether they predict recurrence of DCIS or progression to invasive disease. The ability to make such predictions is key for optimizing treatment decisions, reducing associated risks, and improving the efficiency of healthcare resource allocation.



 We conducted a retrospective cohort study comprising 158 patients with a DCIS diagnosis, which included 57 cases of DCIS recurrence (2–228 months) and 45 cases of progression to invasive disease (12–180 months). Of these, 21 patients received lumpectomy, 92 had lumpectomy followed by radiation therapy, and 45 had total mastectomy. We generated a digital pathology library of whole-slide images (WSIs) from H&E tumor sections and performed machine learning guided nuclear segmentation and classification. A comprehensive set of spatial statistics was calculated for each WSI to characterize the spatial architecture and cellular interactions within the microenvironment. Cross-validation using a linear spline-based hazard regression model (HARE) was applied to a training subset of the data. The resulting survival model was validated on the test subset to predict disease-free survival (DFS) of recurrence and progression.



 We found that the type of DCIS treatment significantly stratified patients based on DFS (log-rank test p<0.0001). Further analysis of patients who received lumpectomy + radiation or mastectomy revealed that decreased mean nearest neighbor distance between tumor cells and lymphocytes, and between fibroblasts and lymphocytes, was significantly associated with improved recurrence-free survival (concordance=0.68). Stratification of samples based on the median risk score derived from our model was discriminatory with a p-value <0.015. Additionally, a decreased nearest neighbor distance between lymphocytes and fibroblasts showed a marginal association with an increased time to invasive disease (p<0.06).



 Integrating spatial ecological statistics derived from digital pathology provides enhanced risk stratification for DCIS, beyond the established effects of initial treatment. Our findings underscore the importance of the tumor microenvironment's ecological dynamics in DCIS recurrence and progression. The predictive risk associated with specific spatial interactions, particularly involving lymphocytes and fibroblasts, suggests that previously unappreciated ecological mechanisms may play a role in the development of early micro-metastases, influencing outcomes even after standard therapies such as mastectomy or lumpectomy with radiation. These insights highlight novel targets for improved prognostic tools and more personalized therapeutic strategies.



 Luis H. Cisneros. Microenvironmental drivers of DCIS recurrence and progression to breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1296ebbd19523c1df1d8c9459fc242375494458" target='_blank'>
              Abstract A016: Microenvironmental drivers of DCIS recurrence and progression to breast cancer
              </a>
            </td>
          <td>
            L. Cisneros
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cellular senescence is a hallmark of cancer and induces senescence-associated inflammatory responses in the tumor microenvironment (TME). CDKN1A, an important cellular senescence marker, plays significant roles in cell proliferation, invasion, migration, and apoptosis. Previous studies have implied its drug resistance role in certain cancer types. However, its impact on immunotherapy efficacy in advanced LUAD remains unclear. Using TCGA and SU2C-MARK cohorts, we investigated CDKN1A’s biological features in advanced LUAD, analyzing pathway regulation, immune infiltration, and immunotherapy associations. Single-cell RNA sequencing (GSE148071) validated TME and cellular communication differences between CDKN1A high/low expressing tumors in advanced LUAD. CDKN1A expression was positively associated with immunosuppressive environment including extracellular matrix, cancer-associated fibroblasts (CAFs) and myeloid-derived suppressor cells (MDSCs) in advanced LUAD for both TCGA and SU2C-MARK cohorts (P < 0.05). CDKN1A showed significantly positive correlations with many senescence genes in advanced LUAD (P < 0.05), which were also positively associated with endothelial cells, epithelial cells, fibroblast cells and macrophages, but negatively associated with immune cells (P < 0.05). In multivariable cox regression, patients with high CDKN1A expression had worse OS (HR = 2.74, 95% CI = 1.31–5.73, P = 0.007) and PFS (HR = 1.78, 95% CI = 1.01–3.11, P = 0.045) than those with low CDKN1A expression when treated with immunotherapy. In contrast, the high CDKN1A expression was not associated with PFS and OS in the TCGA cohort, in which the LUAD patients received standard chemotherapy, suggesting the immunotherapy predictive role instead prognostic role of CDKN1A. Moreover, single-cell analysis revealed that CDKN1A highly expressed tumors were accompanied by an enrichment of stromal and endothelial cells within the tumor microenvironment, along with enhanced activity of SMAD3/4, the downstream transcription factors of TGFB signaling. These tumors exhibited increased cell–cell communication with stromal cells (COL1A1/COL1A2-ITGA1/ITGB1/SDC1) and endothelial cells (NAMPT-ITGAS/ITGB1/INSR). CDKN1A expression was associated with cellular senescence, immunosuppressive environment and exhibited resistance to immunotherapy in advanced LUAD, suggesting a potential combination strategy with senolytic or senomorphic therapies to overcome immunotherapeutic resistance in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f34f805e0422258abbde7dbe983d9c6f010ed5" target='_blank'>
              CDKN1A and cellular senescence are associated with immune resistance in advanced lung adenocarcinoma
              </a>
            </td>
          <td>
            Wang Shen, Feidie Duan, Shuiting Fu, Peng Cui, Yezhen Shi, Lei Zhang, Xiao Mi, Yu Xu, Guoqiang Wang, S. Cai, Xuebing Li, Xiaojun Tang
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) represents a profoundly aggressive and heterogeneous brain neoplasm linked to a bleak prognosis. Palmitoylation plays a key role in the development and progression of GBM, but its molecular mechanism and prognostic significance in GBM are still not fully understood. This study aims to explore the prognostic biomarkers of GBM based on palmitoylation-related genes. Eight scoring methods, including AUCell, UCell, singscore, ssGSEA, JASMINE, VAM, scSE, and viper, were used to score each sample. In addition, the palmitoylation score calculated by the AUCell algorithm is selected as the representative. In order to screen the genes related to GBM survival and build a risk prognosis model, 101 algorithms constructed by 10 kinds of machine learning are arranged and combined for variable screening and model building, and then immune infiltration and immunotherapy evaluation, drug screening, and molecular docking are carried out. We observed that macrophages in GBM cell types have the highest palmitoylation score. The secondary dimensionality reduction clustering of macrophages showed that the palmitoylation of PLCG2 + macrophages was significantly higher than that of other subtypes, and three core prognostic genes (ZDHHC2, ZDHHC4, ZDHHC20) were screened out by machine learning. A higher risk score is significantly related to worse clinical status and most immune labels. Among them, ZDHHC2 was significantly up-regulated in GBM in several verification groups. Molecular docking found that quercetin was the best targeted drug for ZDHHC2. This study revealed for the first time the heterogeneity of palmitoylation at the GBM single-cell level. The identification of ZDHHC2, ZDHHC4, and ZDHHC20 as key regulators of palmitoylation in GBM emphasized their potential as biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9691d337db779456aa047d54e76f827220a4bec" target='_blank'>
              Integrating single-cell RNA-Seq and machine learning to dissect a novel Palmitoylation-related prognostic signature of glioblastoma
              </a>
            </td>
          <td>
            Zhu Zhang, Haojie Zheng, Chunli Yang, Zhiying Lin
          </td>
          <td>2025-12-04</td>
          <td>BMC Neurology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca8660736eface72fe417cf2ee8f74f809dbba1" target='_blank'>
              Effects of mitotically active polyploid giant cancer cells on chemoresistance through interaction with cancer-associated fibroblasts.
              </a>
            </td>
          <td>
            Go Itoh, Yuma Fukushi, Kurara Takagane, Keisuke Kanetaka, Hitomi Suzuki, Yohei Kawasaki, S. Kuriyama, K. Iemura, Kozo Tanaka, Akiteru Goto, Masamitsu Tanaka
          </td>
          <td>2025-12-21</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) is a rare and aggressive soft tissue sarcoma characterized by high genomic instability, resulting in high local recurrence rates and limited effective therapeutic options in advanced stages. Recent progress in cancer immunology research has encouraged investigation into the Tumor Microenvironment (TME) of sarcomas, including MFS, to identify immune-related biomarkers of prognostic and therapeutic relevance. Although data remain limited in MFS, existing evidence suggests a heterogeneous immune landscape, including: i) variable expression of immune checkpoint molecules such as Programmed Cell Death Protein 1 (PD-1) and Programmed Death-Ligand 1 (PD-L1), ii) presence of tumor-infiltrating lymphocytes, iii) alterations in antigen presentation pathways, and iv) a pronounced angiogenic signature. These findings underscore the potential role of immune biomarkers for patients’ clinical stratification and the consequent possibility of developing new immunotherapeutic strategies. This review will focus on the cellular and molecular architecture of immune infiltration, vascular remodeling, and lymphoid neogenesis, assessing their prognostic and predictive value as potential biomarkers. Finally, we will present ongoing clinical trials aimed at modulating the immune-vascular niche to inform innovative therapeutic strategies for this challenging sarcoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f542fc2b8b6a3c1d5def7187e73b0d930cab87" target='_blank'>
              Unveiling the myxofibrosarcoma tumor microenvironment: implications for immunotherapy
              </a>
            </td>
          <td>
            Cecilia Profumo, Massimiliano Grassi, Valentina Rigo, Matteo Mascherini, Paola Monti, Giorgia Arcovito, Giorgia Anselmi, Laura Damele, G. Timon, Danila Comandini, Michela Croce
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [0, 11, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>