<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-14 09:29:23 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="We conducted a spatial analysis of stage III colorectal adenocarcinomas using Hyperion Imaging Mass Cytometry, examining 52 tumors to assess the tumor microenvironment at the single-cell level. This approach identified 10 distinct cell phenotypes in the tumor microenvironment, including stromal and immune cells, with a subset showing a proliferative phenotype. By focusing on spatial neighborhood interactions and tissue niches, particularly regions with tumor-infiltrating lymphocytes, we investigated how cellular organization relates to clinicopathological and molecular features such as microsatellite instability (MSI) and recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4 + T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a63f7d3dd84bf14bb59b2621b8ec0ed0c881cf8" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, HoJoon Lee, Minh Tran, Richard C Dela Cruz, A. Sathe, Xiangqi Bai, Ignacio A Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, , Terence Rhodes
          </td>
          <td>2025-04-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee24fc55015da54029a007a558ae389dd346d690" target='_blank'>
              Mapping the spatial architecture of glioblastoma from core to edge delineates niche-specific tumor cell states and intercellular interactions
              </a>
            </td>
          <td>
            Saad M. Khan, Anthony Z. Wang, Rupen Desai, Colin R. McCornack, Rui Sun, Sonika M. Dahiya, Jennifer Foltz-Stringfellow, Ngima D Sherpa, Lydia Leavitt, T. West, Alexander F. Wang, Aleksandar Krbanjevic, Bryan D. Choi, Eric Leuthardt, B. Patel, Al Charest, Albert H. Kim, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in orchestrating tumor cell behavior and cancer progression. Recent advances in spatial profiling technologies have uncovered novel spatial signatures, including univariate distribution patterns, bivariate spatial relationships, and higher-order structures. These signatures have the potential to revolutionize tumor mechanism and treatment. In this review, we summarize the current state of spatial signature research, highlighting computational methods to uncover spatially relevant biological significance. We discuss the impact of these advances on fundamental cancer biology and translational research, address current challenges and future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f15f8a11af43d80ae3f5bc50b4193451c3cfbd" target='_blank'>
              Mapping Tumor Microenvironment and Treatment Response of Diffuse Midline Glioma Using Multiplexed Immunofluorescence and AI Models
              </a>
            </td>
          <td>
            Sandra Laternser, A. D. De Micheli, Sarah Brüningk, E. McDonough, Antonela Petrovic, Elisabeth Jane Rushing, Christine Surrette, Julia Bugajska, A. Eze, Lindsay Kilburn, Luisa Machado, Susanne Dettwiler, Fabiola Prutek, Noelia Casares Lagar, D. de la Nava, Denise Morinigo, Fiona Ginty, Matthew D. Dun, Michael A. Grotzer, A. G. Stücklin, Sabine Mueller, R. Packer, M. Bornhorst, Marta María Alonso, Javad Nazarian
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL⁺) versus non-generating (TIL⁻) tumors revealed that IL7R expression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL⁻ tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5083118ad8ca69bf9d2b345a596f9d2646c08163" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            K. Hotchkiss, Kenan Zhang, Anna M. Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R. Conejo-Garcia, B. Shaz, John W. Hickey, Mustafa Khasraw
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma invasion into healthy brain tissue remains a major barrier to effective treatment, yet current models fail to capture its full complexity in a scalable and patient-specific manner. Here, we introduce GlioTrace, a novel ex vivo imaging and AI-based analytical framework that enables real-time, spatiotemporal tracking of glioblastoma invasion dynamics in patient-derived glioma cell culture xenograft (PDCX) brain slices. By integrating whole-specimen confocal microscopy, vascular counterstaining, and an advanced computational pipeline combining convolutional neural networks and Hidden Markov Models, GlioTrace identifies distinct invasion modes, including dynamic morphological switching, vessel-guided migration, and immune cell interactions and quantifies patient-specific variations in invasion plasticity. Using GlioTrace, we demonstrate that targeted therapies can selectively modulate invasion phenotypes, revealing spatially and temporally distinct drug responses. This scalable platform provides an unprecedented window into glioblastoma progression and treatment response, offering a powerful tool for precision oncology and anti-invasion therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d85bb045bd5b47db33273aef84595c6d869e5e" target='_blank'>
              Reconstructing the Single-Cell Spatiotemporal Dynamics of Glioblastoma Invasion
              </a>
            </td>
          <td>
            H. Mangukiya, Madeleine Skeppås, Soumi Kundu, Maria Berglund, Adam A. Malik, C. Krona, Sven Nelander
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Gliomas are among the most lethal cancers, with limited treatment options. To uncover hallmarks of therapeutic escape and tumor microenvironment (TME) evolution, we applied spatial proteomics, transcriptomics, and glycomics to 670 lesions from 310 adult and pediatric patients. Single-cell analysis shows high B7H3+ tumor cell prevalence in glioblastoma (GBM) and pleomorphic xanthoastrocytoma (PXA), while most gliomas, including pediatric cases, express targetable tumor antigens in less than 50% of tumor cells, potentially explaining trial failures. Longitudinal samples of isocitrate dehydrogenase (IDH)-mutant gliomas reveal recurrence driven by tumor-immune spatial reorganization, shifting from T-cell and vasculature-associated myeloid cell-enriched niches to microglia and CD206+ macrophage-dominated tumors. Multi-omic integration identified N-glycosylation as the best classifier of grade, while the immune transcriptome best predicted GBM survival. Provided as a community resource, this study opens new avenues for glioma targeting, classification, outcome prediction, and a baseline of TME composition across all stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08309318ca287b808eae53da393baf17b1d09b8" target='_blank'>
              Multi-omic landscape of human gliomas from diagnosis to treatment and recurrence
              </a>
            </td>
          <td>
            Hadeesha Piyadasa, Benjamin Oberlton, Mikaela Ribi, Jolene S Ranek, Inna Averbukh, K. Leow, M. Amouzgar, Candace C. Liu, Noah F. Greenwald, E. Mccaffrey, Rashmi Kumar, Selena Ferrian, Albert G. Tsai, Ferda Filiz, C. Fullaway, M. Bosse, Sricharan Reddy Varra, Alex Kong, Cameron Sowers, M. Gephart, Pablo Nuñez-Perez, EnJun Yang, Michael Travers, Michael Schachter, Samantha Liang, Maria R. Santi, Samantha Bucktrout, P. Gherardini, John Connolly, Kristina Cole, Michael Barish, Christine E. Brown, Derek A. Oldridge, Richard R. Drake, Joanna J. Phillips, Hideho Okada, Robert Prins, Sean C. Bendall, Michael Angelo
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Adult-onset dermatomyositis (DM) is an autoimmune inflammatory myopathy with distinct cutaneous manifestations and a strong malignancy association. Through comparative analysis with cutaneous lupus erythematosus (CLE), our integrated spatial and single-cell transcriptomics analysis revealed unique immune and stromal niches associated with DM subtypes. Unexpectedly, we found an association between cancer-associated DM skin lesions and the presence of dispersed immune infiltrates enriched with macrophages, CD8+ T cells, plasma cells, and B cells with preserved vascular architecture. In contrast, non-cancer associated DM skin exhibited dense myeloid cell infiltrates, including neutrophils, monocytes, and macrophages, with elevated expression of IL1B and CXCL10 localized near injured vascular endothelia. Cytokines produced by these myeloid infiltrates together with local tissue hypoxia triggered dramatic stromal remodeling, leading to loss of vascular-associated fibroblasts. In addition to the CXCL10+ myeloid signature, non-cancer-associated DM skin with pDC presence showed the emergence of specific cellular pairs: PD-L1-expressing mregDCs and activated Tregs expressing NFKB2 and TNF receptors. While both DM and CLE showed strong interferon signatures, DM uniquely displayed IFN-β expression. Together, our study provides the first comprehensive spatial mapping of immune and stromal cells in adult-onset DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62db8748b2521d6e18e161c76b546fccb24b5fcd" target='_blank'>
              Spatial Transcriptomics Identifies Immune-Stromal Niches Associated with Cancer in Adult Dermatomyositis
              </a>
            </td>
          <td>
            Ksenia S. Anufrieva, Neda Shahriari, Ce Gao, Rochelle L. Castillo, Jessica Liu, Sean A Prell, Shideh Kazerounian, Khashayar Afshari, Anastasia N. Kazakova, Erin Theisen, Teresa Bowman, A. LaChance, Kimberly B. Hashemi, Ilya Korsunsky, Mehdi Rashighi, R. A. Vleugels, Kevin Wei
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, M. van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, N. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, R. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, M. Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98eeee496484cc85e8f46ccc055600cb6afe1b1e" target='_blank'>
              A harmonized ovarian cancer scRNA-seq atlas to dissect disease heterogeneity underlying metastatization and chemoresistance
              </a>
            </td>
          <td>
            M. Sallese, Vittorio Aiello, C. E. Villa, Pietro Lo Riso, Giuseppe Testa
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73634c31cfc63813b66fedfb8ecff6a08ba232af" target='_blank'>
              Spatiotemporal organisation of residual disease in mouse and human BRCA1-deficient mammary tumours and breast cancer
              </a>
            </td>
          <td>
            Demeter Túrós, Morgane Decollogny, Astrid Chanfon, Myriam Siffert, L. Romanens, J. Tille, O. Trédan, I. Labidi-Galy, Alberto Valdeolivas, Sven Rottenberg
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have transformed the treatment landscape across multiple cancer types achieving durable responses for a significant number of patients. Despite their success, many patients still fail to respond to ICIs or develop resistance soon after treatment. We sought to identify early treatment features associated with ICI outcome. We leveraged the MC38 syngeneic tumor model because it has variable response to ICI therapy driven by tumor intrinsic heterogeneity. ICI response was assessed based on the level of immune cell infiltration into the tumor – a well-established clinical hallmark of ICI response. We generated a spatial atlas of 48,636 transcriptome-wide spots across 16 tumors using spatial transcriptomics; given the tumors were difficult to profile, we developed an enhanced transcriptome capture protocol yielding high quality spatial data. In total, we identified 8 tumor cell subsets (e.g., proliferative, inflamed, and vascularized) and 4 stroma subsets (e.g., immune and fibroblast). Each tumor had orthogonal histology and bulk-RNA sequencing data, which served to validate and benchmark observations from the spatial data. Our spatial atlas revealed that increased tumor cell cholesterol regulation, synthesis, and transport were associated with a lack of ICI response. Conversely, inflammation and T cell infiltration were associated with response. We further leveraged spatially aware gene expression analysis, to demonstrate that high cholesterol synthesis by tumor cells was associated with cytotoxic CD8 T cell exclusion. Finally, we demonstrate that bulk RNA-sequencing was able to detect immune correlates of response but lacked the sensitivity to detect cholesterol synthesis as a feature of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ca2d699ba974ef22ac49a1f2990095215e2dd0" target='_blank'>
              Bacterial population dynamics during colonization of solid tumors
              </a>
            </td>
          <td>
            Serkan Sayin, Motasem ElGamel, Brittany Rosener, Michael Brehm, Andrew Mugler, Amir Mitchell
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db37f2f209f5470571ed3506376b29858f863c2" target='_blank'>
              Tumor cell heterogeneity drives spatial organization of the intratumoral immune response.
              </a>
            </td>
          <td>
            Miho Tanaka, Lotus Lum, Kenneth H Hu, Piyush Chaudhary, Savannah Hughes, Cecilia Ledezma-Soto, B. Samad, Daphne Superville, Kenneth Ng, Arun Chumber, Ciara Benson, Zoe N Adams, Kelly Kersten, Oscar A. Aguilar, Lawrence Fong, Alexis J. Combes, Matthew F. Krummel, Melissa Q Reeves
          </td>
          <td>2025-06-02</td>
          <td>The Journal of experimental medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3aa6e9e33a49a78ba8f7c54e58b7d68564b492" target='_blank'>
              Applications of Multiplex Immunohistochemistry in Evaluating Spatiotemporal Heterogeneity of T Cells
              </a>
            </td>
          <td>
            Mercedes Machuca-Ostos, Tim de Martines, K. Yoshimura, Junichi Mitsuda, Sumiyo Saburi, Alisa Kimura, Hiroki Morimoto, Koichi Yoshizawa, Nana Sakurai, Nanako Murakami, Kayo Kitamoto, Makoto Yasuda, Y. Sugiyama, H. Ogi, Saya Shibata, Aya Miyagawa-Hayashino, Eiichi Konishi, Kyoko Itoh, Takahiro Tsujikawa, Shigeru Hirano
          </td>
          <td>2025-02-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="A striking characteristic of liver cancer is its extensive heterogeneity, particularly with regard to its varied response to immunotherapy. In this study, we employed multimodal sequencing approaches to explore the various aspects of neoadjuvant nivolumab treatment in liver cancer patients. We used spatially-resolved transcriptomics, single- and bulk-cell transcriptomics, and TCR clonotype analyses to examine the spatiotemporal dynamics of the effects of nivolumab. We observed a significantly higher clonal expansion of T cells in the tumors of patients who responded to the treatment, while lipid accumulation was detected in those of non-responders, likely due to inherent differences in lipid metabolic processes. Furthermore, we found a preferential enrichment of T cells, which was associated with a better drug response. Our results also indicate a functional antagonism between tumor-associated macrophages (TAMs) and CD8 cells and their spatial separation. Notably, we identified a UBASH3B/NR1I2/CEACAM1/HAVCR2 signaling axis, highlighting the intense communication among TAMs, tumor cells, and T-cells that leads to pro-tumorigenic outcomes resulting in poorer nivolumab response. In summary, using integrative multimodal sequencing investigations, combined with the multi-faceted exploration of pre- and post-treatment samples of neoadjuvant nivolumab-treated HCC patients, we identified useful mechanistic determinants of therapeutic response. We also reconstructed the spatiotemporal model that recapitulates the physiological restoration of T cell cytotoxicity by anti-PD1 blockade. Our findings could provide important biomarkers and explain the mechanistic basis differentiating the responders and non-responders. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02314-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ba5970d9a4d124e0362f78801008f40bb86612" target='_blank'>
              Multimodal sequencing of neoadjuvant nivolumab treatment in hepatocellular carcinoma reveals cellular and molecular immune landscape for drug response
              </a>
            </td>
          <td>
            Fanhong Zeng, Qingyang Zhang, Y. Tsui, Huanhuan Ma, Lu Tian, Abdullah Husain, Jingyi Lu, J. Lee, Vanilla Xin Zhang, Po-Man Li, Gary Cheuk-Hang Cheung, Tan-To Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Checkpoint inhibitors have led to remarkable benefits in non-small cell lung cancer (NSCLC), yet response rates remain below expectations. High-dimensional analysis and mechanistic experiments in clinical samples and relevant NSCLC models uncovered the immune composition of lung cancer tissues, providing invaluable insights into the functional properties of tumor-infiltrating T cells and myeloid cells. Among myeloid cells, type 1 conventional dendritic cells (cDC1s) stand out for their unique ability to induce effector CD8 T cells against neoantigens and coordinate antitumoral immunity. Notably, lung resident cDC1 are particularly abundant and long-lived and express a unique tissue-specific gene program, underscoring their central role in lung immunity. Here, we discuss recent insights on the induction and regulation of antitumoral T cell responses in lung cancer, separating it from the tissue-agnostic knowledge generated from heterogeneous tumor models. We focus on the most recent studies dissecting functional states and spatial distribution of lung cDC1 across tumor stages and their impact on T cell responses to neoantigens. Finally, we highlight relevant gaps and emerging strategies to harness lung cDC1 immunostimulatory potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2155ad4b9a79772729f8fb7af74e6020d7c7eb14" target='_blank'>
              Tissue-specific properties of type 1 dendritic cells in lung cancer: implications for immunotherapy
              </a>
            </td>
          <td>
            Lucía López Rodríguez, Roberto Amadio, G. M. Piperno, F. Benvenuti
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Breast cancers present intricate microenvironments comprising heterotypic cellular interactions, yet a comprehensive spatial map remained to be established. Here, we employed the DNA nanoball-based genome-wide in situ sequencing (Stereo-seq) to visualize the geospatial architecture of 30 primary breast tumors and metastatic lymph nodes across different molecular subtypes. This unprecedented high-resolution atlas unveils the fine structure of the tumor vasculature, highlighting heterogeneity in phenotype, spatial distribution, and intercellular communication within both endothelial and perivascular cells. In particular, venular smooth muscle cells are identified as the primary source of CCL21/CCL19 within the microenvironment. In collaboration with ACKR1-positive endothelial cells, they create a chemokine-rich venular niche to synergistically promote lymphocyte extravasation into tumors. High venule density predicts increased immune infiltration and improved clinical outcomes. This study provides a detailed spatial landscape of human breast cancer, offering key insights into the venular regulation of tumor immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3853b930721fb6df8f03ce40a70213e79309b5e" target='_blank'>
              Spatially resolved atlas of breast cancer uncovers intercellular machinery of venular niche governing lymphocyte extravasation
              </a>
            </td>
          <td>
            Xin Wang, Zhanyu Wang, Qijun Liao, Pei Yuan, Junpu Mei, Yin Zhang, Chao Wu, Xiyu Kang, Sufei Zheng, Chen-En Yang, Jiaxiang Liu, Qingyao Shang, Jiangtao Li, Bingning Wang, Liangyu Li, Hui Liu, Weining Hu, Zhensheng Dong, Jie Zhao, Linying Wang, Tao Liu, Yusheng Den, Chengrui Wang, Lijuan Han, Qianjun Chen, Huanming Yang, Xun Xu, Jie He, Zhen Yue, Nan Sun, Xiaodong Fang, Jianming Ying
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
To identify the specific intratumoral and microenvironmental heterogeneity of acral (AM) and mucosal melanoma (MM), we aimed to delineate their distinct cellular compositions, evolutionary trajectories, and subtype-specific therapeutic strategies.


EXPERIMENTAL DESIGN
Single-cell transcriptomic and genomic landscapes were analyzed across 42 melanoma samples (28 AM, 11 MM, and 3 non-acral cutaneous melanoma [CM]), supplemented by in vitro and in vivo validation. Tumor and stromal cells were profiled using scRNA-seq, whole-exome sequencing, and functional assays, including transwell migration, co-culture systems, and xenograft models.


RESULTS
Tumor cells exhibited divergent evolutionary routes, with MM dominated by MGP⁺/PCOLCE⁺ subpopulations showing high epithelial-to-mesenchymal transition (EMT) potential. MM displayed elevated neutrophil infiltration and CXCL3⁺ tumor-associated macrophages, while AM was enriched with PI16⁺ cancer-associated fibroblasts (CAFs) promoting tumor proliferation. Molecular classification revealed MM subtypes: an antigen-presenting subtype linked to favorable outcomes and a proliferative subtype associated with recurrence. TIGIT⁺ Treg cells were enriched in AM, suggesting targeted inhibition potential. Genomic analysis connected BRAF/NRAS mutations to ALDOA⁺ stem-like tumor cells and identified PTGDS as a therapeutic target in triple-WT melanomas.


CONCLUSIONS
Our study provides a comprehensive comparison of AM and MM, uncovering subtype-specific stromal-immune interactions and molecular programs. The findings highlight actionable targets (e.g., TIGIT in AM, CXCL3⁺ macrophages in MM) and propose a framework for precision therapies, biomarker-driven trials, and risk stratification to improve outcomes in these aggressive melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d1f1c177fc4d5a9d6abeefba72d4bbfa28d55fc" target='_blank'>
              Single-cell transcriptomic landscape deciphers intratumoral heterogeneity and subtypes of acral and mucosal melanoma.
              </a>
            </td>
          <td>
            Yunyan Li, Ziyang Cui, Xiaole Song, Yeqing Chen, Cang Li, Junfeng Shi, Wenkang Qian, Guoxin Ren, Jiang Zhou, Chunpu Li, Xiaoqing Ma, Yifan Chen, Dongdong Jia, Yongli Zhang, Zhiling Zhang, Ronghao Zhang, Zhaotian Zhang, Yong Chen, Zhixiang Xu, Wantao Chen, X. Miao, Hongmeng Yu, Jianxin Chen, Kai Wang, Colin R. Goding, Zhi Wei, Tao Li, Rutao Cui
          </td>
          <td>2025-04-07</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Accumulating evidence links pediatric cancers to prenatal transformation events, yet the influence of the developmental stage on oncogenesis remains elusive. We investigated how hematopoietic stem cell developmental stages affect leukemic transformation, disease progression, and therapy response using a novel, humanized model of NUP98::NSD1-driven pediatric acute myeloid leukemia, that is particularly aggressive with WT1 co-mutations. Fetal-derived hematopoietic stem cells readily transform into leukemia, and WT1 mutations further enhance stemness and alter lineage hierarchy. In contrast, stem cells from later developmental stages become progressively resistant to transformation. Single-cell analyses revealed that fetal-origin leukemia stem cells exhibit greater quiescence and reliance on oxidative phosphorylation than their postnatal counterparts. These differences drive distinct therapeutic responses, despite identical oncogenic mutations. In patients, onco-fetal transcriptional programs correlate with worse outcomes. By targeting key vulnerabilities of fetal-origin leukemia cells, we identified combination therapies that significantly reduce aggressiveness, highlighting the critical role of ontogeny in pediatric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are key components of the hepatocellular carcinoma (HCC) tumor microenvironment (TME). regulating tumor proliferation, metastasis, therapy resistance, immune evasion via diverse mechanisms. A deeper understanding of the l diversity of CAFs is essential for predicting patient prognosis and guiding treatment strategies. Methods We examined the diversity of CAFs in HCC by integrating single-cell, bulk, and spatial transcriptome analyses. Results Using a training cohort of 88 HCC single-cell RNA sequencing (scRNA-seq) samples and a validation cohort of 94 samples, encompassing over 1.2 million cells, we classified three fibroblast subpopulations in HCC: HLA-DRB1 + CAF, MMP11 + CAF, and VEGFA + CAF based on highly expressed genes of which, which are primarily located in normal tissue, tumor boundaries, and tumor interiors, respectively. Cell trajectory analysis revealed that VEGFA + CAFs are at the terminal stage of differentiation, which, notably, is tumor-specific. VEGFA + CAFs were significantly associated with patient survival, and the hypoxic microenvironment was found to be a major factor inducing VEGFA + CAFs. Through cellular communication with capillary endothelial cells (CapECs), VEGFA + CAFs promoted intra-tumoral angiogenesis, facilitating tumor progression and metastasis. Additionally, a machine learning model developed using high-expression genes from VEGFA + CAFs demonstrated high accuracy in predicting prognosis and sorafenib response in HCC patients. Conclusions We characterized three fibroblast subpopulations in HCC and revealed their distinct spatial distributions within the tumor. VEGFA + CAFs, which was induced by hypoxic TME, were associated with poorer prognosis, as they promote tumor angiogenesis through cellular communication with CapECs. Our findings provide novel insights and pave the way for individualized therapy in HCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06192-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a50452a86dbb7c37a7b8f94f2df738ddd72d94" target='_blank'>
              3D vascularized microtumors unveil aberrant ccRCC vasculature and differential sensitivity to targeted treatments
              </a>
            </td>
          <td>
            Noémie Brassard-Jollive, Y. Atlas, Camille LM Compère, Corinne Ardidie-Robouant, Philippe Mailly, Morad El Bouchtaoui, Virginie Lelarge, Guillaume Blot, Nathalie Josseaume, Stéphanie De Oliveira, C. Hélary, Christophe Leboeuf, Isabelle Cremer, Mathilde Sibony, Guilhem Bousquet, S. Germain, L. Muller, C. Monnot
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0b044513a32c1bfdbc6767c40e3363e83eee34" target='_blank'>
              Abstract B038: A self-supervised AI model leveraging spatial omics for analyzing tumor microenvironment heterogeneity in breast cancer only with H&E
              </a>
            </td>
          <td>
            Haenara Shin, Dongjoo Lee, Yooeun Kim, D. Lee, Kwon Joong Na, C. Cha, Hosub Park, Hongyoon Choi
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma (GBM), a primary brain tumor, exhibits intratumoral heterogeneity and dynamic spatial-temporal changes. GBM-derived extracellular vesicles (EVs), reflecting tumor characteristics, present potential as liquid-biopsy markers for early diagnosis and monitoring. This study aims to evaluate molecular signatures of plasma-derived EVs from GBM patients using a conventional flow cytometer. EVs have been isolated from glioma patients and healthy controls (HCs) plasma using density gradient ultracentrifugation (DGU). EVs were evaluated by bead-based multiplex analysis in a conventional flow cytometer. Principal component analysis (PCA), hierarchical clustering, and correlation analysis provided comprehensive insights into EV characteristics. EVs successfully isolated were visualized in transmission and scanning electron microscopy (STEM). Bead-based multiplex analysis in flow cytometer detected the level of 37 EV surface markers, including tumor-related, cancer stem cell, endothelial cell, and immune cell- specific antigens. PCA identified the EV surface markers that are most significant for differentiating the subjects, and hierarchical clustering revealed four distinct clusters based on EV surface marker levels. EV molecular signature demonstrated considerable heterogeneity across patient clusters. The presence of CD29 emerged not only as a defining factor for a cluster of patients, but also served as a marker to differentiate patients from HCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4072e04be0714ef76d0ca98717fc6de4c896a86" target='_blank'>
              Identification of distinct profiles of glioblastoma through the immunocapture of extracellular vesicles from patient plasma
              </a>
            </td>
          <td>
            D. Manu, R. Bălaşa, L. Pruteanu, V. Curean, Lucian Barbu-Tudoran, G. Șerban, Rareș Chinezu, A. Bălașa
          </td>
          <td>2025-03-19</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background The tumor boundary of breast cancer represents a highly heterogeneous region. In this area, the interactions between malignant and non-malignant cells influence tumor progression, immune evasion, and drug resistance. However, the spatial transcriptional profile of the tumor boundary and its role in the prognosis and treatment response of breast cancer remain unclear. Method Utilizing the Cottrazm algorithm, we reconstructed the intricate boundaries and identified differentially expressed genes (DEGs) associated with these regions. Cell-cell co-positioning analysis was conducted using SpaCET, which revealed key interactions between tumor-associated macrophage (TAMs) and cancer-associated fibroblasts (CAFs). Additionally, Lasso regression analysis was employed to develop a malignant body signature (MBS), which was subsequently validated using the TCGA dataset for prognosis prediction and treatment response assessment. Results Our research indicates that the tumor boundary is characterized by a rich reconstruction of the extracellular matrix (ECM), immunomodulatory regulation, and the epithelial-to-mesenchymal transition (EMT), underscoring its significance in tumor progression. Spatial colocalization analysis reveals a significant interaction between CAFs and M2-like tumor-associated macrophage (TAM), which contributes to immune exclusion and drug resistance. The MBS score effectively stratifies patients into high-risk groups, with survival outcomes for patients exhibiting high MBS scores being significantly poorer. Furthermore, drug sensitivity analysis demonstrates that high-MB tumors had poor response to chemotherapy strategies, highlighting the role of the tumor boundary in modulating therapeutic efficacy. Conclusion Collectively, we investigate the spatial transcription group and bulk data to elucidate the characteristics of tumor boundary molecules in breast cancer. The CAF-M2 phenotype emerges as a critical determinant of immunosuppression and drug resistance, suggesting that targeting this interaction may improve treatment responses. Furthermore, the MBS serves as a novel prognostic tool and offers potential strategies for guiding personalized treatment approaches in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b10699232e4b3f1c95f3a7d6be8ae3942b8fc0f" target='_blank'>
              Spatial multi-omics analysis of tumor-stroma boundary cell features for predicting breast cancer progression and therapy response
              </a>
            </td>
          <td>
            Yuanyuan Wu, Youyang Shi, Zhanyang Luo, Xiqiu Zhou, Yonghao Chen, Xiaoyun Song, Sheng Liu
          </td>
          <td>2025-03-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Human Immunodeficiency Virus (HIV) is one of the most prevalent viruses, causing significant immune depletion in affected individuals. Current treatments can control HIV and prolong patients’ lives, but new challenges have emerged. Increasing incidence of cancers occur in HIV patients. Esophageal squamous cell carcinoma (ESCC) is one of the most common cancers observed in HIV patients. However, the spatial cellular characteristics of HIV-related ESCC have not been explored, and the differences between HIV-ESCC and typical ESCC remain unclear. Methods We performed spatial transcriptome sequencing on HIV-ESCC samples to depict the microenvironment and employed cell communication analysis and multiplex immunofluorescence to investigate the molecular mechanism in HIV-ESCC. Results We found that HIV-ESCC exhibited a unique cellular composition, with fibroblasts and epithelial cells intermixed throughout the tumor tissue, lacking obvious spatial separation, while other cell types were sparse. Besides, HIV-ESCC exhibited an immune desert phenotype, characterized by a low degree of immune cell infiltration, with only a few SPP1+ macrophages showing immune resistance functions. Cell communication analysis and multiplex immunofluorescence staining revealed that tumor fibroblasts in HIV-ESCC interact with CD44+ epithelial cells via COL1A2, promoting the expression of PIK3R1 in epithelial cells. This interaction activates the PI3K-AKT signaling pathway, which contributes to the progression of HIV-ESCC. Conclusions Our findings depict the spatial microenvironment of HIV-ESCC and elucidate a molecular mechanism in the progression of HIV-ESCC. This will provide us insights into the molecular basis of HIV-ESCC and potential treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02248-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiao-Jun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) represents the most common pathological type of lung cancer, and the combination of neoadjuvant immunotherapy with chemotherapy has emerged as the first-line treatment for NSCLC. Nevertheless, the efficacy of this therapeutic approach remains variable. The present study aims to examine the impact of chemoimmunotherapy in NSCLC patients, with a view to identifying key molecules, critical cell subpopulations, communication patterns and spatial distributions that potentially correlate with therapeutic sensitivity. A total of 16 lung cancer tissue samples were collected from a cohort of 12 NSCLC patients and subjected to single-cell RNA and spatial transcriptome sequencing. Our data demonstrated that the distribution of CD4 + Treg T cells and mCAFs indicated an immunosuppressive tumor microenvironment, while the accumulation of CD4 + Th17 T cells and iCAFs could act as a positive marker for the sensitivity to chemoimmunotherapy. Furthermore, a significant high level of SELENOP-macrophages was observed in tissues from positive responders, and a strong co-localization between SELENOP-macrophages and antigen-presenting cancer associated fibroblasts (CAFs) in the tumor boundaries was identified, indicating the cooperative roles of these two cell types in response to combined therapy. Moreover, SELENOP-macrophages were observed to be accumulated in tertiary lymphoid structures, which further suggested its critical role in recruiting lymphocytes. Furthermore, analysis of cell–cell communication, based on spatial transcriptomics, suggests that the interactions between SELENOP-macrophages, apCAFs, CD4 + and CD8 + T cells were significantly enhanced in responders. In addition, SELENOP-macrophages recruited CD4 + Naïve, Helper and CD8 + Naïve T cells through pathways such as the cholesterol, interleukin, chemokine and HLA when responding to combined therapy. The present study further unveils the dynamic spatial and transcriptional changes in the tumor microenvironment of non-small cell lung cancer in response to combination therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02287-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86da4dd75d392abc980cae7b3d81379fef557c28" target='_blank'>
              Single-cell and spatial transcriptomic analyses revealing tumor microenvironment remodeling after neoadjuvant chemoimmunotherapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Xiaolu Cui, Siyuan Liu, He Song, Jingjing Xu, Yanbin Sun
          </td>
          <td>2025-04-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Hepatoblastoma (HB) is a common paediatric liver malignancy characterised by significant intratumoral heterogeneity and a complex tumour microenvironment (TME). Using single‐cell RNA sequencing (scRNA‐seq), we analysed 43,592 cells from three tumour regions and adjacent normal tissue of an HB patient. Our study revealed distinct cellular compositions and varying degrees of malignancy across different tumour regions, with the T1 region showing the highest malignancy and overexpression of HMGB2 and TOP2A. Survival analysis demonstrated that high HMGB2 expression is associated with poor prognosis and increased recurrence, suggesting its potential as a prognostic marker. Additionally, we identified a diverse immune microenvironment enriched with regulatory T cells (Tregs) and CD8+ effector memory T cells (Tem), indicating potential immune evasion mechanisms. Notably, CTLA‐4 and PD‐1 were highly expressed in Tregs and Tem cells, highlighting their potential as immunotherapy targets. Myeloid cells, including Kupffer cells and dendritic cells, also exhibited distinct functional roles in different tumour regions. This study provides the first comprehensive single‐cell atlas of HB, revealing critical insights into its intratumoral heterogeneity and immune microenvironment. Our findings not only advance the understanding of HB biology but also offer new directions for precision medicine, including the development of targeted therapies and immunotherapeutic strategies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634b11e52e203c006a13f961efc458d554db20a9" target='_blank'>
              Intratumoral Heterogeneity and Immune Microenvironment in Hepatoblastoma Revealed by Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Mingdi Ma, Chen Jin, Qian Dong
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Approximately 15-20% of head and neck cancer squamous cell carcinoma (HNSCC) patients respond favorably to immune checkpoint blockade (ICB). Previous single-cell RNA-Seq (scRNA-Seq) studies identified immune features, including macrophage subset ratios and T-cell subtypes, in HNSCC ICB response. However, the spatial features of HNSCC-infiltrated immune cells in response to ICB treatment need to be better characterized. Methods Here, we perform a systematic evaluation of cell interactions between immune cell types within the tumor microenvironment using spatial omics data using complementary techniques from both 10X Visium spot-based spatial transcriptomics and Nanostring CosMx single-cell spatial omics with RNA gene panel including 435 ligands and receptors. In this study, we used integrated bioinformatics analyses to identify cellular neighborhoods of co-localizing cell types in single-cell spatial transcriptomics and proteomics data. In addition, we used both publicly available scRNA-Seq and in-house spatial RNA-Seq data to identify spatially constrained Ligand-Receptor interactions in Responder patients. Results With 522,399 single cells profiled with both RNA and protein from 26 patients, in addition to spot-resolved spatial RNA-Seq from 8 patients treated with ICB together with bioinformatics analysis of publicly available single-cell and bulk RNA-Seq, we have identified a spatial and cell-type specific context-dependency of myeloid and T cell interaction difference between Responders and Non-Responders. We defined further cellular neighborhood and the sources of chemokine CXCL9/10-CXCR3 interactions in Responders, emerging targets in ICB, as well as CXCL16-CXCR6, CCL4/5-CCR5, and other underappreciated and potential markers and targets for ICB response in HNSCC. In addition, we have contributed a rich data resource of cell-cell Ligand Receptor interactions for the immunotherapy and HNSCC research community. Discussion Our work provides a comprehensive single-cell and spatial atlas of immune cell interactions that correlate with response to ICB in HNSCC. We showcase how integrating multiple technologies and bioinformatics approaches can provide new insights into potential immune-based biomarkers of ICB response. Our results suggested refining future studies using preclinical animal models in a more context-specific manner to elucidate potential underlying mechanisms that lead to improved ICB responses. What is already known on this topic Most cancer patients still do not experience clinical benefits from immune checkpoint blockade (ICB), necessitating the development of response biomarkers and new immunotherapeutic targets. What this study adds Here, we use integrated high-dimensional omics and bioinformatics approaches to identify immune cell-cell interaction markers associated with ICB response in patients with Head and neck squamous cell carcinoma. How this study might affect research, practice or policy We identified spatial and cell-type specificity of Ligand-Receptor interactions between myeloid and T cells in ICB Responder patients that may help inform further mechanistic studies and biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ece16324064b70206dfb8fd687bfc5db3089e8d7" target='_blank'>
              Integrated single-cell and spatial analysis identifies context-dependent myeloid-T cell interactions in head and neck cancer immune checkpoint blockade response
              </a>
            </td>
          <td>
            Athena E. Golfinos-Owens, Taja Ložar, Parth Khatri, Rong Hu, Paul M. Harari, Paul F. Lambert, Megan B Fitzpatrick, Huy Q. Dinh
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The identification of the complex spatial architecture of immune cell infiltration and its interaction mechanisms within tumor ecosystems provides crucial insights into therapeutic responses to neoadjuvant therapy in muscle-invasive bladder cancer (MIBC). This study aims to characterize the spatial features of distinct cell-type niches within the tumor microenvironment (TME) of patients with varying responses to neoadjuvant therapy. Methods We performed spatial transcriptomic profiling on six MIBC specimens obtained from a registered clinical trial (ChiCTR2000032359), generating whole-transcriptome spatial atlases to map the TME architecture. High-throughput analytical frameworks were employed to deconstruct the TME, and key findings were validated through immunohistochemistry and mouse model experiments. Results Our analysis revealed that tissues from complete responders exhibited greater infiltration of T and B cells, with the formation of tertiary lymphoid structure (TLS). Trajectory analysis identified CCL19/CCL21 as the key signaling molecules driving TLS formation in MIBC. Mouse experiments demonstrated that recombinant CCL19/CCL21 protein injections promoted intratumoral TLS formation and enhance the efficacy of immunotherapy. Furthermore, we observed significant intrinsic heterogeneity within individual tumors, which may contribute to the lack of therapeutic efficacy in MIBC. Conclusions This study underscores the critical role of TLS formation in the response to neoadjuvant therapy in MIBC. We identified CCL19/CCL21 as key drivers of TLS formation within MIBC tumors and potential immune-sensitizing agents. Additionally, the intrinsic heterogeneity of tumor should be considered a significant factor influencing therapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06358-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2414e88dc899f0c1cdec00aa234fe0cea16d7" target='_blank'>
              Spatial relationships and interactions of immune cell niches are linked to the pathologic response of muscle-invasive bladder cancer to neoadjuvant therapy
              </a>
            </td>
          <td>
            Wasilijiang Wahafu, Quan Zhou, Xihua Yang, Yongming Yang, Yuanyuan Zhao, Zhu Wang, Xiangpeng Kang, Xiongjun Ye, N. Xing
          </td>
          <td>2025-03-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The intrinsic characteristics of metastatic tumors are of great importance in terms of the development of antimetastatic treatment strategies. Elucidation from a spatial immune perspective has the potential to provide a more comprehensive understanding of the mechanisms underlying immune escape, effectively addressing the limitations of relying solely on the analysis of immune cell subpopulation transcriptional profiles. Advances in spatial omics technology enable researchers to precisely analyze precious liver metastasis samples in a high-throughput manner, revealing spatial alterations in immune cell distribution induced by metastasis and exploring the molecular basis of the remodeling process. The aggregation of specific cell subpopulations in distinct regions not only modifies local immune characteristics but also concurrently affects global biological behaviors of liver metastatic tumors. Identifying specific spatial immune characteristics in pretreatment or early-stage treatment tissue samples may achieve accurate clinical predictions. Moreover, developing strategies that target spatial immune remodeling is a promising avenue for future antimetastatic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ea479cd554af75a0d1dcd42be6ffca84a03793" target='_blank'>
              Spatial immune remodeling of the liver metastases: discovering the path to antimetastatic therapy
              </a>
            </td>
          <td>
            Wenchao Xu, Yibo Wang, Nanzhou Wang, Jianzhou Liu, Li Zhou, Junchao Guo
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e90afb96bda692d274dd22748c2e27453a930" target='_blank'>
              Diving Deep: Profiling Exhausted T Cells in the Tumor Microenvironment Using Spectral Flow Cytometry.
              </a>
            </td>
          <td>
            Karen W Teng, Weng Hua Khoo, Nicholas Ching Wei Ho, S. J. Yang, Douglas C Wilson, Edmond Chua, Shu Wen Samantha Ho
          </td>
          <td>2025-04-03</td>
          <td>Cytometry. Part A : the journal of the International Society for Analytical Cytology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Rationale: Advances in cancer therapies have significantly improved patient survival; however, tumors enriched in cancer stem cells (CSCs) have poor treatment responses. CSCs are a key source of tumor heterogeneity, contributing to therapeutic resistance and unfavorable patient outcomes. In the tumor microenvironment (TME), cell-in-cell (CIC) structures, where one cell engulfs another, have been identified as markers of poor prognosis. Despite their clinical relevance, the mechanisms underlying CIC formation across different tumor cell subpopulations remain largely unknown. Elucidating these processes could provide novel insights and therapeutic opportunities to address aggressive, treatment-resistant cancers. Method: Fluorescent mCherry-carrying colorectal cancer stem cells (CRCSCs) were expanded as spheroids in serum-free media and cocultured with either parental cancer cell-expressing Venus fluorescent protein or CFSE dye-stained immune cells (T cells, M1/M2 macrophages, neutrophils, and NK cells) or treated with EGFR- or PD-L1-targeting antibodies to assess the formation of CIC structures. Genes potentially crucial for the formation of CIC structures were knocked down or overexpressed, and their effects on CIC formation were evaluated. The clinical relevance of the in vitro findings was confirmed through analysis of formalin-fixed, paraffin-embedded (FFPE) human colorectal cancer (CRC) specimens. Results: CRCSCs have a strong predilection for serving as the outer cell in a CIC structure and forming homotypic CIC structures predominantly with parental CRC cells. The frequency of CIC structure formation increased when the cells were exposed to anti-PD-L1 antibody treatment. Both the outer CRCSC in a CIC structure and CRCSCs released from a homotypic CIC structure showed enhanced resistance to the cytotoxicity of NK-92MI cells. Restoration of Stathmin1 (STMN1) expression but not RAC1 knockdown in CRCSCs reduced the homotypic CIC frequency, disrupted the outer cell fate in CIC structures, and increased cell susceptibility to NK-92MI cytotoxicity. In CRC patients, CIC structures are associated with poor tumor differentiation, negative STMN1 expression, and poor prognosis. Conclusion: CSCs play a crucial role in informing CIC structures in CRC. CIC structure formation partially depends on low STMN1 expression and confers a survival advantage under NK cytotoxicity. Targeting this pathway may significantly improve immunotherapy's efficacy for CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab3f42feb06b52b9aecd81de765adfb3a490b01" target='_blank'>
              Colorectal cancer stem cells develop NK cell resistance via homotypic cell-in-cell structures suppressed by Stathmin1
              </a>
            </td>
          <td>
            Yen-Yu Lin, Hsin-Yi Lan, Hao-Wei Teng, Ya-Pei Wang, Wen-Chun Lin, Wei-Lun Hwang
          </td>
          <td>2025-03-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors of the digestive tract, with increasing incidence and mortality rates, posing a significant burden on human health. Its progression relies on various mechanisms, among which the tumor microenvironment and tumor-associated macrophages (TAMs) have garnered increasing attention. Macrophage infiltration in various solid tumors is associated with poor prognosis and is linked to chemotherapy resistance in many cancers. These significant biological behaviors depend on the heterogeneity of macrophages. Tumor-promoting TAMs comprise subpopulations characterized by distinct markers and unique transcriptional profiles, rendering them potential targets for anticancer therapies through either depletion or reprogramming from a pro-tumoral to an anti-tumoral state. Single-cell RNA sequencing technology has significantly enhanced our research resolution, breaking the traditional simplistic definitions of macrophage subtypes and deepening our understanding of the diversity within TAMs. However, a unified elucidation of the nomenclature and molecular characteristics associated with this diversity remains lacking. In this review, we assess the application of conventional macrophage polarization subtypes in colorectal malignancies and explore several unique subtypes defined from a single-cell omics perspective in recent years, categorizing them based on their potential functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Characterizing the variety of cell types in the tumor microenvironment (TME) and their organization into cellular communities is vital for elucidating the biological diversity of cancer and informing therapeutic strategies. Methods Here, we employed a machine learning-based algorithm framework, EcoTyper, to analyze single-cell transcriptomes from 139 patients with head and neck squamous cell carcinoma (HNSC)and gene expression profiles from 983 additional HNSC patients, aiming to delineate the fundamental cell states and ecosystems integral to HNSC. Results A diverse landscape of 66 cell states and 9 ecosystems within the HNSC microenvironment was identified, revealing classical cell types while also expanding upon previous immune classifications. Survival analysis revealed that specific cell states and ecotypes (ecosystems) are associated with patient prognosis, underscoring their potential as indicators of clinical outcomes. Moreover, distinct cell states and ecotypes exhibited varying responses to immunotherapy and chemotherapy, with several showing promise as predictive biomarkers for treatment efficacy. Conclusion Our large-scale integrative transcriptome analysis provides high-resolution insights into the cellular states and ecosystems of HNSC, facilitating the discovery of novel biomarkers and supporting the development of precision therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06299-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be5ea8ecb1627ce7c5b0b610bd601d4a5d3e9545" target='_blank'>
              Overcoming myeloid-driven resistance to CAR T therapy by targeting SPP1
              </a>
            </td>
          <td>
            Sharareh Gholamin, H. Natri, Yuqi Zhao, Shengchao Xu, Maryam Aftabizadeh, Begonya Comin-Anduix, Supraja Saravanakumar, Christian Masia, Robyn A Wong, Lance Peter, Mei-i Chung, Evan D Mee, B. Aguilar, R. Starr, D. Torrejon, D. Alizadeh, Xiwei Wu, Anusha Kalbasi, Antoni Ribas, Stephan Forman, B. Badie, Nicholas E. Banovich, Christine E Brown
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background: Liver progenitor cells (LPCs) with bipotential differentiation capacities are essential for restoring liver homeostasis and hepatocyte population after damage. However, the low proportion and shared markers with epithelial cells make studying LPC heterogeneity difficult, especially in humans. To address this gap, we explored over 259,400 human liver single cells across 4 conditions (fetal, healthy, cirrhotic, and HCC-affected livers). Methods: Human liver tissue samples were analyzed using spatial transcriptomics sequencing technologies to describe the heterogeneity of LPCs. Liver tissue was characterized by LPC heterogeneity at single-cell resolution by employing cellular modules, differentiation trajectories, and gene co-expression patterns. Results: We annotated and identified 1 LPC cluster, 3 LPC subpopulations, and 4 distinct cellular modules, indicating the heterogeneity within LPC and the diversity between LPCs and epithelial cells. LPCs showed spatial colocalization with cholangiocytes and comprised a small proportion (2.95±1.91%) within the merged epithelial cells and LPC populations, exhibiting marked differences in marker expression patterns compared to those in mice. LPCs exhibited distinct cellular states in functional restoration, activation, proliferation, and cell transition. Additionally, the gene co-expression network of LPCs exhibited 3 unique modules, reflecting the distinct connectivity of genes encoding apolipoproteins and heat shock proteins in the gene co-expression network modules. Conclusions: Our study provides valuable insights into the multifaceted heterogeneity of human LPCs. Future studies focusing on spatial gene expression dynamics will contribute to our understanding of the spatial arrangement of liver regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363fd866c375a553c02fda34c7798688e167a041" target='_blank'>
              Integrative single-cell and spatial transcriptome analysis reveals heterogeneity of human liver progenitor cells
              </a>
            </td>
          <td>
            Chuanjun Liu, Kai Wang, Junpu Mei, Ruizhen Zhao, Juan Shen, Wei Zhang, Liangyu Li, Bhaskar Roy, Xiaodong Fang
          </td>
          <td>2025-02-26</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Circulating tumor cells (CTCs) are pivotal biomarkers in tumor metastasis, however, the underlying molecular mechanism of CTCs behavioral heterogeneity during metastasis remains unexplored. Here, an integrative workflow is developed to link behavior characteristics to metabolic profiling within individual CTCs, which simulates the metastatic process on a microfluidic system and combined with single‐cell mass spectrometry (MS) detection. Spheroid‐derived HCT116 cells are tracked and extracted via a temporary vascular system, revealing various arrest patterns under biomimetic vascular shear flow. Downstream MS analysis characterizes 17 cellular metabolites and associates metabolic profiles with de‐adhesion behaviors of the same CTCs, identifying a potential high‐metastatic subpopulation with enhanced arrest ability and evaluating critical metabolites involved in metastasis pathways. Additionally, the metastasis‐inhibiting effect of anti‐tumor drug 5‐fluorouracil by reducing high‐metastatic cells in spheroids is elucidated. This approach offers a valuable opportunity to dissect the interplay of the metastatic behavior and metabolic profiles of CTCs and foster insights into the molecular mechanisms underlying behavioral phenotypes in the tumor metastasis process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3e81bfd80ccc573739b0a4ae2229795ed81e7" target='_blank'>
              Linking Metastatic Behavior and Metabolic Heterogeneity of Circulating Tumor Cells at Single‐Cell Level Using an Integrative Microfluidic System
              </a>
            </td>
          <td>
            Ying Hou, Jiaxu Lin, Hongren Yao, Zengnan Wu, Yongning Lin, Jinming Lin
          </td>
          <td>2025-02-17</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="The role of tumor microenvironment in invasive breast cancer prognosis and treatment is highly appreciated. With the advent of immunotherapy, immunophenotypic characterization in primary tumors is gaining attention as it can improve patient stratification. Here, we discuss the benefits of spatial analysis employing double and multiplex immunostaining, allowing the simultaneous detection of more than one protein on the same tissue section, which in turn helps us provide functional insight into infiltrating immune cells within tumors. We focus on studies demonstrating the prognostic and predictive impact of distinct tumor-infiltrating lymphocyte subpopulations including different CD8(+) T subsets as well as CD4(+) T cells and tumor-associated macrophages in invasive breast carcinoma. The clinical value of immune cell topography is also appreciated. We further refer to how the integration of digital pathology and artificial intelligence in routine practice could enhance the accuracy of multiplex immunostainings evaluation within the tumor microenvironment, maximizing our perception of host immune response, improving in turn decision-making towards more precise immune-associated therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b3640c44bd80cadd9a05e0404e0638d900b1d2" target='_blank'>
              Double–Multiplex Immunostainings for Immune Profiling of Invasive Breast Carcinoma: Emerging Novel Immune-Based Biomarkers
              </a>
            </td>
          <td>
            Sofia D P Theodorou, Konstantinos Ntostoglou, I. Nikas, Dimitrios Goutas, V. Georgoulias, Christos Kittas, Ioannis S. Pateras
          </td>
          <td>2025-03-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc383607243031294dc9e768566a894cfedd572c" target='_blank'>
              Glioblastoma stem cell morphotypes convey distinct cell states and clinically relevant functions
              </a>
            </td>
          <td>
            C. Barelli, Matteo Bonfanti, Filippo Mirabella, Dario Ricca, Negin Alizadehmohajer, Viktoria Sokolova, Alberto Campione, G. Sicuri, C. Peano, Stefania Faletti, Roberto Stefini, N. Kalebic
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor stroma consists mainly of extracellular matrix, fibroblasts, immune cells, and vasculature. Its structure and functions are altered during malignancy: tumor cells transform fibroblasts into cancer-associated fibroblasts, which exhibit immunosuppressive activities on which growth and metastasis depend. These include exclusion of immune cells from the tumor nest, cancer progression, and inhibition of T-cell-based immunotherapy. To understand these complex interactions, we measure the density of different cell types in the stroma using immunohistochemistry techniques on tumor samples from lung cancer patients. We incorporate these data into a minimal dynamical system, explore the variety of outcomes, and finally establish a spatio-temporal model that explains the cell distribution. We reproduce that cancer-associated fibroblasts act as a barrier to tumor expansion, but also reduce the efficiency of the immune response. Our conclusion is that the final outcome depends on the parameter values for each patient and leads to either tumor invasion, persistence, or eradication as a result of the interplay between cancer cell growth, T-cell cytotoxicity, and fibroblast activity. However, despite the existence of a wide range of scenarios, distinct trajectories, and patterns allow quantitative predictions that may help in the selection of new therapies and personalized protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300ea7f39d018c28251113d7be1e7ef9c1ae4343" target='_blank'>
              Mechanistic insight for T-cell exclusion by cancer-associated fibroblasts in human lung cancer
              </a>
            </td>
          <td>
            Joseph Ackermann, Chiara Bernard, P. Sirven, Hélène Salmon, Massimiliano Fraldi, Martine Ben Amar
          </td>
          <td>2025-04-10</td>
          <td>eLife</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Intratumor transcriptional heterogeneity (ITTH) presents a major challenge in cancer treatment, particularly due to limited understanding of the diverse malignant cell populations and their relationship to therapy resistance. While single-cell sequencing has provided valuable insights into tumor composition, its high cost and technical complexity limits its use for large-scale tumor screening. In contrast, several databases collecting bulk RNA sequencing (RNA-seq) data from multiple samples across various cancer types are available and could be used to profile ITTH. Several deconvolution approaches have been developed to infer cellular composition from such data. However, most of these methods rely on predefined markers or single cell reference datasets, limiting the performance of such methods by the quality of used reference data. Although unsupervised approaches do not face such limitations, existing methods have not been specifically adapted to characterize malignant cell states, and focus instead on general cell types. To address these gaps, we introduce CDState, an unsupervised method for inferring malignant cell subpopulations from bulk RNA-seq data. CDState utilizes a Nonnegative Matrix Factorization model improved with sum-to-one constraints and a cosine similarity-based optimization to deconvolve bulk gene expression into distinct cell state-specific profiles, and estimate the abundance of each state across tumor samples. We validate CDState using bulkified single-cell RNA-seq data from five cancer types, showing that it outperforms existing unsupervised deconvolution methods in both cell state proportions and gene expression estimation. Applying CDState to 33 cancer types from TCGA, we identified recurrent malignant cell programs, with epithelial-mesenchymal transition as the main driver of tumor transcriptional heterogeneity. We further link the identified malignant states to patient clinical features, revealing states associated with worse patient prognosis. We show that ITTH is linked with patient survival and clinical features, as well as is associated with varied responses to therapy. Finally, we identified potential genetic drivers of ITTH and malignant cell states, including TP53, KRAS, and PIK3CA, known oncogenic genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16cea30bc9f4619225da6166ac6508ace6b994e" target='_blank'>
              CDState: an unsupervised approach to predict malignant cell heterogeneity in tumor bulk RNA-sequencing data
              </a>
            </td>
          <td>
            Agnieszka Kraft, Josephine Yates, Florian Barkmann, Valentina Boeva
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Microvascular invasion (MVI) is linked to poor prognosis, early recurrence and post-surgical intrahepatic metastasis of hepatocellular carcinoma (HCC) but roles of tumor-associated endothelial cells (TECs) remain unclear. The aim of the current study was to investigate the role of TECs in microvascular invasion in HCC. Methods Single-cell RNA sequencing (scRNA-seq) data from three patients with MVI and two patients with non-MVI HCC were used to identify TECs subpopulations via Seurat R package. Using bioinformatics analysis identified co-expression modules associated with MVI in TECs. Differential gene expression analysis, KME values and Gene Expression Profiling Interactive Analysis (GEPIA) survival were utilized to identify genes with significant involvement. TECs subgroup developmental trajectory was analyzed using monocle2. Five additional spatial transcriptomics (ST) datasets and four HCC postoperative pathological specimens were used to validate the differential expression of subgroups of TECs and hub genes between MVI and non-MVI groups. Results Distinct TECs subgroups had significant heterogeneity between datasets from MVI and non-MVI patients. MVI samples had TECs subgroups with increased levels of the epithelial−mesenchymal transition (EMT), endothelial cell migration and angiogenesis. Opposing EMT development was found in MVI TECs relative to non-MVI TECs. TM4SF1 was highly expressed in TECs undergoing the EMT and is thought to be linked to MVI. Conclusion TECs with elevated TM4SF1 expression facilitate MVI during HCC via an effect on the EMT, suggesting the potential of TM4SF1 as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de61af4341cad2d9fb79114592979b95505d43c" target='_blank'>
              TM4SF1 overexpression in tumor-associated endothelial cells promotes microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Junwu Guo, Liangrui Chen, Binghua Dai, Chengjun Sui, Zhitao Dong, Keji Chen, Kecai Duan, Kunpeng Fang, Aijun Li, Kui Wang, Li Geng
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as cases with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased cellular indexing of transcriptomes and epitopes by sequencing technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We described the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes stratified along a naive → activated axis; patients with a “high activated” signature score appeared to be undergoing a robust melanoma antigen–driven adaptive immune response and, thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise antitumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for patients with high-risk melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies immune cell signatures associated with metastasis
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T. Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Purpose The aim of this study is to explore the pathological niche of cancer metastasis and the site-specific interactions between tumor cells and the microenvironment, understand the mechanisms driving metastasis progression and identify potential therapeutic targets. Methods Data from four lung cancer metastasis datasets (GSE123902, GSE131907, GSE148071, and GSE186344) were downloaded and subjected to stringent quality control and filtering. Cell types were identified using canonical markers, and pseudotime trajectory analysis was performed to evaluate cell differentiation. Functional and pathway enrichment analyses, including ssGSEA and GO/KEGG, were conducted. CellphoneDB was used to analyze intercellular communication, ranking receptor-ligand interactions based on communication strength. Results Eleven cell types were identified after quality control, revealing significant heterogeneity and site-specific functionality in lung cancer metastases. CTLs showed notable activity in antigen presentation and T-cell differentiation pathways, with DNAJB1⁺ CTLs playing a dominant role in cytotoxicity and immune regulation. B cells, myeloid cells, and CAFs were involved in immune modulation, defense, and matrix remodeling through specific signaling pathways. Tumor cell subclusters drove proliferation, migration, and immune evasion via immune-regulatory, Hippo, and TGF-beta pathways. No overlapping pathways were observed across metastatic sites. Cell communication analysis identified PPIA-BSG and APP-CD74 as key axes in brain and lymph node metastases, while FN1-Integrin and CTLA4-CD86 dominated in bone and adrenal metastases, respectively. Conclusions In summary, this study highlights the functional heterogeneity and site-specific interactions of cells in lung cancer metastases, providing insights into the mechanisms shaping metastatic niches and potential therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02269-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c4f954c571841166277c9ca7ec19c26848339cb" target='_blank'>
              Single-cell profiling uncovers the intricate pathological niche diversity in brain, lymph node, bone, and adrenal metastases of lung cancer
              </a>
            </td>
          <td>
            Le Liu, Yuan Zhou, Zhenjun Ye, Zhiyong Chen, Benchao Yuan, Liyi Guo, Haiyan Zhang, Yuanyuan Xu
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dbf2dbcf6c4b1d8a1dbdd8e9dcfc190f14aeaa6" target='_blank'>
              Methylation-Based Deconvolution Unveils Glioblastoma Heterogeneity and Cell-Type Composition Linked to Patient Survival
              </a>
            </td>
          <td>
            Aviel Iluz, Nir Lavi, Hanna Charbit, Mijal Gutreiman, Masha Idelson, Debora Steiner, Etti Ben-Shushan, A. Zick, Amir Eden, A. Mordechai, Moscovici Samuel, Y. Fellig, Alexander Lossos, Joshua Moss, Benjamin E. Reubinoff, I. Lavon
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644814c562e273342aaa2172d8ae291ec9003332" target='_blank'>
              Characterization of tumour heterogeneity through segmentation-free representation learning on multiplexed imaging data.
              </a>
            </td>
          <td>
            Jimin Tan, H. Le, Jiehui Deng, Yingzhuo Liu, Yuan Hao, Michelle Hollenberg, Wenke Liu, Joshua M. Wang, Bo Xia, Sitharam Ramaswami, Valeria Mezzano, Cynthia Loomis, Nina Murrell, Andre L. Moreira, Kyunghyun Cho, Harvey I Pass, Kwok-Kin Wong, Yi Ban, Benjamin G. Neel, A. Tsirigos, David Fenyö
          </td>
          <td>2025-02-20</td>
          <td>Nature biomedical engineering</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Prostate cancer (PC) is the most frequently diagnosed malignancy among men and contributes significantly to cancer-related mortality. While recent advances in in vitro PC modeling systems have been made, there remains a lack of robust preclinical models that faithfully recapitulate the genetic and phenotypic characteristics across various PC subtypes-from localized PC (LPC) to castration-resistant PC (CRPC)-along with associated stromal cells. Here, we established human PC assembloids from LPC and CRPC tissue by reconstituting tumor organoids with corresponding cancer-associated fibroblasts (CAFs), thereby incorporating aspects of the tumor microenvironment (TME). Established PC organoids exhibited high concordance in genomic landscape with parental tumors, and the tumor assembloids showed a higher degree of phenotypic similarity to parental tumors compared to tumor organoids without CAFs. PC assembloids displayed increased proliferation and reduced sensitivity to anti-cancer treatments, indicating that PC assembloids are potent tools for understanding PC biology, investigating interaction between tumor and CAFs, and identifying personalized therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/105fe3ccf7c35aabcf3c81f5242c91ac5e7680b1" target='_blank'>
              Generation of prostate cancer assembloids modeling the patient-specific tumor microenvironment.
              </a>
            </td>
          <td>
            Juhee Lee, Yunhee Kim, Cheol Lee, S. Jeon, Hae Seo, Jongwon Lee, Jungmin Choi, Minyong Kang, Eunjee Kim, Kunyoo Shin
          </td>
          <td>2025-03-31</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are pivotal components of the breast cancer (BC) tumor microenvironment (TME), significantly influencing tumor progression and response to therapy. However, the heterogeneity and specific roles of TAM subpopulations in BC remain inadequately understood. Methods We performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data from BC patients to comprehensively characterize TAM heterogeneity. Utilizing the MetaTiME computational framework and consensus clustering, we identified distinct TAM subtypes and assessed their associations with clinical outcomes and treatment responses. A machine learning-based predictive model was developed to evaluate the prognostic significance of TAM-related gene expression profiles. Results Our analysis revealed three distinct TAM subgroups. Notably, we identified a novel macrophage subtype, M_Macrophage-SPP1-C1Q, characterized by high expression of SPP1 and C1QA, representing an intermediate differentiation state with unique proliferative and oncogenic properties. High infiltration of M_Macrophage-SPP1-C1Q was significantly associated with poor overall survival (OS) and chemotherapy resistance in BC patients. We developed a Random Forest (RF)-based predictive model, Macro.RF, which accurately stratified patients based on survival outcomes and chemotherapy responses, independent of established prognostic parameters. Conclusion This study uncovers a previously unrecognized TAM subtype that drives poor prognosis in BC. The identification of M_Macrophage-SPP1-C1Q enhances our understanding of TAM heterogeneity within the TME and offers a novel prognostic biomarker. The Macro.RF model provides a robust tool for predicting clinical outcomes and guiding personalized treatment strategies in BC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03750-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287d3599b1ab22b44316058fde7d65a5ef96c17d" target='_blank'>
              Integrated single-cell and bulk transcriptomic analysis identifies a novel macrophage subtype associated with poor prognosis in breast cancer
              </a>
            </td>
          <td>
            Qing Wang, Yushuai Yu, Liqiong Ruan, Mingyao Huang, Wei Chen, Xiaomei Sun, Jun Liu, Zirong Jiang
          </td>
          <td>2025-03-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73bf7de11a69b6fed18f060c330341fde4d20ec0" target='_blank'>
              Cellular characterization of the peritumoral brain zone of glioblastomas: heterogenous nature and clinical relevance
              </a>
            </td>
          <td>
            Olaya de Dios, J. Romero, M. A. Ramírez-González, B. Herránz, Alicia Avis, A. Ramos, J. Sepúlveda-Sánchez, R. Gargini, B. Meléndez, Pedro González, L. Jimenez-Roldan, Guillermo García-Posadas, A. Hernández-Laín, Á. Pérez-Núñez, P. Sánchez-Gómez
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The spatial interactions between malignant and immune cells in the tumor microenvironment (TME) play a crucial role in cancer biology and treatment response. Understanding these interactions is critical for predicting prognosis and assessing immunotherapy effectiveness. Conventional methods, which focus on local spatial features, often struggle to achieve robust analysis due to the complex and heterogeneous cellular distributions. We propose a Topological Data Analysis (TDA)-based framework using both global and local spatial features between malignant and immune cells. For the global aspects, we introduce Topological Malignant Clusters (TopMC), a method utilizing persistence diagrams in TDA to capture the global spatial shapes of malignant cell distributions. It quantifies tumor-immune cell infiltration at a global scale by computing distances from the boundaries of the TopMC. Local interactions are evaluated through the density of malignant cells. Using high-resolution multiplex immunofluorescence (mIF) imaging in Diffuse Large B Cell Lymphoma, we integrate distance and density measures into a distance-density space to analyze patterns of malignant-immune cell interactions in TME. This study shows the robustness of the proposed approach to variations in cell distribution, enabling consistent analysis irrespective of whether images are acquired from malignant-enriched or border regions of the tumors. Furthermore, we elucidate the correlation between spatial patterns of immune phenotypes and patient survival probability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f549327f511c89a0763403d23ea2ac5417a3579" target='_blank'>
              Spatial Analysis of Malignant-Immune Cell Interactions in the Tumor Microenvironment Using Topological Data Analysis
              </a>
            </td>
          <td>
            Seol Ah Park, Yongsoo Kim, Paweł Dłotko, Davide Gurnari, Jooyoung Hahn
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Previous studies have highlighted the capacity of brain cancer cells to functionally interact with the tumour microenvironment (TME). This TME-cancer crosstalk crucially contributes to tumour cell invasion and disease progression. In this study, we performed spatial transcriptomic sequencing analysis of a complete annotated tumour-infiltrated brainstem from a single diffuse intrinsic pontine glioma (DIPG) patient. Gene signatures from ten sequential tumour regions were analysed to assess mechanisms of disease progression and oncogenic interactions with the TME. We identified four distinct tumour subpopulations and assessed respective ligand-receptor pairs that actively promote DIPG tumour progression via crosstalk with endothelial, neuronal and immune cell communities. Our analysis found potential targetable mediators of tumour-to-TME communication, including members of the complement component system and the neuropeptide/GPCR ligand-receptor pair ADCYAP1-ADCYAP1R1. These interactions could influence DIPG tumour progression and represent novel therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01952-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ea420e4a0256eeeba7153fed74e4eb3b92c386" target='_blank'>
              Spatial analysis of a complete DIPG-infiltrated brainstem reveals novel ligand-receptor mediators of tumour-to-TME crosstalk
              </a>
            </td>
          <td>
            Anja Kordowski, Onkar Mulay, Xiao Tan, Tuan Vo, Ulrich Baumgartner, Mellissa Maybury, Timothy Hassall, L. Harris, Quan Nguyen, B. Day
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e047830d6d15c84fdba5a27411c84ea5a08197" target='_blank'>
              Multi-Modal Disentanglement of Spatial Transcriptomics and Histopathology Imaging
              </a>
            </td>
          <td>
            Hassaan Maan, Zongliang Ji, Elliot Sicheri, Tiak Ju Tan, Alina Selega, Ricardo Gonzalez, Rahul G Krishnan, Bo Wang, Kieran R. Campbell
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f30684782f70f49100b5a4c8e6aa8a01fed878d3" target='_blank'>
              Single-cell dissection of the genotype-immunophenotype relationship in glioblastoma.
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, Dolores Hambardzumyan, A. Tsankov
          </td>
          <td>2025-04-11</td>
          <td>Brain : a journal of neurology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are highly heterogeneous cells and important components of the breast tumor microenvironment (TME). However, their role and clinical value in ER-positive breast cancer have not been fully clarified. Our study aims to comprehensively characterize the heterogeneity, potential biological functions, and molecular mechanisms of CAFs in ER-positive breast cancer within the tumor microenvironment using multi-omics data, to provide new strategies for the diagnosis and treatment of ER-positive breast cancer patients. In this study, we found that COL1A2(+) MMP1(+) and COL1A2(+) MMP1(-) CAFs were associated with unfavorable prognosis. The dynamic evolution and cell-cell communications of CAFs were analyzed, revealing that COL1A2(+) MMP1(+/-) CAFs show extensive crosstalk with tumor-associated macrophages (TAMs), contributing to an immunosuppressive TME. Moreover, the somatic mutation of TP53 may be a potential indicator for evaluating the infiltration of COL1A2(+) MMP1(+/-) CAFs. Finally, an MRI-based radiomic model was constructed to estimate the abundance of these CAFs. In conclusion, our findings provide a theoretical basis for targeting CAFs and offer a noninvasive approach to evaluate the infiltration level of COL1A2(+) MMP1(+/-) CAFs. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03705-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Osteosarcoma (OS), a prevalent malignant bone tumor, has seen limited progress in treatment efficacy and patient outcomes over decades. Recent insights into the tumor microenvironment (TME) have revealed its crucial role in tumor progression and therapeutic resistance, particularly in OS. This review offers a comprehensive exploration of the OS microenvironment, meticulously dissecting its crucial components: the mesenchymal stromal TME, the immune microenvironment, hypoxia-induced adaptations, and the impact of the physical microenvironment. By demonstrating how these elements collectively drive tumor proliferation, immune evasion, and invasion, this review explores the intricate molecular and cellular dynamics at play. Furthermore, innovative approaches targeting the OS microenvironment, such as immunotherapies, are presented. This review highlights the importance of the TME in OS progression and its potential as a source of novel therapeutic strategies, offering new hope for improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66a91e822690718ee9aef6926172cee6b0fe48f6" target='_blank'>
              Microenvironment matters: insights from the FOSTER consortium on microenvironment-driven approaches to osteosarcoma therapy
              </a>
            </td>
          <td>
            Aurélie Dutour, M. Pasello, Luke Farrow, Mahetab Amer, N. Entz-Werlé, Michaela Nathrath, Katia Scotlandi, Sibylle Mittnacht, Anne Gomez-Mascard
          </td>
          <td>2025-04-10</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spatial transcriptomics is a cutting-edge technology that analyzes gene expression at the cellular level within tissues while integrating spatial location information. This concept, which combines high-plex RNA sequencing with spatial data, emerged in the early 2010s. Spatial transcriptomics has rapidly expanded with the development of technologies such as in situ hybridization, in situ sequencing, in situ spatial barcoding, and microdissection-based methods. Each technique offers advanced mapping resolution and precise spatial assessments at the single-cell level. Over the past decade, the use of spatial transcriptomics on clinical samples has enabled researchers to identify gene expressions in specific diseased foci, significantly enhancing our understanding of cellular interactions and disease processes. In the field of rheumatology, the complex and elusive pathophysiology of diseases such as rheumatoid arthritis, systemic lupus erythematosus, and Sjögren’s syndrome remains a challenge for personalized treatment. Spatial transcriptomics provides insights into how different cell populations interact within disease foci, such as the synovial tissue, kidneys, and salivary glands. This review summarizes the development of spatial transcriptomics and current insights into the pathophysiology of autoimmune rheumatic diseases, focusing on immune cell distribution and cellular interactions within tissues. We also explore the potential of spatial transcriptomics from a clinical perspective and discuss the possibilities for translating this technology to the bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64291da16ef528fb561e1af0f4b9ad8a8ddde042" target='_blank'>
              Comprehensive Integrated Analysis Reveals the Spatiotemporal Microevolution of Cancer Cells in Patients with Bone-Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Yinghua Feng, Xiuli Zhang, Guangpeng Wang, Fei-ya Yang, Ruifang Li, Lu Yin, Dong Chen, Wenkuan Wang, Mingshuai Wang, Zhiyuan Hu, Yuan Sh, N. Xing
          </td>
          <td>2025-04-09</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNAseq) of human tissues has expanded our understanding of the complexity of cellular subsets and their changes in disease. The availability of scRNAseq data in different tissues and disease states provides an opportunity to compare cellular subsets and identify common and unique cellular activation. In this study, we aimed to characterize shared and tissue-specific myeloid and stromal phenotypes and uncover key cellular subtypes involved in pathogenic tissue activation. We analyzed scRNAseq data from 14 public datasets, comprising heart, lung, skin, and synovium in both healthy and diseased states. Our analysis identified distinct and overlapping myeloid and stromal cell populations in these tissues. Despite significant inter-individual variability, we were able to identify both shared and disease-specific changes in these cell populations. These findings provide insights into the conserved and tissue-specific roles of myeloid and stromal cells in health and disease and contribute to a better understanding of tissue pathology and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a4d83ee093d91b7fc9e6f8d4d131194bc8ae33" target='_blank'>
              Human Fibroblast-Myeloid cell tissue atlas across lung, synovium, skin and heart
              </a>
            </td>
          <td>
            L. MacDonald, Olympia M Hardy, Melpomeni Toitou, Gabriela Kania, P. Błyszczuk, R. Micheroli, Thomas D. Otto, Mariola Kurowska-Stolarska, C. Ospelt
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Adoptive cell therapies (ACT) leverage tumor-immune interactions to cure cancer. Despite promising phase I/II clinical trials of chimeric-antigen-receptor natural killer (CAR-NK) cell therapies, molecular mechanisms and cellular properties required to achieve clinical benefits in broad cancer spectra remain underexplored. While in vitro and in vivo experiments are required in this endeavor, they are typically expensive, laborious, and limited to targeted investigations. Here, we present ABMACT (Agent-Based Model for Adoptive Cell Therapy), an in silico approach employing agent-based models (ABM) to simulate the continuous course and dynamics of an evolving tumor-immune ecosystem, consisting of heterogeneous “virtual cells” created based on knowledge and omics data observed in experiments and patients. Applying ABMACT in multiple therapeutic context indicates that to achieve optimal ACT efficacy, it is key to enhance immune cellular proliferation, cytotoxicity, and serial killing capacity. With ABMACT, in silico trials can be performed systematically to inform ACT product development and predict optimal treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a7a9fa08cfb61e58044a46d5cd717b6f33a5eb" target='_blank'>
              Agent-based modeling of cellular dynamics in adoptive cell therapy
              </a>
            </td>
          <td>
            Yujia Wang, Stefano Casarin, May Daher, V. Mohanty, Merve Dede, M. Shanley, R. Başar, Katayoun Rezvani, Ken Chen
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Immune cells exhibit functional heterogeneity beyond what is resolved by classical definitions of subpopulations based on cell surface expression of receptors. To develop efficient and personalized cell-based immunotherapies, we need to resolve this heterogeneity and understand the underlying parameters that dictate cellular responses to specific target cells. For this, new methods are required that can identify and harvest immune cells with specific functions, e.g., high cytotoxic potential, to form clonally expanded cells or to assess molecular or genetic signatures. In this study, we evaluate a system for non-destructive, live cell picking and release in deep, high-aspect ratio microwells and test it for isolation of individual natural killer (NK) cells. We assess its performance at retrieving and releasing beads from microwells and demonstrate its potential for single NK cell isolation with intact viability. We also implement a semi-automated workflow for functional single-cell screening of NK cell behavior in microwell arrays followed by single-cell identification and isolation, demonstrating the potential for functional screening and isolation of serial killing immune cells. Our evaluation concludes that this cell isolation system, in combination with microwell arrays, offers opportunities for improved understanding of NK cell biology with applications towards cell therapy. However, its limited throughput hinders large-scale applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79619eb3501acae9caddc99dc83ce718f2ee3c" target='_blank'>
              Isolation of individual natural killer cells from deep, high-aspect ratio microwell arrays - an evaluative study
              </a>
            </td>
          <td>
            Quentin Verron, Niklas Sandström, Hanna van Ooijen, K. Guldevall, K. Olofsson, T. Frisk, Björn Önfelt
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Extrahepatic cholangiocarcinoma (eCCA) is a rare but refractory cancer with dense desmoplasia. Prognosis-associated stromal cells in eCCA remain poorly characterized. Here, we profiled the tumor cellular composition and identified prognosis-related stromal signatures by single-cell RNA sequencing (scRNA-seq) in eCCA. ECCA patients were further stratified into different categories based on identified stromal signatures. Methods Using scRNA-seq, we profiled the transcriptomes of 37,498 individual cells from eight eCCA biopsies, including five tumor tissues and three paired adjacent normal tissues. Bulk RNA sequencing (bRNA-seq) was also performed on 43 eCCA tumor tissues. Stromal cell composition and heterogeneity were examined through differential gene expression and gene set enrichment analyses. By assessing the expression levels of marker genes in bRNA-seq data, the correlation of stromal cell clusters with survival was explored. The GSVA scores of the cell-specific signature genes of the prognosis-related stromal cell subtypes were calculated and used to stratify eCCA patients. Results The results revealed that tumor stroma in eCCA were composed of hematopoietic progenitor-like cells (HPLCs), fibroblasts (Fb), Schwann cells (Sch), endothelial cells and immune cells. Prognosis-associated stromal cell subpopulations included MKI67 + HPLC, TMEM158 + C3-Fb, FOXP3 + regulatory T cells (Treg), SLIT2 + Sch, TPSD1 + C2-mast cells (MC) and CTSG + C3-MC. Based on these stromal signatures, the eCCA tumors were categorized into three classes: proliferative Group 1 with enrichment of MKI67 + HPLC, inflammatory and fibrotic Group 2 with enrichment of TPSD1 + C2- MC, FOXP3 + Treg and TMEM158 + C3-Fb, and neuronal Group 3 with enrichment of SLIT2 + Sch and CTSG + C3-MC. ECCA patients in Group 3 had a better prognosis when compared to Group 1 and 2, reflecting different impact of stromal subtypes on tumor progression. Conclusion Single-cell transcriptomic analysis reveals prognosis-related stromal signatures that potentiate the stratification of eCCA into proliferative, inflammatory and fibrotic, and neuronal phenotypes, which has important implications on molecular classification and exploring therapeutic targets in eCCA. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-03829-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cc06851451f803822f912fc1af35b083131a67" target='_blank'>
              Single-cell transcriptomic analysis reveals prognosis-related stromal signatures that potentiate stratification of patients with extrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Chen Ni, Rulin Hua, Yuanyuan Yang, Jialu Liang, Wentao Liu, Linlin Wang, Xiaohan Yao, Anqi Li, Long Yu, Ruo Feng, Dekang Lv, Zhihai Qin, Wenlong Zhai
          </td>
          <td>2025-04-09</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8779d676df52501d7ec9c3387c761533f91daa90" target='_blank'>
              Interstitial fluid transport dynamics predict glioblastoma invasion and progression
              </a>
            </td>
          <td>
            Cora M. Carman-Esparza, Caleb A. Stine, Naciye Atay, K. Kingsmore, Maosen Wang, Ryan T. Woodall, Russell C. Rockne, Jessica J. Cunningham, Jenny Munson
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520e7a98e4f661979d2c0ecb98d246afd9f267aa" target='_blank'>
              Translating genetics into tissue: inflammatory cytokine-producing TAMs and PD-L1 tumor expression as poor prognosis factors in cutaneous melanoma
              </a>
            </td>
          <td>
            Celia Barrio-Alonso, Alicia Nieto-Valle, Lucía Barandalla-Revilla, J. Avilés-Izquierdo, V. Parra-Blanco, Paloma Sánchez-Mateos, Rafael Samaniego
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, Europa Azucena González, Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09798a1f6e296d417f2e14a0cda8a7916141ba6" target='_blank'>
              Abstract B114: A 3D Bioprinted Vascularized Tumor-on-a-Chip model for Immuno-Oncology therapeutic development: Recapitulating the Tumor Microenvironment and Immune cell dynamics
              </a>
            </td>
          <td>
            Juliana Navarro Yepes, Queeny Dasgupta, Purboja Purkayastha, Carly Truong, YongDuk Han, Michael Doerfert, C. Mello, Kevin Janson, Ameya Narkar, Sammy S. Datwani, S. Ng, Taci Pereira
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f69de6f588f5dc88e965e9e468e394e888395b6" target='_blank'>
              Single-cell and spatial transcriptomics reveal a tumor-associated macrophage subpopulation that mediates prostate cancer progression and metastasis.
              </a>
            </td>
          <td>
            Shenglin Mei, Hanyu Zhang, Taghreed Hirz, N. Jeffries, Yanxin Xu, Ninib Baryawno, Shulin Wu, Chin-Lee Wu, Akash Patnaik, P. Saylor, David B. Sykes, Douglas M Dahl
          </td>
          <td>2025-03-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Despite the outstanding clinical success of immunotherapy, its therapeutic efficacy in glioblastoma (GBM) is still limited. To identify critical regulators of GBM immunity, we constructed a mouse single-guide RNA (sgRNA) library corresponding to all disease-related immune genes, and performed an in vivo CRISPR knockout (KO) screen in syngeneic GBM mouse models. We demonstrated that the deletion of GDF15 in GBM cells ameliorated the immunosuppressive tumor microenvironment (TME) and enhanced the antitumor efficacy of immune checkpoint blockade (ICB) response. Moreover, we designed unique nanoparticles for efficient encapsulation of CRISPR-Cas9, noninvasive brain delivery and tumor cell targeting, demonstrating an effective and safe strategy for GDF15 gene therapy. The CRISPR-Cas9 nanoparticles, known as ANPSS (Cas9/sgRNA), are easily created by enclosing a single Cas9/sgRNA complex in a polymer shell that is sensitive to glutathione. This shell also contains a dual-action ligand that aids in crossing the blood‒brain barrier, targeting tumor cells, and selectively releasing Cas9/sgRNA. Our encapsulating nanoparticles demonstrated promising GBM targeting, resulting in high GDF15 gene editing efficiency within brain tumors while showing minimal off-target gene editing in high-risk tissues. Treatment with ANPSS (Cas9/sgGDF15) effectively halted tumor growth, reversed immune suppression, and enhanced the efficacy of ICB therapy. These results emphasize the potential role of GDF15 in modulating the immune microenvironment and enhancing the effectiveness of current immunotherapy strategies for GBM. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12951-025-03182-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91dbf9f6daabb9cf44437ddfd0f2bfccdebe0fb" target='_blank'>
              Targeting GDF15 to enhance immunotherapy efficacy in glioblastoma through tumor microenvironment-responsive CRISPR-Cas9 nanoparticles
              </a>
            </td>
          <td>
            Cheng Zou, Xiao Liu, Weizhong Wang, Lei He, Anan Yin, Zhen Cao, Maorong Zhu, Yuxin Wu, Xiaolin Liu, Jiying Ma, Yalong He, Shuning Wang, Wangqian Zhang, Wei Liu, Yingqi Zhang, Jintao Gu, Wei Lin, Kuo Zhang, Meng Li
          </td>
          <td>2025-02-20</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Different patterns of immune and stromal interactions within the tumor microenvironment significantly impact tumor behavior and response to treatment. Recent research has identified distinct immune subtypes—immune desert, excluded, and inflamed—each associated with specific mechanisms that impede anti-tumor immune responses. In the era of precision medicine, accurately stratifying patients into these subtypes is essential for optimizing cancer management and personalizing immunotherapy. The research team at National Cancer Institute in Lithuania addresses critical gaps in this field by focusing on several key areas. Our first focus is on developing reliable and accessible tools for determining patient immune subtypes. We are integrating transcriptomics and histology data to create a robust subtyping pipeline, which we are validating with real-world data and streamlining for clinical use. Another aspect of our work involves creating translatable preclinical models for cancer immunotherapy through utilizing 3D cell cultures and biocompatible scaffolds. We are developing druggable syngeneic murine tumor models that accurately reflect the three immune subtypes, enhancing the relevance of preclinical studies. Using both pharmaceutical and non-pharmaceutical approaches, we are tailoring immunomodulatory interventions to address the unique characteristics of each immune subtype. Joint efforts in these research areas could generate translatable insights, contributing to more precise cancer therapies and overcoming resistance mechanisms. 
  
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8e8d206eced9c976c9ce13db22dd5e4aadd38" target='_blank'>
              Deciphering immune tumor subtypes through microenvironment profiling
              </a>
            </td>
          <td>
            A. Mlynska, Eglė Žymantaitė, Austeja Butkute, N. Dobrovolskienė, J. Krasko, Olha Karaman, Karolina Suveizdė, Beatrice Gudaite, Margarita Žvirblė, Emilija Paberalė, Nijole Matuseviciene, V. Pašukonienė
          </td>
          <td>2025-03-25</td>
          <td>South East European Journal of Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Advances in tissue labeling, imaging, and automated cell identification now enable the visualization of immune cell types in human tumors. However, a framework for analyzing spatial patterns within the tumor microenvironment (TME) is still lacking. To address this, we develop Spatiopath, a null-hypothesis framework that distinguishes statistically significant immune cell associations from random distributions. Using embedding functions to map cell contours and tumor regions, Spatiopath extends Ripley’s K function to analyze both cell-cell and cell-tumor interactions. We validate the method with synthetic simulations and apply it to multi-color images of lung tumor sections, revealing significant spatial patterns such as mast cells accumulating near T cells and the tumor epithelium. These patterns highlight differences in spatial organization, with mast cells clustering near the epithelium and T cells positioned farther away. Spatiopath enables a better understanding of immune responses and may help identify biomarkers for patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80949e1640c0e5682964239d082757c3241b697f" target='_blank'>
              Statistical analysis of spatial patterns in tumor microenvironment images
              </a>
            </td>
          <td>
            Mohamed M Benimam, V. Meas-Yedid, Suvadip Mukherjee, Astri Frafjord, Alexandre Corthay, T. Lagache, Jean-Christophe Olivo-Marin
          </td>
          <td>2025-03-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="BACKGROUND
The heterogeneity of tumors significantly impacts on colorectal cancer (CRC) progression. However, the influence of this heterogeneity on the spatial architecture of CRC remains largely unknown.


METHODS
Spatial transcriptomic (ST) analysis of AOM/DSS-induced colorectal cancer (CRC), integrated with single-cell RNA sequencing, generated a comprehensive spatial atlas of CRC. Pseudotime trajectory, stemness evaluation, and cell-cell communication analyses explored how CD44+ tumor cells at the leading edge remodel the tumor microenvironment (TME). In vitro experiments and immunofluorescence staining of clinical samples validated pleiotrophin (PTN) signaling in promoting cancer-associated fibroblasts (CAFs) phenotypic transition and CRC progression.


RESULTS
Our findings revealed a distinctive layered ring-like structure within CRC tissues, where CD44+ tumor cells exhibiting high stemness were positioned at the tumor's leading edge. Inflammatory CAFs (iCAFs)-like, myofibroblastic CAFs (myCAFs)-like cells and pro-tumorigenic neutrophils primarily located at the tumor edge, in proximity to CD44+ tumor cells. CD44+ tumor cells then triggered the phenotypic transition of CAFs into iCAF-like and myCAF-like cells through PTN signaling.


CONCLUSIONS
Our results provide distinctive insights into how tumor heterogeneity reshapes the TME at the leading edge of tumor, thereby promoting CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge.
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background The identification of molecular features characterizing metastatic disease is a critical area of oncology research, as metastatic foci often exhibit distinct biological behaviors compared to primary tumors. While the focus has largely been on the neoplastic cells themselves, the characterization of the associated stroma remains largely underexplored, with significant implications for understanding metastasis. Main body By employing spatially resolved transcriptomics, we analyzed the transcriptional features of primary breast adenocarcinoma and its associated metastatic foci, on a representative set of microregions. We identified a stromal metastatic (Met) signature, which was subsequently validated across transcriptomic reference human breast cancer (BC) datasets and in spatial transcriptomics of a murine model. Conclusion We discuss the potential of a stromal Met signature to pinpoint metastatic breast cancer, serving as a prognostic tool that can provide a foundation for the exploration of tumor-extrinsic molecular hallmarks of BC metastatic foci. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03353-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabdeb7c0a8725d1b43bd88dd9faad675928ee91" target='_blank'>
              Extraction of a stromal metastatic gene signature in breast cancer via spatial profiling
              </a>
            </td>
          <td>
            G. Bertolazzi, Valeria Cancila, Davide Vacca, Beatrice Belmonte, D. Lecis, Parsa Sirati Moghaddam, A. Di Napoli, M. P. Colombo, G. Pruneri, Giannino Del Sal, Giorgio Scita, Matteo Fassan, A. Vecchione, Silvio Bicciato, Claudio Tripodo
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background As a type of autophagy, aggrephagy degrades the aggregation of misfolded protein in cells and plays an important role in key genetic events for various cancers. However, aggrephagy functions within the tumor microenvironment (TME) in endometrial cancer (EC) remain to be elucidated. Methods A total of 36,227 single cells from single-cell RNA-seq data derived from five EC tumor samples were comprehensively analyzed using a nonnegative matrix factorization (NMF) algorithm for 44 aggrephagy-related genes. Bulk RNA-seq cohorts from public repositories were utilized to assess the prognostic value of aggrephagy-related TME clusters and predict immune checkpoint blockade immunotherapeutic response in EC. Results Fibroblasts, macrophages, CD8+T cells, and lymphatic endothelial cells were categorized into two to five aggrephagy-related subclusters, respectively. CellChat analysis showed that the aggrephagy-related subtypes of TME cells exhibited extensive interactions with tumor epithelial cells, particularly for macrophages. Moreover, aggrephagy regulators may be significantly associated with the pseudotime trajectories of major TME cell types as well as the clinical and biological features of EC. Bulk-seq analysis showed that these aggrephagy-related subclusters had significant predictive value for the survival and immune checkpoint blockade response in EC patients. Notably, immunohistochemical staining results manifested that the TUBA1A+ macrophage subtype was linked to less lymph node metastasis and longer survival, which were consistent with the bioinformatics analysis findings. Conclusions This study provided a novel view of aggrephagy signaling in the EC tumor microenvironment, and intervention of aggrephagy was expected to improve the survival rate of EC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539a7161f9d5e59af0e8571e13d64e98e255b165" target='_blank'>
              Single-cell sequencing reveals the role of aggrephagy-related patterns in tumor microenvironment, prognosis and immunotherapy in endometrial cancer
              </a>
            </td>
          <td>
            Yuquan Yuan, Chunyan Ren, Jin Shu, Keyang Zhu, Ganghui Li, Bao Liu, Jianrong Huang, Yinde Huang, Chengzhi Zhao
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc8b763d2e3ec3323e5145b28e5d2c4495cc407" target='_blank'>
              Abstract B040: Transcriptomic meta-analysis identifies robust, tissue-agnostic gene expression signature of immune checkpoint blockade response
              </a>
            </td>
          <td>
            Y. Guo, Aaron C Tan, Tanmay Kulshrestha, M. Chang, Dawn Lau, Lan Ying Wang, Timothy Tay, Irfahan Kassam, Egor Revkov, Simone Rizzetto, I. Tan, Daniel S W Tan, A. Skanderup
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer-associated fibroblast (CAF) recruitment and activation within the tumor microenvironment (TME) are increasingly acknowledged as drivers of oral squamous cell carcinoma (OSCC) tumor growth and metastasis. Therefore, the mechanisms underlying tumor cell and fibroblast crosstalk warrant further investigation. We discovered that ectopic interferon-stimulated gene 15 (ISG15) expression, which is a promising and novel oncoprotein biomarker elevated in a variety of cancers, enhanced OSCC growth and elevated collagen and α-smooth muscle actin (α-SMA) expression in ISG15-expressing tumors. Analysis of immunohistochemistry revealed high ISG15 expression in human oral tissues correlated with high expression of α-SMA and fibroblast activation protein (FAP). Fibroblast migration and recruitment by ISG15-expressing OSCC cells were confirmed by in vitro and in vivo experiments. Exogenous ISG15 induced fibroblast migration, morphological changes, and vimentin expression. Enrichment of glycolysis pathway genes, as well as increased glycolysis-related gene expression, glucose uptake, and lactate production were observed in ISG15-treated fibroblasts. Lactate release and fibroblast migration were blocked by a competitive inhibitor of glucose metabolism. Furthermore, the knockdown of integrin αL (ITGAL)/CD11a, a subunit of ISG15 receptor lymphocyte functional-associated antigen-1 (LFA-1), in immortalized fibroblasts diminished extracellular ISG15-mediated glycolysis and migration. Our findings suggest that ISG15 derived from OSCC cells interacts with fibroblasts through the LFA-1 receptor, leading to glycolytic reprogramming and promotion of fibroblast migration into the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cdbb5e2d18defd866a0ed69744c9ad3161e60de" target='_blank'>
              Tumor cell-derived ISG15 promotes fibroblast recruitment in oral squamous cell carcinoma via CD11a-dependent glycolytic reprogramming
              </a>
            </td>
          <td>
            Ssu‐Han Wang, Yu-lin Chen, Shih-Han Huang, Yu-Ke Fu, Su-Fang Lin, Shih-Sheng Jiang, Shu-Chen Liu, J. Hsiao, Jang-Yang Chang, Ya-Wen Chen
          </td>
          <td>2025-03-11</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Background Tumor-infiltrating lymphocytes (TILs) and tumor-associated macrophages (TAMs) have increasingly been reported to impact the brain metastatic process of solid tumors. However, data on intra-individual differences between primary tumor and brain metastasis (BM), as well as their correlation with clinical outcome parameters, is scarce. Methods We retrospectively identified patients who received resection of the primary tumor and BM between 01/1990 and 10/2022. Density quantification of TAMs (CD68+, CD163+) and TILs (CD3+, CD8+, CD45RO+, FOXP3+) was performed by immunohistochemical staining of matched tumor tissue samples. Images were processed with QuPath software and heterogeneity of generated heatmaps was measured by Shannon Entropy. Time-to-BM (TTBM) was defined as the time from diagnosis of the primary tumor until the first diagnosis of BM. Results In total, 104 patients (46.2% female; median age 57.3 years at BM diagnosis) were included: 78/104 (75%) non-small cell lung cancer, 18/104 (17%) breast cancer, 8/104 (8%) renal cell carcinomas. Densities of CD3+ (p < 0.001) and CD8+-TILs (p < 0.001) were higher in primary tumor samples, while CD68+ (p = 0.035) and CD163+-TAM densities (p < 0.001) were higher in the matched BM. Higher CD3+, CD8+-TILs and CD163+-TAMs densities in primary tumors were associated with shorter TTBM (p = 0.005, p = 0.015 and p = 0.006, respectively). Higher entropies of CD3+ (p < 0.001) and FOXP3+ (p = 0.011) TILs were observed in primary tumors compared to BM. Longer TTBM was associated with higher entropy of FOXP3+ TILs (p = 0.024) and lower entropy in CD163+ TAMs (p = 0.039). No significant associations of immune cell densities or entropies with OS after BM diagnosis were found. Discussion By utilizing a unique cohort of matched primary tumor and BM tissue samples, we could demonstrate higher TIL densities in primary tumors and higher TAM densities in BM, respectively. Higher cell densities of CD3+, CD8+-TILs and CD163+-TAMs in primary tumors were associated with shorter TTBM, while a larger difference between CD3+ and CD8+ densities between primary tumor and BM was associated with longer TTBM. These findings highlight the potential of targeting TAMs as a therapeutic strategy to mitigate the development of brain metastases. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01939-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91820cda6d428b7441857912e71a4c3a105594b5" target='_blank'>
              Density and entropy of immune cells within the tumor microenvironment of primary tumors and matched brain metastases
              </a>
            </td>
          <td>
            M. Kleinberger, D. Cifci, C. Paiato, E. Tomasich, M. Mair, A. Steindl, Zoltán Spiró, Z. I. Carrero, L. Berchtold, J. Hainfellner, L. Müllauer, G. Heller, M. Preusser, Jakob N Kather, A. Berghoff
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Abstract Motivation Immune cells undergo cytokine-driven polarization in response to diverse stimuli, altering their transcriptional profiles and functional states. This dynamic process is central to immune responses in health and diseases, yet a systematic approach to assess cytokine-driven polarization in single-cell RNA sequencing data has been lacking. Results To address this gap, we developed single-cell unified polarization assessment (Scupa), the first computational method for comprehensive immune cell polarization assessment. Scupa leverages data from the Immune Dictionary, which characterizes cytokine-driven polarization states across 14 immune cell types. By integrating cell embeddings from the single-cell foundation model Universal Cell Embeddings, Scupa effectively identifies polarized cells across different species and experimental conditions. Applications of Scupa in independent datasets demonstrated its accuracy in classifying polarized cells and further revealed distinct polarization profiles in tumor-infiltrating myeloid cells across cancers. Scupa complements conventional single-cell data analysis by providing new insights into dynamic immune cell states, and holds potential for advancing therapeutic insights, particularly in cytokine-based therapies. Availability and implementation The code is available at https://github.com/bsml320/Scupa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/078b81c851163489c04007fcd2f4d464e0013dc1" target='_blank'>
              Scupa: single-cell unified polarization assessment of immune cells using the single-cell foundation model
              </a>
            </td>
          <td>
            Wendao Liu, Zhongming Zhao
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a clinically heterogeneous malignancy with diverse patient outcomes, largely influenced by the tumor microenvironment (TME). Understanding the roles of fibroblasts and macrophages within the TME is essential for developing personalized therapeutic strategies in DLBCL. Methods This study is a multi-omics approach, integrating spatial transcriptomics (n = 11), bulk transcriptomics (n = 2,499), immunohistochemistry (IHC, n = 37), multiplex immunofluorescence (mIF, n = 56), and plasma samples (n = 240) to identify and characterize fibroblast and tumor-associated macrophage subtypes in the TME. Hub genes for LYZ+ fibroblasts and FN1+ macrophages were selected through univariate Cox regression and random forest analyses. Their prognostic significance was validated using IHC, mIF, and autoantibody assays in DLBCL patients treated with R-CHOP and in non-small cell lung cancer (NSCLC) patients receiving immune checkpoint inhibitors (ICIs). Results Fibroblasts and macrophages were classified into two distinct subtypes. Patients with higher LYZ+ fibroblasts infiltration demonstrated superior prognosis, which was associated with increased infiltration of FN1+ macrophages. Key hub genes identified for LYZ+ fibroblasts included LYZ, ANPEP, CSF3R, C15orf48, LILRB4, CLEC7A, and COL7A1, while hub FN1+ macrophages genes included COL1A1, FN1, APOE, DCN, MMP2, SPP1, COL3A1, and COL1A2. Independent prognostic markers in DLBCL treated with R-CHOP and NSCLC treated with ICIs were identified, including LYZ and LILRB4 at both protein and mRNA levels, and COL1A2 autoantibodies (p < 0.05). In DLBCL patients treated with R-CHOP, FN1 mRNA and autoantibody levels were also prognostic markers (p < 0.05). In NSCLC treated with ICIs, COL3A1 autoantibody was prognostic marker (p < 0.05). Conclusions This study identified a prognostically relevant LYZ+ fibroblasts and FN1+ macrophages in DLBCL. The hub genes associated with these subtypes represent potential biomarkers, providing insights into improving patient outcomes in DLBCL. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-03968-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven M Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Recent advancements have revealed the presence of a microbiome within tumor tissues, underscoring the crucial role of the tumor microbiome in the tumor ecosystem. This review delves into the characteristics of the intratumoral microbiome, underscoring its dual role in modulating immune responses and its potential to both suppress and promote tumor growth. We examine state-of-the-art techniques for detecting and analyzing intratumoral bacteria, with a particular focus on their interactions with the immune system and the resulting implications for cancer prognosis and treatment. By elucidating the intricate crosstalk between the intratumoral microbiome and the host immune system, we aim to uncover novel therapeutic strategies that enhance the efficacy of cancer treatments. Additionally, this review addresses the existing challenges and future prospects within this burgeoning field, advocating for the integration of microbiome research into comprehensive cancer therapy frameworks. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5406504325c9482a7770db54158f642d5bcd73b1" target='_blank'>
              Intratumoral microbiome: implications for immune modulation and innovative therapeutic strategies in cancer
              </a>
            </td>
          <td>
            Na Wang, Si Wu, Lanxiang Huang, Yue Hu, Xin He, Jourong He, Ben Hu, Yaqi Xu, Yuan Rong, Chunhui Yuan, Xiantao Zeng, Fubing Wang
          </td>
          <td>2025-02-19</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Laryngeal squamous cell carcinoma (LSCC) is a highly aggressive malignancy with a rising incidence over time. The tumor microenvironment (TME) plays a crucial role in LSCC development, yet the precise cellular characteristics of laryngeal cancer and its TME remain unclear. Here, we employed single-cell RNA sequencing analysis to uncover the heterogeneous populations of tumor and immune cells and investigate the role of the TME in LSCC. This analysis revealed significant heterogeneity among malignant cells, T cells, and macrophages. Notably, regulatory T cells were markedly increased at tumor sites, and macrophage analysis identified an increased presence of the Macrophage-C1-C1QC subset with up-regulated PPIF expression. Bulk RNA-seq further confirmed PPIF up-regulation in exosomes derived from LSCC tissues. Consistently, survival analysis indicated that high PPIF expression was associated with poor prognosis in LSCC. Further analyses suggested that PPIF up-regulation in Macrophage-C1-C1QC cells was associated with the enhancement of their anti-inflammatory phenotype and the promotion of F11R-F11R signaling with malignant cells, allowing LSCC cells to evade macrophage-mediated cytotoxicity. Our study provides new insights into the cellular dynamics of LSCC and highlights the critical role of Macrophage-C1-C1QC and PPIF in LSCC progression, offering potential therapeutic targets for treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93584-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301c190dc2f8d57f1a436734bdde8342d0e72d" target='_blank'>
              Unveiling the role of PPIF and macrophage subtypes in LSCC progression via single-cell and exosome RNA sequencing
              </a>
            </td>
          <td>
            Jianchao Wang, Chunhan Liu, Tailin Wu, Lei Tan, Jiangyan Wang, Wenliang Zhang, Yang Liu, Xudong Wei, Di Zhang
          </td>
          <td>2025-03-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed6afc1fbea4684dd9546a568ad33c0ccc1b6d" target='_blank'>
              Mapping molecular landscapes in triple-negative breast cancer: insights from spatial transcriptomics.
              </a>
            </td>
          <td>
            F. Al-Mansour, H. Almasoudi, Ali M Albarrati
          </td>
          <td>2025-03-22</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diabetes mellitus (DM) is a global health crisis affecting millions, with islet transplantation emerging as a promising treatment strategy to restore insulin production. This review synthesizes the current research on single-cell and spatial transcriptomics in the context of islet transplantation, highlighting their potential to revolutionize DM management. Single-cell RNA sequencing, offers a detailed look into the diversity and functionality within islet grafts, identifying specific cell types and states that influence graft acceptance and function. Spatial transcriptomics complements this by mapping gene expression within the tissue’s spatial context, crucial for understanding the microenvironment surrounding transplanted islets and their interactions with host tissues. The integration of these technologies offers a comprehensive view of cellular interactions and microenvironments, elucidating mechanisms underlying islet function, survival, and rejection. This understanding is instrumental in developing targeted therapies to enhance graft performance and patient outcomes. The review emphasizes the significance of these research avenues in informing clinical practices and improving outcomes for patients with DM through more effective islet transplantation strategies. Future research directions include the application of these technologies in personalized medicine, developmental biology, and regenerative medicine, with the potential to predict disease progression and treatment responses. Addressing ethical and technical challenges will be crucial for the successful implementation of these integrated approaches in research and clinical practice, ultimately enhancing our ability to manage DM and improve patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d2a871c9156c1f2749ccafc8536877666a5a4e" target='_blank'>
              Single-cell genomics and spatial transcriptomics in islet transplantation for diabetes treatment: advancing towards personalized therapies
              </a>
            </td>
          <td>
            Lisha Mou, Tony Bowei Wang, Yuxian Chen, Ziqi Luo, Xinyu Wang, Zuhui Pu
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="T cells play diverse roles in cancer immunology, acting as tumor suppressors, cytotoxic effectors, enhancers of cytotoxic T lymphocyte responses and immune suppressors; providing memory and surveillance; modulating the tumor microenvironment (TME); or activating innate immune cells. However, cancer cells can disrupt T cell function, leading to T cell exhaustion and a weakened immune response against the tumor. The expression of exhausted T cell (Tex) markers plays a pivotal role in shaping the immune landscape of multiple cancers. Our aim was to systematically investigate the role of known T cell exhaustion (Tex) markers across multiple cancers while exploring their molecular interactions, mutation profiles, and potential implications for immunotherapy. The mRNA expression profile of six Tex markers, LAG-3, PDCD1, TIGIT, HAVCR2, CXCL13, and LAYN was investigated in pan-cancer. Utilizing data from The Cancer Genome Atlas (TCGA), Gene Expression Omnibus (GEO), The Cancer Proteome Atlas (TCPA), and other repositories, we characterized the differential expression of the Tex markers, their association with the patients’ survival outcome, and their mutation profile in multiple cancers. Additionally, we analyzed the effects on cancer-related pathways and immune infiltration within the TME, offering valuable insights into mechanisms of cancer immune evasion and progression. Finally, the correlation between their expression and sensitivity to multiple anti-cancer drugs was investigated extensively. Differential expression of all six markers was significantly associated with KIRC and poor prognosis in several cancers. They also played a potential activating role in apoptosis, EMT, and hormone ER pathways, as well as a potential inhibitory role in the DNA damage response and RTK oncogenic pathways. Infiltration of different immune cells was also found to be associated with the expression of the Tex-related genes in most cancer types. These findings underline that the reviving of exhausted T cells can be used to enhance the efficacy of immunotherapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b32105e8581ecae32317ce64eca6316a45d0d52" target='_blank'>
              Characterization of Exhausted T Cell Signatures in Pan-Cancer Settings
              </a>
            </td>
          <td>
            Rifat Tasnim Juthi, Saiful Arefeen Sazed, Manvita Mareboina, A. Zaravinos, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mesothelioma cell heterogeneity encompasses diverse morphological and molecular characteristics observed within tumors, significantly impacting disease progression, treatment outcomes, and the development of targeted therapies. This heterogeneity has long posed challenges for accurate diagnosis and effective treatment, but understanding its complexities offers the potential for novel diagnostic modalities and therapeutic interventions. This study employed single-cell RNA sequencing (scRNA-seq) to investigate mesothelioma cell heterogeneity from various sources, including cell culture (CC), peritoneal lavage (Lav) from the tumor microenvironment, and circulating tumor cells (CTC) in murine models. Gene set enrichment analysis was used to identify distinct gene signatures for each subpopulation. The results revealed unique characteristics for mesothelioma cells depending on their origin. In the CC group, up-regulated genes were primarily involved in tumor cell cycle control, proliferation, and apoptosis. In the CTC group, up-regulated genes were associated with cancer cell stemness. The Lav group showed up-regulated genes facilitating interactions between tumor cells and the microenvironment, such as epithelial–mesenchymal transition and immune responses mediated by IFN-α and IFN-γ. Some pathways were shared among all tumor cells, suggesting the potential for transitioning between functional states under specific conditions. This may be the first study to explore circulating mesothelioma cell heterogeneity using scRNA-seq. The distinct gene signatures identified in each mesothelioma cell subpopulation likely play critical roles in tumor initiation and progression, offering potential novel targets for therapeutic intervention. These findings could help inform the development of more effective, personalized treatments for mesothelioma, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Purpose Bladder cancer (BC) is characterized by high heterogeneity, with non-muscle-invasive (NMIBC) and muscle-invasive (MIBC) stages differing significantly in clinical behavior and outcomes. The transition from NMIBC to MIBC involves extensive tumor microenvironment (TME) remodeling, particularly in endothelial cells (ECs), which drive angiogenesis and modulate immune and extracellular matrix (ECM) interactions. However, the precise roles of ECs in this progression remain poorly defined. Methods Public single-cell RNA sequencing (scRNA-seq) datasets from 47 BC patients were analyzed to characterize endothelial cell heterogeneity and functional states across NMIBC and MIBC. Computational tools such as CellChat were applied to reconstruct cell–cell communication networks, focusing on pathways related to angiogenesis, immune crosstalk, and ECM remodeling. Results Twelve major cell types were identified, with endothelial cells exhibiting distinct transcriptional profiles between NMIBC and MIBC. NMIBC-associated ECs promoted adhesion and migration through HMGB1 and CXCL12 signaling. In contrast, MIBC was enriched in an ADAM10+ endothelial subset associated with vascular remodeling and activation of Wnt signaling via CTNNB1. Key ligand-receptor interactions highlighted the dynamic roles of ECs in TME modulation during BC progression. Conclusions This study reveals stage-specific endothelial cell phenotypes and signaling networks in BC. The identification of an MIBC-specific ADAM10+ endothelial subset underscores its potential role in driving tumor progression and highlights opportunities for stage-adapted vascular-targeted therapies. These findings advance our understanding of BC pathogenesis and provide the foundation for novel therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02297-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7505578630274eaf54930e9821b19f4ea81e3ca3" target='_blank'>
              A single-cell atlas of bladder cancer unveils dynamic cellular composition and endothelial functional shifts during progression
              </a>
            </td>
          <td>
            Hongjian Song, Guixiang Xie, Yaowei Li, Xiaowei Hu, Zongzheng Yang, Yubo Zhao, Qing Shi, Haonan Li, Ziyi Liu, Zhihao Yin, Ziqi Wang, Zhichao Tong, Wanhai Xu
          </td>
          <td>2025-04-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Bladder cancer (BC) is a heterogeneous disease associated with high mortality if not diagnosed early. BC is classified into non-muscle-invasive BC (NMIBC) and muscle-invasive BC (MIBC), with MIBC linked to poor systemic therapy response and high recurrence rates. Current treatments include transurethral resection with Bacillus Calmette-Guérin (BCG) therapy for NMIBC and radical cystectomy with chemotherapy and/or immunotherapy for MIBC. The tumor microenvironment (TME) plays a critical role in cancer progression, metastasis, and therapeutic efficacy. A comprehensive understanding of the TME’s complex interactions holds substantial translational significance for developing innovative treatments. The TME can contribute to therapeutic resistance, particularly in immune checkpoint inhibitor (ICI) therapies, where resistance arises from tumor-intrinsic changes or extrinsic TME factors. Recent advancements in immunotherapy highlight the importance of translational research to address these challenges. Strategies to overcome resistance focus on remodeling the TME to transform immunologically “cold” tumors, which lack immune cell infiltration, into “hot” tumors that respond better to immunotherapy. These strategies involve disrupting cancer-microenvironment interactions, inhibiting angiogenesis, and modulating immune components to enhance anti-tumor responses. Key mechanisms include cytokine involvement [e.g., interleukin-6 (IL-6)], phenotypic alterations in macrophages and natural killer (NK) cells, and the plasticity of cancer-associated fibroblasts (CAFs). Identifying potential therapeutic targets within the TME can improve outcomes for MIBC patients. This review emphasizes the TME’s complexity and its impact on guiding novel therapeutic approaches, offering hope for better survival in MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37a415d4b4c15c220a29189edf11f5cb518dca64" target='_blank'>
              Key immune cells and their crosstalk in the tumor microenvironment of bladder cancer: insights for innovative therapies
              </a>
            </td>
          <td>
            Anna Di Spirito, Sahar Balkhi, Veronica Vivona, Lorenzo Mortara
          </td>
          <td>2025-03-31</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The persistently high mortality rates associated with cancer underscore the imperative need for innovative, efficacious, and safer therapeutic agents, as well as a more nuanced understanding of tumor biology. Patient-derived organoids (PDOs) have emerged as innovative preclinical models with significant translational potential, capable of accurately recapitulating the structural, functional, and heterogeneous characteristics of primary tumors. When integrated with cutting-edge genomic tools such as CRISPR, PDOs provide a powerful platform for identifying cancer driver genes and novel therapeutic targets. This comprehensive review delves into recent advancements in CRISPR-mediated functional screens leveraging PDOs across diverse cancer types, highlighting their pivotal role in high-throughput functional genomics and tumor microenvironment (TME) modeling. Furthermore, this review highlights the synergistic potential of integrating PDOs with CRISPR screens in cancer immunotherapy, focusing on uncovering immune evasion mechanisms and improving the efficacy of immunotherapeutic approaches. Together, these cutting-edge technologies offer significant promise for advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4133d2e7efabd080d8387faf29f0a614a8cf2bb5" target='_blank'>
              Transforming cancer treatment: integrating patient-derived organoids and CRISPR screening for precision medicine
              </a>
            </td>
          <td>
            Ziyi Zhu, Jia-Yang Shen, Paul Chi Lui Ho, Ya Hu, Zhaowu Ma, Lingzhi Wang
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT CD8+ tissue‐resident memory T cells (TRM) are strategically located in peripheral tissues, enabling a rapid response to local infections, which is different from circulating memory CD8+ T cells. Their unique positioning makes them promising targets for vaccines designed to enhance protection at barrier sites and other organs. Recent studies have shown a correlation between CD8+ TRM cells and favorable clinical outcomes in various types of cancer, indicating their potential role in immune checkpoint blockade (ICB) therapies. However, the dual nature of CD8+ TRM cells presents challenges, as their inappropriate activation may lead to autoimmunity and chronic inflammatory conditions. This review highlights significant advancements in the field, focusing on the differentiation pathways and phenotypic heterogeneity of CD8+ TRM cells across different tissues and disease states. We also review their protective roles in various contexts and the implications for vaccine development against infections and treatment strategies for tumors. Overall, this comprehensive review outlines the common features of CD8+ TRM cell differentiation and biological functions, emphasizing their specific characteristics across diverse tissues and disease states, which can guide the design of therapies against infections and tumors while minimizing the risk of autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483c394d11124af6ee70fc4161b20f0da73f559e" target='_blank'>
              Tissue‐Resident Memory CD8+ T Cells: Differentiation, Phenotypic Heterogeneity, Biological Function, Disease, and Therapy
              </a>
            </td>
          <td>
            Luming Xu, Lilin Ye, Qizhao Huang
          </td>
          <td>2025-03-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Summary Spatial and temporal intra-tumor heterogeneity drives tumor evolution and therapy resistance. Existing visualization tools often fail to capture both dimensions simultaneously. To address this, we developed Jellyfish, a tool that integrates phylogenetic and sample trees into a single plot, providing a holistic view of tumor evolution and capturing both spatial and temporal evolution. Available as a JavaScript library and R package, Jellyfish generates interactive visualizations from tumor phylogeny and clonal composition data. We demonstrate its ability to visualize complex subclonal dynamics using data from ovarian high-grade serous carcinoma. Availability and implementation Jellyfish is freely available with MIT license at https://github.com/HautaniemiLab/jellyfish (JavaScript library) and https://github.com/HautaniemiLab/jellyfisher (R package).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0210199a3e67c966717695406003ace2c046fe" target='_blank'>
              Jellyfish: integrative visualization of spatio-temporal tumor evolution and clonal dynamics
              </a>
            </td>
          <td>
            K. Lavikka, Altti Ilari Maarala, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Gastrointestinal metastases are rare in patients with thyroid carcinoma (TC), and their underlying mechanisms remain unclear. Thus, in this study, we aimed to explore the spatial distribution characteristics of TCs and associated gastrointestinal metastatic cells. Methods We used spatial transcriptomics to generate an atlas that captures spatial gene expression patterns in papillary thyroid cancer (PTC), anaplastic thyroid carcinoma (ATC), ATC-associated lymphatic metastasis (ATC-LM), and rare ATC-associated gastric metastasis (ATC-GM). Results We demonstrated that tumor-specific myeloid cells with high SFRP4 expression were correlated with TC dedifferentiation and poor prognosis. Moreover, we validated their close localization to CD44+ tissue stem cells using immunofluorescence staining and spatial transcriptomics. We also demonstrated that ATC-LM and ATC-GM tissues exhibited high levels of CD44+PKHD1L1+ cells, which could serve as markers for these two pathological types. Conclusions These findings highlight the dynamic changes in cell composition, intercellular communication, and potential markers associated with TC dedifferentiation and distant metastasis. Further research based on our findings may contribute to improving diagnostic and therapeutic strategies for patients with TC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06252-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7453b499aa0f4f56b20054ed92fb6b1a0f0d81c" target='_blank'>
              Analysis of thyroid carcinoma composition and spatial architecture in the progression of dedifferentiation, lymphatic metastasis, and gastric metastasis
              </a>
            </td>
          <td>
            Di Wang, Ruichun Lu, Fenglian Yan, Yansong Lin, Hao Wang, Huabao Xiong
          </td>
          <td>2025-02-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, Stephanie Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Intratumor heterogeneity in plantar melanoma orchestrates transcriptional programs that contribute to resistance to target- and immuno-therapies. However, the evolution and spatial distribution of cellular subgroups, as well as their effects on immune environment and patient prognosis, remain unclear. Methods: We analyzed 218,021 cells from 20 plantar melanoma and 6 normal samples using single-cell RNA sequencing to reveal the evolutionary characteristics and communication patterns of tumor subgroups. Spatial transcriptomics and multiplex immunohistochemistry (mIHC) were used to map the spatial distribution of these subgroups, with mIHC scores further evaluating their correlation with patient prognosis. Single-cell multiomics analysis identified key transcription factors associated with chromatin accessibility. In addition, survival analysis was performed using bulk RNA sequencing data from 68 melanoma patients. Results: We identified a continuum of subgroups originating from stem cells via transitional and Schwann cell-like precursor states, ultimately reaching a Schwann cell-like state. This evolution trajectory was supported by integrative evidence, including assessments of stemness, transitional states, RNA velocity, and transcription factors. The histological distribution of these subgroups was validated by spatial transcriptomics and multiple IHC. Notably, Schwann cell-like subgroup, regulated by transcription factor HMGA2, was associated with immune cell dysregulation and a worse prognosis, including increased invasion and lymph node metastasis. Mechanically, inhibition of HMGA2 expression blocked the transition to Schwann-like melanoma fate. Conclusions: This study reveals the unique evolutionary trajectory of plantar melanoma, showing its differentiation towards a Schwann-like fate regulated by HMGA2, leading to a decline in pigment function, enhanced immune tolerance and an increased propensity for lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906aa3f7f8f1b89c37180b73edff27a732dad812" target='_blank'>
              A distinct subgroup with Schwann-like differentiation presents immune resistance and poor prognosis in plantar melanoma
              </a>
            </td>
          <td>
            Jie Tian, Lu Zhang, Zhuang Le, Pingping Lin, Shenxi Zhang, Yicen Yan, Yu Yang, Guohong Zhang, Hang Li, Binbin Lai
          </td>
          <td>2025-03-03</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic alterations are the driving force behind pancreatic cancer (PC) tumorigenesis, but they do not fully account for its diverse phenotypes. Investigating the epigenetic landscapes of PC offers a more comprehensive understanding and could identify targeted therapies that enhance patient survival. In this study, we have developed a new promising methodology of spatial epigenomics that integrates multiplexed molecular imaging with convolutional neural networks. Then, we used it to map epigenetic modification levels in the six most prevalent PC subtypes. We analyzed and semi-quantified the resulting molecular data, revealing significant variability in their epigenomes. DNA and histone modifications, specifically methylation and acetylation, were investigated. Using the same technique, we examined DNA conformational changes to further elucidate the transcriptional regulatory mechanisms involved in PC differentiation. Our results revealed that the foamy-gland and squamous-differentiated subtypes exhibited significantly increased global levels of epigenetic modifications and elevated Z-DNA ratios. Overall, our findings may suggest a potentially reduced efficacy of therapeutics targeting epigenetic regulators for these subtypes. Conversely, the conventional ductal PC subtype has emerged as a promising candidate for treatment with epigenetic modulators. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90087-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Crohn’s disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn’s disease
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Despite significant advancements in identifying novel therapeutic targets and compounds, cancer stem cells (CSCs) remain pivotal in driving therapeutic resistance and tumor progression in gastric cancer (GC). High-resolution knowledge of the transcriptional programs underlying the role of CSC niche in driving tumor stemness and progression is still lacking. Herein, spatial and single-cell RNA sequencing of 32 human gastric mucosa tissues at various stages of malignancy, illuminating the phenotypic plasticity of tumor epithelium and transcriptional trajectory from mature gastric chief cells to the CSC state, which is associated with activation of EGFR and WNT signaling pathways, is conducted. Moreover, the CSCs interact with not only the immunosuppressive CXCL13+ T cells and CCL18+ M2 macrophages to evade immune surveillance, but also the inflammatory cancer-associated fibroblasts (iCAFs) to promote tumorigenesis and maintain stemness, which construct the CSC niche leading to inferior prognosis. Notably, it is uncovered that amphiregulin (AREG) derived from iCAFs promotes tumor stemness by upregulating the expression of SOX9 in tumor cells, and contributes to drug resistance via the AREG-ERBB2 axis. This study provides valuable insight into the characteristics of CSC niche in driving tumor stemness and progression, offering novel perspective for designing effective strategies to overcome GC therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0d6cf84d15c0f17d3af8a79006d512aec0aa37" target='_blank'>
              Dissecting the Spatial and Single-Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer.
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinglong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are central to tumor progression, metastasis, immune evasion, and therapeutic resistance. Characterized by remarkable self-renewal and adaptability, CSCs can transition dynamically between stem-like and differentiated states in response to external stimuli, a process termed “CSC plasticity.” This adaptability underpins their resilience to therapies, including immune checkpoint inhibitors and adoptive cell therapies (ACT). Beyond intrinsic properties, CSCs reside in a specialized microenvironment—the CSC niche—which provides immune-privileged protection, sustains their stemness, and fosters immune suppression. This review highlights the critical role of CSCs and their niche in driving immunotherapy resistance, emphasizing the need for integrative approaches to overcome these challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19a86ea6fc48341fef86fcd4e4833fd36fa9c1e2" target='_blank'>
              Cancer stem cells and niches: challenges in immunotherapy resistance
              </a>
            </td>
          <td>
            Yonglong Pan, Chaoyi Yuan, Chenglong Zeng, Chaoyang Sun, Limin Xia, Guihua Wang, Xiaoping Chen, Bixiang Zhang, Jianfeng Liu, Ze-Yang Ding
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="A significant obstacle in translating innovative breast cancer treatments from bench to bed side is demonstrating efficacy in preclinical settings prior to clinical trials, as the heterogeneity of breast cancer can be challenging to replicate in the laboratory. A significant number of potential medicines have not progressed to clinical trials because preclinical models inadequately replicate the complexities of the varied tumor microenvironment. Consequently, the variety of breast cancer models is extensive, and the selection of a model frequently depends on the specific inquiry presented. This review aims to present an overview of the existing breast cancer models, highlighting their advantages, limitations, and challenges in the context of innovative drug discovery, thereby offering insights that may be advantageous to future translational studies. Conventional monolayer cultures are critical for elucidating the different breast cancer types and their behavior, have limitations in adequately replicating tumor environments. The 3D models such as patient-derived xenografts, cell-derived xenografts and genetically engineered models offer better insights by maintaining tumor microenvironments and cellular heterogeneity. Results can be further enhanced when compared with breast epithelial cells, a negative control to determine early stages by investigating differences between healthy and cancerous mammary cells. While cell lines such as MCF-7, MDA-MB-231 etc are useful in vitro models, they exhibit genetic variations that may affect drug responses over time. Additionally, animal models, particularly rodents, are instrumental in breast cancer research due to their biological resemblances to humans and the relative ease of genetic modification, however, witness a low occurrence of tumors. This review thus concludes that different preclinical models have their associated benefits and pitfalls. Therefore, specific preclinical models can be created by altering the gene expression at the genetic level or could be selected as per specific experimental needs which will enable successful translation of preclinical findings into clinical trials can be possible. See also the graphical abstract(Fig. 1).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e63e0d7fbb28de4c3c415e998efa140b73663b5d" target='_blank'>
              Breast cancer preclinical models: a vital resource for comprehending disease mechanisms and therapeutic development
              </a>
            </td>
          <td>
            Ravneet Kaur, Anuradha Sharma, N. Wijekoon
          </td>
          <td>2025-02-19</td>
          <td>EXCLI Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Ovarian carcinoma represents an aggressive malignancy with poor prognosis and limited therapeutic efficacy. While deubiquitinating (DUB) genes are known to regulate crucial cellular processes and cancer progression, their specific roles in ovarian carcinoma remain poorly understood. Methods We conducted an integrated analysis of single-cell RNA sequencing and bulk transcriptome data from public databases. DUB genes were identified through Genecard database. Using the Seurat package, we performed cell clustering and differential expression analysis. Cell–cell communications were analyzed using CellChat. A DUB-related risk signature (DRS) was developed using machine learning approaches through integration of GEO and TCGA datasets. The prognostic value and immune characteristics of the signature were systematically evaluated. Results Our analysis revealed eight distinct cell subtypes in the tumor microenvironment, including epithelial, fibroblast, myeloid, and Treg cells. DUB-high cells were predominantly found in Treg and myeloid populations, exhibiting elevated expression of tumor-related pathways and enhanced cell–cell communication networks, particularly between fibroblasts and myeloid cells. Conversely, DUB-low cells were enriched in epithelial populations with reduced immune activity. The DRS model demonstrated robust prognostic value across multiple independent cohorts. High-risk patients, as classified by the DRS, showed significantly poorer survival outcomes and distinct immune infiltration patterns compared to low-risk patients. Conclusion This study provides comprehensive insights into DUB gene expression patterns across different cell populations in ovarian carcinoma. The established DRS model offers a promising tool for risk stratification and may guide personalized therapeutic strategies. Our findings highlight the potential role of DUB genes in modulating the tumor immune microenvironment and patient outcomes in ovarian carcinoma. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02267-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186d9d1be19b56c1d6796da9872a6f27a083fcd1" target='_blank'>
              Identification of a deubiquitinating gene-related signature in ovarian cancer using integrated transcriptomic analysis and machine learning framework
              </a>
            </td>
          <td>
            Suwan Hu, Mengting Wang
          </td>
          <td>2025-04-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key players in cancer development and therapy, and they exhibit multifaceted roles in the tumor microenvironment (TME). From their diverse cellular origins, CAFs undergo phenotypic and functional transformation upon interacting with tumor cells and their presence can adversely influence treatment outcomes and the severity of the cancer. Emerging evidence from single-cell RNA sequencing (scRNA-seq) studies have highlighted the heterogeneity and plasticity of CAFs, with subtypes identifiable through distinct gene expression profiles and functional properties. CAFs influence cancer development through multiple mechanisms, including regulation of extracellular matrix (ECM) remodeling, direct promotion of tumor growth through provision of metabolic support, promoting epithelial-mesenchymal transition (EMT) to enhance cancer invasiveness and growth, as well as stimulating cancer stem cell properties within the tumor. Moreover, CAFs can induce an immunosuppressive TME and contribute to therapeutic resistance. In this review, we summarize the fundamental knowledge and recent advances regarding CAFs, focusing on their sophisticated roles in cancer development and potential as therapeutic targets. We discuss various strategies to target CAFs, including ECM modulation, direct elimination, interruption of CAF-TME crosstalk, and CAF normalization, as approaches to developing more effective treatments. An improved understanding of the complex interplay between CAFs and TME is crucial for developing new and effective targeted therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a333ac0501e7572a5af1f1975c3f74e5d8b822" target='_blank'>
              Cancer associated fibroblasts in cancer development and therapy
              </a>
            </td>
          <td>
            Hongyuan Jia, Xingmin Chen, Linling Zhang, Meihua Chen
          </td>
          <td>2025-03-28</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7ab0c0159914b37ff2d6316b406bb08a12959ab" target='_blank'>
              Single-cell transcriptomic atlas of human retina from Chinese donors reveals population-specific cellular diversity
              </a>
            </td>
          <td>
            Sen Lin, Yiwen Tao, Luning Yang, Q. Pan, T. Cai, Yunyan Ye, Jianhui Liu, Yang Zhou, Quanyong Yi, Zen Haut Lu, Lie Chen, Gareth McKay, Richard Rankin, Yongqing Shao, Weihua Meng
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ed63792a7cb4d7298d3e8340ecc0d2e80e3929" target='_blank'>
              EHE cell cultures: a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M. Bellizzi, Anthony N. Snow, Patrick Breheny, M. S. Chimenti, M. Tanas
          </td>
          <td>2025-03-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fc7e6f9b6372a2eebc4b0d3580ed466ef20424" target='_blank'>
              A Tc1- and Th1-T-lymphocyte-rich tumor microenvironment is a hallmark of MSI colorectal cancer.
              </a>
            </td>
          <td>
            Zhihao Huang, T. Mandelkow, N. F. Debatin, M. Lurati, Julia Ebner, J. Raedler, E. Bady, Jan H. Müller, R. Simon, E. Vettorazzi, A. Menz, Katharina Möller, N. Gorbokon, G. Sauter, M. Lennartz, A. Luebke, D. Höflmayer, T. Krech, P. Lebok, C. Fraune, A. Hinsch, Frank Jacobsen, A. Marx, S. Steurer, S. Minner, D. Dum, S. Weidemann, C. Bernreuther, T. Clauditz, E. Burandt, N. Blessin
          </td>
          <td>2025-04-03</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5411691311f3ea382bc8274891edc9e740ebfb77" target='_blank'>
              Non-invasive approach for endoluminal biopsy coupled with single-cell proteomics allows for immune characterization of intracranial aneurysms
              </a>
            </td>
          <td>
            J. Antonios, B. Gultekin, B. Theriault, K. Yalcin, D. Miyagishima, N. Adenu-Mensah, N. Sujijantarat, A. Koo, J. Haynes, P. Cedeno, M. Johnson, R. Hebert, C. Matouk, T. Barak, M. Gunel
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Recent discoveries reveal that tumors host unique communities of microbes, which significantly influence cancer growth, spread, and response to treatments. These microbes interact with cancer cells and their surroundings by releasing chemical substances, altering immune system activity, and directly affecting healthy tissues, thereby shaping how tumors behave and respond to therapy. We explore how these microbial communities differ between cancer types and can serve as indicators of disease progression and treatment success. This review highlights promising new therapeutic strategies that target these microbes, particularly through advances in nanotechnology that allow for the precise delivery of treatments to tumor sites. We also discuss key challenges in studying tumor microbes, including the need for standardized research methods and the careful validation of findings between laboratory models and human patients. This growing field of research offers exciting possibilities for developing more personalized cancer treatments by considering each patient’s unique tumor microbial profile, potentially transforming how we diagnose and treat cancer in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79cbe4af66964c3203d3a9d90215d9ad5ac9b6f0" target='_blank'>
              Decoding the Tumor-Associated Microbiota: From Origins to Nanomedicine Applications in Cancer Therapy
              </a>
            </td>
          <td>
            Ruiqi Wang, Weizheng Li, Hongqian Cao, Lei Zhang
          </td>
          <td>2025-02-27</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Macrophages are a pivotal cell type within the synovial lining and sub-lining of the joint, playing a crucial role in maintaining homeostasis of synovium. Although fate-mapping techniques have been employed to differentiate synovial macrophages from other synovial myeloid cells, no comprehensive study has yet been conducted within the mouse synovial macrophage compartment. In this study, we present, for the first time, lineage tracing results from 18 myeloid-specific fate-mapping models in mouse peripheral blood (PB) and synovial tissue. The identification of synovial macrophages and monocyte-lineage cells through flow cytometry was further validated using cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq) datasets. These findings provide a valuable methodological tool for researchers to select appropriate models for studying the function of synovial myeloid cells and serve as a reference for investigations in other tissue types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0602215fee91dfe1d6560e3a7670fa8d2bc555fa" target='_blank'>
              Comprehensive analysis of myeloid reporter mice
              </a>
            </td>
          <td>
            Yidan Wang, Samuel D. Dowling, Vanessa Rodriguez, Jessica Maciuch, Meghan Mayer, Tyler Therron, Tovah N. Shaw, Miranda G. Gurra, Caroline L. Shah, H. Makinde, Florent Ginhoux, David Voehringer, Cole A. Harrington, Toby Lawrence, John R. Grainger, C. Cuda, Deborah R. Winter, Harris R. Perlman
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell image analysis is crucial for studying drug effects on cellular morphology and phenotypic changes. Most studies focus on single cell types, overlooking the complexity of cellular interactions. Here, we establish an analysis pipeline to extract phenotypic features of cancer cells cultured with fibroblasts. Using high-content imaging, we analyze an oncology drug library across five cancer and fibroblast cell line co-culture combinations, generating 61,440 images and ∼170 million single-cell objects. Traditional phenotyping with CellProfiler achieves an average enrichment score of 62.6% for mechanisms of action, while pre-trained neural networks (EfficientNetB0 and MobileNetV2) reach 61.0% and 62.0%, respectively. Variability in enrichment scores may reflect the use of multiple drug concentrations since not all induce significant morphological changes, as well as the cellular and genetic context of the treatment. Our study highlights nuanced drug-induced phenotypic variations and underscores the morphological heterogeneity of ovarian cancer cell lines and their response to complex co-culture environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4573d60560710e744b74b82ab85a673cb84bfba" target='_blank'>
              Evaluating feature extraction in ovarian cancer cell line co-cultures using deep neural networks
              </a>
            </td>
          <td>
            Osheen Sharma, Greta Gudoityte, Rezan Minozada, Olli Kallioniemi, Riku Turkki, L. Paavolainen, B. Seashore-Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Background Glioma, characterized by its cellular and molecular heterogeneity, presents formidable challenges in treatment strategy and prognostic assessment. The tumor microenvironment (TME) profoundly influences tumor behavior and treatment response, with tumor‐associated neutrophils (TANs) playing a complex but understudied role. This study aimed to investigate the heterogeneity and role of TANs in glioma and to develop a prognostic model. Methods Analysis of scRNA‐seq data identified cellular subpopulations and differentially expressed neutrophil‐related genes (DE‐NRGs). Bulk RNA‐seq was obtained from four independent datasets. Molecular subtypes of glioma samples were determined by consensus clustering. WGCNA was conducted to elucidate the association between gene modules and subtypes. We developed a risk score model. Expression of selected genes was confirmed using immunohistochemistry (IHC). In vitro experiments were also performed for functional verification, including CCK8, EdU, Transwell, and TUNEL assays. Results A total of 108 DE‐NRGs for TANs were identified based on scRNA‐seq data. Two molecular subtypes were characterized, showing significant differences in prognosis and clinical features. Immune‐related analyses demonstrated varied immunological characteristics between subtypes. The risk score model was constructed with 7 genes, including AEBP1, CAVIN1, DCTD, DEPP1, DUSP6, FKBP9, and UGCG. It showed significant prognostic value and was validated across three external datasets. The mutation landscape highlighted higher IDH mutation prevalence in low‐risk groups. Drug sensitivity analysis indicated TMZ resistance in high‐risk groups. In vitro experiments showed that UGCG could promote glioma cell proliferation, migration, and invasion, while decreasing apoptosis. Conclusion This study explored the heterogeneity of TANs and developed a prognostic model, providing insights for prognostic prediction and guiding personalized treatment strategies in glioma. Declaration of Generative AI in Scientific Writing: The authors declare nonuse of generative AI and AI‐assisted technologies in the writing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a8af983a6227909234b80a95e84b835146d4d4" target='_blank'>
              Integrated Analysis to Reveal Heterogeneity of Tumor‐Associated Neutrophils in Glioma
              </a>
            </td>
          <td>
            Wen Wang, Junsheng Li, Qiheng He, Chenglong Liu, Siyu Wang, Zhiyao Zheng, Bojian Zhang, Siqi Mou, Wei Sun, Jizong Zhao
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abee2ab2885700afbd38b00c669afff895861f19" target='_blank'>
              2002 NY-ESO-1-Specific TCR-T Cells Effectively Targets Glioblastoma Cancer Stem Cells
              </a>
            </td>
          <td>
            Michelot Michel, Alan Nguyen, Alexis Parker, Hongqiang Wang, Chen Qian, T. Admasu, Andre E. Boyke, Seokyoung Yoon, Keith L. Black, Rongfu Wang, John S. Yu
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of cancer-related deaths, with the liver being the most frequent site of metastasis in colorectal cancer. Previous studies have predominantly focused on the influence of the primary tumor itself on metastasis, with relatively limited research examining the changes within target organs. Methods Using an orthotopic mouse model of colorectal cancer, single-cell sequencing was employed to profile the transcriptomic landscape of pre-metastatic and metastatic livers. The analysis focused on identifying cellular and molecular changes within the hepatic microenvironment, with particular emphasis on inflammatory pathways and immune cell populations. Results A neutrophil subpopulation with high Prok2 expression was identified, showing elevated levels in the pre-metastatic and metastatic liver. Increased infiltration of Prok2⁺ neutrophils correlated with poor prognosis in liver metastatic colorectal cancer patients. In the liver metastatic niche (MN), these neutrophils showed high App and Cd274 (PD-L1) expression, suppressing macrophage phagocytosis and promoting T-cell exhaustion. Conclusion A Prok2⁺ neutrophil subpopulation infiltrated both pre-metastatic and macro-metastatic liver environments, potentially driving immunosuppression through macrophage inhibition and T-cell exhaustion. Targeting Prok2⁺ neutrophils could represent a novel therapeutic strategy for preventing liver metastasis in colorectal cancer patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06351-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spatial organization of cells in tissues underlies biological function, and recent advances in spatial profiling technologies have enhanced our ability to analyze such arrangements to study biological processes and disease progression. We propose MESA (multiomics and ecological spatial analysis), a framework drawing inspiration from ecological concepts to delineate functional and spatial shifts across tissue states. MESA introduces metrics to systematically quantify spatial diversity and identify hot spots, linking spatial patterns to phenotypic outcomes, including disease progression. Furthermore, MESA integrates spatial and single-cell multiomics data to facilitate an in-depth, molecular understanding of cellular neighborhoods and their spatial interactions within tissue microenvironments. Applying MESA to diverse datasets demonstrates additional insights it brings over prior methods, including newly identified spatial structures and key cell populations linked to disease states. Available as a Python package, MESA offers a versatile framework for quantitative decoding of tissue architectures in spatial omics across health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162d6ff6e3cedba6aedd63bc146790643b33531d" target='_blank'>
              Quantitative characterization of tissue states using multiomics and ecological spatial analysis
              </a>
            </td>
          <td>
            Daisy Yi Ding, Zeyu Tang, Bokai Zhu, Hongyu Ren, Alex K. Shalek, Robert Tibshirani, GP Nolan
          </td>
          <td>2025-04-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is a manifestation of dysfunctional cell states. It emerges from an interplay of intrinsic and extrinsic factors that disrupt cellular dynamics, including genetic and epigenetic alterations, as well as the tumor microenvironment. This complexity can make it challenging to infer molecular causes for treating the disease. This may be addressed by system-wide computer models of cells, as they allow rapid generation and testing of hypotheses that would be too slow or impossible to perform in the laboratory and clinic. However, so far, such models have been impeded by both experimental and computational limitations. In this perspective, we argue that they can now be achieved using deep learning algorithms to integrate omics data and prior knowledge of molecular networks. Such models would have many applications in precision oncology, e.g., for identifying drug targets and biomarkers, predicting resistance mechanisms and toxicity effects of drugs, or simulating cell-cell interactions in the microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8844ef9ba3f77a8686c18b6b08bd0f64a9f348" target='_blank'>
              Towards an interpretable deep learning model of cancer
              </a>
            </td>
          <td>
            Avlant Nilsson, N. Meimetis, Douglas A. Lauffenburger
          </td>
          <td>2025-02-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfff83a3b2e44394201ebb9147dbd5e27d9e3ca" target='_blank'>
              Single-Cell Transcriptomic Analysis Reveals an Aggressive Basal-Like Tumor Cell Subpopulation Associated With Poor Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Chang-Pin Liao, Yuting Zhang, Jing Yang, Shuo Wang, Zhijuan Li, Shuling Chen, Yubin Xie, Lixia Xu, Sui Peng, Xuezhen Zeng, Ming Kuang, Bangde Xiang, Kaiyu Sun, Xiao Zhao
          </td>
          <td>2025-02-24</td>
          <td>Journal of gastroenterology and hepatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction The molecular pathogenesis of ocular adnexal mucosa-associated lymphoid tissue (MALT) lymphoma and IgG4-related ophthalmic disease (IgG4-ROD) remains incompletely understood. Differentiating between the two diseases is vital given that the diagnostic evaluation and treatment approaches can vary significantly; this difficulty in distinction is exacerbated by the absence of specific biomarkers. This study aimed to investigate the differences between these two diseases based on their cellular composition, transcriptional heterogeneity, and the immune microenvironment using single-cell RNA transcriptional sequencing (scRNA-seq) technology. Methods We collected orbital lacrimal gland region tissue samples from three patients with MALT lymphoma and another three with IgG4-ROD and performed single-cell sequencing experiments. Subsequently, we conducted bioinformatics analyses, including cell subpopulation segmentation and inter-group comparison, tumor cell identification, functional enrichment analysis, and pseudotime trajectory analysis. Furthermore, we analyzed the cellular communication between tumor B-cell and T-cell subsets within the immune microenvironment of MALT lymphoma tissues. We performed immunofluorescence assays to verify the co-expression of receptor-ligand pairs. Results A total of six major cell subpopulations were identified, with B-cells and T-cells being the predominant cell types. All B-cell subpopulations in MALT lymphomas are malignant, exhibiting significant intratumoral and intertumoral heterogeneity. Reclustering of the T-cell subpopulation identified five major T-cell subpopulations. Pseudotime analysis revealed that CD4+ naive T-cells in MALT lymphoma patients were highly likely to differentiate into follicular helper T-cells, whereas, in IgG4-ROD patients, CD4+ naive T-cells were highly likely to differentiate into regulatory T-cells. Intercellular communication analysis revealed that the CD27-CD70 immune checkpoint receptor−ligand pair and CXCL13-CXCR5 chemokine receptor−ligand pair were significantly upregulated between malignant B-cells and T-cells subpopulations. Conclusion This study is the first to conduct a comparative single-cell transcriptome sequencing analysis of ocular adnexal MALT lymphoma and IgG4-ROD. Our results reveal the cellular composition, key pathways, and critical immune microenvironment implicated in the development of these two diseases. These findings provide important insights into the pathogenesis of these two diseases and highlight the differences between them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b84eefafe3887c5d3f9997b2994a421a348df" target='_blank'>
              Single-cell RNA-sequencing reveals cellular heterogeneity and immune microenvironment characteristics between ocular adnexal mucosa-associated lymphoid lymphoma and IgG4−related ophthalmic disease
              </a>
            </td>
          <td>
            Yu Yang, Yujiao Wang, Xuelian Jin, W. He
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastomas are the most prevalent primary brain tumors and are associated with a dramatically poor prognosis. Despite an intensive treatment approach, including maximal surgical tumor removal followed by radio- and chemotherapy, the median survival for glioblastoma patients has remained around 18 months for decades. Glioblastoma is distinguished by its highly complex mechanisms of immune evasion and pronounced heterogeneity. This variability is apparent both within the tumor itself, which can exhibit multiple phenotypes simultaneously, and in its surrounding microenvironment. Another key feature of glioblastoma is its “cold” microenvironment, characterized by robust immunosuppression. Recent advances in single-cell RNA sequencing have uncovered new promising insights, revealing previously unrecognized aspects of this tumor. In this review, we consolidate current knowledge on glioblastoma cells and its microenvironment, with an emphasis on their biological properties and unique patterns of molecular communication through signaling pathways. The evidence underscores the critical need for personalized poly-immunotherapy and other approaches to overcome the plasticity of glioblastoma stem cells. Analyzing the tumor microenvironment of individual patients using single-cell transcriptomics and implementing a customized immunotherapeutic strategy could potentially improve survival outcomes for those facing this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf00ce3ce13ab5e7ee7ff1ae10bf120f0be9305e" target='_blank'>
              Molecular biology of the deadliest cancer – glioblastoma: what do we know?
              </a>
            </td>
          <td>
            Aly Ismailov, Aldo Spallone, Alexey Belogurov, Alan Herbert, Maria Poptsova
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Despite remarkable advancements in therapeutic strategies, a considerable proportion of patients with bladder cancer (BC) still experience disease progression and unfavorable prognosis. The heterogeneity and biological functions of tumor endothelial cells (ECs) during BC progression remain poorly understood. We collected scRNA-seq data from BC samples and identified two EC subpopulations through hierarchical clustering analysis. The activity of signaling pathways in distinct EC subpopulations was assessed utilizing AUCell analysis. Gene regulatory networks (GRN) were constructed and analyzed for different EC subpopulations using the pySCENIC algorithm. Additionally, we investigated the association between the abundance of EC subpopulations and both clinical prognosis and immune cell infiltration. The biological effects of ESM1 protein on BC cells were further validated through EdU and Transwell assays. We analyzed 7,519 CD45-negative single cells from BC tissues and discerned two distinct EC subpopulations. The two subpopulations were characterized by high expression of ESM1 (S1 ECs) and CXCL2 (S2 ECs), respectively. In S1 ECs, we observed significant activation of signaling pathways involved in tumor promotion, including angiogenesis and cell proliferation. Additionally, our GRN analysis uncovered notable differences in transcription factor activity between S1 and S2 ECs. Moreover, ESM1 protein promoted proliferation and migration of BC cells. Patients with higher abundance of the S1 EC subpopulation exhibited more unfavorable clinical outcomes and increased infiltration of inhibitory immune cells. Our findings elucidate the transcriptional profiles and biological roles of the high ESM1-expression endothelial cell subpopulation in BC. This subpopulation is associated with poor prognosis and immunosuppressive tumor microenvironment. Accordingly, targeting endothelial cells with high ESM1 expression may offer a novel therapeutic strategy for patients with BC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95731-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4283029ca72adeb8c04b30a9f72593fd7af08134" target='_blank'>
              Single-cell sequencing uncovers a high ESM1-expression endothelial cell subpopulation associated with bladder cancer progression and the immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Baohua Liu, Kai Sun, Qi Hou
          </td>
          <td>2025-03-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive tumor with high metastatic potential which leads to decreased survival rate and resistance to chemotherapy and immunotherapy. Nearly 90% of pancreatic cancer comprises pancreatic ductal adenocarcinoma (PDAC). About 80% of diagnoses takes place at the advanced metastatic stage when it is unresectable, which renders chemotherapy regimens ineffective. There is also a dearth of specific biomarkers for early-stage detection. Advances in next generation sequencing and single cell profiling have identified molecular alterations and signatures that play a role in PDAC progression and subtype plasticity. Most chemotherapy regimens have shown only modest survival benefits, and therefore, translational approaches for immunotherapies and combination therapies are urgently required. In this review, we have examined the immunosuppressive and dense stromal network of tumor immune microenvironment with various metabolic and transcriptional changes that underlie the pro-tumorigenic properties in PDAC in terms of phenotypic heterogeneity, plasticity and subtype co-existence. Moreover, the stromal heterogeneity as well as genetic and epigenetic changes that impact PDAC development is discussed. We also review the PDAC interaction with sequestered cellular and humoral components present in the tumor immune microenvironment that modify the outcome of chemotherapy and radiation therapy. Finally, we discuss different therapeutic interventions targeting the tumor immune microenvironment aimed at better prognosis and improved survival in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e7cd5ce498aa2efbf04ec631daa67571857bc7" target='_blank'>
              Phenotypic heterogeneity and tumor immune microenvironment directed therapeutic strategies in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Remya P. G. Ramesh, Hadida Yasmin, Pretty Ponnachan, Basel Al-Ramadi, Uday Kishore, Ann Mary Joseph
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is among the most aggressive cancers, with rising incidence and limited responsiveness to immunotherapy due to its highly suppressive tumor microenvironment (TME). Tertiary lymphoid structures (TLS), ectopic formation structures of immune cells, are linked to better prognosis and improved immunotherapy responses in PDAC. Understanding TLS’s role in PDAC could enhance immunotherapy effectiveness. Methods This study integrated transcriptomic and clinical data from 310 PDAC patients in GEO database. We performed consensus clustering using tumor-associated TLS (TA-TLS) genes, identifying three distinct molecular subtypes. Single-sample gene set enrichment analysis (ssGSEA) was then employed to calculate a TLS score for each patient, allowing for TLS-based evaluation. Key prognostic genes were identified using an iterative LASSO method, leading to the construction of a risk assessment model, which was validated across independent cohorts. We further analyzed the TLS score using single-cell RNA sequencing (scRNA-seq), visualized key gene expression, and validated protein expression through immunohistochemistry (IHC). Additionally, we explored the effects of DNASE1L3 on cell proliferation and migration, and its immune-related functions using Gene Set Enrichment Analysis (GSEA) and multiplex cytokine analysis. Results Consensus clustering revealed three PDAC molecular subtypes with significant differences in prognosis, TA-TLS gene expression, and TME features. The TLS score effectively stratified patients into high and low groups, correlating with survival outcomes and TME characteristics. Our risk model, validated across cohorts, reliably predicted patient outcomes. Validation studies showed lower expression of DNASE1L3 and IL33 in tumor tissues. scRNA-seq confirmed TLS score associations with immune cells. DNASE1L3 overexpression inhibited PDAC cell proliferation and migration, with cytokine analysis indicating increased immune activity. Conclusions This study elucidated the expression profile of TA-TLS genes in PDAC, constructed a TLS gene-based scoring system, and developed a related risk model. We also explored the functions and potential antitumor mechanisms of key genes, providing evidence and new insights for enhancing TLS-targeted immunotherapy strategies in PDAC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06152-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad4f13e415f9357404f99d30b4979256aef4341" target='_blank'>
              Prognostic implications and characterization of tumor-associated tertiary lymphoid structures genes in pancreatic cancer
              </a>
            </td>
          <td>
            Enkui Zhang, Yongsu Ma, Zonghao Liu, Jixin Zhang, Weikang Liu, Yiran Chen, Guangnian Liu, Xinxin Liu, Fusheng Zhang, Yu Zhu, Yinmo Yang, Xiaodong Tian
          </td>
          <td>2025-03-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) in the tumor microenvironment (TME) are emerging solid-tumor indicators of prognosis and response to immunotherapy. Considering that tumorigenesis requires metabolic reprogramming and subsequent TME remodeling, the discovery of TLS metabolic regulators is expected to produce immunotherapeutic targets. To identify such metabolic regulators, we constructed a metabolism-focused sgRNA library and performed an in vivo CRISPR screening in an orthotopic lung tumor mouse model. Combined with The Cancer Genome Atlas database analysis of TLS-related metabolic hub genes, we found that the loss of Acat1 in tumor cells sensitized tumors to anti-PD1 treatment, accompanied by increased TLS in the TME. Mechanistic studies revealed that ACAT1 resulted in mitochondrial protein hypersuccinylation in lung tumor cells and subsequently enhanced mitochondrial oxidative metabolism, which impeded TLS formation. Elimination of ROS by NAC or Acat1 knockdown promoted B cell aggregation and TLS construction. Consistently, data from tissue microassays of 305 patients with lung cancer showed that TLS were more abundant in non–small cell lung cancer (NSCLC) tissues with lower ACAT1 expression. Intratumoral ACAT1 expression was associated with poor immunotherapy outcomes in patients with NSCLC. In conclusion, our results identified ACAT1 as a metabolic regulator of TLS and a promising immunotherapeutic target in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f530a902aa9799db82e12bb11d7f979721376b" target='_blank'>
              ACAT1 regulates tertiary lymphoid structures and correlates with immunotherapy response in non–small cell lung cancer
              </a>
            </td>
          <td>
            Mengxia Jiao, Yifan Guo, Hongyu Zhang, Haoyu Wen, Peng Chen, Zhiqiang Wang, Baichao Yu, Kameina Zhuma, Yuchen Zhang, Jingbo Qie, Yun Xing, P. Zhao, Zihe Pan, Luman Wang, Dan Zhang, Fei Li, Yijiu Ren, Chang Chen, Yiwei Chu, J. Gu, Ronghua Liu
          </td>
          <td>2025-04-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdedfede7608cb2fb7f1fa7b1c417998bb4fd21" target='_blank'>
              Spatially resolved transcriptomic analysis of the adult human prostate.
              </a>
            </td>
          <td>
            Junyi Hu, Fei Liu, Jing Zhang, Lei Yin, Wanli Cao, Weidong Xu, Yifan Chang, Ye Wang, Jian Wang, Yaxin Hou, Lilong Liu, Sujun Chen, Guanghui Zhu, Junhui Jiang, Zixian Wang, Gong-Hong Wei, H. H. He, Di Gu, Ke Chen, Shancheng Ren
          </td>
          <td>2025-04-01</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Meibomian glands secrete lipid-rich meibum, which prevents tear evaporation. Aging-related Meibomian gland shrinkage may result in part from stem cell exhaustion and is associated with evaporative dry eye disease, a common condition lacking effective treatment. The identities and niche of Meibomian gland stem cells and the signals controlling their activity are poorly defined. Using snRNA-seq, in vivo lineage tracing, ex vivo live imaging, and genetic studies in mice, we identify markers for stem cell populations that maintain distinct regions of the gland and uncover Hedgehog (Hh) signaling as a key regulator of stem cell proliferation. Consistent with this, we show that human Meibomian gland carcinoma exhibits increased Hh signaling. Aged glands display decreased Hh and EGF signaling, deficient innervation, and loss of collagen I in niche fibroblasts, indicating that alterations in both glandular epithelial cells and their surrounding microenvironment contribute to age-related degeneration. These findings suggest new approaches to treat aging-associated Meibomian gland loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd9f723993b9fbc9144678833cdc43c46f60f6e1" target='_blank'>
              Identification of Meibomian gland stem cell populations and mechanisms of aging
              </a>
            </td>
          <td>
            Xuming Zhu, Mingang Xu, Céline Portal, Yvonne Lin, Alyssa Ferdinand, Tien Peng, Edward E. Morrisey, Andrzej A Dlugosz, Joseph M. Castellano, Vivian Lee, John Seykora, Sunny Y Wong, Carlo Iomini, Sarah E Millar
          </td>
          <td>2025-02-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, D. E. Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba54436455eb8d5189fb45709f89284b938e52f3" target='_blank'>
              Abstract B099: Deletion of Piezo1 in CD11b+ Dendritic Cells Amplifies the Local and Systemic Rhabdomyosarcoma Anti-Tumor Response
              </a>
            </td>
          <td>
            Melissa Bonner, David Askew, S. Eid, Suzanne L Tomchuck, muta Abiff, Jay T. Myers, Justin Garyu, A. Huang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Breast Cancer (BC) ranks among the top three most prevalent cancers globally and stands as the principal contributor to cancer-related fatalities among women. In spite of the substantial occurrence rate of BC, the early stage of this disease is generally regarded as curable. However, intra-tumor heterogeneity presents a formidable obstacle to the success of effective treatment. Method In this research, single cell RNA sequencing was utilized to dissect the tumor microenvironment within BC. Slingshot, CytoTRACE and Monocle 2 were applied to illustrate the differentiation process of each subpopulation in the pseudotime sequence. To comprehensively comprehend the tumor cells (TCs) in BC, an analysis of upstream transcription factors was carried out via pySCENIC, while downstream pathway enrichment was conducted through KEGG, GO and GSEA. The prognosis model was established based on the bulk data obtained from TCGA and GEO databases. Knock-down experiments were also implemented to explore the function of the transcription factor CEBPD in the TCs. Results Our in-depth analysis identified eight principal cell types. Notably, TCs were predominantly found within epithelial cells. The classification of TCs further uncovered five unique subpopulations, with one subpopulation characterized by high UGDH expression. This subpopulation was shown to possess distinct metabolic features in metabolism-related investigations. The intricate communication modalities among different cell types were effectively demonstrated by means of CellChat. Additionally, a crucial transcription factor, CEBPD, was identified, which demonstrated a pronounced propensity towards tumors and harbored potential tumor-advancing characteristics. Its role in promoting cancer was subsequently verified through in vitro knock-down experiments. Moreover, a prognostic model was also developed, and a risk score was established based on the genes incorporated in the model. Through comparing the prognoses of different UTRS levels, it was determined that the group with a high UTRS had a less favorable prognosis. Conclusion These outcomes contributed to the elucidation of the complex interrelationships within the BC tumor microenvironment. By specifically targeting certain subpopulations of TCs, novel treatment strategies could potentially be devised. This study shed light on the direction that future research in BC should take, furnishing valuable information that can be utilized to enhance treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f63e96b96bed76e3ae69c6c3838cfd629c2bbf" target='_blank'>
              Exposing the cellular situation: findings from single cell RNA sequencing in breast cancer
              </a>
            </td>
          <td>
            Gaofeng Ni, Xinhan Li, Wenyang Nie, Zhenzhen Zhao, Hua Li, Hongyan Zang
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d026aba47659bcc0e46d8766e11f46d409c8da83" target='_blank'>
              Multidimensional Classification Framework for Human Breast Cancer Cell Lines
              </a>
            </td>
          <td>
            Diogo Dias, Catarina Franco Jones, Ana Catarina Moreira, Gil Gonccalves, Mustafa B. A. Djamgoz, Frederico Castelo Ferreira, Paola Sanjuan‐Alberte, Rosalia Moreddu
          </td>
          <td>2025-02-21</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0f5dce3feb58cd819120465c413a2814ac1e2" target='_blank'>
              Path2Omics: Enhanced transcriptomic and methylation prediction accuracy from tumor histopathology
              </a>
            </td>
          <td>
            Danh-Tai Hoang, E. Shulman, S. R. Dhruba, N. Nair, R. K. Barman, H. Lalchungnunga, Omkar Singh, MacLean P Nasrallah, Eric A Stone, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ea1b0559ba72f593b3a6674f424957c7f5230c" target='_blank'>
              Abstract B016: Genomic features of the tumor microenvironment and antigen presentation machinery vary by clinical and molecular subtypes of breast cancer
              </a>
            </td>
          <td>
            Constandina E. O'Connell, Yoon Jin Cha, Alexander V Lobanov, Cheng Fan, Brooke M. Felsheim, A. Fernandez-Martinez, C. Perou
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="Abstract Breast cancer remains a significant global health challenge due to its complexity, which arises from multiple genetic and epigenetic mutations that originate in normal breast tissue. Traditional machine learning models often fall short in addressing the intricate gene interactions that complicate drug design and treatment strategies. In contrast, our study introduces GEMDiff, a novel computational workflow leveraging a diffusion model to bridge the gene expression states between normal and tumor conditions. GEMDiff augments RNAseq data and simulates perturbation transformations between normal and tumor gene states, enhancing biomarker identification. GEMDiff can handle large-scale gene expression data without succumbing to the scalability and stability issues that plague other generative models. By avoiding the need for task-specific hyper-parameter tuning and specific loss functions, GEMDiff can be generalized across various tasks, making it a robust tool for gene expression analysis. The model’s ability to augment RNA-seq data and simulate gene perturbations provides a valuable tool for researchers. This capability can be used to generate synthetic data for training other machine learning models, thereby addressing the issue of limited biological data and enhancing the performance of predictive models. The effectiveness of GEMDiff is demonstrated through a case study using breast mRNA gene expression data, identifying 307 core genes involved in the transition from a breast tumor to a normal gene expression state. GEMDiff is open source and available at https://github.com/xai990/GEMDiff.git under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3472d3e5ad8a58f2f1d0054cb37345e502bfbbc" target='_blank'>
              GEMDiff: a diffusion workflow bridges between normal and tumor gene expression states: a breast cancer case study
              </a>
            </td>
          <td>
            Xusheng Ai, Melissa C. Smith, F. A. Feltus
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b43aff1a938a3167bd67da194f1242acc86f44b7" target='_blank'>
              Abstract A001: Microdevice Implantation for Immune Modulation and Drug Response Evaluation in Glioblastoma Patients
              </a>
            </td>
          <td>
            Vincenzo Tarallo, Oliver Jonas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background In the course of tumor treatment, radiation therapy (RT) not only kills cancer cells, but also induces complex biological effects in non-malignant cells around cancer cells. These biological effects such as angiogenesis, changes in stromal composition and immune cell infiltration remodel the tumor microenvironment (TME). As one of the major components of the TME, Cancer‑associated fibroblasts (CAFs) are not only involved in tumorigenesis, progression, recurrence, and metastasis but also regulate the tumor-associated immune microenvironment. CAFs and tumor cells or immune cells have complex intercellular communication in the context of tumor radiation. Main content. Different cellular precursors, spatial location differences, absence of specific markers, and advances in single-cell sequencing technology have gradually made the abundant heterogeneity of CAFs well known. Due to unique radioresistance properties, CAFs can survive under high doses of ionizing radiation. However, radiation can induce phenotypic and functional changes in CAFs and further act on tumor cells and immune cells to promote or inhibit tumor progression. To date, the effect of RT on CAFs and the effect of irradiated CAFs on tumor progression and TME are still not well defined. Conclusion In this review, we review the origin, phenotypic, and functional heterogeneity of CAFs and describe the effects of RT on CAFs, focusing on the mutual crosstalk between CAFs and tumor or immune cells after radiation. We also discuss emerging strategies for targeted CAFs therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e7d8569cb5aa3f3831a8692c395636f7587796" target='_blank'>
              Cancer‑associated fibroblasts: a pivotal regulator of tumor microenvironment in the context of radiotherapy
              </a>
            </td>
          <td>
            Linhui Zheng, Wenqi Cai, Yuan Ke, Xiaoyan Hu, Chunqian Yang, Runze Zhang, Huachao Wu, Dong Liu, Haijun Yu, Chaoyan Wu
          </td>
          <td>2025-03-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Tumour‐associated microbiota are integral components of the tumour microenvironment (TME). However, previous studies on intratumoral microbiota primarily rely on bulk tissue analysis, which may obscure their spatial distribution and localized effects. In this study, we applied in situ spatial‐profiling technology to investigate the spatial distribution of intratumoral microbiota in breast cancer and their interactions with the local TME. Using 5R 16S rRNA gene sequencing and RNAscope FISH/CISH on patients’ tissue, we identified significant spatial heterogeneity in intratumoral microbiota, with Fusobacterium nucleatum (F. nucleatum) predominantly localized in tumour cell‐rich areas. GeoMx digital spatial profiling (DSP) revealed that regions colonized by F. nucleatum exhibit significant influence on the expression of RNAs and proteins involved in proliferation, migration and invasion. In vitro studies indicated that co‐culture with F. nucleatum significantly stimulates the proliferation and migration of breast cancer cells. Integrative spatial multi‐omics and co‐culture transcriptomic analyses highlighted the MAPK signalling pathways as key altered pathways. By intersecting these datasets, VEGFD and PAK1 emerged as critical upregulated proteins in F. nucleatum‐positive regions, showing strong positive correlations with MAPK pathway proteins. Moreover, the upregulation of VEGFD and PAK1 by F. nucleatum was confirmed in co‐culture experiments, and their knockdown significantly reduced F. nucleatum‐induced proliferation and migration. In conclusion, intratumoral microbiota in breast cancer exhibit significant spatial heterogeneity, with F. nucleatum colonization markedly altering tumour cell protein expression to promote progression and migration. These findings provide novel perspectives on the role of microbiota in breast cancer, identify potential therapeutic targets, and lay the foundation for future cancer treatments. Key points Intratumoral Fusobacterium nucleatum exhibits significant spatial heterogeneity within breast cancer tissues. F. nucleatum colonization alters the expression of key proteins involved in tumour progression and migration. The MAPK signalling pathway is a critical mediator of F. nucleatum‐induced breast cancer cell proliferation and migration. VEGFD and PAK1 are potential therapeutic targets to mitigate F. nucleatum‐induced tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b181d8c04116c66c673f00453e25d5cfea9b8649" target='_blank'>
              Integrated spatial multi‐omics profiling of Fusobacterium nucleatum in breast cancer unveils its role in tumour microenvironment modulation and cancer progression
              </a>
            </td>
          <td>
            Feng Zhao, Rui An, Yilei Ma, Shaobo Yu, Yuzhen Gao, Yanzhong Wang, Haitao Yu, Xinyou Xie, Jun Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be659580caffb1fbf7776c3ae3608aa7fdd87091" target='_blank'>
              Tumor microenvironment: recent advances in understanding and its role in modulating cancer therapies.
              </a>
            </td>
          <td>
            Disha D Shah, Mehul R. Chorawala, Neha R. Raghani, Rajanikant Patel, Mohammad Fareed, V. Kashid, Bhupendra G. Prajapati
          </td>
          <td>2025-03-18</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract To unravel the mechanism of immune activation and suppression within tumors, a critical step is to identify transcriptional signals governing cell–cell communication between tumor and immune/stromal cells in the tumor microenvironment. Central to this communication are interactions between secreted ligands and cell-surface receptors, creating a highly connected signaling network among cells. Recent advancements in in situ-omics profiling, particularly spatial transcriptomic (ST) technology, provide unique opportunities to directly characterize ligand–receptor signaling networks that power cell–cell communication. In this paper, we propose a novel statistical method, LRnetST, to characterize the ligand–receptor interaction networks between adjacent tumor and immune/stroma cells based on ST data. LRnetST utilizes a directed acyclic graph model with a novel approach to handle the zero-inflated distributions of ST data. It also leverages existing ligand–receptor regulation databases as prior information, and employs a bootstrap aggregation strategy to achieve robust network estimation. Application of LRnetST to ST data of high-grade serous ovarian tumor samples revealed both common and distinct ligand–receptor regulations across different tumors. Some of these interactions were validated through both a MERFISH dataset and a CosMx SMI dataset of independent ovarian tumor samples. These results cast light on biological processes relating to the communication between tumor and immune/stromal cells in ovarian tumors. An open-source R package of LRnetST is available on GitHub at https://github.com/jie108/LRnetST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db23b179c860dc1d8c3febc0dd28265cf6aa4993" target='_blank'>
              Learning directed acyclic graphs for ligands and receptors based on spatially resolved transcriptomic data of ovarian cancer
              </a>
            </td>
          <td>
            Shrabanti Chowdhury, S. Ferri-Borgogno, Peng Yang, Wenyi Wang, Jie Peng, Samuel C Mok, Pei Wang
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="CD8+ tissue-resident memory T cells (CD8+TRM), expressing PD-1 and/or TIM-3, are linked to immunological surveillance in non-small cell lung cancer (NSCLC). However, their prognostic value across activation states and spatial distributions in NSCLC stages is unclear. We analyzed 271 NSCLC patients’ primary tumors and lymph nodes, using multiplex immunohistochemistry and inForm software for cell identification. Statistical analyses included the Mann-Whitney U test and Cox survival analysis. Findings showed CD8+TRM were categorized into four activation states. In locally advanced NSCLC, PD-1−TIM-3+CD8+TRM3, and PD-1+TIM-3+CD8+TRM4 densities were notably higher at invasive margins. Fewer interactions between CD8+TRM and tumor cells were observed in advanced lesions. Decreased PD-1+TIM-3−CD8+TRM2 interactions with tumor cells and increased PD-1+TIM-3+CD8+TRM4 interactions with tumor cells were independently associated with recurrence in patients with early lung adenocarcinoma and squamous carcinoma, respectively. These results suggest that CD8+TRM activation state and distribution are linked to recurrence risk in early-stage NSCLC, emphasizing the importance of CD8+TRM spatial dynamics in prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8927993683d672fa19c819fd8f766d987dddb21" target='_blank'>
              Stage-dependent spatial distribution and prognostic value of CD8+ tissue-resident memory T cells in NSCLC
              </a>
            </td>
          <td>
            Liying Yang, Hao Yang, Miaoqing Zhao, Hongtu Yuan, Jiaxiao Geng, Yushan Yan, Li Wu, Ligang Xing, Jinming Yu, Xiaorong Sun
          </td>
          <td>2025-02-22</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f05272abf51c9f38dd50672d47db4690dbc1bb3" target='_blank'>
              Spatial Profiling Identifies Regionally Distinct Microenvironments and Targetable Immunosuppressive Mechanisms in Pediatric Osteosarcoma Pulmonary Metastases.
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice S Pereira, Demond Williams, Douglass W Morris-Ii, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael C. Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-04-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a highly aggressive brain tumor associated with poor survival outcomes and is driven by a complex tumor microenvironment (TME) that promotes tumor progression and treatment resistance. To explore the role of the TME in GBM, we analyzed glioma-related microarray and single-cell RNA sequencing (scRNA-seq) datasets from the Gene Expression Omnibus (GEO). Functional enrichment and weighted gene coexpression network analyses revealed distinct immune profiles, metabolic alterations, and differences in chemotherapeutic drug sensitivity between the high-risk and low-risk patient groups. scRNA-seq data processed with the ‘Seurat’ package were used to identify differentially expressed genes in pericytes, endothelial cells, and glioma cells, particularly those involved in extracellular matrix (ECM) remodeling. A 17-gene prognostic signature developed through Cox regression and LASSO analyses revealed that key genes (COL1A1, COL4A1, and VIM) were significantly associated with survival outcomes in GBM patients. Drug sensitivity analyses using data from the Genomics of Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) identified potential targeted therapies for GBM, including SB-505,124, staurosporine, and AZD8186. This integrative study underscores the critical roles of the ECM and synaptic remodeling in GBM and suggests novel therapeutic targets to improve personalized treatment strategies for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456759f2943a08a601ac9cd5e20189bf3d373750" target='_blank'>
              Molecular mechanisms and therapeutic targets in glioblastoma multiforme: network and single-cell analyses
              </a>
            </td>
          <td>
            Xiangyu Chen, Xiao Zhong, Feifei Zhang, Xiaomei Zhou, Xiaofeng Yue, Xueru Li
          </td>
          <td>2025-03-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yingchuan Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e24bbb19a6e12af52d533253b1371130f86db3" target='_blank'>
              Integrating genomic, transcriptomic and epigenetic data to identify phenotypically impactful driver pathways in glioblastoma
              </a>
            </td>
          <td>
            Junhua Zhang
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer metastasis, a process in which cancer cells migrate to secondary sites, accounts for 90% of cancer deaths. While machine learning has been used to predict metastasis in a variety of ways, they tend to be specific to tumor types or classification tasks. Here, we provide a pan-cancer machine learning model that is easily interpretable and directly links the transcriptomic and proteomic profiles of hundreds of cancer cell lines from DepMap with metastasis in a tissue-agnostic manner. We show that, for the relatively small sample size here, linear models perform just as well as non-linear models, including neural networks. We also demonstrate that transcriptomics outperforms proteomics for predictive tasks. However, the combination of both transcriptomics and proteomics leads to an overall increase in performance, indicating that these modalities can complement each other. Finally, we demonstrate that those features identified as most important by the joint omics linear model fit are over-represented in biological pathways related to metastasis, suggesting that the model learns relevant information that may be useful in identifying universal biomarkers or therapeutic targets of cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4778538fa7f9077b250641d66da47d7e300ae2d7" target='_blank'>
              Joint linear modeling of transcriptomics and proteomics is predictive of cancer metastasis
              </a>
            </td>
          <td>
            Ronit Sharma, N. Meimetis, Arjana Begzati, S. Nagar, Benjamin Kellman, Hratch M. Baghdassarian
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer cells grow and survive in the tumor microenvironment, which is a complicated process. As a key part of how colorectal cancer (CRC) progresses, tumor-associated macrophages (TAMs) exhibit a double role. Through angiogenesis, this TAM can promote the growth of cancers. Although being able to modify and adjust immune cells is a great advantage, these cells can also exhibit anti-cancer properties including direct killing of cancer cells, presenting antigens, and aiding T cell-mediated responses. The delicate regulatory mechanisms between the immune system and tumors are composed of a complex network of pathways regulated by several factors including hypoxia, metabolic reprogramming, cytokine/chemokine signaling, and cell interactions. Decoding and figuring out these complex systems become significant in building targeted treatment programs. Targeting TAMs in CRC involves disrupting chemokine signaling or adhesion molecules, reprogramming them to an anti-tumor phenotype using TLR agonists, CD40 agonists, or metabolic modulation, and selectively removing TAM subsets that promote tumor growth. Multi-drug resistance, the absence of an accurate biomarker, and drug non-specificity are also major problems. Combining macrophage-targeted therapies with chemotherapy and immunotherapy may revolutionize treatment. Macrophage studies will advance with new technology and multi-omics methodologies to help us understand CRC and build specific and efficient treatments. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-03965-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472095a4c3c0313c90e7e9724e09004bb9a2fb6" target='_blank'>
              From silent partners to potential therapeutic targets: macrophages in colorectal cancer
              </a>
            </td>
          <td>
            Hayat Khizar, K. Ali, Jianwei Wang
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f6763865a99a9f23291b576c73a041b68eb0a51" target='_blank'>
              Bioengineering facets of the tumor microenvironment in 3D tumor models: insights into cellular, biophysical and biochemical interactions.
              </a>
            </td>
          <td>
            Salma T. Rafik, D. Bakkalci, Alexander J MacRobert, Umber Cheema
          </td>
          <td>2025-03-27</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Adoptive cell therapy (ACT) mediates durable and complete regression of various cancers. However, its efficacy is limited by the long-term persistence of cytotoxic T lymphocytes, given their irreversible dysfunction within the tumor microenvironment. Herein, we aimed to establish an artificial lung metastasis model to examine T-lymphocyte subsets, in order to identify potential effective cell subsets for ACT.


METHODS
A metastatic lung melanoma mouse model was established using OVA-expressing melanoma B16 cells. Flow cytometry analysis was conducted to examine the surface markers, transcription factors, and secreted cytokines of tumor-specific CD8+ T cells within metastatic tissues. The infiltrated cells were sorted by flow cytometry for in vitro tumor cell killing assays or in vivo cell infusion therapy combined with chemotherapeutic drugs and immune checkpoint blockade antibodies.


RESULTS
Exhausted CD8+ T cells (Tex) exhibited high heterogeneity in metastatic tissues. Among Tex cells, the CXCR6- precursor cell showed certain memory characteristics, including phenotype, transcription factors, and maintenance, whereas the CXCR6+ subpopulation partially lost these traits. Moreover, CXCR6+ precursor cells effectively replenished effector-like Tex cells in metastatic tissues and exerted direct cytotoxicity against tumor cells. Notably, transferring these tumor-specific CXCR6+ precursor-exhausted T (Texp) cells into recipients induced a substantial regression of metastasis. In addition, these cells could respond to immune checkpoint blockade, which could better control tumor metastasis.


CONCLUSIONS
In our study, a subset of antigen-specific CXCR6-expressing Texp cells was observed within the metastatic tissue. The cells served as a crucial source of effector-like Tex cells and exerted direct cytotoxic effects on tumor cells. Adoptive transfer of CXCR6+ Texp cells effectively mitigated lung metastasis in mice. This study helps elucidate the role of Texp cells in metastasis, thereby offering novel insights into enhancing the efficacy and durability of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb0740f966f6454ce4d396ef8cc094bc08fa069" target='_blank'>
              Tumor-specific CXCR6 positive precursor CD8+ T cells mediate tumor control in metastatic melanoma.
              </a>
            </td>
          <td>
            Yang Song, Ji Chen, Yaqin Zhang, Ning Wu, Yongjun Zhu, Gang Chen, Feng Miao, Zhiming Chen, Yiqing Wang
          </td>
          <td>2025-04-07</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b581086a365c6022e8cfc4df2d851d53411c3c94" target='_blank'>
              494 Multi-Omics Integration and Mapping of Invasion Markers Reveal Tumor-Specific Druggable Targets in Primary Human Glioblastoma
              </a>
            </td>
          <td>
            Paola Suarez-Meade, Jean R. Clemenceau, Isabel Barnfather, Mark Edgar, Aleeshba N. Basil, Tae H. Hwang, A. Quiñones-Hinojosa
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D. Doherty, Brandon A. Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (iCCA) may be genomically subclassified by the presence of potentially actionable molecular aberrations, of which pathogenic alterations in isocitrate dehydrogenase (IDH)1 and fibroblast growth factor receptor (FGFR)2 are the most frequently observed. The impact of these molecular alterations on the tumor immune microenvironment remains incompletely understood. Methods: We performed a high-parameter spatial immune phenotyping of iCCA samples with pathogenic FGFR2 or IDH1 alterations and FGFR2/IDH1 wild-type controls at the single-cell level using CO-Detection by indEXing. Results:﻿ A total of 24 tumors were examined. Tumors with FGFR2 alterations were characterized by fewer CD8+ T cells and “M2-like” macrophages but higher levels of polymorphonuclear myeloid-derived suppressor cells as compared to FGFR2 wild-type tumors. Spatial relationships between polymorphonuclear myeloid-derived suppressor cells and multiple other cell types in the tumor microenvironment (including tumor cells, CD4+, and CD8+ T cells) were enriched in tumors with FGFR2 alterations. Tumors with IDH1 mutations had a trend toward more fibroblasts and were characterized by a closer proximity of tumor cells to CD4+ T cells, and between macrophages and multiple structural tumor microenvironment components as compared to other subtypes. Conclusions: iCCAs with pathogenic FGFR2 fusions/rearrangements and IDH1 mutations have distinct immunophenotypes. Tailoring immunotherapeutic approaches to specific molecular subsets could improve treatment outcomes across the divergent molecularly defined iCCA subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343c87af8f0d2b156de3f86c823598f7aa36e2b7" target='_blank'>
              Deep immune profiling of intrahepatic cholangiocarcinoma with CODEX multiplexed imaging
              </a>
            </td>
          <td>
            M. Baretti, Soumya Shekhar, Vaibhav Sahai, Daniel H. Shu, Kathryn Howe, Valerie Gunchick, N. Assarzadegan, Emma Kartalia, Qingfeng Zhu, E. Hallab, Archit Sheth-Shah, Aya Kondo, Nilofer S Azad, M. Yarchoan
          </td>
          <td>2025-02-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction Pancreatic adenocarcinoma (PAAD) is characterized by a profoundly immunosuppressive tumor microenvironment (TME) that limits the efficacy of immunotherapy. Emerging evidence suggests that tumor-specific metabolic reprogramming may drive disease progression and shape the immune landscape in PAAD. Methods We integrated multi-omics data from TCGA, GEO, and ICGC to identify key metabolism-related genes (MRGs) that influence immune cell infiltration, tumor progression, and patient survival. Based on nine pivotal MRGs (including ANLN, PKMYT1, and HMGA1), we developed and validated a novel metabolic-prognostic index (MPI). Functional enrichment analyses were conducted to elucidate the metabolic pathways associated with different MPI risk groups. In vitro experiments and drug sensitivity analyses were performed to confirm the oncogenic role of selected MRGs and to explore their therapeutic implications. Results The MPI effectively stratified patients into high- and low-risk groups. High-MPI scores correlated with poor overall survival, elevated tumor mutation burden (TMB), and an immunosuppressive TME, evidenced by reduced CD8⁺ T-cell infiltration and increased expression of immune checkpoints (PD-L1, TGF-β). Functional enrichment revealed glycolysis and folate biosynthesis as dominant pathways in high-MPI groups, whereas fatty acid metabolism prevailed in low-MPI groups. Experimental validation underscored the role of ANLN in promoting epithelial-mesenchymal transition (EMT) and immune evasion via NF-κB signaling. ANLN knockdown significantly reduced glycolytic activity, tumor cell migration, and immune evasion. Drug sensitivity analyses indicated resistance to gemcitabine but sensitivity to afatinib in high-MPI patients. Although TIDE analysis predicted immune checkpoint inhibitor (ICI) resistance in high-MPI tumors, a subset of patients showed favorable responses to anti-PD-L1 therapy. Discussion These findings provide a comprehensive framework for understanding how metabolic reprogramming shapes PAAD’s immunosuppressive TME and affects treatment outcomes. By accurately stratifying patients, the MPI serves as a promising tool to guide therapeutic decisions, including targeted therapy selection and immunotherapy prediction, ultimately offering potential for more personalized management of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42608b06fe89e88fa2b364a076f0a8a5c2fa634d" target='_blank'>
              Metabolic reprogramming shapes the immune microenvironment in pancreatic adenocarcinoma: prognostic implications and therapeutic targets
              </a>
            </td>
          <td>
            Weihua Song, Yabin Yu, Siqi Wang, Zhengyi Cui, Qiusi Zhu, Wangrui Liu, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903982cf411415d6dcdb66121d6c8e705f6172b9" target='_blank'>
              The Role of Cancer Stem Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Piotr Świerzewski, Bożena Romanowska-Dixon, Martyna Elas
          </td>
          <td>2025-02-14</td>
          <td>Ophthalmology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24611fd8233fc299c26132a33dc4d39224b104e" target='_blank'>
              Single-Cell Multiomics Reveals Regulatory Mechanisms of CAR T Cell Persistence and Dysfunction in Multiple Myeloma
              </a>
            </td>
          <td>
            Lorea Jordana-Urriza, G. Serrano, Sergio Camara, M. E. Calleja-Cervantes, P. S. Martín-Uriz, A. Zabaleta, A. Oliver-Cáldes, M. Español-Rego, D. Alignani, Teresa Lozano, Saray Rodríguez-Díaz, Elena Iglesias, Valentin Cabañas, J. L. Reguera, V. González-Calle, M. Mateos, F. Sánchez-Guijo, Bruno Paiva, J.J. Lasarte, S. Inogés, A. López-Díaz de Cerio, A. González-Navarro, Manel Juan, C. Fernández de Larrea, Esteban Tamariz, A. Alfonso-Pierola, Paula Rodriguez-Otero, Jesús San-Miguel, Mikel Hernaez, J. R. Rodríguez-Madoz, Felipe Prosper
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Radiation-induced skin injury (RISI) commonly manifests in cancer patients undergoing radiotherapy (RT). However, a universally accepted standard for treating radiation injury has not yet been established. Our objective was to provide a detailed molecular overview of skin pre- and post-radiation therapy, aiming to enhance our understanding of the subclusters and molecular mechanisms contributing to radiodermatitis. Methods C57BL/6 mice were subjected to a single fraction (20 Gy) of RT targeting the right dorsal skin. We then employed integrated single-cell RNA sequencing (scRNA-seq) to analyze skin samples from mice at 7 and 30 days after radiation exposure, as well as from non-irradiated mice. The Seurat analysis pipeline, Cellchat, SCP, and ssGSEA were used to define the cell types and mechanisms involved in radiation-induced skin injury. Reverse transcription polymerase chain reaction (RT-PCR), multiplex immunofluorescent staining, and other datasets (GSE130183, GSE193564, and GSE193807) were used to validate our findings. Results Thirty-two distinct cell clusters encompassing 71,412 cells were identified. We discovered that cycling keratinocytes (KCs), with the BMP signaling pathway enriched, could activate the Wnt pathway, as well as the SMAD pathways, driving the wound healing and fibrosis processes in RISI. Terminally differentiated secretory-papillary fibroblasts (Fibs) are capable of attracting immune cells, which contributes to the pathogenesis of RISI. Lymphatic endothelial cells (ECs) with pro-inflammatory properties play a critical role in the pathogenesis of RISI by facilitating leukocyte migration. Our analysis also highlighted enhanced ligand-receptor interactions, notably the interactions between chemokines like CXCL10, CCL2, and ACKR1, across subclusters of inflammatory KCs, Fibs, ECs, and immune cells, underscoring their pivotal role in leukocyte recruitment in RISI. Conclusions Cycling KCs, secretory-papillary Fibs, and lymphatic ECs play critical roles in RISI progression. Targeting the interactions of these subclusters with immune cells might help improve the severity of RISI. Furthermore, our study provides a valuable resource for understanding the interactions among immune cells in the context of RISI. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00596-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77637e1bbb3f686235eb0539c52ccaa5035493bf" target='_blank'>
              Molecular profiling of skin cells identifies distinct cellular signatures in radiation-induced skin injury across various stages in the murine dataset
              </a>
            </td>
          <td>
            Hongxuan Yu, Tao Zhong, Ying Xu, Zengfu Zhang, Jiachun Ma, Jupeng Yuan, Minglei Wang, Meng Wu, Jinming Yu, Yuequn Ma, Dawei Chen
          </td>
          <td>2025-02-25</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Understanding the proteomic-level heterogeneity of the tumor microenvironment (TME) in colorectal cancer (CRC) is crucial due to its well-known heterogeneity. While heterogenous CRC has been extensively characterized at the molecular subtype level, research into the functional heterogeneity of fibroblasts, particularly their relationship with extracellular matrix (ECM) alterations, remains limited. Addressing this gap is essential for a comprehensive understanding of CRC progression and the development of targeted therapies.


METHODS
24 tissue samples from 21 CRC patients, along with adjacent normal tissues (NAT), were collected and decellularized using a detergent-based method to enrich the ECM component. Proteomic analysis of ECM-enriched samples was performed using tandem mass tag (TMT) spectrometry, followed by statistical analysis including differential expression protein (DEP) analysis. Single-cell RNA sequencing (scRNA-Seq) data from public datasets were integrated and analyzed to delineate cell states within the TME. Bulk tissue RNA-Seq and bioinformatics analysis, including consensus molecular subtype (CMS) classification and single-cell level deconvolution of TCGA bulk RNA-seq data, were conducted to further explore gene expression patterns and TME composition.


RESULTS
Differential cellular origin of the NAT and tumorous ECM proteins were identified, revealing 110 ECM proteins enriched in NAT and 28 ECM proteins in tumor tissues. Desmoplastic and WNT5A+ inflammatory fibroblasts were indicated as the sources of tumor-enriched ECM proteins, while ADAMDEC1+ expressing fibroblasts and PI16+ expressing fibroblast were identified as the sources of NAT-enriched ECM proteins. Deconvolution of bulk RNA-seq of CRC tissues discriminated CMS-specific fibroblast state, reflecting the biological traits of each CMS subtype. Specially, seven ECM genes specific to mesenchymal subtype (CMS4), including PI16+ fibroblast-related 4 genes (SFRP2, PRELP, OGN, SRPX) and desmoplastic fibroblast-related 3 genes (THBS2, CTHRC1, BGN), showed a significant association with poorer survival in patient with CRC.


CONCLUSION
We conducted an extracellular matrix (ECM)-focused profiling of the TME by integrating quantitative proteomics with single-cell RNA sequencing (scRNA-seq) data from CRC patients. We identified the ECM proteins of NAT and tumor tissue, and established a cell-matrisome database. We defined mesenchymal subtype-specific molecules associated with specific fibroblast subtypes showing a significant association with poorer survival in patients with CRC. Our ECM-focused profiling of tumor stroma provides new insights as indicators for biological processes and clinical endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2f30006d88490586d800953c99040977f4d92b" target='_blank'>
              Differential cellular origins of the extracellular matrix of tumor and normal tissues according to colorectal cancer subtypes.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Sangkeun Park, Jun Hyeong Lee, Shin Young Chang, Sang Mi Oh, S. Mun, Junho Kang, Jong-Eun Park, Jung Kyoon Choi, Tae Il Kim, Jin Young Kim, Pilnam Kim
          </td>
          <td>2025-03-03</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c39aa857bbfd9543aa84644d12df58e1a71f0a" target='_blank'>
              Single-cell and spatial RNA sequencing identify divergent microenvironments and progression signatures in early- versus late-onset prostate cancer.
              </a>
            </td>
          <td>
            Yifei Cheng, Bingxin Liu, Junyi Xin, Xiaobin Wu, Wenchao Li, Jinwei Shang, Jiajin Wu, Zhengdong Zhang, Bin Xu, Mulong Du, Gong Cheng, Meilin Wang
          </td>
          <td>2025-04-10</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c654520970222f7cd3694a92d7a5cfe3581a7e0" target='_blank'>
              Identification of new biomarkers of hepatic cancer stem cells through proteomic profiling.
              </a>
            </td>
          <td>
            S. H. Choi, Ha Young Lee, Sung Ho Yun, Sung Jae Jang, Seung-Up Kim, Jun Yong Park, S. Ahn, D. Y. Kim
          </td>
          <td>2025-03-20</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0608cf84711b7461bf4c33725871d3360fc3c9a5" target='_blank'>
              Abstract B047: Stromal senescence and response to immune checkpoint inhibition in melanoma
              </a>
            </td>
          <td>
            Giselle Burton Sojo, Marcel O. Schmidt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Canine cutaneous mast cell tumors (MCTs) are a common, yet clinically challenging tumor type given their variable biological behavior. Although patients with low grade MCTs can often be effectively managed with surgery alone, most dogs with high grade MCTs succumb to their disease despite multimodal therapy. An improved understanding of the immune tumor microenvironment (TME) may help identify novel prognostic and therapeutic targets. Methods In this study, we interrogated the immune transcriptional profiles of the TME in low and high grade MCTs, and quantified intratumoral T cells. Twelve client-owned dogs with MCTs (6 Kiupel low grade with clinically benign behavior and 6 Kiupel high grade with clinically aggressive behavior) that underwent curative-intent surgery were selected. Tumor grade was confirmed by a single veterinary pathologist. RNA was extracted from all tumors followed by immune transcriptional profiling utilizing the NanoString Canine IO panel and analysis using the ROSALIND platform. T cell density was determined by immunohistochemical staining for CD3 and quantified using ImageScope software (Leica Biosystems) following digital slide capture. Lymphocytic infiltrate was further characterized in the TME of one high grade MCT using co-immunofluorescence. Results Immune transcriptional profiling identified 9 differentially expressed genes between low and high grade MCTs (p-adj < 0.05). Programmed cell death protein 1 (PDCD1) and inducible T-cell costimulator ligand (ICOSLG) gene expression were significantly higher in a subset of high grade MCTs. ICOSLG expression positively correlated with T cell score (rs = 0.6434, p = 0.0278). Although the T cell density was not significantly different between low (mean of 76.42 CD3 + /mm2, SD 12 CD3 + /mm2) and high grade MCTs (mean of 129.1 CD3 + /mm2, SD 96.06 CD3 + /mm2), greater variation of T cell densities was observed across high grade MCTs compared to low grade (p = 0.0059). Immunofluorescence of one high grade MCT with marked T cell infiltration revealed organized aggregates of T and B cells consistent with tertiary lymphoid structures (TLS). Conclusions Our data revealed significant differences in the immune TME of low and high grade MCTs and provides rationale to further investigate potential prognostic and therapeutic roles of immune checkpoints in canine MCTs. Supplementary Information The online version contains supplementary material available at 10.1186/s44356-025-00020-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b795d410565b7a2d389acbb8118bd3306a43b7c" target='_blank'>
              Intertumoral heterogeneity of the immune microenvironment in high grade canine mast cell tumors
              </a>
            </td>
          <td>
            K. L. Bardales, L. Jiang, E. Radaelli, C. Assenmacher, J. A. Lenz, M. Atherton
          </td>
          <td>2025-03-14</td>
          <td>Veterinary Oncology (London, England)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Effective cancer therapy involves initiation of a tumor-specific immune response. Consequently, the interest in oncolytic viruses (OV) capable of triggering immunogenic cell death has sparked in recent years. However, the common use of pre-clinical models that fail to mirror patient tumor ecosystems (TES) hinders clinical translation. Here, we provide a condensed view on the intricate interplay between several aspects of TES and OV action and discuss these considerations in the view of recently developed pre-clinical human model systems. Given the urgent demand for innovative cancer treatments, the purpose of this review is to highlight the so-far overlooked complex impact of the tumor microenvironment (TME) on OV permissivity, with the intent to provide a foundation for future, more effective pre-clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e2730ad834a6de61d42867b34d6216e57036c8" target='_blank'>
              Deciphering permissivity of human tumor ecosystems to oncolytic viruses.
              </a>
            </td>
          <td>
            Benjamin Schoeps, Ulrich M Lauer, Knut Elbers
          </td>
          <td>2025-03-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7261a40e65b3cb85eca64d1251012a027c2cf0ef" target='_blank'>
              Epigenetic Symphony in Diffuse Large B-Cell Lymphoma: Orchestrating the Tumor Microenvironment
              </a>
            </td>
          <td>
            A. Caloian, Miruna Cristian, Elena Calin, Andreea-Raluca Pricop, Stelian Mociu, Liliana Seicaru, Sorin Deacu, Nicolae Ciufu, A. Suceveanu, A. Suceveanu, L. Mazilu
          </td>
          <td>2025-04-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="MOTIVATION
Spatial transcriptomics holds great promise for revolutionizing biology and medicine by providing gene expression profiles with spatial information. Until recently, spatial resolution has been limited, but advances in high-throughput in situ imaging technologies now offer new opportunities by covering thousands of genes at a single-cell or even subcellular resolution, necessitating databases dedicated to comprehensive coverage and analysis with user-friendly intefaces.


RESULTS
We introduce the HISSTA database, which facilitates the archival and analysis of in situ transcriptome data at single-cell resolution from various human tissues. We have collected and annotated spatial transcriptome data generated by MERFISH, CosMx SMI, and Xenium techniques, encompassing 112 samples and 28 million cells across 16 tissue types from 63 studies. To decipher spatial contexts, we have implemented advanced tools for cell type annotation, spatial colocalization, spatial cellular communication, and niche analyses. Notably, all datasets and annotations are interactively accessible through Vitessce, allowing users to focus on regions of interest and examine gene expression in detail. HISSTA is a unique database designed to manage the rapidly growing dataset of in situ transcriptomes at single-cell resolution. Given its comprehensive data content and advanced analysis tools with interactive visualizations, HISSTA is poised to significantly impact cancer diagnosis, precision medicine, and digital pathology.


AVAILABILITY AND IMPLEMENTATION
HISSTA is freely accessible at https://kbds.re.kr/hissta/. The source code is available at https://doi.org/10.5281/zenodo.14904523.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8847bc6bc6ac698124acae8ba2d55144bc394fe" target='_blank'>
              HISSTA: a human in situ single-cell transcriptome atlas.
              </a>
            </td>
          <td>
            Jiwon Yu, Jiwoo Moon, Minseo Kim, Gyeol Han, Insu Jang, Jinyoung Lim, Seungmook Lee, Seok-Hwan Yoon, Woong-Yang Park, Byungwook Lee, Sanghyuk Lee
          </td>
          <td>2025-03-31</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3b584bb3e59598c626453f550ef2ecaeb21de4" target='_blank'>
              Abstract A042: Uncovering the mechanism of synergistic anti-tumor effects in FAP-CD40 + PD1-IL2v combination therapy
              </a>
            </td>
          <td>
            Thuy Trinh Nguyen, Leo Kunz, Ines Grazina de Matos, Christine Trumpfheller, P. Umaña
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f7a1649cc4c1f5d7fa9fa78dd55642c87055fdf" target='_blank'>
              PD1 expression in immune cells within the tumor microenvironment of patients with non-small cell and small cell lung cancer
              </a>
            </td>
          <td>
            AYu Kalinchuk, EA Tsarenkova, DM Loos, AA Mokh, EO Rodionov, SV Miller, E. Grigoryeva, LA Tashireva
          </td>
          <td>2025-03-01</td>
          <td>Bulletin of Russian State Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c96757ce9569828ea0b78de759efc296d25cef" target='_blank'>
              Abstract A062: Mechanisms mediating hypoxic memory of type I IFN signaling suppression and implications for immune evasion in luminal breast cancer cells
              </a>
            </td>
          <td>
            Rebecca Marker, Aidan Moriarty, Remi Klotz, Min Yu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2d7a33282302100a7faa70ebce158c29857365" target='_blank'>
              Cancer-associated fibroblasts: multidimensional players in liver cancer
              </a>
            </td>
          <td>
            Yanyun Pan, Yuangang Qiu, Xinbin Zhou, Wei Mao, Xiao-ming Xu
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb5f12dc4ee23891e41c491f92b49e0c1d5cff9" target='_blank'>
              Targeting CXCR4-expressing TAMs in muscle-invasive bladder cancer to enhance tumor control after immunotherapy
              </a>
            </td>
          <td>
            E. Desponds, H. El ahanidi, Nagham Alouche, Hana Zdimerova, Stéphanie Favre, Giulio Zanette, Sina Nassiri, Daniel Benamran, Petros K Tsantoulis, Mohammed Attaleb, Hélène Maby-El Hajjami, Julien Dagher, Vanessa Gourhand, K. Balabanian, M. Espéli, S. Luther, Camilla Jandus, M. Leblond, G. Verdeil
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary Despite multimodal interventions, managing glioblastoma remains challenging. Beyond proliferation, invasion, and evasion, cells are resistant to chemo- and radiotherapy. In GBM, several mechanisms are in play that contribute to the creation and maintenance of a highly immunosuppressive microenvironment, promoting tumor growth and immune escape. Furthermore, the high accumulation of immunosuppressive cells, the pronounced expression of immune checkpoint molecules, and the low number of neoantigens are hallmarks of GBM and contribute to the challenge of therapeutic approaches. In this review, we will discuss how GBM fosters tumor growth and explore treatment options such as small molecules, immunotherapies, and cell therapies, and their potential in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396c9dc36357f3e71cd455bca6bdc89bb76a553a" target='_blank'>
              Immune Cell Interplay in the Fight Against GBM
              </a>
            </td>
          <td>
            Nico Vallieri, Angeliki Datsi
          </td>
          <td>2025-02-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer-associated myeloid cells due to their plasticity play dual roles in both promoting and inhibiting tumor progression. Myeloid cells with immunosuppressive properties play a critical role in anti-cancer immune regulation. Cells of different origin, such as tumor associated macrophages (TAMs), tumor associated neutrophils (TANs), myeloid derived suppressor cells (also called MDSCs) and eosinophils are often expanded in cancer patients and significantly influence their survival, but also the outcome of anti-cancer therapies. For this reason, the variety of preclinical and clinical studies to modulate the activity of these cells have been conducted, however without successful outcome to date. In this review, pro-tumor activity of myeloid cells, myeloid cell-specific therapeutic targets, in vivo studies on myeloid cell re-polarization and the impact of myeloid cells on immunotherapies/genetic engineering are addressed. This paper also summarizes ongoing clinical trials and the concept of chimeric antigen receptor macrophage (CAR-M) therapies, and suggests future research perspectives, offering new opportunities in the development of novel clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8cd651287c208b9fc2804d17f7c89b490c63e13" target='_blank'>
              Harnessing myeloid cells in cancer
              </a>
            </td>
          <td>
            Su-Yeon Park, E. Pylaeva, Vikas Bhuria, A. Gambardella, Giovanna Schiavoni, D. Mougiakakos, Sung-Hoon Kim, Jadwiga Jablonska
          </td>
          <td>2025-03-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9a98022a15f5fa6336dcfa874e713b8fce8db6" target='_blank'>
              AGP-Net: A Universal Network for Gene Expression Prediction of Spatial Transcriptomics
              </a>
            </td>
          <td>
            Yan Yang, Xuesong Li, Liyuan Pan, Guoxun Zhang, Liu Liu, Eric A. Stone
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is responsible for over 900,000 annual deaths worldwide. Emerging evidence supports pro‐carcinogenic bacteria in the colonic microbiome are at least promotional in CRC development and may be causal. We previously showed toxigenic C. difficile from human CRC‐associated bacterial biofilms accelerates tumorigenesis in Apc Min/+ mice, both in specific pathogen‐free mice and in gnotobiotic mice colonized with a defined consortium of bacteria. To further understand host–microbe interactions during colonic tumorigenesis, we combined single‐cell RNA‐sequencing (scRNA‐seq), spatial transcriptomics, and immunofluorescence to define the molecular spatial organization of colonic dysplasia in our consortium model with or without C. difficile. Our data show a striking bipartite regulation of Deleted in Malignant Brain Tumors 1 (DMBT1) in the inflamed versus dysplastic colon. From scRNA‐seq, differential gene expression analysis of normal absorptive colonocytes at 2 weeks postinoculation showed DMBT1 upregulated by C. difficile compared to colonocytes from mice without C. difficile exposure. In contrast, our spatial transcriptomic analysis showed DMBT1 dramatically downregulated in dysplastic foci compared with normal‐adjacent tissue. We further integrated our datasets to generate custom colonic dysplasia scores and ligand‐receptor mapping. Validation with immunofluorescence showed DMBT1 protein downregulated in dysplastic foci from three mouse models of colonic tumorigenesis and in adenomatous dysplasia from human samples. Finally, we used mouse and human organoids to implicate WNT signaling in the downregulation of DMBT1 mRNA and protein. Together, our data reveal cell type‐specific regulation of DMBT1, a potential mechanistic link between bacteria and colonic tumorigenesis. Published 2025. This article is a U.S. Government work and is in the public domain in the USA. The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7def78e585a3b57bba506b3445109c092c108f8" target='_blank'>
              Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia
              </a>
            </td>
          <td>
            Emily H Green, Subhag R. Kotrannavar, Megan E. Rutherford, Hannah M. Lunnemann, Harsimran Kaur, Cody N. Heiser, Hua Ding, A. Simmons, Xiao Liu, Borden Lacy, Kay Washington, M. Shrubsole, Qi Liu, Ken S. Lau, Cynthia L Sears, Robert J. Coffey, Julia L. Drewes, Nicholas O Markham
          </td>
          <td>2025-03-03</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a751c087d4afc3ab315830aaec4e9b6e54e60f3" target='_blank'>
              Abstract A060: Reactive oligodendrocytes promote glioblastoma progression through CCL5/CCR5-mediated glioma stem cell maintenance
              </a>
            </td>
          <td>
            Jason Moffat
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d437c0ab0bc7d73f3a75511dfcad72400900dc7a" target='_blank'>
              Abstract B005: Identification of tumor-reactive TILs from lung cancer organoids co-culture through ruthenium-based proximity labeling
              </a>
            </td>
          <td>
            Weilei Hu, Yifei Bian, Zihan Zhao, Shian Ouyang, Jie P Li, Hongbin Ji
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) plays a critical role in the development, progression, and clinical outcomes of hepatocellular carcinoma (HCC). Despite the critical role of natural killer (NK) cells in tumor immunity, there is limited research on their status within the tumor microenvironment of HCC. In this study, single-cell RNA sequencing (scRNA-seq) analysis of HCC datasets was performed to identify potential biomarkers and investigate the involvement of natural killer (NK) cells in the TME. Methods Single-cell RNA sequencing (scRNA-seq) data were extracted from the GSE149614 dataset and processed for quality control using the “Seurat” package. HCC subtypes from the TCGA dataset were classified through consensus clustering based on differentially expressed genes (DEGs). Weighted gene co-expression network analysis (WGCNA) was employed to construct co-expression networks. Furthermore, univariate and multivariate Cox regression analyses were conducted to identify variables linked to overall survival. The single-sample gene set enrichment analysis (ssGSEA) was used to analyze immune cells and the screened genes. Result A total of 715 DEGs from GSE149614 and 864 DEGs from TCGA were identified, with 25 overlapping DEGs found between the two datasets. A prognostic risk score model based on two genes was then established. Significant differences in immune cell infiltration were observed between high-risk and low-risk groups. Immunohistochemistry showed that HRG expression was decreased in HCC compared to normal tissues, whereas TUBA1B expression was elevated in HCC. Conclusion Our study identified a two-gene prognostic signature based on NK cell markers and highlighted their role in the TME, which may offer novel insights in immunotherapy strategies. Additionally, we developed an accurate and reliable prognostic model, combining clinical factors to aid clinicians in decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2c5ed50f3e7bc86c42a2f717d3b32238dbfc13e" target='_blank'>
              Uncovering the heterogeneity of NK cells on the prognosis of HCC by integrating bulk and single-cell RNA-seq data
              </a>
            </td>
          <td>
            Jiashuo Li, Zhenyi Liu, Gongming Zhang, Xue Yin, Xiaoxue Yuan, Wen Xie, Xiaoyan Ding
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioma, the most common primary cancer of the central nervous system, characterizes significant heterogeneity, presenting major challenges for therapeutic approaches and prognosis. In this study, the interactions between malignant glioma cells and macrophages/monocytes, as well as their influence on tumor progression and treatment responses, were explored using comprehensive single-cell RNA sequencing analysis. Methods RNA-seq data from the TCGA and CGGA databases were integrated and an in-depth analysis of glioma samples was performed using single-cell RNA sequencing, functional enrichment analysis, developmental trajectory analysis, cell–cell communication analysis, and gene regulatory network analysis. Furthermore, a prognostic model based on risk scores was developed, and its predictive performance was assessed through immune cell infiltration analysis and immune treatment response evaluation. Results Fourteen distinct glioma cellular subpopulations, seven primary cell types, and four macrophage/monocyte subtypes were identified. Developmental trajectory analysis offered insights into the origins and heterogeneity of malignant cells as well as macrophages/monocytes. Cell communication analysis revealed the interaction of macrophages and monocytes with malignant cells through several pathways, including the macrophage migration inhibitory factor and secreted phosphoprotein 1 pathways, engaging in key ligand-receptor interactions that influence tumor behavior. Categorization based on these communication characteristics was significantly correlated with overall survival. Immune cell infiltration analysis highlighted variations in immune cell abundance across different subgroups, possibly linked to differing responses to immunotherapy. This predictive model, comprising 29 prognostic genes, demonstrated high accuracy and robustness across multiple independent cohorts. Conclusion This study reveals the complex heterogeneity of the glioma microenvironment and enhances the understanding of diverse characteristics of glioma cell subsets. At the same time, it lays a foundation for the development of therapeutic strategies and prognostic models targeting the glioma microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Skin cancer is the most common malignancy worldwide, comprising three major types: melanoma, basal cell carcinoma (BCC), and squamous cell carcinoma (SCC). A critical goal in advancing our understanding of skin cancer is to identify the shared and distinct biological mechanisms that drive the risk profiles for each skin cancer type. In this study, we integrated tissue-specific chromatin conformation and expression quantitative trait loci data (spatial eQTL) to construct gene regulatory networks (GRNs) for melanocytes, sun-exposed skin, not sun-exposed skin, and blood. These GRNs, which capture spatially regulated gene expressions, were used as instrumental variables along with GWAS summary statistics for melanoma, BCC and SCC, to infer causal relationships between specific gene expression changes and each type of skin cancer. These Mendelian randomization analyses identified 82, 62, and 125 causal genes for melanoma, BCC, and SCC, respectively, with many of these genes not evident from the GWAS data alone. Our analyses revealed distinct mechanisms for each skin cancer type: telomere maintenance and nevus pathways in melanoma, inherited immune traits in BCC, and p53 dysfunction in SCC. Notably, apoptosis and pigmentation emerged as shared biological processes across skin cancers. These findings provide new insights into the genetic drivers of three distinct skin cancers, highlighting new gene targets which can be used to increase diagnostic precision, as well as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1a0a87d4f015e13185fa2728ac9fd439bf3488" target='_blank'>
              Mendelian randomizations with spatial gene networks reveal shared and distinct drivers of risk in major skin cancer types
              </a>
            </td>
          <td>
            Michael Pudjihartono, J. O’Sullivan, W. Schierding
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec33c3cfadc05d6eacf145975ac6bfa380605adc" target='_blank'>
              Artificial intelligence-enabled automated analysis of transmission electron micrographs to evaluate chemotherapy impact on mitochondrial morphology in triple negative breast cancer
              </a>
            </td>
          <td>
            Argenis Arriojas, Lily M. Baek, Mariah J. Berner, Alexander Zhurkevich, Antentor Hinton, M. D. Meyer, L. Dobrolecki, Michael T. Lewis, Kourosh Zarringhalam, Gloria V. Echeverria
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a deadly cancer with an aggressive nature, and how CRC tumor cells manage to translocate and proliferate in a new tissue environment remains not fully understood. Recently, higher-order chromatin structures and spatial genome organization are increasingly implicated in diseases including cancer, but in-depth studies of three-dimensional genome (3D genome) of metastatic cancer are currently lacking, preventing the understanding of the roles of genome organization during metastasis. Here we perform multi-omics profiling of matched normal colon, primary tumor, lymph node metastasis, liver metastasis and normal liver tissue from CRC patients using Hi-C, ATAC-seq and RNA-seq technologies. We find that widespread alteration of 3D chromatin structure is accompanied by dysregulation of genes including SPP1 during the tumorigenesis or metastasis of CRC. Remarkably, the hierarchy of topological associating domain (TAD) changes dynamically, which challenges the traditional view that the TAD structure between tumor and normal tissue is conservative. In addition, we define compartment stability score to measure large-scale alteration in metastatic tumors. To integrate multi-omics data and recognize candidate genes driving cancer metastasis, a pipeline is developed based on Hi-C, RNA-seq and ATAC-seq data. And three candidate genes ARL4C, FLNA, and RGCC are validated to be associated with CRC cell migration and invasion using in vitro knockout experiments. Overall, these data resources and results offer new insights into the involvement of 3D genome in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac5e3595c3552c277cd820f66cb6eb10faa0266" target='_blank'>
              MDA-9/Syntenin as a therapeutic cancer metastasis target: current molecular and preclinical understanding.
              </a>
            </td>
          <td>
            Swadesh K. Das, Paul B Fisher
          </td>
          <td>2025-03-08</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This study comprehensively investigated keratinocyte subpopulation heterogeneity and developmental trajectories during skin aging using single-cell sequencing, transcriptomics, and facial aging-related genome-wide association studies (GWAS) data. We identified three major subpopulations: basal cells (BCs), spinous cells (SCs), and IFI27+ keratinocytes. Single-cell pseudotime analysis revealed that basal cells can differentiate along two distinct paths: toward spinous differentiation or the inflammatory state. With aging, the proportion of IFI27+ cells significantly increased, displaying more active inflammatory and immunomodulatory signals. Through cell–cell communication analysis, we found that the signaling pathways, including NOTCH, PTPR, and PERIOSTIN, exhibited distinct characteristics along different branches. Integration of the GWAS data revealed significant loci on chromosomes 2, 3, 6, and 9 that were spatially correlated with key biological pathways (including antigen processing, oxidative stress, and apoptosis). These findings reveal the complex cellular and molecular mechanisms underlying skin aging, offering potential targets for novel diagnostic approaches and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d6ff4c88e6b6ec5e9e3e8325f958c0ceb91730" target='_blank'>
              Inflammatory Transformation of Skin Basal Cells as a Key Driver of Cutaneous Aging
              </a>
            </td>
          <td>
            Shupeng Liu, Sheng Lu, Zhiping Pang, Jiacheng Li, Meijuan Zhou, Zhenhua Ding, Zhijun Feng
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Malignant pleural mesothelioma (MPM) is a rare and aggressive malignancy with limited therapeutic options. To improve patients management and treatment, more precise stratification strategies are needed. This study aimed to characterize the phenogenomic landscapes of MPM and to understand their influence on patients clinical outcomes. Methods We conducted a phenogenomic analysis on 22 MPM patients using two high throughput approaches: imaging mass cytometry (IMC) with whole exome sequencing (WES). Resulting profiles were addressed for their clinical relevance to predict patients prognosis. Results IMC revealed a highly heterogeneous tumor microenvironment (TME) with distinct tumor cell subpopulations. Notably, we identified a novel sarcomatoid-like cellular cluster associated with poor prognosis. The TME was also infiltrated with immune cells including macrophages and CD4+ T lymphocytes, that were more abundant in patients with favorable clinical outcomes. WES identified a complex genomic landscape with limited prognostic value for individual genetic alterations. However, tumor mutational burden (TMB) emerged as a potential predictive biomarker, inversely correlating with immune cell infiltration, particularly macrophages and CD4+ T lymphocytes. Conclusions Our findings underscore the intricate interplay between the tumor genome, TME composition, and clinical outcomes in MPM. These data support the potential of integrating genomic and TME profiling to develop more precise patient stratification strategies and potentially optimize therapeutic approaches, including immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06193-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352ed6abb5fc9f899a1ac46662fa50876ab4dfda" target='_blank'>
              In Situ Gene Engineering Approach to Overcome Tumor Resistance and Enhance T Cell-Mediated Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Di Zhang, Wenjuan Wang, Mingtan Tang, Chenghao Qu, Zeyu Jiang, Xuerui Li, Yuxia Luan
          </td>
          <td>2025-04-03</td>
          <td>Nano letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The density of tumor-infiltrating lymphocytes (TILs) serves as a valuable indicator for predicting anti-tumor responses, but its broad impact across various types of cancers remains underexplored. We introduce TILScout, a pan-cancer deep-learning approach to compute patch-level TIL scores from whole slide images (WSIs). TILScout achieved accuracies of 0.9787 and 0.9628, and AUCs of 0.9988 and 0.9934 in classifying WSI patches into three categories—TIL-positive, TIL-negative, and other/necrotic—on validation and independent test sets, respectively, surpassing previous studies. The biological significance of TILScout-derived TIL scores across 28 cancers was validated through comprehensive functional and correlational analyses. A consistent decrease in TIL scores with an increase in cancer stage provides direct evidence that the lower TIL content may stimulate cancer progression. Additionally, TIL scores correlated with immune checkpoint gene expression and genomic variation in common cancer driver genes. Our comprehensive pan-cancer survey highlights the critical prognostic significance of TILs within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff714f6b11d2c56c9bba32cdb7dcdd9b8c1deb5a" target='_blank'>
              Prediction and analysis of tumor infiltrating lymphocytes across 28 cancers by TILScout using deep learning
              </a>
            </td>
          <td>
            Huibo Zhang, Lulu Chen, Lan Li, Yang Liu, Barnali Das, Shuang Zhai, Juan Tan, Yan Jiang, Simona Turco, Yi Yao, Dmitrij Frishman
          </td>
          <td>2025-03-19</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The intratumoral mycobiome plays a crucial role in the tumor microenvironment, but its impact on renal cell carcinoma (RCC) remains unclear. We collected and quantitatively profiled the intratumoral mycobiome data from 1044 patients with RCC across four international cohorts, of which 466 patients received immunotherapy. Patients were stratified into mycobiota ecology-depauperate and mycobiota ecology-flourishing (MEF) groups based on fungal abundance. The MEF group had worse prognosis, higher fungal diversity, down-regulated lipid catabolism, and exhausted CD8+ T cells. We developed the intratumoral mycobiota signature and intratumoral mycobiota-related genes expression signature, which robustly predicted prognosis and immunotherapy outcomes in RCC and other cancers. Aspergillus tanneri was identified as a potential key fungal species influencing RCC prognosis. Our findings suggest that the intratumoral mycobiome suppresses lipid catabolism and induces T cell exhaustion in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d247d376bb04aaad37a3a7086a6eccbe76de5ac" target='_blank'>
              Intratumoral mycobiome heterogeneity influences the tumor microenvironment and immunotherapy outcomes in renal cell carcinoma
              </a>
            </td>
          <td>
            Weiming Mou, Zhixing Deng, Lingxuan Zhu, Aimin Jiang, Anqi Lin, Liling Xu, G. Deng, Hongsen Huang, Zeji Guo, Bang Zhu, Shuqi Wu, Tao Yang, Lu Wang, Zaoqu Liu, Ting Wei, Jian Zhang, Liang Cheng, Haojie Huang, Rui Chen, Yi Shao, Quan Cheng, Linhui Wang, Shuofeng Yuan, Peng Luo
          </td>
          <td>2025-04-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized oncology, offering new hope for patients with previously incurable cancers. However, solid tumors remain a significant challenge due to immune evasion, therapeutic resistance, and the immunosuppressive tumor microenvironment. Exosomes, a specialized subset of extracellular vesicles, have emerged as promising tools in cancer therapy owing to their unique role in intercellular communication and immune modulation. These vesicles transport antigens, major histocompatibility complex (MHC) molecules, and immune-modulatory cargo, positioning them as potential platforms for cancer vaccines, drug delivery systems, and combinatorial therapies. Advances in engineered exosomes have improved drug bioavailability, tumor targeting, and immune stimulation, showcasing their potential in personalized medicine. This review highlights their multifaceted role in the tumor microenvironment, and their mechanisms of action in solid cancer therapy. Additionally, we discuss emerging strategies to overcome clinical and technical hurdles, paving the way for novel and effective cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596ee4edf11886df1275d2da9035d5dfc330d982" target='_blank'>
              Exosomes and solid cancer therapy: where are we now?
              </a>
            </td>
          <td>
            T. Zemanek, L. Danišovič, Andreas Nicodemou
          </td>
          <td>2025-02-17</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma, an aggressive and lethal brain tumor, presents enormous clinical challenges, including molecular heterogeneity, high recurrence rates, resistance to conventional therapies, and limited therapeutic penetration across the blood-brain barrier. The glioblastoma microenvironment, characterized by a dynamic interplay of cellular and non-cellular components, is a key driver of tumor growth and therapeutic resistance. Neuroinflammatory cytokines, particularly interleukins and tumor necrosis factor-alpha, play pivotal roles in this microenvironment, contributing to tumor progression and immune evasion. This review highlights oncolytic virotherapy as a promising therapeutic avenue, focusing on its potential to modulate neuroinflammatory responses, induce localized immune reactions, and deliver immunomodulatory factors directly to the tumor site. While encouraging outcomes have been observed, challenges such as overcoming the blood-brain barrier, managing host antiviral immunity, and mitigating potential risks to normal neuronal cells remain critical barriers to clinical translation. By analyzing the intricate interactions of oncolytic viruses with the glioblastoma microenvironment and synthesizing findings from preclinical and clinical trials, this review provides actionable insights into developing personalized and effective therapeutic strategies for this aggressive tumor based on oncolytic virotherapy alone or when using it combined with conventional therapies, immunotherapy, natural killer-cell therapy, chimeric antigen receptor-T cell therapy, and dendritic cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a5a0c8848cf51f5c2b99d53d48fb53289811cd" target='_blank'>
              Unmasking the potential: mechanisms of neuroinflammatory modulation by oncolytic viruses in glioblastoma
              </a>
            </td>
          <td>
            Narimene Beder, S. N. Mirbahari, M. Belkhelfa, Hamid Mahdizadeh, Mehdi Totonchi
          </td>
          <td>2025-02-24</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background and purpose Promoting adaptive immunity with ICIs has drastically improved melanoma prognosis, but not for all patients. Some cases relapse in the first few months, while others keep durable benefit, even after immunotherapy discontinuation. To identify cellular/molecular signatures in peripheral blood that could differentiate advanced from metastatic melanoma and predict dynamics for primary/secondary immune escape, we examined 100 consecutive patients with stage III/IV melanoma scheduled to start ICIs. Materials and methods At melanoma diagnosis, a multiparameter flow cytometric analysis and purification scheme using standard conjugated antibodies were performed for all individuals prior to ICI initiation. In each stage(III/IV) according to their RFS/PFS, we retrospectively selected the cases with the clearest clinical outcomes and focused our analysis on the extreme responders(n=7) and non-responders(n=7) to characterize the transcriptomes of circulating CD4+T-cells by bulk RNA-seq, Differential Expression Analysis(DEA)and Gene Ontology(GO)enrichment analysis. Based on our selected patient cohort, we examined for differentially expressed genes(DEGs)and key-pathways that appear preferentially activated in stage III vs. IV melanoma, and in long vs. short immunotherapy responders. Results Although circulating immune-cells did not numerically differ in both sets of analysis(staging and ICI responsiveness), DEA and GO data showed that patients could be clustered separately, identifying 189vs.92 DEGs in stage IV/III and 101vs.47 DEGs in early progressors/long responders. These DEGs were functionally implicated in distinct pathways. For metastatic cases: inflammatory response(logp-value=-9.2:ADGRE5/2,CYBA,GRN,HMOX1,IRF5,ITGAM), adaptive immunity(logp-value=-7.7:CD1C,CD74,CYBB,NCF2,CTSA,S100A8/9,BCL3,FCER1G), T-cell activation(logp-value=-6.3:BCL3,CD1C,CD74,FCER1G,FGL2)and lipid metabolism/catabolism(logp-value=-2.5/-2.6:ARF3,GPX1,MVD,OCRL,PCCB,CTSA,PNPLA2,NAGLU,GBA2,ABHD4); while in early-progressors to ICIs: immune effector processing(logp-value=-13.7:BCL6,FGR,HLA-DQA1/DQB1,HLA-DRA,HLA-DRB1/DRB5,NKG7,SLC11A1,TYROBP,SPON2,HAVCR2),PD-1(logp-value=-10.2:HLA-DQA1/DQB1,HLA-DRA,HLA-DRB1/DRB5)and IFN signaling(logp-value=-8.5: HLA-DQA1/DQB1,HLA-DRA,HLA-DRB1/DRB5,NCAM1,IFITM3),positive regulation of T-cell activation(logp-value=-7.7:BCL6,HLA-DQA1/DQB1,HLA-DRA,HLA-DRB1/DRB5,SASH3,HAVCR2)and CD28 co-stimulation(logp-value=-10.3:HLA-DQA1/DQB1,HLA-DRA,HLA-DRB1/DRB5), supporting an immune-mediated behavior. Conclusions Specific pathways and marker genes in the peripheral CD4+T-cells may predetermine melanoma staging and immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38cb25a7204b29ac40ea7b1554aafd50200afb9b" target='_blank'>
              Transcriptomic signatures in peripheral CD4+T-lymphocytes may reflect melanoma staging and immunotherapy responsiveness prior to ICI initiation
              </a>
            </td>
          <td>
            Eleni Palli, Matthieu Lavigne, Panagiotis Verginis, T. Alissafi, Amalia Anastasopoulou, G. Lyrarakis, John M. Kirkwood, Helen Gogas, Dimitrios C. Ziogas
          </td>
          <td>2025-03-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aeeccc03e491382af67e5b76f7835dfb0b4a41" target='_blank'>
              508 The Largest Patient-Derived Meningioma Organoid Bank Demonstrates a Robust Model for Transcriptional Analysis of Tumors
              </a>
            </td>
          <td>
            Youssef M. Zohdy, Arman Jahangiri, Fadi Jacob, A. Aksionau, Alejandra Rodas, Zhexing Wen, Gustavo Pradilla, Tomas Garzon-Muvdi
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Localized prostate cancer (PCa) is highly variable in their response to therapies. Although a fraction of this heterogeneity can be explained by clinical factors or genomic and transcriptomic profiling, the proteomic-based profiling of aggressive PCa remains poorly understood. Here, we profiled the genome, transcriptome, proteome and phosphoproteome of 145 cases of localized PCa in Chinese patients. Proteome-based stratification of localized PCa revealed three subtypes with distinct molecular features: immune subgroup, arachidonic acid metabolic subgroup and sialic acid metabolic subgroup with highest biochemical recurrence (BCR) rates. Further, we nominated NANS protein, a key enzyme in sialic acid synthesis as a potential prognostic biomarker for aggressive PCa and validated in two independent cohorts. Finally, taking advantage of cell-derived orthotopic transplanted mouse models, single-cell RNA sequencing (scRNA-seq) and immunofluorescence analysis, we revealed that targeting NANS can reverse the immunosuppressive microenvironment through restricting the sialoglycan-sialic acid-recognizing immunoglobulin superfamily lectin (Siglec) axis, thereby inhibiting tumor growth of PCa. In sum, we integrate multi-omic data to refine molecular subtyping of localized PCa, and identify NANS as a potential prognostic biomarker and therapeutic option for aggressive PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9febd685653f7715435bba7b29d1b13ce378312" target='_blank'>
              Integrated proteogenomic characterization of localized prostate cancer identifies biological insights and subtype-specific therapeutic strategies
              </a>
            </td>
          <td>
            Wei Ou, Xin-Xin Zhang, Bin Li, Ying Tuo, Ren-Xuan Lin, Pengfei Liu, Jianping Guo, Hiocheng Un, Minghao Li, Jiahao Lei, Xiao-Jing Gao, Fu-Fu Zheng, Ling-Wu Chen, Ling-Li Long, Zongren Wang
          </td>
          <td>2025-04-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b339228b07860b7f7b0ff107251f16ed54dc97" target='_blank'>
              Tumor microenvironment as a novel therapeutic target for lymphoid leukemias.
              </a>
            </td>
          <td>
            Shahrzad Mousavi, Soheil Nouri, Arezoo Sadeghipour, Amir Atashi
          </td>
          <td>2025-02-25</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48bced03940233897a4f47121718b37443e74642" target='_blank'>
              Dissection of single-cell landscapes for the development of chimeric antigen receptor T cells in Hodgkin lymphoma.
              </a>
            </td>
          <td>
            A. Gottschlich, R. Grünmeier, Gordon Victor Hoffmann, Sayantan Nandi, V. Kavaka, Philipp Jie Müller, Jakob Jobst, A. Oner, R. Kaiser, J. Gärtig, Ignazio Piseddu, Stephanie Frenz-Wiessner, Savannah D Fairley, Heiko Schulz, V. Igl, Thomas Alexander Janert, Lea di Fina, Maité Mulkers, Moritz Thomas, D. Briukhovetska, D. Simnica, Emanuele Carlini, C. Tsiverioti, Marcel P. Trefny, T. Lorenzini, F. Märkl, Pedro Mesquita, Ruben Brabenec, Thaddäus Strzalkowski, Sophia Stock, Stefanos Michaelides, Johannes Hellmuth, M. Thelen, S. Reinke, W. Klapper, P. Gelebart, Leo Nicolai, Carsten Marr, E. Beltrán, Remco T. A. Megens, Christoph Klein, Fanny Baran-Marszak, A. Rosenwald, Michael von Bergwelt-Baildon, P. Bröckelmann, S. Endres, Sebastian Kobold
          </td>
          <td>2025-04-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e408aa172c15a4c549b8743b41474549ca768526" target='_blank'>
              Immunomodulatory role of exosome-derived content in pediatric medulloblastoma: a molecular subgroup perspective.
              </a>
            </td>
          <td>
            Jessica Oliveira de Santis, G. R. de Sousa, Rosane Gomes Queiroz, M. F. Cândido, Fausto Almeida, C. P. de Rezende, Patricia Cassia de Ruy, G. S. Arini, Beth Coyle, Philippa Wade, M. Brassesco, C. Scrideli, L. Tone, E. Valera
          </td>
          <td>2025-02-17</td>
          <td>Human cell</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f892875ae87d787444ae9dc6d5299e4b5611e9" target='_blank'>
              Abstract B018: Ex vivo modeling of precision immuno-oncology responses in lung cancer
              </a>
            </td>
          <td>
            Bassel Alsaed, Johannes Smolander, Lin Linh, N. Bobik, Hanna Laitinen, L. Lahtinen, Mikko Räsänen, S. Jansouz, K. Peltonen, E. Jokinen, J. Klievink, M. Ainola, Eva Sutinen, P. Saharinen, S. Mustjoki, I. Ilonen, H.M. Haikala
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Various challenges, including tumor heterogeneity and inadequate T cell infiltration, impede the progress of chimeric antigen receptor T cell (CAR-T) therapy for glioblastoma (GBM). To address these obstacles, a multiple step strategy is designed. Initially, literature review and bioinformatics analysis to screen a set of antigens that are heterogeneously expressed in GBM, which are designated as the target-bank, are leveraged. Then, according to the multiplex immunohistochemistry results of each patient's tumor sample, a personalized panel of antigens based on the principle that most cancer cells in tumor tissues can be covered from the target-bank is selected. To target these antigens, Vδ1 T cells are chosen as CAR vehicles because of its high tissue infiltration and off-the-shelf properties, and an optimized protocol for engineering CAR-Vδ1 T cells with high purity and cytotoxicity, low exhaustion, and cytokine release is developed. Next, the specific panel of cocktail CAR-Vδ1 T cells in the GBM organoids that are directly derived from the same patient's tumor is tested. The term "prof" cocktail therapy is coined to describe the approach using precise and rational combination of tumor antigens, organoid-based evaluation, and fitness of Vδ1 T cells. It may accelerate development of effective CAR-T drugs for heterogeneous solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d528650de6d54c949045b7ae03b97d20044ba8e" target='_blank'>
              Rational Design and Organoid-Based Evaluation of a Cocktail CAR-γδ T Cell Therapy for Heterogeneous Glioblastoma.
              </a>
            </td>
          <td>
            Guidong Zhu, Zhongzheng Sun, Yingchao Liu, Jiang Liu, Linpei Guo, Guojing Pei, Ying Jiang, Baowang Miao, Zhen Li, Ping Zhang, Dongqi Tang, Wen Zhang, Chengwei Wang
          </td>
          <td>2025-03-20</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Advancements in personalized neoantigen-based cancer vaccines are ushering in a new era in oncology, targeting unique genetic alterations within tumors to enhance treatment precision and efficacy. Neoantigens, specific to cancer cells and absent in normal tissues, are at the heart of these vaccines, promising to direct the immune system specifically against the tumor, thereby maximizing therapeutic efficacy while minimizing side effects. The identification of neoantigens through genomic and proteomic technologies is central to developing these vaccines, allowing for the precise mapping of a tumor’s mutational landscape. Despite advancements, accurately predicting which neoantigens will elicit strong immune responses remains challenging due to tumor variability and the complexity of immune system interactions. This necessitates further refinement of bioinformatics tools and predictive models. Moreover, the efficacy of these vaccines heavily depends on innovative delivery methods that enhance neoantigen presentation to the immune system. Techniques like encapsulating neoantigens in lipid nanoparticles and using viral vectors are critical for improving vaccine stability and delivery. Additionally, these vaccines contribute towards achieving Sustainable Development Goal 3.8, promoting universal health coverage by advancing access to safe and effective cancer treatments. This review delves into the potential of neoantigen-based vaccines to transform cancer treatment, examining both revolutionary advancements and the ongoing challenges they face.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1914e37039c714aaf0193ccf89aac93e878409a" target='_blank'>
              Advancements and challenges in personalized neoantigen-based cancer vaccines
              </a>
            </td>
          <td>
            Parminder Singh, Mahalaqua Nazil Khatib, Roopashree R, Mandeep Kaur, Manish Srivastava, Amit Barwal, G. V. S. Rajput, Pranchal Rajput, Rukshar Syed, Gajendra Sharma, Sunil Kumar, M. Shabil, Sakshi Pandey, Manvinder Brar, Ganesh Bushi, Rachana Mehta, S. Sah, K. Goh, Prakasini Satapathy, A. Gaidhane, S. Samal
          </td>
          <td>2025-03-14</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Recent oncological research has intensely focused on the tumor immune microenvironment (TME), particularly the functions of CD4 + T lymphocytes. CD4+ T lymphocytes have been implicated in antigen presentation, cytokine release, and cytotoxicity, suggesting their contribution to the dynamics of the TME. Furthermore, the application of single-cell sequencing has yielded profound insights into the phenotypic diversity and functional specificity of CD4+ T cells in the TME. In this review, we discuss the current findings from single-cell analyses, emphasizing the heterogeneity of CD4+ T cell subsets and their implications in tumor immunology. In addition, we review the critical signaling pathways and molecular networks underpinning CD4+ T cell activities, thereby offering novel perspectives on therapeutic targets and strategies for cancer treatment and prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06167-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Osteosarcoma (OS) is the most common malignant bone cancer. Thirty to forty percent of all patients with OS develop tumor recurrence, almost exclusively in the form of lung metastasis, which is associated with a dismal 20% 5-year survival rate. Cancer-associated fibroblasts are a critical cell type within the lung tumor microenvironment that promote immune suppression, drug resistance, and tumor cell survival. Prior work shows tumor cells can co-opt fibroblasts to a protumorigenic phenotype via exosome-mediated intercellular communication. Currently, the mechanisms by which OS exosomes modulate resident lung fibroblast (LF) function have not been evaluated. To investigate this, we isolated exosomes from a panel of six OS cell lines. We assessed the uptake and response of human donor–derived primary LFs (n = 4) to OS exosome treatment in vitro via flow cytometry, confocal fluorescent microscopy, proliferation assays, phosphokinase array, multiplex cytokine analysis, and RNA sequencing. We observed that LFs efficiently take up OS exosomes, which are associated with the induction of MAPK pathway activation, fibroblast proliferation, and significantly enhanced secretion of IL-6, CXCL8, and CCL2 compared with untreated LFs. RNA sequencing of exosome-treated LFs confirmed these responses and revealed significant enrichment of pathways related to cytokine secretion, proliferation, immune cell chemotaxis, migration, proinflammatory, and profibrotic mediators. Finally, in an exosome-educated LF–OS coculture model, exosome-educated LFs conferred significantly increased OS cell survival and proliferation compared with untreated fibroblasts. Together, these data suggest that the OS-derived exosomes can induce a hallmark cancer-associated fibroblast–like inflammatory phenotype in LFs, providing valuable insights into mechanisms that may promote recurrent OS lung metastasis. Significance: These findings provide a critical first step in characterizing the capacity of OS-derived exosomes to reprogram primary LFs toward a tumor-promoting inflammatory phenotype in vitro, offering novel molecular targets for the modulation of fibroblasts in the lung microenvironment as potential therapeutic strategies to prevent OS metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4796bcb4b0cbec4e85b9659ef31e17c57400c98" target='_blank'>
              Osteosarcoma Exosome Priming of Primary Human Lung Fibroblasts Induces an Immune Modulatory and Protumorigenic Phenotype
              </a>
            </td>
          <td>
            E. Palmer, Kathryn E Cronise, Laurel A Haines, Sunetra Das, Aaron Offermann, Carina P Easton, Daniel P Regan
          </td>
          <td>2025-03-18</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0786e178852bb9b6b2a0945adb3bddac988e519" target='_blank'>
              Abstract A030: A deeper understanding of tumor microenvironment modulation induced by IL13Ra2 CAR T cells in a patient with grade 3 ependymoma
              </a>
            </td>
          <td>
            Maryam Aftabizadeh, B. Aguilar, M. D’Apuzzo, B. Badie, Christine E. Brown
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8a88c38aa2fee4b0418baaa6a4ae840b640c29" target='_blank'>
              Emerging strategies of cell and gene therapy targeting tumor immune microenvironment.
              </a>
            </td>
          <td>
            Runtian Wang, Xiaorui Duan, Jian Li, Cheng Zhang, Lin Shen
          </td>
          <td>2025-04-10</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncolytic viruses (OVs) selectively replicate within tumors, directly killing cancer cells and promoting a systemic immune response by releasing tumor antigens. These features make OVs a promising approach in tumor immunotherapy, offering targeted treatment with fewer side effects. Despite these advantages, OVs are primarily administered via intratumoral injection, limiting their effectiveness for advanced, systemic cancers. Among OVs, oncolytic adenoviruses (oAdVs) are the most widely studied due to their well-understood gene regulation, safety, and stability. In this study, a modified oAdV vector, pDC316-oAd-SA, was engineered to express the SIRPα-mIgG1Fc gene, designed to remodel tumor-associated macrophages (TAMs) and enhance anti-tumor immunity. This vector, along with a control virus (Ad-ON), was evaluated both in vitro and in vivo. The modified oAd-SA significantly improved macrophage phagocytosis and showed superior tumor regression in murine models. Additionally, while both oAdVs increased T cell infiltration in the tumor microenvironment, oAd-SA specifically enhanced T cell immune function. The study also revealed that oAdVs modulate TAMs differently across tumor types, with oAd-SA therapy particularly increasing TAM phagocytosis and promoting an anti-tumor response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437049bcd965751b1b3c891343573c9b6d2d9307" target='_blank'>
              Expression of SIRPα-Fc by oncolytic virus enhances antitumor efficacy through tumor microenvironment reprogramming
              </a>
            </td>
          <td>
            Qingzhe Yang, Yongheng Shu, Yanwei Chen, Zhongbing Qi, Shichuan Hu, Yao Zhang, Yu Qin, Xianglin Xu, Jianchuan Hu, Anliang Huang, Pi-Yun Cheng
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Image‐based screening is an efficient drug discovery strategy. Combining high‐content screening (HCS) with artificial intelligence efficiently detects drug‐induced cellular phenotypes from a large pool of compounds. However, previous studies primarily focus on cell classification and cannot interpret models using morphological features. Additionally, the existing morphology‐based studies use only few features, which cannot accurately characterize cell phenotypic perturbations. Herein, π‐PhenoDrug, a deep learning‐based pipeline, is developed for cell phenotype‐driven drug screening. It integrates cell segmentation, morphological profile construction, and phenotype analysis modules of HCS processes. π‐PhenoDrug is applied to evaluate drug response in various human melanoma cell lines. The results demonstrate that π‐PhenoDrug can evaluate drug killing effects across different cell lines via both supervised and unsupervised modes. Furthermore, π‐PhenoDrug identifies drugs with potential killing effects on melanoma cells from a library of diverse compounds. Compared with traditional single‐readout assays, this method is more sensitive to compounds that induce weaker phenotypes, reducing the risk of overlooking effective drugs. These results confirm that π‐PhenoDrug can achieve high‐throughput and accuracy analysis of cell phenotypic data using an unbiased and automated workflow, improving drug discovery efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59d452770f50a368d33283eccf620d3785158da" target='_blank'>
              π‐PhenoDrug: A Comprehensive Deep Learning‐Based Pipeline for Phenotypic Drug Screening in High‐Content Analysis
              </a>
            </td>
          <td>
            Xiao Li, Qinxue Ouyang, Mingfei Han, Xiaoqing Liu, Fuchu He, Yunping Zhu, Ling Leng, Jie Ma
          </td>
          <td>2025-03-02</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Melanoma progression is influenced by complex interactions between tumor cells and the immune microenvironment. This study examined the relationship between microphthalmia-associated transcription factor (MITF) expression and the immune microenvironment in primary melanoma using a modified classification of tumor-infiltrating lymphocytes (TILs) based on conventional BRISK categories. Archival formalin-fixed, paraffin-embedded tissue samples from 81 primary melanoma patients were analyzed via tissue microarray immunohistochemistry to assess MITF protein levels. TIL patterns were categorized into six groups, refining the traditional BRISK classification to distinguish between continuous and discontinuous infiltration, as well as peripheral vs intratumoral distribution. The analysis revealed that melanomas classified under the BRISK B category exhibited the highest MITF expression, often exceeding 50%. In contrast, tumors in the NON-BRISK and ABSENT TIL groups showed significantly lower MITF expression (mean values: 32.73% ± 16.98% and 22.00% ± 10.54%, respectively), with statistically significant differences (Kruskal-Wallis test, P = 0.027; modified classification, P = 0.011). Additionally, the presence of CD20+ B lymphocytes correlated with increased MITF expression (P = 0.009). MITF gene amplification was detected in 29% of cases, though its association with protein expression showed only a trend (P = 0.058). These findings highlight the complex interplay between MITF expression and TIL distribution in melanoma, suggesting that refined TIL classification may offer deeper insights into tumor immunobiology and help predict responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3930e36304cc8998ef22ab7f812e87d874a8d5b0" target='_blank'>
              The impact of MITF expression on tumor-infiltrating lymphocytes in melanoma: Insights into immune microenvironment dynamics.
              </a>
            </td>
          <td>
            D. Vučinić, Matea Lekić, G. Žauhar, G. Zamolo
          </td>
          <td>2025-02-18</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, Scott Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0ef45b0d0460ea2916e782f771c000702057b" target='_blank'>
              1347 Using Living Human Glioblastoma Samples to Develop Tumor-Targeted AAV Gene Therapy Vectors
              </a>
            </td>
          <td>
            M. Guevara, JP McGinnis, Joshua Ortiz-Guzman, Sai Mallannagari, Daniel Curry, Benjamin R. Arenkiel
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Cytotoxic immune cell populations, including CD8+ T cells, play a critical role in anti-tumor immunity, suppressing tumor progression. For this reason, recent therapeutic algorithms include immunotherapeutic agents that target such cell populations for the treatment of patients with a variety of solid tumors. In contrast to solid tumors, such immunotherapeutic strategies failed to show effectiveness in the treatment of patients with myeloid malignancies, including myelodysplastic neoplasms (MDS) and acute myeloid leukemia (AML). For this reason, the study of immunophenotypic and molecular changes in cytotoxic effector cell populations in patients with such disorders would provide the necessary evidence for the development of novel immunotherapies. In this review, we provide an overview of experimental studies describing the modulation of CD8+ T cell populations in patients with myeloid neoplasms, focusing on whether and how specific cell phenotypes are linked to disease progression and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7180f81c8d25196d5cfda7afac4f9f30d1e351bb" target='_blank'>
              The Emerging Role of CD8+ T Cells in Shaping Treatment Outcomes of Patients with MDS and AML
              </a>
            </td>
          <td>
            Athanasios Tasis, Theodoros Spyropoulos, I. Mitroulis
          </td>
          <td>2025-02-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor immune checkpoint therapy (ICT) aims to block immune escape signals between tumor and immune cells. However, low delivery efficiency of immune checkpoint inhibitors (ICIs), narrow single-target approach, and reduced responsiveness notably hinder clinical development of ICT. Here, we developed a nanoliposome-bacteria hybrid system that acts as an antibody (Ab) factory, enabling precise tumor targeting and macrophage activation in hypoxic environments. We reprogrammed attenuated Escherichia coli MG1655 to synthesize CD47 antibodies (aCD47) in response to hypoxic tumor microenvironments while surface conjugating with redox-responsive macrophage colony-stimulating factor-loaded liposomes. This system leverages bacterial tropism to enhance macrophage infiltration and polarization. The low oxygen levels trigger in situ aCD47 expression, blocking the “do not eat me” signal and boosting macrophage antitumor activity. In addition, macrophage antigen presentation activates CD8+CD3+ T cells, amplifying systemic antitumor immunity. Analysis of the gut microbiome shows reduced pathogenicity and improved intestinal tolerance with increased probiotics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aa23d0e3aba41573964b5f29b76e7bf637ae15" target='_blank'>
              Programmable engineered bacteria as sustained-releasing antibody factory in situ for enhancing tumor immune checkpoint therapy
              </a>
            </td>
          <td>
            Xiao-Ting Xie, Meng Guan, Kai Cheng, Yong Li, Bin Zhang, Yi-Tong Zhou, Lin-Fang Tan, Peng-Shuo Dong, Si Chen, Bo Liu, Yuan-Di Zhao, Jin-Xuan Fan
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f8d25128e68026c13f6a41f5701ec7d66c8485" target='_blank'>
              Cancer Cell and Cancer-Associated Fibroblast Communication-Mediated Molecular Subtypes Portray Non-Inflamed Tumor Microenvironment and Guide the Precision Treatment of Bladder Cancer.
              </a>
            </td>
          <td>
            Shenglin Gao, Chuan Liu, Lixin Mao, Yin Chen, Xiaokai Shi, C. Yue, Shouchun Li, X. Qin
          </td>
          <td>2025-02-17</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Macrophages play crucial roles in immune responses and tissue homeostasis. Despite the fact that macrophages were described more than a century ago, they continue to be the cells of intensive interest. Advanced understanding of phenotypic diversity in macrophages holds great promise for development of cell-based therapeutic strategies. The introduction of innovative approaches in cell biology greatly enhances our ability to investigate the unique characteristics of macrophages. The review considers both classical methods to study macrophages and high-tech approaches, including single-cell sequencing, single-cell mass spectrometry, droplet microfluidics, scanning probe microscopy and atomic force spectroscopy. This review will be valuable both to specialists beginning their study of macrophages and to experienced scientists seeking to deepen their understanding of methods at the intersection of biological and physical sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93e451bba104f70c02f78ff9a726f6c564ee32e" target='_blank'>
              Unravelling approaches to study macrophages: from classical to novel biophysical methodologies
              </a>
            </td>
          <td>
            P. Vishnyakova, A. Elchaninov, T. Fatkhudinov, Dmitry Kolesov
          </td>
          <td>2025-02-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cervical cancer (CC) is a major global health issue, ranking sixth in cancer-related mortality. The tumor microenvironment (TME) plays a crucial role in tumor growth. This study explored the cellular composition and immunological landscape of CC using various genomic data sources.Data from the Cancer Genome Atlas and Gene Expression Omnibus were analyzed, including single-cell RNA sequencing, spatial transcriptome analysis, and survival data. Gene set variation analysis (GSVA) identified pathways in CD8+ cells, macrophages, and epithelial cells. Immunohistochemistry assessed marker expression in CC and normal tissues. Tumor immune dysfunction and exclusion (TIDE) scores differentiated high- and low-macrophage groups. Cell–cell communication analyses highlighted interactions between macrophages and epithelial cells.Macrophage markers correlated with overall survival (OS) and disease-free survival (DFS). Epithelial cell subgroups 1 and 4, along with CD8+ T cells, were associated with OS. TIDE scores varied between groups. Specific ligand-receptor interactions were found between macrophages and epithelial cell subgroup 1. Triptolide was effective in epithelial cell subgroup 1, while memantine was more effective in macrophages.Epithelial-macrophage interactions in the TME are crucial for CC progression and treatment, offering a potential immunotherapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6446bb065e28f5d7b90ac2e8452e69b230ac374" target='_blank'>
              Epithelial and macrophage cell interaction in cervical cancer through single-cell RNA-sequencing and spatial analysis
              </a>
            </td>
          <td>
            Zhichao Wang, Long Cheng, Guanghui Li, Huiyan Cheng
          </td>
          <td>2025-04-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fffa6682c8cf8e3d73d30e552a59d8713e1c83f9" target='_blank'>
              Abstract A073: Digital assessment of tertiary lymphoid structures in cancer using whole slide scans
              </a>
            </td>
          <td>
            S. M. Park, Luis Munoz-Erazo, Shelly Lin, J. C. Chun-Jen, Lisa Y.Y. Zhou, J. Rhodes, Taesung Jeon, Sonya Fenton, John L McCall, Roslyn A. Kemp, P. R. Dunbar
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ea7bcdd3ec0f8b344e09113c3c14bed0419bdf" target='_blank'>
              Construction of a Single-cell Atlas of Thyroid Cancer.
              </a>
            </td>
          <td>
            Kaiyu Song, Yaqi Wang, Yuantao Wang, Jiahui Liu, Wenjie Yao, Yongli Chu, Yun Qu, Xicheng Song, Jin Zhou
          </td>
          <td>2025-03-10</td>
          <td>Endocrine, metabolic & immune disorders drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tissue-resident macrophages (TRMs) are attractive cells to therapeutically deliver oligonucleotide and other gene-expression modifying modalities to treat a wide array of diseases ranging from inflammatory to autoimmune, and even cancer. Here, we focus on TRMs located inside the peritoneal cavity lining the abdomen that selectively express a transcription factor GATA6 called large peritoneal macrophages (GLPMs) and successfully demonstrate functional GLPM-selective delivery of a Cy5-fluorophore-labeled siRNA encapsulated in C12–200 cationic-lipidoid-based nanoparticles (siRNA-Cy5 (C12–200)). Despite being TRMs, GLPMs possess a specific migratory ability to peritoneally located liver tissue upon injury incited by acetaminophen (APAP) overdose in mice. A rapid, liver injury-driven tropism of GLPMs carrying siRNA-Cy5 (C12–200) was seen via systemic circulation, which was elegantly demonstrated by using a noninvasive live-cell tracking technique called diffuse in vivo flow cytometry (DiFC). Finally, RNAi-mediated silencing of a well-known pro-inflammatory damage-associated molecular pattern (DAMP) High Mobility Group Box-1 (HMGB1) gene in GLPMs led to the mitigation of liver injury and inflammation via prevention of GLPM modulation to a pro-inflammatory state, which further translated into significant protection from APAP-driven liver injury and a reduction in liver circulating pro-inflammatory cytokines owing to a muted inflammatory response to acute liver injury. Moreover, silencing HMGB1 by a GalNAc-conjugated hepatocyte-targeting siRNA did not reciprocate the findings, further solidifying our results. Together, our data suggested that GLPMs act as delivery carriers by rapidly bringing lipid nanoparticle-encapsulated RNAi modalities to the injured liver and have emerged as a therapeutically viable strategy to address inflammatory diseases, especially those that are more acute in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75cc4fed86dbcf875969e2b126964ab1547a69d6" target='_blank'>
              Treatment of Acute Liver Injury through Selective Tropism of High Mobility Group Box 1 Gene-Silenced Large Peritoneal Macrophages
              </a>
            </td>
          <td>
            Dhaval Oza, Fernando Ivich, Kirsten Deprey, Kelsey Bittner, Keith Bailey, Sarah Goldman, Mikyung Yu, M. Niedre, Ho-Chou Tu, Mansoor M. Amiji
          </td>
          <td>2025-03-18</td>
          <td>ACS Nano</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29965c5d14b51c6573686cfc61d8b621a40178b6" target='_blank'>
              Abstract IA019: Addressing genomic heterogeneity in glioblastoma personalized vaccine design
              </a>
            </td>
          <td>
            Elaine R Mardis, E. Garfinkle, T. Johanns, Katherine E Miller, Gavin P Dunn
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1719d073dfd28cb966f7f3079174856e098d4f7f" target='_blank'>
              Spatial transcriptomics of progression gene signature and tumor microenvironment leading to progression in mycosis fungoides.
              </a>
            </td>
          <td>
            M. Choi, Gyeonghoon Kim, Hwa Jeong Shin, C. Won, Sung Eun Chang, M. Lee, W. Lee
          </td>
          <td>2025-03-31</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf77d83d4281bc62f48275909f7e2fd555ec0bfd" target='_blank'>
              Abstract B059: Mechanisms of Sustainable T-cell Recruitment in a Perfusion-Enabled 3D Interstitial Network
              </a>
            </td>
          <td>
            Duy T. Nguyen, Laxmi Swetha Karanam, Said J Cifuentes Maury, Zoe Zhou
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Intracellular cytokine labeling combined with high‐parametric flow cytometry offers substantial promise in elucidating the nuanced effector functions of cells. However, the establishment of complex multicolor panels is often laborious and the importance of validation processes may be underestimated in research practice. This raises the risk of prematurely translating multicolor panels into in vivo studies. Alternatively, researchers may resort to animal disease models to procure cytokine‐producing cells. Both scenarios raise ethical concerns as they entail the potential for unnecessary animal suffering without yielding novel insights into immunobiology. Here, we perform multicolor panel optimization and validation without the need for stressful animal testing. We designed two spectral flow cytometry panels for cytokine expression analyses across mouse immune and joint cells. Animal testing was replaced by stimulated co‐cultures of T cells, splenocytes, and fibroblast‐like synoviocytes. These cultures were used for multicolor labeling experiments. Our method proved suitable for validating the two cytometry panels, as it provided a complex cellular environment in which a variety of cytokine‐producing populations were identified. In summary, we here present a blueprint for the quality control of single‐cell cytokine assays by cell culture and further introduce multicolor panels that can be employed for studies on inflammatory or infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/048728f487ba63837087f6841943c93b15e7cb5d" target='_blank'>
              Maximizing Insights, Minimizing Animal Testing: A Framework for Validating Multiparametric Single‐Cell Cytokine Analysis Panels
              </a>
            </td>
          <td>
            Johann Aleith, Wendy Bergmann-Ewert, Brigitte Müller-Hilke
          </td>
          <td>2025-03-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ff55c83dc7ac6bb2d4f9aa3c3bfde8b88e10ca" target='_blank'>
              Abstract B019: Mapping the tumor-sentinel node immune migratome demonstrates a key role for CCR7+ dendritic cells in the successful response to immunoradiotherapy
              </a>
            </td>
          <td>
            R. Saddawi-Konefka, Riyam Al Msari, Shiqi Tang, Lauren M. Clubb, Santiago Fassardi, Riley Jones, F. Faraji, Shiruyeh Schokrpur, Bryan S. Yung, M. Allevato, S. Jensen, B. A. Fox, R. Bell, J. S. Gutkind, Andrew Sharabi, Joseph A Califano
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fc0173898db1615005d0ab09ce022fdfc322eb" target='_blank'>
              The Transcriptomic Landscapes of Appendiceal Primary and Metastatic Tumors are Distinct.
              </a>
            </td>
          <td>
            Margaret A. Park, Richard Jacobson, Maria Genilo-Delgado, Amir Mohammadi, Carlos M. Morán-Segura, Solomon Alhassan, Yukihiro Nakanishi, J. Permuth, I. Imanirad, S. Dineen
          </td>
          <td>2025-02-22</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Over the past few decades, immunotherapy has emerged as a powerful strategy to overcome the limitations of conventional cancer treatments. The use of extracellular vesicles, particularly exosomes, which carry cargoes capable of modulating the immune response, has been extensively explored as a potential therapeutic approach in cancer immunotherapy. Exosomes can deliver their cargo to target cells, thereby influencing their phenotype and immunomodulatory functions. They exhibit either immunosuppressive or immune‐activating characteristics, depending on their internal contents. These exosomes originate from diverse cell sources, and their internal contents can vary, suggesting that there may be a delicate balance between immune suppression and stimulation when utilizing them for immunotherapy. Therefore, a thorough understanding of the molecular mechanisms underlying the role of exosomes in cancer progression is essential. This review focuses on the molecular mechanisms driving exosome function and their impact on the tumor microenvironment (TME), highlighting the intricate balance between immune suppression and activation that must be navigated in exosome‐based therapies. Additionally, it underscores the challenges and ongoing efforts to optimize exosome‐based immunotherapies, thereby making a significant contribution to the advancement of cancer immunotherapy research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21f3467f9a19197b3601c9c67c5b67a4de08857" target='_blank'>
              Exosomes: a double‐edged sword in cancer immunotherapy
              </a>
            </td>
          <td>
            Jiayi Chen, Siyuan Hu, Jiayi Liu, Hao Jiang, Simiao Wang, Zhaogang Yang
          </td>
          <td>2025-02-17</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0cea3704c681dc00c1f21af6441282cc9310e" target='_blank'>
              Diversity and heterogeneity in human pancreaticobiliary maljunction revealed by single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Hui-min Mao, Wan-liang Guo, San-li Shi
          </td>
          <td>2025-03-21</td>
          <td>Pediatric surgery international</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths, of which adenocarcinoma is the most common subtype. Despite this, lung adenocarcinoma and its metastasis are poorly understood, due to difficulties in feasibly recapitulating disease progression and predicting clinical benefits of therapy. We outline a methodology to develop immunogenic orthotopic lung adenocarcinoma mouse models, by injecting cell-specific cre viruses into the lung of a genetically engineered mouse, which mirrors cancer progression defined by the International Association for the Study of Lung Cancer. Evaluation of different cre virus/concentrations models demonstrate remarkable consistency in cancer initiation and metastasis, allowing for high throughput, while showing differences in timing and severity, offering greater flexibility when selecting models. Histological and immune profiles reflect clinical observations suggesting similar mechanisms are recapitulated and preliminary data show resultant tumors to be responsive to clinical treatments. We present a clinically relevant, next-generation murine model for studying lung adenocarcinoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90344-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81dd756da5910df487dda469964a4404cb5624ff" target='_blank'>
              Methodology of murine lung cancer mimics clinical lung adenocarcinoma progression and metastasis
              </a>
            </td>
          <td>
            Edison Q Kim, Emily Y. Kim, Eric P Knott, Yujie Wang, Cheng-Bang Chen, Jose R. Conejo-Garcia, M. Wangpaichitr, Diane C Lim
          </td>
          <td>2025-02-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>