<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-30 09:25:39 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The tumor microenvironment (TME) is a complex network of interactions between malignant and host cells, yet its orchestration in advanced high-grade serous ovarian carcinoma (HGSC) remains poorly understood. We present a comprehensive single-cell spatial atlas of 280 metastatic HGSCs, integrating high-dimensional imaging, genomics, and transcriptomics. Using 929 single-cell maps, we identify distinct spatial domains associated with phenotypically heterogeneous cellular compositions, and demonstrate that immune cell co-infiltration at the tumor-stroma interface significantly influences clinical outcomes. To uncover the key drivers of the tumor ecosystem, we developed CEFIIRA (Cell Feature Importance Identification by RAndom forest), which identified tumor cell-intrinsic MHC-II expression as a critical predictor of prolonged survival, independent of clinicomolecular profiles. Validation with external datasets confirmed that MHC-II-expressing cancer cells drive immune infiltration and orchestrate spatial tumor-immune interactions. Our atlas offers novel insights into immune surveillance mechanisms across HGSC clinicomolecular groups, paving the way for improved therapeutic strategies and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/684d76f91d1a8b1bc3e208feca47d927bec6a2ec" target='_blank'>
              Single-cell spatial atlas of high-grade serous ovarian cancer unveils MHC class II as a key driver of spatial tumor ecosystems and clinical outcomes
              </a>
            </td>
          <td>
            Fernando Pérez-Villatoro, Lilian van Wagensveld, Aleksandra Shabanova, Ada Junquera, Ziqi Kang, Iga Niemiec, Matias M. Falco, Ella Anttila, J. Casado, Eric Marcus, Essi Kahelin, Foteini Chamchougia, M. Salko, Saundarya Shah, Salvatore Russo, Jacopo Chiaro, Mikaela Grönholm, G. Sonke, K. K. van de Vijver, Rutgerus Fpm Kruitwagen, Maaike Avan der Aa, A. Virtanen, Vincenzo Cerullo, Anna Vähärautio, P. Sorger, Hugo M. Horlings, A. Färkkilä
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Neuroendocrine and tuft cells are rare, chemosensory epithelial lineages defined by expression of ASCL1 and POU2F3 transcription factors, respectively1,2. Neuroendocrine cancers, including small cell lung cancer (SCLC), frequently display tuft-like subsets, a feature linked to poor patient outcomes3–13. The mechanisms driving neuroendocrine–tuft tumour heterogeneity, and the origins of tuft-like cancers are unknown. Using multiple genetically-engineered animal models of SCLC, we demonstrate that a basal cell of origin (but not the accepted neuroendocrine origin) generates neuroendocrine–tuft-like tumours that highly recapitulate human SCLC. Single-cell clonal analyses of basal-derived SCLC further uncovers unexpected transcriptional states and lineage trajectories underlying neuroendocrine–tuft plasticity. Uniquely in basal cells, introduction of genetic alterations enriched in human tuft-like SCLC, including high MYC, PTEN loss, and ASCL1 suppression, cooperate to promote tuft-like tumours. Transcriptomics of 944 human SCLCs reveal a basal-like subset and a tuft-ionocyte-like state that altogether demonstrate remarkable conservation between cancer states and normal basal cell injury response mechanisms14–18. Together, these data suggest that the basal cell is a plausible origin for SCLC and other neuroendocrine-tuft cancers that can explain neuroendocrine–tuft heterogeneity—offering new insights for targeting lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a9f4695f47fd01e98ecf709150d28f3a1f7f97" target='_blank'>
              Basal cell of origin resolves neuroendocrine–tuft lineage plasticity in cancer
              </a>
            </td>
          <td>
            A. Ireland, Sarah B. Hawgood, Daniel A. Xie, Margaret W. Barbier, Scarlett Lucas-Randolph, Darren R. Tyson, Lisa Y. Zuo, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Sonam Puri, Charles M. Rudin, J. Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="We conducted a spatial analysis using imaging mass cytometry applied to stage III colorectal adenocarcinomas. This study used multiplexed markers to distinguish individual cells and their spatial organization from 52 colorectal cancers. We determined the landscape features of cellular spatial features in the CRC tumor microenvironment. This spatial single-cell analysis identified 10 unique cell phenotypes in the tumor microenvironment that included stromal and immune cells with a subset which had a proliferative phenotype. These special features included spatial neighborhood interactions between single cells as well as different tissue niches, especially the tumor infiltrating lymphocyte regions. We applied a robust statistical analysis to identify significant correlations of cell features with phenotypes such as microsatellite instability or recurrence. We determined that microsatellite stable (MSS) colorectal cancers had an increased risk of recurrence if they had the following features: 1) a low level of stromal tumor-infiltrating lymphocytes, and 2) low interactions between CD4+ T cells and stromal cells. Our results point to the utility of spatial single-cell interaction analysis in defining novel features of the tumor immune microenvironments and providing useful clinical cell-related spatial biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc76d98a517403a79b6bbdaa67daa9d1d6c7537" target='_blank'>
              The single-cell spatial landscape of stage III colorectal cancers
              </a>
            </td>
          <td>
            Andrew Su, Hojoon Lee, Minh Tran, Richard Cruz, A. Sathe, Xiangqi Bai, Ignacio A. Wichmann, Lance Pflieger, Bryce Moulton, Tyler Barker, D. Haslem, David Jones, Lincoln Nadauld, Quan H Nguyen, Hanlee P. Ji, Terence Rhodes
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In bladder cancer (BLCA), the cellular expression of lineage markers with predictive biomarker function can have important implications for tumor progression, treatment response, and survival. However, tumor heterogeneity and the coexistence of distinct tumor subpopulations can complicate the utility of using a primary, clinically assigned tumor signature. In this report, we have applied the reference atlas, Tabula Sapiens, and deep learning model, UniCell, to conduct in-depth transcriptional analysis of unique lineage marker-defined cell clusters in a carcinogen (BBN) induced bladder cancer model. UniCell deconvolution has identified tumor populations, including urothelial, adenocarcinoma, squamous cell carcinoma, and mesenchymal tumor populations, each with cell-intrinsic gene expression signatures relevant to human BLCA progression. The identified tumor clusters contain uniquely basal, luminal, stromal, or hybrid cells in cell lineage marker expression. To understand the significance of these populations during progression, we used trajectory and pseudo-time analysis to show that cells uniquely basal are plastic in lineage identity and can evolve to form tumor populations composed of other lineage marker-defined signatures. Finally, pathway and drug enrichment analysis of tumor cell clusters were used to identify therapeutics that may preferentially target the identified tumor cell populations. These data collectively define a molecular template that may uniquely profile molecular plasticity occurring during progression and response to therapy with important implications for human disease. Significance Tumor heterogeneity is a mechanism for treatment resistance. Our study defines unique tumor subpopulations having differential therapeutic sensitivities and potential for lineage plasticity. Our modeling may impact the treatment of BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e51b8c18f48b163ace1388c13e342c3f5124f692" target='_blank'>
              Mapping of Multilineage Tumor Cell Populations in Mouse Bladder Cancer
              </a>
            </td>
          <td>
            Nadine Schrode, Yang Hu, Haocheng Yu, M. Galsky, J. Sfakianos, R. Sebra, Kristin G Beaumont, David J Mulholland
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has advanced our understanding of tissue regionalization by enabling the visualization of gene expression within whole tissue sections1, but the approach remains dogged by the challenge of achieving single-cell resolution without sacrificing whole genome coverage2,3. Here we present Spotiphy (Spot imager with pseudo single-cell resolution histology), a novel computational toolkit that transforms sequencing-based ST data into single-cell-resolved whole-transcriptome images. In evaluations with Alzheimer’s disease (AD) and normal mouse brains, Spotiphy delivers the most precise cellular compositions. For the first time, Spotiphy reveals novel astrocyte regional specification in mouse brains. It distinguishes sub-populations of DAM (Disease-Associated Microglia) located in different AD mouse brain regions. Spotiphy also identifies multiple spatial domains as well as changes in the patterns of tumor-tumor microenvironment interactions using human breast ST data. Spotiphy enables visualization of cell localization and gene expression in tissue sections, offering key insights into the function of complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20d2df6e0d020ef03f0ffa729e29890d99f779b" target='_blank'>
              Spotiphy enables single-cell spatial whole transcriptomics across the entire section
              </a>
            </td>
          <td>
            Jiyuan Yang, Ziqian Zheng, Yun Jiao, Kaiwen Yu, Sheetal Bhatara, Xu Yang, Sivaraman Natarajan, Jiahui Zhang, John Easton, Koon-Kiu Yan, Junmin Peng, Kaibo Liu, Jiyang Yu
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent studies have identified a unique subtype of cancer-associated fibroblasts (CAFs) termed antigen-presenting CAFs (apCAFs), which remain the least understood CAF subtype. To gain a comprehensive understanding of the origin and function apCAFs, we construct a fibroblast molecular atlas across 14 types of solid tumors. Our integration study unexpectedly reveals two distinct apCAF lineages present in most cancer types: one associated with mesothelial-like cells and the other with fibrocytes. Using a high-resolution single-cell spatial imaging platform, we characterize the spatial niches of these apCAF lineages. We find that mesothelial-like apCAFs are located near cancer cells, while fibrocyte-like apCAFs are associated with tertiary lymphoid structures. Additionally, we discover that both apCAF lineages can up-regulate the secreted protein SPP1, which facilitates primary tumor formation and peritoneal metastasis. Taken together, this study offers an unprecedented resolution in analyzing apCAF lineages and their spatial niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fa0ff2ff7820fdabd0fa0f4d3421c72dfbfc1" target='_blank'>
              Single-cell resolution spatial analysis of antigen-presenting cancer-associated fibroblast niches
              </a>
            </td>
          <td>
            Xiongfeng Chen, Zhuan Zhou, Zeynep Yazgan, Luyu Xie, Francesca Rossi, Yang Liu, Bo Zhang, Patricio M. Polanco, Herbert J. Zeh, Alex C. Kim, Huocong Huang
          </td>
          <td>2024-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3b0a8bb89725a847aae6e2af230a0666ba9d6" target='_blank'>
              Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network
              </a>
            </td>
          <td>
            Debojyoti Chowdhury, Shreyansh Priyadarshi, Sayan Biswas, Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Pancreatic ductal adenocarcinoma (PDAC) represents the complexity of interaction between cancer and cells of the tumor microenvironment (TME). Immune cells affect tumor cell behavior, thus driving cancer progression. Cancer-associated fibroblasts (CAFs) are responsible of the desmoplastic and fibrotic reaction by regulating deposition and remodeling of extracellular matrix (ECM). As tumor-promoting cells abundant in PDAC ECM, CAFs represent promising targets for novel anticancer interventions. However, relevant clinical trials are hampered by the lack of specific markers and elusive differences among CAF subtypes. Indeed, while single-cell transcriptomic analyses have provided important information on the cellular constituents of PDACs and related molecular pathways, studies based on the identification of protein markers in tissues aimed at identifying CAF subtypes and new molecular targets result incomplete. Methods Herein, we applied multiplexed Imaging Mass Cytometry (IMC) at single-cell resolution on 8 human PDAC tissues to depict the PDAC composing cells, and profiling immune cells, endothelial cells (ECs), as well as endocrine cells and tumor cells. Results We focused on CAFs by characterizing up to 19 clusters distinguished by phenotype, spatiality, and interaction with immune and tumor cells. We report evidence that specific subtypes of CAFs (CAFs 10 and 11) predominantly are enriched at the tumor-stroma interface and closely associated with tumor cells. CAFs expressing different combinations of FAP, podoplanin and cadherin-11, were associated with a higher level of CA19-9. Moreover, we identified specific subsets of FAP+ and podoplanin+/cadherin-11+ CAFs enriched in patients with negative prognosis. Discussion The present study provides new general insights into the complexity of the PDAC microenvironment by defining phenotypic heterogeneities and spatial distributions of CAFs, thus suggesting different functions of their subtypes in the PDAC microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c403b727bc52805f8503133b3a285d436787caa0" target='_blank'>
              Depicting the cellular complexity of pancreatic adenocarcinoma by Imaging Mass Cytometry: focus on cancer-associated fibroblasts
              </a>
            </td>
          <td>
            M. Erreni, M. R. Fumagalli, Raffaella D’Anna, M. Sollai, S. Bozzarelli, G. Nappo, Damiano Zanini, Raffaella Parente, Cecilia Garlanda, L. Rimassa, L. Terracciano, S. Biswas, A. Zerbi, Alberto Mantovani, Andrea Doni
          </td>
          <td>2024-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Spatial organization of normal tissues is key for their function, yet the role of distribution of distinct clones of cancer cells in tumor is not usually studied as a potential vulnerability in cancer. This is of particular interest in glioblastoma (GBM), in which extensive intratumor heterogeneity hampers effectiveness of treatment. To better understand tumor evolution and the complexity of cancer ecosystem in GBM, we performed profiling of biopsies collected from MRI-defined regions of the tumors, spanning the surface, tumor core and deep tumor margins, which drive local recurrence after surgery. To test how spatially distinct biopsies differ in their cancer cell composition and to assess the tumor microenvironment heterogeneity, we took advantage of 5-ALA metabolite fluorescence, which accumulates in GBM cells and allows the surgeon to visualize cancer under blue light. We used it to separate cancer cells (predominantly 5-ALAhigh) from tumor microenvironment (5-ALAlow) and subjected the fractions to bulk and single cell transcriptomics, respectively. We found a gradient of cellular states associated with location in the tumor, with the deep margin samples having more 5-ALAlow cancer cells and generally less defined phenotype. We have also developed neurosphere cultures from our collection of spatially distant biopsies. Despite the selection pressure of the in vitro culture conditions, cells from different regions of the tumor adopted different morphologies, suggesting retention of some of their differential features. The differences between cultures were reflected in metabolism, proliferation, and differential sensitivity to a drug panel. Whole exome sequencing and transcriptomic analysis were also performed to identify the selection bottleneck of culture conditions, establish evolutionary trajectory of the distinct regions and neurosphere cultures, and test how strongly our cultures represent both the 5-ALAhigh and 5-ALAlow subpopulations of GBM cells. Our study identified properties associated with different spatial locations within GBM tumors, which will help uncover the cellular and microenvironmental dependencies that could lead to novel therapeutic targets for this highly heterogeneous tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7bfbfe4e2db05f257edb989e202cc92ab018e37" target='_blank'>
              TMOD-22. MULTIREGION BIOPSIES AND CORRESPONDING NEUROSPHERE CULTURES REVEAL SPATIAL DIVERGENCE IN GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>
            Roberto Salatino, Jacob Geisberg, U. Onubogu, Arantxa Romero-Toledo, Banjamin Oakes, O. Szentirmai, Thomas McDonald, Franziska Michor, M. Janiszewska
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In rectal cancer, where part of the patients undergoes chemoradiotherapy, there is a need for improved pretreatment biomarkers applicable to biopsies. Tumor budding (TB) is a biomarker used in colon cancer, and due to its link to epithelial-mesenchymal transition (EMT), is hypothesized to be a potential marker for therapy resistance. Assessment of the utility of tumor buds in rectal biopsies is challenging due to their rarity. As EMT-related processes are also seen in other morphological features beyond tumor buds, we investigated EMT in tumor tissue including morphological features such as tumor cluster size and fibril-like structures. To do so, we leveraged a cohort of colon cancer whole-slide images and another cohort consisting of rectal cancer biopsies, visualized using hyperplex immunofluorescence to identify tumor and EMT-associated proteins. We built a custom image analysis pipeline to detect and segment tumor buds and other morphological features and correlated them with molecular expression intensities. We found strong correlations of EMT up-regulation and morphological transition states, both at the invasive margin and the tumor center. We furthermore observed a link between morpho-molecular transitions and histological growth patterns, which in turn can inform novel biomarkers. Finally, quantification of these morpho-molecular transition states in rectal biopsies showed their impact on survival after neoadjuvant chemoradiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f452e9eb79df45de63d0362c848378c46029149c" target='_blank'>
              Morpho-molecular features of Epithelial Mesenchymal Transition associate with clinical outcome in patients with rectal cancer
              </a>
            </td>
          <td>
            Mauro Gwerder, Cansaran Saygili Demir, Hannah L Williams, A. Lugli, Cristina Graham Martinez, Joanna Kowal, Amjad Khan, Philipp Kirchner, Thibaud Koessler, Martin D. Berger, Martin Weigert, I. Zlobec
          </td>
          <td>2024-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="

 Introduction
 Angioimmunoblastic T-cell lymphoma (AITL) is a T-cell lymphoma characterized by T follicular helper cell (TFH) phenotype. It comes with a strong inflammatory response and highly complex tumor microenvironment, where tumor cells are supported by a scaffold of highly proliferative CD21+ follicular dendritic cells (FDCs) in proximity to high endothelial venules (HEVs). The structured organization of the lymph node, with its distinct B-cell and T-cell zones, is disrupted in lymph node samples from patients with AITL. Based on prior literature in this space, background “benign” inflammatory bystanders and abnormal non-neoplastic vasculature and stromal cells appear to enable tumor cells to escape immune surveillance and unchecked proliferation. We hypothesize that genomic spatial characterization of the tumor microenvironment can lead to improved targeting of these tumor enablers, and for the first time, AITL has been profiled by single cell spatial ATAC-seq, paired with multiplexed immunofluorescence and DBiT-seq for this purpose.
 Methods
 The sample cohort consists of 23 de-identified, archived formalin-fixed paraffin-embedded (FFPE) human lymphoma tissue blocks and 7 OCT-frozen blocks, 6 with paired FFPE. The quality of RNA in each tissue block was assessed via an RNA Integrity Number (RIN) test and blocks with DV200 > 50% (minimal RNA degradation levels) were the ones included in our cohort. For each sample, we performed highly multiplexed immunofluorescence imaging (CODEX) using a custom immune cell panel of 40 markers on the PhenoCycler-Fusion system (Akoya Biosciences). The panel includes cell-type specific markers such as Podoplanin (lymphatic vasculature), CD20, CD3e, as well as all TFH markers. Cell segmentation was performed using a StarDist-based model in QuPath and downstream analysis done using the Seurat package. Serial section was then used for spatial transcriptomics using DBiT-seq to characterize the whole transcriptional landscape of the lymphoma. For the OCT-frozen sections, spatial ATAC-seq was performed on one slide to explore the epigenetic landscape of AITL.
 Results
 Cell type annotation of the CODEX data allowed us to identify the cell types within the tumor microenvironment and quantify their relative proportions. We observed a background of EBV positive B cells marked by the expression of LMP1. Interestingly, we also discovered a subset of non-neoplastic T cells that were EBV-positive. We observed a strong correlation between the proteomic data from CODEX and the spatial transcriptomic data for key cell type populations. Ligand-receptor interaction analysis of the transcriptomic data also showed communication between CXCL13 and its receptor CXCR5, heightened in association with FDCs. We also identified a cluster of IGHM positive cells that highly express KMT2C, one of the histone modification proteins frequently mutated in T-cell lymphomas. Preliminary epigenetic data suggests high chromatin accessibility score in the T-cell rich regions nearby HEVs. BCL6 seems to show high gene activity score in this region, despite its relatively lower transcript and protein expression level. We are currently in the process of fully integrating the RNA profiling and protein expression as well as epigenetic datasets using novel MaxFuse pipeline to provide a comprehensive understanding of the biology underlying AITL.
 Conclusion
 Combining multiplexed immunofluorescence with DBIT-seq and spatial ATAC-seq is providing a comprehensive picture of the mechanisms underlying epigenetic, transcriptional and proteomic landscape of AITL. Data in process of curation reveal novel molecular architecture that may explain underlying cell-cell interactions between TFH tumor cells, reactive B cells and the supporting stromal and vascular apparatus. Specifically, that there may be immune-suppressed EBV+ T cells in the background in addition to spatially segregated activation of tumor cells by surrounding non-neoplastic cells. Broadly, our approach can further the understanding of AITL's complex microenvironment, leading to improved targeting of tumor-enabling signals that facilitate tumor evasion of immune surveillance and drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a499a770e0e4d4798a7a5f85e6f030b569dadf71" target='_blank'>
              Spatial Multiomics Profiling of Angioimmunoblastic T-Cell Lymphoma
              </a>
            </td>
          <td>
            Archibald Enninful, Francine M. Foss, Rong Fan, Mina L Xu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third leading cause of cancer mortality in the United States. Familial adenomatous polyposis (FAP) is a hereditary syndrome that raises the risk of developing CRC, with total colectomy as the only effective prevention. Even though FAP is rare (0.5% of all CRC cases), this disease model is well suited for studying the early stages of malignant transformation as patients form many polyps reflective of pre-cancer states. In order to spatially profile and analyze the pre-cancer and tumor microenvironment, we have performed single-cell multiplexed imaging for 52 samples: 12 normal mucosa,16 FAP mucosa,18 FAP polyps, 2 FAP adenocarcinoma, and 4 sporadic colorectal cancer (CRCs) using Co-detection by Indexing (CODEX) imaging platform. The data revealed significant changes in cell type composition occurring in early stage polyps and during the malignant transformation of polyps to CRC. We observe a decrease in CD4+/CD8+ T cell ratio and M1/M2 macrophage ratio along the FAP disease continuum. Advanced dysplastic polyps show a higher population of cancer associated fibroblasts (CAFs), which likely alter the pre-cancer microenvironment. Within polyps and CRCs, we observe strong nuclear expression of beta-catenin and higher number neo-angiogenesis events, unlike FAP mucosa and normal colon counterparts. We identify an increase in cancer stem cells (CSCs) within the glandular crypts of the FAP polyps and also detect Tregs, tumor associated macrophages (TAMs) and vascular endothelial cells supporting CSC survival and proliferation. We detect a potential immunosuppressive microenvironment within the tumor ‘nest’ of FAP adenocarcinoma samples, where tumor cells tend to segregate and remain distant from the invading immune cells. TAMs were found to infiltrate the tumor area, along with angiogenesis and tumor proliferation. CAFs were found to be enriched near the inflammatory region within polyps and CRCs and may have several roles in supporting tumor growth. Neighborhood analyses between adjacent FAP mucosa and FAP polyps show significant differences in spatial location of cells based on functionality. For example, in FAP mucosa, naive CD4+ T cells alone tend to localize near the fibroblast within the stromal compartment. However, in FAP polyp, CD4+T cells colocalize with the macrophages for T cell activation. Our data are expected to serve as a useful resource for understanding the early stages of neogenesis and the pre-cancer microenvironment, which may benefit early detection, therapeutic intervention and future prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f34b5e9edbfa348b570295378a92ebadba11703" target='_blank'>
              Single-cell spatial mapping reveals alteration of cell type composition and tissue microenvironment during early colorectal cancer formation
              </a>
            </td>
          <td>
            Tuhin K. Guha, Edward D. Esplin, Aaron M. Horning, R. Chiu, Kristina Paul, Annika K. Weimer, Winston R. Becker, Rozelle Laquindanum, M. Mills, D. G. Esplin, Jeanne Shen, Emma Monte, Shannon M. White, Thomas V. Karathanos, Daniel L. Cotter, Joanna Bi, U. Ladabaum, Teri A. Longacre, Christina Curtis, William J. Greenleaf, James M Ford, Michael P. Snyder
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Deciphering the cell state dynamics is crucial for understanding biological processes. Single-cell lineage tracing technologies provide an effective way to track the single-cell lineages by heritable DNA barcodes, but the high missing rates of lineage barcodes and the intra-clonal heterogeneity bring great challenges for dissecting the mechanisms of cell fate decision. Here, we systematically evaluate the feature of single-cell lineage tracing data, and then develop an algorithm scTrace+ to enhance the cell dynamic traces by incorporating multi-faceted transcriptomic similarities into lineage relationships via a Kernelized Probabilistic Matrix Factorization model. We assess its feasibility and performance by conducting ablation and benchmarking experiments on multiple real datasets, and show scTrace+ can accurately predict the fates of cells. Further, scTrace+ effectively identifies some important driver genes implicated in cellular fate decision of diverse biological processes, such as the cell differentiation or the tumor drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b172a55df4793c755c61f2ca21d72c3fa5022eb1" target='_blank'>
              scTrace+: enhance the cell fate inference by integrating the lineage-tracing and multi-faceted transcriptomic similarity information
              </a>
            </td>
          <td>
            Wenbo Guo, Zeyu Chen, Xinqi Li, Jingmin Huang, Qifan Hu, Jin Gu
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intercellular communication between distinct transcriptional states of various cell types in the tumor microenvironment (TME) influence the progression and clinical outcome of the tumors. Details of such clinically relevant cellular state interactions (CSIs) in IDH-mutant gliomas remain obscure. Here we developed CSI-TME, a computational pipeline that deconvolves cell type-specific gene expression from bulk transcriptomic data, identifies distinct cell states, and uncovers prognostic cell state interactions by modelling the clinical data based on the joint activity of cell state pairs. We found a significantly reproducible cell state interaction network (CSIN) that is (i) predominantly pro-tumor, (ii) likely mediated through the interaction between cell surface molecules, (iii) differentially activated in IDH-mut astrocytoma and oligodendroglioma, and (iv) significantly associated with response to immune checkpoint blockade therapy. The distinct malignant cell states involved in CSIs resembled neuronal lineages such as astrocyte-like and oligodendrocyte progenitor cells-like states and captured key interactions between glioma stem cells and immune cells. Integration of CSIN with somatic mutation data suggests the anti-tumor role of CSIs in the early stages that transitions towards pro-tumor role as glioma progresses. In summary, CSI-TME provides valuable insights into the physiology of the TME in IDH-mutant glioma and provides a framework to prioritize ligand-receptor interactions or patients stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f3c50514c2a24aadfd75ca63abf749d7a1de48" target='_blank'>
              Identifying Prognostic Cell State Interactions in the Tumor Microenvironment of IDH-Mutant Gliomas Using CSI-TME
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Kenneth Aldape, Sridhar Hannenhalli
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Carcinogenesis is characterized not only by the uncontrolled growth of malignant cells but also by the disruption of the normal balance of cellular processes and intercellular interactions in the microenvironment that overcome the constraints of tissue homeostasis and support malignant growth. We profiled benign pulmonary dysplasia, carcinoma in situ, and invasive lung carcinomas at single-cell resolution to identify composite changes in cellular processes, signaling, and interactions among tumor-immune-stromal cells in the microenvironment in progressively advanced disease stages. We developed OncoTerrain, a hyperparameter-tuned model that captured synergistic multimodal signatures of an increasingly perturbed microenvironment in malignant disease stages and identified composite microenvironmental changes that supported cancer hallmarks. Key cancer-related changes in transcriptional states, cellular processes, and intercellular interactions involving immune, fibroblast, and stromal cell types preceded tumor initiation and were often synergistic. The microenvironment of increasingly malignant tissues was characterized by immune avoidance, ECM remodeling, and altered cell mobility. There were changes in cell states in fibroblasts, macrophages, and their inter-cellular interactions with other cell types, whereas T-cell activation occurred late. The in-situ carcinomas showed variations in the composite microenvironmental states that corroborated their pathology, which was not apparent at the genome level. A subset of those harbored populations of tumor and non-tumor cells with aggressive characteristics in some but not all aspects of hallmarks of carcinogenesis in the lung. We suspect that the variation in the coordination of microenvironmental cues may influence why some but not all in-situ tumors progress to the advanced stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437dc7b34021d21e799bbd0ca24c7b8aa0ec942c" target='_blank'>
              Disruptive changes in tissue microenvironment prime oncogenic processes at different stages of carcinogenesis in lung
              </a>
            </td>
          <td>
            Vignesh Venkat, Xiaoju Hu, Antara Biswas, Ankit Saxena, Jyoti Malhotra, Gregory Reidlinger, Subhajyoti De
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In this study, we delved into the intricate cellular landscape and interactions within hepatocellular carcinoma (HCC) using single-cell RNA sequencing (scRNA-seq). Our analysis uncovered a diverse array of cell types present in HCC tumors, including endothelial cells, fibroblasts, hepatocytes, Kupffer cells, macrophages, neutrophils, NK cells, pericytes, and proliferating cells. By examining the expression profiles of key marker genes, we elucidated the distinct transcriptional signatures associated with each cell type. Notably, we observed upregulation of certain genes associated with tumor progression and metastasis, highlighting potential therapeutic targets. Additionally, our study revealed the relative proportions of each cell type within HCC tumors, shedding light on the complex cellular composition of these malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516f83808b51104c5136da0299de6bce74fe34c4" target='_blank'>
              Single-Cell RNA Sequencing: Exploring the Cellular Landscape and Interactions in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Wenyi Wei
          </td>
          <td>2024-11-25</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="T cell therapies, such as chimeric antigen receptor (CAR) T cells and T cell receptor (TCR) T cells, are a growing class of anti-cancer treatments. However, expansion to novel indications and beyond last-line treatment requires engineering cells’ dynamic population behaviors. Here we develop the tools for cellular behavior analysis of T cells from live-cell imaging, a common and inexpensive experimental setup used to evaluate engineered T cells. We first develop a state-of-the-art segmentation and tracking pipeline, Caliban, based on human-in-the-loop deep learning. We then build the Occident pipeline to collect a catalog of phenotypes that characterize cell populations, morphology, movement, and interactions in co-cultures of modified T cells and antigen-presenting tumor cells. We use Caliban and Occident to interrogate how interactions between T cells and cancer cells differ when beneficial knock-outs of RASA2 and CUL5 are introduced into TCR T cells. We apply spatiotemporal models to quantify T cell recruitment and proliferation after interactions with cancer cells. We discover that, compared to a safe harbor knockout control, RASA2 knockout T cells have longer interaction times with cancer cells leading to greater T cell activation and killing efficacy, while CUL5 knockout T cells have increased proliferation rates leading to greater numbers of T cells for hunting. Together, segmentation and tracking from Caliban and phenotype quantification from Occident enable cellular behavior analysis to better engineer T cell therapies for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e025cf1fccd7422ab1c920631722b6d643225bf" target='_blank'>
              Cellular behavior analysis from live-cell imaging of TCR T cell–cancer cell interactions
              </a>
            </td>
          <td>
            Archit Verma, Changhua Yu, Stefanie Bachl, Ivan Lopez, Morgan Schwartz, Erick Moen, Nupura Kale, Carter Ching, Geneva Miller, Tom Dougherty, Edward Pao, William Graf, Carl Ward, Siddhartha Jena, Alex Marson, J. Carnevale, David Van Valen, Barbara E. Engelhardt
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Melanoma, a highly metastatic skin cancer, exhibits variations in prognosis and response to therapy based on the site of metastasis. Despite the success of immunotherapy and targeted therapies in melanoma, over half of metastatic melanoma patients will experience disease progression due to therapy resistance. The heterogeneity and plasticity of melanoma cells contribute to metastatic dissemination and therapy resistance. Our aim is to determine whether distinct clones and/or their transcriptional cell states can predict the formation of tumors in various organs and assess how these clones change over time. To identify melanoma clones across different metastatic sites, NOD scid gamma (NSG) mice and C57BL/6 mice were injected subcutaneously, intravenously or intracranially with the same pool of cells of barcoded luciferase expressing YUMMER1.7 murine melanoma cells. Transduction conditions ensured that one DNA barcode integrated into cell genomes at one barcode per cell, serving as a lineage tag. Bioluminescence imaging was performed once a week to monitor tumor growth of mice injected intravenously and intracranially. Subcutaneous tumors were measured by calliper. Mice were euthanized at different time points; day 8, 15, 22, 29 post-implantation and at ethical endpoints. Tumors were harvested and DNA sequencing was performed to identify barcodes expressed by the tumor cells. All mice developed tumors, with 100% penetrance in NSG mice. However, in C57BL/6 mice, 10% of mice intravenously injected and 27% of subcutaneously implanted mice showed complete lesion regression, suggesting that the immune system may be responsible for tumor regression. Analysis of barcodes allowed us to assess the heterogeneity of melanoma tumors at different metastatic sites and their evolution over time. Lineage tracing and clonal heterogeneity will be evaluated using the state of art technology, SPLINTR (Single-cell Profiling and LINeage Tracing), enabling us to match a cells evolution with changes in transcriptional states. Barcode analysis performed before implanting the cells and, at different timepoints in subcutaneous and lung tumors showed that dominant subclones at the baseline were also dominant in subcutaneous and lung tumors in immunocompromised mice. In contrast, dominant subclones in immunocompetent mice was those present in lower proportion at baseline. Additionally, greater variability of subclones was observed especially in lung and brain tumors, across immunocompetent mice, likely as a mechanism of resistance, enabling these tumors to overcome immunoediting. Understanding the variability of clonality between different metastatic sites and over time will improve our comprehension of the role of different subclones in organ- specific metastasis and their transcriptional cell states.
 Citation Format: Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, Davide Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur. Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd114bc5004ce77e46072db707d1b3ec32d44c22" target='_blank'>
              Abstract B001: Intra- and inter-tumoral heterogeneity of melanoma across different metastatic sites
              </a>
            </td>
          <td>
            Veronica L. Aedo Lopez, Reem Saleh, Benjamin Blyth, Xin Du, Dane Vassiliadis, Katie Fennell, D. Moi, Roberta Mazzieri, Riccardo Dolcetti, Karen E. Sheppard, Grant A. McArthur
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract High-dimensional, spatial single-cell technologies, such as CyTOF imaging mass cytometry (IMC), provide detailed information regarding locations of a large variety of cancer and immune cells in microscopic scales in tumor microarray slides obtained from patients prior to immune checkpoint inhibitor (ICI) therapy. An important question is how the initial spatial organization of these cells in the tumor microenvironment (TME) changes with time and regulates tumor growth and eventually outcomes as patients undergo ICI therapy. Utilizing IMC data of melanomas of patients who later underwent ICI therapy, we develop a spatially resolved interacting cell system model that is calibrated against patient response data to address the above question. We find that the tumor fate in these patients is determined by the spatial organization of activated CD8+ T cells, macrophages, and melanoma cells and the interplay between these cells that regulate exhaustion of CD8+ T cells. We find that fencing of tumor cell boundaries by exhausted CD8+ T cells is dynamically generated from the initial conditions that can play a protumor role. Furthermore, we find that specific spatial features such as co-clustering of activated CD8+ T cells and macrophages in the pretreatment samples determine the fate of the tumor progression, despite stochastic fluctuations and changes over the treatment course. Our framework enables the determination of mechanisms of interplay between a key subset of tumor and immune cells in the TME that regulate clinical response to ICIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9225e8a7e492a347e0673b0bb1568c995b67d250" target='_blank'>
              Spatial organization and stochastic fluctuations of immune cells impact clinical responsiveness to immunotherapy in melanoma patients
              </a>
            </td>
          <td>
            Giuseppe Giuliani, William Stewart, Zihai Li, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2024-11-26</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Quiescent cancer cells (QCCs) remain unconquered challenges in cancer treatment. Even after being declared cured, patients remain at risk of cancer recurrence or face overtreatment aimed at eliminating minimal residual QCCs. The lack of effective diagnostic and therapeutic options for QCCs stems from our incomplete understanding of these cells. This knowledge gap is largely due to the scarcity or limitations of preclinical models, particularly those that enable the tracing of QCCs in vivo. By reviewing current QCC tracing tools and their limitations, this review aims to explore more advanced systems for tracing QCCs in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f976d0eb234c239f80d5b7267b1ecf33f0afd1fb" target='_blank'>
              Tracing Quiescent Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Moon Jong Kim
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b3d699dd705e1b450a74d7c903ab854deff777" target='_blank'>
              Increased spatial coupling of integrin and collagen IV in the immunoresistant clear-cell renal-cell carcinoma tumor microenvironment
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, Taylor Peak, Oscar E. Ospina, N. Chakiryan, Anders E Berglund, Paul A Stewart, Jonathan V Nguyen, C. M. Segura, Natasha L. Francis, P. R. Echevarria, J. Chahoud, Roger Li, Kenneth Y. Tsai, Jodi A. Balasi, Yamila Caraballo Peres, J. Dhillon, Lindsey A. Martinez, Warren E. Gloria, Nathan Schurman, Seang-Jae Kim, M. Gregory, James J Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-12-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains an untreatable disease. Understanding GBM’s infiltrative biology at the resection margin is limited, despite causing disease recurrence and progression. To address this, we generated a high-throughput single-nucleus (sn)RNA-seq and snATAC-seq multi-omic dataset from six tumors with distinct genomic drivers and combined it with spatial transcriptomics to characterize the unique molecular phenotype of GBM near the margin. By contrasting GBM-specific biology in matching “Core” vs. “Margin” dissections, we define unique, shared “GBM infiltration” and chromatin accessibility signatures near the margin. We prioritize EGFR as a top differentially expressed and accessible “Margin” marker across GBM subtypes, show its dynamic expression along a core-to-margin infiltration trajectory, and validate its role in migration through CRISPR/Cas9 deletion in two patient-derived models. ChIP-seq studies furthermore corroborate preferential TEAD1 binding at EGFR’s accessible regulatory elements. This validated multi-omic dataset enables further studies into tumor and microenvironment biology in the context of residual GBM disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d032da4813c4397d215d1b17d17829c10107f9f" target='_blank'>
              Spatial multi-omics defines a shared glioblastoma infiltrative signature at the resection margin
              </a>
            </td>
          <td>
            Balagopal Pai, S. Ramos, W. Cheng, T. Joshi, Gabrielle Price, Jessica Tomé-García, G. Nudelman, S. Shroff, K. Beaumont, Yong Raymund, R. Sebra, E. Zaslavsky, N. Tsankova
          </td>
          <td>2024-11-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a fatal adult solid tumour with median overall survival of 18-20 months post-diagnosis; contributing factors to therapeutic inefficacy include acquisition of genomic alterations post-therapy, immune evasion, deregulated hypervascularization, and tumor microenvironmental factors such as hypoxia. Combined with extensive inter- and intra-tumoral heterogeneity at bulk and single-cell level, hypoxia contributes to a gradient of molecular alterations that are specific to different cell populations that make up tumour bulk and reside in specific niches. Hitherto, high-dimensional histopathologic analyses of hypoxic regions within GBM tissue have not been performed. We took a combined spatial and single-cell proteomic profiling approach to investigate the histopathologic features of hypoxia by leveraging a unique clinical study wherein the exogenous hypoxia marker, pimonidazole (PIMO), was administered to GBM-patients preoperatively. Tissue specimens were subjected to imaging mass cytometry, high-resolution imaging, and serial immunohistochemistry using a panel of markers associated with cellular hallmarks of hypoxia, metabolism, proliferation, stemness, angiogenesis, and immune cell types. Our findings showed that PIMO staining is associated with histopathologic features of hypoxia and correlates with specific metabolic, immune, and stemness markers in GBM; specific lymphocyte populations were depleted from hypoxic regions alongside alterations in macrophagic and microglial landscape in a niche-specific manner; hypoxia reduced the proportion of proliferating glioma initiating cells and altered the proliferative, transcriptional, and translational capacity of different cell populations; microvessel density was reduced in hypoxic microenvironment. Cytometry by time-of-flight further validated the altered proportions of specific immune cell types enriched in hypoxic populations of GBM microenvironment. Our study is the first to report use of PIMO to interrogate spatial, single-cell, and phenotypic architecture associated with tissue hypoxia and altered expression of biomarkers associated with hypoxia/glycolysis, immune infiltration, proliferation, and stemness. Identification of targetable biomarkers and mediators of hypoxia-driven habitats in GBM may provide direction for future immunotherapeutic research.
 Citation Format: Shreya Gandhi, Sheila Mansouri, Olivia Singh, Mark Zaidi, Ronald Wu, Andrew Gao, Bradley Wouters, Gelareh Zadeh. Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5960e28aa4e015a793e4ede112dd73ea174cad4c" target='_blank'>
              Abstract B069: Spatial and single-cell proteomic landscaping of the hypoxic microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Shreya Gandhi, S. Mansouri, O. Singh, M. Zaidi, Ronald Wu, A. Gao, Bradley Wouters, G. Zadeh
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 The variable clinical course and response to treatment of Follicular Lymphoma (FL) is strongly influenced by its tumor microenvironment (TME). Indeed, cellular interactions dictate pro- and anti-tumor processes within the TME, although these mechanisms remain incompletely understood. Multiple bulk and single-cell studies have shed light on the TME with prognostic implications. For example, a macrophage gene signature as well as a ‘T cell depleted’ milieu have been associated with inferior clinical outcomes.
 Imaging Mass Cytometry (IMC) permits the multiplexed assessment of over 40 markers concomitantly, superseding the limits of conventional immunohistochemistry, thus enabling detailed single cell spatial profiling of the TME. For comprehensive analysis of tumor immune architecture within FL, we first optimized a 31-plex panel to capture tumor, immune and stromal cell subsets, as well as functional information such as memory and effector status, proliferation, activation and exhaustion. Fixed formalin paraffin embedded (FFPE) biopsies from untreated FL patients enrolled on the GALLIUM trial (NCT01332968), assessing frontline chemo-immunotherapy regimens, were subjected to IMC and 381 high-dimensional images were acquired.
 After quality control, we analysed >3.5 million cells across 340 high-dimensional images. Cell phenotyping identified ~78 cell populations including lymphoma cells, CD4+ and CD8+ T cell subsets (including T follicular helper (Tfh), memory, naïve, exhausted) and a range of non-lymphoid cells e.g. myeloid and stromal cells. As expected, CD20+ B cells were expanded in FL images compared with control lymphoid samples (~56% vs. ~15% respectively p<0.05). Consistent with previous literature, we found an expansion of CD4+ T cells in FL (~20%) compared with control tissue (~12% p<0.05). We have additionally identified sets of samples enriched for CD57+ T cells. The presence of these cells has previously been associated with inferior survival in FL. Furthermore, we have observed that parent lineage abundance (e.g. CD20+ cells, CD3+ cells) identified via our phenotyping of FFPE biopsies were consistent with CIBERSORTx deconvolution analysis from patient-matched RNA-seq samples, improving confidence in our phenotypic labels.
 We next analysed cellular neighborhoods, defining ten distinct FL environments. Analysis was performed i) at the level of neighborhoodcellular composition, and ii) at the level of relative proportions of identified neighborhoods within images. In terms of neighborhood cellular composition, among visually similar tissue components, we found striking variability. For example, we assessed the cellular composition of the neighborhoods making up follicles (as defined by the presence of CD21+ meshworks). In some cases, these neighborhoods were very homogenous with the cellular infiltrate consisting almost exclusively of lymphoma CD20+ cells. In other cases, we found that such neighborhoods were highly heterogenous encompassing diverse immune cell composition.
 By analysing the relative abundance of different neighborhoods, we found sets of variable architectures across images. For example, the ‘lymphoma exclusive’ neighborhood dominated many images. We found that images with high abundance of this neighborhood typically also contained a scattered ‘lymphoma+, CD4/8T+, CD16+‘ neighbourhood additionally exhibiting immunosuppressive Tregs cells, as well as CD57+ T cells. In contrast, a separate notable pattern of neighbourhood abundances was observed in a set of images containing organised follicular structures with detectable CD21+ meshwork, admixed with CD57+/- Tfh-like cells and macrophages. Here, this follicular structure was outlined by a thin layer of CD4+ T cells, CD57+PD1+ T cells and lymphoma cells. These images are completed by an interfollicular region containing neighborhoods comprising mixed immune subsets.
 Ongoing work seeks to examine associations between such variable spatial arrangements with this clinically annotated dataset. This will provide a detailed characterization of how the spatial context and crosstalk of individual cells amid neighbourhoods within the FL TME specifically influences disease behavior, response to treatment and clinical outcome, which will likely extend and compliment those findings already described at bulk and single cell level.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945f841925664791c3d4fe453e1cae6353b65fa0" target='_blank'>
              Highly Heterogeneous Cellular Composition and Neighborhood Architecture within the Follicular Lymphoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Paul Buckley, Julie Chan, Cynthia Bishop, Timothy Woo, K. Todd, Elizabeth H. Phillips, A. Bottos, Christopher Bolen, V. Passerini, Oliver Weigert, Piers E. M. Patten
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Deciphering shared features between patients through unsupervised analyses of tumor single-cell transcriptomes is hindered by the predominant clustering of malignant cells based on the patient’s tumor of origin. In contrast, cancer-associated non-malignant cells within tumors cluster according to their somatic cell type (e.g. macrophage, fibroblast, astrocyte), regardless of the individual patient. We investigated the origin of this contrasting clustering behavior using computational analyses and novel data sampling techniques across 160 tumors representing 9 distinct cancer types. This led to the unexpected disclosing of tumor-specific malignant cell types. We demonstrate that tumor-driven malignant cell clustering is independent from technical or computational biases, nor is it reducible to defined gene sets with tumor-specific expression across all examined cancers. On the opposite, we unveil redundant information dispersed across the transcriptome that encodes a unique identity shared by the malignant cells within each tumor. Additionally, we found a similar genomic encoding for the identity of normal cell types. Finally, we demonstrate that malignant cell identities are maintained across space and over time, as are the identities of normal somatic cells types. These findings suggest the establishment of a distinct type of malignant cells within each tumor, robustly and diffusively encoded across the entire transcribed genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2c0940cbf36006f73fd3a210e9572d4abd3013" target='_blank'>
              A unique malignant cell type per patient tumor is encoded in each cancer cell transcriptome
              </a>
            </td>
          <td>
            M. Saurty-Seerunghen, Elias A. El-Habr, Yossi Eliaz, Léa Bellenger, Christophe Antoniewski, Hervé Chneiweiss, M. Junier
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulation of intracellular signaling networks underpins cancer. However, a systems-level elucidation of how signaling networks within distinct cell subpopulations drive cancer progression in vivo has been unattainable due to technical limitations. We developed INSIGHT (INvestigating SIGnaling network of specific cell subpopulation in Heterogeneous Tissue), a new platform technology combining fluorescence-activated cell sorting with ultra-sensitive mass spectrometry to enable phosphoproteomic characterization of rare and discrete cell subpopulations from fixed tissues. We demonstrated the broad utility of INSIGHT by analyzing the oligodendroglial cell-specific signaling network in the mouse brain. We then applied INSIGHT to investigate the rare, disseminated tumor cell subpopulation in glioblastoma patient-derived xenograft models. INSIGHT uncovered a global rewiring of signaling networks with tumor cell dissemination, marked by a transition from proliferation-associated signaling in the primary tumor cells to signaling associated with postsynapse, neuronal migration, and ion homeostasis in disseminated tumor cells. We reveal interconnections between signaling circuitries within the networks, with numerous proteins, including GluA2, exhibiting altered phosphorylation without protein expression changes, emphasizing the role of post-translational modifications in glioblastoma dissemination. We validated key phosphorylation changes and inferred differentially active kinases with tumor spread to offer new systems-level insights into glioblastoma dissemination mechanisms in vivo. INSIGHT is generally applicable to a wide range of biological systems without genetic engineering and provides quantitative phosphorylation and protein expression data for selected cell subpopulations from heterogeneous tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/decaf507fd537acca35393bd50f3bedca402f1f6" target='_blank'>
              Uncovering the signaling networks of disseminated glioblastoma cells in vivo with INSIGHT
              </a>
            </td>
          <td>
            Ryuhjin Ahn, Alicia D. D’Souza, Lawrence Long, Yufei Cui, D. Burgenske, K. Bakken, Lauren L Ott, Brett L. Carlson, Grace Zhou, Tomer M. Yaron-Barir, Ishwar N. Kohale, Charles A. Whittaker, Cameron T. Flower, Jeffrey Wyckoff, A. Tuma, Jared L. Johnson, Jann Sarkaria, Forest M. White
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary brain cancer in adults, characterized by robust infiltrative growth and invasion within the neural microenvironment. While various stromal cells, including astrocytes, microglia, and vascular cells in the microenvironment, communicate with GBM cells to promote growth and invasion, relatively less is known about another critical component of the tumor stroma: the extracellular matrix (ECM). The matrisome comprises hundreds of ECM genes and affiliated adhesion and signaling components; however, little is understood about (i) the repertoire of matrisome genes and proteins expressed in GBM, (ii) the stromal and cancer cells of origin that express different matrisome factors, and (iii) how these components are interconnected to drive tumor malignancy. Here, we have used in situ single-cell transcriptomics to analyze the spatial expression of nearly 350 genes that constitute the human brain matrisome. Our efforts have identified several differentially expressed matrisome genes involved in ECM adhesion and signaling between cancer and stromal cells. Additionally, matrisome genes associated with low-grade glioma pathogenesis and progression to high-grade GBM have been identified and spatially mapped. Collectively, these experiments reveal the single-cell origins of differentially expressed ECM biomarkers as well as their spatial expression patterns in brain tumors. These findings may lead to new therapeutic efforts to inhibit matrisome components to benefit patients with malignant brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db27897e2873e1f72dc0c04bc6970233914ac8ad" target='_blank'>
              In Situ Single-Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A. Forero, Sumod Sebastian, John E. Morales, Marisol De La Fuente-Granada, Jason T Huse, L. Ballester, Jeffrey S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying malignant transformation. While the cellular compositions of G3/G4 MBs are consistent with a truncated neurodevelopmental hierarchy, it is unknown whether G3/G4 MBs exhibit cellular plasticity. We profiled tumors from 38 patients who underwent surgery for MB at UCSF via single-nucleus RNA sequencing (snRNA-seq). We developed in vivo genetic lineage-tracing systems with a single-cell readout and applied them to patient-derived G3 MB xenografts. High-complexity lentiviral barcode libraries were transduced into D283 and D425 patient-derived cell lines. Barcoded cells were injected into the brains of immunocompromised mice. Both preimplantation barcoded cultures and mature tumors were profiled by single-cell RNA-sequencing (scRNA-seq). This captured both endogenous RNA and barcode transcripts. RL neuro-developmental cell-type labels were transferred from public single-cell data from fetal cerebellum to single-cell data from our patient samples and xenograft models. Remarkably, xenografts produced similar cell types, in similar proportions to those found in human MBs. Moreover, the distribution of cell types found in the xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. We developed approaches to use barcode and expressed point-mutation data to reconstruct intra-clone genetic phylogenies. We then used the transcriptional phenotypes of the observed cells to infer ancestral phenotypes for each clone’s phylogeny. We compared clade to sub-clade differences in cell type against the neural-developmental hierarchy found in the fetal RL. We observed significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. These studies indicate that G3 MB cellular composition is homeostatic and conserved at the level of individual clones. Yet, these tumors exhibit plasticity, as has been observed in other brain cancers such as glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05dba25f98bbf2ce54f337e73be727bd8768d6f8" target='_blank'>
              STEM-20. MULTI-OMICS OF HUMAN MEDULLOBLASTOMAS AND GENETIC LINEAGE TRACING IN VIVO INDICATE CELLULAR PLASTICITY
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Aaron A. Diaz
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 We integrated single-cell RNA-sequencing, Visium spatial transcriptomic, and Xenium in situ transcriptomic data to identify differences in cell composition between the tumor core and infiltrating edge in IDHWT glioblastoma (GBM). Using CellTrek, an algorithm that maps single cells to their most-likely locations in spatial transcriptomic data, we identified extensive spatial organization in GBM. Cell mapping revealed distinct regions enriched for tumor, immune, endothelial, neuronal, and glial cells. These were associated with the location of the tissue sample – edge regions were enriched for neurons and glia (as expected), while core regions were enriched for tumor cells and infiltrating immune cells, particularly macrophages. Within the core samples, the tumor and immune cells tended to occupy distinct spatial niches. Spatial clustering of mapped tumor cells across all samples revealed five spatially segregated tumor cell states, each associated with specific biological processes, including (1) histone methylation, (2) proliferation, (3) wound-healing, EMT, and antigen presentation, (4) hypoxia response, and (5) stem cell differentiation. These tumor cell states exhibited preferential colocalization with specific non-malignant cell types. For instance, antigen-presenting/wound-healing tumor cells tended to colocalize with diverse immune cells, while stem-like tumor cells tended to reside in neuron-rich regions at the tumor edge. Next, to identify ligand-receptor interactions governing this spatial organization, we analyzed known ligand-receptor pairs within all three data types. This revealed known and novel mediators of tumor cell interactions with vasculature, lymphoid cells, myeloid cells, neurons, and glia. While some interactions were shared across all tumor cell states (e.g. all tumor cell states interacted with myeloid cells via APP/CD74 interactions), others were specific to particular tumor cell states (e.g. wound-healing/antigen-presenting tumor cells interacted with macrophages via CSF1/CSF1R, while proliferating and stem-like tumor cells interacted with neurons through multiple known NRXN/NLGN interactions). These interactions suggest potential targets for clinical disruption of the tumor ecosystem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b29e4251f4b5ec080f177e87d960e1da6b0090" target='_blank'>
              EPCO-55. SPATIAL HETEROGENEITY IN THE TUMOR CORE AND INFILTRATING MARGIN OFIDHWT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Saad Khan, Anthony Wang, Rupen Desai, Jennifer Foltz, Eric Leuthardt, B. Patel, S. Dahiya, Albert Kim, Gavin Dunn, Allegra Petti
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20edd8d2bb6126475fe06f906514312b49ac959d" target='_blank'>
              Integrating histopathology and transcriptomics for spatial tumor microenvironment profiling in a melanoma case study
              </a>
            </td>
          <td>
            Óscar Lapuente-Santana, J. Kant, Federica Eduati
          </td>
          <td>2024-11-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a multi-clonal disease, existing as a milieu of clones with unique but related genotypes as initiating clones acquire subsequent mutations. However, bulk sequencing cannot fully capture AML clonal architecture or the clonal evolution that occurs as patients undergo therapy. To interrogate clonal evolution, we performed simultaneous single cell molecular profiling and immunophenotyping on 43 samples from 32 NPM1-mutant AML patients at different stages of disease. Here we show that diagnosis and relapsed AML samples display similar clonal architecture patterns, but signaling mutations can drive increased clonal diversity specifically at relapse. We uncovered unique genotype-immunophenotype relationships regardless of disease state, suggesting leukemic lineage trajectories can be hard-wired by the mutations present. Analysis of longitudinal samples from patients on therapy identified dynamic clonal, transcriptomic, and immunophenotypic changes. Our studies provide resolved understanding of leukemic clonal evolution and the relationships between genotype and cell state in leukemia biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ca51b5a9370b4c12038c29c1cdd25d0324a8d0" target='_blank'>
              Genotype-immunophenotype relationships in NPM1-mutant AML clonal evolution uncovered by single cell multiomic analysis
              </a>
            </td>
          <td>
            Morgan Drucker, Darren Lee, Xuan Zhang, Bailee N. Kain, Michael Bowman, D. Nicolet, Zhe Wang, R. Stone, K. Mrózek, Andrew J Carroll, D. Starczynowski, R. L. Levine, John C. Byrd, Ann-Kathrin Eisfeld, Nathan Salomonis, H. L. Grimes, Robert L Bowman, Linde A. Miles
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Childhood is critical for hematopoietic development and the onset of hematologic diseases. To explore hematopoietic changes from infancy through adolescence, we generated a multi-modal single-cell atlas capturing mRNA and surface protein expression of 90.710 bone marrow (BM) cells. This includes hematopoietic stem/progenitor cells and mesenchymal stromal cells, from seven pediatric individuals and two young adults. We demonstrate that young pediatric BM is distinct from adolescents/young adults (AYA), shifting from B-lineage dominance in early childhood to myeloid and T-lineage bias in adolescence. We uncover two distinct lymphoid progenitors (LyPs) subsets regulating this shift: CD127-positive LyPs with B-lineage output, most abundant in early childhood, and CD127-negative LyPs with lymphoid and myeloid features, more common in AYAs. Age-related changes in stromal composition and signaling, mediated by IL-7 and TGF-β1, correspond with this lineage shift. This study provides an in-depth resource for understanding healthy hematologic development and potential early-life perturbations underlying pediatric hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c596c251202175e6c82f73f5b9ffcf7bd96991a" target='_blank'>
              Single-cell multiomics of pediatric BM reveals age-dependent differences in lineage differentiation linked to stromal cell heterogeneity
              </a>
            </td>
          <td>
            Evelyn S. Hanemaaijer, Konradin F. Müskens, Ireen J. Kal, Brigit M. te Pas, Patrycja Fryzik, Nina Epskamp, Aleksandra Balwierz, T. Candelli, Wim J. de Jonge, T. Margaritis, Mirjam E. Belderbos
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is hallmarked by profound inter- and intratumoral heterogeneity. Current research has mapped key genetic drivers to distinct transcriptional profiles within GBM. Nevertheless, the intricate interactions between genetic alterations and spatial-transcriptional heterogeneity remain elusive. We utilized nestin-Tva C57BL/6 mice to induce de novo GBM via oncogene overexpression alongside shRNA-mediated silencing or deletion of tumor suppressors. We modeled critical driver mutations: PDGFB, NF1, and EGFR. Employing 10X Genomics platform for single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics (Nanostring GeoMx), we integrated data using both horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, analyzed through the SPATA2 software. Advanced computational techniques, including graph neural networks and single-cell deconvolution, were leveraged to unravel the genotypic and spatial variability within tumors and their microenvironments. Our findings uncover persistent cellular heterogeneity across diverse genetic contexts. However, specific genetic mutations precipitated distinct dominant transcriptional states: NF1 and EGFR mutations drove mesenchymal and astrocytic-like states, respectively, whereas PDGFB fostered a neural progenitor/cell-like state. These transcriptional shifts were closely linked to unique microenvironmental modifications, such as elevated neuronal signaling in PDGFB-driven GBMs and significant immune infiltration in NF1 and EGFR models. Through weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) paired with spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive microenvironment in NF1-loss driven GBMs and increased lymphocyte presence around blood vessels in EGFR-amplified GBMs. These patterns were consistent with human spatially resolved transcriptomic data, emphasizing the genetic complexity and heterogeneity of human GBMs. Our genetically engineered mouse models serve as a robust platform for dissecting GBM with defined genetic mutations. Leveraging sophisticated analytical techniques and computational modeling, this study underscores the profound interplay between genetic drivers and the tumor microenvironment, paving the way for targeted therapeutic strategies and precision medicine in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d3f584c4fce5754e72560c1ec2add719af021f" target='_blank'>
              EPCO-18. DECIPHERING SPATIAL DIVERSITY OF GLIOBLASTOMA GENETIC ARCHITECTURE
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 To effectively beat glioblastoma, new pre-clinical models are needed: to dissect the biology of processes like migration and therapy resistance; to understand immune suppression and immunotherapy; and to use as a model for testing in therapeutic development. We have developed a new panel of glioblastoma mouse models, which engineer well-characterised glioblastoma driver mutations into an immortalised eGFP murine astrocyte-like cell. These “GEM-CLeM” are Genetically Engineered Mouse Cell Line Models. When transplanted intra-cranially into immunocompetent mice, these lines formed tumors with the key characteristics of glioblastoma – migratory and invasive, with rapid proliferation and infiltration of suppressive immune cells. The eGFP expression allowed detection of glioblastoma cells in situ and ex vivo, and the ability to readily distinguish tumor and stromal components of the tumors. Importantly, different combinations of driver mutations led to diverse tumor phenotypes. Loss of Pten with over-expression of mutant RAS generated tumors with 100% efficiency and the same time to tumor onset as the commonly used GL261 cell line. However, unlike GL261 tumors the Pten-RASV12 tumors had a highly invasive edge and extensive migration, and were full of pro-tumorigenic macrophages and monocytes, with very few T-cells or dendritic cells. Together, this recapitulated the phenotype of human wildtype IDH1 glioblastoma. In contrast, the very common EGFRvIII mutant did not drive tumor formation in combination with Pten loss. We also modelled mutant IDH glioblastoma by combining loss of p53 with overexpression of IDH1R132H. These were slow growing, densely packed tumors with extensive microglial involvement in addition to pro-tumorigenic macrophages. This GEM-CLeM system combines the benefits of the traditional genetically engineered mouse models with accessible, rapid, reproducible tumor formation. The GEM-CLeM lines are amenable to further genetic manipulation, so cells can be customised with other mutations to ask important biological questions in the presence of an intact tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452f39600e3977617ed3591b2ad73dc07e6976ca" target='_blank'>
              TMIC-66. A NEW MOUSE MODEL THAT COMBINES DRIVER MUTATIONS TO ACCURATELY RECAPITULATE THE DIVERSE PHENOTYPES OF HUMAN GLIOBLASTOMA
              </a>
            </td>
          <td>
            D. Forsythe, Olivia Burn, I. Hermans, Melanie J McConnell
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1266e9e933529788695d2086bf906381800e149" target='_blank'>
              Single-cell and bulk transcriptome analysis reveals tumor cell heterogeneity and underlying molecular program in uveal melanoma
              </a>
            </td>
          <td>
            Ke Li, Jingzhe Huang, Ying Tan, Jie Sun, Meng Zhou
          </td>
          <td>2024-11-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Myelodysplastic syndromes (MDS) represent a heterogenous group of myeloid neoplasms characterized by ineffective hematopoiesis, dysplastic morphology, and risk of transformation to acute leukemia. While neoplastic cell-intrinsic mechanisms driving MDS pathogenesis have been intensely investigated, the spatial architecture and in situ immune microenvironment in human MDS has not been studied at single-cell resolution. We hypothesized that the MDS bone marrow microenvironment is characterized by altered hematopoietic and non-hematopoietic cell distributions and transcriptional profiles including the composition and location of adaptive immune subsets.
 We developed an approach for high-definition spatial transcriptomic analysis of fixed and EDTA-decalcified human marrow core biopsies, enabling interrogation of hundreds of genes simultaneously in the context of their cellular location. We designed a marrow and immune focused 433 gene panel for the Xenium platform (10x Genomics) and applied this to 7 decalcified core marrow biopsies from patients with early-stage untreated MDS, and 3 control marrows, yielding in situ spatial data for 750,688 cells. In parallel, we performed 32-color spectral flow cytometry along with 5' single-cell CITE- and T cell receptor- (TCR) sequencing (N=31 MDS, N=9 controls) capturing 521,853 cells and 145,176 TCRs.
 Single-cell spatial transcriptomics identified >20 distinct cell types including populations that are poorly captured by analyses of single-cell suspensions from bone marrow aspirates: megakaryocytes, osteoblasts, endothelial cells, mesenchymal stromal cells, vascular smooth muscle cells, and adipocytes. Nearest neighbor analyses revealed reduced cell type diversity in MDS compared to controls, particularly surrounding osteoblasts (p=1.7x10-2). Within the immune compartment, we captured B, T, and natural killer (NK) cell types for further stratification into FOXP3+ T regulatory cells, naïve and memory CD4 or CD8 T cells, along with numerous phenotypic and functional markers.
 Spatial data were integrated with single-cell proteomic, RNA, and TCR sequencing in the same individuals to further elucidate the adaptive immune composition and T cell repertoire of the MDS marrow. In MDS samples, B cells were reduced in frequency whereas T cells were the most abundant lymphocyte cell type. Although MDS T cell subset composition was similar to control marrows, the spatial distribution of immune cells in MDS was distinct. Spatial analysis identified aggregates of T, B, and plasmacytoid dendritic cells (pDC) reminiscent of tertiary lymphoid structures in the marrow (which we defined as clusters of B, T, NK and pDCs with > 10 T cells). Immune aggregates were larger in MDS (>75 cells/aggregate, range 83-390) compared to controls (no aggregates >72 cells) with increased proportions of T cells (p=1.2x10-3) and frequencies of CD4 and CD8 T cell subsets. TCF7, a transcription factor linked to T cell stemness, and CXCR4, a key marker of bone marrow homing, were increased in CD4 (p=7.0x10-4 and p=1.5x10-5 respectively) and CD8 (p=1.6x10-2 and p=8.9x10-7 respectively) T cells in MDS immune aggregates. These findings suggest the emergence of immunologically active niches within the MDS marrow and would not have been captured via single-cell RNA-seq alone. Outside of immune aggregates, we found increased memory CD8 T cells (p=1.4x10-2) in MDS. In contrast, control marrows showed no difference in T cell subset frequency between cells inside or outside of immune aggregates. CITE/TCR-seq revealed clonal T cell expansions in MDS within GZMB+ CD8s, distinct from controls (where clonal expansions were found in GZMK+ CD8s), supporting altered CD8 T cell immunity in MDS.
 Our integrative analyses, including single-cell spatial transcriptomic profiling, revealed active remodeling of the adaptive immune system in early stage MDS. The finding of large immune aggregates in the MDS marrow with more T cells bearing signatures of active cell recruitment and growth, combined with increased clonally expanded memory CD8 T cells outside immune aggregates, point toward localized changes in the adaptive immune response in early stage MDS. Spatial analyses revealed location-specific T cell differences within the MDS marrow, underscoring the importance of combined phenotypic and spatial approaches to elucidate disease pathophysiology in MDS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b60776003cdeaafaa3fc36a226c720a5be827ebf" target='_blank'>
              Single-Cell Spatial Transcriptomics Identifies Lymphoid Aggregates with Location-Specific Alterations of Adaptive Immune Cells in Human MDS Bone Marrow
              </a>
            </td>
          <td>
            Robert F. Stanley, Beatrice Zhang, K. Argyropoulos, Brianna Gipson, Kenyon Weis, Alexander M. Lewis, Zoe Katsamakis, Karen Zhao, Catherine Snopkowski, Matthew Zatzman, Ning Fan, R. Chaligné, Umesh Bhanot, M. Roshal, A. Dogan, Benjamin D. Greenbaum, E. Stein, M. R. van den Brink, O. Abdel-Wahab, Susan DeWolf
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Fate-restricted cells can acquire stem cell-like properties through dedifferentiation, enabling them to gain the plasticity required for differentiation into multiple lineages. Tumour plasticity is prominently observed in brain cancers, where transient cell state changes are linked to resistance to conventional therapies. In this study, we demonstrate that a sub-population of dedifferentiated tumour neural stem cells (NSCs) in Drosophila, induced by the knockdown of prospero (pros), can generate its own glial niche. Temporal patterning, known to influence oncogenic competence and tumour malignancy, plays a key role in this process. Specifically, we show that de novo gliogenesis occurs in the more differentiated Syncrip+ (Syp+) NSC population. Modulating Syp levels alters the size of the glial niche, subsequently affecting tumour size. Furthermore, the tumour-associated glial niche expands through cell division and fails to cease proliferation on time due to dysregulated ecdysone signalling, contributing to niche expansion. Our findings reveal that tumours arising via dedifferentiation establish their own supportive glial microenvironment, which sustains tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d30097dbfea218ae747551091d256416f71d4db" target='_blank'>
              Tumour-derived gliogenesis sustains dedifferentiation-dependent tumour growth in the Drosophila CNS
              </a>
            </td>
          <td>
            Edel Alvarez-Ochoa, Qian Dong, Hannah Truong, Louise Y Cheng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Supratentorial ependymomas (ST-EPN) are aggressive pediatric brain tumors that are categorized into distinct molecular subgroups. However, the developmental origins, tumor microenvironment, and phenotypic characteristics of tumor subpopulations across these subgroups are still poorly understood. In this study, we explored the human developmental signatures, global spatial organization, and the morphological and migratory tumor cell state behaviors in ST-EPN tumors at unprecedented resolution. We profiled 42 ST-EPN patients encompassing ZFTA-RELA (n = 20), ZFTA-Clusters 1 to 4 (n=20), and ST-YAP1 (n=4) subgroups by single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and live-cell imaging. We identified recurrent tumor cell states across tumors, that cover both generic cell processes (including cycling and mesenchymal/hypoxia) and human embryonic and fetal brain developmental signatures. ST-EPN subgroups displayed distinct developmental signatures, mapping to two separate temporally restricted progenitors-neuroepithelial-like and embryonic-like cells. Additionally, tumors showed diverging patterns of neuronal or ependymal differentiation, with ZFTA-Cluster 3 tumors as especially distinct from other subgroups in both developmental origin and differentiation. Furthermore, by utilizing 10X Xenium and novel algorithms we discovered that mesenchymal/hypoxia is critical for driving global tissue architecture to become more organized. Notably, recurrent tumors exhibited higher proportion of mesenchymal/hypoxia cells and greater structure than primary tumors. Lastly, by superimposing molecular and functional assays in both in vivo and in vitro models, we found that cell states exhibit distinct morphological and migratory behaviors, with neuronal-like cells being the most invasive. Neuronal-like cells especially exhibited distinct behavioral patterns reminiscent of neuronal migration during development. Moreover, we demonstrated the crucial role of neuronal microenvironment in promoting plasticity of cells toward this neuronal lineage. Taken together, we present a multidimensional framework for investigating cellular states within ST-EPN tumors, offering novel insights into their developmental origins, patterns of spatial organization, and cellular behavioral characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8e8753768336386cb49ecaeea5a571c7f71396" target='_blank'>
              STEM-11. MULTIDIMENSIONAL PROFILING OF TUMOR CELL HETEROGENEITY REVEALS CELL-LINEAGE SPECIFIC FUNCTIONS IN SUPRATENTORIAL EPENDYMOMAS
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, Svenja K. Tetzlaff, Ekin Reyhan, C. A. de Biagi-Junior, C. Cascio, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Bernhard Englinger, Li Jiang, Cuong M Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M Pfister, Marcel Kool, Tomasz J Nowakowski, J. Gojo, Lissa C Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e548b5183bb26502c965fafd756207483f5bccc5" target='_blank'>
              Functional profiling of murine glioma models highlights targetable immune evasion phenotypes
              </a>
            </td>
          <td>
            N. Mikolajewicz, N. Tatari, Jiarun Wei, N. Savage, Adrian Granda Farias, Vassil Dimitrov, David Chen, Z. Zador, Kuheli Dasgupta, Magali Aguilera-Uribe, Yu-Xi Xiao, Seon Yong Lee, P. Mero, D. Mckenna, C. Venugopal, Kevin R. Brown, Hong Han, Sheila K. Singh, Jason Moffat
          </td>
          <td>2024-11-27</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Within the GBM microenvironment, distinct micro-niches arise from independent cancer cell lineages with unique metabolic needs, characterized as Fast Cycling Cells (FCCs) and Slow Cycling Cells (SCCs). FCCs preferentially utilize aerobic glycolysis for energy, whereas treatment-resistant SCCs depend on lipid metabolism. Our study decodes the molecular and spatial heterogeneity within the GBM microenvironment, focusing on the dichotomy of immune infiltrates and the metabolic interactions between SCCs and the immune compartment in human GBM patients.



 We performed spatial transcriptomics and proteomics combined with geospatially resolved single-cell neighborhood analysis of IHC-stained GBM tissues using CellProfiler and the SNAQ algorithm that we developed to investigate the specific immune microenvironment of SCCs and FCCs. Holotomography and fluorescence-based time-lapse imaging coupled with flow cytometry were used to study lipid transfer between immune cells and GBM cells. Finally, the effects of pharmacological and genetic targeting of lipid transfer were investigated both in vitro and in vivo.



 Our study establishes a connection between spatial metabolic heterogeneity and immune diversity, highlighting how specific tumor cell lineages correlate with characteristic immune micro-niches. In particular, SCCs exploit the metabolic properties of recruited immunosuppressive myeloid-derived cells, which metabolically support tumor cells by facilitating lipid transport and supporting SCC metabolism. Our results show that inhibiting lipid transfer affect SCCs, remodels the immune microenvironment, delays tumor growth and improves disease outcome. Our study also demonstrates a correlation between predicted sensitivity to lipid-modifying drugs, such as statins, and the SCC phenotype. This correlation could enable stratification and selection of patients most likely to benefit from such treatments.



 This study establishes critical connections between the TME, cellular metabolism, anti-tumor immunity, and treatment vulnerabilities. Our research provides novel insights into how metabolic exchanges contribute to tumor progression and suggests that targeting metabolic interactions could be an effective therapeutic strategy for GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec73458fa94d4e1053c3c077508f221cb827e3" target='_blank'>
              TMIC-52. DECODING METABOLIC INTERACTIONS IN GLIOBLASTOMA TUMOR MICROENVIRONMENT THROUGH COMPREHENSIVE SPATIAL TRANSCRIPTOMIC AND PROTEOMIC ANALYSIS
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Miruna Anica, Avinash Pittu, Illeana West, Olusegun Sobanjo, Ethan Hodge, Mia Engelbart, Srabasti Sengupta, Nagheme Thomas, C. V. von Roemeling, Maryam Rahman, Matthew R. Sarkisian, Jianping Huang, Jeffrey K Harrison, Duane A Mitchell, L. Deleyrolle
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Research on cell type markers aids investigators in exploring the diverse cellular compositions within gastrointestinal tumors, enhancing our understanding of tumor heterogeneity and its implications for disease progression and treatment response. However, issues such as the integration of large-scale datasets and the lack of standardized cell type identification hinder comprehensive characterization. Here, we developed a user-friendly web interface called PreDigs (Predicted Signatures in Digestive System), which offers 124 tailored scRNA-seq datasets available for download, encompassing over 3.4 million cells. After unsupervised clustering, we unified the identification and naming of subtype labels, ultimately constructing a cell ontology tree that includes 142 cell types, with up to eight hierarchical levels. Meanwhile, we calculated three different context-specific cell-type markers—’Cell Markers’, ‘Subtype Markers’, and ‘TPN Markers’—based on various application requirements within or across tissues. Through the integrated analysis of PreDigs gastrointestinal data, we identified distinct cell subpopulations exclusive to tumors, one of which corresponds to tumor-specific endothelial cells (TEC). Furthermore, PreDigs offers online cell annotation tools that empower users to perform single-cell classification with greater flexibility, accessible at https://www.biosino.org/predigs/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f066c350d98316ff9071f40e0805e670af0f667d" target='_blank'>
              PreDigs: a Database of Context-specific Cell-type Markers and Precise cell subtypes for Digestive Cell Annotation
              </a>
            </td>
          <td>
            Jiayue Meng, Mengyao Han, Yuwei Huang, Liang Li, Yuanhu Ju, Daqing Lv, Xiaoyi Chen, Liyun Yuan, Guoqing Zhang
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 The spatial organization of “immune hubs” in tumor microenvironments has gained recognition across histologies and tissues. Bone marrow offers a valuable platform for investigating immune hubs in the context of acute myeloid leukemia (AML) following hematopoietic stem cell transplant (HSCT). This setting may provide fresh insights into mechanisms of relapse and maintenance of remission after donor lymphocyte infusion (DLI). While it is generally understood that DLI exerts its effects through a graft-versus-leukemia (GVL) response, the specific cellular players and spatial organization driving GVL remain unidentified. Addressing this question can inform the rational design of cancer cellular therapies.
 A major challenge for spatial transcriptomic assays interrogating bone marrow samples is RNA degradation due to decalcification in addition to that caused by formalin fixation and paraffin embedding (FFPE). Furthermore, existing spatial platforms lack single-cell resolution and their throughput is generally limited to a small number of samples. To address these constraints, Singular Genomics is developing the G4XTM, an in-situ next-generation sequencing (NGS) platform that enables rapid sample throughput (up to 20-fold greater than existing platforms), and enables combined readouts of transcriptomics, proteomics, and fluorescent H&E staining in the same FFPE section.
 We used a pre-release version of the G4X to analyze bone marrow biopsies of patients with relapsed AML before and after treatment with DLI. We profiled 24 specimens (6 responder [R] pre-DLI, 7 R post-DLI, 5 non-responder [NR] pre-DLI, 6 NR post-DLI), aiming to elucidate the spatial relationships of immune cellular networks in the marrow microenvironment. Our targeted transcript panel included 153 immune- and marrow-related genes defining specific T cell subsets and other immune cell lineages, as well as 8 protein markers (CD3, CD4, CD8, CD45RA, HLA-DR, CD34, KI67, ATPase). We filtered out cells with transcript counts above the 95th percentile and retained cells with at least 10 transcripts and 3 unique genes. We excluded genes present in less than 5 cells and considered segmented nuclei sizes between 2.6 µm and 14.2 µm, resulting in 193,651 cells (median of 30 transcripts, 11 unique genes per cell). Further pre-processing consisted of batch correction via Harmony (Korsunsky 2019), dimensional reduction and clustering.
 Cell populations across 14 distinct clusters were annotated using gene signatures derived from previously generated single-cell RNA sequencing data (Maurer et al ASH 2023) and refined by considering differentially expressed marker genes and protein expression profiles. We previously demonstrated expansion of an activated CD8 T cell population with high expression of ZNF683 and GZMB uniquely in R post-DLI (Maurer et al ASH 2023). Hence, we interrogated the dynamics and spatial relationships of this specific T cell subset, identified among other lymphoid cells in the in situ data based on gene and protein signatures.
 In confirmation of our previous findings, we found that this T cell subset increased in proportion specifically in R post-DLI samples (3.6% to 6.4% in R pre to post; 2.9% to 1.9% in NR pre to post, p=ns) and exhibited decreased spatial self-association in NRs (8.79% to 2.17% in NR pre to post, 32.5 µm radius, p=0.008). In Rs, these CD8 T cells were more likely to be in the spatial vicinity of myelo-erythroid progenitor and AML cells, perhaps suggestive of active immune surveillance post-DLI (1.62% in NR post vs 6.32% in R post, 32.5 µm radius, p=0.008).
 When examining spatial organization of all cell types, bone marrow of NRs pre-DLI was marked by an overabundance of myeloid cells relative to other cell types. The microenvironment of Rs was more heterogeneous before treatment and showed highest immunologic diversity post-DLI (Shannon Diversity Index: 0.744 in NR post vs. 0.943 in R post, p=0.021). These findings suggest that diversity of marrow composition pre-DLI may be predictive of therapeutic response, though this observation requires validation in larger cohorts.
 We present the first analysis of post-HSCT relapsed AML marrow microenvironment with spatial multi-omic profiling using G4X and find evidence of post-DLI T cell expansion concordant with our scRNA-seq analysis. Further evaluation of spatial organization may lead to novel targets for optimizing adoptive cellular immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02f9e3e994f545d0f9f4bff84f2a37013298007" target='_blank'>
              Deconvolving Immunologic Networks of the AML Marrow Microenvironment with Spatial Multi-Omic Profiling
              </a>
            </td>
          <td>
            K. Maurer, Florian Raths, Kenneth Gouin, Cameron Y Park, Michael Lawson, Jake Koh, Martin M. Fabani, K. Livak, Jerome Ritz, R. Soiffer, Eli Glezer, Catherine J. Wu, Elham Azizi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is a highly aggressive brain cancer with limited treatment options. This research aims to improve our understanding of GBM progression by focusing on extracellular vesicles released by tumor cells that carry important biological information. The authors developed a laboratory model that closely mimics the tumor microenvironment, allowing for the study of extracellular vesicles released from various cell types within the tumor. This model enabled the identification of specific surface markers on extracellular vesicles that could serve as potential noninvasive diagnostic tools for GBM. The findings suggest that these markers may help improve extracellular vesicle isolation and provide insights into the molecular characteristics of the tumor. This study has the potential to significantly impact the research community by offering new approaches to GBM diagnosis and advancing our knowledge of tumor biology, which could ultimately inform future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ccc9a9ae6ff3dcd3e9189e563853f77db227f69" target='_blank'>
              Exploring Extracellular Vesicle Surface Protein Markers Produced by Glioblastoma Tumors: A Characterization Study Using In Vitro 3D Patient-Derived Cultures
              </a>
            </td>
          <td>
            S. Franceschi, F. Lessi, M. Morelli, M. Menicagli, P. Aretini, C. Gambacciani, F. Pieri, G. Grimod, Maria Grazia Trapanese, Silvia Valenti, F. Paiar, A. D. Di Stefano, O. Santonocito, F. Pasqualetti, C. M. Mazzanti
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3c3d2c93cdd85dd75bfc027eecd714525336" target='_blank'>
              Neuroblastoma plasticity during metastatic progression stems from the dynamics of an early sympathetic transcriptomic trajectory
              </a>
            </td>
          <td>
            Benjamin Villalard, A. Boltjes, F. Reynaud, Olivier Imbaud, Karine Thoinet, Ilse Timmerman, S. Croze, Emy Théoulle, Gianluigi Atzeni, Joël Lachuer, Jan J. Molenaar, G. Tytgat, C. Delloye-Bourgeois, Valérie Castellani
          </td>
          <td>2024-11-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="

 Introduction
 The tumor ecosystem in multiple myeloma (MM) involves complex interactions among various cell types, including clonal plasma cells (PCs) and immune cells. Previous research has indicated that the bone marrow (BM) microenvironment's metabolite and lipid profiles differ between MM patients and those with precursor conditions. This study aims to explore if differences in cell composition, metabolic phenotype of clonal PCs, and the spatial organization of various cell types are linked to the early versus late progression of high-risk smoldering MM (SMM) to MM.
 Methods
 We identified high-risk SMM patients using the IMWG 20-2-20 criteria, focusing on early progressors (EP), who progressed to MM within one year, and later progressors (LP), who progressed after more than four years. Using a high-dimensional single-cell spatial imaging platform, CODEX (CO-Detection by indEXing), we identified clonal PCs and immune cell types as well as their metabolic phenotypes at the single-cell level, in paired FFPE BM samples at SMM diagnosis and at the time of progression to MM. Our validated panel included markers such as CD3e, CD4, CD8, CD14, CD20, CD31, CD38, CD45, CD66a, CD68, CD138, HLA-DR, ASCT2, pNRF2, GLUT1, LDHA, ATPA5, CPT1A, and Citrate synthase. We employed single-cell segmentation, artifact cleanup, signal thresholding, and spatial analysis. Repeated measure models accommodated multiple regions of interest (ROIs) per patient and accounted for potential intrapatient correlation, while Poisson models estimated incidence rate ratios (RR) of cell types. Spatial statistical models evaluated cell type segregation and integration at the time of SMM diagnosis.
 Results
 Single cell events were acquired from 6 unique SMM patients who had paired FFPE BM samples from the time of their SMM diagnosis and at the time of their progression to MM. There were 3 EP SMM patients (Time to Progression (TTP):8, 8, 10 mos) and 3 LP SMM patients (TTP: 51, 58, 117 mos). The median number of single cell events identified per patient slide sample was 104,390 (Range: 24,357 - 314,877).
 We identified 8 main cell types: PCs, CD4+ T-cells, CD8+ T-cells, B-cells, granulocytes, Monocyte/Macrophages and two groups of other immune cells. The PCs and the remainder of the non-PCs were further classified based on the expression of the following metabolic markers: ASCT2 (Glutamine importer), CPT1A (Fatty acid metabolism), GLUT1 (Glucose importer), LDHA (Glycolysis), Citrate synthase and ATPA5 (Oxidative Phosphorylation). There was a higher RR of granulocytes (RR 2.1, CI 1.6-2.8, p<0.001) in EP compared to LP samples at the time of SMM diagnosis, but a lower RR of CD4 T cells (RR 0.40, CI 0.3-0.6, p< 0.001). Focusing on metabolic phenotypes of the PCs, we observed higher incidence RR of most metabolic markers: ASCT2 (RR=1.77; p=0.032), ATPA5 (RR=1.25; p<0.001), CPT1a (RR=2.42; p=0.015), and LDHA (RR 2.13; p<0.001) in EP compared to LP samples at the time of SMM diagnosis. However, GLUT1 had a lower incidence RR in EP patients (RR=0.79; p<0.001). The same modeling for non-PCs at the time of SMM diagnosis yielded higher expression of LDHA (RR=1.98; p<0.001) and ASCT2 (RR=3.3; p=0.002).
 Using the x and y coordinates and the corresponding phenotype classifications, we performed spatial statistical modeling based on spatial point processes to generate aggregated measures of how different types of cells of interest interacted. CD4 T cells were more isolated in LPs compared to EPs (p =0.046). Segregation indices for PCs and CD4 T cells indicated that they were more integrated/collocated in EPs. Granulocytes were more isolated in EPs although the difference was not as pronounced (p= 0.07). PCs were more integrated with granulocytes (p=0.06) and B cells (p=0.07) in LP patients. Further analyses of paired samples at the time of progression to MM will be reported at the time of the presentation.
 Conclusion
 This study is the first to provide whole-slide, single-cell, spatial proteomic characterization of clonal PCs and immune cells in the FFPE BMs of high-risk SMM patients with varying progression times to MM. Distinct metabolic pathway markers in clonal PCs and their spatial relationships with immune cell subsets are closely associated with the risk of progression to MM. These findings illuminate the complex interactions between the metabolic states of clonal PCs and their spatial orientation relative to immune cells in the pathogenesis of MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885f74d83e42026f5e0476cb04b6378a52adf535" target='_blank'>
              Single-Cell Spatial Proteomic Characterization of Clonal Plasma Cell and Immune Cell Metabolism within the Bone Marrow Microenvironment in High-Risk Smoldering Multiple Myeloma and Its Association with Progression to Multiple Myeloma
              </a>
            </td>
          <td>
            C. Soekojo, Susan M. Geyer, Emmanuel Contreras Guzman, Kaitlyn R. Whitaker, Shaji K Kumar, Vincent Rajkumar, Joselle M Cook, Yogesh Chawla, F. Buadi, A. Dispenzieri, M. Binder, Rafael Fonseca, Taro Hitosugi, D. Jevremovic, W. Gonsalves, J. V. Villasboas Bisneto
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="
 Diffuse invasion of glioblastoma cells through normal brain tissue is a key contributor to tumor aggressiveness, resistance to conventional therapies, and dismal prognosis in patients. A deeper understanding of how components of the tumor microenvironment (TME) contribute to overall tumor organization and to programs of invasion may reveal opportunities for improved therapeutic strategies. Towards this goal, we applied a novel computational workflow to a spatiotemporally profiled GBM xenograft cohort, leveraging the ability to distinguish human tumor from mouse TME to overcome previous limitations in analysis of diffuse invasion. Our analytic approach, based on unsupervised deconvolution, performs reference-free discovery of cell types and cell activities within the complete GBM ecosystem. We present a comprehensive catalogue of 15 tumor cell programs set within the spatiotemporal context of 90 mouse brain and TME cell types, cell activities, and anatomic structures. Distinct tumor programs related to invasion were aligned with routes of perivascular, white matter, and parenchymal invasion. Furthermore, sub-modules of genes serving as program network hubs were highly prognostic in GBM patients. The compendium of programs presented here provides a basis for rational targeting of tumor and/or TME components. We anticipate that our approach will facilitate an ecosystem-level understanding of immediate and long-term consequences of such perturbations, including identification of compensatory programs that will inform improved combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/678e476a090fe3a350f2e1d603e09efd449e924a" target='_blank'>
              ANGI-05. UNSUPERVISED DECONVOLUTION OF SPATIOTEMPORAL HETEROGENEITY REVEALS HIGH-RESOLUTION INVASION STATES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Varsha Thoppey Manoharan, Aly O. Abdelkareem, Samuel Brown, Aaron H. Gillmor, Courtney Hall, Heewon Seo, Kiran Narta, Sean Grewal, Ngoc Ha Dang, Bo Young Ahn, Kata Osz, Xueqing Lun, Douglas Mahoney, Donna L Senger, Jennifer A. Chan, Sorana A. Morrissy
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Gliomas are lethal malignancies comprised of heterogeneous and dynamic subpopulations of cell states resembling normal neurodevelopmental cell types (radial glia (RG), oligodendrocyte progenitor cell (OPC), neuron progenitor cell (NPC), neuron, etc). While both intrinsic (genetics) and extrinsic (brain environment) cues are coupled to glioma state identity, the functional programs governing glioma cell state and plasticity are unknown. Here we performed a multi-omic interrogation of a large library (n=392) of glioma patient tumors, in vivo orthotopic xenografts and in vitro gliomasphere cultures. Comparisons of matched glioma samples across environments revealed the non-native in vitro environment constrains glioma state diversity specifically enriching for stem-like glioma cell states (RG, immune, vascular). This enrichment was linked to lipid metabolic flexibility, enabling tumors enriched for stem-like states to adapt to various tumor microenvironments. By contrast, “lineage-committed” cell states (OPC, NPC, and neuron) demonstrate restricted lipid metabolism, consequently having a dependence on lipid scavenging from the brain microenvironment for survival. These results connect intra-tumoral heterogeneity and plasticity of glioma to lipid metabolism, leveraging the functional diversity of cellular states to reveal potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1598ae760fc108f7d962df3e2f211396a763ed04" target='_blank'>
              TMET-01. LIPID METABOLIC PROGRAMS DEFINE GLIOMA STATE IDENTITY, PLASTICITY AND DEPENDENCIES
              </a>
            </td>
          <td>
            Jenna K. Minami, Nicholas A. Bayley, C. Tse, Cassidy Andrasz, Dimitri Cadet, Jennifer Salinas, Weihong Yan, Amir Borujerdpur, L. Liau, T. Cloughesy, Kevin Williams, S. Bensinger, A. Bhaduri, Thomas Graeber, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e01bd46496bf7ca9aab2045ddff751c6da9e8f4" target='_blank'>
              Tumor microenvironment remodeling after neoadjuvant chemoradiotherapy in local advanced rectal cancer revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Wen-jing Su, Yuhang Ling, Xiaodong Yang, Yong Wu, Chungen Xing
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Gliobastoma (GBM) is a highly aggressive solid tumor of the brain, characterized by a hypoxic tumor microevironment (TME) leading to histopathological features of necrosis, microvascular proliferation and deregulated hypervascularization. The heterogeneous nature of GBM results in a gradient of intra-tumoral hypoxia that causes molecular changes to specific cell populations within the bulk of the tumor. Our study is the first clinical study to utilize the exogenous oxygen-independent marker pimonidazole (PIMO) to identify the hypoxic TME using high-dimensional spatial and single-cell proteomics of specific cell populations within GBM. A cohort of 35 patients with primary GBM, recurrent GBM or IDH-mutant glioma were administered PIMO prior to surgical resection of tumor tissue for downstream bulk proteomic analysis of bulk tissue by serial immunohistochemistry and imaging mass cytometry (IMC) and single-cell analysis of dissociated tissue by Cytometry by time-of-flight (CyTOF). We utilized high-resolution imaging analyses to validate PIMO as a sensitive and stable marker for hypoxia against a panel of transiently expressed HIF-target genes and examine the inter- and intra-tumoral heterogeneity of hypoxia within each tumor subtype. A custom CyTOF marker panel was developed for further single-cell analyses of T-cell subsets and their functional states, myeloid cell subpopulations, natural killer cells, glial cells, and tumor cells in the hypoxic TME. We demonstrate the utility of PIMO to effectively study the hypoxic TME through spatial and single cell methods and further elucidate mechanisms of hypoxia-driven treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa287ddd81b049b43e5a448648f6d6d942969be3" target='_blank'>
              TMIC-80. HIGH-DIMENSIONAL HISTOPATHOLOGIC EVALUATION OF THE HYPOXIC TUMOR MICROENVIRONMENT IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Shreya Gandhi, O. Singh, S. Mansouri, S. Karimi, Phoebe Lombard, Anna Dvorkin-gheva, Carlos Velasquez, Hafsah Ali, Kaviya Devaraja, Qingxia Wei, Ronald Wu, Mira Li, May Cheung, M. Voisin, A. Gao, David Hedley, Kenneth J Aldape, Bradley Wouters, G. Zadeh
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e6a9e6b43769b80db465f9dd9f678e6c40ef5c" target='_blank'>
              Single cell and spatial transcriptomics highlight the interaction of club-like cells with immunosuppressive myeloid cells in prostate cancer
              </a>
            </td>
          <td>
            A. Kiviaho, Sini K. Eerola, H. Kallio, M. K. Andersen, Miina Hoikka, A. Tiihonen, I. Salonen, Xander Spotbeen, A. Giesen, C. T. Parker, S. Taavitsainen, Olli Hantula, Mikael Marttinen, Ismaïl Hermelo, Mazlina Ismail, Elise Midtbust, M. Wess, W. Devlies, Abhibhav Sharma, S. Krossa, Tomi Häkkinen, Ebrahim Afyounian, K. Vandereyken, Sam Kint, J. Kesseli, T. Tolonen, T.L.J. Tammela, T. Viset, Øystein Størkersen, G. Giskeødegård, M. B. Rye, T. Murtola, A. Erickson, Leena Latonen, G. Bova, Ian G. Mills, S. Joniau, J. Swinnen, Thierry Voet, T. Mirtti, Gerhardt Attard, Frank Claessens, T. Visakorpi, K. Rautajoki, May-Britt Tessem, Alfonso Urbanucci, M. Nykter
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Cell tracking in chimeric models is essential yet challenging, particularly in developmental biology, regenerative medicine, and transplantation studies. Existing methods, such as fluorescent labeling and genetic barcoding, are technically demanding, costly, and often impractical for dynamic, heterogeneous tissues. To address these limitations, we propose a computational framework that leverages sex as a surrogate marker for cell tracking. Our approach uses a machine learning model trained on single-cell transcriptomic data to predict cell sex with high accuracy, enabling clear distinction between donor (male) and recipient (female) cells in sex-mismatched chimeric models. The model identifies specific genes critical for sex prediction and has been validated using public datasets and experimental flow sorting, confirming the biological relevance of the identified cell populations. Applied to skeletal muscle macrophages, our method revealed distinct transcriptional profiles associated with cellular origins. This pipeline offers a robust, cost-effective solution for cell tracking in chimeric models, advancing research in regenerative medicine and immunology by providing precise insights into cellular origins and therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2607809a828288d92a6fadd5cfee754b3bcdb55" target='_blank'>
              CellSexID: Sex-Based Computational Tracking of Cellular Origins in Chimeric Models
              </a>
            </td>
          <td>
            Huilin Tai, Qian Li, Jingtao Wang, Jiahui Tan, Ryann Lang, Basil J. Petrof, Jun Ding
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Background:
 Despite significant advances in single-cell and bulk transcriptome profiling of classic and transformed follicular lymphomas (FL), their tumor microenvironments remain insufficiently understood, limiting the discovery of treatment strategies that target local survival niches. The emerging field of spatial transcriptomics (ST) aims to elucidate the gene-level microarchitecture of lesional biopsies. However, most current spatial methods fail to effectively resolve the high tissue cellularity characteristic of indolent lymphomas while simultaneously maintaining full transcriptome coverage. We optimized tissue preparation and bioinformatic workflows to enhance FL-specific spatial transcriptomics.
 Methods:
 We first evaluated conditions for whole transcriptome recovery using the 10x Visium ST platform with 50 µm sized spots and optimized permeabilization time (40 mins) and section thickness (10 µm). We next applied these optimizations to profile frozen excisional lymph node biopsies from representative patients with Classic and Transformed FL, as well as non-malignant reactive lymphoid tissue. We developed custom FL-specific cell signature matrices by curating and reannotating publicly available 10x single-cell RNA-seq datasets from FL, then extracted single-cell resolution data utilizing the novel CytoSPACE algorithm (Vahid et al, Nat Biotech 2023). In addition to spatial and cell subset specific gene expression, we evaluated spatial copy number variation (CNV) inference using inferCNV and Numbat.
 Results:
 We recovered a median of 10.7k UMI counts per 50 µm spot across samples, allowing evaluation of a median 3,716 unique genes. Single cell mapping with CytoSPACE revealed 35±10 cells per spot totaling to 77±20k cells per sample. In reactive tissue the single cells maps show a classic patterns of polarized follicles and interfollicular T-cell zones. Interestingly, within the interfollicular zones there were immune hotspots of T-regulatory (20±4%), T follicular helper (TFH, 18±6%) and naïve CD8+ T cells (11±5%) situated adjacent to the follicle dark zones. In classic FL tissues we observed enlarged dysmorphic follicles with diminished polarization and interfollicular spaces densely populated with myeloid cells. However, similar appearing but smaller sized immune hotspots were preserved, with T-regulatory cells as the most prevalent subset. Compared to immune hotspots in reactive lymphoid tissue we found lower numbers of TFH cells (p-adj<0.01) and increased non-cytotoxic CD8 T-cells (p-adj=0.02) and NK cells (p-adj<0.05). Interestingly in both classical and transformed FL these hotspots were highly infiltrated by macrophages, activated dendritic cells (ADC), and plasma cells; while these cell types were virtually absent in reactive tissue immune hotspots. Spatial CNV profiling resolved a tumor-specific somatic aneuploidy fingerprint of FL cells for all profiled cases, with per-spot CNV signals highly correlated with malignant B-cell counts as defined by CytoSPACE.
 Conclusion:
 Our study demonstrates the efficacy of Visium ST combined with CytoSPACE single-cell mapping in elucidating the complex microarchitecture of FL tumors. The single cell maps faithfully reproduced the cellular architecture as confirmed by classical histology. We detected different patterns of spatially localized immune hotspots between reactive, classic- and transformed FL tissues. These immune hotspots were marked by a local abundance of T-regulatory cells in both reactive and FL tissues, but enriched in malignant tissue for effector subsets including ADCs, CD8+ T, and NK cells. Modulating immune hotspots may represent a therapeutic opportunity. Spatial tumor fingerprinting using CNV inference enabled high-resolution identification of malignant cells in all patients. Our approach offers a comprehensive view of FL biology within the tumor microenvironment, and additional analyses including spatial T- and B-cell receptor repertoire will be presented.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29991ccc2c9d0cb998eb56b0bbc2b059b4020eeb" target='_blank'>
              Resolving the Microarchitecture of Classic and Transformed Follicular Lymphoma By Single Cell Alignment to Spatial Transcriptomes
              </a>
            </td>
          <td>
            Troy Noordenbos, J. Schroers-Martin, M. Hamilton, Soyeong Jun, Takeshi Sugio, B. Sworder, C. B. Steen, Mari Olsen, C. Liu, Aaron Newman, Brooke Howitt, Y. Natkunam, David B. Miklos, Maximilian Diehn, M. Frank, A. Alizadeh
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Recent advancements in probe-based, full-transcriptome, high-resolution technologies for Formalin-Fixed Paraffin-Embedded (FFPE) tissues, such as Visium CytAssist, Chromium Flex (10X Genomics), and GeoMx DSP (Nanostring), have opened new opportunities for studying decades-old archival samples in biobanks, facilitating the generation of data from extensive cohorts. However, the experimental protocols can be labor-intensive and costly; therefore, it is thus essential for researchers to carefully evaluate the strengths and limitations of each technology in relation to their specific research objectives. Here, we report the results of a comparative analysis of the three methods mentioned above on FFPE archival tumor samples from four non-small cell lung cancer, four breast cancer and six diffuse large B-cell lymphoma. We highlight some relative advantages and disadvantages of each method in the context of operational challenges, bioinformatic analysis and biological discovery. Our results show that: 1) all three methods yielded good-quality, highly reproducible transcriptomic data from serial sections of the same FFPE block; 2) GeoMx data contained mixtures of cell types, even when pre-selecting areas with cell type-specific markers; 3) high-throughput spot-level (Visium) or cell-level (Chromium) data enabled the identification of tumor heterogeneity within and between patients, which could be used to identify targeted therapies. Our data support the use of Visium and Chromium for high-throughput and discovery-driven projects, while the GeoMx platform could be suited for addressing specialized questions on targeted regions. All data generated from this study, including GeoMx, Visium, Chromium, H&E, and expert annotations are publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9f0cd1eca3f861ace59c3be80ff4420ed5c2b7b" target='_blank'>
              Transcriptome Analysis of Archived Tumor Tissues by Visium, GeoMx DSP, and Chromium Methods Reveals Inter- and Intra-Patient Heterogeneity
              </a>
            </td>
          <td>
            Yixing Dong, Chiara Saglietti, Quentin Bayard, Almudena Espin Perez, Sabrina Carpentier, Daria Buszta, S. Tissot, Rémy Dubois, Atanas Kamburov, Senbai Kang, Carla Haignere, R. Sarkis, Sylvie Andre, Marina Alexandre Gaveta, Silvia Lopez Lastra, Nathalie Piazzon, Rita Santos, Katharina Von Loga, Caroline Hoffmann, George Coukos, Solange Peters, V. Soumelis, Eric Y. Durand, L. D. de Leval, R. Gottardo, K. Homicsko, E. Madissoon
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Glioblastoma, the most aggressive form of primary brain cancer, is driven by both intrinsic cellular properties and extrinsic factors from the tumor microenvironment (TME). Here, we leverage our novel human organoid tumor transplantation (HOTT) system to explore how external cues modulate glioblastoma cell type specification, heterogeneity, and migration. Comparing the single-cell transcriptomes of HOTT and matched primary tumors, we found that HOTT recapitulates not only core features of major patient tumor cell types but also those of peritumor non-neoplastic cell types. This enables us to explore tumor-TME interactions and identify PTPRZ1, a receptor tyrosine phosphatase implicated in brain tumor migration, as a key mediator in intercellular communication. This finding was further confirmed by interrogation of published datasets from patients containing non-neoplastic microenvironmental cells. PTPRZ1 activation, resulting from the reactivation of developmental programs in both the tumor and its TME, highlights the relevance of the organoid system for modeling the GBM microenvironment. Moreover, HOTT uniquely allows for perturbations in either patient tumors or their microenvironments, or both, to verify PTPRZ1 function. Tumor knockdown of PTPRZ1 confirmed its role in promoting tumor migration and maintaining progenitor identity. Surprisingly, environmental PTPRZ1 knockdown in human cortical organoids before tumor transplantation inhibited tumor migration and promoted differentiation, even without direct tumor manipulation. Overall, this fundamental result demonstrates that a single treatment delivered to the tumor and its TME can exert opposite effects on the tumor. Consequently, it underscores the necessity of studying human glioblastoma within a human microenvironment context, such as HOTT, especially when evaluating therapeutic efficacy in GBM, as effective treatments must address both the tumor and its complex TME, rather than the tumor alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9ffba54b9658bfd1d4ecc52cd105a03aa1e10dc" target='_blank'>
              TMIC-69. HUMAN ORGANOID TUMOR TRANSPLANTATION IDENTIFIES FUNCTIONAL GLIOBLASTOMA MICROENVIRONMENTAL COMMUNICATION MEDIATED BY PTPRZ1
              </a>
            </td>
          <td>
            Weihong Ge, Ryan L. Kan, Can Yilgor, Elisa Fazzari, Patricia R. Nano, Daria J. Azizad, Matthew Li, Joyce Ito, C. Tse, Andrew Tum, Jessica Scholes, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3304563df16fc68d3eeffdd08dc060638480f4" target='_blank'>
              Single-cell transcriptomic landscape deciphers olfactory neuroblastoma subtypes and intra-tumoral heterogeneity.
              </a>
            </td>
          <td>
            Jingyi Yang, Xiaole Song, Huankang Zhang, Quan Liu, Ruoyan Wei, Luo Guo, Cuncun Yuan, Fu Chen, K. Xue, Yuting Lai, Li Wang, Junfeng Shi, Chengle Zhou, Juan Wang, Yingxuan Yu, Qibing Mei, Li Hu, Huan Wang, Chen Zhang, Qianqian Zhang, Houyong Li, Ye Gu, Weidong Zhao, Huapeng Yu, Jingjing Wang, Zhuo-fu Liu, Han Li, S. Zheng, Juan Liu, Lu Yang, Wanpeng Li, Rui Xu, Jiani Chen, Yumin Zhou, Xian-kui Cheng, Yiqun Yu, Dehui Wang, Xicai Sun, Hongmeng Yu
          </td>
          <td>2024-11-14</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8ac06c638611539d33f6ef5fdb316eabe9f02c" target='_blank'>
              Spatio-temporal transcriptomics of chromothriptic SHH-medulloblastoma identifies multiple genetic clones that resist treatment and drive relapse
              </a>
            </td>
          <td>
            Ilia Kats, Milena Simovic-Lorenz, Hannah Sophia Schreiber, Pooja Sant, Jan-Philipp Mallm, Verena Körber, Albert Li, Pravin Velmurugan, Sophie Heuer, Luisa Kües, Frauke Devens, M. Sill, M. Jugold, Mahmoud Moustafa, Amir Abdollahi, Frank Winkler, A. Korshunov, Stefan M Pfister, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220a065de9e40d9837ef9539b8c30437e0e009d" target='_blank'>
              Advanced single-cell and spatial analysis with high-multiplex characterization of circulating tumor cells and tumor tissue in prostate cancer: Unveiling resistance mechanisms with the CoDuCo in situ assay
              </a>
            </td>
          <td>
            Lilli Bonstingl, Margret Zinnegger, Katja Sallinger, Karin Pankratz, Christin-Therese Müller, Elisabeth Pritz, Corinna Odar, C. Skofler, Christine Ulz, Lisa Oberauner-Wappis, Anatol Borrás-Cherrier, Višnja Somođi, Ellen Heitzer, T. Kroneis, T. Bauernhofer, Amin El-Heliebi
          </td>
          <td>2024-11-16</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Therapy resistance in glioblastoma is in part attributed to intratumoral heterogeneity and treatment-resistant cell populations. Understanding heterogeneity requires more complete tumor sampling at diagnosis and at recurrence. We used a whole-tumor sampling approach to obtain 43 spatially mapped biopsies from 3 glioblastomas at diagnosis and recurrence, Pyclone and ClonEvol imputed clonal evolution from exome data, and weighted gene co-expression network analysis inferred gene expression programs from RNA sequencing. Across the cohort, tumor-wide clonal alterations representing evolutionarily early expansions included canonical changes (i.e. Chr7 gain, EGFR amplification) and a diverse set of copy number variations (Chr19/20 gain), driver mutations (i.e. PTEN, KDR), and fusions (LIMCH1::UCHL1, KANK::DOCK8). In 2/3 patients, the founding clone was the only subclone common to primary and recurrence samples and 37% of cancer drivers across the cohort appeared after evolutionary divergence of the primary and recurrent tumors. We identified thirty-two transcriptional programs, six of which were differentially expressed between timepoints. Programs enriched for mitochondrial activity, endothelium/pericytes and classical glioblastoma subtype signature were more highly expressed in primary samples. Programs enriched for mesenchymal-like state, oligodendrocytes and inhibitory interneurons were more highly expressed in recurrence samples. At diagnosis, 11/16 samples showed higher expression of the classical subtype program compared to mesenchymal-like, but at recurrence the mesenchymal program dominated in 18/18 samples. Expression of the programs correlated more strongly with position relative to the contrast-enhancing rather than T2 tumor centroid (p<0.001). These findings reveal novel diversity and complexity in the genetic roots of individual glioblastoma. Recurrences arose from subclones diverging early in evolution of the primary tumor and contained clonal drivers not detected in the primary. Expression data revealed transition from an intratumorally heterogeneous mixture of classical and mesenchymal signatures to tumor-wide mesenchymal at recurrence and suggests that the contrast enhancing centroid may serve as the biological center of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa1cf111e47e18a468bfde6175297f4bb325c80" target='_blank'>
              EPCO-37. LONGITUDINAL TUMOR-WIDE SAMPLING OF GLIOBLASTOMA REVEALS EARLY EVOLUTIONARY DIVERGENCE OF RECURRENCE AND CLUES TOWARDS IDENTIFYING THE BIOLOGICAL TUMOR CENTER
              </a>
            </td>
          <td>
            Benjamin Lerman, Chibo Hong, Patrick Schupp, Lee Chen, Daniel Zamler, Ivan V. Smirnov, Joanna Phillips, Anny Shai, Janine Lupo, Michael Oldham, Joseph F Costello
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly disease. Over recent decades, a better understanding of the biological mechanisms implicated in its pathogenesis has led to the development of targeted therapies and immunotherapy, resulting in improvements in patient outcomes. To enhance our understanding of lung cancer tumor biology and advance towards precision oncology, we need a comprehensive tumor profile. In recent years, novel in situ spatial multiomics approaches have emerged, offering a more detailed view of the spatial location of tumor and tumor microenvironment cells. These advancements in in situ profiling may unveil further molecular and immune mechanisms in tumor biology that will lead to the discovery of biomarkers for treatment prediction and prognosis. In this review, we provide an overview of current and emerging pathology-based approaches for spatial immune profiling in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e58f728c302c9eea20cd5a44be684287b92352" target='_blank'>
              Novel Spatial Approaches to Dissect the Lung Cancer Immune Microenvironment
              </a>
            </td>
          <td>
            Idania Lubo, Sharia D Hernandez, I. Wistuba, L. S. Solis Soto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology.Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors.Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls.These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Tumor microenvironments (TMEs) encompass complex ecosystems of cancer cells, infiltrating immune cells, and diverse cell types. Intercellular and intracellular signals within the TME significantly influence cancer progression and therapeutic outcomes. Although computational tools are available to study TME interactions, explicitly modeling tumor progression across different cancer types remains a challenge. Methods This study introduces a comprehensive framework utilizing single-cell RNA sequencing (scRNA-seq) data within a multilayer network model, designed to investigate molecular changes across glioma progression stages. The heterogeneous, multilayered network model replicates the hierarchical structure of biological systems, from genetic building blocks to cellular functions and phenotypic manifestations. Results Applying this framework to glioma scRNA-seq data allowed complex network analysis of different cancer stages, revealing significant ligand‒receptor interactions and key ligand‒receptor-transcription factor (TF) axes, along with their associated biological pathways. Differential network analysis between grade III and grade IV glioma highlighted the most critical nodes and edges involved in interaction rewiring. Pathway enrichment analysis identified four essential genes—PDGFA (ligand), PDGFRA (receptor), CREB1 (TF), and PLAT (target gene)—involved in the Receptor Tyrosine Kinases (RTK) signaling pathway, which plays a pivotal role in glioma progression from grade III to grade IV. Discussion These genes emerged as significant features for machine learning in predicting glioma progression stages, achieving 87% accuracy and 93% AUC in a 3-year survival prediction through Kaplan-Meier analysis. This framework provides deeper insights into the cellular machinery of glioma, revealing key molecular relationships that may inform prognosis and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d112e50918401623433091b937b2d39ad5c4da" target='_blank'>
              Integrative analysis of single-cell transcriptomic and multilayer signaling networks in glioma reveal tumor progression stage
              </a>
            </td>
          <td>
            Fereshteh Fallah Atanaki, L. Mirsadeghi, Mohsen Riahi Manesh, K. Kavousi
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a highly malignant, intrinsically resistant and inevitably recurring brain tumor with dismal prognosis. The aggressiveness and lack of effective GBM treatments can be attributed to the highly heterogeneous and plastic nature of GBM tumor cells, which easily confer resistance to standard-of-care (SOC) therapy. While tumor progression has also been attributed to interactions with the tumor microenvironment, quantitative data describing these interactions are still largely missing. Here, we used an multiome approach combining high-dimensional, multiplexed immunohistochemistry and spatial transcriptomics to map evolutions in the spatial, single-cell tissue architecture of 357 paired adult GBM tumor samples derived from 52 patients at diagnosis (ND) and upon recurrence (REC) following SOC treatment. We mapped the spatial distribution of a multitude of GBM tumoral subtypes across this multicentric cohort, through which we identified a high level of heterogeneity defined by specific tumoral niches within and across patients and which evolved when subjected to SOC therapy. In addition, we describe the relationship of the various tumoral niches with their local immune-infiltrates, highlighting an even more immunosuppressive environment following SOC resistance. Finally, by aligning these findings to the observed genomic aberrations and the clinical data of the patients, we are now able to more precisely describe the heterogeneous landscape of glioblastoma and how it evolves under SOC treatment at spatial, single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27f408cb60e0b4c4a336bb204f39691c84c2de06" target='_blank'>
              TMIC-55. CHARACTERIZATION OF THERAPEUTICS EFFECTS IN GLIOBLASTOMA THROUGH INTEGRATED SPATIAL SINGLE-CELL MULTIOME PROFILING OF LONGITUDINAL SAMPLES: UNVEILING CLINICALLY RELEVANT SUBTYPES OF PATHOLOGICAL CHANGES
              </a>
            </td>
          <td>
            M. Vanmechelen, Nazari Pouya, J. Beckervordersandforth, D. Leunissen, B. Cole, B. Decraene, Y. De Visser, Gautam Shankar, M. Verduin, Carmen Bravo, Tom Moors, Jayesh Telang, J. Messiaen, Danielle Eekers, A. Claeys, M. Derweduwe, A. Zurhausen, Francesca M Bosisio, Frank Weyns, T. Daenekindt, Peter Van Eyken, Mieke Govers, Ferdinand Mennens, K. Hovinga, S. De Vleeschouwer, Clement Paul, M. Broen, Marc Vooijs, R. Sciot, A. Antoranz, Jon Pey, Ernst Jan Speel, A. Hoeben, F. De Smet
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="

 Multiple Myeloma (MM) is a genomically heterogeneous cancer of malignant plasma cells (PC) residing in the bone marrow. Despite recent therapeutic advances, it is incompletely understood why some patient populations remain largely therapy-refractory or relapse, even with targeted immunotherapies. Whilst MM classification based on genomic features and gene expression profiling (GEP) is well-described, the role of spatial interactions between tumour cells and the bone marrow tumour microenvironment (TME) in dictating treatment responses is insufficiently defined. Spatial heterogeneity in MM was highlighted by a multi-region whole exome sequencing (WES) study of paired samples from random bone marrow aspirates and distant lesions (Rasche et al Nature Comm 2017, 2022; doi.org/10.1038/s41467-017-00296-y; 10.1038/s41467-022-32145-y), however, it is unknown whether spatially and genomically distant clones interact differently within their immune and stromal compartments. Recent pioneering work, using multiplexed imaging and in situ transcriptomics approaches on MM trephines (Robinson et al, JCI 2023; doi.org/10.1172/JCI167629) demonstrated patterns and mechanisms of tumour-immune cell interactions suggesting TME interactions as a critical therapeutic response determinant.
 Isolated in situ ‘omics’ assays provide limited information due to biased selection of marker panels or discordance between transcriptional and proteomic profiles. We hypothesize that a combined, systematic approach to integrate multiple layers of information (multiplexed imaging, unbiased LC/MS proteomics, transcriptomics, genomics) within their spatial context would significantly advance our understanding of MM TME phenotypes and responses to therapy.
 Consecutive FFPE sections (plasmacytoma or MM; 4 cases) were subjected to spatial transcriptomics (Xenium, 10X Genomics) combined with multiplexed immunofluorescence (IF) or imaging mass cytometry (IMC), laser-capture guided in situ WGS (Rao et al, Genome Med. 2024 doi: 10.1186/s13073-024-01302-x) and LC/MS based spatial proteomic workflows (Davis et al, Nat Commun. 2023 doi: 10.1038/s41467-023-43520-8). A data analysis pipeline utilising machine-learning was implemented. We have developed a multiomics workflow that (1) maps critical regions of interest (ROI) in the TME (using spatial transcriptomics, IHC, IF) and that provides (2) ROI-specific genomic information from WES and/or targeted sequencing, and (3) unbiased proteomic information (LC/MS).
 We mapped tumour, immune and stromal cell types and activation states in the bone marrow environment by using low-plex (30-40 markers) IF or IMC, and a customised probe panel for transcript detection in order to define ROIs. This was followed by laser capture microscopy of defined ROIs to genomically characterise the tumour, with ROI-matched LC/MS proteomics enabled detection of 4000-5000 proteins (current resolution of 200-500 cells per ROI).
 In this proof of concept study with limited sample numbers we identified differences in tumour MHC expression as well as novel NK-cell-myeloma spatial interactions (such as HLA-G/LILRB1) as factors that may control myeloma-immune environments and responses. Shallow WGS of excised ROI sections allows detection of CNA-defined clonal heterogeneity; we are now developing this promising approach to a targeted panel of genomic aberrations in MM. Our next translational step is the deployment of this workflow to longitudinal clinical trial trephine samples with correlation to single cell data from bone marrow aspirates. Taken together, application of this translational platform will provide new insights into MM pathobiology and therapeutic response and resistance mechanisms.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc52b4c61e537e710ecf579d97f6980a75244b" target='_blank'>
              Development of a Spatial Multiomics Platform to Integrate Genomic, Transcriptomic and Proteomic Features for Translational Research in Multiple Myeloma
              </a>
            </td>
          <td>
            S. Gooding, Chen-Yi Wang, Warren Baker, Edmund Watson, Simon Davis, Roman Fischer, Oliver McCallion, J. Hester, Fadi Issa, Srinivasa Rao, Nick Athanasou, A. Thakurta, K. Ramasamy, C. Bjorklund, M. Amatangelo, Anita K. Gandhi, A. Cribbs, A. Laganà, Udo Oppermann
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="

 Introduction
 Acute myeloid leukemia (AML) resides in an immune rich microenvironment. A comprehensive understanding of leukemic cells and their interactions within the leukemic bone marrow microenvironment is needed to better understand disease biology. Recent high-plex proteomics technologies have been developed to interrogate the tumor immune microenvironment, however, these systems suffer from several challenges when being applied in the context of leukemic bone marrow tissue. To address this, we optimized a high throughput bone marrow tissue-microarray based pipeline for high-plex spatial proteomics imaging in bone marrow that reliably captures the immune, tumor, and structural components of the bone marrow microenvironment at the single cell level resolution using both immunofluorescence (IF) and imaging mass cytometry (IMC) on a single slide.
 Methods
 As a proof-of-concept, we organized 16 bone marrow biopsies from 7 adult AML patients into 2 tissue microarrays using 1.5 mm diameter cores with a 4 µm section thickness. Each TMA was then imaged using the COMET multiplex IF platform with a 28-plex panel covering canonical markers for immune cells (n=4), AML cells (n=12), and functional markers (n=12). Using the same slide, we then performed IMC with a 25-plex panel containing 16 overlapping markers to assess concordance between methodologies for a total of 34 unique proteins imaged. Image alignment was guided by tissue structure and co-expression of shared markers onto a reference H&E-stained serial section, which provided structural components such as trabecular bone. Cells were segmented using the convolutional neural network U-Net and manually phenotyped by mean intensity thresholds. Unbiased clustering over each cell's spatial K function was performed to identify 5 distinct regions of cellular organization. We then validated the results of our spatial workflow by staining adjacent sections using the Opal 7-color multiplex IF protocol. Gene set enrichment was performed on an integrated AML transcriptomic dataset consisting of patients from TCGA, MDACC, and BEAT-AML studies (n=480) using a published TLS signature (Cabrita et. al. Nature 2020) to assess prognostic ability.
 Results
 Clustering over the spatial K function revealed 5 regions enriched in AML cells, monocyte/macrophage lineage cells, lymphocytes, unlabeled cells, and one region that contained a relatively even mix of all cell types. Interestingly, we observed the lymphocyte enriched region to only appear in 3 of the 16 tumor cores, taking the form of a dense aggregate of B and T cells, similar in structure and organization to tertiary lymphoid structures (TLS) in solid cancers. B cells in these aggregates displayed increased expression of Ki-67, suggesting proliferating B cells analogous to germinal centers. Furthermore, in tumor cores where these aggregates appeared, we found significant enrichment of monocyte and macrophage lineage cells (p=0.0164), as well as increased CD4 (p=0.017) and CD8 (p=0.041) T cell infiltration into areas of high AML cell density. Finally, we were able to validate the presence of these TLS-like aggregates by staining adjacent sections of the two TMAs using the Opal 7-color multiplex IF protocol.
 We sought to further investigate the presence of TLS-like aggregates in AML by querying large public datasets with a previously validated TLS signature (Cabrita et. al. Nature 2020). Interestingly, the TLS signature strongly correlated to hypoxia, inflammation, and angiogenesis hallmark gene sets, aligning with the propensity of TLSs to form in inflammatory conditions and promote vessel growth in solid cancers. Furthermore, when dichotomizing patients by median TLS score, we found that the signature significantly correlated with overall survival (p=0.016).
 Conclusions
 We find that by utilizing the strengths of both high-plex IF and IMC in our integrative sample-to-analysis workflow, we can reliably spatially characterize the immune, tumor, and structural elements in the bone marrow microenvironment, allowing us to uncover novel cellular structures such as TLS-like aggregates and characterize their effect on the leukemic bone marrow microenvironment. Further, our study's methodologies can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d31ded7167deed36ebdddf6684d5aabc8c3fcf" target='_blank'>
              Multimodal Spatial Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher P. Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, S. Ferri-Borgogno, Akshay V. Basi, Javier A Gomez, Jessica L Root, P. Reville, Andres E. Quesada, C. Bueso-Ramos, Miriam B. Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="
 Glioblastoma (GB) is an inherently heterogeneous and invasive primary brain cancer. Genomic and transcriptomic studies have attempted to classify GB into subtypes that predict survival and that have different therapeutic vulnerabilities. An outstanding question and technical challenge is to visualize the metabolism of a tumour within its native microenvironment and understand to what extent its metabolism is driven by the microenvironment. Patients with GB tumours were infused with [U-13C]glucose and intra-operative multi-regional tumour samples were rapidly snap frozen for analysis using mass spectrometry imaging (MSI). Tissue microenvironment was assessed using multiplexed antibodies in immunocytochemistry (IMC) of contiguous sections. Isotope tracking in human-derived neurospheres and patient-derived xenografts were used to confirm the presence of metabolic phenotypes detected in the human samples and to test the susceptibility of each phenotype to a panel of drugs targeting cellular metabolism. Three metabolic signatures were identified in patient tumours: glycolytic, oxidative and a mixed glycolytic/oxidative phenotype. The phenotypes did not correlate with microenvironmental features, including proliferation rate, immune cell infiltration, and vascularisation. The phenotypes were retained when patient-derived cells were grown in vitro, or as orthotopically implanted xenografts, and were robust to changes in oxygen concentration, demonstrating their cell intrinsic nature. The spatial extent of the regions occupied by cells displaying these distinct metabolic phenotypes are large enough to be detected using clinically applicable metabolic imaging techniques. The phenotypes also demonstrate differential drug sensitivities, suggesting imaging of these phenotypes may be used to stratify patients for personalised therapy trials. This is the first report to demonstrate high resolution imaging of metabolic fluxes in a human tumour in vivo and the presence of different metabolic phenotypes within GB that are tumour cell intrinsic and largely independent of the tumour microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe33b9e0530156643ddb88c0d7e05b51dba89c2" target='_blank'>
              TMET-02. CELL-INTRINSIC METABOLIC PHENOTYPES IDENTIFIED IN GLIOBLASTOMA PATIENTS USING MASS SPECTROMETRY IMAGING OF 13C-LABELED GLUCOSE METABOLISM
              </a>
            </td>
          <td>
            Anastasia Tsyben, Andreas Dannhorn, Gregory Hamm, D. Couturier, Mayen Briggs, R. Mair, K. Brindle
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The dynamic interplay between tumor cells and γδT cells within the tumor microenvironment (TME) significantly influences disease progression and immunotherapy outcome. Here, we delved into the modulation of γδT-cell activation by tumor cell ligands CD112 and CD155, which interact with the activating receptor DNAM-1 on γδT cells. Spatial and single cell RNA sequencing (scRNA-seq), as well as spatial metabolome analysis, from neuroblastoma (NB) revealed that the expression levels and localization of CD112 and CD155 varied across and within tumors, correlating with differentiation status, metabolic pathways, and ultimately disease prognosis and patient survival. Both in vivo tumor xenograft experiments and in vitro co-culture experiments demonstrated that a high CD112/CD155 expression ratio in tumors enhanced γδT-cell-mediated cytotoxicity, while a low-ratio fostered tumor resistance. Mechanistically, CD112 sustained DNAM-1-mediated γδT-cell activation, whereas CD155 downregulated DNAM-1 expression via TRIM21-mediated ubiquitin proteasomal degradation. By interacting with tumor cells differentially expressing CD112 and CD155, intratumoral γδT cells exhibited varying degrees of activation and DNAM-1 expression, representing three major functional subsets. This study underscores the complexity of tumor-immune crosstalk, offering insights into how tumor heterogeneity shapes the immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0f304618ec33893cbfa169baffb49b70570809" target='_blank'>
              Spatial and Single Cell Analyses Reveal Heterogeneity of DNAM-1 Receptor-Ligand Interactions that Instructs Intratumoral γδT-Cell Activity.
              </a>
            </td>
          <td>
            Xiaolin Wang, Hui Wang, Zhengjing Lu, Xiangjun Liu, Wenjia Chai, Wei Wang, Jun Feng, Shen Yang, Wei Yang, Haiyan Cheng, Chenghao Chen, Shihan Zhang, N. Sun, Qiaoyin Liu, Qiliang Li, Wenqi Song, Fang Jin, Qi Zeng, Shengcai Wang, Yan Su, Huanmin Wang, Xinxin Ni, Jingang Gui
          </td>
          <td>2024-11-08</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Background & Objective: Prostate cancer (PCa) cells frequently disseminate to bone where they can remain dormant for extended periods. The precise cellular and molecular mechanisms responsible remain largely unknown. The advent of spatial transcriptomics provides a unique opportunity to study key bone microenvironment-dormant PCa cell interactions in situ. Here, we developed a novel immunocompetent in vivo model of PCa dormancy in bone and employed the technology to address this major gap in knowledge. Methods: We established an in vitro protocol to induce dormancy in multiple PCa cell lines, including the C57BL/6 murine PCa cell line, RM1. Isogenic active and dormant PCA cells underwent bulk RNA Seq. We used an intra- iliac artery (IIA) approach to allow niche-specific seeding of dormant PCa within the bone. Spatial transcriptomics (10X Visium) was performed on tissue sections and R packages (Seurat & CellChat) were used for data analysis. Results: We validated in vitro PCa dormancy via cell cycle analysis, membrane dye retention and ERK/p38 ratio. RNASeq analysis of dormant PCa cells (vs. active) revealed upregulation of immunomodulatory genes such as, GDF15, BDNF, A2M, VGF, NLRP1, and CD55. C57BL/6 immunocompetent mice were inoculated with active or dormant RM1 cells via IIA. Majority of active RM1 mice reached endpoint within 15 days. In contrast, mice bearing dormant RM1 (D-RM1) had extended survival and were removed from study after 30 days. Histological analysis revealed solitary D-RM1 (panCK+/Ki67-/CC3-) on endosteal surfaces compared to macrometastases (high Ki67+) in the active RM1 group. Using IIA PBS injected tumor naïve bones as a control, we applied spatial transcriptomics (10X Visium) to interrogate the microenvironment surrounding active and dormant RM1 PCa cells. Bioinformatic analyses demonstrated higher expression of B cell markers (CD19, CD79a, CD79b, Ighm, and Pax5) and enhanced B cell activation/antigen phagocytosis signature in the D- RM1 group compared to other groups. We validated the presence of CD19 B-cells in proximity to the D-RM1 cells in subsequent tissue sections. In contrast, B cells were located less frequently and to the periphery of active RM1 metastases. Inference analysis using CellChat also identified a significant and distinct signaling pattern between CD55 and CD97 in the B cell enriched clusters in D-RM1 group. CD55/CD97 interaction is a noted mediator of B cell homing. These data suggest a potential role for distinct populations of B cells during the establishment of PCa dormancy in bone. Conclusion: Spatial transcriptomic analysis of the dormant PCa bone microenvironment reveals an enrichment of CD19+ B cells with a distinct signaling pattern. Given their known roles in immune tolerance we posit that infiltrating B cells may play a role in protecting and sustaining newly disseminated dormant PCa cells from immune mediated elimination.
 Citation Format: Mostafa M. Nasr, Ryan T. Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C. Lynch. Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da73f9c20af7ebfb3e86b110c704de45b40046e" target='_blank'>
              Abstract B044: Spatial transcriptomics reveals a distinct B cell signature in the dormant prostate cancer bone microenvironment
              </a>
            </td>
          <td>
            Mostafa M. Nasr, Ryan T Bishop, Haley du Bois, Tao Li, Jeremy S. Frieling, Conor C Lynch
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancers arising from dysregulation of generally operative signaling pathways are often tissue specific, but the mechanisms underlying this paradox are poorly understood. Based on striking cell-type specificity, we postulated that these mechanisms must operate early in cancer development and set out to study them in a model of von Hippel Lindau (VHL) disease. Biallelic mutation of the VHL ubiquitin ligase leads to constitutive activation of hypoxia inducible factors HIF1A and HIF2A and is generally a truncal event in clear cell renal carcinoma. We used an oncogenic tagging strategy in which VHL-mutant cells are marked by tdTomato, enabling their observation, retrieval, and analysis early after VHL-inactivation. Our findings reveal markedly different consequences of HIF1A and HIF2A activation, but that both contribute to renal cell-type specific consequences of VHL-inactivation in the kidney. Early involvement of HIF2A in promoting proliferation within proximal tubular epithelium supports therapeutic targeting of HIF2A early in VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be97f9c0f0b00dada5534b4bd4b5d705fdb68b10" target='_blank'>
              HIFα isoform specific activities drive cell-type specificity of VHL-associated oncogenesis
              </a>
            </td>
          <td>
            Joana Darc Carola Correia Lima, Madeleine Hooker, Ran Li, Ayslan B. Barros, N. Masson, Christopher W. Pugh, D. Mole, Julie Adam, Peter J. Ratcliffe, Samvid Kurlekar
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa9e49413db565400c66d264638af2b81a7c480" target='_blank'>
              Single cell-spatial transcriptomics and bulk multi-omics analysis of heterogeneity and ecosystems in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jiazhou Ye, Yan Lin, Zhiling Liao, Xing Gao, Cheng Lu, Lu Lu, Julu Huang, Xi Huang, Shilin Huang, Hongping Yu, Tao Bai, Jie Chen, Xiaobo Wang, Mingzhi Xie, Min Luo, Jinyan Zhang, Feixiang Wu, Guo-bin Wu, Liang Ma, Bangde Xiang, Lequn Li, Yongqiang Li, Xiaoling Luo, R. Liang
          </td>
          <td>2024-11-15</td>
          <td>NPJ Precision Oncology</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Despite thorough characterizations of cellular compositions within the breast tumor microenvironment (TME), their implications for disease progression and patient prognosis are still poorly understood. Unraveling these effects is vital for identifying potential targets to improve treatment outcomes. In this study, we devised an explainable machine learning (XML) pipeline to scrutinize the associations between TME cellular constituents and relapse-free survival (RFS). By applying our pipeline to estimated cell fractions in the METABRIC and TCGA datasets and comparing these results with associations to pathological complete response (pCR) after neoadjuvant chemotherapy (NAC), we created a comprehensive catalog of the TME’s role based on 5000 patient samples. Our findings reveal an unexpected dichotomy in which macrophages correlate positively with pCR but negatively with RFS, particularly within estrogen receptor-positive (ER+) and Luminal A and B (LumA/B) cancer subtypes. We show that this pattern is driven by heterogeneity in breast tumors characterized by increasing levels of macrophage infiltration. Through imaging mass cytometry (IMC) analysis, we discovered that macrophages tend to accumulate in the vicinity of HLA-ABChi epithelial cells as their frequency increases in tumor tissues and also express elevated levels of HLA-ABC protein. Combining IMC with single-cell RNA sequencing (scRNA-seq) data, we uncovered a significant association between these HLA-ABChi macrophages and regulatory and exhausted T cells (TReg and TEx), suggesting their involvement in immune suppression, likely by creating a chronically activated immunosuppressive TME. Subsequent cell-cell communication analysis predicted interactions between HLA-ABChi macrophages and TEx cells via the ligands SIGLEC9, ALCAM, and CSF1, and with TReg cells through APP, ANGPTL4, and SIGLEC9 signaling. Considering the clinical relevance of macrophages in ER+ (LumA/B) subtypes, our research enhances the characterization of macrophage-driven immune suppression in these tumors and identifies potential targets for immunomodulatory strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ef5aa61ac57e0a39d6c5657cd0986b7fc829d8" target='_blank'>
              Integrated multiomics analysis unveils how macrophages drive immune suppression in breast tumors and affect clinical outcomes
              </a>
            </td>
          <td>
            Y. Azimzade, Mads Haugland Haugen, Vessela N Kristensen, A. Frigessi, Alvaro Köhn-Luque
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM), an aggressive brain malignancy with a cellular hierarchy dominated by GBM stem cells (GSCs), evades antitumor immunity through mechanisms that remain incompletely understood. Like most cancers, GBMs undergo metabolic reprogramming toward glycolysis to generate lactate. Here, we show that lactate production by patient-derived GSCs and microglia/macrophages induces tumor cell epigenetic reprogramming through histone lactylation, an activating modification that leads to immunosuppressive transcriptional programs and suppression of phagocytosis via transcriptional upregulation of CD47, a “don’t eat me” signal, in GBM cells. Leveraging these findings, pharmacologic targeting of lactate production augments efficacy of anti-CD47 therapy. Mechanistically, lactylated histone interacts with the heterochromatin component chromobox protein homolog 3 (CBX3). Although CBX3 does not possess direct lactyltransferase activity, CBX3 binds histone acetyltransferase (HAT) EP300 to induce increased EP300 substrate specificity toward lactyl-CoA and a transcriptional shift toward an immunosuppressive cytokine profile. Targeting CBX3 inhibits tumor growth by both tumor cell–intrinsic mechanisms and increased tumor cell phagocytosis. Collectively, these results suggest that lactate mediates metabolism-induced epigenetic reprogramming in GBM that contributes to CD47-dependent immune evasion, which can be leveraged to augment efficacy of immuno-oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4f3773b00d094cbfb1f9a1b031617c36197710" target='_blank'>
              Lactate reprograms glioblastoma immunity through CBX3-regulated histone lactylation
              </a>
            </td>
          <td>
            Shuai Wang, Tengfei Huang, Qiulian Wu, Huairui Yuan, Xujia Wu, Fanen Yuan, Tingting Duan, S. Taori, Yingming Zhao, Nathaniel W. Snyder, Dimitris Placantonakis, Jeremy N. Rich
          </td>
          <td>2024-11-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69505c7dd30d328b855b8e26bb2a91fa4c9f9efa" target='_blank'>
              Computer-aided analysis reveals metallothionein-positive cancer-associated fibroblasts promote angiogenesis in gastric adenocarcinoma
              </a>
            </td>
          <td>
            Xiaolong Jin, Yu Tian, Haoran Zhu, Yuewen Sun, Zhenxing Zhang
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Adult-type diffuse gliomas present significant treatment challenges due to their complex cellular composition and evolving nature. Understanding this heterogeneity is critical for the development of effective, targeted therapies. We conducted single-cell and single-nucleus RNA sequencing on a unique cohort of 66 tumours (single surgery) and 54 paired samples (two surgeries). This comprehensive dataset of approximately 900,000 cells allows us to dissect conserved cellular landscapes and dynamic evolution across diverse glioma subtypes. Using a novel phylogeny-based approach, we identified an astrocyte-like glioma stem cell (GSC) as the root of diverse glioma lineages. This discovery transforms our understanding of glioma development, pinpointing the GSC as a potential therapeutic target. The GSC’s similarity to healthy adult neural stem cells provides clues as to the origin of glioma heterogeneity and the shared basis of intratumoural diversity. Our analysis uncovered a shared cellular architecture across astrocytomas, oligodendrogliomas, and glioblastomas (GBMs). This architecture encompasses seven recurring malignant cell states: neuro-, oligo-, neuro-oligo-lineage, cycling (G1/S, G2/M), hypoxia-associated, and a quiescent astrocyte-like GSC. Importantly, these cell state proportions vary significantly between glioma subtypes. Additionally, we demonstrate that GBMs display substantially higher T-cell and myeloid cell infiltration than IDH-mutant gliomas. Counterintuitively, this correlates with a poorer prognosis, implying profound T-cell dysfunction and intricate myeloid cell-tumour interactions. Longitudinal analysis further highlights the dynamic nature of glioma cell populations and the immune landscape. Subtype-specific T-cell and myeloid gene expression profiles were identified, including potential targets such as PARD6G (GBM) and CXCL13 (IDH-mutant-astrocytoma), laying the groundwork for tailored immunotherapies and prognostic biomarkers. This study presents a comprehensive model of glioma heterogeneity and evolution. The model reveals a shared astrocyte-like GSC that fuels cellular diversity, coupled with distinct, dynamic immune landscapes in different glioma subtypes. These findings hold extraordinary potential for developing transformative treatment strategies. Future research functionally validating the GSC and its impact on tumour dynamics will be essential for translating these insights into improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a47871bd911080cf561a5ed294b5aa4ee2716f4" target='_blank'>
              STEM-01. SHARED ASTROCYTE-LIKE GLIOMA STEM CELLS DRIVE HETEROGENEITY AND IMMUNE DYNAMICS IN ADULT-TYPE DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Zaman, W. Muskovic, E. Holton, M. Pandya, Rachael Zekanovic, Venessa T Chin, E. Lam, Catia Moutinho, J. Yeung, C. Teo, M. Sughrue, J. Powell
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) resists T-cell killing by developing an immunosuppressive tumor microenvironment. Transcriptomic technologies, such as single-cell RNA sequencing (scRNA-seq), have characterized immune cell phenotypes and putative cell-cell interactions in GBM. However, the tumor microenvironment surrounding T-cells is still not well characterized, motivating the need for spatial analyses. Two publicly-available GBM spatial transcriptomic datasets were analyzed, with one discovery cohort (N=18) and one validation cohort (N=12). Using cell deconvolution with a scRNA-seq reference dataset, areas of high T-cell infiltration were identified and clustered into regional programs based on transcriptomic signatures. To highlight which ligand-receptor interactions are most prominent in these regional programs, a novel spatial enrichment algorithm was applied. The impact of spatially enriched ligand-receptor interactions on overall survival was evaluated using bulk RNA-sequencing data from The Cancer Genome Atlas (TCGA). Three distinct T-cell regional programs (transitional, immunosuppressive, and infiltrative) were found across both the discovery and validation cohort. The infiltrative T-cell regional program resided on the periphery of the tumor with a neural progenitor phenotype (NRGN, VSNL1, SLC17A7). In contrast, the immunosuppressive T-cell region had a higher proportion of malignant cells and a greater mesenchymal tumor signature (ANXA1, CD44, complement pathway). Furthermore, ligand-receptor enrichment analysis showed that the immunosuppressive region had higher tumor and tumor-associated macrophage (TAM) signaling towards CD4 T-cells along the collagen, midkine, and osteopontin pathways. TCGA survival analysis showed that higher collagen and midkine signals were associated with worse overall and disease free survival. Our results identified three distinct regional programs in GBM and highlight a set of signaling pathways associated with T-cell suppression. Our findings suggest that the disruption of these pathways could reduce immunosuppressive signals from tumor and TAMs to T-cells. Subsequent validation with spatial proteomics data in glioblastoma is ongoing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed824dde4f1a4936f2335e4d52e940b75bae961" target='_blank'>
              EPCO-46. SPATIAL TRANSCRIPTOMICS ANALYSIS OF GLIOBLASTOMA REVEALS THREE DISTINCT REGIONAL PROGRAMS OF T-CELL INFILTRATION
              </a>
            </td>
          <td>
            William L Wang, Yiyun Chen, Chiu Ho Mo, Zinaida Good, E. Sotillo, Crystal Mackall
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance.In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature.The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma.We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Fibrocytes were identified as bone marrow-derived myeloid cells that also have fibroblast-like phenotypes, such as ECM production and differentiation to myofibroblasts. Although fibrocytes are known to contribute to various types of tissue fibrosis, their functions in the tumor microenvironment are unclear. We focused on fibrocytes as pivotal regulators of tumor progression. Our previous studies have indicated that fibrocytes induce angiogenesis and cancer stem cell-like phenotypes by secreting various growth factors. In contrast, immune checkpoint inhibitor (ICI)-treated fibrocytes demonstrated antigen-presenting capacity and enhanced antitumor T cell proliferation. Taken together, these findings indicate that fibrocytes have multiple effects on tumor progression. However, the detailed phenotypes of fibrocytes have not been fully elucidated because the isolation of distinct fibrocyte clusters has not been achieved without culturing in ECM-coated conditions or intracellular staining of ECM. The development of single-cell analyses partially resolves these problems. Single-cell RNA sequences in CD45+ immune cells from tumor tissue identified ECM-expressing myeloid-like cells as distinct fibrocyte clusters. In addition, these findings enabled the isolation of tumor-infiltrating fibrocytes as CD45+CD34+ cells. These tumor-infiltrating fibrocytes demonstrated both antigen-presenting ability and differentiation into myofibroblast-like cancer-associated fibroblasts. Considering these functions of fibrocytes in tumor progression, molecular-targeting agents for the migration, activity, and differentiation of fibrocytes are promising therapeutic strategies. Furthermore, identification of specific cell surface markers and master regulators of fibrocytes will advance novel fibrocyte-targeting therapies. In this review, we discuss the multiple roles of tumor-infiltrating fibrocytes and novel cancer therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f015914b4ca53df96678ddef524e2cd85d30fa" target='_blank'>
              Fibrocytes in tumor microenvironment: Identification of their fraction and novel therapeutic strategy.
              </a>
            </td>
          <td>
            A. Mitsuhashi, Yasuhiko Nishioka
          </td>
          <td>2024-11-04</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="

 Despite recent advances in the treatment of acute myeloid leukemia (AML), most patients with AML relapse and succumb to the disease. Achieving complete remissions (CRs) does not equate to cure, especially in TP53 mutated AML. Relapse after initial CRs is caused by measurable/minimal residual disease (MRD), which is resistant to prior therapies due to cell-intrinsic resistance mechanisms or is through protection by the bone marrow (BM) microenvironment. The mechanisms by which the BM microenvironment supports MRD cells are, however, largely unknown. We hypothesized that MRD AML cells possess unique characteristics distinct from leukemia cells prior to therapy and therefore may harbor targetable vulnerabilities. We performed multiplex protein analysis on AML BM biopsy specimens to dissect spatial characteristics of MRD cells and the BM microenvironment using spatial analytical techniques.
 To comprehensively investigate the spatial architecture of TP53 mutated AML, we performed sequential multiplex immunofluorescence (seqIF) staining of formalin-fixed paraffin-embedded BM trephine biopsies using COMET (Lunaphore Technologies) with a 20-antibody panel including CD105 and CD90 for mesenchymal stromal cells (MSCs) and CD3, CD4, CD8, TIGIT, FoxP3, CD56 and CD20 for immune cell markers. We analyzed paired AML BM biopsies from 6 patients with AML with TP53 point mutations at diagnosis and at morphological CR with MRD detected by flow cytometry or PCRafter receiving Aza/Ven- or Ara-C/Idarubicin-based regimens, and 3 normal BM biopsies. Four patients had relapse after achieving CRs. TP53 mutated AML cells expressed high p53 protein levels at diagnosis and at MRD while no or extremely low levels of p53 positive cells were detected in BM biopsies from healthy donors and AML patients with TP53 truncating mutations, served as negative controls. seqIF stained slides were subjected to image analysis, where we segmented BM structures such as MSCs, lipid and bone, and classified hematopoietic cells and leukemia cells into 22 cell categories by combining deep-learning/AI-based methods and supervised classification using Visiopharm software. We determined the distances between any two cells (cell-cell relationships, e.g. AML cells and T-cells) or the distances from any one cell to each area class (cell-structure relationships, e.g. AML cells and MSCs), which allowed us to calculate spatial enrichment scores and evaluate proximity of each component in the BM.
 On average, 25,000 (range 6,700-56,000) cells were analyzed per slide. seqIF detected 36.5 % (10 - 74%) of p53 positive (p53+) cells at diagnosis, and 0.23% (0.1 - 0.5%) at MRD. Clinical blast counts were 40% (22 - 74%) at diagnosis and MRD flow cytometry detected 1.1% (0 - 3.7%), suggesting that in some cases seqIF can detect malignant cells at MRD better than conventional multi-color flow cytometry in TP53 mutated AML. At both diagnosis and MRD, these p53+ cells tend to form clusters, not being randomly distributed throughout the BM. p53+ AML cells were classified using CD14, CD33, CD34, and CD71. p53+/CD71+ cells were more enriched in MRD BM samples compared to p53+/CD33+ cells (27% vs 6.8%). Intriguingly, BMs with p53+/CD71+ enrichment at MRD showed higher CD4/CD8 ratios compared to BMs at diagnosis collected from the same patients (0.68 vs 2.4). Moreover, cell-cell relationship analyses revealed that p53+/CD71+ AML cells are associated with a lower frequency of CD8+ T-cells compared to random distribution of all hematopoietic cells, suggesting that p53+/CD71+ cells are spatially sequestered from anti-AML immune cells. Cell-structure analyses revealed that the area of CD90+ MSCs is smaller than the CD105+ MSC area, consistent with previous reports (Cell 2024;187:1-21). In some cases, p53+ cells are not uniformly distributed in the BM but are proximal to specific BM niches such as CD90 or CD105 areas, suggesting that these specific MSCs support AML cell maintenance.
 In conclusion, novel seqIF technique and spatial analytical approaches revealed novel MRD clusters and spatial characteristics of p53 mutated AML. Additional studies to characterize cell-cell and cell-structure interactions are ongoing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c2ca1bf273b46205728cf824fa6017c30e17e" target='_blank'>
              Spatial Analysis of Human p53 Mutated AML Bone Marrow Reveals Novel MRD Clusters and Spatial Microenvironmental Features of p53 Mutated AML
              </a>
            </td>
          <td>
            Hideaki Mizuno, Yuki Nishida, Edward Ayoub, Akshay V. Basi, Andrea D Bedoy, Javier A Gomez, Jared K. Burks, R. Kanagal-Shamanna, Michael Andreeff
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Single-nucleus transcriptomic studies have revealed glial cell states associated with Alzheimer’s disease; however, these nuclei are dissociated from the complex architecture of the human neocortex. Here, we successfully performed an unbiased distance-based analytic strategy on spatially-registered transcriptomic data. Leveraging immunohistochemistry in the same tissue section, our analyses prioritized SERPINA3 and other genes, such as metallothioneins, as altered in the vicinity of neuritic amyloid plaques. Results were validated at the protein level by immunofluorescence, highlighting that a reactive SERPINA3+ astrocyte subtype, Ast.5, plays a role in the plaque microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9450bbd46dbba1bc34bf873f73b23f7e1a67db" target='_blank'>
              Spatial Transcriptomic Analysis Identifies a SERPINA3-Expressing Astrocytic State Associated with the Human Neuritic Plaque Microenvironment
              </a>
            </td>
          <td>
            Berke Karaahmet, Ya Zhang, Laurine Duquesne, Alina Sigalov, Christina Yung, Alex Kroshilina, David A. Bennett, M. Taga, Hans Klein
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit enhanced resistance to standard of care treatments and have significantly worse overall survival (OS) compared to patients with classical subtype enriched tumors. It is important to develop genomic resources, enabling identification of novel putative targets in a statistically rigorous manner.


EXPERIMENTAL DESIGN
We compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 patient samples, aggregated from publicly available raw data. We mapped cell-type specific scRNAseq gene signatures in bulk RNAseq (n=744) and spatial transcriptomics (ST) (n=22) and performed validation using multiplex immunostaining.


RESULTS
Analysis of tumor cells from our scRNAseq atlas revealed nine distinct populations, two of which aligned with the basal subtype, correlating with worse OS in bulk RNAseq. Deconvolution identified one of the basal populations to be the predominant tumor subtype in non-dissociated ST tissues and in vitro tumor cell and patient-derived organoid lines. We discovered a novel enrichment and spatial association of CXCL10+ cancer associated fibroblasts (CAFs) with basal tumor cells. We identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, suggesting a relationship between these cell types in PDAC tumor microenvironment.


CONCLUSIONS
We show that our scRNAseq atlas (700,000 cells), integrated with ST data, has increased statistical power and is a powerful resource, allowing for expansion of current subtyping paradigms in PDAC. We uncovered a novel signaling niche marked by CXCL10+ CAFs and basal tumor cells that could be explored for future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07269a9cb871b60ea57ca5b2a01a49e35b66cc9b" target='_blank'>
              Human pancreatic cancer single cell atlas reveals association of CXCL10+ fibroblasts and basal subtype tumor cells.
              </a>
            </td>
          <td>
            Ian M Loveless, Samantha B. Kemp, Kailee M Hartway, Jacob T. Mitchell, Yuesong Wu, Samuel D Zwernik, D. Salas-Escabillas, Sydney Brender, Madison George, Yetunde Makinwa, Thais Stockdale, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombwell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, E. Fertig, L. T. Kagohara, Brian Theisen, Howard C Crawford, Nina G. Steele
          </td>
          <td>2024-12-05</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff90b5963386b78b07e8f87307689a97c4a0bc10" target='_blank'>
              iFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells.
              </a>
            </td>
          <td>
            Irene Garcia-Gonzalez, Stefano Gambera, Susana F Rocha, Alvaro Regano, Lourdes García-Ortega, Mariya Lytvyn, Luis Diago-Domingo, Maria S. Sanchez-Muñoz, Aroa Garcia-Cabero, Ivana Zagorac, Wen Luo, Macarena De Andrés-Laguillo, Macarena Fernández-Chacón, Verónica Casquero-García, F. F. Lunella, C. Torroja, F. Sánchez-Cabo, Rui Benedito
          </td>
          <td>2024-12-13</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Reduced therapy response in breast cancer has been correlated with heterogeneity in biomarker composition, expression level, and spatial distribution of cancer cells within a patient tumor. Thus, there is a need for models to replicate cell-cell, cell-stromal, and cell-microenvironment interactions during cancer progression. Traditional two-dimensional (2D) cell culture models are convenient but cannot adequately represent tumor microenvironment histological organization, in vivo 3D spatial/cellular context, and physiological relevance. Recently, three-dimensional (3D) in vitro tumor models have been shown to provide an improved platform for incorporating compositional and spatial heterogeneity and to better mimic the biological characteristics of patient tumors to assess drug response. Advances in 3D bioprinting have allowed the creation of more complex models with improved physiologic representation while controlling for reproducibility and accuracy. This review aims to summarize the advantages and challenges of current 3D in vitro models for evaluating therapy response in breast cancer, with a particular emphasis on 3D bioprinting, and addresses several key issues for future model development as well as their application to other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ccd5bcde9ada94a158d2c3401e2be4034200cf" target='_blank'>
              Modeling intratumor heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            E. McDonough, M. Barroso, Fiona Ginty, David T Corr
          </td>
          <td>2024-12-06</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits a vast heterogeneity in cell types and states. Recent research highlights the reactivation of neurodevelopmental programs in GBM, contributing to this heterogeneity and, ultimately, therapy resistance. To characterize the landscape of progenitor cell types in human primary GBM, we compiled a meta-atlas of single-cell transcriptomes from seven published studies. Using a novel bioinformatic method for gene program analysis, we derived GBM gene expression meta-modules represented across multiple tumors. Our analyses validated previously described GBM gene programs and revealed novel tumor progenitor subtypes. Among these subtypes, we identified a tumor-derived stem cell population exhibiting a mixed vascular identity and transcriptional programs reminiscent of neural stem cells, which we identified as a putative neurovascular progenitor (NVP). To our knowledge, this population has not been previously described in GBM. We validated the existence of NVPs in situ using immunofluorescence from primary patient GBM samples. We then performed an enrichment for NVPs using fluorescence-activated cell sorting direct from patient tumors, followed by single-cell RNA sequencing. Examination of data from traditional model systems, including gliomaspheres and mouse GBM models, confirmed the preservation of NVP identity across systems. In vivo analysis of NVPs in multiple mouse models of GBM elucidate the influence of NVPs on GBM cell type composition, highlighting their role as multipotent progenitors. Finally, we utilized single-cell lineage barcoding to trace the progeny of patient-derived NVP cells in a novel organoid transplantation model, revealing NVP differentiation into vascular-like and neural-like cells. These data indicate that NVP cells play crucial roles in tumor plasticity and may represent a valuable therapeutic target for future exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c96f99928f041516b346c68c5a8d4ece5bdedd" target='_blank'>
              STEM-14. INTEGRATED ANALYSIS OF GLIOBLASTOMA PROGENITOR SUBTYPES REVEALS A TUMOR-DERIVED STEM CELL POPULATION WITH NEURAL AND VASCULAR POTENTIAL
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Matthew Li, Kwanha Yu, Weihong Ge, Patricia R. Nano, Ryan L. Kan, Jose A. Soto, Elizabeth E Crouch, D. Raleigh, Dimitri Cadet, C. Tse, Andrew Tum, Nicholas A. Bayley, Benjamin Deneen, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Melanoma is a highly malignant skin tumor characterized by high metastasis and poor prognosis. Recent studies have highlighted the pivotal role of melanoma stem cells (MSCs)—a subpopulation of cancer stem cells (CSCs)—in driving tumor growth, metastasis, therapeutic resistance, and recurrence. Similar to CSCs in other cancers, MSCs possess unique characteristics, including specific surface markers, dysregulated signaling pathways, and the ability to thrive within complex tumor microenvironment (TME). This review explored the current landscape of MSC research, discussing the identification of MSC-specific surface markers, the role of key signaling pathways such as Wnt/β-catenin, Notch, and Hedgehog (Hh), and how interactions within the TME, including hypoxia and immune cells, contribute to MSC-mediated drug resistance and metastatic behavior. Furthermore, we also investigated the latest therapeutic strategies targeting MSCs, such as small-molecule inhibitors, immune-based approaches, and novel vaccine developments, with an emphasis on their potential to overcome melanoma progression and improve clinical outcomes. This review aims to provide valuable insights into the complex roles of MSCs in melanoma biology and offers perspectives for future research and therapeutic advances against this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fdcf0db27ab5805fb28ff517e6fe41f47f92d66" target='_blank'>
              The roles of cancer stem cells and therapeutic implications in melanoma
              </a>
            </td>
          <td>
            Xiao-Li Mu, Yixin Zhou, Yongxin Yu, Mingyi Zhang, Jiyan Liu
          </td>
          <td>2024-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 To transform cancer drug development pipelines and gain a deeper mechanistic understanding of cancer: immune interactions, we desperately need better experimental models that faithfully recapitulate the cellular heterogeneity, complex and dynamic cell:cell interactions, and therapy responses in humans. To address this need, we developed and characterized a cohort of five new mouse glioma models in the C57BL6/j background. These transplantable tumorsphere models were derived from spontaneous gliomas that formed in genetically engineered mouse models with different oncogenic drivers, including EGFRviii, PDGFA, PTEN, p53, and NF1 mutations. They show a range of responses to standard of care (TMZ+IR) similar to GBM patients and all are resistant, to immune checkpoint inhibitors, anti-PD1+CTLA4, treatment, unlike the GL261 model and similar to GBM patients. We deeply characterized each model, including immune phenotyping and single-cell RNA-sequencing analyses. We also performed a cross-species comparison of human and mouse GBM and stromal cell phenotypes at the single-cell level. Our results show that new mouse models recapitulate the heterogeneity of glioma cell states observed in individual patients and, importantly, also the complex and immune suppressive TME. We demonstrate at the single-cell level that glioma-associated T and myeloid cell types are molecularly similar and conserved across species. Leveraging these data sets and experimental models, we are testing now novel immunotherapy targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcbcbc1f841c1a6de776b418447465f25bcaddf" target='_blank'>
              IMMU-66. PHYSIOLOGICALLY RELEVANT MURINE GLIOMA MODELS THAT REPRESENT GBM HETEROGENEITY AND ENABLE THE NEXT GENERATION OF DISCOVERY AND THERAPY EFFICACY STUDIES
              </a>
            </td>
          <td>
            J. Maldonado, Caiyi Wang, Jia-Shiun Leu, H. N. Tran, Nourhan Abdelfattah, Kyuson Yun
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Background
 Malignancies exhibit variable cellular distribution patterns and the relationship between these topographic variations, underlying biological processes, and clinical outcomes remain poorly understood. Point process analyses, widely used in ecology, can elucidate the spatial distribution of points in complex systems but have rarely been applied to tumor heterogeneity. We recently demonstrated (Hoppe et al, Cancer Discovery 2023), that cells co-expressing high MYC and BCL2 but lacking BCL6 (M+2+6-), in Diffuse Large B Cell Lymphoma (DLBCL) are consistently correlated with poor survival compared to other MYC/BCL2/BCL6 combinations. Machine learning approaches can be applied to understand nuances of cellular point patterns and help with correlating with clinicopathological variables. Here, we developed a code frame that can be generalized across tissue regions accounting for heterogeneity to quantitatively study spatial patterns of M+2+6- cells in DLBCL.
 Methods
 We developed a scalable automated workflow for generating spatial point patterns. The individual steps of the pipeline have been consolidated into a standalone package that can be executed in a facile manner for any image type, without dependencies on any proprietary software. Using multiplexed fluorescent immunohistochemistry (mfIHC) in four cohorts of DLBCL (n=449), spatial point patterns were derived, and Geyer's point process model was applied. Machine learning classification models were benchmarked for spatial statistics derived from the point process model. A multi-omic analysis, including single-cell transcriptomic analyses of 22 DLBCL samples, was then conducted. Spatial transcriptomic technique, Stereoseq, was also conducted on 2 DLBCL samples.
 Results
 The workflow consists of the following parts: 1) The python script using the OpenCV package to manipulate the kernel size and intensity of the spatial coordinates overlaid on the images; 2) A QuPath groovy script to automate the import and export of the images and the parameter thresholds for the pixel classifier; 3) An R script to build the spatial point patterns from coordinates and save different oncogene co-expression as marks within the accurate geojson annotations; and 4) an R script to obtain measures of quality in terms of minimizing the number of points excluded while generating accurate spatial point pattern windows.
 After applying the pipeline, we see that patients could be divided into two: one group showed “clustered” spatial organization, while the other displayed a “dispersed” M+2+6- cell distribution. We achieved an accuracy of 98% in classifying patients as “dispersed” and “clustered” across four cohorts, through the random forest model. Cases with “dispersed” M+2+6- cells had shorter overall survival across all analyzed cohorts (P < 0.05 in 4/4 cohorts). Patients enriched in the “dispersed” phenotype, predominantly belonged to the ABC cell of origin subtype. We derived a “dispersed” pattern gene signature through multi-omic analyses which expressed genes implicated in cell migration and adhesion. Validation of the dispersed signature was conducted using Stereoseq, where M+2+6- cells enriched in the signature displayed greater values of L function across distances. Patients enriched in the dispersed phenotype displayed lower infiltration of immune subtypes though deconvolution hinting at a possible immunologically cold microenvironment.
 Conclusion
 This study demonstrates the clinical relevance studying the spatial distribution of malignant cell subpopulations through point pattern analysis. We anticipate that this machine learning pipeline can be developed for clinical use, enabling the classification of spatial phenotypes in DLBCL biopsies for patient stratification.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bd0b4673621cba332787c4ab367ac2dd048cda" target='_blank'>
              Machine Learning Classification of Spatial Patterns of Malignant Cells Reveals Implications in Prognosis and Tumor Microenvironment Composition in Lymphoma
              </a>
            </td>
          <td>
            Shruti Sridhar, M. Hoppe, P. Jaynes, Gayatri Kumar, Siddham Jasoria, Ziwei Meng, Y. Peng, S. de Mel, Limei Poon, E. Chan, W. Chng, S. Tan, S. Hue, Siok‑Bian Ng, C. Nagarajan, N. Grigoropoulos, Shaoying Li, Joseph D. Khoury, P. Farinha, A. Mottok, David W. Scott, Xiaofei Ye, Q. Pan-Hammarström, Vaibhav Rajan, Kasthuri Kannan, A. D. Jeyasekharan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="The phenotype of circulating tumor cells (CTCs) offers valuable insights into monitoring cancer metastasis and recurrence. While microfluidics presents a promising approach for capturing these rare cells in blood, the phenotypic profiling of CTCs remains technically challenging. Herein, we developed a nanoporous micropillar array chip enabling highly efficient capture and in situ phenotypic analysis of CTCs through enhanced and tunable on-chip immunoaffinity binding. The nanoporous micropillar array addresses the fundamental limits in fluidic mass transfer, surface stagnant flow boundary effect, and interface topographic and multivalent reactions simultaneously within a single device, resulting in a synergistic enhancement of CTC immunocapture efficiency. The CTC capture efficiency increased by approximately 40% for cancer cells with low surface marker expressing. By manipulating fluidic velocity (hydrodynamic drag force) on the chip, a cell adhesion gradient was generated in the capture chamber, enabling individual CTCs with varying expression levels of epithelial cellular adhesion molecules to be immunocaptured at the corresponding spatial locations where equilibrium drag force is provided. The clinical utility of the nanoporous micropillar array was demonstrated by accurately distinguishing early and advanced stages of breast cancer and further longitudinally monitoring treatment response. We envision that the nanoporous micropillar array chip will provide an in situ capture and molecular profiling approach for CTCs and enhance the clinical application of CTC liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4168617dcff4a12ca2fb61825d5b9644d135db" target='_blank'>
              Profiling Phenotypic Heterogeneity of Circulating Tumor Cells through Spatially Resolved Immunocapture on Nanoporous Micropillar Arrays.
              </a>
            </td>
          <td>
            Lianyu Lu, Yaohui Wang, Yue Ding, Yuqing Wang, Zhi Zhu, Jinsong Lu, Liu Yang, Peng Zhang, Chao Yang
          </td>
          <td>2024-11-04</td>
          <td>ACS nano</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Diffuse midline gliomas, H3K27-altered (DMG) are highly malignant brain tumors with a 2-year survival rate of less than 10 percent. These tumors are primarily localized to the pons and thalamus regions of the brain. The cellular compartment of the tumor microenvironment (TME) which surrounds a tumor is composed of tumor and non-tumor cells, such as immune cells and oligodendrocytes. Communication between tumor cells and non-tumor cells in the TME can promote tumor growth and targeting communication networks could improve patient outcomes. Using a bioinformatics approach and publicly available pediatric DMG single-cell RNA-sequence
 (scRNA-seq) data, several methods of communication (secreted signaling, cell-cell contact, and extracellular matrix) within the DMG TME were identified. Based on the high communication probability, outgoing and incoming networks from non-tumor cells and tumor cells were inferred, such as the Complement, Galectin, and CD96 pathways. Specific ligand-receptor interactions between non-tumor and tumor cells were also found, including LGALS9-HAVCR2, PDGFB/PDGFRA, and COL4A5/ITGAV. A subsequent literature review was performed to validate these identified pathways in other tumors, including gliomas, and inform future in vitro experiments with primary DMG cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86e4e55f141e1e3cca1b3dde7a5016937939f79" target='_blank'>
              TMIC-65. USING BIOINFORMATICS TO INVESTIGATE THE TUMOR MICROENVIRONMENT OF DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Havisha Kalpatthi, Andrea F Cruz, S. Agnihotri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Rationale:
 Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases like atherosclerosis and pulmonary hypertension, but their role in tumorigenesis is unknown.


 Objective:
 To investigate SMC diversity and plasticity in tumor angiogenesis and its underlying molecular drivers.


 Methods and Results:
 Here we use SMC-specific lineage-tracing mouse models and single cell RNA sequencing (scRNA-seq) to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that around 10% of SMCs adopt a phenotype which includes features traditionally associated with macrophage-like cells, suggesting they may contribute to the immune profile of the tumor microenvironment (TME). These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Signaling from bone marrow stromal antigen 2 (BST2) on the surface of tumor cells to PIRA2 on SMC promotes this phenotypic transition, leading to a SMC phenotype that is more proliferative, migratory, and phagocytic. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and the cells that were able to transition had a comparatively higher inflammatory signal typically associated with anti-tumor effect.


 Conclusions:
 As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. The novel concept of SMC contribution to the immune landscape suggests manipulation of SMC phenotypic switching as an approach to altering cancer pathogenesis. Further translational work is needed to understand how this phenotypic switch could influence response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049a7db1d4940952698e1c93fbbc4e08101322fd" target='_blank'>
              Abstract 4134531: BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, Wei Feng Ma, Clint Miller, Nick J. Leeper
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent advances in fate mapping and single-cell technologies have revealed how the dynamics and function of tissue-resident macrophages are shaped by their environment. However, macrophages in sensory organs such as the cochlea where the central nervous system and peripheral nervous system meet remain understudied. Combining single-cell transcriptomics, fate mapping, and parabiosis experiments, we show that five types of myeloid cells including three tissue-resident macrophage subpopulations, coexist in the mouse cochlea. The three macrophage subsets showed different potential functions in relationship with their specific topography across cochlear compartments. Further analysis revealed that they were partially derived from yolk sac progenitors during development, while in adulthood, most cochlear macrophages were long-term resident. Finally, we showed that cochlear macrophage morphology and density changed during aging. Our findings show that cochlea is a microenvironment with a unique heterogeneity of macrophages in terms of gene expression, spatial distribution, ontogeny, and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26fe62c7df589b534663cdc760f68dc26a4ad8b4" target='_blank'>
              Single-cell, spatial, and fate-mapping analyses uncover niche dependent diversity of cochlear myeloid cells
              </a>
            </td>
          <td>
            Aude Chiot, Max J. Felgner, Dillon Brownell, Katherine H. Rott, Alina P. Bogachuk, Dennis-Dominik Rosmus, Takahiro Masuda, Audrey Ching, Patrick J Atkinson, Marco Prinz, Karen Sachs, Alan G. Cheng, Peter Wieghofer, Bahareh Ajami
          </td>
          <td>2024-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Aggressive human prostate tumors traverse a contractile smooth muscle pseudo-capsule, in a process termed extracapsular extension (ECE), defining the pT3 pathologic stage associating with increased biochemical recurrence, bone metastases, and cancer-specific mortality. While T3 lesions can be detected by non-invasive mpMRI imaging, how the tumor invades into and through the contractile muscle is unknown. Using a mouse xenograft model system of ECE, we investigated the transcriptomic consequences of human tumor invasion into and through the contractile smooth muscle of the mouse diaphragm. Both human and mouse gene transcripts were documented and analyzed. Tumor cells were intraperitoneally injected into male NSG mice and 6 weeks later, three tumor compartments (pre-invasive, muscle-invasive, and super-invasive cells that had reached the superior diaphragm surface) were analyzed for differential bulk human and mouse gene expression. Whole genome transcriptomic sequencing and GO pathway analysis revealed that approximately 414 human genes were differentially expressed between the non- invasive, muscle-resident, and super-invasive tumor populations and at least 5 enriched pathways were involved. The unique expression gene patterns of muscle-resident tumor cells were reversible when compared to the pre-invasive and super-invasive expression patterns. Significant increases in g-H2AX and nuclear deformation were observed in the muscle-resident tumor clusters as compared to the non-invasive tumor mass. No differences in tumor cell proliferation, as detected by Ki67 staining, were found between the tumor clusters in the three compartments. Immunohistochemistry staining for a damage response cytokeratin (KRT6A) and integrin (CD49f) revealed heterogeneity within the muscle resident tumors as compared to the non- invasive cells. Taken together, these data suggest a hostile contractile muscle environment elicits specific responses by the invading tumor. Single cell analysis within each of the tumor compartments is currently underway to define the spatial heterogeneity of gene expression in invasive tumor cell clusters within the contractile muscle. A dynamic reciprocity of the tumor/muscle microenvironment is suggested by the analysis of the bulk transcriptomic sequencing demonstrating a significant increase in mouse muscle bio-synthetic gene transcription as a consequence of the human tumor penetrating the tissue. Taken together, these data indicate that human tumors, during the act of traversing the contractile muscle layer, respond to this unique microenvironment by transiently altering transcription, while sustaining nuclear damage which results in the reprogramming of the muscle. This new information suggests that novel tumor or muscle biomarkers might indicate early muscle invasion events and assist new high-resolution image analysis for precision diagnostic and/or therapeutic decisions. (Supported in part by P30 CA23074; F30 CA143924, UACC Team Science Award).
 Citation Format: Kendra D Marr, Beatrice S Knudsen, Rafael Sainz, Kelvin W Pond, Noel E Warfel, Belinda E Sun, Anne E Cress. Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d695d2e17a8bb732f87fee7797bcc07c303850" target='_blank'>
              Abstract A055: Tumor muscle invasion promotes tumor heterogeneity and normal muscle reprogramming
              </a>
            </td>
          <td>
            Kendra D. Marr, Beatrice S. Knudsen, Rafael Sainz, Kelvin W Pond, Noel E. Warfel, Belinda E Sun, A. Cress
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Introduction:
 Hematopoietic stem cells are in constant clonal competition, as a result variant clones resistant to inflammatory pressures emerge with aging resulting in clonal hematopoiesis. Multiple independent clones can coexist, and potentially come into equilibrium. However, with an acute leukemia a clone overtakes the system, thus inhibiting normal hematopoiesis. We sought to explore the possibility of leveraging the increased competitive ability of CHIP clones to dampen expansion of an aggressive malignant clone. As proof of principle, we tested whether introduction of Tet2-/- cells as competitors could dampen the expansion of MPLW515L cells, an aggressive MPN mouse model.
 Methods:
 We used a transduction-transformation model of MPL W515L as an aggressive MPN model. This mouse model usually results in death approximately 3-4 weeks post-transplant. Whole bone marrow from wild-type C57B/6 (CD45.1) mice was transduced with Green Fluorescent Protein (GFP) tagged retrovirus to induce expression of hMPLW515L. We transduced bone marrow with a low viral multiplicity of infection (MOI) resulting in approximately 1-5% GFP+ cells to extend survival beyond the expected 3-4 weeks post-transplant. The transduced bone marrow was mixed with unmanipulated whole bone marrow from either WT or Tet2-/- (both in CD45.1/2) to result in a 90%/10% mixture and transplanted into lethally irradiated CD45.2 mice. Peripheral blood counts, % GFP+ cells, and chimerism was followed biweekly. Surviving mice were sacrificed and analyzed at 5 months post-transplant.
 Results:
 There was early death from expansion of MPLW515L GFP+ cells within 3-4 weeks in 10% of both the WT and Tet2-/- competitor groups. However, of the surviving mice the percentage of GFP+ cells in the peripheral blood was consistently lower in the Tet2-/- competitor group compared to the WT competitor group (for example mean of 55% GFP+ in WT competitor group vs 23% at day 105 post-transplant). At five months post-transplant we sacrificed the cohort to assess the composition of MPLW515L cells in the hematopoietic stem and progenitor compartments. The percentage of GFP+ cells was higher in all cell populations assessed in the WT competitor group compared to the Tet2-/- competitor group. Moreover, spleen and liver weights were higher in the WT versus Tet2-/- competitor group.
 Conclusions:
 This work demonstrates proof of principle that clones with increased competitive ability and low malignant potential, such as Tet2-/- cells, can be used as a buffer for the expansion of highly aggressive clones. This concept of introduction of non-malignant competitor clones could be potentially be explored as a novel therapeutic approach in aggressive hematologic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bdd911e73f56ee0ae933fdda9f4ee1c54e2a20" target='_blank'>
              Leveraging Clonal Competition between Chip and Aggressive Malignant Clones As a Novel Therapeutic Approach
              </a>
            </td>
          <td>
            K. K. Guillermo, Xin (Helen) Huang, Jianhong C Heidmann, Angela G. Fleischman
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and most aggressive malignant primary brain tumor in adults. Among the most lethal human malignancies, GBM is highly resistant to cytotoxic treatments and molecularly targeted therapies, and also generally refractory to T cell–based immunotherapies, largely due to their immunologically cold nature, i.e., characterized by a paucity of tumor T cell infiltrates that result from the immunosuppressive microenvironment. Here, we identify a distinct mechanism driven by spatiotemporal interaction of leukocytes with tumor stromal cells, namely, mesenchymal-like population of endothelial cells (ECs) form an immunosuppressive vascular niche that regulates macrophage polarization and induces GBM resistance to CAR T immunotherapy. Our single-cell transcriptome analyses of human and mouse GBM tumors identify a subpopulation of ECs acquire robust immunosuppressive phonotypes to drive alternative macrophage polarization. These ECs are spatially localized proximately to immunosuppressive macrophages and selectively express Twist1. Furthermore, we reveal a Twist1/SATB1-mediated sequential transcriptional activation mechanism, through which Twist1+ tumor ECs produce osteopontin to locally promote immunosuppressive macrophage phenotypes. Genetic or pharmacological ablation of Twist1 reverses macrophage-mediated immunosuppression and enhances T cell infiltration and activation, leading to reduced GBM growth and extended mouse survival. Importantly, pharmacological inhibition and lipid nanoparticle-mediated targeted inactivation of Twist1 overcome GBM resistance to Egfrviii CAR T cell immunotherapy. Thus, these findings uncover a spatially restricted mechanism controlling vasculature-mediated tumor immunity and suggest that targeting endothelial Twist1 may offer exciting opportunities for GBM immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa787a775e12e3cc0befad1784ebce83795d53f9" target='_blank'>
              IMMU-22. TWIST1-MEDIATED ENDOTHELIAL PLASTICITY DRIVES GLIOBLASTOMA RESISTANCE TO CAR T IMMUNOTHERAPY
              </a>
            </td>
          <td>
            Fan Yang, Md Naushad Akhtar, Yanqing Gong, Yi Fan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Pericytes are crucial for the development, stabilization, and functional regulation of microvasculature, especially in the retina. In diabetic retinopathy (DR), early loss of pericytes is a key event that drives microvascular dysfunction. Despite their critical role, the mechanisms underlying the functional heterogeneity of pericytes in DR remain poorly understood, impeding the development of effective therapeutic strategies. Methods We employed single-cell RNA sequencing to construct a comprehensive single- cell atlas of non-diabetic and diabetic retinas. Using bioinformatic clustering and subcluster analysis, we identified a specific pericyte subcluster associated with diabetic microvascular complications. Differential gene expression analysis and immunofluorescence validation highlighted PTTG1 as a potential key regulator of pericyte dysfunction. To investigate its functional role, we emplyed CRISPR/Cas9 and adenoviral vectors to modulate PTTG1 expression in vitro and in vivo. Combined transcriptomic and metabolomic approaches were used to explore the mechanistic pathways through which PTTG1 influences pericyte biology and vascular function. Results We identified a novel pericyte subcluster characterized by elevated expression of PTTG1, which was strongly correlated with diabetic microvascular dysfunction. Silencing PTTG1 using CRISPR/Cas9 and siRNA in vitro mitigated pericyte dysfunction under high- glucose conditions. Targeted knockdown of PTTG1 using viral vectors improved retinal vascular integrity and reduced neovascularization in diabetic mice. Transcriptomic and untargeted metabolomic analyses revealed that PTTG1 knockdown reprogrammed pericyte energy metabolism by modulating glycolysis pathway genes, reducing oxidative stress, and restoring pericyte function, ultimately alleviating microvascular dysfunction in DR. Conclusions PTTG1 plays a critical role in regulating pericyte dysfunction and maintaining vascular homeostasis in diabetic retinopathy. By modulating key metabolic pathways and pericyte phenotypes, PTTG1 represents a promising therapeutic target for treating diabetic microvascular complications. These insights offer a novel molecular framework for developing targeted therapies aimed at restoring retinal vascular health in diabetic patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2cfb264dd76b8c3dd1f5b9db0dce030fd7e6a3" target='_blank'>
              PTTG1-Mediated Pericyte Dysfunction Drives Diabetes-Induced Microvascular Dysfunction
              </a>
            </td>
          <td>
            Linyu Zhang, Ling Ren, Jingyue Zhang, Min Xia, Xiaosa Li, Mudi Yao, Fanfei Ma, Chang Jiang, Jin Yao, Biao Yan
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Aging impacts the bone marrow microenvironment by inducing inflammation. We hypothesize that aging increases heritable epigenetic heterogeneity, leading to transcriptome heterogeneity within hematopoietic stem and progenitor cells (HSPC). This resulting heterogenous state, combined with the aged microenvironment-driven selection, shifts the functional capacity of the hematopoietic system, contributing to the functional decline of the hematopoietic system as well as the evolution of diseases such as myelodysplastic syndrome and leukemias. Our research aims to investigate clonal heterogeneity in both young and old mice and highlight the phenotypic patterns selected in each case.
 We employed CellTag indexing and single-cell RNA sequencing to track HSPC, identify the selected clones in young and old mice BM, and characterize their molecular signatures. We generated 10x Genomics single-cell RNA-seq for tagged HSPC after 15-day culturing (in vitro) and two months post-transplantation into mice (in vivo). Tracing the expansion of HSPC clones in young and old BM revealed critical changes in HSPC heterogeneity and selection in older mice.
 We observed an increase in transcriptional heterogeneity of old HSPC as compared to young samples (P<0.01, Wilcox test). Genes with higher gene expression variability in older mice coincide with pathways pivotal for HSPC fitness, e.g., ribosome synthesis, inflammation response mechanisms (JAK-STAT), hematopoietic cell lineage, AML, and cancer pathways (FDR<0.1, GSEA). At the clonal composition level, young HSPCs exhibited more oligoclonal reconstitution than old HSPC, indicated by lower Shannon entropy in young HSPCs compared to old (P<0.05 Wilcox test). This result suggests stricter clonal quality control in the young microenvironment, potentially leading to increased purifying selection that removes clones with significant transcriptional profile alterations and reduced fitness. Moreover, comparing the transcriptional profiles of clones showing robust expansion after transplantation (selected or “winner” clones) with negatively selected clones revealed several common pathways shared between young and old selected clones after expansion (FDR<0.1, GSEA), including cell cycle activity, stress response, WNT signaling, and metabolism - identifying candidate mediators of HSPC fitness in vivo. However, major differences between old and young selected clones were evident before expansion. In young, selected clones, signatures related to inflammatory response were observed.Strikingly, we observed a significant positive expression correlation (P<0.0001, Wilcox test) among the selected clones only in young mice, not in old mice. The high transcriptomic similarity among selected clones is consistent with strict clonal quality control in the young BM microenvironment. Notably, we traced the selected clones back among the tagged HSPCs in vitro (scRNA-seq) and observed higher transcriptomic similarity among selected clones from young vs. old donor mice even before transplantation (P<0.0001, Wilcox test). Collectively, these results underscore the selective nature of the young BM, contrasting with the decline of HSPC quality control in old BM, contributing to functional hematopoietic decline and fueling pre-malignant evolution.
 These studies reveal striking differences in clonal dynamics and quality control among HSPC in young and old mice. Clonal tracing of hematopoietic clones provides insight into how clonal competition and microenvironmental selection can contribute to hematopoietic decline and malignant evolution. Better understanding of the pathways that contribute to HSPC fitness and functional decline with aging could lead to approaches to track and perhaps even counter these aging-associated changes, which is expected to contribute to improved hematopoietic function and reduced risk of malignancy in the elderly.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff59a979a8b7594abbd6b4f17c49725e50fa5fe2" target='_blank'>
              Title: In Vivo Clonal Tracing of Hematopoietic Stem and Progenitor Cells Reveals Increased Clonal Heterogeneity during Aging, Alongside Critical Changes in Selection Patterns
              </a>
            </td>
          <td>
            Lamis Naddaf, Marco De Dominici, Xiaowen Chen, Parveen Kumar, James S Chavez, Travis Roeder, Shilpita Karmakar, Eric M. Pietras, Hideyuki Oguro, James DeGregori, Sheng Li
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer progression and response to therapy are inextricably reliant on the coevolution of a supportive tissue microenvironment. This is particularly evident in pancreatic ductal adenocarcinoma, a tumor type characterized by expansive and heterogeneous stroma. Herein, we employed single-cell RNA sequencing and spatial transcriptomics of normal, inflamed, and malignant pancreatic tissues to contextualize stromal dynamics associated with disease and treatment status, identifying temporal and spatial trajectories of fibroblast differentiation. Using analytical tools to infer cellular communication, together with a newly developed assay to annotate genomic alterations in cancer cells, we additionally explored the complex intercellular networks underlying tissue circuitry, highlighting a fibroblast-centric interactome that grows in strength and complexity in the context of malignant transformation. Our study yields new insights on the stromal remodeling events favoring the development of a tumor-supportive microenvironment and provides a powerful resource for the exploration of novel points of therapeutic intervention in pancreatic ductal adenocarcinoma. Significance: Pancreatic cancer remains a high unmet medical need. Understanding the interactions between stroma and cancer cells in this disease may unveil new opportunities for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106968e3649b3283e96d5bfd01ae138fc25d6cc4" target='_blank'>
              Dynamic Evolution of Fibroblasts Revealed by Single-Cell RNA Sequencing of Human Pancreatic Cancer
              </a>
            </td>
          <td>
            S. Dimitrieva, Jon M Harrison, Jonathan Chang, M. Piquet, M. Mino-Kenudson, Millicent Gabriel, Vivek Sagar, Heiko Horn, K. Lage, Julie Kim, Gang Li, Shaobu Weng, Cynthia Harris, Anupriya S Kulkarni, David T Ting, M. Qadan, Peter J Fagenholz, Cristina R Ferrone, Angelo L Grauel, Tyler Laszewski, Alina Raza, Markus Riester, Tim Somerville, Joel P Wagner, Glenn Dranoff, J. Engelman, Audrey Kauffmann, Rebecca Leary, Andrew L Warshaw, K. Lillemoe, C. F. Fernandez-Del Castillo, David A Ruddy, Andrew S. Liss, Viviana Cremasco
          </td>
          <td>2024-11-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9e868d681b765f550dc5f5febbd2934a88e0fb" target='_blank'>
              Single-cell RNA sequencing reveals the landscape of the cellular ecosystem of primary hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zeli Yin, Yilin Song, Liming Wang
          </td>
          <td>2024-11-14</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Pancreatic cancer has a 5-year survival rate of merely 12%. The high rate of late-stage diagnoses underscores the need for reliable biomarkers for early detection and disease monitoring. Circulating tumor cells (CTCs) have emerged as a promising biomarker, yet their detection remains challenging due to their rarity and phenotypic diversity. This study evaluates the Parsortix® system, a microfluidic device designed to enrich CTCs based on size and deformability, using pancreatic cancer cell lines. As increasing evidence indicates that during epithelial to mesenchymal transition (EMT) a cell’s deformability increases, we evaluated to what extent the Parsortix® system was biased towards epithelial cancer cells. First, the EMT stage of three pancreatic cancer cell lines, CAPAN-1, PANC-1 and MIA PaCa-2, was assessed using immunocytochemistry, flow cytometry and proteomics. CAPAN-1 cells were classified as epithelial, MIA PaCa-2 cells exhibited a mesenchymal-like phenotype, and PANC-1 cells demonstrated a hybrid phenotype. Then, by spiking these cells into blood samples, we determined the Parsortix® system’s ability to recover the cancer cells. Our results indicated that epithelial and hybrid phenotypes are more efficiently captured (62.6 ± 18.5% and 65.4 ± 11.1%) than mesenchymal cancer cells (32.8 ± 10.2%). To confirm these findings, spike-in experiments were repeated using an EMT inducible cell line. Again, significantly lower recovery rates were found for the cells in a mesenchymal-like state (31.5 ± 6.4%) compared to those in an epithelial state (47.56 ± 7.2%). In conclusion, the Parsortix® system may underestimate the presence of mesenchymal CTCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da39e1701a00ff88015fb62ba16d3ce2af9aae2e" target='_blank'>
              Biases in the Parsortix® system observed with pancreatic cancer cell lines
              </a>
            </td>
          <td>
            N. Vandenbussche, Renske Imschoot, Béatrice Lintermans, Lode Denolf, Joachim Taminau, Charlotte Fieuws, G. Berx, Kris Gevaert, Kathleen B M Claes
          </td>
          <td>2024-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="

 Background: ENKTCL is characterized by heterogeneity in genomic mutations, EBV-encoded gene expression, and host HLA-variants conferring differing susceptibility to viral genes. However, the cellular architecture of the ENKTCL tumor microenvironment (TME) and its potential association with sensitivity to chemotherapy across diverse racial and ethnic groups globally remain poorly understood.
 Aim: We sought to decipher the heterogeneous landscape of ENKTCL TME in diagnostic patient samples at the cellular level and in situ to identify key cell populations, their spatial distribution within the TME, and cell-to-cell interactions that correlated with chemotherapy response.
 Methods: We conducted high-resolution, in situ spatial transcriptomic profiling using the 10x Genomics Xenium platform on 12 formalin-fixed, paraffin-embedded (FFPE) ENKTCL tumor samples. We designed a custom ENKTCL-specific 480 target gene panel through systematic analyses of publicly available clinical, genomic, and transcriptomic data. Specific probes for ~20 genes related to EBV infection and viral replication were also incorporated to capture viral dynamics and effects on the TME. Importantly, the cohort incorporated patients from distinct ethnic backgrounds across the United States (n=7) and South America (n=5) to enable a more heterogeneous representation of this disease. Within the cohort, 5 cases were male and 7 female, and the median age of diagnosis was 52 (range, 25-71). The median Ann Arbor stage at diagnosis was I (range, I-IV). Of the 12 cases, 11 had nasal, and 1 case had cutaneous involvement as the primary site of disease. Overall, 6 patients had advanced stage, and 6 patients had localized disease. Based on the PINK score, patients were identified as low risk (n=3), intermediate risk (n=2), and high risk (n=7). The cohort received distinct but heterogeneous frontline treatments that included SMILE (n=4), CHOEP (n=1), DeVIC (n=3), MTX/Asp/Dex (n=1), cisplatinum with radiotherapy (n=2), and palliative radiation therapy (n=1). After frontline treatment, patients exhibited CR (n=5), PR (n=4), or PD (n=3). Following frontline therapy, 1 patient underwent allogeneic stem cell transplant. The median time for relapse was 5 m (range, 4-6). Overall, 3 patients underwent second-line treatment, with GemOx (n=1), Brentuximab Vedotin (n=1), and SMILE (n=1), achieving PR (n=2) and PD (n=1). Third and fourth-line treatments included romidepsin (n=1), palliative chemotherapy (n=1), cytotoxic T-lymphocytes on a clinical trial (n=1) and pralatrexate (n=1).
 Results: In situ transcriptomics enabled an in-depth, unbiased interrogation of the cellular and transcriptional landscape of the ENKTCL TME. After quality control, we identified a total of 2,122,592 cells from all samples in the cohort, spanning 9 major lineages, including tumor-infiltrating lymphocytes such as NK, T, B, plasma, and myeloid cells, as well as fibroblasts, endothelial, epithelial, and mast cells, characterized based on canonical marker expression. Intensive lymphocyte infiltration was observed, with over 700,000 T and NK cells and approximately 650,000 myeloid cells composing the largest compartments. More than 100,000 cells were identified as EBV-infected, including macrophages, NK, NKT, and transformed B cells. Notably, EBV-infected NK, B, and mast cells were more abundant in CR patients, while changes were also observed in macrophages, with higher abundances in the PD and relapsed patient groups. Only the PR group presented significantly elevated abundances of EBV-infected NKT cells, while CD8+ T cells were less prevalent in CR patients. We uncovered numerous differences between groups at the organizational, spatial, cellular, and molecular levels, pointing to specific ENKTCL TME endophenotypes associated with outcomes.
 Conclusion: This study represents the first use of spatial transcriptomics to interrogate the ENKTCL TME in situ. The extensive insights gained from cell co-localization and cell-cell communication signals will prioritize novel biomarkers that underlie responsiveness to chemotherapy. Correlative multi-omic analyses with genomic alterations and HLA specificity in a first-of-its-kind multiregional and highly multiplexed cohort are underway and will be reported at the meeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef2c86e544f6b65915a65faa6c86f9d298fbd4a" target='_blank'>
              Unveiling the Tumor Microenvironment in Extranodal NK/T-Cell Lymphoma (ENKTCL) Patients with in Situ Spatial Transcriptomics
              </a>
            </td>
          <td>
            D. Karagkouni, Maria J. J. Fernandez Turizo, Nikolaos Kalavros, Sean M McCabe, Alexandra W Lenart, Shivani Nanda, Athanasios Ploumakis, Antonella Amaral, Anna B Rider, A. Sohani, Mwanasha H Merrill, E. Jacobsen, M. E. Cabrera, Pablo Villegas, Ioannis S Vlachos, Salvia Jain
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Metastatic disease is the common cause of death in colorectal cancer. To better understand the cellular landscape of colorectal metastasis, we profiled primary tumors and liver, lung, and peritoneal metastases using single-cell RNA sequencing. Our analysis revealed a subset of tumor epithelial cells with a highly plastic transcription profile and, inversely, low expression of stemness markers. Tumor cells with plasticity markers were spatially enriched at the tumor- stroma interface with exposure to tissue microenvironmental factors. Cell-free malignant ascites, which recapitulates the tumor microenvironment, impaired expression of proliferation and stem- like markers in favor of plasticity markers and enhanced migration of tumor cells. Furthermore, Co-culture of colorectal tumor cells with CD11b+ macrophages isolated from malignant ascites from patients was sufficient to recapitulate tumor cell plasticity. Profiling of tumor-associated macrophages identified enrichment for a lipid-handling features and high spatial correlation with plastic tumor cells. Taken together, we demonstrate that tumor-associated macrophages contribute to disease progression through development of pro-malignant features.
 Citation Format: Matthew A. Cottam, Muhammad B. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, Kamran Idrees. Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38b2321f9b6c18e29ebc7faf6bc4bebdd5ef49f8" target='_blank'>
              Abstract B019: Regulatory macrophages contribute to tumor cell plasticity in colorectal cancer metastases
              </a>
            </td>
          <td>
            Matthew A. Cottam, M. Mirza, Emily N. Arner, Marium Siddiqui, Clare Lipscombe, Zaid Hatem, Katy E. Beckermann, K. Idrees
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Malignant growth and survival of cancer is supported through reprogrammed lipid metabolism. In gliomas, genetic alterations can drive lipid metabolic dysregulation revealing therapeutic opportunities to exploit lipid metabolic vulnerabilities within genetically defined subsets of glioma tumors. However, it remains unclear whether inter- and intra-tumoral transcriptomic heterogeneity in gliomas relates to distinct lipid programs and potential dependencies. Here we set out to characterize pan-glioma lipid metabolic heterogeneity through genomic, transcriptomic, and lipidomic profiling of a large and diverse cohort of patient tumor samples. We define key axes of lipid metabolic variability across gliomas and identify relationships between lipid metabolic gene expression programs and glioma lipidomic profiles. Intriguingly, intersection of lipid gene expression signatures with single cell RNA sequencing revealed patterns of lipid metabolic heterogeneity that distinguish neurodevelopmental cellular states of gliomas, mirroring patterns of lipid metabolic diversity across cell types within the normal brain. Consequently, tumors with opposing cellular state enrichment have distinct lipid metabolism and environmental dependencies. Tumors enriched for Radial Glia-like states have high capacity for de novo fatty acid synthesis while tumors enriched for oligodendrocyte progenitor-like states are dependent on exogenous sources of lipids for survival and growth. These findings connect inter- and intra-tumoral heterogeneity of cellular states to variability in lipid metabolic reprogramming to reveal future avenues for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb17270c6d8bbb278033ffba4c666b5e6ba0ea5" target='_blank'>
              TMET-17. LIPID METABOLIC HETEROGENEITY OF GLIOMA CELLULAR STATES REVEALS ENVIRONMENTAL DEPENDENCIES
              </a>
            </td>
          <td>
            Nicholas A. Bayley, Jenna K. Minami, C. Tse, Henan Zhu, Jennifer Salinas, Cassidy Andrasz, Weihong Yan, Dimitri Cadet, Amir Borujerdpur, Kevin Williams, A. Bhaduri, S. Bensinger, L. Liau, T. Cloughesy, Thomas Graeber, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b389c3620d8a50a335c990feb75f3a4d357ca8e6" target='_blank'>
              Integration of single-cell and spatial transcriptome sequencing identifies CDKN2A as a senescent biomarker in endothelial cells implicating hepatocellular carcinoma malignancy
              </a>
            </td>
          <td>
            Yue Ma, Chenhe Yi, Ning Cai, Jinhong Chen
          </td>
          <td>2024-11-06</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cancer progression poses a clinical hurdle due to tumor heterogeneity that contributes to treatment resistance and immune evasion. While the adaptive immune system can effectively eliminate cancer in certain instances, tumor escape remains challenging. The immune microenvironment plays an important role in this process. One unanswered question is the role of ECM geometry, also known as Tumor Associated Collagen Signatures (TACS), and its effects on tumor immune recognition. In solid malignancies, ECM geometry has been associated with disease stage and T cell infiltration. Specifically, empirically observed ECM topologies are frequently categorized based on fiber arrangement: random fibers (TACS1), circumferentially aligned fibers (TACS2), and radially arranged fibers (TACS3). Even though a clear negative correlation between TACS and patient survival has been established, the specific roles and extent of TACS in T cell-driven cancer evolution remain uncharacterized. The precise mechanisms underlying how TACS influences cell movement are still not fully elucidated, with divergent opinions on whether and how the ECM mediates immune cell infiltration. At present, we currently lack a physical model relating the impact of TACS on the spatial co-evolution between an adaptive immune repertoire and a heterogeneous population of evading cancer cells. Here, we developed the EVO-ACT (EVOlutionary Agent-based Cancer T cell interaction) model, which is based on the Gillespie algorithm, to study the effects of TACS on tumor evolution and dynamical tumor-T cell interactions. Our analysis of tumor-T cell interactions across three TACS types reveals distinct migration patterns, with TACS3 showing the highest efficiency, TACS1 intermediate, and TACS2 the lowest. TACS impacts T cells more than tumors, with stronger chemokine gradients aiding T cell infiltration. Our model suggests that despite TACS2's limited efficiency, it cannot fully impede T cell infiltration. Varied migration efficiencies lead to diverse T cell infiltration patterns and immunoediting levels. TACS3 shows a benefit to immune recognition and higher survival, contrary to lower survival in TACS3 patients observed clinically in breast cancer. Our results indicate that only after introducing phenotypic changes, such as Epithelial-Mesenchymal Transition (EMT), and noting that EMT occurs exclusively after the TACS2-TACS3 progression, can our findings successfully explain the clinically observed trends. Considering mesenchymal tumor cells' ability to upregulate PD-1/PD-L1, we explore TACS-specific tumor evolution's impact on PD-1/PD-L1 inhibitor responses and patient survival. We observe higher tumor escape rates and lower survival in TACS3 compared to TACS2. Thus, TACS3 alone does not account for lower survival in breast cancer; TACS3-associated late-stage cancer byproducts like EMT and elevated checkpoint expression significantly decrease survival.
 Citation Format: Yijia Fan, Jason Tomas George. ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52d9e3eca63d450bccf177de70595f8faf5e6be" target='_blank'>
              Abstract A031: ECM architecture drives distinct evolutionary patterns of tumor immune escape and elimination in breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Jason Tomas George
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer evolution follows complex trajectories involving diverse genomic alterations and clonal dynamics, making it challenging to validate computational methods for single-cell DNA sequencing analysis. Here we present cancerSimCraft, a comprehensive framework for simulating cancer genome data at both clonal and single-cell resolution. cancerSimCraft combines deterministic rules with stochastic processes to model various genomic events including CNVs, SNVs, and WGDs. The framework enables integration of real cancer genome patterns with user-defined parameters, supporting customizable simulation designs that reflect both empirical data and theoretical models. Through systematic benchmarking, we demonstrate cancerSimCraft’s utility in evaluating computational methods under various conditions, particularly focusing on the impact of dataset size and parameter sensitivity. We further demonstrated its application in exploring clonal evolution and mutation patterns through controlled in silico experiments. These comprehensive simulation capabilities make cancerSimCraft a valuable resource for both computational method development and theoretical studies in cancer genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617e592c6c24284ee3c33b23e1a58a1555ca1d22" target='_blank'>
              cancerSimCraft: A Multi-resolution Cancer Genome Simulator with Comprehensive Ground Truth Tracking
              </a>
            </td>
          <td>
            Haijing Jin, Nicholas Navin, Ken Chen
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Modern cancer management comprises a variety of treatment strategies. Immunotherapy, while successful at treating many cancer subtypes, is often hindered by tumor immune evasion and T cell exhaustion as a result of an immunosuppressive tumor microenvironment (TME). In solid malignancies, the extracellular matrix (ECM) embedded within the TME plays a central role in T cell recognition and cancer growth by providing structural support and regulating cell behavior. Relative to healthy tissues, tumor associated ECM signatures include increased fiber density and alignment. These and other differentiating features contributed to variation in clinically observed tumor-specific ECM configurations, collectively referred to as Tumor-Associated Collagen Signatures (TACS) 1–3. TACS is associated with disease progression and immune evasion. This review explores our current understanding of how ECM geometry influences the behaviors of both immune cells and tumor cells, which in turn impacts treatment efficacy and cancer evolutionary progression. We discuss the effects of ECM remodeling on cancer cells and T cell behavior and review recent in silico models of cancer-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9d5f20d0547b05a0b2945d91b671d9fb6945ef" target='_blank'>
              Understanding the interplay between extracellular matrix topology and tumor-immune interactions: Challenges and opportunities
              </a>
            </td>
          <td>
            Yijia Fan, Alvis Chiu, Feng Zhao, Jason T. George
          </td>
          <td>2024-11-07</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Adult gliomas exhibit a high level of heterogeneity presenting a substantial challenge hampering development of targeted therapies. Bulk RNAseq studies have identified a mesenchymal gene expression signature across the glioma continuum, based on which a mesenchymal subtype of glioblastoma (MES-GBM) has been described. However, the mesenchymal phenotype of gliomas remains inconsistently defined and the origin of the mesenchymal signature has been debated). Single cell transcriptomic profiling has allowed resolution of gene expression profiles of tumor and non-tumor cells, leading to recent identification of cancer-associated fibroblasts in gliomas and improved understanding of the mesenchymal gene expression signature.



 We performed longitudinal scRNAseq analysis of organotypic human glioma slices from 8 patients cultured ex vivo for up to 3 weeks.



 We found 2 of the 6 IDHwt glioblastomas exhibit the mesenchymal phenotype based on Consensus Methylation Profiling (NIH) and scRNAseq analysis, annotated as Mes-1 and Mes-2. The gene expression profiles of mesenchymal glioblastomas show limited concordance with that of either the proneural or classical glioblastoma signatures or of cell types in physiological human brain, with Mes-2 deviating more extensively from non-mesenchymal gliomas. Instead, single cell transcriptomics suggests a hyper-secretory phenotype and overlaps broadly with that of cancer-associated fibroblasts (CAFs) and both express FGF2, other growth factors, and cytokines. Notably, Mes-2 cells express minimal to no levels of genes associated with brain cell lineages, such as PTPRZ1 and SCG3, which are abundantly expressed in other glioma subtypes.



 Taken together, our data reveals extended depth of glioma heterogeneity that distinguishes MES-GBM from other glioma subtypes and suggests a phenotypic and lineage linkage between MES-GBM and glioma-associated fibroblasts. Thus, new diagnostic and treatment strategies leveraging auto- and paracrine mechanisms underpinning tumor and non-tumor interactions including disruption of the FGF2-FGFR1 circuitry may apply specifically to MES-GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612fc5e5441e08af69c2465ff558c82864f962e0" target='_blank'>
              TMIC-56. LONGITUDINAL SINGLE CELL RNA SEQUENCING UNVEILS PHENOTYPIC CO-OPTION AND LINEAGE TRAITS OF MESENCHYMAL GLIOBLASTOMA AND GLIOMA-ASSOCIATED FIBROBLASTS
              </a>
            </td>
          <td>
            Jiyong Liang, Longfei Huo, Jihong Xu, Madhuri Saidane, Xiaohua Chen, Katherine Zhang, Pratibha Sharma, Truc Kuo, Lisa M. Norberg, Gregory Buchold, Chetna Wathoo, Malcolm F. McDonald, Frederick F Lang, Ganesh Rao, Vinay K. Puduvalli
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d643827078776ad2a6449f6a7b6358707684af74" target='_blank'>
              Cell-cell interactions mediating primary and metastatic breast cancer dormancy.
              </a>
            </td>
          <td>
            Nicholas A Lenart, Shreyas S Rao
          </td>
          <td>2024-11-25</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a heterogeneous pediatric brain tumor with variable clinical outcomes. Since current treatments facing limitations due to tumor resistance, less than 10% of relapsed patients survive beyond five years, with poorly understood recurrence mechanisms. To obtain cellular diversity and genetic characters in primary and recurrent MB, we conducted comprehensive analyses utilizing the single-cell/nucleus RNA sequencing (sc/snRNA-seq), snATAC-seq and spatial transcriptomics, complemented with bulk RNA sequencing and matched primary-recurrent whole exon profiles. SHH and Group_3 subgroups revealed distinct cellular subsets, including cycling, differentiated and quiescent tumor cells. Recurrent or disseminated MB displayed a dramatically different cellular ecosystem compared to primary tumors, with varying proportions of shared cell types. Local recurrence exhibited a higher enrichment of cycling tumor cells, while differentiated subset was notably present in disseminated lesions, indicating that local recurrences harbored the stronger proliferative capacity, whereas disseminations relatively well differentiated. Additionally, chromosomal alterations were assessed, reflecting distinct genetic subclones, such as chr7q gain and chr11 loss in Group_3 disseminations. Notably, the emergence of a subpopulation termed “high cellular plasticity (HCP)” during MB progression was noted, characterized by dynamic adaptability and stemness properties. As validated, HCP state was associated with increased chromatin accessibility, contributing to tumor recurrence and immune microenvironment remodeling. Functionally, inhibiting markers associated with HCP cells, such as PTPRZ1, efficiently suppressed MB metastasis in preclinical models. In conclusion, our findings fill the critical knowledge gaps in understanding the cellular diversity, chromosomal alterations and biological dynamics in MB recurrence and dissemination, offering potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e99ed55df676c0aad6e2ec0b1d04b50f24de8e" target='_blank'>
              EPCO-02. HIGH CELLULAR PLASTICITY STATE OF HUMAN MEDULLOBLASTOMA LOCAL RECURRENCE AND DISTANT DISSEMINATION
              </a>
            </td>
          <td>
            Hailong Liu, Jiankang Li, T. Jiang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Pre-leukemic disorders, such as MDS or MPN, are well-established models for characterizing disease evolution and transformation into leukemia, allowing the study of clinically-relevant key genetic events such as loss-of-function TET2 mutations. However, the mechanisms by which a TET2-mutant clone gains competitive advantage over its wild-type (WT) counterparts remains unclear due to challenges in modelling cell competition efficiently at scale.
 Here, we report a novel in vitro model of single-cell derived clonal competition, demonstrating the existence of TET2-mutant competitor clones that impair wild-type expansion. Leveraging the recently reported PVA-based expansion cultures (Nature, 2019), we developed a cell competition culture protocol which allows single cells to be grown and the resulting clones to be split.
 To validate the system, single EPCR+ CD45+ Sca-1high CD150+ CD48- (ESLAM Sca+) WT cells were expanded for 14 days, after which clones were split and distributed in equivalent cell numbers across multiple wells. 14 days later, flow cytometric analysis and in vivo functional assays showed that the behavior of the original clone was maintained.
 Using the CD45.1/CD45.2 congenic system to discriminate between WT (CD45.1) and TET2 clones (CD45.2) we set up clonal competition cultures: Single ESLAM Sca+ cells from WT and TET2 knockout mice (Tet2-/-) were isolated and expanded for 14 days. Clones were split and distributed across 36 competition wells, each with 50% WT cells from one clone and 50% Tet2-/- cells from another clone, creating a matrix of 3 x 3 competition cultures alongside a WT and Tet2-/- control for each clone. Flow cytometry analysis of clonal composition revealed the presence of a previously unidentified type of TET2 “super competitor” clone. These super competitors impaired the growth and expansion of wild-type clones, and in some instances, completely depleted their WT counterparts. Albeit rare (1 super competitor identified in approximately every 12 analyzed TET2 competitor clones, n = 192), TET2 super competitors consistently impair or completely suppress the expansion of their wild-type counterparts across all the competition wells.
 Interestingly, the data showed that the HSC frequency within the TET2 super competitor clones was lower than that in WT and TET2 competitor clones, denoting that suppression of WT competitors might be attributed to extrinsic factors, rather than being driven by the HSCs themselves. Transcriptomic analysis of the different clone types highlighted a molecular signature of inflammation and iron metabolism.
 To further explore this phenomenon, we harvested media from the 28-day competition cultures, and performed Data-Independent Acquisition Mass Spectrometry (DIA-MS). In agreement with the RNA-seq data, DIA-MS analysis revealed the secretion of inflammatory molecules into the media by enhanced TET2 super competitor clones, including Ifit3m, Cd14, and C3, which are central to the innate immune response and macrophage function.
 Thus, our data have demonstrated that, while the proportion of phenotypic HSCs within TET2 super competitor clones is reduced, secretion of inflammatory molecules potentiates the expansion of such clones and hinders the self-renewal potential of the WT rival clones.
 In conclusion, following the development of a novel in vitro clonal level competition model, we have identified a novel type of pre-leukemic competitor clone. This enables the investigation of molecular and cellular mechanisms with clinical relevance, to better understand how malignant HSCs enhance self-renewal and suppress normal HSC clones in blood cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ba6d7de81b002e6963a30d3deef66f881c23f2f" target='_blank'>
              A Novel in Vitro Model of Pre-Leukemic Competition Reveals “Super-Competitive” TET2-Mutant Hematopoietic Stem Cell Clones
              </a>
            </td>
          <td>
            Lilia Cabrera-Cosme, Juan Rubio-Lara, Ellie Bennett, Joanna Milek, M. Jassinskaja, Sandy MacDonald, Karen Hogg, Nicola K. Wilson, Berthold Gottgens, David Kent
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The development of T cell receptor-engineered T cells (TCR-T) targeting intracellular antigens is a promising strategy for treating solid tumors; however, the mechanisms underlying their effectiveness remain poorly understood. In this study, we employed advanced techniques to investigate the functional state of T cells engineered with retroviral vectors to express a TCR specific for the NY-ESO-1 157-165 peptide in the HLA-A*02:01 context. Flow cytometry revealed a predominance of naïve T cells. Gene expression profiling using NanoString technology revealed upregulation of genes encoding chemokine receptors CCR2 and CCR5, indicating enhanced migration towards tumor sites. In the SK-Mel-37 xenograft model, these transduced T cells achieved complete tumor eradication. Furthermore, single-cell RNA sequencing (scRNA-seq) conducted 14 days post-TCR T cell infusion provided a comprehensive analysis of the in vivo adaptation of these cells, identifying a distinct subset of CD8+ effector T cells with an NK cell-like gene expression profile. Our findings indicate that NY-ESO-1 TCR-transduced T cells have the potential to mediate dual antitumor effects through both antigen-independent NK-like and antigen-specific CTL-like responses. This study underscores the potential of NY-ESO-1 TCR-T cells as potent tumor-eradicating agents, highlighting the importance of harnessing their versatile functional capabilities to refine and enhance therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c35da049d73d1a38187e30a25bcb51a1113dcd7" target='_blank'>
              Decoding NY-ESO-1 TCR T cells: transcriptomic insights reveal dual mechanisms of tumor targeting in a melanoma murine xenograft model
              </a>
            </td>
          <td>
            A. Alsalloum, S. Alrhmoun, O. Perik-Zavosdkaia, M. Fisher, M. Volynets, J. Lopatnikova, R. Perik-Zavodskii, J. Shevchenko, Julia Philippova, Olga Solovieva, Evgenii Zavjalov, V. Kurilin, H. Shiku, A. Silkov, S. Sennikov
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Recent advancements in in situ molecular profiling technologies, including spatial proteomics and transcriptomics, have enabled detailed characterization of the microenvironment at cellular and subcellular levels. While these techniques provide rich information about individual cells’ spatial coordinates and expression profiles, extracting biologically meaningful spatial structures from the data remains a significant challenge. Current methodologies often rely on unsupervised clustering followed by cell type annotation based on differentially expressed genes within each cluster and most of the time will require other information as the reference (e.g., HE-stained images). This is labor-intensive and demands extensive domain knowledge. To address these challenges, we propose a supervised graph contrastive learning framework, MARVEL. MARVEL is a supervised graph contrastive learning method that can effectively embed local microenvironments represented by cell neighbor graphs into a continuous representation space, facilitating various downstream microenvironment annotation scenarios. By leveraging partially annotated examples as strong positives, our approach mitigates the common issues of false positives encountered in conventional graph contrastive learning. Using real-world annotated data, we demonstrate that MARVEL outperforms existing methods in three key microenvironment-related tasks: transductive microenvironment annotation, inductive microenvironment querying, and the identification of novel microenvironments across different slices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/152ed49b0be6d463a8a4a25e29a66310bcce4454" target='_blank'>
              MARVEL: Microenvironment Annotation by Supervised Graph Contrastive Learning
              </a>
            </td>
          <td>
            Yan Cui, Hongzhi Wen, Robert Yang, Xi Luo, Hui Liu, Yuyin Xie
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Spatial transcriptomics is revolutionizing our understanding of cellular dynamics and interactions in the tumor microenvironment (TME). By mapping distinct cellular populations and their interactions, we can gain valuable insights into tumor behavior. To investigate these dynamics in breast cancer, we conducted Xenium In-situ spatial transcriptomics on twenty formalin-fixed paraffin-embedded (FFPE) samples taken from eight patients undergoing neoadjuvant chemotherapy. These samples have been taken pre-treatment, mid-treatment and post-treatment from four cases with pathological complete response (pCR), and four cases with residual disease (RD) at time of surgery. Our analyses have found distinct cellular composition changes between pCR and RD cases. Patients with RD demonstrated an increase in endothelial, perivascular-like (PVL) and epithelial cells from pre- to post-treatment. This was accompanied by a decrease in myeloid and malignant epithelial cells at mid-treatment. In contrast, patients with pCR demonstrated increases in cancer-associated fibroblasts (CAFs) and myeloid cells, with reductions in plasmablasts and cancer epithelial cells at mid-treatment. This suggests that an increase in CAF and myeloid cells observed between pre-treatment and mid-treatment samples may be predictive of response. We performed ligand-receptor (LR) analyses, which demonstrated diverse myeloid interactions in RD cases and diverse CAF interactions in pCR cases in pre-treatment samples. Mid-treatment samples exhibited a decrease in interactions for CAFs, myeloid and T cells in pCR and RD cases. The most frequently identified CAF LR interactions, as senders, were between TGFB1/TGFB3 and TGFBR1/TGFBR2, while the most frequently identified myeloid LR interactions were between PDCD1LG2 - PDCD1 (encoding PDL2 & PD1), CCL8 - ACKR1 (encoding Atypical Chemokine Receptor 1) and CCL2 - ACKR1. Our findings indicate that the dynamic changes in cellular composition and interactions within the TME differs between responders and non-responders. To further explore these differences, we plan to perform spatially constrained analyses to identify unique cellular niches that may serve as potential biomarkers for informing novel therapeutic strategies.
 Citation Format: Jeremy Mo, Hanyun Zhang, Kate Harvey, Alexander Swarbrick. Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d85f0bd80cb01173f30a8b1b70f6e0a4a64eb5" target='_blank'>
              Abstract C040: Spatial transcriptomics unveils disparities in tumor microenvironment dynamics between breast cancer responders and non-responders
              </a>
            </td>
          <td>
            Jeremy Mo, Hanyun Zhang, K. Harvey, Alexander Swarbrick
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma remains one of the deadliest brain malignancies. First-line therapy consists of maximal surgical tumor resection, chemotherapy, and radiotherapy. Malignant cells escape surgical resection by migrating into the surrounding healthy brain tissue, where they give rise to the recurrent tumor. Gene expression profiling allows glioblastoma tumor cores to be classified into mesenchymal, proneural, and classical subtypes, each with distinct genetic alterations and cellular compositions. In contrast, the adjacent brain parenchyma where infiltrating malignant cells escape surgical resection is less characterized in patients. Using single-cell and multicellular resolution spatial transcriptomics on tissues from both the tumor core and infiltrated brain tissue (n = 11), we compared the transcriptional profiles of malignant cells in these regions. Malignant cells near regions with microvascular proliferation showed increased expression of genes related to vascular homeostasis. Within tumor cores, proneural and mesenchymal subtypes exhibited distinct spatial gene expression patterns, although these differences were reduced in the infiltrated brain tissue, apart from gene expression due to chromosomal alterations specific to the proneural subtype. We identified two transcriptional patterns of brain infiltration, with the dominant pattern showing a transition from mesenchymal to proneural states. This transition was accompanied by increased expression of genes linked to neurodevelopmental pathways and glial cell differentiation. Additionally, one gene module upregulated in infiltrating malignant cells was predictive of poor patient survival and enriched for genes associated with differentiated neuronal cells. Together, our findings provide an updated view of the spatial landscape of glioblastomas and infiltrated brain tissue, furthering our understanding of the malignant cells that invade healthy brain. This insight offers new avenues for targeted therapy after surgical resection of the primary tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11ef6dd4e17522c0f5c98bcdd5e7ccdaddf85f0" target='_blank'>
              ANGI-08. SPATIAL TRANSCRIPTOMIC COMPARISONS OF GLIOBLASTOMA TISSUE REVEAL UPREGULATION OF NEURODEVELOPMENTAL PATHWAYS AND SYNAPTIC GENES IN INFILTRATING TUMOR CELLS
              </a>
            </td>
          <td>
            D. Harwood, V. Pedersen, N. Bager, A. Schmidt, T. Stannius, A. Areškevičiūtė, Knud Josefsen, D. Nørøxe, David Scheie, Hannah Rostalski, M. Lü, A. Locallo, U. Lassen, F. Bagger, J. Weischenfeldt, D. Heiland, Kristoffer Vitting-Seerup, S. Michaelsen, B. Kristensen
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 The organized layered structure of the human neocortex contrasts sharply with the chaotic architecture of malignant CNS tumors. The heterogenous nature of glioblastoma (GBM) poses significant challenges to research and treatment, given that effective strategies require consistent biological patterns. Here, we leverage recent advances in spatial biology, integrating spatially resolved transcriptomics and metabolomics to identify drivers of GBM-specific architectural evolution. We performed spatial metabolomics (MALDI) and transcriptomics (Visium n=43 and MERFISH n=25) on glioblastoma patients, with computational analysis conducted using the SPATA2 R package. Our findings reveal that most spatial recurrent patterns in glioblastoma emerge in a gradient-like organization, particularly influenced by proximity to areas of necrosis and hypoxia. Samples without hypoxic regions demonstrated less spatial organization. Metabolomic and transcriptomic profiling indicated enhanced responses to hypoxia, reduced cell proliferation, and elevated expression indicative of the S phase in cell division near necrotic regions. The vicinity to hypoxic areas was associated with increased extracellular matrix remodeling and epithelial-to-mesenchymal transition, suggesting that the invasive capabilities of GBM cells originate from these hypoxic niches. By further integrating spatially resolved TCR-sequencing, we identified that hypoxic gradients drive defined T cell responses with a higher likelihood of T cell dysfunction within a distance of 235 µm (sd: 132 µm) to hypoxia. This suggests that the presence of necrosis, hypoxia, and hypoxia-associated metabolites could serve as potential predictive markers for T-cell behavior and exhaustion in glioblastoma. Our study offers an in-depth analysis of the influence of two prevalent niches in glioblastoma: necrosis and hypoxia. We reveal that the complex architecture of glioblastoma is not solely dictated by genetic randomness and stochastic occurrences but is significantly influenced by spatial proximity to key factors like necrosis and hypoxia. This insight contributes to an evolving model that seeks to unravel the intricate biology underpinning this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b4118e50fd65f0b3670d52a4f2ef27daee8cbf" target='_blank'>
              BIOS-07. GRADUAL METABOLIC ALTERATIONS RESTRUCTURE THE SPATIAL ARCHITECTURE AND IMMUNITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Kueckelhaus, Jasim Kada-Benotmane, O. Schnell, Jürgen Beck, Giulia Villa, D. Heiland
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 We’ve curated a comprehensive multi-omics database of medulloblastoma (MB) serving as a resource to understand spatiotemporal organization and guide therapeutic strategies. Herein, our analysis encompasses single-nucleus transcriptome (n = 51), chromatin accessibility (n = 46), and spatial transcriptomic profiles (n = 31) from human MB samples and patient-derived xenograft lines spanning four subgroups, augmented by bulk RNA (n = 322), whole-genome short-sequencing (n = 279), and bisulfite sequencing (n = 300) from matched samples. We delineate the malignant cellular hierarchy, identifying MB cancer stem-like cells (MB-CSCs), cycling cells, and more differentiated populations. Gene signatures of different cell subpopulations strongly correlate to clinical outcomes, while spatial character of patient-derived materials emphasizes the geographically heterogeneous nature of MB. Examination of somatic copy number variants (CNVs) and transcriptional signatures at single-nucleus resolution reveals geographically defined subclones harboring distinct CNVs, suggesting a competitive agglomeration of co-existing multi-clones for MB. Alongside the trajectorial transition, we observe distinct chromosomal alterations in apical MB-CSCs compared to more differentiated cells, indicative of extensive subclone-primed spatiotemporal evolution. Further analysis of reagent-gene interactions of subclonal lineages contributes to predicting the druggable candidates for treatment optimization. Integration of spatial architecture data highlights the recapitulation of subclone-determined niches through colocation with identified malignant lineages and their own inflammatory or fibrotic adaptions. Taken together, this comprehensive database, comprising five or six kinds of datatypes on the same MB cases, facilitates combinatorial analyses across different genomic modalities, offering genetic and geographic insights into therapeutic advancements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c132829ee848aad60862b63f0ed4457631be0db6" target='_blank'>
              EPCO-01. FROM HIERARCHY TO GEOGRAPHY: MULTI-CLONES DRIVE THE CELLULAR DIVERSITY AND EVOLUTION OF HUMAN MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Hailong Liu, Jiankang Li, T. Jiang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CDK4/6 inhibitors such as palbociclib block cell cycle progression and improve outcomes for many ER+/HER2- breast cancer patients. Unfortunately, many patients are initially resistant to the drug or develop resistance over time in part due to heterogeneity among individual tumor cells. To better understand these mechanisms of resistance, we used multiplex, single-cell imaging to profile cell cycle proteins in ER+ breast tumor cells under increasing palbociclib concentrations. We then applied spherical principal component analysis (SPCA), a dimensionality reduction method that leverages the inherently cyclical nature of the high-dimensional imaging data, to look for changes in cell cycle behavior in resistant cells. SPCA characterizes data as a hypersphere and provides a framework for visualizing and quantifying differences in cell cycles across treatment-induced perturbations. The hypersphere representations revealed shifts in the mean cell state and population heterogeneity. SPCA validated expected trends of CDK4/6 inhibitor response such as decreased expression of proliferation markers (Ki67, pRB), but also revealed potential mechanisms of resistance including increased expression of cyclin D1 and CDK2. Understanding the molecular mechanisms that allow treated tumor cells to evade arrest is critical for identifying targets of future therapies. Ultimately, we seek to further SPCA as a tool of precision medicine, targeting treatments by individual tumors, and extending this computational framework to interpret other cyclical biological processes represented by high-dimensional data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de25c3330f7fce9f46a524989a6ef4e9398a4c" target='_blank'>
              Spherical Manifolds Capture Drug-Induced Changes in Tumor Cell Cycle Behavior.
              </a>
            </td>
          <td>
            Olivia Wen, Samuel C Wolff, Wayne Stallaert, Didong Li, Jeremy E. Purvis, Tarek M. Zikry
          </td>
          <td>2024-11-21</td>
          <td>Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer has the highest global incidence among all cancers, affecting more than 2 million individuals annually. Despite the availability of new drugs and novel treatment combinations, it is postulated that the incidence and mortality of breast cancer will rise by 40.8% and 51.9% respectively by 2040. Such dire statistics are associated with the clonal evolution of cancer cells that leads to therapeutic resistance and consequent relapse in breast cancer patients. On the other hand, the tumor microenvironment (TME) comprising of tumor cells, cancer-associated immune cells, re-programmed stromal cells, and the extracellular matrix (ECM) creates an immunosuppressive niche facilitating immune evasion. This review focuses on a critical cellular component of the tumor microenvironment, the tumor-associated macrophages (TAMs) in breast cancer immunotherapy. Macrophages are inherently plastic and can convert from an anti-tumor M1 phenotype to a pro-tumor M2 phenotype based on microenvironmental cues. Cancer cells facilitate these cues, allowing the tumor-associated macrophages to gain M2 phenotype and mediate immune evasion. Therefore, knowledge of the distinct role of tumor-associated macrophages in immune evasion can help design therapeutics such as engineered macrophages, M2 targeting drugs, and novel macrophage-mediated drug delivery strategies for long-term survival in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6e4305fc222b5a36786a300bfa76f97eae0c28" target='_blank'>
              Reprogramming the tumor microenvironment – macrophages emerge as key players in breast cancer immunotherapy
              </a>
            </td>
          <td>
            Ana Sami, Afsheen Raza
          </td>
          <td>2024-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Tumor cells undergo metabolic alterations that support their proliferation and simultaneously modify immune populations, resulting in an immunosuppressive tumor microenvironment. This study aims to compare the metabolic expression of immune cells within the tumor microenvironment to those in a healthy environment. However, there is a lack of single-cell transcriptomic data for normal brain cells, necessitating an analysis of different methods to compare immune cells in tumors with those in a healthy brain. To address this, we explored cell-to-cell metabolic variability using public single-cell RNA-seq (scRNA-seq) data of glioblastoma (n=338564), and from immune cells in the brains of post-mortem individuals (n=14177), fetuses (n=40000), and epilepsy patients (n=466), as well as from immune blood cells (n=329762). We employed ssGSEA to generate activity scores for 70 key metabolic pathways from KEGG for each individual cell. Known marker genes were utilized to identify normal cell populations (macrophages, T cells, oligodendrocytes) in datasets lacking cell type annotations. Our results revealed that blood cells exhibited high metabolic activity in macrophages but showed moderate activity in other cell types. Immune blood cells also demonstrated high oxidative phosphorylation (OXPHOS) but low fatty acid activity. Fetal brain cells displayed a lack of heterogeneity and showed high fatty acid activity with low OXPHOS activity. Brain cells from post-mortem individuals displayed heterogeneity but exhibited low metabolic activity across nearly all cells and pathways. Additionally, the hierarchy of pathway activity differed from the other datasets. Epilepsy data revealed a metabolic activity pattern similar to tumor-infiltrating immune cells, with comparable activity hierarchy and heterogeneity, although the sample size was limited for detailed comparison. Understanding the differences between types of datasets is important to choose the most appropriate comparison, allowing us to observe the metabolic alterations in immune cells caused by glioblastoma and to develop better immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b202ba6c76d43caa459c400f6e3ca08ac9d6785c" target='_blank'>
              EPCO-11. BIASES IN HEALTHY BRAIN SINGLE-CELL DATASETS: FINDING THE BEST COMPARISON FOR GLIOBLASTOMA STUDIES
              </a>
            </td>
          <td>
            F. Carcanholo, Renan L. S. Simões, E. Santos, Tathiane Malta
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent evidence suggests epigenetic heterogeneity may be the primary driver of cell phenotype and treatment resistance in glioblastoma (GBM). Epigenetic heterogeneity is defined by alterations of chromatin histone modifications with activating marks such as H3K27 acetylation and repressive marks such as H3K27 trimethylation. Combinations of these histone marks govern the activity of functional genomic elements such as enhancers or promoters. Single-cell (“sc-”) technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we report the development of an optimized method for performing tissue processing and simultaneous single-cell epigenetic and transcriptomic analyses in flash frozen GBM samples.



 Building on our prior work, we developed a new protocol, multiomic single-cell cleavage-under-targets and tagmentation, or “mscCUT&TAG”. We optimized nuclei isolation, cleanup and permeabilization from flash frozen GBM and reaction conditions for the scCUT&TAG portion of the assay. Critically, these changes maintained high-quality, intact nuclei in suspension while limiting well-described technical issues such as nuclei clumping and sample loss. Furthermore, we incorporated the ability to multiplex both patient samples and histone marks.



 We applied 10X scRNA-seq, 10X Multiome (RNA+ATAC-seq), and mscCUT&TAG for five histone marks to the same GBM sample, enabling an integrated evaluation of the tumor’s transcriptome, chromatin accessibility and epigenetic regulation at single-cell resolution. mscCUT&TAG enabled the first assessment of single-cell genome segmentation in GBM. We integrated across modalities to identify specific tumor cell populations, cell-type specific coordinated changes in epigenetics and transcriptome, and define functional regulatory elements.



 We developed a novel multiomic protocol, mscCUT&TAG, that allows for a comprehensive analysis of epigenetic heterogeneity in GBM. We anticipate this protocol will enhance our understanding of fundamental biology and potential therapeutic avenues in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc234be87fcb8f1eb7cc14aaf65833786391806" target='_blank'>
              EPCO-19. MULTIOMIC PROFILING OF EPIGENETIC HETEROGENEITY IN GLIOBLASTOMA AT SINGLE-CELL RESOLUTION USING MULTIOMIC SINGLE-CELL CUT&TAG
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, Eliese Moelker, P. Fecci, A. Patel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc282ab7c49fa5f1e3334e1737972fbc08f7f900" target='_blank'>
              Single cell analysis identified a basal cell transition state associated with the development and progression of bladder cancer
              </a>
            </td>
          <td>
            Yang Li, Pengjie Shi, Yuhong Ding, Zhipeng Yao, Lilong Liu, Junyi Hu, Zhenghao Liu, Jinxu Li, Ke Chen, Yaxin Hou
          </td>
          <td>2024-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) defines a heterogeneous set of neuroepithelial cancers of the posterior fossa. The MB tumor microenvironment (MB-TME) influences tumor progression and therapy response and has emerged as a growing target for novel therapeutic approaches. Prior single-cell analysis has characterized the MB-TME composition but does not reveal the spatial orientation of components. To address this gap, we performed 10X Visium spatial sequencing on sixteen pediatric MB samples obtained at surgical resection with samples representing the four molecular subtypes (WNT, N=1; SHH, N=6; Group 3, N=2; Group 4, N=5) and two matching samples at diagnosis and relapse. Spatial voxel clustering yielded clusters corresponding to malignant, immune, and stromal components of the TME, including tumor-associated astrocytes (TAAs), macrophages (TAMs), and vascular endothelium. Ten distinct clusters emerged representing malignant cell states, defined primarily by programs of cell cycle progression and neuronal differentiation. Notably, these MB cell clusters were differentially abundant between the MB subgroups, highlighting heterogeneity between disease subtypes. Moreover, examining the patterns of cellular spatial distribution in high-risk (HR) patients revealed dense regions of quiescent MB progenitors, while standard-risk patients showed greater heterogeneity within their TME. Proximity-dependent intercellular signaling further highlighted increased signaling by TAAs and vascular endothelium with decreased heat shock response signaling in HR patients. Comparing samples from relapse to diagnosis, TAMs, TAAs, and vascular endothelium were found to constitute a greater proportion of the TME. Subsequent niche-dependent gene expression and pathway analysis for TAAs localized near tumor vasculature in relapsed samples revealed increased expression of metastasis-associated pathways. Collectively, the results of our study enable new insight into the heterogeneity of the MB-TME, with close spatial association of quiescent MB progenitors correlating with high-risk clinical features. Quiescent MB progenitors have been documented to confer resistance through cellular dormancy, and identifying its molecular drivers may reveal potential targets for anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab43977fead30af3a35caa1b5f09c7a002ddad" target='_blank'>
              TMIC-04. SPATIAL SEQUENCING OF MEDULLOBLASTOMA REVEALS BIOLOGICAL NICHES CORRELATED WITH HIGH-RISK FEATURES AND RELAPSE
              </a>
            </td>
          <td>
            Marina E. Michaud, Franklin Chien, Mojtaba Bakhtia, M. Snuderl, Chanel Schroff, Matthew Schneiderjan, Tobey J. MacDonald, Manoj Bhasin
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte-SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomic analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in 3-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8536f19d4a9f802b3a9ad739c338954491fd6245" target='_blank'>
              Deciphering Colorectal Cancer-Hepatocyte Interactions: A Multiomic Platform for Interrogation of Metabolic Crosstalk in the Liver-Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E. Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Introduction Macrophages are essential in maintaining homeostasis, combating infections, and influencing the process of various diseases, including cancer. Macrophages originate from diverse lineages: Notably, tissue‐resident macrophages (TRMs) differ from hematopoietic stem cells and circulating monocyte‐derived macrophages based on genetics, development, and function. Therefore, understanding the recruited and TRM populations is crucial for investigating disease processes. Methods By searching literature databses, we summarized recent relevant studies. Research has shown that tumor‐associated macrophages (TAMs) of distinct origins accumulate in tumor microenvironment (TME), with TRM‐derived TAMs closely resembling gene signatures of normal TRMs. Results Recent studies have revealed that TRMs play a crucial role in cancer progression. However, organ‐specific effects complicate TRM investigations. Nonetheless, the precise involvement of TRMs in tumors is unclear. This review explores the multifaceted roles of TRMs in cancer, presenting insights into their origins, proliferation, the latest research methodologies, their impact across various tumor sites, their potential and strategies as therapeutic targets, interactions with other cells within the TME, and the internal heterogeneity of TRMs. Conclusions We believe that a comprehensive understanding of the multifaceted roles of TRMs will pave the way for targeted TRM therapies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad16516b6455af131bbe30bbcb75f6a839b39296" target='_blank'>
              Tissue‐Resident Macrophages in Cancer: Friend or Foe?
              </a>
            </td>
          <td>
            J. Chi, Qing-cui Gao, Dan Liu
          </td>
          <td>2024-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) often exhibit stem-like attributes that depend on an intricate stemness-promoting cellular ecosystem within their niche. The interplay between CSCs and their niche has been implicated in tumor heterogeneity and therapeutic resistance. Normal stem cells (NSCs) and CSCs share stemness features and common microenvironmental components, displaying significant phenotypic and functional plasticity. Investigating these properties across diverse organs during normal development and tumorigenesis is of paramount research interest and translational potential. Advancements in next-generation sequencing (NGS), single-cell transcriptomics, and spatial transcriptomics have ushered in a new era in cancer research, providing high-resolution and comprehensive molecular maps of diseased tissues. Various spatial technologies, with their unique ability to measure the location and molecular profile of a cell within tissue, have enabled studies on intratumoral architecture and cellular cross-talk within the specific niches. Moreover, delineation of spatial patterns for niche-specific properties such as hypoxia, glucose deprivation, and other microenvironmental remodeling are revealed through multilevel spatial sequencing. This tremendous progress in technology has also been paired with the advent of computational tools to mitigate technology-specific bottlenecks. Here we discuss how different spatial technologies are used to identify NSCs and CSCs, as well as their associated niches. Additionally, by exploring related public data sets, we review the current challenges in characterizing such niches, which are often hindered by technological limitations, and the computational solutions used to address them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548e0b0064280f307e085a25804779bf2b1c655a" target='_blank'>
              Deciphering normal and cancer stem cell niches by spatial transcriptomics: opportunities and challenges.
              </a>
            </td>
          <td>
            Hirak Sarkar, Eunmi Lee, Sereno Lopez-Darwin, Yibin Kang
          </td>
          <td>2024-11-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults, characterized by poor overall survival and an urgent need for more effective treatment strategies. Macroscopic heterogeneity, as observed through MR imaging, divides the tumor into distinct regions: a tumor-cell dense contrast-enhancing tumor core and a non-contrast-enhancing infiltration zone that appears hyperintense on T2/FLAIR imaging. The infiltration zone contains both infiltrating tumor cells and non-malignant cells from the local brain microenvironment. Although genomic and transcriptomic differences between the tumor core and the infiltration zone have been reported, functional differences, such as differential drug sensitivities, remain largely unknown.



 To investigate this, we collected region-specific glioblastoma patient tissue samples from both the contrast-enhancing tumor core and the non-contrast-enhancing but 5-aminolevulinic acid (5-ALA) fluorescent infiltration zone during surgery. Using pharmacoscopy, a microscopy-based single-cell drug screening platform, we assessed drug responses across regions and patients.



 To determine whether drug responses within a single tumor region were more similar compared to responses across different regions we collected multiple tissue samples from each tumor region in seven patients. We identified region-specific drug responses for temozolomide and lomustine, among other drugs, in some of the patients. However, drug sensitivities were not consistent within the same region across different patients. Thus, across patients, the region-specific drug responses were obscured by patient heterogeneity. To address intra- and inter-tumor drug response heterogeneity, we developed a complementarity score based on region-specific drug response data of 60 drugs across 23 paired samples. The complementarity score identified promising drug combinations that demonstrated strong anti-tumor activity across tumor regions and patients.



 Monotherapy approaches have largely failed in glioblastoma, despite extensive molecular profiling efforts aimed at identifying tumor-specific vulnerabilities. Identifying drug combinations that synergistically target the locoregional heterogeneity of glioblastoma may open up new avenues of personalized therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bab37d6465b9764681d6770f3a30b2ef1d8dd5" target='_blank'>
              DDDR-05. SPATIALLY RESOLVED DRUG SENSITIVITY PROFILING IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, Sohyon Lee, R. Wegmann, J. Mena, F. Vasella, J. Friesen, Luca Regli, E. Le Rhun, A. Zeitlberger, M. Neidert, N. Tatari, G. Hutter, Michael Weller, B. Snijder, T. Weiss
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Diffuse infiltration is an aggressive feature of high-grade gliomas with survival implications. The contribution of crosstalk between non-neoplastic and neoplastic cells to tumor infiltration remains largely understudied due to the lack of profiling techniques that retain spatial information. Spatial multi-omic profiling is a promising approach to comprehensively analyze transcript-omics, prote-omics and metabol-omics on the same tissue section while preserving information about the spatial organization of cells. Integration of these spatial studies allows for inferring the consequences of complex cell-cell communication underlying tumor infiltration. We hypothesize that the glioma edge is enriched with pro-infiltration ligands-receptors driving tumor infiltration. Strategies that disrupt these ligand-receptor networks may suppress glioma infiltration and improve clinical outcomes.



 We utilized a glioma tissue micro-array (TMA) to establish the neoplastic and non-neoplastic heterogeneity in the GBM infiltration edge. Each TMA slide consists of pathologist annotated tumor core and edge samples of Glioblastoma (IDH Wildtype, WHO Grade 4; n=10), Diffuse Astrocytoma (IDH Mutant, WHO Grade 3; n=3), Oligodendroglioma (IDH mutant, WHO Grade 2; n=5) and 2 non-brain control samples. We performed spatial multi-omics profiling on adjacent sections of the TMA using 10x Xenium spatial transcriptomics, imaging mass cytometry (IMC) and mass spectrometry imaging (MSI). Integration, visualization, and quantification of the spatial data was done on VisioPharm.



 In GBM, we identified candidate mRNA transcripts and proteins for ligands enriched at the infiltrating edge (compared to tumor core; p<0.0001) that correlated with a poor progression free survival (PFS; r2=0.22). These candidate ligands were also significantly enriched at the edge of oligodendroglioma WHO Grade 2 and astrocytoma WHO Grade 3.



 Spatial multi-omics profiling on a TMA consisting of Glioma WHO grades 2-4 identified differentially expressed and targetable pro-tumor infiltration ligands associated with lower PFS in GBM. Functional studies to uncover the role of metabolites enriched at the glioma edge for the expression of the identified pro-infiltration ligands are ongoing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf0234947ec5e04cf1bc6ff024f2ae5bb086359" target='_blank'>
              EPCO-07. INTEGRATING SPATIALLY RESOLVED MULTI-OMICS DATA TO UNCOVER DYSFUNCTIONAL METABOLISM DRIVEN NETWORKS THAT ENHANCE INFILTRATION OF DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            Yanxia Ma, Nhat Nguyen, Sanjay Singh, Akshay V. Basi, Duncan Mak, Javier Gomez, Jared Burks, Erin Seely, Frederick F Lang, C. Ene
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The mechanisms that regulate cancer dormancy remain poorly understood. Using a mouse model of resectable pancreatic adenocarcinoma (PDAC), we identified Dec2 as a gene that was upregulated in metastatic dormant tumor cells. Deletion of Dec2 from tumor cells substantially increased mouse survival after resection due to an immune-mediated mechanism as the survival benefit was abrogated in immunodeficient conditions. Dec2 promoted immune evasion by repressing multiple components of the MHC-I dependent antigen presentation pathway in tumor cells. Dec2 is a regulator of circadian rhythms, and we found several components of the antigen presentation pathway oscillated in a circadian manner that was lost upon deletion of Dec2. Moreover, T-cell mediated tumor cell killing varied depending on the time of day. We suggest that lowered MHC-I presentation of antigens during rest phase is a natural effect of the circadian clock, which is exploited by Dec2-overexpressing pancreatic tumors to evade the immune system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42eeeb16fbe1bfd7a981ff5b86b3bf2e08aeb4a" target='_blank'>
              The circadian gene Dec2 promotes pancreatic cancer dormancy by regulating tumor cell antigen presentation to facilitate immune evasion
              </a>
            </td>
          <td>
            Chris R. Harris, Lan Wang, C. Dudgeon, Orjola Prela, Juliana Cazarin de Menezes, Ching-Hua Shih, Christina Davidson, A. Casabianca, Subhajyoti De, Wade Narrow, Jennifer Becker, V. Balachandran, P. Grandgenett, Michael A. Hollingsworth, Jean L. Grem, Minsoo Kim, Yeonsun Hong, Scott Gerber, Paula M. Vertino, Chongfeng Gao, Anna Repesh, Zachary L. Klamer, Yansheng Hao, Brian J Altman, Brian Haab, Darren R. Carpizo
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Summary Haematopoietic stem and progenitor cell (HSPC) autologous gene therapies are promising treatment for a variety of blood disorders. Investigation of the long‐term HSPC clonal dynamics and other measures of safety and durability following lentiviral‐mediated gene therapies in predictive models are crucial for assessing risks and benefits in order to inform decisions regarding wider utilization. We established an autologous lentivirally barcoded HSPC transplantation model in rhesus macaque (RM), a model offering insights into haematopoiesis and gene therapies with direct relevance to human. Healthy young adult RMs underwent total body irradiation, followed by transplantation of autologous HSPCs transduced with a lentiviral vector containing a diverse genetic barcode library, uniquely labelling individual HSPCs and their progeny. With up to 131 months of follow‐up, we now report quantitative clonal dynamics, characterizing the number, diversity, stability and lineage bias of hundreds of thousands of HSPC clones tracked in five RMs. We documented long‐term stable and multi‐lineage output from a highly polyclonal pool of HSPCs. Clonal succession after stable haematopoietic reconstitution was minimal. There was no evidence for accelerated acquisition of acquired somatic mutations following autologous lentivirally transduced HSPC transplantation. Our results provide relevant insights into long‐term HSPC behaviours in vivo following transplantation and gene therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572ce898b59869c94f73c33f1fe01408d0f30361" target='_blank'>
              Long‐term tracking of haematopoietic clonal dynamics and mutations in non‐human primate undergoing transplantation of lentivirally barcoded haematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Rohan V. Hosuru, Jack Yang, Yifan Zhou, A. Gin, T. Hayal, So Gun Hong, Cynthia E. Dunbar, Chuanfeng Wu
          </td>
          <td>2024-11-10</td>
          <td>British Journal of Haematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Effective immune synapses are a prerequisite for desirable immune surveillance and control of tumour growth. Cell-to-cell interactions form the basis of immune synapses, resulting in either immunoprotected or immunosuppressed manifestations in the tumour brain that could explain glioblastoma (GBM) recurrence. Using state-of-the-art multiplex imaging, spatial statistics, and spatially aware deep learning tools in 14 patient-matching tissues from primary and recurrent GBM, we detected spatial immune networks (SINets). SINets across 105 regions of interest comprising 2.3 million cells show stronger spatial engagement between myeloid and lymphoid lineages at recurrence. SINets’ architecture in primary tumours is associated with relapse timing. Patients, where the SINets are more modular, recur sooner than patients with denser SiNets. In addition, this association presents sexual dimorphism in our cohort (N=3 females, and N=4 males). Phenotype-to-phenotype spatial coexistence also contributes to enhancing phenotype prediction. A spatial-aware deep learning model trained with single-cell data from primary tumours reaches an average balanced accuracy of 0.76 in primary tumour cellular communities unseen during training, compared to a spatial-agnostic model that reached an average balanced accuracy of 0.69. In summary, we detected a distinct spatial engagement in glioblastoma undergoing chemo-radiation treatment. The spatial patterns of coexistence represent ecological niches that contribute to defining cell occurrence and may underpin effective immune synapses. Overall, the spatiotemporal elements provide a newer approach for dissecting glioblastoma’s complexity and discovering opportunities for stratifying patients by harnessing spatial information encoded in multicellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8383db42b59d05ff4b66f525f28516202250c998" target='_blank'>
              TMIC-68. DEEP LEARNING OF THE SPATIAL IMMUNE ENGAGEMENT IN GLIOBLASTOMA UNDER TEMOZOLOMIDE TREATMENT
              </a>
            </td>
          <td>
            Simon P. Castillo, Afrooz Jahedi, Pravesh Gupta, J. Huse, K. Kannan, Yinyin Yuan, Krishna P. Bhat
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Recently, we have shown that primary brain tumors form highly organized, small-world scale-free networks, reminiscent of early networks in neurodevelopment and highly resistant to therapy. Such putative cancer-cell intrinsic neural mechanisms are becoming an increasing focus in the field of Cancer Neuroscience, but it has not yet been shown whether tumors of non-CNS origin also have the ability to form multicellular networks which sustain proliferative signaling and increase resistance to therapy. We observed synchronized calcium activity using longitudinal microscopy both in vivo and in vitro that were reminiscent of gap junction coupled networks. Indeed, dye transfer experiments verified functional coupling of brain metastases cells in vitro. Similarly, inhibition of gap junctions with different pharmacological agents significantly reduced calcium oscillations and tumour proliferation in vitro. Furthermore, gap junctions are significantly upregulated in brain-tropic sublines compared to parental cells. To understand the functional relevance of these gap-junction coupled tumor cell networks, we performed bulk RNA-Sequencing of two melanoma metastasis models in monoculture under gap junction inhibition and control: We observed reduced calcium communication and a concordant downregulation of neurodevelopmental and synaptic pathways within the tumor cells upon network disconnection, highlighting a potential recapitulation of neural-like features in metastases to the brain. To investigate the therapeutic potential of targeting these networks, we treated mice with brain metastases with two brain-penetrant gap junction blockers and observed significantly reduced tumor burden. Finally, we investigated the presence of heterotypic networks between brain metastases cells and the brain microenvironment with our SR101 dye transfer pipeline. Contrary to our findings in primary brain tumors, no gap junction connections to cells of the brain microenvironment in those models with reduced tumor growth upon network inhibition could be detected, supporting a cancer cell intrinsic gap junction mediated network and highlighting a selectivity of these homogenous tumor networks and potential therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f760d30de9fc9f379968ef3f9925cc574445048" target='_blank'>
              CNSC-27. INTRATUMORAL CANCER CELL NETWORKS IN BRAIN METASTASES
              </a>
            </td>
          <td>
            Niels Olshausen, N. Hebach, Joana S Schlag, Calvin Thommek, Yvonne Yang, D. Hoffmann, C. Mayer, D. Westphal, M. Breckwoldt, Wolfgang Wick, F. Winkler, M. Karreman
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is associated with a high risk of distant metastasis, particularly to the lung and liver. Both intrinsic characteristics of cancer cells and the tumor microenvironment (TME) influence TNBC growth and metastasis. The TME, comprising stromal, immune, and endothelial cells, exhibits heterogeneity both between and within cell types. Identifying the molecular characteristics of TME cells that affect cancer progression is crucial. We aim to investigate this using single-cell RNA sequencing of patient-derived xenograft (PDX) models to explore TME cells and genes involved in TNBC metastasis. We established PDX models by transplanting tumor tissues from 26 TNBC patients into immunodeficient mice. These models were categorized based on their metastatic potential and patient outcomes: non-metastatic (n=5) and metastatic (n=4). Single-cell RNA sequencing of nine PDX tumors revealed significant differences in gene expression of murine stromal cells between metastatic and non-metastatic models. We identified ten genes (Serpinb2, Spp1, Tnc, Thbs1, Timp1, Il11, Mt2, Crabp1, Cck, Mt1) that were highly expressed in the stromal cells of metastatic PDX models. To test if TNBC cells influence these fibroblast genes, we exposed NIH3T3 fibroblasts to conditioned media from the 4T1 TNBC cell line. In three independent experiments, only Crabp1 expression in NIH3T3 cells was consistently increased by 4T1- conditioned media. CRABP1, a retinoic acid-binding protein, is known for its role in differentiation and proliferation by regulating MAPK signaling. Although Crabp1-positive cancer-associated fibroblasts have been noted, its role in TNBC metastasis was previously unexplored. We investigated CRABP1 function using a CRABP1 knockdown NIH3T3 cell line. CRABP1 knockdown led to reduced invasion and migration of 4T1 cells in trans-well assays and decreased invasiveness of 4T1 spheroids in a collagen matrix. Additionally, CRABP1 knockdown NIH3T3 cells showed reduced proliferation in vitro and fewer alpha-SMA positive cancer-associated fibroblasts in co-injected 4T1 tumors in BALB/C mice. In summary, CRABP1, identified through single-cell RNA sequencing of stromal cells in TNBC PDX models, is a potential regulator of TNBC metastasis, affecting cancer cell migration, invasion, and fibroblast proliferation.
 Citation Format: Woohang Heo, Yujeong Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, Hyeong-Gon Moon. Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55fcf53482b3293e557961f04bd5df55bfd4945" target='_blank'>
              Abstract A034: Single cell RNA sequencing of triple negative breast cancer patient-derived xenograft model identifies CRABP1 of cancer-associated fibroblast as a key regulator of breast cancer metastasis
              </a>
            </td>
          <td>
            W. Heo, Y. Her, Sieun Yang, Dakyung Lee, Rokhyun Kim, Jong-Il Kim, H. Moon
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="

 Introduction:
 Acute myeloid leukemia (AML) is a heterogeneous disease characterized by the accumulation of abnormal hematopoietic stem and progenitor cells. TP53 mutations, present in 8-30% of AML cases, are associated with high risk of relapse and poor prognosis. To investigate the resistance mechanisms, we analyzed 52 longitudinal single-cell RNA sequencing (scRNA-seq) samples from 26 TP53 mutant (TP53-mut) patients. In parallel, we employed single-cell DNA and protein sequencing (scDNA+protein seq) and single-cell mass cytometry (CyTOF) in 10 longitudinal samples from 5 TP53-mut AML patients. This approach allowed us to establish a multiomic (genomic, transcriptomic, proteomic) link in the context of TP53-mut AML to identify potential resistance mechanisms.
 Methods: For scRNA-seq, we sequenced a total of 97,980 cells from 26 paired samples before and after treatment, 19 of which were TP53-mut. For single-cell DNA+protein analysis, we sequenced a total of 44,550 cells from these 10 samples. Samples with at least 70% viability were stained with the Total-Seq™-D Human Heme Oncology Cocktail, containing 45 oligo-conjugated antibodies for surface markers relevant to AML. For mutation analysis, we utilized a validated AML custom panel (Morita et al., 2020) covering recurrent mutations in 37 genes. For CyTOF, we multiplexed samples and stained them with 50 antibodies characterizing lineage, apoptosis, and signaling pathways. We successfully constructed a machine learning model for integrating scDNA+protein and CyTOF using our gradient-boosting algorithm established pipelines and were able to predict DNA mutation status in CyTOF data with over 95% accuracy.
 Results:
 To elucidate the mechanisms underlying TP53-mut AML persistence, we conducted a comprehensive analysis integrating single-cell transcriptional, DNA, and proteomic profiling. Firstly, we developed an advanced pipeline for blast calling in AML, which combined (i) CNV inference using Numbat, (ii) automated cell type annotation (CellTypist), and (iii) surface marker validation using clinical flow cytometry lab gating strategies. This pipeline demonstrated a high correlation between inferred and clinically reported blasts in diagnostic samples (r2= 0.745) exceeding previously reported correlations in scRNA AML studies. However, it yielded poor correlation in post-treatment samples (r2= 0.533).
 Clonal evolution analysis using scDNA+protein revealed that TP53-mut clones undergo monocytic differentiation as an escape mechanism. More specifically, the resistant clone shifted from CD34+ blasts pre-treatment to CD16+ monocytic cells post-treatment. Pseudotime analysis of CyTOF data showed consistent findings: post-treatment AML cells lose stemness and express monocytic differentiation markers, suggesting monocytic differentiation as a mechanism of resistance.
 Proteomic profiling of resistant cells in TP53-mut AML blasts, annotated in the CyTOF dataset using machine learning models from scDNA+protein data, revealed increased expression of Ki67, BCL-XL, CD68, and CD11b, and decreased expression of HLA-DR and CD34, suggesting an upregulation of anti-apoptotic mechanisms and escape from immune surveillance. TP53 mutation prediction correlated with accumulation of p53 protein, further validating our model performance. Unexpectedly, clonal architecture analysis showed TP53 mutations in lymphocyte subpopulations. For instance, Patient 1 had TP53 mutations in 16.5% of CD4+ T cells, 35.9% of CD8+ T cells, and 19.2% of NK cells. The presence of TP53 mutations in T- and NK cells has not previously been observed. We demonstrate that TP53 mutations are causative of T-cell exhaustion (L Li, et al., M Andreeff, ASH 2024). Additionally, we have identified and validated MYC (Y Nishida et al., M Andreeff, ASH 2024) and PLK4 (E Ayoub et al., M Andreeff, ASH 2022) as potential targets for TP53-mut AML, and we are currently investigating their translational potential in TP53-mut AML.
 Conclusion:
 This study establishes a genotype-phenotype connection through single-cell multiomic profiling of TP53-mutated AML and maps the clonal evolution and immunophenotypic dynamics during treatment. We propose monocytic differentiation and T-cell exhaustion due to the presence of TP53 mutations in T- and NK cells as a potential mechanism of resistance, providing new insights into AML persistence and therapy escape.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77fabbbcc45ac650e21203effd221f236f275c5d" target='_blank'>
              Single-Cell Multiomics Unveils Venetoclax-Resistant Monocytic Differentiation and Immune Evasion in TP53 Mutant AML Clones
              </a>
            </td>
          <td>
            Edward Ayoub, David Lehotzky, Li Li, M. Muftuoglu, Yuki Nishida, V. Mohanty, Jake S Leighton, Tallie Patsilevas, Mahesh Basyal, Andrea D Bedoy, Torsten Haferlach, Koichi Takahashi, Ken Chen, Nicholas Navin, G. Issa, G. Getz, Michael Andreeff
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) presents diverse malignancies with varying biological and clinical behaviors, driven by an interplay between cancer cells and tumor microenvironment. Deciphering these interactions is crucial for personalized diagnostics and treatment. This study explores the prognostic impact of tumor proliferation and immune response patterns, assessed by computational pathology indicators, on breast cancer-specific survival (BCSS) models in estrogen receptor-positive HER2-negative (ER+HER2–) and triple-negative BC (TNBC) patients. Materials and methods Whole-slide images of tumor surgical excision samples from 252 ER+HER2– patients and 63 TNBC patients stained for estrogen and progesterone receptors, Ki67, HER2, and CD8 were analyzed. Digital image analysis (DIA) was performed for tumor tissue segmentation and quantification of immunohistochemistry (IHC) markers; the DIA outputs were subsampled by hexagonal grids to assess the spatial distributions of Ki67-positive tumor cells and CD8-positive (CD8+) cell infiltrates, expressed as Ki67-entropy and CD8-immunogradient indicators, respectively. Prognostic models for BCSS were generated using multivariable Cox regression analysis, integrating clinicopathological and computational IHC indicators. Results In the ER+HER2– BC, multivariable Cox regression revealed that high CD8+ density within the tumor interface zone (IZ) (HR: 0.26, p = 0.0056), low immunodrop indicator of CD8+ density (HR: 2.93, p = 0.0051), and low Ki67-entropy (HR: 5.95, p = 0.0.0061) were independent predictors of better BCSS, while lymph node involvement predicted worse BCSS (HR: 3.30, p = 0.0013). In TNBC, increased CD8+ density in the IZ stroma (HR: 0.19, p = 0.0119) and Ki67-entropy (HR: 3.31, p = 0.0250) were independent predictors of worse BCSS. Combining these independent indicators enhanced prognostic stratification in both BC subtypes. Conclusions Computational biomarkers, representing spatial properties of the tumor proliferation and immune cell infiltrates, provided independent prognostic information beyond conventional IHC markers in BC. Integrating Ki67-entropy and CD8-immunogradient indicators into prognostic models can improve patient stratification with regard to BCSS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e50ee2898c062787fae58412a5fdbeb3e64a19" target='_blank'>
              Spatial distributions of CD8 and Ki67 cells in the tumor microenvironment independently predict breast cancer-specific survival in patients with ER+HER2– and triple-negative breast carcinoma
              </a>
            </td>
          <td>
            Dovile Zilenaite-Petrulaitiene, A. Rasmusson, Ruta Barbora Valkiuniene, A. Laurinavičienė, Linas Petkevičius, A. Laurinavičius
          </td>
          <td>2024-11-22</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Intratumoral heterogeneity promotes tumor progression, treatment resistance, and recurrence in glioblastoma (GBM), and the cellular tumor core and infiltrating edge represent transcriptionally distinct cellular ecosystems. In this study, we characterize and validate the molecular differences across the core-edge spatial axis in GBM by applying whole-transcriptome GeoMx Digital Spatial Profiling (DSP) to 5 IDH-wildtype tumors. Three 3-micron tissue cores were extracted from each tumor and arranged into a tissue microarray (TMA). Core and edge regions of interest (ROIs) were chosen through coregistration of H&E and immunofluorescent TMA scans. Following DSP, data for 44 ROIs (30 core and 14 edge) and 8530 RNA targets were bioinformatically analyzed. Single-sample gene set enrichment analysis aligned the DSP core and edge regional signatures to those of publicly annotated datasets, substantiating our profiling methodology. Unsupervised analyses revealed ROI clusters unique to each tumor specimen for both pre- and post-normalized data. To mitigate a potential patient-level effect in subsequent differential gene expression and pathway analyses, we leveraged a linear mixed-effect model using core versus edge as a fixed effect and tumor specimen as a random effect. Our analyses showed upregulation of epigenomic markers in core ROIs and neuronal markers in edge ROIs. Further, core and edge signatures were incongruent with the canonical GBM subtypes (Mesenchymal, Classical, Proneural) and malignant cell states (OPC-like, NPC-like, AC-like, MES-like). Cellular deconvolution analysis highlighted differential enrichment of neoplastic cells in core ROIs and neuroglia (astrocytes and oligodendrocytes) in edge ROIs. In summary, we leverage spatial profiling to further validate the geographic molecular heterogeneity of GBM, which is characterized by divergent neuronal programs within the infiltrating edge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7cc77c63633549b85f3d94e4ad3932eaf68377d" target='_blank'>
              EPCO-48. RESOLVING INTRATUMORAL TRANSCRIPTIONAL HETEROGENEITY ACROSS THE CORE-EDGE SPATIAL AXIS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Adam Luo, Kelly Casella, Ezra G. Baraban, Calixto-Hope G. Lucas, Eugene Shenderov
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Background. Virtually every MM patient transitions through the asymptomatic precursor conditions MGUS and SMM prior to developing MM. This provides a window for intervention to prevent progression to frank malignancy, and also exciting opportunities to investigate mechanisms underlying such progression. In contrast to solid tumours, relatively little is known about how the immune system is shaped by, and in turn shapes, MM through progression from precursor conditions to frank malignancy.
 Aim. To define the contribution of tumour features to T cell differentiation during disease evolution from precursor disease to MM.
 Methods. We generated the “pre-myeloma atlas”, the largest single-cell RNA sequencing (scRNA-seq) dataset of precursor disease and MM to date. It encompassed >1,000,000 cells from 295 newly-generated and published scRNA-seq samples from MM (n=54 patients), precursor conditions (n=78), and non-cancer controls (n=102). We enhanced the atlas through including single-cell T cell receptor (TCR) sequencing (42% samples), bone marrow and peripheral blood sampling, and modelling clinical data including age and tumour burden. Malignant clones (tumour cells) were identified by clonal immunoglobulin expression and scored with pan-cancer transcriptional pathways. Viral-reactive TCRs were acquired from VDJdb and neoantigen-reactive TCRs predicted with NetMHCpan4.1 and TEINet.
 Results. Within the pre-myeloma atlas, we identified CD4+ and CD8+ T cell subsets across the spectrum of differentiation from naïve (SELL+CCR7+) to terminal memory (GZMB+) phenotypes. Notably, we identified exhausted T cells in only a single MM patient (1 of 54), suggesting T cell exhaustion is not a pervasive feature of MM. Comparing controls with precursor and symptomatic patients, we found T cell composition was comparable through disease evolution. However, the abundance of T cell subsets was altered: in MM, earlier differentiated subsets such as naïve CD8+ were depleted relative to controls (P<0.001) whereas terminal memory subsets were enriched (P<0.003). Principal component analysis of T cell composition captured this shift from early to terminal subsets, which we termed T cell skewing. T cell skewing rose with disease severity and was enhanced in SMM and MM independent of patient age (SMM P=0.004; MM P<0.001). Longitudinal sampling of precursor patients who subsequently progressed revealed T cell skewing occurs longitudinally at progression. Together, this suggests exaggerated T cell memory differentiation occurs in MM.
 TCR repertoire clonality (indicative of T cell proliferation in response to antigen) was higher in MM relative to controls (P=0.06) and corrected with T cell skewing (R=0.71, P<0.001). Additionally, the frequency of TCR clones possessing similar antigen-recognition domains (indicative of shared antigen-specificity) rose between SMM and MM (P<0.002) and correlated with T cell skewing (R=0.4, P=0.02). Importantly, in silico analyses of T cell specificity suggested (1) TCR clones annotated as viral-reactive were not associated with T cell skewing, and (2) TCR clones predicted to bind autologous tumour neoantigens occupied terminal memory states. Together, these results suggest antigen, possibly tumour in origin, drive T cell differentiation in MM.
 Finally, to interrogate mechanisms of tumour-immune cross-talk, we compared T cell and tumour features. We observed a positive correlation between T cell skewing with paraprotein (R=0.43, P=0.04), that serves as a proxy for tumour burden. Additionally, effector T cells (IFNG+TNF+) were associated with tumour marrow infiltration (aspirate % CD138+; R=055, P<0.001). When analysing the transcriptional activity of tumour cells, we found effector T cells also correlated with cell death-associated pathways (R=0.66, P<0.001). We noted sharing of TCRs between these two T cell subsets, suggesting a differentiation process between these subsets possibly regulated by tumour-intrinsic processes and tumour burden.
 Conclusion. Our results show that, unlike solid cancers, MM is not associated with T-cell exhaustion and instead defined by a pattern of T-cell differentiation resembling antigen-driven terminal memory differentiation. T cell associations with tumour burden and antigen could suggest anti-tumour immunity drives a novel form of cancer-associated T-cell differentiation in MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729020d6ba64084dfb7bfe1195058f87d0ac8af9" target='_blank'>
              Tumour-Intrinsic Features Shape T-Cell Differentiation through Myeloma Disease Evolution
              </a>
            </td>
          <td>
            K. Foster, Elise Rees, L. Ainley, Eileen M Boyle, L. Lee, Gwennan Ward, D. Galas-Filipowicz, Anna Mikolajczak, Emma J Lyon, Dylan Jankovic, Jasmine Rahman, Mahima Turakhia, Imran Uddin, Gordon Beattie, Yvette Hoade, Catherine Zhu, J. Reading, Ieuan G Walker, Michael A. Chapman, K. Ramasamy, Javier Herrero, B. Chain, Sergio Quezada, K. Yong
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background Glioblastoma is the most aggressive malignant brain tumor with poor survival due to its invasive nature driven by cell migration, with unclear linkage to transcriptomic information. The aim of this study was to develop a physics-based framework connecting to transcriptomics to predict patient-specific glioblastoma cell migration. Methods and Results We applied a physics-based motor-clutch model, a cell migration simulator (CMS), to parameterize the migration of glioblastoma cells and define physical biomarkers on a patient-by-patient basis. We reduced the 11-dimensional parameter space of the CMS into 3 principal physical parameters that govern cell migration: motor number—describing myosin II activity, clutch number—describing adhesion level, and F-actin polymerization rate. Experimentally, we found that glioblastoma patient-derived (xenograft) cell lines across mesenchymal (MES), proneural, and classical subtypes and 2 institutions (N = 13 patients) had optimal motility and traction force on stiffnesses around 9.3 kPa, with otherwise heterogeneous and uncorrelated motility, traction, and F-actin flow. By contrast, with the CMS parameterization, we found that glioblastoma cells consistently had balanced motor/clutch ratios to enable effective migration and that MES cells had higher actin polymerization rates resulting in higher motility. The CMS also predicted differential sensitivity to cytoskeletal drugs between patients. Finally, we identified 18 genes that correlated with the physical parameters, suggesting transcriptomic data alone could potentially predict the mechanics and speed of glioblastoma cell migration. Conclusions We describe a general physics-based framework for parameterizing individual glioblastoma patients and connecting to clinical transcriptomic data that can potentially be used to develop patient-specific anti-migratory therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04123a654e8ca28f714460bf7a348c04580a354d" target='_blank'>
              Cell migration simulator-based biomarkers for glioblastoma
              </a>
            </td>
          <td>
            Jay Hou, M. McMahon, Tyler Jubenville, J. Sarkaria, Clark C Chen, David J. Odde
          </td>
          <td>2024-11-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Phenotypic plasticity drives non-genetic tumor adaptation in response to various microenvironmental and therapeutic pressures, consequently promoting tumor heterogeneity and progression. In malignant gliomas, the intrinsic and extrinsic mechanisms underlying phenotypic lineage plasticity and its contribution to drug resistance remain unclear. The epidermal growth factor receptor (EGFR) is frequently altered in glioma and serves as a critical regulator of normal radial glia (RG) cells – multipotent progenitors that give rise to cortical neurons and glia in the developing brain. Here, through interrogation of a large panel of diverse glioblastoma (GBM) patient-derived gliomaspheres and orthotopic xenografts, we find that enrichment of RG-like cells is significantly correlated with responses to pharmacological ablation of EGFR, highlighting EGFR as a crucial lineage survival factor for a population of malignant RG-like cells in GBM. Single-cell RNA sequencing and quantitative proteomics reveal that adaptation to EGFR inhibition is linked to a temporal contraction in RG-like cells and concomitant increase in lineage-restricted neuronal and oligodendrocyte progenitor (NPC/OPC)-like states exclusively in the orthotopic brain environment. This lineage transition is coupled to heightened RAS signaling gene expression programs and sustained activation of MAPK signaling, despite robust and durable inhibition of EGFR activation. Furthermore, constitutively active MEK – but not AKT – is sufficient to rescue tumor proliferation under EGFRi, suggesting that the brain microenvironment modulates RAS-MAPK oncogenic signaling to drive lineage transitions following EGFR blockade. Collectively, these results highlight a critical link between oncogenic signaling and neurodevelopmental lineage plasticity in malignant gliomas, presenting a compelling therapeutic opportunity to block tumor cell state transitions for more durable tumor responses in the native glioma tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b338b52a7c0c64400d62a989938ab592a2bae18" target='_blank'>
              CSIG-24. BRAIN MICROENVIRONMENT-DEPENDENT NEURODEVELOPMENTAL LINEAGE PLASTICITY PROMOTES ADAPTATION TO ONCOGENE INHIBITION IN MALIGNANT GLIOMAS
              </a>
            </td>
          <td>
            Dimitri Cadet, Cassidy Andrasz, Nicholas A. Bayley, Elisa Fazzari, E. Fernández, Q. Okobi, Thomas Graeber, A. Bhaduri, T. Cloughesy, David Nathanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood. To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing. Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles. Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f68ff076c678ea3a1de296cefcb3a2b2a3560d" target='_blank'>
              EPCO-26. LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, Avishay Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, Franck Bielle, A. Laurenge, Anna Luisa Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Zheng Zhao, Qianghu Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes. The UW/Fred Hutch metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting cell-free DNA (cfDNA) and matched normal tissue, primary tumor, and metastatic tumor samples from advanced BLCA patients, with a focus on these variant subtype tumors. This dataset, consisting of 20 patients with up to five metastases per patient, is a unique opportunity to assess how cfDNA is able to capture heterogeneity across BLCA. We assessed both inter-patient heterogeneity between BLCA subtypes and inter-tumoral heterogeneity within each patient using epigenetic and genomic profiling. Using nucleosome profiling, we identified the activity of lineage markers and transcription factors that distinguish healthy individuals from BLCA patients, and between BLCA subtypes. Furthermore, we conducted a detailed analysis of whole genome (30X) and targeted (4,000X) cfDNA sequencing in their ability to capture the evolutionary history of tumor mutations. Detection of tumor mutations by cfDNA is highly dependent on clonality, with >90% of founder tumor mutations captured by either sequencing method, while capture is decreased for either subclonal mutations or those private to a single metastasis. Importantly, deleterious mutations in BLCA driver genes had close to 100% detection using targeted panel sequencing, even for subclonal mutations. These findings lay the groundwork for the use of cfDNA in clinical BLCA decision-making, including tracking the emergence of pathogenic, targetable mutations and variant subtypes.
 Citation Format: Samantha L Schuster, Pushpa Itagi, Sonali Arora, Patricia C Galipeau, Thomas W Persse, Michael Yang, Allie Kreitman, Alan Min, Funda Vakar-Lopez, John K Lee, Petros Grivas, Robert B Montgomery, Jonathan L Wright, Andrew C Hsieh, Hung- Ming Lam, Gavin Ha. Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7eb84ec36801f20a2ac043ba0422da74b9249b7" target='_blank'>
              Abstract B040: Cell-free DNA captures inter- and intra-patient heterogeneity in advanced bladder cancer
              </a>
            </td>
          <td>
            Samantha L Schuster, P. Itagi, Sonali Arora, Patricia Galipeau, Thomas Persse, Michael Yang, Allie Kreitman, Alan Min, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common malignancy of the central nervous system, characterized by a dismal prognosis and inevitable recurrence despite aggressive standard-of-care therapy. Extensive colonization of the surrounding brain parenchyma precludes complete surgical resection, posing a significant therapeutic challenge. While well-studied in model systems, the molecular features and epigenetic regulators of invasive GBM cells remain under-explored directly in clinical samples. To address this gap, we performed a multimodal single-cell sequencing characterization of GBM tumor samples from anatomically distinct regions collected using MRI guidance. Our results demonstrate an enrichment of progenitor-like (neural- and oligodendrocyte-like) malignant states and neurons at the tumor margins. In contrast, differentiated-like states and myeloid cells were more abundant in the tumor core. Peri-tumoral progenitor-like malignant states expressed a unique neuronal signature associated with synaptic signaling, neurogenesis, and Notch signaling. Further, the expression of this neuronal signature was correlated with increased invasiveness. Analysis of matched primary-recurrent GBM patient cohorts revealed an expansion of this neuronal activity program, which was associated with worse overall survival. Motifs of proneural transcription factors implicated in neuronal lineage differentiation were also found to be differentially accessible in progenitor-like malignant states marked by the neuronal invasive signature, potentially contributing to the remodeling of cell states at the invasive margin. Further, cell-cell interaction analysis predicted greater communication between neurons and peri-tumoral progenitor-like malignant states, mediated predominantly through neurexin-neuroligin trans-synaptic signaling, facilitating synaptogenesis and the integration of tumor cells into neural circuits. Altogether, these findings suggest a model in which GBM invasive cells communicate with normal brain neurons in peri-tumoral regions, exploiting neurodevelopmental pathways to facilitate invasion and potentially seed recurrence. Our characterization of invasive GBM cells provides insight into the mechanisms of brain invasion and highlights potential therapeutic vulnerabilities of malignant invasive cells. Understanding these mechanisms opens new avenues for targeted therapies aimed at curtailing the invasive and adaptive capabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d487d52385a969d9534435102163444173f4c0" target='_blank'>
              EPCO-28. UNVEILING INVASIVE MECHANISMS OF GLIOBLASTOMA CELLS THROUGH MULTIMODAL SINGLE-CELL SEQUENCING
              </a>
            </td>
          <td>
            Yiyan Wu, Benson Wu, Y. Ellenbogen, S. Mansouri, Joan Kant, Xuyao Li, O. Singh, Parnian Habibi, Pathum Kossinna, Sandra Ruth Lau Rodriguez, A. Gao, G. Zadeh, Federico Gaiti
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Introduction
 Myeloproliferative neoplasms (MPN) are the consequence of acquired mutations in hematopoietic stem and progenitor cells (HSPCs). Specific mutations and mutational burden determine a patient's risk of disease progression to lethal post-MPN AML.
 Recent technological advancements linking single-cell gene expression and genotype have facilitated the deconvolution of tumor heterogeneity. However, these pipelines are limited in their ability to genotype low-expressed transcripts (e.g., JAK2) and multiple pathogenic loci. To overcome this, we developed a novel single-cell LOng-read genotyping of TRanscripts pipeline called LOTRSeq to explore cancer heterogeneity through target enrichment and long-read sequencing.
 Methods
 LOTRSeq genotypes 10X 3' single-cell RNA sequencing (scRNASeq) barcoded transcripts by a biotin-streptavidin probe-based pull down of 30 genes frequently mutated in MPN/AML, followed by Oxford Nanopore Technology (ONT) long-read sequencing. We performed scRNASeq of 50,439 CD34+ HSPCs of n=8 patients with JAK2V617F-driven MPN (n=4 PV, n=1 ET, n=3 MF) and n=4 post-MPN AML (2 paired samples) followed by LOTRSeq. To assess transcriptional differences between mutated and wildtype HSPCs of MPN/post-MPN AML patients compared to healthy HSPCs, we integrated CD34+ HSPCs scRNASeq data of healthy subjects (n=6, Ainciburu et al. 2023).
 Results
 We utilized a pool of cDNA generated from 4 AML cell lines with 8 different 10X barcodes and ratios to validate the LOTRSeq pipeline for the deconvolution of similar barcodes and genotyping of lowly expressed genes. Target enrichment achieved up to 100X enrichment of transcripts over whole transcriptome sequencing. Read proportions assigned to deconvoluted barcodes highly correlated with cell mix proportions (R=0.93, p=0.0007). We achieved 95% sensitivity to call JAK2V617F and 80% sensitivity to call TP53M133K, with 84% and 78% accuracy, respectively. In MPN and post-MPN AML samples, expressed variant allele frequencies (VAF) of pseudo bulked long-reads highly correlated with genomic VAF (n=6, R=0.9, p=0.01). In addition, 60-80% of cells were genotyped, with genotyping % strongly correlating with Nanopore sequencing output (R=0.85, p=0.0002).
 Single-cell analysis and clustering of CD34+ HSPCs agnostic to genotype reveals that cells cluster according to a deterministic progenitor phenotype. The HSPCs of JAK2V617F-driven MPN largely maintain transcriptional heterogeneity similar to healthy but with expanded erythroid primed cluster (p=0.0047) that correlates with the percentage of JAK2V617F cells (R=0.73, p=0.039). Consistent with previous publications (Nam et al. 2019, van Egeren et al. 2021), mutant and wildtype cells did not cluster apart. However, JAK2V617F cells compared to wildtype and healthy, showed a higher probability of converging towards an erythroid cell fate.
 Sequential analysis of a patient's bone marrow with primary MF and post-MPN AML showed linear clonal evolution of a JAK2V617F and IDH1R132S mutant clone at leukemic transformation, leading to the expansion of a transcriptional HSC-like cluster. Another paired ET and post-MPN AML with a 2-year transformation period showed branching evolution where we identified at diagnosis 27% JAK2V617F cells and 22% TP53Y236C in independent clones and 8% TP53R282W within the TP53Y236C clone. Upon leukemic transformation, the JAK2V617F clone was detected only in 0.6% of cells, outcompeted by a clone with biallelic loss of TP53 through missense mutations Y236C and R282W. This clonal population overlapped transcriptionally with Granulocyte-Megakaryocyte-Progenitor like cells.
 Conclusion
 We have developed a novel scRNASeq pipeline to concurrently genotype a panel of 30 MPN/AML driver genes and examine the transcriptional impact of genetic heterogeneity. We observe a progressive loss of transcriptional heterogeneity with disease progression associated with acquiring additional gene mutations and clonal evolution. Discrete gene mutations are associated with specific effects on transcriptional identity. This knowledge will be invaluable in understanding and monitoring the impact of mutation acquisition and clonal evolution during MPN disease progression and response to therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c97c5f1c1dc16e43acdc1eab98c9dc41b372d18" target='_blank'>
              Single Cell Genotype-Phenotype Analysis Demonstrates That Post-MPN AML Is Driven By Clonal Evolution and Loss of Transcriptional Heterogeneity within Stem Cell Compartments
              </a>
            </td>
          <td>
            Jasmin Straube, Julian Grabek, Leanne T. Cooper, Rohit N. Haldar, Caroline McNamara, David M Ross, Andrew Perkins, Megan J Bywater, Steven W Lane
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="How cells respond to different signals leading to defined lineages is an open question to understand physiological differentiation leading to the formation of organs and tissues. Among the various morphogens, retinoic acid signaling, via the RXR/RAR nuclear receptors activation, is a key morphogen of nervous system development and brain homeostasis. Here we analyze gene expression in ∼80,000 cells covering 16 days of monolayer mouse stem cell differentiation driven by the pan-RAR agonist all-trans retinoic acid, the RARα agonist BMS753 or the activation of both RARβ and RARγ receptors (BMS641+BMS961). Furthermore, we have elucidated the role of these retinoids for driving nervous tissue formation within 90 days of brain organoid cultures, by analyzing > 8,000 distinct spatial regions over 28 brain organoids. Despite a delayed progression in BMS641+BMS961, RAR-specific agonists led to a variety of neuronal subtypes, astrocytes and oligodendrocyte precursors. Spatially-resolved transcriptomics performed in organoids revealed spatially distinct RAR isotype expression leading to specialization signatures associated to matured tissues, including a variety of neuronal subtypes, retina-like tissue structure signatures and even the presence of microglia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6687609670fc431d4f7c052030f236bcbfd2efc5" target='_blank'>
              Decoding transcriptional identity during Neuron-Astroglia Cell Fate driven by RAR-specific agonists
              </a>
            </td>
          <td>
            Ariel O Galindo-Albarrán, Aysis Koshy, Maria Grazia Mendoza-Ferri, M. Mendoza-Parra
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc425805bf39c40a323743bba1f215c7c5c8c55" target='_blank'>
              Digital spatial profiling for pathologists.
              </a>
            </td>
          <td>
            B. Donati, G. Manzotti, F. Torricelli, Cristian Ascione, Riccardo Valli, G. Santandrea, Moira Ragazzi, Eleonora Zanetti, A. Ciarrocchi, S. Piana
          </td>
          <td>2024-11-05</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Lineage plasticity and histologic transformation from prostate adenocarcinoma to neuroendocrine prostate cancer (NEPC) occurs in up to 15-20% of patients with castration-resistant prostate cancer (CRPC) as mechanism of treatment resistance and is associated with aggressive disease and poor prognosis. NEPC tumors typically display small cell carcinoma morphology with loss of androgen receptor (AR) expression and gain of neuroendocrine (NE) lineage markers. However, there is a spectrum of phenotypes that are observed during the lineage plasticity process, and the clinical significance of mixed histologies or those that co-express AR and NE markers or lack all markers is not well defined. Translational research studies investigating NEPC have used variable definitions making clinical trial design challenging. Here we discuss the diagnostic workup of metastatic biopsies to help guide the reproducible classification of phenotypic CRPC subtypes. We recommend classifying CRPC tumors based on histomorphology (adenocarcinoma, small cell carcinoma, poorly differentiated carcinoma, other morphologic variant, or mixed morphology) and immunohistochemical markers with a priority for AR, NKX3.1, INSM1, synaptophysin and cell proliferation based on Ki-67 positivity, with additional markers to be considered based on the clinical context. Ultimately, a unified workup of metastatic CRPC biopsies can improve clinical trial design and eventually practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ba90573d033c02c8e63a055556c5045dc3b6fc" target='_blank'>
              Framework for the Pathology Workup of Metastatic Castration-Resistant Prostate Cancer Biopsies.
              </a>
            </td>
          <td>
            Michael C Haffner, Michael J. Morris, C. C. Ding, Erolcan Sayar, Rohit Mehra, Brian D. Robinson, Lawrence D. True, Martin Gleave, Tamara L. Lotan, Rahul Aggarwal, Jiaoti Huang, Massimo Loda, Peter S Nelson, Mark A. Rubin, H. Beltran
          </td>
          <td>2024-11-26</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Mutations in the histone methyltransferase EZH2, particularly the Y641 hotspot mutation, have been implicated in hematologic malignancies, yet the effect of timing and cellular context on their oncogenic potential has remained unknown. In this study, we utilized a conditional allele with tissue-specific Cre drivers to investigate the effects of Ezh2Y641F mutations at various stages of development, with a focus on the hematopoietic system. We found that ubiquitous heterozygous Ezh2Y641F expression at birth, or conditional expression in hematopoietic or mesenchymal stem cells, led to decreased survival due to hematopoietic defects and bone marrow failure, with no evidence of malignancy. In contrast, Ezh2Y641F expression in committed B cells drives lymphoma formation, highlighting the lineage-specific oncogenic activity of the mutation. Transcriptomic analysis of B cell progenitors revealed key pathway alterations between Cre models such as altered IL2-Stat5 signaling pathway, differential expression of E2F targets, and altered GTPase pathway expression driven by upregulation of Guanylate Binding Proteins (GBPs) in Mx1-Cre Ezh2Y641F pro-B cells. We further found that the GBP locus is regulated by Ezh2-mediated H3K27me3, it is associated with poorer survival in Acute Myeloid Leukemia patients and has variable effects on apoptosis in human lymphoma and leukemia cell lines. These findings suggest that the Ezh2Y641F mutation may alter immune regulatory pathways, cell differentiation and apoptosis, with potential implications for disease progression. Our results highlight the critical role of mutation timing and cellular context in EZH2-driven hematopoietic disease, resulting in distinct downstream changes that shape the oncogenic impact of EZH2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58373fa943a83c479e1a45c9450b16a84a7d051a" target='_blank'>
              Developmental Stage and Cellular Context Determine Oncogenic and Molecular Outcomes of Ezh2Y641F Mutation in Hematopoiesis
              </a>
            </td>
          <td>
            Sarah M. Zimmerman, Samantha J. Procasky, Sofia R. Smith, Jie-Yu Liu, Chad Torrice, George P. Souroullas
          </td>
          <td>2024-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Little progress has been made to delineate the inflammatory microenvironment in pediatric high-grade glioma (pHGG) and diffuse midline glioma (DMG). We utilize molecularly defined human samples and immunocompetent mouse models to study how tumor location and genetic driver mutations influence the tumor microenvironment (TME). We first induced tumors of each pHGG/DMG molecular subtype (H3WT, H3G34R, H3.1K27M, H3.3K27M) into both the cortical hemisphere and midline of separate mice and performed survival studies and flow cytometry. The main driving factor shaping the immune landscape and survival is the driver mutation, not tumor location. Next, we performed single cell RNA sequencing on murine H3WT cortical pHGG, H3WT DMG, H3.1 and H3.3K27M DMGs, paired with multiplex flow cytometry. H3K27M DMGs have sparse T cell infiltration and limited T cell recruiting chemokine production. The main non-neoplastic cell type in all tumors was infiltrating myeloid cells and microglia. H3WT DMGs had the greatest myeloid cell and monocyte-derived macrophage (MDM) infiltration, while H3.3K27M DMGs were enriched for microglia. Disease associated gene signatures were found in tumor-associated macrophages (TAMs) that are found in other neurodegenerative diseases, marked by down-regulation of inflammatory signaling, interferon responses, and up-regulation of metabolism-related pathways. H3.3K27M DMGs were enriched for disease-associated TAMs. These gene signatures were then identified in human pHGG/DMG scRNA sequencing data. CD16 monocytes and microglia were enriched for diseased signatures, while CD14 monocytes were pro-inflammatory. We demonstrate that reprogramming this diseased signature in mice results in increased T cell infiltration and extension of survival in H3K27M DMG. Last, through pharmacologic inhibition of CCR1 and CCR5, we demonstrate preventing TAM infiltration in DMG results in enhanced survival, comparable to radiation therapy. Together, this work provides the foundation for developing immunotherapies targeted at specific subgroups of pHGG and DMGs, such as CAR-T-cell, oncolytic viral therapy, and checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fcf1de80664843474f36fd47a61ca2ddc3739c7" target='_blank'>
              TMIC-06. TUMOR ASSOCIATED MACROPHAGES DRIVE TUMOR PROGRESSION AND ARE TRANSCRIPTIONALLY SHAPED BY HISTONE MUTATIONS IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            James L. Ross, Montserrat Puigdelloses-Vallcorba, Gonzalo Piñero, N. Soni, Wes Thomason, J. DeSisto, Angelo Angione, N. Tsankova, Maria Castro, M. Schniederjan, Nitin Wadhwani, G. P. Raju, Peter Morgenstern, Oren J. Becher, Adam L Green, A. Tsankov, D. Hambardzumyan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in cancer development and metastasis. This review summarizes the current research on how the TME promotes metastasis through molecular pathways, focusing on key components, such as cancer-associated fibroblasts, immune cells, endothelial cells, cytokines, and the extracellular matrix. Significant findings have highlighted that alterations in cellular communication within the TME enable tumor cells to evade immune surveillance, survive, and invade other tissues. This review highlights the roles of TGF-β and VEGF signaling in promoting angiogenesis and extracellular matrix remodeling, which facilitate metastasis. Additionally, we explored how metabolic reprogramming of tumor and stromal cells, influenced by nutrient availability in the TME, drives cancer progression. This study also evaluated the therapeutic strategies targeting these interactions to disrupt metastasis. By providing a multidisciplinary perspective, this study suggests that understanding the molecular basis of the TME can lead to more effective cancer therapies and identify potential avenues for future research. Future research on the TME should prioritize unraveling the molecular and cellular interactions within this complex environment, which could lead to novel therapeutic strategies and personalized cancer treatments. Moreover, advancements in technologies such as single-cell analysis, spatial transcriptomics, and epigenetic profiling offer promising avenues for identifying new therapeutic targets and improving the efficacy of immunotherapies, particularly in the context of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada71caaa487afafe1604b436411fabfcdc1d067" target='_blank'>
              Tumor microenvironment and cancer metastasis: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Ç. BİRAY AVCI, Bakiye Goker Bagca, M. Nikanfar, Leila S. Takanlou, M. S. Takanlou, Alireza Nourazarian
          </td>
          <td>2024-11-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Brain metastasis occurs in 15 % of breast cancer patients and is one of the major causes of breast cancer related morbidity. Brain metastatic breast cancer (BMBC) microenvironment is a heterogeneous niche of cellular and acellular components of which the innate immune cells, microglia, and macrophages, comprise the major fraction. However, the role of these innate immune cells in the tumor microenvironment (TME) in mediating cancer progression and response to therapy is not well understood. To address this need, herein, we developed three dimensional (3D) co-culture spheroids using brain metastatic triple negative breast cancer (TNBC) cell line MDA-MB-231Br or patient derived brain metastatic TNBC cell line F2-7 with THP1 monocytes. Under co-culture conditions MDA-MB-231Br co-culture spheroids assumed highly protrusive morphology with enhanced mesenchymal and proliferative characteristics. Additionally, phenotypic determination of cocultured macrophages revealed elevated levels of markers associated with alternatively activated macrophages i.e. CD163, CD206, and c-Myc. Furthermore, we found that under coculture conditions immunosuppressive and evasive markers PD1, PDL1, CD47 and CD24 were significantly upregulated. Similarly, F2-7 and monocyte cocultures also led to an immunosuppressive environment with elevated levels of M2 type markers and associated anti-inflammatory cytokines. We further found that the presence of M2 type macrophages in the microenvironment of BMBC spheroids led to acquired resistance against Paclitaxel, a clinically relevant chemotherapeutic drug against BMBC. In sum, in our coculture conditions, brain metastatic TNBC cells polarized monocytes into immunosuppressive and pro-tumorigenic tumor associated macrophages leading to a pro-tumoral microenvironment. Thus, 3D co-culture spheroids of BMBC cells and monocytes can serve as robust platform for studying cancer cell and innate immune cell interactions occurring in the BMBC environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2798dacea531ff71fb1f6304110b24fadde96a0" target='_blank'>
              TMIC-48. THREE DIMENSIONAL CANCER CELL-MONOCYTE CO-CULTURE SPHEROIDS RECAPITULATE IMMUNOSUPPRESSIVE AND PRO-TUMORIGENIC MICROENVIRONMENTS IN BRAIN METASTATIC BREAST CANCER
              </a>
            </td>
          <td>
            S. Gurung, Kyra Berger, Shreyas S Rao
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Our research aims to understand the adaptive, ergo potentially metastatic, responses of prostate cancer to changing microenvironments. Emerging evidence implicates a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study revealed that PACC presence in patient prostate tumors at the time of radical prostatectomy was predictive of future metastasis. To test for a causative relationship between PACC state biology and metastasis in prostate cancer, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) from animal models. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Specifically, we applied this approach to analysis of subcutaneous, caudal artery, and intracardiac murine models. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. Furthermore, in vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites. Additional in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence of pro-metastatic mechanisms specific to the PACC state. Implications: Considering that many anti-cancer agents induce the PACC state, our data positions the increased metastatic competency of PACC state cells as an important unforeseen ramification of neoadjuvant regimens, which may help explain clinical correlations between chemotherapy and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c130a571d99ddcd76ddd1df86bc68f827f48ad3" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic.
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T A Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, S. Amend
          </td>
          <td>2024-12-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Establishing pre-clinical patient derived orthtopical xenograft models for IDH-mutant and H3K27M-mutant gliomas has presented significant histological challenges. There is increasing interest in developing faithful patient-derived ex vivo 3D models to replicate these type of gliomas, aiming to accelerate drug screening efforts.



 We developed a rapid approach for establishing nanoliter-sized miniature glioma organoids, termed NanoGlios, by leveraging emulsion microfluidics. We tested the robustness of NanoGlios for neurosphere culture, glioma xenografts, and primary patient samples, focusing on H3K27M-mutant diffuse midline gliomas (DMGs) and IDH-mutant gliomas. We compared the morphological features of NanoGlios to their counterparts using high-content imaging, H&E staining, and histological analysis. Additionally, we evaluated the mutation and gene expression profiles of NanoGlios relative to their counterparts. Furthermore, we conducted proof-of-concept drug screening of NanoGlios with various inhibitors and established an AI-enhanced high-throughput phenotypic imaging-based screening platform.



 We successfully established NanoGlios from various sample sources, including neurospheres, xenografts, and primary tissues harboring H3K27M or IDH1/2 mutations. NanoGlios derived from xenografts or primary tissues captured more heterogeneous patterns as compared to the neuropsheres. We were able to generate thousands of NanoGlios and perform drug assays for hundreds of conditions at passage 0 within two weeks from both resected glioma tissues and xenografts. The AI phenotypic screening enabled us to identify differential responses of NanoGlios to ONC-201 treatment and drug responses to mutant IDH inhibitors Vorasidenib and Ivosidenib, which was not able to be detected by conventional cell viability bulk assays.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3753eee1d63df13c824fea975bf6805f689c10ca" target='_blank'>
              TMOD-36. NANOGLIOS: A MINIATURIZED PATIENT AVATAR ENABLING RAPID AND HIGH-THROUGHPUT PHENOTYPIC SCREENING FOR GLIOMAS
              </a>
            </td>
          <td>
            Zhaohui Wang, Wei Huang, Satoru Kawakita, Francisco Bustamante, Rene Larios, Chongming Jiang, Gregory P Lekovic, Richard G Everson, L. Liau, Albert Lai, Zhaohui Wang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="

 Introduction
 Myeloid neoplasms post cytotoxic therapy (MN-pCT) are a catastrophic complication of cancer therapy with an aggressive disease course and dismal patient outcomes. They frequently arise from preexisting clonal hematopoiesis (CH). Large sequencing studies have reported differences in the mutational spectra of age-related CH (ARCH), therapy-related CH (t-CH), de novo MN, and MN-pCT, respectively, suggesting functional differences in the capacity of the various driver gene mutations to initiate MN-pCT. Here, we utilized clinical sequencing data, meta-analytic statistical methods, newly established CRISPR/Cas9-edited mouse hematopoietic stem and progenitor cell (HSPC) lines, and murine mixed bone marrow chimeras to model the evolution of MN-pCT from CH under the selective pressure of anti-cancer therapy.
 Results
 We compared targeted next-generation sequencing data from MN-pCT and de novo MN patients collected at our institution. We found a significantly lower frequency of DNMT3A mutations in the MN-pCT group, while mutations in PPM1D and TP53 were more frequent. To determine the frequency of typical ARCH mutations (i.e., in the genes DNMT3A, TET2, ASXL1) as opposed to mutations in genes associated with t-CH (i.e., ATM, CHEK2, PPM1D, and TP53) in cohorts comprising the general population, therapy-exposed non-myeloid cancer patients, or MN-pCT patients, we performed a meta-analysis from 21 publicly available sequencing studies encompassing over 760,000 individuals. Estimation of cohort-specific mutational frequencies using a random effects logistic regression model revealed a significantly higher frequency of above mutations in therapy-exposed non-myeloid cancer patients (i.e., t-CH) as compared to the general population (i.e., ARCH) with highest odds ratios for PPM1D and TP53. Importantly, only TP53 mutations were strongly enriched in MN-pCT samples compared to patients with t-CH.
 To study the development of MN-pCT from CH in vitro, we generated CRISPR/Cas9-edited conditionally immortalized HSPC lines by reversible Hoxb8 overexpression with mutations in Dnmt3a, Tet2, Asxl1, Atm, Chek2, Ppm1d, or Trp53. First, we tested the capacity of HSPCs with these ARCH or t-CH mutations to clonally expand upon treatment with common anti-cancer drugs. Chek2- and Ppm1d-mutant HSPCs moderately outcompeted their wild-type counterparts mostly after treatment with double-strand break-inducing agents, whereas Trp53-/- HSPCs demonstrated rapid and almost complete outgrowth in the presence of a wide range of conventional cytostatic drugs as well as various targeted agents - e.g., PARP, BRAF, and BCL2 inhibitors. By contrast, cells with ARCH mutations did not expand upon treatment with anti-cancer drugs. Accordingly, Trp53-/- HSPCs displayed strongest defects in apoptosis and G1 cell cycle arrest as well as signs of persistent DNA damage as shown by γ-H2AX staining when compared to HSPCs with mutations in the other tested genes. Next, we assessed the capacity for malignant transformation after treatment with anti-cancer drugs in in vitro immortalization assays. Only Trp53-/- HSPCs were able to generate immortalized cell lines that harbored complex karyotypes reminiscent of those found in patients with TP53-mutant MN.To investigate clonal expansion and malignant transformation in vivo, we generated murine mixed bone marrow chimeras with 5-10% CRISPR/Cas9-edited HSPCs with mutations in Atm, Chek2, Ppm1d, or Trp53. After sublethal irradiation to mimic anti-cancer therapy, clonal expansion was observed in peripheral blood of Chek2-, Ppm1d- and again strongest in Trp53-mutant mice. Strikingly, only mice with Trp53-/- HSPCs displayed significantly shortened survival driven by the development of MN-pCT.
 Discussion and Outlook
 Using clinical sequencing data together with pre-clinical modelling in vitro and in vivo, we demonstrate that mutations in CHEK2, PPM1D, and TP53 but not classic ARCH mutations confer a clonal advantage to HSPCs under the selective pressure of anti-cancer drugs, thereby explaining their enrichment in t-CH. However, only TP53 mutations are sufficient to drive MN-pCT development in vitro and in vivo. Our findings further support the development of mitigation strategies to prevent MN-pCT in cancer patients at risk - in particular those with TP53 mutations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c04a0fd4f4254b49fa65a518d8e33c6826db227" target='_blank'>
              Modelling the Evolution of Clonal Hematopoiesis to Myeloid Malignancies Under the Selective Pressure of Anti-Cancer Therapy
              </a>
            </td>
          <td>
            Christian Koch, Jonas Fullin, Ebru Topçu, Nancy Klemm, Karolina Zielinska, Monika Hebeisen, Joelle Tchinda, M. Manz, S. Boettcher
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND
A spatially resolved, niche-level analysis of tumour microenvironments (TME) can provide insights into cellular interactions and their functional impacts in gastric cancers (GC).


OBJECTIVE
Our goal was to translate the spatial organisation of GC ecosystems into a functional landscape of cellular interactions involving malignant, stromal and immune cells.


DESIGN
We performed spatial transcriptomics on nine primary GC samples using the Visium platform to delineate the transcriptional landscape and dynamics of malignant, stromal and immune cells within the GC tissue architecture, highlighting cellular crosstalks and their functional consequences in the TME.


RESULTS
GC spatial transcriptomes with substantial cellular heterogeneity were delineated into six regional compartments. Specifically, the fibroblast-enriched TME upregulates epithelial-to-mesenchymal transformation and immunosuppressive response in malignant and TME cells, respectively. Cell type-specific transcriptional dynamics revealed that malignant and endothelial cells promote the cellular proliferations of TME cells, whereas the fibroblasts and immune cells are associated with procancer and anticancer immunity, respectively. Ligand-receptor analysis revealed that CCL2-expressing fibroblasts promote the tumour progression via JAK-STAT3 signalling and inflammatory response in tumour-infiltrated macrophages. CCL2+ fibroblasts and STAT3-activated macrophages are co-localised and their co-abundance was associated with unfavourable prognosis. We experimentally validated that CCL2+ fibroblasts recruit myeloid cells and stimulate STAT3 activation in recruited macrophages. The development of immunosuppressive TME by CCL2+ fibroblasts were also validated in syngeneic mouse models.


CONCLUSION
GC spatial transcriptomes revealed functional cellular crosstalk involving multiple cell types among which the interaction between CCL2+ fibroblasts and STAT3-activated macrophages plays roles in establishing immune-suppressive GC TME with potential clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe72f4555eff64ae7e607a98dd2b2dd8cb471c6e" target='_blank'>
              Spatial dissection of tumour microenvironments in gastric cancers reveals the immunosuppressive crosstalk between CCL2+ fibroblasts and STAT3-activated macrophages.
              </a>
            </td>
          <td>
            Sung Hak Lee, Dagyeong Lee, Junyong Choi, Hye Jeong Oh, In-Hye Ham, Daeun Ryu, Seo-Yeong Lee, Dong-Jin Han, Sunmin Kim, Youngbeen Moon, In-Hye Song, K. Song, Hyeseong Lee, Seungho Lee, Hoon Hur, Tae-Min Kim
          </td>
          <td>2024-11-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and malignant type of brain tumor in adults and no effective therapies exist to combat this disease. Surgical resection, radio- and temozolomide-based-chemotherapy are insufficient mainly due to inter- and intra-tumoral heterogeneity of GBM tumors, therefore personalized approaches are required. Additionally, diverse tumor microenvironment with a leading role of glioblastoma stem cells (GSCs) in hypoxic core underlie the aggressiveness and recurrence of GBM. However, the molecules and pathways which directly promote tumor aggressiveness together with the driving process of epithelial-to-mesenchymal transition (EMT) and the components of extracellular matrix (ECM), remain only partially covered with prior studies focused mainly on coding oncogenes and tumor suppressor genes. Over the past years, a special attention has been paid to non-coding, circular RNAs (circRNAs) differentially expressed in various cancers, including GBM. Despite high abundance and stability, the exact biological roles of most individual circRNAs have not been fully revealed, so far. Besides anti-cancer targets, circRNAs have been considered as essential biomarkers and stratifying criterion of malignancy. Thus, studies on functions of differentially expressed circRNAs in GBM are an emerging field to be explored. In herein study, the biological role of circRNA CLIP2 (circCLIP2) was found highly overexpressed in primary tumor tissues. Upon the knock-down of circCLIP2 in 2D and 3D in vitro models, functional tests performed clearly indicate a significant contribution of circCLIP2 to aggressive potential of GBM as showed by decreased rates of proliferation, migration and invasion. Additional studies indicated predominant expression of circCLIP2 in GSC fractions additionally elevated by hypoxic conditions. The increased rates of proliferation and migration as well as molecular analyses were also confirmed in in vivo settings where xenograft model was applied. The expression of EMT marker Snail2 and ECM-degrading enzyme metalloproteinase 9 (MMP9) were significantly downregulated in the tumors formed from circCLIP2 silenced cells. Taken together, these results indicate a strong contribution of circCLIP2 in the aggressive phenotype of GBM which is due to a modulation of GSC proliferative potential as well as migration and invasion via EMT together with the modulator of ECM composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41e1172182008f639a17a4a3feb76b12f54727b" target='_blank'>
              Circular RNA - circCLIP2 is predominantly expressed in GSC niche and enhances glioblastoma aggressiveness via EMT and ECM signaling
              </a>
            </td>
          <td>
            J. Misiorek, Ż. Zarębska, K. Kuczyński, J. Latowska-Łysiak, A. Grabowska, P. Głodowicz, M. Sajek, Anna Karlik, Dorota Wronka, A. Barciszewska, Łukasz Przybył, K. Rolle
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Immune cell microenvironment is a critical component regulating myeloma progression and efficacy of various immunotherapies. Myeloma grows in complex foci or clusters within the bone marrow, and the composition and organization of these structures is poorly understood. Interestingly, we have found non-malignant long lived plasma cells also form dynamic clusters in the bone marrow suggesting a common survival niches. In patients, analyses of myeloma tumor microenvironment (TME) typically rely on imprecise bone marrow aspirates, which can collect random samplings of the marrow parenchyma as well as peripheral blood, and may contain variable and unknown proportion of myeloma foci or the relevant TME. To precisely profile immune microenvironment precisely within myeloma foci, we inoculated photoactivatable-GFP (PA-GFP) expressing immunocompetent mice with Vk*MYC GFP cells, and using two-photon microscopy in the tibia, we photo-labeled immune cells within the foci and analyzed by flow. We discovered the composition within foci was distinct compared to bulk bone marrow within the same marrow, which mimicked naïve marrow composition, suggesting unique TME were limited to myeloma foci volume. We also conducted longitudinal bulk analyses of total BM with tumor growth and found that distinct changes in myeloid cell composition with tumor burden, suggesting multiple phases in TME evolution. Finally, using genetic ablation of myeloid cells using DTR expressing mice, we found that macrophage and monocytes promoted tumor spreading to other bones, while plasmacytoid dendritic cells suppressed spreading. These findings highlight the dynamics and precision of the myeloma TME and how it regulates tumor progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6123419360db216c7324fff2b27cc2739904d28" target='_blank'>
              Analysis of Spatial and Temporal Dynamics of the Myeloma Immune Microenvironment
              </a>
            </td>
          <td>
            David R Fooksman, Ilseyar Akhmetzyanova
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Introduction: Fludarabine, a purine nucleotide analogue, is integral to preparative regimens for CAR-T cell therapy and allo-SCT in treating B-cell pediatric acute lymphoblastic leukemia (B-ALL). By disrupting DNA synthesis and repair, fludarabine induces apoptosis in proliferating leukemia cells. It also plays a vital role in immunosuppression, lymphodepletion, and tumor burden reduction, optimizing the efficacy of subsequent therapies. Despite its critical role, the molecular impact of fludarabine on leukemia cells and its efficacy factors remain inadequately understood.
 Methods: We utilized a cutting-edge single-cell spatial proteomics assay, Molecular Pixelation (MPX, Karlsson et al. Nat Methods 2024), an optics-free DNA sequencing-based method to analyze the cell surface interactome in response to fludarabine. The glucocorticoid-resistant ETV6::RUNX1 cell line REH and the BCR::ABL1 cell line SUP-B15 were incubated with and without fludarabine for 72 hours at varying concentrations, with repeated technical repetitions. Approximately 1,000 surviving single cells per condition were analyzed using the MPX assay (Pixelgen Technologies), wherein antibody-oligonucleotide conjugates generated nanometer-sized molecular pixels, read out by sequencing on a NovaSeq X Plus instrument (Illumina). Spatial proteomics networks for 76 proteins were computationally reconstructed from DNA-sequencing reads by inferring the relative locations of antibody-oligonucleotide conjugates, creating ~1,000 spatially connected neighborhoods per cell. Orthogonal immunocytochemistry (ICC) data were generated to validate the top protein interactions.
 Results: We have previously shown that ex-vivo drug screening in combination with single-cell transcriptome sequencing (scRNA-seq) readout can elucidate transcriptional effects of fludarabine (Gezelius et al. NAR Genomics and Bioinformatics 2024). This approach confirmed fludarabine's disruption of DNA synthesis and repair in dividing cells, and in addition revealed previously unknown extensive dysregulation of transcripts encoding cell surface proteins. The scRNA-seq data do not adequately resolve the spatial nor functional dynamics of the cell surface proteome, leaving this important aspect of response to fludarabine unexplored. In this study, we take advantage of recent advancements in single-cell spatial proteomics profiling to spatially elucidate how the cell surface proteome organization and function is modulated during fludarabine treatment.
 Spatial statistical analysis revealed novel patterns in cell surface organization during fludarabine treatment. The MPX assay identified 27 proteins with significantly altered abundance post-treatment (adj. Wilcoxon p < 0.01), consistent with the increased abundance of cell-surface proteins observed in our previous scRNA-seq data. Additionally, 26 proteins exhibited changes in polarity (adj. Wilcoxon p < 0.01) after fludarabine treatment, with most proteins shifting from a dispersed to a polarized state. We also detected 138 protein pairs with modified co-localization patterns. Notably, fludarabine enhanced the abundance, polarization, and co-localization of cadherin proteins CD82 and CD53, suggesting a potential therapeutic scaffold. This finding was further validated through orthogonal ICC experiments.
 Conclusions: This study advances our understanding of the spatial organization of the cell surface proteome in leukemia cells following fludarabine exposure. The findings provide new insights into fludarabine's effects at the single-cell level and may inform the development of more effective targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df1ba5108871a33c10f05770da635143d8f7eec" target='_blank'>
              Mapping the Spatial Proteome of Individual Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            J. Nordlund, M. Globisch, Henrik Gezelius, Anna Pia Enblad, O. Krali, Mariya Lysenkova-Wiklander, Martin Åberg, Arja Harila, A. Raine, Claes Andersson
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Transforming growth factor-beta (TGFβ) has dual roles in cancer, initially suppressing tumors but later promoting metastasis and immune evasion. Efforts to inhibit TGFβ have been largely unsuccessful due to significant toxicity and indiscriminate immunosuppression. Leucine-rich repeat-containing protein 15 (LRRC15) is a TGFβ-regulated antigen expressed by mesenchymal-derived cancer cells and cancer-associated fibroblasts (CAFs). In preclinical studies, ablation of TGFβ-driven LRRC15+ CAFs increased tumor infiltration of CD8+ T cells. However, the underlying pathobiological mechanisms prompting TGFβ’s upregulation of LRRC15 expression are unclear. Using an integrated approach combining functional compound screening with single-cell RNA sequencing, we reveal key genomic features regulating TGFβ’s ability to increase LRRC15 expression on cancer cells. Construction of gene regulatory networks converged our analyses on four key genes—MMP2, SPARC, TGFβR2, and WNT5B—central to TGFβ-induced LRRC15 pathobiology. Validation of these genes in cell models and their use in predicting immunotherapy responses highlight their potential in refining immunotherapy strategies and personalizing co-treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41bd13d4e06b8bfdb4417486610098a9d8cd845" target='_blank'>
              Key Regulatory Elements of the TGFβ-LRRC15 Axis Predict Disease Progression and Immunotherapy Resistance Across Cancer Types
              </a>
            </td>
          <td>
            Claire M. Storey, Michael Cheng, Mohamed Altai, Julie Park, Julie Tran, Smiths S. Lueong, Daniel L. J. Thorek, Liqun Mao, Wahed Zedan, Constance Yuen, Alexander Ridley, Marija Trajovic-Arsic, Ken Herrmann, Sumit K. Subuhdi, B. Siddiqui, Katharina Lückerath, J. Siveke, R. Damoiseaux, Xia Yang, David Ulmert
          </td>
          <td>2024-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) treatment is hindered by a dearth of representative mouse GBM preclinical models in immunocompetent mice. Here, we characterized five murine GBM stem-like cell (mGSC) models derived from lentivirus-induced tumors in transgenic mice that are driven by the activation of the Nf1-Ras signaling pathway and inactivation of Tp53.



 MGSC lines (005, RIG, NF53, C1, and C3) were cultured as spheres in serum-free stem cell media. Whole exome sequencing (WES) was employed to quantify single nucleotide polymorphisms (SNPs). Stem cell properties were characterized by stemness in vitro and tumorigenicity after intracerebral implantation in C57BL/6 mice. Tumor phenotypes and the immune microenvironment were characterized by immunohistochemistry, flow cytometry, and RNA sequencing.



 WES revealed a large variation in coding sequence SNPs across mGSC lines (~20-fold), likely influenced by the mixed backgrounds of the parental mice. MGSCs exhibited variable clonogenic sphere formation and CD133 expression levels. In vivo, they consistently initiated lethal malignant gliomas, with median survival ranging from 29 to 82 days, and showed strong CD44 expression and variable invasiveness. The tumor microenvironment featured an abundance of CD68+ macrophages and uniform high PD-L1+ myeloid cells, while T cell infiltration varied among the models, with low mutation burden C1 and C3 exhibiting fewer tumor-infiltrating T cells.



 Upon orthotopic implantation in immunocompetent mice, mGSCs generate tumors characteristic of human GBM. Despite similar strategies to generate these mGSCs, they exhibited a range of phenotypes and immune profiles in mGSC-derived orthotopic tumors. These mGSCs provide new preclinical GBM models for developing GBM immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85632e943cebc73850dcb33bb8236cdd57ebd815" target='_blank'>
              immunocompetent murine glioblastoma stem-like cell models exhibiting distinct phenotypes
              </a>
            </td>
          <td>
            Kimia Kardani, S. Ghouse, Muzammil Arif Din Abdul Jabbar, Namita Rajasubramanian, Judit Sánchez Gil, A. Stemmer-Rachamimov, Yasushi Soda, R. Martuza, Toshiro Hara, Hiroaki Wakimoto, S. Rabkin
          </td>
          <td>2024-12-07</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="

 The acquired BTKC481S mutation is the most common resistance mechanism to Bruton's tyrosine kinase (BTK) inhibition in chronic lymphocytic leukemia (CLL). Yet, its variant allele frequency (VAF) is often low (<10%), raising the question of how wild-type (WT) cells survive in the context of low abundance of mutant (MUT) cells. To investigate this, we modeled the clonal dynamics of BTK-MUT and WT cells longitudinally in a cohort of 42 patients resistant to BTK inhibitors (BTKi). In 21 patients, clonal dynamics deviated from a simple model where MUT cells increase at the expense of WT cells. Instead, both MUT and WT cell numbers increased at relapse, suggesting that BTK-MUT and WT cells cooperate to drive resistance to BTKi.
 To explore the phenotypic variability that may underlie clonal cooperation, we expanded our genotyping of transcriptomes (GoT) profiling of seven CLL samples taken at relapse after BTKi [60,580 cells with a median of 8,713 cells per patient (6,819-9,838)] and carrying low VAFs (3-43%). GoT genotypes a mutation of interest (i.e. BTKC481S) jointly with single-cell transcriptomes of primary samples. This integration allows to link genotype and phenotype, and by comparing BTK-MUT and BTK-WT cells within the same patient, this method eliminates inter-patient variability and confounding factors, thus focusing on changes in expression and intratumoral heterogeneity (ITH).
 Transcriptomic ITH in CLL is largely driven by the recirculation of CLL cells between peripheral blood and lymphoid tissues. CXCR4hiCD5low CLL cells (i.e. quiescent fraction) migrate to lymph node (LN) proliferation centers (PR) via a CXCL12 gradient. In PR, they receive microenvironmental signals, upregulate mir155 (encoded by MIR155HG), increase BCR signaling, proliferate, and exit as CXCR4lowCD5hi cells (i.e. proliferative fraction). BTKi impairs CXCR4 signaling and homing to LN. Here, we curated gene signatures for the quiescent (CXCR4, CD24), proliferative (CD27, CXCR3, CD52, ITGAX) and MIR155HGhi fractions (MIR155HG, MYC, CD83, CDK4) leveraging published scRNA-seq datasets. We classified cells into the 3 fractions and uncovered an increased quiescent fraction (p=0.058) and a decreased proliferative fraction (p<0.05) in BTK-WT cells compared to BTK-MUT cells across all patients, suggesting that BTK-WT have an impaired homing capacity to LN.
 We reasoned that cancer cell states are plastic and continuous, requiring tailored methods to account for these features. We employed topic modeling, an algorithm that decomposes each gene expression profile into a set number of topics (k). Topics represent groups of correlated genes, linked to specific biological functions. We could annotate 3/5 topics to the fractions described above in most patients, with an increase of the MIR155HGhi topic (MIR155HG and MYC targets) in BTK-MUT cells (p<0.001). We also identified patient-specific topics that could not be captured through conventional methods.
 In one patient, we identified a topic enriched in interferon-stimulated genes (IFIT1, IFIT2, IFIT3, IFI44L, STAT1) that was specific to BTK-WT cells (p<0.001) within the quiescent fraction. As interferons protect CLL cells from apoptosis and BTKi sensitive cells are still responsive to interferons [Xia, J Immunol 2020], our data suggests that BTK-WT cells may survive by responding to circulating interferons without homing into LN.
 In the same patient, we observed a cell state enriched in BTK-MUT cells expressing IL4R, which is associated with resistance to BTKi through the IL4R-IL4 axis. Intriguingly, a cell state specific to BTK-WT cells in the same patient overexpressed Chronic Lymphocytic Leukemia Up-Regulated 1 (CLLU1), a gene that distinguishes CLL from normal B cells and other lymphoid malignancies and is a marker of poor prognosis. IL4R expression was mutually exclusive with CLLU1, suggesting a putative paracrine activation of the IL4 pathway, as CLLU1 was previously proposed to have structural homology to IL4. Notably, we observe subclonal expression of CLLU1 in 6/7 patients.
 Taken together, we propose two models of clonal collaboration between BTK-WT and BTK-MUT cells, involving paracrine signaling between interferons and CLLU1/IL4. We are currently investigating the relevance of these signaling axes in larger cohorts of longitudinally collected patient samples, including through in vitro functional validation assays.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02d5ba94fa66b53963b26f372346d282d334e6f" target='_blank'>
              Topic Modeling of Genotyping of Transcriptomes Reveals Collaboration between BTKC481S-Mutant and Wild Type Cells in Btki-Resistant Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Ramon Massoni-Badosa, Joshua S. Schiffman, Barun Pradhan, Rebecca M. Murray, Andrew H. Lipsky, Debra Van Egeren, Danny Luan, Shirley Chen, R. Chaligné, Kai Beattie, Alyssa Indart, Nada Chraiki, N. Dusaj, E. Bhavsar, Chelston R Ang, Anna S. Nam, Federico Gaiti, P. Chamely, Nathaniel D. Omans, Preeti Trisal, Anita K. Gandhi, Kyu-Tae Kim, Rafael C. Schulman, Franco Izzo, Gavriel Mullokandov, Chingiz Underbayev, J. Burger, A. Wiestner, Elisa ten Hacken, J. Allan, J. Woyach, R. Furman, D. Landau
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="A noticeable characteristic of pancreatic ductal adenocarcinoma (PDAC) tumors is a dense tumor microenvironment with abundant and dense, desmoplastic stroma woven tightly with both cellular and matrix components. The high stromal density is associated with higher intratumor pressures which, until the last decade, was largely assumed to be tumor protective, confirmed by early studies demonstrating that altering the stroma was effective in genetically engineered models of PDAC. However, clinical trials using these approaches have been disappointing. There is increasing recognition that stroma heterogeneity is much greater than initially thought with an explosion of investigation into cancer-associated fibroblast (CAF) subpopulations led by experimental and single-cell transcriptomic studies. This review summarizes and attempts to harmonize the current transcriptomic data of CAF subpopulations. Understanding the heterogeneity of CAFs, the matrix, and other tumor microenvironment features will be critical to developing effective therapeutic approaches. Identifying model systems that best recapitulate the clinical behavior and treatment response of human PDAC will be important. Examining subpopulations as defined by clinical outcome will remain a critical step in defining clinically impactful CAF subtypes in larger clinical cohorts. The future of precision oncology in PDAC will depend on the integration of precision tumor epithelial and precision stroma approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50e456e62579aeacb26fe2456d06aace9ecba04" target='_blank'>
              CAFomics: convergence to translation for precision stroma approaches.
              </a>
            </td>
          <td>
            Ian C McCabe, X. Peng, Joseph F Kearney, Jen Jen Yeh
          </td>
          <td>2024-11-08</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80cae72455d0f13cb986669bd3a06f266bb4aab" target='_blank'>
              Spatial transcriptomics reveals substantial heterogeneity in triple-negative breast cancer with potential clinical implications
              </a>
            </td>
          <td>
            Xiaoxiao Wang, D. Venet, F. Lifrange, D. Larsimont, Mattia Rediti, L. Stenbeck, F. Dupont, G. Rouas, Andrea Joaquin Garcia, L. Craciun, L. Buisseret, Michail Ignatiadis, M. Carausu, N. Bhalla, Yuvarani Masarapu, Eva Gracia Villacampa, L. Franzén, Sami Saarenpää, L. Kvastad, K. Thrane, J. Lundeberg, F. Rothé, Christos Sotiriou
          </td>
          <td>2024-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Abstract Background Intratumoral heterogeneity (ITH) and subclonal antigen expression blunt antitumor immunity and are associated with poor responses to immune-checkpoint blockade immunotherapy (ICB) in patients with cancer. The underlying mechanisms however thus far remained elusive, preventing the design of novel treatment approaches for patients with high ITH tumors. Methods We developed a mouse model of lung adenocarcinoma with defined expression of different neoantigens (NeoAg), enabling us to analyze how these impact antitumor T-cell immunity and to study underlying mechanisms. Data from a large cancer patient cohort was used to study whether NeoAg architecture characteristics found to define tumor immunogenicity in our mouse models are linked to ICB responses in patients with cancer. Results We demonstrate that concurrent expression and clonality define NeoAg architectures which determine the immunogenicity of individual NeoAg and drive immune evasion of tumors with heterogenous NeoAg expression. Mechanistically, we identified concerted interplays between concurrent T-cell responses induced by cross-presenting dendritic cells (cDC1) mirroring the tumor NeoAg architecture during T-cell priming in the lymph node. Depending on the characteristics and clonality of respective NeoAg, this interplay mutually benefited concurrent T-cell responses or led to competition between T-cell responses to different NeoAg. In tumors with heterogenous NeoAg expression, NeoAg architecture-induced suppression of T-cell responses against branches of the tumor drove immune evasion and caused resistance to ICB. Therapeutic RNA-based vaccination targeting immune-suppressed T-cell responses synergized with ICB to enable control of tumors with subclonal NeoAg expression. A pan-cancer clinical data analysis indicated that competition and synergy between T-cell responses define responsiveness to ICB in patients with cancer. Conclusions NeoAg architectures modulate the immunogenicity of NeoAg and tumors by dictating the interplay between concurrent T-cell responses mediated by cDC1. Impaired induction of T-cell responses supports immune evasion in tumors with heterogenous NeoAg expression but is amenable to NeoAg architecture-informed vaccination, which in combination with ICB portrays a promising treatment approach for patients with tumors exhibiting high ITH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15e0ff3925c0c2401500133c42352b7c2b38ca5d" target='_blank'>
              Neoantigen architectures define immunogenicity and drive immune evasion of tumors with heterogenous neoantigen expression
              </a>
            </td>
          <td>
            Malte Roerden, Andrea B Castro, Yufei Cui, Noora Harake, Byungji Kim, Jonathan Dye, L. Maiorino, Forest M. White, Darrell Irvine, Kevin Litchfield, S. Spranger
          </td>
          <td>2024-11-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a highly malignant brain tumor characterized by diffuse infiltration and significant heterogeneity, without effective treatment. Mesenchymal-like (MES) glioma cells drive tumor aggression by promoting brain invasion and tumor microenvironment remodeling. Previous spatial transcriptomic analyses identified Collagen, particularly COL1A, as a key extracellular matrix (ECM) component, enriched in MES areas termed oncostreams. Depletion of COL1A1 disrupted oncostreams, influencing the tumor microenvironment to support cell survival, and migration. Reactive astrocytes and macrophages, essential components of the brain microenvironment, impact glioma progression. However, the interplay between glioma-secreted collagen and the brain microenvironment remains poorly understood. To elucidate this, we generated orthotopic mouse glioma models with high-collagen (NPD) and low-collagen (NPDshCol1A1). Our data revealed that downregulation of COL1A1 in glioma cells alters the abundance and spatial distribution of reactive astrocytes and macrophages/microglia. Immunofluorescence and flow cytometry analysis showed that tumors with low-collagen had decreased GFAP+ reactive astrocyte cells at the invasive edge and perivascular niche compared to high-collagen gliomas. Moreover, our results shows that high collagen levels may activate astrocytes through distant communication, as evidenced by the higher presence of reactive astrocytes in the contralateral cortex of NPD tumors. However, low-collagen tumors exhibited an increase in astrocytes (GFAP+) in the tumor core. Furthermore, high-collagen gliomas demonstrated increased macrophage infiltration, promoting an immunosuppressive environment. Using multiplex immunohistochemistry assays, we observed that reactive astrocytes along the invasive tumor edge were associated with glioma cells expressing elevated levels of COL1A1 and TGF-β ligand. In summary, our results show that glioma cell-expressed COL1A1 promotes the activation of peritumoral and perivascular brain-resident astrocytes and macrophages. The interactions of reactive astrocyte with glioma cells enriched in collagen form a feedback loop that enhances tumor invasion and vascular proliferation, suggesting that targeting the ECM-astrocyte interactions could disrupt these processes and offer a therapeutic avenue for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a6c055a8b8d19c7f6efc8b6c7f773b8db68c08" target='_blank'>
              CNSC-55. UNCOVERING THE ROLE OF COLLAGEN EXTRACELLULAR MATRIX IN GLIOBLASTOMA BRAIN MICROENVIRONMENT TRANSFORMATION
              </a>
            </td>
          <td>
            Melika Hashemi, Durgadevi Selvam, Reid J Hale, Katelynn Shockley, Ailin Zhyrgalbekovna, Andrea Comba
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Introduction: Properties of tumor angiogenesis impact on tumor growth and composition of the tumor microenvironment. Tumor vessels are composed of pericytes and other mural cells. Detailed characterization of the cellular composition of the microvasculature can suggest functional vessel and case properties relevant for outcome, response to treatment and immune surveillance. Results: Based on single cell RNA-seq from three human stage II/III colon cancers and pilot in-situ multi-antibody staining of six tumors two main clusters of perivascular cells were identified; B1 (MCAM+, MYH11-) and B2 (MCAM+, MYH11+) cells. B1 cells displayed pericyte features including expression of RGS5, whereas B2 cells showed high expression of smooth muscle cell markers like ACTG2 and Desmin. Desmin-high B2 cells were predominantly located in muscular layers, whereas Desmin-low/MYH11-high B2 cells showed perivascular enrichment. An extended novel analyses workflow of digital image analysis was used to profile peri-endothelial composition in 19 intervals; 11 1-µm intervals from 0 to 10 µm and 8 5-µm intervals from 10 to 50 µm from endothelial areas. Each of these expansion areas was quantitated regarding density of PDGFRB+/- cells including B1/B2 cells, and a MYH11+/MCAM- cells. Density of macrophage subsets (CD68-positive subsets defined by CD163 and CD11c) were also quantitated. In general, B1 cells showed stronger spatial association with endothelial cells than B2 cells, and this spatial association was stronger for the PDGFRB+ B1 subset. Survival analysis indicated that prognostic relevance was strongest when PDGFRB+ (favorable prognosis) cells were closest to endothelial cells. Regarding perivascular macrophage density, M0 and transit M1/M2 macrophages were predominantly enriched in the peri-endothelial regions, whereas density of M1 and M2 increased gradually as distance from vessels increased. M2 density was overall associated with poor prognosis, whereas high density of other macrophage subsets overall was associated with good prognosis. Ongoing studies are using a clustering-based approach to identify vessel subsets, and prognosis associations are being consolidated in additional cohorts. Summary: This study provides a novel approach for high resolution profiling of perivascular status in CRC, and possibly other tumor types, that could be of specific importance to identify tumor features associated with response to angiogenesis-directed therapies.
 Citation Format: Linglong( 凌 龙 ) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, Simon Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, Artur Mezheyeuski, David Edler, Anna Martling, Fredrik Pontén, Bengt Glimelius, Tobias Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman. Mapping the vascular walls of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b99632bcf5e6a6bbd94c1d54966c1ecf742fe" target='_blank'>
              Abstract B029: Mapping the vascular walls of colorectal cancer
              </a>
            </td>
          <td>
            Linglong(凌龙) Huang( 黄 ), Mercedes Herrera, Jonas Sjölund, Vladimir Chocoloff, S. Joost, Rasul M. Tabiev, Lina Wik Leiss, Carina Strell, Luis Nunes, A. Mezheyeuski, D. Edler, A. Martling, Fredrik Pontén, B. Glimelius, T. Sjöblom, Maria Kasper, Kristian Pietras, Arne Östman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eeb1135b7c75ad3978d67f0a6864e002ffcff1b" target='_blank'>
              Comprehensive single-cell aging atlas of healthy mammary tissues reveals shared epigenomic and transcriptomic signatures of aging and cancer.
              </a>
            </td>
          <td>
            B. Angarola, Siddhartha Sharma, Neerja Katiyar, Hyeon Gu Kang, Djamel Nehar-Belaid, SungHee Park, Rachel Gott, Giray N Eryilmaz, Mark A. LaBarge, Karolina Palucka, Jeffrey H. Chuang, Ron Korstanje, D. Ucar, Olga Anczukόw
          </td>
          <td>2024-11-25</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Recurrence and metastasis are the major issues for papillary thyroid cancer (PTC). Current morphological and molecular classification systems are not satisfied for PTC diagnosis due to lacking variant-specific morphological criteria and high signal-to-noise in mutation-based diagnosis, respectively. Importantly, intratumor heterogeneity is largely lost in current molecular classification system, which can be resolved by single cell RNA sequencing (scRNA-seq). However, scRNA-seq loses spatial information and morphological features. Herein, scRNA-seq is integrated and spatially-resolved transcriptomics (SRT) to elaborate the mechanisms underlying the spatial heterogeneity, malignancy and metastasis of PTCs by associating transcriptome and local morphology. This results demonstrated that PTC cells evolved with multiple routes, driven by the enhanced aerobic metabolism and the suppressed mRNA translation and protein synthesis and the involvement of cell-cell interaction. Two curated malignant and metastatic footprints can discriminate PTC cells from normal thyrocytes. Ferroptosis resistance contributed to PTC evolution. This results will advance the knowledge of intratumor spatial heterogeneity and evolution of PTCs at spatial and single-cell levels, and propose better diagnostic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870c25c5831bd8c1742183d121d62d6970a89470" target='_blank'>
              Spatial and Single-Cell Transcriptomics Unraveled Spatial Evolution of Papillary Thyroid Cancer.
              </a>
            </td>
          <td>
            Guangzhe Zheng, Shaobo Chen, Wanqi Ma, Quanshu Wang, Li Sun, Changwen Zhang, Ge Chen, Shuping Zhang, Shuguang Chen
          </td>
          <td>2024-11-14</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="

 Background: Acute myeloid leukemia (AML) is a heterogeneous disease characterized by arrested differentiation and clonal expansion. While certain efforts have been devoted to integrating single-cell data in AML, there is a clear need to build an interactive atlas fully annotated with clinical, demographic, molecular, cytogenetic and treatment data, providing comprehensive sources and customized functions for analysis. To mend this gap, we developed an interactive Comprehensive Atlas of Single cells in AML (iCASA) platform which can enhance the understanding of the cellular communication, transcriptomic regulation, immune repertoires, and cell population changes over the course of AML evolution in various treatment settings.
 Methods: Single cell RNA profiling (scRNAseq) of 73 bone marrow (BM) samples from 43 AML patients and 9 healthy BM samples including our inhouse previously published and newly generated data were included. Batch effects were evaluated by Harmony. Cell types were annotated by integration of canonical markers, flow cytometry, conventional cytogenetics, inferCNV, FISH, and manual revision. Reference mapping of AML cells was projected by Symphony. Data-sharing platform Shiny App was built in R for interactive use.
 Results: AML BM samples included 22 newly diagnosed patients who received venetoclax-based therapy, 13 newly diagnosed patients who received frontline therapy without venetoclax and 8 relapsed/refractory patients. Of those, 19 patients had paired samples (2 to 3) pre and post therapy, providing longitudinal assessment of AML-immune cell interactions. The median age of the population at time of diagnosis was 71 years (range 38 - 89). The most frequent mutation was TP53 (39%), followed by IDH2 (23%), and DNMT3A (23%) and 44% patients had del7/7q. After preprocessing, 404,819 high-quality cells were retained. To facilitate the public utility of our resource and drive future discoveries, the iCASA features were divided into 8 tabs: 1) Dimensionality reduction visualizes distribution of cells by UMAP or tSNE where pathways of interest can be projected to assess biological functions in different cellular states. 2) Differential expression analysis identifies differential genes across cell types, normal versus tumor, with interactive visualization such as heatmaps and volcano plots. 3) Correlation analysis performs pairwise gene correlation analysis at pseudo-bulk level for modules of interest. 4) Signature Scoring of user-provided gene sets or predefined pathway signatures enable users to look at pathway activities across different cell types and clinical subgroups. 5) Survival analysis provides overall survival curve based on gene expression, pathway scores, or gene regulator modules (including transcription factor activity) 6) Cell-cell communication predicts ligand-receptor interaction between cell types by CellChat. Visualization includes circle plot, chord diagram, and heatmap. 7) Trajectory analysis determines the dynamic process of cell differentiation. 8) Immune repertoires (scTCR and scBCR) paired with immune cell subtypes to examine markers of immune response.
 The iCASA platform will be available on a free web-access portal for users to explore. The interactive interface makes it accessible even to those without prior experience in computational biology. Additionally, users can download the raw data for use on local machines. As an application of the portal, we identified 12 broad cell clusters and 38 specific cell states spanning hematopoietic stem cells to differentiated cells. T cells were the dominant lineage in the niche (56%). AML cells were mapped by a linear mixture model to hematopoietic cells from 9 healthy donors to determine their cellular states. The overall cellular states showed significant diverse distribution across patients (p < 2.2x10-16), indicating high intra-tumoral heterogeneity of AML patients. The web portal is now in demo mode and expected to go live in January 2025.
 Conclusions: We provided a large scRNAseq resource of hematopoietic differentiation in AML patients. This integrated information is a valuable resource to characterize dynamic changes in AML patients with chemotherapy, understand response to combined therapies. Ongoing analyses to derive biological insights include identifying meta clusters associated with cellular states, evaluating dynamic changes in different myeloid subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d49617268780472c259a570fae0d0b19343e04" target='_blank'>
              ICASA: An Interactive Comprehensive Atlas of Single Cells in AML from 82 Bone Marrow Samples
              </a>
            </td>
          <td>
            Bofei Wang, Poonam N Desai, Jae Jun Ku, N. Daver, P. Reville, Hussein A. Abbas
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 All glioblastoma (GBM) molecular subsets share the common trait of accelerated progression following necrosis, which cannot be adequately explained by cellular proliferation arising from accumulated genetic alterations. We suggest that necrosis is much more than a passive phenomenon related to rapid growth but rather is a driving force causing tumor microenvironment (TME) restructuring and biologic progression. yet mechanisms remain poorly understood due to a lack of animal models to study necrosis as a primary variable. In GBM, the most malignant primary brain tumor, vascular pathology and central necrosis precede rapid, radial expansion. To reveal spatio-temporal changes directly following vascular damage in the brain, we developed methodology to induce clinically relevant thrombosis vaso-occlusion within GBMs in immunocompetent RCAS/tv-a mouse model to study TME restructuring by intravital microscopy and demonstrate its impact on glioma progression in real-time. We found that following Rose Bengal photothrombosis, GBMs rapidly expanded radially, with glioma migration away from central necrosis and tumor-associated macrophages (TAMs) and glioma stem cells (GSCs) increased dramatically in the peri-necrotic zones. Through the single cell tracking analysis, we found that TAMs displaying increased displacement toward necrosis and greater overall migration distances. Collectively, this model introduces necrosis as the primary variable and captures glioma growth dynamics and TME restructuring in a manner that will facilitate the development of therapies that antagonize these mechanisms to improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b22bc4e43477f4a1b90ea25ae0545995cb46b8f" target='_blank'>
              TMOD-14. ESTABLISHING A CLINICALLY RELEVANT MOUSE MODEL TO STUDY NECROSIS AS A PRIMARY VARIABLE IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Jiabo Li, Ling-Kai Shih, Steven M. Markwell, Cheryl Olson, David Sullivan, Nicholas Lam, Daniel Brat
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150e626daa762108097ae88ebcf26c0674670dd5" target='_blank'>
              Isolation and identification of patient-derived liver cancer stem cells and development of personalized treatment strategies
              </a>
            </td>
          <td>
            Tingting Guo, Shuai Zhang, Weiping Zeng, Yan Liang, Jinghe Xie, Shoupei Liu, Yaqi Qiu, Ying Fu, Y. Ou, Keqiang Ma, Bailin Wang, Weili Gu, Yuyou Duan
          </td>
          <td>2024-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b397d7b281fa7e047bec3772e9d91f03230a0f4" target='_blank'>
              Profiling the cancer-prone microenvironment in a zebrafish model for MPNST.
              </a>
            </td>
          <td>
            Cheryl Cero, John S. House, Vincenzo Verdi, J. L. Ferguson, Dereje D. Jima, Aubrie A. Selmek, Olivia M. Patania, Jennifer E. Dwyer, Bih-Rong Wei, Dillon Lloyd, H. Shive
          </td>
          <td>2024-11-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 The clonal architecture of acute myeloid leukemia (AML) can be highly complex and heterogeneous. This genetic diversity complicates the treatment of AML, as different clones may respond variably to therapies, contributing to treatment resistance and disease relapse. Understanding the mechanisms and implications of clonal evolution in AML is crucial for developing more effective treatment strategies that can target the diverse cellular populations.
 The generation of patient-derived xenograft (PDX) models has revolutionized the study of AML. One of the main uses of PDX models is to study how a leukemic sample might respond to different therapies. Here, we used single-cell DNA sequencing (scDNA-seq) and surface protein profiling to monitor the clonal evolution and differentiation of an isocitrate dehydrogenase 1 (IDH1)-mutated leukemic sample in response to ivosidenib (IVO), a mutant IDH1 inhibitor, either as monotherapy or in combination with venetoclax (VEN) or azacitidine (AZA). Single-cell mutation analysis of the engrafted cell population showed that the initial evolutionary path was linear characterized by the sequential acquisition of DNMT3AR882H (clone 1), WT1R467W (clone 2), IDH1R132H (clone 3), NPM1Trp288Cysfs*12 (clone 4), FLT3-ITD (clone 5), and WT1R439C (clone 6) mutations. Three subclones branched off at the end of this linear path with each acquiring either a TET2Q1548del (clone 7), DNMT3AD531del (clone 8), or KITH40QfsTer6 (clone 9) mutation. The overall clonal composition of the input primary sample and sample from vehicle-treated animals were largely similar. To quantify the differences in therapeutic response between clones, we developed a hierarchical multinomial Bayesian model that estimated the fold change in absolute cell numbers for each clone compared with vehicle treatment. This analysis revealed that both single-agent VEN and IVO+VEN were generally more effective in targeting earlier-stage clones (clones 3-5) than later-stage clones (clones 6-9). In contrast, single-agent AZA and IVO+AZA were effective in eliminating clone 6 as well as earlier-stage clones. Notably, the combination of IVO+AZA was effective in reducing all the clones by approximately the same magnitude, including clones 7 and 8 which demonstrated lower sensitivity to AZA monotherapy. Given that the WT1R439C mutation, acquired in clone 6, distinguished early-stage from late-stage clones, these findings suggest that WT1 mutations might contribute to IVO and VEN resistance and that the addition of AZA can overcome this mechanism of resistance. Consistent with these findings, the combination of IVO+AZA strongly upregulated the expression of the myeloid markers CD11b, CD14, and CD15 to a greater extent than with either single agent. These findings provide evidence that AZA and IVO could synergize to overcome the differentiation block in IDH1-mutated AML cells, reflecting the superior clinical efficacy of this regimen over single-agent IVO.
 To broaden the applicability of PDX models to study the competition between genetic clones that co-exist infrequently in an individual sample, we generated mixed PDX (mPDX) models by co-engrafting two or more leukemic samples in the same animal and used scDNA-seq to deconvolute the clonal composition and sample origin of the engrafted cells. As proof-of-principle, we generated an IDH1R132H/IDH2R140Q mPDX to model isoform switching in both directions by treating the animals with either IVO or enasidenib (ENA), a mutant IDH2 inhibitor. We observed the expected differentiation response and depletion of IDH1-mutated or IDH2-mutated clones by IVO or ENA treatment as single agents, respectively. Further studies are underway to demonstrate whether dual mutant IDH1 and IDH2 inhibition (IVO+ENA) will circumvent resistance driven by isoform switching.
 In summary, we demonstrate the utility of applying single-cell proteogenomic analysis in traditional and mixed PDX models to gain crucial insights into mechanisms of resistance and potential strategies to overcome it.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad2ac5a87afb275fd53cef27ee87a6e28e0b788" target='_blank'>
              Single-Cell Proteogenomic Analysis of Clonal Evolution in Patient-Derived Xenograft Models of AML Treated with IDH Inhibitors
              </a>
            </td>
          <td>
            Alex Liu, S. Cathelin, Yitong Yang, Farzaneh Aboualizadeh, Amina Abow, G. Basi, Lance Li, David L Dai, Abdula Maher, Eric Grignano, Mohsen Hosseini, Dhanoop Manikoth Ayyathan, Vivian Wang, Troy Ketela, Dylan M Marchione, Adriana Tron, A. Arruda, Mark D. Minden, Steven M Chan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 DICER1 cancer predisposition syndrome is a heritable condition where affected patients have increased risk of neoplasms including fusion-negative rhabdomyosarcoma (FN-RMS). The syndrome is defined by germline heterozygous loss of function mutations in DICER1, but it remains unclear exactly how these mutations predispose children to cancer. To gain insight into the role of Dicer1 loss in DICER1 cancer predisposition, we deleted the Dicer1 gene both in the germline or conditionally within the tumor cell in our previously established mouse model of FN-RMS. One would expect the germline loss of one allele to provide a sensitized background in the tumor cell to acquire a second hit mutation. However, we observed faster tumor onset and increased penetrance in mice with germline heterozygous Dicer1 loss (Dicer1 +/- ) but not in mice when Dicer1 loss was restricted to the tumor cells (Dicer1 Flox/+ ), demonstrating that haploinsufficient tumor suppressor activity of Dicer1 loss requires non-cell autonomous cell types. Single cell RNA sequencing (scRNA-Seq) revealed significant expansion of immature neutrophils in Dicer1 +/- tumors, which were enriched for proteases associated with neutrophil extracellular traps (NETs). NETs are webs of chromatin and proteases released when neutrophils in response to inflammatory stimuli in process termed NETosis. We show Dicer1 +/- murine and human tumors are enriched for neutrophils and tumor-bearing mice had abundant circulating NETs. NETs cause significant damage to the local extracellular matrix (ECM) and have been implicated in tumor promotion. In addition, neutrophil enrichment, we found that Dicer1 +/- tumors show significant ECM remodeling and are increased C5a, a potent neutrophil chemoattractant and NET-priming factor. These results suggest NETosis is upregulated in Dicer1 +/- FN-RMS tumors. Ligand-receptor pair analysis (iTALK) on the scRNA- Seq data revealed a putative interaction between a NET-derived ligand and EGFR and IGF1R on tumor cells, suggesting direct tumor promoting signaling from NET-to-tumor. We validated this novel FN-RMS promotion mechanism in vitro using human FN-RMS cell lines and observed that this NET-derived ligand promoted FN-RMS growth through both EGFR and IGF1R signaling. NET release requires citrullination of histone H3 mediated by peptidyl arginine deiminase 4 (PADI4) to facilitate decondensation of chromatin. We intercrossed the Padi4 -/- allele into our tumor model to genetically block NETosis and observed complete rescue of the accelerated tumor onset and increased penetrance observed in Dicer1 +/- mice. These results demonstrate that NETosis promotes the growth of Dicer1 +/- FN-RMS tumors and is required for Dicer1 +/- haploinsufficiency. These findings may be applicable to many DICER1 syndrome- associated cancers, as neutrophils which get converted to tumor promoters by heterozygous DICER1 loss are present in DICER1 syndrome patients regardless of tumor type and suggest targeting neutrophils/NET-release may reduce cancer risk in DICER1 +/- individuals.
 Citation Format: Randolph K Larsen, Jason A Hanna, Hongjian Jin, Kristen B Reed, Darden W Kimbrough, Kyna Voung, Myron K Evans, Casey G Langdon, Catherine J Drummond, Matthew R Garcia, David Finkelstein, Patrick A Schreiner, Jerold E Rehg, Kris Ann P Schultz, Mark E Hatley. DICER1 cancer predisposition tumor promotion mediated non-cell autonomously via neutrophils and NETosis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266771b89aebf534b5e742d8cc356c61afa9ade6" target='_blank'>
              Abstract PR010: DICER1 cancer predisposition tumor promotion mediated non-cell autonomously via neutrophils and NETosis
              </a>
            </td>
          <td>
            Randolph K. Larsen, Jason A. Hanna, Hongjian Jin, Kristin B Reed, Darden W. Kimbrough, Kyna Voung, Myron K Evans, Casey G Langdon, Catherine J Drummond, Matthew R. Garcia, David Finkelstein, Patrick A Schreiner, J. Rehg, Kris Ann P Schultz, M. Hatley
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past century, colorectal cancer (CRC) has become one of the most devastating cancers impacting the human population. To gain a deeper understanding of the molecular mechanisms driving this solid tumor, researchers have increasingly turned their attention to the tumor microenvironment (TME). Spatial transcriptomics and proteomics have emerged as a particularly powerful technology for deciphering the complexity of CRC tumors, given that the TME and its spatial organization are critical determinants of disease progression and treatment response. Spatial transcriptomics enables high-resolution mapping of the whole transcriptome. While spatial proteomics maps protein expression and function across tissue sections. Together, they provide a detailed view of the molecular landscape and cellular interactions within the TME. In this review, we delve into recent advances in spatial biology technologies applied to CRC research, highlighting both the methodologies and the challenges associated with their use, such as the substantial tissue heterogeneity characteristic of CRC. We also discuss the limitations of current approaches and the need for novel computational tools to manage and interpret these complex datasets. To conclude, we emphasize the importance of further developing and integrating spatial transcriptomics into CRC precision medicine strategies to enhance therapeutic targeting and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b317770be242dc222a3004766249a5b9de43dcc" target='_blank'>
              Current state and future prospects of spatial biology in colorectal cancer
              </a>
            </td>
          <td>
            Francisco G. Carranza, Fernando C. Diaz, Maria Ninova, Enrique Velazquez-Villarreal
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Introduction:
 A precise depiction of intratumoral clonal diversity reveals the processes of clonal evolution and diversity within tumors. Comprehensive genetic analyses and inference of clonal structure in B-cell lymphomas, such as diffuse large B-cell lymphomas (DLBCL) and follicular lymphomas (FL), have been performed using variant allele fractions (VAFs) obtained from bulk samples. However, this method is limited in terms of predicting precise clonal structure, cell-level co-occurrence, and chronological order of genetic alterations. Moreover, the clonal relationship between the genotype and phenotype at the single-cell level has not been well characterized in B-cell lymphomas.
 Methods:
 Simultaneous single-cell DNA sequencing (scDNA-seq) and cell surface protein analyses were performed using the Tapestri platform (Mission Bio). We analyzed 32 samples from 30 patients with B-cell lymphomas, consisting of 12 DLBCLs and 18 FLs. In two cases, both lymph node and bone marrow tumor samples were analyzed. We designed 298 amplicons to cover 61 recurrently mutated genes in FL and DLBCL samples. We also selected 68 custom antibodies with oligotags to capture cell surface protein expression related to B cell differentiation (e.g., CD10, CD19, CD20, CD138, and CD83) and interactions with the tumor microenvironment (e.g., MHC PD-1, PD-L1, CTLA-4, LAG3, BTLA, and LIGHT). The sequenced data were processed using the Tapestri Pipeline for variant calling and multiple filtering processes (Miles L et al., Nature 2020). Phylogenetic trees were inferred using the COMPASS package (Sollier E et al., Nat Commun 2023).
 Results:
 We sequenced a median of 6,501 cells per sample (range: 1,039-18,099). A total of 223,336 cells were annotated using conventional B/T cell lineage markers to identify B cells (120,384), CD4-T-cells (38,863), and CD8-T-cells (21,523), with a median coverage of 78 reads per cell per amplicon. Through scDNA-seq, 97 variants were detected (median: 3.5; range: 1-10 per sample) which were used for further analysis. Notably, the average VAFs derived from scDNA-seq exhibited good concordance with the VAFs obtained from matched bulk whole-exome sequencing data (n=15), confirming the reliability of scDNA-seq (R=0.89).
 The most frequent mutations identified through scDNA-seq in DLBCL cases were CREBBP, TP53, B2M, and MYD88 (n=4, 33%), and CARD11 and HIST1H1E (n=3, 25%). In FL cases, CREBBP mutations were the most frequently detected (n=8, 44%), followed by EZH2, TNFRSF14, and STAT6 mutations (n=4, 23%). Notably, all CREBBP mutations detected in FL cases were found in the initiating cells of pyrogenic trees, suggesting a strong role in initiating mutations in FL. MEF2B, HIST1H1E, and GNA13 mutations were also found in the initiating cells, whereas KMT2D mutations were mainly detected in subclonal cells in FL. Consistent with the findings of previous bulk studies, all TP53 mutations were found in the subclonal cells in both DLBCL and FL cases. Strikingly, all six STAT6 mutations (D419G (n=3), D419N, N421K, and N430T) were detected in subclonal cells, suggesting a contribution to clonal expansion and progressive FL disease.
 Spatially separated tumors were found to share founder genetic alterations at the single-cell level, and additional alterations were acquired in lymph node (e.g., CREBBP CN-LOH and CTSS Y132D), but not in bone marrow.
 scDNA-seq also provided information on co-occurrence and mutual exclusiveness at the cellular level. In the case of both MYD88 (L265P) and CD79B mutations (MCD-type DLBCL), although these two gene mutations were concurrently found in most lymphoma cells, 39 of 1,523 lymphoma cells had only MYD88 mutation. Similarly, in an FL case harboring both CREBBP and EP300 mutations, these mutations diverged into two independent branches, indicating mutual exclusion at the cellular level.
 Conclusion:
 scDNA sequencing revealed the clonal structures and evolutionary processes of B-cell lymphomas at the cellular level. Specifically, some driver mutations were found to be mutually exclusive, which contrasts with the findings from previous bulk-sequencing studies that showed their co-occurrence at the patient level. Simultaneous profiling of single-cell DNA and cell surface proteins is ongoing, which will provide further insights into the genetic and phenotypic profiles of intratumor heterogeneity in B-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a085eaf5798489d72f8e4a8edabf00a264c2c0e1" target='_blank'>
              Single-Cell DNA Sequencing Reveals Cell-Level Genetic Evolutionary Processes and Clonal Structure in B-Cell Lymphomas
              </a>
            </td>
          <td>
            Hiroyuki Murakami, R. Chijimatsu, H. Ujiie, Y. Naoi, Kazutaka Sunami, Kazuhiko Yamamoto, Soichiro Fujii, Tomofumi Yano, K. Seike, Hideaki Fujiwara, Noboru Asada, N. Fujii, Ken-Ichi Matsuoka, T. Urata, Kazuhiro Ikeuchi, Hiroki Kobayashi, Yasuharu Sato, Shuta Tomida, Y. Maeda, D. Ennishi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive and life-threatening cancer, poses a significant therapeutic challenge, particularly for patients with distant metastasis, who face a 5-year survival rate of only approximately 22.5%. This underscores the urgent need for innovative therapeutic approaches. Emerging evidence suggests that sensory and parasympathetic innervation within the tumor microenvironment (TME) plays a crucial role in driving metastasis and immune evasion, partly through the promotion of M2 macrophages and cancer stem cells. To uncover the mechanisms behind tumor-specific innervation in metastatic melanoma, we employed a systems biology approach integrating bulk RNAseq from diverse melanoma cell lines, TCGA-SKCM data analysis, single-cell RNAseq, and spatial transcriptomics. This comprehensive analysis led us to identify a previously uncharacterized gene regulatory network (GRN) orchestrating tumor innervation. This GRN is driven by axon guidance cues (Netrin, Slit, Semaphorin), their receptors (DCC, Robo, Neuropilin, ITGB1), and transcription factors (RAS, RND1, TFAP2, CAMKII). We further discovered that the pioneer factor FOXC2, known for its association with metastasis, plays a critical role in opening the promoters of innervation-specific genes. Additionally, we identified Ephrin and CXCR4-mediated crosstalk between neurons and tumor cells. Machine learning models validated the accuracy of this GRN in predicting melanoma metastasis, and a neural network-based model confirmed the enrichment of nerve cells near metastatic tumor cells within the TME in TCGA immunohistochemistry data. The functional relevance of this network was further supported by CRISPR-mediated knockout studies, which disrupted tumor-specific signaling pathways. Notably, we also observed the enrichment of genes linked to mRNA processing bodies and neuron-specific membrane-less condensates in metastatic melanoma. These findings highlight the critical role of neuronal infiltration in melanoma metastasis. Currently, therapeutic strategies targeting this aspect of the TME remain limited. Our identified gene set offers potential as a metastatic predictor, and targeting the neural component of the TME represents a promising new therapeutic avenue for melanoma.
 Citation Format: Anjana Goli, Subhajit Dutta. Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a5d79cdb084a3fb556d9c17280ebbf7e5cef77" target='_blank'>
              Abstract B024: Deciphering the neurogenic tumor microenvironment: A novel gene network driving innervation in metastatic melanoma offers prognostic and therapeutic insights
              </a>
            </td>
          <td>
            Anjana Goli, Subhajit Dutta
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Diffuse gliomas are the most prevalent primary malignant brain tumors in adults and are characterized by invasive growth and poor prognosis. Tumor cells infiltrate the brain parenchyma and spread to distant sites. The aim of this study is to characterize the spatial heterogeneity, mode, and direction of tumor development to identify key areas for localized treatment. Neuronavigation was employed to collect n=134 image-guided samples from n=16 adult patients with diffuse gliomas, including n=7 IDH wild-type (IDHwt) and n=9 IDH mutant (IDHmut) tumors from regions with and without MRI abnormalities. Collected samples were subjected to targeted and shallow whole-genome sequencing. Somatic variants were used to determine the evolutionary processes governing heterogeneity. Examination of the dN/dS ratio, Tajima’s D score, and MOBSTER model suggested that all tumors are consistent with neutral evolution, as previously reported. Considering that genetic heterogeneity was a result of stochastic mutations, we used phylogeographic relationships between samples to dissect the spatiotemporal development of these tumors. Evolutionary (patristic) and cartesian (Euclidean) distances between sample pairs from the same patient were correlated in n=2/9 IDHmut and n=2/7 IDHwt patients, suggesting that migratory or punctuated evolutionary processes play a major role in tumor diffusion. An examination of spatial diffusion revealed that the latest descendant appeared peripherally in n=5/9 IDHmut patients, suggesting boundary-driven growth. In the remaining cases, three patients exhibited centrally-driven growth, whereas a fourth patient displayed a mixed pattern. In contrast, in n=4/7 IDHwt patients, descendants appeared closer to the tumor center, suggesting centrally-driven growth. In the remaining three IDHwt patients, one patient showed a diffusion that was consistent with boundary-driven growth, while the other two patients showed a mixed pattern. Taken together our data suggest that IDHwt tumors tend to develop centrally, whereas IDHmut tumors preferably develop outward. Our data could aid in tailoring localized treatment based on projected growth patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5d56574d35051c969272c65183fe55a4346f24" target='_blank'>
              CNSC-54. CENTRAL AND BOUNDARY-DRIVEN GROWTH PATTERNS DOMINATE RESPECTIVELY IDH WILD-TYPE AND MUTANT GLIOMAS
              </a>
            </td>
          <td>
            Maria Kyriakidou, Konnie Urbaniak, Mimi Mbegbu, Russell Rockne, P. Wesseling, R. Eijgelaar, K. Anderson, R. Verhaak, Phillip de Witt-Hamer, N. Verburg, Sergio Branciamore, Floris P Barthel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract High-grade serous ovarian cancer (HG-SOC), accounting for 70–80% of ovarian cancer deaths, is characterized by a widespread and rapid metastatic nature, influenced by diverse cell types, cell–cell interactions, and acellular components of the tumour microenvironment (TME). Within this tumour type, autocrine and paracrine activation of the endothelin-1 receptors (ET-1R), expressed in tumour cells and stromal elements, drives metastatic progression. The lack of three-dimensional models that faithfully recapitulate the unique HG-SOC TME has been the bottleneck in performing drug screening for personalized medicine. Herein, we developed HG-SOC tumouroids by engineering a dense central artificial cancer mass (ACM) containing HG-SOC cells, nested within a compressed hydrogel recapitulating the stromal compartment comprising type I collagen, laminin, fibronectin, and stromal cells (fibroblasts and endothelial cells). ET-1-stimulated HG-SOC cells in the tumouroids showed an altered migration pattern and formed cellular aggregates, mimicking micrometastases that invaded the stroma. Compared with control cells, ET-1-stimulated tumouroids showed a higher number of invasive bodies, which were reduced by treatment with the dual ET-1 receptor (ET-1R) antagonist macitentan. In addition, ET-1 increased the size of the invading aggregates compared with control cells. This study establishes an experimental 3D multicellular model eligible for mechanical research, investigating the impact of matrix stiffness and TME interactions, which will aid drug screening to guide therapeutic decisions in HG-SOC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15db46d3514b1f218b5237d92ed54e0d18d30187" target='_blank'>
              Endothelin-1 receptor blockade impairs invasion patterns in engineered 3D high-grade serous ovarian cancer tumouroids
              </a>
            </td>
          <td>
            Judith Pape, Umber Cheema, Piera Tocci, Rosanna Sestito, Ilenia Masi, Marilena Loizidou, A. Bagnato, L. Rosanò
          </td>
          <td>2024-11-01</td>
          <td>Clinical Science (London, England : 1979)</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="

 Graft-versus-leukemia (GvL) responses enable successful allogeneic hematopoietic cell transplantation (HCT), a cornerstone treatment for high-risk blood cancers such as acute myeloid leukemia (AML) or advanced chronic lymphocytic leukemia (CLL). Yet, monitoring of GvL effects remains challenging, including early detection of emerging immune escape or tracking of clonal evolution during phases of incomplete leukemia control. We hypothesized that mitochondrial DNA (mtDNA) mutation analysis would be an effective tool for sensitive monitoring of clonal evolution and measurable residual disease (MRD) that could yield new insights into the dynamics and cellular phenotypes of GvL response or resistance.
 To establish mtDNA mutations as natural genetic barcodes capable of resolving leukemic evolution, we analyzed 126 serially collected bulk RNA and whole-exome sequencing (WES) datasets of CD19+ sorted CLL cells from 34 individuals (median interval first-to-last sample 2,184 days, range 280-4,402). We discerned high correlation between changes in heteroplasmy of mtDNA mutations (RNA-seq) and cancer cell fraction (CCF, calculated from WES data) (r=0.84; p<0.001). In particular, heteroplasmy and CCF changes were smaller in genetically stable (n=8) compared to evolving (n=18) CLL (coefficient of variation 24 vs 60 [heteroplasmy] and 31 vs 57 [CCF]; p<0.001). Following HCT (n=8), shorter vs longer days-to-relapse (median 310 vs 1,082) was associated with lesser vs more changes in heteroplasmy (p=0.034), suggesting the ability to measure cumulative GvL effects with mtDNA mutations. In 3 cases, we obtained single cell (sc) DNA-seq profiles (Tapestri) from circulating CLL before and after HCT and directly detected co-evolution of somatic nuclear with mtDNA mutations. Thus, mtDNA mutations track with leukemia clones and their dynamics are highly consistent with changes in somatic nuclear DNA mutations.
 To assess mtDNA mutations as sensitive markers of MRD, we determined their median limit-of-detection in bulk RNA-seq data as 0.3% (0.008-73%). This high sensitivity enabled tracking of resistant leukemia selected by GvL, such as in one CLL case, where a mtDNA mutation (6426G>A) expanded from 0.2% (pre-HCT) to 54% heteroplasmy (post-HCT) over a period of 6 years. By scDNA-seq, we validated that 6426G>A marked a CLL subclone represented by 3 of 2,416 cells (0.12%) pre-HCT that expanded in the post-HCT relapse.
 Donor/recipient deconvolution using mtDNA mutations is another approach for sensitively measuring post-HCT chimerism and MRD. We confirmed that mtDNA mutations could track rare cells at frequencies of 0.1-0.3% by single cell ATAC sequencing (scATAC-seq) in in-silico and in-vitro simulations of post-HCT MRD. In 10 primary human bone marrow samples, we defined residual leukemia cell phenotypes with scATAC-seq and capture of surface markers (ASAP-seq) at time of incipient AML relapse post-HCT. We also tracked residual donor hematopoiesis in 14 bone marrow (BM) samples of 4 frank post-HCT AML relapse cases with ASAP-seq, detectable in 0.1%-0.6% of cells and mostly of erythroid lineage. Together, mtDNA-based MRD tracking is highly sensitive, and when coupled with multimodal single cell sequencing, can define the phenotypes of residual leukemia.
 Finally, we charted clonal evolution in 21 BM samples of 5 AML cases with ASAP-seq, either at recurrence post-HCT (n=3) or during ipilimumab-based relapse treatment (n=5; ETCTN/CTEP 10026, NCT02890329). By mtDNA analysis, we defined a median of 7 AML subclones per case (range 5-12). In contrast to 2 instances with little evidence of clonal evolution after matched unrelated HCT, in 1 case, haploidentical transplantation led to pronounced subclonal shifts, revealing differential intensity of GvL. Throughout decitabine/ipilimumab treatment in post-HCT AML relapse, we found clonal and phenotypic evolution even in cases that stabilized only transiently, consistent with activated GvL countered by immune escape. In 3 cases, we validated the clonal shifts with scDNA-seq and directly demonstrated co-segregation and co-evolution of mitochondrial with somatic nuclear DNA mutations. In sum, mtDNA mutations resolve post-HCT immune pressures through dissection of clonal evolution.
 Together, mtDNA mutations enable the sensitive monitoring of the phenotypes and selection of residual leukemia cells after HCT and reveal differential activity of GvL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e887d7fda5f301c323fb075aa55a5c1f0f8334d7" target='_blank'>
              Leukemia Evolution and Resistance to Graft-Versus-Leukemia Responses Revealed By Mitochondrial DNA Mutations
              </a>
            </td>
          <td>
            L. Penter, Nicoletta Cieri, K. Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A Lareau, Leif S Ludwig, D. Neuberg, Haesook T. Kim, Shuqiang Li, Lars Bullinger, Jerome Ritz, G. Getz, Jacqueline S. Garcia, R. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="
 Circulating Tumor Cells (CTCs) hold the promise of comprehensive yet noninvasive molecular characterization of cancer cells during the course of tumor evolution and progression. To dissect the epigenetic profile of prostate CTCs, we undertook paired DNA methylation and RNA sequencing of single CTCs isolated from blood specimens of patients with prostate cancer. Across the genome, our study identified 40 core Partial Methylation Domains (PMDs) that arise early in tumorigenesis and are shared by all individual CTCs across multiple patients. Key among these is a single genomic locus harboring multiple interferon-inducible genes along with all the CD1 gene family members, implicated in innate immunity. Based on mouse models, we propose that early biallelic silencing of multiple genes residing within a single PMD may contribute to the escape from immune surveillance in prostate and potentially other cancers. Studies of CTC heterogeneity within individual patients are limited by the very small number of cells that can be isolated from standard 10ml blood volumes. To enable such analyses, we applied a new high throughput microfluidic platform, capable of processing an entire leukapheresis product, interrogating 2-3 L of blood volume equivalent. Applying this technology to patients with metastatic prostate cancer, we successfully enriched thousands of CTCs from individual cases, enabling the characterization of intra-patient heterogeneity at the level of cell morphology and marker expression, chromosome copy number variation and exome-wide genotyping, and transcriptionally-defined tumor cell subpopulations. Such high-volume microfluidic enrichment of CTCs constitutes a new dimension in liquid biopsies.
 Citation Format: Daniel A Haber, Shyamala Maheswaran, Hongshan Guo, Joanna Vuille, Shih-Bo Huang, Ben S Wittner, Lecia V Sequist, Richard J Lee, Patricia AR Brunker, David T Miyamoto, Avanish Mishra, Mehmet Toner. Tumor cell-based liquid biopsy to characterize prostate cancer [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr IA005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe44dae01b6a504aeda595e4d36a826d45670615" target='_blank'>
              Abstract IA005: Tumor cell-based liquid biopsy to characterize prostate cancer
              </a>
            </td>
          <td>
            D. A. Haber, S. Maheswaran, Hongshan Guo, Joanna A. Vuille, Shih-Bo Huang, Ben S. Wittner, L. Sequist, Richard J. Lee, Patricia A. R. Brunker, D. Miyamoto, Avanish Mishra, Mehmet Toner
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="

 Myeloid neoplasms are clonal disorders driven by acquisition of somatic mutations in hematopoietic stem cells (HSCs). HSCs acquire mutations at each cell division, most of which are thought to be effectively neutral, while only recurrent driver mutations in a relatively small number of cancer-causative genes are thought to promote clonal expansion of HSCs, or clonal hematopoiesis (CH), and leukemia progression. However, nearly 50% of CH cases are not associated with an annotated driver mutation, suggesting that the true number of drivers might be much larger than catalogued to date. Since CH infrequently progresses to myelodysplastic syndrome (MDS) or acute myeloid leukemia (AML), the dynamics of antecedent preleukemic clonal evolution in individuals who develop myeloid neoplasms remain poorly understood.
 Here, we use induced pluripotent stem cell (iPSC) reprogramming to track genomic and functional evolution of preleukemic clones in 18 patients with MDS or AML. We generated a panel of 42 iPSC lines spanning all stages of premalignant evolution marked by 1, 2, or 3 annotated driver mutations and isogenic wild-type (WT) clones derived from normal HSPCs in the same patient. To measure genome-wide dynamics of mutation acquisition during clonal evolution, we performed whole exome sequencing of MDS/AML patient samples, preleukemic iPSC clones, and matched WT iPSCs. We defined preleukemic mutations as somatic variants present in the primary MDS/AML patient sample and at least one preleukemic clone but absent in the isogenic WT iPSCs. We found that coding mutation burden in 1-, 2-, and 3-driver clones increased linearly. In a neutral process, the ratio of amino acid-changing to neutral mutations is expected to remain constant. By contrast, we found that 2- and 3-drivers clones harbored a significantly higher proportion of amino acid-changing nonsynonymous and indel mutations compared to 1-driver clones. A regression model incorporating driver and neutral mutation counts showed a significant increase in amino acid-changing mutations during progression relative to neutral counts. These findings show that clonal progression is marked by increasing mutation burden with an excess of amino acid-changing variants, supporting a model of intensified positive selection for unannotated “cryptic” drivers. These genes are likely not annotated as drivers because they are infrequently mutated in myeloid malignancies.
 To understand how cryptic drivers impact HSC fitness, we first sought to better define the impact of recurrent preleukemic DNMT3A R882H mutations. DNMT3A mutations enhanced hematopoietic output of iPSC-derived HSPCs and reduced expression of immune response pathways and HLA genes encoding the major histocompatibility complex class II (MHC-II). MHC-II expression was concordantly reduced on the surface of DNMT3A-mutant HSPCs and dendritic cells. We next carried out single cell transcriptomics on individuals with DNMT3A-mutant CH. Consistent with the iPSC findings, DNMT3A-mutant HSPCs and dendritic cells had decreased expression of MHC-II genes compared to age-matched controls. These data implicate MHC-II as an axis regulated by common preleukemic DNMT3A mutations. We then extended this analysis to candidate cryptic drivers in our dataset. Of these, NSD1 and IRF1 were infrequently mutated (<1%) in patients with MDS and AML and previously implicated as regulators of HLA expression. To investigate how inactivating NSD1 and IRF1 mutations affect hematopoiesis, we performed knockdown (KD) of these genes in human CD34+ cord blood HSPCs. NSD1-KD expanded CD34+CD133+ HSPCs, while IRF1-KD blocked differentiation into myeloid lineages. Notably, both NSD1- and IRF1-KD displayed significant reduction of MHC-II on CD34+CD133+ HSPCs, suggesting that both common and cryptic drivers converge on MHC-II down-regulation.
 In conclusion, tracking preleukemic hematopoiesis by iPSC reprogramming, we present evidence that unannotated cryptic driver mutations are under positive selection in preleukemic evolution and nominate MHC-II reduction as convergent pathway by which common and cryptic drivers promote clonal fitness. We propose that downregulation of antigen presentation via MHC class II may be a common mechanism by which cryptic drivers contribute to clonal expansion.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a2b320f8f46d084f755068cfdbe4a0533e7de7" target='_blank'>
              iPSC Reprogramming Identifies the Role of Cryptic Drivers in Preleukemic Clonal Evolution
              </a>
            </td>
          <td>
            Martina Sarchi, Sintra Stewart, Jason Kim, Ksenia Safina, S. Busch, Hayden Gizinski, Nelli Aydinyan, Andreea Reilly, J. P. Creamer, Rochelle Bergantinos, Jasper Hsu, Jonathan D. Good, Malgorzata Olszewska, A. Gallì, Luca Malcovati, Sioḃán Keel, David Wu, J. Abkowitz, Pamela S Becker, E. Papapetrou, K. Harris, Peter van Galen, S. Gulsuner, Sergei Doulatov
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Desmosomes are transmembrane protein complexes critical for cell-cell adhesion in epithelial tissues and other cell types. In skin cancers like melanoma, epithelial cells called keratinocytes are the predominant cells in the primary melanoma microenvironment, yet it remains largely unknown whether this cell type is subject to genetic alterations during melanoma development and its role in early melanoma progression. In this study, using an integrative analysis of tumor mutations and protein biophysical interactions, we identify a high frequency of genetic alterations in desmosome-related genes in human cancers, with the highest occurrences in cutaneous melanoma. Notably, over 70% of cutaneous melanoma cases exhibit mutations in desmosome genes and desmosome gene mutations are associated with lowered expression of the desmosome complex in melanoma. Using spatial transcriptomics and immunohistochemistry analyses, we find that the decrease in desmosome gene expression predominantly occurs in keratinocytes within the tumor microenvironment rather than in the melanoma cells themselves. To study the functional significance of this alteration in melanoma development, we used a human melanoma/keratinocyte co-culture system using primary melanoma cell lines. Our studies show that knockdown of desmosome genes in keratinocytes markedly increased proliferation of adjacent melanoma cells in melanoma/keratinocyte co-cultures. Additionally, melanoma cell proliferation is enhanced when exposed to media preconditioned by desmosome-deficient keratinocytes. These observations suggest that the gradual accumulation of mutations in desmosome genes within neighboring keratinocytes may create a conducive environment that primes melanoma cells for neoplastic transformation. This study underscores the critical role of genetic alterations in the tumor microenvironment in melanoma progression and highlights the potential for targeting desmosome-related pathways in therapeutic strategies against melanoma.
 Citation Format: Mohita Tagore. Desmosome mutations in the microenvironment regulate melanoma proliferation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cede4aac61d054a3a470265c4e2db5a458ea7c19" target='_blank'>
              Abstract A042: Desmosome mutations in the microenvironment regulate melanoma proliferation
              </a>
            </td>
          <td>
            Mohita M. Tagore
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Measurable residual disease (MRD) assessment is important in acute myeloid leukemia (AML), as post-remission MRD indicates higher relapse risk. Current techniques, however, lack precision in individual outcome prediction, complicated by AML's clonal heterogeneity. Novel single cell MRD (scMRD) workflows based on DNA mutations offer a potential breakthrough, simultaneously capturing genotypic and immunophenotypic changes at a single-cell level. The Tapestri® platform (Mission Bio) incorporates a microfluidic droplet technology with cell barcoding beads and gene-specific primers that enables amplification and comprehensive identification of low-frequency subclones that may lead to overt disease or resistance. With this approach, complex heterogeneity and differentiation between leukemic clones and clonal hematopoiesis can be deconstructed.
 Objective: This study represents the first large-scale scMRD analysis in AML using this novel scMRD technology to deconvolute clonal heterogeneity to identify potential relapse markers and to differentiate between leukemic clones and clonal hematopoiesis.
 Methods: We analyzed a cohort of 69 AML patients in first complete remission (CR) as assessed by bone marrow morphology from the BLAST trial (NCT02502968). For 11 patients, we analyzed paired samples from both first CR and relapse. We performed single-cell DNA + protein sequencing on a total of 250,000 cells across 23 multiplexed pools. Our methodology included 40 selected mutation hotspot genes and a 19-plex antibody oligonucleotide cocktail for AML-specific surface markers as well as patient-specific hash tag antibodies for enhanced demultiplexing. We used magnetic activated cell sorting (MACS) to enrich CD34/CD117-positive blasts to ensure high-quality samples for subsequent analysis.
 Results: Our analyses revealed a diverse genetic landscape with 25 different mutated genes. DNMT3A and NPM1:p.W228Cfs mutations were the most common alterations. The genetic distribution included a predominance of preleukemic genes (51.5%), chromatin remodelers (12.1%), splice factors (9.1%), transcription factors (6.1%), cell cycle regulators (6.1%), and other functional categories (15.2%).
 Lower MRD (and clonal hematopoiesis) levels were consistently associated with improved overall survival (OS) and relapse-free survival (RFS) across different thresholds (<5%, <10%, <20%, <30%, <40% and <50% MRD burden per patient), with patients in the <5% MRD group showing a significant difference in RFS (p = 0.0013). The median overall survival in this group was 18.0 months, while the median relapse-free survival was 11,7 months. OS and RFS remained significantly different in all MRD groups even after exclusion of DNMT3A and TET2 mutations, which most likely represented clonal hematopoiesis.
 In paired samples, TP53 mutations were notably prominent during follow-up, remaining the dominant clone, indicating a mixed composition, and the persistence of treatment resistant clones. Overall, our study highlighted the dynamic nature of clonal evolution under therapeutic pressure by demonstrating continuous clonal diversity during clinical remission and notable changes in clonal composition during relapse.
 Conclusion: In this AML cohort, our study provides insights into the genetic landscape and clonal dynamics of AML in remission. Thus, our findings emphasize the importance and the potential of single-cell DNA + protein sequencing in refining MRD assessment compared to current techniques.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db74b338da09cd55d0a1a600dcec1d4e9bacb1a1" target='_blank'>
              Clonal Dynamics of Leukemic and Clonal Hematopoiesis Mutations Predict Relapse in Single Cell MRD Analysis of AML in First Complete Remission
              </a>
            </td>
          <td>
            Enise Ceran, A. K. Merbach, Christian Rohde, Robert Durruthy-Durruthy, Adam Sciambi, Michelle Lotze, Gema Fuerte, Matteo Cattaneo, Ivan Lukic, Maxi Wass, Judith Schaffrath, Christoph Röllig, R. Schlenk, Marina Scheller, Simone Kowoll, Jörg Steighardt, B. Edemir, Beatrice Ludwig-Kraus, L. P. Mueller, Sabine Edemir, Abi Vainstein-Haras, E. Sorani, Irit Gliko Kabir, S. Kadosh, C. Gromann, Andreas Wienke, Claudia Wickenhauser, C. Baldus, U. Platzbecker, H. Serve, M. Bornhäuser, Carsten Müller-Tidow
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680ac6121a0aae5814a42089f80d9a0e4a9ab7a0" target='_blank'>
              Revealing the structural microenvironment of high metastatic risk uveal melanomas following decellularisation
              </a>
            </td>
          <td>
            K. Aughton, Joshua Hattersley, Sarah E Coupland, H. Kalirai
          </td>
          <td>2024-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Despite recent advances in the treatment of melanoma, many patients with metastatic disease still succumb to their disease. To identify tumor-intrinsic modulators of immunity to melanoma, we performed a whole-genome CRISPR screen in melanoma and identified Setdb1 as well as all components of the HUSH complex. We found that loss of Setdb1 leads to increased immunogenicity and complete tumor clearance in a CD8+ T-cell dependent manner. Mechanistically, loss of Setdb1 causes de-repression of endogenous retroviruses (ERVs) in melanoma cells and triggers tumor-cell intrinsic type-I interferon signaling, upregulation of MHC-I expression, and increased CD8+ T-cell infiltration. Importantly, spontaneous immune clearance observed in Setdb1-/- tumors results in subsequent protection from other ERV-expressing tumor lines, supporting the functional anti-tumor role of ERV-specific CD8+ T-cells found in the Setdb1-/- microenvironment. Blocking the type-I interferon receptor in mice grafted with Setdb1-/- tumors decreases immunogenicity by decreasing MHC-I expression, leading to decreased T-cell infiltration and increased melanoma growth, comparable to Setdb1wt tumors. Together, these results provide key in vivo evidence of a critical role for Setdb1 and type-I interferons in generating an inflamed tumor microenvironment, and potentiating tumor-cell intrinsic immunogenicity in melanoma. This study further emphasizes regulators of ERV expression and type-I interferon expression as potential therapeutic targets for augmenting anti-cancer immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ad3ee60cf098ddf889ff6e96bfd86efcd38c883" target='_blank'>
              Setdb1-loss induces type-I interferons and immune clearance of melanoma.
              </a>
            </td>
          <td>
            M. McGeary, William Damsky, Andrew J Daniels, Sabine M. Lang, Qingji Xu, E. Song, Clotilde Huet-Calderwood, Hua Jane Lou, Sateja Paradkar, G. Micevic, S. Kaech, D. Calderwood, Benjamin E. Turk, Q. Yan, Akiko Iwasaki, M. Bosenberg
          </td>
          <td>2024-11-26</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5daf8fc78edec4dc67cc0ac87030a2ee471c5078" target='_blank'>
              Cancer-associated fibroblasts: heterogeneity, tumorigenicity and therapeutic targets
              </a>
            </td>
          <td>
            Keke Lv, Tianlin He
          </td>
          <td>2024-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 TET2 mutations are clinically significant mutations in clonal hematopoiesis of indeterminate potential (CHIP) and myeloid malignancies driving inflammation and clonal progression, but the underlying mechanism of how TET2 loss leads to these phenotypes is not understood. Here, we used in vivo genome-wide genetic perturbations in primary WT or Tet2 KO Cas9+ hematopoietic stem and progenitor cells (HSPCs) in a model of zymosan serositis to decipher the mechanisms of Tet2 KO inflammation.
 Previous experimental approaches using CRISPR/Cas9 to study factors governing hematopoiesis in vivo have been hampered by heterogeneity in cellular expansion during hematopoietic reconstitution. To overcome this limitation, we developed an in vivo hematopoietic stem and progenitor (HSPC)-based CRISPR/Cas9 screening system with high-diversity (> 10 million) unique molecular identifiers (UMIs). After transplanting Cas9-expressing Lin-Sca1+c-Kit+ (LSK) cells transduced with UMI-labeled guide-RNAs (gRNAs), we compared neutrophils and monocytes recruited during zymosan stimulus compared to their counterparts in the bone marrow. This genome-wide CRISPR knockout screen revealed that Tet2 KO myeloid cells were dependent on lipid metabolic genes for inflammatory recruitment but not WT myeloid cells. In agreement, Tet2 KO neutrophils and macrophages accumulated neutral lipids as confirmed on BODIPY 493/503 neutral lipid staining and lipidomic analysis.
 The top dependency for Tet2 KO inflammation was OGT, O-linked N-acetylglucosamine transferase, responsible for attaching O-GlcNAc modifications onto proteins. To investigate whether OGT activity is changed in Tet2 KO, we performed nutrient tracing studies using uniformly labeled C13 glucose and IP-MS for O-GlcNAc in WT and Tet2 KO bone-marrow derived macrophages (BMDMs), which revealed increased utilization of UDP-GlcNAc, the substrate for OGT. To determine whether OGT controls neutral lipid accumulation, we knocked down OGT or used (Z)-PUGNAc, an OGA inhibitor to mimic increased OGT activity, which both showed that OGT controls neutral lipid accumulation.
 As OGT is among the top binding partners for TET2, we investigated whether OGT binding to TET2 is critical for inflammation. Mapping studies show that OGT binds to the disordered region on TET2 and when this interaction is disrupted, TET2 mutants have increased inflammation. Conversely, the OGT-binding TET2 peptide is sufficient to reduce inflammation in Tet2 KO whereas the mutant TET2 peptide that does not bind OGT had little effect, suggesting that TET2 restrains OGT to control inflammation.
 On chromatin, TET2 negatively regulates OGT and unrestrained OGT leads to increased OGT and O-GlcNAcylation signal on ChIP-Seq in BMDMs. These regions overlap with SREBP1/LXR binding sites, driving inflammatory lipid metabolism. A therapeutic target of OGT hyperactivation in TET2 mutants is ACLY, serving as an important node for lipid accumulation and inflammation in Tet2 KO.
 In summary, through a novel in vivo CRISPR screening system combined with metabolomics, proteomics and epigenetic studies, we reveal that TET2 is a negative regulator of a master nutrient sensor OGT. In response to nutrient cues, TET2 physiologically binds OGT on chromatin thus restraining its activity. Upon TET2 loss, OGT binding to genomic loci is increased causing metabolic hyperactivity and lipid accumulation, ultimately driving inflammation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed55307614642020e6ed237821f0ad9fd6a5486e" target='_blank'>
              Loss of Negative Metabolic Feedback Drives TET2 Mutant Inflammation
              </a>
            </td>
          <td>
            Geon Geon Kim, Christopher B. Hergott, Aidan P. Miller, A. Deik, Meaghan Boileau, Kevin Bullock, Kerry Pierce, Angelina Choy, Wesley J Shin, Marie E McConkey, Justin CT Loke, Birgitta A. Ryback, Michael N Trinh, Justine Rutter, Hong Yue, Paul Park, Shourya SR Burman, Eric Fischer, Scott A. Armstrong, Clary Clish, Benjamin L Ebert
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Ewing sarcoma (EwS) is an aggressive bone and soft tissue tumor and EwS patients often succumb to their disease years after initial treatment due to relapses at metastatic sites. Novel therapeutic strategies are needed that can successfully eliminate microscopic residual disease foci that persist at the end of primary therapy. Extracellular matrix (ECM) proteins in the tumor microenvironment (TME) provide critical pro-survival and pro-invasion signals to tumor cells. Our published and unpublished data show that the glycoprotein tenascin-C (TNC) is enriched in EwS metastases and that TNC and other pro-tumorigenic ECM proteins are deposited by subpopulations of CAF-like tumor cells that are activated in response to tumor and TME-derived TGF-beta ligands. TNC is abundantly produced by multiple human tumors of epithelial and non-epithelial lineage but is otherwise rarely expressed outside of development and wound healing. Importantly, TNC is specifically enriched in metastatic lesions where it has been implicated as a mediator of metastatic competence and treatment resistance. In the current work we have tested whether the TNC-rich TME of disseminated EwS tumor foci could be leveraged to direct therapies to sites of residual micrometastatic disease. To achieve this, we generated monovalent and bivalent anti-human TNC VHHs (hTNC-VHH) using a mammalian expression platform. Camelid-derived hTNC-VHH sequences were sourced from published literature. VHHs are single domain heavy chain only antibody fragments with higher stability and tissue penetration than conventional monoclonal antibodies. Using EwS tumor spheroids in collagen-rich 3D culture, hTNC-VHH penetrated dense ECM matrices and bound through multiple cell layers in under 10 minutes. hTNC-VHH accumulated in TNC-positive but not TNC-knockout EwS spheroids and were retained beyond 96 hours. To assess their potential to drive protein and immune therapies to TNC-rich TMEs, we fused hTNC-VHH to immune modulating CD3- and CD28-activating VHHs. These hTNC-VHH-CD3/CD28-conjugates promoted immune cell activation and tumor cell death in PBMC-tumor co-culture assays as determined by CD25 flow cytometry and fluorescence microscopy, respectively. To assess in vivo localization, fluorescently-tagged hTNC-VHH were intravenously injected into mice that had been xenografted with EwS cells. Tissue microscopy confirmed selective binding of hTNC-VHH to small lung and liver EwS micrometastases in less than 4 hours. VHHs were not retained in non-tumor bearing organs including the brain and heart or in tumor-free lung and liver regions. Experiments are ongoing to assess anti-tumor efficacy and pharmacodynamics of hTNC-VHH conjugates in EwS xenografts. Together these findings suggest that the ECM-remodeling properties of CAF-like EwS cells can be exploited to recruit novel ECM-targeting protein therapeutics to micrometastases. If successful, this innovative approach could eradicate microscopic residual disease and prevent metastatic EwS recurrence.
 Citation Format: Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M. Garcia, James M. Olson, Elizabeth R. Lawlor. Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C050.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9942d5319f0b477046d50c6a8b105f9899b2930d" target='_blank'>
              Abstract C050: Leveraging ECM deposition by CAF-like Ewing sarcoma tumor cells to target micrometastases with matrix-binding VHHs
              </a>
            </td>
          <td>
            Emma D. Wrenn, Jason P. Price, Raymond O. Ruff, Nicolas M Garcia, James M. Olson, Elizabeth R. Lawlor
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d320d942b07625b0b1d0c674abc05688c751b9d0" target='_blank'>
              Adipocytes reprogram the proteome of breast cancer cells in organotypic three-dimensional cell cultures
              </a>
            </td>
          <td>
            Karla Tovar-Hernández, Yarely M. Salinas-Vera, Ángeles Carlos-Reyes, A. P. García-Hernández, L. Marchat, Gilberto Mandujano-Lázaro, Emmanuel Ríos-Castro, Andrea Velasco-Suárez, Ivonne Mendez-Gómez, Angeles C. Tecalco-Cruz, Eloisa Ibarra-Sierra, C. López-Camarillo
          </td>
          <td>2024-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Medullary thyroid carcinoma (MTC) is a rare and aggressive tumor, often requiring systemic treatment in advanced or metastatic stages, where drug resistance presents a significant challenge. Given the role of cancer stem cells (CSCs) in cancer recurrence and drug resistance, we aimed to identify CSC subpopulations within two MTC cell lines harboring pathogenic variants in the two most common MEN2-associated codons. We analyzed 15 stemness-associated markers, along with well-established thyroid stem cell markers (CD133, CD44, and ALDH1), a novel candidate (DLK1), and multidrug resistance proteins (MRP1 and MRP3). The ability to efflux the fluorescent dye Hoechst 3342 and form spheroids, representing CSC behavior, was also assessed. MZ-CRC-1 cells (p.M918T) displayed higher expressions of canonical markers, DLK1, and MRP proteins than TT cells (p.C634W). MZ-CRC-1 cells also formed more spheroids and showed less dye accumulation (p < 0.0001). Finally, we observed that DLK1+ cells (those expressing DLK1) in both cell lines exhibited significantly higher levels of stemness markers compared to DLK1− cells (those lacking DLK1 expression). These findings underscore DLK1’s role in enhancing the stemness phenotype, providing valuable insights into MTC progression and resistance and suggesting potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46fc9f36dee4eea1c7adc632b8fd58baacccb9b" target='_blank'>
              DLK1 Is Associated with Stemness Phenotype in Medullary Thyroid Carcinoma Cell Lines
              </a>
            </td>
          <td>
            Danilo Dias da Silva, R. P. Araldi, Mariana Rocha Belizario, Welbert Gomes Rocha, R. M. Maciel, J. Cerutti
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Cell state plasticity is retained by all stem-like and differentiated cells within glioblastoma (GBM) tumors. GBM cells can exploit these state-shifting capacities to evade and resist therapeutics. Thus, there is a need to identify and target the regulators of GBM cell state transitions. To address this question, we use a fully human platform where hiSPC-derived organoids serve as a recipient environment to host cells engrafted directly from surgically resected GBMs. This allows us to precisely examine how microenvironmental variables impact the molecular signatures of GBM cells. We acutely isolate GBM cells from surgical resections, label cells with a GFP-lentivirus for 4 hours, and then engraft these tumor cells into parallelly patterned region-specific organoids that mimic dorsal forebrain, ventral forebrain, midbrain, and hindbrain identities. These regions were chosen as they capture derivatives of the three primary brain vesicles and because these regions contain unique cell types and signaling molecules. We performed paired single-cell RNA sequencing on tumor cells pre-engraftment and 14 days post-engraftment. Universal to all regional organoids, we find that GBM cells shift towards a neural progenitor-like and neuron-like signature within organoids, regardless of the cell state of the parent tumor prior to engraftment. To support our findings, we also observed that patient-derived glioma cell lines converge on a neural progenitor-like state within organoids. Thus, the dominance of this shared GBM cell state within organoids implies the existence of shared extrinsic factors in these immature (neurogenic) organoids that favor a progenitor-like or neuron-like identity across all four regions. We are now asking whether this bias towards neuronal and/or neural progenitor identity is a result of engrafting into young neurogenic organoids, or if the same phenomenon could be true in late-stage (post-gliogenic) mature organoids. We are also using this valuable platform and data to identify the extrinsic signals that GBM cells rely on to fulfill their plastic potential, adapt to the microenvironment, and resist therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f97acab6d83148ea03065d74fff70ff890b1b78" target='_blank'>
              TMIC-33. MICROENVIRONMENT-DRIVEN CHANGES IN GLIOBLASTOMA CELL STATE SIGNATURES EXAMINED USING REGION-SPECIFIC HUMAN BRAIN ORGANOIDS
              </a>
            </td>
          <td>
            Tarun Bhatia, Saisrinidhi Ganta, Anson Sing, Alexia King, Caitlin Sojka, Kimberly Hoang, E. Nduom, Renee D Read, Jeffrey Olson, Steven Sloan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most malignant and aggressive primary brain tumor, is driven by both intrinsic genetic mutations and extrinsic factors from the tumor microenvironment (TME). GBM manipulates the TME to enhance its survival and proliferation through mechanisms such as forming functional synapses with adjacent normal cells and secreting immunosuppressive factors. Our laboratory has developed a novel human organoid tumor transplantation (HOTT) model, which allows for the interrogation of tumor-normal cell crosstalk at a molecular level and facilitates the dissection of specific signaling pathways that drive tumor progression and maintenance. Using single-cell transcriptomics on human GBM HOTTs, we identified the PTN-PTPRZ1 ligand-receptor interaction as a critical mediator of tumor-microenvironment communication. PTPRZ1, a receptor tyrosine phosphatase, has been previously implicated in glioma cell migration, and our data further elucidate its role in modulating tumor behavior. Specifically, we demonstrated that knockdown of PTPRZ1 within the TME of human cortical organoids significantly enhances glioma cell migration and inhibits differentiation, underscoring the essential role of PTPRZ1 in maintaining the tumor’s invasive phenotype. Further pharmacological inhibition of PTPRZ1’s catalytic activity with a small molecule revealed that these effects on cell fate determination are independent of its phosphatase activity. This suggests that PTPRZ1 exerts critical non-catalytic functions that are pivotal for GBM progression. To further investigate these non-catalytic mechanisms, we propose utilizing single-cell transcriptomics coupled with proximity labeling and proteomics. This approach will allow us to comprehensively map PTPRZ1-mediated signaling networks and elucidate their impact on tumor biology, potentially unveiling novel therapeutic targets. Our findings revealed a novel axis of GBM biology mediated by PTPRZ1 and highlighted the importance of considering microenvironmental interactions in the development of effective therapeutics for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e2eb14f30c8f2631ab0122c079e7474889f0947" target='_blank'>
              TMIC-77. ELUCIDATING THE ROLE OF PTPRZ1 SIGNALING IN DRIVING GLIOBLASTOMA PROGRESSION USING THE NOVEL HUMAN ORGANOID TUMOR TRANSPLANTATION SYSTEM
              </a>
            </td>
          <td>
            Ryan L. Kan, Weihong Ge, Can Yilgor, Elisa Fazzari, Patricia R. Nano, Daria J. Azizad, Matthew Li, Joyce Ito, C. Tse, Hong A. Tum, Jessica Scholes, Kunal S. Patel, David Nathanson, A. Bhaduri
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 The intra- and inter-patient heterogeneity in high-grade glioma (HGG) continues to contribute to its poor prognosis. Clinical biopsies are often harvested from limited regions and typically are not image localized. Thus, they fail to capture the diversity within tumor regions, immune expression or normal cell abundances that play key roles in tumor development. It is important to gain an understanding of these subpopulation ecologies, their spatial resolution, and interactions between them that may then be exploited for future therapeutic benefit. Further, these may differ by patient characteristics such as sex, age at diagnosis and treatment status. Using an ongoing image-localized biopsy collection protocol, we have so far evaluated the bulk transcriptomics of 202 multi-regional biopsies from 58 patients to characterize HGG heterogeneity. These samples were processed through Monocle, a reverse graph embedding algorithm that groups samples into states and orders them along developmental trajectories. Deconvolution methods were previously used to predict relative abundances of 7 normal, 6 glioma, and 5 immune cell subpopulations for each sample, which we have now overlaid on the Monocle graph. Monocle classified HGG into 4 main states along a three-pronged trajectory. These states reveal distinct population ecologies with associated enriched gene pathways. We also note significant immune pathway enrichments that differ between state and patient-reported sex. Together, these algorithms reveal a simple transcriptomic trajectory that helps us understand the development and evolution of HGG. Characterizing the in vivo diversity within and between high grade gliomas is important for understanding prognosis, stratifying future treatments and ultimately improving patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537edcf404e379d243b38f5fbd9729b85401085d" target='_blank'>
              EPCO-40. TRANSCRIPTIONAL TRAJECTORY INFERENCE OF IMAGE-LOCALIZED, MULTI-REGIONAL HIGH-GRADE GLIOMA BIOPSIES REVEALS DISTINCT POPULATION ECOLOGIES AND SEX-ASSOCIATED ENRICHED PATHWAYS
              </a>
            </td>
          <td>
            L. Curtin, Kamila Bond, K. Singleton, Pamela Jackson, C. Sereduk, C. Krishna, R. Zimmerman, D. Patra, K. Donev, Maciej Mrugala, N. Tran, Leland Hu, B. Bendok, O. Al-Dalahmah, P. Canoll, Kristin R Swanson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000ea5f5a04e65fe0427f079acb1a4f46ea53f2a" target='_blank'>
              Deciphering the topological landscape of glioma using a network theory framework
              </a>
            </td>
          <td>
            Mengchao Yao, Yang Su, Ruiqi Xiong, Xile Zhang, Xiaomei Zhu, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma IDH wild type (GB), the most common malignant primary brain tumor, is characterized by rapid proliferation, extensive infiltration into surrounding brain tissue, and significant resistance to current therapies. Median survival is only 15 months despite extensive clinical efforts. The tumor microenvironment (TME) in GB is highly specialized, supporting the tumor’s aggressive behavior and its ability to evade conventional treatments. One critical component is the aberrant vascular network that complicates the delivery of chemotherapy across the blood–brain barrier. Antiangiogenic therapies emerged as a promising option but have shown limited efficacy in extending the survival of these patients. Comprehension of the complex vascular network of GB may be a key to overcoming the limitations of current therapies. Pericytes are gaining recognition within the context of the TME. These mural cells are essential for vascular integrity and may contribute to tumor progression and therapeutic resistance. Although their role has been evidenced in other tumors, they remain underexplored in GB. Pericytes are known to respond to tumor hypoxia and interact with vascular endothelia, influencing responses to DNA damage and antiangiogenic treatments. They actively regulate not only angiogenesis but also the different vasculogenic strategies for tumor neovascularization. Additionally, they affect leukocyte trafficking and tumor-associated macrophages. This review aims to integrate the various functions controlled by pericytes to favor deeper investigation into their actionable potential. Pericytes may represent a promising target for novel therapeutic strategies in order to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835c71d63c5567604bb678c8f9634fc408771476" target='_blank'>
              Pericytes in Glioblastoma: Hidden Regulators of Tumor Vasculature and Therapy Resistance
              </a>
            </td>
          <td>
            Irene Salazar-Saura, María Pinilla-Sala, J. Megías, Lara Navarro, Esther Roselló-Sastre, T. San-Miguel
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Gastric cancer (GC) is characterized by a complex and heterogeneous tumor microenvironment (TME) that significantly influences disease progression and treatment outcomes. The tumor stroma, which is composed of a variety of cell types such as cancer-associated fibroblasts, immune cells and vascular components, displays significant spatial and temporal diversity. These stromal elements engage in dynamic crosstalk with cancer cells, shaping their proliferative, invasive and metastatic potential. Furthermore, the TME is instrumental in facilitating resistance to traditional chemotherapy, specific treatments and immunotherapy strategies. Understanding the underlying mechanisms by which the GC microenvironment evolves and supports tumor growth and therapeutic resistance is critical for developing effective treatment strategies. The present review explores the latest progress in understanding the intricate interactions between cancer cells and their immediate environment in GC, highlighting the implications for disease pathogenesis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78f0fda7748f1b417eb850faad66bf082f52653" target='_blank'>
              Role and value of the tumor microenvironment in the progression and treatment resistance of gastric cancer (Review)
              </a>
            </td>
          <td>
            Heng Yun, Fangde Dong, Xiaoqin Wei, Xinyong Yan, Ronglong Zhang, Xiuyu Zhang, Yulin Wang
          </td>
          <td>2024-11-26</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Intra-tumor heterogeneity (ITH) describes the coexistence of multiple genetically distinct subclones within the tumor of a patient resulting from somatic evolution, clonal diversification, and selection. It is a main causal driver of therapy resistance and treatment failure in the clinic by already containing subclones that are resistant to therapy or by subclones acquiring resistance to therapy. Core-binding factor (CBF) acute myeloid leukemia (AML) is cytogenetically defined by a translocation of chromosome 8 and 21 [t(8;21)(q22;q22)] or an inversion/translocation of chromosome 16 [inv(16)(p13.1q22) or t(16;16)(p13.1q22)] resulting in RUNX1::RUNX1T1 and CBFB::MYH11 fusion genes, respectively. Despite being classified as AML with favorable prognosis up to 45% of patients relapse and, therefore, understanding the history of somatic events of this heterogeneous AML subtype can contribute to improve clinical outcome.
 To unravel ITH and clonal evolution for both small-scale (single nucleotide variants, insertions and deletions) and large-scale (somatic copy-number alterations (SCNAs), gene fusions) genetic alterations in CBF AML, we analyzed longitudinal samples from 9 patients with t(8;21) or inv(16) AML by whole exome, error-corrected targeted and nanopore long-read bulk sequencing (24 samples). Single-cell DNA sequencing (scDNAseq) of longitudinal samples was performed via the MissionBio Tapestri® platform using custom targeted sequencing panels including information on patient-specific somatic variants, SCNAs and CBF fusions. Samples were collected at the time of diagnosis (D), complete remission (CR), and relapse (Rel). We used a novel approach for reconstructing the order of somatic events in our patients that allowed us to detect SCNAs in phylogenetic trees inferred from somatic variants and CBF fusions.
 In total, we identified 399 variants via bulk sequencing. In the diagnosis samples (n=9) 230 variants were detected and 169 variants in the relapse samples (n=8). Additionally, 7 SCNAs in 5 patients and patient-specific gene fusion breakpoints for all patients were identified using bulk and Nanopore sequencing. We were able to detect patient specific CBF fusions in single cells for 8/9 patients and for those 8 patients we detected between 61% and 96% (mean: 86%) of the variants identified via bulk sequencing. Additionally, we identified 10 variants by scDNAseq in the regions covered by the custom amplicon panels, that have not been detected in bulk sequencing, including FLT3-ITD.
 In total, we inferred tumor phylogenies for 8 patients with detectable gene fusion at D and inferred clonal evolution under chemotherapy pressure for 3 patients. We were able to detect SCNAs by applying our method in small tumor subclones that were not detectable with bulk data nor using existing single-cell tools. By reconstructing the order of somatic evolutionary events from scDNAseq, we can demonstrate that the CBF gene fusions were the first events to occur in disease development in our patients. For patients with samples at D and Rel (n=3) we observed a persisting founding clone throughout the course of the disease. We identified remaining tumor cells in available CR samples (n=6) ranging from 4 to 34 cells (0.16%-1.54%) that harbor at least one variant or the CBF fusion. Among the 148 cells with any detectable variant/fusion, only 5 cells would have been detected when only investigating the fusion. Thus, simultaneous tracking of multiple genotypes showed a clear superiority compared to current strategies, which focus on fusion detection for MRD purposes. By assigning those cells to tumor clones inferred from samples at D and Rel, we can demonstrate that the AML clones were not completely eradicated during treatment.
 In summary, our study presents a detailed picture of ITH and clonal evolution including somatic variants, gene fusions and SCNAs at single-cell resolution of 9 CBF AML patients. We reconstructed the mutational order of somatic events within one tumor sample including acquisition of CBF gene fusions and SCNAs, which adds a new layer into clonal evolution of CBF AML. Moreover, we were able to detect and assign remaining tumor cells at CR to inferred tumor clones of each patient. This highlights the importance of improving the understanding of tumor development and, in more detail, the necessity of identifying early events during tumorigenesis in CBF AML patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27c14f92afccda3056b80626f5b41472934ea1b" target='_blank'>
              Intra-Tumor Heterogeneity in Core-Binding Factor Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina Strzelecka, Natalia Estrada, Klara Kopp, A. Dolnik, Marlon Tilgner, Coral Fustero, Felicitas Thol, F. Heidel, M. Heuser, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Sex-determining region Y-box 2 (SOX2) has recently been used as a marker for pluripotency to identify cellular glioblastoma invasion pathologically. This can be used to identify areas of tumor invasion consisting of sparse individual cells, likely highlighting the furthest margin of tumor invasion. This study aligned large format SOX2 stained tissue samples to conventional MRI to determine demographic factors that impact the presence of SOX2 positive invasion outside the T1 contrast-enhanced tumor margin.



 The cohort for this study consisted of 36 IDH-wildtype glioblastoma patients (23 male, 13 female, mean age=61.9). Tissue samples from areas of suspected tumor invasion and normal tissue were collected at autopsy, stained for SOX2, digitized at 40X resolution, and automatically segmented for SOX2 positive staining cell percent (SOX2+%) at MR resolution using a color deconvolution algorithm. Contrast enhanced T1 scans from acquisitions close to death were annotated manually for tumor presence and aligned to the T2-weighted FLAIR image. Tissue data was then aligned to MR images using a custom in-house software that relies on manually defined control points to generate a nonlinear transform that warps tissue into MR space while correcting for tissue shrinkage. Mean SOX2+% was computed for each patient outside of contrast enhancement and compared across age and sex as demographic factors.



 Males were found to have a higher SOX2+% than females outside contrast enhancement (t=2.09, p=0.045), while no association with age was observed (r=0.146, p=0.397).



 Sex differences were observed for non-enhancing SOX2 positivity, which may relate to the more aggressive tumors identified in male patients in some studies. Further research into the mechanisms behind sex differences in tumor growth patterns and molecular characteristics impacting the infiltration of tumor beyond the MR defined region is needed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcf5f100471745ba67f49e284ea228b13d9895de" target='_blank'>
              ANGI-11. SEX DIFFERENCES IN SOX2 GLIOBLASTOMA EXPRESSION OUTSIDE MRI-DEFINED CONTRAST ENHANCEMENT AT AUTOPSY
              </a>
            </td>
          <td>
            S. Bobholz, A. Lowman, Daniel Aaronsen, A. Winiarz, S. Duenweg, Michael Flatley, Biprojit Nath, F. Kyereme, Jennifer Connelly, Kelly Mrachek, Max O Krucoff, Anjishnu Banerjee, Peter S LaViolette
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment-naïve primary and post-HAIC HCC, and integrated single-nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post-HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post-HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme-K and PD-1 (PD-1+CD8+ Tex-int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD-1+CD8+ Tex-int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC-based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single-Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy.
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26feb24c74a70e25f404994931c40c73dd0fe64" target='_blank'>
              Deciphering molecular landscape of breast cancer progression and insights from functional genomics and therapeutic explorations followed by in vitro validation
              </a>
            </td>
          <td>
            Bushra Khan, Rowaid M. Qahwaji, Mashael S. Alfaifi, Tanwir Athar, Abdullah Khan, Mohammad Mobashir, I. Ashankyty, K. Imtiyaz, Areej Alahmadi, M. M. Rizvi
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="

 Background: Acute myeloid leukemia (AML) is a heterogeneous disease characterized by variable leukemic states that primarily originate in the bone marrow (medullary) and can also occur in extramedullary (EM) tissues, indicating its adaptability to different environments. The spatial dynamics of medullary and extramedullary AML remain underexplored. Understanding these dynamics is crucial for identifying the microenvironments in AML and improving diagnostic approaches.
 Methods: We conducted Visium spatial transcriptomics (ST), suspension single cell RNA, and GeoMx-based Digital Spatial Profiling (DSP) on AML medullary and extramedullary patient samples. For Visium ST, we applied two assay versions (v1 and CytAssist v2) on paired bone marrow (BM) and EM samples from two AML patients (n=8 total samples). We compared Visium assays on these decalcified and non-decalcified leukemic tissues and adapted median absolute deviation (MAD) for spatial spot filtering. Spatial deconvolution was performed using in-house generated BM suspension scRNA-seq reference from 16 samples (9 healthy, 7 AML patients). Findings were validated using GeoMx-based DSP in three independent AML patients, providing a comprehensive spatial map of medullary and extramedullary leukemia.
 Results: Visium ST patients: Patient 1 (PT1) had medullary and EM leukemia, while Patient 2 (PT2) had isolated EM leukemia with no BM involvement. RNA fragmentation on these samples varied, measured as the DV200 values before library preparation (range 24-62%; mean DV200 = 44.5, standard deviation = 16.6). Post-library DNA traces for v1 were lower in BM samples (24-29%) compared to v2 (89-91%), while EM tissues showed similar post-library traces for both assays (v1: 87-83%, v2: 88-91%). V2 assays detected more oligonucleotides (mean: 11072.37) and genes (mean 4735.67) compared to v1 (mean oligonucleotides: 1292.76, and mean genes detected: 955.19).
 Considering the variability in each tissue, we adapted MAD to filter spots based on mitochondrial content and expression profiles. We next performed shared nearest neighbor (SNN) modularity optimization-based unsupervised clustering, revealing overlap between determined clusters and hematopathologist-identified histopathological tissue annotations. To spatially delineate spot-level cellular information in our BM samples, we deconvolved individual spots using probabilistic label transfer workflow with our scRNA-seq BM reference. High leukemic score (HLS) spots defined with increased AML population and showed co-localization with monocytes and granulocyte-monocyte progenitors (GMP) in BM and macrophages in EM tissues.
 Spatial cell-cell communication and pathway analysis indicated CXCL12-CXCR4 and PI3K/AKT/mTOR pathway promoted trans-differentiation in inflammatory niches. Additionally, deconvolution of AML cells showed that primitive-like AML cells were located near the endosteal niche, while committed-like cells were spatially distant from the bone. These patterns were validated with GeoMx-based DSP, suggesting that AML cells at different states of differentiation locate in distinct niches within the BM.
 Conclusions: To our knowledge, this is the first study applying Visium ST to both medullary and extramedullary AML tissues from paired samples to delineate biologic niche-specific differences. Our study elucidates the spatial niches of AML using multimodal ST, identifying distinct inflammatory niches with significant involvement of the CXCL12-CXCR4 axis, promoting trans-differentiation via the PI3K/AKT/mTOR pathway, aiding in AML migration and infiltration into EM sites. Spatial analysis revealed that primitive-like AML cells predominantly locate near the endosteal niche in BM, while committed-like and GMP-like populations situate more distally. All our data will deposit in an interactive public website that allows other researchers to navigate and explore. Spatial analysis is ongoing for 17 additional BM AML patients with concomitant proteomics profiling.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aa6cd0576a8088d7655c27ebc5fe848511b18ec" target='_blank'>
              Multimodal Spatial Transcriptomic Profiling Elucidates Niche-Specific Dynamics in Medullary and Extramedullary Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Enes Dasdemir, Ivo Veletic, Christopher P. Ly, Andres E. Quesada, F. Jelloul, Pamella Borges, Sreyashi Basu, Sonali Jindal, Zhiqiang Wang, Alexander J. Lazar, Khalida M Wani, D. Antunes, P. Reville, Preethi H. Gunaratne, Robert J Tower, Padmanee Sharma, Hussein A. Abbas
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="
 Changes in the glioma immune microenvironment at recurrence may contribute to disease progression and therapeutic resistance. In prior studies we demonstrated that the circumscribed glioma, pleomorphic xanthoastrocytoma (PXA), was relatively immune enriched including activated cytotoxic CD8+ T cells and Iba1+ microglia/macrophages. While PXA are often low-grade, a significant fraction can recur and progress to higher grade. To determine how the tumor and its immune microenvironment changes with recurrence, we examined a cohort of tumors from patients with multiple resections. Using single-nucleus RNA sequencing (snRNAseq) and spatial molecular imaging we investigate the temporal changes in the tumor and its microenvironment. To begin, we analyzed samples from three patients with paired initial and recurrent tumor. A total of 22,465 immune and non-immune cells were analyzed. Compared to the initial, the recurrent tumor tended to have decreased numbers of T cells and T cells had lower expression of CD2 and GZMA. Pathway analysis for DE genes in TAMs demonstrated enrichment of genes associated with NTRK signaling, while pathways related to aerobic respiration and mitochondrial translation were decreased suggesting a less inflammatory phenotype. Tumor cells at recurrence demonstrated decreased expression of CD274 (PD-L1) and of the chemokine CXCL14, previously identified as a factor driving an anti-tumor CD8+ T cell response. In a cohort of 5 patients with paired initial and recurrent tumors, we observed increased numbers of CD163+ TAMs. Together these data suggest changes in the tumor-associated immune response at recurrence that may be relevant to therapy. Using murine models for the disease we are investigating the factors that regulate these changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b88f373ffe2f117f963a4d52016101c59ac0f40" target='_blank'>
              TMIC-84. MOLECULAR EVOLUTION OF THE TUMOR AND ITS MICROENVIRONMENT AT RECURRENCE
              </a>
            </td>
          <td>
            Elaheh Hashemi, Akshaya R.L, Anupam Kumar, Viva Voong, Lin Wang, Aaron A. Diaz, Joanna Phillips
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 DNA methylation profiling has become an important diagnostic and exploratory tool in neuro-oncology. Intra-tumoral molecular heterogeneity is a crucial factor in the treatment resistance of IDH-wildtype glioblastoma (GBM). However, the intra-tumoral heterogeneity of methylation profiles in GBM remains poorly elucidated. This study aimed to investigate the intra-tumoral heterogeneity of DNA methylation subclasses and deconvolve methylation profiles to infer the cellular composition of the tumor.



 We conducted genome-wide DNA methylation analysis and applied the DKFZ/Heidelberg CNS tumor classifier to FFPE tissue samples from a multi-sampling cohort with multiple samples per patient and a single-sampling cohort with one sample per patient. Only samples classified as GBM were included. We conducted two different methylation-based deconvolution analyses. Deconvolution 1 inferred the fractions of tumor subtypes (RTK_I, RTK_II, MES_TYP, and MES_ATYP) and cell types in the microenvironment. Deconvolution 2 estimated the abundance of malignant cell states (stem-like and differentiated cell components) and cell types in the microenvironment.



 The multi-sampling cohort included 32 patients with 113 samples, and the single-sampling cohort included 87 patients with 87 samples. In the multi-sampling cohort, 8 patients (25%) exhibited intra-patient heterogeneity in methylation subclasses across their samples. Deconvolution 1 indicated that each tumor subclass component was heterogeneously distributed within samples, and the abundance of the myeloid cell component correlated with the MES_TYP component across samples. Deconvolution 2 similarly revealed heterogeneity in the immune component abundance across samples within patients. Integrating the results of Deconvolution 1 and Deconvolution 2 using the entire cohort revealed significant correlations between RTK_I and stem-like, RTK_II and differentiated, and MES_TYP and immune components. These findings were also replicated in TCGA dataset.



 Our findings underscore the importance of considering intra-tumoral heterogeneity in methylation profiles and the biological characteristics of each methylation subclass in developing novel GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bc927be7c787e923aae21a191879fcd45aa8fd" target='_blank'>
              EPCO-15. INTRA-TUMORAL HETEROGENEITY OF DNA METHYLATION PROFILING AND CELLULAR COMPOSITION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yuji Matsumoto, Omkar Singh, Jose A. Garcia, Sunwoo Kwak, Zied Abdullaev, Nelson Freeburg, Dana Silverbush, Kenneth J Aldape, C. Davatzikos, M. Nasrallah
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common malignant brain tumor in adults. GBMs are known for their highly invasive and proliferative properties and are associated with a dismal prognosis (~12-18 months post-diagnosis). Neuroanatomical disparities in the occurrence of GBMs suggest that in addition to microenvironmental impacts, tumor cells are impacted by the cellular and molecular diversity across varying regions of this nervous system. This idea has been difficult to study due to the lack of human-specific models that effectively recapitulate the biological behaviors of GBMs in region-specific microenvironments. To address these challenges, we engrafted GFP-tagged patient-derived glioma cell lines into human induced pluripotent stem cell (hiPSC)-derived organoids patterned to mimic the most forebrain (dorsal cortex) and hindbrain (spinal cord) regions of the nervous system. Immunostaining assays at 2, 7, 14, and 21 days post-engraftment compared the extent of infiltration, as measured by the distance traveled by glioma cells, in both regions. We found that glioma cells infiltrated a significantly greater distance in the dorsal cortical organoids compared to the spinal cord organoids. This suggests a more conducive environment for glioma infiltration in the forebrain vs. hindbrain at baseline. We are currently determining if these regional biases in infiltration capacity are modified with either optogenetic stimulation of the host brain organoids or in the presence of interregional interactions in an assembloid system. Thus, using this fully human-specific platform, we aim to gain insights into the infiltrative behaviors of GBM cells, which will advance our understanding of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045df47e99729e03f5d67a0ffd1abdf99975dbdd" target='_blank'>
              TMIC-34. INFILTRATIVE BEHAVIOR OF PATIENT-DERIVED GLIOBLASTOMA CELL LINES IN REGION-SPECIFIC HUMAN BRAIN ORGANOIDS
              </a>
            </td>
          <td>
            Saisrinidhi Ganta, Tarun Bhatia, Alexia King, Anson Sing, Renee D Read, Steven Sloan
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor microenvironment (TME) plays a crucial role in the oncogenesis, early distant metastasis, and limited immunotherapeutic responses of pancreatic ductal adenocarcinoma (PDAC). Myeloid cell populations, particularly tumor-associated macrophages (TAMs), located within this immunosuppressive niche, are pivotal in these processes. With aid of single cell RNA sequencing (scRNA-seq), multiple studies have successfully validated distinct macrophage subtypes pathological functions in different cancer types, yet the correlation of TAMs in hepatic metastatic and primary PDAC lesions remains insufficiently understood. In this study, we reanalyzed samples from both primary and hepatic metastatic PDACs to elucidate the functional conservation and variability of TAMs. Pseudotime trajectory inference, together with pseudobulk analysis based on subtype level, manifests TAMs undergo similar metabolism reprogramming and pro-inflammatory style transitions with specific signaling pathway activations. Compared to primary tumors, hypoxic conditions are alleviated, which largely depends on the physiological context and the preliminary ligand-receptor interactive network within the metastatic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a56bbf6854596a35b27cf40482bd3f5c23fafab" target='_blank'>
              In-depth analysis of pathological function of tumor-associated macrophage populations in hepatic metastatic and primary PDAC
              </a>
            </td>
          <td>
            Fanyu Meng
          </td>
          <td>2024-12-12</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic progression is responsible for the majority of cancer-related deaths. A better understanding of the underlying molecular mechanisms driving metastasis therefore remains of utmost importance. Here we take efforts to excavate mutant driver factors of metastases in the level of signaling pathways. We introduce two models, EntCDP and ModSDP, to detect the similarity and specificity of pathways in a pan-cancer context. The simulation studies confirm their feasibility and identification accuracy. Using mutation profiles of 17 cancer types with primary and metastatic cohorts from MSK-IMPACT, we apply the two models to investigate the transition of signaling systems from primary to metastatic tumors mainly from five perspectives. 1) We initially perform a comprehensive comparison of 15 primary-metastatic cancer pairs. Nearly all shared common driver genes or pathways, while specific driver gene sets of primary or/and metastatic tumors can be seen for almost each pair, except for breast cancer. 2) We use ModSDP to identify the specificity of metastases with the same primary site and highlight calcium signaling in breast cancer with bone metastasis, FoxO signaling for melanoma/lung lymph node metastasis, etc. 3) Oppositely, we use EntCDP to identify common factors yielding three frequently metastatic sites (liver, lung and brain), and recognized cushing syndrome, microRNAs in cancer and MAPK signaling pathway, separately. 4) Regarding high-tropism metastatic patterns, we investigated the relationship between the metastatic tumor and primary tumors in both locations. In the typical pattern of colorectal cancer metastasis to the liver, we detected hepatic signals in primary colorectal cancer. 5) Finally, we focused on the common and specific characteristics of relevant cancer types, such as gender-related cancers and gastrointestinal cancers. Study on gynecologic tumors suggest endocrine hormone change and virus infection as their common risk factors, while for male we highlight hedgehog signaling and EPH-related receptors as invasive potential of prostate cancer. Multiple interesting findings revealed by this study may be helpful for the understanding of the extent of signal changes during tumor metastasis. We expect that this study will provide a valuable resource for transforming our knowledge about signals in cancer metastasis into alternative clinical practice for advanced patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facca2ecc611248ea30b446f5773d2c64fea06c3" target='_blank'>
              Pan-cancer study on transition of signaling systems from primary to metastatic tumors
              </a>
            </td>
          <td>
            Wenjia Zhou, Junhua Zhang
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Fatty acid binding protein 7 (FABP7) is a multifunctional chaperone involved in lipid metabolism and signaling. It is primarily expressed in astrocytes and neural stem cells (NSCs), as well as their derived malignant glioma cells within the central nervous system. Despite growing evidence for FABP7’s tumor-intrinsic onco-metabolic functions, its mechanistic role in regulating the brain tumor immune microenvironment (TIME) and its impact on prognosis at the molecular level remain incompletely understood. Utilizing combined transcriptome profiling and pan-cancer analysis approaches, we report that FABP7 mediates the expression of multiple onco-immune drivers, collectively impacting tumor immunity and clinical outcomes across brain cancer subtypes. An analysis of a single-cell expression atlas revealed that FABP7 is predominantly expressed in the glial lineage and malignant cell populations in gliomas, with nuclear localization in their parental NSCs. Pathway and gene enrichment analysis of RNA sequencing data from wild-type (WT) and Fabp7-knockout (KO) mouse brains, alongside control (CTL) and FABP7-overexpressing (FABP7 OV) human astrocytes, revealed a more pronounced effect of FABP7 levels on multiple cancer-associated pathways. Notably, genes linked to brain cancer progression and tumor immunity (ENO1, MUC1, COL5A1, and IL11) were significantly downregulated (>2-fold) in KO brain tissue but were upregulated in FABP7 OV astrocytes. Furthermore, an analysis of data from The Cancer Genome Atlas (TCGA) showed robust correlations between the expression of these factors, as well as FABP7, and established glioma oncogenes (EGFR, BRAF, NF1, PDGFRA, IDH1), with stronger associations seen in low-grade glioma (LGG) than in glioblastoma (GBM). TIME profiling also revealed that the expression of FABP7 and the genes that it modulates was significantly associated with prognosis and survival, particularly in LGG patients, by influencing the infiltration of immunosuppressive cell populations within tumors. Overall, our findings suggest that FABP7 acts as an intracellular regulator of pro-tumor immunomodulatory genes, exerting a synergistic effect on the TIME and clinical outcomes in brain cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15939c78a79b5a83578829fc7d538cf232f24932" target='_blank'>
              Integrated Transcriptome Profiling and Pan-Cancer Analyses Reveal Oncogenic Networks and Tumor-Immune Modulatory Roles for FABP7 in Brain Cancers
              </a>
            </td>
          <td>
            Yool Lee, Carlos C. Flores, Micah Lefton, Sukanya Bhoumik, Yuji Owada, Jason R. Gerstner
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="It is important to systematically identify tumor suppressor genes (TSGs) to improve our understanding of tumorigenesis and develop strategies for early diagnosis and mitigating disease progression. In the present study, we used an in vivo genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)-CRISPR-associated protein 9 (Cas9) screen and identified FPS/FES-related (FER) as a TSG. Single-cell RNA sequencing (scRNA-seq) revealed that normal cells with low FER expression exhibited elevated malignant transformation potential and stemness properties. FER knockout promoted the tumorigenic transformation, characterized by high colony-forming efficiency and suspension growth ability, acquired tumorigenicity in vivo, increased metabolic activity, dedifferentiation properties, and immune evasion. Moreover, analysis revealed that low FER expression tumors share molecular phenotypes with FER knockout cells, suggesting the consistent role of FER in tumor initiation and progression. Taken together, our findings not only provide insights into the essential role of FER as a tumor suppressor in tumor initiation and progression but also highlight its potential as a target for future clinical diagnosis. © 2024 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0ef168e6b6a500c7fdbd949d7e67ca980d8052" target='_blank'>
              CRISPR-Cas9 screening identifies the role of FER as a tumor suppressor.
              </a>
            </td>
          <td>
            Jiaqi Wang, Ran Yang, Feng-Po Wang, Junlei Zhang, Yutong Dong, Jiangjun Wang, Meng Yu, Yixiao Xu, Lianlian Liu, Yuda Cheng, Chen Zhang, Yi Yang, Wubin Yang, Jiali Wang, Guangxing Chen, Yi Huang, Yanping Tian, Rui Jian, Bing Ni, Wei Wu, Y. Ruan
          </td>
          <td>2024-12-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 High-grade glioma (HGG) is the most common and aggressive form of primary brain cancer. Standard of care surgery employs Magnetic Resonance Imaging to identify the contrast enhancing (CE) core for resection, leaving a non-enhancing (NE) invasive rim of tumor which contributes to recurrence. The hypoxic tumor microenvironment is thought to drive invasion into healthy brain, and understanding cellular responses to hypoxia may provide insight into combatting recurrence. Here, we apply a graph network to integrate regionally heterogeneous data from a unique dataset of spatially-localized CE and NE biopsies from a large cohort of patients with HGG to identify gene networks regulating hypoxia response and invasion.



 RNA-sequencing of 1159 differentially expressed genes across 159 IDH-wt biopsies (74 patients) from NE/CE regions were represented as nodes in a Neo4j graph network. Candidate genes for network random walks occurred exclusively within NE communities detected by modularity optimization and were localized by cell-type using single-cell RNA-sequencing (scRNAseq). CSF2RA was selected as source (walk length = 5; 10,000 walks/source node). Frequently occurring genes (one-tailed test) in walks were ranked by correlation with CSF2RA expression in NE samples and analyzed by gene set enrichment analysis for biological processes (clusterProfiler). CSF2RA and hypoxia gene expression were visualized with DoHeatmap in composite NE/CE scRNAseq profiles.



 CSF2RA, an immune receptor, was upregulated (logFC = 0.789) in NE samples, scRNAseq myeloid populations, and Glycolytic/Plurimetabolic glioma subtype. CSF2RA random walk genes (n=46) enriched metabolic/proliferative signatures. CSF2RA expression correlated negatively with hypoxia genes globally and regionally (NE/CE), while NE/CE scRNAseq revealed distinct cellular expression patterns of CSF2RA and hypoxia response genes.



 CSF2RA expression may prime NE glioma cells for invasion by altering hypoxia response. The genetic network bridging CSF2RA and hypoxia response may be a targetable mechanism of tumor recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d079992b8d959a801dbb717c3b250b9954d47a5" target='_blank'>
              ANGI-10. NETWORK ANALYSIS ESTABLISHES CSF2RA’S REGIONAL INFLUENCE ON HYPOXIA RESPONSE IN THE INVASIVE RIM OF HIGH GRADE GLIOMA
              </a>
            </td>
          <td>
            M. Flick, V. Stevens, C. Sereduk, C. Plaisier, R. Barajas, Leland Hu, K. Kannan, N. Tran
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Clonal hematopoiesis (CH) occurs when a hematopoietic stem cell (HSC) acquires a somatic mutation that confers a fitness advantage, thereby leading to its clonal expansion. The incidence of clonal hematopoiesis dramatically increases with aging, and aging-related inflammation in the bone marrow (BM) environment has been posited to be a driver of CH. However, the origins and mechanisms of inflammation that promote the outgrowth of clonal HSCs remain incompletely understood. Recent evidence has shown that epigenetic dysregulation of heterochromatin during aging can reactivate retrotransposons in a cell-type specific manner to induce a type I interferon response, suggesting a possible source for BM inflammation. However, the contributions of retrotransposons to inflammation in the BM environment remain largely unexplored.
 To investigate the cellular origins of retrotransposon-mediated inflammation within the bone marrow, we performed single-cell RNA sequencing on a cohort of five young (2 months) and five aged (>24 months) C57BL/6 mice. We identified six major BM environment cell types (chondrocytes, endothelial cells, fibroblasts, mesenchymal stromal cells, osteolineage cells, and pericytes) and amongst these cell types, we found that mesenchymal stromal cells (MSCs) robustly reactivated retrotransposon expression during aging. We further validated these findings by purifying MSCs from young and aged mice for bulk RNA-sequencing, additionally finding that type I interferon genes were the most significantly enriched genes in aged MSCs by Gene Ontology enrichment analysis. Our data thus suggested a link between aging-associated reactivation of retrotransposons in the BM environment and innate immune activation.
 We next sought to understand whether retrotransposon reactivation can non-cell-autonomously promote the expansion of HSCs harboring mutations in Dnmt3a, the most commonly mutated gene in CH. To do so we utilized mice deficient in Mpp8, which is an epigenetic regulator that normally acts to maintain retrotransposon silencing. We transplanted Vav1-cre;Dnmt3afl-R878H/+ (Dnmt3a-mutated) HSCs and WT HSCs at a 1:5 ratio into busulfan-conditioned recipient Mpp8-/- mice, which recapitulate conditions of retrotransposon activation in the BM environment, as well as WT recipient mice. No difference was observed in peripheral blood donor chimerism over the course of five months, likely due to the well-described differentiation defects that characterize Dnmt3a-deficient HSCs. However, in a terminal BM analysis at 5 months, Dnmt3a-mutated cells demonstrated more robust expansion in Mpp8-/- recipients as compared with WT recipients across hematopoietic stem and progenitor cell compartments (HSC p=0.06, LSK p=0.06, CMP p=0.03, GMP p=0.49, MEP p=0.34) and the absolute frequency of Dnmt3a-mutated HSCs was higher in Mpp8-/- recipients as compared to WT recipients (p=0.057). Together, these results provide the first evidence of a cell-extrinsic role for retrotransposon reactivation in driving CH.
 To further investigate whether retrotransposons can directly induce a type I interferon response in MSCs, we ectopically overexpressed two retrotransposons that we found to be upregulated in MSCs during aging, Lx8 and B1, using a murine BM stromal cell line, MS5. We found that both overexpressed retrotransposons robustly induced the expression of IFN-beta as well as downstream type I interferon stimulated genes (ISGs). Additionally, when we co-cultured MS5 cells in the presence of interferon-alpha with a competing mixture of WT and Dnmt3a-mutated HSCs for 7 days ex vivo, we found that the Dnmt3a-mutated cells outcompeted WT cells (p=0.07).
 Together, our data support a role for a retrotransposon-mediated type I interferon response in cell-extrinsically promoting the preferential outgrowth of DNMT3A-mutated HSCs during aging. Given the high prevalence of CH amidst an expanding aging population, a better understanding of the mediators of HSC expansion in CH is key to mitigating its associated diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcce4d4a006aba0c6ebfcbbf3d6a6fc5fb499fcd" target='_blank'>
              Retrotransposon Reactivation in Mesenchymal Stromal Cells Promotes Clonal Expansion of Dnmt3a-Mutated HSCs
              </a>
            </td>
          <td>
            Qiongdi Zhang, Joshua Theriot, Donghong Cai, Jian Xu, Sisi Zheng
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction: Survival rates in advanced head and neck cancer (HNC) remain poor with no clinically approved biomarkers to predict treatment efficacy. HNC is a highly heterogenous disease and the lack of available models that capture patient heterogeneity and the complexity of the tumor microenvironment (TME) contributes to these poor patient outcomes. Microphysiological systems (MPS) are complex multicellular in vitro models that can recapitulate three-dimensional organ structure and function. The low cell number requirements for MPS permits the creation of patient-specific TME models, where all the cell types can be isolated from a single piece of tumor tissue. These patient-specific MPS represent a promising strategy for elucidating the contribution of the TME to treatment resistance, predicting the best treatment for each patient and improving patient outcomes in HNC. Here, we report a patient-specific MPS that contains multiple TME compartments and recapitulates patient heterogeneity. Methods: Tumor tissue was collected from patients with oral cavity carcinoma undergoing standard of care surgery followed by radiation or chemoradiation. Tissue from each patient was digested and tumor infiltrating leukocytes (TIL) were isolated using CD45 magnetic beads, while epithelial cells and fibroblasts were expanded in culture using specific media formulations. Tissue was also snap frozen for single cell RNA sequencing analysis. To build the MPS, a custom designed microfluidic device was filled with a hydrogel matrix containing patient-matched TIL, fibroblasts and epithelial spheroids to mimic the solid tumor. Lumens molded from the hydrogel were seeded with blood or lymphatic endothelial cells to create vasculature. Multiple replicate MPS can be built from each patient, which are cultured in an optimized media formulation to support the multiple primary cell types. Patient-specific MPS were treated with radiation or cisplatin + radiation to mimic the treatment the patient received. Multiple orthogonal endpoints were measured including tumor cell migration and viability, cytokine secretion and angiogenesis. Single cell sequencing was also performed on the original tumor tissue and matched MPS. Results: We have successfully created patient-specific models of the HNC TME. Single cell RNA sequencing analysis of the MPS suggests that cellular heterogeneity is being retained. Patient characteristics varied between MPS with some patient TME MPS showing highly migratory tumor cells or angiogenic responses. Treatment responses were measured for each patient-specific MPS and ongoing work will characterize the contributions of the TME to treatment resistance. These data demonstrate the feasibility of using patient-specific MPS to investigate the contribution of the TME to treatment response and resistance and lays important groundwork for using MPS to identify patient treatment responses.
 Citation Format: Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, Sheena C. Kerr. Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5867639a61561e0c0780d2ce0be172eff5b37091" target='_blank'>
              Abstract B034: Engineering the tumor microenvironment: Patient-specific microphysiological systems of the head and neck cancer tumor microenvironment
              </a>
            </td>
          <td>
            Adeel Ahmed, Marcos Lares, Fauzan Ahmed, Adam R. Burr, Paul M. Harari, David J. Beebe, S. Kerr
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represents the most prevalent and deadly primary brain tumor in adults, and it remains highly refractory to current therapeutic interventions. Significant intratumor heterogeneity drives rapid invasion, immune escape, and therapeutic resistance. Therapeutic paradigms based on deconvoluting key molecular mechanisms driving gliomagenesis are urgently needed. Quaking (Qki, encoded by Qk) is frequently altered in GBM patients; Qk mutations or deletions occur in 35% percent of cases, and the Qki protein is absent in 50-60% of cases. Our lab has characterized Qki as important tumor suppressor in GBM, where deletion of Qk enables pre-malignant neural stem cells to expand outside their niche and develop into tumors that recapitulate human GBM. We discovered that Qki functions as a transcriptional coactivator of lipid metabolism genes, whereby Qki loss disrupts lipid homeostasis and compromises the integrity of various membrane structures. Several downstream processes governed by Qki have been delineated; however, upstream regulators of Qki have yet to be identified. Recurrent cancer mutations to specific Qki arginine sites implicate the dysregulation of arginine methylation, a common post-translational modification catalyzed by protein arginine methyltransferases (PRMTs). Leveraging in vitro methylation assays, we demonstrate that Qki is selectively methylated by the PRMT coactivator-associated arginine methyltransferase 1 (CARM1), specifically in the Quaking regulatory domain where cancer mutations are reported. We observe that Qki interacts with known CARM1 substrates, functioning in the transcriptional complex assembled by CARM1. Through mutagenesis studies, we identified arginine 242 (R242) and arginine 256 (R256) as specific CARM1-mediated methylation sites on Qki. Importantly, mutations to these sites compromise Qki’s activity as a transcriptional coactivator. Moreover, brain-specific deletion of CARM1 in vivo results in neurological phenotypes reminiscent of Qki deletion. Altogether, we provide evidence that CARM1 represents a novel tumor suppressor in GBM that functions through the arginine methylation of Qki.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c4f80d5e1592ad297d2b32f6d52b2a08309ce04" target='_blank'>
              EPCO-17. CARM1 METHYLATES GLIOBLASTOMA TUMOR SUPPRESSOR QUAKING TO REGULATE ITS TRANSCRIPTIONAL COACTIVATOR FUNCTION
              </a>
            </td>
          <td>
            Kaylene Lu, Tanner Wright, Seula Shin, Mark T. Bedford, Jian Hu
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is among the most aggressive brain cancers, and it contains glioma stem cells (GSCs) that drive tumor initiation, progression, and recurrence. These cells resist conventional therapies, contributing to high recurrence rates in GBM patients. Developing in vitro models that mimic the tumor microenvironment (TME), particularly the GSC niche, is crucial for understanding GBM growth and therapeutic resistance. Three-dimensional (3D) spheroid models provide a more physiologically relevant approach than traditional two-dimensional (2D) cultures, recapitulating key tumor features like hypoxia, cell heterogeneity, and drug resistance. This review examines scaffold-free and scaffold-based methods for generating 3D GBM spheroids, focusing on their applications in studying the cancer stem cell niche. The discussion encompasses methods such as the hanging drop, low-adhesion plates, and magnetic levitation, alongside advancements in embedding spheroids within extracellular matrix-based hydrogels and employing 3D bioprinting to fabricate more intricate tumor models. These 3D culture systems offer substantial potential for enhancing our understanding of GBM biology and devising more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea7b7a45a32163a4dbbbdd90c32adea5a5aa6ad9" target='_blank'>
              Recapitulating Glioma Stem Cell Niches Using 3D Spheroid Models for Glioblastoma Research
              </a>
            </td>
          <td>
            Hyunji Jo, Seulgi Lee, Min-Hyeok Kim, Sungsu Park, Seo-Yeon Lee
          </td>
          <td>2024-11-01</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="

 The incidence of clonal hematopoiesis of indeterminate potential (CHIP) increases exponentially with aging. Still, rare clones harboring the prototypical clonal hematopoiesis (CH) mutations are nearly ubiquitous in middle aged individuals, suggesting that cell intrinsic or micro-environmental changes observed with aging can promote the expansion of malignant clones to pathologically relevant levels, increasing the risk of hematological malignancies and a multitude of sequelae associated with CHIP.
 To address if aging can alter the fitness dynamics regulating the rate of expansion of malignant clones, we utilized a CRISPR based approach in hematopoietic stem and progenitor cells (HSPC) from mice of different ages to introduce mutations in the Tet2 and Dnmt3a genes or in the Rosa26 safe harbor locus (as competitors), and analyzed their rate of expansion after transplantation in aged matched mice.
 Our data show that Dnmt3a KO cells expand with similar rates in young and old recipient mice after primary and secondary transplant. In contrast, Tet2 KO cells expand at a significantly faster rate in old HSPC in old mice (20 months old) in comparison to young HSPC in young mice (2 months old) and in all hematopoietic populations in the peripheral blood and the bone marrow. Importantly, by heterochronic transplantation of young or old donor HSPC in old or young recipient mice respectively, we observe that the rate of expansion of the Tet2 mutant cells is solely determined by the age of the donor, suggesting that cell intrinsic age-associated changes in the HSPC population facilitate the expansion of the Tet2 KO clones.
 By using donor mice of different ages, we observed that the rate of expansion of the Tet2 KO cells is significantly accelerated by 7 months of age in both male and female mice. Aging is associated with shifts in the relative frequencies of HSPC sub-populations - therefore we sorted myeloid biased HSC, lymphoid biased HSC, short-term (ST) HSC and multipotent progenitor cells (MPP) from old mice, and after Tet2 CRISPR, transplanted the cells into young recipient mice. Both myeloid and lymphoid biased HSC show a similar accelerated rate of Tet2 KO cell expansion, suggesting that the increased myeloid bias with age is not correlated with the Tet2 KO adaptation. In ST-HSC and MPP, we did not observe a selection for the Tet2 clones in the short term. As expected, we noticed a dwindling engraftment in mice transplanted with control ST-HSC and MPP with time; however, Tet2 KO cells exhibited much more robust and prolonged maintenance in these multi-potent progenitor populations, suggesting that the loss of Tet2 can extend the limited self-renewal of progenitor cells.
 Next, we performed competitive transplantations with donor cells with differing genotype (WT or Tet2 KO) or age (2 months or 20 months) to directly compare their relative fitness. Our results show that young and old Tet2 KO HSPC have similar fitness levels, and young Tet2 KO HSPC having a small advantage over young control cells. In contrast, young WT HSPC have a much larger fitness advantage than old WT HSPC. Thus, the enhanced Tet2 KO expansion with age is due to an intrinsic defect of the WT population that cannot effectively compete with Tet2 mutant clones.
 To elucidate the molecular mechanisms for the fitness advantage of Tet2 KO cells with aging, we performed scRNAseq on WT or Tet2 KO HSPC from 2, 12 and 20 month old mice isolated 12 weeks after transplantation into age matched recipient mice. By performing GSEA analyses, we found that aging increased signatures related to mTORC1 activity, ribosomal biogenesis and mitochondrial oxidative phosphorylation, as well as pathways related to RUNX1 and p53 activity, all of which were reverted in Tet2 KO cells to young-like levels.
 In conclusion, our study supports the hypothesis that TET2 mutations observed in CH are selectively advantageous in aging due to cell intrinsic changes occurring in HSC. While the specific age-associated alteration(s) in HSC leading to the adaptation of TET2 mutant clones remain to be defined, we find that several transcriptional changes observable with aging are reverted by the loss of Tet2, likely endowing mutant cells with a larger fitness differential than that observable in a young, highly fit HSC population. Interventions aimed at maintaining HSC clones with higher fitness in aging can be a powerful preventative factor against the emergence of CHIP and its negative consequences.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb4383e22c456c17f49687a230a0253d990b5d6c" target='_blank'>
              Hematopoietic Stem Cell Aging Promotes Expansion of Tet2 Mutant Clones By Cell Intrinsic Mechanisms
              </a>
            </td>
          <td>
            Marco De Dominici, Lamis Naddaf, Vadym Zaberezhnyy, Andrew Goodspeed, Sheng Li, James DeGregori
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="

 Background: Hematopoiesis is arranged in a hierarchy of stem and progenitor cells capable of producing distinct cell fates. Blockade of these differentiation processes is a critical feature of myeloid leukemia development, with different mutation classes resulting in distinct maturation states. Understanding the cellular and transcriptional context for differentiation blockade and biases may offers insights into maximizing differentiation therapy and tracking disease evolution.
 Single cell RNA-seq (scRNA) have provided unprecedented resolution into cellular and clonal heterogeneity. Computational approaches have made recovering differentiation dynamics tractably possible. However, many of these tools were developed through the lens of embryogenesis, a setting not well tuned for the hierarchical, multi-potency of hematopoiesis. Current approaches do not evaluate discontinuous differentiation processes present in malignant leukemia. To address these gaps, we developed SuperJump: a jump-diffusion based supervised cell-fate model. We deploy this approach in 1) normal murine, aged hematopoiesis, and 2) murine models of acute myeloid leukemia with distinct orders of mutation. We identify cells poised for differentiation, and their underlying transcriptional networks.
 Approach: SuperJump is split into 3 phases. In phase 1: we use a semi-supervised method to select positive gene expression signatures from known terminal cell types. We aggregate these signatures as a differentiation entropy score. Here we query multiple cell fates in one model, whereas other strategies aim to rediscover the biological hierarchy for each fate separately.
 In phase 2, we develop a cell-cell transition matrix using a stochastic model-based strategy. Our innovative jump-diffusion model allows for discontinuous interactions between cells to determine the transition probabilities, allowing for a broader reach of neighboring cell states.
 In phase 3, we aim to identify cell transition likelihood using an absorbing Markov chain. We specify cell types as sources and sinks of differentiation. Sink cells are identified as those with the highest membership to their own cell-type. The likelihood for a cell to transition to any given sink is calculated and termed “fate propensity”. Differentiation-poised cells are identified within each population through an outlier detection statistic. Finally, poising is then associated with RNA expression and inferred transcription factor activity.
 Results: First, we ran SuperJump on a well-established model of myeloid fate-bias in normal hematopoiesis from young and middle-aged mice (n=52967 cells). We queried B cells, neutrophils and monocytes as sink cells, and sought to identify poising bias with aging. In the middle-aged sample, we observed myeloid-poised cells as early as HSC (0.08%), a modest expansion in MPPs (0.25%) and a significant degree of poising in middle-aged GMPs (18.9%). This was in stark contrast to young mice which showed no myeloid poising at any stage. We found that myeloid-poised MPPs and GMPs were associated Irf8 expression (MPPs: p=7.52x10-6,GMPs: p= 7.13x10-39) and MPPs poised for B-cells were associated with Pax5 activity (MPPs: p=2.84x10-2). These data highlight early stages of differentiation poising in HSPCs to specific cell fates.
 We next investigated a multi-recombinase murine leukemia model mutant for Dnmt3a, Npm1 and Flt3 (n=5215 cells). In this model we varied the order of mutation between Npm1 and Flt3 and report how mutation order influences cell-fate trajectories. When Flt3 is the last mutation, we observe an expansion of common dendritic progenitors (CDP) which are poised for plasmacytoid dendritic cell differentiation. These cells were enriched for Tcf4 (p=2.200x10-16) and Socs2 (p=5.023x10-16), potentially linking Stat5 signaling and active proliferation towards pDC fate. How mutation order relates to cell fate will be further explored.
 Conclusion: Here we deploy a new approach for identifying differentiation trajectories in normal and malignant hematopoiesis. Our approach is innovative in the use of supervised differentiation signatures, deployment of jump-diffusion to model discontinuous differentiation, and the identification of poised cell states. Our approach allows for insights into early progenitor fates and underlying mechanisms driving them.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c380559ad8b11f15f81e4bab4543c7d71a629396" target='_blank'>
              Super-Jump: State Intent Inference and Lineage Tracking through Semi-Supervised Jump Diffusion Modeling in Normal Hematopoiesis and Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Michael S Bowman, Maria Telpoukhovskaia, Jennifer J. Trowbridge, Robert L Bowman
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 DNA methylation profiling has become a pivotal tool in neuro-oncology. However, little is known about the impact of methylation profiling on radiological imaging in IDH-wildtype glioblastoma (GBM). This study explored the potential of radiomics to non-invasively predict molecular characteristics based on the methylation profiling of GBM.



 This study included a multi-sampling cohort of 32 patients with 113 samples and a single-sampling cohort of 87 patients with 87 samples, which underwent genome-wide methylation analysis and were classified as GBM by the DKFZ/Heidelberg CNS tumor classifier. We performed two different methylation-based deconvolution analyses. Deconvolution 1 estimated the fractions of tumor subtypes (RTK_I, RTK_II, MES_TYP, and MES_ATYP) and cell types in the microenvironment. Deconvolution 2 inferred the abundance of malignant cell states (stem-like and differentiated cell components) and cell types in the microenvironment. We derived 6084 radiomic features from preoperative multi-parametric MRI in each patient.



 In the multi-sampling cohort, Deconvolution 1 and Deconvolution 2 demonstrated that tumor subclass components and immune components exhibited heterogeneous distribution across samples within patients; however, the stem-like to differentiated cell ratio was preserved across samples within patients. In the entire cohort, patients with tumors exhibiting a high proportion of the stem-like component, defined as stem-like tumors, had significantly shorter overall survival. Stem-like tumors exhibited significantly higher levels of the RTK_I component and lower proportions of the RTK_II component compared to non-stem-like tumors. A machine learning model that utilizes support vector machines to classify stem-like tumors using radiomic features achieved a mean AUC of 0.71 (95% confidence interval: 0.56–0.85) in nested cross-validation.



 We showed that radiomethylomics can be used to predict tumor composition and patient survival from preoperative MRI in GBM. Based on these promising results we will increase our power with additional samples to facilitate a personalized approach to GBM management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff84a274afa2828c086628b34b40943cc6ef06da" target='_blank'>
              NIMG-27. RADIOMETHYLOMICS FOR NON-INVASIVE PREDICTION OF TUMOR PHENOTYPES AND SURVIVAL IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yuji Matsumoto, Omkar Singh, Jose A. Garcia, Sunwoo Kwak, Zied Abdullaev, Nelson Freeburg, Dana Silverbush, Kenneth J Aldape, C. Davatzikos, M. Nasrallah
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Adult-type diffuse gliomas, the most common primary brain tumours, remain a clinical challenge in oncology with limited treatment options, restricted anti-tumour immune response and a dismal patient prognosis. Despite their immunosuppressive microenvironment, the formation of lymphoid aggregates containing adaptive immune cells has been reported in gliomas. However, the cellular compositions, immunological function, and relevance of lymphoid aggregates for adaptive anti-tumour immunity is not well understood. Therefore, we performed a comprehensive, unbiased analysis of lymphoid aggregation in 642 adult-type diffuse gliomas using a multi-modal approach, combining DNA methylation and RNA sequencing with spatial transcriptome and proteome profiling. Overall, B cell aggregates and tertiary lymphoid structures (TLS) were observed in 15% of tumours, with the highest prevalence in primary glioma localised to the temporoparietal lobe. The presence of TLS was associated with an improved overall survival. Gliomas containing TLS displayed a remodelled perivascular space, characterised by transcriptional upregulation and spatial redistribution of collagens associated with barrier functions. Spatial transcriptome and proteome profiling revealed heterogenous B:T cell interactions that were associated with elevated CD8 T-cell numbers and differences in IgA and IgG plasma cell forming capacity, suggestive of dynamic adaptive immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8853943d0d644fd815ddb5d51b11bdcf65c71b9" target='_blank'>
              TMIC-39. SPATIAL MULTIOMICS PROFILING REVEALS HETEROGENEITY OF B:T CELL INTERACTIONS AND PLASMA CELL FORMATION IN TERTIARY LYMPHOID STRUCTURES IN HUMAN GLIOMAS
              </a>
            </td>
          <td>
            P. Cakmak, J. Lun, Aakanksha Singh, J. Macas, Michael C. Burger, Lucie Marie Hasse, Elke Hattingen, M. Köhler, J. Schupp, Tatjana Starzetz, Eike Steidl, Karlheinz Plate, Y. Reiss, K. Imkeller
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Juvenile Myelomonocytic Leukemia (JMML) is a RAS-driven aggressive childhood leukemia characterised by aberrant proliferation of myelomonocytic cells and very poor prognosis. Unlike other leukemias, children with JMML have no disease modifying treatments and notably Bone Marrow (BM) transplantation, the only curative treatment, is associated with a 40% risk of relapse. Our previous work characterized JMML leukemia stem cells (LSC), Louka et al, J Exp Med (2021); however the mechanisms through which the JMML LSC exert clonal dominance and treatment resistance are incompletely understood. It is increasingly recognized that aberrant cell-cell communication networks play a key role in cancer development and treatment resistance; an unexplored area in JMML. We therefore set out to investigate the JMML hematopoietic compartments and malignant BM niche using a single cell(sc) multi-omic approach including scRNA seq, cell surface protein expression, CyTOF and spatial transcriptomics from primary patient BM, generating a comprehensive spatial and molecular cellular atlas from diagnosis to disease relapse.
 ScRNAseq analysis (10 JMML, 4 controls, n=38506 single cells) identified expansion of HSC and progenitor cells (HSPC) and a JMML-specific differentiation trajectory leading to expansion of the myelomonocytic series; reduction of the conventional erythroid series and erythroid progenitors with aberrant expression of fetal hemoglobin, in line with the disease features. We also observed marked expansion of dendritic cells (DCs), including conventional (cDC) and plasmacytoid dendritic cells (pDC) and validated this with multiparameter flow cytometry (19 JMML cDC1 p=0.0043, cDC2 p=0.0002, pDC p= 0.0002, 6 controls) and CyTOF analysis (21 JMML, 6 controls). Differential gene expression analysis identified JMML DCs among the top 3 cellular compartments with highly diverse gene expression signatures compared to normal controls, predominantly characterised by upregulation of inflammatory pathways and impaired antigen presentation. Cell-to-cell interaction analysis demonstrated a high number of pathological interactions between JMML LSC and mature populations, many involving JMML LSC and DCs, suggesting that DCs may play a crucial role in remodelling the JMML LSC BM niche and driving disease development.
 Spatial transcriptomics (ST) (n= 791564 cells) using a bespoke panel of 600 genes provided a detailed, high resolution atlas of the JMML BM architecture at diagnosis and at relapse, by capturing all hematopoietic and stroma BM elements. This confirmed a marked increase of JMML DCs compared to normal controls at diagnosis and a further expansion at disease relapse, including a marked expansion of cDCs ( cDC1 and cDC2), however at relapse numbers of pDC remained unchanged. We employed a bespoke ST analytical pipeline to accurately demonstrate increased proximity of HSPC and cDC in JMML compared to other cellular compartments, indicating a high probability of interaction between HSPC and cDC.
 In order to functionally investigate these putative pathological cellular interactions, we established a JMML LSC and DC co-culture proliferation assay. This demonstrated increased proliferation of JMML LSC when cultured in the presence of disease-associated DCs, while a similar expansion was not observed when HSPC from normal controls were co-cultured with DCs from the same donor (JMML n=9 p<0.0001 , normal controls n=6). This was particularly observed when JMML LSC were co-cultured with cDC and not with pDC, in line with our ST observations at relapse. Cell-to-cell interaction network analysis identified an active AXL/MERTK (JMML-LSC) - GAS6 (DCs) axis in JMML and increased GAS6 protein was detected in supernatant of JMML LSC-DC co-cultures. We further demonstrated increased numbers of JMML LSC when cultured in the presence of exogenous GAS6, which was reversed in the presence of specific AXL/MERTK inhibitors or CRISPR-mediated MERTK knockout in primary patient CD34+cells.
 In conclusion, our work represents the first report suggesting that malignant DCs can cell-extrinsically support the maintenance and proliferation of LSC. Using spatial transcriptomics we comprehensively characterized the cellular landscape in JMML BM at diagnosis and relapse and uncovered novel pathobiological cell-to-cell interactions with potential for further preclinical development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d03563acc9224753c731a81b3f6f4fa95fb7adcd" target='_blank'>
              Spatial Transcriptomics Identifies Aberrant Communication between Dendritic Cells and Leukemia Stem Cells As a Therapeutically Targetable Axis in Juvenile Myelomonocytic Leukemia
              </a>
            </td>
          <td>
            E. Louka, Kristian Gurashi, S. Wen, M. Salio, Christina Simoglou Karali, Roman Doll, Sally-Ann Clark, Anastasia Kosmidou, Rosalin Cooper, Daniel Royston, Deena Iskander, Nawshad Hayder, Renuka Teague, Cristina Tormo Garcia, Maria Greco, Paola Vargas Gutierrez, Joshua Bull, Anupama Rao, Irene Roberts, A. Mead
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 As breast cancer migrates into the brain, it acquires distinct genomic, immunologic and phenotypic changes through interactions with the local microenvironment. We sought to determine how targetable gene expression changes mediate breast cancer extravasation into and proliferation within the brain. We created a brain metastatic cell line (“brain-seeking”) via serial intracardiac injection of mCherry+ 4T1 breast cancer cells in Balb/c mice and flow cytometry isolation of micrometastases. We conducted in vitro and in-vivo studies to uncover immunological and genetic differences between primary breast cancer and brain metastases. In culture, we observed significant upregulation in adhesion (p=0.01) and invasion (p=0.0004) along with upregulation of adhesion-promoting genes such as Sdc3, Serpine2 or ECM1 in brain-seeking cells. We observed significant differences in survival of Balb/c mice injected intracranially with brain-seeking cells vs. breast cancer cells (p=0.04). In the mammary fat pad (MFP), parental 4T1 cells caused increased mortality vs. brain-seeking cells (p=0.003). This rejection of brain-seeking 4T1 cells in the MFP could be due to prior adaptation to the brain microenvironment, while rejection of parental 4T1 cells in the brain could be due to lacking said adaptations. Consistent with this finding, of 20,382 analyzed genes in both cell lines, 108 related to immune regulation were significantly upregulated in brain-seeking cells including Cxcl1, Dusp1, C3. Flow cytometry showed increased cytotoxic and helper T cells in intracranially injected parental vs. brain-seeking cells (p <0.001). In the MFP, however, 4T1 parental cells displayed CD3+ T cell depletion and an increased number of neutrophils (p <0.05) while the “brain-seeking cells” showed increased numbers of CD3+ and CD4+ cells (p <0.01) and macrophages (p<0.05). This suggests that brain metastases evolve from their primary tumor through adhesive and invasive adaptations promoting extravasation, along with unique immunological adaptations that allow their proliferation free of immune surveillance in the brain microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40f83672977d31ed510d2a8f39a12e5698eb1" target='_blank'>
              TMIC-01. TARGETABLE GENE EXPRESSION CHANGES MEDIATE EXTRAVASATION OF BREAST CANCER CELLS FROM THE CIRCULATION AND LOCAL PROLIFERATION IN THE BRAIN
              </a>
            </td>
          <td>
            Venina Kalistratova, R. Morshed, Eric Chalif, Sabraj A. Gill, Harmon Khela, Meeki K. Lad, Ankita Sati, Aakriti Mathur, M. Aghi
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 The extracellular matrix (ECM) plays a critical role in cancer progression, with its biophysical properties influencing cancer behavior. ECM stiffness can induce long-lasting adaptations in cancer cells, promoting sustained invasion through a process known as “mechanical memory” (MM). Glioblastoma (GBM) exhibits spatial heterogeneity in stiffness, with stiffer ECM found at the tumor periphery where invasive cells reside and promote recurrence. We hypothesize that increased stiffness at the tumor periphery primes GBM cells for sustained invasion through MM.



 Cell lines were established from image-guided human GBM samples from tumor margin and core (n=10 patients, 20 cell lines). Cells from random locations were conditioned in physiologically relevant extracellular matrices for two weeks. Human organotypic assay (n=3), Migration Assays, Atomic Force Microscopy (AFM), Immunocytochemistry, Western Blot, and RNA-sequencing (n=6) were performed.



 GBM cells at higher stiff surfaces displayed increased migration (p<0.01). Magnetic Resonance Elastography (n=9) confirmed GBM heterogeneity, with softer cores (0.2-0.5kPa) transitioning to stiffer margins (>2kPa). Organotypic scaffolds mimicking the tumor margin promoted enhanced migration of GFP-labeled GBM cells. Margin cells and cells conditioned on stiffer matrices demonstrated higher migratory potential compared to their core and soft-conditioned counterparts (p<0.05). These trends persist over time and across multiple passages. ECM stiffness exposure causes nanomechanical, cytoskeletal, and transcriptional adaptations. AFM shows that cells from the margin (n=6) are softer (p<0.01) and more deformable (p<0.01). RNAseq identified upregulation of MM-specific genes in GBM margin cells. The mechanosensitive transcription factor YAP1 emerged as a key driver of MM. Notably, YAP1 is targetable by the FDA-approved drug Verteporfin. YAP1 expression was elevated in margin and stiff-conditioned cells (p<0.05). Stiff-conditioned cells and margin cells displayed increased sensitivity to Verteporfin treatment (p<0.05), with treatment significantly reducing margin migration compared to the core (p<0.001).



 Stiffer GBM margins promote the formation of aggressive phenotypes through MM. Targeting YAP1 with Verteporfin, a readily available drug, presents a promising strategy to halt GBM invasion and improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d3df0979f0c2cba7865ec958f4264c5339cc38" target='_blank'>
              TMIC-67. EMERGING ROLE OF EXTRACELLULAR MATRIX STIFFNESS AS A HALLMARK OF GLIOBLASTOMA: INSIGHT TO MECHANICAL MEMORY FORMATION AND PROLONGED INVASION IN THE TUMOR PERIPHERY
              </a>
            </td>
          <td>
            Paola Suarez Meade, Rachel Whitehead, Jean Clemenceau, E. Norton, Tanmay Kulkarni, Changjin Hong, Caroline Teeter, Manuela Aramburu, K. Chaichana, Steven S. Rosenfeld, Rajappa S. Kenchappa, H. Guerrero-Cázares, P. Schiapparelli, Tae H Hwang, Konstantinos Konstantopoulos, A. Quiñones-Hinojosa
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Personalized radiotherapy based on the intrinsic sensitivity of individual tumors is anticipated, however, it has yet to be realized. To explore cancer radiosensitivity, analysis in combination with omics data is important. The Cancer Cell Line Encyclopedia (CCLE) provides multi-layer omics data for hundreds of cancer cell lines. However, the radiosensitivity counterpart is lacking. To address this issue, we aimed to establish a database of radiosensitivity, as assessed by the gold standard clonogenic assays, for the CCLE cell lines by collecting data from the literature. A deep learning-based screen of 33,284 papers identified 926 relevant studies, from which SF2 (survival fraction after 2 Gy irradiation) data were extracted. The median SF2 (mSF2) was calculated for each cell line, generating an mSF2 database comprising 285 cell lines from 28 cancer types. The mSF2 showed a normal distribution among higher and lower cancer-type hierarchies, demonstrating a large variation across and within cancer types. In selected cell lines, mSF2 correlated significantly with the single-institution SF2 obtained using standardized experimental protocols or with integral survival, a radiosensitivity index that correlates with clonogenic survival. Notably, the mSF2 for blood cancer cell lines was significantly lower than that for solid cancer cell lines, which is in line with the empirical knowledge that blood cancers are radiosensitive. Furthermore, the CCLE-derived protein levels of NFE2L2 and SQSTM1, which are involved in antioxidant damage responses that confer radioresistance, correlated significantly with mSF2. These results suggest the robustness and potential utility of the mSF2 database, linked to multi-layer omics data, for exploring cancer radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532979073d1bb70a6b201c4bde03ed34d813df58" target='_blank'>
              Establishment of cancer cell radiosensitivity database linked to multi-layer omics data.
              </a>
            </td>
          <td>
            T. Oike, Ryosuke Kambe, Narisa Dewi Maulany Darwis, Atsushi Shibata, Tatsuya Ohno
          </td>
          <td>2024-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Brain tumors arise in close association with neurons, suggesting that these non-neoplastic cells may be critical stromal drivers of brain tumor initiation and growth. Previously, we have shown that murine low-grade optic glioma formation and progression in the setting of Neurofibromatosis type 1 (NF1) is dictated by neurons and neuronal activity. In these studies, these neuronal dependencies reflected neuronal activity-driven enzymatic cleavage of a growth factor (neuroligin-3) from oligodendrocyte precursors cells (tumor initiation) or neuronal production of a paracrine factor to stimulate T cell support of optic glioma growth (tumor progression). Since neurons typically communicate with other neurons through neurotransmitters, we sought to explore the possibility that neurotransmitters operate to modulate low-grade glioma growth using humanized mouse models of pilocytic astrocytoma (PA). Leveraging single cell RNA sequencing of three independent sets of pediatric PAs, we identified neurotransmitter pathway enrichment in the tumor cells. This neurotransmitter pathway enrichment reflected aberrant expression of specific neurotransmitter receptors, which we confirmed in three independently generated PA tissue microarrays and in five distinct primary PA cell lines grown in vitro. Moreover, this aberrant neurotransmitter receptor expression established differential neurotransmitter PA growth dependencies in vitro. Importantly, interruption of neurotransmitter signaling in human PA xenografts attenuated tumor growth and ERK activation in Rag1-/- mice in vivo. Finally, we discovered crosstalk between neurotransmitter receptor and receptor tyrosine kinase signaling that revealed another target for therapeutic inhibition. Taken together, these findings elucidate a previously unknown neurotransmitter PA growth dependency amenable to therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23e8358003ff9c22c4c3138ccf4ef59222584db" target='_blank'>
              CNSC-43. NEUROTRANSMITTER DEPENDENCY UNDERLIES TUMOR GROWTH IN HUMANIZED PEDIATRIC LOW-GRADE GLIOMA MODELS
              </a>
            </td>
          <td>
            C. Anastasaki, Rasha Barakat, Rui Mu, Joshua P Koleske, Chloe M Kernan, Yunqing Gao, Olivia M Cobb, Corrine Gardner, Hongjin Shu, Lakshmi Ramachandran Nair, S. Dahiya, Charles G. Eberhart, Joanna J Phillips, Steven J Mennerick, Jennifer Strahle, Fausto J. Rodriguez, David H. Gutmann
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Complete tumour resection in glioblastoma patients is not possible. Residual therapy-resistant edge-derived cells drive tumour recurrence and infiltrative expansion in glioblastoma. Features that are specific to malignant edge-derived cells may serve as predictive biomarkers to allow individual tailoring of the treatment plan to slow down tumour invasion and prevent tumour recurrence in GBM patients following standard therapy. Intratumoural spatial heterogeneity facilitates therapeutic resistance and recurrence in glioblastoma. Our knowledge on glioblastoma heterogeneity is mostly restricted to the surgically resectable tumour core, while the functional characterization of tumour cells at the infiltrating edge remains largely elusive due to the presence of normal functional brain tissue in the peritumoural lesion. Edge-derived cells exhibit larger capacity for infiltrative expansion and are the main drivers of treatment failure and tumour recurrence, making them action targets for novel treatment approaches in glioblastoma. To resolve the transcriptional heterogeneity of GBM within the spatial context, we profiled gene expression of different tumour regions (“edge” and “core”) obtained at initial surgical resection from primary and recurrent IDH-WT GBM patients with Visium 10x and GeoMx. We show that infiltrative edge-derived cells are spatially segregated and are characterized by regionally shared distinct genomic and transcriptomic signatures that promote invasiveness and underly disease recurrence. Tumour edge is enriched for neuronal signatures, while tumour core displays larger transcriptional subpopulation diversity and abundance of proliferative cells with a high capacity for self-renewal, consistent with the proliferative and cancer stem cell-enriched phenotype in early GBM progressors. Upregulated DEGs of tumour edge cells are significantly associated with ion regulation of transport, chemical synaptic transmission, and nervous system development. These modules may represent tumour cell hijacking of neuronal programs specifically at the tumour periphery as described in the context of glioma-neuron synaptic communication and formation of neurite-like microtubes. Upregulation of ion regulation transport at the tumour periphery indicates enhanced neuronal activity and excitability driving infiltrating growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40797756057776083a00011e6e40dee03507caf6" target='_blank'>
              EPCO-38. IDENTIFYING THE MOLECULAR SIGNATURE OF INFILTRATING EDGE IN GLIOBLASTOMA AS DRIVERS OF TUMOUR INVASION AND RECURRENCE
              </a>
            </td>
          <td>
            Alyona Ivanova, Shamini Ayyadhury, M. Wu, David G Munoz, Trevor Pugh, Sunit Das
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is a highly heterogeneous hematologic malignancy characterized by a poor prognosis, primarily due to resistance to primary chemotherapy and a high relapse rate among patients in remission. This underscores the limitations of conventional chemotherapy in achieving a cure for the majority of AML patients. Spatial transcriptome (ST) technology has emerged as a powerful tool for studying the spatial gene expression and cell-cell interaction patterns, yet its application to calcium-rich tissues like bone marrow has been challenging. In this study, we firstly introduced a novel approach for the paraffin embedding and decalcification of bone marrow tissue, which is fully compatible with Visium glass slides. To validate our method, we collected bone marrow tissues form AML patients and performed spatial transcriptomic sequencing alongside single-cell RNA sequencing. We totally got 5364 ± 2049 (AML-b8) and 4892 ± 2096 (AML-b9) unique molecular identifiers (UMIs) per spot, which is significantly higher than the published studies of bone marrow, and equal to those of soft tissues. This demonstrates that our approach overcomes the limitations of spatial transcriptome data collection in calcium-rich tissues. Utilizing marker genes, we accurately mapped the spatial distribution of megakaryocytes (MKs) and bone trabecular in bone marrow, confirmed through HE staining. For the first time, we integrated spatial transcriptome data, single-cell sequencing and histological analysis to provide a comprehensive characterization of bone marrow in AML patients. After confirming the cell type of each spot, we present the maps of bone marrow tissues in AML patients that could provide both spatial information and gene expression level. Furthermore, we found that the proportion of blasts in our ST results more accurately reflects the tumor burden in AML patients compared to bone marrow smear and flow cytometry, due to the preservation of tissue architecture. Our analysis distinguished seven subpopulations within the tumor cells of bone marrow, including leukemia stem cells (LSKs), highlighting the potential of our approach in elucidating the distribution and transcriptomic profiles of LSKs in AML patients. In our ST models, we observed a significant correlation between genes' expression level and their distance to endosteal niche, such as S100A8, S100A9, CYBB, CHMP3, PACS2, UNC93B1, CMKLR1 and TFRC. Functional analysis of intra-tumor hubs revealed key pathways, including Toll-like receptor cascades, apoptosis process and so on, which are pivotal in bone marrow samples. In conclusion, we present a pioneering method for bone marrow spatial transcriptome analysis, offering a complete and detailed profile of bone marrow in AML patients. This work paves the way for a deeper understanding of AML pathophysiology and may lead to the development of more effective therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1ed9f35bdf12e25119bc30e92359c1da8a97ae" target='_blank'>
              Integrating Spatial Transcriptome, Single-Cell Sequencing and Histopathology Reveals Bone Marrow Spatial Heterogenous in AML Patients
              </a>
            </td>
          <td>
            Xiangjie Lin, Huafeng Wang, Jie Jin
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 The general accessibility of the brain to immune cells has been a subject of intense debate and reinterpretation within the last decade. While glioblastoma’s brain tumor ecosystem is typically considered immunosuppressed, we report the presence of a tumor-reactive immune cell niche within the cranial bone marrow (CB) situated adjacent to treatment-naive tumors. At time of initial diagnosis, glioblastoma patients underwent clinical and anatomical imaging, which revealed an unexpected co-morbid association between the CB and the tumor. Following this observation, we conducted an extensive experimental study including multiplex immunofluorescence, immunoprofiling, functional assays, and single-cell analysis, which identified an active lymphoid niche within the patient’s CB. CD8+ T cell effector cells residing in this niche exhibited acute and durable antitumor responses, and, compared to cells from the distal bone marrow, showed increased expression of the lymphoid egress marker S1PR1. Single-cell T cell receptor sequencing revealed a rich reservoir of non-exhausted tumor-shared CD8+ T cell clonotypes within the CB lymphoid niche, suggesting re-circulation between the proximal bone and the tumor tissue. These clonotypes were characterized by a high predicted tumor-reactivity, significantly surpassing levels found in blood and distal bone marrow. We discovered the entire developmental spectrum of CD8+ T cells in the CB, indicating that anatomical proximity may play a critical role in the process of early anti-tumor response. Correspondingly, cranial enhancement of CXCR4 radiolabel may serve as a surrogate marker, indicating focal association with improved progression-free survival. Our findings support the preservation and further exploitation of these cranioencephalic units to improve clinical care in glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9453b8f2a360f4185cfca9759a99c3c8079cf254" target='_blank'>
              IMMU-71. CRANIOENCEPHALIC FUNCTIONAL LYMPHOID UNITS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, K. Hermann, Florian Rambow, Igor Cima, U. Sure, Björn Scheffler
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7713144e1809faa54b6475ca46362db89d138cc3" target='_blank'>
              In vitro model of retinoblastoma derived tumor and stromal cells for tumor microenvironment (TME) studies
              </a>
            </td>
          <td>
            Emily Alefeld, André Haase, Dario Van Meenen, Bettina Budeus, Oliver Dräger, Natalia Miroschnikov, Saskia Ting, Deniz Kanber, Eva Biewald, Nikolaos Bechrakis, Nicole Dünker, M. Busch
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is characterized by inter- and intra-tumor molecular heterogeneity and demonstrated cell plasticity. Sex differences have been observed in various aspects: a male:female GBM incidence ratio is 1.6:1; standard of care treatments (debulking, radiation, and temozolomide) benefit females more than males. 14 human GBM samples from two study sites were subject to spatial profiling on the Vizgen Merscope platform to examine whether distinct biological and therapeutic niches in GBM were evident, to understand how the tumor microenvironments (TME) contributed to the heterogeneity of GBM, and to interrogate sex difference in the immune landscape of GBM. Multiple sets of spatial probes were designed to capture the cell identities (cancer stem cells, astrocytes, oligodendrocytes, microglia, macrophages, etc.) and the spatially-resolved expression patterns of genes with different functions. Two of the 14 GBM samples were also profiled on another FISH-based spatial platform (Veranome). The data from the two platforms exhibited a linear correlation. We explored the gene-gene co-expression network, and calculated the interaction probabilities between genes which were then compared for sex difference. We found that female samples showed significantly lower (two-fold) CD4+/CD8A+ T-cell count ratios than male samples. We also observed sample-specific spatial niches of different GBM cell subtypes and immune cell populations. Our large-scale spatial transcriptomics study revealed intrinsic sex-based differences in immune response to tumor and illuminated microenvironmental influences on GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f9e48ec997a8efb730a81775200a0b844dc3558" target='_blank'>
              EPCO-44. SPATIAL TRANSCRIPTOMICS ENABLES STATISTICAL MODELING OF SEX DIFFERENCE IN GENE CO-EXPRESSION NETWORK AND IMMUNE LANDSCAPE OF GLIOBLASTOMA
              </a>
            </td>
          <td>
            Yue Hao, Angela Baker, G. Reid, Jingqin Luo, Brian Hilbush, Marcos Otero, Christopher Streck, Karen L. Fink, George J. Snipes, Bruce E. Mickey, Andrew E. Sloan, J. Barnholtz-Sloan, Michael E. Berens
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Despite aggressive therapy combining surgical resection, radiation and chemotherapy, glioblastoma (GBM) almost always recurs. Currently, there is no standard treatment for recurrent GBM, and patient inclusion in clinical trials is based on the genetic and molecular profile of the primary tumor even though the effects of chemoradiation on the transcriptional subtype in matched primary and recurrent GBM remain largely understudied. Paralleling the GLASS consortium, we sought to perform a longitudinal assessment of GBM recurrence after therapy based on molecular subtype with the hypothesis that recurrent tumors exhibit a distinct transcriptional landscape different from treatment-naive tumors. To address this, we utilized primary tumor-derived glioma stem cells (GSCs) representing classical and proneural subtypes. We then undertook in vivo intracranial implantation studies and performed standard concomitant treatment that mimics the Stupp protocol. Upon bulk RNAseq data analyses of untreated (n=21) and recurrent tumors (n=17) we identified that while the molecular subtype of recurrent tumors does not change post therapy, their transcriptional profile is more heterogenous compared to untreated tumors for both subtypes. Hierarchical cluster analyses revealed distinct sub-clusters of recurrent tumors. Common mechanisms of recurrence include upregulation of angiogenesis pathways and PI3k/Akt/mTOR signaling pathways. Interestingly, recurrent tumors exhibit upregulation of genes involved in cellular metabolism and membrane transport/ potential. The source of heterogeneity in recurrent tumors in both subtypes appears to be due to the diversity in expression of gene sets involved in hallmark of cancer pathways such as cell cycle, Notch and Wnt signaling pathways and heterogeneity in oncogenic signatures and cell states resembling fetal or neural cells. Further research into the recurrent tumor microenvironment by spatial multi-omics approaches using this pre-clinical recurrent GBM model will elucidate the linkage between cellular as well as molecular heterogeneity and enable identification of targets that can help drive therapeutic decisions for recurrent GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbf4b1b944267298359c5f873eaa544818a4a7d9" target='_blank'>
              TMIC-46. TRANSCRIPTIONAL HETEROGENEITY AND MECHANISTIC PATHWAYS OF RECURRENT GLIOBLASTOMA: INSIGHTS FROM A PRE-CLINICAL RECURRENT TUMOR MODEL
              </a>
            </td>
          <td>
            Mohammad Fayyad Zaman, Marc Daou, Lynette M. Phillips, Sanjay Singh, L. Long, J. Gumin, Daniel Ledbetter, Frederick F Lang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Glioblastoma multiforme (GBM) remains the most lethal and prevalent primary brain tumor worldwide. Despite advances in understanding its biology and multimodal therapy options, the overall prognosis for GBM patients remains bleak, primarily due to the high recurrence rate, which is intimately linked to tumor resistance against standard therapeutic interventions. Glioblastoma stem cells (GSCs) contribute to therapeutic resistance and cellular heterogeneity through their self-renewal properties and ability to adapt. Thus, identifying new molecular targets driving GBM progression is crucial for developing effective therapies. Recent advancements in genome biology have revealed that long non-coding RNAs (lncRNAs) play roles in various biological functions, and the dysregulation of numerous lncRNAs has been shown to be associated with specific cancers. To identify GSC-associated lncRNAs, a comprehensive analysis was performed using single-cell RNA sequencing datasets from GBM tumors, GBM organoids, GSC-enriched GBM tumors, as well as normal human brain samples. A chromatin-enriched lncRNA, GIHCG, was identified as being highly expressed in GSCs compared to non-neoplastic neural stem cells. The GIHCG gene, located on human chromosome 12q14.1, is frequently amplified in glioblastoma and impacts overall survival. Bioinformatic and functional analyses revealed that GIHCG is amplified in 13% of GBM patient samples, and high expression of GIHCG is a negative prognostic marker for GBM patients, with significantly reduced overall survival compared to the low expression group. In addition, the depletion of GIHCG significantly reduced GSC proliferation and migration. Further, the knockdown of GIHCG decreased both mRNA and protein expression levels of critical GSC stemness factors, including SOX2, NESTIN, and OLIG2. Moreover, GIHCG depletion reduced the sphere formation capacity of GSCs, demonstrating the functional importance of GIHCG in the maintenance of GSC stemness. These results indicate that GIHCG is essential for GSC proliferation, migration, and stemness, underscoring its potential therapeutic value in RNA-based therapies for combating glioblastoma malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d912d5117a292a29ded4aacf7d2e6cd8ccb4fe" target='_blank'>
              STEM-19. TARGETINGGIHCG LNCRNA IN GLIOBLASTOMA STEM CELLS: A POTENTIAL THERAPEUTIC STRATEGY
              </a>
            </td>
          <td>
            R. Hazra
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Although immune checkpoint blockade therapy has shown promising results in a small subset of patients with colorectal cancer with high microsatellite instability, the majority of patients with colorectal cancer do not respond to immune checkpoint blockade therapy. The main obstacle to the success of immunotherapy in cancer treatment is the exhaustion of tumor-infiltrating lymphocytes (TIL). Elucidating the spatial organization of immune checkpoints within the tumor microenvironment (TME) could pave the way for the development of novel prognostic tools and therapeutic strategies to enhance antitumor immune responses. To clarify the spatial and functional diversity of TILs in the colorectal TME, we performed multiplexed IHC to examine the exhaustion of TILs in the TME, the expression of PD-1 and T-cell immunoglobulin and mucin domain–containing protein 3 (TIM-3), which are major biomarkers of T-cell exhaustion, and least absolute shrinkage and selection operator method–Cox analyses of the correlation between colorectal cancer prognosis and TME features. For proof of concept, the antitumor efficacy of TIM-3 and PD-1 dual blockade in colorectal cancer was further evaluated in a CT26 s.c. tumor model of human colorectal cancer. We found that the spatial context of PD-1 and TIM-3 successfully predicted the overall survival of patients with colorectal cancer independent of tumor–node–metastasis stage. Dual targeting of PD-1 and TIM-3 in mouse tumor models inhibited tumor progression and reduced T-cell exhaustion, indicating a potential strategy for improving the clinical treatment of colorectal cancer. Significance: The identification of specific spatial patterns of immune checkpoint expression that correlate with overall survival in patients with colon cancer suggests a potential prognostic tool for risk stratification and treatment selection. These findings pave the way for the development of novel therapeutic strategies to enhance antitumor immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b7e667f607ec054318ced952999594c8c938db" target='_blank'>
              Spatial Context of Immune Checkpoints as Predictors of Overall Survival in Patients with Resectable Colorectal Cancer Independent of Standard Tumor–Node–Metastasis Stages
              </a>
            </td>
          <td>
            Hao Kong, Qingxin Yang, Chunwei Wu, Xiangji Wu, Xinrui Yan, Li-Bin Huang, Lu Chen, Zongguang Zhou, Ping Wang, Hong Jiang
          </td>
          <td>2024-11-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The evolution of cellular heterogeneity in IDH-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. To address it, we assembled a longitudinal cohort of 121 primary and recurrent GBM specimens from 59 patients, with extensive clinical annotations, and profiled it by single-nucleus RNA-sequencing and bulk tumor DNA sequencing. In most cases, longitudinal samples diverged in their composition of cell types and cell states. However, almost all theoretical trajectories were observed in our cohort such that the overall distribution of cell types and cell states was comparable between primary and recurrent samples. The most consistent longitudinal effect (66% of patients) was a lower malignant cell fraction at recurrence and a reciprocal increase in proportions of glio-neuronal TME cell types; in some cases, this was further accompanied by a coordinated shift of malignant cells towards neuronal-like states. MGMT methylation and radiation-related small deletion phenotypes were linked to particular trajectories, with depletion of mesenchymal-like cells and enrichment of hypoxia-related malignant cells, respectively. Importantly, changes in malignant states were also associated with specific changes in TME composition. In summary, our analysis highlights diverse longitudinal GBM trajectories that are shaped by treatment response and TME interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea653ba81a99c699e1e7affffaf3c3f66e7e9bb" target='_blank'>
              EPCO-34. DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. N. G. Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, Franck Bielle, A. Laurenge, Anna Luisa Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer spheroids are spherical, three-dimensional (3D), in vitro assemblies of cancer cells, which are gaining importance as a useful model in cancer behavior studies. Designed to simulate key features of the in vivo tumor microenvironment, spheroids offer reliable insights for drug screening and testing applications. We observed contrasting phenotypes in 3D cervical cancer (CC) cultures. Thus, in this study, we compared the proteomes of 3D and traditional two-dimensional (2D) cultures of CC cell lines, HeLa, SiHa, and C33A. When cultured in in-house poly-(2-hydroxyethyl methacrylate)-coated plates under conditions suitable for 3D spheroid formation, these CC cell lines yielded spheroids exhibiting different features. Proteomic analysis of cells cultured in 2D and 3D cultures revealed similar protein profiles but remarkable differences in the expression levels of some proteins. In SiHa and C33A cells, the upregulation of key proteins required for spheroid formation was insufficient for the formation of compact spheroids. In contrast, HeLa cells could form compact spheroids because they upregulated the proteins, including cadherin-binding, cytoskeleton, and adhesion proteins, necessary for spheroid formation during the remodeling process. Overall, this study unravels the mechanisms underlying the formation of spheroids in the commonly used CC cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815cc2d468c5e900a469f3a75fe62c8fb5f3a378" target='_blank'>
              Proteomic Analysis Reveals Cadherin, Actin, and Focal Adhesion Molecule-Mediated Formation of Cervical Cancer Spheroids
              </a>
            </td>
          <td>
            P. Molika, K. Leetanaporn, W. Chiangjong, Pongsakorn Choochuen, R. Navakanitworakul
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Background Angiogenesis is indispensable for the sustained survival and progression of both embryonic development and tumorigenesis. This intricate process is tightly regulated by a multitude of pro‐angiogenic genes. The presence of gene modules facilitating angiogenesis has been substantiated in both embryonic development and the context of tumor proliferation. However, it remains unresolved whether the pro‐angiogenic gene modules expressed during embryonic development also exist in tumors. Methods This study performed a pan‐cancer single‐cell RNA sequencing (scRNA‐seq) analysis on samples from 332 patients across seven cancer types: thyroid carcinoma, lung cancer, breast cancer, hepatocellular carcinoma, colorectal cancer, ovarian carcinoma, and prostate adenocarcinoma. Data processing was carried out using the Seurat R package, with rigorous quality control to filter high‐quality cells and mitigate batch effects across datasets. We used principal component analysis (PCA), shared nearest neighbor graph‐based clustering, and Uniform Manifold Approximation and Projection (UMAP) to visualize cell types and identify distinct cell clusters. Myeloid cell subpopulations were further analyzed for the expression of embryonic pro‐angiogenic gene modules (EPGM) and tumor pro‐angiogenic gene modules (TPGM). Results The analysis identified nine major cell types within the tumor microenvironment, with myeloid cells consistently exhibiting elevated expression of both tumor pro‐angiogenic gene modules (TPGM) and EPGM across all tumor types. In particular, myeloid cells, including macrophages and monocytes, showed high EPGM expression, indicating an active role of embryonic pro‐angiogenesis pathways in tumors. A subset analysis revealed 20 distinct myeloid subtypes with varying EPGM and TPGM expression across different cancers. Treatment and disease stage influenced these gene expressions, with certain subtypes, such as HSPAhi/STAT1+ macrophages in breast cancer, displaying reduced pro‐angiogenic gene activity post‐treatment. Conclusion This study provides evidence that tumors may exploit EPGM to enhance vascularization and support sustained growth, as evidenced by the elevated EPGM expression in tumor‐associated myeloid cells. The consistent presence of EPGM in TAMs across multiple cancer types suggests a conserved mechanism wherein tumors harness embryonic angiogenic pathways to facilitate their progression. Distinct EPGM expression patterns in specific myeloid cell subsets indicate potential therapeutic targets, particularly in cases where EPGM activation contributes to resistance against anti‐angiogenic therapies. These findings shed new light on the molecular mechanisms underlying tumor angiogenesis and highlight the prognostic relevance of EPGM expression in cancer, underscoring its potential as a biomarker for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f200ed8424d51876bd86eacf393553e56f7950" target='_blank'>
              Pan‐Cancer Single‐Cell Transcriptomic Analysis Reveals Divergent Expression of Embryonic Proangiogenesis Gene Modules in Tumorigenesis
              </a>
            </td>
          <td>
            Zeshuai Wang, Yiyi Su, Lisha Zhao, Wei Liu, Jiaqi Zhang, Wei Yang, Hanjie Li, Mingqian Feng, Hao Wang, Zhuo Song
          </td>
          <td>2024-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is now the first and second leading cause of cancer-related deaths in men and women under 50, respectively, upending prior predictions. While many risk factors have been identified, the etiology of early onset disease is poorly understood, and it is unclear why some benign polyps progress to CRC while most do not. The genetic drivers of colon tumor initiation and progression to malignancy are not sufficient to drive disease on their own, as evidenced by the near ubiquitous presence of oncogenic mutations within morphologically “normal” adult tissues like the colon. This raises several fundamental questions: 1) what are the pioneering molecular and cellular events underlying benign-to-malignant transition following oncogenic mutation? 2) Is this sequence of events predetermined by cancer genetics or are additional (micro)environmental stimuli necessary? 3) How does the benign-to-malignant transition reshape tumor clonal architecture and is this immunogenic? We have begun to address these questions using a combination of rare human colon polyps with evidence of early malignant transition and innovative new mouse models that enable control of stepwise colon cancer progression. These autochthonous models combine in vivo CRISPR-Cas9 and an inducible “split-Cre” system to decouple focal tumor initiation in the distal colon (via Apc knockout) from spatiotemporal control of secondary driver events of malignant progression (e.g., Kras G12D, Trp53 knockout) in rare cells of benign adenomas. We have validated the in vivo functionality of these models, generating early “intramucosal carcinoma” and “adenocarcinoma-in-adenoma”—physiologically relevant and elusive stages of human colon cancer development that have never been reliably captured in model systems to date. Comparing these tumors with spatial transcriptomic data from human specimens, we report distinct transcriptional programs of early progression, a unique stromal reaction associated with early progression, and restructuring of glandular architecture and the stem cell niche in early carcinoma. By delineating the earliest events underlying cancer progression, this work promises to inform new strategies for early detection and prevention. We anticipate that the models will also be particularly valuable in functionally interrogating host and environmental factors that impact risk of early onset cancer.
 Citation Format: Peter M.K. Westcott, Yihan Qin, Daniel Zhang, Nikita Persaud, Zakeria Aminzada, Nischal Bhandari, Colin McLaughlin, William Rideout III, Santiago Naranjo, Song Han, Rodrigo Romero, Claire Regan, Jonathan Preall, Semir Beyaz, Tyler Jacks, Zhen Zhao, Sepideh Gholami. Early changes to the colon tumor microenvironment during benign-to- malignant transition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accd8252a49f53325d258371518d1ee2669b74af" target='_blank'>
              Abstract PR016: Early changes to the colon tumor microenvironment during benign-to- malignant transition
              </a>
            </td>
          <td>
            P. Westcott, Yihan Qin, Daniel Zhang, Nikita Persaud, Zakeria Aminzada, Nischal Bhandari, Colin McLaughlin, William Rideout, S. Naranjo, Song Han, Rodrigo Romero, C. Regan, Jonathan Preall, Semir Beyaz, Tyler Jacks, Zhen Zhao, Sepideh Gholami
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Central nervous system neuroblastoma with FOXR2 activation (NB-FOXR2) is a high-grade tumor of the brain hemispheres and a newly identified molecular entity. Tumors express dual neuronal and glial markers, leading to frequent misdiagnoses, and limited information exists on the role of FOXR2 in their genesis. To identify their cellular origins, we profiled the transcriptomes of NB-FOXR2 tumors at the bulk and single-cell levels and integrated these profiles with large single-cell references of the normal brain. NB-FOXR2 tumors mapped to LHX6+/DLX+ lineages derived from the medial ganglionic eminence, a progenitor domain in the ventral telencephalon. In vivo prenatal Foxr2 targeting to the ganglionic eminences in mice induced postnatal cortical tumors recapitulating human NB-FOXR2 specific molecular signatures. Profiling of FOXR2 binding on chromatin in murine models revealed an association with ETS transcriptional networks, as well as direct binding of FOXR2 at key transcription factors that coordinate initiation of gliogenesis. These data indicate that NB-FOXR2 originate from LHX6+/DLX+ interneuron lineages, a lineage-of-origin distinct from that of other FOXR2-driven brain tumors, highlight the susceptibility of ventral telencephalon-derived interneurons to FOXR2-driven oncogenesis, and suggest that FOXR2-induced activation of glial programs may explain the mixed neuronal and oligodendroglial features in these tumors. More broadly, this work underscores systematic profiling of brain development as an efficient approach to orient oncogenic targeting for in vivo modeling, critical for the study of rare tumors and development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62cff560a5d91b324b57873507549af7d3c89c0" target='_blank'>
              FOXR2 targets LHX6+/DLX+ neural lineages to drive CNS neuroblastoma.
              </a>
            </td>
          <td>
            Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Michael McNicholas, S. Hébert, Adam Ptack, D. Faury, Jessica W. Tsai, A. Korshunov, T. Phoenix, B. Ellezam, David T W Jones, Michael D. Taylor, P. Bandopadhayay, Manav Pathania, Nada Jabado, Claudia L Kleinman
          </td>
          <td>2024-11-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="NPM1-mutated AML is one of the largest entities in international classification systems of myeloid neoplasms, which are based on integrating morphologic and clinical data with genomic data. Previous research, however, indicates that bulk transcriptomics-based subtyping may improve prognostication and therapy guidance. Here, we characterized the heterogeneity in NPM1-mutated AML by performing single-cell RNA-sequencing and spectral flow cytometry on 16 AML belonging to three distinct subtypes previously identified by bulk transcriptomics. Using single-cell expression profiling we generated a comprehensive atlas of NPM1-mutated AML, collectively reconstituting complete myelopoiesis. The three NPM1-mutated transcriptional subtypes showed consistent differences in the proportions of myeloid cell clusters with distinct patterns in lineage commitment and maturational arrest. In all samples, malignant cells were detected across different myeloid cell clusters, indicating that NPM1-mutated AML are heavily skewed but not fully arrested in myelopoiesis. Same-sample multi-color spectral flow cytometry recapitulated these skewing patterns, indicating that the three NPM1-mutated subtypes can be consistently identified across platforms. Moreover, our analyses highlighted differences in the abundance of rare hematopoietic stem cells suggesting that skewing occurs early in myelopoiesis. To conclude, by harnessing single-cell RNA-sequencing and spectral flow cytometry, we provide a detailed description of three distinct and reproducible patterns in lineage skewing in NPM1-mutated AML that may have potential relevance for prognosis and treatment of patients with NPM1-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d2c7d3f716516ca564f7acecb6aa3ff7231897" target='_blank'>
              Resolving inter- and intra-patient heterogeneity in NPM1-mutated AML at single-cell resolution
              </a>
            </td>
          <td>
            E. O. Karakaslar, E. Argiro, N. E. Struckman, R. H. Shirali, J. Severens, W. Honders, S. L. Kloet, H. Veelken, M. Reinders, M. Griffioen, E. B. van den Akker
          </td>
          <td>2024-12-11</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Adoptive cell therapy using Chimeric Antigen Receptor (CAR) T cells has revolutionized cancer immunotherapy for a number of hematologic diseases. However, challenges remain in Chronic Lymphocytic Leukemia (CLL) where durable remission rates are low. Previously, we found that CAR vector-mediated disruption of the epigenetic regulator TET2 led to the clonal expansion of CD8+ CAR T cells in a CLL patient (Fraietta et al., Nature 2018). TET2 is an ɑ-ketoglutarate (ɑKG)-dependent dioxygenase that catalyzes the conversion of methyl-Cytosine in DNA to hydroxy-methyl-cytosine, which culminates in the removal of methylation marks in silenced genes. The TET2-disrupted CAR T cells exhibited immunophenotypic and epigenetic characteristics of early memory T cells. Given that TET2 is a known tumor suppressor gene, complete ablation via CRISPR/Cas9 could risk malignant transformation of engineered T cells (Jain et al., Nature 2023). Isocitrate dehydrogenase 1 (IDH1) mutant R132H had previously demonstrated exhibition of neomorphic activity that producing excess amounts of D-2-hydroxyglutarate (D-2HG), which is a metabolic inhibitor of TET2 via competitor ɑKG (Figueroa et al., Cancer Cell 2010). We hypothesized that IDH1 neomorph overexpression in CAR T cells safely inhibits TET2 activity and thereby phenocopies TET2 genetic disruption as observed in our patient.
 To investigate the biological impact of IDH1 R132H neomorph overexpression (IDH1NM OE) in CAR T cells, we developed a bicistronic expression system for CD19-specific 2nd generation CAR with 4-1BB/CD3ζ to co-express IDH1 R132H neomorph (IDH1NM) and used wildtype IDH1 (IDH1WT) as a control. Western blotting confirmed IDH1WT and IDH1NM overexpression in manufactured CAR19 T cells. IDH1NM OE led to significant D-2HG accumulation in CAR19 T cells, unlike IDH1WT. The expression of IDH1NM did not affect the cytolytic activity of CAR T cells, confirming the preservation of CAR their effector function. IDH1NM OE, however, significantly augmented CAR T cell response in a repeat tumor challenge model, similar to our previously published TET2 knock-down studies in normal donor T cells (Fraietta et al., Nature 2018). Further, IDH1NM CAR T cells showed significantly increased outgrowth in the CCR7+CD45RA+ central memory phenotype during chronic CAR stimulation. Additionally, we observed that effector cytokine levels, including IFNɣ and TNFɑ, were significantly reduced along with a decrease in CD107a, a major degranulation marker, while IL-2 levels were increased which is consistent with our TET2 knock-down studies. This suggests that IDH1NM OE improves the maintenance of early memory as well as effector function in the face of repeated tumor cell encounter.
 Next, we aimed to identify the relevance between metabolic reprogramming and memory function mediated by IDH1NM OE in CAR T cells, based on our hypothesis that IDH1NM OE impact ɑKG-dependent dioxygenase activity including various histone demethylase as well as TET2 in CAR T cells. We observed an increase in methylated histone and a decrease in oxidated cytosine in IDH1NM CAR T cells which experienced repeated CAR stimulation in vitro. This suggests that metabolic reprogramming by IDH1NM OE in CAR T cells leads to epigenetic alterations toward a memory-like phenotype in CAR T cells. Our studies therefore confirm that the metabolic reprogramming of CAR T cells via IDH1 NM OE phenocopies TET2 disruption. Pre-clinical mouse studies are ongoing and will be presented at the ASH.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bced5e74ec75c24287dced3600cf0ae522fdb7ac" target='_blank'>
              Metabolic Reprogramming By IDH1 Neomorph Epigenetically Guides CAR T Cell Differentiation
              </a>
            </td>
          <td>
            Kunho Chung, Dongxu Jiang, Xiaorong Gu, Jie Xu, Céline Grégoire, Beatriz Coutinho de Oliveira, B. Jha, J. Melenhorst
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4422b0759e87ed388f1ba9e5bcbfed55cb314237" target='_blank'>
              Spatial transcriptomics reveals strong association between SFRP4 and extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            M. K. Andersen, S. Krossa, Elise Midtbust, Christine A. Pedersen, M. Wess, T. S. Høiem, T. Viset, Øystein Størkersen, Ingunn Nervik, E. Sandsmark, H. Bertilsson, G. Giskeødegård, M. B. Rye, May-Britt Tessem
          </td>
          <td>2024-11-08</td>
          <td>Communications Biology</td>
          <td>4</td>
          <td>26</td>
        </tr>

        <tr id="Hematopoietic malignancies emerge through the gradual acquisition of genetic mutations within hematopoietic stem and progenitor cells (HSPCs). Mutations that occur early in disease progression impart a selective growth advantage to HSPCs, which allows them to expand and contribute to a substantial percentage of mature blood cells. This increased expansion is termed clonal hematopoiesis (CH) and is a preleukemic phase associated with an increased risk of developing leukemia. Inhibitor of DNA binding 1 (ID1) protein is a transcriptional regulator of proliferation/differentiation of neuronal, muscle, hematopoietic and other cells, and is frequently overexpressed in cancer. Id1 is expressed at low levels in normal HSCs and is induced by growth factors and other mediators of inflammatory stress and promotes HSPC proliferation in vitro and in vivo. Since chronic inflammation is associated with the progression of hematopoietic malignancies, reducing Id1 expression during CH may be therapeutic. Mutations in TET2 are frequently observed in patients with CH, and Tet2−/−mice develop CH that progress to hematopoietic malignancies. Id1 is upregulated in murine Tet2−/− HSPCs and in AML, CMML and MDS patient samples with TET2 mutations. Genetic ablation of Id1 in Tet2−/− HSPCs reduces HSPC expansion/self-renewal/CH, extramedullary hematopoiesis, myeloid skewing, genetic instability and delays the onset of disease. Mechanistically, p16 expression, senescence and apoptosis were increased and proliferation decreased in Tet2−/−; Id1−/− HSPCs. Thus, ID1 may represent a potential therapeutic target to reduce CH, hematopoietic hyperplasia, and delay the onset of disease. One Sentence Summary Genetic ablation of Id1 in Tet2−/− mice rescues clonal hematopoiesis by increasing CDKI expression, apoptosis, senescence, and differentiation, and reducing cell growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3312a0ebaea58379a205a9959b262e76e84831" target='_blank'>
              Id1 Promotes Clonal Hematopoiesis in Mice with Tet2 Loss of Function
              </a>
            </td>
          <td>
            Shweta Singh, Tanmoy Sarkar, K. Gudmundsson, Brad L. Jakubison, Holly Morris, Sandra Burkett, Karim Baktiar, Gary T. Pauly, D. Sigano, Joel P. Schneider, Jonathan Keller
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) may originate from somatic mutations within the subventricular zone (SVZ). We investigated putative GBM origin-cells in the SVZ with somatic mutations implicated in GBM development. We hypothesized these origin-cells have unique properties compared to tumor-derived GBM tumorspheres (TSs). We isolated, characterized, and analyzed mouse transcriptomes of GBM origin-cells and GBM TSs, and sought to identify potential human GBM origin-cells within the SVZ. Using a mouse model with somatic mutations (Trp53/Pten/hEGFRvIII) induced within the SVZ, we demonstrated that mutated cells in the SVZ migrate to the cortex and form GBM. We isolated mouse GBM (mGBM) origin-cells from the SVZ and mGBM TSs from the tumors, comparing their characteristics, transcriptome, and tumorigenic potential upon orthotopic or SVZ injection. To identify potential human GBM origin-cells, single-nucleus RNA sequencing (snRNAseq) was performed on human GBM SVZ specimens. Our investigation revealed intriguing properties and distinct gene expression patterns in GBM origin-cells. mGBM origin-cells from the SVZ somatic mutation model demonstrated enhanced stemness and invasiveness compared to control SVZ cells. These cells exhibited site-specific tumorigenic capacity: cortical injection of mGBM origin-cells failed to induce tumor formation, while re-injection into the SVZ led to migration to the cortex and tumor development. RNA sequencing revealed significant gene expression differences between mGBM origin-cells and mGBM TSs despite similar mutation profiles, suggesting distinct identities. snRNAseq analysis of human GBM SVZ specimens (pathologically tumor-free) identified cells with chromosomal aberrations characteristic of early GBM development, including chromosome 7 gain / 10 loss. These findings suggest potential human GBM origin-cells within the SVZ. This study describes a distinct population of GBM origin-cells within the SVZ, different from tumor-derived cancer stem cells. These GBM origin-cells exhibit unique properties, including enhanced stemness and invasiveness, and display site-specific tumorigenic capacity, forming tumors only in the SVZ environment. Further research into the factors that cause GBM origin-cells in the SVZ to migrate and form tumors in the cortex is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a544bba9a9968cc22ab71ada85d9eea9182155fd" target='_blank'>
              STEM-04. UNVEILING GLIOBLASTOMA ORIGIN-CELLS: ISOLATION AND CHARACTERIZATION FROM THE SUBVENTRICULAR ZONE
              </a>
            </td>
          <td>
            R. Choi, Se-Young Jo, Eunchae Yeo, Hyeong-Cheol Oh, E. Jo, Jin-Kyung Shim, Hyun Jung Kim, Joo Ho Lee, J. Yoo, Seonah Choi, J. Moon, E. Kim, Se Hoon Kim, Jong Hee Chang, Hyun Seok Kim, Jeong Ho Lee, Hoon Kim, Sangwoo Kim, Seok-Gu Kang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Epigenetic treatment has been studied as a means to augment anti-tumor immune cell activity by increasing the expression of antiviral response genes. Previous studies, including our own, have revealed an additional consequence of epigenetic treatment: reactivation of transposable elements (TEs). Normally, most TEs are epigenetically repressed in healthy somatic tissues. However, epigenetic treatment can reactivate TEs as previously unannotated promoters, particularly in rapidly proliferating cells. These reactivated TEs could generate unique proteins that may act as novel antigens for immunotherapy. To study epigenetic treatment as a novel method to augment immunotherapy in cancers with a low number of somatic mutation-derived antigens, we chose glioblastoma as a model. Specifically, we investigated if drug-induced TE expression increases the antigen repertoire upon combinatorial treatment with DNMT inhibitor (Decitabine) and HDAC inhibitor (Panobinostat), agents currently being tested for clinical use in cancers, including CNS tumors. Using patient-derived primary glioblastoma stem cell (GSC) lines and control cell lines (astrocytes and fibroblasts), we first profiled treatment-induced changes in the epigenetic landscape using WGBS-seq and ATAC-seq. Then, using nanoCAGE-seq, both short-read and long-read RNA-seq, we identified treatment-induced, TE-derived transcripts that are predominantly expressed in glioblastoma cells along with upregulation of antiviral gene programs. Finally, using LC-MS/MS, we validated that tumor-enriched reactivated TEs, which mostly originate from the LTR class, encode HLA-I presented antigens in primary GSCs. Notably, epigenetic therapy also reactivated many TEs in proliferating control cell lines, and thus we provide evidence that targeted approaches like CRISPRa might reduce the side effects of activating nonspecific TE loci. Overall, our findings suggest that the therapeutic utilization of epigenetic treatment-induced, TE-derived antigens holds great promise but also provides a cautionary tale regarding the careful selection of tumor-enriched antigen targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb41ebbe9bc1c8d312f1b15f580716fe9d040ce3" target='_blank'>
              EPCO-08. EPIGENETIC THERAPY POTENTIATES TRANSPOSABLE ELEMENT TRANSCRIPTION TO CREATE TUMOR-ENRICHED ANTIGENS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, Hyung Joo Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert Kim, Ting Wang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood.



 We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques.



 We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001).



 We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab5bf963aa397423b3e3483e91bf239995470ed" target='_blank'>
              A05 - HPB Interrogating the spatial biology of pancreatic cancer-associated fibroblasts
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, R. Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-11-01</td>
          <td>British Journal of Surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor growth and progression involve coordinated regulation by internal, microenvironmental, and systemic signals and often display conspicuous sexual dimorphism. The mechanisms governing the integration and coordination of these signals, along with their sex-based differences, remain largely unknown. Using a Drosophila tumor model originating from nonreproductive tissue, we show that female-biased tumor growth involves multifaceted communications among tumor cells, hemocytes, and neuroendocrine insulin-producing cells (IPCs). Notch-active tumor cells recruit hemocytes carrying the tumor necrosis factor–α (TNF-α) homolog Eiger to the tumor microenvironment (TME), activating the c-Jun N-terminal kinase (JNK) pathway in tumor cells, instigating the sexually dimorphic up-regulation of cytokine Unpaired 2 (Upd2). Upd2, in turn, exerts a distal influence by modulating the release of a Drosophila insulin-like peptide (Dilp2) from IPCs. Dilp2 then activates the insulin signaling in the tumor, thereby fostering sexual-dimorphic tumor growth. Together, these findings reveal a relay mechanism involving the TME and systemic signals that collectively control the sexual dimorphism of tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4504e9cb20bc72b2973f3610b4d0ff83ae5829a" target='_blank'>
              Sex-dimorphic tumor growth is regulated by tumor microenvironmental and systemic signals
              </a>
            </td>
          <td>
            Xianfeng Wang, Hongcun Bao, Yi-Chun Huang, Anindita Barua, Chun-Ming Lai, Jie Sun, Youfang Zhou, Fei Cong, Shangyu Gong, Chih-Hsuan Chang, Wu-Min Deng
          </td>
          <td>2024-12-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA methylation can give rise to robust biomarkers of aging, yet most studies profile it at the bulk tissue level, which masks cell type-specific alterations that may follow distinct aging trajectories. Long-read sequencing technology enables methylation profiling of extended DNA fragments, which allows mapping to their cell type of origin. In this study, we introduce a framework for evaluating cell type-specific aging using long-read sequencing data, without the need for cell sorting. Leveraging cell type-specific methylation patterns, we map long-read fragments to individual cell types and generate cell type-specific methylation profiles, which are used as input to a newly developed probabilistic aging model, LongReadAge, capable of predicting epigenetic age at the cell-type level. We apply LongReadAge to track aging of myeloid cells and lymphocytes from bulk leukocyte data as well as circulating cell-free DNA, demonstrating robust performance in predicting age despite limited shared features across samples. This approach provides a novel method for profiling the dynamics of epigenetic aging at cell-type resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5709038484acf3d57f1f45cf8c79fcec1017a895" target='_blank'>
              Profiling Epigenetic Aging at Cell-Type Resolution Through Long-Read Sequencing
              </a>
            </td>
          <td>
            A. Eames, M. Moqri, J. Poganik, V. N. Gladyshev
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Astrocytes are the most abundant glial cell in the brain and have been shown to adopt reactive phenotypes upon interaction with glioma cells. While tumor-associated astrocyte (TAA) reactivity is linked to increased inflammation, proliferation, invasion, and treatment resistance of glioblastoma (GBM), the mechanisms by which cancer cells reprogram TAA are poorly defined. We have previously shown that mitochondrial transfer from astrocytes to GBM cells via GAP43-dependent microtubes alters GBM cellular metabolism and increases tumorigenicity. However, the potential role of this communication mechanism in TAA function remains unknown. Patient-derived GBM cells (PDCs) were transduced with a mito-GFP lentiviral vector and were co-cultured with hTERT-immortalized, mito-mCherry human astrocytes. Using flow cytometry, we observed that the frequency of mitochondrial transfer to hTERT astrocytes was dependent on GBM cell type and noted transfer rates as high as ~40% in co-cultures with L1 PDCs. Utilizing high-resolution confocal microscopy, we subsequently confirmed co-localization of fluorescent protein signal with MitoTracker Deep Red and observed increases in mitochondrial biomass in transfer-positive astrocytes. To evaluate whether mitochondrial transfer occurs in the tumor microenvironment, we implanted immunocompromised mice with L1 mito-mCherry PDCs. Approximately 17.7% of mouse TAAs in the tumor core were positive for GBM mitochondria, whereas long-distance transfer was not observed in the contralateral hemisphere. Functionally, acquisition of tumor mitochondria promoted astrocyte proliferation as measured by cells in G2/M phase and significantly increased total cellular reactive oxygen species (ROS) levels. These findings highlight a process by which astrocytes may become reactive in response to increased ROS from GBM mitochondrial transfer. Future studies will investigate the cellular response to increased ROS in astrocytes and investigate potential links to inflammatory and immunosuppressive signaling. Elucidating how GBM-to-astrocyte mitochondrial transfer reprograms TAA would help identify therapeutic vulnerabilities that can be exploited by blocking mitochondrial transfer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cbc62bc1e5a0a05dd5859b8fa1fd47b66f39774" target='_blank'>
              TMIC-35. MITOCHONDRIAL TRANSFER PROMOTES ASTROCYTE REACTIVITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Jonathan Mitchell, Defne Bayik, Dionysios C. Watson
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Introduction. During metastasis, tumor cells interact extensively with endogenous cells in distant organs. These interactions change the behavior of these surrounding cell populations, often inducing a more pro-tumor microenvironment. The lung is one of the most common sites of breast cancer (BC) metastasis, and in the lung, alveolar epithelial cells are the most common cell type. My data suggest that the lung alveolar epithelium contributes to metastatic outgrowth, and I hypothesize that BC lung micrometastases activate surrounding lung epithelial cells which, in turn, support the outgrowth of BC metastases within the lung. The overall purpose of these studies is to identify factors secreted by resident lung epithelial cells that could be used as metastasis-specific therapeutic targets and/or agents. Methods. To test this hypothesis, I utilized immunocompetent preclinical BC metastasis models to study lung metastatic outgrowth, the stage at which most patients are diagnosed with metastatic disease. A custom imaging panel was developed to quantify lung wound repair, and single-cell RNA-sequencing (scRNAseq) was performed on mouse lungs with high or low metastatic burden to identify genes in the lung epithelium that produce potentially targetable pro-metastatic secreted factors. No-contact co-culture experiments were used to directly study reciprocal paracrine interactions between lung type II alveolar epithelial (AT2) and BC cells. Results. A wound repair-related phenotype, characterized by chronic inflammation, developed within the lung microenvironment during metastatic outgrowth, including an increase in the number and activation of AT2 cells surrounding metastases as they grow. Single-cell RNA-sequencing of mouse lungs with a high vs. low metastatic burden showed that metastatic outgrowth significantly changed AT2 gene expression resulting in a modified secretome. No-contact co-culture experiments indicated that TNBC cells directly alter AT2 gene expression, while AT2 secreted factors promoted TNBC growth. I investigated the possible mechanism(s) responsible for these effects and discovered that AT2 pulmonary surfactant protein and lipid levels are altered by BC-derived secreted factors. Interestingly, treatment with the naturally derived calf surfactant Infasurf, which contains native surfactant proteins/lipids and is FDA-approved for use in premature infants, inhibited BC proliferation. Conclusion. Low levels of surfactant are commonly associated with pulmonary disease and lung cancer. My BC lung metastasis data suggest that something similar may be occurring in the lungs during metastatic outgrowth. Overall, my studies demonstrate the potential for targeting lung epithelial cells in the metastatic microenvironment, in addition to directly targeting malignant cells, as an effective way to treat and manage BC lung metastases.
 Citation Format: Jessica L. Christenson, Nicole S. Spoelstra, Jennifer K. Richer. Reciprocal activation of breast cancer metastases and the lung epithelium during metastatic outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b54efc3b640295c6631376423e51f0df8729df6" target='_blank'>
              Abstract A001: Reciprocal activation of breast cancer metastases and the lung epithelium during metastatic outgrowth
              </a>
            </td>
          <td>
            Jessica L. Christenson, Nicole S Spoelstra, Jennifer K Richer
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Within the tumor microenvironment (TME), cancer-associated fibroblasts (CAFs) were shown to be an active and pivotal cell population, supporting many protumorigenic mechanisms. Podoplanin (PDPN)-positive CAFs are of special interest since their abundance correlated with a worse prognosis for patients of different cancer entities, including malignant melanoma. In this study, we applied a loss-of-function approach in an in vivo mouse melanoma model to evaluate the contribution of CAF-specific PDPN expression to melanoma formation and progression. Surprisingly, despite its prominent expression in CAFs deletion of PDPN in this cell type did neither affect the onset, nor growth of MM tumors. These data imply that PDPN expression in CAFs represents a biomarker for poor prognosis but does not serve as a useful target for stroma-directed therapy of malignant melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3666add015c6558ff2185b82cf13e728aaa3557e" target='_blank'>
              CAF Specific Expression of Podoplanin May Be Dispensable for the Malignancy of Malignant Melanoma.
              </a>
            </td>
          <td>
            Saskia Tauch, Bettina Kast, S. Lohr, Lowis Kemm, Melanie Sator‐Schmitt, N. Gengenbacher, H. Augustin, Peter Angel
          </td>
          <td>2024-11-08</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Abstract Background Pituitary neuroendocrine tumours (PitNETs) are common intracranial tumours that are highly heterogeneous with unpredictable growth patterns. The driver genes and mechanisms that are crucial for tumour progression in somatotroph PitNETs are poorly understood. Methods In this study, we performed integrative spatial transcriptomics (ST) and single‐cell RNA sequencing (scRNA‐seq) analysis on somatotroph tumours and normal pituitary samples to comprehensively characterize the differences in cellular characteristics. Results By analyzing combined copy number variations (CNVs), tumour tissues were divided into two regions, which included the CNVhigh and CNVlow areas. The protumour genes DLK1 and RCN1 were highly expressed in the CNVhigh area, which might be related to tumour progression and could be targeted for precision therapy. We also found that the transforming growth factor beta signalling pathway participated in tumour progression and identified heterogeneity in the expression profiles of key genes. We assessed the intertumoral and intratumoral heterogeneity in somatotroph PitNETs and emphasized the importance of individualized treatment. Conclusion In summary, we visualized the cellular distribution and transcriptional differences in normal pituitary and somatotroph PitNETs by ST and scRNA‐seq for the first time. This study provides a strong theoretical foundation to comprehensively understand the crucial mechanisms involved in tumour progression and develop new strategies to treat somatotroph PitNETs. Key points The first‐ever visualization of cellular distributions in normal and tumor pituitary tissues. The inter‐ and intra‐tumoral transcriptomic heterogeneity of somatotroph PitNETs was comprehensively revealed. Identification of potential protumor factors and critical signaling pathways, opening new avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e48794431ba31899bb2431ecb1aff5c2c812ca3" target='_blank'>
              Comprehensive mapping of somatotroph pituitary neuroendocrine tumour heterogeneity using spatial and single‐cell transcriptomics
              </a>
            </td>
          <td>
            Jialin Wang, Xuejing Li, Jing Guo, Zan Yuan, Xinyu Tong, Zehao Xiao, Meng Liu, Changxiaofeng Liu, Hongyun Wang, L. Gong, Chuzhong Li, Yazhuo Zhang, Weiyan Xie, Chunhui Liu
          </td>
          <td>2024-11-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Introduction:
 In several subtypes of acute lymphoblastic leukemia (ALL), lymphoblasts often switch to a myeloid (mostly monocytic) lineage during the early phase of treatment. This unusual but well-documented phenomenon can cause diagnostic problems, particularly in the assessment of response to treatment. The cause of the lineage switch and its dynamics are not yet fully understood. Modern multiomic methods provide a unique opportunity to elucidate the progression of B-to-myeloid transdifferentiation and to clarify how different subtypes of ALL with a propensity for lineage instability differ. Previous studies have focused only on blasts at diagnosis (Dx); in our study, we also incorporated blasts from peripheral blood (PB) during or after the initial corticosteroid prephase of treatment.
 Methods:
 We used 10x Genomics' Single Cell 5‘ v2 platform to simultaneously evaluate gene expression and cell surface protein expression at the single-cell level from 3 pediatric patients with different subtypes of ALL prone to phenotypic instability (DUX4 rearrangement, ZNF384 rearrangement, and PAX5 P80R mutation). Blasts from bone marrow (BM) or PB from Dx were analyzed together with PB from day 8 (n=1) or day 5 (n=1) of a BFM-type treatment protocol. The samples were stained with a panel of TotalSeq-C antibody-oligonucleotide conjugates and processed using the Single Cell 5‘ workflow. Sequencing was conducted on an Illumina NextSeq2000 platform. A mass cytometry panel combining myeloid and lymphoid markers was used to measure protein expression in paired samples (Dx and on-treatment sample) from patients with a B-to-myeloid switch with DUX4r (n=3), PAX5 P80R (n=1), and ZNF384r (n=1) subtypes.
 Both RNA and protein counts were derived with a pipeline based on The Single-cell Pediatric Cancer Atlas project (Hawkins, bioRxiv, doi: 10.1101/2024.04.19.590243) and further data processing, including background removal and doublet detection, was carried out in accordance with current recommendations (Heumos, Nat Rev Genet 2023). Our trajectory inference framework, tviblindi (Stuchly, eLife 2024), was used to infer dynamic processes from the static snapshots provided by single-cell measurements.
 Results:
 In all subtypes, we were able to detect an intermediate population connecting B- and switched blasts, favoring a model in which the switch occurs via transdifferentiation. In the PAX5 P80R patient, we observed an abrupt change to a myeloid phenotype on day 5 of treatment. However, we identified the intermediate population at Dx on a transcriptomic level. In the ZNF384r patient, lineage instability was already evident at Dx, with a portion of the blasts showing differentiation towards the myeloid lineage while harboring a clonal immunoglobulin heavy chain (IGH) rearrangement on a DNA level as determined from sorted myeloid blasts. The expression of this IGH rearrangement decreased in myeloid blasts together with IGHM. In the DUX4r patient, B-to-myeloid transdifferentiation was detected on day 8 of the corticoid pre-phase, accompanied by a smaller B lymphoid blast population showing increased protein expression of CD45 and an altered transcriptome distinguishing this cluster from B lymphoid blasts at Dx.
 The immunophenotype of the intermediate populations was typical of B-blasts, but a shared set of myeloid genes was upregulated in their transcriptomes (e.g., CSF2RA, CEBPD, CD86, CD68, CYBB, S100A4, TYROBP, SRGN). Trajectory inference analysis was consistent with a transdifferentiation path through these intermediate populations in all samples, with an expected lag between transcriptomic and immunophenotypic alterations. Notably, CD371 was upregulated at the onset of the switch, along with other myeloid markers.
 Summary:
 Although the dynamics of the switch varied between subtypes, we observed an intermediate cellular population of varying size in patients with DUX4r, ZNF384r, and PAX5 P80R ALL, which was immunophenotypically consistent with B-blasts while sharing a common myeloid-primed transcriptomic signature. Furthermore, pseudotemporal ordering with tviblindi placed these intermediate populations before the emergence of monocytic cells in all samples, supporting our transdifferentiation hypothesis.
 Supported by NU23-05-00353, NW24-07-0026, NU23J-03-00026, UNCE/24/MED/003, and LX22NPO5102.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b35edb5cfdbe6fbcb5b8a93b11f21d8f5581b1d" target='_blank'>
              Single-Cell Multiomics Reveals a Shared Mechanism of Lineage Switch in DUX4r, ZNF384r, and PAX5 P80R Subtypes of Pediatric Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Vojen Sadílek, Barbora Cabalkova, Aneta Skotnicova, Michaela Reiterova, M. Vaskova, M. Žaliová, O. Hrusak, J. Trka, Tomas Kalina, E. Froňková, E. Mejstrikova, J. Stuchlý
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Phenotypic plasticity is a hallmark of cancer and increasingly realized as a mechanism of resistance to androgen receptor (AR)-targeted therapy. Now that many prostate cancer (PCa) patients are treated upfront with AR-targeted agents, it's critical to identify actionable mechanisms that drive phenotypic plasticity, to prevent the emergence of resistance. We showed that loss of tristetraprolin (TTP, gene ZFP36) increased NF-κB activation, and was associated with higher rates of aggressive disease and early recurrence in primary PCa. We also examined the clinical and biological impact of ZFP36 loss with co-loss of PTEN, a known driver of PCa. Analysis of multiple independent primary PCa cohorts demonstrated that PTEN and ZFP36 co-loss was associated with increased recurrence risk. Engineering prostate-specific Zfp36 deletion in vivo, induced prostatic intraepithelial neoplasia, and, with Pten co-deletion, resulted in rapid progression to castration-resistant adenocarcinoma. Zfp36 loss altered the cell state driven by Pten loss, demonstrated by enrichment of EMT, inflammation, TNFα/NF-κB, IL6-JAK/STAT3 gene sets. Additionally, our work revealed that ZFP36 loss also induced enrichment of multiple gene sets involved in mononuclear cell migration, chemotaxis, and proliferation. Use of the NF-κB inhibitor, dimethylaminoparthenolide (DMAPT) induced marked therapeutic responses in tumors with PTEN and ZFP36 co-loss and reversed castration resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a8debd5a52234a8e6e2eecf0d0e1cd5849b25a" target='_blank'>
              Low tristetraprolin expression activates phenotypic plasticity and primes transition to lethal prostate cancer in mice.
              </a>
            </td>
          <td>
            Katherine L. Morel, Beatriz German, Anis A Hamid, Jagpreet S Nanda, S. Linder, A. Bergman, Henk van der Poel, Ingrid Hofland, E. Bekers, S. Trostel, Deborah L. Burkhart, S. Wilkinson, Anson T. Ku, Minhyung Kim, Jina Kim, Duanduan Ma, Jasmine T. Plummer, Sungyong You, Xiaofeng A. Su, W. Zwart, A. Sowalsky, Christopher J. Sweeney, Leigh Ellis
          </td>
          <td>2024-11-19</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Chronic antigenic stimulation is central to marginal zone lymphoma (MZL) development. While the pharmacological inhibition of the B-cell receptor (BCR) signaling is initially effective, secondary resistance frequently develops. We conducted a CRISPR interference (CRISPRi) screen investigating enhancer-associated long non-coding RNAs (elncRNAs) in MZL cells to identify transcripts crucial to shape their dependence on BCR pathway activation. We identified LOC730338, an elncRNA associated with A-to-I RNA editing, which we renamed ADARreg. Silencing ADARreg re-sensitized tumor cells to BCR pathway inhibition by modulating ADAR2 nuclear translocation and altering intronic RNA editing. The process preferentially affected genes that enhance immune responses, such as STING, IRF3, and p65. ADARreg knockdown also increased lymphoma cell sensitivity to NK cell-mediated cytotoxicity. Our results indicate that targeting elncRNAs like ADARreg represents a potential strategy to overcome drug resistance in lymphoma, also opening new therapeutic opportunities for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b0f5b32756fe2cd1cc1438cd5631c98fcc3016" target='_blank'>
              Enhancer lncRNA LOC730338 limits the innate immune response against lymphoma cells, inhibiting ADAR2-dependent alternative transcription
              </a>
            </td>
          <td>
            Luciano Cascione, F. Guidetti, S. Ramnarayanan, Andrea Rinaldi, F. Spriano, Alex Zadro, C. Tarantelli, Nicolas Munz, A. Arribas, Rory Johnson, Francesco Bertoni, S. Napoli
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="

 Chronic graft-vs-host disease (cGVHD) is a common and often morbid immune related complication of allogeneic hematopoietic cell transplantation (HCT), and a major obstacle to the success of this cellular therapy. Its presentation is heterogeneous and the pathophysiology of cGVHD is incompletely understood. Sclerodermatous GVHD (Scl GVHD) is a severe form of cGVHD and is associated with poor prognosis and low sensitivity to immune suppression which is the current first line therapy. Here with the spatial transcriptomics analysis using the skin biopsy tissues demonstrated the dysregulation of LGR5+ fibroblast cells in cGVHD patient skins. LGR5 gene is known for functioning in excessive ECM degradation and hence reduced in cGVHD patients. We demonstrated the upregulation of ECM components deposition in the patient skins. In addition, we expanded our cohort and profiled 20 patient and 6 control skin tissue biopsies using next-generation spatial molecular profiling to gain deep insights into the immune landscape and pathogenesis of Scl cGVHD. In summary, we have demonstrated a distinct LGR5-expressing fibroblast subset as the central axis of perturbation in SSc patients. Targeted therapies directed against this pathway might begin a new era for SSc therapies.
 Introduction: Chronic graft-vs-host disease (cGVHD) is a major obstacle to the success of allogeneic hematopoietic stem cell transplantation (HCT) in patients. This debilitating condition is characterized by chronic inflammation, cell-mediated and humoral immunity, and ultimately tissue fibrosis. There is currently little or no understanding of the molecular pathogenesis of chronic cGVHD resulting in poor effective treatment strategies. Sclerodermatous GVHD (sclGVHD) is one of the more severe forms of cGVHD associated with poor prognosis and low sensitivity to immune suppressive therapy.
 Methods: We used the Nanostring CosMx platform to analyze biopsies of skin lesions from twenty patients and six controls to obtain in situ, single cell transcriptomic and protein expression profiles. Samples were prepared as follows: First, a slide (~5 µm thickness) of FFPE lesional tissue was permeabilized for target retrieval and hybridized with RNA-specific probes and antibodies to bind to the targets. The slide was then processed within the flow cell of the CoxMX SMI which images through cycles of in situ hybridization, fluorescence detection, UV photocleavage, and finally washes to remove probes. Immediately following transcriptomics, the protein panel will be run to detect fluorescence from protein targets using tagged antibodies.
 The 1000-plex Human Universal Cell Typing and Cell State Panel was used, which contains 950 target probes, as well as negative controls and Segmentation Markers. This panel provides robust tissue mapping, cell typing, and analysis of cell states and a wide range of signaling interactions including our immunotherapy targets of interest, fibroblast markers, as well as other markers of pathogenesis pathways. Segmentation Markers were used for field of view (FOV) selection in instrument software and provide vital input for allocating transcripts to individual cells. For instance, DAPI and CD298/B2M are used to segmentalize the nuclear and cell membrane compartments, respectively.
 Data Analysis: Data were analyzed using the R-programming and CosMx software analysis interface (AtoMx), which includes QC normalization, graphical visualization, cell typing (with marker identification) and spatial clustering. Data was compared to publicly available reference libraries of gene expression.
 Results: To investigate cGVHD immunopathogenesis, in the context of tissue architecture we integrated spatial RNA sequencing on a tissue microarray assembled with total 20 skin biopsies of GVHD patients and 6 skin biopsies from normal controls. We demonstrated a tight correlation of dysregulation of LGR5+ fibroblast in Sclerodermatous cGVHD patient compared to control skin biopsies. We also demonstrated the ECM components (Col1A1, COL3A1, COL6A2) in sclerodermatous cGVHD skin biopsies. We also observed the macrophages switching from M1(Tissue resident macrophages, TRMs) to M2 (Tumor associated macrophages, TAMs) in diseased skin biopsies.
 Conclusion: We spatially profiled 26 skin biopsies with 20 sclerodermatous cGVHD patients and identified a novel biomarker that can be targeted for the treatment of disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c462b27d7f0802c4a2660d2a84060d5156ccc5" target='_blank'>
              Spatial Characterization of Molecular Pathways Suggests the Dysregulation of LGR5 Expressing Fibroblasts in Patients with sclerodermatous Chronic Graft-Vs-Host Disease
              </a>
            </td>
          <td>
            Marjia Afrin, Sahar Nasr, Cristabelle De Souza, Brooke Howitt, Antonia Maria Mueller, S. Arai, J. Shizuru, G. Wernig
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Hypoxia is significantly associated with cancer progression and treatment outcomes. Nevertheless, the precise molecular mechanisms underlying the hypoxia-induced immunosuppressive microenvironment in high-grade serous ovarian cancer (HGSOC) are still not fully understood. Methods By analyzing five independent transcriptomic datasets, we investigated the effect of hypoxia on prognosis and tumor microenvironment (TME) in HGSOC. The hypoxia levels and the intercellular communication signaling pathways were studied by using single-cell analysis. Furthermore, the Hypoxia-TME classifier was developed and then validated in the multiple HGSOC datasets. In addition, we also investigated the prognostic significance, genetic variations, signaling pathways, and the potential for immunotherapy benefits in different Hypoxia-TME subgroups. Results Hypoxia was identified as a crucial risk factor in HGSOC, and strongly correlated with an immunosuppressive microenvironment characterized by alterations in the composition and distribution of immune cells. Single-cell analysis elucidated the heterogeneity inherent within the TME in HGSOC, and demonstrated an association between the hypoxic TME and fibroblasts as well as macrophages. CellChat analysis identified SPP1-CD44 and CXCL12-CXCR4 as the principal signaling axes through which macrophages and fibroblasts interact with T cells, respectively. Moreover, a personalized Hypoxia-TME classifier was constructed and validated through the integration of the hypoxia (18 genes) and TME (7 immune cells) scores. It was observed that patients in the Hypoxialow/TMEhigh subgroup displayed a significantly better prognosis than other subgroups. Different subgroups exhibited unique genomic alterations and variations in signaling pathway differences, including TGF-β and Wnt/β-catenin pathways, which are closely associated with various biological functions. Finally, our results indicated that patients in the Hypoxialow/TMEhigh subgroup exhibit a better response to immunotherapy, suggesting the potential utility of the Hypoxia-TME classifier as a new biomarker in HGSOC. Conclusion Our study revealed hypoxia-induced immunosuppressive microenvironment, and developed Hypoxia-TME classifier to distinguish the prognosis, immune characteristics, and potential benefits of immunotherapy in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20687a43e7305e0122a3a8346c6bbe4322680deb" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptome reveals hypoxia-induced immunosuppressive microenvironment to predict immunotherapy response in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Qingshan Chen, Yue Zhang, Chao Wang, Hui Ding, Liqun Chi
          </td>
          <td>2024-11-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Drug resistance is always a challenge in cancer treatment, whether for chemotherapy, targeting, or immunotherapy. Although tumor cell lines are derived from cancer patients, they gradually lost the original characteristics, including heterogeneity and tumor microenvironment (TME), during the long period of in vitro culturing. Therefore, it is urgent to use patient-derived tumor models instead of cancer cell lines to study tumor drug resistance. Herein, we developed a microarray device that serves as a platform for high-throughput and three-dimensional culture of breast cancer patient-derived organoids (BCOs) and investigated their resistance to adriamycin (ADM). Coupled with fluorescence microscopy, this system enabled on-chip drug response monitoring and cell viability assessment without the consumption of a large number of tumor cells. The organoids were divided into a resistant BCO group (RBCO) and a sensitive BCO group (SBCO) according to their half-inhibitory concentration (IC50). Different from cancer cell lines, BCOs demonstrated obvious heterogeneity in drug treatment. Ivermectin (IVM), a broad-spectrum antiparasitic agent approved by the Food and Drug Administration (FDA), was observed to synergistically augment ADM-induced cytotoxicity in organoids. The BCO chip provides a promising platform for investigation of drug resistance and preclinical drug screening based on clinical samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3dff74841f7fcae2716181c34b93fe5db0a2817" target='_blank'>
              Patient-Derived Organoids on a Microarray for Drug Resistance Study in Breast Cancer.
              </a>
            </td>
          <td>
            Tianyuan Fang, Xinlun Xie, Wei Lu, Zi-Yin Hong, Wenbo Peng, Jun Zhou, Min Wang, Bo Yao
          </td>
          <td>2024-11-05</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Our understanding of how individual mutations, whether present in all or just a fraction of the leukemia cells, affect cellular responses to therapy is limited. Leukemia mouse models provide unique possibilities to explore how therapy affects the evolution of genetically distinct clones and to identify mechanisms of resistance allowing transfer to human disease. Herein we investigated how different therapies influenced survival, clonal evolution, and resistance patterns in mouse KMT2A::MLLT3 leukemia with subclonal KrasG12D.
 Bone marrow (BM) from a leukemia expressing KMT2A::MLLT3-mCherry in all cells and a de novo KrasG12D in 38% of cells, were re-transplanted to sublethally irradiated recipients1. Upon engraftment, treatment was started with either chemotherapy mimicking the induction regimen given for AML patients (cytarabine for 5 days + doxorubicin for 3 days, n=4), or the MEK inhibitor Trametinib either alone for 28 days (n=7), or after chemotherapy (n=6); controls received vehicle.
 Our results show that each treatment significantly prolonged survival with a median latency of 25 days for chemotherapy, 27 days for Trametinib alone and 33.5 days for chemotherapy followed by Trametinib, versus 20 days for the controls. At the time of leukemia development, most of the cells in the BM expressed KMT2A::MLLT3-mCherry, and mice displayed splenomegaly and leukocytosis.
 Next, we investigated how treatment impacted the evolution of the KrasG12D-mutated KMT2A::MLLT3 cells, through targeted sequencing. In control mice receiving vehicle, the KrasG12D-containing leukemia cells had a competitive advantage over cells expressing KMT2A::MLLT3-alone and increased to clonal dominance in all mice. However, upon chemotherapy treatment the frequency of KMT2A::MLLT3+KrasG12D cells stable at around 40%, indicating that chemotherapy suppressed their expansion, but could not eradicate these cells. However, tertiary transplantation followed by repeated chemotherapy treatment caused the KrasG12D-containing leukemia cells to expand to clonal dominance indicating a competitive advantage. Across the other treatment combinations, most mice relapsed with the KrasG12D-mutated cells expanding to clonal dominance, except for mice receiving chemotherapy followed by Trametinib where two mice relapsed with either reduced or maintained frequency of Kras-mutated cells.
 In 37% of the mice, the variant allele frequency (VAF) of the KrasG12D-mutation was above 0.5, in line with allelic imbalance at the Kras locus, with mice of those mice receiving targeted treatment suggesting target-dependent mechanisms of resistance and a selective pressure for increased Ras-levels during MEK-inhibition. In line with this, we have previously shown that upon secondary transplantation of BM cells from the KrasG12D-donor mouse1, there is strong selective pressure for additional KrasG12D with 40% of the cells, acquiring an additional chromosome 6, where Kras is located, or a Robertsonian translocation involving two chromosomes 6. We will now investigate if the same mechanism occur in these mice and link that to the VAF and the individual treatments.
 Next, to evaluate how the specific treatments affected gene expression patterns and to determine how resistance occur, RNA sequencing (n=19) revealed 3 major clusters, containing either 1) the untreated mice, 2) mice that received Trametinib either alone or with chemotherapy and 3) mice that received chemotherapy. However, resistance upon treatment with chemotherapy was characterized by an up-regulation of fatty acid metabolism and ROS signaling pathways whereas resistance upon treatment with Trametinib was characterized by up-regulation of signaling pathways such as JAK/STAT, TP53, and TNFα suggesting different mechanisms of resistance and ways to target the resistant cells.
 Taken together, the different treatments given impacted latency and how the KrasG12D-mutated cells expanded and targeted sequencing highlighted that allelic imbalance at the Kras-locus occur upon resistance with Tramatenib. Finally, the transcriptomic analysis proposed deregulation of distinct pathways depending on the specific treatment providing means for targeting of the resistant cells.
 Reference:
 1. Hyrenius-Wittsten, A., et al., De novo activating mutations drive clonal evolution and enhance clonal fitness in KMT2A-rearranged leukemia. Nat Commun, 2018. 9(1): p. 1770.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7608bb73bcaa1d348e8b78e5c46afe3a4f6cb59" target='_blank'>
              Unraveling How Treatment Shapes Clonal Evolution and Resistance in Murine KMT2A-Rearranged Leukemia with Subclonal KrasG12D
              </a>
            </td>
          <td>
            Ton Falqués, Mattias Pilheden, L. Moura-Castro, Qirui Zhang, Louise Ahlgren, Helena Sturesson, Kajsa Paulsson, Axel Hyrenius Wittsten, Anna K. Hagstroem-Andersson
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Despite recent advances in the biology of IDH-wildtype glioblastoma, it remains a devastating disease with median survival of less than 2 years. However, the molecular underpinnings of the heterogeneous response to the current standard-of-care treatment regimen consisting of maximal safe resection, adjuvant radiation, and chemotherapy with temozolomide remain unknown.


METHODS
Comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients was performed to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses.


RESULTS
While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (eg, EGFR, PTEN, and NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Furthermore, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation had shorter interval to recurrence and were significantly enriched in NF1, TP53, and RB1 alterations and the mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and increased methylation at 4 specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes.


CONCLUSIONS
Glioblastoma undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8c70c5b87e0a8ebd1138c27ed60e44214d7859" target='_blank'>
              Longitudinal multimodal profiling of IDH-wildtype glioblastoma reveals the molecular evolution and cellular phenotypes underlying prognostically different treatment responses.
              </a>
            </td>
          <td>
            Calixto-Hope G Lucas, N. Al-Adli, Jacob S. Young, Rohit Gupta, R. Morshed, Jasper Wu, Ajay Ravindranathan, Anny Shai, N. Oberheim Bush, Jennie W Taylor, John de Groot, J. Villanueva‐Meyer, M. Pekmezci, Arie Perry, Andrew W Bollen, P. Theodosopoulos, M. Aghi, E. Chang, Shawn L Hervey-Jumper, D. Raleigh, Annette M Molinaro, Joseph F Costello, Aaron A. Diaz, Jennifer L. Clarke, N. Butowski, Joanna J. Phillips, Susan M Chang, Mitchell S. Berger, D. A. Solomon
          </td>
          <td>2024-11-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Intrapatient tumor heterogeneity contributes to immunotherapy and targeted therapy resistance in cutaneous metastatic melanoma patients. More than 50% of melanoma patients develop brain metastases (MBM) during disease progressionand these patients have a very poor prognosis with a median overall survival of 4 months. To assess how tumors persist through treatments and adapt to and rely on organ-specific microenvironments, we utilized samples from a patient who had received anti-CTLA4, BRAFi/MEKi, and BRAFi/MEKi+CDK4/6i treatments during the LOGIC2 clinical trial. Eight short-term melanoma cultures (STMCs) were established from different metastatic site biopsies at chronological time points throughout the patient’s treatment and subsequent post-mortem resection of the bone, ovary, and brain metastatic sites. While the initial treatment naïve metastatic tumor in the breast was BRAF V600E mutant, NRAS Q61 mutations were acquired after treatment with CDK4/6i in the bone and ovary tumors. We performed single-cell RNA sequencing and identified that each STMC consists of 2-10 different subpopulations. By cell viability assay, we determined all STMCs were resistant to CDK4/6i, however had varying degrees of response to BRAFi and MEKi in vitro. Two STMCs (MM150604, MM150859) were sensitive to both BRAFi and MEKi. We identified HER3 was upregulated in the sensitive STMCs and more specifically, HER3 expression was elevated in the MBM STMC and in other MBM patient tumors in publicly available datasets. In publicly available data of the MBM tumor microenvironment (TME), neurons produce NRG1 and NRG2 while astrocytes and pericytes produce NRG2, exogenous sources of NRGs in the MBM TME that act as ligands for the HER3 receptor. Treatment with recombinant NRG1 partially rescued the growth of MBM cells and restored MAPK and AKT-mTOR signaling in the presence of BRAFi/MEKi suggesting that the brain microenvironment may play a role in drug tolerance. This novel finding demonstrates a unique role of HER3 in MBM and targeting the NRG-HER3 signaling axis could have therapeutic potential in melanoma patients with active brain metastases.
 Citation Format: Haley P. Wilson, Glenn L. Mersky, Jelan I. Haj, Jessica L.F. Teh, Phil F. Cheng, Signe Caksa, Manal Mustafa, Isabella Trachtenberg, Francis J. Waltrich Jr, Casey D. Stefanski, Vivian Chua, Dan A. Erkes, Mitchell P. Levesque, Reinhard Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin. Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C048.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44bfe1a0d7f48e3ce31e054b1efd8507580f7dfc" target='_blank'>
              Abstract C048: Intrapatient heterogeneity and targeting microenvironment protection mechanisms in melanoma metastasis
              </a>
            </td>
          <td>
            Haley P Wilson, Glenn L Mersky, Jelan I Haj, J. Teh, Phil F Cheng, Signe Caksa, Manal U Mustafa, Isabella V Trachtenberg, Francis J. Waltrich, Casey D Stefanski, Vivian Chua, Dan A Erkes, Mitchell Levesque, R. Dummer, Timothy J. Purwin, Claudia Capparelli, Andrew E. Aplin
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ccd7ec4d58343288a03559cb704fa6e29135122" target='_blank'>
              Single-cell transcriptome unveils unique transcriptomic signatures of human organ-specific endothelial cells.
              </a>
            </td>
          <td>
            Rui-Ze Niu, Hong-Yan Xu, Hui Tian, Dan Zhang, Chun-Yu He, Xiao-Lan Li, Yu-Ye Li, Juan He
          </td>
          <td>2024-11-07</td>
          <td>Basic research in cardiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary This study identified tumor-infiltrating B cells (TIBs) and tissue-resident memory T cells (TRMs) as potential prognostic indicators in brain metastases (BMs) derived from gastrointestinal (GI) cancers. Higher densities of TIBs and TRMs in the BM tissues significantly correlated with improved overall survival after BM diagnosis. These findings suggest that quantifying TIB and TRM levels in surgically resected BM samples could provide valuable prognostic information to guide treatment decisions and follow-up strategies for patients with this lethal condition. Additionally, we revealed distinct spatial distributions and characteristics of these lymphocyte subsets, which advanced our understanding of the BM immune microenvironment. Further studies with larger cohorts are needed to validate these findings and explore their potential therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05174bd705c5070b577974c3c577fb1cea4365e3" target='_blank'>
              Tumor-Infiltrating B Cells and Tissue-Resident Memory T Cells as Prognostic Indicators in Brain Metastases Derived from Gastrointestinal Cancers
              </a>
            </td>
          <td>
            Masasuke Ohno, Shunichiro Kuramitsu, Kimihiro Yamashita, Toru Nagasaka, S. Haimoto, Mitsugu Fujita
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The inherent characteristics of glioblastoma multiforme (GBM), including tumoral heterogeneity and invasive capacity, combined with the presence of the blood-brain tumor barrier (BBTB), present challenges in developing effective treatment for GBM. Especially, the margins of GBM, where GBM cells infiltrate normal brain tissue, exhibit high resistance to therapies. Despite the difficulties in controlling tumor progression from this region, the GBM margin remains a critical area to be studied. Here we report a microengineered model that mimics the BBTB within the GBM margin, incorporating a 3D network of normal astrocytes and GBM cells isolated from patients newly diagnosed with GBM. The interaction between GBM cells and stromal cells results in increased vascular permeability, reactive gliosis, and alterations in astrocyte behavior to foster tumor invasiveness and progression. We compare patient-specific tumor responses to conventional chemotherapy in our BBTB on a chip model with clinical outcomes, demonstrating the capability of the model to predict personalized drug responses. Our BBTB model may serve as a personalized tool to examine the interactions between tumors and normal brain tissue, ultimately facilitating the screening of personalized medicine for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096e09643bdd46f475549696da124c6efcbed83f" target='_blank'>
              TMIC-47. HUMAN BLOOD-BRAIN TUMOR BARRIER ON A CHIP TO INVESTIGATE PERSONALIZED TREATMENT FOR GLIOBLASTOMA PATIENTS
              </a>
            </td>
          <td>
            Jaejoon Lim, Sujin Cho, Gaeun Lee, Minsu Ryoo, Yu Jin Kim, J. Sim, Dongkil Kim, Youn‐Jung Kang, Song Ih Ahn, Jungho Ahn
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="A fundamental question in biology, central to our understanding of cancer and other pathologies, is determining how different cell types coordinate to form and maintain tissues. Recognizing the distinct features and capabilities of the cells that compose these tissues is critical. Unfortunately, the complexity of tissues often hinders our ability to distinguish between neighboring cell types and, in turn, scrutinize their transcriptomes and generate reliable and tractable cell models for studying their inherently different biologies. We have recently introduced a novel method that permits the identification and purification of the 12 cell types that compose the human breast—nearly all of which could be reliably propagated in the laboratory. Here, we explore the nature of these cell types. We sequence mRNAs from each purified population and investigate transcriptional patterns that reveal their distinguishing features. We describe the differentially expressed genes and enriched biological pathways that capture the essence of each cell type, and we highlight transcripts that display intriguing expression patterns. These data, analytic tools, and transcriptional analyses form a rich resource whose exploration provides remarkable insights into the inner workings of the cell types composing the breast, thus furthering our understanding of the rules governing normal cell and tissue function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96577d34ee726674a61ddf8250fdd165d06c753" target='_blank'>
              Transcriptomic analysis of the 12 major human breast cell types reveals mechanisms of cell and tissue function
              </a>
            </td>
          <td>
            Katelyn Del Toro, Rosalyn Sayaman, Kate Thi, Y. Licon-Munoz, William C. Hines
          </td>
          <td>2024-11-01</td>
          <td>PLOS Biology</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most common and deadly primary brain tumor, exerts profound local and systemic immunosuppressive effects through a variety of mechanisms. Tumor-derived extracellular vesicles (EVs) are increasingly appreciated as critical mediators of these effects. In surveying the tumor-immune microenvironment in samples of ultrasonic aspirate collected during surgical resection of GBM (n=13), we identified several key immunomodulatory markers expressed across a range of myeloid cell populations. Particularly, we found TIGIT, classically understood as a T cell inhibitory protein, expressed across a range of myeloid cell populations in patient samples, including in myeloid-derived suppressor cells (MDSCs, 46.84±16.78% of total monocytic MDSCs). Using EVs collected from patient-derived cell lines, we were able to recapitulate these effects upon co-culture with monocytes derived from the peripheral blood of healthy donors (n=7). Furthermore, Induced MDSCs significantly inhibit T cells proliferation (70.97±8.454%, compared to control 19.4±5.71; p<0.03), concomitant with a surge in Th2 immune responses via TIGIT/CD155 axis. This effect was reversed after TIGIT silencing, indicating a TIGIT-dependent immune suppression in the GBM microenvironment. Our study indicates EV-induced TIGIT upregulation in myeloid cells as a crucial immunosuppressive mechanism in glioblastoma, with potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28d7c950cbc4e5f2f351073303f4e60365488489" target='_blank'>
              IMMU-56. THE TIGIT/CD155 AXIS PLAYS A CRITICAL ROLE IN EXTRACELLULAR VESICLE-MEDIATED PROGRAMMING OF THE GLIOBLASTOMA MICROENVIRONMENT
              </a>
            </td>
          <td>
            Mohamad Asad, Julio Inocencio, Stefan Mitrasinovic, Minori Aoiki, I. Parney, Celina Crisman, Patrick Lasala, Emad Eskandar, B. Himes
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The leptomeninges play a pivotal role in the central nervous system (CNS), serving both as a barrier and as a conduit for fluid and cellular transport. Despite their critical functions, our understanding of leptomeningeal development and maturation during human embryogenesis remains limited. This study seeks to bridge this gap. We conducted single-nucleus RNA sequencing on leptomeningeal tissues from eight human embryos, capturing developmental stages from early fetal to late mid-fetal phases. Our bioinformatic analyses encompassed cell type classification, identification of layer-specific markers, mapping of the arachnoid barrier maturation trajectory, and joint analyses with mouse and aged human leptomeningeal scRNA datasets. Key bioinformatic findings were validated through immunostaining in selected samples. Our study revealed a complex cellular heterogeneity within the developing leptomeninges, identifying distinct subpopulations of fibroblasts, immune cells, and vascular cells. We mapped the transcriptomic dynamics of fibroblast cell types throughout fetal brain development, highlighting a clear maturation process from early fetal to late mid-fetal stages. Comparative analysis with mouse data allowed us to distinguish human-specific layer markers while confirming several conserved markers shared between humans and mice. Joint analysis with aged human datasets identified two unique arachnoid clusters specific to aging leptomeninges. Moreover, we traced the developmental trajectory of the arachnoid barrier, detailing the transcriptomic shifts associated with its gradual formation. Notably, immune cells in early fetal stages were predominantly M2-type macrophages, underscoring a distinctive immune environment. Finally, we explored the molecular interactions between fibroblasts and other cell types, highlighting their coordinated roles in orchestrating leptomeningeal development. Together, our findings provide a comprehensive overview of the cellular and molecular landscape of the developing human leptomeninges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c35538a8c7a55a082c80621304b3bf8d3408c46d" target='_blank'>
              Deciphering the temporal transcriptional landscape of human fetal leptomeninges.
              </a>
            </td>
          <td>
            Licheng Sun, Ping Liu, Jingjing Guo, Chuantao Fang, Li Li, Yi Liu, Yanfeng Tan, Wei Zhang, Rui Zhao, Fayong Zhang, Jianbo Xiao, Rui Dong, Shaojie Ma, Xinyu Mei, Dashi Qi
          </td>
          <td>2024-12-07</td>
          <td>Brain : a journal of neurology</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>